<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <div xmlns="" tagx="article">
      <head>
         <style type="text/css"> table, tr, td {border : thin;}</style>
         <span class="citation_publisher">Oxford University Press</span>
         <h2 class="citation-title">DNA barcodes from four loci provide poor resolution of taxonomic groups in the genus
            Crataegus
         </h2>
         <span class="doi">10.1093/aobpla/plv045</span>
         <div class="contrib-group">contrib: 
            	  CONT
            	  
            <span class="citation_author">Mehdi Zarrei</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Nadia Talent</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Maria Kuzmina</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Jeanette Lee</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Jensen Lund</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Paul R. Shipley</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Saša Stefanović</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Timothy A. Dickinson</span>
            <span class="citation_author_institution_ref"></span>
            	AFF
            	  
            <span class="citation_author_institution">[1 The Centre for Applied Genomics, The Hospital for Sick Children]</span>
            	AFF
            	  
            <span class="citation_author_institution">[2 Green Plant Herbarium, Department of Natural History]</span>
            	AFF
            	  
            <span class="citation_author_institution">[3 50 Stone Road East, Guelph, ON]</span>
            	AFF
            	  
            <span class="citation_author_institution">[4 109 Lakeshore Ave., Edgewood, BC]</span>
            	AFF
            	  
            <span class="citation_author_institution">[5 Department of Chemistry, I.K. Barber School of Arts and Sciences]</span>
            	AFF
            	  
            <span class="citation_author_institution">[6 Department of Ecology and Evolutionary Biology]</span>
            	AFF
            	  
            <span class="citation_author_institution">[7 Department of Biology]</span>
         </div>
         <span class="email">[tim.dickinson@utoronto.ca]</span>
         <span class="conflict">[]</span>
         <span class="contribs">[Associate Editor: Kermit Ritland]</span>
         <span class="citation_pubdate">[2015-4-29]</span>
         <span class="citation_copyright">[Copyright 2015: ]</span>
         <span class="citation_licence">[Licence This is an Open Access article distributed under the terms of the Creative
            Commons Attribution License (http://creativecommons.org/licenses/by/4.0/), which permits
            unrestricted reuse, distribution, and reproduction in any medium, provided the original
            work is properly cited.]
         </span>
         <span class="citation_funding">[Canada Foundation for Innovation http://dx.doi.org/10.13039/501100000196Ontario Research
            FundOntario Genomics Institute http://dx.doi.org/10.13039/5011000000922008-OGI-ICI-03Natural
            Sciences and Engineering Research Council of Canada Strategic Research Project grantUniversity
            of Western Ontario'sROM Reproductions, Acquisitions, and Research FundDepartment of
            Natural HistoryUniversity of Toronto http://dx.doi.org/10.13039/501100003579]
         </span>
         <span class="citation_pagecount">[]</span>
         <span class="citation_data">[]</span>
      </head>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The leaves and fruits of some of the approximately 230 species of hawthorn (
            <i>Crataegus</i>) yield natural health products with significant therapeutic effects on symptoms of
            cardiovascular disease. DNA barcoding could be a valuable tool for authenticating
            these products, but only a small fraction of the 93 taxa that we examined were distinguished,
            even though all major clades and eight out of ten taxonomic sections of the genus
            were included. DNA barcoding as currently practised thus has limited utility in 
            <i>Crataegus</i>. Hybridization, lineage sorting due to incomplete concerted evolution in ITS2, and
            limited variation in plastid loci are implicated.
         </p>
      </div>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">DNA barcodes can facilitate identification of organisms especially when morphological
            characters are limited or unobservable. To what extent this potential is realized
            in specific groups of plants remains to be determined. Libraries of barcode sequences
            from well-studied authoritatively identified plants represented by herbarium voucher
            specimens are needed in order for DNA barcodes to serve their intended purpose, where
            this is possible, and to understand the reasons behind their failure to do so, when
            this occurs. We evaluated four loci, widely regarded as universal DNA barcodes for
            plants, for their utility in hawthorn species identification. Three plastid regions,
            
            <i>mat</i>K, 
            <i>rbcL</i>a and 
            <i>psb</i>A
            <i>-trn</i>H, and the internal transcribed spacer 2 (ITS2) of nuclear ribosomal DNA discriminate
            only some of the species of 
            <i>Crataegus</i> that can be recognized on the basis of their morphology etc. This is, in part, because
            in Rosaceae tribe Maleae most individual plastid loci yield relatively little taxonomic
            resolution and, in part, because the effects of allopolyploidization have not been
            eliminated by concerted evolution of the ITS regions. Although individual plastid
            markers provided generally poor resolution of taxonomic groups in 
            <i>Crataegus</i>, a few species were notable exceptions. In contrast, analyses of concatenated sequences
            of the 3 plastid barcode loci plus 11 additional plastid loci gave a well-resolved
            maternal phylogeny. In the ITS2 tree, different individuals of some species formed
            groups with taxonomically unrelated species. This is a sign of lineage sorting due
            to incomplete concerted evolution in ITS2. Incongruence between the ITS2 and plastid
            trees is best explained by hybridization between different lineages within the genus.
            In aggregate, limited between-species variation in plastid loci, hybridization and
            a lack of concerted evolution in ITS2 all combine to limit the utility of standard
            barcoding markers in 
            <i>Crataegus</i>. These results have implications for authentication of hawthorn materials in natural
            health products.
         </p>
      </div>
      <div id="s1">
         <h2>Introduction</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The main aim of DNA barcoding is to identify an unknown organism by comparing a DNA
            sequence from the unknown with records in a database of identified sequences, based
            on some measure of genetic similarity (
            <span ref-type="bibr" rid="PLV045C41">
               <sup>
                  <a href="#PLV045C41">Hebert et al. 2003a</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="PLV045C11">
               <sup>
                  <a href="#PLV045C11">Chase et al. 2005</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="PLV045C81">
               <sup>
                  <a href="#PLV045C81">Ross et al. 2008</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="PLV045C86">
               <sup>
                  <a href="#PLV045C86">Seberg and Petersen 2009</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="PLV045C45">
               <sup>
                  <a href="#PLV045C45">Hollingsworth et al. 2011</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="PLV045C28">
               <sup>
                  <a href="#PLV045C28">Dong et al. 2012</a>
               </sup>
            </span>) or combination of diagnostic sequence characters (
            <span ref-type="bibr" rid="PLV045C78">
               <sup>
                  <a href="#PLV045C78">Reid et al. 2011</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="PLV045C106">
               <sup>
                  <a href="#PLV045C106">Weitschek et al. 2013</a>
               </sup>
            </span>). The Barcode of Life Data Systems (BOLD) database (
            <span ref-type="bibr" rid="PLV045C77">
               <sup>
                  <a href="#PLV045C77">Ratnasingham and Hebert 2007</a>
               </sup>
            </span>) plays an important role as a centralized and curated depository for DNA barcode
            sequences that is effective in archiving and making accessible detailed voucher information
            for each organism from which the sequence was obtained. The Consortium for the Barcode
            of Life (CBOL) Plant Barcoding Group 
            <span ref-type="bibr" rid="PLV045C10">
               <sup>
                  <a href="#PLV045C10">(2009)</a>
               </sup>
            </span> identified two plastid protein-coding loci, 
            <i>mat</i>K and 
            <i>rbcL</i>, as the barcode regions for generating a library of identified sequences for plants.
            Additional discrimination on species-level taxonomy was proved by using supplementary
            markers: plastid 
            <i>psb</i>A-
            <i>trn</i>H intergenic spacer and internal transcribed spacer 2 (ITS2) from nuclear ribosomal
            DNA (
            <span ref-type="bibr" rid="PLV045C53">
               <sup>
                  <a href="#PLV045C53">Kress et al. 2009</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="PLV045C13">
               <sup>
                  <a href="#PLV045C13">Chen et al. 2010</a>
               </sup>
            </span>). The usefulness and challenges of using different plant DNA barcodes were summarized
            by 
            <span ref-type="bibr" rid="PLV045C45">
               <sup>
                  <a href="#PLV045C45">Hollingsworth et al. (2011)</a>
               </sup>
            </span>. It remains to be seen to what extent these markers can assist identification in
            closely related plant taxa that are difficult to identify because, for example, there
            is a limited number of characters, not all of which are necessarily present at the
            phenological stage of a given specimen.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Hawthorns (
            <i>Crataegus</i>, Rosaceae tribe Maleae) are frequent and often locally abundant throughout the North
            Temperate Zone in high light intensity habitats where the moisture regime permits
            the establishment of woody vegetation. Within the Rosaceae subfamily Amygdaloideae
            the genus is relatively large (
            <span ref-type="bibr" rid="PLV045C25">
               <sup>
                  <a href="#PLV045C25">Dickinson et al. 2007</a>
               </sup>
            </span>), with 50–100 Old World species (
            <span ref-type="bibr" rid="PLV045C14">
               <sup>
                  <a href="#PLV045C14">Christensen 1992</a>
               </sup>
            </span>), and 150–200 New World ones (
            <span ref-type="bibr" rid="PLV045C71">
               <sup>
                  <a href="#PLV045C71">Phipps 2015</a>
               </sup>
            </span>). Species identification depends on a suite of characters; some of which can only
            be recorded at the time of flowering, while others must be observed in fruiting material.
            In addition, identification has been further complicated by the much larger number
            of species that have been described in 
            <i>Crataegus</i>, for which comprehensive synonymies have only recently become available (
            <span ref-type="bibr" rid="PLV045C14">
               <sup>
                  <a href="#PLV045C14">Christensen 1992</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="PLV045C71">
               <sup>
                  <a href="#PLV045C71">Phipps 2015</a>
               </sup>
            </span>).
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Hawthorn fruits are edible and a few species are cultivated for their fruits in East
            Asia (
            <i>C. pinnatifida</i>), Europe (
            <i>C. azarolus</i>, 
            <i>C. germanica</i>) and in the New World (
            <i>C. mexicana</i>, 
            <i>C. opaca</i>). Hawthorns are also used in traditional medicine in all of these places, and there
            is a sizable market for natural health products (NHPs) made from hawthorn leaves,
            flowers and fruits (
            <span ref-type="bibr" rid="PLV045C32">
               <sup>
                  <a href="#PLV045C32">Edwards et al. 2012</a>
               </sup>
            </span>). Hawthorn chemistry is fairly well known, but most species studied are Eurasian
            ones (Table 1 in 
            <span ref-type="bibr" rid="PLV045C32">
               <sup>
                  <a href="#PLV045C32">Edwards et al. 2012</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="PLV045C110">
               <sup>
                  <a href="#PLV045C110">Yang and Liu 2012</a>
               </sup>
            </span>). Studies of the effects of hawthorn preparations in treating hyperlipidaemia and
            cardiovascular conditions have been the subject of meta-analyses that suggest there
            are significant health benefits to be obtained from these preparations (
            <span ref-type="bibr" rid="PLV045C38">
               <sup>
                  <a href="#PLV045C38">Guo et al. 2008</a>
               </sup>
            </span>). To date, however, virtually all published studies of the therapeutic components
            and the therapeutic effects of hawthorn preparations have used Eurasian 
            <i>Crataegus</i> species (
            <span ref-type="bibr" rid="PLV045C110">
               <sup>
                  <a href="#PLV045C110">Yang and Liu 2012</a>
               </sup>
            </span>). Similarly, hawthorn NHPs sold in Europe and North America predominantly employ
            raw materials from only a limited number of Eurasian species.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In the southern interior of British Columbia, hawthorns are particularly abundant
            as a consequence of their weediness, and of agricultural activity and land-use changes
            (
            <span ref-type="bibr" rid="PLV045C72">
               <sup>
                  <a href="#PLV045C72">Phipps and O'Kennon 2002</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="PLV045C26">
               <sup>
                  <a href="#PLV045C26">Dickinson et al. 2008</a>
               </sup>
            </span>). The diversity of described hawthorn species is higher here than anywhere else in
            western North America, comprising widespread species like red-fruited 
            <i>C. chrysocarpa</i> (
            <i>C.</i> sect. 
            <i>Coccineae</i>) and 
            <i>C. macracantha</i> (
            <i>C.</i> sect. 
            <i>Macracanthae</i>), as well as black-fruited ones (
            <i>C. douglasii</i>, 
            <i>C. suksdorfii</i> in 
            <i>C.</i> sect. 
            <i>Douglasia</i>) and their hybrids in 
            <i>Crataegus</i> series 
            <i>Douglasianae</i> and 
            <i>Purpureofructus</i> (
            <span ref-type="bibr" rid="PLV045C113">
               <sup>
                  <a href="#PLV045C113">Zarrei et al. 2014</a>
               </sup>
            </span>) and the naturalized Eurasian red-fruited hawthorn, 
            <i>C. monogyna</i>. Native hawthorn species are common throughout the southern half of British Columbia
            (south of 56° North latitude). In addition to using their wood for hammers, tool handles
            and digging sticks (
            <span ref-type="bibr" rid="PLV045C102">
               <sup>
                  <a href="#PLV045C102">Turner 2014b</a>
               </sup>
            </span>), coastal and interior First Nations in British Columbia used the fruits of native
            hawthorns for food, often smashed so as to break up the pyrenes (and the seeds inside;
            
            <span ref-type="bibr" rid="PLV045C102">
               <sup>
                  <a href="#PLV045C102">Turner 2014b</a>
               </sup>
            </span>). First Nation names for both black- and red-fruited hawthorns are documented for
            the southern half of British Columbia and the adjacent USA, the latter species (
            <i>C. chrysocarpa</i>) mainly in the southern and interior portion of this range (
            <span ref-type="bibr" rid="PLV045C101">
               <sup>
                  <a href="#PLV045C101">Turner 2014a</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="PLV045C102">
               <sup>
                  <a href="#PLV045C102">b</a>
               </sup>
            </span>). In fact, hawthorn species across North America are named by First Nations people,
            and ethnobotanical uses of hawthorn species are documented for food and for treatments
            of various conditions, including gastrointestinal, dermatological and heart ailments
            (
            <span ref-type="bibr" rid="PLV045C3">
               <sup>
                  <a href="#PLV045C3">Arnason et al. 1981</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="PLV045C32">
               <sup>
                  <a href="#PLV045C32">Edwards et al. 2012</a>
               </sup>
            </span>).
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Together with widespread use of hawthorn NHPs that are manufactured almost exclusively
            from a small number of Eurasian species, the diversity and abundance of hawthorns
            in British Columbia has led to recognition of a potential economic opportunity for
            locally sourced NHPs. For example, the Naturally Grown Herb and Spice Producers Cooperative
            (HerbPro; 
            <div tagxxx="uri">http://herbpronaturals.ca/HerbPro/</div>) has established an agroforestry demonstration farm with hawthorn as the main orchard
            alley cropping tree. HerbPro has developed a fresh/frozen puree of 
            <i>C. chrysocarpa</i> fruit and 
            <i>C. monogyna</i> leaf and has had farm-gate sales since 2008. Anecdotal evidence has shown that the
            puree has positive health benefits for those with cardiovascular disease and hypertension.
            The raw ingredients for NHPs manufactured and marketed in Canada must be identified
            and the species listed on product labelling. Collaboration between HerbPro and the
            academic authors has focussed on evaluating the chemistry and taxonomy of western
            North American hawthorns, developing identification tools that can be used in the
            field, and investigating the extent to which DNA barcodes can be used to confirm identifications
            and authenticate raw ingredients in post-harvest processing (e.g. dried and/or powdered
            forms of fruit, leaf and flower). Our collaboration also includes limited trials of
            hawthorn preparations in an animal model of human metabolic syndrome (F. Borthwick
            
            <i>et al.</i>, unpubl. data—presented in part as 
            <span ref-type="bibr" rid="PLV045C114">
               <sup>
                  <a href="#PLV045C114">Dickinson et al. 2014</a>
               </sup>
            </span>).
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Hawthorn preparations, as employed in the animal trials described above and as used
            in NHP formulations, comprise not only biopolymers (nucleic acids, proteins) but also
            complex mixtures of much lower molecular weight secondary metabolites. Characterization
            of these secondary metabolites (the metabolome) by means of analytical instrumentation
            such as mass spectrometer or nuclear magnetic resonance (NMR) spectrometer (metabolomics;
            
            <span ref-type="bibr" rid="PLV045C69">
               <sup>
                  <a href="#PLV045C69">Oliver et al. 1998</a>
               </sup>
            </span>) provides another means by which to describe hawthorn species. Since the therapeutic
            effects of hawthorn preparations appear to be mediated by the metabolome (notably,
            phenolic compounds; 
            <span ref-type="bibr" rid="PLV045C32">
               <sup>
                  <a href="#PLV045C32">Edwards et al. 2012</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="PLV045C110">
               <sup>
                  <a href="#PLV045C110">Yang and Liu 2012</a>
               </sup>
            </span>), it is valid to ask to what extent the variation in the metabolomes of 
            <i>Crataegus</i> species parallels the variation in DNA sequences. Given that the DNA sequence variation
            reflects genealogical relationships (we infer the latter from the former), we wish
            to see whether or not metabolomics resemblances reflect phylogenetic relationships.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To the extent that the molecular data are sufficient (sequence variation in ITS and
            two chloroplast loci) in a study of 
            <i>Stachys</i> (Lamiaceae; 
            <span ref-type="bibr" rid="PLV045C83">
               <sup>
                  <a href="#PLV045C83">Salmaki et al. 2013</a>
               </sup>
            </span>), we know that phytochemical variation (
            <span ref-type="bibr" rid="PLV045C100">
               <sup>
                  <a href="#PLV045C100">Tundis et al. 2014</a>
               </sup>
            </span>) is not necessarily congruent with phylogeny. Non-metric multidimensional scaling
            of data for several classes of secondary metabolites (mono-, sesqui- and diterpenes,
            iridoids, phenylethanoid glycosides, flavonoids) contrasts species of 
            <i>Stachys</i> and those of 
            <i>Sideritis</i>, while species of the genus 
            <i>Betonica</i> are located in a subset of the chemospace occupied by the 
            <i>Stachys</i> species (
            <span ref-type="bibr" rid="PLV045C100">
               <sup>
                  <a href="#PLV045C100">Tundis et al. 2014</a>
               </sup>
            </span>). In the nuclear and plastid phylogenies, however, 
            <i>Betonica</i> is one of the outgroups for 
            <i>Stachys</i>, and the species of 
            <i>Sideritis</i> are nested among those of 
            <i>Stachys</i> (
            <span ref-type="bibr" rid="PLV045C83">
               <sup>
                  <a href="#PLV045C83">Salmaki et al. 2013</a>
               </sup>
            </span>).
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Here we examine the utility of the proposed plant DNA barcode loci, comparing the
            information they provide for hawthorns with that obtained using additional loci and
            molecular phylogenetic methods. In doing so, we seek to (i) sample the species diversity
            of hawthorns extensively in order to generate a vouchered library of DNA sequences
            deposited in BOLD (see 
            <div tagxxx="ext-link">
               <b>Supporting Information—Table S1</b>
            </div>; dx.doi.org/10.5883/DS-NAMCRAT; 
            <span ref-type="bibr" rid="PLV045C77">
               <sup>
                  <a href="#PLV045C77">Ratnasingham and Hebert 2007</a>
               </sup>
            </span>) and GenBank (
            <span ref-type="bibr" rid="PLV045C5">
               <sup>
                  <a href="#PLV045C5">Benson et al. 2011</a>
               </sup>
            </span>) from 
            <i>Crataegus</i> specimens that have already been identified by expert taxonomists; (ii) assess the
            utility of the universal plant DNA barcoding markers in species identification, particularly
            in discrimination of medicinally important taxa; (iii) compare the preceding results
            with those of phylogenetic analyses of the cpDNA barcoding loci augmented by 11 additional
            cpDNA loci chosen for the variability they exhibit in 
            <i>Crataegus</i> and other Maleae (
            <span ref-type="bibr" rid="PLV045C59">
               <sup>
                  <a href="#PLV045C59">Lo and Donoghue 2012</a>
               </sup>
            </span>) and (iv) examine whether polyploidization, apomixis and possibly incomplete concerted
            evolution of ITS limit the usefulness of DNA barcoding in plant groups where these
            factors may be frequent (
            <span ref-type="bibr" rid="PLV045C45">
               <sup>
                  <a href="#PLV045C45">Hollingsworth et al. 2011</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="PLV045C112">
               <sup>
                  <a href="#PLV045C112">Zarrei et al. 2012</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="PLV045C113">
               <sup>
                  <a href="#PLV045C113">2014</a>
               </sup>
            </span>). (v) We provide a preliminary NMR metabolomics dataset with which we examine the
            relationship between similarities in the therapeutically significant phenolic composition
            of leaf tissue with that of phylogenetic relationships seen in the cpDNA sequence
            data.
         </p>
      </div>
      <div id="s2">
         <h2>Methods</h2>
         <div id="s2a">
            <h3>Plant materials</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We sampled DNA barcode loci for 355 
               <i>Crataegus</i> specimens in the Royal Ontario Museum Green Plant Herbarium (TRT; 
               <div tagxxx="ext-link">
                  <b>Supporting Information—Table S1</b>
               </div>; BOLD doi:10.5883/DS-NAMCRAT), selected as follows. A total of 275 samples of primarily
               North American species came from hawthorn trees in Ontario, the southeastern USA,
               Colorado, Utah, Wyoming and the Pacific Northwest for which we had leaf tissue dried
               on silica gel, as well as voucher specimens (
               <span ref-type="bibr" rid="PLV045C93">
                  <sup>
                     <a href="#PLV045C93">Talent and Dickinson 2005</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C60">
                  <sup>
                     <a href="#PLV045C60">Lo et al. 2007</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="PLV045C61">
                  <sup>
                     <a href="#PLV045C61">2009</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C26">
                  <sup>
                     <a href="#PLV045C26">Dickinson et al. 2008</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C18">
                  <sup>
                     <a href="#PLV045C18">Coughlan 2012</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C112">
                  <sup>
                     <a href="#PLV045C112">Zarrei et al. 2012</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">2014</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C19">
                  <sup>
                     <a href="#PLV045C19">Coughlan et al. 2014</a>
                  </sup>
               </span>). For these individuals up to 10 accessions per species (or cytotype) were sampled
               
               <b>[see 
                  <div tagxxx="ext-link">Supporting Information—</div>
                  <div tagxxx="ext-link">Table S1</div>]
               </b>. This material also included vouchered samples collected at the arboreta and botanical
               gardens acknowledged in our earlier papers, together with samples of Eurasian species
               provided to us by K. I. Christensen, A. A. Dönmez and T. Romankova 
               <b>[see</b>
               <div tagxxx="ext-link">
                  <b>Supporting Information—Table S1</b>
               </div>
               <b>]</b>. A further 80 TRT herbarium specimens of North American species from the J.B. Phipps
               Hawthorn Research Collection 
               <b>[see</b>
               <div tagxxx="ext-link">
                  <b>Supporting Information—Table S1</b>
               </div>
               <b>]</b> were sampled as part of the Canadian Centre for DNA Barcoding project, ‘DNA Barcode
               Flora of Canada’. Of the samples, 38 
               <b>[</b>
               <div tagxxx="ext-link">
                  <b>Supporting Information—Table S1]</b>
               </div>, together with samples from 6 additional individuals, were used for more intensive
               study of cpDNA sequence variation in mainly Pacific Northwest 
               <i>Crataegus</i>
               <div tagxxx="ext-link">
                  <b>[Supporting Information—Table S2]</b>
               </div> (cf. 
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>). Finally, a further eight accessions were part of samples used to explore the utility
               of the two low-copy nuclear markers described below (AT1 and PEPC; 
               <div tagxxx="ext-link">
                  <b>Supporting Information—Tables S3 and S4</b>
               </div>). DNA for some of these individuals was obtained from seeds. These 369 
               <i>Crataegus</i> samples represent all of the clades found by 
               <span ref-type="bibr" rid="PLV045C60">
                  <sup>
                     <a href="#PLV045C60">Lo et al. (2007</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="PLV045C61">
                  <sup>
                     <a href="#PLV045C61">2009</a>
                  </sup>
               </span>), as well as all four sections native to North America and four of six sections native
               to Eurasia (i.e. not including any of the four species in sections 
               <i>Cuneatae</i> and 
               <i>Henryanae</i>). Our sample also comprises four named nothosections, as well as 35 series, two named
               nothoseries and 83 species (plus five varieties, one hybrid and four named nothospecies;
               
               <div tagxxx="ext-link">
                  <b>Supporting Information—Tables S1 and S2</b>
               </div>). Although just over one-third of our 369 accessions belong to 
               <i>C.</i> section 
               <i>Douglasia</i>, the taxonomic breadth of our sample is sufficient to let us relate the infrageneric
               classification 
               <b>[</b>
               <div tagxxx="ext-link">
                  <b>Supporting Information—Table S1]</b>
               </div> to the phylogenetic structure found by 
               <span ref-type="bibr" rid="PLV045C60">
                  <sup>
                     <a href="#PLV045C60">Lo et al. (2007)</a>
                  </sup>
               </span> in a similarly wide sampling of the genus. Most of the samples studied come from
               individuals whose ploidy level and breeding system have been documented either by
               means of chromosome counts and embryological studies (
               <span ref-type="bibr" rid="PLV045C24">
                  <sup>
                     <a href="#PLV045C24">Dickinson et al. 1996</a>
                  </sup>
               </span>), or by means of flow cytometry of leaf and seed tissues (
               <span ref-type="bibr" rid="PLV045C93">
                  <sup>
                     <a href="#PLV045C93">Talent and Dickinson 2005</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="PLV045C94">
                  <sup>
                     <a href="#PLV045C94">2007a</a>
                  </sup>
               </span>; 
               <div tagxxx="ext-link">
                  <b>Supporting Information—Tables S1 and S2</b>
               </div>).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Based on the results of 
               <span ref-type="bibr" rid="PLV045C9">
                  <sup>
                     <a href="#PLV045C9">Campbell et al. (2007)</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="PLV045C74">
                  <sup>
                     <a href="#PLV045C74">Potter et al. (2007)</a>
                  </sup>
               </span> and 
               <span ref-type="bibr" rid="PLV045C60">
                  <sup>
                     <a href="#PLV045C60">Lo et al. (2007)</a>
                  </sup>
               </span>, sequences from 
               <i>Amelanchier</i>, 
               <i>Cotoneaster</i>, 
               <i>Malus</i>, 
               <i>Pyrus</i> and 
               <i>Sorbus</i> were chosen for outgroup rooting of all but one of the trees produced in this study
               
               <b>[see</b>
               <div tagxxx="ext-link">
                  <b>Supporting Information—Tables S1–S4</b>
               </div>
               <b>]</b>. The tree built from sequences of the plastid barcode loci 
               <b>[see 
                  <div tagxxx="ext-link">Supporting Information—</div>
                  <div tagxxx="ext-link">Fig. S1</div>]
               </b> was rooted using the sequences from 
               <i>Crataegus brachyacantha</i>, following the results of 
               <span ref-type="bibr" rid="PLV045C60">
                  <sup>
                     <a href="#PLV045C60">Lo et al. (2007)</a>
                  </sup>
               </span>, and based on the similarity of the submarginal venation seen in leaves of this species
               (unique in 
               <i>Crataegus</i>; Fig. 1 in 
               <span ref-type="bibr" rid="PLV045C26">
                  <sup>
                     <a href="#PLV045C26">Dickinson et al. 2008</a>
                  </sup>
               </span>) to that of 
               <i>Hesperomeles</i> (
               <span ref-type="bibr" rid="PLV045C48">
                  <sup>
                     <a href="#PLV045C48">Kelly 2008</a>
                  </sup>
               </span>), a genus shown by 
               <span ref-type="bibr" rid="PLV045C58">
                  <sup>
                     <a href="#PLV045C58">Li et al. (2012)</a>
                  </sup>
               </span> to be sister to 
               <i>Crataegus</i>.
            </p>
         </div>
         <div id="s2b">
            <h3>Molecular methods</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Four DNA barcodes (
               <i>rbcL</i>, 
               <i>mat</i>K, 
               <i>psb</i>A
               <i>-trn</i>H and ITS2; 
               <span ref-type="bibr" rid="PLV045C12">
                  <sup>
                     <a href="#PLV045C12">Chase et al. 2007</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C10">
                  <sup>
                     <a href="#PLV045C10">CBOL Plant Working Group 2009</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C45">
                  <sup>
                     <a href="#PLV045C45">Hollingsworth et al. 2011</a>
                  </sup>
               </span>) were generated directly from genomic DNA for the 355 
               <i>Crataegus</i> accessions in the NAMCRAT dataset. DNA was extracted and amplified from leaf tissue
               using Canadian Centre for DNA Barcoding (CCDB) protocols (
               <span ref-type="bibr" rid="PLV045C47">
                  <sup>
                     <a href="#PLV045C47">Ivanova et al. 2011</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C54">
                  <sup>
                     <a href="#PLV045C54">Kuzmina and Ivanova 2011a</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="PLV045C55">
                  <sup>
                     <a href="#PLV045C55">b)</a>
                  </sup>
               </span>. With ITS2, additional primers (
               <span ref-type="bibr" rid="PLV045C108">
                  <sup>
                     <a href="#PLV045C108">White et al. 1990</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C90">
                  <sup>
                     <a href="#PLV045C90">Stanford et al. 2000</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C13">
                  <sup>
                     <a href="#PLV045C13">Chen et al. 2010</a>
                  </sup>
               </span>) were needed for successful amplification in some cases. The barcoding sample overlapped
               partially with that for which additional plastid loci were sequenced (below), and
               for which cloned ITS2 sequences were studied (
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>). The successfully amplified amplicons were then sequenced on the 3730xl DNA Analyser
               (Applied Biosystems) with both forward and reverse primers to assure accuracy.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To increase the discriminatory power of the plastid markers especially in view of
               the potential consequences of long generation times associated with woodiness in 
               <i>Crataegus</i> (
               <span ref-type="bibr" rid="PLV045C88">
                  <sup>
                     <a href="#PLV045C88">Smith and Donoghue 2008</a>
                  </sup>
               </span>), an additional 11 plastid markers (
               <span ref-type="bibr" rid="PLV045C59">
                  <sup>
                     <a href="#PLV045C59">Lo and Donoghue 2012</a>
                  </sup>
               </span>) were sequenced for a subsample of individuals 
               <b>[see</b>
               <div tagxxx="ext-link">
                  <b>Supporting Information—Table S2</b>
               </div>
               <b>]</b>, using the DNA extraction protocols described in 
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. (2014)</a>
                  </sup>
               </span>. They are as follows: 
               <i>trn</i>G-
               <i>trn</i>S (
               <span ref-type="bibr" rid="PLV045C40">
                  <sup>
                     <a href="#PLV045C40">Hamilton 1999</a>
                  </sup>
               </span>), 
               <i>rpl</i>2-
               <i>trn</i>H (
               <span ref-type="bibr" rid="PLV045C103">
                  <sup>
                     <a href="#PLV045C103">Vaillancourt and Jackson 2000</a>
                  </sup>
               </span>), 
               <i>rpl</i>20-
               <i>rps</i>12 (
               <span ref-type="bibr" rid="PLV045C40">
                  <sup>
                     <a href="#PLV045C40">Hamilton 1999</a>
                  </sup>
               </span>), 
               <i>trn</i>L-
               <i>trn</i>F (
               <span ref-type="bibr" rid="PLV045C92">
                  <sup>
                     <a href="#PLV045C92">Taberlet et al. 1991</a>
                  </sup>
               </span>), 
               <i>atp</i>B-
               <i>rbcL</i> (
               <span ref-type="bibr" rid="PLV045C9">
                  <sup>
                     <a href="#PLV045C9">Campbell et al. 2007</a>
                  </sup>
               </span>), 
               <i>rps</i>16 intron (
               <span ref-type="bibr" rid="PLV045C9">
                  <sup>
                     <a href="#PLV045C9">Campbell et al. 2007</a>
                  </sup>
               </span>), 
               <i>rpl</i>16 intron (
               <span ref-type="bibr" rid="PLV045C9">
                  <sup>
                     <a href="#PLV045C9">Campbell et al. 2007</a>
                  </sup>
               </span>), 
               <i>trn</i>C-
               <i>yfc</i>6, 
               <i>acc</i>D (forward, 5′-AGAATGGGTACCTCGA-3′; reverse, 5′-GTGTGGTGATCAAGTAGTTA-3′, designed
               here), 
               <i>rpo</i>C1 (
               <span ref-type="bibr" rid="PLV045C8">
                  <sup>
                     <a href="#PLV045C8">Burgess et al. 2011</a>
                  </sup>
               </span>) and 
               <i>atp</i>F-
               <i>atp</i>H (
               <span ref-type="bibr" rid="PLV045C8">
                  <sup>
                     <a href="#PLV045C8">Burgess et al. 2011</a>
                  </sup>
               </span>). For this component of the project, the plastid barcode loci were amplified and
               sequenced using the following primers: 
               <i>mat</i>K (forward, 5′-ACCCCATTCATCTGGAAATCTTGGTTC-3′; reverse, 5′-CGTACAGTACTTTTGTGTTTACGAG-3′,
               designed here), 
               <i>rbcL</i>a (forward, 
               <span ref-type="bibr" rid="PLV045C57">
                  <sup>
                     <a href="#PLV045C57">Levin et al. 2003</a>
                  </sup>
               </span>; reverse, 
               <span ref-type="bibr" rid="PLV045C52">
                  <sup>
                     <a href="#PLV045C52">Kress and Erickson 2007</a>
                  </sup>
               </span>) and 
               <i>psb</i>A
               <i>-trn</i>H (forward, 
               <span ref-type="bibr" rid="PLV045C97">
                  <sup>
                     <a href="#PLV045C97">Tate and Simpson 2003</a>
                  </sup>
               </span>; reverse, 
               <span ref-type="bibr" rid="PLV045C85">
                  <sup>
                     <a href="#PLV045C85">Sang et al. 1997</a>
                  </sup>
               </span>). The plastid amplicons were directly sequenced using the PCR primers. Post-PCR steps
               for all markers were followed as in 
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. (2014</a>
                  </sup>
               </span>). Cycle sequencing reactions were performed using the BigDye
               <sup>®</sup> Terminator v3.1 kit (Applied Biosystems, Inc., Foster City, CA, USA). Cleaned cycle
               sequencing products were sequenced on an ABI 3730 (Applied Biosystems) DNA Analyser
               at the Royal Ontario Museum (Toronto, Canada). Sequences were proofed and edited using
               Geneious Pro. v.5.6 (
               <span ref-type="bibr" rid="PLV045C30">
                  <sup>
                     <a href="#PLV045C30">Drummond et al. 2012</a>
                  </sup>
               </span>) and assembled using Geneious Pro. v.5.6 or BioEdit v.7.0.5.3 (
               <span ref-type="bibr" rid="PLV045C39">
                  <sup>
                     <a href="#PLV045C39">Hall 1999</a>
                  </sup>
               </span>).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We also investigated two low-copy nuclear markers, i.e. the partial Phosphoenolpyruvate
               Carboxylase (PEPC) gene and the Pentatricopeptide region (PPR) homologue to the AT1G09680
               gene in Arabidopsis (AT1), in smaller samples representing mainly 
               <i>Crataegus</i> section 
               <i>Douglasia</i>
               <b>[see 
                  <div tagxxx="ext-link">Supporting Information—</div>
                  <div tagxxx="ext-link">Tables S3 and S4</div>]
               </b>. Details of the methods used are given in 
               <b>
                  <div tagxxx="ext-link">Supporting Information—</div>
               </b>
               <div tagxxx="ext-link">
                  <b>File S1</b>
               </div>.
            </p>
         </div>
         <div id="s2c">
            <h3>Chemistry sample preparation</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Leaves from 14 hawthorn samples representing four species (Table 
               <span ref-type="table" rid="PLV045TB1">
                  <a href="#PLV045TB1">1</a>
               </span>) were lyophilized for 24 h and then ground to a fine powder using a food mill. The
               milled plant material was passed through a 520 µm sieve (30 mesh) to remove any insufficiently
               milled particles. Two hundred and fifty milligrams of filtered material were extracted
               sequentially with three aliquots of deuterated methanol (methanol-d4) containing 5
               mM 4,4-dimethyl-4-silapentane-1-sulfonic acid (DSS; 1.0, 0.5, 0.5 mL), centrifuged,
               and the supernatants collected and combined. This extract was filtered through a 0.45
               µm polyfluorotetraethylene syringe filter prior to transfer into an NMR tube for analysis.
               
               <div id="PLV045TB1">
                  <span class="label">Table 1.</span>
                  <caption>
                     <p>Sources of the 
                        <i>Crataegus</i> leaf samples lyophilized and extracted for the metabolomics assays (Fig. 
                        <span ref-type="fig" rid="PLV045F2">2</span>). All WKHGC samples were collected from hawthorn individuals grown from bare root
                        stock planted in 2005, 2006 and 2007 as orchard alley cropping trees at the Naturally
                        Grown Herb and Spice Producers Cooperative (HerbPro) agroforestry demonstration farm
                        in Edgewood BC. Ploidy levels in parentheses are those known for the taxon in question;
                        all four 
                        <i>C. suksdorfii</i> shown to be diploid by flow cytometry.
                     </p>
                  </caption>
                  <table>
                     <colgroup>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                     </colgroup>
                     <thead>
                        <tr>
                           <th>Taxon</th>
                           <th>Sample</th>
                           <th>Locality</th>
                           <th>Voucher</th>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>
                              <i>C. monogyna</i> Jacq. (2
                              <i>x</i>)
                           </td>
                           <td></td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>WKHGC491</td>
                           <td>Edgewood BC (49.805, −118.156)</td>
                           <td></td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>WKHGC493</td>
                           <td>Edgewood BC (49.805, −118.156)</td>
                           <td></td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>WKHGC620</td>
                           <td>Edgewood BC (49.805, −118.156)</td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>
                              <i>C. okanaganensis</i> J.B. Phipps and O'Kennon (4
                              <i>x</i>)
                           </td>
                           <td></td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>WKHGC490</td>
                           <td>Edgewood BC (49.805, −118.156)</td>
                           <td></td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>WKHGC495</td>
                           <td>Edgewood BC (49.805, −118.156)</td>
                           <td></td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>WKHGC619</td>
                           <td>Edgewood BC (49.805, −118.156)</td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>
                              <i>C. douglasii</i> Lindl. (4
                              <i>x</i>)
                           </td>
                           <td></td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>WKHGC492</td>
                           <td>Edgewood BC (49.805, −118.156)</td>
                           <td></td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>WKHGC617</td>
                           <td>Edgewood BC (49.805, −118.156)</td>
                           <td></td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>WKHGC488</td>
                           <td>Edgewood BC (49.805, −118.156)</td>
                           <td></td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>WKHGC494</td>
                           <td>Edgewood BC (49.805, −118.156)</td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>
                              <i>C. suksdorfii</i> (Sarg.) Kruschke—2
                              <i>x</i>
                           </td>
                           <td></td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>JC042</td>
                           <td>OR, Jackson Co. (42.4307, −123.094)</td>
                           <td>TRT00020321</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>JC060</td>
                           <td>OR, Linn Co. (44.5626, −123.152)</td>
                           <td>TRT00020146</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>JC139</td>
                           <td>OR, Columbia Co. (46.1106, −122.984)</td>
                           <td>TRT00020243</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>2013-01</td>
                           <td>OR, Josephine Co. (42.278, −123.647) Deer Creek Center</td>
                           <td>TRT00028421</td>
                        </tr>
                     </div>
                  </table>
               </div>
            </p>
         </div>
         <div id="s2d">
            <h3>Nuclear magnetic resonance spectra acquisition</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A Varian MercuryPlus 400 MHz NMR instrument was used to acquire the metabolomic data
               (Varian, Inc., Palo Alto, CA, USA). One-dimensional proton spectra NMR experiments
               were acquired at 25 °C over a spectral width of 4201.7 Hz with an observe pulse of
               11.80 µs (90°) using a PRESAT water suppression sequence including a PURGE cycle.
               The acquisition time was 3.899 s, resulting in a dataset of 16 000 points. A total
               of 128 transients were acquired for each spectrum. Spectra were processed using MestReNova
               version 9.0.1 (Mestrelab Research S.L., Santiago de Compostela, Spain) and were manually
               phased and baseline corrected using a Whittaker Smoother base point detection with
               spline fitting. The spectra were then referenced, binned to chemical shift widths
               of 0.005 ppm and normalized to the DSS reference peak. A transposed data matrix of
               chemical shifts and the intensity values of the superimposed spectra was exported
               as a comma separated value file (*.CSV) and imported to Microsoft Excel for formatting
               prior to import into multivariate statistical analysis software. Statistical analysis
               was performed on the region of the spectrum where the majority of the signals are
               due to plant phenolic compounds (6.0–8.0 ppm), resulting in a data matrix of 44 samples
               with 400 observables (intensities in each chemical shift bin). The data were then
               imported into the multivariate statistical analysis software, Solo version 7.3.1 (Eigenvector
               Research, Inc., Wenatchee, WA, USA).
            </p>
         </div>
         <div id="s2e">
            <h3>Data analyses</h3>
            <div id="s2e1">
               <h3>Barcoding markers</h3>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The DNA sequences were edited and uploaded into the BOLD system (
                  <span ref-type="bibr" rid="PLV045C77">
                     <sup>
                        <a href="#PLV045C77">Ratnasingham and Hebert 2007</a>
                     </sup>
                  </span>). Following the guidelines proposed by the 
                  <span ref-type="bibr" rid="PLV045C10">
                     <sup>
                        <a href="#PLV045C10">CBOL Plant Working Group (2009)</a>
                     </sup>
                  </span> and 
                  <span ref-type="bibr" rid="PLV045C45">
                     <sup>
                        <a href="#PLV045C45">Hollingsworth et al. (2011)</a>
                     </sup>
                  </span>, the suitability of DNA barcoding for hawthorns was assessed against three main criteria:
                  (i) universality and marker amplification success, (ii) sequence quality and coverage
                  and (iii) discriminatory power among species. The data were analysed in two steps.
                  Each marker was first analysed individually. Because plastid markers in Maleae appear
                  to be inherited maternally (
                  <span ref-type="bibr" rid="PLV045C17">
                     <sup>
                        <a href="#PLV045C17">Corriveau and Coleman 1988</a>
                     </sup>
                  </span>; 
                  <span ref-type="bibr" rid="PLV045C46">
                     <sup>
                        <a href="#PLV045C46">Ishikawa et al. 1992</a>
                     </sup>
                  </span>; 
                  <span ref-type="bibr" rid="PLV045C76">
                     <sup>
                        <a href="#PLV045C76">Raspé 2001</a>
                     </sup>
                  </span>) and because they are found on a single chromosome as a linkage group (
                  <span ref-type="bibr" rid="PLV045C29">
                     <sup>
                        <a href="#PLV045C29">Doyle 1992</a>
                     </sup>
                  </span>), all three plastid markers were concatenated into a single matrix. Due to the paralogy
                  issues associated with the ITS2 marker in 
                  <i>Crataegus</i> (discussed in 
                  <span ref-type="bibr" rid="PLV045C113">
                     <sup>
                        <a href="#PLV045C113">Zarrei et al. 2014</a>
                     </sup>
                  </span>), the ITS2 data were not combined with those from the plastid markers in the final
                  analyses.
               </p>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To assess the universality of markers, we report the per cent sequencing success for
                  each locus (Table 
                  <span ref-type="table" rid="PLV045TB2">
                     <a href="#PLV045TB2">2</a>
                  </span>). The assessment of the sequence quality was performed via the CCBC automated informatics
                  pipeline following the guidelines of the 
                  <span ref-type="bibr" rid="PLV045C10">
                     <sup>
                        <a href="#PLV045C10">CBOL (2009)</a>
                     </sup>
                  </span>. Character-based analysis [maximum parsimony (MP) and Bayesian inference] has also
                  proved to be a useful tool for species identification (
                  <span ref-type="bibr" rid="PLV045C64">
                     <sup>
                        <a href="#PLV045C64">Lowenstein et al. 2009</a>
                     </sup>
                  </span>; 
                  <span ref-type="bibr" rid="PLV045C49">
                     <sup>
                        <a href="#PLV045C49">Kelly et al. 2010</a>
                     </sup>
                  </span>). However, since 
                  <span ref-type="bibr" rid="PLV045C41">
                     <sup>
                        <a href="#PLV045C41">Hebert et al. (2003a</a>
                     </sup>
                  </span>, 
                  <span ref-type="bibr" rid="PLV045C42">
                     <sup>
                        <a href="#PLV045C42">b)</a>
                     </sup>
                  </span> proposed the use of genetic distance as a standard method for analyses of barcode
                  data, the majority of barcoding studies have followed this distance-based approach
                  (reviewed in 
                  <span ref-type="bibr" rid="PLV045C98">
                     <sup>
                        <a href="#PLV045C98">Taylor and Harris 2012</a>
                     </sup>
                  </span>). Neighbour-Joining (NJ) trees (
                  <span ref-type="bibr" rid="PLV045C82">
                     <sup>
                        <a href="#PLV045C82">Saitou and Nei 1987</a>
                     </sup>
                  </span>) were built using both the tools made available by BOLD (Jalview 2, 
                  <span ref-type="bibr" rid="PLV045C105">
                     <sup>
                        <a href="#PLV045C105">Waterhouse et al. 2009</a>
                     </sup>
                  </span>; Kimura 2 parameter, after alignment by MUSCLE, 
                  <span ref-type="bibr" rid="PLV045C31">
                     <sup>
                        <a href="#PLV045C31">Edgar 2004</a>
                     </sup>
                  </span>) and PAUP* v. 4b10 (
                  <span ref-type="bibr" rid="PLV045C91">
                     <sup>
                        <a href="#PLV045C91">Swofford 2003</a>
                     </sup>
                  </span>).
                  
                  <div id="PLV045TB2">
                     <span class="label">Table 2.</span>
                     <caption>
                        <p>Barcoding information for the four markers investigated here for 355 samples representing
                           93 
                           <i>Crataegus</i> taxa 
                           <b>[see 
                              <div tagxxx="ext-link">Supporting Information—</div>
                              <div tagxxx="ext-link">Table S1</div>]
                           </b>. 
                           <sup>1</sup>Percentage of individuals successfully sequenced. 
                           <sup>2</sup>Outgroups excluded; calculated using MEGA v. 6.0 (
                           <span ref-type="bibr" rid="PLV045C96">
                              <sup>
                                 <a href="#PLV045C96">Tamura et al. 2013</a>
                              </sup>
                           </span>).
                        </p>
                     </caption>
                     <table>
                        <colgroup>
                           <col></col>
                           <col></col>
                           <col></col>
                           <col></col>
                           <col></col>
                        </colgroup>
                        <thead>
                           <tr>
                              <th>Marker</th>
                              <th>
                                 <i>mat</i>K
                              </th>
                              <th>
                                 <i>rbcL</i>a
                              </th>
                              <th>
                                 <i>psb</i>A-
                                 <i>trn</i>H
                              </th>
                              <th>ITS2</th>
                           </tr>
                        </thead>
                        <div tagxxx="tbody">
                           <tr>
                              <td>Aligned sequence length (bp)</td>
                              <td>783</td>
                              <td>552</td>
                              <td>429</td>
                              <td>671</td>
                           </tr>
                           <tr>
                              <td>Unaligned length (mean); excluding end gaps</td>
                              <td>321 (705.7 ± 99.3) 783</td>
                              <td>485 (551.3 ± 5.3) 552</td>
                              <td>183 (274.9 ± 22.0) 429</td>
                              <td>112 (312 ± 92.5) 587</td>
                           </tr>
                           <tr>
                              <td>Pairwise % identity</td>
                              <td>99.3</td>
                              <td>99.8</td>
                              <td>87.6</td>
                              <td>79.8</td>
                           </tr>
                           <tr>
                              <td>Number of taxa successfully amplified and sequences</td>
                              <td>82</td>
                              <td>93</td>
                              <td>81</td>
                              <td>65</td>
                           </tr>
                           <tr>
                              <td>Number of samples successfully amplified and sequenced</td>
                              <td>255</td>
                              <td>340</td>
                              <td>290</td>
                              <td>192</td>
                           </tr>
                           <tr>
                              <td>% sequencing success
                                 <sup>1</sup>
                              </td>
                              <td>71.8</td>
                              <td>95.8</td>
                              <td>81.7</td>
                              <td>54.1</td>
                           </tr>
                           <tr>
                              <td>Overall mean sequence divergence
                                 <sup>2</sup>
                              </td>
                              <td>0.001</td>
                              <td>0.002</td>
                              <td>0.128</td>
                              <td>0.135</td>
                           </tr>
                        </div>
                     </table>
                  </div>
               </p>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Our evaluation of the success of the different barcode loci corresponds to the tree-based
                  methods described by 
                  <span ref-type="bibr" rid="PLV045C81">
                     <sup>
                        <a href="#PLV045C81">Ross et al. (2008)</a>
                     </sup>
                  </span>. These authors used simulations to conclude that other approaches to using genetic
                  distance data to evaluate barcoding success (BLAST, comparisons of the distances between
                  unknowns and candidate reference taxa) will fail when not all candidate species are
                  represented unambiguously in the set of reference taxa. Thus we know that because
                  of the relatively limited sequence variation seen within the Maleae with individual
                  chloroplast loci (
                  <span ref-type="bibr" rid="PLV045C33">
                     <sup>
                        <a href="#PLV045C33">Evans et al. 2000</a>
                     </sup>
                  </span>; 
                  <span ref-type="bibr" rid="PLV045C9">
                     <sup>
                        <a href="#PLV045C9">Campbell et al. 2007</a>
                     </sup>
                  </span>; 
                  <span ref-type="bibr" rid="PLV045C74">
                     <sup>
                        <a href="#PLV045C74">Potter et al. 2007</a>
                     </sup>
                  </span>) the correspondence between a barcode sequence and a taxon may be ambiguous. The
                  abundance of ITS2 paralogs found by 
                  <span ref-type="bibr" rid="PLV045C113">
                     <sup>
                        <a href="#PLV045C113">Zarrei et al. (2014)</a>
                     </sup>
                  </span> will have the same effect.
               </p>
            </div>
            <div id="s2e2">
               <h3>Additional plastid markers</h3>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">All plastid sequences were submitted to GenBank (
                  <span ref-type="bibr" rid="PLV045C5">
                     <sup>
                        <a href="#PLV045C5">Benson et al. 2011</a>
                     </sup>
                  </span>), and the accession numbers are available in 
                  <div tagxxx="ext-link">
                     <b>Supporting Information—Table S2</b>
                  </div>. The sequences were aligned with CLUSTAL X (
                  <span ref-type="bibr" rid="PLV045C99">
                     <sup>
                        <a href="#PLV045C99">Thompson et al. 1997</a>
                     </sup>
                  </span>) or Geneious alignment option of Geneious Pro v.5.6 (
                  <span ref-type="bibr" rid="PLV045C30">
                     <sup>
                        <a href="#PLV045C30">Drummond et al. 2012</a>
                     </sup>
                  </span>). This initial alignment was adjusted manually in BioEdit v.7.0.5.3 or Geneious Pro
                  v.5.6 to minimize steps in the most parsimonious trees (e.g. 
                  <span ref-type="bibr" rid="PLV045C51">
                     <sup>
                        <a href="#PLV045C51">Koch et al. 2010</a>
                     </sup>
                  </span>). Because all 14 plastid markers (including the three barcode markers) used in this
                  study are linked, the concatenated sequence matrix was analysed. However, separate
                  parsimony analyses were conducted for each marker to investigate the possible incongruence
                  (discussed in 
                  <span ref-type="bibr" rid="PLV045C7">
                     <sup>
                        <a href="#PLV045C7">Bull et al. 1993</a>
                     </sup>
                  </span>) among datasets. Two different phylogenetic analyses were run for the concatenated
                  regions: (i) MP analyses using PAUP* v. 4b10 (
                  <span ref-type="bibr" rid="PLV045C91">
                     <sup>
                        <a href="#PLV045C91">Swofford 2003</a>
                     </sup>
                  </span>) and (ii) Bayesian analyses (BIs; 
                  <span ref-type="bibr" rid="PLV045C111">
                     <sup>
                        <a href="#PLV045C111">Yang and Rannala 1997</a>
                     </sup>
                  </span>) using MrBayes v. 3.2.0 (
                  <span ref-type="bibr" rid="PLV045C79">
                     <sup>
                        <a href="#PLV045C79">Ronquist and Huelsenbeck 2003</a>
                     </sup>
                  </span>; 
                  <span ref-type="bibr" rid="PLV045C80">
                     <sup>
                        <a href="#PLV045C80">Ronquist et al. 2010</a>
                     </sup>
                  </span>). The details of each analysis are the same as in 
                  <span ref-type="bibr" rid="PLV045C113">
                     <sup>
                        <a href="#PLV045C113">Zarrei et al. (2014)</a>
                     </sup>
                  </span>. Indels were coded as separate presence/absence characters using SeqState version
                  1.4.1 (
                  <span ref-type="bibr" rid="PLV045C66">
                     <sup>
                        <a href="#PLV045C66">Müller 2005</a>
                     </sup>
                  </span>) with modified complex coding option originally described by 
                  <span ref-type="bibr" rid="PLV045C87">
                     <sup>
                        <a href="#PLV045C87">Simmons and Ochoterena (2000)</a>
                     </sup>
                  </span> and appended to the end of matrices.
               </p>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In the parsimony analysis, the character state changes were equally weighted and character
                  changes were interpreted under ACCTRAN optimization (
                  <span ref-type="bibr" rid="PLV045C1">
                     <sup>
                        <a href="#PLV045C1">Agnarsson and Miller 2008</a>
                     </sup>
                  </span>). A two-stage strategy of Fitch parsimony (
                  <span ref-type="bibr" rid="PLV045C37">
                     <sup>
                        <a href="#PLV045C37">Fitch 1971</a>
                     </sup>
                  </span>) search was undertaken following 
                  <span ref-type="bibr" rid="PLV045C113">
                     <sup>
                        <a href="#PLV045C113">Zarrei et al. (2014</a>
                     </sup>
                  </span>). The phylogenetic reliability was assessed using non-parametric bootstrapping. The
                  bootstrap support (BS) was estimated using 1000 bootstrap pseudoreplicates with simple
                  taxon addition and TBR swapping but permitting only 10 trees per replicate to be held.
                  The consistency index (CI), rescaled consistency index (RC) and 
                  <span ref-type="bibr" rid="PLV045C34">
                     <sup>
                        <a href="#PLV045C34">Farris's (1989)</a>
                     </sup>
                  </span> retention index (RI) were calculated to measure the amount of homoplasy in the dataset.
                  The best-fit model for each region in the plastid concatenated matrix is provided
                  in 
                  <div tagxxx="ext-link">
                     <b>Supporting Information—Table S5</b>
                  </div>. These models were selected by Akaike information criterion (AIC; 
                  <span ref-type="bibr" rid="PLV045C2">
                     <sup>
                        <a href="#PLV045C2">Akaike 1974</a>
                     </sup>
                  </span>), as implemented in MrModeltest v. 2.3 (
                  <span ref-type="bibr" rid="PLV045C68">
                     <sup>
                        <a href="#PLV045C68">Nylander 2004</a>
                     </sup>
                  </span>).
               </p>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">For the BIs, two simultaneous runs with four chains each were run for 20 million generations.
                  In each run, every 2000th tree was sampled. The completion of the BI was determined
                  when the average standard deviation of split frequencies ≤0.05 (
                  <span ref-type="bibr" rid="PLV045C79">
                     <sup>
                        <a href="#PLV045C79">Ronquist and Huelsenbeck 2003</a>
                     </sup>
                  </span>) for the combined two runs was assumed and the complete convergence between the Bayesian
                  Markov chain Monte Carlo runs was reached. Convergence of an independent search was
                  further explored by plotting likelihood scores vs generations using the program Tracer
                  v1.5 (
                  <span ref-type="bibr" rid="PLV045C75">
                     <sup>
                        <a href="#PLV045C75">Rambaut and Drummond 2007</a>
                     </sup>
                  </span>). The burn-in phase for each run—the first 25 % of sampled trees—was discarded during
                  computing the phylogram consensus tree based on the average branch lengths (50 % majority
                  rule) of the remaining trees (15 000 trees) using 
                  <i>sumt</i> command implemented in MrBayes. Support for Bayesian topologies was estimated using
                  node posterior probabilities (PPs) from the posterior distribution of topologies.
               </p>
            </div>
            <div id="s2e3">
               <h3>Metabolomic data</h3>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Classes were defined for each sample according to its species identification. Before
                  multivariate analysis, mean centring and Pareto scaling, where each variable is divided
                  by the square of its standard deviation, were applied to the dataset (
                  <span ref-type="bibr" rid="PLV045C104">
                     <sup>
                        <a href="#PLV045C104">van den Berg et al. 2006</a>
                     </sup>
                  </span>). Following this, a hierarchical cluster analysis (HCA) was performed. The HCA dendrogram
                  was generated using Ward's minimum variance method with Mahalanobis distance and generalized
                  least squares weighting to 
                  <i>α</i> = 0.001.
               </p>
            </div>
         </div>
      </div>
      <div id="s3">
         <h2>Results</h2>
         <div id="s3a">
            <h3>Barcoding markers</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Across all four regions investigated, a total of 1077 assembled DNA sequences were
               obtained from 355 samples (mean = 3.03 regions sequenced per sample; BOLD dx.doi.org/10.5883/DS-NAMCRAT,
               Table 
               <span ref-type="table" rid="PLV045TB2">
                  <a href="#PLV045TB2">2</a>
               </span>; 
               <div tagxxx="ext-link">
                  <b>Supporting Information—Table S1</b>
               </div>). The overall average sample sequencing success was 75.8 %, ranging from 54.1 % for
               ITS2 to 95.8 % for 
               <i>rbcL</i>a (Table 
               <span ref-type="table" rid="PLV045TB2">
                  <a href="#PLV045TB2">2</a>
               </span>). Internal transcribed spacer 2 had the lowest species sequencing success (69.9 %),
               compared with 100 % for 
               <i>rbcL</i>a (Table 
               <span ref-type="table" rid="PLV045TB2">
                  <a href="#PLV045TB2">2</a>
               </span>). The sequences were retrieved for all the markers using both the forward and reverse
               primers. The variation in both the sequence length (because of insertions or deletions)
               and additive polymorphic sites (APS) was detected in the sequences of ITS2 due to
               the presence of multiple copies of ITS2 in the genome of each individual (see Discussion
               for more detail). No APS were detected in the traces obtained from the plastid markers.
               However, displacement of electropherograms was observed in 
               <i>psb</i>A
               <i>-trn</i>H tracers due to the presence of homopolymer runs (
               <span ref-type="bibr" rid="PLV045C20">
                  <sup>
                     <a href="#PLV045C20">Devey et al. 2009</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C35">
                  <sup>
                     <a href="#PLV045C35">Fazekas et al. 2010</a>
                  </sup>
               </span>). Sequencing with the reverse primer resolved this problem. Pairwise percentage sequence
               identity varied from a low of 79.8 % (ITS2; estimated using Geneious Pro. v.5.6, 
               <span ref-type="bibr" rid="PLV045C30">
                  <sup>
                     <a href="#PLV045C30">Drummond et al. 2012</a>
                  </sup>
               </span>; Table 
               <span ref-type="table" rid="PLV045TB2">
                  <a href="#PLV045TB2">2</a>
               </span>), to a high of 99.8 % (
               <i>rbcL</i>a; Table 
               <span ref-type="table" rid="PLV045TB2">
                  <a href="#PLV045TB2">2</a>
               </span>). The lowest sequence identity among the plastid barcode markers investigated here
               was 87.6 % for the 
               <i>psb</i>A
               <i>-trn</i>H spacer (Table 
               <span ref-type="table" rid="PLV045TB2">
                  <a href="#PLV045TB2">2</a>
               </span>).
            </p>
            <div id="s3a1">
               <h3>Resolution of the barcode loci</h3>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Despite earlier success using barcode sequence data to confirm the parentage of two
                  hybrids (
                  <span ref-type="bibr" rid="PLV045C15">
                     <sup>
                        <a href="#PLV045C15">Christensen et al. 2014</a>
                     </sup>
                  </span>), only a very limited number of 
                  <i>Crataegus</i> species are diagnosable using individual plastid barcode loci (Table 
                  <span ref-type="table" rid="PLV045TB3">
                     <a href="#PLV045TB3">3</a>
                  </span>; cf. 
                  <span ref-type="bibr" rid="PLV045C36">
                     <sup>
                        <a href="#PLV045C36">Fineschi et al. 2005</a>
                     </sup>
                  </span>). In addition to the six taxa (five species and one hybrid) for which at least one
                  locus provided a diagnostic position (Table 
                  <span ref-type="table" rid="PLV045TB3">
                     <a href="#PLV045TB3">3</a>
                  </span>), single-nucleotide polymorphisms also diagnose two interesting groupings including
                  the autotriploid 
                  <i>C. gaylussacia</i> with its probable progenitor, diploid 
                  <i>C. suksdorfii</i> (
                  <i>mat</i>K, Table 
                  <span ref-type="table" rid="PLV045TB3">
                     <a href="#PLV045TB3">3</a>
                  </span>; 
                  <span ref-type="bibr" rid="PLV045C113">
                     <sup>
                        <a href="#PLV045C113">Zarrei et al. 2014</a>
                     </sup>
                  </span>). The other grouping of interest is that of 
                  <i>C.</i> × 
                  <i>canescens</i> together with (i) 
                  <i>C.</i> sect. 
                  <i>Crataegus</i> and its hybrids with 
                  <i>C. punctata</i> (
                  <span ref-type="bibr" rid="PLV045C15">
                     <sup>
                        <a href="#PLV045C15">Christensen et al. 2014</a>
                     </sup>
                  </span>), (ii) 
                  <i>C. brachyacantha</i>, and 
                  <i>C. spathulata</i> (
                  <i>rbcL</i>a, Table 
                  <span ref-type="table" rid="PLV045TB3">
                     <a href="#PLV045TB3">3</a>
                  </span>) and (iii) 
                  <i>C. germanica</i> (
                  <i>C.</i> sect. 
                  <i>Mespilus</i>), and all of the 
                  <i>C.</i> ser. 
                  <i>Cerrones</i> taxa in the sample (
                  <i>psb</i>A
                  <i>-trn</i>H, Table 
                  <span ref-type="table" rid="PLV045TB3">
                     <a href="#PLV045TB3">3</a>
                  </span>). On the basis of their molecular phylogeny of 
                  <i>Crataegus</i>, 
                  <span ref-type="bibr" rid="PLV045C60">
                     <sup>
                        <a href="#PLV045C60">Lo et al. (2007</a>
                     </sup>
                  </span>) suggested that 
                  <i>C.</i> × 
                  <i>canescens</i> (originally described as 
                  <i>Mespilus canescens</i> J.B. Phipps) was likely a hybrid involving 
                  <i>C. germanica</i>, 
                  <i>C. brachyacantha</i> and an unknown third taxon.
                  
                  <div id="PLV045TB3">
                     <span class="label">Table 3.</span>
                     <caption>
                        <p>Single-nucleotide changes in the three chloroplast markers investigated in 
                           <i>Crataegus</i> for 355 samples representing 93 distinct taxa 
                           <b>[see 
                              <div tagxxx="ext-link">Supporting Information—</div>
                              <div tagxxx="ext-link">Table S1</div>]
                           </b>. With these loci the accessions belonging to six taxa formed distinct single-taxon
                           clusters in the corresponding NJ tree calculated by BOLD; these clusters result from
                           the polymorphisms shown (position in the consensus sequence). In bold, polymorphisms
                           and positions scored as diagnostic by BOLD (
                           <i>n</i> ≥ 3). Ploidy levels shown are based on data from earlier studies (
                           <span ref-type="bibr" rid="PLV045C93">
                              <sup>
                                 <a href="#PLV045C93">Talent and Dickinson 2005</a>
                              </sup>
                           </span>; 
                           <span ref-type="bibr" rid="PLV045C19">
                              <sup>
                                 <a href="#PLV045C19">Coughlan et al. 2014</a>
                              </sup>
                           </span>; 
                           <span ref-type="bibr" rid="PLV045C113">
                              <sup>
                                 <a href="#PLV045C113">Zarrei et al. 2014</a>
                              </sup>
                           </span>). In addition to 
                           <i>C.</i> × 
                           <i>canescens</i> (
                           <span ref-type="bibr" rid="PLV045C60">
                              <sup>
                                 <a href="#PLV045C60">Lo et al. 2007</a>
                              </sup>
                           </span>), 
                           <i>C. spathulata</i> (
                           <span ref-type="bibr" rid="PLV045C60">
                              <sup>
                                 <a href="#PLV045C60">Lo et al. 2007</a>
                              </sup>
                           </span>) and 
                           <i>C. nigra</i> (
                           <span ref-type="bibr" rid="PLV045C113">
                              <sup>
                                 <a href="#PLV045C113">Zarrei et al. 2014</a>
                              </sup>
                           </span>) are suspected of being (paleo-) hybrids.
                        </p>
                     </caption>
                     <table>
                        <colgroup>
                           <col></col>
                           <col></col>
                           <col></col>
                           <col></col>
                        </colgroup>
                        <thead>
                           <tr>
                              <th>Marker</th>
                              <th>
                                 <i>mat</i>K
                              </th>
                              <th>
                                 <i>rbcL</i>a
                              </th>
                              <th>
                                 <i>psb</i>A-
                                 <i>trn</i>H
                              </th>
                           </tr>
                        </thead>
                        <div tagxxx="tbody">
                           <tr>
                              <td>
                                 <i>C.</i> × 
                                 <i>canescens</i> (allotriploid; 
                                 <i>n</i> = 6)
                              </td>
                              <td></td>
                              <td>T → C (5) with 
                                 <i>N</i> = 21 diploid 
                                 <i>C.</i> sect. 
                                 <i>Crataegus</i> and 4 
                                 <i>C.</i> × 
                                 <i>ninae-celottiae</i>
                              </td>
                              <td>G → C (287) with 
                                 <i>N</i> = 10 diploid 
                                 <i>C. germanica</i> and 17 
                                 <i>C.</i> ser. 
                                 <i>Cerrones</i>
                              </td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>A→G (57)</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>
                                 <b>C → G (121)</b>
                              </td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>A→G (391)</td>
                           </tr>
                           <tr>
                              <td>
                                 <i>C. brachyacantha</i> (diploid; 
                                 <i>n</i> = 9)
                              </td>
                              <td></td>
                              <td>A→G (57)</td>
                              <td></td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>
                                 <b>G → C (244)</b>
                              </td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>
                                 <b>G → A (357)</b>
                              </td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>A →G (391)</td>
                           </tr>
                           <tr>
                              <td>
                                 <i>C. spathulata</i> (diploid; 
                                 <i>n</i> = 3)
                              </td>
                              <td>
                                 <b>T → A (238)</b>
                              </td>
                              <td>A→G (57)</td>
                              <td>G→A (91)</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>
                                 <b>A → G (621)</b>
                              </td>
                              <td>A→G (391)</td>
                           </tr>
                           <tr>
                              <td>
                                 <i>C. gaylussacia</i> (autotriploid; 
                                 <i>n</i> = 6)
                              </td>
                              <td>C→T (288) with 
                                 <i>N</i> = 7 diploid 
                                 <i>C. suksdorfii</i>
                              </td>
                              <td>
                                 <b>C → T (417)</b>
                              </td>
                              <td></td>
                           </tr>
                           <tr>
                              <td>
                                 <i>C. nigra</i> (diploid; 
                                 <i>n</i> = 3)
                              </td>
                              <td>G→T (277)</td>
                              <td>
                                 <b>A → C (120)</b>
                              </td>
                              <td></td>
                           </tr>
                           <tr>
                              <td>
                                 <i>C. pinnatifida</i> (diploids, triploids, tetraploids and hexaploids known; 
                                 <i>n</i> = 3)
                              </td>
                              <td>T→G (75) with 
                                 <i>N</i>=3 of seven 
                                 <i>C. macracantha</i>
                              </td>
                              <td></td>
                              <td>G→T (197)</td>
                           </tr>
                           <tr>
                              <td></td>
                              <td>
                                 <b>G → A (354)</b> with 
                                 <i>N</i> = 2 
                                 <i>C. hupehensis</i>
                              </td>
                              <td></td>
                              <td></td>
                           </tr>
                        </div>
                     </table>
                  </div>
               </p>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Combining the sequences of the three plastid barcode loci into a single alignment
                  yielded an NJ tree 
                  <b>[see 
                     <div tagxxx="ext-link">Supporting Information—</div>
                     <div tagxxx="ext-link">Fig. S1</div>]
                  </b> in which some more and better-resolved clusters appeared, but few of these comprised
                  only a single species, or even only closely related species (e.g. belonging to the
                  same taxonomic series 
                  <b>[see 
                     <div tagxxx="ext-link">Supporting Information—</div>
                     <div tagxxx="ext-link">Fig. S1</div>]
                  </b>). Also, topological relationships between groups expected from earlier analyses of
                  more informative loci (
                  <span ref-type="bibr" rid="PLV045C60">
                     <sup>
                        <a href="#PLV045C60">Lo et al. 2007</a>
                     </sup>
                  </span>, 
                  <span ref-type="bibr" rid="PLV045C61">
                     <sup>
                        <a href="#PLV045C61">2009</a>
                     </sup>
                  </span>; 
                  <span ref-type="bibr" rid="PLV045C113">
                     <sup>
                        <a href="#PLV045C113">Zarrei et al. 2014</a>
                     </sup>
                  </span>) were not recovered in the NJ tree based only on 
                  <i>rbcL</i>a, 
                  <i>mat</i>K and 
                  <i>psb</i>A
                  <i>-trn</i>H data 
                  <b>[see 
                     <div tagxxx="ext-link">Supporting Information</div>
                     <div tagxxx="ext-link">—Fig. S1</div>]
                  </b>.
               </p>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The NJ tree based on ITS2 sequences obtained by direct amplification of genomic DNA
                  according to the BOLD protocol 
                  <b>[see 
                     <div tagxxx="ext-link">Supporting Information—</div>
                     <div tagxxx="ext-link">Fig. S2</div>]
                  </b> is similarly problematic. As described earlier (
                  <span ref-type="bibr" rid="PLV045C27">
                     <sup>
                        <a href="#PLV045C27">Dickinson et al. 2011</a>
                     </sup>
                  </span>), while some diploids formed well-resolved single-species clusters (
                  <div tagxxx="ext-link">
                     <b>Supporting Information—Fig. S2</b>
                  </div>; e.g. 
                  <i>C. brachyacantha</i>, 
                  <i>C. germanica</i>, 
                  <i>C. spathulata</i>), others did not (
                  <div tagxxx="ext-link">
                     <b>Supporting Information—Fig. S2</b>
                  </div>; e.g. 
                  <i>C. marshallii</i>, 
                  <i>C. punctata</i>). Polyploids were found in heterogeneous clusters that in some cases did not include
                  the most closely related diploid species (
                  <div tagxxx="ext-link">
                     <b>Supporting Information—Fig. S2</b>
                  </div>; e.g. 
                  <i>C</i>. ser. 
                  <i>Cerrones</i>).
               </p>
            </div>
         </div>
         <div id="s3b">
            <h3>Phylogenetic analysis of 14 plastid markers</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">No incongruence was detected between datasets in separate parsimony analyses of the
               14 markers. In contrast to clustering based on one or a few barcode loci, phylogenetic
               analysis of a total of 560 sequences for 14 markers 
               <b>[see 
                  <div tagxxx="ext-link">Supporting Information</div>
                  <div tagxxx="ext-link">—Table S6</div>]
               </b> yielded much greater resolution, and recovered the groups and topological relationships
               expected from earlier analyses (in Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>, branches are labelled so as to correspond as nearly as possible with those in Fig. 4
               of 
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>). The concatenated plastid matrix comprised 10 570 sites; of which, 487 (4.6 %) were
               variable and 190 (1.84 %) were parsimony-informative (including outgroups). The loci
               with highest percentages of parsimony-informative sites were 
               <i>rpl</i>2-
               <i>trn</i>H (3.65 %) and 
               <i>psb</i>A
               <i>-trn</i>H (3.35 %). Two markers, 
               <i>rpo</i>C1 (0.18 %) and 
               <i>rbcL</i>a (0.36 %), yielded the fewest parsimony-informative sites. Due to the very low resolution
               of the trees resulting from the individual markers, these trees are not shown. The
               first and second stage of parsimony analysis (
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>) of the concatenated plastid matrix generated 18 trees, each with: 587 steps, a CI
               of 0.86, a RC of 0.77 and a RI of 0.89 (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>). The ingroup is well supported (PP = 1; BS = 99).
               
               <div tagxxx="fig">
                  <span class="label">Figure 1.</span>
                  <caption>
                     <p>
                        <i>Crataegus</i> phylogram based on sequence data from 14 plastid loci 
                        <b>[see 
                           <div tagxxx="ext-link">Supporting Information—</div>
                           <div tagxxx="ext-link">Table S2</div>]
                        </b>; majority rule consensus of 15 000 Bayesian trees. Numbers above branches are PP;
                        numbers below the branches are percent BS. For ease of comparison, clades are labelled
                        as in Fig. 4 of 
                        <span ref-type="bibr" rid="PLV045C113">
                           <sup>
                              <a href="#PLV045C113">Zarrei et al. (2014)</a>
                           </sup>
                        </span>. Scale for branch lengths is substitutions per site. 
                        <i>Crataegus suksdorfii</i> 3
                        <i>x</i>* = sampled from Haida Gwaii; 
                        <i>C. suksdorfii</i> 3
                        <i>x</i>^ = sampled from Vancouver Island; 
                        <i>C. rivularis</i>* sampled from Colorado (NT273); 
                        <i>C. rivularis</i>^ sampled from Wyoming (2001–42). An asterisk following the bootstrap value indicates
                        the branches collapse in the strict consensus tree of 18 trees. The infrageneric classification
                        mapped onto the tree is based on 
                        <span ref-type="bibr" rid="PLV045C15">
                           <sup>
                              <a href="#PLV045C15">Christensen et al. (2014)</a>
                           </sup>
                        </span>, Dickinson 
                        <i>et al</i>. (in prep.) and 
                        <span ref-type="bibr" rid="PLV045C71">
                           <sup>
                              <a href="#PLV045C71">Phipps (2015)</a>
                           </sup>
                        </span>.
                     </p>
                  </caption>
                  <div tagxxx="graphic"></div>
               </div>
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Topological relationships between groups expected from earlier analyses (
               <span ref-type="bibr" rid="PLV045C60">
                  <sup>
                     <a href="#PLV045C60">Lo et al. 2007</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="PLV045C61">
                  <sup>
                     <a href="#PLV045C61">2009</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>) were found here (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>), as was not the case in the analysis of only three concatenated plastid loci 
               <b>[see 
                  <div tagxxx="ext-link">Supporting Information—</div>
                  <div tagxxx="ext-link">Fig. S1</div>]
               </b> or in that of the ITS2 sequences amplified from genomic DNA 
               <b>[see 
                  <div tagxxx="ext-link">Supporting Information—</div>
                  <div tagxxx="ext-link">Fig. S2</div>]
               </b>. 
               <i>Crataegus germanica</i> (
               <i>C.</i> section 
               <i>Mespilus</i>) and 
               <i>C. brachyacantha</i> (
               <i>C.</i> section 
               <i>Brevispinae</i>) branch as successive sisters to the remaining ingroup taxa. 
               <i>Crataegus</i> section 
               <i>Crataegus</i> forms a well-supported group (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>, clade A; PP = 1, BS = 94). Its sister clade (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>, Clade B) comprises 
               <i>C.</i> sections 
               <i>Coccineae</i> and 
               <i>Macracanthae</i> mainly of eastern North America (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>, clade E) and an unlabelled clade (star, Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>). This unlabelled clade consists of the remaining Eurasian species (in 
               <i>C.</i> sect. 
               <i>Sanguineae</i>, branches G
               <div tagxxx="sub">1</div> and G
               <div tagxxx="sub">2</div>) and the primarily western North American 
               <i>C.</i> sect. 
               <i>Douglasia</i> (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>, clades F and H).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Clade D comprises species belonging to 
               <i>C.</i> section 
               <i>Sanguineae</i> and 
               <i>C.</i> section 
               <i>Douglasia</i> (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>). The phylogenetic relationship of 
               <i>C. maximoviczii</i> (
               <i>C.</i> section 
               <i>Sanguineae</i>; Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>, clade G
               <div tagxxx="sub">1</div>) is not resolved and this species is in a polytomy with clades D and F. Clade F comprises
               
               <i>C.</i> series 
               <i>Cerrones</i> (PP = 1; BS = 95) from the central Rocky Mountains, and the southern British Columbia
               allotetraploid, 
               <i>C. shuswapensis</i> (
               <i>C.</i> series 
               <i>Douglasianae</i>).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">As noted earlier (
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>), the behaviour of well-documented recent intersectional hybrids (
               <span ref-type="bibr" rid="PLV045C15">
                  <sup>
                     <a href="#PLV045C15">Christensen et al. 2014</a>
                  </sup>
               </span>) helps elucidate the allopolyploid status of a number of species in 
               <i>C.</i> section 
               <i>Douglasia</i>. Three individuals of 
               <i>C.</i> × 
               <i>ninae-celottiae</i> (=
               <i>C. monogyna</i> × 
               <i>C. punctata</i>; 
               <span ref-type="bibr" rid="PLV045C15">
                  <sup>
                     <a href="#PLV045C15">Christensen et al. 2014</a>
                  </sup>
               </span>) analysed here are grouped with other members of sect. 
               <i>Crataegus</i> in clade A with PP = 1 and BS = 99 (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>). In contrast, the two individuals of 
               <i>C.</i> × 
               <i>cogswellii</i> (=diploid 
               <i>C. suksdorfii</i>× 
               <i>C. monogyna</i>; 
               <span ref-type="bibr" rid="PLV045C15">
                  <sup>
                     <a href="#PLV045C15">Christensen et al. 2014</a>
                  </sup>
               </span>) in our sample form a clade (
               <i>I</i>
               <div tagxxx="sub">3</div>, Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>) with diploid 
               <i>C. suksdorfii</i> with PP = 1 and BS = 98. Both nothospecies arose from hybridization over the past
               200 years between native North American diploids and the introduced Eurasian species,
               
               <i>C. monogyna</i> (
               <span ref-type="bibr" rid="PLV045C15">
                  <sup>
                     <a href="#PLV045C15">Christensen et al. 2014</a>
                  </sup>
               </span>). The cladistic relationships between these hybrids and their parents seen in a phylogeny
               based on maternally inherited loci provide a good indication of the predominant direction
               in which hybridization has taken place (
               <span ref-type="bibr" rid="PLV045C15">
                  <sup>
                     <a href="#PLV045C15">Christensen et al. 2014</a>
                  </sup>
               </span>).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The same logic applies to the cladistic relationships between allopolyploids in 
               <i>C.</i> series 
               <i>Douglasianae</i> (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>, clade H, mostly) and those in 
               <i>C.</i> nothoseries 
               <i>Montaninsulae</i> and 
               <i>Purpureofructus</i> (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>, clades E
               <div tagxxx="sub">2</div> and E
               <div tagxxx="sub">3</div>). These relationships (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>) suggest that tetraploid 
               <i>C. chrysocarpa</i> (or another member of 
               <i>C.</i> series 
               <i>Rotundifoliae</i>) is the female parent of 
               <i>C. okanaganensis</i> (also tetraploid; clade E
               <div tagxxx="sub">2</div>, PP = 0.71, BS = 69), whereas tetraploid 
               <i>C. macracantha</i> (or another member of 
               <i>C.</i> sect. 
               <i>Macracanthae</i>) is the female parent of the remaining section 
               <i>Douglasia</i> species in clade E
               <div tagxxx="sub">3</div> (PP = 1, BS = 99). Members of 
               <i>C.</i> sect. 
               <i>Coccineae</i> (probably also either 
               <i>C. chrysocarpa</i> or 
               <i>C. macracantha</i>, as these species are the only ones with any appreciable range in the trans-Mississippi
               west) appear to have been the male parents of the other allopolyploids in 
               <i>C.</i> section 
               <i>Douglasia</i> (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>; clades I
               <div tagxxx="sub">2</div>, 
               <i>C. castlegarensis</i> and 
               <i>C. enderbyensis</i>, and I
               <div tagxxx="sub">4</div>, 
               <i>C. douglasii</i> and 
               <i>C. okennoni</i>; 
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>). As in the case of the two diploid nothospecies derived from 
               <i>C. monogyna</i> (
               <span ref-type="bibr" rid="PLV045C15">
                  <sup>
                     <a href="#PLV045C15">Christensen et al. 2014</a>
                  </sup>
               </span>), the presence of cloned ITS2 variants from both 
               <i>C.</i> section 
               <i>Coccineae</i> and 
               <i>C.</i> section 
               <i>Douglasia</i> in these tetraploid nothospecies clinched their hybrid status (
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>). Finally, the same logic elucidates the origin of polyploid 
               <i>C. suksdorfii</i> (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>, clade I4; 
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>).
            </p>
         </div>
         <div id="s3c">
            <h3>Metabolomic analysis of Crataegus species</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Comparison of four 
               <i>Crataegus</i> species (Fig. 
               <span ref-type="fig" rid="PLV045F2">2</span>) with respect to their 
               <sup>1</sup>H NMR metabolomics data demonstrates the greater similarity of the two allotetraploids
               in the sample to each other, relative to their putative common ancestor, diploid 
               <i>C. suksdorfii. Crataegus monogyna</i> is most dissimilar, and in this respect these limited metabolomics results parallel
               the phylogenetic relationships determined from both chloroplast DNA sequence data
               (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>) and nuclear DNA (
               <span ref-type="bibr" rid="PLV045C60">
                  <sup>
                     <a href="#PLV045C60">Lo et al. 2007</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="PLV045C61">
                  <sup>
                     <a href="#PLV045C61">2009</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>). This analysis targeted phenolic compounds because they have demonstrated cardioprotective
               activity. As specialized metabolites they are also more likely to vary more between
               species than within samples compared with primary metabolites such as sugars.
               
               <div tagxxx="fig">
                  <span class="label">Figure 2.</span>
                  <caption>
                     <p>Hierarchical cluster analysis of 
                        <sup>1</sup>H NMR metabolomics data of phenolic compounds (6–8 ppm) from four 
                        <i>Crataegus</i> species (Table 
                        <span ref-type="table" rid="PLV045TB1">
                           <a href="#PLV045TB1">1</a>
                        </span>).
                     </p>
                  </caption>
                  <div tagxxx="graphic"></div>
               </div>
            </p>
         </div>
      </div>
      <div id="s4">
         <h2>Discussion</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Using data from 93 mostly species-level 
            <i>Crataegus</i> taxa that represent all major clades and 8 out of 10 sections of the genus, we assess
            DNA barcode markers using the following three criteria: (i) universality and marker
            amplification success, (ii) sequence quality and coverage and (iii) discriminatory
            power among species. We will also comment on the additional information that our DNA
            barcode data provide and on the serious limitations that our results place on using
            DNA barcodes to identify hawthorn species and authenticate hawthorn NHPs.
         </p>
         <div id="s4a">
            <h3>Barcode universality and amplification success in Crataegus</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The plastid markers met the first criterion for DNA barcoding, namely that amplification
               succeeded with almost all hawthorn accessions tested (87.1–100 % of the taxa amplified;
               Table 
               <span ref-type="table" rid="PLV045TB2">
                  <a href="#PLV045TB2">2</a>
               </span>). In contrast, amplification of the ITS2 region was generally less successful, and
               obtaining the data analysed here required recourse to additional primer pairs (65
               taxa amplified; Table 
               <span ref-type="table" rid="PLV045TB2">
                  <a href="#PLV045TB2">2</a>
               </span>).
            </p>
         </div>
         <div id="s4b">
            <h3>Crataegus barcode sequence quality and coverage</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The plastid markers also met the second criterion for DNA barcoding, namely that sequence
               quality was generally high for each of the markers investigated here (Table 
               <span ref-type="table" rid="PLV045TB2">
                  <a href="#PLV045TB2">2</a>
               </span>). However, the sequencing success for 
               <i>mat</i>K was lower than that for the rest of the plastid markers. The ITS2 region had the
               lowest rate of sequencing success (69.9 %). Sequence quality was also low so that,
               overall, ITS2 yielded the fewest barcode sequences (Table 
               <span ref-type="table" rid="PLV045TB2">
                  <a href="#PLV045TB2">2</a>
               </span>).
            </p>
         </div>
         <div id="s4c">
            <h3>The limits of plastid DNA barcode discrimination of Crataegus species</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">There are more than 200 
               <i>Crataegus</i> species native to North America (
               <span ref-type="bibr" rid="PLV045C71">
                  <sup>
                     <a href="#PLV045C71">Phipps 2015</a>
                  </sup>
               </span>). We have sampled 93 mostly species-level taxa, and our accessions represent all
               four native sections, one naturalized section and the four nothosections of the genus
               that are currently recognized (
               <span ref-type="bibr" rid="PLV045C60">
                  <sup>
                     <a href="#PLV045C60">Lo et al. 2007</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C6">
                  <sup>
                     <a href="#PLV045C6">Brouillet et al. 2010+</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C15">
                  <sup>
                     <a href="#PLV045C15">Christensen et al. 2014</a>
                  </sup>
               </span>). In addition, our accessions represent a comprehensive sample of 
               <i>Crataegus</i> in western North America, and of 
               <i>C.</i> section 
               <i>Douglasia</i>, thus matching recommendations in 
               <span ref-type="bibr" rid="PLV045C73">
                  <sup>
                     <a href="#PLV045C73">Piredda et al. (2011)</a>
                  </sup>
               </span> for ensuring that the correct relationship between barcode sequences and species
               is obtained. Individually, the nucleotide diversity of the three plastid DNA barcode
               markers was low and distinguishes only three species and one nothospecies (Table 
               <span ref-type="table" rid="PLV045TB2">
                  <a href="#PLV045TB2">2</a>
               </span>; five in all, if the European species 
               <i>C. nigra</i> is included). Similar results using just the 
               <i>psb</i>A
               <i>-trn</i>H barcode locus have been obtained in 
               <i>Fragaria</i>, where only two species out of 21 could be distinguished (
               <span ref-type="bibr" rid="PLV045C115">
                  <sup>
                     <a href="#PLV045C115">Njuguna and Bassil 2011</a>
                  </sup>
               </span>). In the NJ tree for the concatenated plastid loci (317 individuals with data for
               at least two loci), only one more species forms a distinct single-taxon cluster (
               <div tagxxx="ext-link">
                  <b>Supporting Information—Fig. S1</b>
               </div>, diploid 
               <i>C. germanica</i>). We have little doubt that these tallies could be marginally improved with further
               sampling of species, and of individuals within species, but not to a large extent.
               Comparison of the plastid and nuclear genomes (
               <span ref-type="bibr" rid="PLV045C109">
                  <sup>
                     <a href="#PLV045C109">Wolfe et al. 1987</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C16">
                  <sup>
                     <a href="#PLV045C16">Clegg et al. 1994</a>
                  </sup>
               </span>), as well as studies of the molecular systematics of the Rosaceae in general and
               the Maleae in particular (
               <span ref-type="bibr" rid="PLV045C33">
                  <sup>
                     <a href="#PLV045C33">Evans et al. 2000</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C9">
                  <sup>
                     <a href="#PLV045C9">Campbell et al. 2007</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C74">
                  <sup>
                     <a href="#PLV045C74">Potter et al. 2007</a>
                  </sup>
               </span>; cf. 
               <span ref-type="bibr" rid="PLV045C67">
                  <sup>
                     <a href="#PLV045C67">Nikiforova et al. 2013</a>
                  </sup>
               </span>), all point to low levels of sequence variation in plastid loci especially in recently
               evolved species. As a result, in 
               <i>Crataegus</i> a reasonable degree of phylogenetic resolution was only obtained by concatenating
               sequence data from 14 plastid loci (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>; cf. 
               <span ref-type="bibr" rid="PLV045C59">
                  <sup>
                     <a href="#PLV045C59">Lo and Donoghue 2012</a>
                  </sup>
               </span>), but this is not a solution acceptable from the current barcoding point of view.
            </p>
         </div>
         <div id="s4d">
            <h3>The limits of ITS2 DNA barcode discrimination of Crataegus species</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The intragenomic variability of the ITS2 region in 
               <i>Crataegus</i> has negative consequences for DNA barcoding. Several different copies of ITS2 (ribotypes)
               are present in the genomes not only of polyploids, but also diploids (
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>), including recent hybrids (e.g. 
               <i>C.</i>× 
               <i>cogswellii</i> and 
               <i>C.</i>× 
               <i>ninae-celottiae</i>; 
               <span ref-type="bibr" rid="PLV045C63">
                  <sup>
                     <a href="#PLV045C63">Love and Feigen 1978</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C107">
                  <sup>
                     <a href="#PLV045C107">Wells and Phipps 1989</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C15">
                  <sup>
                     <a href="#PLV045C15">Christensen et al. 2014</a>
                  </sup>
               </span>). Both indels and APS among ribotypes limit direct amplification of sequences from
               genomic DNA. For example, high-quality sequences could not be obtained from the recent
               hybrids 
               <i>C.</i>× 
               <i>cogswellii</i> and 
               <i>C.</i>× 
               <i>ninae-celottiae</i>. Several indels at different sites caused electropherogram displacements. Although
               bidirectionally sequencing this short marker (mean sequence length = 314) contributed
               to sequence recovery, the problem associated with identifying the paralogs still remained.
               Cloning the ITS2 amplicons revealed the extent of hybridization in polyploid black-fruited
               taxa of 
               <i>Crataegus</i> by demonstrating the co-occurrence in a single individual of ITS2 sequences from
               sections 
               <i>Coccineae</i> and 
               <i>Douglasia</i> (
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>). This highlights the limited utility of ITS2 as a barcoding marker particularly
               for allopolyploid taxa (
               <span ref-type="bibr" rid="PLV045C45">
                  <sup>
                     <a href="#PLV045C45">Hollingsworth et al. 2011)</a>
                  </sup>
               </span>.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The DNA barcoding protocol of direct sequencing of ITS2 is thus likely to lead to
               incorrect species identifications by randomly amplifying only a single ribotype, especially
               from a polyploid in which the number of paralogs is unknown. In the case of diploid
               
               <i>C. spathulata</i>, all three individuals form a single, distinct cluster (clade C
               <div tagxxx="sub">1</div>, 
               <div tagxxx="ext-link">
                  <b>Supporting Information—Fig. S2</b>
               </div>; see also 
               <i>C. brachyacantha</i>, clade A
               <div tagxxx="sub">2</div>, and 
               <i>C. germanica</i>, clade A
               <div tagxxx="sub">1</div>). However, an equally distinct cluster (
               <div tagxxx="ext-link">
                  <b>Supporting Information—Fig. S2</b>
               </div>, clade C
               <div tagxxx="sub">2</div>) comprises all three individuals of diploid 
               <i>C. marshallii</i> (series 
               <i>Apiifoliae</i> in section 
               <i>Crataegus</i>) plus 
               <i>C. displar</i>, a tetraploid in series 
               <i>Lacrimatae</i> (section 
               <i>Coccineae</i>). Another example is 
               <i>C. pinnatifida</i> (section 
               <i>Crataegus</i>); ITS2 sequences were obtained for two of the individuals of this species 
               <b>[see 
                  <div tagxxx="ext-link">Supporting Information</div>
                  <div tagxxx="ext-link">—Table S1</div>]
               </b>. One of these (TRT099) forms a distinct cluster with 
               <i>C. hupehensis</i> (section 
               <i>Hupehenses</i>; 
               <div tagxxx="ext-link">
                  <b>Supporting Information—Fig. S2</b>
               </div>), while the other (TRT100) clusters with 
               <i>C. wilsonii</i> (section 
               <i>Sanguineae</i>; 
               <div tagxxx="ext-link">
                  <b>Supporting Information—Fig. S2</b>
               </div>). In fact, few of the clusters found on the NJ tree of the ITS2 sequences 
               <b>[see 
                  <div tagxxx="ext-link">Supporting Information—</div>
                  <div tagxxx="ext-link">Fig. S2</div>]
               </b> contain only a single species, even though these clusters may comprise taxa belonging
               to the same series. We can offer no explanation for the examples described above of
               disparate taxa forming small, well-defined clades. On the other hand, the way in which
               some tetraploid individuals from section 
               <i>Douglasia</i> are found interspersed among individuals from section 
               <i>Coccineae</i> in this tree 
               <b>[see 
                  <div tagxxx="ext-link">Supporting Information—</div>
                  <div tagxxx="ext-link">Fig. S2</div>]
               </b> is undoubtedly related to the way the 
               <i>Douglasia</i> taxa concerned were found by 
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. (2014)</a>
                  </sup>
               </span> to comprise individuals containing both 
               <i>Coccineae</i> and 
               <i>Douglasia</i> ITS2 sequences.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The problem associated with ITS2 as a barcode marker stems partly from the limited
               sequence variation that this locus exhibits in 
               <i>Crataegus</i> (
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>), and partly from the incomplete homogenization of this locus by concerted evolution
               (
               <span ref-type="bibr" rid="PLV045C4">
                  <sup>
                     <a href="#PLV045C4">Arnheim 1983</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>). The rate at which molecular forces homogenize ITS2 is evidently lower than that
               at which hybridization and polyploidization have added ribotypes to an individual's
               genome (in a context where gametophytic apomixis produces unreduced female gametes
               which then undergo either parthenogenesis or fertilization; 
               <span ref-type="bibr" rid="PLV045C93">
                  <sup>
                     <a href="#PLV045C93">Talent and Dickinson 2005</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="PLV045C94">
                  <sup>
                     <a href="#PLV045C94">2007a</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="PLV045C95">
                  <sup>
                     <a href="#PLV045C95">b)</a>
                  </sup>
               </span>. Similar results are seen with low-copy number nuclear genes like AT1 
               <b>[see 
                  <div tagxxx="ext-link">Supporting Information</div>
                  <div tagxxx="ext-link">—Fig. S3</div>]
               </b> and PEPC (
               <div tagxxx="ext-link">
                  <b>Supporting Information—Fig. S4</b>
               </div>
               <b>and File S1</b>). Neither of these issues has been considered in recent evaluations of ITS2 as a
               suitable DNA barcode marker in medicinal plants (
               <span ref-type="bibr" rid="PLV045C13">
                  <sup>
                     <a href="#PLV045C13">Chen et al. 2010</a>
                  </sup>
               </span>) or in the Rosaceae (
               <span ref-type="bibr" rid="PLV045C70">
                  <sup>
                     <a href="#PLV045C70">Pang et al. 2011</a>
                  </sup>
               </span>). In fact, 
               <span ref-type="bibr" rid="PLV045C13">
                  <sup>
                     <a href="#PLV045C13">Chen et al. (2010)</a>
                  </sup>
               </span> did not sequence 
               <i>Crataegus</i> ITS2 themselves, relying instead on sequences deposited in GenBank by E. Y. Y. Lo
               (
               <div tagxxx="ext-link">Table S5</div> in 
               <span ref-type="bibr" rid="PLV045C13">
                  <sup>
                     <a href="#PLV045C13">Chen et al. 2010)</a>
                  </sup>
               </span>. 
               <span ref-type="bibr" rid="PLV045C70">
                  <sup>
                     <a href="#PLV045C70">Pang et al. (2011</a>
                  </sup>
               </span>, their 
               <div tagxxx="ext-link">Table S1</div>), only sequenced ITS2 in three varieties of 
               <i>C. pinnatifida</i> and relied again almost exclusively on sequences deposited in GenBank by Lo (
               <div tagxxx="ext-link">Table S2</div> in 
               <span ref-type="bibr" rid="PLV045C70">
                  <sup>
                     <a href="#PLV045C70">Pang et al. 2011)</a>
                  </sup>
               </span>. In both these studies, only the BLAST- and distance-based methods of evaluation
               of 
               <span ref-type="bibr" rid="PLV045C81">
                  <sup>
                     <a href="#PLV045C81">Ross et al. (2008)</a>
                  </sup>
               </span> were used. Both these studies implicitly assumed a one-to-one correspondence between
               species and unique ITS2 sequences. Such a correspondence is necessary for the success
               of the BLAST- and distance-based methods of evaluation (
               <span ref-type="bibr" rid="PLV045C81">
                  <sup>
                     <a href="#PLV045C81">Ross et al. 2008</a>
                  </sup>
               </span>). These methods will break down when there is a one-to-many relationship between
               species and sequences, as is the case when not all of the possible ITS2 paralogs have
               been included in the reference dataset, and any given paralog may occur in more than
               one species. As we have shown elsewhere (
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>), the ITS sequences obtained in our earlier work (
               <span ref-type="bibr" rid="PLV045C60">
                  <sup>
                     <a href="#PLV045C60">Lo et al. 2007</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="PLV045C61">
                  <sup>
                     <a href="#PLV045C61">2009</a>
                  </sup>
               </span>) are likely to underestimate the intragenomic diversity at this locus in many 
               <i>Crataegus</i> taxa.
            </p>
         </div>
         <div id="s4e">
            <h3>Metabolomics data</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Relationships inferred from molecular data (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>) are also implied in the resemblances obtained with the small sample studied here
               for the flavonoid component of the metabolome (Fig. 
               <span ref-type="fig" rid="PLV045F2">2</span>). More metabolomics data from a more representative sample are needed, however, before
               we can tell whether these data track the relationships seen in 
               <i>Crataegus</i> molecular data or not, and whether chemical tests will be more useful than DNA barcode
               loci in discriminating hawthorn species used in NHPs.
            </p>
         </div>
         <div id="s4f">
            <h3>Limited utility of DNA barcode data for studies of Crataegus phylogeny</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Sequence data from DNA barcode loci have proved useful in clarifying relationships
               between recent 
               <i>Crataegus</i> hybrids and their parents where the latter were both diploids and from different
               clades (
               <span ref-type="bibr" rid="PLV045C15">
                  <sup>
                     <a href="#PLV045C15">Christensen et al. 2014</a>
                  </sup>
               </span>; cf. 
               <span ref-type="bibr" rid="PLV045C60">
                  <sup>
                     <a href="#PLV045C60">Lo et al. 2007</a>
                  </sup>
               </span>). For the variety of reasons described above, these data are nevertheless inadequate
               for either barcoding purposes or for revealing evolutionary relationships across the
               entire genus 
               <b>[see 
                  <div tagxxx="ext-link">Supporting Information—</div>
                  <div tagxxx="ext-link">Figs S1 and S2</div>]
               </b>. Plastid loci can be chosen much more strategically if appreciable phylogenetic resolution
               is sought from a limited number of loci (
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>). Alternatively, alignments of sequences of considerably more than just three plastid
               loci can be analysed together in order to obtain a useful degree of phylogenetic resolution.
               When this approach is taken (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>), the results robustly corroborate earlier work on the same sample (
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Comparison can also be made between our results and those of 
               <span ref-type="bibr" rid="PLV045C59">
                  <sup>
                     <a href="#PLV045C59">Lo and Donoghue (2012</a>
                  </sup>
               </span>; a study of intergeneric relationships in the Maleae) as they relate to 
               <i>Crataegus</i>. Their Fig. 1 is a tree based on ITS (cloned only when direct sequencing gave ambiguous
               nucleotides) plus the same 11 plastid loci studied here in addition to the barcoding
               loci 
               <i>rbcL</i>, 
               <i>mat</i>K and 
               <i>psb</i>A
               <i>-trn</i>H, for a larger sample of hawthorn species than we have used (7 of the 47 Crataegus
               accessions, and 20 of the 33 species in 
               <div tagxxx="ext-link">
                  <b>Supporting Information—Table S2</b>
               </div> are shared with the Lo and Donoghue study). Even after Lo and Donoghue removed the
               two genera responsible for a lack of congruence between their ITS and plastid data,
               their tree based on the combined datasets shows support values that are marginally
               lower than those obtained here, with just 14 plastid loci, for branches A, B, C and
               E (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>). In both their study and ours, it is noteworthy that 
               <i>C.</i> series 
               <i>Cerrones</i> (clade F, Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>) is sister to clade D, comprising both East Asian 
               <i>C.</i> section 
               <i>Sanguineae</i> (clades G
               <div tagxxx="sub">1</div> and G
               <div tagxxx="sub">2</div>, Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>) and North American 
               <i>C.</i> series 
               <i>Douglasianae</i> (clade H), suggesting that the origin of section 
               <i>Sanguineae</i> involved an east to west trans-Beringian migration, from western North America into
               eastern Asia. These results also warrant possibly recognizing the 
               <i>Cerrones</i> as a section, that is, at the same taxonomic rank as sections 
               <i>Douglasia</i> and 
               <i>Sanguineae</i>  (Fig. 
               <span ref-type="fig" rid="PLV045F1">1</span>) 
               <b>[see 
                  <div tagxxx="ext-link">Supporting Information—</div>
                  <div tagxxx="ext-link">Table S1</div>]
               </b>.
            </p>
         </div>
         <div id="s4g">
            <h3>Limited utility of DNA barcode data for Crataegus NHP development</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Various authors have pointed out that DNA barcoding will not be useful in particular
               plant groups. 
               <span ref-type="bibr" rid="PLV045C89">
                  <sup>
                     <a href="#PLV045C89">Spooner (2009)</a>
                  </sup>
               </span> examined the utility of barcoding in wild potatoes and concluded that plastid loci
               may lack the necessary polymorphism, while ITS may exhibit too much intraspecific
               variation, much as we have seen here. 
               <span ref-type="bibr" rid="PLV045C45">
                  <sup>
                     <a href="#PLV045C45">Hollingsworth et al. (2011)</a>
                  </sup>
               </span> list factors that will work against barcoding success: breeding system, hybridization,
               polyploidy, long generation times (or reduced mutation rates), narrow taxon concepts,
               species history and seed dispersal. Like many other genera in the Rosaceae, 
               <i>Crataegus</i> is a woody perennial that exhibits a nexus of frequent hybridization, production
               and fertilization of unreduced gametes, hence polyploidy and accompanying shifts to
               self compatibility (
               <span ref-type="bibr" rid="PLV045C25">
                  <sup>
                     <a href="#PLV045C25">Dickinson et al. 2007</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C44">
                  <sup>
                     <a href="#PLV045C44">Hojsgaard et al. 2014</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>). Arguably as a result of the patterns of morphological variation associated with
               frequent apomixis and selfing (
               <span ref-type="bibr" rid="PLV045C23">
                  <sup>
                     <a href="#PLV045C23">Dickinson and Phipps 1985</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C21">
                  <sup>
                     <a href="#PLV045C21">Dickinson 1986</a>
                  </sup>
               </span>; cf. 
               <span ref-type="bibr" rid="PLV045C62">
                  <sup>
                     <a href="#PLV045C62">Lo et al. 2010</a>
                  </sup>
               </span>), the (morphological) species concepts used in 
               <i>Crataegus</i> have been quite narrow, and in many cases do not likely reflect a high degree of
               genetic differentiation. Concerning species history, 
               <span ref-type="bibr" rid="PLV045C45">
                  <sup>
                     <a href="#PLV045C45">Hollingsworth et al. (2011)</a>
                  </sup>
               </span> refer to the effects of recent, rapid radiation on the one hand, and on the other
               hand, to the maintenance of genetic polymorphisms in large populations, as also contributing
               to the failure of DNA barcoding within a group. In the case of 
               <i>Crataegus</i>, it seems likely that many hybridization and polyploidization events have been relatively
               recent, post-Pleistocene (&lt;12 000 years) in any case, and possibly just in the last
               millennium, as a consequence of First Nation and then European land-clearing activities
               (
               <span ref-type="bibr" rid="PLV045C65">
                  <sup>
                     <a href="#PLV045C65">Marie-Victorin 1938</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C26">
                  <sup>
                     <a href="#PLV045C26">Dickinson et al. 2008</a>
                  </sup>
               </span>).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In fact, the morphology visible in hawthorn flowering and fruiting voucher specimens
               is adequate for identification of most western North American species (
               <span ref-type="bibr" rid="PLV045C22">
                  <sup>
                     <a href="#PLV045C22">Dickinson 2012</a>
                  </sup>
               </span>) because, being long-lived woody perennials, wild hawthorns are readily marked and
               vouchered at both seasons for comparison (
               <span ref-type="bibr" rid="PLV045C43">
                  <sup>
                     <a href="#PLV045C43">Hildreth et al. 2007</a>
                  </sup>
               </span>). It is also significant that in this geographic area there are only a limited number
               of common species, almost all of which have now been included in flow cytometric surveys
               of ploidy level and breeding system (
               <div tagxxx="ext-link">
                  <b>Supporting Information—Tables S1 and S2</b>
               </div>; 
               <span ref-type="bibr" rid="PLV045C93">
                  <sup>
                     <a href="#PLV045C93">Talent and Dickinson 2005</a>
                  </sup>
               </span>; Talent unpubl. data). In contrast, the 
               <i>Crataegus</i> flora of eastern North America is much richer, and is less well represented in molecular
               phylogenetic studies (
               <span ref-type="bibr" rid="PLV045C61">
                  <sup>
                     <a href="#PLV045C61">Lo et al. 2009</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C113">
                  <sup>
                     <a href="#PLV045C113">Zarrei et al. 2014</a>
                  </sup>
               </span>). As a consequence, only a few groups can be considered even moderately well-characterized
               [e.g. series 
               <i>Aestivales</i> and 
               <i>Brevispinae</i>, and the North American series assigned to section 
               <i>Crataegus</i> by 
               <span ref-type="bibr" rid="PLV045C71">
                  <sup>
                     <a href="#PLV045C71">Phipps (2015)</a>
                  </sup>
               </span>]. More data are available from flow cytometry, and these have informed regional studies
               (
               <span ref-type="bibr" rid="PLV045C56">
                  <sup>
                     <a href="#PLV045C56">Lance 2014</a>
                  </sup>
               </span>), but coverage is not yet complete. Fortunately, a comprehensive floristic treatment
               of North American 
               <i>Crataegus</i>, complete with extensive synonymies, is available now as a part of the 
               <i>Flora North America</i> project (
               <span ref-type="bibr" rid="PLV045C71">
                  <sup>
                     <a href="#PLV045C71">Phipps 2015</a>
                  </sup>
               </span>). By providing testable taxonomic hypotheses, this treatment should stimulate the
               collection and analysis of new data with which to make sense of the taxonomic complexity
               that has been the hallmark of eastern North American hawthorns for over 100 years.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Production of hawthorn NHPs that employ native North American 
               <i>Crataegus</i> species as raw materials thus should not be impeded by an inability to employ DNA
               barcoding to authenticate source taxa. Data on the predominant agamospermy of polyploid
               
               <i>Crataegus</i> (all western species except for two; 
               <span ref-type="bibr" rid="PLV045C94">
                  <sup>
                     <a href="#PLV045C94">Talent and Dickinson 2007a</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="PLV045C95">
                  <sup>
                     <a href="#PLV045C95">b</a>
                  </sup>
               </span>) suggest that orchards raised from seeds or developed by grafting from previously
               identified and vouchered genotypes can be harvested over multiple years, thus minimizing
               the need for raw material authentication. Limited trials in an animal model of human
               metabolic syndrome of hawthorn preparations from native 
               <i>C. chrysocarpa</i> fruit and BC-grown 
               <i>C. monogyna</i> leaves have shown significant improvements over untreated controls (F. Borthwick
               
               <i>et al.</i>, unpubl. data—presented in part as 
               <span ref-type="bibr" rid="PLV045C114">
                  <sup>
                     <a href="#PLV045C114">Dickinson et al. 2014</a>
                  </sup>
               </span>). In addition to information on the taxonomy and phylogeny of hawthorns, information
               on hawthorn chemistry, analytical methods and the validation of those methods are
               increasingly available (
               <span ref-type="bibr" rid="PLV045C50">
                  <sup>
                     <a href="#PLV045C50">Kirakosyan et al. 2005</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="PLV045C32">
                  <sup>
                     <a href="#PLV045C32">Edwards et al. 2012</a>
                  </sup>
               </span>). Discovery of metabolomic variation between hawthorn species suggests the possibility
               that different therapeutic outcomes may be obtained when North American NHPs are administered
               rather than the more ubiquitous hawthorn NHPs originating from European species.
            </p>
         </div>
      </div>
      <div id="s5">
         <h2>Conclusions</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We have examined the utility of DNA barcoding in a sample of 355 accessions representing
            93 mostly species-level taxa from all major clades known to date, in a moderately
            large plant genus, 
            <i>Crataegus</i>. Our sequence data and voucher information (including specimen images) represent
            well-studied authoritatively identified individuals and are publicly available on
            the BOLD website (dx.doi.org/10.5883/DS-NAMCRAT; sequence data also on GenBank, 
            <div tagxxx="ext-link">
               <b>Supporting Information—Tables S1–S4</b>
            </div>) where they can be consulted, downloaded and reanalysed. The preliminary results
            from the small metabolomics dataset studied here show the promise of NMR chemotaxonomic
            data for studies of 
            <i>Crataegus</i> in relation to variation in therapeutic applications or NHP raw material authentication.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Our analyses of these sequence data from three plastid loci and ITS2 generally failed
            to recover either the cladistic structure or the morphology-based infrageneric classification
            of 
            <i>Crataegus</i>. We attribute this result to the lack of variation within Rosaceae tribe Maleae in
            the plastid loci chosen as barcodes. The use of biparentally inherited ITS2 as a DNA
            barcode is confounded by the frequency of allopolyploidy in 
            <i>Crataegus</i> combined with incomplete homogenization of this locus by concerted evolution. Incomplete
            concerted evolution of this kind leading to the presence of multiple gene copies is
            also common in the human genome (
            <span ref-type="bibr" rid="PLV045C116">
               <sup>
                  <a href="#PLV045C116">Zarrei et al. 2015</a>
               </sup>
            </span>).
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In 
            <i>Crataegus</i>, resolution of species or of groups of closely related species, depends on having
            more phylogenetically informative sites than can be provided by a small number of
            plastid loci. Relatively well-resolved phylogenetic analyses required concatenated
            sequences from a total of 14 plastid loci (Fig. 
            <span ref-type="fig" rid="PLV045F1">1</span>). High-resolution phylogenetic analyses based on the nuclear genome will require
            data from very low-copy number loci, the paralogs of which are readily identifiable.
            Since phenolic compounds, widely purported to be beneficial in cardiovascular health,
            vary in composition between species, there is considerable potential for using this
            chemical information to choose an optimal species for NHP formulation. However, development
            of hawthorn NHPs using North American species will require a taxonomy bolstered by
            molecular data that are interpretable in the field using morphological characters.
            This is now largely available for western North America, but remains a challenge in
            the much more complex 
            <i>Crataegus</i> flora of eastern North America.
         </p>
      </div>
      <div id="s6">
         <h2>Accession Numbers</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">See 
            <div tagxxx="ext-link">
               <b>Supporting Information—Tables S1–S4</b>
            </div> for GenBank accession numbers for the DNA sequences studied here.
         </p>
      </div>
      <div id="s7">
         <h2>Sources of Funding</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We gratefully acknowledge the 
            <div tagxxx="funding-source">Canada Foundation for Innovation</div> and the 
            <div tagxxx="funding-source">Ontario Research Fund</div> for supporting the Canadensys infrastructure that enabled TRT to database, image
            and curate voucher specimens for this project. Part of the laboratory work for this
            project was undertaken at the Canadian Centre of DNA Barcoding at Guelph where DNA
            barcoding was funded by the Government of Canada through Genome Canada and the 
            <div tagxxx="funding-source">Ontario Genomics Institute</div> (
            <div tagxxx="award-id">2008-OGI-ICI-03</div>). Fieldwork, and labwork in Toronto and Kelowna, were supported by a 
            <div tagxxx="funding-source">Natural Sciences and Engineering Research Council of Canada Strategic Research Project
               grant
            </div>, ‘Hawthorn as a new Canadian agroforestry crop for functional foods and nutraceuticals,’
            STPGP 381073-09 to T.A.D., S.S., P.R.S. and Spencer D. Proctor, who is a co-PI on
            the Strategic Research Project grant, but was not involved in the research that resulted
            in this manuscript. The generous support of the Louise Hawley Stone Charitable Trust
            enabled the Royal Ontario Museum Green Plant Herbarium to accept the 
            <div tagxxx="funding-source">University of Western Ontario's</div> gift of the J.B. Phipps Hawthorn Research Collection. Additional support from the
            
            <div tagxxx="funding-source">ROM Reproductions, Acquisitions, and Research Fund</div> and the 
            <div tagxxx="funding-source">Department of Natural History</div>, as well as from the Department of Ecology and Evolutionary Biology at the 
            <div tagxxx="funding-source">University of Toronto</div> is also gratefully acknowledged.
         </p>
      </div>
      <div id="s8">
         <h2>Contributions by the Authors</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">M.Z. designed the initial sampling, and carried out the sequencing and data analyses
            represented in Fig. 
            <span ref-type="fig" rid="PLV045F1">1</span> and 
            <div tagxxx="ext-link">
               <b>Supporting Information Figs S1–S4</b>
            </div>. N.T. carried out most of the flow cytometric analyses and curates these data. M.K.
            sampled additional individuals and carried out all of the DNA barcode sequencing.
            J. Lee took part in fieldwork and record-keeping for the project. J. Lund and P.R.S.
            collected and analysed the metabolomics data used here. S.S. and T.A.D. advised on
            sampling and data analysis. The manuscript was prepared by M.Z. and T.A.D., based
            on contributions from, and revisions by, all of the authors.
         </p>
      </div>
      <div id="s9">
         <h2>Conflict of Interest Statement</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">J. Lee is on the board of directors of the Naturally Grown Herb &amp; Spice Producers
            Cooperative (HerbPro), and chairs their value added committee that is focussed on
            development of hawthorn-based products.
         </p>
      </div>
      <div id="s10">
         <h2>Supporting Information</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The following additional information is available in the online version of this article
            –
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
            <b>Table S1.</b> Voucher information and GenBank accession numbers for the three plastid DNA barcode
            loci, and nrITS2 amplified directly from genomic DNA as a DNA barcode locus, used
            in the current study (
            <div tagxxx="ext-link">
               <b>Supporting Information—Figs S1 and S2</b>
            </div>; BOLD Dataset NAMCRAT; further details of the barcode sequences are available at
            dx.doi.org/10.5883/DS-NAMCRAT).
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
            <b>Table S2.</b> Phylogenetic analysis of 14 
            <i>Crataegus</i> plastid DNA loci (Fig. 
            <span ref-type="fig" rid="PLV045F1">1</span>). Voucher information and GenBank accession numbers for 11 supplementary plastid
            DNA loci and the three plastid DNA barcode loci (
            <div tagxxx="ext-link">
               <b>Supporting Information—Table S1</b>
            </div>; further details of the barcode sequences are available at dx.doi.org/10.5883/DS-NAMCRAT).
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
            <b>Table S3.</b>
            <i>Crataegus</i> voucher information and GenBank accession numbers for the AT1 region used in the
            current study.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
            <b>Table S4.</b>
            <i>Crataegus</i> voucher information and GenBank accession numbers for the PEPC region used in the
            current study.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
            <b>Table S5.</b> Models of nucleotide evolution in 
            <i>Crataegus</i> selected using the AIC for analyses of plastid (Fig. 
            <span ref-type="fig" rid="PLV045F1">1</span>; 
            <div tagxxx="ext-link">
               <b>Supporting Information—Table S2</b>
            </div>) and nuclear 
            <b>[see 
               <div tagxxx="ext-link">Supporting Information</div>
               <div tagxxx="ext-link">—Figs S3 and S4</div>
            </b>; 
            <div tagxxx="ext-link">
               <b>Supporting Information—Tables S3 and S4</b>
            </div>
            <b>]</b> markers.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
            <b>Table S6.</b> Comparison of 
            <i>Crataegus</i> sequence variation between different markers utilized for the phylogenetic analyses
            (Fig. 
            <span ref-type="fig" rid="PLV045F1">1</span>; 
            <div tagxxx="ext-link">
               <b>Supporting Information—Figs S3 and S4</b>
            </div>).
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
            <b>File S1; AT1 and PEPC Supporting Information.</b> Low-copy nuclear gene methods, data analyses and results.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
            <b>Figure S1.</b> Neighbour-Joining tree for three concatenated 
            <i>Crataegus</i> plastid DNA barcode markers (
            <i>mat</i>K, 
            <i>rbcL</i>a and 
            <i>psb</i>A
            <i>-trn</i>H). The numbers preceding the taxonomic information are 
            <i>Crataegus</i> sample identifiers (
            <div tagxxx="ext-link">
               <b>Supporting Information—Table S1</b>
            </div>; BOLD Dataset NAMCRAT). Ploidy level for the vouchered individual is given where
            known (in parentheses, if voucher information unavailable, but species has a single
            characteristic ploidy level). Scale for branch lengths in substitutions per site.
            Leaves are coloured to show 
            <i>Crataegus</i> infrageneric classification 
            <b>[see</b>
            <div tagxxx="ext-link">
               <b>Supporting Information—Table S1</b>
            </div>
            <b>]</b>, with labelling of clades following that in 
            <span ref-type="bibr" rid="PLV045C60">
               <sup>
                  <a href="#PLV045C60">Lo et al. (2007)</a>
               </sup>
            </span>: blue rectangles, section 
            <i>Mespilus</i> (clade A
            <div tagxxx="sub">1</div>); blue text, section 
            <i>Brevispinae</i> (clade A
            <div tagxxx="sub">2</div>); red rectangles, section 
            <i>Crataegus</i> (clade B); orange text, sections 
            <i>Coccineae</i> and 
            <i>Macracanthae</i> (clade D); purple rectangles, series 
            <i>Cerrones</i> (clade E
            <div tagxxx="sub">1</div>); purple text, series 
            <i>Douglasianae</i> (clade E
            <div tagxxx="sub">2</div>) and green rectangles, section 
            <i>Sanguineae</i> (clade E
            <div tagxxx="sub">3</div>). Clade C as described earlier (
            <span ref-type="bibr" rid="PLV045C60">
               <sup>
                  <a href="#PLV045C60">Lo et al. 2007</a>
               </sup>
            </span>), and in the text.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
            <b>Figure S2.</b> Neighbour-Joining tree for ITS2 amplified directly from 
            <i>Crataegus</i> genomic DNA as a DNA barcode locus. The numbers preceding the taxonomic information
            are 
            <i>Crataegus</i> sample identifiers (
            <div tagxxx="ext-link">
               <b>Supporting Information—Table S1</b>
            </div>; BOLD Dataset NAMCRAT). Ploidy level for the vouchered individual is given where
            known (in parentheses, if voucher information unavailable, but species has a single
            characteristic ploidy level). This figure includes sequences (filled diamonds) from
            four accessions of 
            <i>C. douglasii</i> (TRT157, 175, 177 and 184; 
            <div tagxxx="ext-link">
               <b>Supporting Information—Table S1</b>
            </div>), two of 
            <i>C. orbicularis</i> (MKTRT587, 588; 
            <div tagxxx="ext-link">
               <b>Supporting Information—Table S1</b>
            </div>) and one of 
            <i>C. sheila-phippsiae</i> (MKTRT617; 
            <div tagxxx="ext-link">
               <b>Supporting Information—Table S1</b>
            </div>) with sequences that were not barcode compliant, but nevertheless could be fitted
            into an alignment. Scale for branch lengths in substitutions per site. Leaves are
            coloured, and clades are labelled, as described for 
            <div tagxxx="ext-link">
               <b>Supporting Information—Fig. S1</b>
            </div>.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
            <b>Figure S3</b>. The Bayesian phylogram for AT1, the PPR homologue of the AT1G09680 gene in 
            <i>Arabidopsis</i>. Branch support values are indicated as PP above branches, and bootstrap (BS) values
            below branches. Branches with PP &lt;0.5 are shown as polytomies. BS values &lt;50 % are
            not shown. Asterisks indicate branches retained in the strict consensus of 243470
            trees. Scale for branch lengths in substitutions per site. The numbers preceding the
            taxonomic information are 
            <i>Crataegus</i> sample identifiers 
            <b>[see 
               <div tagxxx="ext-link">Supporting Information—</div>
               <div tagxxx="ext-link">Table S3</div>]
            </b> and clone numbers (separated by a hyphen). Clades a–d are described in the 
            <div tagxxx="ext-link">
               <b>Supporting Information Text</b>
            </div>
            <b>—File S1</b>. Leaves are coloured as described for 
            <div tagxxx="ext-link">
               <b>Supporting Information—Fig. S1</b>
            </div>.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
            <b>Figure S4</b>. The Bayesian phylogram for PEPC sequences. Branch support values are indicated as
            PP above branches. Branches with PP &lt;0.5 are shown as polytomies (PP values &lt;0.7 are
            not shown). Asterisks indicate branches retained in the strict consensus of 102 420
            trees. Leaves are coloured as in 
            <div tagxxx="ext-link">
               <b>Supporting Information—Fig. S3</b>
            </div>. Scale for branch lengths in substitutions per site. S, short paralog; L, long paralog.
            The numbers preceding the taxonomic information are 
            <i>Crataegus</i> sample identifiers 
            <b>[see 
               <div tagxxx="ext-link">Supporting Information—</div>
               <div tagxxx="ext-link">Table S4</div>]
            </b> and clone numbers (separated by hyphen). Clade a is described in the 
            <div tagxxx="ext-link">
               <b>Supporting Information Text—</b>
            </div>
            <b>File S1</b>. Leaves are coloured as described for 
            <div tagxxx="ext-link">
               <b>Supporting Information—Fig. S1</b>
            </div>.
         </p>
         <div tagxxx="supplementary-material">
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Additional Information</div>
            </caption>
            <div tagxxx="media"></div>
            <div tagxxx="media"></div>
            <div tagxxx="media"></div>
            <div tagxxx="media"></div>
            <div tagxxx="media"></div>
            <div tagxxx="media"></div>
            <div tagxxx="media"></div>
            <div tagxxx="media"></div>
            <div tagxxx="media"></div>
            <div tagxxx="media"></div>
            <div tagxxx="media"></div>
            <div tagxxx="media"></div>
         </div>
      </div>
      <div tagxxx="back">
         <div tagxxx="ack">
            <div tagxxx="title">Acknowledgements</div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We thank A. Anderson, the late K. I. Christensen, G. Cooley, J. M. Coughlan, C. Delong,
               A. A. Dönmez, W. Earle, M. Graham, J. Herrick, B. Jensen, R. and M. Johnson, R. W.
               Lance, T. Lewis, E. Y. Y. Lo, R. M. Love, H. Massicotte and L. Tackaberry, H. Moothoo,
               S. Nguyen, R. and J. Pojar, G. Powell, B. Rogers, T. Romankova, C. Shaw, D. Smith,
               T. Steen, R. Thompson, P. Zika and others mentioned in 
               <div tagxxx="ext-link">
                  <b>Supporting Information—Tables S1–S4</b>
               </div> for assistance with plant collecting. The following organizations kindly provided
               us with access to plant material growing under their care, or on their property: the
               Arkansas Natural Heritage Commission; the Arnold Arboretum of Harvard University;
               the Deer Creek Center of the Siskiyou Field Institute; the Jardin Botanique de Montréal,
               the Morton Arboretum; the Nature Conservancy; Point Reyes National Seashore and the
               University of California Botanical Garden, Berkeley. We are indebted to J. M. Coughlan
               for sharing her flow cytometry data with us. James B. Phipps kindly identified several
               of our 2007 and earlier collections. For assistance in the herbarium, managing the
               voucher material, we are indebted to K. Buck, R. Martin and C. Ng. Dan Potter and
               an anonymous reviewer provided many suggestions that greatly improved portions of
               the text. This work could not have been completed without the help of A. Rodrigues,
               T. Braukmann, K. Choffe and O. Haddrath. Paula Brown (British Columbia Institute of
               Technology) was instrumental in helping us obtain the NSERC Strategic Research Project
               grant. Her lab at BCIT was responsible for storing and curating collections of fruits
               and leaves made for this project.
            </p>
         </div>
         <div tag="ref-list">
            <ul>
               <div tagxxx="title">Literature Cited</div>
               <li tag="ref">
                  <a name="PLV045C1"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Agnarsson</span>
                           <span tagx="given-names">I</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Miller</span>
                           <span tagx="given-names">JA</span>,
                        </span>
                     </span>
                     <span tagx="year">2008</span>
                     <span tagx="article-title">Is ACCTRAN better than DELTRAN?</span>
                     <span tagx="source">Cladistics</span>
                     <span tagx="volume">24</span>:
                     <span tagx="fpage">1</span>–
                     <span tagx="lpage">7</span>. 
                     <div tagxxx="ext-link">doi:10.1111/j.1096-0031.2008.00229.x</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C2"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Akaike</span>
                           <span tagx="given-names">H</span>,
                        </span>
                     </span>
                     <span tagx="year">1974</span>
                     <span tagx="article-title">A new look at the statistical model identification</span>. 
                     <span tagx="source">IEEE Transactions on Automatic Control</span>
                     <span tagx="volume">19</span>:
                     <span tagx="fpage">716</span>–
                     <span tagx="lpage">723</span>. 
                     <div tagxxx="ext-link">doi:10.1109/TAC.1974.1100705</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C3"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Arnason</span>
                           <span tagx="given-names">T</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Hebda</span>
                           <span tagx="given-names">RJ</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Johns</span>
                           <span tagx="given-names">T</span>,
                        </span>
                     </span>
                     <span tagx="year">1981</span>
                     <span tagx="article-title">Use of plants for food and medicine by native peoples of eastern Canada</span>. 
                     <span tagx="source">Canadian Journal of Botany</span>
                     <span tagx="volume">59</span>:
                     <span tagx="fpage">2189</span>–
                     <span tagx="lpage">2325</span>. 
                     <div tagxxx="ext-link">doi:10.1139/b81-287</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C4"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Arnheim</span>
                           <span tagx="given-names">N</span>,
                        </span>
                     </span>
                     <span tagx="year">1983</span>
                     <span tagx="article-title">Concerted evolution of multigene families</span>. In: 
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Nei</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Koehn</span>
                           <span tagx="given-names">R</span>,
                        </span>
                     </span>, eds. 
                     <span tagx="source">Evolution of genes and proteins</span>. 
                     <span class="publisher-loc'">[Sunderland, MA]</span>: 
                     <span class="publisher-name'">[Sinauer]</span>, 
                     <span tagx="fpage">38</span>–
                     <span tagx="lpage">61</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C5"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Benson</span>
                           <span tagx="given-names">DA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Karsch-Mizrachi</span>
                           <span tagx="given-names">I</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Lipman</span>
                           <span tagx="given-names">DJ</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Ostell</span>
                           <span tagx="given-names">J</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Sayers</span>
                           <span tagx="given-names">EW</span>,
                        </span>
                     </span>
                     <span tagx="year">2011</span>
                     <span tagx="article-title">GenBank</span>. 
                     <span tagx="source">Nucleic Acids Research</span>
                     <span tagx="volume">39</span>:
                     <span tagx="fpage">D32</span>–
                     <span tagx="lpage">D37</span>. 
                     <div tagxxx="ext-link">doi:10.1093/nar/gkq1079</div>
                     <span class="pub-id'">[21071399]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C6"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Brouillet</span>
                           <span tagx="given-names">L</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Coursol</span>
                           <span tagx="given-names">F</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Meades</span>
                           <span tagx="given-names">SJ</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Favreau</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Anions</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Bélisle</span>
                           <span tagx="given-names">P</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Desmet</span>
                           <span tagx="given-names">P</span>,
                        </span>
                     </span>
                     <span tagx="year">2010+</span>
                     <span tagx="article-title">Crataegus 
                        <i>L. in VASCAN, the database of vascular plants of Canada</i>
                     </span>. 
                     <div tagxxx="comment">
                        <div tagxxx="uri">http://data.canadensys.net/vascan/taxon/1047</div>
                     </div> (
                     <div tagxxx="comment">28 August 2014</div>).
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C7"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Bull</span>
                           <span tagx="given-names">JJ</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Huelsenbeck</span>
                           <span tagx="given-names">JP</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Cunningham</span>
                           <span tagx="given-names">CW</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Swofford</span>
                           <span tagx="given-names">DL</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Waddell</span>
                           <span tagx="given-names">PJ</span>,
                        </span>
                     </span>
                     <span tagx="year">1993</span>
                     <span tagx="article-title">Partitioning and combining data in phylogenetic analysis</span>. 
                     <span tagx="source">Systematic Biology</span>
                     <span tagx="volume">42</span>:
                     <span tagx="fpage">384</span>–
                     <span tagx="lpage">397</span>. 
                     <div tagxxx="ext-link">doi:10.1093/sysbio/42.3.384</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C8"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Burgess</span>
                           <span tagx="given-names">KS</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Fazekas</span>
                           <span tagx="given-names">AJ</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Kesanakurti</span>
                           <span tagx="given-names">PR</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Graham</span>
                           <span tagx="given-names">SW</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Husband</span>
                           <span tagx="given-names">BC</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Newmaster</span>
                           <span tagx="given-names">SG</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Percy</span>
                           <span tagx="given-names">DM</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Hajibabaei</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Barrett</span>
                           <span tagx="given-names">SCH</span>,
                        </span>
                     </span>
                     <span tagx="year">2011</span>
                     <span tagx="article-title">Discriminating plant species in a local temperate flora using the 
                        <i>rbcL</i>+
                        <i>mat</i>K DNA barcode
                     </span>. 
                     <span tagx="source">Methods in Ecology and Evolution</span>
                     <span tagx="volume">2</span>:
                     <span tagx="fpage">333</span>–
                     <span tagx="lpage">340</span>. 
                     <div tagxxx="ext-link">doi:10.1111/j.2041-210X.2011.00092.x</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C9"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Campbell</span>
                           <span tagx="given-names">CS</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Evans</span>
                           <span tagx="given-names">RC</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Morgan</span>
                           <span tagx="given-names">DR</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Arsenault</span>
                           <span tagx="given-names">MP</span>,
                        </span>
                     </span>
                     <span tagx="year">2007</span>
                     <span tagx="article-title">Phylogeny of subtribe Pyrinae (formerly the Maloideae, Rosaceae): limited resolution
                        of a complex evolutionary history
                     </span>. 
                     <span tagx="source">Plant Systematics and Evolution</span>
                     <span tagx="volume">266</span>:
                     <span tagx="fpage">119</span>–
                     <span tagx="lpage">145</span>. 
                     <div tagxxx="ext-link">doi:10.1007/s00606-007-0545-y</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C10"></a>
                  <div tagxxx="mixed-citation">
                     <span class="collab'">[CBOL Plant Working Group]</span>. 
                     <span tagx="year">2009</span>
                     <span tagx="article-title">A DNA barcode for land plants</span>. 
                     <span tagx="source">Proceedings of the National Academy of Sciences of the USA</span>
                     <span tagx="volume">106</span>:
                     <span tagx="fpage">12794</span>–
                     <span tagx="lpage">12797</span>. 
                     <div tagxxx="ext-link">doi:10.1073/pnas.0905845106</div>
                     <span class="pub-id'">[19666622]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C11"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Chase</span>
                           <span tagx="given-names">MW</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Salamin</span>
                           <span tagx="given-names">N</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Wilkinson</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Dunwell</span>
                           <span tagx="given-names">JM</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Kesanakurthi</span>
                           <span tagx="given-names">RP</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Haidar</span>
                           <span tagx="given-names">N</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Savolainen</span>
                           <span tagx="given-names">V</span>,
                        </span>
                     </span>
                     <span tagx="year">2005</span>
                     <span tagx="article-title">Land plants and DNA barcodes: short-term and long-term goals</span>. 
                     <span tagx="source">Philosophical Transactions of the Royal Society B: Biological Sciences</span>
                     <span tagx="volume">360</span>:
                     <span tagx="fpage">1889</span>–
                     <span tagx="lpage">1895</span>. 
                     <div tagxxx="ext-link">doi:10.1098/rstb.2005.1720</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C12"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Chase</span>
                           <span tagx="given-names">MW</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Cowan</span>
                           <span tagx="given-names">RS</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Hollingsworth</span>
                           <span tagx="given-names">PM</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">van den Berg</span>
                           <span tagx="given-names">C</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Madriñán</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Petersen</span>
                           <span tagx="given-names">G</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Seberg</span>
                           <span tagx="given-names">O</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Jørgsensen</span>
                           <span tagx="given-names">T</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Cameron</span>
                           <span tagx="given-names">KM</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Carine</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Pedersen</span>
                           <span tagx="given-names">N</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Hedderson</span>
                           <span tagx="given-names">TAJ</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Conrad</span>
                           <span tagx="given-names">F</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Salazar</span>
                           <span tagx="given-names">GA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Richardson</span>
                           <span tagx="given-names">JE</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Hollingsworth</span>
                           <span tagx="given-names">ML</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Barraclough</span>
                           <span tagx="given-names">TG</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Kelly</span>
                           <span tagx="given-names">L</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Wilkinson</span>
                           <span tagx="given-names">M</span>,
                        </span>
                     </span>
                     <span tagx="year">2007</span>
                     <span tagx="article-title">A proposal for a standardised protocol to barcode all land plants</span>. 
                     <span tagx="source">Taxon</span>
                     <span tagx="volume">56</span>:
                     <span tagx="fpage">295</span>–
                     <span tagx="lpage">299</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C13"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Chen</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Yao</span>
                           <span tagx="given-names">H</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Han</span>
                           <span tagx="given-names">J</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Liu</span>
                           <span tagx="given-names">C</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Song</span>
                           <span tagx="given-names">J</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Shi</span>
                           <span tagx="given-names">L</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Zhu</span>
                           <span tagx="given-names">Y</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Ma</span>
                           <span tagx="given-names">X</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Gao</span>
                           <span tagx="given-names">T</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Pang</span>
                           <span tagx="given-names">X</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Luo</span>
                           <span tagx="given-names">K</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Li</span>
                           <span tagx="given-names">Y</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Li</span>
                           <span tagx="given-names">X</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Jia</span>
                           <span tagx="given-names">X</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Lin</span>
                           <span tagx="given-names">Y</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Leon</span>
                           <span tagx="given-names">C</span>,
                        </span>
                     </span>
                     <span tagx="year">2010</span>
                     <span tagx="article-title">Validation of the ITS2 region as a novel DNA barcode for identifying medicinal plant
                        species
                     </span>. 
                     <span tagx="source">PLoS ONE</span>
                     <span tagx="volume">5</span>:
                     <span tagx="fpage">e8613</span>
                     <div tagxxx="ext-link">doi:10.1371/journal.pone.0008613</div>
                     <span class="pub-id'">[20062805]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C14"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Christensen</span>
                           <span tagx="given-names">KI</span>,
                        </span>
                     </span>
                     <span tagx="year">1992</span>
                     <span tagx="article-title">Revision of 
                        <i>Crataegus</i> sect. 
                        <i>Crataegus</i> and nothosect. 
                        <i>Crataeguineae</i> (Rosaceae-Maloideae) in the Old World
                     </span>. 
                     <span tagx="source">Systematic Botany Monographs</span>
                     <span tagx="volume">35</span>:
                     <span tagx="fpage">1</span>–
                     <span tagx="lpage">199</span>. 
                     <div tagxxx="ext-link">doi:10.2307/25027810</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C15"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Christensen</span>
                           <span tagx="given-names">KI</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Zarrei</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Kuzmina</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Talent</span>
                           <span tagx="given-names">N</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Lin</span>
                           <span tagx="given-names">C</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>
                     </span>
                     <span tagx="year">2014</span>
                     <span tagx="article-title">
                        <i>Crataegus</i> × 
                        <i>ninae-celottiae</i> and 
                        <i>C</i>. × 
                        <i>cogswellii</i> (Rosaceae, Maleae), two spontaneously formed intersectional nothospecies
                     </span>. 
                     <span tagx="source">PhytoKeys</span>
                     <span tagx="volume">36</span>:
                     <span tagx="fpage">1</span>–
                     <span tagx="lpage">26</span>. 
                     <div tagxxx="ext-link">doi:10.3897/phytokeys.36.6784</div>
                     <span class="pub-id'">[24843290]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C16"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Clegg</span>
                           <span tagx="given-names">MT</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Gaut</span>
                           <span tagx="given-names">BS</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Learn</span>
                           <span tagx="given-names">GH</span>
                           <div tagxxx="suffix">Jr</div>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Morton</span>
                           <span tagx="given-names">BR</span>,
                        </span>
                     </span>
                     <span tagx="year">1994</span>
                     <span tagx="article-title">Rates and patterns of chloroplast DNA evolution</span>. 
                     <span tagx="source">Proceedings of the National Academy of Sciences of the USA</span>
                     <span tagx="volume">91</span>:
                     <span tagx="fpage">6795</span>–
                     <span tagx="lpage">6801</span>. 
                     <div tagxxx="ext-link">doi:10.1073/pnas.91.15.6795</div>
                     <span class="pub-id'">[8041699]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C17"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Corriveau</span>
                           <span tagx="given-names">JL</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Coleman</span>
                           <span tagx="given-names">AW</span>,
                        </span>
                     </span>
                     <span tagx="year">1988</span>
                     <span tagx="article-title">Rapid screening method to detect potential biparental inheritance of plastid DNA and
                        results for over 200 angiosperm species
                     </span>. 
                     <span tagx="source">American Journal of Botany</span>
                     <span tagx="volume">75</span>:
                     <span tagx="fpage">1443</span>–
                     <span tagx="lpage">1458</span>. 
                     <div tagxxx="ext-link">doi:10.2307/2444695</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C18"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Coughlan</span>
                           <span tagx="given-names">JM</span>,
                        </span>
                     </span>
                     <span tagx="year">2012</span>
                     <span tagx="article-title">Mechanisms for range size and distribution variation in polyploid complex black-fruited
                        hawthorn (Crataegus series Douglasianae): biogeographic implications for the maintenance
                        of cytotype diversity
                     </span>. 
                     <div tagxxx="comment">MSc Thesis</div>, 
                     <span class="publisher-name'">[University of Toronto]</span>, 
                     <span class="publisher-loc'">[Canada]</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C19"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Coughlan</span>
                           <span tagx="given-names">JM</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Stefanović</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>
                     </span>
                     <span tagx="year">2014</span>
                     <span tagx="article-title">Relative resource allocation to dispersal and competition demonstrates the putative
                        role of hybridity in geographical parthenogenesis
                     </span>. 
                     <span tagx="source">Journal of Biogeography</span>
                     <span tagx="volume">41</span>:
                     <span tagx="fpage">1603</span>–
                     <span tagx="lpage">1613</span>. 
                     <div tagxxx="ext-link">doi:10.1111/jbi.12316</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C20"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Devey</span>
                           <span tagx="given-names">DS</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Chase</span>
                           <span tagx="given-names">MW</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Clarkson</span>
                           <span tagx="given-names">JJ</span>,
                        </span>
                     </span>
                     <span tagx="year">2009</span>
                     <span tagx="article-title">A stuttering start to plant DNA barcoding: microsatellites present a previously overlooked
                        problem in non-coding plastid regions
                     </span>. 
                     <span tagx="source">Taxon</span>
                     <span tagx="volume">58</span>:
                     <span tagx="fpage">7</span>–
                     <span tagx="lpage">15</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C21"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>
                     </span>
                     <span tagx="year">1986</span>
                     <span tagx="article-title">Topodeme differentiation in Ontario taxa of 
                        <i>Crataegus</i> (Rosaceae: Maloideae): leaf morphometric evidence
                     </span>. 
                     <span tagx="source">Canadian Journal of Botany</span>
                     <span tagx="volume">64</span>:
                     <span tagx="fpage">2738</span>–
                     <span tagx="lpage">2747</span>. 
                     <div tagxxx="ext-link">doi:10.1139/b86-363</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C22"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>
                     </span>
                     <span tagx="year">2012</span>
                     <span tagx="article-title">Overview of BC hawthorns</span>. 
                     <div tagxxx="comment">
                        <div tagxxx="uri">http://labs.eeb.utoronto.ca/dickinson/BC_Crataegus/</div> (31 August 2014)
                     </div>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C23"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Phipps</span>
                           <span tagx="given-names">JB</span>,
                        </span>
                     </span>
                     <span tagx="year">1985</span>
                     <span tagx="article-title">Studies in 
                        <i>Crataegus</i> L. (Rosaceae: Maloideae). XIII. Degree and pattern of phenotypic variation in 
                        <i>Crataegus</i> sect. 
                        <i>Crus-galli</i> in Ontario
                     </span>. 
                     <span tagx="source">Systematic Botany</span>
                     <span tagx="volume">10</span>:
                     <span tagx="fpage">322</span>–
                     <span tagx="lpage">337</span>. 
                     <div tagxxx="ext-link">doi:10.2307/2418596</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C24"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Belaoussoff</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Love</span>
                           <span tagx="given-names">RM</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Muniyamma</span>
                           <span tagx="given-names">M</span>,
                        </span>
                     </span>
                     <span tagx="year">1996</span>
                     <span tagx="article-title">North American black-fruited hawthorns. I. Variation in floral construction, breeding
                        system correlates, and their possible evolutionary significance in 
                        <i>Crataegus</i> sect. 
                        <i>Douglasii</i> London
                     </span>. 
                     <span tagx="source">Folia Geobotanica and Phytotaxonomica</span>
                     <span tagx="volume">31</span>:
                     <span tagx="fpage">355</span>–
                     <span tagx="lpage">371</span>. 
                     <div tagxxx="ext-link">doi:10.1007/BF02815380</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C25"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Lo</span>
                           <span tagx="given-names">E</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Talent</span>
                           <span tagx="given-names">N</span>,
                        </span>
                     </span>
                     <span tagx="year">2007</span>
                     <span tagx="article-title">Polyploidy, reproductive biology, and Rosaceae: understanding evolution and making
                        classifications
                     </span>. 
                     <span tagx="source">Plant Systematics and Evolution</span>
                     <span tagx="volume">266</span>:
                     <span tagx="fpage">59</span>–
                     <span tagx="lpage">78</span>. 
                     <div tagxxx="ext-link">doi:10.1007/s00606-007-0541-2</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C26"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Lo</span>
                           <span tagx="given-names">EYY</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Talent</span>
                           <span tagx="given-names">N</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Love</span>
                           <span tagx="given-names">RM</span>,
                        </span>
                     </span>
                     <span tagx="year">2008</span>
                     <span tagx="article-title">Black-fruited hawthorns of western North America—one or more agamic complexes?</span>
                     <span tagx="source">Botany</span>
                     <span tagx="volume">86</span>:
                     <span tagx="fpage">846</span>–
                     <span tagx="lpage">865</span>. 
                     <div tagxxx="ext-link">doi:10.1139/B08-072</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C27"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Zarrei</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Kuzmina</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Stefanović</span>
                           <span tagx="given-names">S</span>,
                        </span>
                     </span>
                     <span tagx="year">2011</span>
                     <span tagx="article-title">Mixed success with DNA barcoding of hawthorns (
                        <i>Crataegus</i> L., Rosaceae)
                     </span>. 
                     <span tagx="source">Botany 2011. Healing the planet</span>. 
                     <span class="publisher-loc'">[St. Louis, MO]</span>
                     <div tagxxx="comment">Abstract 
                        <div tagxxx="uri">http://2011.botanyconference.org/engine/search/index.php?func=detail&amp;aid=720</div> (31 August 2014)
                     </div>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C114"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Proctor</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Shipley</span>
                           <span tagx="given-names">PR</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Lee</span>
                           <span tagx="given-names">J</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Coughlan</span>
                           <span tagx="given-names">J</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Zarrei</span>
                           <span tagx="given-names">M</span>,
                        </span>
                     </span>
                     <span tagx="year">2014</span>
                     <span tagx="article-title">Natural health products and 
                        <i>Crataegus</i> of the Pacific Northwest
                     </span>. In: 
                     <span tagx="source">Annual Meeting, Canadian Botanical Association</span>, 
                     <span class="publisher-loc'">[Montréal]</span>, 
                     <span class="publisher-name'">[Québec]</span>, 
                     <div tagxxx="comment">15–18 June 2014</div>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C28"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dong</span>
                           <span tagx="given-names">W</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Liu</span>
                           <span tagx="given-names">J</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Yu</span>
                           <span tagx="given-names">J</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Wang</span>
                           <span tagx="given-names">L</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Zhou</span>
                           <span tagx="given-names">S</span>,
                        </span>
                     </span>
                     <span tagx="year">2012</span>
                     <span tagx="article-title">Highly variable chloroplast markers for evaluating plant phylogeny at low taxonomic
                        levels and for DNA barcoding
                     </span>. 
                     <span tagx="source">PLoS ONE</span>
                     <span tagx="volume">7</span>:
                     <span tagx="fpage">e35071</span>
                     <div tagxxx="ext-link">doi:10.1371/journal.pone.0035071</div>
                     <span class="pub-id'">[22511980]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C29"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Doyle</span>
                           <span tagx="given-names">JJ</span>,
                        </span>
                     </span>
                     <span tagx="year">1992</span>
                     <span tagx="article-title">Gene trees and species trees: molecular systematics as one-character taxonomy</span>. 
                     <span tagx="source">Systematic Botany</span>
                     <span tagx="volume">17</span>:
                     <span tagx="fpage">144</span>–
                     <span tagx="lpage">163</span>. 
                     <div tagxxx="ext-link">doi:10.2307/2419070</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C30"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Drummond</span>
                           <span tagx="given-names">AJ</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Ashton</span>
                           <span tagx="given-names">B</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Buxton</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Cheung</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Cooper</span>
                           <span tagx="given-names">A</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Duran</span>
                           <span tagx="given-names">C</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Field</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Heled</span>
                           <span tagx="given-names">J</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Kearse</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Markowitz</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Moir</span>
                           <span tagx="given-names">R</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Stones-Havas</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Sturrock</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Thierer</span>
                           <span tagx="given-names">T</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Wilson</span>
                           <span tagx="given-names">A</span>,
                        </span>
                     </span>
                     <span tagx="year">2012</span>
                     <span tagx="article-title">Geneious v5.6</span>. 
                     <div tagxxx="comment">
                        <div tagxxx="uri">http://www.geneious.com</div> (15 April 2012)
                     </div>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C31"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Edgar</span>
                           <span tagx="given-names">RC</span>,
                        </span>
                     </span>
                     <span tagx="year">2004</span>
                     <span tagx="article-title">MUSCLE: a multiple sequence alignment method with reduced time and space complexity</span>. 
                     <span tagx="source">BMC Bioinformatics</span>
                     <span tagx="volume">5</span>:
                     <span tagx="fpage">113</span>
                     <div tagxxx="ext-link">doi:10.1186/1471-2105-5-113</div>
                     <span class="pub-id'">[15318951]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C32"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Edwards</span>
                           <span tagx="given-names">JE</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Brown</span>
                           <span tagx="given-names">PN</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Talent</span>
                           <span tagx="given-names">N</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Shipley</span>
                           <span tagx="given-names">PR</span>,
                        </span>
                     </span>
                     <span tagx="year">2012</span>
                     <span tagx="article-title">A review of the chemistry of the genus 
                        <i>Crataegus</i>
                     </span>. 
                     <span tagx="source">Phytochemistry</span>
                     <span tagx="volume">79</span>:
                     <span tagx="fpage">5</span>–
                     <span tagx="lpage">26</span>. 
                     <div tagxxx="ext-link">doi:10.1016/j.phytochem.2012.04.006</div>
                     <span class="pub-id'">[22608128]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C33"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Evans</span>
                           <span tagx="given-names">RC</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Alice</span>
                           <span tagx="given-names">LA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Campbell</span>
                           <span tagx="given-names">CS</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Kellogg</span>
                           <span tagx="given-names">EA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>
                     </span>
                     <span tagx="year">2000</span>
                     <span tagx="article-title">The granule-bound starch synthase (GBSSI) gene in the Rosaceae: multiple loci and
                        phylogenetic utility
                     </span>. 
                     <span tagx="source">Molecular Phylogenetics and Evolution</span>
                     <span tagx="volume">17</span>:
                     <span tagx="fpage">388</span>–
                     <span tagx="lpage">400</span>. 
                     <div tagxxx="ext-link">doi:10.1006/mpev.2000.0828</div>
                     <span class="pub-id'">[11133193]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C34"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Farris</span>
                           <span tagx="given-names">JS</span>,
                        </span>
                     </span>
                     <span tagx="year">1989</span>
                     <span tagx="article-title">The retention index and the rescaled consistency index</span>. 
                     <span tagx="source">Cladistics</span>
                     <span tagx="volume">5</span>:
                     <span tagx="fpage">417</span>–
                     <span tagx="lpage">419</span>. 
                     <div tagxxx="ext-link">doi:10.1111/j.1096-0031.1989.tb00573.x</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C35"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Fazekas</span>
                           <span tagx="given-names">AJ</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Steeves</span>
                           <span tagx="given-names">R</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Newmaster</span>
                           <span tagx="given-names">S</span>,
                        </span>
                     </span>
                     <span tagx="year">2010</span>
                     <span tagx="article-title">Improving sequencing quality from PCR products containing long mononucleotide repeats</span>. 
                     <span tagx="source">BioTechniques</span>
                     <span tagx="volume">48</span>:
                     <span tagx="fpage">277</span>–
                     <span tagx="lpage">285</span>. 
                     <div tagxxx="ext-link">doi:10.2144/000113369</div>
                     <span class="pub-id'">[20569204]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C36"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Fineschi</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Salvini</span>
                           <span tagx="given-names">D</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Turchini</span>
                           <span tagx="given-names">D</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Pastorelli</span>
                           <span tagx="given-names">R</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Vendramin</span>
                           <span tagx="given-names">GG</span>,
                        </span>
                     </span>
                     <span tagx="year">2005</span>
                     <span tagx="article-title">
                        <i>Crataegus monogyna</i> Jacq. and 
                        <i>C. laevigata</i> (Poir.) DC. (Rosaceae, Maloideae) display low level of genetic diversity assessed
                        by chloroplast markers
                     </span>. 
                     <span tagx="source">Plant Systematics and Evolution</span>
                     <span tagx="volume">250</span>:
                     <span tagx="fpage">187</span>–
                     <span tagx="lpage">196</span>. 
                     <div tagxxx="ext-link">doi:10.1007/s00606-004-0228-x</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C37"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Fitch</span>
                           <span tagx="given-names">WM</span>,
                        </span>
                     </span>
                     <span tagx="year">1971</span>
                     <span tagx="article-title">Toward defining the course of evolution: minimum change for a specific tree topology</span>. 
                     <span tagx="source">Systematic Zoology</span>
                     <span tagx="volume">20</span>:
                     <span tagx="fpage">406</span>–
                     <span tagx="lpage">426</span>. 
                     <div tagxxx="ext-link">doi:10.2307/2412116</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C38"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Guo</span>
                           <span tagx="given-names">R</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Pittler</span>
                           <span tagx="given-names">MH</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Ernst</span>
                           <span tagx="given-names">E</span>,
                        </span>
                     </span>
                     <span tagx="year">2008</span>
                     <span tagx="article-title">Hawthorn extract for treating chronic heart failure</span>. 
                     <span tagx="source">Cochrane Database of Systematic Reviews</span>
                     <div tagxxx="issue">1</div>:
                     <span tagx="fpage">CD005312</span>.
                     <span class="pub-id'">[18254076]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C39"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Hall</span>
                           <span tagx="given-names">TA</span>,
                        </span>
                     </span>
                     <span tagx="year">1999</span>
                     <span tagx="article-title">BioEdit: a user-friendly biological sequence alignment editor and analysis program
                        for Windows 95/98/NT
                     </span>. 
                     <span tagx="source">Nucleic Acids Symposium Series</span>
                     <span tagx="volume">41</span>:
                     <span tagx="fpage">95</span>–
                     <span tagx="lpage">98</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C40"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Hamilton</span>
                           <span tagx="given-names">MB</span>,
                        </span>
                     </span>
                     <span tagx="year">1999</span>
                     <span tagx="article-title">Four primer pairs for the amplification of chloroplast intergenic regions with intraspecific
                        variation
                     </span>. 
                     <span tagx="source">Molecular Ecology</span>
                     <span tagx="volume">8</span>:
                     <span tagx="fpage">521</span>–
                     <span tagx="lpage">523</span>.
                     <span class="pub-id'">[10199016]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C41"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Hebert</span>
                           <span tagx="given-names">PDN</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Cywinska</span>
                           <span tagx="given-names">A</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Ball</span>
                           <span tagx="given-names">SL</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">deWaard</span>
                           <span tagx="given-names">JR</span>,
                        </span>
                     </span>
                     <span tagx="year">2003a</span>
                     <span tagx="article-title">Biological identifications through DNA barcodes</span>. 
                     <span tagx="source">Proceedings of the Royal Society B Biological Sciences</span>
                     <span tagx="volume">270</span>:
                     <span tagx="fpage">313</span>–
                     <span tagx="lpage">321</span>. 
                     <div tagxxx="ext-link">doi:10.1098/rspb.2002.2218</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C42"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Hebert</span>
                           <span tagx="given-names">PDN</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Ratnasingham</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">deWaard</span>
                           <span tagx="given-names">JR</span>,
                        </span>
                     </span>
                     <span tagx="year">2003b</span>
                     <span tagx="article-title">Barcoding animal life: cytochrome c oxidase subunit 1 divergences among closely related
                        species
                     </span>. 
                     <span tagx="source">Proceedings of the Royal Society B Biological Sciences</span>
                     <span tagx="volume">270</span>:
                     <span tagx="fpage">S96</span>–
                     <span tagx="lpage">S99</span>. 
                     <div tagxxx="ext-link">doi:10.1098/rsbl.2003.0025</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C43"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Hildreth</span>
                           <span tagx="given-names">J</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Hrabeta-Robinson</span>
                           <span tagx="given-names">E</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Applequist</span>
                           <span tagx="given-names">W</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Betz</span>
                           <span tagx="given-names">J</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Miller</span>
                           <span tagx="given-names">J</span>,
                        </span>
                     </span>
                     <span tagx="year">2007</span>
                     <span tagx="article-title">Standard operating procedure for the collection and preparation of voucher plant specimens
                        for use in the nutraceutical industry
                     </span>. 
                     <span tagx="source">Analytical and Bioanalytical Chemistry</span>
                     <span tagx="volume">389</span>:
                     <span tagx="fpage">13</span>–
                     <span tagx="lpage">17</span>. 
                     <div tagxxx="ext-link">doi:10.1007/s00216-007-1405-x</div>
                     <span class="pub-id'">[17572883]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C44"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Hojsgaard</span>
                           <span tagx="given-names">D</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Greilhuber</span>
                           <span tagx="given-names">J</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Pellino</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Paun</span>
                           <span tagx="given-names">O</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Sharbel</span>
                           <span tagx="given-names">TF</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Hörandl</span>
                           <span tagx="given-names">E</span>,
                        </span>
                     </span>
                     <span tagx="year">2014</span>
                     <span tagx="article-title">Emergence of apospory and bypass of meiosis via apomixis after sexual hybridisation
                        and polyploidisation
                     </span>. 
                     <span tagx="source">New Phytologist</span>
                     <span tagx="volume">204</span>:
                     <span tagx="fpage">1000</span>–
                     <span tagx="lpage">1012</span>.
                     <span class="pub-id'">[25081588]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C45"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Hollingsworth</span>
                           <span tagx="given-names">PM</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Graham</span>
                           <span tagx="given-names">SW</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Little</span>
                           <span tagx="given-names">DP</span>,
                        </span>
                     </span>
                     <span tagx="year">2011</span>
                     <span tagx="article-title">Choosing and using a plant DNA barcode</span>. 
                     <span tagx="source">PLoS ONE</span>
                     <span tagx="volume">6</span>:
                     <span tagx="fpage">e19254</span>
                     <div tagxxx="ext-link">doi:10.1371/journal.pone.0019254</div>
                     <span class="pub-id'">[21637336]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C46"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Ishikawa</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Kato</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Imakawa</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Mikami</span>
                           <span tagx="given-names">T</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Shimamoto</span>
                           <span tagx="given-names">Y</span>,
                        </span>
                     </span>
                     <span tagx="year">1992</span>
                     <span tagx="article-title">Organelle DNA polymorphism in apple cultivars and rootstocks</span>. 
                     <span tagx="source">Theoretical and Applied Genetics</span>
                     <span tagx="volume">83</span>:
                     <span tagx="fpage">963</span>–
                     <span tagx="lpage">967</span>. 
                     <div tagxxx="ext-link">doi:10.1007/BF00232957</div>
                     <span class="pub-id'">[24202920]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C47"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Ivanova</span>
                           <span tagx="given-names">N</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Kuzmina</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Fazekas</span>
                           <span tagx="given-names">A</span>,
                        </span>
                     </span>
                     <span tagx="year">2011</span>
                     <span tagx="article-title">Glass fiber plate DNA extraction protocol</span>. 
                     <span tagx="source">CCDB protocols</span>. 
                     <span class="publisher-loc'">[Guelph, ON]</span>: 
                     <span class="publisher-name'">[Canadian Centre for DNA Barcoding, University of Guelph]</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C49"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Kelly</span>
                           <span tagx="given-names">LJ</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Ameka</span>
                           <span tagx="given-names">GK</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Chase</span>
                           <span tagx="given-names">MW</span>,
                        </span>
                     </span>
                     <span tagx="year">2010</span>
                     <span tagx="article-title">DNA barcoding of African Podostemaceae (river-weeds): a test of proposed barcode regions</span>. 
                     <span tagx="source">Taxon</span>
                     <span tagx="volume">59</span>:
                     <span tagx="fpage">251</span>–
                     <span tagx="lpage">260</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C48"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Kelly</span>
                           <span tagx="given-names">LM</span>,
                        </span>
                     </span>
                     <span tagx="year">2008</span>
                     <span tagx="article-title">
                        <i>Image of Rosaceae</i> Hesperomeles heterophylla
                     </span>. 
                     <div tagxxx="comment">
                        <div tagxxx="uri">hesp_hetero_margin.jpg</div>, PlantSystematics.org. 800px × 531px (15 July 2014)
                     </div>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C50"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Kirakosyan</span>
                           <span tagx="given-names">A</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Kaufman</span>
                           <span tagx="given-names">PB</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Seymour</span>
                           <span tagx="given-names">EM</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Bolling</span>
                           <span tagx="given-names">SF</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Warber</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Grigoryan</span>
                           <span tagx="given-names">K</span>,
                        </span>
                     </span>
                     <span tagx="year">2005</span>
                     <span tagx="article-title">Different aspects of the exploration and sustainable use of value-added medicinal
                        products in hawthorn (
                        <i>Crataegus</i> spp.)
                     </span>. 
                     <span tagx="source">Research Advances in Agricultural and Food Chemistry</span>
                     <span tagx="volume">6</span>:
                     <span tagx="fpage">95</span>–
                     <span tagx="lpage">109</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C51"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Koch</span>
                           <span tagx="given-names">MA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Karl</span>
                           <span tagx="given-names">R</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Kiefer</span>
                           <span tagx="given-names">C</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Al-Shehbaz</span>
                           <span tagx="given-names">IA</span>,
                        </span>
                     </span>
                     <span tagx="year">2010</span>
                     <span tagx="article-title">Colonizing the American Continent: systematics of the genus 
                        <i>Arabis</i> in North America (Brassicaceae)
                     </span>. 
                     <span tagx="source">American Journal of Botany</span>
                     <span tagx="volume">97</span>:
                     <span tagx="fpage">1040</span>–
                     <span tagx="lpage">1057</span>. 
                     <div tagxxx="ext-link">doi:10.3732/ajb.0900366</div>
                     <span class="pub-id'">[21622474]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C52"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Kress</span>
                           <span tagx="given-names">WJ</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Erickson</span>
                           <span tagx="given-names">DL</span>,
                        </span>
                     </span>
                     <span tagx="year">2007</span>
                     <span tagx="article-title">A two-locus global DNA barcode for land plants: the coding rbcL gene complements the
                        non-coding trnH-psbA spacer region
                     </span>. 
                     <span tagx="source">PLoS ONE</span>
                     <span tagx="volume">2</span>:
                     <span tagx="fpage">e508</span>
                     <div tagxxx="comment">
                        <div tagxxx="ext-link">doi:10.1371/journal.pone.0000508</div>
                     </div>.
                     <span class="pub-id'">[17551588]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C53"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Kress</span>
                           <span tagx="given-names">WJ</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Erickson</span>
                           <span tagx="given-names">DL</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Jones</span>
                           <span tagx="given-names">FA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Swenson</span>
                           <span tagx="given-names">NG</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Perez</span>
                           <span tagx="given-names">R</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Sanjur</span>
                           <span tagx="given-names">O</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Bermingham</span>
                           <span tagx="given-names">E</span>,
                        </span>
                     </span>
                     <span tagx="year">2009</span>
                     <span tagx="article-title">Plant DNA barcodes and a community phylogeny of a tropical forest dynamics plot in
                        Panama
                     </span>. 
                     <span tagx="source">Proceedings of the National Academy of Sciences of the USA</span>
                     <span tagx="volume">106</span>:
                     <span tagx="fpage">18621</span>–
                     <span tagx="lpage">18626</span>. 
                     <div tagxxx="ext-link">doi:10.1073/pnas.0909820106</div>
                     <span class="pub-id'">[19841276]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C54"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Kuzmina</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Ivanova</span>
                           <span tagx="given-names">N</span>,
                        </span>
                     </span>
                     <span tagx="year">2011a</span>
                     <span tagx="article-title">PCR amplification for plants and fungi</span>. 
                     <span tagx="source">CCDB protocols</span>. 
                     <span class="publisher-loc'">[Guelph, ON]</span>: 
                     <span class="publisher-name'">[Canadian Centre for DNA Barcoding, University of Guelph]</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C55"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Kuzmina</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Ivanova</span>
                           <span tagx="given-names">N</span>,
                        </span>
                     </span>
                     <span tagx="year">2011b</span>
                     <span tagx="article-title">Primer sets for plants and fungi</span>. 
                     <span tagx="source">CCDB Protocols</span>. 
                     <span class="publisher-loc'">[Guelph, ON]</span>: 
                     <span class="publisher-name'">[Canadian Centre for DNA Barcoding, University of Guelph]</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C56"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Lance</span>
                           <span tagx="given-names">RW</span>,
                        </span>
                     </span>
                     <span tagx="year">2014</span>
                     <span tagx="source">Haws: a guide to hawthorns of the Southeastern United States</span>. 
                     <span class="publisher-loc'">[Mills River, NC]</span>: 
                     <span class="publisher-name'">[By the author]</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C57"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Levin</span>
                           <span tagx="given-names">RA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Wagner</span>
                           <span tagx="given-names">WL</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Hoch</span>
                           <span tagx="given-names">PC</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Nepokroeff</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Pires</span>
                           <span tagx="given-names">JC</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Zimmer</span>
                           <span tagx="given-names">EA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Sytsma</span>
                           <span tagx="given-names">KJ</span>,
                        </span>
                     </span>
                     <span tagx="year">2003</span>
                     <span tagx="article-title">Family-level relationships of Onagraceae based on chloroplast 
                        <i>rbc</i>L and 
                        <i>ndh</i>F data
                     </span>. 
                     <span tagx="source">American Journal of Botany</span>
                     <span tagx="volume">90</span>:
                     <span tagx="fpage">107</span>–
                     <span tagx="lpage">115</span>. 
                     <div tagxxx="ext-link">doi:10.3732/ajb.90.1.107</div>
                     <span class="pub-id'">[21659085]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C58"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Li</span>
                           <span tagx="given-names">Q-Y</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Guo</span>
                           <span tagx="given-names">W</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Liao</span>
                           <span tagx="given-names">W-B</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Macklin</span>
                           <span tagx="given-names">JA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Li</span>
                           <span tagx="given-names">J-H</span>,
                        </span>
                     </span>
                     <span tagx="year">2012</span>
                     <span tagx="article-title">Generic limits of Pyrinae: insights from nuclear ribosomal DNA sequences</span>. 
                     <span tagx="source">Botanical Studies</span>
                     <span tagx="volume">53</span>:
                     <span tagx="fpage">151</span>–
                     <span tagx="lpage">164</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C59"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Lo</span>
                           <span tagx="given-names">EYY</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Donoghue</span>
                           <span tagx="given-names">MJ</span>,
                        </span>
                     </span>
                     <span tagx="year">2012</span>
                     <span tagx="article-title">Expanded phylogenetic and dating analyses of the apples and their relatives (Pyreae,
                        Rosaceae)
                     </span>. 
                     <span tagx="source">Molecular Phylogenetics and Evolution</span>
                     <span tagx="volume">63</span>:
                     <span tagx="fpage">230</span>–
                     <span tagx="lpage">243</span>. 
                     <div tagxxx="ext-link">doi:10.1016/j.ympev.2011.10.005</div>
                     <span class="pub-id'">[22293154]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C60"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Lo</span>
                           <span tagx="given-names">EYY</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Stefanović</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>
                     </span>
                     <span tagx="year">2007</span>
                     <span tagx="article-title">Molecular reappraisal of relationships between 
                        <i>Crataegus</i> and 
                        <i>Mespilus</i> (Rosaceae, Pyreae)—two genera or one?
                     </span>
                     <span tagx="source">Systematic Botany</span>
                     <span tagx="volume">32</span>:
                     <span tagx="fpage">596</span>–
                     <span tagx="lpage">616</span>. 
                     <div tagxxx="ext-link">doi:10.1600/036364407782250562</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C61"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Lo</span>
                           <span tagx="given-names">EYY</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Stefanović</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Christensen</span>
                           <span tagx="given-names">KI</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>
                     </span>
                     <span tagx="year">2009</span>
                     <span tagx="article-title">Evidence for genetic association between East Asian and Western North American 
                        <i>Crataegus</i> L. (Rosaceae) and rapid divergence of the Eastern North American lineages based on
                        multiple DNA sequences
                     </span>. 
                     <span tagx="source">Molecular Phylogenetics and Evolution</span>
                     <span tagx="volume">51</span>:
                     <span tagx="fpage">157</span>–
                     <span tagx="lpage">168</span>. 
                     <div tagxxx="ext-link">doi:10.1016/j.ympev.2009.01.018</div>
                     <span class="pub-id'">[19405185]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C62"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Lo</span>
                           <span tagx="given-names">EYY</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Stefanović</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Ritland</span>
                           <span tagx="given-names">K</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>
                     </span>
                     <span tagx="year">2010</span>
                     <span tagx="article-title">Fine-scale comparisons of genetic variability in seed families of asexually and sexually
                        reproducing 
                        <i>Crataegus</i> (Hawthorn; Rosaceae)
                     </span>. 
                     <span tagx="source">American Journal of Botany</span>
                     <span tagx="volume">97</span>:
                     <span tagx="fpage">1014</span>–
                     <span tagx="lpage">1024</span>. 
                     <div tagxxx="ext-link">doi:10.3732/ajb.0900091</div>
                     <span class="pub-id'">[21622471]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C63"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Love</span>
                           <span tagx="given-names">R</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Feigen</span>
                           <span tagx="given-names">M</span>,
                        </span>
                     </span>
                     <span tagx="year">1978</span>
                     <span tagx="article-title">Interspecific hybridization between native and naturalized 
                        <i>Crataegus</i> (Rosaceae) in western Oregon
                     </span>. 
                     <span tagx="source">Madroño</span>
                     <span tagx="volume">25</span>:
                     <span tagx="fpage">211</span>–
                     <span tagx="lpage">217</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C64"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Lowenstein</span>
                           <span tagx="given-names">JH</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Amato</span>
                           <span tagx="given-names">G</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Kolokotronis</span>
                           <span tagx="given-names">SO</span>,
                        </span>
                     </span>
                     <span tagx="year">2009</span>
                     <span tagx="article-title">The real maccoyii: identifying tuna sushi with DNA barcodes—contrasting characteristic
                        attributes and genetic distances
                     </span>. 
                     <span tagx="source">PLoS ONE</span>
                     <span tagx="volume">4</span>:
                     <span tagx="fpage">e7866</span>
                     <div tagxxx="ext-link">doi:10.1371/journal.pone.0007866</div>
                     <span class="pub-id'">[19924239]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C65"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Marie-Victorin</span>
                           <span tagx="given-names">F</span>,
                        </span>
                     </span>
                     <span tagx="year">1938</span>
                     <span tagx="article-title">Phytogeographical problems of eastern Canada</span>. 
                     <span tagx="source">American Midland Naturalist</span>
                     <span tagx="volume">19</span>:
                     <span tagx="fpage">489</span>–
                     <span tagx="lpage">558</span>. 
                     <div tagxxx="ext-link">doi:10.2307/2420466</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C66"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Müller</span>
                           <span tagx="given-names">K</span>,
                        </span>
                     </span>
                     <span tagx="year">2005</span>
                     <span tagx="article-title">SeqState—primer design and sequence statistics for phylogenetic DNA datasets</span>. 
                     <span tagx="source">Applied Bioinformatics</span>
                     <span tagx="volume">4</span>:
                     <span tagx="fpage">65</span>–
                     <span tagx="lpage">69</span>. 
                     <div tagxxx="ext-link">doi:10.2165/00822942-200504010-00008</div>
                     <span class="pub-id'">[16000015]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C67"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Nikiforova</span>
                           <span tagx="given-names">SV</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Cavalieri</span>
                           <span tagx="given-names">D</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Velasco</span>
                           <span tagx="given-names">R</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Goremykin</span>
                           <span tagx="given-names">V</span>,
                        </span>
                     </span>
                     <span tagx="year">2013</span>
                     <span tagx="article-title">Phylogenetic analysis of 47 chloroplast genomes clarifies the contribution of wild
                        species to the domesticated apple maternal line
                     </span>. 
                     <span tagx="source">Molecular Biology and Evolution</span>
                     <span tagx="volume">30</span>:
                     <span tagx="fpage">1751</span>–
                     <span tagx="lpage">1760</span>. 
                     <div tagxxx="ext-link">doi:10.1093/molbev/mst092</div>
                     <span class="pub-id'">[23676769]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C115"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Njuguna</span>
                           <span tagx="given-names">W</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Bassil</span>
                           <span tagx="given-names">NV</span>,
                        </span>
                     </span>
                     <span tagx="year">2011</span>
                     <span tagx="article-title">DNA barcoding: unsuccessful for species identification in 
                        <i>Fragaria</i> L
                     </span>. 
                     <span tagx="source">Acta Horticulturae</span>
                     <span tagx="volume">918</span>:
                     <span tagx="fpage">349</span>–
                     <span tagx="lpage">356</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C68"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Nylander</span>
                           <span tagx="given-names">JAA</span>,
                        </span>
                     </span>
                     <span tagx="year">2004</span>
                     <span tagx="source">MrModeltest v2.3 program</span>. 
                     <div tagxxx="comment">Distributed by the author</div>
                     <span class="publisher-name'">[Evolutionary Biology Centre, Uppsala University]</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C69"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Oliver</span>
                           <span tagx="given-names">SG</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Winson</span>
                           <span tagx="given-names">MK</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Kell</span>
                           <span tagx="given-names">DB</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Baganz</span>
                           <span tagx="given-names">F</span>,
                        </span>
                     </span>
                     <span tagx="year">1998</span>
                     <span tagx="article-title">Systematic functional analysis of the yeast genome</span>. 
                     <span tagx="source">Trends in Biotechnology</span>
                     <span tagx="volume">16</span>:
                     <span tagx="fpage">373</span>–
                     <span tagx="lpage">378</span>. 
                     <div tagxxx="ext-link">doi:10.1016/S0167-7799(98)01214-1</div>
                     <span class="pub-id'">[9744112]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C70"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Pang</span>
                           <span tagx="given-names">XH</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Song</span>
                           <span tagx="given-names">JY</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Zhu</span>
                           <span tagx="given-names">YJ</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Xu</span>
                           <span tagx="given-names">HX</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Huang</span>
                           <span tagx="given-names">LF</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Chen</span>
                           <span tagx="given-names">SL</span>,
                        </span>
                     </span>
                     <span tagx="year">2011</span>
                     <span tagx="article-title">Applying plant DNA barcodes for Rosaceae species identification</span>. 
                     <span tagx="source">Cladistics</span>
                     <span tagx="volume">27</span>:
                     <span tagx="fpage">165</span>–
                     <span tagx="lpage">170</span>. 
                     <div tagxxx="ext-link">doi:10.1111/j.1096-0031.2010.00328.x</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C71"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Phipps</span>
                           <span tagx="given-names">JB</span>,
                        </span>
                     </span>
                     <span tagx="year">2015</span>
                     <span tagx="article-title">Crataegus</span>. In: 
                     <span class="collab'">[Flora of North America Editorial Committee]</span>, ed. 
                     <span tagx="source">Flora of North America North of Mexico</span>. 
                     <span tagx="volume">Vol. 9</span>
                     <span class="publisher-loc'">[New York]</span>: 
                     <span class="publisher-name'">[Oxford University Press]</span>, 
                     <span tagx="fpage">491</span>–
                     <span tagx="lpage">643</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C72"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Phipps</span>
                           <span tagx="given-names">JB</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">O'Kennon</span>
                           <span tagx="given-names">RJ</span>,
                        </span>
                     </span>
                     <span tagx="year">2002</span>
                     <span tagx="article-title">New taxa of Crataegus (Rosaceae) from the northern Okanagan-southwestern Shuswap diversity
                        center
                     </span>. 
                     <span tagx="source">SIDA, Contributions to Botany</span>
                     <span tagx="volume">20</span>:
                     <span tagx="fpage">115</span>–
                     <span tagx="lpage">144</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C73"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Piredda</span>
                           <span tagx="given-names">R</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Simeone</span>
                           <span tagx="given-names">MC</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Attimonelli</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Bellarosa</span>
                           <span tagx="given-names">R</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Schirone</span>
                           <span tagx="given-names">B</span>,
                        </span>
                     </span>
                     <span tagx="year">2011</span>
                     <span tagx="article-title">Prospects of barcoding the Italian wild dendroflora: oaks reveal severe limitations
                        to tracking species identity
                     </span>. 
                     <span tagx="source">Molecular Ecology Resources</span>
                     <span tagx="volume">11</span>:
                     <span tagx="fpage">72</span>–
                     <span tagx="lpage">83</span>. 
                     <div tagxxx="ext-link">doi:10.1111/j.1755-0998.2010.02900.x</div>
                     <span class="pub-id'">[21429102]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C74"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Potter</span>
                           <span tagx="given-names">D</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Eriksson</span>
                           <span tagx="given-names">T</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Evans</span>
                           <span tagx="given-names">RC</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Oh</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Smedmark</span>
                           <span tagx="given-names">JEE</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Morgan</span>
                           <span tagx="given-names">DR</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Kerr</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Robertson</span>
                           <span tagx="given-names">KR</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Arsenault</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Campbell</span>
                           <span tagx="given-names">CS</span>,
                        </span>
                     </span>
                     <span tagx="year">2007</span>
                     <span tagx="article-title">Phylogeny and classification of Rosaceae</span>. 
                     <span tagx="source">Plant Systematics and Evolution</span>
                     <span tagx="volume">266</span>:
                     <span tagx="fpage">5</span>–
                     <span tagx="lpage">43</span>. 
                     <div tagxxx="ext-link">doi:10.1007/s00606-007-0539-9</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C75"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Rambaut</span>
                           <span tagx="given-names">A</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Drummond</span>
                           <span tagx="given-names">AJ</span>,
                        </span>
                     </span>
                     <span tagx="year">2007</span>
                     <span tagx="source">
                        <i>Tracer</i>
                        <i>v1.4</i>
                     </span>. 
                     <div tagxxx="comment">
                        <div tagxxx="uri">http://tree.bio.ed.ac.uk/software/tracer/</div> (15 April 2012)
                     </div>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C76"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Raspé</span>
                           <span tagx="given-names">O</span>,
                        </span>
                     </span>
                     <span tagx="year">2001</span>
                     <span tagx="article-title">Inheritance of the chloroplast genome in 
                        <i>Sorbus aucuparia</i> L. (Rosaceae)
                     </span>. 
                     <span tagx="source">Journal of Heredity</span>
                     <span tagx="volume">92</span>:
                     <span tagx="fpage">507</span>–
                     <span tagx="lpage">509</span>. 
                     <div tagxxx="ext-link">doi:10.1093/jhered/92.6.507</div>
                     <span class="pub-id'">[11948219]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C77"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Ratnasingham</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Hebert</span>
                           <span tagx="given-names">PDN</span>,
                        </span>
                     </span>
                     <span tagx="year">2007</span>
                     <span tagx="article-title">BOLD: the barcode of life data system (http://
                        <div tagxxx="uri">www.barcodinglife.org</div>)
                     </span>. 
                     <span tagx="source">Molecular Ecology Notes</span>
                     <span tagx="volume">7</span>:
                     <span tagx="fpage">355</span>–
                     <span tagx="lpage">364</span>. 
                     <div tagxxx="ext-link">doi:10.1111/j.1471-8286.2007.01678.x</div>
                     <span class="pub-id'">[18784790]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C78"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Reid</span>
                           <span tagx="given-names">BN</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Le</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">McCord</span>
                           <span tagx="given-names">WP</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Iverson</span>
                           <span tagx="given-names">JB</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Georges</span>
                           <span tagx="given-names">A</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Bergmann</span>
                           <span tagx="given-names">T</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Amato</span>
                           <span tagx="given-names">G</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Desalle</span>
                           <span tagx="given-names">R</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Naro-Maciel</span>
                           <span tagx="given-names">E</span>,
                        </span>
                     </span>
                     <span tagx="year">2011</span>
                     <span tagx="article-title">Comparing and combining distance-based and character-based approaches for barcoding
                        turtles
                     </span>. 
                     <span tagx="source">Molecular Ecology Resources</span>
                     <span tagx="volume">11</span>:
                     <span tagx="fpage">956</span>–
                     <span tagx="lpage">967</span>. 
                     <div tagxxx="ext-link">doi:10.1111/j.1755-0998.2011.03032.x</div>
                     <span class="pub-id'">[21635698]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C79"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Ronquist</span>
                           <span tagx="given-names">F</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Huelsenbeck</span>
                           <span tagx="given-names">JP</span>,
                        </span>
                     </span>
                     <span tagx="year">2003</span>
                     <span tagx="article-title">MrBayes 3: Bayesian phylogenetic inference under mixed models</span>. 
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="volume">19</span>:
                     <span tagx="fpage">1572</span>–
                     <span tagx="lpage">1574</span>. 
                     <div tagxxx="ext-link">doi:10.1093/bioinformatics/btg180</div>
                     <span class="pub-id'">[12912839]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C80"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Ronquist</span>
                           <span tagx="given-names">F</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">van deMark</span>
                           <span tagx="given-names">P</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Huelsenbeck</span>
                           <span tagx="given-names">JP</span>,
                        </span>
                     </span>
                     <span tagx="year">2010</span>
                     <span tagx="article-title">Bayesian phylogenetic analyses using MrBayes</span>. In: 
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Lemey</span>
                           <span tagx="given-names">P</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Salemi</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Vandamme</span>
                           <span tagx="given-names">AM</span>,
                        </span>
                     </span>, eds. 
                     <span tagx="source">The phylogenetic handbook: a practical approach to phylogenetic analysis and hypothesis
                        testing
                     </span>. 
                     <span class="publisher-loc'">[Cambridge]</span>: 
                     <span class="publisher-name'">[University Cambridge Press]</span>, 
                     <span tagx="fpage">2010</span>–
                     <span tagx="lpage">2066</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C81"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Ross</span>
                           <span tagx="given-names">HA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Murugan</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Li</span>
                           <span tagx="given-names">WLS</span>,
                        </span>
                     </span>
                     <span tagx="year">2008</span>
                     <span tagx="article-title">Testing the reliability of genetic methods of species identification via simulation</span>. 
                     <span tagx="source">Systematic Biology</span>
                     <span tagx="volume">57</span>:
                     <span tagx="fpage">216</span>–
                     <span tagx="lpage">230</span>. 
                     <div tagxxx="ext-link">doi:10.1080/10635150802032990</div>
                     <span class="pub-id'">[18398767]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C82"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Saitou</span>
                           <span tagx="given-names">N</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Nei</span>
                           <span tagx="given-names">M</span>,
                        </span>
                     </span>
                     <span tagx="year">1987</span>
                     <span tagx="article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees</span>. 
                     <span tagx="source">Molecular Biology and Evolution</span>
                     <span tagx="volume">4</span>:
                     <span tagx="fpage">406</span>–
                     <span tagx="lpage">425</span>.
                     <span class="pub-id'">[3447015]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C83"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Salmaki</span>
                           <span tagx="given-names">Y</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Zarre</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Ryding</span>
                           <span tagx="given-names">O</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Lindqvist</span>
                           <span tagx="given-names">C</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Bräuchler</span>
                           <span tagx="given-names">C</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Heubl</span>
                           <span tagx="given-names">G</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Barber</span>
                           <span tagx="given-names">J</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Bendiksby</span>
                           <span tagx="given-names">M</span>,
                        </span>
                     </span>
                     <span tagx="year">2013</span>
                     <span tagx="article-title">Molecular phylogeny of tribe Stachydeae (Lamiaceae subfamily Lamioideae)</span>. 
                     <span tagx="source">Molecular Phylogenetics and Evolution</span>
                     <span tagx="volume">69</span>:
                     <span tagx="fpage">535</span>–
                     <span tagx="lpage">551</span>. 
                     <div tagxxx="ext-link">doi:10.1016/j.ympev.2013.07.024</div>
                     <span class="pub-id'">[23916621]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C85"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Sang</span>
                           <span tagx="given-names">T</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Crawford</span>
                           <span tagx="given-names">DJ</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Stuessy</span>
                           <span tagx="given-names">TF</span>,
                        </span>
                     </span>
                     <span tagx="year">1997</span>
                     <span tagx="article-title">Chloroplast DNA phylogeny, reticulate evolution and biogeography of 
                        <i>Paeonia</i> (Paeoniaceae)
                     </span>. 
                     <span tagx="source">American Journal of Botany</span>
                     <span tagx="volume">84</span>:
                     <span tagx="fpage">1120</span>–
                     <span tagx="lpage">1136</span>. 
                     <div tagxxx="ext-link">doi:10.2307/2446155</div>
                     <span class="pub-id'">[21708667]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C86"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Seberg</span>
                           <span tagx="given-names">O</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Petersen</span>
                           <span tagx="given-names">G</span>,
                        </span>
                     </span>
                     <span tagx="year">2009</span>
                     <span tagx="article-title">How many loci does it take to DNA barcode a Crocus?</span>
                     <span tagx="source">PLoS ONE</span>
                     <span tagx="volume">4</span>:
                     <span tagx="fpage">e4598</span>
                     <div tagxxx="ext-link">doi:10.1371/journal.pone.0004598</div>
                     <span class="pub-id'">[19240801]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C87"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Simmons</span>
                           <span tagx="given-names">MP</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Ochoterena</span>
                           <span tagx="given-names">H</span>,
                        </span>
                     </span>
                     <span tagx="year">2000</span>
                     <span tagx="article-title">Gaps as characters in sequence-based phylogenetic analyses</span>. 
                     <span tagx="source">Systematic Biology</span>
                     <span tagx="volume">49</span>:
                     <span tagx="fpage">369</span>–
                     <span tagx="lpage">381</span>. 
                     <div tagxxx="ext-link">doi:10.1093/sysbio/49.2.369</div>
                     <span class="pub-id'">[12118412]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C88"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Smith</span>
                           <span tagx="given-names">SA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Donoghue</span>
                           <span tagx="given-names">MJ</span>,
                        </span>
                     </span>
                     <span tagx="year">2008</span>
                     <span tagx="article-title">Rates of molecular evolution are linked to life history in flowering plants</span>. 
                     <span tagx="source">Science</span>
                     <span tagx="volume">322</span>:
                     <span tagx="fpage">86</span>–
                     <span tagx="lpage">89</span>. 
                     <div tagxxx="ext-link">doi:10.1126/science.1163197</div>
                     <span class="pub-id'">[18832643]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C89"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Spooner</span>
                           <span tagx="given-names">DM</span>,
                        </span>
                     </span>
                     <span tagx="year">2009</span>
                     <span tagx="article-title">DNA barcoding will frequently fail in complicated groups: an example in wild potatoes</span>. 
                     <span tagx="source">American Journal of Botany</span>
                     <span tagx="volume">96</span>:
                     <span tagx="fpage">1177</span>–
                     <span tagx="lpage">1189</span>. 
                     <div tagxxx="ext-link">doi:10.3732/ajb.0800246</div>
                     <span class="pub-id'">[21628268]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C90"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Stanford</span>
                           <span tagx="given-names">AM</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Harden</span>
                           <span tagx="given-names">R</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Parks</span>
                           <span tagx="given-names">CR</span>,
                        </span>
                     </span>
                     <span tagx="year">2000</span>
                     <span tagx="article-title">Phylogeny and biogeography of Juglans (Juglandaceae) based on 
                        <i>mat</i>K and ITS sequence data
                     </span>. 
                     <span tagx="source">American Journal of Botany</span>
                     <span tagx="volume">87</span>:
                     <span tagx="fpage">872</span>–
                     <span tagx="lpage">882</span>. 
                     <div tagxxx="ext-link">doi:10.2307/2656895</div>
                     <span class="pub-id'">[10860918]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C91"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Swofford</span>
                           <span tagx="given-names">DL</span>,
                        </span>
                     </span>
                     <span tagx="year">2003</span>
                     <span tagx="source">PAUP*: phylogenetic analysis using parsimony (*and Other Methods)</span>. 
                     <div tagxxx="comment">Version 4.0b10 for Macintosh (PPC/Altivec) ed</div>
                     <span class="publisher-loc'">[Sunderland, MA]</span>: 
                     <span class="publisher-name'">[Sinauer Associates]</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C92"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Taberlet</span>
                           <span tagx="given-names">P</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Gielly</span>
                           <span tagx="given-names">L</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Pautou</span>
                           <span tagx="given-names">G</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Bouvet</span>
                           <span tagx="given-names">J</span>,
                        </span>
                     </span>
                     <span tagx="year">1991</span>
                     <span tagx="article-title">Universal primers for amplification of three non-coding regions of chloroplast DNA</span>. 
                     <span tagx="source">Plant Molecular Biology</span>
                     <span tagx="volume">17</span>:
                     <span tagx="fpage">1105</span>–
                     <span tagx="lpage">1109</span>. 
                     <div tagxxx="ext-link">doi:10.1007/BF00037152</div>
                     <span class="pub-id'">[1932684]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C93"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Talent</span>
                           <span tagx="given-names">N</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>
                     </span>
                     <span tagx="year">2005</span>
                     <span tagx="article-title">Polyploidy in 
                        <i>Crataegus</i> and 
                        <i>Mespilus</i> (Rosaceae, Maloideae): evolutionary inferences from flow cytometry of nuclear DNA
                        amounts
                     </span>. 
                     <span tagx="source">Canadian Journal of Botany</span>
                     <span tagx="volume">83</span>:
                     <span tagx="fpage">1268</span>–
                     <span tagx="lpage">1304</span>. 
                     <div tagxxx="ext-link">doi:10.1139/b05-088</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C94"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Talent</span>
                           <span tagx="given-names">N</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>
                     </span>
                     <span tagx="year">2007a</span>
                     <span tagx="article-title">Endosperm formation in aposporous 
                        <i>Crataegus</i> (Rosaceae, Spiraeoideae, tribe Pyreae): parallels to Ranunculaceae and Poaceae
                     </span>. 
                     <span tagx="source">New Phytologist</span>
                     <span tagx="volume">173</span>:
                     <span tagx="fpage">231</span>–
                     <span tagx="lpage">249</span>. 
                     <div tagxxx="ext-link">doi:10.1111/j.1469-8137.2006.01918.x</div>
                     <span class="pub-id'">[17204072]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C95"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Talent</span>
                           <span tagx="given-names">N</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>
                     </span>
                     <span tagx="year">2007b</span>
                     <span tagx="article-title">The potential for ploidy level increases and decreases in Crataegus (Rosaceae, Spiraeoideae,
                        tribe Pyreae)
                     </span>. 
                     <span tagx="source">Canadian Journal of Botany</span>
                     <span tagx="volume">85</span>:
                     <span tagx="fpage">570</span>–
                     <span tagx="lpage">584</span>. 
                     <div tagxxx="ext-link">doi:10.1139/B07-028</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C96"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Tamura</span>
                           <span tagx="given-names">K</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Stecher</span>
                           <span tagx="given-names">G</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Peterson</span>
                           <span tagx="given-names">D</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Filipski</span>
                           <span tagx="given-names">A</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Kumar</span>
                           <span tagx="given-names">S</span>,
                        </span>
                     </span>
                     <span tagx="year">2013</span>
                     <span tagx="article-title">MEGA6: molecular evolutionary genetics analysis version 6.0</span>. 
                     <span tagx="source">Molecular Biology and Evolution</span>
                     <span tagx="volume">30</span>:
                     <span tagx="fpage">2725</span>–
                     <span tagx="lpage">2729</span>. 
                     <div tagxxx="ext-link">doi:10.1093/molbev/mst197</div>
                     <span class="pub-id'">[24132122]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C97"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Tate</span>
                           <span tagx="given-names">JA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Simpson</span>
                           <span tagx="given-names">BB</span>,
                        </span>
                     </span>
                     <span tagx="year">2003</span>
                     <span tagx="article-title">Paraphyly of Tarasa (Malvaceae) and diverse origins of the polyploid species</span>. 
                     <span tagx="source">Systematic Botany</span>
                     <span tagx="volume">28</span>:
                     <span tagx="fpage">723</span>–
                     <span tagx="lpage">737</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C98"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Taylor</span>
                           <span tagx="given-names">HR</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Harris</span>
                           <span tagx="given-names">WE</span>,
                        </span>
                     </span>
                     <span tagx="year">2012</span>
                     <span tagx="article-title">An emergent science on the brink of irrelevance: a review of the past 8 years of DNA
                        barcoding
                     </span>. 
                     <span tagx="source">Molecular Ecology Resources</span>
                     <span tagx="volume">12</span>:
                     <span tagx="fpage">377</span>–
                     <span tagx="lpage">388</span>. 
                     <div tagxxx="ext-link">doi:10.1111/j.1755-0998.2012.03119.x</div>
                     <span class="pub-id'">[22356472]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C99"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Thompson</span>
                           <span tagx="given-names">JD</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Gibson</span>
                           <span tagx="given-names">TJ</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Plewniak</span>
                           <span tagx="given-names">F</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Jeanmougin</span>
                           <span tagx="given-names">F</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Higgins</span>
                           <span tagx="given-names">DG</span>,
                        </span>
                     </span>
                     <span tagx="year">1997</span>
                     <span tagx="article-title">The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment
                        aided by quality analysis tools
                     </span>. 
                     <span tagx="source">Nucleic Acids Research</span>
                     <span tagx="volume">25</span>:
                     <span tagx="fpage">4876</span>–
                     <span tagx="lpage">4882</span>. 
                     <div tagxxx="ext-link">doi:10.1093/nar/25.24.4876</div>
                     <span class="pub-id'">[9396791]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C100"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Tundis</span>
                           <span tagx="given-names">R</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Peruzzi</span>
                           <span tagx="given-names">L</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Menichini</span>
                           <span tagx="given-names">F</span>,
                        </span>
                     </span>
                     <span tagx="year">2014</span>
                     <span tagx="article-title">Phytochemical and biological studies of 
                        <i>Stachys</i> species in relation to chemotaxonomy: a review
                     </span>. 
                     <span tagx="source">Phytochemistry</span>
                     <span tagx="volume">102</span>:
                     <span tagx="fpage">7</span>–
                     <span tagx="lpage">39</span>. 
                     <div tagxxx="ext-link">doi:10.1016/j.phytochem.2014.01.023</div>
                     <span class="pub-id'">[24661611]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C101"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Turner</span>
                           <span tagx="given-names">NJ</span>,
                        </span>
                     </span>
                     <span tagx="year">2014a</span>
                     <span tagx="source">Ancient pathways, ancestral knowledge: ethnobotany and ecological wisdom of indigenous
                        peoples of Northwestern North America—Appendix 2B. Names of native plant species in
                        indigenous languages of Northwestern North America
                     </span>
                     <span class="publisher-loc'">[. Victoria]</span>, 
                     <span class="publisher-loc'">[BC]</span>: 
                     <span class="publisher-name'">[University of Victoria]</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C102"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Turner</span>
                           <span tagx="given-names">NJ</span>,
                        </span>
                     </span>
                     <span tagx="year">2014b</span>
                     <span tagx="source">Ancient pathways, ancestral knowledge: ethnobotany and ecological wisdom of indigenous
                        peoples of Northwestern North America Volume One—the history and practice of indigenous
                        plant knowledge
                     </span>. 
                     <span class="publisher-loc'">[Montreal and Kingston]</span>: 
                     <span class="publisher-name'">[McGill-Queen’s University Press]</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C103"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Vaillancourt</span>
                           <span tagx="given-names">RE</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Jackson</span>
                           <span tagx="given-names">HD</span>,
                        </span>
                     </span>
                     <span tagx="year">2000</span>
                     <span tagx="article-title">A chloroplast DNA hypervariable region in eucalypts</span>. 
                     <span tagx="source">Theoretical and Applied Genetics</span>
                     <span tagx="volume">101</span>:
                     <span tagx="fpage">473</span>–
                     <span tagx="lpage">477</span>. 
                     <div tagxxx="ext-link">doi:10.1007/s001220051505</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C104"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">van den Berg</span>
                           <span tagx="given-names">RA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Hoefsloot</span>
                           <span tagx="given-names">HCJ</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Westerhuis</span>
                           <span tagx="given-names">JA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Smilde</span>
                           <span tagx="given-names">AK</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">van der Werf</span>
                           <span tagx="given-names">MJ</span>,
                        </span>
                     </span>
                     <span tagx="year">2006</span>
                     <span tagx="article-title">Centering, scaling, and transformations: improving the biological information content
                        of metabolomics data
                     </span>. 
                     <span tagx="source">BMC Genomics</span>
                     <span tagx="volume">7</span>:
                     <span tagx="fpage">142</span>
                     <div tagxxx="ext-link">doi:10.1186/1471-2164-7-142</div>
                     <span class="pub-id'">[16762068]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C105"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Waterhouse</span>
                           <span tagx="given-names">AM</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Procter</span>
                           <span tagx="given-names">JB</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Martin</span>
                           <span tagx="given-names">DMA</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Clamp</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Barton</span>
                           <span tagx="given-names">GJ</span>,
                        </span>
                     </span>
                     <span tagx="year">2009</span>
                     <span tagx="article-title">Jalview version 2—a multiple sequence alignment editor and analysis workbench</span>. 
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="volume">25</span>:
                     <span tagx="fpage">1189</span>–
                     <span tagx="lpage">1191</span>. 
                     <div tagxxx="ext-link">doi:10.1093/bioinformatics/btp033</div>
                     <span class="pub-id'">[19151095]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C106"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Weitschek</span>
                           <span tagx="given-names">E</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Van Velzen</span>
                           <span tagx="given-names">R</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Felici</span>
                           <span tagx="given-names">G</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Bertolazzi</span>
                           <span tagx="given-names">P</span>,
                        </span>
                     </span>
                     <span tagx="year">2013</span>
                     <span tagx="article-title">BLOG 2.0: a software system for character-based species classification with DNA Barcode
                        sequences. What it does, how to use it
                     </span>. 
                     <span tagx="source">Molecular Ecology Resources</span>
                     <span tagx="volume">13</span>:
                     <span tagx="fpage">1043</span>–
                     <span tagx="lpage">1046</span>.
                     <span class="pub-id'">[23350601]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C107"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Wells</span>
                           <span tagx="given-names">TC</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Phipps</span>
                           <span tagx="given-names">JB</span>,
                        </span>
                     </span>
                     <span tagx="year">1989</span>
                     <span tagx="article-title">Studies in 
                        <i>Crataegus</i> (Rosaceae: Maloideae). XX. Interserial hybridization between 
                        <i>Crataegus monogyna</i> (series 
                        <i>Oxycanthae</i>) and 
                        <i>Crataegus punctata</i> (series 
                        <i>Punctatae</i>) in southern Ontario
                     </span>. 
                     <span tagx="source">Canadian Journal of Botany</span>
                     <span tagx="volume">67</span>:
                     <span tagx="fpage">2465</span>–
                     <span tagx="lpage">2472</span>. 
                     <div tagxxx="ext-link">doi:10.1139/b89-316</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C108"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">White</span>
                           <span tagx="given-names">TJ</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Bruns</span>
                           <span tagx="given-names">T</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Lee</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Taylor</span>
                           <span tagx="given-names">J</span>,
                        </span>
                     </span>
                     <span tagx="year">1990</span>
                     <span tagx="article-title">Amplifications and direct sequencing of fungal ribosomal RNA genes for phylogenetics</span>. In: 
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Innis</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Gelfand</span>
                           <span tagx="given-names">D</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Sninsky</span>
                           <span tagx="given-names">J</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">White</span>
                           <span tagx="given-names">T</span>,
                        </span>
                     </span>, eds. 
                     <span tagx="source">PCR protocols: a guide to methods and applications</span>. 
                     <span class="publisher-loc'">[San Diego, CA]</span>: 
                     <span class="publisher-name'">[Academic Press]</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C109"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Wolfe</span>
                           <span tagx="given-names">KH</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Li</span>
                           <span tagx="given-names">WH</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Sharp</span>
                           <span tagx="given-names">PM</span>,
                        </span>
                     </span>
                     <span tagx="year">1987</span>
                     <span tagx="article-title">Rates of nucleotide substitution vary greatly among plant mitochondrial, chloroplast,
                        and nuclear DNAs
                     </span>. 
                     <span tagx="source">Proceedings of the National Academy of Sciences of the USA</span>
                     <span tagx="volume">84</span>:
                     <span tagx="fpage">9054</span>–
                     <span tagx="lpage">9058</span>. 
                     <div tagxxx="ext-link">doi:10.1073/pnas.84.24.9054</div>
                     <span class="pub-id'">[3480529]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C110"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Yang</span>
                           <span tagx="given-names">B</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Liu</span>
                           <span tagx="given-names">P</span>,
                        </span>
                     </span>
                     <span tagx="year">2012</span>
                     <span tagx="article-title">Composition and health effects of phenolic compounds in hawthorn (
                        <i>Crataegus</i> spp.) of different origins
                     </span>. 
                     <span tagx="source">Journal of the Science of Food and Agriculture</span>
                     <span tagx="volume">92</span>:
                     <span tagx="fpage">1578</span>–
                     <span tagx="lpage">1590</span>. 
                     <div tagxxx="ext-link">doi:10.1002/jsfa.5671</div>
                     <span class="pub-id'">[22488722]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C111"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Yang</span>
                           <span tagx="given-names">Z</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Rannala</span>
                           <span tagx="given-names">B</span>,
                        </span>
                     </span>
                     <span tagx="year">1997</span>
                     <span tagx="article-title">Bayesian phylogenetic inference using DNA sequences: a Markov chain Monte Carlo method</span>. 
                     <span tagx="source">Molecular Biology and Evolution</span>
                     <span tagx="volume">14</span>:
                     <span tagx="fpage">717</span>–
                     <span tagx="lpage">724</span>. 
                     <div tagxxx="ext-link">doi:10.1093/oxfordjournals.molbev.a025811</div>
                     <span class="pub-id'">[9214744]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C112"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Zarrei</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Talent</span>
                           <span tagx="given-names">N</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Coughlan</span>
                           <span tagx="given-names">JM</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Stefanović</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>
                     </span>
                     <span tagx="year">2012</span>
                     <span tagx="article-title">Nuclear ribosomal ITS2 DNA sequences illuminate hybridization in a diploid-polyploid
                        agamic complex of 
                        <i>Crataegus</i> (Rosaceae)
                     </span>. 
                     <span tagx="source">Botany 2012. The next generation</span>. 
                     <span class="publisher-loc'">[Columbus, OH]</span>
                     <div tagxxx="comment">Abstract 
                        <div tagxxx="uri">http://2012.botanyconference.org/engine/search/index.php?func=detail&amp;aid=536</div> (31 August 2014)
                     </div>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C113"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Zarrei</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Stefanović</span>
                           <span tagx="given-names">S</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Dickinson</span>
                           <span tagx="given-names">TA</span>,
                        </span>
                     </span>
                     <span tagx="year">2014</span>
                     <span tagx="article-title">Reticulate evolution in North American black-fruited hawthorns (
                        <i>Crataegus</i> section 
                        <i>Douglasia</i>; Rosaceae): evidence from nuclear ITS2 and plastid sequences
                     </span>. 
                     <span tagx="source">Annals of Botany</span>
                     <span tagx="volume">114</span>:
                     <span tagx="fpage">253</span>–
                     <span tagx="lpage">269</span>. 
                     <div tagxxx="ext-link">doi:10.1093/aob/mcu116</div>
                     <span class="pub-id'">[24984714]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="PLV045C116"></a>
                  <div tagxxx="mixed-citation">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Zarrei</span>
                           <span tagx="given-names">M</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">MacDonald</span>
                           <span tagx="given-names">JR</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Merico</span>
                           <span tagx="given-names">D</span>,
                        </span>, 
                        <span class="name'">
                           <span tagx="surname">Scherer</span>
                           <span tagx="given-names">SW</span>,
                        </span>
                     </span>
                     <span tagx="year">2015</span>
                     <span tagx="article-title">A copy number variation map of the human genome</span>. 
                     <span tagx="source">Nature Reviews Genetics</span>
                     <span tagx="volume">16</span>:
                     <span tagx="fpage">172</span>–
                     <span tagx="lpage">183</span>. 
                     <div tagxxx="ext-link">doi:10.1002/jsfa.5671</div>
                  </div>
               </li>
            </ul>
         </div>
      </div>
   </div>
</html>