<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <div xmlns="" tagx="article">
      <head>
         <style type="text/css"> table, tr, td {border : thin;}</style>
         <span class="citation_publisher">Public Library of Science</span>
         <h2 class="citation-title">Universal Single-Probe RT-PCR Assay for Diagnosis of Dengue Virus Infections</h2>
         <span class="doi">10.1371/journal.pntd.0003416</span>
         <div class="contrib-group">contrib: 
            	  CONT
            	  
            <span class="citation_author">Erik Alm</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Birgitta Lesko</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Gunnel Lindegren</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Clas Ahlm</span>
            <span class="citation_author_institution_ref">3</span>
            <span class="citation_author">Sandra Söderholm</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Kerstin I. Falk</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Nina Lagerqvist</span>
            <span class="citation_author_institution_ref">1</span>
            	  CONT
            	  
            <div tagxxx="contrib">
               <span class="name'">
                  <span tagx="surname">Armstrong</span>
                  <span tagx="given-names">Philip M.</span>,
               </span>
               <div tagxxx="role">Editor</div>
               <span ref-type="aff" rid="edit1"></span>
            </div>
         </div>
         <span class="email">[nina.lagerqvist@folkhalsomyndigheten.se]</span>
         <span class="conflict">[The authors have declared that no competing interests exist.]</span>
         <span class="contribs">[Conceived and designed the experiments: NL EA. Performed the experiments: NL SS GL.
            Analyzed the data: EA NL BL KIF. Contributed reagents/materials/analysis tools: CA.
            Wrote the paper: NL EA BL KIF CA. Performed the sequence analysis: EA.]
         </span>
         <span class="citation_pubdate">[2014-12-18]</span>
         <span class="citation_copyright">[Copyright 2014: Alm et al]</span>
         <span class="citation_licence">[Licence This is an open-access article distributed under the terms of the Creative
            Commons Attribution License, which permits unrestricted use, distribution, and reproduction
            in any medium, provided the original author and source are credited.]
         </span>
         <span class="citation_funding">[The authors are grateful to the Swedish Civil Contingencies Agency (MSB) for financial
            support. The funders had no role in the study design, data collection and analysis,
            decision to publish, or preparation of the manuscript.]
         </span>
         <span class="citation_pagecount">[]</span>
         <span class="citation_data">[Data AvailabilityThe authors confirm that all data underlying the findings are fully
            available without restriction. All relevant data are within the paper and its Supporting
            Information files.]
         </span>
         <div tagxxx="notes">
            <div tagxxx="title">Data Availability</div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The authors confirm that all data underlying the findings are fully available without
               restriction. All relevant data are within the paper and its Supporting Information
               files.
            </p>
         </div>
      </head>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <div id="">
            <h2>Background</h2>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Dengue is a mosquito-borne viral disease that has become more prevalent in the last
               few decades. Most patients are viremic when they present with symptoms, and early
               diagnosis of dengue is important in preventing severe clinical complications associated
               with this disease and also represents a key factor in differential diagnosis. Here,
               we designed and validated a hydrolysis-probe-based one-step real-time RT-PCR assay
               that targets the genomes of dengue virus serotypes 1–4.
            </p>
         </div>
         <div id="">
            <h2>Methodology/Principal Findings</h2>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The primers and probe used in our RT-PCR assay were designed to target the 3′ untranslated
               region of all complete genome sequences of dengue virus available in GenBank (n = 3,305).
               Performance of the assay was evaluated using in vitro transcribed RNA, laboratory-adapted
               virus strains, external control panels, and clinical specimens. The linear dynamic
               range was found to be 10
               <sup>4</sup>–10
               <sup>11</sup> GCE/mL, and the detection limit was between 6.0×10
               <sup>2</sup> and 1.1×10
               <sup>3</sup> GCE/mL depending on target sequence. The assay did not cross-react with human RNA,
               nor did it produce false-positive results for other human pathogenic flaviviruses
               or clinically important etiological agents of febrile illnesses. We used clinical
               serum samples obtained from returning travelers with dengue-compatible symptomatology
               (n = 163) to evaluate the diagnostic relevance of our assay, and laboratory diagnosis
               performed by the RT-PCR assay had 100% positive agreement with diagnosis performed
               by NS1 antigen detection. In a retrospective evaluation including 60 archived serum
               samples collected from confirmed dengue cases 1–9 days after disease onset, the RT-PCR
               assay detected viral RNA up to 9 days after appearance of symptoms.
            </p>
         </div>
         <div id="">
            <h2>Conclusions/Significance</h2>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The validation of the RT-PCR assay presented here indicates that this technique can
               be a reliable diagnostic tool, and 
               <i>hence</i> we suggest that it be introduced as the method of choice during the first 5 days
               of dengue symptoms.
            </p>
         </div>
      </div>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <div tagxxx="title">Author Summary</div>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Dengue is the most common mosquito-borne viral disease affecting humans, with an estimated
            100 million cases annually. Being able to achieve early and correct detection of all
            four serotypes of dengue virus can have an impact on the diagnosis of individual patients,
            such as febrile travelers returning home, but can also be useful in endemic countries,
            particularly during the early stage of an outbreak. After analyzing all the complete
            genome sequences of dengue virus serotypes 1 to 4 published in GenBank, we constructed
            a one-step real-time RT-PCR assay that can be used in different settings to detect
            dengue virus in clinical samples. Extensive evaluation of the performance of this
            assay by testing samples from dengue patients confirmed the usefulness of the method
            in detecting early stage infection. In short, our dengue virus RT-PCR proved to be
            a reliable diagnostic tool and a relevant alternative and/or complement to NS1 antigen-detecting
            tests and serological methods for diagnosis of acute dengue.
         </p>
      </div>
      <div id="s1">
         <h2>Introduction</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Dengue has gradually become one of the leading causes of morbidity in tropical and
            subtropical regions 
            <span ref-type="bibr" rid="pntd.0003416-WHO1">
               <sup>
                  <a href="#pntd.0003416-WHO1">[1]</a>
               </sup>
            </span>, and this disease is caused by infection with any of four genetically related dengue
            virus (DENV) serotypes (designated 1 to 4). DENV has a positive-stranded RNA genome
            and belongs to the family 
            <i>Flaviviridae</i> and the genus 
            <i>Flavivirus</i>. The geographical areas in which DENV circulates have expanded in recent years, and
            all four serotypes are now found in Asia, Africa, and the Americas 
            <span ref-type="bibr" rid="pntd.0003416-Guzman1">
               <sup>
                  <a href="#pntd.0003416-Guzman1">[2]</a>
               </sup>
            </span>. Endemic dengue has been reported from more than 100 countries 
            <span ref-type="bibr" rid="pntd.0003416-WHO1">
               <sup>
                  <a href="#pntd.0003416-WHO1">[1]</a>
               </sup>
            </span>, many of which are popular tourist destinations. Thus diagnostic challenges include
            both patients residing in endemic countries and travelers returning home with fever
            after visiting such regions.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Most DENV infections are asymptomatic or are manifested by a sudden onset of fever
            4 to 6 days after infection 
            <span ref-type="bibr" rid="pntd.0003416-Guzman2">
               <sup>
                  <a href="#pntd.0003416-Guzman2">[3]</a>
               </sup>
            </span> that is accompanied by nonspecific signs and symptoms such as nausea, headache, rash,
            and muscle and joint pain, without any severe sequelae 
            <span ref-type="bibr" rid="pntd.0003416-Low1">
               <sup>
                  <a href="#pntd.0003416-Low1">[4]</a>
               </sup>
            </span>. However, some patients develop dengue with severe symptoms, which was formerly termed
            dengue hemorrhagic fever or dengue shock syndrome 
            <span ref-type="bibr" rid="pntd.0003416-Alexander1">
               <sup>
                  <a href="#pntd.0003416-Alexander1">[5]</a>
               </sup>
            </span>. This critical phase begins at defervescence and is characterized by plasma leakage
            leading to respiratory distress and shock, internal hemorrhage, and organ impairment,
            manifestations that can have a deadly outcome if proper clinical management is delayed
            
            <span ref-type="bibr" rid="pntd.0003416-WHO2">
               <sup>
                  <a href="#pntd.0003416-WHO2">[6]</a>
               </sup>
            </span>.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Infection with one serotype confers long-term protective immunity against that particular
            serotype only, whereas subsequent infection with a heterologous serotype may predispose
            a person to develop severe disease 
            <span ref-type="bibr" rid="pntd.0003416-Zompi1">
               <sup>
                  <a href="#pntd.0003416-Zompi1">[7]</a>
               </sup>
            </span>. It is not fully understood why some individuals resolve DENV infections without
            any complications, whereas others develop severe symptoms, and no single factor has
            yet been identified as the sole contributor to disease severity 
            <span ref-type="bibr" rid="pntd.0003416-Martina1">
               <sup>
                  <a href="#pntd.0003416-Martina1">[8]</a>
               </sup>
            </span>. It has been suggested that the infecting DENV subtype or strain influences disease
            outcome 
            <span ref-type="bibr" rid="pntd.0003416-Steel1">
               <sup>
                  <a href="#pntd.0003416-Steel1">[9]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pntd.0003416-Messer1">
               <sup>
                  <a href="#pntd.0003416-Messer1">[10]</a>
               </sup>
            </span>, but the genetic characteristics of the infecting virus are currently not taken into
            account in the management and clinical care of acutely ill patients 
            <span ref-type="bibr" rid="pntd.0003416-WHO2">
               <sup>
                  <a href="#pntd.0003416-WHO2">[6]</a>
               </sup>
            </span>.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Considering the broad spectrum of clinical presentations of dengue, as well as the
            differential diagnosis to other diseases, laboratory confirmation is essential. Fever
            is the most common presenting symptom and often coincides with viremia 
            <span ref-type="bibr" rid="pntd.0003416-Gubler1">
               <sup>
                  <a href="#pntd.0003416-Gubler1">[11]</a>
               </sup>
            </span> and non-structural protein 1 (NS1) antigenemia 
            <span ref-type="bibr" rid="pntd.0003416-Moi1">
               <sup>
                  <a href="#pntd.0003416-Moi1">[12]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pntd.0003416-Tricou1">
               <sup>
                  <a href="#pntd.0003416-Tricou1">[13]</a>
               </sup>
            </span>. NS1 is a soluble protein, and it has been shown to be present in serum for up to
            nine days after onset of fever in a proportion of DENV-1 infected patients 
            <span ref-type="bibr" rid="pntd.0003416-Alcon1">
               <sup>
                  <a href="#pntd.0003416-Alcon1">[14]</a>
               </sup>
            </span> and is therefore the target for several diagnostic tests. Ever since these assays
            were introduced, they have been widely used for early dengue diagnosis 
            <span ref-type="bibr" rid="pntd.0003416-Fry1">
               <sup>
                  <a href="#pntd.0003416-Fry1">[15]</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pntd.0003416-Muller1">
               <sup>
                  <a href="#pntd.0003416-Muller1">[17]</a>
               </sup>
            </span>. Unfortunately, recent evaluations have shown that although the NS1 antigen detection
            tests offer high specificity 
            <span ref-type="bibr" rid="pntd.0003416-Hang1">
               <sup>
                  <a href="#pntd.0003416-Hang1">[18]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pntd.0003416-Andries1">
               <sup>
                  <a href="#pntd.0003416-Andries1">[19]</a>
               </sup>
            </span>, they differ markedly in sensitivity 
            <span ref-type="bibr" rid="pntd.0003416-Hang1">
               <sup>
                  <a href="#pntd.0003416-Hang1">[18]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pntd.0003416-Bessoff1">
               <sup>
                  <a href="#pntd.0003416-Bessoff1">[20]</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pntd.0003416-Ramirez1">
               <sup>
                  <a href="#pntd.0003416-Ramirez1">[22]</a>
               </sup>
            </span>.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Serological diagnosis is usually achieved by detecting DENV-specific IgM and IgG antibodies,
            which start to appear during the early post-febrile period 
            <span ref-type="bibr" rid="pntd.0003416-Halstead1">
               <sup>
                  <a href="#pntd.0003416-Halstead1">[23]</a>
               </sup>
            </span>. A primary infection is characterized by slow and low-titer antibody responses, and
            IgM antibodies represent the dominant isotype. IgM antibodies appear by day 2 to 5
            of illness 
            <span ref-type="bibr" rid="pntd.0003416-Chanama1">
               <sup>
                  <a href="#pntd.0003416-Chanama1">[24]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pntd.0003416-Matheus1">
               <sup>
                  <a href="#pntd.0003416-Matheus1">[25]</a>
               </sup>
            </span>, whereas IgG antibodies generally become detectable at the end of the first week
            after onset of symptoms 
            <span ref-type="bibr" rid="pntd.0003416-Guzman1">
               <sup>
                  <a href="#pntd.0003416-Guzman1">[2]</a>
               </sup>
            </span>. The opposite is typically observed in secondary infections: IgM antibody levels
            are undetectable or low, and IgG antibody titers rise rapidly 
            <span ref-type="bibr" rid="pntd.0003416-Chanama1">
               <sup>
                  <a href="#pntd.0003416-Chanama1">[24]</a>
               </sup>
            </span>. Consequently, laboratory diagnosis based entirely on detection of IgM antibodies
            can give false-negative results in patients that have previously been infected with
            DENV 
            <span ref-type="bibr" rid="pntd.0003416-Chanama1">
               <sup>
                  <a href="#pntd.0003416-Chanama1">[24]</a>
               </sup>
            </span>. Unspecific reactions are also an important issue to consider in dengue serodiagnosis.
            Commercially available tests for rapid detection of IgM antibodies have been shown
            to produce false-positive results in sera positive for anti-DENV IgG, malaria, and
            rheumatoid factor 
            <span ref-type="bibr" rid="pntd.0003416-Hunsperger1">
               <sup>
                  <a href="#pntd.0003416-Hunsperger1">[26]</a>
               </sup>
            </span>, and assays detecting anti-DENV IgG antibodies can be vulnerable to cross-reactions
            caused by antibodies against related viruses 
            <span ref-type="bibr" rid="pntd.0003416-Guzman2">
               <sup>
                  <a href="#pntd.0003416-Guzman2">[3]</a>
               </sup>
            </span>.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Due to the mentioned limitations of the conventional assays, the use of molecular
            methods to detect viral genomes for early diagnosis of dengue has increased. Several
            specific and sensitive PCR-based assays have been developed during the past years
            
            <span ref-type="bibr" rid="pntd.0003416-Waggoner1">
               <sup>
                  <a href="#pntd.0003416-Waggoner1">[27]</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pntd.0003416-Huhtamo1">
               <sup>
                  <a href="#pntd.0003416-Huhtamo1">[34]</a>
               </sup>
            </span>, and one DENV serotype-specific multiplex real-time RT-PCR has been approved by the
            US Food and Drug Administration for use in clinical settings 
            <span ref-type="bibr" rid="pntd.0003416-Santiago1">
               <sup>
                  <a href="#pntd.0003416-Santiago1">[35]</a>
               </sup>
            </span>. Some of those assays are less sensitive gel-based PCR methods 
            <span ref-type="bibr" rid="pntd.0003416-SanchezSeco1">
               <sup>
                  <a href="#pntd.0003416-SanchezSeco1">[32]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pntd.0003416-Lanciotti1">
               <sup>
                  <a href="#pntd.0003416-Lanciotti1">[33]</a>
               </sup>
            </span>, whereas there are other techniques such as the multiplex real-time RT-PCR that require
            advanced thermal cyclers that detect the four specific fluorophores for which these
            systems have been validated 
            <span ref-type="bibr" rid="pntd.0003416-Waggoner2">
               <sup>
                  <a href="#pntd.0003416-Waggoner2">[28]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pntd.0003416-Johnson1">
               <sup>
                  <a href="#pntd.0003416-Johnson1">[31]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pntd.0003416-Santiago1">
               <sup>
                  <a href="#pntd.0003416-Santiago1">[35]</a>
               </sup>
            </span>. The common denominator for most of these assays is the large number of primers and/or
            probes that are needed to detect the genomes of the four DENV serotypes 
            <span ref-type="bibr" rid="pntd.0003416-Waggoner1">
               <sup>
                  <a href="#pntd.0003416-Waggoner1">[27]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pntd.0003416-Waggoner2">
               <sup>
                  <a href="#pntd.0003416-Waggoner2">[28]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pntd.0003416-Ito1">
               <sup>
                  <a href="#pntd.0003416-Ito1">[30]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pntd.0003416-Johnson1">
               <sup>
                  <a href="#pntd.0003416-Johnson1">[31]</a>
               </sup>
            </span>, which may complicate modifications of primer and probe sequences when new genetic
            variants of DENV emerge. Accordingly, we developed a new universal DENV RT-PCR assay
            that has a single fluorophore-labeled hydrolysis probe and three primers, and represents
            a system designed to target the 3′ untranslated region (UTR) of all complete genome
            sequences (n = 3,305) of the four DENV serotypes that have been published thus far.
            We used in vitro transcribed RNA, laboratory-adapted virus strains, external control
            panels, and clinical samples to evaluate the DENV RT-PCR assay, and the results showed
            that the method is highly sensitive and specific for early diagnosis of suspected
            dengue.
         </p>
      </div>
      <div id="s2">
         <h2>Methods</h2>
         <div id="s2a">
            <h3>Ethics statement and clinical specimens</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Serum samples were obtained from the biobank repository of the Public Health Agency
               of Sweden (formerly the Swedish Institute for Communicable Disease Control) as stipulated
               in the regulations for use of such material in diagnostic development and quality
               assessment. The Swedish Ethical Review Act (2003:16), Ethical Review of Research Involving
               Humans (
               <div tagxxx="ext-link">http://www.epn.se/media/45159/the_etical_review_act.pdf</div>) is not applicable, for this reason, no informed consent was required. These samples
               were used for diagnostic development and quality assessment.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">During January and February 2014, sera (n = 163) were collected from all patients
               who had returned from travel abroad showing dengue-compatible symptomatology and epidemiology.
               Information on day of disease onset and travel history was available in most cases.
               Furthermore, serum samples that had been obtained from confirmed dengue patients (n = 60)
               during the years 2011–2013 were analyzed retrospectively; the day of disease onset
               was known for these patients. Clinical specimens from patients with non-dengue diagnoses
               were collected during 2014.
            </p>
         </div>
         <div id="s2b">
            <h3>Viruses and external control panels</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The specificity of the DENV RT-PCR assay was evaluated by testing RNA extracted from
               preparations of the following: DENV-1 (strains West Pac., Hawaii, and 8356/10), DENV-2
               (strains NewGuinea C. and 4397/11), DENV-3 (strains H-87 and 3140/09), DENV-4 (strains
               H-241 and 3274/09), other human pathogenic members of the 
               <i>Flaviviridae</i> family (Japanese encephalitis virus [JEV; strain Nakayama], tick-borne encephalitis
               virus [TBEV; strains Hochosterwitz, Sofjin, and Latvia], West Nile virus [WNV; strains
               Eg101, MgAn 786/6/1995, WN_0304, and Ug_1937], Zika virus [ZIKV; strain MR766], Usutu
               virus [USUV; strain g39], and yellow fever virus [YFV; strain Asibi]), and six non-flaviviruses
               representing the three virus families 
               <i>Togaviridae</i> (chikungunya virus [CHIKV; strains 23161 and Malaysia], 
               <i>Areanaviridae</i> (Lassa virus [LASV; strain Josiah]), and 
               <i>Bunyaviridae</i> (Rift Valley fever virus [RVFV; strain ZH548], Dobrava virus [DOBV; strain H119/99],
               Hantaan virus [HTNV; strain 76–118], and Seoul virus [SEOV; strain R22]). Conditions
               for virus propagation are described in 
               <span ref-type="supplementary-material" rid="pntd.0003416.s008">S1 Text</span>. Handling of infectious material was performed in biosafety level 3 or 4 containment
               laboratories depending on classification of each virus.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Three external DENV control panels were obtained from Quality Control for Molecular
               Diagnosis (QCMD, 
               <div tagxxx="ext-link">http://www.qcmd.org/</div>) in the years 2011, 2012, and 2013. These panels consisted of 12 samples each: 10
               containing DENV serotypes 1–4 and two control samples.
            </p>
         </div>
         <div id="s2c">
            <h3>RNA extraction</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Total RNA was extracted from 140-µL aliquots of clinical specimens or supernatant
               of infected cells using a QIAamp viral RNA mini kit (Qiagen) according to the protocol
               suggested by the manufacturer. The RNA was eluted with 60 µL of elution buffer and
               stored at −80°C pending analysis. RNA extracted from supernatants of virus-infected
               cells was assayed at a dilution of 1∶10 or 1∶20.
            </p>
         </div>
         <div id="s2d">
            <h3>Design of primers and probe</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To find the most conserved region in the DENV genome, all whole genome sequences that
               were available at the design stage (November 1, 2013) were downloaded from the National
               Center for Biotechnology Information (NCBI) and used for assay design. Multiple sequence
               alignments containing all genomic sequences were created using Clustal Omega v 1.2.0
               
               <span ref-type="bibr" rid="pntd.0003416-Sievers1">
                  <sup>
                     <a href="#pntd.0003416-Sievers1">[36]</a>
                  </sup>
               </span>. Primers and probes were designed using in-house software capable of optimizing sequence
               conservation and estimating chemical properties of the oligonucleotides simultaneously.
               Melting temperatures (Tm) were verified using Primer Express v3.0 (Applied Biosystems,
               LifeTechnologies). The theoretical specificity of the system was investigated using
               BLAST against the NCBI nucleotide database.
            </p>
         </div>
         <div id="s2e">
            <h3>One-step real-time RT-PCR</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The DENV one-step real-time RT-PCR assay was carried out in 25 µL of reaction mixture
               containing 5 µL of RNA template, TaqMan Fast Virus 1-Step Master Mix (Life Technologies),
               nuclease-free H
               <div tagxxx="sub">2</div>O (Quanta), each primer at 0.9 µM, and 0.2 µM probe. The primers and probe were purchased
               from Life Technologies, and their sequences are presented in 
               <span ref-type="table" rid="pntd-0003416-t001">
                  <a href="#pntd-0003416-t001">Table 1</a>
               </span>. The minor-groove-binding (MGB)-probe was labeled with 6-carboxy fluorescein (FAM)
               reporter dye and a non-fluorescent quencher (NFQ). Amplification and detection of
               the 64-nucleotide amplicon were performed in a StepOne Plus real-time PCR system (Applied
               Biosystem, Life Technologies). Thermal cycling parameters were as follows: reverse
               transcription at 50°C for 5 min, inactivation at 95°C for 20 s followed by 40 cycles
               of fluorescence detection at 95°C for 3 s, and annealing at 60°C for 30 s. The baseline
               and threshold were set using the auto-baseline and threshold feature in StepOne Software
               v2.2.2 (Life Technologies). Samples were considered negative (not determined) if no
               target amplification was recorded after 40 cycles. Samples were tested in duplicate
               if not stated otherwise. The concentration of detected virus RNA or in vitro transcribed
               RNA was measured in genome copy equivalents (GCE)/mL. To ensure adequate RNA extraction
               from clinical samples, mRNA of human beta-actin was assessed using a TaqMan probe-based
               RT-PCR assay (Life Technologies). Dilutions and pipetting were performed using a robotic
               QIAgility workstation (Qiagen). The CDC DENV-1-4 Real-Time RT-PCR Assay was performed
               in singleplex reactions according to the manufacturer's instructions (Centers for
               Disease Control and Prevention). Amplification and detection were performed in a 7500
               Fast DX Real-time PCR instrument (Applied Biosystems).
            </p>
            <div id="pntd-0003416-t001">
               <div tagxxx="object-id">10.1371/journal.pntd.0003416.t001</div>
               <span class="label">Table 1</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Characteristics of the primers and probe targeting the 3′ UTR of DENV serotypes 1–4.</div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <colgroup>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                     </colgroup>
                     <thead>
                        <tr>
                           <td>Name</td>
                           <td>Sequence (5′-3′)</td>
                           <td>Position
                              <span ref-type="table-fn" rid="nt101">
                                 <sup>
                                    <a href="#nt101">1</a>
                                 </sup>
                              </span>
                           </td>
                           <td>Tm
                              <span ref-type="table-fn" rid="nt102">
                                 <sup>
                                    <a href="#nt102">2</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>DENV_F</td>
                           <td>
                              <div tagxxx="named-content">GCATATTGACGCTGGGARAGAC</div>
                           </td>
                           <td>10632–10653</td>
                           <td>59.7</td>
                        </tr>
                        <tr>
                           <td>DENV_R1-3</td>
                           <td>
                              <div tagxxx="named-content">TTCTGTGCCTGGAATGATGCTG</div>
                           </td>
                           <td>10674–10695</td>
                           <td>63.1</td>
                        </tr>
                        <tr>
                           <td>DENV_R4</td>
                           <td>
                              <div tagxxx="named-content">YTCTGTGCCTGGATWGATGTTG</div>
                           </td>
                           <td>10674–10695</td>
                           <td>63.1</td>
                        </tr>
                        <tr>
                           <td>DENV_P</td>
                           <td>6FAM-
                              <div tagxxx="named-content">CAGAGATCCTGCTGTC</div>-MGB(NFQ)
                           </td>
                           <td>10654–10669</td>
                           <td>67.0</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <table-wrap-foot xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div class="fn" id="nt101">
                     <span class="label">1</span>
                     <p>Genome positions are given according to the DENV-1 NCBI reference sequence (GenBank
                        acc. no. NC_001477).
                     </p>
                  </div>
                  <div class="fn" id="nt102">
                     <span class="label">2</span>
                     <p>The mean melting temperature (Tm) is shown for degenerate primers (R, A/G; W, A/T;
                        Y, G/A/C).
                     </p>
                  </div>
                  <div class="fn" id="nt103">
                     <span class="label"></span>
                     <p>6FAM = 6-carboxy fluorescein; MGB = minor-groove-binding; NFQ = non-fluorescent quencher.</p>
                  </div>
               </table-wrap-foot>
            </div>
         </div>
         <div id="s2f">
            <h3>Evaluation of RT-PCR assay performance</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The dynamic range and the efficiency of amplification of the DENV RT-PCR assay were
               determined by assaying triplicates of 10-fold serial dilutions of in vitro transcribed
               RNA (BioSynthesis Inc.). The RNA transcripts (denoted RNA[DENV_1–3] and RNA[DENV_4])
               were 88 and 84 nt, and were based on the sequences of DENV-1 (GenBank acc. no. FJ687428)
               and DENV-4 (GenBank acc. no. FJ882597) complementary to the DENV_R1–3 and DENV_R4
               primers, respectively. The sequences of the in vitro transcribed RNA can be found
               in 
               <span ref-type="supplementary-material" rid="pntd.0003416.s009">S2 Text</span>.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The limit of detection (LoD) was determined by assaying twofold serial dilutions close
               to the detection limit of the assay of in vitro transcribed RNA. The RNA transcripts
               were tested in eight replicates in three separate experiments. LoD was defined as
               the last dilution in which RNA was detected in all replicates.
            </p>
         </div>
         <div id="s2g">
            <h3>Gel-based RT-PCR and sequencing</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">DENV-specific RNA was amplified in 25-µL reaction mixtures by primer pairs 
               <div tagxxx="named-content">5′-TCAATATGCTGAAACGCGHGAG-3′</div> and 
               <div tagxxx="named-content">5′-GCGCCTTCNGNNGACATCCA-3′</div> using a OneStep RT-PCR kit (Qiagen). These primers target positions 132–153 and 764–783
               of the DENV-1 NCBI reference sequence (GenBank acc. no. NC_001477). The reaction mixtures
               were placed in a Veriti Thermal Cycler (Applied Biosystems) and were cycled at 50°C
               for 30 min, 95°C for 15 min, followed by 50 cycles of 95°C for 20 s, 50°C for 30 s,
               and 70°C for 1 min, and a final extension for 10 min at 72°C. The amplified product
               (652 nt) was analyzed by electrophoresis and purified using a QIAquick PCR Purification
               kit (Qiagen). The sequence was determined by bi-directional Sanger sequencing (GATC
               Biotech) and compared with sequences in GenBank using BLASTn.
            </p>
         </div>
         <div id="s2h">
            <h3>Antigen and antibody detection methods</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Soluble NS1 DENV antigen was detected using the immunochromatographic one-step assay
               SD Bioline Dengue NS1 Ag Rapid Test (Standard Diagnostics Inc.) according to the manufacturer's
               protocol. The presence of DENV-specific IgM and IgG antibodies in clinical serum samples
               was determined by Panbio Dengue IgM capture ELISA (Alere) and an in-house immunofluorescence
               assay (IFA) 
               <span ref-type="bibr" rid="pntd.0003416-Vene1">
                  <sup>
                     <a href="#pntd.0003416-Vene1">[37]</a>
                  </sup>
               </span>, respectively. The IgM assay was performed according to the manufacturer's guidelines,
               and a result was considered positive if&gt;11 pan-bio units (PBU) were obtained. IFA
               was performed according to a previously published protocol 
               <span ref-type="bibr" rid="pntd.0003416-Vene1">
                  <sup>
                     <a href="#pntd.0003416-Vene1">[37]</a>
                  </sup>
               </span>, and a titer of ≥40 was regarded as a positive result. The appropriate assay/assays
               were selected based on the time elapsed after onset of symptoms as reported by the
               clinician. Inasmuch as no single assay can correctly diagnose all dengue patients
               at the various times they may seek medical attention, a diagnosis of “confirmed dengue”
               was reached based on positive results in at least two of three assays (ELISA, IFA,
               or NS1 antigen detection), or a fourfold rise in IgG titers between paired sera (collected
               at least 7 days apart).
            </p>
         </div>
      </div>
      <div id="s3">
         <h2>Results/Discussion</h2>
         <div id="s3a">
            <h3>Assay design</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The DENV one-step real-time RT-PCR assay was designed to target all complete genome
               sequences of DENV serotypes 1–4 available in GenBank at the time (November 1, 2013).
               The dataset used to design the primers and probe comprised a total of 3,305 DENV genomes
               representing 1,735 DENV-1, 929 DENV-2, 517 DENV-3, and 124 DENV-4 sequences. These
               features distinguish our assay from other published PCR assays, most of which are
               developed based on a selected number of sequences or designed to detect currently
               circulating DENV strains 
               <span ref-type="bibr" rid="pntd.0003416-Waggoner1">
                  <sup>
                     <a href="#pntd.0003416-Waggoner1">[27]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pntd.0003416-Waggoner2">
                  <sup>
                     <a href="#pntd.0003416-Waggoner2">[28]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pntd.0003416-Ito1">
                  <sup>
                     <a href="#pntd.0003416-Ito1">[30]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pntd.0003416-Huhtamo1">
                  <sup>
                     <a href="#pntd.0003416-Huhtamo1">[34]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pntd.0003416-Santiago1">
                  <sup>
                     <a href="#pntd.0003416-Santiago1">[35]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pntd.0003416-Kong1">
                  <sup>
                     <a href="#pntd.0003416-Kong1">[38]</a>
                  </sup>
               </span>. However, a study showing simultaneous circulation of new and old strains has highlighted
               the importance of being able to detect diverse strains of this virus 
               <span ref-type="bibr" rid="pntd.0003416-Fahri1">
                  <sup>
                     <a href="#pntd.0003416-Fahri1">[39]</a>
                  </sup>
               </span>. Furthermore, we included 12 sylvatic DENV sequences in our dataset. This was done
               because there have been observations of spillover of sylvatic DENV to humans 
               <span ref-type="bibr" rid="pntd.0003416-Cardosa1">
                  <sup>
                     <a href="#pntd.0003416-Cardosa1">[40]</a>
                  </sup>
               </span>–
               <span ref-type="bibr" rid="pntd.0003416-Franco1">
                  <sup>
                     <a href="#pntd.0003416-Franco1">[42]</a>
                  </sup>
               </span>, which has resulted in small outbreaks or individual cases of severe dengue.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Very few regions in the DENV genome are sufficiently conserved for a universal DENV
               RT-PCR system, and the variability of the entire genome is shown in 
               <span ref-type="supplementary-material" rid="pntd.0003416.s001">S1 Figure</span> in the Supporting Information. A candidate region with relatively high sequence conservation
               was identified in the 3′ UTR genomic positions 10632–10695 (numbering relative to
               DENV-1, GenBank acc. no. NC_001477) (
               <span ref-type="supplementary-material" rid="pntd.0003416.s001">S1 Figure</span>). This region is large enough to accommodate a TaqMan system with a short MGB-modified
               probe.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The region 10654–10669 is exceptionally conserved; three of the 3,305 genomes studied
               were found to contain two mutations in this region (GenBank acc. no. JQ922554, FJ639735,
               and FJ639819), and only nine genomes contained a single mutation. One of the sequences
               with two mutations was collected in 1963 (JQ922554) and the indels in the 3′ UTR were
               not confirmed according to the authors for the remaining two sequences (FJ639735 and
               FJ639819). For this reason, and because the surrounding sequence could accommodate
               both forward and reverse primers in conserved regions when using only a few ambiguous
               nucleotides, we designed the DENV_P TaqMan probe to target the 10654–10669 region.
               DENV-4 differs from the other serotypes at positions 10670–10695. Consequently, the
               DENV_R4 primer was designed to specifically target DENV-4, whereas the DENV_R1–3 primer
               was designed to target DENV-1, -2, and -3. The primers and probe sequences and positions
               are shown in 
               <span ref-type="fig" rid="pntd-0003416-g001">Fig. 1</span>, in which deviations from the consensus sequence are indicated by vertical bars and
               percentages of the total dataset. 
               <span ref-type="supplementary-material" rid="pntd.0003416.s002">S2 Figure</span> in the Supporting Information shows the geographic and temporal distributions of
               the sequences of the four DENV serotypes and their deviations from the primer and
               probe sequences. The 3′ UTR of DENV has a complex secondary structure 
               <span ref-type="bibr" rid="pntd.0003416-Friebe1">
                  <sup>
                     <a href="#pntd.0003416-Friebe1">[43]</a>
                  </sup>
               </span>. The secondary structure at the primer and probe binding site is shown in 
               <span ref-type="supplementary-material" rid="pntd.0003416.s003">S3 Figure</span> in the Supporting Information.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pntd.0003416.g001</div>
               <span class="label">Figure 1</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Overview of the primers and probe in the DENV RT-PCR assay.</div>
                  <p>The reverse primers DENV_R1–3 (A) and DENV_R4 (B) were specifically designed to target
                     DENV serotypes 1–3 and serotype 4, respectively. Vertical bars and percentages show
                     the fraction of sequences with nucleotides deviating from the consensus of DENV serotypes
                     1–3 (A) and serotype 4 (B). Percentages below 1 are not shown. Numbers indicate genomic
                     positions.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The assay target site (length 64 nt) was matched against all non-DENV sequences in
               the NCBI nucleotide database by using BLASTn with very loose match criteria (word-size = 7,
               E-cutoff = 1,000, match/mismatch cost +1/−1, Gap cost 5/2). The hits with the lowest
               E-value were ZIKV. These ZIKV sequences (GenBank acc. no. KF268950, KF268949, and
               KF268948) matched only the forward primer and a part of the probe-binding site, hence
               it is very unlikely that any non-specific amplification will occur when using this
               assay. An alignment showing the 3′ UTR sequences of pathogenic flaviviruses is included
               in the Supporting Information, 
               <span ref-type="supplementary-material" rid="pntd.0003416.s010">S1 File</span>.
            </p>
         </div>
         <div id="s3b">
            <h3>Assay performance</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The performance of the DENV RT-PCR assay was evaluated using a known concentration
               of in vitro transcribed RNA. In this assessment, we used two RNA transcripts, RNA[DENV_1–3]
               and RNA[DENV_4], which accommodate binding for reverse primer DENV_R1–3 and DENV_R4,
               respectively.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The linear dynamic range and the efficiency of amplification were determined by assaying
               triplicates of 10-fold dilutions of transcript RNA (
               <span ref-type="fig" rid="pntd-0003416-g002">Fig. 2A</span>). The DENV RT-PCR assay could distinguish differences in concentration over a range
               of 9.6×10
               <sup>3</sup> to 9.6×10
               <sup>10</sup> GCE/mL RNA[DENV_1–3] and, similarly, 9.1×10
               <sup>3</sup> to 9.1×10
               <sup>10</sup> GCE/mL RNA[DENV_4] (
               <span ref-type="fig" rid="pntd-0003416-g002">Fig. 2A</span>). Values representing the efficiency of amplification for the corresponding template
               concentrations were 105.6% (R
               <sup>2</sup> = 0.999, y-intercept 39.4) and 102.6% (R
               <sup>2</sup> = 0.999, y-intercept 37.8), respectively (
               <span ref-type="fig" rid="pntd-0003416-g002">Fig. 2A</span>). The intra- and inter-assay coefficients of variation at 4.6×10
               <sup>3</sup> GCE/mL RNA[DENV_1–3] were 0.8% and 1.0%, respectively, and corresponding coefficients
               at 4.8×10
               <sup>3</sup> GCE/mL RNA[DENV_4] were 0.5% and 1.4%.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pntd.0003416.g002</div>
               <span class="label">Figure 2</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Dynamic range and limit of detection of the DENV RT-PCR assay.</div>
                  <p>(A) The linear dynamic range of the DENV RT-PCR assay was determined by testing triplicates
                     of 10-fold serially diluted in vitro transcribed RNA. The sequences of the transcript
                     RNA (RNA[DENV_R1–3] and RNA[DENV_R4]) were matched with the two reverse primers DENV_R1–3
                     and DENV_R4, respectively. Each dot represents the mean Cq-value from three replicates,
                     the error bars indicate the 95% confidence interval, and the lines illustrate the
                     result of the lin-log regression analysis. (B and C) Limit of detection was determined
                     by assaying eight replicates of twofold serially diluted RNA transcripts in three
                     separate experiments, and the results of testing are shown for RNA[DENV_R1–3] (B)
                     and RNA[DENV_R4] (C). Horizontal lines indicate mean values, boxes denote the 25th
                     to 75th percentiles and whiskers the 5–95% percentiles, and dots represent outliers.
                     The number of positives per total number of replicates tested is given above each
                     box. Limit of detection was defined as the last dilution in which transcript RNA was
                     detected in all 24 replicates. GCE = genome copy equivalents.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">LoD was determined empirically by testing twofold serial dilutions of transcript RNA
               in the analytical range of the expected detection limit. The DENV RT-PCR assay could
               detect all 24 replicates of 1.1×10
               <sup>3</sup> GCE/mL (corresponding to approximately 14 GCE/reaction) RNA[DENV_1–3] and 24 replicates
               of 6.0×10
               <sup>2</sup> GCE/mL (approximately 7 GCE/reaction) RNA[DENV_4] (
               <span ref-type="fig" rid="pntd-0003416-g002">Fig. 2</span>, B and C).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To determine the performance of the DENV RT-PCR assay in common matrices associated
               with dengue laboratory diagnostics 
               <span ref-type="bibr" rid="pntd.0003416-WHO2">
                  <sup>
                     <a href="#pntd.0003416-WHO2">[6]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pntd.0003416-Araujo1">
                  <sup>
                     <a href="#pntd.0003416-Araujo1">[44]</a>
                  </sup>
               </span>, we compared the slopes of the standard curves generated by amplification of RNA
               extracted from DENV spiked in human serum, plasma, and cerebrospinal fluid (CSF) with
               the slopes of such curves for RNA extracted from DENV diluted in water. Due to the
               use of serotype-specific reverse primers, we included DENV-1 and DENV-4 in the analysis.
               Depending on the type of co-extracted inhibitor, a variety of inhibition mechanisms
               can occur during the PCR process 
               <span ref-type="bibr" rid="pntd.0003416-Opel1">
                  <sup>
                     <a href="#pntd.0003416-Opel1">[45]</a>
                  </sup>
               </span>. By diluting the RNA, and thereby diluting the co-extracted inhibitor, the negative
               effect of inhibition in different matrices can be detected. With the exception of
               DENV-4 diluted in plasma (p = 0.036), the slopes of the lin-log standard curves generated
               from RNA diluted fivefold were not significantly different from the slopes obtained
               from viral RNA extracted from water (DENV-1 in serum p = 0.62, plasma p = 0.05, and
               CSF p = 0.38; DENV-4 in serum p = 0.77 and CSF p = 0.69; two-way ANOVA, three replicates),
               which indicates that, in general, PCR inhibition was minimal in samples of serum,
               plasma, and CSF.
            </p>
         </div>
         <div id="s3c">
            <h3>Assay specificity</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Three external control panels obtained from QCMD were used to evaluate the DENV RT-PCR
               assay regarding its capacity to detect all four DENV serotypes (
               <span ref-type="supplementary-material" rid="pntd.0003416.s004">S1 Table</span>). Each panel consisted of ten samples containing various concentrations of DENV (10
               <sup>3</sup> to 10
               <sup>6</sup> GCE/mL), a non-DENV flavivirus control, and one negative control. Independent of
               genome copy number, the DENV RT-PCR assay detected all samples containing DENV, and
               no unspecific reactions were observed in control samples (
               <span ref-type="supplementary-material" rid="pntd.0003416.s004">Table S1</span>).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To exclude possible cross-reaction between the DENV RT-PCR assay and human RNA, we
               analyzed 46 serum samples collected from patients with non-dengue diagnoses. None
               of these samples tested positive in the DENV RT-PCR assay (
               <span ref-type="table" rid="pntd-0003416-t002">
                  <a href="#pntd-0003416-t002">Table 2</a>
               </span>), whereas all were found to be positive for beta-actin mRNA, thus indicating adequate
               RNA extraction. To analyze the ability of the DENV RT-PCR assay to detect DENV in
               clinical samples, we tested 15 serum samples obtained from patients who had presented
               with dengue-compatible symptomatology and were confirmed to be DENV positive by sequencing.
               Of these samples, eight were positive for DENV-1, three for DENV-2, and four for DENV-3,
               and the DENV RT-PCR assay detected the DENV in all of those samples (
               <span ref-type="table" rid="pntd-0003416-t002">
                  <a href="#pntd-0003416-t002">Table 2</a>
               </span>). Unfortunately, we were unable to locate serum samples known to be positive for
               DENV-4.
            </p>
            <div id="pntd-0003416-t002">
               <div tagxxx="object-id">10.1371/journal.pntd.0003416.t002</div>
               <span class="label">Table 2</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">DENV RT-PCR assay specificity.</div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <colgroup>
                        <col></col>
                        <col></col>
                     </colgroup>
                     <thead>
                        <tr>
                           <td>Sample</td>
                           <td>No. positive (no. tested)</td>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>Clinical samples (not dengue)</td>
                           <td>0 (46)</td>
                        </tr>
                        <tr>
                           <td>DENV-1 clinical samples
                              <span ref-type="table-fn" rid="nt104">
                                 <sup>
                                    <a href="#nt104">1</a>
                                 </sup>
                              </span>
                           </td>
                           <td>8 (8)</td>
                        </tr>
                        <tr>
                           <td>DENV-2 clinical samples
                              <span ref-type="table-fn" rid="nt104">
                                 <sup>
                                    <a href="#nt104">1</a>
                                 </sup>
                              </span>
                           </td>
                           <td>3 (3)</td>
                        </tr>
                        <tr>
                           <td>DENV-3 clinical samples
                              <span ref-type="table-fn" rid="nt104">
                                 <sup>
                                    <a href="#nt104">1</a>
                                 </sup>
                              </span>
                           </td>
                           <td>4 (4)</td>
                        </tr>
                        <tr>
                           <td>DENV-1
                              <span ref-type="table-fn" rid="nt105">
                                 <sup>
                                    <a href="#nt105">2</a>
                                 </sup>
                              </span>
                           </td>
                           <td>3 (3)</td>
                        </tr>
                        <tr>
                           <td>DENV-2
                              <span ref-type="table-fn" rid="nt105">
                                 <sup>
                                    <a href="#nt105">2</a>
                                 </sup>
                              </span>
                           </td>
                           <td>2 (2)</td>
                        </tr>
                        <tr>
                           <td>DENV-3
                              <span ref-type="table-fn" rid="nt105">
                                 <sup>
                                    <a href="#nt105">2</a>
                                 </sup>
                              </span>
                           </td>
                           <td>2 (2)</td>
                        </tr>
                        <tr>
                           <td>DENV-4
                              <span ref-type="table-fn" rid="nt105">
                                 <sup>
                                    <a href="#nt105">2</a>
                                 </sup>
                              </span>
                           </td>
                           <td>2 (2)</td>
                        </tr>
                        <tr>
                           <td>Non-DENV flaviviruses
                              <span ref-type="table-fn" rid="nt105">
                                 <sup>
                                    <a href="#nt105">2</a>
                                 </sup>
                              </span>
                           </td>
                           <td>0 (11)</td>
                        </tr>
                        <tr>
                           <td>Non-flaviviruses
                              <span ref-type="table-fn" rid="nt105">
                                 <sup>
                                    <a href="#nt105">2</a>
                                 </sup>
                              </span>
                           </td>
                           <td>0 (7)</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <table-wrap-foot xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div class="fn" id="nt104">
                     <span class="label">1</span>
                     <p>The DENV serotype was determined by Sanger sequencing.</p>
                  </div>
                  <div class="fn" id="nt105">
                     <span class="label">2</span>
                     <p>Laboratory-adapted strains of flavivirus (DENV, JEV, TBEV, USUV, WNV, YFV, and ZIKV)
                        and non-flavivirus (CHIKV, DOBV, HTNV, LASV, RVFV, and SEOV) are specified in Methods.
                     </p>
                  </div>
               </table-wrap-foot>
            </div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Potential cross-reactivity with genetically related viruses was evaluated by assaying
               the nucleic acid of six human pathogenic flaviviruses, which were represented by a
               total of 11 strains (
               <span ref-type="table" rid="pntd-0003416-t002">
                  <a href="#pntd-0003416-t002">Table 2</a>
               </span>). In view of the non-specific clinical presentation and course of dengue, differential
               diagnosis of this disease should also take several other conditions into consideration
               
               <span ref-type="bibr" rid="pntd.0003416-Gibbons1">
                  <sup>
                     <a href="#pntd.0003416-Gibbons1">[46]</a>
                  </sup>
               </span>. These include infections with CHIKV and RVFV, which are etiological agents of febrile
               illnesses that have been reported to have habitats that partially overlap those of
               DENV 
               <span ref-type="bibr" rid="pntd.0003416-Singh1">
                  <sup>
                     <a href="#pntd.0003416-Singh1">[47]</a>
                  </sup>
               </span>. Accordingly, we also assessed our DENV RT-PCR method by assaying RNA of CHIKV and
               RVFV and other viruses. The primers and probe did not detect the genomes of non-DENV
               flaviviruses or non-flaviviruses (
               <span ref-type="table" rid="pntd-0003416-t002">
                  <a href="#pntd-0003416-t002">Table 2</a>
               </span>).
            </p>
         </div>
         <div id="s3d">
            <h3>Clinical performance</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We evaluated the clinical performance of the DENV RT-PCR assay by analyzing serum
               samples from residents of Sweden who presented with febrile illness after returning
               from travel to dengue-endemic regions. The serum samples were obtained from January
               to February 2014 and were tested consecutively using our DENV RT-PCR assay, the NS1
               antigen rapid test, IgM capture ELISA, and/or in-house IFA-detecting anti-DENV IgG
               antibodies (
               <span ref-type="fig" rid="pntd-0003416-g003">Fig. 3</span>). The integrity of the extraction reagents and the successful recovery of RNA from
               the analyzed patient serum samples were verified by detection of beta-actin mRNA.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pntd.0003416.g003</div>
               <span class="label">Figure 3</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Flow-chart showing the laboratory test results for samples from returning travelers
                     with dengue-compatible symptomatology.
                  </div>
                  <p>Serum samples collected during January and February 2014 were tested consecutively
                     by the newly developed DENV RT-PCR method, an NS1 antigen detection test, IgM capture
                     ELISA, and/or an in-house IFA detecting DENV-specific IgG antibodies. The criteria
                     for a positive result in each individual assay are explained in 
                     <span ref-type="sec" rid="s2">Methods</span>. The results of the laboratory analysis of the first sample arriving at the Public
                     Health Agency of Sweden are shown. w/o = without.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We used the novel DENV RT-PCR method to analyze a total of 163 serum samples from
               patients with suspected dengue, and the assay detected viral RNA in 23 of the samples
               (
               <span ref-type="fig" rid="pntd-0003416-g003">Fig. 3</span>). Ten of the samples that were positive by DENV RT-PCR contained detectable levels
               of both IgM and IgG antibodies, indicating that this assay is still useful as a diagnostic
               tool in patients presenting with anti-DENV-specific antibodies. Only 20 of the 23
               samples that were positive by the DENV RT-PCR assay were also positive by the NS1
               antigen test, indicating low sensitivity of the latter method. The three samples that
               were only positive in the DENV RT-PCR were also confirmed positive using the CDC DENV-1-4
               RT-PCR Assay. One sample that tested positive solely for NS1 antigen (faint line)
               was not positive for DENV in our assay, and the convalescence sample from this patient
               (collected more than 10 days after disease onset) did not contain detectable levels
               of anti-DENV IgG antibodies (
               <span ref-type="fig" rid="pntd-0003416-g003">Fig. 3</span>). These results strongly suggest a false-positive result in the NS1 antigen test.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Three samples tested negative for IgM antibodies but were positive for viral RNA and
               IgG antibodies (titers 40, 40, and 2,860; 
               <span ref-type="fig" rid="pntd-0003416-g003">Fig. 3</span>), which implies secondary infections or false-negative results in the IgM capture
               ELISA. Eleven serum samples were found to be positive for DENV IgM antibodies, although
               no other method could confirm an acute dengue infection. The low specificity of commercial
               IgM tests has been highlighted in a previous study in which false-positive results
               were observed in samples from both patients with other infections and healthy donors
               
               <span ref-type="bibr" rid="pntd.0003416-Blacksell1">
                  <sup>
                     <a href="#pntd.0003416-Blacksell1">[48]</a>
                  </sup>
               </span>.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To summarize, in six acutely ill patients who would not have been confirmed as having
               dengue based on analysis of their first serum sample by use of the diagnostic algorithm
               applied in our laboratory (i.e., positive in two out of three tests), the virus was
               detected by our DENV RT-PCR assay. One of these DENV RT-PCR-positive patients also
               had detectable IgG antibody levels, four were also positive for NS1 antigen but not
               IgG or IgM, and the sixth patient was negative in all other tests (
               <span ref-type="fig" rid="pntd-0003416-g003">Fig. 3</span>). These six samples were confirmed DENV positive using the CDC DENV-1-4 Real Time
               RT-PCR Assay, whereof two contained DENV-2, two contained DENV-3, and two samples
               contained DENV-4.
            </p>
         </div>
         <div id="s3e">
            <h3>Diagnostic application</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The DENV RT-PCR assay was further evaluated using 60 archived serum samples that were
               collected in 2012 and 2013 from returning travelers for whom a dengue diagnosis was
               confirmed and the day of disease onset was reported (
               <span ref-type="fig" rid="pntd-0003416-g004">Fig. 4</span>). The samples represented the following amounts of time elapsed after onset of symptoms:
               1–3 days (n = 5), 4 days (n = 10), 5 days (n = 7), 6 days (n = 11), 7 days (n = 12),
               8 days (n = 8), and 9 days (n = 7). The laboratory diagnosis was performed as described
               in 
               <span ref-type="sec" rid="s2">Methods</span> (
               <span ref-type="fig" rid="pntd-0003416-g004">Fig. 4A</span>). For the patients who had provided samples 1–3 days after disease onset, analysis
               of a second covalence sample confirmed the dengue diagnosis. To determine the approximate
               virus levels in samples collected 1–9 days after symptoms appeared, we included a
               standard containing 10
               <sup>3</sup> to 10
               <sup>8</sup> GCE/mL RNA[DENV_R1–3] (
               <span ref-type="fig" rid="pntd-0003416-g004">Fig. 4B</span>). When possible, the infecting serotypes of the DENV positive samples were determined
               using the CDC DENV-1-4 Real-Time RT-PCR Assay, for detailed information see 
               <span ref-type="supplementary-material" rid="pntd.0003416.s005">S2 Table</span> in the Supporting Information.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pntd.0003416.g004</div>
               <span class="label">Figure 4</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Results of the DENV RT-PCR assay performed on serum samples obtained 1 to 9 days after
                     onset of symptoms.
                  </div>
                  <p>(A) Archived serum samples that had been collected from 60 patients on days 1–3 (n = 5),
                     4 (n = 10), 5 (n = 7), 6 (n = 11), 7 (n = 12), 8 (n = 8), and 9 (n = 7) after disease
                     onset were tested by the DENV RT-PCR assay, an NS1 antigen detection test, IgM capture
                     ELISA, and IFA detecting DENV-specific IgG antibodies. The criteria for a positive
                     result in each of these analyses are explained in 
                     <span ref-type="sec" rid="s2">Methods</span>. The curves show the percent of samples testing positive in the individual assays
                     for the specified days after disease onset. (B) Viral load in samples collected on
                     days 1–9 after onset of symptoms. Each dot represents the mean of results for duplicate
                     samples from a single patient. GCE = genome copy equivalents.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">There was 100% positive agreement 
               <span ref-type="bibr" rid="pntd.0003416-Cicchetti1">
                  <sup>
                     <a href="#pntd.0003416-Cicchetti1">[49]</a>
                  </sup>
               </span> between the DENV RT-PCR assay and NS1 antigen detection, and the DENV RT-PCR method
               revealed viral RNA in two additional samples in which no NS1 antigen could be detected
               (
               <span ref-type="fig" rid="pntd-0003416-g004">Fig. 4</span>). Furthermore, the DENV RT-PCR assay detected viral RNA in all samples up to 5 days
               after disease onset (
               <span ref-type="fig" rid="pntd-0003416-g004">Fig. 4A</span>), and a detection rate of at least 70% has previously been reported for RT-PCR in
               this time interval 
               <span ref-type="bibr" rid="pntd.0003416-Moi1">
                  <sup>
                     <a href="#pntd.0003416-Moi1">[12]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pntd.0003416-Waggoner2">
                  <sup>
                     <a href="#pntd.0003416-Waggoner2">[28]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pntd.0003416-Lindegren1">
                  <sup>
                     <a href="#pntd.0003416-Lindegren1">[50]</a>
                  </sup>
               </span>. It is generally accepted that when DENV IgM levels become detectable, the viral
               load will have declined below measureable levels 
               <span ref-type="bibr" rid="pntd.0003416-WHO2">
                  <sup>
                     <a href="#pntd.0003416-WHO2">[6]</a>
                  </sup>
               </span>. Nevertheless, in our study, 82% and 57% of the samples collected 6 and 9 days, respectively,
               after disease onset tested positive for viral RNA by the DENV RT-PCR assay, even though
               all patients had seroconverted (IgM and/or IgG) at the indicated time points (
               <span ref-type="fig" rid="pntd-0003416-g004">Fig. 4A</span>). Similar to the results presented in 2008 by Dumoulin et al. 
               <span ref-type="bibr" rid="pntd.0003416-Dumoulin1">
                  <sup>
                     <a href="#pntd.0003416-Dumoulin1">[51]</a>
                  </sup>
               </span>, we were unable to detect DENV in samples obtained 10 days after appearance of symptoms.
            </p>
         </div>
         <div id="s3f">
            <h3>Concluding remarks</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Here we present an alternative to the more complicated multiplex RT-PCR assays for
               detection of DENV serotypes 1–4. Having the capacity to detect a wide range of genetic
               DENV variants would enable diagnostic laboratories to resolve traveler-associated
               infections in non-endemic countries and to quickly respond to outbreaks in endemic
               countries. However, many laboratories lack proper tools for molecular diagnosis of
               DENV infection 
               <span ref-type="bibr" rid="pntd.0003416-Domingo1">
                  <sup>
                     <a href="#pntd.0003416-Domingo1">[52]</a>
                  </sup>
               </span>. Although the vast majority of the patients included in the present study had been
               infected with DENV in Southeast Asia, some of them were Swedish residents returning
               from travel in Africa, the Caribbean, and South America, and they were all diagnosed
               with the virus by use of the DENV RT-PCR assay. This observation strongly indicates
               that the primers and probe used in our assay target genomes of DENV circulating in
               different regions. However, the assay has yet to be evaluated in clinical settings
               in endemic regions where other clinical profiles such as secondary infections may
               be overrepresented compared to our setting where we almost exclusively diagnose primary
               infections. Although we did not observe PCR inhibition in our clinical material, it
               can be valuable to include an inhibitory control 
               <span ref-type="bibr" rid="pntd.0003416-Dingle1">
                  <sup>
                     <a href="#pntd.0003416-Dingle1">[53]</a>
                  </sup>
               </span> if less preserved samples are to be analyzed using the DENV RT-PCR.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Finally, the DENV RT-PCR assay showed 100% positive agreement with laboratory diagnosis
               performed using NS1 antigen detection tests. Viral RNA could be detected in samples
               obtained up to 5 days after disease onset and in more than half of the samples collected
               6 to 9 days after appearance of symptoms. Hence the DENV RT-PCR assay is the method
               of choice during the first 5 days of disease, but after that period it should be supplemented
               with serology-based techniques.
            </p>
         </div>
      </div>
      <div id="s4">
         <h2>Supporting Information</h2>
         <div tagxxx="supplementary-material">
            <span class="label">S1 Figure</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>DENV genome sequence diversity.</b> Variability plot showing the percentage of sequences that deviate from the majority
                  consensus at each nucleotide position. The gray line shows per position deviations
                  and the black line shows ±10 positions unweighed moving average.
               </p>
               <p>(TIF)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S2 Figure</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>Geographic and temporal distribution of DENV 3′ UTR sequences and their deviations
                     at the DENV RT-PCR assay target site.
                  </b> A total of 3021 DENV sequences covering the entire DENV RT-PCR binding site were
                  found in the NCBI nucleotide collection (17 September 2014) of which 1125 were DENV-1
                  (A), 958 DENV-2 (B), 797 DENV-3 (C), and 141 DENV-4 (D). Perfect match (grey) corresponds
                  to no mismatches in the primers or probe. Mild mismatches (green) corresponds to one
                  or a few mismatches near the 5′ end of primers and perfect match to the probe. Moderate
                  mismatches (yellow) corresponds to one or a few mismatches in nucleotides not positioned
                  in the 3′ region of the primer and maximum one mismatch in the probe. Severe mismatch
                  (red) corresponds to one mismatch close to the 3′ end in primers, several evenly distributed
                  mismatches in the primer, or two mismatches in the probe. Fatal mismatch (blue) corresponds
                  to mismatch in the final nucleotide in the 3′ end of primers or three or more mismatches
                  in the probe. A) Of 1125 DENV-1 sequences covering the complete amplicon, seven have
                  one mismatch in the probe. Two sequences (GenBank acc. no. FJ639735 and FJ639819)
                  have two mismatches in the probe, the authors of these sequences report that the indels
                  in the 3′ UTR have not been validated. B) Of 958 DENV-2 sequences covering the complete
                  amplicon, eight have one mismatch in the probe, none of the sequences have more than
                  one mismatch. C) Of the 797 DENV-3 sequences covering the complete amplicon, one has
                  one mismatch in the probe. One sequence (GenBank acc. no. JQ922554), collection date
                  1963, has two mismatches in the probe. D) Of the 141 DENV-4 sequences covering the
                  complete amplicon, one has one mismatch in the probe. The remaining DENV1-4 sequences
                  covering the complete amplicon, presented in A–D, have perfect match with the probe.
                  The sequences included in this analysis covers the complete amplicon and includes
                  complete genomes, incomplete genomes, 3′ UTR sequences, and polyprotein sequences.
                  For the design of primers and probe only complete genome sequences were used.
               </p>
               <p>(PDF)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S3 Figure</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>Schematic illustration of the secondary structure of the DENV RT-PCR binding region
                     in the 3′ UTR.
                  </b> The structure is adapted from Friebe and Harris 2010 
                  <span ref-type="bibr" rid="pntd.0003416-Friebe1">
                     <sup>
                        <a href="#pntd.0003416-Friebe1">[43]</a>
                     </sup>
                  </span>. The binding sites of the primers and probe are highlighted.
               </p>
               <p>(TIF)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S1 Table</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>Performance of the DENV RT-PCR assay on external control panels.</b>
               </p>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S2 Table</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>Serological and RT-PCR results days 1–9 after symptom onset.</b>
               </p>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S1 Checklist</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>STARD checklist.</b>
               </p>
               <p>(DOC)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S1 Flow Diagram</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>STARD flow diagram.</b>
               </p>
               <p>(PDF)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S1 Text</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>Virus propagation.</b>
               </p>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S2 Text</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>In vitro transcribed RNA sequences.</b>
               </p>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S1 File</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>Alignment of the 3′ UTR sequences of pathogenic flaviviruses.</b> The alignment was created using Clustal Omega v 1.2.0 
                  <span ref-type="bibr" rid="pntd.0003416-Sievers1">
                     <sup>
                        <a href="#pntd.0003416-Sievers1">[36]</a>
                     </sup>
                  </span>. The following 3′ UTR sequences are included: DENV1 (GenBank acc. no. NC_001477),
                  DENV2 (GenBank acc. no. NC_001474), DENV3 (GenBank acc. no. NC_001475), DENV4 (GenBank
                  acc. no. NC_002640), JEV (GenBank acc. no. NC_001437), St. Louis encephalitis virus
                  (GenBank acc. no. NC_007580), TBEV (GenBank acc. no. NC_001672), USUV (GenBank acc.
                  no. NC_006551), WNV (GenBank acc. no. NC_001563), YFV (GenBank acc. no. NC_002031),
                  and ZIKV (GenBank acc. no. NC_012532).
               </p>
               <p>(FA)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
      </div>
      <div tagxxx="back">
         <div tagxxx="ack">
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We gratefully acknowledge M. Andersson for technical assistance and S. Vene for valuable
               scientific advice. We also thank the European Virus Archive (EVA) for providing ZIKV
               strain MR766 and DENV strains 8356/10, 4397/11, 3140/09, and 3274/09.
            </p>
         </div>
         <div tag="ref-list">
            <ul>
               <div tagxxx="title">References</div>
               <li tag="ref">
                  <a name="pntd.0003416-WHO1"></a>
                  <span class="label">1</span>
                  <div tagxxx="mixed-citation">WHO (2013) Dengue and severe dengue. Available: 
                     <div tagxxx="ext-link">http://www.who.int/mediacentre/factsheets/fs117/en/</div>. Accessed 30 May 2014.
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Guzman1"></a>
                  <span class="label">2</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Guzman</span>
                        <span tagx="given-names">MG</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Halstead</span>
                        <span tagx="given-names">SB</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Artsob</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Buchy</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Farrar</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">Dengue: a continuing global threat</span>. 
                     <span tagx="source">Nat Rev Microbiol</span>
                     <span tagx="volume">8</span>: 
                     <span tagx="fpage">S7</span>–
                     <span tagx="lpage">16</span>.
                     <span class="pub-id'">[21079655]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Guzman2"></a>
                  <span class="label">3</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Guzman</span>
                        <span tagx="given-names">MG</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kouri</span>
                        <span tagx="given-names">G</span>,
                     </span> (
                     <span tagx="year">2004</span>) 
                     <span tagx="article-title">Dengue diagnosis, advances and challenges</span>. 
                     <span tagx="source">Int J Infect Dis</span>
                     <span tagx="volume">8</span>: 
                     <span tagx="fpage">69</span>–
                     <span tagx="lpage">80</span>.
                     <span class="pub-id'">[14732325]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Low1"></a>
                  <span class="label">4</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Low</span>
                        <span tagx="given-names">JG</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ong</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tan</span>
                        <span tagx="given-names">LK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chaterji</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chow</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">The early clinical features of dengue in adults: challenges for early clinical diagnosis</span>. 
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="volume">5</span>: 
                     <span tagx="fpage">e1191</span>.
                     <span class="pub-id'">[21655307]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Alexander1"></a>
                  <span class="label">5</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Alexander</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Balmaseda</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Coelho</span>
                        <span tagx="given-names">IC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Dimaano</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hien</span>
                        <span tagx="given-names">TT</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">Multicentre prospective study on dengue classification in four South-east Asian and
                        three Latin American countries
                     </span>. 
                     <span tagx="source">Trop Med Int Health</span>
                     <span tagx="volume">16</span>: 
                     <span tagx="fpage">936</span>–
                     <span tagx="lpage">948</span>.
                     <span class="pub-id'">[21624014]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-WHO2"></a>
                  <span class="label">6</span>
                  <div tagxxx="mixed-citation">WHO (2009) Dengue: Guidlines for Diagnosis, Treatment, Prevention adn Control. Geneva:
                     WHO. 
                     <div tagxxx="ext-link">http://whqlibdoc.who.int/publications/2009/9789241547871_eng.pdf?ua=1</div>. Accessed 30 May 2014.
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Zompi1"></a>
                  <span class="label">7</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Zompi</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Montoya</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Pohl</span>
                        <span tagx="given-names">MO</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Balmaseda</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Harris</span>
                        <span tagx="given-names">E</span>,
                     </span> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Dominant cross-reactive B cell response during secondary acute dengue virus infection
                        in humans
                     </span>. 
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="volume">6</span>: 
                     <span tagx="fpage">e1568</span>.
                     <span class="pub-id'">[22448292]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Martina1"></a>
                  <span class="label">8</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Martina</span>
                        <span tagx="given-names">BE</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Koraka</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Osterhaus</span>
                        <span tagx="given-names">AD</span>,
                     </span> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Dengue virus pathogenesis: an integrated view</span>. 
                     <span tagx="source">Clin Microbiol Rev</span>
                     <span tagx="volume">22</span>: 
                     <span tagx="fpage">564</span>–
                     <span tagx="lpage">581</span>.
                     <span class="pub-id'">[19822889]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Steel1"></a>
                  <span class="label">9</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Steel</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gubler</span>
                        <span tagx="given-names">DJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Bennett</span>
                        <span tagx="given-names">SN</span>,
                     </span> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">Natural attenuation of dengue virus type-2 after a series of island outbreaks: a retrospective
                        phylogenetic study of events in the South Pacific three decades ago
                     </span>. 
                     <span tagx="source">Virology</span>
                     <span tagx="volume">405</span>: 
                     <span tagx="fpage">505</span>–
                     <span tagx="lpage">512</span>.
                     <span class="pub-id'">[20663532]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Messer1"></a>
                  <span class="label">10</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Messer</span>
                        <span tagx="given-names">WB</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gubler</span>
                        <span tagx="given-names">DJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Harris</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sivananthan</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">de Silva</span>
                        <span tagx="given-names">AM</span>,
                     </span> (
                     <span tagx="year">2003</span>) 
                     <span tagx="article-title">Emergence and global spread of a dengue serotype 3, subtype III virus</span>. 
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="volume">9</span>: 
                     <span tagx="fpage">800</span>–
                     <span tagx="lpage">809</span>.
                     <span class="pub-id'">[12899133]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Gubler1"></a>
                  <span class="label">11</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gubler</span>
                        <span tagx="given-names">DJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Suharyono</span>
                        <span tagx="given-names">W</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tan</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Abidin</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sie</span>
                        <span tagx="given-names">A</span>,
                     </span> (
                     <span tagx="year">1981</span>) 
                     <span tagx="article-title">Viraemia in patients with naturally acquired dengue infection</span>. 
                     <span tagx="source">Bull World Health Organ</span>
                     <span tagx="volume">59</span>: 
                     <span tagx="fpage">623</span>–
                     <span tagx="lpage">630</span>.
                     <span class="pub-id'">[6976230]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Moi1"></a>
                  <span class="label">12</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Moi</span>
                        <span tagx="given-names">ML</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Omatsu</span>
                        <span tagx="given-names">T</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tajima</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lim</span>
                        <span tagx="given-names">CK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kotaki</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">Detection of dengue virus nonstructural protein 1 (NS1) by using ELISA as a useful
                        laboratory diagnostic method for dengue virus infection of international travelers
                     </span>. 
                     <span tagx="source">J Travel Med</span>
                     <span tagx="volume">20</span>: 
                     <span tagx="fpage">185</span>–
                     <span tagx="lpage">193</span>.
                     <span class="pub-id'">[23577865]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Tricou1"></a>
                  <span class="label">13</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Tricou</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vu</span>
                        <span tagx="given-names">HT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Quynh</span>
                        <span tagx="given-names">NV</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nguyen</span>
                        <span tagx="given-names">CV</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tran</span>
                        <span tagx="given-names">HT</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">Comparison of two dengue NS1 rapid tests for sensitivity, specificity and relationship
                        to viraemia and antibody responses
                     </span>. 
                     <span tagx="source">BMC Infect Dis</span>
                     <span tagx="volume">10</span>: 
                     <span tagx="fpage">142</span>.
                     <span class="pub-id'">[20509940]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Alcon1"></a>
                  <span class="label">14</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Alcon</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Talarmin</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Debruyne</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Falconar</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Deubel</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2002</span>) 
                     <span tagx="article-title">Enzyme-linked immunosorbent assay specific to Dengue virus type 1 nonstructural protein
                        NS1 reveals circulation of the antigen in the blood during the acute phase of disease
                        in patients experiencing primary or secondary infections
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">40</span>: 
                     <span tagx="fpage">376</span>–
                     <span tagx="lpage">381</span>.
                     <span class="pub-id'">[11825945]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Fry1"></a>
                  <span class="label">15</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Fry</span>
                        <span tagx="given-names">SR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Meyer</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Semple</span>
                        <span tagx="given-names">MG</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Simmons</span>
                        <span tagx="given-names">CP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sekaran</span>
                        <span tagx="given-names">SD</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">The diagnostic sensitivity of dengue rapid test assays is significantly enhanced by
                        using a combined antigen and antibody testing approach
                     </span>. 
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="volume">5</span>: 
                     <span tagx="fpage">e1199</span>.
                     <span class="pub-id'">[21713023]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Chaiyaratana1"></a>
                  <span class="label">16</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Chaiyaratana</span>
                        <span tagx="given-names">W</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chuansumrit</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Pongthanapisith</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tangnararatchakit</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lertwongrath</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Evaluation of dengue nonstructural protein 1 antigen strip for the rapid diagnosis
                        of patients with dengue infection
                     </span>. 
                     <span tagx="source">Diagn Microbiol Infect Dis</span>
                     <span tagx="volume">64</span>: 
                     <span tagx="fpage">83</span>–
                     <span tagx="lpage">84</span>.
                     <span class="pub-id'">[19232857]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Muller1"></a>
                  <span class="label">17</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Muller</span>
                        <span tagx="given-names">DA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Young</span>
                        <span tagx="given-names">PR</span>,
                     </span> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">The flavivirus NS1 protein: molecular and structural biology, immunology, role in
                        pathogenesis and application as a diagnostic biomarker
                     </span>. 
                     <span tagx="source">Antiviral Res</span>
                     <span tagx="volume">98</span>: 
                     <span tagx="fpage">192</span>–
                     <span tagx="lpage">208</span>.
                     <span class="pub-id'">[23523765]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Hang1"></a>
                  <span class="label">18</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Hang</span>
                        <span tagx="given-names">VT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nguyet</span>
                        <span tagx="given-names">NM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Trung</span>
                        <span tagx="given-names">DT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tricou</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Yoksan</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Diagnostic accuracy of NS1 ELISA and lateral flow rapid tests for dengue sensitivity,
                        specificity and relationship to viraemia and antibody responses
                     </span>. 
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="volume">3</span>: 
                     <span tagx="fpage">e360</span>.
                     <span class="pub-id'">[19156192]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Andries1"></a>
                  <span class="label">19</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Andries</span>
                        <span tagx="given-names">AC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Duong</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ngan</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ong</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Huy</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Field evaluation and impact on clinical management of a rapid diagnostic kit that
                        detects dengue NS1, IgM and IgG
                     </span>. 
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="volume">6</span>: 
                     <span tagx="fpage">e1993</span>.
                     <span class="pub-id'">[23301110]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Bessoff1"></a>
                  <span class="label">20</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Bessoff</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Delorey</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sun</span>
                        <span tagx="given-names">W</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hunsperger</span>
                        <span tagx="given-names">E</span>,
                     </span> (
                     <span tagx="year">2008</span>) 
                     <span tagx="article-title">Comparison of two commercially available dengue virus (DENV) NS1 capture enzyme-linked
                        immunosorbent assays using a single clinical sample for diagnosis of acute DENV infection
                     </span>. 
                     <span tagx="source">Clin Vaccine Immunol</span>
                     <span tagx="volume">15</span>: 
                     <span tagx="fpage">1513</span>–
                     <span tagx="lpage">1518</span>.
                     <span class="pub-id'">[18685015]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Colombo1"></a>
                  <span class="label">21</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Colombo</span>
                        <span tagx="given-names">TE</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vedovello</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Araki</span>
                        <span tagx="given-names">CS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Cogo-Moreira</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">dos Santos</span>
                        <span tagx="given-names">IN</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">Dengue-4 false negative results by Panbio(R) Dengue Early ELISA assay in Brazil</span>. 
                     <span tagx="source">J Clin Virol</span>
                     <span tagx="volume">58</span>: 
                     <span tagx="fpage">710</span>–
                     <span tagx="lpage">712</span>.
                     <span class="pub-id'">[24238889]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Ramirez1"></a>
                  <span class="label">22</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Ramirez</span>
                        <span tagx="given-names">AH</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Moros</span>
                        <span tagx="given-names">Z</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Comach</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Zambrano</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Bravo</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Evaluation of dengue NS1 antigen detection tests with acute sera from patients infected
                        with dengue virus in Venezuela
                     </span>. 
                     <span tagx="source">Diagn Microbiol Infect Dis</span>
                     <span tagx="volume">65</span>: 
                     <span tagx="fpage">247</span>–
                     <span tagx="lpage">253</span>.
                     <span class="pub-id'">[19733994]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Halstead1"></a>
                  <span class="label">23</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Halstead</span>
                        <span tagx="given-names">SB</span>,
                     </span> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">Dengue</span>. 
                     <span tagx="source">Lancet</span>
                     <span tagx="volume">370</span>: 
                     <span tagx="fpage">1644</span>–
                     <span tagx="lpage">1652</span>.
                     <span class="pub-id'">[17993365]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Chanama1"></a>
                  <span class="label">24</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Chanama</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Anantapreecha</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">A</span>
                        <span tagx="given-names">An</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sa-gnasang</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kurane</span>
                        <span tagx="given-names">I</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2004</span>) 
                     <span tagx="article-title">Analysis of specific IgM responses in secondary dengue virus infections: levels and
                        positive rates in comparison with primary infections
                     </span>. 
                     <span tagx="source">J Clin Virol</span>
                     <span tagx="volume">31</span>: 
                     <span tagx="fpage">185</span>–
                     <span tagx="lpage">189</span>.
                     <span class="pub-id'">[15465410]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Matheus1"></a>
                  <span class="label">25</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Matheus</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Pham</span>
                        <span tagx="given-names">TB</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Labeau</span>
                        <span tagx="given-names">B</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Huong</span>
                        <span tagx="given-names">VT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lacoste</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2014</span>) 
                     <span tagx="article-title">Kinetics of Dengue Non-Structural Protein 1 Antigen and IgM and IgA Antibodies in
                        Capillary Blood Samples from Confirmed Dengue Patients
                     </span>. 
                     <span tagx="source">Am J Trop Med Hyg</span>
                     <span tagx="volume">90</span>: 
                     <span tagx="fpage">438</span>–
                     <span tagx="lpage">443</span>.
                     <span class="pub-id'">[24470561]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Hunsperger1"></a>
                  <span class="label">26</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Hunsperger</span>
                        <span tagx="given-names">EA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Yoksan</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Buchy</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nguyen</span>
                        <span tagx="given-names">VC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sekaran</span>
                        <span tagx="given-names">SD</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Evaluation of commercially available anti-dengue virus immunoglobulin M tests</span>. 
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="volume">15</span>: 
                     <span tagx="fpage">436</span>–
                     <span tagx="lpage">440</span>.
                     <span class="pub-id'">[19239758]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Waggoner1"></a>
                  <span class="label">27</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Waggoner</span>
                        <span tagx="given-names">JJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Abeynayake</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sahoo</span>
                        <span tagx="given-names">MK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gresh</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tellez</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">Development of an internally controlled real-time reverse transcriptase PCR assay
                        for pan-dengue virus detection and comparison of four molecular dengue virus detection
                        assays
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">51</span>: 
                     <span tagx="fpage">2172</span>–
                     <span tagx="lpage">2181</span>.
                     <span class="pub-id'">[23637298]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Waggoner2"></a>
                  <span class="label">28</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Waggoner</span>
                        <span tagx="given-names">JJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Abeynayake</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sahoo</span>
                        <span tagx="given-names">MK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gresh</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tellez</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">Single-reaction, multiplex, real-time rt-PCR for the detection, quantitation, and
                        serotyping of dengue viruses
                     </span>. 
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="volume">7</span>: 
                     <span tagx="fpage">e2116</span>.
                     <span class="pub-id'">[23638191]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Das1"></a>
                  <span class="label">29</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Das</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Pingle</span>
                        <span tagx="given-names">MR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Munoz-Jordan</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rundell</span>
                        <span tagx="given-names">MS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rondini</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2008</span>) 
                     <span tagx="article-title">Detection and serotyping of dengue virus in serum samples by multiplex reverse transcriptase
                        PCR-ligase detection reaction assay
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">46</span>: 
                     <span tagx="fpage">3276</span>–
                     <span tagx="lpage">3284</span>.
                     <span class="pub-id'">[18685000]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Ito1"></a>
                  <span class="label">30</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Ito</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Takasaki</span>
                        <span tagx="given-names">T</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Yamada</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nerome</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tajima</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2004</span>) 
                     <span tagx="article-title">Development and evaluation of fluorogenic TaqMan reverse transcriptase PCR assays
                        for detection of dengue virus types 1 to 4
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">42</span>: 
                     <span tagx="fpage">5935</span>–
                     <span tagx="lpage">5937</span>.
                     <span class="pub-id'">[15583346]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Johnson1"></a>
                  <span class="label">31</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Johnson</span>
                        <span tagx="given-names">BW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Russell</span>
                        <span tagx="given-names">BJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lanciotti</span>
                        <span tagx="given-names">RS</span>,
                     </span> (
                     <span tagx="year">2005</span>) 
                     <span tagx="article-title">Serotype-specific detection of dengue viruses in a fourplex real-time reverse transcriptase
                        PCR assay
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">43</span>: 
                     <span tagx="fpage">4977</span>–
                     <span tagx="lpage">4983</span>.
                     <span class="pub-id'">[16207951]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-SanchezSeco1"></a>
                  <span class="label">32</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Sanchez-Seco</span>
                        <span tagx="given-names">MP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rosario</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hernandez</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Domingo</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Valdes</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">Detection and subtyping of dengue 1–4 and yellow fever viruses by means of a multiplex
                        RT-nested-PCR using degenerated primers
                     </span>. 
                     <span tagx="source">Trop Med Int Health</span>
                     <span tagx="volume">11</span>: 
                     <span tagx="fpage">1432</span>–
                     <span tagx="lpage">1441</span>.
                     <span class="pub-id'">[16930266]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Lanciotti1"></a>
                  <span class="label">33</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Lanciotti</span>
                        <span tagx="given-names">RS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Calisher</span>
                        <span tagx="given-names">CH</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gubler</span>
                        <span tagx="given-names">DJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chang</span>
                        <span tagx="given-names">GJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vorndam</span>
                        <span tagx="given-names">AV</span>,
                     </span> (
                     <span tagx="year">1992</span>) 
                     <span tagx="article-title">Rapid detection and typing of dengue viruses from clinical samples by using reverse
                        transcriptase-polymerase chain reaction
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">30</span>: 
                     <span tagx="fpage">545</span>–
                     <span tagx="lpage">551</span>.
                     <span class="pub-id'">[1372617]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Huhtamo1"></a>
                  <span class="label">34</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Huhtamo</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hasu</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Uzcategui</span>
                        <span tagx="given-names">NY</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Erra</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nikkari</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">Early diagnosis of dengue in travelers: comparison of a novel real-time RT-PCR, NS1
                        antigen detection and serology
                     </span>. 
                     <span tagx="source">J Clin Virol</span>
                     <span tagx="volume">47</span>: 
                     <span tagx="fpage">49</span>–
                     <span tagx="lpage">53</span>.
                     <span class="pub-id'">[19963435]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Santiago1"></a>
                  <span class="label">35</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Santiago</span>
                        <span tagx="given-names">GA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vergne</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Quiles</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Cosme</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vazquez</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">Analytical and clinical performance of the CDC real time RT-PCR assay for detection
                        and typing of dengue virus
                     </span>. 
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="volume">7</span>: 
                     <span tagx="fpage">e2311</span>.
                     <span class="pub-id'">[23875046]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Sievers1"></a>
                  <span class="label">36</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Sievers</span>
                        <span tagx="given-names">F</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Wilm</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Dineen</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gibson</span>
                        <span tagx="given-names">TJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Karplus</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using
                        Clustal Omega
                     </span>. 
                     <span tagx="source">Mol Syst Biol</span>
                     <span tagx="volume">7</span>: 
                     <span tagx="fpage">539</span>.
                     <span class="pub-id'">[21988835]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Vene1"></a>
                  <span class="label">37</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Vene</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Mangiafico</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Niklasson</span>
                        <span tagx="given-names">B</span>,
                     </span> (
                     <span tagx="year">1995</span>) 
                     <span tagx="article-title">Indirect immunofluorescence for serological diagnosis of dengue virus infections in
                        Swedish patients
                     </span>. 
                     <span tagx="source">Clin Diagn Virol</span>
                     <span tagx="volume">4</span>: 
                     <span tagx="fpage">43</span>–
                     <span tagx="lpage">50</span>.
                     <span class="pub-id'">[15566826]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Kong1"></a>
                  <span class="label">38</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Kong</span>
                        <span tagx="given-names">YY</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Thay</span>
                        <span tagx="given-names">CH</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tin</span>
                        <span tagx="given-names">TC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Devi</span>
                        <span tagx="given-names">S</span>,
                     </span> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">Rapid detection, serotyping and quantitation of dengue viruses by TaqMan real-time
                        one-step RT-PCR
                     </span>. 
                     <span tagx="source">J Virol Methods</span>
                     <span tagx="volume">138</span>: 
                     <span tagx="fpage">123</span>–
                     <span tagx="lpage">130</span>.
                     <span class="pub-id'">[17000012]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Fahri1"></a>
                  <span class="label">39</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Fahri</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Yohan</span>
                        <span tagx="given-names">B</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Trimarsanto</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sayono</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hadisaputro</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">Molecular surveillance of dengue in Semarang, Indonesia revealed the circulation of
                        an old genotype of dengue virus serotype-1
                     </span>. 
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="volume">7</span>: 
                     <span tagx="fpage">e2354</span>.
                     <span class="pub-id'">[23951374]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Cardosa1"></a>
                  <span class="label">40</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Cardosa</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ooi</span>
                        <span tagx="given-names">MH</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tio</span>
                        <span tagx="given-names">PH</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Perera</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">EC</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Dengue virus serotype 2 from a sylvatic lineage isolated from a patient with dengue
                        hemorrhagic fever
                     </span>. 
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="volume">3</span>: 
                     <span tagx="fpage">e423</span>.
                     <span class="pub-id'">[19399166]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Vasilakis1"></a>
                  <span class="label">41</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Vasilakis</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tesh</span>
                        <span tagx="given-names">RB</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Weaver</span>
                        <span tagx="given-names">SC</span>,
                     </span> (
                     <span tagx="year">2008</span>) 
                     <span tagx="article-title">Sylvatic dengue virus type 2 activity in humans, Nigeria, 1966</span>. 
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="volume">14</span>: 
                     <span tagx="fpage">502</span>–
                     <span tagx="lpage">504</span>.
                     <span class="pub-id'">[18325274]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Franco1"></a>
                  <span class="label">42</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Franco</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Palacios</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Martinez</span>
                        <span tagx="given-names">JA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vazquez</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Savji</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">First report of sylvatic DENV-2-associated dengue hemorrhagic fever in West Africa</span>. 
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="volume">5</span>: 
                     <span tagx="fpage">e1251</span>.
                     <span class="pub-id'">[21829739]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Friebe1"></a>
                  <span class="label">43</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Friebe</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Harris</span>
                        <span tagx="given-names">E</span>,
                     </span> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">Interplay of RNA elements in the dengue virus 5′ and 3′ ends required for viral RNA
                        replication
                     </span>. 
                     <span tagx="source">J Virol</span>
                     <span tagx="volume">84</span>: 
                     <span tagx="fpage">6103</span>–
                     <span tagx="lpage">6118</span>.
                     <span class="pub-id'">[20357095]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Araujo1"></a>
                  <span class="label">44</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Araujo</span>
                        <span tagx="given-names">F</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nogueira</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Araujo Mde</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Perdigao</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Cavalcanti</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Dengue in patients with central nervous system manifestations, Brazil</span>. 
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="volume">18</span>: 
                     <span tagx="fpage">677</span>–
                     <span tagx="lpage">679</span>.
                     <span class="pub-id'">[22469335]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Opel1"></a>
                  <span class="label">45</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Opel</span>
                        <span tagx="given-names">KL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chung</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">McCord</span>
                        <span tagx="given-names">BR</span>,
                     </span> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">A study of PCR inhibition mechanisms using real time PCR</span>. 
                     <span tagx="source">J Forensic Sci</span>
                     <span tagx="volume">55</span>: 
                     <span tagx="fpage">25</span>–
                     <span tagx="lpage">33</span>.
                     <span class="pub-id'">[20015162]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Gibbons1"></a>
                  <span class="label">46</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gibbons</span>
                        <span tagx="given-names">RV</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vaughn</span>
                        <span tagx="given-names">DW</span>,
                     </span> (
                     <span tagx="year">2002</span>) 
                     <span tagx="article-title">Dengue: an escalating problem</span>. 
                     <span tagx="source">BMJ</span>
                     <span tagx="volume">324</span>: 
                     <span tagx="fpage">1563</span>–
                     <span tagx="lpage">1566</span>.
                     <span class="pub-id'">[12089096]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Singh1"></a>
                  <span class="label">47</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Singh</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Mittal</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rizvi</span>
                        <span tagx="given-names">MM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chhabra</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sharma</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">The first dominant co-circulation of both dengue and chikungunya viruses during the
                        post-monsoon period of 2010 in Delhi, India
                     </span>. 
                     <span tagx="source">Epidemiol Infect</span>
                     <span tagx="volume">140</span>: 
                     <span tagx="fpage">1337</span>–
                     <span tagx="lpage">1342</span>.
                     <span class="pub-id'">[21906409]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Blacksell1"></a>
                  <span class="label">48</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Blacksell</span>
                        <span tagx="given-names">SD</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Jarman</span>
                        <span tagx="given-names">RG</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gibbons</span>
                        <span tagx="given-names">RV</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tanganuchitcharnchai</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Mammen</span>
                        <span tagx="given-names">MP</span>
                        <div tagxxx="suffix">Jr</div>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Comparison of seven commercial antigen and antibody enzyme-linked immunosorbent assays
                        for detection of acute dengue infection
                     </span>. 
                     <span tagx="source">Clin Vaccine Immunol</span>
                     <span tagx="volume">19</span>: 
                     <span tagx="fpage">804</span>–
                     <span tagx="lpage">810</span>.
                     <span class="pub-id'">[22441389]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Cicchetti1"></a>
                  <span class="label">49</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Cicchetti</span>
                        <span tagx="given-names">DV</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Feinstein</span>
                        <span tagx="given-names">AR</span>,
                     </span> (
                     <span tagx="year">1990</span>) 
                     <span tagx="article-title">High agreement but low kappa: II. Resolving the paradoxes</span>. 
                     <span tagx="source">J Clin Epidemiol</span>
                     <span tagx="volume">43</span>: 
                     <span tagx="fpage">551</span>–
                     <span tagx="lpage">558</span>.
                     <span class="pub-id'">[2189948]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Lindegren1"></a>
                  <span class="label">50</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Lindegren</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vene</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lundkvist</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Falk</span>
                        <span tagx="given-names">KI</span>,
                     </span> (
                     <span tagx="year">2005</span>) 
                     <span tagx="article-title">Optimized diagnosis of acute dengue fever in Swedish travelers by a combination of
                        reverse transcription-PCR and immunoglobulin M detection
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">43</span>: 
                     <span tagx="fpage">2850</span>–
                     <span tagx="lpage">2855</span>.
                     <span class="pub-id'">[15956408]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Dumoulin1"></a>
                  <span class="label">51</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Dumoulin</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Marti</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Panning</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hatz</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hirsch</span>
                        <span tagx="given-names">HH</span>,
                     </span> (
                     <span tagx="year">2008</span>) 
                     <span tagx="article-title">Pan-dengue virus detection by PCR for travelers returning from the tropics</span>. 
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">46</span>: 
                     <span tagx="fpage">3104</span>–
                     <span tagx="lpage">3106</span>.
                     <span class="pub-id'">[18650359]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Domingo1"></a>
                  <span class="label">52</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Domingo</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Niedrig</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Teichmann</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kaiser</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rumer</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">2nd International external quality control assessment for the molecular diagnosis
                        of dengue infections
                     </span>. 
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="volume">4</span>
                     <div tagxxx="comment">DOI: 
                        <div tagxxx="ext-link">10.1371/journal.pntd.0000833</div>
                     </div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0003416-Dingle1"></a>
                  <span class="label">53</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Dingle</span>
                        <span tagx="given-names">KE</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Crook</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Jeffery</span>
                        <span tagx="given-names">K</span>,
                     </span> (
                     <span tagx="year">2004</span>) 
                     <span tagx="article-title">Stable and Noncompetitive RNA Internal Control for Routine Clinical Diagnostic Reverse
                        Transcription-PCR
                     </span>. 
                     <span tagx="source">Clin Microbiol</span>
                     <span tagx="volume">42</span>: 
                     <span tagx="fpage">1003</span>–
                     <span tagx="lpage">1011</span>.
                  </div>
               </li>
            </ul>
         </div>
      </div>
   </div>
</html>