<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <div xmlns="" tagx="article">
      <head>
         <style type="text/css"> table, tr, td {border : thin;}</style>
         <span class="citation_publisher">Public Library of Science</span>
         <h2 class="citation-title">First International External Quality Assessment Study on Molecular and Serological
            Methods for Yellow Fever Diagnosis
         </h2>
         <span class="doi">10.1371/journal.pone.0036291</span>
         <div class="contrib-group">contrib: 
            	  CONT
            	  
            <span class="citation_author">Cristina Domingo</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Camille Escadafal</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Leonid Rumer</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Jairo A. Méndez</span>
            <span class="citation_author_institution_ref">2</span>
            <span class="citation_author">Paquita García</span>
            <span class="citation_author_institution_ref">3</span>
            <span class="citation_author">Amadou A. Sall</span>
            <span class="citation_author_institution_ref">4</span>
            <span class="citation_author">Anette Teichmann</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Oliver Donoso-Mantke</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Matthias Niedrig</span>
            <span class="citation_author_institution_ref">1</span>
            	  CONT
            	  
            <div tagxxx="contrib">
               <span class="name'">
                  <span tagx="surname">Coffey</span>
                  <span tagx="given-names">Lark L.</span>,
               </span>
               <div tagxxx="role">Editor</div>
               <span ref-type="aff" rid="edit1"></span>
            </div>
         </div>
         <span class="email">[Domingo-CarrascoC@rki.de]</span>
         <span class="conflict">[]</span>
         <span class="contribs">[Conceived and designed the experiments: CD MN. Performed the experiments: CD CE AT.
            Analyzed the data: CD LR ODM MN. Contributed reagents/materials/analysis tools: JAM
            PG AAS. Wrote the paper: CD.]
         </span>
         <span class="citation_pubdate">[2012-5-3]</span>
         <span class="citation_copyright">[Copyright 2012: ]</span>
         <span class="citation_licence">[Licence ]</span>
         <span class="citation_funding">[]</span>
         <span class="citation_pagecount">[]</span>
         <span class="citation_data">[]</span>
      </head>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <div id="">
            <h2>Objective</h2>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We describe an external quality assurance (EQA) study designed to assess the efficiency
               and accurateness of molecular and serological methods used by expert laboratories
               performing YF diagnosis.
            </p>
         </div>
         <div id="">
            <h2>Study Design</h2>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">For molecular diagnosis evaluation, a panel was prepared of 14 human plasma samples
               containing specific RNA of different YFV strains (YFV-17D, YFV South American strain
               [Brazil], YFV IvoryC1999 strain), and specificity samples containing other flaviviruses
               and negative controls. For the serological panel, 13 human plasma samples with anti-YFV-specific
               antibodies against different strains of YFV (YFV-17D strain, YFV IvoryC1999 strain,
               and YFV Brazilian strain), as well as specificity and negative controls, were included.
            </p>
         </div>
         <div id="">
            <h2>Results</h2>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Thirty-six laboratories from Europe, the Americas, Middle East, and Africa participated
               in these EQA activities. Only 16% of the analyses reported met all evaluation criteria
               with optimal performance. Serial dilutions of YFV-17D showed that in general the methodologies
               reported provided a suitable sensitivity. Failures were mainly due to the inability
               to detect wild-type strains or the presence of false positives. Performance in the
               serological diagnosis varied, mainly depending on the methodology used. Anti-YFV IgM
               detection was not performed in 16% of the reports using IIF or ELISA techniques, although
               it is preferable for the diagnosis of YFV acute infections. A good sensitivity profile
               was achieved in general; however, in the detection of IgM antibodies a lack of sensitivity
               of anti-YFV antibodies against the vaccine strain 17D was observed, and of the anti-YFV
               IgG antibodies against a West African strain. Neutralization assays showed a very
               good performance; however, the unexpected presence of false positives underlined the
               need of improving the running protocols.
            </p>
         </div>
         <div id="">
            <h2>Conclusion</h2>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">This EQA provides information on each laboratory's efficacy of RT-PCR and serological
               YFV diagnosis techniques. The results indicate the need for improving serological
               and molecular diagnosis techniques and provide a follow-up of the diagnostic profiles.
            </p>
         </div>
      </div>
      <div id="s1">
         <h2>Introduction</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Yellow fever (YF) is a viral disease which is symptomatic in about 5–20% of infected
            people. In its mildest form YF is characterized by non-specific fever and headache,
            while the severe form of the disease is characterized by high fever, jaundice, bleeding,
            and eventually shock and multiple organ failure. Among those who develop severe disease
            20–50% may die. To date, there are no antivirals for specific treatment of the infection,
            and YF vaccination of the population at risk is the best approach to prevent and control
            the disease 
            <span ref-type="bibr" rid="pone.0036291-WHO1">
               <sup>
                  <a href="#pone.0036291-WHO1">[1]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pone.0036291-Monath1">
               <sup>
                  <a href="#pone.0036291-Monath1">[2]</a>
               </sup>
            </span>.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Yellow fever virus (YFV) is transmitted to humans by bites of infected mosquitoes.
            In the jungle setting, mosquitoes and monkeys maintain the virus 
            <i>via</i> an enzootic cycle. The virus can also be transmitted vertically from female mosquitoes
            to eggs as a maintenance mechanism from one year to the next. A second transmission
            cycle, the urban cycle, involves humans and the mosquito species 
            <i>Aedes (Stegomyia) aegypti</i>. An intermediary cycle is maintained in humid and semi-humid African savannas with
            viral transmission to humans and to non-human primates through different 
            <i>Aedes sp.</i> Mosquito control is not possible in areas of sylvatic transmission, thus eradication
            of YF is not a plausible option 
            <span ref-type="bibr" rid="pone.0036291-WHO2">
               <sup>
                  <a href="#pone.0036291-WHO2">[3]</a>
               </sup>
            </span>.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">YF remains an important public health problem for people in endemic regions in Africa
            and South America and is considered an emerging disease, with a dramatic upsurge in
            the number of cases in recent years 
            <span ref-type="bibr" rid="pone.0036291-Anonymous1">
               <sup>
                  <a href="#pone.0036291-Anonymous1">[4]</a>
               </sup>
            </span>. YF cases occur every year in Africa. For example, during 2011 cases from Senegal,
            Sierra Leone, Uganda, and Ivory Coast have been reported to the World Health Organization
            (WHO) (
            <div tagxxx="ext-link">http://www.who.int</div>). In the Americas, the circulation of jungle YF has experienced an unexpected increase
            since 2008, affecting Argentina, Brazil, Colombia, Paraguay, and Venezuela on the
            continent and Trinidad and Tobago in the Caribbean. Remarkably, a number cases of
            urban transmission have been reported in Bolivia 
            <span ref-type="bibr" rid="pone.0036291-Anonymous2">
               <sup>
                  <a href="#pone.0036291-Anonymous2">[5]</a>
               </sup>
            </span>, and Paraguay has dealt with an outbreak of apparently urban YF 
            <span ref-type="bibr" rid="pone.0036291-VanderStuyft1">
               <sup>
                  <a href="#pone.0036291-VanderStuyft1">[6]</a>
               </sup>
            </span>. The real incidence of YFV infections worldwide is unknown due to the non-specific
            nature of the symptoms leading to misdiagnoses, together with insufficient reporting
            and ground-surveillance, and it is estimated to be over 200,000 cases per year 
            <span ref-type="bibr" rid="pone.0036291-WHO1">
               <sup>
                  <a href="#pone.0036291-WHO1">[1]</a>
               </sup>
            </span>.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The clinical diagnosis of YF and the identification of YFV vaccine-associated adverse
            events (YFVAE) are particularly difficult because of similar symptoms of a wide range
            of diseases (e.g. dengue, leptospirosis, viral hepatitis, malaria, hemorrhagic viral
            diseases), therefore laboratory confirmation is essential. As criteria for laboratory
            YF diagnosis, WHO recommends the detection of YFV-specific IgM or a fourfold or greater
            rise in serum IgG levels in the absence of recent YF vaccination and a negative diagnosis
            for other flaviviruses. Isolation of YFV, positive post-mortem liver histopathology,
            detection of YFV antigen in tissues by immunohistochemistry, or detection of YFV genome
            in blood or organs by RT-PCR also confirm the presence of YFV infection 
            <span ref-type="bibr" rid="pone.0036291-WHO3">
               <sup>
                  <a href="#pone.0036291-WHO3">[7]</a>
               </sup>
            </span>. Recently, we have reported the detection of YFV-17D genome in the urine of healthy
            YF vaccinees and vaccinees with serious post- vaccination adverse events, which seems
            very promising for the investigation in outbreak situations by non-invasive sampling
            methods 
            <span ref-type="bibr" rid="pone.0036291-Domingo1">
               <sup>
                  <a href="#pone.0036291-Domingo1">[8]</a>
               </sup>
            </span>.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Molecular methods for the detection of the viral genome offer a rapid, sensitive,
            and highly specific alternative for early serological diagnosis during the viraemic
            phase of infection or in post-mortem tissues. After the viraemic phase the use of
            serological methods represents a good option to confirm the infection, but generally
            two samples are required to be taken at least two weeks apart. Serological diagnosis
            commonly includes the use of haemagglutination, ELISA, indirect immunofluorescence
            (IIF) and seroneutralization assays, but flavivirus serological cross-reactions constitute
            a major obstacle in achieving confirmed diagnoses or reliable serosurveys in endemic
            areas where other flaviviruses circulate (e.g. dengue, Zika, St. Louis encephalitis,
            or West Nile viruses). Seroneutralization is considered as the most specific serological
            technique. However, the assays are laborious and time consuming and are only available
            in expert laboratories. But the availability of commercial assays for the serological
            diagnosis of YF has increased the implementation of such techniques. Commercial assays
            are, in general, thoroughly standardized and offer good standards of sensitivity,
            specificity, and reproducibility in diagnostic laboratories in order to receive the
            authorisation for in vitro diagnostics.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The performance of the different techniques applied in YF diagnosis may vary between
            laboratories, and so far no external quality assessment (EQA) studies addressing their
            accomplishment have been performed. This international EQA is an important tool to
            evaluate the performance of protocols currently in use in diagnostic laboratories
            and to highlight weaknesses in their methodologies and operating procedures.
         </p>
      </div>
      <div id="s2">
         <h2>Materials and Methods</h2>
         <div id="s2a">
            <h3>Participants</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Institutions involved in laboratory diagnostics of YFV infection were invited to participate
               in this study. Invitees consisted of members of the European Network of “Imported”
               Viral Diseases (ENIVD), national/regional YFV reference laboratories, and diagnostic
               laboratories contacted through the Pan American Health Organization (PAHO) or the
               African Network of Laboratories for polioviruses and hemorrhagic fevers diagnosis.
               The study was announced as an EQA study on YFV molecular and serological diagnostic
               methods proficiency, which included certifying and publishing the results in a comparative
               and anonymous manner.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Thirty-six laboratories from Europe (n = 28), the Americas (n = 7), Middle East Asia
               (n = 1), and Africa (n = 1) participated in these EQA activities, and reports including
               32 and 31 data sets of results were returned for the molecular (28 laboratories) and
               serological (28 laboratories) diagnosis EQAs, respectively.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Twenty laboratories participated both in the molecular diagnostics EQA and the serological
               diagnostics EQA from Europe (n = 15), the Americas (n = 4), and Africa (n = 1): IRBA-IMTSSA,
               Marseille, France; Bernhard-Nocht-Institut, Hamburg, Germany; Institut für Mikrobiologie
               der Bundeswehr, Munich, Germany; Aristotle University of Thessaloniki, School of Medicine,
               Greece; Istituto Nazionale per le Malattie Infettive “L. Spallanzani”, Rome, Italy;
               Fondazione IRCCS Policlinico San Mateo, Pavia, Italy; Norwegian Institute of Public
               Health, Oslo, Norway; CEVD/INS, Aguas de Moura, Portugal; Institute of Microbiology
               and Immunology, University of Ljubljana, Ljubljana, Slovenia; Instituto de Salud Carlos
               III, Madrid, Spain; Hospital Clínic i Provincial de Barcelona, Barcelona, Spain; Spiez
               Laboratory, Spiez, Switzerland; University of Geneva, Geneva, Switzerland; Erasmus
               Medical Centre, Rotterdam, The Netherlands; Health Protection Agency, CEPR, Porton
               Down, Salisbury, United Kingdom; Centro Nacional de Enfermedades Tropicales, CENETROP,
               Santa Cruz, Bolivia; Instituto Nacional de Salud, Bogotá, Colombia; Institut Pasteur
               de la Guyane, Cayenne Cedex, French Guiana; Laboratorio Central de Salud Pública,
               Asunción, Paraguay; Special Pathogens Unit, National Institute for Communicable Diseases,
               National Health Laboratory Service, SPU/NICD-NHLS, Johannesburg, South Africa.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Eight laboratories participated exclusively in the YFV molecular diagnostics EQA,
               from Europe (n = 7) and the Middle East (n = 1): Institute of Virology, Medical University
               Vienna, Vienna, Austria; Slovak Academy of Science, Bratislava, Slovakia; Assistance
               Publique-Hôpitaux de Marseilles, Hôpital de la Timone, AP-HM TIMONE, Marseille, France;
               Army Medical and Veterinary Research Center, Rome, Italy; Centre for Biothreat Preparedness,
               Helsinki, Finland; Universitätsklinikum Freiburg, Freiburg, Germany; Institut für
               Virologie, Marburg, Germany; National Center for Zoonotic Viruses, MOH-PHL, Tel-Hashomer,
               Israel.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Eight laboratories participated exclusively in the serological diagnosis EQA, from
               Europe (n = 5) and the Americas (n = 3); Instituut voor Tropische Geneeskunde, Antwerp,
               Belgium; Institute of Public Health, Ostrava, Czech Republic; Euroimmun AG, Lübeck,
               Germany; National Center for Epidemiology, Budapest, Hungary; Crucell Switzerland
               AG, Berne, Switzerland; Bio Manguinhos, Rio de Janeiro, Brazil; Viral Zoonoses National
               Microbiology Laboratory, Winnipeg, Manitoba, Canada; Instituto Nacional de Higiene
               y Medicina Tropical “LIP”, Guayaquil, Ecuador.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The European Network for the Diagnostics of ‘Imported’ Viral Diseases -Collaborative
               Laboratory Response Network (ENIVD-CLRN) established and coordinated this EQA as in
               other EQAs previously performed 
               <span ref-type="bibr" rid="pone.0036291-Niedrig1">
                  <sup>
                     <a href="#pone.0036291-Niedrig1">[9]</a>
                  </sup>
               </span>–
               <span ref-type="bibr" rid="pone.0036291-Domingo2">
                  <sup>
                     <a href="#pone.0036291-Domingo2">[12]</a>
                  </sup>
               </span>.
            </p>
         </div>
         <div id="s2b">
            <h3>Specimen preparation</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The molecular diagnosis EQA panel consisted of inactivated YFV preparations generated
               from Vero E6 cell culture supernatants infected with different YFV strains: the vaccine
               strain (YFV-17D), a South American strain (Brazil), and a West African strain (IvoryC1999)
               
               <span ref-type="bibr" rid="pone.0036291-Bae1">
                  <sup>
                     <a href="#pone.0036291-Bae1">[13]</a>
                  </sup>
               </span>. Supernatants were inactivated by heating for 1 h at 56°C and by gamma irradiation
               (25 kilogray [kGy]) to ensure their non-infectivity. The inactivated supernatant viral
               load was estimated after heat inactivation and additionally after gamma irradiation
               by an in-house real-time RT-qPCR with a 95% detection limit in copy number estimated
               in 6.48 copies/reaction (rxn) (95% CI: 2.35–235 copies/rxn) (C. Domingo, unpublished
               results).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The inactivated material was diluted in serum plasma to prepare a set of ten positive
               samples that included five serial 10-fold dilution series of YFV-17D (3×10
               <sup>6</sup> Genome equivalents [GE]/sample to 3×10
               <sup>2</sup> GE/sample), two YFV (Brazil) dilutions (10
               <sup>4</sup> GE/sample and 10
               <sup>3</sup> GE/sample), and three YFV (strain IvoryC1999; GenBank Acc. No.: AY603338) dilutions
               (2×10
               <sup>4</sup> GE/sample to 69 GE/sample). As specificity controls we prepared two additional plasma
               samples, one of them containing West Nile virus (WNV [New York]), Japanese encephalitis
               virus (JEV [strain SA-14-02]), St. Louis encephalitis virus (SLEV [Parton]), and tick-borne
               encephalitis virus (TBEV [strain Absettarov]). The second plasma sample contained
               the four dengue serotypes (DENV-1 VR344 [strain Thai 1958], DENV-2 VR345 [TH-36 strain],
               DENV-3 VR216 [H87 strain], and DENV-4 VR217 [H241 strain]). Two negative control plasma
               samples were also included (
               <span ref-type="table" rid="pone-0036291-t001">
                  <a href="#pone-0036291-t001">Table 1</a>
               </span>).
            </p>
            <div id="pone-0036291-t001">
               <div tagxxx="object-id">10.1371/journal.pone.0036291.t001</div>
               <span class="label">Table 1</span>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">EQA results of the 28 participant laboratories in the molecular diagnosis panel.</div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <colgroup>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                     </colgroup>
                     <thead>
                        <tr>
                           <td>Sample ID</td>
                           <td>#2</td>
                           <td>#9</td>
                           <td>#12</td>
                           <td>#4</td>
                           <td>#14</td>
                           <td>#10</td>
                           <td>#5</td>
                           <td>#13</td>
                           <td>#1</td>
                           <td>#6</td>
                           <td>#11</td>
                           <td>#3</td>
                           <td>#8</td>
                           <td>#7</td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>Viral load in sample</td>
                           <td>3×10
                              <sup>6</sup>
                           </td>
                           <td>3×10
                              <sup>5</sup>
                           </td>
                           <td>3×10
                              <sup>4</sup>
                           </td>
                           <td>3×10
                              <sup>3</sup>
                           </td>
                           <td>3×10
                              <sup>2</sup>
                           </td>
                           <td>10
                              <sup>4</sup>
                           </td>
                           <td>10
                              <sup>3</sup>
                           </td>
                           <td>2×10
                              <sup>4</sup>
                           </td>
                           <td>2×10
                              <sup>3</sup>
                           </td>
                           <td>69</td>
                           <td>NEG</td>
                           <td>NEG</td>
                           <td>NEG</td>
                           <td>NEG</td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>Lab. no.</td>
                           <td>RT-PCR technique</td>
                           <td>17D</td>
                           <td>17D</td>
                           <td>17D</td>
                           <td>17D</td>
                           <td>17D</td>
                           <td>Brazil</td>
                           <td>Brazil</td>
                           <td>Ivory Coast</td>
                           <td>Ivory Coast</td>
                           <td>Ivory Coast</td>
                           <td>SLEV/JEVWNV/TBEV</td>
                           <td>DENV1-4</td>
                           <td>neg</td>
                           <td>neg</td>
                           <td>TARGET</td>
                           <td>Score
                              <span ref-type="table-fn" rid="nt101">
                                 <sup>
                                    <a href="#nt101">*</a>
                                 </sup>
                              </span>
                           </td>
                           <td>Classification</td>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>16a</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt102">
                                 <sup>
                                    <a href="#nt102">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>26</td>
                           <td>OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>8</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt103">
                                 <sup>
                                    <a href="#nt103">b</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>PANFLAV</td>
                           <td>24
                              <span ref-type="table-fn" rid="nt114">
                                 <sup>
                                    <a href="#nt114">¶</a>
                                 </sup>
                              </span>
                           </td>
                           <td>OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>17b</td>
                           <td>RT nested PCR
                              <span ref-type="table-fn" rid="nt104">
                                 <sup>
                                    <a href="#nt104">c</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>PANFLAV</td>
                           <td>24
                              <span ref-type="table-fn" rid="nt114">
                                 <sup>
                                    <a href="#nt114">¶</a>
                                 </sup>
                              </span>
                           </td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>1</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt102">
                                 <sup>
                                    <a href="#nt102">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>24</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>27</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt105">
                                 <sup>
                                    <a href="#nt105">d</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>24</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>28</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt106">
                                 <sup>
                                    <a href="#nt106">e</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>24</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>15</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt102">
                                 <sup>
                                    <a href="#nt102">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>22</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>17a</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt107">
                                 <sup>
                                    <a href="#nt107">f</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>22</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>6</td>
                           <td>RT nested PCR
                              <span ref-type="table-fn" rid="nt113">
                                 <sup>
                                    <a href="#nt113">x</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>PANFLAV</td>
                           <td>20
                              <span ref-type="table-fn" rid="nt114">
                                 <sup>
                                    <a href="#nt114">¶</a>
                                 </sup>
                              </span>
                           </td>
                           <td>OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>22a</td>
                           <td>RT nested PCR
                              <span ref-type="table-fn" rid="nt108">
                                 <sup>
                                    <a href="#nt108">g</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>ND</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>PANFLAV</td>
                           <td>20
                              <span ref-type="table-fn" rid="nt114">
                                 <sup>
                                    <a href="#nt114">¶</a>
                                 </sup>
                              </span>
                           </td>
                           <td>OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>16b</td>
                           <td>Heminest Rt-PCR
                              <span ref-type="table-fn" rid="nt104">
                                 <sup>
                                    <a href="#nt104">c</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>PANFLAV</td>
                           <td>20
                              <span ref-type="table-fn" rid="nt114">
                                 <sup>
                                    <a href="#nt114">¶</a>
                                 </sup>
                              </span>
                           </td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>2</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt102">
                                 <sup>
                                    <a href="#nt102">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>20</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>9</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt102">
                                 <sup>
                                    <a href="#nt102">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>20</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>14</td>
                           <td>RT nested PCR
                              <span ref-type="table-fn" rid="nt108">
                                 <sup>
                                    <a href="#nt108">g</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>PANFLAV</td>
                           <td>18
                              <span ref-type="table-fn" rid="nt114">
                                 <sup>
                                    <a href="#nt114">¶</a>
                                 </sup>
                              </span>
                           </td>
                           <td>OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>4</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt102">
                                 <sup>
                                    <a href="#nt102">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>18</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>10</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt106">
                                 <sup>
                                    <a href="#nt106">e</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>14</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>11</td>
                           <td>RT-PCR
                              <span ref-type="table-fn" rid="nt109">
                                 <sup>
                                    <a href="#nt109">h</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>14</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>20</td>
                           <td>Heminest RT-PCR
                              <span ref-type="table-fn" rid="nt104">
                                 <sup>
                                    <a href="#nt104">c</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>PANFLAV</td>
                           <td>16
                              <span ref-type="table-fn" rid="nt114">
                                 <sup>
                                    <a href="#nt114">¶</a>
                                 </sup>
                              </span>
                           </td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>3b</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt106">
                                 <sup>
                                    <a href="#nt106">e</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>14</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>5</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt113">
                                 <sup>
                                    <a href="#nt113">x</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>14</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>13</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt102">
                                 <sup>
                                    <a href="#nt102">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>ND</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>14</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>18</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt106">
                                 <sup>
                                    <a href="#nt106">e</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>14</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>3a</td>
                           <td>RT nested PCR
                              <span ref-type="table-fn" rid="nt107">
                                 <sup>
                                    <a href="#nt107">f</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>12</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>19</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt102">
                                 <sup>
                                    <a href="#nt102">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>12</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>22b</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt106">
                                 <sup>
                                    <a href="#nt106">e</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>ND</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>12</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>7</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt110">
                                 <sup>
                                    <a href="#nt110">i</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>YFV</td>
                           <td>10</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>21</td>
                           <td>TaqMan RT-PCR
                              <span ref-type="table-fn" rid="nt106">
                                 <sup>
                                    <a href="#nt106">e</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>10</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>25</td>
                           <td>RT-nested PCR
                              <span ref-type="table-fn" rid="nt113">
                                 <sup>
                                    <a href="#nt113">x</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>ND</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>ND</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>8</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>12</td>
                           <td>SYBR-RT-PCR
                              <span ref-type="table-fn" rid="nt103">
                                 <sup>
                                    <a href="#nt103">b</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>PANFLAV</td>
                           <td>6</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>23</td>
                           <td>RT-PCR
                              <span ref-type="table-fn" rid="nt111">
                                 <sup>
                                    <a href="#nt111">j</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>PANFLAV</td>
                           <td>0</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>24</td>
                           <td>RT-nested PCR
                              <span ref-type="table-fn" rid="nt112">
                                 <sup>
                                    <a href="#nt112">k</a>
                                 </sup>
                              </span>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>YFV</td>
                           <td>NS</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>26</td>
                           <td>RT-nested PCR
                              <span ref-type="table-fn" rid="nt107">
                                 <sup>
                                    <a href="#nt107">f</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>
                              <i>(−)</i>
                           </td>
                           <td>+</td>
                           <td>+</td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>
                              <i>(+)</i>
                           </td>
                           <td>YFV</td>
                           <td>NS</td>
                           <td>NON OPTIMAL</td>
                        </tr>
                        <tr>
                           <td>CORRECT RESULTS (%)</td>
                           <td>
                              <b>96.8</b>
                           </td>
                           <td>
                              <b>90.6</b>
                           </td>
                           <td>
                              <b>86.2</b>
                           </td>
                           <td>
                              <b>71.8</b>
                           </td>
                           <td>
                              <b>53.1</b>
                           </td>
                           <td>
                              <b>67.7</b>
                           </td>
                           <td>
                              <b>50</b>
                           </td>
                           <td>
                              <b>50</b>
                           </td>
                           <td>
                              <b>48.3</b>
                           </td>
                           <td>
                              <b>25</b>
                           </td>
                           <td>
                              <b>84.3</b>
                           </td>
                           <td>
                              <b>25</b>
                           </td>
                           <td>
                              <b>93.7</b>
                           </td>
                           <td>
                              <b>93.7</b>
                           </td>
                           <td></td>
                        </tr>
                     </div>
                  </table>
               </div>
               <table-wrap-foot xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div class="fn" id="nt101">
                     <span class="label">*</span>
                     <p>The score reflects the quality of the diagnostic results: +, positive result (score
                        2 points); 
                        <i>(+)</i>, false positive (score -2 points); −, correct negative result (score 2 points); 
                        <i>(−)</i>, false negative (score 0 points).
                     </p>
                  </div>
                  <div class="fn" id="nt102">
                     <span class="label">a</span>
                     <p>
                        <span ref-type="bibr" rid="pone.0036291-Drosten1">
                           <sup>
                              <a href="#pone.0036291-Drosten1">[20]</a>
                           </sup>
                        </span>;
                     </p>
                  </div>
                  <div class="fn" id="nt103">
                     <span class="label">b</span>
                     <p>adapted from 
                        <span ref-type="bibr" rid="pone.0036291-Moureau1">
                           <sup>
                              <a href="#pone.0036291-Moureau1">[23]</a>
                           </sup>
                        </span>;
                     </p>
                  </div>
                  <div class="fn" id="nt104">
                     <span class="label">c</span>
                     <p>
                        <span ref-type="bibr" rid="pone.0036291-Scaramozzino1">
                           <sup>
                              <a href="#pone.0036291-Scaramozzino1">[19]</a>
                           </sup>
                        </span>;
                     </p>
                  </div>
                  <div class="fn" id="nt105">
                     <span class="label">d</span>
                     <p>
                        <span ref-type="bibr" rid="pone.0036291-Charlier1">
                           <sup>
                              <a href="#pone.0036291-Charlier1">[24]</a>
                           </sup>
                        </span>;
                     </p>
                  </div>
                  <div class="fn" id="nt106">
                     <span class="label">e</span>
                     <p>
                        <span ref-type="bibr" rid="pone.0036291-Bae2">
                           <sup>
                              <a href="#pone.0036291-Bae2">[21]</a>
                           </sup>
                        </span>;
                     </p>
                  </div>
                  <div class="fn" id="nt107">
                     <span class="label">f</span>
                     <p>in-house;</p>
                  </div>
                  <div class="fn" id="nt108">
                     <span class="label">g</span>
                     <p>
                        <span ref-type="bibr" rid="pone.0036291-SanchezSeco1">
                           <sup>
                              <a href="#pone.0036291-SanchezSeco1">[18]</a>
                           </sup>
                        </span>;
                     </p>
                  </div>
                  <div class="fn" id="nt109">
                     <span class="label">h</span>
                     <p>
                        <span ref-type="bibr" rid="pone.0036291-Brown1">
                           <sup>
                              <a href="#pone.0036291-Brown1">[22]</a>
                           </sup>
                        </span>;
                     </p>
                  </div>
                  <div class="fn" id="nt110">
                     <span class="label">i</span>
                     <p>
                        <span ref-type="bibr" rid="pone.0036291-Mantel1">
                           <sup>
                              <a href="#pone.0036291-Mantel1">[16]</a>
                           </sup>
                        </span>;
                     </p>
                  </div>
                  <div class="fn" id="nt111">
                     <span class="label">j</span>
                     <p>
                        <span ref-type="bibr" rid="pone.0036291-Kuno1">
                           <sup>
                              <a href="#pone.0036291-Kuno1">[15]</a>
                           </sup>
                        </span>;
                     </p>
                  </div>
                  <div class="fn" id="nt112">
                     <span class="label">k</span>
                     <p>
                        <span ref-type="bibr" rid="pone.0036291-Heraud1">
                           <sup>
                              <a href="#pone.0036291-Heraud1">[17]</a>
                           </sup>
                        </span>;
                     </p>
                  </div>
                  <div class="fn" id="nt113">
                     <span class="label">x</span>
                     <p>not provided.</p>
                  </div>
                  <div class="fn" id="nt114">
                     <span class="label">¶</span>
                     <p>Sequencing performed; NS: Negative Score; ND: not done; PANFLAV: panflavivirus; YFV:
                        yellow fever virus.
                     </p>
                  </div>
               </table-wrap-foot>
            </div>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Sample preparations were tested by an in-house real-time quantitative RT-PCR to validate
               the quality of the samples.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">For the serological diagnosis, a panel of 13 samples was prepared by diluting anti-YFV-positive
               sera from YF vaccinees and from wild-type YFV infections with fresh frozen plasma
               previously confirmed as negative for flaviviruses. After dilution, the samples were
               heat inactivated (56°C, 1 h). The proficiency panel consisted of a set of nine samples
               which included four serial 2-fold dilutions of YFV-17D-positive sera (IgM and IgG
               positive), two dilutions from a West African wild-type YFV infection serum (IgM negative,
               IgG positive), and three dilutions from a positive South American wild-type YFV infection
               serum (IgM and IgG positive). As specificity controls, we included aliquots of two
               sera containing IgM and IgG antibodies reactive for other flaviviruses (WNV, DENV)
               and two additional negative sera as controls (
               <span ref-type="table" rid="pone-0036291-t002">
                  <a href="#pone-0036291-t002">Table 2</a>
               </span>).
            </p>
            <div id="pone-0036291-t002">
               <div tagxxx="object-id">10.1371/journal.pone.0036291.t002</div>
               <span class="label">Table 2</span>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">EQA results of the 28 participating laboratories in the serological panel.</div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <colgroup>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                     </colgroup>
                     <thead>
                        <tr>
                           <td>Sample ID</td>
                           <td>#6</td>
                           <td>#11</td>
                           <td>#7</td>
                           <td>4#</td>
                           <td>#5</td>
                           <td>#10</td>
                           <td>#14</td>
                           <td>#13</td>
                           <td>#1</td>
                           <td>#2</td>
                           <td>#9z</td>
                           <td>#8</td>
                           <td>#12</td>
                           <td></td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>αYF17D</td>
                           <td>αYF17D</td>
                           <td>αYF17D</td>
                           <td>αYF17D</td>
                           <td>αYF-AFR</td>
                           <td>αYF-AFR</td>
                           <td>αYF-SA</td>
                           <td>αYF-SA</td>
                           <td>αYF-SA</td>
                           <td>αWNV</td>
                           <td>αDEN</td>
                           <td>neg</td>
                           <td>neg</td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>Dilution</td>
                           <td>1∶5</td>
                           <td>1∶10</td>
                           <td>1∶20</td>
                           <td>1∶40</td>
                           <td>1∶10</td>
                           <td>1∶20</td>
                           <td>1∶20</td>
                           <td>1∶40</td>
                           <td>1∶80</td>
                           <td>1∶20</td>
                           <td>1∶20</td>
                           <td></td>
                           <td></td>
                           <td>Score*</td>
                        </tr>
                        <tr>
                           <td>Lab. no.</td>
                           <td>Assay</td>
                           <td>IgM/IgG+/+</td>
                           <td>IgM/IgG+/+</td>
                           <td>IgM/IgG+/+</td>
                           <td>IgM/IgG+/−</td>
                           <td>IgM/IgG−/+</td>
                           <td>IgM/IgG−/−</td>
                           <td>IgM/IgG+/+</td>
                           <td>IgM/IgG+/+</td>
                           <td>IgM/IgG+/+</td>
                           <td>IgM/IgG−/−</td>
                           <td>IgM/IgG−/−</td>
                           <td>IgM/IgG−/−</td>
                           <td>IgM/IgG−/−</td>
                           <td>IgG</td>
                           <td>IgM</td>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>18</td>
                           <td>IIF
                              <span ref-type="table-fn" rid="nt116">
                                 <sup>
                                    <a href="#nt116">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>−/+</td>
                           <td>−/+</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/+</td>
                           <td>+/+</td>
                           <td>−/+</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>22</td>
                           <td>16</td>
                        </tr>
                        <tr>
                           <td>12b</td>
                           <td>IIF
                              <span ref-type="table-fn" rid="nt116">
                                 <sup>
                                    <a href="#nt116">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>−/+</td>
                           <td>−/+</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/+</td>
                           <td>+/+</td>
                           <td>+/+</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>22</td>
                           <td>18</td>
                        </tr>
                        <tr>
                           <td>27</td>
                           <td>IIF</td>
                           <td>+/+</td>
                           <td>−/+</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/+</td>
                           <td>+/+</td>
                           <td>+/+</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>22</td>
                           <td>20</td>
                        </tr>
                        <tr>
                           <td>4</td>
                           <td>IIF
                              <span ref-type="table-fn" rid="nt116">
                                 <sup>
                                    <a href="#nt116">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>−/+</td>
                           <td>−/+</td>
                           <td>−/+</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/+</td>
                           <td>+/+</td>
                           <td>+/+</td>
                           <td>−/−</td>
                           <td>−/+</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>20</td>
                           <td>18</td>
                        </tr>
                        <tr>
                           <td>6</td>
                           <td>IIF
                              <span ref-type="table-fn" rid="nt116">
                                 <sup>
                                    <a href="#nt116">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>−/+</td>
                           <td>−/−</td>
                           <td>−/eq</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/+</td>
                           <td>+/+</td>
                           <td>eq/+</td>
                           <td>−/eq</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>20</td>
                           <td>18</td>
                        </tr>
                        <tr>
                           <td>7</td>
                           <td>IIF
                              <span ref-type="table-fn" rid="nt118">
                                 <sup>
                                    <a href="#nt118">c</a>
                                 </sup>
                              </span>
                           </td>
                           <td>−/+</td>
                           <td>−/+</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/+</td>
                           <td>+/+</td>
                           <td>eq/+</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/+</td>
                           <td>20</td>
                           <td>18</td>
                        </tr>
                        <tr>
                           <td>8</td>
                           <td>IIF
                              <span ref-type="table-fn" rid="nt116">
                                 <sup>
                                    <a href="#nt116">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+/+</td>
                           <td>−/+</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/+</td>
                           <td>−/+</td>
                           <td>+/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>20</td>
                           <td>18</td>
                        </tr>
                        <tr>
                           <td>29</td>
                           <td>IIF
                              <span ref-type="table-fn" rid="nt116">
                                 <sup>
                                    <a href="#nt116">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+/+</td>
                           <td>−/+</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/+</td>
                           <td>+/+</td>
                           <td>+/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>20</td>
                           <td>20</td>
                        </tr>
                        <tr>
                           <td>10</td>
                           <td>IIF
                              <span ref-type="table-fn" rid="nt117">
                                 <sup>
                                    <a href="#nt117">b</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+/+</td>
                           <td>+/+</td>
                           <td>+/+</td>
                           <td>−/+</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/+</td>
                           <td>+/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>20</td>
                           <td>22</td>
                        </tr>
                        <tr>
                           <td>5</td>
                           <td>IIF
                              <span ref-type="table-fn" rid="nt117">
                                 <sup>
                                    <a href="#nt117">b</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+/+</td>
                           <td>+/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/+</td>
                           <td>+/+</td>
                           <td>+/+</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/−</td>
                           <td>+/+</td>
                           <td>18</td>
                           <td>18</td>
                        </tr>
                        <tr>
                           <td>20a</td>
                           <td>IIF</td>
                           <td>+/+</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/+</td>
                           <td>+/+</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>18</td>
                           <td>18</td>
                        </tr>
                        <tr>
                           <td>3</td>
                           <td>IIF
                              <span ref-type="table-fn" rid="nt116">
                                 <sup>
                                    <a href="#nt116">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+/−</td>
                           <td>+/−</td>
                           <td>+/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/+</td>
                           <td>+/+</td>
                           <td>+/+</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/−</td>
                           <td>−/−</td>
                           <td>18</td>
                           <td>22</td>
                        </tr>
                        <tr>
                           <td>2</td>
                           <td>IIF
                              <span ref-type="table-fn" rid="nt117">
                                 <sup>
                                    <a href="#nt117">b</a>
                                 </sup>
                              </span>
                           </td>
                           <td>nd/+</td>
                           <td>nd/+</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/+</td>
                           <td>nd/+</td>
                           <td>nd/+</td>
                           <td>nd/+</td>
                           <td>nd/+</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>18</td>
                           <td>nd</td>
                        </tr>
                        <tr>
                           <td>25</td>
                           <td>IIF
                              <span ref-type="table-fn" rid="nt116">
                                 <sup>
                                    <a href="#nt116">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+/+</td>
                           <td>eq/−</td>
                           <td>−/−</td>
                           <td>−/eq</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/+</td>
                           <td>+/+</td>
                           <td>eq/−</td>
                           <td>eq/−</td>
                           <td>−/−</td>
                           <td>eq/−</td>
                           <td>−/−</td>
                           <td>16</td>
                           <td>18</td>
                        </tr>
                        <tr>
                           <td>12a</td>
                           <td>IIF
                              <span ref-type="table-fn" rid="nt117">
                                 <sup>
                                    <a href="#nt117">b</a>
                                 </sup>
                              </span>
                           </td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/+</td>
                           <td>nd/+</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>16</td>
                           <td>nd</td>
                        </tr>
                        <tr>
                           <td>13</td>
                           <td>IIF
                              <span ref-type="table-fn" rid="nt119">
                                 <sup>
                                    <a href="#nt119">d</a>
                                 </sup>
                              </span>
                           </td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/+</td>
                           <td>nd/+</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>16</td>
                           <td>nd</td>
                        </tr>
                        <tr>
                           <td>28</td>
                           <td>IIF
                              <span ref-type="table-fn" rid="nt116">
                                 <sup>
                                    <a href="#nt116">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/+</td>
                           <td>+/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>14</td>
                           <td>16</td>
                        </tr>
                        <tr>
                           <td>14a</td>
                           <td>IIF
                              <span ref-type="table-fn" rid="nt116">
                                 <sup>
                                    <a href="#nt116">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/+</td>
                           <td>+/−</td>
                           <td>+/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>14</td>
                           <td>20</td>
                        </tr>
                        <tr>
                           <td>11</td>
                           <td>IIF
                              <span ref-type="table-fn" rid="nt117">
                                 <sup>
                                    <a href="#nt117">b</a>
                                 </sup>
                              </span>
                           </td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/+</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>nd/−</td>
                           <td>14</td>
                           <td>nd</td>
                        </tr>
                        <tr>
                           <td>1</td>
                           <td>IIF
                              <span ref-type="table-fn" rid="nt116">
                                 <sup>
                                    <a href="#nt116">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/−</td>
                           <td>+/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>12</td>
                           <td>16</td>
                        </tr>
                        <tr>
                           <td>16</td>
                           <td>ELISA
                              <span ref-type="table-fn" rid="nt121">
                                 <sup>
                                    <a href="#nt121">f</a>
                                 </sup>
                              </span>
                              <sup>,</sup>
                              <span ref-type="table-fn" rid="nt122">
                                 <sup>
                                    <a href="#nt122">g</a>
                                 </sup>
                              </span>
                           </td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/−</td>
                           <td>+/−</td>
                           <td>+/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>12</td>
                           <td>18</td>
                        </tr>
                        <tr>
                           <td>19</td>
                           <td>ELISA
                              <span ref-type="table-fn" rid="nt117">
                                 <sup>
                                    <a href="#nt117">b</a>
                                 </sup>
                              </span>
                           </td>
                           <td>+/eq</td>
                           <td>+/−</td>
                           <td>+/+</td>
                           <td>eq/+</td>
                           <td>−/−</td>
                           <td>−/−</td>
                           <td>+/−</td>
                           <td>+/−</td>
                           <td>−/+</td>
                           <td>+/+</td>
                           <td>−/+</td>
                           <td>+/+</td>
                           <td>−/+</td>
                           <td>10</td>
                           <td>20</td>
                        </tr>
                        <tr>
                           <td>26</td>
                           <td>ELISA
                              <span ref-type="table-fn" rid="nt117">
                                 <sup>
                                    <a href="#nt117">b</a>
                                 </sup>
                              </span>
                           </td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>nd/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>nd/nd</td>
                           <td>+/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>nd</td>
                           <td>14</td>
                        </tr>
                        <tr>
                           <td>23</td>
                           <td>ELISA</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>+/nd</td>
                           <td>+/nd</td>
                           <td>+/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>nd</td>
                           <td>18</td>
                        </tr>
                        <tr>
                           <td>22</td>
                           <td>ELISA
                              <span ref-type="table-fn" rid="nt120">
                                 <sup>
                                    <a href="#nt120">e</a>
                                 </sup>
                              </span>
                           </td>
                           <td>eq/nd</td>
                           <td>eq/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>+/nd</td>
                           <td>+/nd</td>
                           <td>+/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>−/nd</td>
                           <td>nd</td>
                           <td>22</td>
                        </tr>
                        <tr>
                           <td>24</td>
                           <td>NT</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>−</td>
                           <td>−</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>24</td>
                        </tr>
                        <tr>
                           <td>30</td>
                           <td>NT</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>−</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>+</td>
                           <td>24</td>
                        </tr>
                        <tr>
                           <td>9</td>
                           <td>NT</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>+</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>18</td>
                        </tr>
                        <tr>
                           <td>15</td>
                           <td>NT</td>
                           <td>+</td>
                           <td>+</td>
                           <td>+</td>
                           <td>−</td>
                           <td>+</td>
                           <td>−</td>
                           <td>−</td>
                           <td>+</td>
                           <td>+</td>
                           <td>−</td>
                           <td>−</td>
                           <td>+</td>
                           <td>+</td>
                           <td>18</td>
                        </tr>
                        <tr>
                           <td>14b</td>
                           <td>NT</td>
                           <td>+</td>
                           <td>+</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>+</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>−</td>
                           <td>16</td>
                        </tr>
                        <tr>
                           <td>20b</td>
                           <td>HAI</td>
                           <td>&lt;1∶20</td>
                           <td>&lt;1∶20</td>
                           <td>&lt;1∶20</td>
                           <td>&lt;1∶20</td>
                           <td>&lt;1∶20</td>
                           <td>&lt;1∶20</td>
                           <td>1∶20</td>
                           <td>&lt;1∶20</td>
                           <td>&lt;1∶20</td>
                           <td>&lt;1∶20</td>
                           <td>&lt;1∶20</td>
                           <td>&lt;1∶20</td>
                           <td>&lt;1∶20</td>
                           <td>6</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <table-wrap-foot xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div class="fn" id="nt115">
                     <span class="label"></span>
                     <p>The score reflects the quality of the diagnostic results: +, positive results; −,
                        negative result; nd, not done; eq, equivocal. NT, seroneutralization test; HAI, haemagglutination
                        inhibition test; Wrong results are marked in grey.
                     </p>
                  </div>
                  <div class="fn" id="nt116">
                     <span class="label">a</span>
                     <p>EUROIMMUN;</p>
                  </div>
                  <div class="fn" id="nt117">
                     <span class="label">b</span>
                     <p>In-house assay;</p>
                  </div>
                  <div class="fn" id="nt118">
                     <span class="label">c</span>
                     <p>
                        <span ref-type="bibr" rid="pone.0036291-Takacs1">
                           <sup>
                              <a href="#pone.0036291-Takacs1">[25]</a>
                           </sup>
                        </span>;
                     </p>
                  </div>
                  <div class="fn" id="nt119">
                     <span class="label">d</span>
                     <p>Bernhard Nocht Institut für Tropenmedizin, Hamburg;</p>
                  </div>
                  <div class="fn" id="nt120">
                     <span class="label">e</span>
                     <p>
                        <span ref-type="bibr" rid="pone.0036291-Talarmin1">
                           <sup>
                              <a href="#pone.0036291-Talarmin1">[32]</a>
                           </sup>
                        </span>;
                     </p>
                  </div>
                  <div class="fn" id="nt121">
                     <span class="label">f</span>
                     <p>
                        <span ref-type="bibr" rid="pone.0036291-Ansari1">
                           <sup>
                              <a href="#pone.0036291-Ansari1">[33]</a>
                           </sup>
                        </span>;
                     </p>
                  </div>
                  <div class="fn" id="nt122">
                     <span class="label">g</span>
                     <p>
                        <span ref-type="bibr" rid="pone.0036291-Forshey1">
                           <sup>
                              <a href="#pone.0036291-Forshey1">[34]</a>
                           </sup>
                        </span>.
                     </p>
                  </div>
               </table-wrap-foot>
            </div>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">For both panels, aliquots of 100 µl each were number-coded, freeze dried for 24 h
               (Christ, AlphaI-5, Hanau, Germany) and stored at 4°C until dispatch.
            </p>
         </div>
         <div id="s2c">
            <h3>Validation of the panel sets and dispatch</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To validate the molecular panel, we tested three different sets of EQA samples before
               distribution by the Robert Koch Institute (RKI), Berlin, Germany. After reconstitution
               with 100 µl of water, the samples were extracted using the QIAamp viral RNA minikit
               (Qiagen, Hilden, Germany) according to the manufacturer's instructions. As mentioned
               above, we estimated the YFV genome copies present in these samples by an in-house
               real-time RT-PCR (
               <span ref-type="table" rid="pone-0036291-t001">
                  <a href="#pone-0036291-t001">Table 1</a>
               </span>).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Similarly, two sets of samples for the serological EQA panel were validated before
               distribution. After reconstitution with 100 µl of water, samples were analyzed by
               IIF for the presence of specific YFV antibodies using commercially available kits
               (FK 2665-1010-G and FK 2665-1010-M, Anti-yellow fever virus IIFT, Euroimmun, Lübeck,
               Germany) following the manufacturer's instructions. These assays have demonstrated
               a specificity of 94.7% and 96.7% in the detection of IgG and IgM antibodies in two
               panels of patient sera involving 300 and 294 sera each. Similarly, their sensitivity
               was determined as 94.7% for IgG and 94.4% for IgM (stated by manufacturer). A microneutralization
               assay in Vero E6 cells was also carried out to confirm the presence and specificity
               of the antibodies as previously described 
               <span ref-type="bibr" rid="pone.0036291-Doblas1">
                  <sup>
                     <a href="#pone.0036291-Doblas1">[14]</a>
                  </sup>
               </span>.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The EQA panels were distributed to participants with full instructions. Samples were
               shipped at ambient temperature by post to participating laboratories. We requested
               participant laboratories to resuspend the samples in 100 µl of water and to analyse
               the material as serum samples for YFV molecular/serological diagnosis as done routinely.
               They were asked to report their results and any problems encountered, as well as to
               provide information on the protocol details (RT-PCR method, extraction procedure,
               serological methods, sera dilution) using a common form included in the documentation.
            </p>
         </div>
         <div id="s2d">
            <h3>Evaluation of the results</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To assure anonymous participation, an individual numerical identification code was
               assigned to the results sent by each laboratory. This number was followed by a letter
               (a, b) in case different laboratory results were received based on different methods.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A scoring system was established for sensitivity and specificity obtained by each
               participant laboratory. For the molecular diagnostics EQA evaluation we assigned two
               points for correct results (100% = 28), and penalised false-positive results with
               -2 points. We considered those methods as non-optimal which failed to detect one or
               more strains of YFV, or presented false-positive results in the negative samples.
               In those cases when a false-positive amplification result was obtained by RT-PCR in
               the “non-specificity control” samples (#3 and #11), which were however correctly identified
               by sequencing, the result was considered correct.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">For the serological diagnostics EQA evaluation we also assigned two points for correct
               results (100% = 26), whereas false-negative/-positive results were not scored. Equivocal
               or borderline results were considered as positive. IgM and IgG results were considered
               separately.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The complete panel of results was sent to the participants in an anonymous manner
               where they could only identify the results from their own laboratory.
            </p>
         </div>
         <div id="s2e">
            <h3>Statistical Analysis</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Data collected were entered into Microsoft Excel (Microsoft Corp., Bellingham, WA,
               USA) and analysed using SPSS 14.0 for Windows (SPSS Inc., Chicago, IL, USA).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Logit analysis was used to evaluate the effect of viral RNA concentration on the RT-PCR
               performance by using cumulative fractions of positive results reported for each test
               sample of the 10-fold dilution series of YFV-17D. The result reflects the performance
               of a hypothetical average laboratory.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <span ref-type="sec" rid="s3">Results</span> with respect to categorised variables were analysed by McNemar's test. T-test and
               Mann-Whitney tests were used to estimate the effect of the real-time RT-PCR format
               on the performance. P-value&lt;0.05 was considered to indicate statistical significance.
            </p>
         </div>
      </div>
      <div id="s3">
         <h2>Results</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A total of 36 laboratories participated in this EQA. Among the participants, 20 laboratories
            (56%) reported both serological and molecular results, indicating that they included
            both approaches in their diagnostic algorithm. However, 22% (8 out of 36) of the participants
            only applied either molecular or serological techniques, respectively.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A total of 32 laboratory results were received for the molecular diagnosis study (
            <span ref-type="table" rid="pone-0036291-t001">
               <a href="#pone-0036291-t001">Table 1</a>
            </span>), including four double sets from laboratories using two methods each (sets 3ab,
            16ab, 17ab, 22ab). A total of 31 laboratory results were received for the serological
            diagnosis study (
            <span ref-type="table" rid="pone-0036291-t002">
               <a href="#pone-0036291-t002">Table 2</a>
            </span>), including three double sets from laboratories using two methods each (sets 12ab,
            14ab, 20ab).
         </p>
         <div id="s3a">
            <h3>Molecular diagnosis results</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A variety of tests were used for screening and identification of YFV genome by participating
               laboratories; these included RT-PCR (n = 2, 6% of the laboratory results) 
               <span ref-type="bibr" rid="pone.0036291-Kuno1">
                  <sup>
                     <a href="#pone.0036291-Kuno1">[15]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pone.0036291-Mantel1">
                  <sup>
                     <a href="#pone.0036291-Mantel1">[16]</a>
                  </sup>
               </span>, RT-nested PCR (n = 8, 25%) 
               <span ref-type="bibr" rid="pone.0036291-Kuno1">
                  <sup>
                     <a href="#pone.0036291-Kuno1">[15]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pone.0036291-Heraud1">
                  <sup>
                     <a href="#pone.0036291-Heraud1">[17]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pone.0036291-SanchezSeco1">
                  <sup>
                     <a href="#pone.0036291-SanchezSeco1">[18]</a>
                  </sup>
               </span>, hemi-nested RT-PCR (n = 2, 6%) 
               <span ref-type="bibr" rid="pone.0036291-Scaramozzino1">
                  <sup>
                     <a href="#pone.0036291-Scaramozzino1">[19]</a>
                  </sup>
               </span>, TaqMan (n = 19, 60%) 
               <span ref-type="bibr" rid="pone.0036291-Mantel1">
                  <sup>
                     <a href="#pone.0036291-Mantel1">[16]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pone.0036291-Drosten1">
                  <sup>
                     <a href="#pone.0036291-Drosten1">[20]</a>
                  </sup>
               </span>–
               <span ref-type="bibr" rid="pone.0036291-Charlier1">
                  <sup>
                     <a href="#pone.0036291-Charlier1">[24]</a>
                  </sup>
               </span>, and SYBR Green 
               <span ref-type="bibr" rid="pone.0036291-Moureau1">
                  <sup>
                     <a href="#pone.0036291-Moureau1">[23]</a>
                  </sup>
               </span> (n = 1, 3%)-based real-time RT-PCR assays (
               <span ref-type="table" rid="pone-0036291-t001">
                  <a href="#pone-0036291-t001">Table 1</a>
               </span>). As many as ten published protocols (indicated in 
               <span ref-type="table" rid="pone-0036291-t001">
                  <a href="#pone-0036291-t001">Table 1</a>
               </span>) were used by participants and only three methods were established “in house” (9.37%).
               Eight laboratories used the TaqMan RT-PCR developed by Drosten et al. 
               <span ref-type="bibr" rid="pone.0036291-Drosten1">
                  <sup>
                     <a href="#pone.0036291-Drosten1">[20]</a>
                  </sup>
               </span>, and six of them reported using the TaqMan RT-PCR developed by Bae et al. 
               <span ref-type="bibr" rid="pone.0036291-Bae2">
                  <sup>
                     <a href="#pone.0036291-Bae2">[21]</a>
                  </sup>
               </span>, in both cases with varying performance depending on the reporting laboratory.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Performance varied among the 28 laboratories (
               <span ref-type="table" rid="pone-0036291-t001">
                  <a href="#pone-0036291-t001">Table 1</a>
               </span>). Five out of 32 (16%) analyses reported met all criteria with optimal performance;
               13 out of 32 (41%) test results achieved non-optimal results due to the failure to
               detect one or more YFV strains (in 12 sets of results wild-type strains were not detected,
               whereas in one data set the YF-17D strain was not detected). One laboratory did not
               report any positive results. Additionally, 78% (25 out of 32) of laboratory results
               reported false positives; 21 laboratory reports falsely identified YFV in samples
               containing other flaviviruses (#3 and #11) and four do so in negative samples (
               <span ref-type="table" rid="pone-0036291-t001">
                  <a href="#pone-0036291-t001">Table 1</a>
               </span>).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Serial dilutions of YFV-17D (samples #2, #9, #12, #4, and #14) were used in order
               to test the sensitivity of the different methods. To estimate the effect of virus
               concentration on RT-PCR performance, Logit analysis was carried out using cumulative
               RT-PCR-positive results reported for each sample of the 10-fold YFV-17D dilution series
               (
               <span ref-type="fig" rid="pone-0036291-g001">Figure 1</span>). The data demonstrated that 50% of positive performance could be expected at a concentration
               of 2.8×10
               <sup>2</sup> GE/sample (95% confidence interval [CI] 11.7–1.2×10
               <sup>3</sup> GE/sample). Ninety-five percent performance could be expected for 1.5×10
               <sup>6</sup> GE/sample (95% CI 1.9×10
               <sup>5</sup>–2.8×10
               <sup>8</sup> GE/sample).
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pone.0036291.g001</div>
               <span class="label">Figure 1</span>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Logit analysis of laboratory tests with a correct result (y axis) for YFV-17D related
                     to viral RNA concentration in positive samples (x axis).
                  </div>
                  <p>Data points represent individual samples in the panel. Thick line is the regression
                     line calculated on the basis of a logit model (dose–response curve), and thin lines
                     are 95% confidence intervals.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Comparison of the results obtained for similar concentrations of different YFV strains
               (
               <span ref-type="table" rid="pone-0036291-t001">
                  <a href="#pone-0036291-t001">Table 1</a>
               </span>) also suggests the effect of virus type on the performance. Thus the results for
               YFV-17D were significantly better than those for the IvoryC1999 strain (McNemar's
               test was performed for sample #12 versus sample #13 and for sample #4 versus sample
               #1). The performance for YFV-17D was also significantly higher than that for the Brazilian
               strain (McNemar's test performed for sample #12 versus sample #10 and for sample #4
               versus the sample #5). At the same time no essential difference in test performance
               was found when comparing Brazilian to Ivory Coast YFV samples.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Only four laboratories used two different molecular techniques for the evaluation,
               an advantageous approach to exclude false positives/negatives in routine diagnosis.
               The four laboratories used a combination of TaqMan RT-PCR and RT-nested PCR techniques.
               Laboratories no. 3 and 16 achieved better scores when using the TaqMan protocols,
               whereas laboratories no. 17 and 22 did with the RT-nested PCR ones (
               <span ref-type="table" rid="pone-0036291-t001">
                  <a href="#pone-0036291-t001">Table 1</a>
               </span>).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Logit analysis was also carried out separately for results obtained with different
               RT-PCR techniques. For TaqMan-based real-time RT-PCR methods 50% performance could
               be expected when viral genome concentration was equal to 1.6×10
               <sup>2</sup> GE/sample (95% CI [0.1–584]) while for the RT nested PCR protocols it was achieved
               at 5×10
               <sup>3</sup> GE/sample (95% CI [87–3.7×10
               <sup>4</sup>]). 95% of certainty was achieved with TaqMan real-time RT-PCR at 1.1×10
               <sup>4</sup> GE/sample (95% CI [3×10
               <sup>3</sup>–2×10
               <sup>7</sup>]), compared to RT nested PCR at 1×10
               <sup>8</sup> GE/sample (95% CI [2.4×10
               <sup>6</sup>–2.8×10
               <sup>15</sup>]), suggesting a better sensitivity of the TaqMan protocols. However, t-test comparing
               the TaqMan real-time methods' performance to that of RT nested PCR protocols did not
               reveal any significant difference in performance of the methods applied. In general,
               it seemed that the success of the analysis depended rather on the performance of the
               individual laboratories than of the format of the RT-PCR.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We requested further information on the number of copies of YFV genome in the samples
               sent to the participants in order to estimate the laboratories' experience in viral
               load determination. Only eight results out of 32 (25%) reported quantitative results
               (data not shown), although 20 laboratories reported results obtained by real-time-based
               procedures which are suitable to provide quantitative data. Among the reported results,
               five provided the quantification estimation as Ct values which give very limited data
               to accurately estimate the samples' viral load in the absence of calibrated standards.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Another point to consider in view of the results reported is the use of methods for
               the generic detection of flavivirus genome. Nine laboratories reported results using
               different pan-flavivirus protocols 
               <span ref-type="bibr" rid="pone.0036291-SanchezSeco1">
                  <sup>
                     <a href="#pone.0036291-SanchezSeco1">[18]</a>
                  </sup>
               </span>–
               <span ref-type="bibr" rid="pone.0036291-Drosten1">
                  <sup>
                     <a href="#pone.0036291-Drosten1">[20]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pone.0036291-Moureau1">
                  <sup>
                     <a href="#pone.0036291-Moureau1">[23]</a>
                  </sup>
               </span>, and five of them correctly differentiated the YFV samples from other flaviviruses
               in samples #3 and #11 by sequencing. This approach (detection plus sequencing) is
               suitable for diagnosis purposes as long as a good sensitivity is reached.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Some non-optimal results were due to the presence of false positives, mainly in sample
               #3 (only 25% of correct results) containing the four dengue serotypes and sample #11
               containing other flaviviruses (
               <span ref-type="table" rid="pone-0036291-t001">
                  <a href="#pone-0036291-t001">Table 1</a>
               </span>). Four laboratories reported false positive results in samples containing only human
               plasma, which indicates the need to improve laboratory procedures since carry-over
               contamination must be suspected.
            </p>
         </div>
         <div id="s3b">
            <h3>Serological diagnostics results</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Most of the serology laboratory results were obtained by using IIF for the detection
               of specific YFV antibodies (20 out of 31, 65%), but also ELISA (5 out of 31, 16%),
               seroneutralization (5 out of 31, 16%), and haemagglutination (1 set of results, 3%)
               assays.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Performance varied, depending mainly on the method used and the subclass of antibodies
               to be detected but also on the performing laboratory (
               <span ref-type="table" rid="pone-0036291-t002">
                  <a href="#pone-0036291-t002">Table 2</a>
               </span>).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Four out of 25 reports (16%) using IIF or ELISA tests did not include results for
               IgM antibodies, whereas in the case of IgG detection only three laboratories (12%)
               did not report results, testing only the presence of IgM as marker of infection.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Of the 21 test results for IgM analysis, ten (48%) did not report the presence of
               anti-YFV-17D IgM antibodies even in the first serial dilution of YFV-17D-positive
               sera. Among them, six results out of ten (60%) were obtained using a commercial IIF
               test (EUROIMMUN), three out of ten (30%) using “in-house” ELISA, and one out of ten
               (10%) using a published IIF protocol 
               <span ref-type="bibr" rid="pone.0036291-Takacs1">
                  <sup>
                     <a href="#pone.0036291-Takacs1">[25]</a>
                  </sup>
               </span>. In general, however, in the serial dilutions of samples containing anti-YFV antibodies
               against the South-American YFV strain, a good sensitivity profile was observed. Only
               seven out of 21 test results referring to IgM (33%) did not report anti-YF IgM antibodies
               in sample #1, the sera sample most diluted. Of these, two out seven (28.5%) laboratory
               results were obtained using an “in-house” ELISA test, two out of seven (28.5%) using
               an “in-house” IIF method, and three out of seven (42.5%) using a commercial IIF assay
               (
               <span ref-type="table" rid="pone-0036291-t002">
                  <a href="#pone-0036291-t002">Table 2</a>
               </span>).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Two laboratories out of 25 (8%) performing IIF or ELISA assays reported false-positive
               IgM detection in sample #2 which contained antibodies against WNV. However, as many
               as four laboratories (16%) reported false-positive results in samples #8 and/or #12
               (negative control samples).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Regarding anti-YFV IgG results reported during this EQA, a better performance was
               apparently achieved by laboratories using IIF protocols, as they obtained a higher
               score in the evaluation (
               <span ref-type="table" rid="pone-0036291-t002">
                  <a href="#pone-0036291-t002">Table 2</a>
               </span>).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Two laboratories out of the 22 (9%) sending IgG results completely failed to detect
               IgG antibodies. No laboratory reported the presence of IgG antibodies against YFV
               (African origin) in samples #5 and #10. In the serial dilution containing anti-YFV
               antibodies against a South-American YFV strain used as sensitivity control, 86% of
               the data reported the presence of anti-YFV IgG antibodies in sample #14, 68% in sample
               #13, and 45.5% in sample #1. However, in eight test results (36%) the presence of
               anti-YFV IgG (17D) was not detected in the lowest serial dilution of the YFV-17D-positive
               sera. Among these, four laboratories out of eight (50%) used a commercial IIF assay
               (EUROIMMUN), two (25%) an “in-house” IIF test, one (12.5%) an IIF assay purchased
               from the Bernhard-Nocht-Institut (Hamburg), and one (12.5%) used two-in house ELISAs.
               In 13 reports (59%) IgG detection in YFV-17D samples failed for sample #11, and in
               18 reports (82%) for sample #7 (highest dilution). Unexpectedly, none of the laboratories
               using anti-YFV ELISA detected the presence of IgG properly. Laboratory no. 19 reported
               the detection of IgG in samples #1 and #7 but not in previous dilutions of the same
               samples.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">False IgG positives were reported in six laboratory results, three of them in samples
               #2 and #9 containing antibodies against other flaviviruses, and another three in sample
               #12, a negative control serum. The fact that other laboratories using the same commercial
               assays did not report these false positives results, suggests a problem in the operational
               procedures rather than a non-specific cross-reactivity of the assays.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In general, no difference in performance was found for IIF and ELISA assays or for
               the usage of commercial versus “in-house” assays (data not shown), nor was a difference
               found in the detection accuracy for IgM and IgG anti-YFV antibodies (McNemar's test).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The data obtained by seroneutralization assays do not allow to distinguish among IgM
               and IgG antibodies since the assay determines the presence of total neutralizing antibodies.
               Seroneutralization assays are expected to be the most specific ones in flavivirus
               serological diagnostics. In this study they showed a good performance with high scores
               (
               <span ref-type="table" rid="pone-0036291-t002">
                  <a href="#pone-0036291-t002">Table 2</a>
               </span>). Indeed, even in the low-titre sample #5 (IgM−/IgG+) laboratories nos. 15 and 30
               reported a correct positive result. However, the presence of false-positive results
               for samples #8 and #12 raises the question whether these results are due to a higher
               sensitivity or a strong background in the test. Control samples for the cytotoxicity
               effect and controls for virus infectivity must be included in each assay to be able
               to interpret the results correctly.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The haemagglutination assay which was only used by laboratory no. 20 showed a very
               low sensitivity, with only sample #14 reported as positive (
               <span ref-type="table" rid="pone-0036291-t002">
                  <a href="#pone-0036291-t002">Table 2</a>
               </span>).
            </p>
         </div>
      </div>
      <div id="s4">
         <h2>Discussion</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">This is the first international EQA on YF diagnostics. The increasing importance of
            this disease in Africa and the Americas, and the risk of expansion to other areas,
            makes it necessary to assure that the methods used for YFV diagnostics and surveillance
            are working properly where they are already implemented.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Among the participating laboratories 20 out of 36 (56%) reported both serological
            and molecular results, indicating that they routinely include both approaches in their
            diagnostic algorithm. However, 22% (eight out of 36) of the participants applied only
            molecular or serological techniques, respectively, leaving room for the presence of
            false-negative results depending not only on the accuracy of the methods but also
            on the time period between the onset of disease and the diagnosis. We recommend combining
            both molecular and serological methods to provide the best accuracy in the diagnosis
            of wild-type YFV infections and YFVAE, an approach which increases the diagnosis window
            while minimizing the risk of false-negatives results. Likewise we also recommend the
            use of paired samples, when possible, to validate the results obtained in acute samples
            by confirming the presence/absence of seroconversion. This strategy is useful for
            the detection of false positives due to cross-contaminations in the case of molecular
            diagnosis or false isolated IgM positives in the case of serological diagnosis.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Regarding molecular diagnosis EQA, the participants using TaqMan real-time RT-PCR-based
            techniques overall showed a better performance with a higher sensitivity than other
            assays. However, the major limitation for the implementation of these assays is the
            costs of both thermocyclers and reagents which hamper a generalized application in
            the field.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">One of the main weaknesses observed during this EQA was the inability of some protocols
            to detect the YFV genome of wild-type strains. Five out of eight laboratories which
            failed to recognize all wild-type YFV strains used the TaqMan real-time RT-PCR described
            by Bae et al. 
            <span ref-type="bibr" rid="pone.0036291-Bae2">
               <sup>
                  <a href="#pone.0036291-Bae2">[21]</a>
               </sup>
            </span>, two out of eight used “in-house” methods, and one applied the method previously
            described by Brown et al. 
            <span ref-type="bibr" rid="pone.0036291-Brown1">
               <sup>
                  <a href="#pone.0036291-Brown1">[22]</a>
               </sup>
            </span> who already pointed out that false negatives might occur when using this protocol.
            One laboratory used a TaqMan real-time RT-PCR previously described exclusively for
            the detection of YFV vaccine strain 17D 
            <span ref-type="bibr" rid="pone.0036291-Mantel1">
               <sup>
                  <a href="#pone.0036291-Mantel1">[16]</a>
               </sup>
            </span>, which would explain the results obtained. However, the methods mentioned above showed
            a very good sensitivity profile against YFV-17D, making them useful tools to identify
            viral genome in suspected YFVAE or research, but obviously not to identify suspected
            wild-type cases. The presence of mismatches between the oligonucleotide and the viral
            target sequences might explain the failure in genome amplification. This denotes the
            need to adapt and up-date regularly oligonucleotide sequences in use in diagnosis
            laboratories to detect the presence of mutations in the circulating strains which
            may compromise the ability of the assay to amplify and/or detect the targeted sequence
            leading to false negative results. However, among those laboratories which failed
            to detect only one of the wild-type strains (South American [Brazil] or West African
            [IvoryC1999]), two laboratories reported the use of the TaqMan real-time RT-PCR described
            by Drosten et al. 
            <span ref-type="bibr" rid="pone.0036291-Drosten1">
               <sup>
                  <a href="#pone.0036291-Drosten1">[20]</a>
               </sup>
            </span>, one laboratory used the RT-nested PCR by Kuno 
            <span ref-type="bibr" rid="pone.0036291-Kuno1">
               <sup>
                  <a href="#pone.0036291-Kuno1">[15]</a>
               </sup>
            </span>, and one laboratory used an “in-house” protocol. Other participants applied the method
            published by Drosten et al., with optimal detection of all YFV strains. Therefore,
            it seems in these cases that failure in detecting some strains might more likely be
            due to the specific performance of the laboratories rather than to the techniques
            themselves, suggesting the need to revise the running protocols of these laboratories
            to improve the quality of their results.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">One laboratory completely failed in the detection of all positive samples. It can
            not be excluded that the gamma irradiation of the samples for inactivation could have
            resulted in nicked RNA that would affect those methods amplifying fragments of around
            600 bp 
            <span ref-type="bibr" rid="pone.0036291-Lemmer1">
               <sup>
                  <a href="#pone.0036291-Lemmer1">[26]</a>
               </sup>
            </span> as it is the case of the method used by this laboratory 
            <span ref-type="bibr" rid="pone.0036291-Heraud1">
               <sup>
                  <a href="#pone.0036291-Heraud1">[17]</a>
               </sup>
            </span>. However, other laboratories reported good results with bigger amplification targets
            so probably other factors might have influenced the performance of this laboratory.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Similarly, the presence of false-positive results in flavivirus RNA-free samples also
            denoted the need to optimize laboratory practices in order to avoid the occurrence
            of cross-contamination.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">It is remarkable that for the sample containing genome of the four dengue serotypes
            (sample #3) only 25% of the results were correct. In the meantime the possibility
            of the presence of contaminant traces of YFV genome due to the processing and preparation
            of the samples has been excluded, and these results could indicate some degree of
            non-specificity in the techniques. As dengue is a common differential diagnosis for
            YFV, such specificity issues should be taken into consideration while interpreting
            dengue or YFV diagnostic testing results.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Twenty-eight percent of the tests reported used a generic flavivirus assay for the
            detection of YFV RNA, and 55.5% of these correctly differentiated the presence of
            YFV genome from that of other flaviviruses by consecutive sequencing. This approach
            might be advantageous for diagnostic purposes in laboratories which need to cover
            the possibility of infections with different flaviviruses without holding available
            a specific protocol for each of them. However, we must point out that the use of these
            generic methods must always be accompanied by sequence identification, mainly in endemic
            areas where usually more than one of the flaviviruses circulates.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">As in previous EQA studies, data regarding viral load were scarce even though the
            use of real-time techniques was prominent 
            <span ref-type="bibr" rid="pone.0036291-Domingo2">
               <sup>
                  <a href="#pone.0036291-Domingo2">[12]</a>
               </sup>
            </span>.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Regarding the serological results of this EQA, one of the main observations was the
            fact that 16% of the participating laboratories do not routinely include the analysis
            of specific anti-YFV IgM in their diagnostic algorithms. It is well known that for
            the serological diagnosis of an acute YFV infection it is preferable to test for the
            presence of IgM which appears for a short period of time soon after the infection,
            confirming a recent contact with the virus. The presence of IgG does not provide a
            good proof of recent YFV infection and requires the analysis of a second sample to
            confirm a rise in the antibody titres. In general, the sole detection of IgG would
            only confirm a previous contact with YFV, even by vaccination, or with another flavivirus
            as serological cross-reactivity is more pronounced for IgG detection than for IgM,
            as suggested by this EQA and previous reports 
            <span ref-type="bibr" rid="pone.0036291-DonosoMantke1">
               <sup>
                  <a href="#pone.0036291-DonosoMantke1">[27]</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pone.0036291-Calisher1">
               <sup>
                  <a href="#pone.0036291-Calisher1">[29]</a>
               </sup>
            </span>. It is important to note that in general the existence of this cross-reactivity is
            the major weakness of the serological diagnosis, not only for YFV but for all flaviviruses.
            Seroneutralization assays are considered the “gold standard” for specific identification
            of a positive immune response. However, some degree of non-specificity can not be
            excluded and such a technique requires the use of well-characterized controls. Additionally,
            seroneutralization assays can perform poorly in samples with multiple/subsequent infections
            unless multiple samples are available. Also the material, the expertise, and the time
            required to perform a seroneutralization assay do not make it a proper choice for
            early diagnosis.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Fortunately, from the results of this EQA it has become obvious that false positives
            due to cross-reactivity are not the main limitation of the serological techniques
            used by the participating laboratories, and that most of the specificity difficulties
            could be solved by proper standardization of the protocols and the use of adequate
            controls during the assays.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">One of the main issues that deserve more attention is the apparent lack of sensitivity
            regarding anti-YFV-17D IgM detection in 48% of the results reported, while a good
            sensitivity profile was observed in wild-type infection sera. This should be taken
            into account when assessing the protection provided by the YFV vaccines by determining
            IgM levels.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In the case of IgG detection, apparently the quality of the results depends mostly
            on the performing laboratories since their results differ even when using the same
            technique. The lack of detection of IgG in sample #5 has no major impact on the conclusions
            of this EQA since the samples from West African wild type infection contained the
            lower titre of antibodies compared to the others, and required very sensitive techniques
            for detection, which could only be achieved by seroneutralization assays in this EQA.
            Similarly, the presence of false-positive results in those laboratories with other
            correct results makes it unclear whether a low threshold of detection could imply
            a higher risk for false-positive results to occur.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We conclude that the main differences in the molecular diagnosis results might be
            more related to the handling of assays and specimens, pointing out the importance
            of regularly revising and improving the operational protocols. The different profiles
            of strain detection obtained with each protocol must be taken into account by the
            laboratories which must select those protocols most suitable for their particular
            diagnosis purpose (wild-type infections vs. YFVAE).
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In general, commercial serological assays showed a very good sensitivity profile in
            this study for both IgM and IgG detection of wild-type origin, but not for antibody
            detection against the vaccine strain YFV-17D. Commercial serological assays used for
            diagnostic purposes show a better specificity when compared to “in-house” ones. However,
            their use raises some concern when evaluating the immune response elicited after vaccination.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">No difference in the sensitivity detection of IgM and IgG anti-YF antibodies has been
            found in this EQA, in contrast with the results observed in previous EQA studies on
            other arboviral infections 
            <span ref-type="bibr" rid="pone.0036291-Niedrig2">
               <sup>
                  <a href="#pone.0036291-Niedrig2">[11]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pone.0036291-Crill1">
               <sup>
                  <a href="#pone.0036291-Crill1">[30]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pone.0036291-Domingo3">
               <sup>
                  <a href="#pone.0036291-Domingo3">[31]</a>
               </sup>
            </span>. However, the different titres of IgM and IgG in the samples included in this EQA
            make this comparison quite general.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The low participation of endemic countries in this EQA, even though widely announced,
            points out the need to encourage more laboratories to implement YFV diagnosis techniques
            and to participate routinely in quality assurance programmes.
         </p>
      </div>
      <div tagxxx="back">
         <div tagxxx="ack">
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The authors are grateful to J. Kraft for excellent assistance during the EQA activity
               and to U. Erikli for copy-editing.
            </p>
         </div>
         <div tagxxx="fn-group">
            <div class="fn" id="">
               <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <b>Competing Interests: </b>The authors have declared that no competing interests exist.
               </p>
            </div>
            <div class="fn" id="">
               <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <b>Funding: </b>This study was performed by the European Network for the Diagnostics of ‘Imported’
                  Viral Diseases-Collaborative Laboratory Response Network (ENIVD-CLRN) funded by the
                  European Centre for Disease Prevention and Control (ECDC), contract no. ECDC/2008/011.
                  The funders had no role in study design, data collection and analysis, decision to
                  publish, or preparation of the manuscript.
               </p>
            </div>
         </div>
         <div tag="ref-list">
            <ul>
               <div tagxxx="title">References</div>
               <li tag="ref">
                  <a name="pone.0036291-WHO1"></a>
                  <span class="label">1</span>
                  <span class="element-citation'">
                     <span class="collab'">[WHO Media Centre]</span>
                     <span tagx="year">2009</span>
                     <span tagx="article-title">Yellow fever Fact Sheet n°100. World Health Organization.</span>
                     <div tagxxx="comment">Available: 
                        <div tagxxx="ext-link">http://www.who.int/mediacentre/factsheets/fs100/en/</div>. Accessed 2012 Apr 10
                     </div>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Monath1"></a>
                  <span class="label">2</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Monath</span>
                           <span tagx="given-names">TP</span>,
                        </span>
                     </span>
                     <span tagx="year">2001</span>
                     <span tagx="article-title">Yellow fever: an update.</span>
                     <span tagx="source">Lancet Infect Dis</span>
                     <span tagx="volume">1</span>
                     <span tagx="fpage">11</span>
                     <span tagx="lpage">20</span>
                     <span class="pub-id'">[11871403]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-WHO2"></a>
                  <span class="label">3</span>
                  <span class="element-citation'">
                     <span class="collab'">[WHO]</span>
                     <span tagx="year">1998</span>
                     <span tagx="source">WHO Report on Global Surveillance of Epidemic-prone Infectious Diseases – Yellow fever</span>
                     <span class="publisher-loc'">[Geneva]</span>
                     <span class="publisher-name'">[WHO]</span>
                     <div tagxxx="comment">Available: 
                        <div tagxxx="ext-link">http://www.who.int/csr/resources/publications/yellowfev/CSR_ISR_2000_1</div>. Accessed 2012 Apr 10
                     </div>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Anonymous1"></a>
                  <span class="label">4</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Anonymous</span>,
                        </span>
                     </span>
                     <span tagx="year">2006</span>
                     <span tagx="article-title">Yellow fever situation in Africa and South America, 2005.</span>
                     <span tagx="source">Wkly Epidemiol Rec</span>
                     <span tagx="volume">81</span>
                     <span tagx="fpage">317</span>
                     <span tagx="lpage">324</span>
                     <span class="pub-id'">[16924775]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Anonymous2"></a>
                  <span class="label">5</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Anonymous</span>,
                        </span>
                     </span>
                     <span tagx="year">2008</span>
                     <span tagx="article-title">Outbreak news. Yellow fever, Paraguay.</span>
                     <span tagx="source">Wkly Epidemiol Rec</span>
                     <span tagx="volume">83</span>
                     <span tagx="fpage">105</span>
                     <span class="pub-id'">[18350685]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-VanderStuyft1"></a>
                  <span class="label">6</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Van der Stuyft</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gianella</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Pirard</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Cespedes</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Lora</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">1999</span>
                     <span tagx="article-title">Urbanisation of yellow fever in Santa Cruz, Bolivia.</span>
                     <span tagx="source">Lancet</span>
                     <span tagx="volume">353</span>
                     <span tagx="fpage">1558</span>
                     <span tagx="lpage">1562</span>
                     <span class="pub-id'">[10334253]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-WHO3"></a>
                  <span class="label">7</span>
                  <span class="element-citation'">
                     <span class="collab'">[WHO]</span>
                     <span tagx="year">2003</span>
                     <span tagx="source">WHO-recommended standards for surveillance of selected vaccine-preventable diseases</span>
                     <span class="publisher-loc'">[Geneva]</span>
                     <span class="publisher-name'">[WHO]</span>
                     <span tagx="fpage">40</span>
                     <span tagx="lpage">44</span>
                     <div tagxxx="comment">Available: 
                        <div tagxxx="ext-link">http://whqlibdoc.who.int/hq/2003/WHO_V&amp;B_03.01.pdf</div>. Accessed 2012 Apr 10
                     </div>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Domingo1"></a>
                  <span class="label">8</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Domingo</span>
                           <span tagx="given-names">C</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Yactayo</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Agbenu</span>
                           <span tagx="given-names">E</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Demanou</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Schulz</span>
                           <span tagx="given-names">AR</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">2011</span>
                     <span tagx="article-title">Detection of yellow fever 17D genome in urine.</span>
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">49</span>
                     <span tagx="fpage">760</span>
                     <span tagx="lpage">762</span>
                     <span class="pub-id'">[21106799]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Niedrig1"></a>
                  <span class="label">9</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Niedrig</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Donoso-Mantke</span>
                           <span tagx="given-names">O</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Schadler</span>
                           <span tagx="given-names">R</span>,
                        </span>
                     </span>
                     <span tagx="year">2007</span>
                     <span tagx="article-title">The European Network for Diagnostics of Imported Viral Diseases (ENIVD)–12 years of
                        strengthening the laboratory diagnostic capacity in Europe.
                     </span>
                     <span tagx="source">Euro Surveill</span>
                     <span tagx="volume">12</span>
                     <span tagx="fpage">E070419 070415</span>
                     <span class="pub-id'">[17868616]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Panning1"></a>
                  <span class="label">10</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Panning</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Charrel</span>
                           <span tagx="given-names">RN</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Donoso Mantke</span>
                           <span tagx="given-names">O</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Landt</span>
                           <span tagx="given-names">O</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Niedrig</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">2009</span>
                     <span tagx="article-title">Coordinated implementation of chikungunya virus reverse transcription-PCR.</span>
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="volume">15</span>
                     <span tagx="fpage">469</span>
                     <span tagx="lpage">471</span>
                     <span class="pub-id'">[19239767]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Niedrig2"></a>
                  <span class="label">11</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Niedrig</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Zeller</span>
                           <span tagx="given-names">H</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Schuffenecker</span>
                           <span tagx="given-names">I</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Drosten</span>
                           <span tagx="given-names">C</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Emmerich</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">2009</span>
                     <span tagx="article-title">International diagnostic accuracy study for the serological detection of chikungunya
                        virus infection.
                     </span>
                     <span tagx="source">Clin Microbiol Infect</span>
                     <span tagx="volume">15</span>
                     <span tagx="fpage">880</span>
                     <span tagx="lpage">884</span>
                     <span class="pub-id'">[19624513]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Domingo2"></a>
                  <span class="label">12</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Domingo</span>
                           <span tagx="given-names">C</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Niedrig</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Teichmann</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Kaiser</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Rumer</span>
                           <span tagx="given-names">L</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">2010</span>
                     <span tagx="article-title">2nd international external quality control assessment for the molecular diagnosis
                        of dengue infections.
                     </span>
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="volume">4</span>
                     <span tagx="fpage">e833</span>
                     <span class="pub-id'">[20957194]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Bae1"></a>
                  <span class="label">13</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Bae</span>
                           <span tagx="given-names">HG</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Drosten</span>
                           <span tagx="given-names">C</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Emmerich</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Colebunders</span>
                           <span tagx="given-names">R</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Hantson</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">2005</span>
                     <span tagx="article-title">Analysis of two imported cases of yellow fever infection from Ivory Coast and The
                        Gambia to Germany and Belgium.
                     </span>
                     <span tagx="source">J Clin Virol</span>
                     <span tagx="volume">33</span>
                     <span tagx="fpage">274</span>
                     <span tagx="lpage">280</span>
                     <span class="pub-id'">[16036176]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Doblas1"></a>
                  <span class="label">14</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Doblas</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Domingo</span>
                           <span tagx="given-names">C</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Bae</span>
                           <span tagx="given-names">HG</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Bohorquez</span>
                           <span tagx="given-names">CL</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">de Ory</span>
                           <span tagx="given-names">F</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">2006</span>
                     <span tagx="article-title">Yellow fever vaccine-associated viscerotropic disease and death in Spain.</span>
                     <span tagx="source">J Clin Virol</span>
                     <span tagx="volume">36</span>
                     <span tagx="fpage">156</span>
                     <span tagx="lpage">158</span>
                     <span class="pub-id'">[16597510]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Kuno1"></a>
                  <span class="label">15</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Kuno</span>
                           <span tagx="given-names">G</span>,
                        </span>
                     </span>
                     <span tagx="year">1998</span>
                     <span tagx="article-title">Universal diagnostic RT-PCR protocol for arboviruses.</span>
                     <span tagx="source">J Virol Methods</span>
                     <span tagx="volume">72</span>
                     <span tagx="fpage">27</span>
                     <span tagx="lpage">41</span>
                     <span class="pub-id'">[9672130]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Mantel1"></a>
                  <span class="label">16</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Mantel</span>
                           <span tagx="given-names">N</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Aguirre</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gulia</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Girerd-Chambaz</span>
                           <span tagx="given-names">Y</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Colombani</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">2008</span>
                     <span tagx="article-title">Standardized quantitative RT-PCR assays for quantitation of yellow fever and chimeric
                        yellow fever–dengue vaccines.
                     </span>
                     <span tagx="source">J Virol Methods</span>
                     <span tagx="volume">151</span>
                     <span tagx="fpage">40</span>
                     <span tagx="lpage">46</span>
                     <span class="pub-id'">[18501437]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Heraud1"></a>
                  <span class="label">17</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Heraud</span>
                           <span tagx="given-names">JM</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Hommel</span>
                           <span tagx="given-names">D</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Hulin</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Deubel</span>
                           <span tagx="given-names">V</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Poveda</span>
                           <span tagx="given-names">JD</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">1999</span>
                     <span tagx="article-title">First case of yellow fever in French Guiana since 1902.</span>
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="volume">5</span>
                     <span tagx="fpage">429</span>
                     <span tagx="lpage">432</span>
                     <span class="pub-id'">[10341180]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-SanchezSeco1"></a>
                  <span class="label">18</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Sanchez-Seco</span>
                           <span tagx="given-names">MP</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Rosario</span>
                           <span tagx="given-names">D</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Domingo</span>
                           <span tagx="given-names">C</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Hernandez</span>
                           <span tagx="given-names">L</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Valdes</span>
                           <span tagx="given-names">K</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">2005</span>
                     <span tagx="article-title">Generic RT-nested-PCR for detection of flaviviruses using degenerated primers and
                        internal control followed by sequencing for specific identification.
                     </span>
                     <span tagx="source">J Virol Methods</span>
                     <span tagx="volume">126</span>
                     <span tagx="fpage">101</span>
                     <span tagx="lpage">109</span>
                     <span class="pub-id'">[15847925]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Scaramozzino1"></a>
                  <span class="label">19</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Scaramozzino</span>
                           <span tagx="given-names">N</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Crance</span>
                           <span tagx="given-names">JM</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Jouan</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">DeBriel</span>
                           <span tagx="given-names">DA</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Stoll</span>
                           <span tagx="given-names">F</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">2001</span>
                     <span tagx="article-title">Comparison of flavivirus universal primer pairs and development of a rapid, highly
                        sensitive heminested reverse transcription-PCR assay for detection of flaviviruses
                        targeted to a conserved region of the NS5 gene sequences.
                     </span>
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">39</span>
                     <span tagx="fpage">1922</span>
                     <span tagx="lpage">1927</span>
                     <span class="pub-id'">[11326014]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Drosten1"></a>
                  <span class="label">20</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Drosten</span>
                           <span tagx="given-names">C</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gottig</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Schilling</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Asper</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Panning</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">2002</span>
                     <span tagx="article-title">Rapid detection and quantification of RNA of Ebola and Marburg viruses, Lassa virus,
                        Crimean-Congo hemorrhagic fever virus, Rift Valley fever virus, dengue virus, and
                        yellow fever virus by real-time reverse transcription-PCR.
                     </span>
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">40</span>
                     <span tagx="fpage">2323</span>
                     <span tagx="lpage">2330</span>
                     <span class="pub-id'">[12089242]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Bae2"></a>
                  <span class="label">21</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Bae</span>
                           <span tagx="given-names">HG</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Nitsche</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Teichmann</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Biel</span>
                           <span tagx="given-names">SS</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Niedrig</span>
                           <span tagx="given-names">M</span>,
                        </span>
                     </span>
                     <span tagx="year">2003</span>
                     <span tagx="article-title">Detection of yellow fever virus: a comparison of quantitative real-time PCR and plaque
                        assay.
                     </span>
                     <span tagx="source">J Virol Methods</span>
                     <span tagx="volume">110</span>
                     <span tagx="fpage">185</span>
                     <span tagx="lpage">191</span>
                     <span class="pub-id'">[12798247]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Brown1"></a>
                  <span class="label">22</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Brown</span>
                           <span tagx="given-names">TM</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Chang</span>
                           <span tagx="given-names">GJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Cropp</span>
                           <span tagx="given-names">CB</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Robbins</span>
                           <span tagx="given-names">KE</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Tsai</span>
                           <span tagx="given-names">TF</span>,
                        </span>
                     </span>
                     <span tagx="year">1994</span>
                     <span tagx="article-title">Detection of yellow fever virus by polymerase chain reaction.</span>
                     <span tagx="source">Clin Diagn Virol</span>
                     <span tagx="volume">2</span>
                     <span tagx="fpage">41</span>
                     <span tagx="lpage">51</span>
                     <span class="pub-id'">[15566752]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Moureau1"></a>
                  <span class="label">23</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Moureau</span>
                           <span tagx="given-names">G</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Temmam</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gonzalez</span>
                           <span tagx="given-names">JP</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Charrel</span>
                           <span tagx="given-names">RN</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Grard</span>
                           <span tagx="given-names">G</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">2007</span>
                     <span tagx="article-title">A real-time RT-PCR method for the universal detection and identification of flaviviruses.</span>
                     <span tagx="source">Vector Borne Zoonotic Dis</span>
                     <span tagx="volume">7</span>
                     <span tagx="fpage">467</span>
                     <span tagx="lpage">477</span>
                     <span class="pub-id'">[18020965]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Charlier1"></a>
                  <span class="label">24</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Charlier</span>
                           <span tagx="given-names">N</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Molenkamp</span>
                           <span tagx="given-names">R</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Leyssen</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Paeshuyse</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Drosten</span>
                           <span tagx="given-names">C</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">2004</span>
                     <span tagx="article-title">Exchanging the yellow fever virus envelope proteins with Modoc virus prM and E proteins
                        results in a chimeric virus that is neuroinvasive in SCID mice.
                     </span>
                     <span tagx="source">J Virol</span>
                     <span tagx="volume">78</span>
                     <span tagx="fpage">7418</span>
                     <span tagx="lpage">7426</span>
                     <span class="pub-id'">[15220415]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Takacs1"></a>
                  <span class="label">25</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Takacs</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Berencsi</span>
                           <span tagx="given-names">G</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Mezey</span>
                           <span tagx="given-names">I</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Brojnas</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Barcsay</span>
                           <span tagx="given-names">E</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">1994</span>
                     <span tagx="article-title">Detection of transfusion-associated hepatitis caused by non-A, non-B, non-C flavivirus.</span>
                     <span tagx="source">Acta Microbiol Immunol Hung</span>
                     <span tagx="volume">41</span>
                     <span tagx="fpage">83</span>
                     <span tagx="lpage">89</span>
                     <span class="pub-id'">[7921854]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Lemmer1"></a>
                  <span class="label">26</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Lemmer</span>
                           <span tagx="given-names">K</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Donoso-Mantke</span>
                           <span tagx="given-names">O</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Bae</span>
                           <span tagx="given-names">HG</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Groen</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Drosten</span>
                           <span tagx="given-names">C</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">2004</span>
                     <span tagx="article-title">External quality control assessment in PCR diagnostics of dengue virus infections.</span>
                     <span tagx="source">J Clin Virol</span>
                     <span tagx="volume">30</span>
                     <span tagx="fpage">291</span>
                     <span tagx="lpage">296</span>
                     <span class="pub-id'">[15163416]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-DonosoMantke1"></a>
                  <span class="label">27</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Donoso Mantke</span>
                           <span tagx="given-names">O</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Lemmer</span>
                           <span tagx="given-names">K</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Biel</span>
                           <span tagx="given-names">SS</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Groen</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Schmitz</span>
                           <span tagx="given-names">H</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">2004</span>
                     <span tagx="article-title">Quality control assessment for the serological diagnosis of dengue virus infections.</span>
                     <span tagx="source">J Clin Virol</span>
                     <span tagx="volume">29</span>
                     <span tagx="fpage">105</span>
                     <span tagx="lpage">112</span>
                     <span class="pub-id'">[14747029]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Niedrig3"></a>
                  <span class="label">28</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Niedrig</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Donoso Mantke</span>
                           <span tagx="given-names">O</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Altmann</span>
                           <span tagx="given-names">D</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Zeller</span>
                           <span tagx="given-names">H</span>,
                        </span>
                     </span>
                     <span tagx="year">2007</span>
                     <span tagx="article-title">First international diagnostic accuracy study for the serological detection of West
                        Nile virus infection.
                     </span>
                     <span tagx="source">BMC Infect Dis</span>
                     <span tagx="volume">7</span>
                     <span tagx="fpage">72</span>
                     <span class="pub-id'">[17608925]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Calisher1"></a>
                  <span class="label">29</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Calisher</span>
                           <span tagx="given-names">CH</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Karabatsos</span>
                           <span tagx="given-names">N</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Dalrymple</span>
                           <span tagx="given-names">JM</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Shope</span>
                           <span tagx="given-names">RE</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Porterfield</span>
                           <span tagx="given-names">JS</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">1989</span>
                     <span tagx="article-title">Antigenic relationships between flaviviruses as determined by cross-neutralization
                        tests with polyclonal antisera.
                     </span>
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">70</span>
                     <div tagxxx="issue">Pt 1</div>
                     <span tagx="fpage">37</span>
                     <span tagx="lpage">43</span>
                     <span class="pub-id'">[2543738]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Crill1"></a>
                  <span class="label">30</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Crill</span>
                           <span tagx="given-names">WD</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Chang</span>
                           <span tagx="given-names">GJ</span>,
                        </span>
                     </span>
                     <span tagx="year">2004</span>
                     <span tagx="article-title">Localization and characterization of flavivirus envelope glycoprotein cross-reactive
                        epitopes.
                     </span>
                     <span tagx="source">J Virol</span>
                     <span tagx="volume">78</span>
                     <span tagx="fpage">13975</span>
                     <span tagx="lpage">13986</span>
                     <span class="pub-id'">[15564505]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Domingo3"></a>
                  <span class="label">31</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Domingo</span>
                           <span tagx="given-names">C</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">de Ory</span>
                           <span tagx="given-names">F</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Sanz</span>
                           <span tagx="given-names">JC</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Reyes</span>
                           <span tagx="given-names">N</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gascon</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">2009</span>
                     <span tagx="article-title">Molecular and serologic markers of acute dengue infection in naive and flavivirus-vaccinated
                        travelers.
                     </span>
                     <span tagx="source">Diagn Microbiol Infect Dis</span>
                     <span tagx="volume">65</span>
                     <span tagx="fpage">42</span>
                     <span tagx="lpage">48</span>
                     <span class="pub-id'">[19679234]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Talarmin1"></a>
                  <span class="label">32</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Talarmin</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Labeau</span>
                           <span tagx="given-names">B</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Lelarge</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Sarthou</span>
                           <span tagx="given-names">JL</span>,
                        </span>
                     </span>
                     <span tagx="year">1998</span>
                     <span tagx="article-title">Immunoglobulin A-specific capture enzyme-linked immunosorbent assay for diagnosis
                        of dengue fever.
                     </span>
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">36</span>
                     <span tagx="fpage">1189</span>
                     <span tagx="lpage">1192</span>
                     <span class="pub-id'">[9574674]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Ansari1"></a>
                  <span class="label">33</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Ansari</span>
                           <span tagx="given-names">MZ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Shope</span>
                           <span tagx="given-names">RE</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Malik</span>
                           <span tagx="given-names">S</span>,
                        </span>
                     </span>
                     <span tagx="year">1993</span>
                     <span tagx="article-title">Evaluation of vero cell lysate antigen for the ELISA of flaviviruses.</span>
                     <span tagx="source">J Clin Lab Anal</span>
                     <span tagx="volume">7</span>
                     <span tagx="fpage">230</span>
                     <span tagx="lpage">237</span>
                     <span class="pub-id'">[8395596]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="pone.0036291-Forshey1"></a>
                  <span class="label">34</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Forshey</span>
                           <span tagx="given-names">BM</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Guevara</span>
                           <span tagx="given-names">C</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Laguna-Torres</span>
                           <span tagx="given-names">VA</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Cespedes</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Vargas</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="year">2010</span>
                     <span tagx="article-title">Arboviral etiologies of acute febrile illnesses in Western South America, 2000–2007.</span>
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="volume">4</span>
                     <span tagx="fpage">e787</span>
                     <span class="pub-id'">[20706628]</span>
                  </span>
               </li>
            </ul>
         </div>
      </div>
   </div>
</html>