<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <div xmlns="" tagx="article">
      <head>
         <style type="text/css"> table, tr, td {border : thin;}</style>
         <span class="citation_publisher"></span>
         <h2 class="citation-title">Structure of malaria invasion protein RH5 with erythrocyte basigin and blocking antibodies</h2>
         <span class="doi">10.1038/nature13715</span>
         <div class="contrib-group">contrib: 
            	  CONT
            	  
            <span class="citation_author">Katherine E. Wright</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Kathryn A. Hjerrild</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Jonathan Bartlett</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Alexander D. Douglas</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Jing Jin</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Rebecca E. Brown</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Joseph J. Illingworth</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Rebecca Ashfield</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Stine B. Clemmensen</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Willem A. de Jongh</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Simon J. Draper</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Matthew K. Higgins</span>
            <span class="citation_author_institution_ref"></span>
         </div>
         <span class="email">[matthew.higgins@bioch.ox.ac.uk]</span>
         <span class="conflict">[]</span>
         <span class="contribs">[]</span>
         <span class="citation_pubdate">[2014-8-17]</span>
         <span class="citation_copyright">[Copyright : ]</span>
         <span class="citation_licence">[Licence Users may view, print, copy, and download text and data-mine the content in
            such documents, for the purposes of academic research, subject always to the full
            Conditions of use:http://www.nature.com/authors/editorial_policies/license.html#terms]
         </span>
         <span class="citation_funding">[]</span>
         <span class="citation_pagecount">[]</span>
         <span class="citation_data">[]</span>
      </head>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Invasion of host erythrocytes is essential to the life cycle of 
         <i>Plasmodium</i> parasites and development of the pathology of malaria. The stages of erythrocyte
         invasion, including initial contact, apical reorientation, junction formation, and
         active invagination, are directed by coordinated release of specialised apical organelles
         and their parasite protein contents
         <sup>
            <span ref-type="bibr" rid="R1">
               <sup>
                  <a href="#R1">1</a>
               </sup>
            </span>
         </sup>. Among these proteins, and central to invasion by all species, are two parasite protein
         families, the reticulocyte-binding protein homologue (RH) and erythrocyte-binding
         like (EBL) proteins, that mediate host-parasite interactions
         <sup>
            <span ref-type="bibr" rid="R2">
               <sup>
                  <a href="#R2">2</a>
               </sup>
            </span>
         </sup>. RH5 from 
         <i>Plasmodium falciparum</i> (PfRH5) is the only member of either family demonstrated to be necessary for erythrocyte
         invasion in all tested strains, through its interaction with the erythrocyte surface
         protein basigin (CD147, EMMPRIN)
         <sup>
            <span ref-type="bibr" rid="R3">
               <sup>
                  <a href="#R3">3</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="R4">
               <sup>
                  <a href="#R4">4</a>
               </sup>
            </span>
         </sup>. Indeed, antibodies targeting PfRH5 or basigin efficiently block parasite invasion
         
         <i>in vitro</i>
         <sup>
            <span ref-type="bibr" rid="R4">
               <sup>
                  <a href="#R4">4</a>
               </sup>
            </span>-
            <span ref-type="bibr" rid="R9">
               <sup>
                  <a href="#R9">9</a>
               </sup>
            </span>
         </sup>, making PfRH5 an excellent vaccine candidate. Here we present crystal structures
         of PfRH5 in complex with basigin and two distinct inhibitory antibodies. PfRH5 adopts
         a novel fold in which two three-helical bundles come together in a kite-like architecture,
         presenting binding sites for basigin and inhibitory antibodies at one tip. This provides
         the first structural insight into erythrocyte binding by the 
         <i>Plasmodium</i> RH protein family and identifies novel inhibitory epitopes to guide design of a new
         generation of vaccines against the blood-stage parasite.
      </p>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Each 
         <i>Plasmodium</i> species contains at least one RH protein. These are often large, of low sequence
         complexity and with no homology to proteins of known structure. PfRH5 is unusual in
         being significantly shorter than its homologues (~60kDa for PfRH5 vs 200-375kDa for
         other RH proteins). It lacks their C-terminal transmembrane segment, but associates
         peripherally with the membrane and with PfRH5 interacting protein (PfRipr)
         <sup>
            <span ref-type="bibr" rid="R10">
               <sup>
                  <a href="#R10">10</a>
               </sup>
            </span>
         </sup>. Although it shares only ~20% pairwise sequence identity with other PfRH proteins
         <sup>
            <span ref-type="bibr" rid="R3">
               <sup>
                  <a href="#R3">3</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="R11">
               <sup>
                  <a href="#R11">11</a>
               </sup>
            </span>
         </sup>, PfRH5 is remarkably conserved, with only five common non-synonymous single nucleotide
         polymorphisms (SNPs)
         <sup>
            <span ref-type="bibr" rid="R7">
               <sup>
                  <a href="#R7">7</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="R8">
               <sup>
                  <a href="#R8">8</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="R12">
               <sup>
                  <a href="#R12">12</a>
               </sup>
            </span>
         </sup>. Crucially, antibodies raised against one PfRH5 variant neutralise parasites of all
         tested heterologous strains, containing these and other less common SNPs
         <sup>
            <span ref-type="bibr" rid="R6">
               <sup>
                  <a href="#R6">6</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="R8">
               <sup>
                  <a href="#R8">8</a>
               </sup>
            </span>
         </sup>, and anti-PfRH5 monoclonal antibodies that prevent parasite growth 
         <i>in vitro</i> can directly block the PfRH5:basigin interaction
         <sup>
            <span ref-type="bibr" rid="R5">
               <sup>
                  <a href="#R5">5</a>
               </sup>
            </span>
         </sup>. Moreover, acquisition of anti-PfRH5 antibodies during natural infection correlates
         with clinical outcome and these antibodies can inhibit parasite growth 
         <i>in vitro</i>
         <sup>
            <span ref-type="bibr" rid="R13">
               <sup>
                  <a href="#R13">13</a>
               </sup>
            </span>
         </sup>. These findings have generated intense excitement about PfRH5 as a next-generation
         blood-stage malaria vaccine target and emphasised the need for structural information
         to guide rational immunogen design.
      </p>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Structural studies of PfRH5 required a protein construct lacking flexible regions
         but still capable of binding basigin. Long disordered regions were predicted within
         residues 1-140 and 248-296 (
         <span ref-type="fig" rid="F4">Extended Data Fig. 1a</span>), and in cultured parasite lines, PfRH5 is processed by removal of the N-terminus
         to generate a ~45 kDa fragment
         <sup>
            <span ref-type="bibr" rid="R3">
               <sup>
                  <a href="#R3">3</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="R10">
               <sup>
                  <a href="#R10">10</a>
               </sup>
            </span>
         </sup>. We therefore designed PfRH5ΔNL, encompassing residues 140-526 but lacking 248-296,
         and showed it to bind basigin by surface plasmon resonance (SPR) with an affinity
         of 1.3 μM (
         <span ref-type="fig" rid="F1">Fig. 1C</span>), comparable to the affinity of full-length PfRH5 for basigin (1.1μM)
         <sup>
            <span ref-type="bibr" rid="R4">
               <sup>
                  <a href="#R4">4</a>
               </sup>
            </span>
         </sup>.
      </p>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To ensure that PfRH5ΔNL contains epitopes required to elicit an inhibitory immune
         response, we raised rabbit polyclonal IgG and tested their ability to neutralise parasites
         by a growth-inhibitory activity (GIA) assay (
         <span ref-type="fig" rid="F1">Fig. 1D</span>). IgG raised against PfRH5ΔNL protein showed a potent inhibitory effect, similar
         to that of IgG raised by immunisation of rabbits with viral vectors expressing full-length
         PfRH5
         <sup>
            <span ref-type="bibr" rid="R6">
               <sup>
                  <a href="#R6">6</a>
               </sup>
            </span>
         </sup>, or full-length PfRH5 recombinant protein
         <sup>
            <span ref-type="bibr" rid="R8">
               <sup>
                  <a href="#R8">8</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="R9">
               <sup>
                  <a href="#R9">9</a>
               </sup>
            </span>
         </sup>. We also tested binding of PfRH5ΔNL to a panel of mouse monoclonal antibodies previously
         characterized for PfRH5 binding and growth-inhibitory activity
         <sup>
            <span ref-type="bibr" rid="R5">
               <sup>
                  <a href="#R5">5</a>
               </sup>
            </span>
         </sup>. PfRH5ΔNL bound to growth-inhibitory antibodies including QA1, QA5, and 9AD4, but
         not to non-inhibitory 4BA7 and RB3 (
         <span ref-type="fig" rid="F5">Extended Data Fig. 2</span>). Thus, PfRH5ΔNL induces a growth-inhibitory immune response, and contains the epitopes
         targeted by inhibitory antibodies.
      </p>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">For structural studies, PfRH5ΔNL was mixed with basigin or fragments of growth-inhibitory
         monoclonal antibodies, 9AD4 or QA1. The complexes were trimmed using GluC protease
         and lysines chemically methylated before crystallisation. Crystals formed and data
         were collected to 3.1Å (PfRH5ΔNL:basigin), 2.3Å (PfRH5ΔNL:9AD4) and 3.1Å resolution
         (PfRH5ΔNL:QA1). Structures were determined using molecular replacement (
         <span ref-type="table" rid="T1">
            <a href="#T1">Extended Data Table 1</a>
         </span>).
      </p>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">PfRH5 adopts a rigid, flat, ‘kite-shaped’ architecture with a pseudo-two-fold rotation
         symmetry and no similarity to known structures (
         <span ref-type="fig" rid="F1">Fig. 1A</span>). Each half is predominantly built from a three helical bundle, with the outermost
         helices containing significant kinks or breaks. The N-terminal half begins with a
         short, two-stranded β-sheet that crosses the long axis of the kite at its centre.
         This is followed by a single, short helix and two long, kinked helices connected by
         the truncated loop (containing 58 residues in full-length PfRH5). The C-terminal half
         is simpler, consisting of three long helices that span the entire length of the domain
         and finishing with a flexible C-terminus. One disulphide bond (C345-C351) stabilises
         the loop that links the two halves of the structure, while another links the second
         and third helices (C224-C317), leaving one unpaired cysteine (C329).
      </p>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">PfRH5 is predominantly rigid, with five copies in the three different crystal forms
         aligning with an rmsd of 0.9Å over 95% of residues (
         <span ref-type="fig" rid="F4">Extended Data Fig. 1b</span>). Only the C-terminus (residue 496-end) and the loop linking helices 4 and 5 (residues
         396-406) adopt different positions in different crystal forms. A molecular envelope
         derived from small angle X-ray scattering (SAXS) analysis of full-length PfRH5 in
         solution exhibits a similar flat structure (
         <span ref-type="fig" rid="F1">Fig. 1B</span>, 
         <span ref-type="fig" rid="F6">Extended Data Figure 3</span>). This envelope is elongated relative to PfRH5ΔNL, most likely due to residues missing
         in this construct or not ordered in the crystal structure (22 residues at the C-terminus,
         the flexible loop, and perhaps part of the extended N-terminus).
      </p>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">As members of the 
         <i>Plasmodium</i> RH family share little sequence identity, sequence alignments and structure-based
         threading were used to predict whether other members contain the PfRH5 fold. In each
         protein analysed (
         <i>P. falciparum</i> RH1, RH2a, RH2b, RH3 and RH4; 
         <i>P. vivax</i> RBP-1 and RBP-2; 
         <i>P. reichenowi</i> RH5; and 
         <i>P. yoelii</i> Py01365), N-terminal PfRH5-like domains were identified with high confidence, despite
         sequence identities of 14-22% and a lack of totally conserved residues or disulphide
         bonds (
         <span ref-type="fig" rid="F7">Extended Data Fig. 4a</span>). Similar residues are located primarily in the interior of the domain, where they
         may stabilise the fold (
         <span ref-type="fig" rid="F7">Extended Data Fig. 4b</span>). In PfRH4, the only other RH protein with a known erythrocyte receptor, the complement
         receptor 1 (CR1) binding fragment contains the putative PfRH5 fold
         <sup>
            <span ref-type="bibr" rid="R14">
               <sup>
                  <a href="#R14">14</a>
               </sup>
            </span>
         </sup>. These PfRH5-like domains are therefore excellent candidates for ligand-binding modules
         in other RH proteins.
      </p>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Basigin binds at the tip of PfRH5, distant from the flexible loop and C-terminus,
         with both domains and the intervening linker directly contacting PfRH5 (
         <span ref-type="fig" rid="F2">Fig. 2A,D</span>). Most of the contact area (~1350 Å
         <sup>2</sup>) occurs through hydrogen bonds between the backbone of strands A and G of the basigin
         N-terminal domain and loops at the tip of PfRH5 (
         <span ref-type="table" rid="T2">
            <a href="#T2">Extended Data Table 2a</a>
         </span>). PfRH5 residues F350 and W447 stabilise this interaction by packing into hydrophobic
         pockets on basigin. The limited involvement of basigin side chains will reduce the
         potential for basigin escape mutants that prevent PfRH5 binding and impair parasite
         invasion.
      </p>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The basigin C-terminal domain and H102 in the linker also directly contact PfRH5 (
         <span ref-type="table" rid="T2">
            <a href="#T2">Extended Data Table 2a</a>
         </span>). The three loops at the tip of the basigin C-terminal domain (linking strands B
         and C, strands D and E and strands F and G) interact with the second and fourth helices
         of PfRH5 through hydrogen bonds and a hydrophobic patch contributed by residues VPP
         from the BC loop. However, flexibility of the basigin linker allows different orientations
         of the C-terminal domains in the two copies in the asymmetric unit of the crystal.
         Chain B interacts through the BC and DE loops (a ~650 Å
         <sup>2</sup> interface) while chain D interacts through the BC and FG loops (~480 Å
         <sup>2</sup>) (
         <span ref-type="fig" rid="F2">Fig. 2b</span>), leading to a maximum difference of ~18Å in the position of the C-terminus in the
         two complexes. Flexibility is also predicted from SAXS analysis of the complex in
         solution (
         <span ref-type="fig" rid="F6">Extended Data Fig. 3</span>). While PfRH5 and the basigin N-terminal domain fit the SAXS envelope, the C-terminal
         domain only partially fits, consistent with a flexible interaction with PfRH5.
      </p>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">PfRH5 is highly conserved, with just twelve non-synonymous SNPs in 227 field isolates,
         and only five at frequencies of 10% or greater
         <sup>
            <span ref-type="bibr" rid="R7">
               <sup>
                  <a href="#R7">7</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="R8">
               <sup>
                  <a href="#R8">8</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="R12">
               <sup>
                  <a href="#R12">12</a>
               </sup>
            </span>
         </sup>. These SNPs are distributed across the structure but do not affect residues that
         directly contact basigin (
         <span ref-type="fig" rid="F8">Extended Data Fig. 5</span>). By contrast, in sequenced laboratory strains, eight PfRH5 SNPs are associated with
         increased ability to invade 
         <i>Aotus</i> erythrocytes
         <sup>
            <span ref-type="bibr" rid="R15">
               <sup>
                  <a href="#R15">15</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="R16">
               <sup>
                  <a href="#R16">16</a>
               </sup>
            </span>
         </sup>. A number of these (I204, N347, Y358 and E362) are in or close to the basigin binding
         site, and may affect host tropism. Basigin residues which, when mutated, affect PfRH5
         affinity (F27, Q100 and H102)
         <sup>
            <span ref-type="bibr" rid="R17">
               <sup>
                  <a href="#R17">17</a>
               </sup>
            </span>
         </sup> are also located at the interface.
      </p>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The two PfRH5:basigin complexes in the asymmetric unit pack together through basigin-mediated
         contacts, including a ~911 Å
         <sup>2</sup> interface between the two basigin C-terminal domains, bringing their C-termini into
         close proximity (
         <span ref-type="fig" rid="F9">Extended Data Fig. 6</span>). As yet, the role of PfRH5 in invasion is uncertain, but it is tempting to speculate
         that this 2:2 complex assembles during invasion, mediating a signalling event in either
         parasite or erythrocyte to trigger an essential downstream process. This would leave
         one face of PfRH5 available for binding of PfRipr
         <sup>
            <span ref-type="bibr" rid="R10">
               <sup>
                  <a href="#R10">10</a>
               </sup>
            </span>
         </sup> and other, as yet unidentified, binding partners. However, in solution (at concentrations
         ≤24 μM) we observe no 2:2 complex, either through SAXS (
         <span ref-type="fig" rid="F6">Extended Data Fig. 3</span>) or analytical ultracentrifugation (
         <span ref-type="fig" rid="F2">Fig. 2C</span>, 
         <span ref-type="fig" rid="F10">Extended Data Fig 7</span>). Whether such a complex assembles at high local concentrations during invasion remains
         to be elucidated.
      </p>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To identify inhibitory epitopes, complexes of PfRH5ΔNL with Fab fragments from three
         inhibitory monoclonal antibodies were studied by crystallography and SAXS. QA1 and
         QA5 were previously shown to block PfRH5-basigin binding and parasite growth. 9AD4
         does not block PfRH5-basigin binding 
         <i>in vitro</i>, but is one of the most effective antibodies currently available for inhibiting parasite
         growth
         <sup>
            <span ref-type="bibr" rid="R5">
               <sup>
                  <a href="#R5">5</a>
               </sup>
            </span>
         </sup>. Crystal structures of PfRH5 bound to QA1 and 9AD4 were confirmed by SAXS, while
         a model for PfRH5:QA5 was derived from SAXS analysis, guided by a previously identified
         linear epitope (residues 201-213 from helix 2)
         <sup>
            <span ref-type="bibr" rid="R5">
               <sup>
                  <a href="#R5">5</a>
               </sup>
            </span>
         </sup>.
      </p>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The antibodies bind to three distinct sites, close to the vertex of PfRH5 (
         <span ref-type="fig" rid="F3">Fig. 3</span>, 
         <span ref-type="fig" rid="F11">Extended Data Fig. 8</span>, 
         <span ref-type="table" rid="T2">
            <a href="#T2">Extended Data Table 2b,c</a>
         </span>). QA1 binds to loops at the PfRH5 tip, overlapping the basigin N-terminal domain
         binding site. QA5 predominantly interacts with PfRH5 helix 2, overlapping the basigin
         C-terminal domain binding site. In contrast, 9AD4 binds helices 2 and 3, close to,
         but not overlapping, either basigin binding site. This is likely to allow intact 9AD4
         IgG to impede erythrocyte binding when PfRH5 and basigin are both membrane-tethered.
         This reveals inhibitory epitopes in, or close to, the basigin binding sites, that
         can be targeted to block parasite invasion.
      </p>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In summary, PfRH5 adopts a novel architecture, formed, as in many families of parasite
         surface proteins
         <sup>
            <span ref-type="bibr" rid="R18">
               <sup>
                  <a href="#R18">18</a>
               </sup>
            </span>
         </sup>, from a robust α-helical scaffold. This maintains the overall fold by retaining residues
         required for helical packing, while allowing significant surface sequence variation.
         Sequence homology identifies this fold at the N-terminus of other RH proteins, where
         it is likely to act as a ligand-binding module.
      </p>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Characterisation of the PfRH5:basigin complex prompts a range of future experiments
         to investigate the role of PfRH5 in erythrocyte invasion. Furthermore, monoclonal
         antibodies that block parasite growth bind at or close to the basigin-binding site.
         Immunogens containing these regions of PfRH5 will be important components of a vaccine
         to prevent 
         <i>P. falciparum</i> erythrocyte invasion, thereby crippling the parasite responsible for the deadliest
         form of human malaria.
      </p>
      <div id="S1">
         <h2>Methods</h2>
         <div id="S2">
            <h3>Basigin cloning, expression and purification</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A section of the basigin gene encoding immunoglobin domains 1 and 2 of the short isoform
               (residues 22-205) was amplified from cDNA using primers: 5′-CCGGATCCGCTGCCGGAACCGTGTTC-3′
               and CCCATATGCTAGTGTGACCGCACTCTCAGG-3′. PCR products were cloned into a modified pET15b
               vector (Novagen), which encodes an N-terminal hexa-histidine tag followed by a tobacco
               etch virus (TEV) protease cleavage site. TEV cleavage leaves an additional glycine
               at the N-terminus from the cleavage site.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Basigin was expressed in bacterial strain Origami B (DE3) (Novagen) by incubation
               overnight at 25°C after induction with 1mM IPTG. The protein was purified by nickel-nitrilotriacetic
               acid (Ni
               <sup>2+</sup>-NTA; Qiagen) affinity chromatography, followed by buffer exchange into PBS using
               a PD-10 desalting column (GE Healthcare), and overnight cleavage with His-tagged TEV
               protease at 4°C, before a second Ni
               <sup>2+</sup>-NTA column. The flow-through was concentrated using an Amicon Ultra centrifugal filter
               device (molecular mass cutoff, 3,000 Da). Finally, gel filtration was performed with
               a Superdex 200 16/60 column (GE Healthcare) in 20 mM HEPES (pH 7.5) and 150 mM NaCl.
            </p>
         </div>
         <div id="S3">
            <h3>Expression and purification of 9AD4, QA1 and QA5</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Anti-PfRH5 monoclonal antibodies were described previously
               <sup>
                  <span ref-type="bibr" rid="R5">
                     <sup>
                        <a href="#R5">5</a>
                     </sup>
                  </span>
               </sup>. Hybridomas expressing 9AD4, QA1 and QA5 were grown in Dulbecco’s Modified Eagle’s
               Medium (DMEM; Sigma) supplemented with 4 mM L-glutamine (Sigma), 0.01 M HEPES (Life
               Technologies), 100 U penicillin and 0.1 mg/ml streptomycin (Sigma), and 20% fetal
               calf serum (Gibco). They were then transferred into CD Hybridoma medium (Life Technologies)
               with glutamine, penicillin, and streptomycin. The cells were harvested after 7-10
               days. The cell culture supernatant was exchanged into 20 mM phosphate pH 7.0 with
               a tangential flow filtration device (Pall).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The sample was then loaded onto a HiTrap Protein G HP column (GE Healthcare), eluted
               in 0.1 M glycine-HCl (pH 3.0), and immediately neutralised with 0.1 M Tris (pH 8.0).
               The sample was exchanged into 100 mM phosphate (pH 6.4), 300 mM NaCl, 2 mM EDTA, 5
               mM L-cysteine (pH 6.4), and 1.5 mM β-mercaptoethanol using PD-10 columns (GE Healthcare).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Antibody fragments were generated by addition of papain agarose (Sigma), and overnight
               incubation at 37°C. The papain agarose was removed by centrifugation, and the sample
               loaded onto a HiTrap Protein A HP column (GE Healthcare). The material that did not
               bind to the column was gel filtered on a Superdex 200 16/60 column (GE Healthcare)
               in 20 mM HEPES (pH 7.5) and 150 mM NaCl.
            </p>
         </div>
         <div id="S4">
            <h3>PfRH5 cloning, expression and purification</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A synthetic gene was designed encoding full-length PfRH5 residues E26-Q526 from 
               <i>Plasmodium falciparum</i> strain 7G8, codon optimised for expression in 
               <i>Drosophila melanogaster</i>, with four putative glycosylation sites removed by mutation of asparagines to alanines
               (Life Technologies). The gene was flanked by an EcoRI site and Kozak sequence at the
               5′ end, and a NotI site at the 3′ end. Long-range disorder was predicted using POODLE-L
               <sup>
                  <span ref-type="bibr" rid="R19">
                     <sup>
                        <a href="#R19">19</a>
                     </sup>
                  </span>
               </sup>. A shortened mutant, PfRH5ΔN, lacking the flexible N-terminus, was also synthesised,
               encompassing residues K140-Q526. A further mutant, PfRH5ΔNL, was generated by removal
               of a predicted disordered loop (residues N248-M296) in addition to the N-terminal
               truncation. All constructs contained a C-terminal hexa-histidine tag. PfRH5 constructs
               were subcloned into pExpreS
               <sup>2</sup>-1 (ExpreS
               <sup>2</sup>ion Biotechnologies, Horsholm, Denmark).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Constructs were transfected into Schneider 2 (S2) cells using ExpreS
               <sup>2</sup> Insect-TRx5 (ExpreS
               <sup>2</sup>ion) transfection reagent, and a polyclonal cell line was selected over three weeks
               in EX-CELL 420 insect cell media with L-glutamine (Sigma) supplemented with 10% fetal
               calf serum (Gibco) and 1.5 mg/ml zeocin. The stable cell lines were then expanded
               in EX-CELL 420 media without fetal calf serum or zeocin. Four days after the final
               split, the cells were harvested, and the cell culture supernatant exchanged into 20
               mM phosphate, pH 7.4, 250 mM NaCl, 20 mM imidazole, and 0.005% Tween-20, using a tangential
               flow filtration device (Pall). Protein purification from the supernatant then proceeded
               through Ni
               <sup>2+</sup>-NTA (HisTrap FF; GE Healthcare) and ConA lectin-binding chromatography (HiTrap ConA
               4B; GE Healthcare).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Full-length PfRH5, PfRH5ΔN and PfRH5ΔNL for biophysical analysis were subjected to
               gel filtration on a Superdex 200 16/60 column (GE Healthcare), and concentrated with
               an Amicon centrifugal filter device (molecular weight cut-off, 10,000 Da).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To prepare PfRH5ΔNL for crystallisation, the sample was cleaved overnight with endoproteinase
               GluC (Sigma) at 20°C. At this point, basigin, 9AD4, or QA1 was added to the protein.
               To methylate the complex, 0.02 M dimethylamine-borane complex (ABC) and 0.04 M formaldehyde
               were added and the reaction incubated at 4°C for 1 hour. An additional 0.02 M ABC
               and 0.04 M formaldehyde were added; after another 1 hour, 0.01 M ABC was added and
               the reaction was incubated at 4°C overnight. Finally, the complexes were gel filtered
               and concentrated as detailed above.
            </p>
         </div>
         <div id="S5">
            <h3>Surface plasmon resonance</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Surface plasmon resonance experiments were carried out using a BIAcore T200 instrument
               (GE Healthcare). Experiments were performed at 20°C in 10 mM HEPES (pH 7.4), 150 mM
               NaCl, 3 mM EDTA, 0.005% Tween 20, 2 mg/ml dextran, and 1 mg/ml salmon sperm DNA (Sigma
               Aldrich). Basigin was immobilized on a CM5 chip (GE Healthcare) by amine coupling
               (GE Healthcare kit) to a total of 850 RU. A concentration series of PfRH5ΔNL (8, 4,
               2, 1, 0.5, 0.25, 0.125, and 0.0625 μM) was injected over the basigin-coated chip for
               120 s at 30 μl/min, followed by a 300 s dissociation time. The chip surface was then
               regenerated with 30 s of 2 M NaCl. Specific binding of PfRH5ΔNL was obtained by subtracting
               the response from a blank surface from that of the basigin-coated surface. The kinetic
               sensorgrams were fitted to a global 1:1 interaction model, allowing determination
               of the dissociation constant, 
               <i>K
                  <div tagxxx="sub">d</div>
               </i>, using BIAevaluation software 1.0 (GE Healthcare).
            </p>
         </div>
         <div id="S6">
            <h3>Assays of growth inhibition activity (GIA)</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Rabbit immunisations were carried out by Biogenes (Germany). Female ZiKa rabbits (n=4)
               were immunised intramuscularly (i.m.) with 20μg PfRH5ΔNL protein formulated in complete
               Freund’s adjuvant on day 0, followed by two booster immunisations i.m. on days 28
               and 56 with the same dose of protein formulated in incomplete Freund’s adjuvant. Control
               rabbits received the same immunisation schedule with 50μg ovalbumin protein. Sera
               were collected two weeks after the final immunisation and shipped frozen.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Total IgG was purified from rabbit sera using protein G columns (Pierce). The 
               <i>P. falciparum</i> 3D7 and 7G8 lines were maintained in continuous culture using fresh O
               <sup>+</sup> erythrocytes at 2% haematocrit and synchronised by two incubations in 5% sorbitol
               6-8 h apart. Synchronised trophozoites were adjusted to 0.3% parasitaemia and then
               incubated for 42 h with the various IgG concentrations (tested in triplicate). Final
               parasitaemia was determined by biochemical determination of parasite lactate dehydrogenase
               <sup>
                  <span ref-type="bibr" rid="R20">
                     <sup>
                        <a href="#R20">20</a>
                     </sup>
                  </span>
               </sup>. Percentage growth inhibition is expressed relative to wells containing IgG from
               control immunised rabbits
               <sup>
                  <span ref-type="bibr" rid="R6">
                     <sup>
                        <a href="#R6">6</a>
                     </sup>
                  </span>
               </sup>. The mean of the three replicate wells was taken to obtain the final data for each
               individual rabbit at each tested IgG concentration. Experiments were performed twice
               against each strain of parasite with very similar results. A representative experiment
               is shown in 
               <span ref-type="fig" rid="F1">Fig. 1D</span>. The mean ± standard error of mean are plotted for the results obtained with the
               purified IgG from the four immunised rabbits.
            </p>
         </div>
         <div id="S7">
            <h3>ELISA</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Maxisorp plates were coated with anti-PfRH5 mouse monoclonal antibody
               <sup>
                  <span ref-type="bibr" rid="R5">
                     <sup>
                        <a href="#R5">5</a>
                     </sup>
                  </span>
               </sup> (50μL/well at 5μg/mL) and left at 4°C overnight. Following six washes in PBS with
               0.05% Tween-20 (PBS-T), plates were blocked with 150 μL/well Casein block solution
               (Pierce) for 1 h, followed by another six PBS-T washes. PfRH5ΔNL protein was added
               to wells in triplicate (50 μL/well) at four dilutions in blocking buffer (800, 200,
               50 and 12.5 ng/mL) and incubated at room temperature for 2 h. Following another six
               PBS-T washes, plates were incubated with polyclonal anti-full-length PfRH5 rabbit
               serum
               <sup>
                  <span ref-type="bibr" rid="R6">
                     <sup>
                        <a href="#R6">6</a>
                     </sup>
                  </span>
               </sup> diluted 1:1000 in blocking solution, 50 μL/well for 1 h. Following another PBS-T
               wash, plates were incubated with goat anti-rabbit IgG alkaline phosphatase diluted
               1:5000 in blocking solution, 50 μL/well for 1 h. Following a final six PBS-T washes,
               and two PBS washes, plates were developed and optical density read at 405nm as previously
               described
               <sup>
                  <span ref-type="bibr" rid="R21">
                     <sup>
                        <a href="#R21">21</a>
                     </sup>
                  </span>
               </sup>.
            </p>
         </div>
         <div id="S8">
            <h3>Crystallisation, data collection and data processing of PfRH5 complexes</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Crystallisation was achieved using vapour diffusion in sitting drops. A TTP Labtech
               Mosquito LCP robot was employed to mix 100 nl of each protein complex at 8 mg/ml with
               100 nl of well solutions from commercially available crystal screens. Crystals were
               first obtained in conditions from the Proplex (RH5ΔNL:basigin), JCSG+ (RH5ΔNL:9AD4)
               and Morpheus (RH5ΔNL:QA1) crystal screens (Molecular Dimensions). The well solution
               for PfRH5ΔNL:basigin was 0.2 M lithium sulphate, 0.1 M MES (pH 6.0), and 20% (w/v)
               PEG 4000. For PfRH5ΔNL:9AD4, the well solution was 24% (w/v) PEG 1500 and 20% (v/v)
               glycerol. PfRH5ΔNL:QA1 was crystallised in a 0.12 M mix of 1,6-hexanediol, 1-butanol,
               1,2-propanediol, 2-propanol, 1,4-butanediol, 1,3-propanediol; 0.1 M MES-imidazole
               (pH 6.5); 20% (w/v) PEG 550 MME; and 10% (w/v) PEG 20,000. Crystals were obtained
               at 4°C for PfRH5ΔNL:basigin and PfRH5ΔNL:9AD4, and at 18°C for PfRH5ΔNL:QA1. Crystal
               seeds were generated by adding crystals in well solution to a Seed Bead (Hampton Research)
               and vortexing for 15 seconds. New crystal drops were then set up by mixing 100 nl
               of 8 mg/ml protein complex, 50 nl of well solution, 50 nl of seeds, and 50 nl of Silver
               Bullets or Silver Bullets Bio additives (Hampton Research). RH5ΔNL:basigin crystals
               were cryoprotected by transfer into well solution supplemented with 25% ethylene glycol,
               then cryocooled by plunging into nitrogen. PfRH5ΔNL:9AD4 crystals were directly cryocooled.
               PfRH5ΔNL:QA1 crystals were cryoprotected in well solution with a total of 32% (w/v)
               PEG 550 MME and 16% (w/v) PEG 20,000 before cryocooling.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Data were collected at the Diamond Light Source, on beamline I04-1 for the PfRH5ΔNL:basigin
               complex (wavelength 0.92 Å), and on beamline I04 for both PfRH5ΔNL:Fab complexes (wavelength
               0.97949 Å). Data reduction was performed using iMosflm
               <sup>
                  <span ref-type="bibr" rid="R22">
                     <sup>
                        <a href="#R22">22</a>
                     </sup>
                  </span>,
                  <span ref-type="bibr" rid="R23">
                     <sup>
                        <a href="#R23">23</a>
                     </sup>
                  </span>
               </sup> and scala
               <sup>
                  <span ref-type="bibr" rid="R24">
                     <sup>
                        <a href="#R24">24</a>
                     </sup>
                  </span>
               </sup> from the CCP4 processing suite
               <sup>
                  <span ref-type="bibr" rid="R25">
                     <sup>
                        <a href="#R25">25</a>
                     </sup>
                  </span>
               </sup>. A composite of the light chain from PDB entry 3QQ9 and the heavy chain from 3HR5
               was used as a molecular replacement model for 9AD4 in Phaser
               <sup>
                  <span ref-type="bibr" rid="R26">
                     <sup>
                        <a href="#R26">26</a>
                     </sup>
                  </span>
               </sup>. Density modification was performed with PARROT
               <sup>
                  <span ref-type="bibr" rid="R27">
                     <sup>
                        <a href="#R27">27</a>
                     </sup>
                  </span>
               </sup>. Buccaneer
               <sup>
                  <span ref-type="bibr" rid="R28">
                     <sup>
                        <a href="#R28">28</a>
                     </sup>
                  </span>
               </sup> was employed to build helices into initial PfRH5ΔNL density, initiating a cycle of
               model building and refinement using REFMAC
               <sup>
                  <span ref-type="bibr" rid="R29">
                     <sup>
                        <a href="#R29">29</a>
                     </sup>
                  </span>
               </sup>, BUSTER
               <sup>
                  <span ref-type="bibr" rid="R30">
                     <sup>
                        <a href="#R30">30</a>
                     </sup>
                  </span>
               </sup>, and Coot
               <sup>
                  <span ref-type="bibr" rid="R31">
                     <sup>
                        <a href="#R31">31</a>
                     </sup>
                  </span>
               </sup>. PfRH5ΔNL was used as a molecular replacement model in Phaser
               <sup>
                  <span ref-type="bibr" rid="R26">
                     <sup>
                        <a href="#R26">26</a>
                     </sup>
                  </span>
               </sup>, together with basigin (PDB code 3B5H) for the PfRH5ΔNL:basigin structure, or with
               a composite of the heavy chain from 2ZN9 and the light chain from 1I7Z for the PfRH5ΔNL:QA1
               structure. Refinement was accomplished with BUSTER
               <sup>
                  <span ref-type="bibr" rid="R30">
                     <sup>
                        <a href="#R30">30</a>
                     </sup>
                  </span>
               </sup>. Ramachandran statistics, determined using PROCHECK
               <sup>
                  <span ref-type="bibr" rid="R32">
                     <sup>
                        <a href="#R32">32</a>
                     </sup>
                  </span>
               </sup>, were as follows: for PfRH5ΔNL:basigin, 93% of residues fell in the allowed regions
               and 7% in the additional allowed region; for PfRH5ΔNL:9AD4, 93.9% were in the allowed
               region, 5.8% were in the additional allowed region, and 0.2% were in the generously
               allowed region; and for PfRH5ΔNL:QA1, 86.0% were in the allowed region, 13.3% were
               in the additional allowed region, and 0.7% were in the generously allowed region.
               There were no residues in the disallowed region of the Ramachandran plot in any of
               the structures.
            </p>
         </div>
         <div id="S9">
            <h3>Alignment of RH protein sequences</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Sequences of 
               <i>P. falciparum</i> RH1, RH4, RH2a, RH2b, and RH3; 
               <i>P. vivax</i> RBP-1 and RBP-2; 
               <i>P. reichenowi</i> RH5; and 
               <i>P. yoelii</i> Py01365 were aligned against PfRH5 using the Clustal Omega server
               <sup>
                  <span ref-type="bibr" rid="R33">
                     <sup>
                        <a href="#R33">33</a>
                     </sup>
                  </span>
               </sup>, and an alignment figure generated with ESPript
               <sup>
                  <span ref-type="bibr" rid="R34">
                     <sup>
                        <a href="#R34">34</a>
                     </sup>
                  </span>
               </sup>. The homologous portion of each sequence was then threaded onto the PfRH5 structure
               using the Phyre2 server
               <sup>
                  <span ref-type="bibr" rid="R35">
                     <sup>
                        <a href="#R35">35</a>
                     </sup>
                  </span>
               </sup>.
            </p>
         </div>
         <div id="S10">
            <h3>Small-angle X-ray scattering data collection and processing</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Small-angle X-ray scattering (SAXS) data were collected at beamline BM29 at the European
               Synchrotron Radiation Facility (ESRF, Grenoble, France). Scattering was detected using
               a Pilatus image reader at 20°C.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Full-length PfRH5:basigin, PfRH5:QA1 and PfRH5:QA5 complexes were prepared by mixing
               the individual proteins in equimolar amounts, and gel filtering on a Superdex 200
               16/60 column in 20 mM HEPES (pH 7.5), 150 mM NaCl. PfRH5:9AD4 samples were prepared
               by mixing PfRH5 and 9AD4 in equimolar amounts for 10 minutes. 60 μl samples were prepared
               in serial dilutions between 0.2-2.0 mg/ml.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Ten consecutive frames of 10 s each were recorded for each protein sample, with buffer
               samples measured in between. The frames were inspected for any evidence of radiation
               damage, and any affected images were excluded from further processing. Data were normalised
               to the intensity of the incident beam, images were averaged, and buffer scattering
               subtracted using PRIMUS
               <sup>
                  <span ref-type="bibr" rid="R36">
                     <sup>
                        <a href="#R36">36</a>
                     </sup>
                  </span>,
                  <span ref-type="bibr" rid="R37">
                     <sup>
                        <a href="#R37">37</a>
                     </sup>
                  </span>
               </sup>. Guinier plots were inspected to assess the presence of aggregation
               <sup>
                  <span ref-type="bibr" rid="R38">
                     <sup>
                        <a href="#R38">38</a>
                     </sup>
                  </span>
               </sup>. Composite curves were generated by scaling and merging appropriate data sets. The
               radius of gyration (R
               <div tagxxx="sub">g</div>) was determined from the Guinier plot using AutoRg
               <sup>
                  <span ref-type="bibr" rid="R37">
                     <sup>
                        <a href="#R37">37</a>
                     </sup>
                  </span>
               </sup>. The distance distribution function (P(r)) was obtained by indirect Fourier transform,
               generating an estimate of the maximum particle dimension (D
               <div tagxxx="sub">max</div>) and the Porod volume of the hydrated particle
               <sup>
                  <span ref-type="bibr" rid="R37">
                     <sup>
                        <a href="#R37">37</a>
                     </sup>
                  </span>,
                  <span ref-type="bibr" rid="R39">
                     <sup>
                        <a href="#R39">39</a>
                     </sup>
                  </span>
               </sup>. An estimate of the molecular weight was obtained by dividing the Porod volume by
               1.7. Twenty 
               <i>ab initio</i> shape reconstructions were generated with DAMMIF
               <sup>
                  <span ref-type="bibr" rid="R40">
                     <sup>
                        <a href="#R40">40</a>
                     </sup>
                  </span>
               </sup> and averaged with DAMAVER
               <sup>
                  <span ref-type="bibr" rid="R41">
                     <sup>
                        <a href="#R41">41</a>
                     </sup>
                  </span>
               </sup>. The normalized spatial discrepancy (NSD) parameter was used to diagnose the similarity
               of the reconstructions
               <sup>
                  <span ref-type="bibr" rid="R42">
                     <sup>
                        <a href="#R42">42</a>
                     </sup>
                  </span>
               </sup>. SITUS and SCULPTOR
               <sup>
                  <span ref-type="bibr" rid="R43">
                     <sup>
                        <a href="#R43">43</a>
                     </sup>
                  </span>
               </sup> were used to dock atomic structures into the SAXS envelopes.
            </p>
         </div>
         <div id="S11">
            <h3>Analytical ultracentrifugation</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Analytical ultracentrifugation experiments were carried out using a Beckman An-60
               Ti rotor in a Beckman Optima XL-1 analytical ultracentrifuge at 20°C. Samples were
               prepared at concentrations of 0.5-1.0 mg/ml in 20 mM HEPES (pH 7.5), and 150 mM NaCl.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The samples for sedimentation velocity experiments were full-length PfRH5, basigin,
               and a PfRH5:basigin complex prepared by gel filtration. Scans were recorded using
               interference optics. Multiple scans were fit to a continuous size distribution using
               SEDFIT
               <sup>
                  <span ref-type="bibr" rid="R44">
                     <sup>
                        <a href="#R44">44</a>
                     </sup>
                  </span>,
                  <span ref-type="bibr" rid="R45">
                     <sup>
                        <a href="#R45">45</a>
                     </sup>
                  </span>
               </sup>. The buffer density and viscosity were calculated with SEDNTERP
               <sup>
                  <span ref-type="bibr" rid="R46">
                     <sup>
                        <a href="#R46">46</a>
                     </sup>
                  </span>
               </sup>. The sample partial specific volumes were estimated from the amino acid sequences,
               also using SEDNTERP.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">During the time course of a sedimentation equilibrium experiment, degradation of full-length
               PfRH5 was observed. Therefore experiments were performed using PfRH5ΔN. Basigin, PfRH5ΔN
               and a gel filtered PfRH5ΔN:basigin complex were analysed at 22 μM, 9 μM, and 9 μM,
               respectively. The run proceeded for 20 hours at different speeds and scans recorded
               the UV absorbance at 280 nm. The buffer density and viscosity, together with sample
               partial specific volumes, were calculated as above
               <sup>
                  <span ref-type="bibr" rid="R46">
                     <sup>
                        <a href="#R46">46</a>
                     </sup>
                  </span>
               </sup>. The data were fit to an ideal monodisperse model using SEDPHAT
               <sup>
                  <span ref-type="bibr" rid="R47">
                     <sup>
                        <a href="#R47">47</a>
                     </sup>
                  </span>
               </sup>, with the assumption that at concentrations of ~10 times the K
               <div tagxxx="sub">D</div>, PfRH5ΔN is fully complexed with basigin.
            </p>
         </div>
      </div>
      <div id="S12">
         <h2>Extended Data</h2>
         <div tagxxx="fig">
            <span class="label">Extended Data Figure 1</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">PfRH5 disorder predictions and structural alignment</div>
               <p>
                  <b>a,</b> Long-range disorder was predicted by POODLE-L
                  <sup>
                     <span ref-type="bibr" rid="R19">
                        <sup>
                           <a href="#R19">19</a>
                        </sup>
                     </span>
                  </sup> and was used to determine domain boundaries for the PfRH5ΔNL crystallisation construct.
                  The disorder predictions are shown above the sequence of PfRH5, with values of &gt;0.5
                  indicative of disorder. The residues visible in the PfRH5ΔNL crystal structure are
                  shown below the PfRH5 sequence as secondary structure elements (sheets as blue arrows,
                  and helices as tubes in rainbow coloring) linked by blue lines. The missing loop (248-296)
                  is shown as a break in the blue line. The secretion signal sequence is indicated (black
                  underline). 
                  <b>b,</b> Two copies of PfRH5 from the PfRH5:basigin structure (red and orange), two copies
                  from the PfRH5:QA1 structure (blue and cyan), and one copy from PfRH5:9AD4 (green)
                  structure were aligned using Coot
                  <sup>
                     <span ref-type="bibr" rid="R31">
                        <sup>
                           <a href="#R31">31</a>
                        </sup>
                     </span>
                  </sup>, giving an RMSD of 1.7 Å. The C-terminus and the loop between helices 4 and 5 were
                  the only regions showing significant differences. For the remaining 95% of PfRH5,
                  the RMSD is 0.9 Å.
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <span class="label">Extended Data Figure 2</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Investigation of the interaction of PfRH5ΔNL with a panel of mouse mAbs using ELISA</div>
               <p>Five mAbs that bind to PfRH5 were coated on a ELISA plate and probed using PfRH5ΔNL
                  at concentrations of 12.5, 50, 200 or 800 ng/ml. Antibodies 9AD5, QA1 and QA5 interacted
                  with PfRH5ΔNL while RB3 and 4BA7 did not. Indeed, RB3 and 4BA7 bind to the flexible
                  N-terminus and the truncated loop, respectively, both features lacking in PfRH5ΔNL
                  <sup>
                     <span ref-type="bibr" rid="R5">
                        <sup>
                           <a href="#R5">5</a>
                        </sup>
                     </span>
                  </sup>. The error bars are standard error of mean (n=3).
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <span class="label">Extended Data Figure 3</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">SAXS analysis of the PfRH5:basigin complex</div>
               <p>
                  <b>a</b>, The theoretical scattering calculated from the average of 20 
                  <i>ab initio</i> reconstructions (continuous lines, with PfRH5 in orange and PfRH5:basigin in blue)
                  plotted with the experimental scattering intensity curves (diamonds). The data are
                  presented as the natural logarithm of the intensity. Guinier plots are displayed in
                  the inset. 
                  <b>b</b>, The distance distribution function, P(r), of PfRH5 (orange) and PfRH5:basigin (blue).
                  
                  <b>c</b>, To the left, the crystal structure of PfRH5ΔNL (yellow) was docked into the average
                  
                  <i>ab initio</i> SAXS envelope of full-length PfRH5 (grey). Extra density corresponding to some or
                  all of the truncated regions is visible at the bottom of the kite-like structure,
                  near the C-terminus. To the right, the crystal structure of PfRH5ΔNL:basigin is docked
                  into the average 
                  <i>ab initio</i> SAXS envelope of full-length PfRH5:basigin (grey). PfRH5ΔNL is yellow. In dark blue
                  and cyan are basigin molecules from the two PfRH5ΔNL:basigin complexes in the asymmetric
                  unit, superimposed based on the structure of PfRH5ΔNL. 
                  <b>d</b>, Summary of SAXS parameters. The radius of gyration (R
                  <div tagxxx="sub">g</div>) was determined from the Guinier plot using AutoRg
                  <sup>
                     <span ref-type="bibr" rid="R37">
                        <sup>
                           <a href="#R37">37</a>
                        </sup>
                     </span>
                  </sup>, and the maximum particle dimension (Dmax) and the Porod volume
                  <sup>
                     <span ref-type="bibr" rid="R39">
                        <sup>
                           <a href="#R39">39</a>
                        </sup>
                     </span>
                  </sup> were calculated using GNOM
                  <sup>
                     <span ref-type="bibr" rid="R37">
                        <sup>
                           <a href="#R37">37</a>
                        </sup>
                     </span>
                  </sup>. An estimate of the molecular weight was obtained by dividing the Porod volume by
                  1.7. 
                  <i>Ab initio</i> modeling was used to generate 20 shape reconstructions from the data. The normalised
                  spatial discrepancy parameter (NSD) diagnoses the similarity of these models
                  <sup>
                     <span ref-type="bibr" rid="R42">
                        <sup>
                           <a href="#R42">42</a>
                        </sup>
                     </span>
                  </sup>. The models were averaged and the fit of the average model to the experimental data
                  is indicated by the χ value.
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <span class="label">Extended Data Figure 4</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">A conserved PfRH5-like fold in other 
                  <i>Plasmodium</i> RH proteins
               </div>
               <p>
                  <b>a,</b>
                  <i>P. falciparum</i> RH1, RH4, RH2b, RH2a and RH3 (a pseudogene); 
                  <i>P. vivax</i> RBP-1 and RBP-2; 
                  <i>P. reichenowi</i> RH5; and 
                  <i>P. yoelii</i> Py01365 were aligned using Clustal Omega
                  <sup>
                     <span ref-type="bibr" rid="R33">
                        <sup>
                           <a href="#R33">33</a>
                        </sup>
                     </span>
                  </sup> and were threaded using the Phyre2 server
                  <sup>
                     <span ref-type="bibr" rid="R35">
                        <sup>
                           <a href="#R35">35</a>
                        </sup>
                     </span>
                  </sup>, giving more than 98% confidence of fold conservation over &gt;260 residues in each
                  case. The secondary structure of PfRH5 is shown below the sequence in a rainbow colour
                  scheme as in 
                  <span ref-type="fig" rid="F1">Fig. 1A</span>. Residues from PfRH5 that interact with basigin, QA1 and 9AD4 are indicated above
                  the sequence by blue, red or green stars respectively. Cysteine residues that make
                  disulphide bonds are indicated by pink numbers, with residues sharing the same number
                  forming a disulphide bond. 
                  <b>b,</b> PfRH5 is shown in yellow, with residues similar among RH proteins (from the alignment
                  in 
                  <span ref-type="fig" rid="F7">extended data figure 4</span>) highlighted as pink sticks. The majority of the similar residues appear to play
                  a structural role stabilising the architecture of the domain.
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <span class="label">Extended Data Figure 5</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Location of PfRH5 polymorphisms, and residues of PfRH5 and basigin implicated in host
                  tropism
               </div>
               <p>
                  <b>a</b> and 
                  <b>b</b>, Indicated are the locations of PfRH5 SNPs that are common (10% frequency or greater;
                  red sticks) or uncommon (blue sticks) among 227 field isolates
                  <sup>
                     <span ref-type="bibr" rid="R7">
                        <sup>
                           <a href="#R7">7</a>
                        </sup>
                     </span>,
                     <span ref-type="bibr" rid="R8">
                        <sup>
                           <a href="#R8">8</a>
                        </sup>
                     </span>,
                     <span ref-type="bibr" rid="R12">
                        <sup>
                           <a href="#R12">12</a>
                        </sup>
                     </span>
                  </sup>, as well as additional SNPs observed in lab strains (green sticks)
                  <sup>
                     <span ref-type="bibr" rid="R15">
                        <sup>
                           <a href="#R15">15</a>
                        </sup>
                     </span>,
                     <span ref-type="bibr" rid="R16">
                        <sup>
                           <a href="#R16">16</a>
                        </sup>
                     </span>
                  </sup>. 
                  <b>b</b>, Basigin (blue) is shown in addition to PfRH5 (yellow). SNPs Y203, I204, N347, Y358,
                  and E362 are localized in or near the PfRH5:basigin interface. Not visible in this
                  orientation is lab strain polymorphism K429. 
                  <b>c</b>, Highlighted are basigin residues F27, Q100, and H102, which affect the affinity
                  for PfRH5 when mutated
                  <sup>
                     <span ref-type="bibr" rid="R17">
                        <sup>
                           <a href="#R17">17</a>
                        </sup>
                     </span>
                  </sup> (orange sticks). Also shown are two SNPs of PfRH5, namely N347 and I204 (pink sticks),
                  found in the PfRH5:basigin binding interface and linked to the strain’s ability to
                  invade 
                  <i>Aotus</i> monkey erythrocytes
                  <sup>
                     <span ref-type="bibr" rid="R15">
                        <sup>
                           <a href="#R15">15</a>
                        </sup>
                     </span>
                  </sup>.
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <span class="label">Extended Data Figure 6</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Arrangement of two PfRH5:basigin complexes in the asymmetric unit of the crystal</div>
               <p>One complex, shown in yellow (PfRH5) and blue (basigin), interacts with the second,
                  shown in silver (PfRH5) and cyan (basigin), primarily through packing between the
                  two C-terminal domains of basigin. The two C-termini of basigin are in close proximity
                  (top view).
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <span class="label">Extended Data Figure 7</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Analysis of the PfRH5:basigin complex using analytical ultracentrifugation</div>
               <p>
                  <b>a-b,</b> Sedimentation velocity analysis. The continuous sedimentation coefficient distributions
                  that best fit the data are shown for basigin (top), full-length PfRH5 (middle), and
                  a gel filtered PfRH5:basigin complex (bottom). The inset shows the fitting residuals.
                  
                  <b>c-d,</b> Sedimentation equilibrium analysis. PfRH5ΔN (residues 140-526), basigin, and a gel
                  filtered PfRH5ΔN:basigin complex were analysed. The runs lasted 20 hours at different
                  speeds, as indicated in the inset legends. UV absorbance was monitored at 280 nm.
                  The residuals are shown below fitted data. The calculated molecular weights are consistent
                  with the formation of a 1:1 complex between PfRH5ΔN and basigin.
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <span class="label">Extended Data Figure 8</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">SAXS of PfRH5 in complex with growth-inhibitory Fab fragments</div>
               <p>
                  <b>a</b>, The theoretical scattering calculated from the average of 20 
                  <i>ab initio</i> reconstructions (continuous lines, with PfRH5 in orange, PfRH5:9AD4 in green, PfRH5:QA1
                  in red, and PfRH5:QA5 in blue) plotted with the experimental scattering intensity
                  curves (black diamonds). The data are presented as the natural logarithm of the intensity.
                  The Guiner plots are displayed in the inset. 
                  <b>b</b>, The distance distribution function, P(r), with colours as in 
                  <b>a</b>. 
                  <b>c</b>, The crystal structures of PfRH5ΔNL:QA1(left) and PfRH5ΔNL:9AD4 (middle) were docked
                  into the corresponding full-length PfRH5:Fab envelopes (grey). PfRH5ΔNL is shown in
                  yellow, QA1 in red, and 9AD4 in green. PfRH5ΔNL and a Fab fragment (cyan) were docked
                  into the PfRH5:QA5 SAXS envelope to generate a model of the PfRH5:QA5 structure (right).
                  
                  <b>d,</b> A summary of SAXS parameters. The radius of gyration (R
                  <div tagxxx="sub">g</div>) was determined from the Guinier plot using AutoRg
                  <sup>
                     <span ref-type="bibr" rid="R37">
                        <sup>
                           <a href="#R37">37</a>
                        </sup>
                     </span>
                  </sup>, and the maximum particle dimension (Dmax) and the Porod volume
                  <sup>
                     <span ref-type="bibr" rid="R39">
                        <sup>
                           <a href="#R39">39</a>
                        </sup>
                     </span>
                  </sup> were calculated using GNOM
                  <sup>
                     <span ref-type="bibr" rid="R37">
                        <sup>
                           <a href="#R37">37</a>
                        </sup>
                     </span>
                  </sup>. An estimate of the molecular weight was obtained by dividing the Porod volume by
                  1.7. 
                  <i>Ab initio</i> modeling was used to generate 20 shape reconstructions from the data. The normalised
                  spatial discrepancy parameter (NSD) diagnoses the similarity of these models
                  <sup>
                     <span ref-type="bibr" rid="R42">
                        <sup>
                           <a href="#R42">42</a>
                        </sup>
                     </span>
                  </sup>. The models were averaged and the fit of the average model to the experimental data
                  is indicated by the χ value.
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
            <div id="T1">
               <span class="label">Extended Data Table 1</span>
               <caption>
                  <div tagxxx="title">Crystallographic data collection and refinement statistics</div>
               </caption>
               <table>
                  <thead>
                     <tr>
                        <th></th>
                        <th>PfRH5△NL:basigin</th>
                        <th>PfRH5△NL:9AD4</th>
                        <th>PfRH5△NL:QAl</th>
                     </tr>
                  </thead>
                  <div tagxxx="tbody">
                     <tr>
                        <td>
                           <b>Data collection</b>
                        </td>
                        <td></td>
                        <td></td>
                        <td></td>
                     </tr>
                     <tr>
                        <td>Space group</td>
                        <td>P2
                           <div tagxxx="sub">1</div>2
                           <div tagxxx="sub">1</div>2
                           <div tagxxx="sub">1</div>
                        </td>
                        <td>P2
                           <div tagxxx="sub">1</div>2
                           <div tagxxx="sub">1</div>2
                           <div tagxxx="sub">1</div>
                        </td>
                        <td>P22
                           <div tagxxx="sub">1</div>2
                           <div tagxxx="sub">1</div>
                        </td>
                     </tr>
                     <tr>
                        <td>Cell dimensions</td>
                        <td></td>
                        <td></td>
                        <td></td>
                     </tr>
                     <tr>
                        <td> 
                           <i>a, b, c</i> (Å)
                        </td>
                        <td>75.28, 109.36, 151.95</td>
                        <td>39.79, 86.66, 324.04</td>
                        <td>65.12, 137.30, 228.58</td>
                     </tr>
                     <tr>
                        <td> 
                           <i>α,β,γ</i>(°)
                        </td>
                        <td>90.00, 90.00, 90.00</td>
                        <td>90.00, 90.00, 90.00</td>
                        <td>90.00, 90.00, 90.00</td>
                     </tr>
                     <tr>
                        <td>Resolution (Å)</td>
                        <td>67.46 – 3.10(3.27-3.10)</td>
                        <td>41.86-2.30 (2.42-2.30)</td>
                        <td>47.24-3.10(3.27-3.10)</td>
                     </tr>
                     <tr>
                        <td>
                           <i>R</i>
                           <div tagxxx="sub">pim</div>
                        </td>
                        <td>0.080 (0.297)</td>
                        <td>0.047 (0.375)</td>
                        <td>0.049 (0.380)</td>
                     </tr>
                     <tr>
                        <td>
                           <i>I</i>/σ
                           <i>I</i>
                        </td>
                        <td>9.4 (2.7)</td>
                        <td>9.8 (2.0)</td>
                        <td>9.2 (2.3)</td>
                     </tr>
                     <tr>
                        <td>Completeness (%)</td>
                        <td>99.8 (99.8)</td>
                        <td>96.2 (83.3)</td>
                        <td>95.5 (96.3)</td>
                     </tr>
                     <tr>
                        <td>Redundancy</td>
                        <td>6.1 (6.2)</td>
                        <td>5.0 (4.4)</td>
                        <td>4.5 (4.6)</td>
                     </tr>
                     <tr>
                        <td>
                           <b>Refinement</b>
                        </td>
                        <td></td>
                        <td></td>
                        <td></td>
                     </tr>
                     <tr>
                        <td>Resolution (Å)</td>
                        <td>67.46-3.10</td>
                        <td>41.86-2.30</td>
                        <td>47.24-3.10</td>
                     </tr>
                     <tr>
                        <td>No. reflections</td>
                        <td>23357</td>
                        <td>49336</td>
                        <td>36259</td>
                     </tr>
                     <tr>
                        <td>
                           <i>R</i>
                           <div tagxxx="sub">work</div>/ 
                           <i>R</i>
                           <div tagxxx="sub">free</div> (%)
                        </td>
                        <td>21.71 / 24.45</td>
                        <td>22.10 / 25.83</td>
                        <td>23.6 / 28.1</td>
                     </tr>
                     <tr>
                        <td>No. atoms</td>
                        <td>7629</td>
                        <td>6005</td>
                        <td>11478</td>
                     </tr>
                     <tr>
                        <td>Wilson B-factors</td>
                        <td>48.0</td>
                        <td>41.1</td>
                        <td>90.1</td>
                     </tr>
                     <tr>
                        <td>R.m.s deviations</td>
                        <td></td>
                        <td></td>
                        <td></td>
                     </tr>
                     <tr>
                        <td> Bond lengths (Å)</td>
                        <td>0.010</td>
                        <td>0.014</td>
                        <td>0.013</td>
                     </tr>
                     <tr>
                        <td> Bond angles (°)</td>
                        <td>1.5</td>
                        <td>1.7</td>
                        <td>1.9</td>
                     </tr>
                  </div>
               </table>
            </div>
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
            <div id="T2">
               <span class="label">Extended Data Table 2</span>
               <caption>
                  <div tagxxx="title">Description of interactions between PfRH5 with basigin or monoclonal antibodies QA1
                     and 9AD4
                  </div>
               </caption>
               <table>
                  <thead>
                     <tr>
                        <th>a</th>
                        <th></th>
                        <th></th>
                        <th></th>
                        <th></th>
                        <th></th>
                        <th></th>
                     </tr>
                     <tr>
                        <th>
                           <hr></hr>
                        </th>
                     </tr>
                     <tr>
                        <th>PfRH5</th>
                        <th>Basigin</th>
                        <th></th>
                     </tr>
                     <tr>
                        <th>Chain</th>
                        <th>Residue</th>
                        <th>Group</th>
                        <th>Chain</th>
                        <th>Residue</th>
                        <th>Group</th>
                        <th>Interaction</th>
                     </tr>
                  </thead>
                  <div tagxxx="tbody">
                     <tr>
                        <td></td>
                        <td></td>
                        <td></td>
                        <td>
                           <b>
                              <i>N-terminal domain:</i>
                           </b>
                        </td>
                        <td></td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/C</td>
                        <td>S197</td>
                        <td>side chain</td>
                        <td>B/D</td>
                        <td>Q100</td>
                        <td>side chain NH2</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/C</td>
                        <td>S197</td>
                        <td>side chain</td>
                        <td>B/D</td>
                        <td>E84</td>
                        <td>side chain</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/C</td>
                        <td>F350</td>
                        <td>side chain</td>
                        <td>B/D</td>
                        <td></td>
                        <td>hydrophobic</td>
                        <td>hydrophobic</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/C</td>
                        <td>N352</td>
                        <td>side chain CO</td>
                        <td>B/D</td>
                        <td>N98</td>
                        <td>backbone NH</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/C</td>
                        <td>N354</td>
                        <td>side chain NH2</td>
                        <td>B/D</td>
                        <td>N98</td>
                        <td>backbone CO</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/C</td>
                        <td>R357</td>
                        <td>side chain</td>
                        <td>B/D</td>
                        <td>V26</td>
                        <td>backbone CO</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/C</td>
                        <td>W447</td>
                        <td>side chain</td>
                        <td>B/D</td>
                        <td></td>
                        <td>hydrophobic</td>
                        <td>hydrophobic</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/C</td>
                        <td>W447</td>
                        <td>backbone CO</td>
                        <td>B/D</td>
                        <td>T28</td>
                        <td>backbone NH</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/C</td>
                        <td>R448</td>
                        <td>side chain</td>
                        <td>B/D</td>
                        <td>T25</td>
                        <td>side chain</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/C</td>
                        <td>T449</td>
                        <td>side chain</td>
                        <td>B/D</td>
                        <td>V26</td>
                        <td>backbone NH</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/C</td>
                        <td>T449</td>
                        <td>backbone NH</td>
                        <td>B/D</td>
                        <td>V26</td>
                        <td>backbone CO</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td></td>
                        <td></td>
                        <td></td>
                        <td>
                           <b>
                              <i>Linker:</i>
                           </b>
                        </td>
                        <td></td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/C</td>
                        <td>Y200</td>
                        <td>side chain</td>
                        <td>B/D</td>
                        <td>H102</td>
                        <td>side chain</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td></td>
                        <td></td>
                        <td></td>
                        <td>
                           <b>
                              <i>C-terminal domain:</i>
                           </b>
                        </td>
                        <td></td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/C</td>
                        <td></td>
                        <td>hydrophobic</td>
                        <td>B/D</td>
                        <td>V131</td>
                        <td>side chain</td>
                        <td>hydrophobic</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A</td>
                        <td></td>
                        <td>hydrophobic</td>
                        <td>B</td>
                        <td>P133</td>
                        <td>side chain</td>
                        <td>hydrophobic</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A</td>
                        <td>D207</td>
                        <td>side chain</td>
                        <td>B</td>
                        <td>Q164</td>
                        <td>side chain NH2</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>C</td>
                        <td>E362</td>
                        <td>side chain</td>
                        <td>D</td>
                        <td>K191</td>
                        <td>side chain</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>C</td>
                        <td>E362</td>
                        <td>side chain</td>
                        <td>D</td>
                        <td>S190</td>
                        <td>side chain</td>
                        <td>hydrogen bond</td>
                     </tr>
                  </div>
               </table>
               <table>
                  <thead>
                     <tr>
                        <th>b</th>
                        <th></th>
                        <th></th>
                        <th></th>
                        <th></th>
                        <th></th>
                        <th></th>
                     </tr>
                     <tr>
                        <th>
                           <hr></hr>
                        </th>
                     </tr>
                     <tr>
                        <th>PfRH5</th>
                        <th>QA1</th>
                        <th></th>
                     </tr>
                     <tr>
                        <th>Chain</th>
                        <th>Residue</th>
                        <th>Group</th>
                        <th>Chain</th>
                        <th>Residue</th>
                        <th>Group</th>
                        <th>Interaction</th>
                     </tr>
                  </thead>
                  <div tagxxx="tbody">
                     <tr>
                        <td></td>
                        <td></td>
                        <td></td>
                        <td>
                           <b>
                              <i>Heavy chain:</i>
                           </b>
                        </td>
                        <td></td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/D</td>
                        <td>K196</td>
                        <td>side chain</td>
                        <td>B/E</td>
                        <td>N101</td>
                        <td>side chain CO</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/D</td>
                        <td>K196</td>
                        <td>side chain</td>
                        <td>B/E</td>
                        <td>D31</td>
                        <td>backbone CO</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>A/D</td>
                        <td>S197</td>
                        <td>backbone NH</td>
                        <td>B/E</td>
                        <td>D53</td>
                        <td>side chain</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/D</td>
                        <td>Y346</td>
                        <td>side chain</td>
                        <td>B/E</td>
                        <td>Y33</td>
                        <td>side chain</td>
                        <td>pi stacking</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/D</td>
                        <td>N348</td>
                        <td>side chain CO</td>
                        <td>B/E</td>
                        <td>G105</td>
                        <td>backbone NH</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/D</td>
                        <td>N352</td>
                        <td>side chain</td>
                        <td>B/E</td>
                        <td>Y33</td>
                        <td>side chain</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/D</td>
                        <td>N352</td>
                        <td>side chain</td>
                        <td>B/E</td>
                        <td>Y59</td>
                        <td>side chain</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/D</td>
                        <td>N354</td>
                        <td>backbone NH</td>
                        <td>B/E</td>
                        <td>Y57</td>
                        <td>side chain</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/D</td>
                        <td>K452</td>
                        <td>side chain</td>
                        <td>B/E</td>
                        <td>D104</td>
                        <td>side chain</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td></td>
                        <td></td>
                        <td></td>
                        <td>
                           <b>
                              <i>Light chain:</i>
                           </b>
                        </td>
                        <td></td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/D</td>
                        <td>N347</td>
                        <td>side chain NH2</td>
                        <td>C/F</td>
                        <td>S95</td>
                        <td>backbone CO</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/D</td>
                        <td>N348</td>
                        <td>backbone NH</td>
                        <td>C/F</td>
                        <td>Y36</td>
                        <td>side chain</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/D</td>
                        <td>N349</td>
                        <td>backbone NH</td>
                        <td>C/F</td>
                        <td>Y34</td>
                        <td>side chain</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/D</td>
                        <td>N349</td>
                        <td>side chain CO</td>
                        <td>C/F</td>
                        <td>W96</td>
                        <td>side chain</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/D</td>
                        <td>F350</td>
                        <td>side chain</td>
                        <td>C/F</td>
                        <td></td>
                        <td>hydrophobic</td>
                        <td>hydrophobic</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A/D</td>
                        <td>Q451</td>
                        <td>backbone CO</td>
                        <td>C/F</td>
                        <td>Y34</td>
                        <td>side chain</td>
                        <td>hydrogen bond</td>
                     </tr>
                  </div>
               </table>
               <table>
                  <thead>
                     <tr>
                        <th>c</th>
                        <th></th>
                        <th></th>
                        <th></th>
                        <th></th>
                        <th></th>
                        <th></th>
                     </tr>
                     <tr>
                        <th>
                           <hr></hr>
                        </th>
                     </tr>
                     <tr>
                        <th>PfRH5</th>
                        <th>9AD4</th>
                        <th></th>
                     </tr>
                     <tr>
                        <th>Chain</th>
                        <th>Residue</th>
                        <th>Group</th>
                        <th>Chain</th>
                        <th>Residue</th>
                        <th>Group</th>
                        <th>Interaction</th>
                     </tr>
                  </thead>
                  <div tagxxx="tbody">
                     <tr>
                        <td></td>
                        <td></td>
                        <td></td>
                        <td>
                           <b>
                              <i>Heavy chain:</i>
                           </b>
                        </td>
                        <td></td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A</td>
                        <td>A205</td>
                        <td>backbone CO</td>
                        <td>B</td>
                        <td>Y103</td>
                        <td>backbone CO</td>
                        <td>hydrogen bond (H
                           <div tagxxx="sub">2</div>0)
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A</td>
                        <td>F209</td>
                        <td>side chain</td>
                        <td>B</td>
                        <td></td>
                        <td>hydrophobic</td>
                        <td>hydrophobic</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A</td>
                        <td>Y335</td>
                        <td>side chain</td>
                        <td>B</td>
                        <td>Y103</td>
                        <td>backbone CO</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A</td>
                        <td>N338</td>
                        <td>side chain CO</td>
                        <td>B</td>
                        <td>W107</td>
                        <td>side chain</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A</td>
                        <td>L339</td>
                        <td>side chain</td>
                        <td>B</td>
                        <td></td>
                        <td>hydrophobic</td>
                        <td>hydrophobic</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A</td>
                        <td>E341</td>
                        <td>side chain</td>
                        <td>B</td>
                        <td>S52</td>
                        <td>side chain</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A</td>
                        <td>E341</td>
                        <td>side chain</td>
                        <td>B</td>
                        <td>M54</td>
                        <td>backbone CO</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A</td>
                        <td>E341</td>
                        <td>side chain</td>
                        <td>B</td>
                        <td>A55</td>
                        <td>backbone CO</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A</td>
                        <td>E341</td>
                        <td>side chain</td>
                        <td>B</td>
                        <td>Y56</td>
                        <td>backbone CO</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A</td>
                        <td>Q342</td>
                        <td>side chain CO</td>
                        <td>B</td>
                        <td>N53</td>
                        <td>side chain NH2</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A</td>
                        <td></td>
                        <td>hydrophobic</td>
                        <td>B</td>
                        <td>F101</td>
                        <td>side chain</td>
                        <td>hydrophobic</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td></td>
                        <td></td>
                        <td></td>
                        <td>
                           <b>
                              <i>Light chain:</i>
                           </b>
                        </td>
                        <td></td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A</td>
                        <td>K212</td>
                        <td>side chain</td>
                        <td>C</td>
                        <td>Y32</td>
                        <td>backbone CO</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A</td>
                        <td>D331</td>
                        <td>side chain</td>
                        <td>C</td>
                        <td>Y32</td>
                        <td>side chain</td>
                        <td>hydrogen bond</td>
                     </tr>
                     <tr>
                        <td>
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>A</td>
                        <td>N334</td>
                        <td>side chain NH2</td>
                        <td>C</td>
                        <td>Y31</td>
                        <td>side chain</td>
                        <td>hydrogen bond (H
                           <div tagxxx="sub">2</div>0)
                        </td>
                     </tr>
                  </div>
               </table>
            </div>
         </p>
      </div>
      <div tagxxx="back">
         <div tagxxx="ack">
            <div tagxxx="title">Acknowledgements</div>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">MKH is a Wellcome Trust Investigator (101020/Z/13/Z). KEW is funded by a Wellcome
               Trust Ph.D. studentship. SJD holds a UK Medical Research Council (MRC) Career Development
               Fellowship (G1000527), is a Jenner Investigator and Lister Institute Research Prize
               Fellow. The project was also funded by the European Vaccine Initiative (EVI) (InnoMalVac);
               the UK MRC (MR/K025554/1), the European Community’s Seventh Framework Programme (FP7/2007-2013,
               grant agreement number 242095 – EVIMalaR); a Wellcome Trust Training Fellowship (089455/2/09/z
               to ADD). We thank Julie Furze and Daniel Alanine (Jenner Institute); David Staunton
               and Edward Lowe (Department of Biochemistry, Oxford University); Adam Round (ESRF);
               and Ralf Flaig and Jose Brandao-Neto (Diamond Light Source).
            </p>
         </div>
         <div tagxxx="fn-group">
            <div class="fn" id="FN2">
               <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Atomic coordinates and structure factors are deposited at the Protein Data Bank with
                  accession codes 4U0Q, 4U0R and 4U1G.
               </p>
            </div>
            <div class="fn" id="FN3">
               <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Reprints and permissions information is available at 
                  <div tagxxx="ext-link">www.nature.com/reprints</div>.
               </p>
            </div>
            <div class="fn" id="FN4">
               <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The authors declare competing financial interests. ADD, JJI and SJD are named on patent
                  applications relating to PfRH5 and/or other malaria vaccines. WAJ is named on patents
                  relating to 
                  <i>Drosophila</i> S2 protein expression and is co-founder of ExpreS
                  <sup>2</sup>ion Biotechnologies.
                  
               </p>
            </div>
         </div>
         <div tag="ref-list">
            <ul>
               <div tagxxx="title">References</div>
               <li tag="ref">
                  <a name="R1"></a>
                  <span class="label">1</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Cowman</span>
                           <span tagx="given-names">AF</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Crabb</span>
                           <span tagx="given-names">BS</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Invasion of red blood cells by malaria parasites</span>
                     <span tagx="source">Cell</span>
                     <span tagx="year">2006</span>
                     <span tagx="volume">124</span>
                     <span tagx="fpage">755</span>
                     <span tagx="lpage">766</span>
                     <span class="pub-id'">[16497586]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R2"></a>
                  <span class="label">2</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Tham</span>
                           <span tagx="given-names">WH</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Healer</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Cowman</span>
                           <span tagx="given-names">AF</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Erythrocyte and reticulocyte binding-like proteins of 
                        <i>Plasmodium falciparum</i>
                     </span>
                     <span tagx="source">Trends Parasitol</span>
                     <span tagx="year">2012</span>
                     <span tagx="volume">28</span>
                     <span tagx="fpage">23</span>
                     <span tagx="lpage">30</span>
                     <span class="pub-id'">[22178537]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R3"></a>
                  <span class="label">3</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Baum</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Reticulocyte-binding protein homologue 5 - an essential adhesin involved in invasion
                        of human erythrocytes by 
                        <i>Plasmodium falciparum</i>
                     </span>
                     <span tagx="source">Int. J. Parasitol</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">39</span>
                     <span tagx="fpage">371</span>
                     <span tagx="lpage">380</span>
                     <span class="pub-id'">[19000690]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R4"></a>
                  <span class="label">4</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Crosnier</span>
                           <span tagx="given-names">C</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Basigin is a receptor essential for erythrocyte invasion by 
                        <i>Plasmodium falciparum</i>
                     </span>
                     <span tagx="source">Nature</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">480</span>
                     <span tagx="fpage">534</span>
                     <span tagx="lpage">537</span>
                     <span class="pub-id'">[22080952]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R5"></a>
                  <span class="label">5</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Douglas</span>
                           <span tagx="given-names">AD</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Neutralization of 
                        <i>Plasmodium falciparum</i> merozoites by antibodies against PfRH5
                     </span>
                     <span tagx="source">J. Immunol</span>
                     <span tagx="year">2014</span>
                     <span tagx="volume">192</span>
                     <span tagx="fpage">245</span>
                     <span tagx="lpage">258</span>
                     <span class="pub-id'">[24293631]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R6"></a>
                  <span class="label">6</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Douglas</span>
                           <span tagx="given-names">AD</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">The blood-stage malaria antigen PfRH5 is susceptible to vaccine-inducible cross-strain
                        neutralizing antibody
                     </span>
                     <span tagx="source">Nat. Commun</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">2</span>
                     <span tagx="fpage">601</span>
                     <span class="pub-id'">[22186897]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R7"></a>
                  <span class="label">7</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Williams</span>
                           <span tagx="given-names">AR</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Enhancing blockade of 
                        <i>Plasmodium falciparum</i> erythrocyte invasion: assessing combinations of antibodies against PfRH5 and other
                        merozoite antigens
                     </span>
                     <span tagx="source">PLoS Pathog</span>
                     <span tagx="year">2012</span>
                     <span tagx="volume">8</span>
                     <span tagx="fpage">e1002991</span>
                     <span class="pub-id'">[23144611]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R8"></a>
                  <span class="label">8</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Bustamante</span>
                           <span tagx="given-names">LY</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">A full-length recombinant 
                        <i>Plasmodium falciparum</i> PfRH5 protein induces inhibitory antibodies that are effective across common PfRH5
                        genetic variants
                     </span>
                     <span tagx="source">Vaccine</span>
                     <span tagx="year">2013</span>
                     <span tagx="volume">31</span>
                     <span tagx="fpage">373</span>
                     <span tagx="lpage">379</span>
                     <span class="pub-id'">[23146673]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R9"></a>
                  <span class="label">9</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Reddy</span>
                           <span tagx="given-names">KS</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Bacterially expressed full-length recombinant 
                        <i>Plasmodium falciparum</i> RH5 protein binds erythrocytes and elicits potent strain-transcending parasite-neutralizing
                        antibodies
                     </span>
                     <span tagx="source">Infect. Immun</span>
                     <span tagx="year">2014</span>
                     <span tagx="volume">82</span>
                     <span tagx="fpage">152</span>
                     <span tagx="lpage">164</span>
                     <span class="pub-id'">[24126527]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R10"></a>
                  <span class="label">10</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Chen</span>
                           <span tagx="given-names">L</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">An EGF-like protein forms a complex with PfRh5 and is required for invasion of human
                        erythrocytes by 
                        <i>Plasmodium falciparum</i>
                     </span>
                     <span tagx="source">PLoS Pathog</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">7</span>
                     <span tagx="fpage">e1002199</span>
                     <span class="pub-id'">[21909261]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R11"></a>
                  <span class="label">11</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Rodriguez</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Lustigman</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Montero</span>
                           <span tagx="given-names">E</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Oksov</span>
                           <span tagx="given-names">Y</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Lobo</span>
                           <span tagx="given-names">CA</span>,
                        </span>
                     </span>
                     <span tagx="article-title">PfRH5: a novel reticulocyte-binding family homolog of 
                        <i>Plasmodium falciparum</i> that binds to the erythrocyte, and an investigation of its receptor
                     </span>
                     <span tagx="source">PLoS ONE</span>
                     <span tagx="year">2008</span>
                     <span tagx="volume">3</span>
                     <span tagx="fpage">e3300</span>
                     <span class="pub-id'">[18827878]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R12"></a>
                  <span class="label">12</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Manske</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Analysis of 
                        <i>Plasmodium falciparum</i> diversity in natural infections by deep sequencing
                     </span>
                     <span tagx="source">Nature</span>
                     <span tagx="year">2012</span>
                     <span tagx="volume">487</span>
                     <span tagx="fpage">375</span>
                     <span tagx="lpage">379</span>
                     <span class="pub-id'">[22722859]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R13"></a>
                  <span class="label">13</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Tran</span>
                           <span tagx="given-names">TM</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Naturally acquired antibodies specific for 
                        <i>Plasmodium falciparum</i> reticulocyte-binding protein homologue 5 inhibit parasite growth and predict protection
                        from malaria
                     </span>
                     <span tagx="source">J. Infect. Dis</span>
                     <span tagx="year">2014</span>
                     <span tagx="volume">209</span>
                     <span tagx="fpage">789</span>
                     <span tagx="lpage">798</span>
                     <span class="pub-id'">[24133188]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R14"></a>
                  <span class="label">14</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Tham</span>
                           <span tagx="given-names">WH</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Complement receptor 1 is the host erythrocyte receptor for 
                        <i>Plasmodium falciparum</i> PfRh4 invasion ligand
                     </span>
                     <span tagx="source">Proc. Natl Acad. Sci. USA</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">107</span>
                     <span tagx="fpage">17327</span>
                     <span tagx="lpage">17332</span>
                     <span class="pub-id'">[20855594]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R15"></a>
                  <span class="label">15</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Hayton</span>
                           <span tagx="given-names">K</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Erythrocyte binding protein PfRH5 polymorphisms determine species-specific pathways
                        of 
                        <i>Plasmodium falciparum</i> invasion
                     </span>
                     <span tagx="source">Cell Host Microbe</span>
                     <span tagx="year">2008</span>
                     <span tagx="volume">4</span>
                     <span tagx="fpage">40</span>
                     <span tagx="lpage">51</span>
                     <span class="pub-id'">[18621009]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R16"></a>
                  <span class="label">16</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Hayton</span>
                           <span tagx="given-names">K</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Various PfRH5 polymorphisms can support 
                        <i>Plasmodium falciparum</i> invasion into the erythrocytes of owl monkeys and rats
                     </span>
                     <span tagx="source">Mol. Biochem. Parasitol</span>
                     <span tagx="year">2013</span>
                     <span tagx="volume">187</span>
                     <span tagx="fpage">103</span>
                     <span tagx="lpage">110</span>
                     <span class="pub-id'">[23305874]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R17"></a>
                  <span class="label">17</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Wanaguru</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Liu</span>
                           <span tagx="given-names">W</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Hahn</span>
                           <span tagx="given-names">BH</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Rayner</span>
                           <span tagx="given-names">JC</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Wright</span>
                           <span tagx="given-names">GJ</span>,
                        </span>
                     </span>
                     <span tagx="article-title">RH5-Basigin interaction plays a major role in the host tropism of 
                        <i>Plasmodium falciparum</i>
                     </span>
                     <span tagx="source">Proc. Natl Acad. Sci. USA</span>
                     <span tagx="year">2013</span>
                     <span tagx="volume">110</span>
                     <span tagx="fpage">20735</span>
                     <span tagx="lpage">20740</span>
                     <span class="pub-id'">[24297912]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R18"></a>
                  <span class="label">18</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Higgins</span>
                           <span tagx="given-names">MK</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Carrington</span>
                           <span tagx="given-names">M</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Sequence variation and structural conservation allows development of novel function
                        and immune evasion in parasite surface protein families
                     </span>
                     <span tagx="source">Protein Sci</span>
                     <span tagx="year">2014</span>
                     <span tagx="volume">23</span>
                     <span tagx="fpage">354</span>
                     <span tagx="lpage">365</span>
                     <span class="pub-id'">[24442723]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R19"></a>
                  <span class="label">19</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Hirose</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Shimizu</span>
                           <span tagx="given-names">K</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Kanai</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Kuroda</span>
                           <span tagx="given-names">Y</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Noguchi</span>
                           <span tagx="given-names">T</span>,
                        </span>
                     </span>
                     <span tagx="article-title">POODLE-L: a two-level SVM prediction system for reliably predicting long disordered
                        regions
                     </span>
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="year">2007</span>
                     <span tagx="volume">23</span>
                     <span tagx="fpage">2046</span>
                     <span tagx="lpage">2053</span>
                     <span class="pub-id'">[17545177]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R20"></a>
                  <span class="label">20</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Miura</span>
                           <span tagx="given-names">K</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Anti-apical-membrane-antigen-1 antibody is more effective than anti-42-kilodalton-merozoite-surface-protein-1
                        antibody in inhibiting plasmodium falciparum growth, as determined by the in vitro
                        growth inhibition assay
                     </span>
                     <span tagx="source">Clin. Vaccine Immunol</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">16</span>
                     <span tagx="fpage">963</span>
                     <span tagx="lpage">968</span>
                     <span class="pub-id'">[19439523]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R21"></a>
                  <span class="label">21</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Sheehy</span>
                           <span tagx="given-names">SH</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Phase Ia clinical evaluation of the 
                        <i>Plasmodium falciparum</i> blood-stage antigen MSP1 in ChAd63 and MVA vaccine vectors
                     </span>
                     <span tagx="source">Mol. Ther</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">19</span>
                     <span tagx="fpage">2269</span>
                     <span tagx="lpage">2276</span>
                     <span class="pub-id'">[21862998]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R22"></a>
                  <span class="label">22</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Battye</span>
                           <span tagx="given-names">TG</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Kontogiannis</span>
                           <span tagx="given-names">L</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Johnson</span>
                           <span tagx="given-names">O</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Powell</span>
                           <span tagx="given-names">HR</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Leslie</span>
                           <span tagx="given-names">AG</span>,
                        </span>
                     </span>
                     <span tagx="article-title">iMOSFLM: a new graphical interface for diffraction-image processing with MOSFLM</span>
                     <span tagx="source">Acta Crystallogr. D Biol. Crystallogr</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">67</span>
                     <span tagx="fpage">271</span>
                     <span tagx="lpage">281</span>
                     <span class="pub-id'">[21460445]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R23"></a>
                  <span class="label">23</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Leslie</span>
                           <span tagx="given-names">AG</span>,
                        </span>
                     </span>
                     <span tagx="article-title">The integration of macromolecular diffraction data</span>
                     <span tagx="source">Acta Crystallogr. D Biol. Crystallogr</span>
                     <span tagx="year">2006</span>
                     <span tagx="volume">62</span>
                     <span tagx="fpage">48</span>
                     <span tagx="lpage">57</span>
                     <span class="pub-id'">[16369093]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R24"></a>
                  <span class="label">24</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Evans</span>
                           <span tagx="given-names">PR</span>,
                        </span>
                     </span>
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Sawyer</span>
                           <span tagx="given-names">L</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Isaacs</span>
                           <span tagx="given-names">N</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Bailey</span>
                           <span tagx="given-names">S</span>,
                        </span>
                     </span>
                     <span tagx="source">Data reduction</span>
                     <span tagx="fpage">114</span>
                     <span tagx="lpage">122</span>
                     <div tagxxx="conf-name">Proceedings of the CCP4 Study Weekend</div>
                     <div tagxxx="conf-loc">Daresbury Laboratory, Warrington, UK</div>
                     <div tagxxx="conf-date">1993</div>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R25"></a>
                  <span class="label">25</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Winn</span>
                           <span tagx="given-names">MD</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Overview of the CCP4 suite and current developments</span>
                     <span tagx="source">Acta Crystallogr. D Biol. Crystallogr</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">67</span>
                     <span tagx="fpage">235</span>
                     <span tagx="lpage">242</span>
                     <span class="pub-id'">[21460441]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R26"></a>
                  <span class="label">26</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">McCoy</span>
                           <span tagx="given-names">AJ</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Phaser crystallographic software</span>
                     <span tagx="source">J. Appl. Crystallogr</span>
                     <span tagx="year">2007</span>
                     <span tagx="volume">40</span>
                     <span tagx="fpage">658</span>
                     <span tagx="lpage">674</span>
                     <span class="pub-id'">[19461840]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R27"></a>
                  <span class="label">27</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Zhang</span>
                           <span tagx="given-names">KY</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Cowtan</span>
                           <span tagx="given-names">K</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Main</span>
                           <span tagx="given-names">P</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Combining constraints for electron-density modification</span>
                     <span tagx="source">Methods Enzymol</span>
                     <span tagx="year">1997</span>
                     <span tagx="volume">277</span>
                     <span tagx="fpage">53</span>
                     <span tagx="lpage">64</span>
                     <span class="pub-id'">[18488305]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R28"></a>
                  <span class="label">28</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Cowtan</span>
                           <span tagx="given-names">K</span>,
                        </span>
                     </span>
                     <span tagx="article-title">The Buccaneer software for automated model building</span>
                     <span tagx="source">Acta Crystallogr. D Biol. Crystallogr</span>
                     <span tagx="year">2006</span>
                     <span tagx="volume">62</span>
                     <span tagx="fpage">1002</span>
                     <span tagx="lpage">1011</span>
                     <span class="pub-id'">[16929101]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R29"></a>
                  <span class="label">29</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Murshudov</span>
                           <span tagx="given-names">GN</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">REFMAC5 for the refinement of macromolecular crystal structures</span>
                     <span tagx="source">Acta Crystallogr. D Biol. Crystallogr</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">67</span>
                     <span tagx="fpage">355</span>
                     <span tagx="lpage">367</span>
                     <span class="pub-id'">[21460454]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R30"></a>
                  <span class="label">30</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Bricogne</span>
                           <span tagx="given-names">G</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="source">BUSTER version 2.10.0</span>
                     <span tagx="year">2011</span>
                     <span class="publisher-name'">[Global Phasing Ltd.]</span>
                     <span class="publisher-loc'">[Cambridge, UK]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R31"></a>
                  <span class="label">31</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Emsley</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Lohkamp</span>
                           <span tagx="given-names">B</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Scott</span>
                           <span tagx="given-names">WG</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Cowtan</span>
                           <span tagx="given-names">K</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Features and development of Coot</span>
                     <span tagx="source">Acta Crystallogr. D Biol. Crystallogr</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">66</span>
                     <span tagx="fpage">486</span>
                     <span tagx="lpage">501</span>
                     <span class="pub-id'">[20383002]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R32"></a>
                  <span class="label">32</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Laskowski</span>
                           <span tagx="given-names">RA</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">MacArthur</span>
                           <span tagx="given-names">MW</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Moss</span>
                           <span tagx="given-names">DS</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Thornton</span>
                           <span tagx="given-names">JM</span>,
                        </span>
                     </span>
                     <span tagx="article-title">PROCHECK - a program to check the stereochemical quality of protein structures</span>
                     <span tagx="source">J. App. Cryst</span>
                     <span tagx="year">1993</span>
                     <span tagx="volume">26</span>
                     <span tagx="fpage">283</span>
                     <span tagx="lpage">291</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R33"></a>
                  <span class="label">33</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Sievers</span>
                           <span tagx="given-names">F</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using
                        Clustal Omega
                     </span>
                     <span tagx="source">Mol. Syst. Biol</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">7</span>
                     <span tagx="fpage">539</span>
                     <span class="pub-id'">[21988835]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R34"></a>
                  <span class="label">34</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Gouet</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Courcelle</span>
                           <span tagx="given-names">E</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Stuart</span>
                           <span tagx="given-names">DI</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Metoz</span>
                           <span tagx="given-names">F</span>,
                        </span>
                     </span>
                     <span tagx="article-title">ESPript: analysis of multiple sequence alignments in PostScript</span>
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="year">1999</span>
                     <span tagx="volume">15</span>
                     <span tagx="fpage">305</span>
                     <span tagx="lpage">308</span>
                     <span class="pub-id'">[10320398]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R35"></a>
                  <span class="label">35</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Kelley</span>
                           <span tagx="given-names">LA</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Sternberg</span>
                           <span tagx="given-names">MJE</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Protein structure prediction on the Web: a case study using the Phyre server</span>
                     <span tagx="source">Nat. Protoc</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">4</span>
                     <span tagx="fpage">363</span>
                     <span tagx="lpage">371</span>
                     <span class="pub-id'">[19247286]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R36"></a>
                  <span class="label">36</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Konarev</span>
                           <span tagx="given-names">PV</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Volkov</span>
                           <span tagx="given-names">VV</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Sokolova</span>
                           <span tagx="given-names">AV</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Koch</span>
                           <span tagx="given-names">MHJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Svergun</span>
                           <span tagx="given-names">DI</span>,
                        </span>
                     </span>
                     <span tagx="article-title">PRIMUS: a Windows PC-based system for small-angle scattering data analysis</span>
                     <span tagx="source">J. Appl. Crystallogr</span>
                     <span tagx="year">2003</span>
                     <span tagx="volume">36</span>
                     <span tagx="fpage">1277</span>
                     <span tagx="lpage">1282</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R37"></a>
                  <span class="label">37</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Petoukhov</span>
                           <span tagx="given-names">MV</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Konarev</span>
                           <span tagx="given-names">PV</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Kikhney</span>
                           <span tagx="given-names">AG</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Svergun</span>
                           <span tagx="given-names">DI</span>,
                        </span>
                     </span>
                     <span tagx="article-title">ATSAS 2.1 - towards automated and web-supported small-angle scattering data analysis</span>
                     <span tagx="source">J. Appl. Crystallogr</span>
                     <span tagx="year">2007</span>
                     <span tagx="volume">40</span>
                     <span tagx="fpage">S223</span>
                     <span tagx="lpage">S228</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R38"></a>
                  <span class="label">38</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Guinier</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Fournet</span>
                           <span tagx="given-names">G</span>,
                        </span>
                     </span>
                     <span tagx="source">Small-angle Scattering of X-rays</span>
                     <span tagx="year">1955</span>
                     <span class="publisher-name'">[John Wiley and Sons]</span>
                     <span class="publisher-loc'">[New York, USA]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R39"></a>
                  <span class="label">39</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Porod</span>
                           <span tagx="given-names">G</span>,
                        </span>
                     </span>
                     <span tagx="source">General Theory Small Angle X-ray Scattering</span>
                     <span tagx="year">1982</span>
                     <span class="publisher-name'">[Academic Press, Inc.]</span>
                     <span class="publisher-loc'">[London, UK]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R40"></a>
                  <span class="label">40</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Franke</span>
                           <span tagx="given-names">D</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Svergun</span>
                           <span tagx="given-names">DI</span>,
                        </span>
                     </span>
                     <span tagx="article-title">DAMMIF, a program for rapid ab-initio shape determination in small-angle scattering</span>
                     <span tagx="source">J. Appl. Crystallogr</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">42</span>
                     <span tagx="fpage">342</span>
                     <span tagx="lpage">346</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R41"></a>
                  <span class="label">41</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Konarev</span>
                           <span tagx="given-names">PV</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Petoukhov</span>
                           <span tagx="given-names">MV</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Volkov</span>
                           <span tagx="given-names">VV</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Svergun</span>
                           <span tagx="given-names">DI</span>,
                        </span>
                     </span>
                     <span tagx="article-title">ATSAS 2.1, a program package for small-angle scattering data analysis</span>
                     <span tagx="source">J. Appl. Crystallogr</span>
                     <span tagx="year">2006</span>
                     <span tagx="volume">39</span>
                     <span tagx="fpage">277</span>
                     <span tagx="lpage">286</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R42"></a>
                  <span class="label">42</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Kozin</span>
                           <span tagx="given-names">MB</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Svergun</span>
                           <span tagx="given-names">DI</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Automated matching of high- and low-resolution structural models</span>
                     <span tagx="source">J. Appl. Crystallogr</span>
                     <span tagx="year">2001</span>
                     <span tagx="volume">34</span>
                     <span tagx="fpage">33</span>
                     <span tagx="lpage">41</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R43"></a>
                  <span class="label">43</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Birmanns</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Rusu</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Wriggers</span>
                           <span tagx="given-names">W</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Using Sculptor and Situs for simultaneous assembly of atomic components into low-resolution
                        shapes
                     </span>
                     <span tagx="source">J. Struct. Biol</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">173</span>
                     <span tagx="fpage">428</span>
                     <span tagx="lpage">435</span>
                     <span class="pub-id'">[21078392]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R44"></a>
                  <span class="label">44</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Schuck</span>
                           <span tagx="given-names">P</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Size-distribution analysis of macromolecules by sedimentation velocity ultracentrifugation
                        and Lamm equation modeling
                     </span>
                     <span tagx="source">Biophys. J</span>
                     <span tagx="year">2000</span>
                     <span tagx="volume">78</span>
                     <span tagx="fpage">1606</span>
                     <span tagx="lpage">1619</span>
                     <span class="pub-id'">[10692345]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R45"></a>
                  <span class="label">45</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Schuck</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Perugini</span>
                           <span tagx="given-names">MA</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gonzales</span>
                           <span tagx="given-names">NR</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Howlett</span>
                           <span tagx="given-names">GJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Schubert</span>
                           <span tagx="given-names">D</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Size-distribution analysis of proteins by analytical ultracentrifugation: Strategies
                        and application to model systems
                     </span>
                     <span tagx="source">Biophys. J</span>
                     <span tagx="year">2002</span>
                     <span tagx="volume">82</span>
                     <span tagx="fpage">1096</span>
                     <span tagx="lpage">1111</span>
                     <span class="pub-id'">[11806949]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R46"></a>
                  <span class="label">46</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Laue</span>
                           <span tagx="given-names">TM</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Shah</span>
                           <span tagx="given-names">BD</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Ridgeway</span>
                           <span tagx="given-names">TM</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Pelletier</span>
                           <span tagx="given-names">SL</span>,
                        </span>
                     </span>
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Harding</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Rowe</span>
                           <span tagx="given-names">A</span>,
                        </span>
                     </span>
                     <span tagx="source">Analytical Ultracentrifugation in Biochemistry and Polymer Science</span>
                     <span tagx="year">1992</span>
                     <span tagx="fpage">90</span>
                     <span tagx="lpage">125</span>
                     <span class="publisher-name'">[Royal Society of Chemistry]</span>
                     <span class="publisher-loc'">[Cambridge, UK]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="R47"></a>
                  <span class="label">47</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Vistica</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Sedimentation equilibrium analysis of protein interactions with global implicit mass
                        conservation constraints and systematic noise decomposition
                     </span>
                     <span tagx="source">Anal. Biochem</span>
                     <span tagx="year">2004</span>
                     <span tagx="volume">326</span>
                     <span tagx="fpage">234</span>
                     <span tagx="lpage">256</span>
                     <span class="pub-id'">[15003564]</span>
                  </span>
               </li>
            </ul>
         </div>
      </div>
      <div tagxxx="floats-group">
         <div tagxxx="fig">
            <span class="label">Figure 1</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">The structure of PfRH5</div>
               <p>
                  <b>a</b>, Three views of PfRH5ΔNL (from the PfRH5ΔNL:9AD4 structure), and a schematic topology
                  diagram, coloured in rainbow from blue (N-terminus) to red (C-terminus). Disulphide
                  bonds are indicated on the topology diagram by red lines. 
                  <b>b</b>, PfRH5ΔNL structure docked into a SAXS envelope of full-length PfRH5. 
                  <b>c</b>, SPR analysis of the PfRH5ΔNL:basigin interaction. 
                  <b>d</b>, 
                  <i>In vitro</i> growth inhibition activity (GIA) of IgG from rabbits immunised with PfRH5ΔNL against
                  3D7 (red) and 7G8 (blue) 
                  <i>P. falciparum</i> strains. The error bars are standard error of mean (n=3).
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <span class="label">Figure 2</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">The structure of the PfRH5:basigin complex</div>
               <p>
                  <b>a</b>, The structure of PfRH5ΔNL (yellow) bound to basigin (blue). 
                  <b>b</b>, A top view of the PfRH5ΔNL:basigin complex showing the two conformations of basigin
                  (blue and cyan) found in the asymmetric unit, aligned on PfRH5ΔNL. 
                  <b>c</b>, Equilibrium analytical ultracentrifugation analysis of PfRH5:basigin indicating
                  a 1:1 complex. 
                  <b>d</b>, Close-up of the PfRH5:basigin binding site. Basigin residues in the N-terminal domain
                  (pink), the linker (His102, orange stick), and the C-terminal domain (green) contact
                  PfRH5 (grey surface). In the alternative basigin conformation in the asymmetric unit,
                  the cyan loop contacts PfRH5.
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <span class="label">Figure 3</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Structural analysis of binding of invasion-inhibitory antibody fragments to PfRH5</div>
               <p>
                  <b>a</b>, Crystal structures of PfRH5ΔNL (yellow) bound to inhibitory antibody fragments QA1
                  (red) and 9AD4 (green). Close-up views of the PfRH5 epitopes (red) are shown with
                  antibodies as grey surfaces. 
                  <b>b</b>, Top view of PfRH5ΔNL:9AD4 crystal structure with superimposed basigin (blue) aligned
                  on PfRH5. 
                  <b>c,</b> Top view of a model of PfRH5:QA5, in a SAXS-derived envelope, with the putative QA5
                  epitope highlighted red
                  <sup>
                     <span ref-type="bibr" rid="R5">
                        <sup>
                           <a href="#R5">5</a>
                        </sup>
                     </span>
                  </sup>. 
                  <b>d</b>, Schematic showing binding sites for the N- and C-terminal domains of basigin (BSG-N
                  and BSG-C; blue), QA1 (red), 9AD4 (green) and QA5 (cyan), on the structure of PfRH5ΔNL.
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
      </div>
   </div>
</html>