<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <div xmlns="" tagx="article">
      <head>
         <style type="text/css"> table, tr, td {border : thin;}</style>
         <span class="citation_publisher">Public Library of Science</span>
         <h2 class="citation-title">New Insights into Flavivirus Evolution, Taxonomy and Biogeographic History, Extended
            by Analysis of Canonical and Alternative Coding Sequences
         </h2>
         <span class="doi">10.1371/journal.pone.0117849</span>
         <div class="contrib-group">contrib: 
            	  CONT
            	  
            <span class="citation_author">Gregory Moureau</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Shelley Cook</span>
            <span class="citation_author_institution_ref">2</span>
            <span class="citation_author">Philippe Lemey</span>
            <span class="citation_author_institution_ref">3</span>
            <span class="citation_author">Antoine Nougairede</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Naomi L. Forrester</span>
            <span class="citation_author_institution_ref">4</span>
            <span class="citation_author">Maxim Khasnatinov</span>
            <span class="citation_author_institution_ref">5</span>
            <span class="citation_author">Remi N. Charrel</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Andrew E. Firth</span>
            <span class="citation_author_institution_ref">6</span>
            <span class="citation_author">Ernest A. Gould</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Xavier de Lamballerie</span>
            <span class="citation_author_institution_ref">1</span>
            	  CONT
            	  
            <div tagxxx="contrib">
               <span class="name'">
                  <span tagx="surname">Lee</span>
                  <span tagx="given-names">Young-Min</span>,
               </span>
               <div tagxxx="role">Academic Editor</div>
               <span ref-type="aff" rid="edit1"></span>
            </div>
         </div>
         <span class="email">[gregory.moureau@univ-amu.fr]</span>
         <span class="conflict">[Competing Interests: Naomi Forrester is a PLOS ONE Editorial Board Member. This does
            not alter the authors’ adherence to all the PLOS ONE policies on sharing data and
            materials.]
         </span>
         <span class="contribs">[Conceived and designed the experiments: GM AN SC PL AEF. Performed the experiments:
            GM NLF MK SC PL AEF. Analyzed the data: GM AN SC PL AEF EAG XDL. Contributed reagents/materials/analysis
            tools: GM RNC XDL. Wrote the paper: GM SC NLF MK AEF PL EAG XDL.]
         </span>
         <span class="citation_pubdate">[2015-2-26]</span>
         <span class="citation_copyright">[Copyright 2015: Moureau et al]</span>
         <span class="citation_licence">[Licence This is an open access article distributed under the terms of the Creative
            Commons Attribution License, which permits unrestricted use, distribution, and reproduction
            in any medium, provided the original author and source are credited]
         </span>
         <span class="citation_funding">[AEF is supported by Wellcome Trust grant 088789. This work was supported by Aix-Marseille
            University, IRD, EHESP, the project European Virus Archive (under EU grant agreement
            n° 228292), the European Union Seventh Framework Programme - projects SILVER and PREDEMICS,
            under grant agreements 260644 and 278433 respectively. This study was also partially
            funded by EU grant FP7-261504 EDENext and is catalogued by the EDENext Steering Committee
            as EDENext299 (http://www.edenext.eu). The funders had no role in study design, data
            collection and analysis, decision to publish, or preparation of the manuscript.]
         </span>
         <span class="citation_pagecount">[]</span>
         <span class="citation_data">[Data AvailabilityAll 10 complete genome sequences are available from the GenBank database
            (accession numbers KF917535 to KF917541, KJ469370 to KJ469372). 4 further complete
            genome sequences are already available from the GenBank database (accession numbers
            EU082199, EU082200, JX477685, JX477686).]
         </span>
         <div tagxxx="notes">
            <div tagxxx="title">Data Availability</div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">All 10 complete genome sequences are available from the GenBank database (accession
               numbers KF917535 to KF917541, KJ469370 to KJ469372). 4 further complete genome sequences
               are already available from the GenBank database (accession numbers EU082199, EU082200,
               JX477685, JX477686).
            </p>
         </div>
      </head>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To generate the most diverse phylogenetic dataset for the flaviviruses to date, we
            determined the genomic sequences and phylogenetic relationships of 14 flaviviruses,
            of which 10 are primarily associated with 
            <i>Culex</i> spp. mosquitoes. We analyze these data, in conjunction with a comprehensive collection
            of flavivirus genomes, to characterize flavivirus evolutionary and biogeographic history
            in unprecedented detail and breadth. Based on the presumed introduction of yellow
            fever virus into the Americas via the transatlantic slave trade, we extrapolated a
            timescale for a relevant subset of flaviviruses whose evolutionary history, shows
            that different 
            <i>Culex</i>-spp. associated flaviviruses have been introduced from the Old World to the New World
            on at least five separate occasions, with 2 different sets of factors likely to have
            contributed to the dispersal of the different viruses. We also discuss the significance
            of programmed ribosomal frameshifting in a central region of the polyprotein open
            reading frame in some mosquito-associated flaviviruses.
         </p>
      </div>
      <div id="sec001">
         <h2>INTRODUCTION</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The flaviviruses constitute a fascinating group of diverse arboviruses that exhibit
            uniquely clear correlations between phylogenetic relationships and virus-vector-host
            interactions [
            <span ref-type="bibr" rid="pone.0117849.ref001">
               <sup>
                  <a href="#pone.0117849.ref001">1</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pone.0117849.ref004">
               <sup>
                  <a href="#pone.0117849.ref004">4</a>
               </sup>
            </span>]. The genus 
            <i>Flavivirus</i> includes an unusually large number of taxonomically recognised species (more than
            50 at the present time, of which more than 40 are human pathogens) with a global distribution.
            The genus also includes a large and increasing number of unclassified or “tentative”
            species. Pathogenic mosquito- and/or tick-borne flaviviruses cause a variety of clinical
            diseases in a wide range of vertebrate species. These disease syndromes include mild/severe
            febrile illness, “flu-like” syndromes with a rash, or in other cases severe encephalitis
            or haemorrhagic disease. Dengue with / without warning signs and severe dengue, is
            the most devastating arboviral disease in tropical and, increasingly, sub-tropical
            areas of the world (300 to 400 million cases each year) [
            <span ref-type="bibr" rid="pone.0117849.ref005">
               <sup>
                  <a href="#pone.0117849.ref005">5</a>
               </sup>
            </span>]. Yellow fever virus and Japanese encephalitis virus also considerably contribute
            to the human flavivirus disease burden. Other flavivirus diseases, including West
            Nile encephalitis, Usutu encephalitis, Zika fever, Bagaza encephalitis and duck egg
            drop syndrome, are recognised as “emerging diseases”.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Flavivirus ecological networks are varied, complex, and poorly understood. Importantly,
            most of the natural pathogens are transmitted by arthropods (
            <i>i</i>.
            <i>e</i>. they are “arboviruses”). However, flaviviruses with no known vector (NKV), or that
            infect only insects (
            <i>i</i>.
            <i>e</i>., insect-specific flaviviruses—ISFVs), have also been identified. This remarkable
            diversity is associated with broad genetic variability, complex mechanisms of pathogenesis
            and intriguing virus/vector/host associations.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The taxonomy of the flaviviruses is constantly being updated to reflect newly-identified
            viruses and advances in analytical methods. Interestingly, the first mammalian viruses
            to be identified included 4 arboviruses, three of which were flaviviruses: louping
            ill virus, yellow fever virus and dengue virus [
            <span ref-type="bibr" rid="pone.0117849.ref006">
               <sup>
                  <a href="#pone.0117849.ref006">6</a>
               </sup>
            </span>]. This has had a deep and long-lasting influence on the development of virological
            research and more specifically, on taxonomy [
            <span ref-type="bibr" rid="pone.0117849.ref007">
               <sup>
                  <a href="#pone.0117849.ref007">7</a>
               </sup>
            </span>] and phylogeography.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">With the publication of many new complete genomic flavivirus sequences [
            <span ref-type="bibr" rid="pone.0117849.ref003">
               <sup>
                  <a href="#pone.0117849.ref003">3</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pone.0117849.ref008">
               <sup>
                  <a href="#pone.0117849.ref008">8</a>
               </sup>
            </span>] and the data generated in the current study, it is now timely and appropriate to
            re-examine the phylogenetic relationships in the context of flavivirus vector-host
            relationships, evolution and biogeographical characteristics.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The concept that the phylogenetic relationships of the tick-borne flaviviruses (TBFV)
            may correlate with their epidemiology, disease association and biogeography was first
            proposed in 1996 with the publication of the clinal evolution concept of the tick-borne
            encephalitic flaviviruses [
            <span ref-type="bibr" rid="pone.0117849.ref009">
               <sup>
                  <a href="#pone.0117849.ref009">9</a>
               </sup>
            </span>]. These relationships were then corroborated and extended by the inclusion of the
            mosquito-borne and non-vectored flaviviruses [
            <span ref-type="bibr" rid="pone.0117849.ref001">
               <sup>
                  <a href="#pone.0117849.ref001">1</a>
               </sup>
            </span>] but analyses were still based on a limited number of recognised flaviviruses and
            only partial gene sequence data (44 species based on the envelope gene). Subsequent
            analyses [
            <span ref-type="bibr" rid="pone.0117849.ref003">
               <sup>
                  <a href="#pone.0117849.ref003">3</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pone.0117849.ref008">
               <sup>
                  <a href="#pone.0117849.ref008">8</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pone.0117849.ref010">
               <sup>
                  <a href="#pone.0117849.ref010">10</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pone.0117849.ref011">
               <sup>
                  <a href="#pone.0117849.ref011">11</a>
               </sup>
            </span>], using more extensive datasets improved our understanding of these virus-vector-host
            relationships in the context of their evolution and dispersal [
            <span ref-type="bibr" rid="pone.0117849.ref002">
               <sup>
                  <a href="#pone.0117849.ref002">2</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pone.0117849.ref012">
               <sup>
                  <a href="#pone.0117849.ref012">12</a>
               </sup>
            </span>]. Additional flaviviruses including Lammi virus [
            <span ref-type="bibr" rid="pone.0117849.ref013">
               <sup>
                  <a href="#pone.0117849.ref013">13</a>
               </sup>
            </span>], N’goye virus [
            <span ref-type="bibr" rid="pone.0117849.ref014">
               <sup>
                  <a href="#pone.0117849.ref014">14</a>
               </sup>
            </span>], Alkhumra haemorrhagic fever virus [
            <span ref-type="bibr" rid="pone.0117849.ref015">
               <sup>
                  <a href="#pone.0117849.ref015">15</a>
               </sup>
            </span>], Usutu virus [
            <span ref-type="bibr" rid="pone.0117849.ref016">
               <sup>
                  <a href="#pone.0117849.ref016">16</a>
               </sup>
            </span>], New Mapoon virus [
            <span ref-type="bibr" rid="pone.0117849.ref017">
               <sup>
                  <a href="#pone.0117849.ref017">17</a>
               </sup>
            </span>], and Marisma mosquito virus [
            <span ref-type="bibr" rid="pone.0117849.ref018">
               <sup>
                  <a href="#pone.0117849.ref018">18</a>
               </sup>
            </span>] have subsequently been discovered. Together with the increasing numbers of documented
            species and strains of ISFV, that do not appear to be arboviruses [
            <span ref-type="bibr" rid="pone.0117849.ref019">
               <sup>
                  <a href="#pone.0117849.ref019">19</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pone.0117849.ref023">
               <sup>
                  <a href="#pone.0117849.ref023">23</a>
               </sup>
            </span>], these discoveries have shed new light on our perception of the evolution and taxonomy
            of this complex genus.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Based on the flavivirus arthropod vectors and vertebrate hosts, current phylogenies
            recognise three major groups in addition to the ISFVs [
            <span ref-type="bibr" rid="pone.0117849.ref001">
               <sup>
                  <a href="#pone.0117849.ref001">1</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pone.0117849.ref003">
               <sup>
                  <a href="#pone.0117849.ref003">3</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pone.0117849.ref008">
               <sup>
                  <a href="#pone.0117849.ref008">8</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pone.0117849.ref010">
               <sup>
                  <a href="#pone.0117849.ref010">10</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pone.0117849.ref013">
               <sup>
                  <a href="#pone.0117849.ref013">13</a>
               </sup>
            </span>]: the tick-borne, mosquito-borne, and no known-vector flaviviruses (TBFV, MBFV and
            NKV respectively). The TBFV are sub-divided into pathogenic flaviviruses primarily
            associated with 
            <i>Ixodes</i> spp., and apathogenic flaviviruses associated with 
            <i>Ornithodorus</i> spp. ticks that primarily feed on or parasitize, seabirds. The mosquito-borne flaviviruses
            (MBFV) are sub-divided into those primarily associated with 
            <i>Culex</i> spp. (ornithophilic) mosquitoes and those primarily associated with 
            <i>Aedes</i> spp. (mammalophilic) mosquitoes. In contrast, the viruses with no known vectors (NKV)
            are divided into those primarily associated either with rodents or bats. These specific
            lineage NKVs are defined in the text as NKV
            <sup>SL</sup>. In addition, the three viruses (Sokoluk, Entebbe bat and Yokose virus) are exceptional
            in that they diverged with the MBFV but arthropod vectors have not been associated
            with these viruses. In other words, they diverged within the MBFV—
            <i>Aedes</i> spp.-associated clade but appear to have lost this mosquito association [
            <span ref-type="bibr" rid="pone.0117849.ref010">
               <sup>
                  <a href="#pone.0117849.ref010">10</a>
               </sup>
            </span>]. These NKV are defined in the text as NKV-like.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">For further clarity, we will also distinguish ISFVs which are very divergent from
            the other members of the genus 
            <i>Flavivirus</i> by referring to them as ISFV
            <sup>SL</sup>. Finally, we will distinguish recently discovered viruses such as LAMV, which fall
            phylogenetically within the MBFV group but appear to infect only insects, as ISF-like.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The genus 
            <i>Flavivirus</i> also includes 2 highly divergent genetic lineages not associated with any recognized
            group in the ninth report of the International Committee on Taxonomy of Viruses (ICTV)
            [
            <span ref-type="bibr" rid="pone.0117849.ref007">
               <sup>
                  <a href="#pone.0117849.ref007">7</a>
               </sup>
            </span>], represented by Tamana bat virus [TABV] [
            <span ref-type="bibr" rid="pone.0117849.ref011">
               <sup>
                  <a href="#pone.0117849.ref011">11</a>
               </sup>
            </span>] and Ngoye virus [
            <span ref-type="bibr" rid="pone.0117849.ref014">
               <sup>
                  <a href="#pone.0117849.ref014">14</a>
               </sup>
            </span>]. Recently, another highly divergent lineage, Mogiana tick virus [MGTV], was isolated
            in Brazil [
            <span ref-type="bibr" rid="pone.0117849.ref024">
               <sup>
                  <a href="#pone.0117849.ref024">24</a>
               </sup>
            </span>].
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Whilst early interpretations of the evolutionary and taxonomic relationships within
            the genus 
            <i>Flavivirus</i> have proved informative, they were restricted to some extent by the lack of complete
            genomic sequence data, and robust analytical methods. As a result, it has not yet
            proven possible to resolve the issue that phylogenies based on complete genomes and,
            separately, the NS3 gene show different branching characteristics from those based
            on the NS5 gene, despite a lack of evidence for recombination within the respective
            data sets [
            <span ref-type="bibr" rid="pone.0117849.ref010">
               <sup>
                  <a href="#pone.0117849.ref010">10</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pone.0117849.ref011">
               <sup>
                  <a href="#pone.0117849.ref011">11</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pone.0117849.ref025">
               <sup>
                  <a href="#pone.0117849.ref025">25</a>
               </sup>
            </span>]. In this manuscript we report the genomic sequences of fourteen flaviviruses, for
            which previously only limited data were available. Based on these more comprehensive
            datasets, we attempt to resolve hitherto unanswered questions relating to the flaviviruses,
            and explore frame shift characteristics. Additionally, we identify viruses that appear
            to have been introduced from the Old World into the New World, estimate the likely
            times prior to the present that these introductions occurred, and discuss factors
            that probably contributed to the global dispersal of these viruses.
         </p>
      </div>
      <div id="sec002">
         <h2>MATERIALS AND METHODS</h2>
         <div id="sec003">
            <h3>Viruses</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">MBFVs included in the study were all subcultured at least once in C6/36 cells. Batu
               cave virus [BCV], Jutiapa virus [JUTV], Phnom Penh bat virus [PPBV] and Sokuluk virus
               [SOKV] were amplified in the mammalian cell-line BHK21 and Sitiawan virus [STWV] was
               amplified in Vero cell cultures. STWV virus was kindly provided by Dr. Yuji Kono as
               inactivated nucleic acid in RNA-Now lysis buffer.
            </p>
         </div>
         <div id="sec004">
            <h3>Nucleic acid preparation</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Viral RNA was either extracted using the BioRobot EZ1 (Viral RNA Mini kit: Qiagen)
               or RNA-Now (Biogentex) using the manufacturer’s recommendations. Reverse transcription
               was carried out using Taqman Reverse transcription reagents (Applied Biosystems) under
               standard conditions with random hexamers as primers.
            </p>
         </div>
         <div id="sec005">
            <h3>Polymerase chain reaction (PCR) in the conserved region of the genome</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">PCR was targeted at the E, NS3 and NS5 gene-conserved regions using consensus degenerate
               primers [
               <span ref-type="bibr" rid="pone.0117849.ref008">
                  <sup>
                     <a href="#pone.0117849.ref008">8</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref026">
                  <sup>
                     <a href="#pone.0117849.ref026">26</a>
                  </sup>
               </span>–
               <span ref-type="bibr" rid="pone.0117849.ref028">
                  <sup>
                     <a href="#pone.0117849.ref028">28</a>
                  </sup>
               </span>]. Sequences for the NS3 gene region were obtained using NS3-FS (5’-GGIGTIYTICAYACIATGTGGCAYGTIAC-3’)/NS3-FR
               (5’-TKICKICCIAYICKICCICKICKYTGIGCNGY-3’) primers in first round PCR, followed by nested
               PCR using X1 (5’-YIRTIGGIYTITAYGGIWWYGG-3’)/X2 (5’-RTTIGCICCCATYTCISHDATRTCIGT-3’)
               primers, with standard conditions and a hybridisation temperature of 45°C.
            </p>
         </div>
         <div id="sec006">
            <h3>Sequencing strategy</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Specific primers were designed from the 3 conserved region sequences defined above
               and long-range PCR was conducted to complete the sequencing of gap between the E to
               NS3 genes, and NS3 to NS5 genes, using the cMaster RTplusPCR system (Eppendorf) [
               <span ref-type="bibr" rid="pone.0117849.ref003">
                  <sup>
                     <a href="#pone.0117849.ref003">3</a>
                  </sup>
               </span>]. A long PCR product sequencing protocol (LoPPs) was employed to sequence amplicons
               [
               <span ref-type="bibr" rid="pone.0117849.ref029">
                  <sup>
                     <a href="#pone.0117849.ref029">29</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref030">
                  <sup>
                     <a href="#pone.0117849.ref030">30</a>
                  </sup>
               </span>].
            </p>
         </div>
         <div id="sec007">
            <h3>PCR amplification of the 5’ and 3’ end of coding sequences</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The 5’-terminal region of the genome sequence was obtained using an E-gene specific
               reverse primer and a forward degenerate primer in the 5’UTR designed using an alignment
               of available 5’ UTRs of 
               <i>Culex</i>-spp. associated flavivirus sequences (5’-CULEX-S1: 5’-AGwiGTTCryCTGyGTGArCT-3’; position
               1–21 of the Japanese encephalitis virus genome). Semi-nested PCR with a second inner
               virus-specific primer was also used when necessary. The 3'-terminal region of the
               genome was obtained using a similar strategy with virus-specific forward primers in
               the NS5 region and a reverse degenerate primer in the 3’UTR (3’UTR-MOS: 5’-GGTCTCCWMTAACCTCTAG-3’).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Sequencing was conducted with the primers used for amplification, or with M13 primers
               after cloning in a pCR2 cloning vector (Invitrogen).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Complete polyprotein open-reading frame (ORF) sequences, excluding the partial untranslated
               regions (UTR) regions obtained by this protocol were used for further analyses.
            </p>
         </div>
         <div id="sec008">
            <h3>Next Generation Sequencing (NGS)</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Resequencing of seven of the eleven new complete polyprotein ORF sequences (AROAV,
               CPCV, ITV, KOUV, NTAV, TMUV and YAOV) was performed using the Ion PGM Sequencer (Life
               Technologies SAS, Saint Aubin, France) [
               <span ref-type="bibr" rid="pone.0117849.ref031">
                  <sup>
                     <a href="#pone.0117849.ref031">31</a>
                  </sup>
               </span>] and a random reverse transcription-amplification protocol. Reads, of minimum length
               30 nucleotides, were trimmed using CLC Genomic Workbench 6.5 (QIAGEN Company), with
               a minimum of 99% quality per base and mapped to reference sequences previously obtained
               by the Sanger method. Parameters were set such that each accepted read had to map
               to the reference sequence for at least 50% of its length, with a minimum of 80% identity
               to the reference.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">BCV, JUTV and PPBV sequences were obtained using the same NGS method and 
               <i>de novo</i> assembly.
            </p>
         </div>
         <div id="sec009">
            <h3>Sequence analysis</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Sequences were refined using Sequencher 4.8 (Gene Codes, Ann Arbor, MI) and combined
               with other flavivirus sequences available in the Genbank database, to obtain a dataset
               including a representative of at least one sequence for each species available as
               full polyprotein ORF for the genus 
               <i>Flavivirus</i>. Genbank accession numbers of sequences used for the analysis are noted after each
               virus abbreviation on 
               <span ref-type="fig" rid="pone.0117849.g001">Fig. 1</span> (tree). Complete polyprotein ORF amino acid alignments were generated using both
               Clustal W2 [
               <span ref-type="bibr" rid="pone.0117849.ref032">
                  <sup>
                     <a href="#pone.0117849.ref032">32</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref033">
                  <sup>
                     <a href="#pone.0117849.ref033">33</a>
                  </sup>
               </span>] and MUSCLE [
               <span ref-type="bibr" rid="pone.0117849.ref034">
                  <sup>
                     <a href="#pone.0117849.ref034">34</a>
                  </sup>
               </span>] available at the EMBL server (
               <div tagxxx="ext-link">http://www.ebi.ac.uk/Tools/</div>) and refined manually, for comparison. Nucleotide alignments were then deduced using
               amino acid (AA) alignments as a template using the TranAlign software available via
               the EMBOSS server (
               <div tagxxx="ext-link">http://emboss.bioinformatics.nl/cgi-bin/emboss/tranalign</div>). The effect of removing regions of ambiguous alignment via the GBlocks algorithm
               [
               <span ref-type="bibr" rid="pone.0117849.ref035">
                  <sup>
                     <a href="#pone.0117849.ref035">35</a>
                  </sup>
               </span>] using less stringent parameters was also investigated.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pone.0117849.g001</div>
               <span class="label">Fig 1</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Bayesian phylogeny of the ORF ‘global genus’ amino acid dataset.</div>
                  <p>Only posterior probabilities of 0.9 are included. The tree is midpoint rooted. Bar,
                     0.3 substitutions per site.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Amino acid phylogenetic trees were reconstructed using Markov chain Monte Carlo (MCMC)
               analysis implemented in MrBayes v3.1.2 [
               <span ref-type="bibr" rid="pone.0117849.ref036">
                  <sup>
                     <a href="#pone.0117849.ref036">36</a>
                  </sup>
               </span>]. The analysis was performed using the WAG substitution model with gamma distributed
               rate variation among sites and using default priors. Five independent Markov chains
               were run for 10 million generations, with the first 10% of samples discarded as burn-in.
               Stationarity was confirmed based on effective sample sizes &gt;400 using Tracer v1.4.1
               [
               <span ref-type="bibr" rid="pone.0117849.ref037">
                  <sup>
                     <a href="#pone.0117849.ref037">37</a>
                  </sup>
               </span>]. A maximum clade credibility tree was summarised using TreeAnnotator which annotates
               all nodes with posterior probability support values. A range of nucleotide analyses
               was also conducted (both including and excluding third codon positions).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Secondary RNA structures were investigated using the RNAfold webserver (
               <div tagxxx="ext-link">http://rna.tbi.univie.ac.at/cgi-bin/RNAfold.cgi</div>) and pknotsRG [
               <span ref-type="bibr" rid="pone.0117849.ref038">
                  <sup>
                     <a href="#pone.0117849.ref038">38</a>
                  </sup>
               </span>]. Synonymous site conservation was analysed as described previously [
               <span ref-type="bibr" rid="pone.0117849.ref039">
                  <sup>
                     <a href="#pone.0117849.ref039">39</a>
                  </sup>
               </span>].
            </p>
         </div>
         <div id="sec010">
            <h3>Bayesian inference of a time-measured evolutionary history</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A second data set of polyprotein ORF flaviviruses was compiled by excluding the highly
               divergent sequences of all ISFVs and including additional sequences of YFV that were
               isolated from either the New World or the Old World. An amino acid alignment was produced
               using Clustal W2 and regions of ambiguous alignment were removed using the GBlocks
               algorithm with standard parameters.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A time-measured evolutionary history was inferred using MCMC analysis implemented
               in BEAST [
               <span ref-type="bibr" rid="pone.0117849.ref040">
                  <sup>
                     <a href="#pone.0117849.ref040">40</a>
                  </sup>
               </span>]. Based on model testing using Prottest, we used the LG substitution model [
               <span ref-type="bibr" rid="pone.0117849.ref041">
                  <sup>
                     <a href="#pone.0117849.ref041">41</a>
                  </sup>
               </span>] with gamma distributed rate variation among sites in conjunction with a relaxed
               uncorrelated lognormal molecular clock model [
               <span ref-type="bibr" rid="pone.0117849.ref042">
                  <sup>
                     <a href="#pone.0117849.ref042">42</a>
                  </sup>
               </span>] and a Bayesian skygrid tree-prior [
               <span ref-type="bibr" rid="pone.0117849.ref043">
                  <sup>
                     <a href="#pone.0117849.ref043">43</a>
                  </sup>
               </span>].
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Time-calibration of the evolutionary history was based on the recognised slave-trade
               introduction of YFV to Brazil [
               <span ref-type="bibr" rid="pone.0117849.ref012">
                  <sup>
                     <a href="#pone.0117849.ref012">12</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref044">
                  <sup>
                     <a href="#pone.0117849.ref044">44</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref045">
                  <sup>
                     <a href="#pone.0117849.ref045">45</a>
                  </sup>
               </span>]. Specifically, based on estimates from 
               <div tagxxx="ext-link">www.slavevoyages.org</div>, we constrained the common ancestor of the American YFVs to have existed before 1860,
               and the divergence from the West-African YFVs to have occurred after 1561. The divergence
               of the South American YFV from the African YFV strains represents the upper boundary
               for the introduction. Therefore, this node is assumed to be younger than 449 years.
               The common ancestor of all South American YFV strains represents the lower boundary
               for the slave trade introduction, and this node is therefore assumed to be older than
               150 years. Given the long evolutionary time-scale and the fact that we performed our
               analysis at the amino acid level to allow the estimation of relatively deeper divergence
               times, instead of resorting to potentially saturated nucleotides, and to avoid a potential
               disconnect between short-term and long-term evolutionary rates [
               <span ref-type="bibr" rid="pone.0117849.ref046">
                  <sup>
                     <a href="#pone.0117849.ref046">46</a>
                  </sup>
               </span>], we did not consider sampling time differences when estimating the timed history.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We also incorporated a two-state discrete diffusion model for the Old World and New
               World locations in our analyses and jointly estimated the ancestral geographical states
               with the evolutionary history [
               <span ref-type="bibr" rid="pone.0117849.ref047">
                  <sup>
                     <a href="#pone.0117849.ref047">47</a>
                  </sup>
               </span>]. The BEAST analysis was run for 20 million generations and diagnosized using Tracer.
               Trees were summarized using TreeAnnotator and visualized using FigTree (
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span>).
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pone.0117849.g002</div>
               <span class="label">Fig 2</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Maximum clade credibility tree summarized from the Bayesian molecular clock analysis.</div>
                  <p>All nodes attained maximal probability support except for those indicated by a * (posterior
                     probability &lt; 0.95). The node age uncertainty is indicated using 95% highest posterior
                     density (HPD) interval bars in blue. Old World and New World ancestral states are
                     indicated by black and red branches/nodes respectively. Nodes of interest are listed
                     A to N as referred in the text.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
         </div>
         <div id="sec011">
            <h3>Biogeographical data</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Available ecological data, geographical dispersal patterns, host association and pathogenicity
               for all mosquito-borne flaviviruses were retrieved from the CRORA viral database (Centre
               collaborateur OMS de Reference et de Recherche sur les Arbovirus, Institut Pasteur
               de Dakar, Africa 
               <div tagxxx="ext-link">http://www.pasteur.fr/recherche/banques/CRORA/</div>) and the International Catalogue of Arboviruses [
               <span ref-type="bibr" rid="pone.0117849.ref048">
                  <sup>
                     <a href="#pone.0117849.ref048">48</a>
                  </sup>
               </span>]. Three viruses tentatively assigned to the genus 
               <i>Flavivirus</i>, TABV [
               <span ref-type="bibr" rid="pone.0117849.ref011">
                  <sup>
                     <a href="#pone.0117849.ref011">11</a>
                  </sup>
               </span>], Ngoye virus [
               <span ref-type="bibr" rid="pone.0117849.ref014">
                  <sup>
                     <a href="#pone.0117849.ref014">14</a>
                  </sup>
               </span>] and Mogiana tick [
               <span ref-type="bibr" rid="pone.0117849.ref024">
                  <sup>
                     <a href="#pone.0117849.ref024">24</a>
                  </sup>
               </span>] virus were not included in analyses because they are genetically too divergent to
               be incorporated without ambiguities using currently-available methods [
               <span ref-type="bibr" rid="pone.0117849.ref010">
                  <sup>
                     <a href="#pone.0117849.ref010">10</a>
                  </sup>
               </span>].
            </p>
         </div>
      </div>
      <div id="sec012">
         <h2>RESULTS</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Using our sequencing strategy, we obtained complete coding sequence for 10 
            <i>Culex</i>-spp. associated flaviviruses (Aroa virus [AROAV], Naranjal virus [NJLV], Cacipacore
            virus [CPCV], Koutango virus [KOUV], Yaounde virus [YAOV], Stratford virus [STRV],
            Israel turkey meningoencephalomyelitis virus [ITV], Ntaya virus [NTAV], Tembusu virus
            [TMUV] and STWV. Additionally, the remaining non-vectored viruses (NKV-like) in the
            MBFV group, viz., SOKV and 3 NKV
            <sup>SL</sup> flaviviruses BCV, JUTV and PPBV were sequenced. With the addition of these new sequences,
            and by taking into account other recently-published data [
            <span ref-type="bibr" rid="pone.0117849.ref049">
               <sup>
                  <a href="#pone.0117849.ref049">49</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pone.0117849.ref052">
               <sup>
                  <a href="#pone.0117849.ref052">52</a>
               </sup>
            </span>], we have produced the most comprehensive review and phylogeny of flavivirus full
            polyprotein ORF sequences and phylogeographic information to date.
         </p>
         <div id="sec013">
            <h3>Phylogenetic analysis</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <span ref-type="fig" rid="pone.0117849.g001">Fig. 1</span> presents a phylogenetic tree based on all the sequence data yielding the highest
               posterior probabilities out of all analyses performed. The tree is based on amino
               acid sequences aligned using ClustalW2. There are high levels of support at all nodes
               (with a posterior probability ≥ to 0.98 for all nodes, with the exception of one node
               intra WNV lineage 2 at 0.6 and one node intra DENV-3 strains at 0,5). Clades suggested
               by this analysis correspond to the recognised virus-vector-host groups defined above
               and previously [
               <span ref-type="bibr" rid="pone.0117849.ref001">
                  <sup>
                     <a href="#pone.0117849.ref001">1</a>
                  </sup>
               </span>].
            </p>
         </div>
         <div id="sec014">
            <h3>TBFVs</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The phylogenetic relationships of the TBFVs agree with previous studies [
               <span ref-type="bibr" rid="pone.0117849.ref003">
                  <sup>
                     <a href="#pone.0117849.ref003">3</a>
                  </sup>
               </span>] and are not elaborated upon further in this manuscript.
            </p>
         </div>
         <div id="sec015">
            <h3>NKVs</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The inclusion of the complete polyprotein ORF sequence for SOKV corroborates and extends
               previous suggested groupings for the three NKV-like bat-associated viruses, namely
               Entebbe bat virus [ENTV], Yokose virus [YOKV] and SOKV which cluster within the 
               <i>Aedes</i>-associated mosquito-borne virus clade. These three NKV-like viruses represent the
               ICTV “Entebbe bat virus group” [
               <span ref-type="bibr" rid="pone.0117849.ref007">
                  <sup>
                     <a href="#pone.0117849.ref007">7</a>
                  </sup>
               </span>] and form a basal lineage for the YFV group and the Edge Hill virus [EHV] group.
               In addition, our analysis confirms that this NKV-like group of viruses currently sits
               in the clade that contains 
               <i>Aedes</i> spp. MBFVs with a robust branching pattern supported by high posterior probabilities.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Sokoluk virus was isolated in 1970 from 
               <i>Vespertilio pipistrellus</i> bats in Kyrgyzstan [
               <span ref-type="bibr" rid="pone.0117849.ref053">
                  <sup>
                     <a href="#pone.0117849.ref053">53</a>
                  </sup>
               </span>]. According to the 9
               <sup>th</sup> report of the ICTV [
               <span ref-type="bibr" rid="pone.0117849.ref007">
                  <sup>
                     <a href="#pone.0117849.ref007">7</a>
                  </sup>
               </span>] SOKV is a member of the Entebbe bat virus group, and appears to be a strain of the
               species ENTV. 
               <i>Entebbe bat virus</i> is an African virus isolated from the insectivorous migratory bat species 
               <i>Tadarida</i> (Chaerephon) 
               <i>limbata</i>. It therefore seems reasonable to propose that SOKV probably emerged in Africa and
               dispersed to Asia via transmission across areas of overlapping bat habitats. Importantly,
               ENTV and SOKV have been shown to replicate in C6/36 mosquito cells 
               <i>in vitro</i> [
               <span ref-type="bibr" rid="pone.0117849.ref054">
                  <sup>
                     <a href="#pone.0117849.ref054">54</a>
                  </sup>
               </span>] whereas the recognised NKV
               <sup>SL</sup> that form a genetically distant clade (Rio bravo [RBV], Montana myotis leukoencephalitis
               virus [MMLV], etc.) do not replicate in mosquito cells [
               <span ref-type="bibr" rid="pone.0117849.ref055">
                  <sup>
                     <a href="#pone.0117849.ref055">55</a>
                  </sup>
               </span>]. This observation together with the phylogenetic position of these viruses in the
               MBFV clade is consistent with the idea that the transmission or maintenance cycles
               of YOKV, ENTV and SOKV (or ancestral representatives of these viruses), may have involved
               mosquitoes in the past.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The addition of the complete ORF sequences of BCV, JUTV and PPBV corroborates and
               extends our knowledge regarding the poorly-described NKV
               <sup>SL</sup> group. Apoi virus (a Japanese virus isolated from 
               <i>Apodemus</i> spp. and 
               <i>Clethrionomys</i> spp.) constitutes a separate group which appears to have diverged earlier from the
               other viruses, which are distributed into two groups that include viruses isolated
               from rodents and bats respectively [
               <span ref-type="bibr" rid="pone.0117849.ref025">
                  <sup>
                     <a href="#pone.0117849.ref025">25</a>
                  </sup>
               </span>]. Bat-associated viruses constitute 2 sub-groups including MMLV and RBV, PPBV and
               BCV, respectively. Rodent-associated viruses are represented by Modoc virus and JUTV
               in our analysis. The other representatives of rodent and bat NKVs were not included
               in this analysis due to the absence of complete ORF sequences.
            </p>
         </div>
         <div id="sec016">
            <h3>MBFVs</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Using partial genome sequence-based phylogenies, the MBFVs were previously divided
               into 2 major groups reflecting their vector associations (
               <i>i</i>.
               <i>e</i>., 
               <i>Aedes-</i> and 
               <i>Culex</i>-associated groups), the principal vertebrate host and associated diseases [
               <span ref-type="bibr" rid="pone.0117849.ref001">
                  <sup>
                     <a href="#pone.0117849.ref001">1</a>
                  </sup>
               </span>]. Our analysis, based on complete ORF sequence data, confirmed this division of MBFVs
               into 2 major epidemiologically distinct vector groups, 
               <i>i</i>.
               <i>e</i>., those primarily associated either with 
               <i>Culex</i> spp. or 
               <i>Aedes</i> spp mosquitoes. However, additional ISFV-“like” MBFVs became available for phylogenetic
               analysis. Lammi virus [LAMV] was isolated from 
               <i>A</i>. 
               <i>cinereus</i> mosquitoes in Finland [
               <span ref-type="bibr" rid="pone.0117849.ref013">
                  <sup>
                     <a href="#pone.0117849.ref013">13</a>
                  </sup>
               </span>], Ilomantsie virus [ILOV] was isolated from 
               <i>Ochlerotatus</i> mosquitoes in Finland (Huhtamo et 
               <i>al</i>., submitted paper), Marisma mosquito virus [MMV] from 
               <i>O</i>. 
               <i>caspius</i> in Spain [
               <span ref-type="bibr" rid="pone.0117849.ref018">
                  <sup>
                     <a href="#pone.0117849.ref018">18</a>
                  </sup>
               </span>], Donggang virus [DGV] from 
               <i>Aedes</i> mosquitoes in China (Unpublished data, GenBank acc. Number: NC_016997), Chaoyang
               virus [CHAOV] from 
               <i>A</i>. 
               <i>vexans nipponii</i> in Korea [
               <span ref-type="bibr" rid="pone.0117849.ref056">
                  <sup>
                     <a href="#pone.0117849.ref056">56</a>
                  </sup>
               </span>] and in China (Article published in Chinese, GenBank acc. Number: FJ883471), Nounané
               virus [NOUV] from 
               <i>Uranotaenia</i> spp. mosquitoes in Ivory Coast [
               <span ref-type="bibr" rid="pone.0117849.ref057">
                  <sup>
                     <a href="#pone.0117849.ref057">57</a>
                  </sup>
               </span>] and Barkedji virus [BJV] from 
               <i>Culex perexiguus</i> in Israel [
               <span ref-type="bibr" rid="pone.0117849.ref058">
                  <sup>
                     <a href="#pone.0117849.ref058">58</a>
                  </sup>
               </span>] and Senegal (Unpublished data, GenBank acc. Number: EU078325). The mosquito vector
               was originally unknown for CHAOV (China) and BJV (Senegal) but these viruses have
               also been recently described in other locations [
               <span ref-type="bibr" rid="pone.0117849.ref058">
                  <sup>
                     <a href="#pone.0117849.ref058">58</a>
                  </sup>
               </span>]. Thus, based on this more recent evidence the probable vectors for CHAOV and BJV
               are 
               <i>Aedes</i> and 
               <i>Culex</i> species respectively.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">These viruses form two distinct groups (
               <span ref-type="fig" rid="pone.0117849.g001">Fig. 1</span>) that diverge from the two previously recognised groups 
               <i>Aedes</i> spp.-associated and 
               <i>Culex</i> spp.-associated MBFV. Moreover, the branching pattern of the tree implies that they
               emerged after the 
               <i>Aedes</i> spp.-associated virus lineage but before the 
               <i>Culex</i> spp.-associated virus lineage. Currently, they sit in the major clade of 
               <i>Aedes</i>-associated viruses and represent an integral part of the evolutionary continuum amongst
               the MBFVs. These two groups of Eurasian viruses, containing LAMV, ILOV, DGV and CHAOV
               and the African virus group containing BJV and NOUV, appear to replicate only in mosquito
               cells, possibly with transitory replication in vertebrate cell lines [
               <span ref-type="bibr" rid="pone.0117849.ref013">
                  <sup>
                     <a href="#pone.0117849.ref013">13</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref057">
                  <sup>
                     <a href="#pone.0117849.ref057">57</a>
                  </sup>
               </span>]. This is compatible with 2 independent events during which these MBFVs lost their
               ability to infect vertebrate cells. Recently, Nanay virus was isolated from 
               <i>Culex ocossa</i> in Peru [
               <span ref-type="bibr" rid="pone.0117849.ref059">
                  <sup>
                     <a href="#pone.0117849.ref059">59</a>
                  </sup>
               </span>]. This virus was only partially sequenced in the E and NS5 gene but it seems to be
               closely related to NOUV, leaving the question of a potentially more widespread dispersal
               of this new ISFV-like virus in the NW. This is not surprising, in view of other papers
               which reported the widespread distribution of ISFV and the huge extent of undersampling.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <i>Aedes</i>-associated flaviviruses (
               <span ref-type="fig" rid="pone.0117849.g001">Fig. 1</span>) include the dengue virus [DENV] group, the yellow fever virus [YFV] group, the EHV
               group, and the Kedougou virus [KEDV] group. Results from our analyses are in accordance
               with the most recently published studies [
               <span ref-type="bibr" rid="pone.0117849.ref008">
                  <sup>
                     <a href="#pone.0117849.ref008">8</a>
                  </sup>
               </span>].
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The complete ORF sequences of 10 
               <i>Culex spp</i>.-associated flaviviruses were determined and included in the phylogenetic tree (
               <span ref-type="fig" rid="pone.0117849.g001">Fig. 1</span>). Thus, all viruses currently known to fall within this group have now been characterized.
            </p>
            <div id="sec017">
               <h3>CPCV</h3>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">CPCV is a bird-associated virus, isolated in Brazil from the blood of the black-faced
                  antbird. This virus has never been found in mosquitoes. However, this could reflect
                  insufficient sampling of field materials. In the most recent phylogenies (
                  <span ref-type="fig" rid="pone.0117849.g001">Fig. 1</span>), CPCV always roots the JEV group with a posterior probability of 1. Recently, it
                  was isolated in Brazil from a human presenting clinically with leptospirosis and/or
                  yellow fever-like illness [
                  <span ref-type="bibr" rid="pone.0117849.ref060">
                     <sup>
                        <a href="#pone.0117849.ref060">60</a>
                     </sup>
                  </span>].
               </p>
            </div>
            <div id="sec018">
               <h3>YAOV, KOUV and WNV</h3>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">YAOV and KOUV represent ancestral lineages of WNV in the phylogenetic tree (
                  <span ref-type="fig" rid="pone.0117849.g001">Fig. 1</span>). YAOV has been isolated in Africa in the Cameroon, Central African Republic, Congo,
                  Senegal and Ivory Coast from birds, mammals and both 
                  <i>Culex</i> and 
                  <i>Aedes</i> mosquito species and has never been identified as a human pathogen [
                  <span ref-type="bibr" rid="pone.0117849.ref061">
                     <sup>
                        <a href="#pone.0117849.ref061">61</a>
                     </sup>
                  </span>]. In contrast, KOUV was isolated in Senegal from rodents (
                  <i>Tatera kempi</i> and 
                  <i>Mastomys sp</i>.) and also from a human following a laboratory infection [
                  <span ref-type="bibr" rid="pone.0117849.ref048">
                     <sup>
                        <a href="#pone.0117849.ref048">48</a>
                     </sup>
                  </span>].
               </p>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The sequences of two viruses isolated in Eastern Europe and Russia, namely Rabensburg
                  virus (RABV) and Krasnodar virus (KRDV) respectively, from a pool of 
                  <i>Culex pipiens</i> mosquitoes and 
                  <i>Dermacentor marginatus</i> ticks, were also included in the phylogenetic analysis because they are related to,
                  but show significant divergence, from WNV [
                  <span ref-type="bibr" rid="pone.0117849.ref062">
                     <sup>
                        <a href="#pone.0117849.ref062">62</a>
                     </sup>
                  </span>,
                  <span ref-type="bibr" rid="pone.0117849.ref063">
                     <sup>
                        <a href="#pone.0117849.ref063">63</a>
                     </sup>
                  </span>]. In addition, the Indian lineage WNV strain India 804994 isolated from a human,
                  was included in the analysis [
                  <span ref-type="bibr" rid="pone.0117849.ref064">
                     <sup>
                        <a href="#pone.0117849.ref064">64</a>
                     </sup>
                  </span>].
               </p>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The phylogeny (
                  <span ref-type="fig" rid="pone.0117849.g001">Fig. 1</span>) supports previous suggestions [
                  <span ref-type="bibr" rid="pone.0117849.ref065">
                     <sup>
                        <a href="#pone.0117849.ref065">65</a>
                     </sup>
                  </span>] that WNV has an African origin, as the African YAOV roots the WNV group.
               </p>
            </div>
            <div id="sec019">
               <h3>MVEV and ALFV</h3>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Other members within the JEV serocomplex including JEV, Usutu virus [USUV], Murray
                  Valley encephalitis virus [MVEV] and Alfuy virus [ALFV], form a strongly supported
                  sister group to these viruses, all sharing a common ancestor with CPCV. Notably, the
                  2 Australian viruses MVEV and ALFV share the same ecological niche and are considered
                  to be a single species [
                  <span ref-type="bibr" rid="pone.0117849.ref007">
                     <sup>
                        <a href="#pone.0117849.ref007">7</a>
                     </sup>
                  </span>], with ALFV being a strain of MVEV. In common with other JEV serocomplex viruses,
                  MVEV and ALFV have been isolated birds and mosquitoes. MVEV causes hundreds of human
                  cases of encephalitis annually in Australia. In contrast, there is only one unconfirmed
                  case of mild polyarticular disease (in 1987) doe to ALFV [
                  <span ref-type="bibr" rid="pone.0117849.ref048">
                     <sup>
                        <a href="#pone.0117849.ref048">48</a>
                     </sup>
                  </span>]. Previous studies in laboratory animals showed that ALFV is less neuroinvasive than
                  MVEV following peripheral challenge [
                  <span ref-type="bibr" rid="pone.0117849.ref066">
                     <sup>
                        <a href="#pone.0117849.ref066">66</a>
                     </sup>
                  </span>].
               </p>
            </div>
            <div id="sec020">
               <h3>ITV and BAGV</h3>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Within the recognized Ntaya virus group [
                  <span ref-type="bibr" rid="pone.0117849.ref007">
                     <sup>
                        <a href="#pone.0117849.ref007">7</a>
                     </sup>
                  </span>], we have determined the polyprotein ORF sequences of NTAV, ITV, TMUV and STWV. ITV
                  and Bagaza virus [BAGV] are bird-associated viruses that cause encephalitis in poultry
                  and wild birds. Both of these viruses appear to have their evolutionary origins in
                  Africa, although ITV is a frequent cause of avian disease in Israel and BAGV was recently
                  identified as the aetiological agent of bird fatalities in birds in southern Spain
                  [
                  <span ref-type="bibr" rid="pone.0117849.ref067">
                     <sup>
                        <a href="#pone.0117849.ref067">67</a>
                     </sup>
                  </span>,
                  <span ref-type="bibr" rid="pone.0117849.ref068">
                     <sup>
                        <a href="#pone.0117849.ref068">68</a>
                     </sup>
                  </span>]. BAGV was also isolated in India and human exposure was implied by detection of
                  BAGV neutralizing antibodies in 15% of the human population [
                  <span ref-type="bibr" rid="pone.0117849.ref069">
                     <sup>
                        <a href="#pone.0117849.ref069">69</a>
                     </sup>
                  </span>]. No human exposures have been reported for ITV.
               </p>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">During the preparation of this manuscript, the sequences of 5 strains of ITV were
                  reported [
                  <span ref-type="bibr" rid="pone.0117849.ref070">
                     <sup>
                        <a href="#pone.0117849.ref070">70</a>
                     </sup>
                  </span>]. The authors suggested that ITV and BAGV should be considered a single species with
                  2 different clades representing “old isolates” and “recent isolates”. The sequence
                  presented in our analysis is an isolate from 1959, included in the clade of “old isolates”
                  (
                  <span ref-type="fig" rid="pone.0117849.g001">Fig. 1</span>) and is 6% divergent at the nucleotide level from all other ITV and BAGV sequences.
               </p>
            </div>
            <div id="sec021">
               <h3>TMUV, STWV and DEDSV</h3>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">TMUV, STWV and the recently described duck egg drop syndrome virus [DEDSV] [
                  <span ref-type="bibr" rid="pone.0117849.ref051">
                     <sup>
                        <a href="#pone.0117849.ref051">51</a>
                     </sup>
                  </span>] and layer flavivirus (unpublished data, Genbank acc nb.: JF926699) are also closely
                  related southern Asian strains, mainly isolated in Thailand, China and Korea, but
                  they are phylogenetically distinct from ITV and BAGV [
                  <span ref-type="bibr" rid="pone.0117849.ref071">
                     <sup>
                        <a href="#pone.0117849.ref071">71</a>
                     </sup>
                  </span>–
                  <span ref-type="bibr" rid="pone.0117849.ref073">
                     <sup>
                        <a href="#pone.0117849.ref073">73</a>
                     </sup>
                  </span>]. They also cause severe disease pathologies in domestic birds (chicken, duck, etc.)
                  but have not been associated with human disease.
               </p>
            </div>
            <div id="sec022">
               <h3>NTAV</h3>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">NTAV was originally isolated in Uganda. During the preparation of this manuscript,
                  the sequence of another strain of Ntaya virus was reported [
                  <span ref-type="bibr" rid="pone.0117849.ref074">
                     <sup>
                        <a href="#pone.0117849.ref074">74</a>
                     </sup>
                  </span>]. The strain provisionally designated IPD/A was collected in the Cameroon region
                  of Africa in 1966. This virus differs by only 0.08% in nucleotide sequence when compared
                  with the original strain that was sequenced in our study (
                  <span ref-type="fig" rid="pone.0117849.g001">Fig. 1</span>). Positive human serological evidence in many regions of Africa confirms that NTAV
                  does infect humans [
                  <span ref-type="bibr" rid="pone.0117849.ref074">
                     <sup>
                        <a href="#pone.0117849.ref074">74</a>
                     </sup>
                  </span>].
               </p>
            </div>
            <div id="sec023">
               <h3>AROAV group</h3>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">With determination of the ORF sequences of AROAV and NJLV, the sequences of all viruses
                  recognised to date within the AROAV group are now known. Both AROAV and NJLV were
                  isolated from sentinel hamsters in South America (Venezuela and Ecuador respectively).
                  NJLV has also been isolated from 
                  <i>Culex</i> spp., mosquitoes.
               </p>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Bussuquara virus [BSQV] and Iguape virus [IGUV] are 2 other representatives of this
                  group and have been respectively isolated in sentinel monkeys and sentinel mice, in
                  Brazil. As shown in 
                  <span ref-type="fig" rid="pone.0117849.g001">Fig. 1</span>, NJLV and BSQV form a sister group, as do AROAV and IGUV. These 4 viruses have all
                  been isolated from mammals and each has its own ecological and biological niche. With
                  the exception of BSQV which causes symptoms in humans that include fever, headache
                  and arthralgia, other members are not known to be human pathogens.
               </p>
            </div>
            <div id="sec024">
               <h3>STRV and KOKV</h3>
               <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">With the complete ORF determination of STRV in this study, all of the recognised viruses
                  known to date in the KOKV group have now been sequenced. Interestingly, STRV has only
                  been isolated from 
                  <i>Aedes vexans</i> mosquitoes whereas KOKV was also isolated from 
                  <i>Culex annulirostris</i> [
                  <span ref-type="bibr" rid="pone.0117849.ref048">
                     <sup>
                        <a href="#pone.0117849.ref048">48</a>
                     </sup>
                  </span>], both in Australia and Papua New Guinea. STRV is not known to cause human pathology,
                  whereas serological evidence in humans has been reported for KOKV and occasionally
                  it is responsible for acute polyarthritic disease (3 cases) with fever, headache and
                  lethargy [
                  <span ref-type="bibr" rid="pone.0117849.ref075">
                     <sup>
                        <a href="#pone.0117849.ref075">75</a>
                     </sup>
                  </span>]. Additionally, the KOKV group also appears to include 2 new members designated TS5273
                  and New Mapoon virus [NMV] (CY1014). New Mapoon virus is included with complete polyprotein
                  ORF sequence in 
                  <span ref-type="fig" rid="pone.0117849.g001">Fig. 1</span> [
                  <span ref-type="bibr" rid="pone.0117849.ref017">
                     <sup>
                        <a href="#pone.0117849.ref017">17</a>
                     </sup>
                  </span>,
                  <span ref-type="bibr" rid="pone.0117849.ref076">
                     <sup>
                        <a href="#pone.0117849.ref076">76</a>
                     </sup>
                  </span>].
               </p>
            </div>
         </div>
         <div id="sec025">
            <h3>ISFV
               <sup xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">SL</sup>
            </h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">During the past decade, many new ISFV
               <sup>SL</sup> have been isolated and their sequence data are consistent with the concept that they
               should be classified as a fourth major group of flaviviruses. Moreover it was recently
               observed that the ISFV
               <sup>SL</sup> could be sub-divided into 2 sub-groups: 
               <i>Stegomyia</i> (
               <i>Aedes</i>) associated viruses and 
               <i>Culex</i> associated viruses based on all inferred phylogenies. With the addition of recent
               discoveries such as Nakiwogo virus (NAKV) isolated from 
               <i>Mansonia</i> [
               <span ref-type="bibr" rid="pone.0117849.ref020">
                  <sup>
                     <a href="#pone.0117849.ref020">20</a>
                  </sup>
               </span>] and Palm Creek virus (PCV) isolated from 
               <i>Coquillettidia</i> [
               <span ref-type="bibr" rid="pone.0117849.ref077">
                  <sup>
                     <a href="#pone.0117849.ref077">77</a>
                  </sup>
               </span>] the ISFV
               <sup>SL</sup> group is becoming increasingly complex. Indeed the phylogeny now shows a potential
               third and/or fourth sub-group that includes NAKV and PCV. This is not surprising considering
               that the likely mosquito vectors of the two viruses, 
               <i>Coquillettidia</i> and 
               <i>Mansonia</i> spp., are considered by morphological data to be sister groups (Harbach &amp; Kitching,
               1998). Also, these viruses are significantly undersampled and more vector distribution
               studies and virus discovery would help to clarify their phylogenetic status.
            </p>
         </div>
         <div id="sec026">
            <h3>A timed evolutionary perspective for Old to New World introductions</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">From a phylogeographic point of view, the evolutionary origins and dispersal patterns
               of many of the flaviviruses can be tentatively deduced by considering their association
               between phylogenetic clustering and geographic location, knowledge of historical anthropological
               patterns, host/vector associations and estimated times from the present, of divergence
               from a common ancestor. For example, all the 
               <i>Culex</i>-spp. associated virus clades are rooted by Old World 
               <i>Aedes</i>-spp. associated viruses.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To provide formal support for these observations, we performed a separate Bayesian
               phylogenetic analysis under a relaxed molecular clock model. A summary of this analysis
               is represented by the maximum clade credibility tree, which has strong branching support
               for most of the clades; all nodes had high posterior probabilities (&gt; 0.95) except
               those labelled with a star (*). Time-calibration for the tree, was based on the principle
               that YFV was introduced from the Old World into the New World during the slave-trade
               [
               <span ref-type="bibr" rid="pone.0117849.ref012">
                  <sup>
                     <a href="#pone.0117849.ref012">12</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref044">
                  <sup>
                     <a href="#pone.0117849.ref044">44</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref045">
                  <sup>
                     <a href="#pone.0117849.ref045">45</a>
                  </sup>
               </span>]. Although this provides only a single calibration point in the evolutionary history,
               it allows us to position other relative divergence times (
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span>). In this tree, viruses included in the highly divergent ISFV groups were removed
               to avoid the extrapolation of divergence times too far into the past, based on a relatively
               recent calibration.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In 
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span>, the black nodes and branches indicate that the ancestral viral lineage was inferred
               to have existed in the Old World (OW) as opposed to the red nodes and branches for
               the New World (NW) virus lineages. Mean divergence time estimates and 95% credibility
               intervals (translucent light blue bars) are shown for each node in the tree. The phylogenetic
               estimate identifies, with high confidence, 11 independent nodes that represent introductions
               of mosquito-borne virus lineages (identified as A to E, F1–5 and G1–2) from the OW
               to the NW (
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span>). 
               <span ref-type="table" rid="pone.0117849.t001">
                  <a href="#pone.0117849.t001">Table 1</a>
               </span> summarizes the credible intervals for each node (or 95% HPD interval) and the median
               estimate.
            </p>
            <div id="pone.0117849.t001">
               <div tagxxx="object-id">10.1371/journal.pone.0117849.t001</div>
               <span class="label">Table 1</span>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <colgroup>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                     </colgroup>
                     <thead>
                        <tr>
                           <th>Virus</th>
                           <th>Interval low</th>
                           <th>Nodes of divergence with the common ancestor</th>
                           <th>Interval high</th>
                           <th>
                              <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span> nodes
                           </th>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>WNV NY99 and KUNV</td>
                           <td>137</td>
                           <td>277</td>
                           <td>475</td>
                           <td>
                              <b>A</b>
                           </td>
                        </tr>
                        <tr>
                           <td>CPCV</td>
                           <td>3217</td>
                           <td>5239</td>
                           <td>7453</td>
                           <td>
                              <b>B</b>
                           </td>
                        </tr>
                        <tr>
                           <td>SLEV</td>
                           <td>4356</td>
                           <td>6928</td>
                           <td>9806</td>
                           <td>
                              <b>C</b>
                           </td>
                        </tr>
                        <tr>
                           <td>ILHV group</td>
                           <td>4107</td>
                           <td>6801</td>
                           <td>9496</td>
                           <td>
                              <b>D</b>
                           </td>
                        </tr>
                        <tr>
                           <td>AROAV group</td>
                           <td>5703</td>
                           <td>9351</td>
                           <td>13013</td>
                           <td>
                              <b>E</b>
                           </td>
                        </tr>
                        <tr>
                           <td>DENV-1 gen3 NW / OW</td>
                           <td>157</td>
                           <td>321</td>
                           <td>515</td>
                           <td>
                              <b>F1</b>
                           </td>
                        </tr>
                        <tr>
                           <td>DENV-3 gen3 NW / OW</td>
                           <td>89</td>
                           <td>186</td>
                           <td>311</td>
                           <td>
                              <b>F2</b>
                           </td>
                        </tr>
                        <tr>
                           <td>DENV-2 gen As-Am NW / OW</td>
                           <td>74</td>
                           <td>148</td>
                           <td>250</td>
                           <td>
                              <b>F3</b>
                           </td>
                        </tr>
                        <tr>
                           <td>DENV-2 gen Am NW / OW</td>
                           <td>245</td>
                           <td>450</td>
                           <td>664</td>
                           <td>
                              <b>F4</b>
                           </td>
                        </tr>
                        <tr>
                           <td>DENV-4 gen2 NW / OW</td>
                           <td>115</td>
                           <td>225</td>
                           <td>377</td>
                           <td>
                              <b>F5</b>
                           </td>
                        </tr>
                        <tr>
                           <td>YFV split OW / NW</td>
                           <td>281</td>
                           <td>385</td>
                           <td>450</td>
                           <td>
                              <b>G1</b>
                           </td>
                        </tr>
                        <tr>
                           <td>YFV NW</td>
                           <td>150</td>
                           <td>199</td>
                           <td>277</td>
                           <td>
                              <b>G2</b>
                           </td>
                        </tr>
                        <tr>
                           <td>DTV and POWV</td>
                           <td>308</td>
                           <td>601</td>
                           <td>1025</td>
                           <td>
                              <b>H</b>
                           </td>
                        </tr>
                        <tr>
                           <td>FE and Sib TBEV group and Europe TBEV group/LIV</td>
                           <td>649</td>
                           <td>1087</td>
                           <td>1610</td>
                           <td>
                              <b>I</b>
                           </td>
                        </tr>
                        <tr>
                           <td>GGEV/TSEV group and rest of Europe TBEV/LIV</td>
                           <td>465</td>
                           <td>783</td>
                           <td>1133</td>
                           <td>
                              <b>J</b>
                           </td>
                        </tr>
                        <tr>
                           <td>Europe TBEV and LIV</td>
                           <td>328</td>
                           <td>572</td>
                           <td>844</td>
                           <td>
                              <b>K</b>
                           </td>
                        </tr>
                        <tr>
                           <td>LIV and SSEV</td>
                           <td>195</td>
                           <td>369</td>
                           <td>569</td>
                           <td>
                              <b>L</b>
                           </td>
                        </tr>
                        <tr>
                           <td>TBEV Neud and TBEV Hypr</td>
                           <td>57</td>
                           <td>141</td>
                           <td>254</td>
                           <td>
                              <b>M</b>
                           </td>
                        </tr>
                        <tr>
                           <td>Bat's NKV split NW /OW</td>
                           <td>4370</td>
                           <td>7421</td>
                           <td>10414</td>
                           <td>
                              <b>N</b>
                           </td>
                        </tr>
                        <tr>
                           <td>Rodent's NKV split OW /NW</td>
                           <td>6065</td>
                           <td>9580</td>
                           <td>13750</td>
                           <td>
                              <b>O</b>
                           </td>
                        </tr>
                     </div>
                  </table>
               </div>
            </div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Estimated times of divergence that belong to the “current time”, i.e., the past 500
               hundred years were authenticated against historical facts relating to the slave trade
               and introduction of yellow fever virus into the Americas [
               <span ref-type="bibr" rid="pone.0117849.ref078">
                  <sup>
                     <a href="#pone.0117849.ref078">78</a>
                  </sup>
               </span>].
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Similar estimates for divergence between OW ancestral DENV and NW strains (150 to
               450 years ago) are also consistent with the concept of human introduction of these
               viruses into the NW via the Slave trade and other commercial exchanges (nodes F1 to
               F5—
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span>).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The divergence between Far eastern/Siberian TBEV and European TBEV is estimated to
               have occurred 1087 [1610–649] years ago (node I—
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span>)[
               <span ref-type="bibr" rid="pone.0117849.ref079">
                  <sup>
                     <a href="#pone.0117849.ref079">79</a>
                  </sup>
               </span>] and, the divergence between European TBEV and LIV/SSEV occurred about 572 [844–328]
               years ago (node K—
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span>). The estimates for LIV are consistent with historical records of the dispersal patterns
               of this virus in the British Isles [
               <span ref-type="bibr" rid="pone.0117849.ref080">
                  <sup>
                     <a href="#pone.0117849.ref080">80</a>
                  </sup>
               </span>].
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The first outbreak of WNV in North America occurred in August 1999 in New York (
               <div tagxxx="ext-link">http://www.cdc.gov/mmwr/preview/mmwrhtml/mm4838a1.htm</div>). The virus was most likely introduced from North Africa [
               <span ref-type="bibr" rid="pone.0117849.ref081">
                  <sup>
                     <a href="#pone.0117849.ref081">81</a>
                  </sup>
               </span>]. In the OW, the eastward dispersal of WNV, probably via migratory birds and/or shipping,
               seems to have occurred many years earlier leading to the appearance of Kunjin virus
               (KUNV) in Australia. 
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span> (node A) shows that KUNV diverged from WNV approximately 277 [475–137] years ago.
               This corresponds quite closely with the dates of the early emigrants from the British
               Isles to Australia. The shipping routes included stopovers in Africa, during which
               KUNV, or an ancestral lineage, could have gained access to the ships via infected
               mosquitoes. The virus could then have been carried from Africa to Australia in these
               mosquitoes on the ships that transported the British emigrants.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In the Old World, Powassan virus (POWV) is found in Russia [
               <span ref-type="bibr" rid="pone.0117849.ref082">
                  <sup>
                     <a href="#pone.0117849.ref082">82</a>
                  </sup>
               </span>] and since the ancestral lineages of this virus are also OW viruses, it is reasonable
               to assume that POWV originated in the OW [
               <span ref-type="bibr" rid="pone.0117849.ref012">
                  <sup>
                     <a href="#pone.0117849.ref012">12</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref083">
                  <sup>
                     <a href="#pone.0117849.ref083">83</a>
                  </sup>
               </span>]. However, POWV is also found in the NW in Canada and the United States of America,
               although no other related tick-borne encephalitic viruses have been found in the New
               World. The ancestral lineage of POWV in the NW has diverged to produce 2 lineages
               [
               <span ref-type="bibr" rid="pone.0117849.ref079">
                  <sup>
                     <a href="#pone.0117849.ref079">79</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref084">
                  <sup>
                     <a href="#pone.0117849.ref084">84</a>
                  </sup>
               </span>] namely the current POWV lineage and a variant that adapted to deer ticks, currently
               referred to as deer tick virus (DTV). This is depicted in 
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span> as node H, where the ancestral lineage is displayed in black on the assumption of
               an OW origin.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">There has been considerable discussion concerning the significance of these observations
               [
               <span ref-type="bibr" rid="pone.0117849.ref079">
                  <sup>
                     <a href="#pone.0117849.ref079">79</a>
                  </sup>
               </span>] and the hypothesis that POW was introduced into North America using the Beringian
               land bridge that connected Asia and North America between 15,000–11,000 years ago
               has recently been used to estimate modelling the temporal origin and evolution of
               flaviviruses (see the 
               <span ref-type="sec" rid="sec028">discussion</span> section)[
               <span ref-type="bibr" rid="pone.0117849.ref085">
                  <sup>
                     <a href="#pone.0117849.ref085">85</a>
                  </sup>
               </span>].
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Based on the phylogenetic and phylogeographic evidence, we deduce that mosquito-borne
               flaviviruses, corresponding to those identified at nodes A to E, F1–5 and G1–2 (
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span>), originated in the OW and were introduced into the New World 11 times. This does
               not preclude the possibility that in some cases ancestral African lineages may have
               dispersed to the NW before emerging as the currently recognised viruses. According
               to the TMCRA, these 11 events can be divided into two categories, those that have
               an estimated TMRCA that overlaps with the period of commercial trading by ships crossing
               the Atlantic Ocean, approximately during the past 500 years (YFV, node G1–2; DENV
               1–4, node F1–5; WNV, node A—
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span>) and those for which the estimated TMRCA pre-dates this period of trading (AROAV
               group, node E; ILHV group, node D; SLEV, node C; CPCV, node B—
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span>).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">On the other hand, the TMRCA estimates for the other 6 NW lineages suggest that these
               viruses potentially could have been introduced to the NW more than one thousand years
               before the initiation of historic commercial and slave trading across the Atlantic
               Ocean (AROAV group (5700–13000; 
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span> node E); ILHV group (4100–9500; 
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span> node D); SLEV (4400–9800; 
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span> node C); CPCV (3200–7500; 
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span> node B); NW bat NKV
               <sup>SL</sup> (4400–10400; node N) and NW rodent NKV
               <sup>SL</sup> (6100–13800; node O). These estimate are less accurate, due to the relatively short
               term calibration date we have used for YFV as reference, and could be thousands or
               even more years in the past. All apparent virus migrations during this period are
               referred to as having occurred “before the slave trade”.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Node E of 
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span> displays a TMRCA of 9400 years, which represents the common ancestor of the NW Aroa
               virus (AROAV) and Kokobera virus (KOKV) groups, i.e. groups of viruses that dispersed
               respectively westward to the NW and eastward to Australia, in the OW.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A similar pattern was observed at node D (TMRCA 6800 years before the present). The
               ILHV group emerged presumably in Africa and dispersed westward to the NW. On the other
               hand, the BAGV group remained in Africa, eventually emerging in Europe and the Middle
               East, whereas the TMUV related ancestral lineages emerged and dispersed eastward into
               Asia (node D—
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span>).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">On the other hand, the JEV/USUV/ALFV/MVEV group shares a common ancestor with a TMRCA
               of about 2000–3000 years before the present, implying that an ancestral lineage dispersed
               eastwards out of Africa and appears to have dispersed to south East Asia before emerging
               as JEV and then dispersing widely, throughout Asia [
               <span ref-type="bibr" rid="pone.0117849.ref086">
                  <sup>
                     <a href="#pone.0117849.ref086">86</a>
                  </sup>
               </span>].
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Nodes N and O in 
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span> identify the no known vector NKV
               <sup>SL</sup> viruses. In common with the arboviruses, there are distinct lineages in the OW and
               NW (coloured black and red respectively). The TMRCA of the OW NKV, Apoi virus (APOIV),
               which was isolated from rodents in Japan, pre-dates all the other recognised NKV
               <sup>SL</sup>, and the MRCA of this lineage diverged to produce descendant NKV and arthropod-borne
               virus lineages. This leaves open the possibility that the association of flaviviruses
               with tick and mosquito vectors may have been an acquired trait from a non-vectored
               ancestral virus [
               <span ref-type="bibr" rid="pone.0117849.ref025">
                  <sup>
                     <a href="#pone.0117849.ref025">25</a>
                  </sup>
               </span>]. Nodes N and O indicate that the NKVs were introduced to the NW on at least two
               independent occasions. Based on the TMRCA predictions, the NKV diverged over a period
               of 4000 to 14,000 years ago. It has been suggested (Varelas-Wesley &amp; Calisher, 1982)
               that these NKVs could have been introduced into the Americas during the Miocene/Pliocene
               period, possibly by migrating rodents and/or bats. This idea is consistent with the
               TMRCA predictions presented in 
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span>. Nevertheless, the alternative possibility that these NW viruses emerged in the OW
               and were introduced more recently via rodents into the NW, following the development
               of trading via ships between the OW and the NW should not be ruled out. Two African
               viruses Dakar Bat and Bukulasa bat virus are not included in this analysis as only
               partial sequences for these viruses are currently available.
            </p>
         </div>
         <div id="sec027">
            <h3>A search for sequence elements associated with ribosomal-1 frameshifting</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Many viruses harbour sequences that induce a proportion of translating ribosomes to
               shift-1 nt and continue translating in the new reading frame to produce a 'transframe'
               fusion protein [
               <span ref-type="bibr" rid="pone.0117849.ref087">
                  <sup>
                     <a href="#pone.0117849.ref087">87</a>
                  </sup>
               </span>]. Where functionally utilized, this is referred to as programmed-1 ribosomal frameshifting
               (-1 PRF). The eukaryotic-1 frameshift site typically consists of a 'slippery' heptanucleotide
               sequence fitting the consensus motif X_XXY_YYZ, where XXX represents any three identical
               nucleotides; YYY represents AAA or UUU; Z represents A, C or U; and underscores separate
               zero-frame codons. In the tandem slippage model, the P-site anticodon re-pairs from
               XXY to XXX, whereas the A-site anticodon re-pairs from YYZ to YYY, thus allowing for
               perfect re-pairing except at the wobble position. Certain deviations from the canonical
               XXX of the slippery site are tolerated in the P-site, including UCC in some members
               of the JEV serogroup, GGU in cardioviruses and some luteoviruses, GUU in equine arteritis
               virus (family 
               <i>Arteriviridae</i>), and GGA in many ISFVs besides insect nidoviruses of the family 
               <i>Mesoniviridae</i> and some umbraviruses and dianthoviruses. The efficiency of frameshifting depends
               on the identity of the slippery site nucleotides but is typically less than 1% in
               the absence of additional stimulatory elements. Thus, most known instances of eukaryotic-1
               frameshifting are stimulated (typically to a level between 1% and 50%) by the presence
               of a 3' stable RNA secondary structure, such as a pseudoknot or stem-loop, that is
               separated from the slippery heptanucleotide by a 'spacer' region of 5–9 nt.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Sequence elements associated with-1 PRF were previously described in the JEV serogroup
               viruses for JEV, WNV, USUV, MVEV and ALFV [
               <span ref-type="bibr" rid="pone.0117849.ref088">
                  <sup>
                     <a href="#pone.0117849.ref088">88</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref089">
                  <sup>
                     <a href="#pone.0117849.ref089">89</a>
                  </sup>
               </span>](
               <span ref-type="fig" rid="pone.0117849.g003">Fig. 3a</span>). In these viruses, -1 PRF occurs (with an estimated efficiency of 20–50%) when ribosomes
               are positioned on the codons encoding the 8th and 9th amino acids of NS2A. When PRF
               occurs, ribosomes translate a 43-codon ORF in the-1 reading frame relative to the
               polyprotein ORF and then terminate. The resulting 52 amino acid 'transframe' polypeptide
               is not cleaved at the NS1|NS2A cleavage site and thus frameshifting results in the
               production of a C-terminally extended version of NS1, known as NS1'. Our study extends
               the range of viruses that possess this frameshift site to include CPCV, YAOV and KOUV.
               The overlapping ORF displayed a constant length (43 codons) for all viruses within
               the JEV serogroup, with the exception of SLEV which lacks the frameshift site.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pone.0117849.g003</div>
               <span class="label">Fig 3</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Frameshift stimulatory elements in viruses of the JEV serogroup.</div>
                  <p>(a) Previously identified frameshift site (Y_CCU_UUU; Y = C or U; orange) and 3'-adjacent
                     stable pseudoknot structure responsible for stimulating-1 PRF in the NS2A-encoding
                     region of JEV and related flaviviruses. Stems 1 and 2 of the pseudoknot are indicated
                     in blue and red respectively. (b) The shift site and pseudoknot are preserved in the
                     newly sequenced KOUV but not in YAOV or CPCV. Substitutions that preserve the base-pairings
                     in stem 1 (blue) or stem 2 (red) of the pseudoknot are indicated in pale blue and
                     orange respectively. In YAOV, a simple stem-loop (brown) was predicted at an appropriate
                     spacing from the shift site to act as a stimulator of-1 PRF. CPCV maintains the shift
                     site but multiple possible 3'-proximal structures (not shown) could be predicted.
                     SLEV sequences lack a suitable shift site at this genomic location. (c) Predicted
                     frameshift stimulatory elements (shift site and 3'-adjacent stem-loop) in YAOV.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A nucleotide alignment of the putative stimulatory elements responsible for the-1
               PRF is presented in 
               <span ref-type="fig" rid="pone.0117849.g003">Fig. 3b</span>. The slippery heptanucleotide Y_CCU_UUU (highlighted in orange on 
               <span ref-type="fig" rid="pone.0117849.g003">Fig. 3b</span>) is absolutely conserved for CPCV, YAOV and KOUV as with other members of the JEV
               group except for SLEV. Predicted 3' RNA stimulatory elements are highlighted in 
               <span ref-type="fig" rid="pone.0117849.g003">Fig. 3b</span>. KOUV retains the canonical stable 3' pseudoknot structure that stimulates-1 PRF
               in other JEV serogroup flaviviruses (
               <span ref-type="fig" rid="pone.0117849.g003">Fig. 3a</span>) [
               <span ref-type="bibr" rid="pone.0117849.ref089">
                  <sup>
                     <a href="#pone.0117849.ref089">89</a>
                  </sup>
               </span>]. In YAOV, however, a stem-loop structure was predicted instead of a pseudoknot (
               <span ref-type="fig" rid="pone.0117849.g003">Fig. 3c</span>). The CPCV sequence also lacks the potential to form the canonical JEV serogroup
               pseudoknot. In this case, however, several possible alternative structures could be
               predicted and, in the absence of experimental analysis or comparative genomic inference,
               it remains unclear which structure (if any) might be functionally relevant for CPCV.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Analysis of nucleotide conservation at synonymous sites within alignments of related
               flavivirus sequences has previously been successful in identifying additional coding
               ORFs overlapping internal regions of the polyprotein ORF and accessed via-1 PRF [
               <span ref-type="bibr" rid="pone.0117849.ref088">
                  <sup>
                     <a href="#pone.0117849.ref088">88</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref090">
                  <sup>
                     <a href="#pone.0117849.ref090">90</a>
                  </sup>
               </span>]. Due to the sequence constraints imposed by simultaneous coding in two overlapping
               reading frames, besides maintaining functional frameshift-stimulatory elements, such
               sites (if functionally important and phylogenetically conserved) are associated with
               greatly increased nucleotide conservation at synonymous sites in the polyprotein reading
               frame relative to the genome average. To investigate the potential presence of frameshifting
               in other flaviviruses, we constructed sequence alignments of selected flavivirus clades,
               and analyzed conservation at synonymous sites as described previously [
               <span ref-type="bibr" rid="pone.0117849.ref039">
                  <sup>
                     <a href="#pone.0117849.ref039">39</a>
                  </sup>
               </span>]. A selection of these analyses is presented in Figs. 
               <span ref-type="fig" rid="pone.0117849.g004">4</span> and 
               <span ref-type="fig" rid="pone.0117849.g005">5</span>.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pone.0117849.g004</div>
               <span class="label">Fig 4</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Synonymous site conservation analysis for selected flavivirus clades.</div>
                  <p>Alignments of 249 JEV serogroup, 49 NTAV/TMUV clade, 89 DENV, 6 DGV/LAMV clade and
                     29 CxFV/QBV clade polyprotein ORF sequences were analyzed for synonymous site variability
                     as decribed previously (Firth et al., 2011 PMID 21525127). The accession numbers of
                     all sequences used in the analysis are available on request. Red lines indicate the
                     probability (
                     <i>p</i>-value) of obtaining not more than the observed number of synonymous substitutions,
                     in a 25-codon sliding window, under a null model of neutral evolution at synonymous
                     sites. Dashed grey lines indicate an approximate 5% false positive threshold after
                     correcting for multiple tests (i.e. ~136 x 25-codon windows in the ~3400-codon polyprotein
                     ORF). Statistically significant peaks in synonymous site conservation are indicative
                     of overlapping functional elements, either coding or non-coding. Genome maps are shown
                     for each clade. UTR lengths may be uncertain for less well-studied clades. Known and
                     predicted overlapping ORFs accessed via-1 PRF are shown in pink. The predicted overlapping
                     ORF in the DGV/LAMV clade is much shorter in DGV than in other members of the clade;
                     the long form of the ORF is indicated. Note that 
                     <i>p</i>-values can not be directly compared between different clades because the statistical
                     significance (i.e. p-value) of observed reductions in synonymous site variabilty depends
                     on the diversity of the specific sequence alignment being analyzed.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pone.0117849.g005</div>
               <span class="label">Fig 5</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Synonymous site conservation analysis for selected flavivirus clades.</div>
                  <p>Alignments of 6 UGSV/BOUV clade, 6 WESSV/SEPV clade, 56 YFV, and 144 POWV/TBEV clade
                     polyprotein ORF sequences were analyzed for synonymous site variability as decribed
                     previously (Firth et al., 2011 PMID 21525127). The accession numbers of all sequences
                     used in the analysis are available on request. Red lines indicate the probability
                     (
                     <i>p</i>-value) of obtaining not more than the observed number of synonymous substitutions,
                     in a 25-codon sliding window, under a null model of neutral evolution at synonymous
                     sites. Dashed grey lines indicate an approximate 5% false positive threshold after
                     correcting for multiple tests (i.e. ~136 x 25-codon windows in the ~3400-codon polyprotein
                     ORF). Statistically significant peaks in synonymous site conservation are indicative
                     of overlapping functional elements, either coding or non-coding. Genome maps are shown
                     for each clade. UTR lengths may be uncertain for less well-studied clades and are
                     omitted for the UGSV/BOUV clade due to lack of sequence data. Known and predicted
                     overlapping ORFs accessed via-1 PRF are shown in pink. Note that 
                     <i>p</i>-values can not be directly compared between different clades because the statistical
                     significance (i.e. p-value) of observed reductions in synonymous site variabilty depends
                     on the diversity of the specific sequence alignment being analyzed.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Most flaviviruses exhibit enhanced synonymous site conservation at the 5' end of the
               polyprotein ORF. This is presumably associated with non-coding elements such as functional
               RNA structures involved in replication and/or translation enhancement [
               <span ref-type="bibr" rid="pone.0117849.ref091">
                  <sup>
                     <a href="#pone.0117849.ref091">91</a>
                  </sup>
               </span>–
               <span ref-type="bibr" rid="pone.0117849.ref096">
                  <sup>
                     <a href="#pone.0117849.ref096">96</a>
                  </sup>
               </span>]. Several flavivirus clades also exhibit localized regions of statistically significantly
               enhanced synonymous site conservation in internal regions of the polyprotein ORF,
               notably the JEV serogroup, the ISFVs, the CHAOV-LAMV-DGV-ILOV clade (
               <span ref-type="fig" rid="pone.0117849.g004">Fig. 4</span>), and the WESSV-SEPV clade (
               <span ref-type="fig" rid="pone.0117849.g005">Fig. 5</span>).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In the ISFVs (CxFV-QBV clade shown in 
               <span ref-type="fig" rid="pone.0117849.g004">Fig. 4</span>), enhanced synonymous site conservation is apparent in the NS2A/2B-encoding region;
               this corresponds to a long overlapping ORF that is accessed via-1 PRF [
               <span ref-type="bibr" rid="pone.0117849.ref090">
                  <sup>
                     <a href="#pone.0117849.ref090">90</a>
                  </sup>
               </span>]. Many more ISFV sequences are now available, and the overlapping ORF is conserved
               in all except for the cell culture adapted original isolate of CFAV (GenBank acc.
               M91671) [
               <span ref-type="bibr" rid="pone.0117849.ref023">
                  <sup>
                     <a href="#pone.0117849.ref023">23</a>
                  </sup>
               </span>]. The overlapping ORF ranges from 270 to 293 codons in CxFV, QBV, PCV, NAKV and 
               <i>Cx theileri</i> flavivirus, and 253 to 257 codons in KRV, CFAV, HANKV, AEFV and 
               <i>Ochlerotatus caspius</i> flavivirus. The frameshift site generally appears to be G_GAU_UUY (Y = U or C), with
               notable exceptions appearing to be G_UUU_UUU in NAKV and A_AAU_UUU_UUC (potential
               tandem shift sites) in PCV.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In the CHAOV-LAMV-DGV-ILOV clade enhanced synonymous site conservation is apparent
               in the region encoding NS2B. This is associated with a conserved G_GAU_UUU slippery
               heptanucleotide and a 3'-adjacent predicted stem-loop structure that stimulates-1
               PRF in dual reporter assays and whose functionality is supported by compensatory substitutions
               (i.e. paired substitutions that preserve the predicted base-pairings) [
               <span ref-type="bibr" rid="pone.0117849.ref052">
                  <sup>
                     <a href="#pone.0117849.ref052">52</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref090">
                  <sup>
                     <a href="#pone.0117849.ref090">90</a>
                  </sup>
               </span>]. Here, the putative frameshift ORF has variable length (107 codons in LAMV and CHAOV,
               71 codons in ILOV, but only 6 codons in DGV)[
               <span ref-type="bibr" rid="pone.0117849.ref052">
                  <sup>
                     <a href="#pone.0117849.ref052">52</a>
                  </sup>
               </span>], perhaps explaining why the conservation peak does not extend throughout the 107-codon
               ORF annotated on the genome map in 
               <span ref-type="fig" rid="pone.0117849.g004">Fig. 4</span>.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A dramatic and statistically significant peak in synonymous site conservation in an
               internal region was observed in one other clade of flaviviruses—the WESSV-SEPV clade
               (
               <span ref-type="fig" rid="pone.0117849.g005">Fig. 5</span>). This could represent an overlapping non-coding RNA element or yet another-1 PRF
               site. Inspection of the sequences corresponding to the conservation peak revealed
               a conserved slippery heptanucleotide, G_GUU_UUU [the same shift site that is utilized
               for-1 PRF in cardioviruses and some species of luteovirus, besides both-1 and-2 PRF
               in porcine reproductive and respiratory syndrome arterivirus [
               <span ref-type="bibr" rid="pone.0117849.ref097">
                  <sup>
                     <a href="#pone.0117849.ref097">97</a>
                  </sup>
               </span>–
               <span ref-type="bibr" rid="pone.0117849.ref099">
                  <sup>
                     <a href="#pone.0117849.ref099">99</a>
                  </sup>
               </span>]], and the potential for a 3'-adjacent stem-loop to form at the appropriate spacing
               potentially to act as a stimulator of-1 PRF (
               <span ref-type="fig" rid="pone.0117849.g006">Fig. 6</span>). In this case the overlapping ORF has just 7 codons, with the termination codon
               embedded within the predicted stem-loop structure. Frameshifting here would result
               in a greatly truncated version of the NS2A protein with a distinct 7 amino acid C-terminal
               end encoded by the short overlapping ORF.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pone.0117849.g006</div>
               <span class="label">Fig 6</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Predicted frameshift stimulatory elements in WESSV and SEPV.</div>
                  <p>Frameshifting is predicted to occur on a conserved G_GUU_UUU heptanucleotide (orange)
                     in the NS2A-encoding region, stimulated by a 3'-adjacent stem-loop structure. The-1
                     frame stop codon is indicated in red. The WESSV RefSeq NC_012735 is shown; nucleotide
                     differences in the SEPV RefSeq NC_008719 are indicated in blue.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A statistically significant peak in synonymous site conservation was also observed
               in the NS3-encoding region of TBEV and related flaviviruses (
               <span ref-type="fig" rid="pone.0117849.g005">Fig. 5</span>). However this was not thought to represent a PRF site, first, because the degree
               of conservation was relatively modest compared to what is typically observed at a
               PRF site, and, second, because we were unable to identify a canonical PRF-compatible
               shift site at this genomic location in these viruses. Instead, it appears to correspond
               to a small conserved RNA stem-loop structure.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Internal frameshift sites and/or overlapping ORFs were not predicted using these methods
               for other flavivirus groups such as SLEV, YFV and DENV. It should be noted, however,
               that the synonymous site conservation analysis would not necessarily detect cases
               of frameshifting where there is not an overlapping ORF subject to purifying selection
               and where the frameshift stimulatory elements comprise just a few codons (e.g. primary
               sequence rather than secondary structure stimulators) and/or are located in the nascent
               peptide rather than the RNA sequence. Also these analyses do not yet provide statistically
               useful results for all flavivirus clades (e.g. where there is insufficient sequence
               divergence within a clade and too much sequence divergence between phylogenetically
               adjacent clades to observe the effects of purifying selection at synonymous sites).
            </p>
         </div>
      </div>
      <div id="sec028">
         <h2>DISCUSSION</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">As phylogenetic methods have developed and more complete genome (ORF) sequence data
            have become available, increasingly robust analyses and interpretations of the flaviviruses
            have become possible. For example, we now know that the African virus, Kadam virus
            (KADV), sits at the root of the TBEV lineages and KADV diverged from a common ancestor
            of the seabird tick-associated viruses (
            <span ref-type="fig" rid="pone.0117849.g001">Fig. 1</span>).
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Our phylogeny strongly supports the NS3 branching pattern of divergence defined previously
            [
            <span ref-type="bibr" rid="pone.0117849.ref010">
               <sup>
                  <a href="#pone.0117849.ref010">10</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pone.0117849.ref011">
               <sup>
                  <a href="#pone.0117849.ref011">11</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pone.0117849.ref025">
               <sup>
                  <a href="#pone.0117849.ref025">25</a>
               </sup>
            </span>] in which the TBFVs and NKVs share a common ancestor. The analysis confirms previous
            suggestions that the 
            <i>Culex</i>-associated flaviviruses evolved from ancestral 
            <i>Aedes</i>-associated viruses [
            <span ref-type="bibr" rid="pone.0117849.ref008">
               <sup>
                  <a href="#pone.0117849.ref008">8</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pone.0117849.ref012">
               <sup>
                  <a href="#pone.0117849.ref012">12</a>
               </sup>
            </span>]. However, the recent discovery of several new viruses included in the ISFV-like
            group, now reveals the possibility of two potentially new virus groups, viz., the
            LAMV group and the NOUV group. They are geographically very widely dispersed but the
            mechanisms for their dispersal are unknown. Currently exclusively OW viruses found
            in Africa, Europe or Asia have been characterised. However, with the addition of the
            newly isolated Nanay virus in Peru [
            <span ref-type="bibr" rid="pone.0117849.ref059">
               <sup>
                  <a href="#pone.0117849.ref059">59</a>
               </sup>
            </span>], which is closely related to NOUV, the ISFV-like group now seems to extend to the
            NW and this will need to be re-visited when genomic sequences of NW isolates become
            available.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">These recently-discovered apparently non-pathogenic mosquito-associated viruses (LAMV
            and NOUV group) are phylogenetically closely-related to human pathogenic viruses,
            such as DENV, YFV, WNV, JEV. However, they do not appear to replicate in vertebrate
            cells [
            <span ref-type="bibr" rid="pone.0117849.ref013">
               <sup>
                  <a href="#pone.0117849.ref013">13</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pone.0117849.ref052">
               <sup>
                  <a href="#pone.0117849.ref052">52</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pone.0117849.ref057">
               <sup>
                  <a href="#pone.0117849.ref057">57</a>
               </sup>
            </span>] a property which to date has precluded the possibility of isolating such viruses
            from field material by intracerebral inoculation of newborn mice. Possibly, they represent
            independent lineages of viruses that, like the NKV, have lost a particular genetic
            trait not required for their long-term survival.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Within the MBFVs, three NKV-like viruses, ENTV, SOKV and YOKV, have not been associated
            with any known vector. Nevertheless, they are genetically closely related to the YFV
            and EHV groups, have only been found in the Old World, and are associated with migratory
            bats. YOKV and SOKV were both isolated in Asia but appear to have their ancestral
            roots in Africa, as do many of the related viruses in the EHV group. Thus, flaviviruses
            seem able to lose or acquire a variety of genetic traits as illustrated by acquisition
            of the requirement for a tick or mosquito vector and therefore becoming arboviruses,
            loss of ability to infect vertebrate cells, exemplified by LAMV and NOUV, loss of
            the requirement for vector transmission illustrated by the ENTV group and originally
            proposed by Mattingly in 1960 [
            <span ref-type="bibr" rid="pone.0117849.ref100">
               <sup>
                  <a href="#pone.0117849.ref100">100</a>
               </sup>
            </span>] and supported by Kuno and Chang [
            <span ref-type="bibr" rid="pone.0117849.ref101">
               <sup>
                  <a href="#pone.0117849.ref101">101</a>
               </sup>
            </span>], inability to infect vertebrate cells, as illustrated by the ISFVs and acquisition
            of a frameshift as typified by the JEV serogroup viruses (discussed in detail later).
         </p>
         <div id="sec029">
            <h3>No known vector flaviviruses specific lineage (NKV
               <sup xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">SL</sup>)
            </h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In contrast with the MBFV and TBFV groups, the NKV
               <sup>SL</sup> group remains poorly described. Based on the phylogenetic and TMRCA data, viruses
               within this group have been introduced to the NW on at least two separate occasions
               and these introductions may have occurred thousands of years ago. Estimations for
               the time of divergence between these OW and NW NKV
               <sup>SL</sup> ranged between 4000 to 14000 years ago (
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span> nodes N and O). Despite the recognised large errors in TMRCA estimates, they all
               pre-date, by thousands of years, the recognised commercial trading period across the
               Atlantic Ocean that commenced 400–500 years ago. Therefore, viruses could have dispersed
               to the NW from the OW thousands of years before transoceanic trading was taking place.
               The possible mechanisms of dispersal remain a mystery. It is now established that
               OW human populations (presumably of both Asian and European origin) became established
               in the Americas during the period of TMRCA estimates, and possibly even before these
               estimated times. The question arises, during such migrations is it likely that infected
               animals could have been transported (
               <i>e</i>.
               <i>g</i>., rodents). The arrival of viruses in the NW could alternatively have occurred via
               the gradual dispersal of OW rodents and migratory bats. On the other hand, these transoceanic
               dispersions might have occurred after the development of trading between the OW and
               the NW.
            </p>
         </div>
         <div id="sec030">
            <h3>Insect-specific flaviviruses (ISFVs)</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Until about ten years ago, only one ISFV, cell fusing agent virus (CFAV), was recognised.
               Currently, at least 9 genetically distinguishable ISFVs have been isolated and their
               sequences determined. This group of mosquito- and potentially sandfly-borne viruses
               is an extremely genetically diverse group with a divergence of up to 61% at the amino
               acid level. This compares with the genetic distances between the three current genera
               in the family 
               <i>Flaviviridae</i>. Moreover, they infect only invertebrate hosts and DNA forms of ISFV genomes are
               generated during infection of cell cultures [
               <span ref-type="bibr" rid="pone.0117849.ref026">
                  <sup>
                     <a href="#pone.0117849.ref026">26</a>
                  </sup>
               </span>]. Integrated DNA sequences have been identified in mosquito genomes and isolated
               DNA forms have been detected in field samples [
               <span ref-type="bibr" rid="pone.0117849.ref018">
                  <sup>
                     <a href="#pone.0117849.ref018">18</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref026">
                  <sup>
                     <a href="#pone.0117849.ref026">26</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref102">
                  <sup>
                     <a href="#pone.0117849.ref102">102</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref103">
                  <sup>
                     <a href="#pone.0117849.ref103">103</a>
                  </sup>
               </span>]. Currently, there is no clear indication of the biological significance of these
               DNA forms amongst ISFVs. We therefore propose that the distinct differences of the
               ISFVs justify their inclusion as a separate genus in the family 
               <i>Flaviviridae</i>. These viruses are also interesting because similar ISFVs have been isolated from
               mosquito species that inhabit different ecological niches, raising the question how
               might this occur? Possible mechanisms of virus transfer between mosquito species include
               the diffusion of viruses between larvae at sites shared by different mosquito species,
               or biting midges could become infected when taking blood meals from infected mosquitoes
               [
               <span ref-type="bibr" rid="pone.0117849.ref104">
                  <sup>
                     <a href="#pone.0117849.ref104">104</a>
                  </sup>
               </span>–
               <span ref-type="bibr" rid="pone.0117849.ref108">
                  <sup>
                     <a href="#pone.0117849.ref108">108</a>
                  </sup>
               </span>] and they could then act as vectors of the virus if they subsequently feed on different
               mosquito species.
            </p>
         </div>
         <div id="sec031">
            <h3>A perspective on Old World to New World flavivirus introductions</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">It is recognised that YFV and DENV were transported frequently to the Americas from
               Africa on the ships that transported slaves across the Atlantic Ocean during the centuries
               of slave trading [
               <span ref-type="bibr" rid="pone.0117849.ref044">
                  <sup>
                     <a href="#pone.0117849.ref044">44</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref045">
                  <sup>
                     <a href="#pone.0117849.ref045">45</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref078">
                  <sup>
                     <a href="#pone.0117849.ref078">78</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref109">
                  <sup>
                     <a href="#pone.0117849.ref109">109</a>
                  </sup>
               </span>–
               <span ref-type="bibr" rid="pone.0117849.ref111">
                  <sup>
                     <a href="#pone.0117849.ref111">111</a>
                  </sup>
               </span>]. Based on this assumption, we estimated divergence times and ancestral OW/NW relationships
               to provide a deeper understanding of the evolution and dispersal patterns of flaviviruses.
               Since virus evolution characterised by lineage-specific and non-constant substitution
               rates (Pettersson 2014), it would be unwise to assert that a single calibration point
               can produce precise estimates for both short/recent times, and long/deep times. Not
               surprisingly, our estimates, based on a calibration point from the recent historical
               period and involving mosquito-borne viruses, differ significantly from those of Pettersson
               and Omar—whose calibration point was derived from a more distant historical time period
               and involved a tick-borne virus. An important consequence of the latter calibration
               is the hypothetical origin of mosquito-borne flavivirus diversification during the
               last glaciation period, whilst the former calibration suggests that the major part
               of mosquito-borne flavivirus diversification which involves human epidemiology, occurred
               after the end of the most recent major Ice Age [
               <span ref-type="bibr" rid="pone.0117849.ref012">
                  <sup>
                     <a href="#pone.0117849.ref012">12</a>
                  </sup>
               </span>].
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Our choice was to use an ‘independently confirmed’ historical hypothesis [
               <span ref-type="bibr" rid="pone.0117849.ref044">
                  <sup>
                     <a href="#pone.0117849.ref044">44</a>
                  </sup>
               </span>] and to exclude from analysis highly divergent viruses for which genomic and biological
               information remains scarce (insect specific viruses, Tamana bat virus, Ngoye virus,
               Mogiana tick virus. . .). Therefore, based on these assumptions our estimates are
               more accurate for the recent historical period than for the deepest nodes of the trees,
               and for mosquito-borne viruses than for tick-borne or no-known vector flaviviruses.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Our results demonstrate that such movement of arboviruses from Africa to the Americas
               is not unique to YFV and DENV. The ancestral history presented in 
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span> provides clear indications that multiple introductions of viruses have occurred from
               the OW to the NW during relatively recent millenia. All of the 
               <i>Culex</i>-associated viruses that circulate in Europe, Asia, Australia and/or the Americas,
               appear to have their evolutionary roots in Africa. Given the OW diversity of flaviviruses
               and the comparatively lower number of NW viruses, this analysis attempts to formalise
               a parsimonious argument for an ‘Out-of-Africa’ history. Unless there is a matching,
               but largely unsampled flavivirus diversity in the NW, this remains the most plausible
               interpretation of their phylogenetic distribution.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Given that we rely on a single, relatively recent calibration to estimate old divergence
               times, we do not consider our estimates to be a precise historical record of flavivirus
               evolution, but it does allow us to distinguish 2 major temporal periods during which
               flaviviruses may have been introduced from the OW into the NW. All historical and
               phylogenetic data are consistent with the concept that there was a recent period of
               400 to 500 years during which African viruses such as YFV and DENV were transported
               on slave and commercial ships across the Atlantic Ocean to the Americas, [
               <span ref-type="bibr" rid="pone.0117849.ref012">
                  <sup>
                     <a href="#pone.0117849.ref012">12</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref044">
                  <sup>
                     <a href="#pone.0117849.ref044">44</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref045">
                  <sup>
                     <a href="#pone.0117849.ref045">45</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref078">
                  <sup>
                     <a href="#pone.0117849.ref078">78</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref111">
                  <sup>
                     <a href="#pone.0117849.ref111">111</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref112">
                  <sup>
                     <a href="#pone.0117849.ref112">112</a>
                  </sup>
               </span>] which is why we base our estimates on this for the divergence time estimation. This
               “recent evolutionary period” includes viruses that have gradually emerged during the
               past few decades viruses (e.g., WNV, HIV, SARS, CoV, HepC virus, USUV, MERS CoV) and
               are still being dispersed via human movement and commercial transportation.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Based on published estimates for the times of divergence in the tree [
               <span ref-type="bibr" rid="pone.0117849.ref009">
                  <sup>
                     <a href="#pone.0117849.ref009">9</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref044">
                  <sup>
                     <a href="#pone.0117849.ref044">44</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref113">
                  <sup>
                     <a href="#pone.0117849.ref113">113</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref114">
                  <sup>
                     <a href="#pone.0117849.ref114">114</a>
                  </sup>
               </span>] it also appears that these introductions to the New World may have occurred gradually
               over the period of time corresponding to the movement of humans from Africa to the
               New World during the four or five hundred years of the Slave trade. Support for the
               concept that DENV can be included with YFV also comes from the known presence of the
               American genotype dengue viruses in South America that have African ancestral lineages.
               These viruses were presumably also transported to the Americas over the same range
               of time as the introduction of yellow fever viruses [
               <span ref-type="bibr" rid="pone.0117849.ref012">
                  <sup>
                     <a href="#pone.0117849.ref012">12</a>
                  </sup>
               </span>] and even more recently as slave trading was finally abolished [
               <span ref-type="bibr" rid="pone.0117849.ref115">
                  <sup>
                     <a href="#pone.0117849.ref115">115</a>
                  </sup>
               </span>]. It has been recorded that febrile syndromes and even haemorrhagic fever, clinically
               compatible with dengue fever, were diagnosed clinically in southern parts of North
               America over a long period of time [
               <span ref-type="bibr" rid="pone.0117849.ref111">
                  <sup>
                     <a href="#pone.0117849.ref111">111</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref116">
                  <sup>
                     <a href="#pone.0117849.ref116">116</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref117">
                  <sup>
                     <a href="#pone.0117849.ref117">117</a>
                  </sup>
               </span>]. The most recent and persuasive example of an OW introduction to the NW was the
               spectacular appearance of West Nile virus (WNV) in New York in August 1999 [
               <span ref-type="bibr" rid="pone.0117849.ref118">
                  <sup>
                     <a href="#pone.0117849.ref118">118</a>
                  </sup>
               </span>]. Since the first cases of WNV encephalitis in birds and humans were discovered in
               the area of the Bronx zoo and relatively close to a major international airport, it
               is considered possible that WNV was inadvertently introduced into North America either
               via infected birds or mosquitoes transported to the New York area by air transport
               [
               <span ref-type="bibr" rid="pone.0117849.ref012">
                  <sup>
                     <a href="#pone.0117849.ref012">12</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref065">
                  <sup>
                     <a href="#pone.0117849.ref065">65</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref119">
                  <sup>
                     <a href="#pone.0117849.ref119">119</a>
                  </sup>
               </span>].
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span> also identifies 4 examples of other flaviviruses (CPCV—node B; SLEV—node C; ILHV
               group—node C; AROAV group—node E) that have been introduced from the OW to the NW.
               However, the TMRCA for these introduced viruses predates the period of slave trading,
               by thousands of years and will be referred to as “ancient period” viruses.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">If we consider the possibility of a more ancient introduction of viruses, i.e. between
               4000 to 14000 years ago, we have to consider the possibility that ancestral viruses
               emerged in the OW and were then introduced into the NW many years before the slave
               trade period, possibly by birds, rodents, bats, arthropods and/or humans. The tree
               shows that introductions to the NW were multiple and independent. The first introduction
               is represented in 
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span>—node E for the AROAV group. Except for BSQV, these viruses have only been isolated
               from rodents. The TMRCA for this node is 5700 to 13000 years ago. An ancestral virus
               present in the OW could have been introduced into the NW via rodents or migratory
               birds and then adapted to other species such as rodents. Other predicted introductions
               are presented in 
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span> nodes-C, B and D for ILHV/ROCV, SLEV and CPCV. These viruses are bird associated
               and may therefore have been introduced via migratory birds or bats. However, if birds
               were a major cause of virus introduction into the NW, it seems surprising that viruses
               such as WNV have only been successfully introduced once, as suggested by the numerous
               phylogenetic analyses of many North America WNV isolates [
               <span ref-type="bibr" rid="pone.0117849.ref081">
                  <sup>
                     <a href="#pone.0117849.ref081">81</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref120">
                  <sup>
                     <a href="#pone.0117849.ref120">120</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref121">
                  <sup>
                     <a href="#pone.0117849.ref121">121</a>
                  </sup>
               </span>]. Additionally, Nanay virus isolated in 
               <i>Culex</i> mosquitoes in Peru might represent another example of potential virus introduction
               from the OW to the NW.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Recent discoveries of viruses in the ISF group such as Culex flavivirus [
               <span ref-type="bibr" rid="pone.0117849.ref020">
                  <sup>
                     <a href="#pone.0117849.ref020">20</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref021">
                  <sup>
                     <a href="#pone.0117849.ref021">21</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref122">
                  <sup>
                     <a href="#pone.0117849.ref122">122</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref123">
                  <sup>
                     <a href="#pone.0117849.ref123">123</a>
                  </sup>
               </span>] show that 
               <i>Culex</i> spp.-associated viruses contain members from both the OW and the NW. It has been
               proposed that in the past, several different insect-specific flaviviruses have been
               introduced independently into Latin America, from the OW, rather than a single virus
               having been introduced with subsequent divergence to generate the different viruses
               found in the Americas [
               <span ref-type="bibr" rid="pone.0117849.ref049">
                  <sup>
                     <a href="#pone.0117849.ref049">49</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref124">
                  <sup>
                     <a href="#pone.0117849.ref124">124</a>
                  </sup>
               </span>].
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The NKV
               <sup>SL</sup> Apoi virus, an OW virus isolated in Japan from 
               <i>Apodemus</i> mice (
               <i>Muridae</i> family, 
               <i>Murinae</i> sub-family) roots all other NKV
               <sup>SL</sup>. MODV and JUTV were the first recognised rodent NKV
               <sup>SL</sup> to be introduced to the NW (
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span>—node O) about 9500 years ago. MODV was isolated in North America from 
               <i>Peromyscus</i> mice (
               <i>Muridae</i> family, 
               <i>Sigmodontinae</i> sub-family) whereas JUTV was isolated in South America. These viruses were possibly
               introduced into the NW via rodents, when the OW and NW land masses in the northern
               hemisphere were joined by ice.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Rio Bravo virus (RBV) and Montana myotis leukoencephalitis virus (MMLV, represent
               a second introduction of NKVs into the NW (
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span>—node N). Their ancestral lineage appears to have diverged from an ancestral lineage
               of the OW bat NKVs.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Studies on Arenaviruses and Hantaviruses originally led to the development of the
               concept that viruses have co-evolved with their rodent hosts over time-scales of millions
               of years during which the viruses were assumed to have been transported during the
               gradual introduction of rodents from the OW into the NW [
               <span ref-type="bibr" rid="pone.0117849.ref125">
                  <sup>
                     <a href="#pone.0117849.ref125">125</a>
                  </sup>
               </span>–
               <span ref-type="bibr" rid="pone.0117849.ref127">
                  <sup>
                     <a href="#pone.0117849.ref127">127</a>
                  </sup>
               </span>]. However more recent studies do not appear to support this concept since for hantaviruses,
               estimates of virus divergence are in the order of thousands of years, as they exhibit
               short-term substitution rates of 10–2 to 10–4 substitutions/site/year [
               <span ref-type="bibr" rid="pone.0117849.ref128">
                  <sup>
                     <a href="#pone.0117849.ref128">128</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref129">
                  <sup>
                     <a href="#pone.0117849.ref129">129</a>
                  </sup>
               </span>]. It is hence more likely that Hantaviruses adapted relatively recently to their
               rodent hosts.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In the case of rodent-associated flaviviruses, and particularly NKV
               <sup>SL</sup>, only limited genetic data are available and due to presumed undersampling of viruses,
               to date there is no clear indication of the exact period of introduction of viruses
               to the rodent species in the NW. However, by analogy with the hantaviruses, and taking
               into account the TMRCA presented in 
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span>, it seems unlikely that flaviviruses have coevolved with their rodent hosts over
               millions of years.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Although the uncertainty in the TMRCA is quite significant, there are clearly two
               periods of emergence, 
               <i>i</i>.
               <i>e</i>. an “ancient period” in the order of magnitude of thousands of years ago, and a more
               “recent” period in the order of decades to a few hundred years ago. We note that it
               is important to be aware of the fact that estimates for the “ancient period” including
               estimates for NK
               <sup>SL</sup>, are based on a single calibration event of an 
               <i>Aedes</i> vectored virus (YFV) during the “recent period”. This is almost certainly not an
               accurate estimation of the evolutionary dynamics that occurred during the “ancient
               period”, which in the case of the NKV
               <sup>SL</sup> are not vectored by arthropods.
            </p>
         </div>
         <div id="sec032">
            <h3>Frameshifting in the MBFVs and ISFVs</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">It was previously reported that members of the JEV serocomplex express a transframe
               fusion protein, NS1', via-1 PRF [
               <span ref-type="bibr" rid="pone.0117849.ref089">
                  <sup>
                     <a href="#pone.0117849.ref089">89</a>
                  </sup>
               </span>]. The frameshift-stimulatory elements—a 'slippery' heptanucleotide sequence and the
               potential to form a stable 3'-adjacent pseudoknot structure—are conserved in all recognised
               members of the JEV serocomplex with the exception of SLEV [
               <span ref-type="bibr" rid="pone.0117849.ref088">
                  <sup>
                     <a href="#pone.0117849.ref088">88</a>
                  </sup>
               </span>]. However, the sequences of CPCV, YAOV and KOUV had not been determined when this
               conclusion was reached. We now show that the frameshift site is also conserved in
               these newly sequenced viruses. Thus, the ability to produce NS1' via PRF appears to
               have been acquired as a genetic trait after the branching point which separates the
               New World virus SLEV (
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span>—node C) from another New World virus (CPCV) and the remaining Old World JEV serocomplex
               viruses (
               <span ref-type="fig" rid="pone.0117849.g002">Fig. 2</span>—node B).
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The CPCV sequence also lacks the potential to form the canonical JEV-serogroup 3'
               RNA stable pseudoknot that is associated with highly efficient-1 PRF (20–50%; [
               <span ref-type="bibr" rid="pone.0117849.ref089">
                  <sup>
                     <a href="#pone.0117849.ref089">89</a>
                  </sup>
               </span>]); thus it is possible that frameshifting in CPCV is relatively inefficient.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The significance of the NS1' protein is not yet fully understood but it has been reported
               that it plays a role in viral neuroinvasiveness and reduced neurovirulence [
               <span ref-type="bibr" rid="pone.0117849.ref089">
                  <sup>
                     <a href="#pone.0117849.ref089">89</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref130">
                  <sup>
                     <a href="#pone.0117849.ref130">130</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pone.0117849.ref131">
                  <sup>
                     <a href="#pone.0117849.ref131">131</a>
                  </sup>
               </span>].
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Frameshifting at a very similar genomic location occurs in the ISFVs but results in
               the translation of a much longer overlapping ORF [
               <span ref-type="bibr" rid="pone.0117849.ref090">
                  <sup>
                     <a href="#pone.0117849.ref090">90</a>
                  </sup>
               </span>]. Potential frameshift sites have also been bioinformatically predicted for NOUV,
               CHAOV, LAMV and KEDV (Firth 
               <i>et al</i>., 2010); ILOV and DGV [
               <span ref-type="bibr" rid="pone.0117849.ref052">
                  <sup>
                     <a href="#pone.0117849.ref052">52</a>
                  </sup>
               </span>]; and SEPV and WESSV (see above). These potential frameshift sites are also located
               within the genomic region encoding NS2A/NS2B. Thus, the acquisition of PRF at internal
               regions of the polyprotein ORF appears to be a common (though not ubiquitous) theme
               of flavivirus evolution. Aside from the ability to produce new functional proteins,
               where PRF is efficient, it may also play a role in downregulating production of the
               3'-encoded replicative proteins and more quickly recycling the host cell translational
               machinery for increased production of the 5'-encoded structural proteins.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In conclusion, with the addition of 14 new flavivirus ORF sequences, the estimation
               of phylogenies via Bayesian methods, plus biogeographic and bioinformatic considerations,
               we have identified 11 likely introductions of mosquito-borne flaviviruses from the
               OW to the NW over two separate temporal periods. We have also demonstrated that similar
               introductions have occurred eastwards from Africa to Australia, again over two distinct
               time periods. Clearly there have been far more introductory movements of flaviviruses
               from one part of the world to another than we have described here. However, more data
               for a larger number of samples will be required before we can draw specific and more
               detailed conclusions. Finally, in the context of flavivirus evolution and dispersal,
               we have extended our current understanding of frameshifting amongst the flaviviruses.
            </p>
         </div>
      </div>
      <div tagxxx="back">
         <div tag="ref-list">
            <ul>
               <div tagxxx="title">REFERENCES</div>
               <li tag="ref">
                  <a name="pone.0117849.ref001"></a>
                  <span class="label">1</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gaunt</span>
                        <span tagx="given-names">MW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sall</span>
                        <span tagx="given-names">AA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">de Lamballerie</span>
                        <span tagx="given-names">X</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Falconar</span>
                        <span tagx="given-names">AK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Dzhivanian</span>
                        <span tagx="given-names">TI</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2001</span>) 
                     <span tagx="article-title">Phylogenetic relationships of flaviviruses correlate with their epidemiology, disease
                        association and biogeography
                     </span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">82</span>: 
                     <span tagx="fpage">1867</span>–
                     <span tagx="lpage">1876</span>. 
                     <span class="pub-id'">[11457992]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref002"></a>
                  <span class="label">2</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Solomon</span>
                        <span tagx="given-names">T</span>,
                     </span> (
                     <span tagx="year">2008</span>) 
                     <span tagx="article-title">Pathogenic flaviviruses</span>. 
                     <span tagx="source">Lancet</span>
                     <span tagx="volume">371</span>: 
                     <span tagx="fpage">500</span>–
                     <span tagx="lpage">509</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/S0140-6736(08)60238-X</div>
                     </div>
                     <span class="pub-id'">[18262042]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref003"></a>
                  <span class="label">3</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Grard</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Moureau</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Charrel</span>
                        <span tagx="given-names">RN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lemasson</span>
                        <span tagx="given-names">JJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gonzalez</span>
                        <span tagx="given-names">JP</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">Genetic characterization of tick-borne flaviviruses: new insights into evolution,
                        pathogenetic determinants and taxonomy
                     </span>. 
                     <span tagx="source">Virology</span>
                     <span tagx="volume">361</span>: 
                     <span tagx="fpage">80</span>–
                     <span tagx="lpage">92</span>. 
                     <span class="pub-id'">[17169393]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref004"></a>
                  <span class="label">4</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Zanotto</span>
                        <span tagx="given-names">PM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gao</span>
                        <span tagx="given-names">GF</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gritsun</span>
                        <span tagx="given-names">T</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Marin</span>
                        <span tagx="given-names">MS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Jiang</span>
                        <span tagx="given-names">WR</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">1995</span>) 
                     <span tagx="article-title">An arbovirus cline across the northern hemisphere</span>. 
                     <span tagx="source">Virology</span>
                     <span tagx="volume">210</span>: 
                     <span tagx="fpage">152</span>–
                     <span tagx="lpage">159</span>. 
                     <span class="pub-id'">[7793067]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref005"></a>
                  <span class="label">5</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Bhatt</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gething</span>
                        <span tagx="given-names">PW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Brady</span>
                        <span tagx="given-names">OJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Messina</span>
                        <span tagx="given-names">JP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Farlow</span>
                        <span tagx="given-names">AW</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">The global distribution and burden of dengue</span>. 
                     <span tagx="source">Nature</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1038/nature12879</div>
                     </div>
                     <span class="pub-id'">[24390349]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref006"></a>
                  <span class="label">6</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Calisher</span>
                        <span tagx="given-names">CH</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span> (
                     <span tagx="year">2003</span>) 
                     <span tagx="article-title">Taxonomy of the virus family Flaviviridae</span>. 
                     <span tagx="source">Adv Virus Res</span>
                     <span tagx="volume">59</span>: 
                     <span tagx="fpage">1</span>–
                     <span tagx="lpage">19</span>. 
                     <span class="pub-id'">[14696325]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref007"></a>
                  <span class="label">7</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Pletnev</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Heinz</span>
                        <span tagx="given-names">FX</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Meyers</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Thiel</span>
                        <span tagx="given-names">H-J</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2011</span>) 
                     <div tagxxx="chapter-title">Flaviviridae</div> In: 
                     <span class="name'">
                        <span tagx="surname">King</span>
                        <span tagx="given-names">AMQ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Adams</span>
                        <span tagx="given-names">MJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Carstens</span>
                        <span tagx="given-names">EB</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lefkowitz</span>
                        <span tagx="given-names">EJ</span>,
                     </span>, editors. 
                     <span tagx="source">Virus Taxonomy—Ninth Report of the International Committee on Taxonomy of Viruses</span>. 
                     <span class="publisher-loc'">[Oxford]</span>: 
                     <span class="publisher-name'">[Elsevier]</span> pp. 
                     <span tagx="fpage">1003</span>–
                     <span tagx="lpage">1020</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/j.bjid.2014.12.003</div>
                     </div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref008"></a>
                  <span class="label">8</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Grard</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Moureau</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Charrel</span>
                        <span tagx="given-names">RN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">EC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">Genomics and evolution of Aedes-borne flaviviruses</span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">91</span>: 
                     <span tagx="fpage">87</span>–
                     <span tagx="lpage">94</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1099/vir.0.014506-0</div>
                     </div>
                     <span class="pub-id'">[19741066]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref009"></a>
                  <span class="label">9</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Zanotto</span>
                        <span tagx="given-names">PM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gao</span>
                        <span tagx="given-names">GF</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Harvey</span>
                        <span tagx="given-names">PH</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">EC</span>,
                     </span> (
                     <span tagx="year">1996</span>) 
                     <span tagx="article-title">Population dynamics of flaviviruses revealed by molecular phylogenies</span>. 
                     <span tagx="source">Proceedings of the National Academy of Sciences</span>
                     <span tagx="volume">93</span>: 
                     <span tagx="fpage">548</span>–
                     <span tagx="lpage">553</span>. 
                     <span class="pub-id'">[8570593]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref010"></a>
                  <span class="label">10</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Cook</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">EC</span>,
                     </span> (
                     <span tagx="year">2005</span>) 
                     <span tagx="article-title">A multigene analysis of the phylogenetic relationships among the flaviviruses (Family:
                        Flaviviridae) and the evolution of vector transmission
                     </span>. 
                     <span tagx="source">Arch Virol</span>. 
                     <span class="pub-id'">[16380812]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref011"></a>
                  <span class="label">11</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">de Lamballerie</span>
                        <span tagx="given-names">X</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Crochu</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Billoir</span>
                        <span tagx="given-names">F</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Neyts</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">de Micco</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2002</span>) 
                     <span tagx="article-title">Genome sequence analysis of Tamana bat virus and its relationship with the genus Flavivirus</span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">83</span>: 
                     <span tagx="fpage">2443</span>–
                     <span tagx="lpage">2454</span>. 
                     <span class="pub-id'">[12237426]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref012"></a>
                  <span class="label">12</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">de Lamballerie</span>
                        <span tagx="given-names">X</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Zanotto</span>
                        <span tagx="given-names">PM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">EC</span>,
                     </span> (
                     <span tagx="year">2003</span>) 
                     <span tagx="article-title">Origins, evolution, and vector/host coadaptations within the genus Flavivirus</span>. 
                     <span tagx="source">Adv Virus Res</span>
                     <span tagx="volume">59</span>: 
                     <span tagx="fpage">277</span>–
                     <span tagx="lpage">314</span>. 
                     <span class="pub-id'">[14696332]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref013"></a>
                  <span class="label">13</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Huhtamo</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Putkuri</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kurkela</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Manni</span>
                        <span tagx="given-names">T</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vaheri</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Characterization of a novel flavivirus from mosquitoes in northern europe that is
                        related to mosquito-borne flaviviruses of the tropics
                     </span>. 
                     <span tagx="source">J Virol</span>
                     <span tagx="volume">83</span>: 
                     <span tagx="fpage">9532</span>–
                     <span tagx="lpage">9540</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1128/JVI.00529-09</div>
                     </div>
                     <span class="pub-id'">[19570865]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref014"></a>
                  <span class="label">14</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Grard</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lemasson</span>
                        <span tagx="given-names">JJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sylla</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Dubot</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Cook</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">Ngoye virus: a novel evolutionary lineage within the genus Flavivirus</span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">87</span>: 
                     <span tagx="fpage">3273</span>–
                     <span tagx="lpage">3277</span>. 
                     <span class="pub-id'">[17030860]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref015"></a>
                  <span class="label">15</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Charrel</span>
                        <span tagx="given-names">RN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Zaki</span>
                        <span tagx="given-names">AM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Fakeeh</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Yousef</span>
                        <span tagx="given-names">AI</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">de Chesse</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2005</span>) 
                     <span tagx="article-title">Low diversity of Alkhurma hemorrhagic fever virus, Saudi Arabia, 1994–1999</span>. 
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="volume">11</span>: 
                     <span tagx="fpage">683</span>–
                     <span tagx="lpage">688</span>. 
                     <span class="pub-id'">[15890119]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref016"></a>
                  <span class="label">16</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Bakonyi</span>
                        <span tagx="given-names">T</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kolodziejek</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Weissenbock</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nowotny</span>
                        <span tagx="given-names">N</span>,
                     </span> (
                     <span tagx="year">2004</span>) 
                     <span tagx="article-title">Complete genome analysis and molecular characterization of Usutu virus that emerged
                        in Austria in 2001: comparison with the South African strain SAAR-1776 and other flaviviruses
                     </span>. 
                     <span tagx="source">Virology</span>
                     <span tagx="volume">328</span>: 
                     <span tagx="fpage">301</span>–
                     <span tagx="lpage">310</span>. 
                     <span class="pub-id'">[15464850]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref017"></a>
                  <span class="label">17</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Nisbet</span>
                        <span tagx="given-names">DJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lee</span>
                        <span tagx="given-names">KJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">van den Hurk</span>
                        <span tagx="given-names">AF</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Johansen</span>
                        <span tagx="given-names">CA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kuno</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2005</span>) 
                     <span tagx="article-title">Identification of new flaviviruses in the Kokobera virus complex</span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">86</span>: 
                     <span tagx="fpage">121</span>–
                     <span tagx="lpage">124</span>. 
                     <span class="pub-id'">[15604438]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref018"></a>
                  <span class="label">18</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Vazquez</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sanchez-Seco</span>
                        <span tagx="given-names">MP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Palacios</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Molero</span>
                        <span tagx="given-names">F</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Reyes</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">Novel Flaviviruses Detected in Different Species of Mosquitoes in Spain</span>. 
                     <span tagx="source">Vector Borne Zoonotic Dis</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1089/vbz.2011.0736</div>
                     </div>
                     <span class="pub-id'">[22141733]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref019"></a>
                  <span class="label">19</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Cook</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Bennett</span>
                        <span tagx="given-names">SN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">EC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">De Chesse</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Moureau</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">Isolation of a new strain of the flavivirus cell fusing agent virus in a natural mosquito
                        population from Puerto Rico
                     </span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">87</span>: 
                     <span tagx="fpage">735</span>–
                     <span tagx="lpage">748</span>. 
                     <span class="pub-id'">[16528021]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref020"></a>
                  <span class="label">20</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Cook</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Moureau</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Harbach</span>
                        <span tagx="given-names">RE</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Mukwaya</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Goodger</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Isolation of a novel species of flavivirus and a new strain of Culex flavivirus (Flaviviridae)
                        from a natural mosquito population in Uganda
                     </span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">90</span>: 
                     <span tagx="fpage">2669</span>–
                     <span tagx="lpage">2678</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1099/vir.0.014183-0</div>
                     </div>
                     <span class="pub-id'">[19656970]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref021"></a>
                  <span class="label">21</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Kim</span>
                        <span tagx="given-names">DY</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Guzman</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Bueno</span>
                        <span tagx="given-names">R</span>
                        <div tagxxx="suffix">Jr</div>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Dennett</span>
                        <span tagx="given-names">JA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Auguste</span>
                        <span tagx="given-names">AJ</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Characterization of Culex Flavivirus (Flaviviridae) strains isolated from mosquitoes
                        in the United States and Trinidad
                     </span>. 
                     <span tagx="source">Virology</span>
                     <span tagx="volume">386</span>: 
                     <span tagx="fpage">154</span>–
                     <span tagx="lpage">159</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/j.virol.2008.12.034</div>
                     </div>
                     <span class="pub-id'">[19193389]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref022"></a>
                  <span class="label">22</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Sang</span>
                        <span tagx="given-names">RC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gichogo</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gachoya</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Dunster</span>
                        <span tagx="given-names">MD</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ofula</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2003</span>) 
                     <span tagx="article-title">Isolation of a new flavivirus related to cell fusing agent virus (CFAV) from field-collected
                        flood-water Aedes mosquitoes sampled from a dambo in central Kenya
                     </span>. 
                     <span tagx="source">Arch Virol</span>
                     <span tagx="volume">148</span>: 
                     <span tagx="fpage">1085</span>–
                     <span tagx="lpage">1093</span>. 
                     <span class="pub-id'">[12756616]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref023"></a>
                  <span class="label">23</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Stollar</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Thomas</span>
                        <span tagx="given-names">VL</span>,
                     </span> (
                     <span tagx="year">1975</span>) 
                     <span tagx="article-title">An agent in the Aedes aegypti cell line (Peleg) which causes fusion of Aedes albopictus
                        cells
                     </span>. 
                     <span tagx="source">Virology</span>
                     <span tagx="volume">64</span>: 
                     <span tagx="fpage">367</span>–
                     <span tagx="lpage">377</span>. 
                     <span class="pub-id'">[806166]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref024"></a>
                  <span class="label">24</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Maruyama</span>
                        <span tagx="given-names">SR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Castro-Jorge</span>
                        <span tagx="given-names">LA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ribeiro</span>
                        <span tagx="given-names">JM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gardinassi</span>
                        <span tagx="given-names">LG</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Garcia</span>
                        <span tagx="given-names">GR</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2014</span>) 
                     <span tagx="article-title">Characterisation of divergent flavivirus NS3 and NS5 protein sequences detected in
                        Rhipicephalus microplus ticks from Brazil
                     </span>. 
                     <span tagx="source">Memorias do Instituto Oswaldo Cruz</span>
                     <span tagx="volume">109</span>: 
                     <span tagx="fpage">38</span>–
                     <span tagx="lpage">50</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1590/0074-0276130166</div>
                     </div>
                     <span class="pub-id'">[24626302]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref025"></a>
                  <span class="label">25</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Billoir</span>
                        <span tagx="given-names">F</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">de Chesse</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tolou</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">de Micco</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2000</span>) 
                     <span tagx="article-title">Phylogeny of the genus flavivirus using complete coding sequences of arthropod-borne
                        viruses and viruses with no known vector
                     </span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">81</span>: 
                     <span tagx="fpage">781</span>–
                     <span tagx="lpage">790</span>. 
                     <span class="pub-id'">[10675416]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref026"></a>
                  <span class="label">26</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Crochu</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Cook</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Attoui</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Charrel</span>
                        <span tagx="given-names">RN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">De Chesse</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2004</span>) 
                     <span tagx="article-title">Sequences of flavivirus-related RNA viruses persist in DNA form integrated in the
                        genome of Aedes spp. mosquitoes
                     </span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">85</span>: 
                     <span tagx="fpage">1971</span>–
                     <span tagx="lpage">1980</span>. 
                     <span class="pub-id'">[15218182]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref027"></a>
                  <span class="label">27</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gaunt</span>
                        <span tagx="given-names">MW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span> (
                     <span tagx="year">2005</span>) 
                     <span tagx="article-title">Rapid subgroup identification of the flaviviruses using degenerate primer E-gene RT-PCR
                        and site specific restriction enzyme analysis
                     </span>. 
                     <span tagx="source">J Virol Methods</span>
                     <span tagx="volume">128</span>: 
                     <span tagx="fpage">113</span>–
                     <span tagx="lpage">127</span>. 
                     <span class="pub-id'">[15927275]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref028"></a>
                  <span class="label">28</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Moureau</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Temmam</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gonzalez</span>
                        <span tagx="given-names">JP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Charrel</span>
                        <span tagx="given-names">RN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Grard</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">A real-time RT-PCR method for the universal detection and identification of flaviviruses</span>. 
                     <span tagx="source">Vector Borne Zoonotic Dis</span>
                     <span tagx="volume">7</span>: 
                     <span tagx="fpage">467</span>–
                     <span tagx="lpage">477</span>. 
                     <span class="pub-id'">[18020965]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref029"></a>
                  <span class="label">29</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Emonet</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Grard</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Brisbarre</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Moureau</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Temmam</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">LoPPS: a long PCR product sequencing method for rapid characterisation of long amplicons</span>. 
                     <span tagx="source">Biochem Biophys Res Commun</span>
                     <span tagx="volume">344</span>: 
                     <span tagx="fpage">1080</span>–
                     <span tagx="lpage">1085</span>. 
                     <span class="pub-id'">[16643852]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref030"></a>
                  <span class="label">30</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Emonet</span>
                        <span tagx="given-names">SF</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Grard</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Brisbarre</span>
                        <span tagx="given-names">NM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Moureau</span>
                        <span tagx="given-names">GN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Temmam</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">Long PCR Product Sequencing (LoPPS): a shotgun-based approach to sequence long PCR
                        products
                     </span>. 
                     <span tagx="source">Nat Protoc</span>
                     <span tagx="volume">2</span>: 
                     <span tagx="fpage">340</span>–
                     <span tagx="lpage">346</span>. 
                     <span class="pub-id'">[17406595]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref031"></a>
                  <span class="label">31</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Rothberg</span>
                        <span tagx="given-names">JM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hinz</span>
                        <span tagx="given-names">W</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rearick</span>
                        <span tagx="given-names">TM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Schultz</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Mileski</span>
                        <span tagx="given-names">W</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">An integrated semiconductor device enabling non-optical genome sequencing</span>. 
                     <span tagx="source">Nature</span>
                     <span tagx="volume">475</span>: 
                     <span tagx="fpage">348</span>–
                     <span tagx="lpage">352</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1038/nature10242</div>
                     </div>
                     <span class="pub-id'">[21776081]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref032"></a>
                  <span class="label">32</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Larkin</span>
                        <span tagx="given-names">MA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Blackshields</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Brown</span>
                        <span tagx="given-names">NP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chenna</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">McGettigan</span>
                        <span tagx="given-names">PA</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">Clustal W and Clustal X version 2.0</span>. 
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="volume">23</span>: 
                     <span tagx="fpage">2947</span>–
                     <span tagx="lpage">2948</span>. 
                     <span class="pub-id'">[17846036]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref033"></a>
                  <span class="label">33</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Thompson</span>
                        <span tagx="given-names">JD</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Higgins</span>
                        <span tagx="given-names">DG</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gibson</span>
                        <span tagx="given-names">TJ</span>,
                     </span> (
                     <span tagx="year">1994</span>) 
                     <span tagx="article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through
                        sequence weighting, position-specific gap penalties and weight matrix choice
                     </span>. 
                     <span tagx="source">Nucleic acids research</span>
                     <span tagx="volume">22</span>: 
                     <span tagx="fpage">4673</span>–
                     <span tagx="lpage">4680</span>. 
                     <span class="pub-id'">[7984417]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref034"></a>
                  <span class="label">34</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Edgar</span>
                        <span tagx="given-names">RC</span>,
                     </span> (
                     <span tagx="year">2004</span>) 
                     <span tagx="article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. 
                     <span tagx="source">Nucleic acids research</span>
                     <span tagx="volume">32</span>: 
                     <span tagx="fpage">1792</span>–
                     <span tagx="lpage">1797</span>. 
                     <span class="pub-id'">[15034147]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref035"></a>
                  <span class="label">35</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Talavera</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Castresana</span>
                        <span tagx="given-names">J</span>,
                     </span> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">Improvement of phylogenies after removing divergent and ambiguously aligned blocks
                        from protein sequence alignments
                     </span>. 
                     <span tagx="source">Syst Biol</span>
                     <span tagx="volume">56</span>: 
                     <span tagx="fpage">564</span>–
                     <span tagx="lpage">577</span>. 
                     <span class="pub-id'">[17654362]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref036"></a>
                  <span class="label">36</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Huelsenbeck</span>
                        <span tagx="given-names">JP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ronquist</span>
                        <span tagx="given-names">F</span>,
                     </span> (
                     <span tagx="year">2001</span>) 
                     <span tagx="article-title">MRBAYES: Bayesian inference of phylogenetic trees</span>. 
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="volume">17</span>: 
                     <span tagx="fpage">754</span>–
                     <span tagx="lpage">755</span>. 
                     <span class="pub-id'">[11524383]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref037"></a>
                  <span class="label">37</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Drummond</span>
                        <span tagx="given-names">AJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rambaut</span>
                        <span tagx="given-names">A</span>,
                     </span> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">BEAST: Bayesian evolutionary analysis by sampling trees</span>. 
                     <span tagx="source">BMC Evol Biol</span>
                     <span tagx="volume">7</span>: 
                     <span tagx="fpage">214</span>
                     <span class="pub-id'">[17996036]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref038"></a>
                  <span class="label">38</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Reeder</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Steffen</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Giegerich</span>
                        <span tagx="given-names">R</span>,
                     </span> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">pknotsRG: RNA pseudoknot folding including near-optimal structures and sliding windows</span>. 
                     <span tagx="source">Nucleic acids research</span>
                     <span tagx="volume">35</span>: 
                     <span tagx="fpage">W320</span>–
                     <span tagx="lpage">324</span>. 
                     <span class="pub-id'">[17478505]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref039"></a>
                  <span class="label">39</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Firth</span>
                        <span tagx="given-names">AE</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Wills</span>
                        <span tagx="given-names">NM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gesteland</span>
                        <span tagx="given-names">RF</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Atkins</span>
                        <span tagx="given-names">JF</span>,
                     </span> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">Stimulation of stop codon readthrough: frequent presence of an extended 3' RNA structural
                        element
                     </span>. 
                     <span tagx="source">Nucleic acids research</span>
                     <span tagx="volume">39</span>: 
                     <span tagx="fpage">6679</span>–
                     <span tagx="lpage">6691</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1093/nar/gkr224</div>
                     </div>
                     <span class="pub-id'">[21525127]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref040"></a>
                  <span class="label">40</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Drummond</span>
                        <span tagx="given-names">AJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Suchard</span>
                        <span tagx="given-names">MA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Xie</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rambaut</span>
                        <span tagx="given-names">A</span>,
                     </span> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Bayesian phylogenetics with BEAUti and the BEAST 1.7</span>. 
                     <span tagx="source">Mol Biol Evol</span>
                     <span tagx="volume">29</span>: 
                     <span tagx="fpage">1969</span>–
                     <span tagx="lpage">1973</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1093/molbev/mss075</div>
                     </div>
                     <span class="pub-id'">[22367748]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref041"></a>
                  <span class="label">41</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Abascal</span>
                        <span tagx="given-names">F</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Zardoya</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Posada</span>
                        <span tagx="given-names">D</span>,
                     </span> (
                     <span tagx="year">2005</span>) 
                     <span tagx="article-title">ProtTest: selection of best-fit models of protein evolution</span>. 
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="volume">21</span>: 
                     <span tagx="fpage">2104</span>–
                     <span tagx="lpage">2105</span>. 
                     <span class="pub-id'">[15647292]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref042"></a>
                  <span class="label">42</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Drummond</span>
                        <span tagx="given-names">AJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ho</span>
                        <span tagx="given-names">SY</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Phillips</span>
                        <span tagx="given-names">MJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rambaut</span>
                        <span tagx="given-names">A</span>,
                     </span> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">Relaxed phylogenetics and dating with confidence</span>. 
                     <span tagx="source">PLoS biology</span>
                     <span tagx="volume">4</span>: 
                     <span tagx="fpage">e88</span>
                     <span class="pub-id'">[16683862]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref043"></a>
                  <span class="label">43</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gill</span>
                        <span tagx="given-names">MS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lemey</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Faria</span>
                        <span tagx="given-names">NR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rambaut</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Shapiro</span>
                        <span tagx="given-names">B</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">Improving Bayesian population dynamics inference: a coalescent-based model for multiple
                        loci
                     </span>. 
                     <span tagx="source">Mol Biol Evol</span>
                     <span tagx="volume">30</span>: 
                     <span tagx="fpage">713</span>–
                     <span tagx="lpage">724</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1093/molbev/mss265</div>
                     </div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref044"></a>
                  <span class="label">44</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Bryant</span>
                        <span tagx="given-names">JE</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">EC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Barrett</span>
                        <span tagx="given-names">AD</span>,
                     </span> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">Out of Africa: a molecular perspective on the introduction of yellow fever virus into
                        the Americas
                     </span>. 
                     <span tagx="source">PLoS Pathog</span>
                     <span tagx="volume">3</span>: 
                     <span tagx="fpage">e75</span>
                     <span class="pub-id'">[17511518]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref045"></a>
                  <span class="label">45</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Tabachnick</span>
                        <span tagx="given-names">WJ</span>,
                     </span> (
                     <span tagx="year">1991</span>) 
                     <span tagx="article-title">Evolutionary genetics and arthropod-borne diseases: The yellow fever mosquito, Aedes
                        aegypti
                     </span>. 
                     <span tagx="source">Am J Entomol</span>
                     <span tagx="volume">37</span>:
                     <span tagx="fpage">14</span>–
                     <span tagx="lpage">24</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref046"></a>
                  <span class="label">46</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Worobey</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Telfer</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Souquiere</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hunter</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Coleman</span>
                        <span tagx="given-names">CA</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">Island biogeography reveals the deep history of SIV</span>. 
                     <span tagx="source">Science</span>
                     <span tagx="volume">329</span>: 
                     <span tagx="fpage">1487</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1126/science.1193550</div>
                     </div>
                     <span class="pub-id'">[20847261]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref047"></a>
                  <span class="label">47</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Lemey</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rambaut</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Drummond</span>
                        <span tagx="given-names">AJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Suchard</span>
                        <span tagx="given-names">MA</span>,
                     </span> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Bayesian phylogeography finds its roots</span>. 
                     <span tagx="source">PLoS computational biology</span>
                     <span tagx="volume">5</span>: 
                     <span tagx="fpage">e1000520</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1371/journal.pcbi.1000520</div>
                     </div>
                     <span class="pub-id'">[19779555]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref048"></a>
                  <span class="label">48</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Karabatsos</span>
                        <span tagx="given-names">N</span>,
                     </span> (
                     <span tagx="year">1985</span>) 
                     <span tagx="source">International Catalogue of Arboviruses</span>; 
                     <span class="name'">
                        <span tagx="surname">Karabatsos</span>
                        <span tagx="given-names">N</span>,
                     </span>, editor. 
                     <span class="publisher-loc'">[San Antonio, Texas]</span>: 
                     <span class="publisher-name'">[The American Society of Tropical Medecine and Hygiene]</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref049"></a>
                  <span class="label">49</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Cook</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Moureau</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kitchen</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">de Lamballerie</span>
                        <span tagx="given-names">X</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Molecular evolution of the insect-specific flaviviruses</span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">93</span>: 
                     <span tagx="fpage">223</span>–
                     <span tagx="lpage">234</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1099/vir.0.036525-0</div>
                     </div>
                     <span class="pub-id'">[22012464]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref050"></a>
                  <span class="label">50</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Huhtamo</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Moureau</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Cook</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Julkunen</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Putkuri</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Novel insect-specific flavivirus isolated from northern Europe</span>. 
                     <span tagx="source">Virology</span>
                     <span tagx="volume">433</span>: 
                     <span tagx="fpage">471</span>–
                     <span tagx="lpage">478</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/j.virol.2012.08.038</div>
                     </div>
                     <span class="pub-id'">[22999256]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref051"></a>
                  <span class="label">51</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Liu</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lu</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Li</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Moureau</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Deng</span>
                        <span tagx="given-names">YQ</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Genomic and antigenic characterization of the newly emerging Chinese duck egg-drop
                        syndrome flavivirus: genomic comparison with Tembusu and Sitiawan viruses
                     </span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">93</span>: 
                     <span tagx="fpage">2158</span>–
                     <span tagx="lpage">2170</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1099/vir.0.043554-0</div>
                     </div>
                     <span class="pub-id'">[22764316]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref052"></a>
                  <span class="label">52</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Huhtamo</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Cook</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Moureau</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Uzcátegui</span>
                        <span tagx="given-names">NY</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sironen</span>
                        <span tagx="given-names">T</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2014</span>) 
                     <span tagx="source">Novel flaviviruses from mosquitoes: Mosquito-specific evolutionary lineages within
                        the phylogenetic group of mosquito-borne flaviviruses
                     </span>. 
                     <span class="publisher-loc'">[Virology]</span>: 
                     <span class="publisher-name'">[In press]</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref053"></a>
                  <span class="label">53</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Lvov</span>
                        <span tagx="given-names">DK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tsyrkin</span>
                        <span tagx="given-names">YM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Karas</span>
                        <span tagx="given-names">FR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Timopheev</span>
                        <span tagx="given-names">EM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gromashevski</span>
                        <span tagx="given-names">VL</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">1973</span>) 
                     <span tagx="article-title">"Sokuluk" virus, a new group B arbovirus isolated from Vespertilio pipistrellus Schreber,
                        1775, bat in the Kirghiz S.S.R
                     </span>. 
                     <span tagx="source">Arch Gesamte Virusforsch</span>
                     <span tagx="volume">41</span>: 
                     <span tagx="fpage">170</span>–
                     <span tagx="lpage">174</span>. 
                     <span class="pub-id'">[4727779]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref054"></a>
                  <span class="label">54</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Varelas-Wesley</span>
                        <span tagx="given-names">I</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Calisher</span>
                        <span tagx="given-names">CH</span>,
                     </span> (
                     <span tagx="year">1982</span>) 
                     <span tagx="article-title">Antigenic relationships of flaviviruses with undetermined arthropod-borne status</span>. 
                     <span tagx="source">Am J Trop Med Hyg</span>
                     <span tagx="volume">31</span>: 
                     <span tagx="fpage">1273</span>–
                     <span tagx="lpage">1284</span>. 
                     <span class="pub-id'">[6293325]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref055"></a>
                  <span class="label">55</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Kuno</span>
                        <span tagx="given-names">G</span>,
                     </span> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">Host range specificity of flaviviruses: correlation with in vitro replication</span>. 
                     <span tagx="source">J Med Entomol</span>
                     <span tagx="volume">44</span>: 
                     <span tagx="fpage">93</span>–
                     <span tagx="lpage">101</span>. 
                     <span class="pub-id'">[17294926]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref056"></a>
                  <span class="label">56</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Lee</span>
                        <span tagx="given-names">JS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Grubaugh</span>
                        <span tagx="given-names">ND</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kondig</span>
                        <span tagx="given-names">JP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Turell</span>
                        <span tagx="given-names">MJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kim</span>
                        <span tagx="given-names">HC</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">Isolation and genomic characterization of Chaoyang virus strain ROK144 from Aedes
                        vexans nipponii from the Republic of Korea
                     </span>. 
                     <span tagx="source">Virology</span>
                     <span tagx="volume">435</span>: 
                     <span tagx="fpage">220</span>–
                     <span tagx="lpage">224</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/j.virol.2012.10.020</div>
                     </div>
                     <span class="pub-id'">[23127596]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref057"></a>
                  <span class="label">57</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Junglen</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kopp</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kurth</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Pauli</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ellerbrok</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">A new flavivirus and a new vector: characterization of a novel flavivirus isolated
                        from uranotaenia mosquitoes from a tropical rain forest
                     </span>. 
                     <span tagx="source">J Virol</span>
                     <span tagx="volume">83</span>: 
                     <span tagx="fpage">4462</span>–
                     <span tagx="lpage">4468</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1128/JVI.00014-09</div>
                     </div>
                     <span class="pub-id'">[19224998]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref058"></a>
                  <span class="label">58</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Kolodziejek</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Pachler</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Bin</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Mendelson</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Shulman</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">Barkedji virus, a novel mosquito-borne flavivirus identified in Culex perexiguus mosquitoes,
                        Israel, 2011
                     </span>. 
                     <span tagx="source">J Gen Virol</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1099/vir.0.060822-0</div>
                     </div>
                     <span class="pub-id'">[24362963]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref059"></a>
                  <span class="label">59</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Evangelista</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Cruz</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Guevara</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Astete</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Carey</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">Characterization of a novel flavivirus isolated from Culex (Melanoconion) ocossa mosquitoes
                        from Iquitos, Peru
                     </span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">94</span>: 
                     <span tagx="fpage">1266</span>–
                     <span tagx="lpage">1272</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1099/vir.0.050575-0</div>
                     </div>
                     <span class="pub-id'">[23515021]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref060"></a>
                  <span class="label">60</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Batista</span>
                        <span tagx="given-names">WC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tavares Gda</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vieira</span>
                        <span tagx="given-names">DS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Honda</span>
                        <span tagx="given-names">ER</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Pereira</span>
                        <span tagx="given-names">SS</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">Notification of the first isolation of Cacipacore virus in a human in the State of
                        Rondonia, Brazil
                     </span>. 
                     <span tagx="source">Rev Soc Bras Med Trop</span>
                     <span tagx="volume">44</span>: 
                     <span tagx="fpage">528</span>–
                     <span tagx="lpage">530</span>. 
                     <span class="pub-id'">[21860907]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref061"></a>
                  <span class="label">61</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Williams</span>
                        <span tagx="given-names">RAJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vázquez</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Asante</span>
                        <span tagx="given-names">I</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Bonney</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Odoom</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Yaoundé-like virus in resident wild bird, Ghana</span>. 
                     <span tagx="source">African Journal of Microbiology Research</span>
                     <span tagx="volume">6</span>: 
                     <span tagx="fpage">1966</span>–
                     <span tagx="lpage">1969</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref062"></a>
                  <span class="label">62</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Hubalek</span>
                        <span tagx="given-names">Z</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Halouzka</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Juricova</span>
                        <span tagx="given-names">Z</span>,
                     </span> (
                     <span tagx="year">1999</span>) 
                     <span tagx="article-title">West Nile fever in Czechland</span>. 
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="volume">5</span>: 
                     <span tagx="fpage">594</span>–
                     <span tagx="lpage">595</span>. 
                     <span class="pub-id'">[10458974]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref063"></a>
                  <span class="label">63</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Lvov</span>
                        <span tagx="given-names">DK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Butenko</span>
                        <span tagx="given-names">AM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gromashevsky</span>
                        <span tagx="given-names">VL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kovtunov</span>
                        <span tagx="given-names">AI</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Prilipov</span>
                        <span tagx="given-names">AG</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2004</span>) 
                     <span tagx="article-title">West Nile virus and other zoonotic viruses in Russia: examples of emerging-reemerging
                        situations
                     </span>. 
                     <span tagx="source">Arch Virol Suppl</span>: 
                     <span tagx="fpage">85</span>–
                     <span tagx="lpage">96</span>. 
                     <span class="pub-id'">[15119764]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref064"></a>
                  <span class="label">64</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Bondre</span>
                        <span tagx="given-names">VP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Jadi</span>
                        <span tagx="given-names">RS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Mishra</span>
                        <span tagx="given-names">AC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Yergolkar</span>
                        <span tagx="given-names">PN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Arankalle</span>
                        <span tagx="given-names">VA</span>,
                     </span> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">West Nile virus isolates from India: evidence for a distinct genetic lineage</span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">88</span>: 
                     <span tagx="fpage">875</span>–
                     <span tagx="lpage">884</span>. 
                     <span class="pub-id'">[17325360]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref065"></a>
                  <span class="label">65</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">May</span>
                        <span tagx="given-names">FJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Davis</span>
                        <span tagx="given-names">CT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tesh</span>
                        <span tagx="given-names">RB</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Barrett</span>
                        <span tagx="given-names">AD</span>,
                     </span> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">Phylogeography of West Nile virus: from the cradle of evolution in Africa to Eurasia,
                        Australia, and the Americas
                     </span>. 
                     <span tagx="source">J Virol</span>
                     <span tagx="volume">85</span>: 
                     <span tagx="fpage">2964</span>–
                     <span tagx="lpage">2974</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1128/JVI.01963-10</div>
                     </div>
                     <span class="pub-id'">[21159871]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref066"></a>
                  <span class="label">66</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">May</span>
                        <span tagx="given-names">FJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lobigs</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lee</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gendle</span>
                        <span tagx="given-names">DJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Mackenzie</span>
                        <span tagx="given-names">JS</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">Biological, antigenic and phylogenetic characterization of the flavivirus Alfuy</span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">87</span>: 
                     <span tagx="fpage">329</span>–
                     <span tagx="lpage">337</span>. 
                     <span class="pub-id'">[16432019]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref067"></a>
                  <span class="label">67</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Aguero</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Fernandez-Pinero</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Buitrago</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sanchez</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Elizalde</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">Bagaza virus in partridges and pheasants, Spain, 2010</span>. 
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="volume">17</span>: 
                     <span tagx="fpage">1498</span>–
                     <span tagx="lpage">1501</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.3201/eid1708.110077</div>
                     </div>
                     <span class="pub-id'">[21801633]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref068"></a>
                  <span class="label">68</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gamino</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gutierrez-Guzman</span>
                        <span tagx="given-names">AV</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Fernandez-de-Mera</span>
                        <span tagx="given-names">IG</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ortiz</span>
                        <span tagx="given-names">JA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Duran-Martin</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Natural Bagaza virus infection in game birds in southern Spain</span>. 
                     <span tagx="source">Veterinary research</span>
                     <span tagx="volume">43</span>: 
                     <span tagx="fpage">65</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1186/1297-9716-43-65</div>
                     </div>
                     <span class="pub-id'">[22966904]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref069"></a>
                  <span class="label">69</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Bondre</span>
                        <span tagx="given-names">VP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sapkal</span>
                        <span tagx="given-names">GN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Yergolkar</span>
                        <span tagx="given-names">PN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Fulmali</span>
                        <span tagx="given-names">PV</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sankararaman</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Genetic characterization of Bagaza virus (BAGV) isolated in India and evidence of
                        anti-BAGV antibodies in sera collected from encephalitis patients
                     </span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">90</span>: 
                     <span tagx="fpage">2644</span>–
                     <span tagx="lpage">2649</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1099/vir.0.012336-0</div>
                     </div>
                     <span class="pub-id'">[19570951]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref070"></a>
                  <span class="label">70</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Fernandez-Pinero</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Davidson</span>
                        <div tagxxx="suffix">II</div>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Elizalde</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Perk</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Khinich</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2014</span>) 
                     <span tagx="article-title">Bagaza virus and Israel turkey meningoencephalomyelitis virus are a single virus species</span>. 
                     <span tagx="source">J Gen Virol</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1099/vir.0.000037</div>
                     </div>
                     <span class="pub-id'">[25537376]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref071"></a>
                  <span class="label">71</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Pandey</span>
                        <span tagx="given-names">BD</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Karabatsos</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Cropp</span>
                        <span tagx="given-names">B</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tagaki</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tsuda</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">1999</span>) 
                     <span tagx="article-title">Identification of a flavivirus isolated from mosquitos in Chiang Mai Thailand</span>. 
                     <span tagx="source">Southeast Asian J Trop Med Public Health</span>
                     <span tagx="volume">30</span>: 
                     <span tagx="fpage">161</span>–
                     <span tagx="lpage">165</span>. 
                     <span class="pub-id'">[10695806]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref072"></a>
                  <span class="label">72</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Kono</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tsukamoto</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Abd Hamid</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Darus</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lian</span>
                        <span tagx="given-names">TC</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2000</span>) 
                     <span tagx="article-title">Encephalitis and retarded growth of chicks caused by Sitiawan virus, a new isolate
                        belonging to the genus Flavivirus
                     </span>. 
                     <span tagx="source">Am J Trop Med Hyg</span>
                     <span tagx="volume">63</span>: 
                     <span tagx="fpage">94</span>–
                     <span tagx="lpage">101</span>. 
                     <span class="pub-id'">[11358004]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref073"></a>
                  <span class="label">73</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Su</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Li</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hu</span>
                        <span tagx="given-names">X</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Yu</span>
                        <span tagx="given-names">X</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Wang</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">Duck egg-drop syndrome caused by BYD virus, a new Tembusu-related flavivirus</span>. 
                     <span tagx="source">PLoS One</span>
                     <span tagx="volume">6</span>: 
                     <span tagx="fpage">e18106</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1371/journal.pone.0018106</div>
                     </div>
                     <span class="pub-id'">[21455312]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref074"></a>
                  <span class="label">74</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Dilcher</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sall</span>
                        <span tagx="given-names">AA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hufert</span>
                        <span tagx="given-names">FT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Weidmann</span>
                        <span tagx="given-names">M</span>,
                     </span> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">Full-length genome sequence of Ntaya virus</span>. 
                     <span tagx="source">Virus Genes</span>
                     <span tagx="volume">46</span>: 
                     <span tagx="fpage">162</span>–
                     <span tagx="lpage">164</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1007/s11262-012-0825-7</div>
                     </div>
                     <span class="pub-id'">[22996737]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref075"></a>
                  <span class="label">75</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Boughton</span>
                        <span tagx="given-names">CR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hawkes</span>
                        <span tagx="given-names">RA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Naim</span>
                        <span tagx="given-names">HM</span>,
                     </span> (
                     <span tagx="year">1986</span>) 
                     <span tagx="article-title">Illness caused by a Kokobera-like virus in south-eastern Australia</span>. 
                     <span tagx="source">The Medical journal of Australia</span>
                     <span tagx="volume">145</span>: 
                     <span tagx="fpage">90</span>–
                     <span tagx="lpage">92</span>. 
                     <span class="pub-id'">[3016489]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref076"></a>
                  <span class="label">76</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">May</span>
                        <span tagx="given-names">FJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Clark</span>
                        <span tagx="given-names">DC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Pham</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Diviney</span>
                        <span tagx="given-names">SM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Williams</span>
                        <span tagx="given-names">DT</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">Genetic divergence among members of the Kokobera group of flaviviruses supports their
                        separation into distinct species
                     </span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">94</span>: 
                     <span tagx="fpage">1462</span>–
                     <span tagx="lpage">1467</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1099/vir.0.049940-0</div>
                     </div>
                     <span class="pub-id'">[23426358]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref077"></a>
                  <span class="label">77</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Hobson-Peters</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Yam</span>
                        <span tagx="given-names">AW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lu</span>
                        <span tagx="given-names">JW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Setoh</span>
                        <span tagx="given-names">YX</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">May</span>
                        <span tagx="given-names">FJ</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">A new insect-specific flavivirus from northern Australia suppresses replication of
                        West Nile virus and Murray Valley encephalitis virus in co-infected mosquito cells
                     </span>. 
                     <span tagx="source">PLoS One</span>
                     <span tagx="volume">8</span>: 
                     <span tagx="fpage">e56534</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1371/journal.pone.0056534</div>
                     </div>
                     <span class="pub-id'">[23460804]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref078"></a>
                  <span class="label">78</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Bloom</span>
                        <span tagx="given-names">KJ</span>,
                     </span> (
                     <span tagx="year">1993</span>) 
                     <span tagx="source">The Mississippi Valley’s Great Yellow Fever Epidemic of 1878</span>. In: 
                     <span class="name'">
                        <span tagx="surname">Press</span>
                        <span tagx="given-names">LSU</span>,
                     </span>, editor. 
                     <span class="publisher-name'">[Medical History LA]</span>: 
                     <span class="publisher-loc'">[BatonRouge]</span> pp. 
                     <span tagx="fpage">1</span>–
                     <span tagx="lpage">290</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1111/gwat.12306</div>
                     </div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref079"></a>
                  <span class="label">79</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Heinze</span>
                        <span tagx="given-names">DM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Forrester</span>
                        <span tagx="given-names">NL</span>,
                     </span> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Revisiting the clinal concept of evolution and dispersal for the tick-borne flaviviruses
                        by using phylogenetic and biogeographic analyses
                     </span>. 
                     <span tagx="source">J Virol</span>
                     <span tagx="volume">86</span>: 
                     <span tagx="fpage">8663</span>–
                     <span tagx="lpage">8671</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1128/JVI.01013-12</div>
                     </div>
                     <span class="pub-id'">[22674986]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref080"></a>
                  <span class="label">80</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">McGuire</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">EC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gao</span>
                        <span tagx="given-names">GF</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Reid</span>
                        <span tagx="given-names">HW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span> (
                     <span tagx="year">1998</span>) 
                     <span tagx="article-title">Tracing the origins of louping ill virus by molecular phylogenetic analysis</span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">79</span> (
                     <div tagxxx="issue">Pt 5</div>): 
                     <span tagx="fpage">981</span>–
                     <span tagx="lpage">988</span>.
                     <span class="pub-id'">[9603312]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref081"></a>
                  <span class="label">81</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Mann</span>
                        <span tagx="given-names">BR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">McMullen</span>
                        <span tagx="given-names">AR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Swetnam</span>
                        <span tagx="given-names">DM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Barrett</span>
                        <span tagx="given-names">AD</span>,
                     </span> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">Molecular epidemiology and evolution of West Nile virus in North America</span>. 
                     <span tagx="source">International journal of environmental research and public health</span>
                     <span tagx="volume">10</span>: 
                     <span tagx="fpage">5111</span>–
                     <span tagx="lpage">5129</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.3390/ijerph10105111</div>
                     </div>
                     <span class="pub-id'">[24135819]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref082"></a>
                  <span class="label">82</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Leonova</span>
                        <span tagx="given-names">GN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kondratov</span>
                        <span tagx="given-names">IG</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ternovoi</span>
                        <span tagx="given-names">VA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Romanova</span>
                        <span tagx="given-names">EV</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Protopopova</span>
                        <span tagx="given-names">EV</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Characterization of Powassan viruses from Far Eastern Russia</span>. 
                     <span tagx="source">Arch Virol</span>
                     <span tagx="volume">154</span>: 
                     <span tagx="fpage">811</span>–
                     <span tagx="lpage">820</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1007/s00705-009-0376-y</div>
                     </div>
                     <span class="pub-id'">[19363586]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref083"></a>
                  <span class="label">83</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gritsun</span>
                        <span tagx="given-names">TS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nuttall</span>
                        <span tagx="given-names">PA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span> (
                     <span tagx="year">2003</span>) 
                     <span tagx="article-title">Tick-borne flaviviruses</span>. 
                     <span tagx="source">Adv Virus Res</span>
                     <span tagx="volume">61</span>: 
                     <span tagx="fpage">317</span>–
                     <span tagx="lpage">371</span>. 
                     <span class="pub-id'">[14714436]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref084"></a>
                  <span class="label">84</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Telford</span>
                        <span tagx="given-names">SR</span>
                        <div tagxxx="suffix">3rd</div>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Armstrong</span>
                        <span tagx="given-names">PM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Katavolos</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Foppa</span>
                        <span tagx="given-names">I</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Garcia</span>
                        <span tagx="given-names">AS</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">1997</span>) 
                     <span tagx="article-title">A new tick-borne encephalitis-like virus infecting New England deer ticks, Ixodes
                        dammini
                     </span>. 
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="volume">3</span>: 
                     <span tagx="fpage">165</span>–
                     <span tagx="lpage">170</span>. 
                     <span class="pub-id'">[9204297]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref085"></a>
                  <span class="label">85</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Pettersson</span>
                        <span tagx="given-names">JH</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Fiz-Palacios</span>
                        <span tagx="given-names">O</span>,
                     </span> (
                     <span tagx="year">2014</span>) 
                     <span tagx="article-title">Dating the origin of the genus Flavivirus in the light of Beringian biogeography</span>. 
                     <span tagx="source">J Gen Virol</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1099/vir.0.000037</div>
                     </div>
                     <span class="pub-id'">[25537376]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref086"></a>
                  <span class="label">86</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gao</span>
                        <span tagx="given-names">X</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Liu</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Wang</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Fu</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Guo</span>
                        <span tagx="given-names">Z</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">Southernmost Asia is the source of Japanese encephalitis virus (genotype 1) diversity
                        from which the viruses disperse and evolve throughout Asia
                     </span>. 
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="volume">7</span>: 
                     <span tagx="fpage">e2459</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1371/journal.pntd.0002459</div>
                     </div>
                     <span class="pub-id'">[24069502]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref087"></a>
                  <span class="label">87</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Firth</span>
                        <span tagx="given-names">AE</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Brierley</span>
                        <span tagx="given-names">I</span>,
                     </span> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Non-canonical translation in RNA viruses</span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">93</span>: 
                     <span tagx="fpage">1385</span>–
                     <span tagx="lpage">1409</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1099/vir.0.042499-0</div>
                     </div>
                     <span class="pub-id'">[22535777]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref088"></a>
                  <span class="label">88</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Firth</span>
                        <span tagx="given-names">AE</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Atkins</span>
                        <span tagx="given-names">JF</span>,
                     </span> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">A conserved predicted pseudoknot in the NS2A-encoding sequence of West Nile and Japanese
                        encephalitis flaviviruses suggests NS1' may derive from ribosomal frameshifting
                     </span>. 
                     <span tagx="source">Virol J</span>
                     <span tagx="volume">6</span>: 
                     <span tagx="fpage">14</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1186/1743-422X-6-14</div>
                     </div>
                     <span class="pub-id'">[19196463]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref089"></a>
                  <span class="label">89</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Melian</span>
                        <span tagx="given-names">EB</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hinzman</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nagasaki</span>
                        <span tagx="given-names">T</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Firth</span>
                        <span tagx="given-names">AE</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Wills</span>
                        <span tagx="given-names">NM</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">NS1' of flaviviruses in the Japanese encephalitis virus serogroup is a product of
                        ribosomal frameshifting and plays a role in viral neuroinvasiveness
                     </span>. 
                     <span tagx="source">J Virol</span>
                     <span tagx="volume">84</span>: 
                     <span tagx="fpage">1641</span>–
                     <span tagx="lpage">1647</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1128/JVI.01979-09</div>
                     </div>
                     <span class="pub-id'">[19906906]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref090"></a>
                  <span class="label">90</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Firth</span>
                        <span tagx="given-names">AE</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Blitvich</span>
                        <span tagx="given-names">BJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Wills</span>
                        <span tagx="given-names">NM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Miller</span>
                        <span tagx="given-names">CL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Atkins</span>
                        <span tagx="given-names">JF</span>,
                     </span> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">Evidence for ribosomal frameshifting and a novel overlapping gene in the genomes of
                        insect-specific flaviviruses
                     </span>. 
                     <span tagx="source">Virology</span>
                     <span tagx="volume">399</span>: 
                     <span tagx="fpage">153</span>–
                     <span tagx="lpage">166</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/j.virol.2009.12.033</div>
                     </div>
                     <span class="pub-id'">[20097399]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref091"></a>
                  <span class="label">91</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Clyde</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Harris</span>
                        <span tagx="given-names">E</span>,
                     </span> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">RNA secondary structure in the coding region of dengue virus type 2 directs translation
                        start codon selection and is required for viral replication
                     </span>. 
                     <span tagx="source">J Virol</span>
                     <span tagx="volume">80</span>: 
                     <span tagx="fpage">2170</span>–
                     <span tagx="lpage">2182</span>. 
                     <span class="pub-id'">[16474125]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref092"></a>
                  <span class="label">92</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Clyde</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Barrera</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Harris</span>
                        <span tagx="given-names">E</span>,
                     </span> (
                     <span tagx="year">2008</span>) 
                     <span tagx="article-title">The capsid-coding region hairpin element (cHP) is a critical determinant of dengue
                        virus and West Nile virus RNA synthesis
                     </span>. 
                     <span tagx="source">Virology</span>
                     <span tagx="volume">379</span>: 
                     <span tagx="fpage">314</span>–
                     <span tagx="lpage">323</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/j.virol.2008.06.034</div>
                     </div>
                     <span class="pub-id'">[18676000]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref093"></a>
                  <span class="label">93</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Groat-Carmona</span>
                        <span tagx="given-names">AM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Orozco</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Friebe</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Payne</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kramer</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">A novel coding-region RNA element modulates infectious dengue virus particle production
                        in both mammalian and mosquito cells and regulates viral replication in Aedes aegypti
                        mosquitoes
                     </span>. 
                     <span tagx="source">Virology</span>
                     <span tagx="volume">432</span>: 
                     <span tagx="fpage">511</span>–
                     <span tagx="lpage">526</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/j.virol.2012.06.028</div>
                     </div>
                     <span class="pub-id'">[22840606]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref094"></a>
                  <span class="label">94</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Liu</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Wimmer</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Paul</span>
                        <span tagx="given-names">AV</span>,
                     </span> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Cis-acting RNA elements in human and animal plus-strand RNA viruses</span>. 
                     <span tagx="source">Biochimica et biophysica acta</span>
                     <span tagx="volume">1789</span>: 
                     <span tagx="fpage">495</span>–
                     <span tagx="lpage">517</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/j.bbagrm.2009.09.007</div>
                     </div>
                     <span class="pub-id'">[19781674]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref095"></a>
                  <span class="label">95</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Tuplin</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Evans</span>
                        <span tagx="given-names">DJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Buckley</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Jones</span>
                        <span tagx="given-names">IM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">Replication enhancer elements within the open reading frame of tick-borne encephalitis
                        virus and their evolution within the Flavivirus genus
                     </span>. 
                     <span tagx="source">Nucleic acids research</span>
                     <span tagx="volume">39</span>: 
                     <span tagx="fpage">7034</span>–
                     <span tagx="lpage">7048</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1093/nar/gkr237</div>
                     </div>
                     <span class="pub-id'">[21622960]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref096"></a>
                  <span class="label">96</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Khromykh</span>
                        <span tagx="given-names">AA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Meka</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Guyatt</span>
                        <span tagx="given-names">KJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Westaway</span>
                        <span tagx="given-names">EG</span>,
                     </span> (
                     <span tagx="year">2001</span>) 
                     <span tagx="article-title">Essential role of cyclization sequences in flavivirus RNA replication</span>. 
                     <span tagx="source">J Virol</span>
                     <span tagx="volume">75</span>: 
                     <span tagx="fpage">6719</span>–
                     <span tagx="lpage">6728</span>. 
                     <span class="pub-id'">[11413342]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref097"></a>
                  <span class="label">97</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Domier</span>
                        <span tagx="given-names">LL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">McCoppin</span>
                        <span tagx="given-names">NK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Larsen</span>
                        <span tagx="given-names">RC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">D'Arcy</span>
                        <span tagx="given-names">CJ</span>,
                     </span> (
                     <span tagx="year">2002</span>) 
                     <span tagx="article-title">Nucleotide sequence shows that Bean leafroll virus has a Luteovirus-like genome organization</span>. 
                     <span tagx="source">Journal of General Virology</span>
                     <span tagx="volume">83</span>: 
                     <span tagx="fpage">1791</span>–
                     <span tagx="lpage">1798</span>. 
                     <span class="pub-id'">[12075101]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref098"></a>
                  <span class="label">98</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Loughran</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Firth</span>
                        <span tagx="given-names">AE</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Atkins</span>
                        <span tagx="given-names">JF</span>,
                     </span> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">Ribosomal frameshifting into an overlapping gene in the 2B-encoding region of the
                        cardiovirus genome
                     </span>. 
                     <span tagx="source">Proc Natl Acad Sci U S A</span>
                     <span tagx="volume">108</span>: 
                     <span tagx="fpage">E1111</span>–
                     <span tagx="lpage">1119</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1073/pnas.1102932108</div>
                     </div>
                     <span class="pub-id'">[22025686]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref099"></a>
                  <span class="label">99</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Fang</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Treffers</span>
                        <span tagx="given-names">EE</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Li</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tas</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sun</span>
                        <span tagx="given-names">Z</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Efficient-2 frameshifting by mammalian ribosomes to synthesize an additional arterivirus
                        protein
                     </span>. 
                     <span tagx="source">Proc Natl Acad Sci U S A</span>
                     <span tagx="volume">109</span>: 
                     <span tagx="fpage">E2920</span>–
                     <span tagx="lpage">2928</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1073/pnas.1211145109</div>
                     </div>
                     <span class="pub-id'">[23043113]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref100"></a>
                  <span class="label">100</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Mattingly</span>
                        <span tagx="given-names">PF</span>,
                     </span> (
                     <span tagx="year">1960</span>) 
                     <span tagx="article-title">Ecological aspects of the evolution of mosquito-borne virus diseases</span>. 
                     <span tagx="source">Transactions of the Royal Society of Tropical Medicine &amp; Hygiene</span>
                     <span tagx="volume">54</span>: 
                     <span tagx="fpage">97</span>–
                     <span tagx="lpage">112</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1093/trstmh/tru112</div>
                     </div>
                     <span class="pub-id'">[25059525]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref101"></a>
                  <span class="label">101</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Kuno</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chang</span>
                        <span tagx="given-names">GJ</span>,
                     </span> (
                     <span tagx="year">2005</span>) 
                     <span tagx="article-title">Biological transmission of arboviruses: reexamination of and new insights into components,
                        mechanisms, and unique traits as well as their evolutionary trends
                     </span>. 
                     <span tagx="source">Clin Microbiol Rev</span>
                     <span tagx="volume">18</span>: 
                     <span tagx="fpage">608</span>–
                     <span tagx="lpage">637</span>. 
                     <span class="pub-id'">[16223950]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref102"></a>
                  <span class="label">102</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Roiz</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vazquez</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Seco</span>
                        <span tagx="given-names">MP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tenorio</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rizzoli</span>
                        <span tagx="given-names">A</span>,
                     </span> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Detection of novel insect flavivirus sequences integrated in Aedes albopictus (Diptera:
                        Culicidae) in Northern Italy
                     </span>. 
                     <span tagx="source">Virol J</span>
                     <span tagx="volume">6</span>: 
                     <span tagx="fpage">93</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1186/1743-422X-6-93</div>
                     </div>
                     <span class="pub-id'">[19575816]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref103"></a>
                  <span class="label">103</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Sanchez-Seco</span>
                        <span tagx="given-names">MP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vazquez</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Collao</span>
                        <span tagx="given-names">X</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hernandez</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Aranda</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Surveillance of arboviruses in Spanish wetlands: detection of new flavi- and phleboviruses</span>. 
                     <span tagx="source">Vector Borne Zoonotic Dis</span>
                     <span tagx="volume">10</span>: 
                     <span tagx="fpage">203</span>–
                     <span tagx="lpage">206</span>.
                     <span class="pub-id'">[19485777]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref104"></a>
                  <span class="label">104</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Edwards</span>
                        <span tagx="given-names">FW</span>,
                     </span> (
                     <span tagx="year">1922</span>) 
                     <span tagx="article-title">On Some Malayan and Other Species of Culicoides, with a Note on the Genus Lasiohelea</span>. 
                     <span tagx="source">Bulletin of entomological research</span>
                     <span tagx="volume">13</span>: 
                     <span tagx="fpage">161</span>–
                     <span tagx="lpage">167</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref105"></a>
                  <span class="label">105</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Laird</span>
                        <span tagx="given-names">M</span>,
                     </span> (
                     <span tagx="year">1946</span>) 
                     <span tagx="article-title">A ceratopogonine midge (Culicoides anophelis Edwards, 1922) sucking engorged blood
                        from a mosquito (Armigeres lacuum Edwards, 1922) at Palmalmal, New Britain
                     </span>. 
                     <span tagx="source">Transactions and Proceedings of the Royal Society of New Zealand</span>
                     <span tagx="volume">76</span>: 
                     <span tagx="fpage">158</span>–
                     <span tagx="lpage">161</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref106"></a>
                  <span class="label">106</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Chu Feng</span>
                        <span tagx="given-names">I</span>,
                     </span> (
                     <span tagx="year">1959</span>) 
                     <span tagx="article-title">FOUR SPECIES OF THE GENUS CULICOIDES (DIPTERA, HELEIDAE) FROM HAINAN ISLAND, SOUTH
                        CHINA
                     </span>. 
                     <span tagx="source">ACTA ENTOMOLOGICA SINICA</span>
                     <span tagx="volume">9</span>: 
                     <span tagx="fpage">161</span>–
                     <span tagx="lpage">166</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref107"></a>
                  <span class="label">107</span>
                  <div tagxxx="mixed-citation">Chhilar JS, Chaudhry S (2011) First Report of a Biting Midge Culicoides Anophelis
                     Parasitzing Mosquito Anopheles stephensi from North-Western India.
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref108"></a>
                  <span class="label">108</span>
                  <div tagxxx="mixed-citation">Reddy CVS, Hafeez M (2011) Studies on certain aspects of prevalence of Culicoides
                     species.
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref109"></a>
                  <span class="label">109</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Coutard</span>
                        <span tagx="given-names">B</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Malet</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Morin</span>
                        <span tagx="given-names">B</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Jamal</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Weaver</span>
                        <span tagx="given-names">SC</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Understanding the alphaviruses: Recent research on important emerging pathogens and
                        progress towards their control
                     </span>. 
                     <span tagx="source">Antiviral Research</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/j.antiviral.2009.12.007</div>
                     </div>
                     <span class="pub-id'">[20035788]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref110"></a>
                  <span class="label">110</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">de Lamballerie</span>
                        <span tagx="given-names">X</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Zanotto</span>
                        <span tagx="given-names">PM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">EC</span>,
                     </span> (
                     <span tagx="year">2001</span>) 
                     <span tagx="article-title">Evolution, epidemiology, and dispersal of flaviviruses revealed by molecular phylogenies</span>. 
                     <span tagx="source">Adv Virus Res</span>
                     <span tagx="volume">57</span>: 
                     <span tagx="fpage">71</span>–
                     <span tagx="lpage">103</span>. 
                     <span class="pub-id'">[11680389]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref111"></a>
                  <span class="label">111</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gubler</span>
                        <span tagx="given-names">DJ</span>,
                     </span> (
                     <span tagx="year">1987</span>) 
                     <span tagx="article-title">Dengue and dengue hemorrhagic fever in the Americas</span>. 
                     <span tagx="source">Puerto Rico health sciences journal</span>
                     <span tagx="volume">6</span>: 
                     <span tagx="fpage">107</span>–
                     <span tagx="lpage">111</span>. 
                     <span class="pub-id'">[3313490]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref112"></a>
                  <span class="label">112</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Strode</span>
                        <span tagx="given-names">GK</span>,
                     </span> (
                     <span tagx="year">1951</span>) 
                     <span tagx="source">Yellow Fever</span>. 
                     <span class="publisher-loc'">[New York]</span>: 
                     <span class="publisher-name'">[McGraw-Hill]</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref113"></a>
                  <span class="label">113</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Kramer</span>
                        <span tagx="given-names">LD</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chandler</span>
                        <span tagx="given-names">LJ</span>,
                     </span> (
                     <span tagx="year">2001</span>) 
                     <span tagx="article-title">Phylogenetic analysis of the envelope gene of St. Louis encephalitis virus</span>. 
                     <span tagx="source">Arch Virol</span>
                     <span tagx="volume">146</span>: 
                     <span tagx="fpage">2341</span>–
                     <span tagx="lpage">2355</span>. 
                     <span class="pub-id'">[11811684]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref114"></a>
                  <span class="label">114</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Moss</span>
                        <span tagx="given-names">SR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Turner</span>
                        <span tagx="given-names">SL</span>,
                     </span> (
                     <span tagx="year">2004</span>) 
                     <span tagx="article-title">Evolution and dispersal of encephalitic flaviviruses</span>. 
                     <span tagx="source">Arch Virol Suppl</span>: 
                     <span tagx="fpage">65</span>–
                     <span tagx="lpage">84</span>. 
                     <span class="pub-id'">[15119763]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref115"></a>
                  <span class="label">115</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Allicock</span>
                        <span tagx="given-names">OM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lemey</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tatem</span>
                        <span tagx="given-names">AJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Pybus</span>
                        <span tagx="given-names">OG</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Bennett</span>
                        <span tagx="given-names">SN</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Phylogeography and population dynamics of dengue viruses in the Americas</span>. 
                     <span tagx="source">Mol Biol Evol</span>
                     <span tagx="volume">29</span>: 
                     <span tagx="fpage">1533</span>–
                     <span tagx="lpage">1543</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1093/molbev/msr320</div>
                     </div>
                     <span class="pub-id'">[22319149]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref116"></a>
                  <span class="label">116</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gubler</span>
                        <span tagx="given-names">DJ</span>,
                     </span> (
                     <span tagx="year">1997</span>) 
                     <div tagxxx="chapter-title">The emergence of dengue/dengue haemorrhagic fever as a global public health problem</div> In: 
                     <span class="name'">
                        <span tagx="surname">Saluzzo</span>
                        <span tagx="given-names">JF</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Dodet</span>
                        <span tagx="given-names">B</span>,
                     </span>, editors. 
                     <span tagx="source">Factors in the Emergence of Arbovirus Diseases</span>. 
                     <span class="publisher-loc'">[Paris]</span>: 
                     <span class="publisher-name'">[Elsevier]</span> pp. 
                     <span tagx="fpage">83</span>–
                     <span tagx="lpage">92</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref117"></a>
                  <span class="label">117</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gubler</span>
                        <span tagx="given-names">DJ</span>,
                     </span> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">Dengue/dengue haemorrhagic fever: history and current status</span>. 
                     <span tagx="source">Novartis Foundation Symposium</span>
                     <span tagx="volume">277</span>: 
                     <span tagx="fpage">3</span>–
                     <span tagx="lpage">16</span>; discussion 16–22, 71–13, 251–253. 
                     <span class="pub-id'">[17319151]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref118"></a>
                  <span class="label">118</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Briese</span>
                        <span tagx="given-names">T</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Jia</span>
                        <span tagx="given-names">XY</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Huang</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Grady</span>
                        <span tagx="given-names">LJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lipkin</span>
                        <span tagx="given-names">WI</span>,
                     </span> (
                     <span tagx="year">1999</span>) 
                     <span tagx="article-title">Identification of a Kunjin/West Nile-like flavivirus in brains of patients with New
                        York encephalitis
                     </span>. 
                     <span tagx="source">Lancet</span>
                     <span tagx="volume">354</span>: 
                     <span tagx="fpage">1261</span>–
                     <span tagx="lpage">1262</span>. 
                     <span class="pub-id'">[10520637]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref119"></a>
                  <span class="label">119</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Lanciotti</span>
                        <span tagx="given-names">RS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Roehrig</span>
                        <span tagx="given-names">JT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Deubel</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Smith</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Parker</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">1999</span>) 
                     <span tagx="article-title">Origin of the West Nile virus responsible for an outbreak of encephalitis in the northeastern
                        United States
                     </span>. 
                     <span tagx="source">Science</span>
                     <span tagx="volume">286</span>: 
                     <span tagx="fpage">2333</span>–
                     <span tagx="lpage">2337</span>. 
                     <span class="pub-id'">[10600742]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref120"></a>
                  <span class="label">120</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Reisen</span>
                        <span tagx="given-names">WK</span>,
                     </span> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">Ecology of West Nile virus in North America</span>. 
                     <span tagx="source">Viruses</span>
                     <span tagx="volume">5</span>: 
                     <span tagx="fpage">2079</span>–
                     <span tagx="lpage">2105</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.3390/v5092079</div>
                     </div>
                     <span class="pub-id'">[24008376]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref121"></a>
                  <span class="label">121</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Roehrig</span>
                        <span tagx="given-names">JT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Layton</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Smith</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Campbell</span>
                        <span tagx="given-names">GL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nasci</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2002</span>) 
                     <span tagx="article-title">The emergence of West Nile virus in North America: ecology, epidemiology, and surveillance</span>. 
                     <span tagx="source">Curr Top Microbiol Immunol</span>
                     <span tagx="volume">267</span>: 
                     <span tagx="fpage">223</span>–
                     <span tagx="lpage">240</span>. 
                     <span class="pub-id'">[12082991]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref122"></a>
                  <span class="label">122</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Huanyu</span>
                        <span tagx="given-names">W</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Haiyan</span>
                        <span tagx="given-names">W</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Shihong</span>
                        <span tagx="given-names">F</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Guifang</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hong</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Isolation and identification of a distinct strain of Culex Flavivirus from mosquitoes
                        collected in Mainland China
                     </span>. 
                     <span tagx="source">Virol J</span>
                     <span tagx="volume">9</span>: 
                     <span tagx="fpage">73</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1186/1743-422X-9-73</div>
                     </div>
                     <span class="pub-id'">[22452813]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref123"></a>
                  <span class="label">123</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Morales-Betoulle</span>
                        <span tagx="given-names">ME</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Monzon Pineda</span>
                        <span tagx="given-names">ML</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sosa</span>
                        <span tagx="given-names">SM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Panella</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lopez</span>
                        <span tagx="given-names">MR</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2008</span>) 
                     <span tagx="article-title">Culex flavivirus isolates from mosquitoes in Guatemala</span>. 
                     <span tagx="source">J Med Entomol</span>
                     <span tagx="volume">45</span>: 
                     <span tagx="fpage">1187</span>–
                     <span tagx="lpage">1190</span>. 
                     <span class="pub-id'">[19058647]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref124"></a>
                  <span class="label">124</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Harbach</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kitching</span>
                        <span tagx="given-names">IJ</span>,
                     </span> (
                     <span tagx="year">1998</span>) 
                     <span tagx="article-title">Phylogeny and classification of the Culicidae (Diptera)</span>. 
                     <span tagx="source">Systematic Entomology</span>
                     <span tagx="volume">23</span>: 
                     <span tagx="fpage">327</span>–
                     <span tagx="lpage">370</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref125"></a>
                  <span class="label">125</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Charrel</span>
                        <span tagx="given-names">RN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">de Lamballerie</span>
                        <span tagx="given-names">X</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Fulhorst</span>
                        <span tagx="given-names">CF</span>,
                     </span> (
                     <span tagx="year">2001</span>) 
                     <span tagx="article-title">The Whitewater Arroyo virus: natural evidence for genetic recombination among Tacaribe
                        serocomplex viruses (family Arenaviridae)
                     </span>. 
                     <span tagx="source">Virology</span>
                     <span tagx="volume">283</span>: 
                     <span tagx="fpage">161</span>–
                     <span tagx="lpage">166</span>. 
                     <span class="pub-id'">[11336541]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref126"></a>
                  <span class="label">126</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Engel</span>
                        <span tagx="given-names">SR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hogan</span>
                        <span tagx="given-names">KM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Taylor</span>
                        <span tagx="given-names">JF</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Davis</span>
                        <span tagx="given-names">SK</span>,
                     </span> (
                     <span tagx="year">1998</span>) 
                     <span tagx="article-title">Molecular systematics and paleobiogeography of the South American sigmodontine rodents</span>. 
                     <span tagx="source">Mol Biol Evol</span>
                     <span tagx="volume">15</span>: 
                     <span tagx="fpage">35</span>–
                     <span tagx="lpage">49</span>. 
                     <span class="pub-id'">[9491603]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref127"></a>
                  <span class="label">127</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Hughes</span>
                        <span tagx="given-names">AL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Friedman</span>
                        <span tagx="given-names">R</span>,
                     </span> (
                     <span tagx="year">2000</span>) 
                     <span tagx="article-title">Evolutionary diversification of protein-coding genes of hantaviruses</span>. 
                     <span tagx="source">Mol Biol Evol</span>
                     <span tagx="volume">17</span>: 
                     <span tagx="fpage">1558</span>–
                     <span tagx="lpage">1568</span>. 
                     <span class="pub-id'">[11018161]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref128"></a>
                  <span class="label">128</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Ramsden</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">EC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Charleston</span>
                        <span tagx="given-names">MA</span>,
                     </span> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Hantavirus evolution in relation to its rodent and insectivore hosts: no evidence
                        for codivergence
                     </span>. 
                     <span tagx="source">Mol Biol Evol</span>
                     <span tagx="volume">26</span>: 
                     <span tagx="fpage">143</span>–
                     <span tagx="lpage">153</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1093/molbev/msn234</div>
                     </div>
                     <span class="pub-id'">[18922760]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref129"></a>
                  <span class="label">129</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Ramsden</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Melo</span>
                        <span tagx="given-names">FL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Figueiredo</span>
                        <span tagx="given-names">LM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">EC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Zanotto</span>
                        <span tagx="given-names">PM</span>,
                     </span> (
                     <span tagx="year">2008</span>) 
                     <span tagx="article-title">High rates of molecular evolution in hantaviruses</span>. 
                     <span tagx="source">Mol Biol Evol</span>
                     <span tagx="volume">25</span>: 
                     <span tagx="fpage">1488</span>–
                     <span tagx="lpage">1492</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1093/molbev/msn093</div>
                     </div>
                     <span class="pub-id'">[18417484]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref130"></a>
                  <span class="label">130</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Sun</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Yu</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Deubel</span>
                        <span tagx="given-names">V</span>,
                     </span> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Japanese encephalitis virus NS1' protein depends on pseudoknot secondary structure
                        and is cleaved by caspase during virus infection and cell apoptosis
                     </span>. 
                     <span tagx="source">Microbes and infection / Institut Pasteur</span>
                     <span tagx="volume">14</span>: 
                     <span tagx="fpage">930</span>–
                     <span tagx="lpage">940</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/j.micinf.2012.03.007</div>
                     </div>
                     <span class="pub-id'">[22504173]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0117849.ref131"></a>
                  <span class="label">131</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Ye</span>
                        <span tagx="given-names">Q</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Li</span>
                        <span tagx="given-names">XF</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Zhao</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Li</span>
                        <span tagx="given-names">SH</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Deng</span>
                        <span tagx="given-names">YQ</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">A single nucleotide mutation in NS2A of Japanese encephalitis-live vaccine virus (SA14–14–2)
                        ablates NS1' formation and contributes to attenuation
                     </span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">93</span>: 
                     <span tagx="fpage">1959</span>–
                     <span tagx="lpage">1964</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1099/vir.0.043844-0</div>
                     </div>
                     <span class="pub-id'">[22739060]</span>
                  </div>
               </li>
            </ul>
         </div>
      </div>
   </div>
</html>