<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <div xmlns="" tagx="article">
      <head>
         <style type="text/css"> table, tr, td {border : thin;}</style>
         <span class="citation_publisher">BioMed Central</span>
         <h2 class="citation-title">3D structured illumination microscopy of mammalian embryos and spermatozoa</h2>
         <span class="doi">10.1186/s12861-015-0092-7</span>
         <div class="contrib-group">contrib: 
            	  CONT
            	  
            <span class="citation_author">Jens Popken</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Maik Dahlhoff</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Tuna Guengoer</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Eckhard Wolf</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Valeri Zakhartchenko</span>
            <span class="citation_author_institution_ref"></span>
            	AFF
            	  
            <span class="citation_author_institution">[ ]</span>
            	AFF
            	  
            <span class="citation_author_institution">[ ]</span>
            	AFF
            	  
            <span class="citation_author_institution">[ ]</span>
         </div>
         <span class="email">[]</span>
         <span class="conflict">[]</span>
         <span class="contribs">[]</span>
         <span class="citation_pubdate">[2015-11-26]</span>
         <span class="citation_copyright">[Copyright : ]</span>
         <span class="citation_licence">[Licence Open AccessThis article is distributed under the terms of the Creative Commons
            Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/),
            which permits unrestricted use, distribution, and reproduction in any medium, provided
            you give appropriate credit to the original author(s) and the source, provide a link
            to the Creative Commons license, and indicate if changes were made. The Creative Commons
            Public Domain Dedication waiver (http://creativecommons.org/publicdomain/zero/1.0/)
            applies to the data made available in this article, unless otherwise stated.]
         </span>
         <span class="citation_funding">[]</span>
         <span class="citation_pagecount">[]</span>
         <span class="citation_data">[]</span>
      </head>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <div id="">
            <h2>Background</h2>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Super-resolution fluorescence microscopy performed via 3D structured illumination
               microscopy (3D-SIM) is well established on flat, adherent cells. However, blastomeres
               of mammalian embryos are non-adherent, round and large. Scanning whole mount mammalian
               embryos with 3D-SIM is prone to failure due to the movement during scanning and the
               large distance to the cover glass.
            </p>
         </div>
         <div id="">
            <h2>Results</h2>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Here we present a highly detailed protocol that allows performing 3D-SIM on blastomeres
               of mammalian embryos with an image quality comparable to scans in adherent cells.
               This protocol was successfully tested on mouse, rabbit and cattle embryos and on rabbit
               spermatozoa.
            </p>
         </div>
         <div id="">
            <h2>Conclusions</h2>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Our protocol provides detailed instructions on embryo staining, blastomere isolation,
               blastomere attachment, embedding, correct oil predictions, scanning conditions, and
               oil correction choices after the first scan. Finally, the most common problems are
               documented and solutions are suggested. To our knowledge, this protocol presents for
               the first time a highly detailed and practical way to perform 3D-SIM on mammalian
               embryos and spermatozoa.
            </p>
         </div>
      </div>
      <div id="Sec1">
         <h2>Background</h2>
         <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">With an axial resolution of about 100 nm and a lateral resolution of about 300 nm
            3D structured illumination microscopy (3D-SIM) features an 8-fold volumetric resolution
            improvement over confocal microscopy [
            <span ref-type="bibr" rid="CR1">
               <sup>
                  <a href="#CR1">1</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="CR2">
               <sup>
                  <a href="#CR2">2</a>
               </sup>
            </span>]. This improvement is achieved by redirecting the laser beams through a mobile grid
            and scanning each section 15 times with a slightly altered grid location and orientation
            (3 angles with 5 phases each). The signals produced at the borders of this mobile
            grid allow a software to compute super-resolution images [
            <span ref-type="bibr" rid="CR3">
               <sup>
                  <a href="#CR3">3</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="CR6">
               <sup>
                  <a href="#CR6">6</a>
               </sup>
            </span>]. This scanning and calculating procedure has three requirements for specimens. Firstly,
            15 scans per section require superior bleach resistant signal providing molecules.
            This is especially important when scanning thick objects since the total exposure
            time throughout all sections increases with the total number of z-sections scanned.
            Secondly, all signal providing molecules must remain at their 3D location during the
            entire scanning process, otherwise the calculation of the final image stack will fail.
            Since complete stacks are scanned for each angle before rotating the grid, even the
            slightest movement may reduce image quality markedly. Even individual signal providing
            molecules moving through the object such as particles can greatly interfere with the
            algorithm. Thirdly, 3D-SIM is a form of wide-field microscopy. This means that sharp
            images can only be acquired in an area at a certain distance to the cover glass. Signals
            outside the optimal range will appear less crisp and image artifacts such as artificial
            generation of signals can occur. This requires the use of the correct oil for a specific
            distance to the cover glass. Because of these three requirements flat and adherent
            cells are the optimal choice for 3D-SIM. However, blastomeres of mammalian embryos
            are non-adherent, large and surrounded by the zona pellucida. We have developed a
            protocol for 3D-SIM on mammalian embryos and generated high quality data for multiple
            studies [
            <span ref-type="bibr" rid="CR7">
               <sup>
                  <a href="#CR7">7</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="CR8">
               <sup>
                  <a href="#CR8">8</a>
               </sup>
            </span>]. Here we present a detailed experimental protocol how to achieve superior image
            quality of mammalian blastomeres using 3D-SIM.
         </p>
      </div>
      <div id="Sec2">
         <h2>Methods</h2>
         <div id="Sec3">
            <h3>Ethics statement</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">All animal procedures and experiments with embryos were approved by the Government
               of Upper Bavaria (permit number 55.2-1-54-2532-34-09) and performed in accordance
               with the German Animal Welfare Act and European Union Normative for Care and Use of
               Experimental Animals.
            </p>
         </div>
         <div id="Sec4">
            <h3>Recovery and culture of mouse embryos</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Female mice were injected interperitoneally (IP) with pregnant mare’s serum (PMS)
               between 1:00 and 4:00 PM of Day 1. On Day 3, forty-two hours after the PMS injection,
               the mice received an IP injection of human chorionic gonadotropin (HCG). Immediately
               following injection, females of the FVB/N inbred strain were mated with males of the
               same mouse strain. Ovulation occurs approximately 12 h after HCG injection, at which
               time the eggs can be fertilized. Females were screened every morning and evening for
               vaginal plugs to see if mating has occurred and sacrificed by cervical dislocation
               at the same day after finding a vaginal plug (0.5 dpc). For embryo recovery, females
               were euthanized by cervical dislocation under isoflurane anesthesia. The oviduct was
               removed and flushed with M2 medium (Sigma, Taufkirchen, Germany) containing 0.4 %
               bovine serum albumin (BSA) (Roth, Karlsruhe, Germany). Zygotes were collected under
               a stereo microscope (Zeiss, Jena, Germany) with 20x magnification and transferred
               to microdrops of M16 medium (Sigma) on a culture dish covered with paraffin oil (Roth).
               Embryos were cultivated at 37 °C and 5 % CO
               <div tagxxx="sub">2</div> in an incubator until the appropriate stage for fixation.
            </p>
         </div>
         <div id="Sec5">
            <h3>Recovery and culture of rabbit embryos</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Recovery of embryos was performed as described [
               <span ref-type="bibr" rid="CR9">
                  <sup>
                     <a href="#CR9">9</a>
                  </sup>
               </span>]. Female Zika rabbits were first superovulated by injection of 100 IU (international
               units) of equine chorionic gonadotropin (ECG; Intergonan, Intervet) intramuscularly
               and 100 IU of human chorionic gonadotropin (HCG; Ovogest, Intervet) intravenously
               72 h later. 18-20 h post-HCG injection and after natural mating 
               <i>in vivo</i> fertilized zygotes were flushed from the explanted oviducts of rabbits in warm phosphate
               buffered saline (PBS) supplemented with 4 mg/ml bovine serum albumin. Rabbit embryos
               were cultured in Quinn’s medium (SAGE, Trumbull, CT) containing 2.5 % fetal calf serum
               (FCS) in a humidified atmosphere of 5 % CO
               <div tagxxx="sub">2</div> in air at 38.5 °C until the appropriate stage for fixation.
            </p>
         </div>
         <div id="Sec6">
            <h3>Recovery and culture of bovine embryos</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <i>In vitro</i> fertilization of bovine embryos was performed as described [
               <span ref-type="bibr" rid="CR10">
                  <sup>
                     <a href="#CR10">10</a>
                  </sup>
               </span>]. Cumulus-oocyte complexes (COCs) were obtained by aspiration from ovaries of slaughtered
               cows. COCs were matured in modified Parker’s medium containing TCM199 supplemented
               with 5 % estrous cow serum (ECS) and 0.2 U/ml o-FSH (Ovagen; ICPbio) for 20–22 h at
               39 °C, 5 % CO
               <div tagxxx="sub">2</div> in air and maximum humidity. Matured COCs were washed with the fertilization medium
               Tyrode's albumin lactate pyruvate (FERT-TALP) supplemented with sodium pyruvate (2.2 mg/ml),
               heparin sodium salt (2 mg/ml), and BSA (6 mg/ml) and transferred to 400-μl droplets
               of medium. Frozen spermatozoa were thawed at 38 °C. 100 μl thawed sperm suspension
               covered by 1 ml capacitation medium was subjected to the swim-up procedure for 60 min.
               The COCs and spermatozoa (2 x 10
               <sup>6</sup> cells/ml) were co-incubated for 18 h at 39 °C and 5 % CO
               <div tagxxx="sub">2</div> in humidified air. Presumptive zygotes were mechanically denuded by vortexing, washed
               3x in SOF culture medium enriched with 5 % ECS, BME 100x (20 μl/ml; Invitrogen) and
               MEM (Minimum Essential Medium) 100x (10 μl/ml, Invitrogen), and transferred to 400-μl
               droplets of medium covered with mineral oil. Embryos were grown at 39 °C in a humidified
               atmosphere of 5 % CO
               <div tagxxx="sub">2</div>, 5 % O
               <div tagxxx="sub">2</div>, and 90 % N
               <div tagxxx="sub">2</div> until they reached the appropriate stage for fixation.
            </p>
         </div>
         <div id="Sec7">
            <h3>Cover glass preparation</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Clean cover glasses were covered by polylysine (33 μl of a 1:100 Polylysine dilution
               in aqua bidest for 5 min) applying it in the exact center of each cover glass. If
               the covered area was too large the Vectashield applied to this prepared area at the
               final steps of the embedding procedure touched the constructed chamber’s walls and
               could leave the blastomeres uncovered and dried out (see below). When finished with
               the last cover glass, but at least 5 min after the start with the first one, the drop
               of polylysine solution was rinsed off the cover glasses in the same order with 500 μl-1000 μl
               aqua bidest. No drops were left, since after drying they could leave marks. The polylysinated
               side in the same corner of each cover glass was marked with a permanent marker and
               with a character or number that can only be read correctly from one side (for example
               "1"). Even though no liquid was visible on the slides they were dried for at least
               2 h.
            </p>
         </div>
         <div id="Sec8">
            <h3>Slide preparation</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Clean glasses were covered with 4 layers of Tesa-Film. To cover a glass its center
               was pressed at a 45° angle against the sticky side of the film (Fig. 
               <span ref-type="fig" rid="Fig1">1a</span>). Rotating a finger slowly in a circular pattern, the film was carefully pressed
               against the glass and attached onto the glass without generating bubbles. Then the
               film was cut off before and after the attached region on the glass. This procedure
               was repeated 3 more times to obtain 4 layers of Tesa-Film exactly on the top of each
               other on the glass. The reason for building such a high chamber lies in the effect
               of the objective pressing down and thereby reducing the distance between the cover
               glass and the slide inside the chamber. According to the equation of continuity A1*v1 = A2*v2
               (A1, A2 = cross-sectional areas before and after distance reduction; v1, v2 = speed
               of the embedding medium before and after distance reduction) the flow speed of the
               embedding medium inside the chamber will increase when pressing against the cover
               glass with the objective. A larger initial volume in the chamber equals a smaller
               relative volume decrease which equals a smaller speed increase under the pressure
               of the objective. Furthermore, the slide was placed under a stereo microscope and
               using a scalpel and tweezers the pattern shown in Fig. 
               <span ref-type="fig" rid="Fig1">1b</span> was cut into all 4 layers. The central area should be as large as possible to prevent
               contact between embedding medium and film when placing the cover glass on top of this
               structure. The parts shown in Fig. 
               <span ref-type="fig" rid="Fig1">1c</span> were removed with tweezers except for the center part. This central area of the film
               remained attached during storage and was removed shortly before placing the cover
               glass on top of this construct in order to minimize dust within the central chamber.
               <div tagxxx="fig">
                  <span class="label">Fig. 1</span>
                  <caption>
                     <p>Preparation of slides. 
                        <b>a</b> The film was attached to the slide. The black arrow marks the spot where the film
                        was pressed against the slide in a circular motion. 
                        <b>b</b> The film layers were cut with a scalpel. The dashed lines mark the cutting positions.
                        
                        <b>c</b> The slide was prepared for storage. The grey areas inside the outer border mark Tesa-Film
                        removed with 'tweezers'
                     </p>
                  </caption>
                  <div tagxxx="graphic"></div>
               </div>
            </p>
         </div>
         <div id="Sec9">
            <h3>Fixation</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Unless noted otherwise fixation of embryos and all subsequent steps were performed
               at room temperature (RT). For preservation of the 3D shape of embryos and nuclei it
               was necessary to avoid any deforming pressure and prevent embryos from drying out
               at any step of the following procedures. Embryos were briefly washed in 38 °C 1x phosphate
               buffered saline (PBS), fixed in 2 % paraformaldehyde (PFA) in PBS for 10 min, washed
               twice in PBS and then stored at 4 °C in PBS until further use. The maximum allowed
               storage duration depended on the concentration of target proteins for staining. Target
               proteins with a low concentration such as histones were stained and scanned shortly
               after fixation because the achievable signal strength diminishes with time. Target
               proteins with a high concentration such as nuclear lamins or nucleoporins were stained
               and scanned even after months of storage in PBS with acceptable signal strengths compared
               to the background signals.
            </p>
         </div>
         <div id="Sec10">
            <h3>Immunostaining</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Background caused by PFA was quenched using 20 mM glycine in PBS for 10 min. After
               washing twice with PBS embryos were permeabilized with 0.5 % Triton-X 100 for 15-30 min.
               After washing twice with PBS unspecific background signals were reduced by incubation
               in 2 % BSA for 2-4 h at RT. When targeting proteins with a low concentration, such
               as histones, embryos were blocked before staining for 4 h or longer in 2 % BSA in
               PBS to differentiate actual signals from background more efficiently. Four hours of
               blocking could result in a more focused localization of the signals leading to a slightly
               altered appearance compared to blocking for 2 h. When targeting proteins with a high
               concentration such as nuclear lamins or nucleoporins embryos were blocked before staining
               for only 2 h since the background was relatively less intense. However, maximum image
               quality was achieved with a blocking for 4 h. Then embryos were sequentially incubated
               in 40 μl of primary and secondary antibody solutions contained within PCR-tube caps,
               diluted in PBS with 2 % BSA. The source of the first antibody was different from the
               species of the embryo or the zona pellucida to avoid staining problems (see Troubleshooting
               below). In the case of using the same source as the embryo’s species, the zona pellucida
               was removed prior to staining as described below. Tables 
               <span ref-type="table" rid="Tab1">
                  <a href="#Tab1">1</a>
               </span> and 
               <span ref-type="table" rid="Tab2">
                  <a href="#Tab2">2</a>
               </span> show examples of primary and secondary antibodies used successfully with 3D-SIM.
               Specimens were incubated with primary antibodies overnight at 4 °C. After washing
               in PBS for 5 times with 2 % BSA the appropriate secondary antibodies, diluted in PBS
               with 2 % BSA, were applied for 1 h, again followed by washing in PBS for 5 times with
               2 % BSA and 5 times without BSA. BSA had two effects. It blocked background signals
               and made embryos less sticky. While the first property was desired the second one
               was counter-effective when trying to attach blastomeres on a cover glass for minimal
               movement. In order to maximize stickiness even after 5 times of washing away BSA with
               PBS the following fixation step was used to bind the remaining BSA. After washing
               away the PFA embryos were extremely sticky and in order to prevent their attachment
               to any surface PBS was constantly pumped under the embryos to keep them floating within
               the PBS and transporting the embryos not within the transferpettor’s pipette but in
               a tiny drop at the tip of it. If the embryo has attached to a surface recovery was
               possible by blowing the embryo with the PBS very strongly against it. However, the
               pressure produced by a standard size opening of a pipette on a transferpettor may
               not suffice. In this case, the tip of a new glass pipette was melted until the opening
               was barely visible. Coating the pipette with silicone and the surfaces with silicone
               or agar may prevent embryos from attaching prior to the final embedding step. A cover
               glass with the polylysinated side upwards was put into the middle of the cover of
               a 3.5 cm well. A drop of 10-20 μl PBS was added in the center of the glass. Fixation
               of antibodies was performed with 4 % PFA in PBS for 10 min, followed by washing twice
               in PBS. PFA could cause increased background signals in the 488 nm channel. Therefore,
               fixing antibodies with only 2 % PFA in PBS was used to reduce such additional background
               signals. Chromatin was counterstained with DAPI (Life Technologies, Catalog No. D1306;
               4′,6-diamidino-2-phenylindole) 25 μg/ml diluted in PBS for 15 min followed by washing
               twice in PBS.
               <div id="Tab1">
                  <span class="label">Table 1</span>
                  <caption>
                     <p>Examples of primary antibodies used successfully with 3D-SIM</p>
                  </caption>
                  <table>
                     <thead>
                        <tr>
                           <th>Host</th>
                           <th>Target</th>
                           <th>Official name</th>
                           <th>Dilution</th>
                           <th>Company</th>
                           <th>ID</th>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>Goat</td>
                           <td>Epitope at C-terminus of Lamin B1. Detecting Lamin B1 and, to a lesser extent, Lamin
                              B2
                           </td>
                           <td>Lamin B Antibody (M-20)</td>
                           <td>1:100</td>
                           <td>Santa Cruz</td>
                           <td>SC-6217</td>
                        </tr>
                        <tr>
                           <td>Mouse</td>
                           <td>NUP153</td>
                           <td>Anti-NUP153 [QE5] antibody</td>
                           <td>1:200</td>
                           <td>Abcam</td>
                           <td>ab24700</td>
                        </tr>
                        <tr>
                           <td>Rabbit</td>
                           <td>H3K4me3</td>
                           <td>Anti-Histone H3 (tri methyl K4) antibody - ChIP Grade</td>
                           <td>1:100</td>
                           <td>Abcam</td>
                           <td>ab8580</td>
                        </tr>
                        <tr>
                           <td>Mouse</td>
                           <td>H3K9me3</td>
                           <td>Histone H3K9me3 antibody (mAb)</td>
                           <td>1:100</td>
                           <td>Active Motif</td>
                           <td>39285/6</td>
                        </tr>
                        <tr>
                           <td>Rat</td>
                           <td>RNAPII-S5p</td>
                           <td>Monoclonal antibody against CTD phosphoserine epitope 5</td>
                           <td>1:20</td>
                           <td>Gift from Dirk Eick (see Markaki et al. [
                              <span ref-type="bibr" rid="CR23">
                                 <sup>
                                    <a href="#CR23">23</a>
                                 </sup>
                              </span>])
                           </td>
                        </tr>
                        <tr>
                           <td>Mouse</td>
                           <td>B23</td>
                           <td>Monoclonal Anti-B23 antibody produced in mouse</td>
                           <td>1:1000</td>
                           <td>Sigma</td>
                           <td>B0556</td>
                        </tr>
                     </div>
                  </table>
                  <table-wrap-foot>
                     <p>The host should be different from the species of the embryo. Otherwise, the zona pellucida
                        should be removed prior to staining
                     </p>
                  </table-wrap-foot>
               </div>
               <div id="Tab2">
                  <span class="label">Table 2</span>
                  <caption>
                     <p>Examples of secondary antibodies used successfully with 3D-SIM</p>
                  </caption>
                  <table>
                     <thead>
                        <tr>
                           <th>Host</th>
                           <th>Target</th>
                           <th>Fluorophore</th>
                           <th>Excitation [nm]</th>
                           <th>Dilution</th>
                           <th>Company</th>
                           <th>ID</th>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>Donkey</td>
                           <td>Mouse IgG (H + L)</td>
                           <td>Alexa</td>
                           <td>488</td>
                           <td>1:400</td>
                           <td>Invitrogen</td>
                           <td>A-21202</td>
                        </tr>
                        <tr>
                           <td>Donkey</td>
                           <td>Mouse IgG (H + L)</td>
                           <td>Alexa</td>
                           <td>594</td>
                           <td>1:500</td>
                           <td>Invitrogen</td>
                           <td>A-21203</td>
                        </tr>
                        <tr>
                           <td>Donkey</td>
                           <td>Goat IgG (H + L)</td>
                           <td>Alexa</td>
                           <td>488</td>
                           <td>1:400</td>
                           <td>Invitrogen</td>
                           <td>A-11055</td>
                        </tr>
                        <tr>
                           <td>Donkey</td>
                           <td>Goat IgG (H + L) preadsorbed</td>
                           <td>Alexa</td>
                           <td>594</td>
                           <td>1:400</td>
                           <td>Abcam</td>
                           <td>ab150136</td>
                        </tr>
                        <tr>
                           <td>Donkey</td>
                           <td>Rabbit IgG (H + L)</td>
                           <td>Dylight (Alexa bleaches less)</td>
                           <td>488</td>
                           <td>1:300</td>
                           <td>Jackson Immuno Research</td>
                           <td>711-485-152</td>
                        </tr>
                        <tr>
                           <td>Donkey</td>
                           <td>Rabbit IgG (H + L)</td>
                           <td>Dylight (Alexa bleaches less)</td>
                           <td>594</td>
                           <td>1:500</td>
                           <td>Jackson Immuno Research</td>
                           <td>711-505-152</td>
                        </tr>
                        <tr>
                           <td>Goat</td>
                           <td>Mouse IgG (H + L) preadsorbed</td>
                           <td>Alexa</td>
                           <td>594</td>
                           <td>1:300</td>
                           <td>Invitrogen</td>
                           <td>A-11032</td>
                        </tr>
                        <tr>
                           <td>Donkey</td>
                           <td>Rat IgG (H + L)</td>
                           <td>Alexa</td>
                           <td>594</td>
                           <td>1:300</td>
                           <td>Invitrogen</td>
                           <td>A-21209</td>
                        </tr>
                     </div>
                  </table>
               </div>
            </p>
         </div>
         <div id="Sec11">
            <h3>Isolation of blastomeres and attachment to the cover glass</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The 3.5 cm well containing the polylysinated cover glass, the drop of PBS and the
               embryo were transported to a microscope with at least one micromanipulator (two are
               preferred). An empty injection needle was mounted into the right micromanipulator
               holder. Similarly, a needle with a diameter smaller than an embryo was mounted into
               the left micromanipulator holder. This second needle was helpful to stabilize the
               position of the embryo but was not absolutely necessary for the subsequent steps.
               Once the embryo was placed inside the drop of PBS on top of the polylysinated cover
               glass adhesion forces attached the zona pellucida to the glass. An area of the zona
               pellucida slightly remote from the blastomeres was pressed against the cover glass
               with the injection needle. Then the needle was moved back and forth while keeping
               the zona pressed against the glass. The slit in the zona pellucida was enlarged by
               vibrating the needle via knocking against the micromanipulator with a finger (Fig. 
               <span ref-type="fig" rid="Fig2">2a</span>). Afterwards the injection needle was inserted into the zona and stretched along
               the surface of the cover glass away from the embryo to open the zona even more. The
               embryo inside the open zona pellucida was rotated with the injection needle to line
               up the border between two blastomeres. Then the injection needle was placed along
               this border and moved back and forth to separate the blastomeres (Fig. 
               <span ref-type="fig" rid="Fig2">2b</span>). Additional knocking against the micromanipulator could facilitate this process.
               A single blastomere was removed from the zona pellucida and the zona pellucida was
               moved with the remaining blastomeres away (Fig. 
               <span ref-type="fig" rid="Fig2">2c and d</span>). The remaining blastomeres were moved inside the zona pellucida to a different remote
               location from the previous blastomere to prevent bleaching from scanning a neighbouring
               blastomere (Fig. 
               <span ref-type="fig" rid="Fig2">2e</span>). The procedure was repeated for all blastomeres. The initial attachment of the blastomeres
               was automatically facilitated by the adhesion forces of the polylysinated surface.
               The attached blastomeres were left immobile otherwise this could lead to tearing and
               stretching artifacts. When working with blastocysts and the task was to scan only
               trophectoderm (TE) cells a blastocyst was let to sink on the glass and the monolayer
               of cells closest to the glass was scanned (Fig. 
               <span ref-type="fig" rid="Fig3">3a</span>). The cell-free space inside blastocysts (blastocoel) can be large enough so that
               TE cells on the far side of the blastocyst do not interfere with the scans. If there
               were multiple layers of TE cells then individual TE cells were isolated and placed
               next to each other in remote locations as a monolayer. If inner cell mass (ICM) and
               TE cells were scanned the layer(s) of TE cells were cut like the zona pellucida. Tearing
               and stretching of cells remote from the slit were avoided. TE cells were then gently
               attached to the glass while the sphere of ICM cells was residing on top of this monolayer.
               This ICM sphere was then detached from the monolayer of TE cells by gently pressing
               against it with the side of the injection needle (Fig. 
               <span ref-type="fig" rid="Fig3">3b</span>). The ICM sphere was then moved to a remote location and individual cells were isolated
               as described above for other embryos (Fig. 
               <span ref-type="fig" rid="Fig3">3c</span>). To maximize image quality it was necessary to attach single blastomeres to the
               glass not touching each other (Fig. 
               <span ref-type="fig" rid="Fig3">3d</span>). After attachment of all blastomeres to their target location additional attachments
               were performed to prevent movements disabling 3D-SIM reconstruction (Fig. 
               <span ref-type="fig" rid="Fig2">2f and g</span>). The outermost part of blastomeres was pressed down with the injection needle against
               the cover glass. Then the 3.5 cm well was carefully rotated by 90° and the newly accessible
               outermost parts were pressed down. The zona pellucida was moved outside the drop so
               that it could not swim through the medium while scanning. When scanning sperm cells
               attached to the zona pellucida after an IVF procedure the zona was cut in smaller
               parts and these parts were pressed against the glass for maximum adhesion.
               <div tagxxx="fig">
                  <span class="label">Fig. 2</span>
                  <caption>
                     <p>Isolation and attachment of individual blastomeres. 
                        <b>a</b> The zona pellucida was cut open. 
                        <b>b</b> A blastomere was isolated. 
                        <b>c</b> The isolated blastomere was moved outside the zona pellucida through the cut. 
                        <b>d</b> The zona pellucida with the remaining blastomeres was moved to the target location
                        for the next blastomere. 
                        <b>e</b> Positions of blastomeres on the cover glass were chosen as far away from each other
                        as possible, but not too close to the border of the PBS drop. 
                        <b>f</b> Once all blastomeres had been placed on the cover glass their outermost parts were
                        pressed against the glass. The dashed line marks the section shown in G. 
                        <b>g</b> The attachment surface was increased even further by stretching out the pressed down
                        periphery of blastomeres by moving the pressed down needle away from the blastomere.
                        The circled N marks the position of the nucleus in this blastomere. 
                        <b>h</b> The finalized slide with the reversed cover glass attached in the center chamber
                        and sealed with nail polish. The marker "1" was dissolved by the nail polish
                     </p>
                  </caption>
                  <div tagxxx="graphic"></div>
               </div>
               <div tagxxx="fig">
                  <span class="label">Fig. 3</span>
                  <caption>
                     <p>Isolation of blastomeres from blastocysts. 
                        <b>a</b> The trophectoderm (TE) layer was cut open. 
                        <b>b</b> The inner cell mass was moved away from the TE layer. 
                        <b>c</b> Blastomere isolation. 
                        <b>d</b> Individual blastomeres were placed remotely from each other
                     </p>
                  </caption>
                  <div tagxxx="graphic"></div>
               </div>
            </p>
         </div>
         <div id="Sec12">
            <h3>Embedding</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The closed well was transported back to the stereo microscope. The excess of PBS was
               removed from the cover glass while keeping all blastomeres covered with it. The cover
               glass was washed two times with 10-20 μl of PBS and covered slowly with 10-20 μl of
               Vectashield. The excess of Vectashield was removed from the cover glass while keeping
               all blastomeres covered with it. The slide with the removed central part of the Tesa-Film
               was placed under the stereo microscope. The cover glass with the drop facing down
               was put above the center part of the Tesa-Film frame and slowly lowered until the
               drop reached the slide avoiding the attachment of the drop with the Tesa-Film. The
               edges of the cover glass rested on the outer frame of the Tesa-Film. All sides of
               the outer frame were treated with a very small amount of nail polish using a fine
               brush. A contact between nail polish and Vectashield was avoided, while ensuring the
               complete sealing of the outer frame. After the nail polish had dried out the slide
               with the cover glass was held down at an angle under the stereo microscope and all
               blastomeres were marked with arrows on the back of the slide, but not on the cover
               glass (Fig. 
               <span ref-type="fig" rid="Fig2">2h</span>). The slide was stored at 4 °C horizontally with the cover glass facing down to prevent
               movement of the Vectashield drop and detachment of the blastomeres.
            </p>
         </div>
         <div id="Sec13">
            <h3>Oil selection for microscopy</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Selection of the correct oil required to produce crisp images at the desired distance
               to the cover glass was performed in two steps. Firstly, the oil was estimated using
               embryos at various stages of development based on empirical testing (Table 
               <span ref-type="table" rid="Tab3">
                  <a href="#Tab3">3</a>
               </span>). We found that smaller blastomeres required oils with lower refractive indexes.
               Secondly, the correct oil was determined by analyzing preview and reconstructed images
               provided by 3D-SIM and its accompanying software package.
               <div id="Tab3">
                  <span class="label">Table 3</span>
                  <caption>
                     <p>Optimal refractive indices of oils per stage based on empirical testing</p>
                  </caption>
                  <table>
                     <thead>
                        <tr>
                           <th>Stage/cell type</th>
                           <th>Optimal refractive index of oil</th>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>Zygote (centrally located pronuclei)</td>
                           <td>&gt;1.534</td>
                        </tr>
                        <tr>
                           <td>2-cell</td>
                           <td>1.528–1.534</td>
                        </tr>
                        <tr>
                           <td>4-cell</td>
                           <td>1.526–1.528</td>
                        </tr>
                        <tr>
                           <td>8-cell</td>
                           <td>1.518–1.526</td>
                        </tr>
                        <tr>
                           <td>16-cell, morula and blastocyst</td>
                           <td>1.512–1.518</td>
                        </tr>
                        <tr>
                           <td>Spermatozoa, fibroblasts and cell lines</td>
                           <td>1.512–1.514</td>
                        </tr>
                     </div>
                  </table>
               </div>
            </p>
         </div>
         <div id="Sec14">
            <h3>Determination of the blastomere coordinates</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">A small drop of the estimated oil was applied on the clean cover glass. With the specimen
               localization microscope of the 3D-SIM setup the arrows in transmission light mode
               were found and the locations of the associated blastomeres were marked in the software.
            </p>
         </div>
         <div id="Sec15">
            <h3>Determination of the correct oil before scanning</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The slide was transferred to the 3D-SIM microscope and the objective was moved in
               the z-direction at the position of the first blastomere. The z-position of the objective
               was compared before moving it towards the cover glass and afterwards. In our setup,
               smaller z-position values required oils with a higher refractive index, because the
               layer of the specimen was located at a greater distance from the cover glass. With
               this understanding, the preview image was used to predict the selection of the correct
               oil. In round specimens such as nuclei, wide-field microscopy apparently generates
               intense circular patterns that form cone shapes above and below the specimens (Fig. 
               <span ref-type="fig" rid="Fig4">4</span>). The key to find the oil that provides a focused image in the vertical center of
               the specimen is to ensure that the distance of the upper and lower congregation points
               to the specimen is equal above and below the specimen (Fig. 
               <span ref-type="fig" rid="Fig4">4a–c</span>). This can be assessed by moving in the z-direction above and below the specimen
               and counting sections between the top and bottom cone focal points and the first clearly
               visible grid on the nucleus’ top and bottom side. If the distance of the upper and
               lower congregation points was equal above and below the specimen then the focal section
               of the oil was in the specimen center. This was verified by checking for sharp lines
               of the grid in the mid-section. If the distance was not the same, then the oil with
               a higher refractive index was used in the case when the side of the nucleus with the
               lower z-position had a shorter distance between the cone focal point and the first
               clearly visible grid on the nucleus compared to the side of the nucleus with the higher
               z-position. If this scenario was reversed then the oil with a lower refractive index
               was used.
               <div tagxxx="fig">
                  <span class="label">Fig. 4</span>
                  <caption>
                     <p>Determination of the correct oil before the first scan. 
                        <b>a1–5</b> Preview images of the first focal point of grid-free signals (
                        <b>a1</b>), the first occurrence of the grid on the specimen (
                        <b>a2</b>), the mid-section (
                        <b>a3</b>), the last occurrence of the grid (
                        <b>a4</b>) and the second focal point of grid-free signals. 
                        <b>b1–5</b> show the same images as in (
                        <b>a1–5</b>) with optimized contrast for improved visualization of the grid. 
                        <b>c1</b> Orthogonal view of the first of the three non-reconstructed scans with different
                        angles. The dashed lines mark sections (
                        <b>a1</b>–
                        <b>a5</b>). 
                        <b>c2</b> Orthogonal view of the second of three non-reconstructed scans. Note the reduction
                        of signal strength between the two scans caused by bleaching. 
                        <b>d1</b> Orthogonal view of the reconstructed scan. The central dashed line marks the section
                        of D2. 
                        <b>d2</b> Reconstructed mid-section with full resolution
                     </p>
                  </caption>
                  <div tagxxx="graphic"></div>
               </div>
            </p>
         </div>
         <div id="Sec16">
            <h3>3D structured illumination microscopy (3D-SIM) and quantitative image evaluation</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">3D-SIM of embryonic nuclei was performed on a DeltaVision OMX V3 system (Applied Precision
               Imaging/GE Healthcare) with a lateral (x,y) resolution of ~120 nm and an axial (z)
               resolution of ~300 nm. The system was equipped with a 100x/1.40 NA PlanApo oil immersion
               objective (Olympus), Cascade II:512 EMCCD cameras (Photometrics) and 405, 488 and
               593 nm diode lasers. Image stacks were acquired with a z-distance of 125 nm and with
               15 raw SIM images per plane (5 phases, 3 angles). The SI raw data were computationally
               reconstructed with channel specific measured OTFs using the softWoRX 4.0 software
               package (Applied Precision). Images from the different color channels were registered
               with alignment parameters obtained from calibration measurements with 0.2 μm diameter
               TetraSpeck beads (Invitrogen). The voxel size of the reconstructed images is 39.5 nm
               in x,y and 125 nm in z with 32-bit depth. For all subsequent image processing and
               data analysis, images were converted to 16-bit composite tiff-stacks. Image stacks
               were processed using ImageJ 1.45b (
               <div tagxxx="ext-link">http://rsb.info.nih.gov/ij/</div>).
            </p>
         </div>
         <div id="Sec17">
            <h3>Scanning</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Before scanning the specimen was centered by comparing the distance of its border
               to the borders of the frame. This was also checked throughout all z-sections to use
               entire 3D-stacks. The key to great scans of perfectly immobile and clean specimens
               in clean Vectashield was to minimize bleaching during the scan. With highly concentrated
               target proteins such as lamins or nucleoporins a special property of the Alexa brand
               of secondary antibodies was used to not only reduce bleaching, but even stop or reverse
               bleaching. Alexa antibodies displayed two bleaching properties. Firstly, they bleached
               during a first intense scan (Table 
               <span ref-type="table" rid="Tab4">
                  <a href="#Tab4">4</a>
               </span>). Secondly, after such a first intense scan, their photon count was much more stable
               during a second less intense scan (Table 
               <span ref-type="table" rid="Tab5">
                  <a href="#Tab5">5</a>
               </span>). To obtain optimal results, Alexa stained specimens were scanned with a highly concentrated
               target protein in two steps. In the first step scanning was performed using the CCD
               camera with high laser intensity. This first reconstructed scan was inspected using
               the orthogonal viewer of the DV viewer software or with FIJI (ImageJ with appropriate
               plugins for viewing DV files) to check if the correct oil had been selected (Fig. 
               <span ref-type="fig" rid="Fig4">4</span>). In that case, the two cones on the top and on the bottom of the sample had the
               same height. If the intensity of the signals did not allow for a second scan with
               the settings according to Table 
               <span ref-type="table" rid="Tab5">
                  <a href="#Tab5">5</a>
               </span> then the first scan was repeated. The second scan was performed with a highly sensitive
               EMCCD camera with far less intense laser settings. A large photon count reduction
               was observed between the beginning and the end of the first, high intensity scan.
               The photon count stayed more stable in the second, low intensity scan. However, even
               the initial scan yielded very good results in many cases. This could be due to the
               fact that the bleaching of the fluorescent beads used to configure the algorithm of
               the software was proportional to the bleaching of the initial scan and therefore resembled
               the configured situation better than a reduced bleaching situation. Generally however,
               less bleaching was better than more bleaching. If the image quality was suboptimal
               when scanning multiple wavelengths at the same time then those were scanned in separate
               runs and the channels were combined after the scans using FIJI or the DV software
               provided with the microscope. The reduced image quality in parallel scans could be
               due to fluorophores being bleached by other wavelengths as well, adding additional
               bleaching during parallel scanning. For optimal results each channel was bleached
               independently until the scenario described in Table 
               <span ref-type="table" rid="Tab5">
                  <a href="#Tab5">5</a>
               </span> was achieved for all channels at the beginning of the final scan. If the target was
               not highly concentrated multiple scans were avoided. In this case each channel was
               scanned independently with the second scan settings.
               <div id="Tab4">
                  <span class="label">Table 4</span>
                  <caption>
                     <p>Microscope settings and expected photon counts for the first scan</p>
                  </caption>
                  <table>
                     <thead>
                        <tr>
                           <th>Fluorophor/staining</th>
                           <th>Mode</th>
                           <th>Exposure</th>
                           <th>Laser</th>
                           <th>Initial photon count</th>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>Alexa</td>
                           <td>CCD 5 MHz</td>
                           <td>25 ms</td>
                           <td>50 %</td>
                           <td>17000</td>
                        </tr>
                        <tr>
                           <td>DAPI</td>
                           <td>CCD 5 MHz</td>
                           <td>50 ms-120 ms (lower is better)</td>
                           <td>100 %</td>
                           <td>8000 - 16000</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <div id="Tab5">
                  <span class="label">Table 5</span>
                  <caption>
                     <p>Microscope settings and expected photon counts for the second scan</p>
                  </caption>
                  <table>
                     <thead>
                        <tr>
                           <th>Fluorophor/staining</th>
                           <th>Camera</th>
                           <th>EMCCD gain</th>
                           <th>Exposure</th>
                           <th>Laser</th>
                           <th>Initial photon count</th>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>Alexa</td>
                           <td>EMCCD 5 MHz</td>
                           <td>2800-3300</td>
                           <td>30 ms</td>
                           <td>10 %</td>
                           <td>17000</td>
                        </tr>
                        <tr>
                           <td>DAPI</td>
                           <td>CCD 5 MHz</td>
                           <td>-</td>
                           <td>50 ms-120 ms (lower is better)</td>
                           <td>100 %</td>
                           <td>8000 – 16000</td>
                        </tr>
                     </div>
                  </table>
               </div>
            </p>
         </div>
         <div id="Sec18">
            <h3>Determination of the correct oil after the first scan</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Not all objects generated cones of artificially increased signals above and below
               the object. Non-round nucleus-like objects observed in early stage rabbit embryos
               did not generate intense cones (see the orthogonal views in Fig. 
               <span ref-type="fig" rid="Fig5">5</span>). In this case, the correct oil was determined after the first scan. A highly concentrated
               target protein generating line-like signals along the z-axis like a nuclear lamina
               staining was the best indicator of whether to increase or lower the chosen oil refractive
               index and for this purpose orthogonal views of the first scan were generated. This
               was performed with the DV viewer software or with FIJI. Objects like nucleoli were
               identified in the stack and in the preview window to determine whether the top or
               the bottom of the z-stack were closer to the cover glass. To determine whether the
               top or the bottom of the scanned stack was closer to the cover glass the objective
               was moved up and down in the z-axis with the preview window and the z-position was
               compared with the objective’s position when it was at a large distance to the cover
               glass before scanning. The area of the z-stack was identified in the orthogonal viewer
               with the highest signal intensity and the sharpest lines (Fig. 
               <span ref-type="fig" rid="Fig5">5</span>). This focus area was moved further away from the glass by changing to oil with a
               higher refractive index or moved closer to the glass by changing to oil with a lower
               refractive index. Once the correct oil had been identified all channels were scanned.
               <div tagxxx="fig">
                  <span class="label">Fig. 5</span>
                  <caption>
                     <p>Effect of oils with different refractive indices. The same nucleus stained for lamin
                        B from a rabbit 8-cell stage 
                        <i>in vivo</i> embryo scanned with 1.520 (
                        <b>a</b>), 1.526 (
                        <b>b</b>) and 1.530 (
                        <b>c</b>) oils. Note the movement of the optimal focal section away from the cover glass with
                        higher oils. The sides of the orthogonal views are color-coded for high quality sections
                        at around the optimal focal section (green), medium quality sections further away
                        (yellow) and low quality sections even further away (red)
                     </p>
                  </caption>
                  <div tagxxx="graphic"></div>
               </div>
            </p>
         </div>
         <div id="Sec19">
            <h3>Computing the final images</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">To ensure high quality scans the scanned stack was computed immediately after scanning.
               If the computed result was not optimal the same area was scanned again with different
               settings. In large scans the central area of 256 by 256 pixels was cut out and only
               this area was reconstructed for preview purposes. The final images were reconstructed
               with the optical transfer function (OTF) files generated using the optimal oil for
               fluorescent beads attached to the cover glass. In our case the refractive index of
               the oil optimal for beads was 1.512. Drift correction was activated to counteract
               any movements and produce images with a maximum quality. This setting did not have
               any negative effects on scans with perfectly still specimens.
            </p>
         </div>
         <div id="Sec20">
            <h3>List of materials and reagents</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>PMS:</b> pregnant mare’s serum
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>HCG:</b> human chorionic gonadotropin
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>Isoflurane</b>
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>BSA:</b> bovine serum albumin
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>M2:</b> medium
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>M16:</b> medium
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>ECG:</b> equine chorionic gonadotropin
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>Quinn’s</b> medium
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>FCS:</b> fetal calf serum
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>Parker’s</b> medium
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>TCM199:</b> medium
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>ECS:</b> estrous cow serum
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>FSH:</b> follicular stimulating hormone
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>FERT-TALP:</b> fertilization medium Tyrode's albumin lactate pyruvate supplemented
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>Heparin:</b> sodium salt
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>SOF:</b> synthetic oviductal fluid
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>BME:</b> Basal Medium Eagle
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>MEM:</b> Minimum Essential Medium
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>Polylysine</b>
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>Vectashield</b>
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>PFA:</b> paraformaldehyde
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>Glycine</b>
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>Triton-X 100</b>
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
               <b>DAPI:</b> 4′,6-diamidino-2-phenylindole
            </p>
         </div>
      </div>
      <div id="Sec21">
         <h2>Results and Discussion</h2>
         <div id="Sec22">
            <h3>Troubleshooting</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">In the case of a suboptimal image quality of scanned blastomeres we tested the staining
               and the microscope first using flat, adherent cells stained using the same conditions
               as stated above. If 
               <b>channels were misaligned</b> (Fig. 
               <span ref-type="fig" rid="Fig6">6</span>a) in adherent cells we recalibrated the microscope using fluorescent beads with the
               same temperature during scanning as for calibration. Misalignment could be checked
               best with prominent structures in multiple channels such as a lamin B and a nucleoporin
               staining or a nucleoli and a DAPI staining in nuclei with nucleoli lined by dense
               chromatin. If 
               <b>antibodies for internuclear signals were also located at the nuclear periphery</b> (Fig. 
               <span ref-type="fig" rid="Fig6">6</span>b) we used only 2 % BSA in PBS as blocking buffer since other additives could reduce
               accessibility of chromatin for antibodies. The antibody itself could also not be specific.
               <div tagxxx="fig">
                  <span class="label">Fig. 6</span>
                  <caption>
                     <p>Potential problems. 
                        <b>a1</b>. Misaligned B23 (green), H3K4me3 (red) and DAPI (grey) channel. 
                        <b>a2</b> shows an enlargement of the box in 
                        <b>a1</b>. Arrows mark the gap between the signals that should be aligned. 
                        <b>b</b>. H3K4me3 staining at the periphery occasionally caused by using the blocking buffer
                        developed by Markaki et al. [
                        <span ref-type="bibr" rid="CR21">
                           <sup>
                              <a href="#CR21">21</a>
                           </sup>
                        </span>] and utilized by Huebner et al. [
                        <span ref-type="bibr" rid="CR22">
                           <sup>
                              <a href="#CR22">22</a>
                           </sup>
                        </span>] without such an effect. 
                        <b>c1</b>. Movement during scanning may have caused short lines instead of round nuclear pores
                        as visualized by a NUP153 staining. 
                        <b>c2</b>. The DAPI channel from the same nucleus as shown in 
                        <b>c2</b> is blurry. 
                        <b>c3</b>. The lines of lamin B of the same nucleus as shown in 
                        <b>c1</b> and 
                        <b>c2</b> are duplicated. 
                        <b>d1</b>. Blurry X/Y section of a DAPI stained nucleus next to another DAPI stained nucleus
                        at the right top border. 
                        <b>d2</b>. The X/Z section marked in D1 shows lines emanating from all four corners caused
                        by the second nucleus (marked by arrows; compare with Figure 
                        <span ref-type="fig" rid="Fig4">4</span> D1). 
                        <b>d3</b>. These lines interfere with the signals produced by the central nucleus. 
                        <b>e</b>. Rabbit anti CDX2 staining of a rabbit blastocyst recorded by confocal microscopy
                        shows a normal staining pattern outside the zona pellucida, a strongly stained zona
                        pellucida and an altered staining pattern of blastomeres inside the zona pellucida
                        when compared with nuclei outside the zona pellucida
                     </p>
                  </caption>
                  <div tagxxx="graphic"></div>
               </div>
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The wrong oil led to suboptimal results as described above. 
               <b>Bleaching</b> was minimized by the pre-bleaching strategy outlined above. In the case of bleaching
               the top sections of a nucleus in an orthogonal view were brighter than the bottom
               sections. 
               <b>Artificial short lines emanating from the real signals</b> (Fig. 
               <span ref-type="fig" rid="Fig6">6</span>c) indicated sample movement during scanning. This could be either due to an improper
               attachment of the sample to the cover glass or movements of other materials inside
               the sample (loose fluorophores or particles). The solution was to re-attach the specimen
               and to fix the specimen again. To achieve this with the same sample, PBS was added
               to the hardened nail polish seal while gently disconnecting the bond between the nail
               polish and the cover glass with very thin tweezers. Once the nail polish seal and
               the Tesa-Film were disconnected from the cover glass on all sides the cover glass
               was lifted up using tweezers, turned in the way that the drop of Vectashield faced
               up and then placed inside a 3.5 cm well. All subsequent steps were performed by exchanging
               chemicals on the cover glass with the blastomeres still attached. Vectashield was
               washed away gently but completely with PBS for 5 times while not blowing against the
               attached blastomeres. The specimen was fixed with 4 % PFA in PBS for 10 min, washed
               twice with PBS and restained with DAPI 25 μg/ml diluted in PBS for 15 min followed
               by washing twice in PBS. The well was moved to a microscope featuring micromanipulators.
               The periphery of blastomeres was pressed onto the cover glass with an injection needle
               attached to a micromanipulator as described above. The well was moved back to a stereo microscope
               and the sample was washed gently without blowing against the blastomeres with PBS
               for 5 times. The different light direction options of the stereo microscope were used
               to check the medium for floating particles. All particles were removed with a pipette.
               The specimens were washed twice with Vectashield and checked again for particles.
               All particles were removed with a pipette. The cover glass was turned in the way that
               the Vectashield was facing down and then was slowly lowered to a fresh clean slide
               while keeping it horizontally aligned. The slide was sealed with nail polish and scanned
               again. Another factor for optimal image quality was to ensure that there were 
               <b>no additional nuclei in close vicinity</b> (Fig. 
               <span ref-type="fig" rid="Fig6">6</span>d) to the nucleus scanned. Nuclei in the vicinity could cause artificial patterns
               reducing the image quality. If the 
               <b>zona pellucida displayed strong signals</b> (Fig. 
               <span ref-type="fig" rid="Fig6">6</span>e) while blastomeres within the zona pellucida had low to no signals the secondary
               antibody was most likely targeting the embryo’s species. The zona pellucida could
               be saturated with target proteins. This is a common problem with mouse and rabbit
               embryos as most secondary antibodies are created in these species. The solution to
               this problem was to remove the zona pellucida before the first contact with antibodies.
               Handling sticky embryos after removal of 2 % BSA in PBS without a zona pellucida could
               be challenging. This required constant blowing under the embryos to prevent attachment
               to the bottom of the Eppendorf tube cap. When moving embryos between caps it was necessary
               to keep embryos in a drop of liquid at the tip of the transferpettor’s pipette to
               prevent attachment inside the pipette. In order to maximize stickiness even after
               5 times of washing away BSA with PBS the following fixation step was used to bind
               the remaining BSA. After washing away the PFA embryos were extremely sticky and in
               order to prevent their attachment to any surface PBS was constantly pumped under the
               embryos to keep them floating within the PBS and transporting the embryos not within
               the transferpettor’s pipette but in a tiny drop at the tip of it.
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">To avoid the stickiness of embryos with or without the zona pellucida a more simple
               solution than a complicated pipetting system could be the use of chambers/slides/dishes
               covered by agar or silicon and pipettes covered with silicon. However, such additional
               reagents in direct contact with the solutions containing the embryos may create particles
               disturbing the scanning and calculating procedure. It should also be noted that the
               embryo’s stickiness is increased during prolonged storage of the fixed embryos at
               4 °C, as prolonged storage can alter the surface of blastomeres and of the zona pellucida.
            </p>
         </div>
         <div id="Sec23">
            <h3>Examples of 3D-SIM and confocal microscopy scans</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Figure 
               <span ref-type="fig" rid="Fig7">7</span> features 3D-SIM scans of nuclei in blastomeres from cattle, rabbit and mouse embryos
               and rabbit spermatozoa. A confocal scan shows the difference in resolution.
               <div tagxxx="fig">
                  <span class="label">Fig. 7</span>
                  <caption>
                     <p>Examples of 3D-SIM and confocal microscopy scans. 
                        <b>a</b>. 3D-SIM scans of a nucleus in a trophectoderm blastomere from a bovine blastocyst.
                        
                        <b>b</b>. 3D-SIM scans of a nucleus in a blastomere from a rabbit 21-cell embryo. 
                        <b>c</b>. 3D-SIM scans of a nucleus in a blastomere from a mouse morula. The background may
                        have increased slightly since this morula was stored for 2 years in PBS under mineral
                        oil between fixation and staining. 
                        <b>d</b>. 3D-SIM scans of a rabbit spermatozoon. 
                        <b>e</b>. Confocal microscopy scans of a nucleus in a blastomere from a rabbit 2-cell stage
                        embryo. 
                        <b>a1</b>–
                        <b>e1</b>. DAPI. 
                        <b>a2</b>–
                        <b>e2</b>. NUP153. 
                        <b>a3</b>–
                        <b>e3</b>. Lamin B. 
                        <b>a4</b>–
                        <b>e4</b>. Composite of DAPI (grey), NUP153 (green) and lamin B (red). 
                        <b>a5</b>–
                        <b>d5</b>. DAPI without a correction of the background signals. 
                        <b>a6</b>–
                        <b>d6</b>. NUP153 without a correction of the background signals. 
                        <b>a7</b>–
                        <b>d7</b>. Lamin B without a correction of the background signals
                     </p>
                  </caption>
                  <div tagxxx="graphic"></div>
               </div>
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">This figure showcases the image quality possible with our 3D-SIM protocol by resolving
               individual nuclear pores.
            </p>
         </div>
         <div id="Sec24">
            <h3>Comparison with alternative microscopy solutions</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">While 3D-SIM surpasses the image resolution of confocal imaging [
               <span ref-type="bibr" rid="CR11">
                  <sup>
                     <a href="#CR11">11</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="CR12">
                  <sup>
                     <a href="#CR12">12</a>
                  </sup>
               </span>], the stabilization of specimens, the determination of the correct oil, potentially
               increased bleaching by scanning each section far more often than during confocal microscopy
               and the requirement of reconstructing the final image make this method more demanding.
               Alternatively, confocal scanning without deconvolution may be the preferred method
               if signal intensities between multiple samples have to be compared, since algorithms
               can alter the signal intensity and create artifacts [
               <span ref-type="bibr" rid="CR13">
                  <sup>
                     <a href="#CR13">13</a>
                  </sup>
               </span>]. Wide-field microscopy also introduces new artifacts such as spherically increased
               signals to the specimens, which were not observed in confocal scans. GFP or RFP signals
               can get bleached too much with our setup and may require GFP or RFP boosters featuring
               Alexa fluorophores.
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">While the resolution of 3D-SIM is lower than the resolution achieved with electron
               microscopy [
               <span ref-type="bibr" rid="CR14">
                  <sup>
                     <a href="#CR14">14</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="CR15">
                  <sup>
                     <a href="#CR15">15</a>
                  </sup>
               </span>], 3D-SIM offers the advantage of using lasers with multiple wavelengths for multiple
               target co-localization studies not possible with electron microscopy. Additionally,
               embedding specimens in solid blocks and cutting/milling the blocks for scanning sections
               is necessary for electron microscopy and may introduce artifacts such as stretching
               of sections or cut marks [
               <span ref-type="bibr" rid="CR16">
                  <sup>
                     <a href="#CR16">16</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="CR17">
                  <sup>
                     <a href="#CR17">17</a>
                  </sup>
               </span>]. 3D-SIM can be performed in a liquid embedding medium that does not dry out and
               does not require cutting/milling and therefore does not change the morphology of specimens
               as demonstrated in completely spherical nuclei in 3D. However, liquid embedding media
               with a different density than PBS may potentially cause temporary deformation. A temporarily
               deformed zona pellucida was observed in earlier tests when embedding whole embryos
               with a zona pellucida in one step from PBS to Vectashield. Usually, the deformed zona
               pellucida recovered its original shape after some time. This temporary problem can
               be circumvented by washing embryos with increasing concentrations of the embedding
               medium in PBS before the final embedding step in pure embedding medium. Spherical
               nuclei in all dimensions prove that deformation based on medium density differences
               is either temporary or does not affect blastomeres. Earlier tests with embedding media
               that harden such as ProLong Gold led to alterations in morphology and an increased
               background signal and are therefore not recommended. This may be due to volume reductions
               during vaporization of the solvent.
            </p>
         </div>
         <div id="Sec25">
            <h3>Applicability to alternative super-resolution fluorescence microscopy solutions</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">This protocol essentially aims to simulate the stability of fluorescent molecule localizations
               of adherent cells by attaching blastomeres to polylysinated cover glasses. Furthermore,
               it aims to simulate their proximity to the cover glass by using the correct oil to
               work around the distance to the cover glass. Therefore, because this protocol essentially
               turns blastomeres into simulated adherent cells close to the cover glass, it is feasible
               that this protocol is also compatible with other super-resolution microscopy solutions
               proven to work with adherent cells such as photoactivated localization microscopy
               (PALM) [
               <span ref-type="bibr" rid="CR18">
                  <sup>
                     <a href="#CR18">18</a>
                  </sup>
               </span>], stochastic optical reconstruction microscopy (STORM) [
               <span ref-type="bibr" rid="CR19">
                  <sup>
                     <a href="#CR19">19</a>
                  </sup>
               </span>] and stimulated-emission-depletion microscopy STED [
               <span ref-type="bibr" rid="CR20">
                  <sup>
                     <a href="#CR20">20</a>
                  </sup>
               </span>].
            </p>
         </div>
      </div>
      <div id="Sec26">
         <h2>Conclusions</h2>
         <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">To our knowledge, this protocol presents for the first time a highly detailed and
            practical way to perform super-resolution fluorescence microscopy on areas within
            blastomeres of mammalian embryos which can be remotely located from the cover glass.
            It underscores the importance of stabilizing the location of all signal providing
            molecules throughout the scanning process and demonstrates the process of achieving
            stability in non-adherent cells. Furthermore, this protocol gives practical advice
            in selecting the oil with the correct refractive index before the first scan is performed.
            Selection of the correct oil is mandatory for performing high quality wide-field microscopy
            at large distances from the cover glass such as nuclei in large, round blastomeres
            of early mammalian embryos.
         </p>
         <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">While the image resolution surpasses confocal imaging, the stabilization of specimens,
            the determination of the correct oil, potentially increased bleaching by scanning
            each section more often than in confocal microscopy and the requirement of reconstructing
            the final image make this method more demanding. The problem of bleaching is far greater
            in large blastomeres that require scanning of many z-sections than in flat adherent
            cells. This protocol gives practical advice on how to utilize the non-linear bleaching
            properties of Alexa fluorophores to minimize bleaching in large stacks.
         </p>
         <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Finally, this protocol serves as a guide to generate super-resolution fluorescence
            scans of areas within blastomeres in a highly robust and reliable way.
         </p>
      </div>
      <div tagxxx="back">
         <div tagxxx="glossary">
            <div tagxxx="title">Abbreviations</div>
            <div tagxxx="def-list">
               <div tagxxx="def-item">
                  <div tagxxx="term">3D</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">three-dimensional</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">B23</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">nucleolar phosphoprotein B23</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">BSA</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Bovine serum albumin</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">CCD</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Charge-coupled device</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">CDX2</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Caudal-type homeobox 2</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">DAPI</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">4′,6-diamidino-2-phenylindole</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">EMCCD</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Electron multiplying charge coupled device</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">GFP</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Green fluorescent protein</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">H3K4me3</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Histone 3 lysine 4 trimethylated</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">ICM</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Inner cell mass</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">IVF</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
                        <i>in vitro</i> fertilization
                     </p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">NUP153</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Nucleoporin 153 kDa</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">PBS</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Phosphate buffered saline</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">PFA</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Paraformaldehyde</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">RFP</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Red fluorescent protein</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">RT</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Room temperature</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">SIM</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Structured illumination microscopy</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">TE</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Trophectoderm</p>
                  </div>
               </div>
            </div>
         </div>
         <div tagxxx="fn-group">
            <div class="fn" id="">
               <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
                  <b>Competing interests</b>
               </p>
               <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The authors declare that they have no competing interests.</p>
            </div>
            <div class="fn" id="">
               <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
                  <b>Authors’ contributions</b>
               </p>
               <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Conceived and designed the experiments: JP VZ. Performed the experiments: JP TG VZ.
                  Analyzed the data: JP. Wrote the paper: JP EW VZ. All authors read and approved the
                  final manuscript.
               </p>
            </div>
         </div>
         <div tagxxx="ack">
            <div tagxxx="title">Acknowledgements</div>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">We gratefully acknowledge the support provided by Thomas Cremer. This study was supported
               by grants from the Deutsche Forschungsgemeinschaft to E.W. and V.Z. (CR 59/26, FOR
               1041, ZA 425/1-3). In addition, research of E.W. and V.Z. was supported by the EU
               grant Plurisys, HEALTH-F4-2009-223485 FP7 Health 534 project. The authors are members
               of COST Action BM1308 “Sharing Advances on Large Animal Models – SALAAM”.
            </p>
         </div>
         <div tag="ref-list">
            <ul>
               <div tagxxx="title">References</div>
               <li tag="ref">
                  <a name="CR1"></a>
                  <span class="label">1.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Schermelleh</span>
                           <span tagx="given-names">L</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Heintzmann</span>
                           <span tagx="given-names">R</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Leonhardt</span>
                           <span tagx="given-names">H</span>,
                        </span>
                     </span>
                     <span tagx="article-title">A guide to super-resolution fluorescence microscopy</span>
                     <span tagx="source">J Cell Biol</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">190</span>
                     <div tagxxx="issue">2</div>
                     <span tagx="fpage">165</span>
                     <span tagx="lpage">75</span>
                     <span class="pub-id'">[10.1083/jcb.201002018]</span>
                     <span class="pub-id'">[20643879]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR2"></a>
                  <span class="label">2.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Cremer</span>
                           <span tagx="given-names">C</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Masters</span>
                           <span tagx="given-names">BR</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Resolution enhancement techniques in microscopy</span>
                     <span tagx="source">EPJ H</span>
                     <span tagx="year">2013</span>
                     <span tagx="volume">38</span>
                     <div tagxxx="issue">3</div>
                     <span tagx="fpage">281</span>
                     <span tagx="lpage">344</span>
                     <span class="pub-id'">[10.1140/epjh/e2012-20060-1]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR3"></a>
                  <span class="label">3.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Gustafsson</span>
                           <span tagx="given-names">MG</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Surpassing the lateral resolution limit by a factor of two using structured illumination
                        microscopy
                     </span>
                     <span tagx="source">J Microsc</span>
                     <span tagx="year">2000</span>
                     <span tagx="volume">198</span>
                     <div tagxxx="issue">2</div>
                     <span tagx="fpage">82</span>
                     <span tagx="lpage">7</span>
                     <span class="pub-id'">[10.1046/j.1365-2818.2000.00710.x]</span>
                     <span class="pub-id'">[10810003]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR4"></a>
                  <span class="label">4.</span>
                  <div tagxxx="mixed-citation">Heintzmann R, Cremer CG. Laterally modulated excitation microscopy: improvement of
                     resolution by using a diffraction grating. International Society for Optics and Photonic;
                     1999. p. 185–96.
                  </div>
               </li>
               <li tag="ref">
                  <a name="CR5"></a>
                  <span class="label">5.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Gustafsson</span>
                           <span tagx="given-names">MG</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Extended resolution fluorescence microscopy</span>
                     <span tagx="source">Curr Opin Struct Biol</span>
                     <span tagx="year">1999</span>
                     <span tagx="volume">9</span>
                     <div tagxxx="issue">5</div>
                     <span tagx="fpage">627</span>
                     <span tagx="lpage">8</span>
                     <span class="pub-id'">[10.1016/S0959-440X(99)00016-0]</span>
                     <span class="pub-id'">[10508771]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR6"></a>
                  <span class="label">6.</span>
                  <div tagxxx="mixed-citation">Cremer C. Optics far beyond the diffraction limit. In: Traeger F, editor. Springer
                     Handbook of Lasers and Optics: Springer Berlin Heidelberg; 2012. p. 1359–97.
                  </div>
               </li>
               <li tag="ref">
                  <a name="CR7"></a>
                  <span class="label">7.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Popken</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Brero</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Koehler</span>
                           <span tagx="given-names">D</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Schmid</span>
                           <span tagx="given-names">VJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Strauss</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Wuensch</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Reprogramming of fibroblast nuclei in cloned bovine embryos involves major structural
                        remodeling with both striking similarities and differences to nuclear phenotypes of
                        
                        <i>in vitro</i> fertilized embryos
                     </span>
                     <span tagx="source">Nucleus</span>
                     <span tagx="year">2014</span>
                     <span tagx="volume">5–5</span>
                     <span tagx="fpage">542</span>
                     <span tagx="lpage">54</span>
                     <span class="pub-id'">[10.4161/19491034.2014.970107]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR8"></a>
                  <span class="label">8.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Popken</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Graf</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Krebs</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Blum</span>
                           <span tagx="given-names">H</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Schmid</span>
                           <span tagx="given-names">VJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Strauss</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Remodeling of the nuclear envelope and lamina during bovine preimplantation development
                        and its functional implications
                     </span>
                     <span tagx="source">PLoS One</span>
                     <span tagx="year">2015</span>
                     <span tagx="volume">10</span>
                     <div tagxxx="issue">5</div>
                     <span tagx="fpage">e0124619</span>
                     <span class="pub-id'">[10.1371/journal.pone.0124619]</span>
                     <span class="pub-id'">[25932910]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR9"></a>
                  <span class="label">9.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Zakhartchenko</span>
                           <span tagx="given-names">V</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Flisikowska</span>
                           <span tagx="given-names">T</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Li</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Richter</span>
                           <span tagx="given-names">T</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Wieland</span>
                           <span tagx="given-names">H</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Durkovic</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Cell-mediated transgenesis in rabbits: chimeric and nuclear transfer animals</span>
                     <span tagx="source">Biol Reprod</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">84</span>
                     <div tagxxx="issue">2</div>
                     <span tagx="fpage">229</span>
                     <span tagx="lpage">37</span>
                     <span class="pub-id'">[10.1095/biolreprod.110.087098]</span>
                     <span class="pub-id'">[20944083]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR10"></a>
                  <span class="label">10.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Hiendleder</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Wirtz</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Mund</span>
                           <span tagx="given-names">C</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Klempt</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Reichenbach</span>
                           <span tagx="given-names">HD</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Stojkovic</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Tissue-specific effects of in vitro fertilization procedures on genomic cytosine methylation
                        levels in overgrown and normal sized bovine fetuses
                     </span>
                     <span tagx="source">Biol Reprod</span>
                     <span tagx="year">2006</span>
                     <span tagx="volume">75</span>
                     <div tagxxx="issue">1</div>
                     <span tagx="fpage">17</span>
                     <span tagx="lpage">23</span>
                     <span class="pub-id'">[10.1095/biolreprod.105.043919]</span>
                     <span class="pub-id'">[16554415]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR11"></a>
                  <span class="label">11.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Schermelleh</span>
                           <span tagx="given-names">L</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Carlton</span>
                           <span tagx="given-names">PM</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Haase</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Shao</span>
                           <span tagx="given-names">L</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Winoto</span>
                           <span tagx="given-names">L</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Kner</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Subdiffraction multicolor imaging of the nuclear periphery with 3D structured illumination
                        microscopy
                     </span>
                     <span tagx="source">Science</span>
                     <span tagx="year">2008</span>
                     <span tagx="volume">320</span>
                     <div tagxxx="issue">5881</div>
                     <span tagx="fpage">1332</span>
                     <span tagx="lpage">6</span>
                     <span class="pub-id'">[10.1126/science.1156947]</span>
                     <span class="pub-id'">[18535242]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR12"></a>
                  <span class="label">12.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Huang</span>
                           <span tagx="given-names">B</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Babcock</span>
                           <span tagx="given-names">H</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Zhuang</span>
                           <span tagx="given-names">X</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Breaking the diffraction barrier: super-resolution imaging of cells</span>
                     <span tagx="source">Cell</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">143</span>
                     <div tagxxx="issue">7</div>
                     <span tagx="fpage">1047</span>
                     <span tagx="lpage">58</span>
                     <span class="pub-id'">[10.1016/j.cell.2010.12.002]</span>
                     <span class="pub-id'">[21168201]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR13"></a>
                  <span class="label">13.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Wallace</span>
                           <span tagx="given-names">W</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Schaefer</span>
                           <span tagx="given-names">LH</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Swedlow</span>
                           <span tagx="given-names">JR</span>,
                        </span>
                     </span>
                     <span tagx="article-title">A workingperson's guide to deconvolution in light microscopy</span>
                     <span tagx="source">BioTechniques</span>
                     <span tagx="year">2001</span>
                     <span tagx="volume">31</span>
                     <div tagxxx="issue">5</div>
                     <span tagx="fpage">1076</span>
                     <span tagx="lpage">8</span>
                     <span class="pub-id'">[11730015]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR14"></a>
                  <span class="label">14.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Wei</span>
                           <span tagx="given-names">D</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Jacobs</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Modla</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Zhang</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Young</span>
                           <span tagx="given-names">CL</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Cirino</span>
                           <span tagx="given-names">R</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">High-resolution three-dimensional reconstruction of a whole yeast cell using focused-ion
                        beam scanning electron microscopy
                     </span>
                     <span tagx="source">BioTechniques</span>
                     <span tagx="year">2012</span>
                     <span tagx="volume">53</span>
                     <div tagxxx="issue">1</div>
                     <span tagx="fpage">41</span>
                     <span tagx="lpage">8</span>
                     <span class="pub-id'">[22780318]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR15"></a>
                  <span class="label">15.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Huang</span>
                           <span tagx="given-names">B</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Bates</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Zhuang</span>
                           <span tagx="given-names">X</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Super-resolution fluorescence microscopy</span>
                     <span tagx="source">Annu Rev Biochem</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">78</span>
                     <span tagx="fpage">993</span>
                     <span tagx="lpage">1016</span>
                     <span class="pub-id'">[10.1146/annurev.biochem.77.061906.092014]</span>
                     <span class="pub-id'">[19489737]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR16"></a>
                  <span class="label">16.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Mollenhauer</span>
                           <span tagx="given-names">HH</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Artifacts caused by dehydration and epoxy embedding in transmission electron microscopy</span>
                     <span tagx="source">Microsc Res Tech</span>
                     <span tagx="year">1993</span>
                     <span tagx="volume">26</span>
                     <div tagxxx="issue">6</div>
                     <span tagx="fpage">496</span>
                     <span tagx="lpage">512</span>
                     <span class="pub-id'">[10.1002/jemt.1070260604]</span>
                     <span class="pub-id'">[8305727]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR17"></a>
                  <span class="label">17.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Al-Amoudi</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Studer</span>
                           <span tagx="given-names">D</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Dubochet</span>
                           <span tagx="given-names">J</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Cutting artefacts and cutting process in vitreous sections for cryo-electron microscopy</span>
                     <span tagx="source">J Struct Biol</span>
                     <span tagx="year">2005</span>
                     <span tagx="volume">150</span>
                     <div tagxxx="issue">1</div>
                     <span tagx="fpage">109</span>
                     <span tagx="lpage">21</span>
                     <span class="pub-id'">[10.1016/j.jsb.2005.01.003]</span>
                     <span class="pub-id'">[15797735]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR18"></a>
                  <span class="label">18.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Zhong</span>
                           <span tagx="given-names">H</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Photoactivated localization microscopy (PALM): an optical technique for achieving
                        ~10-nm resolution
                     </span>
                     <span tagx="source">Cold Spring Harb Protoc</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">2010</span>
                     <div tagxxx="issue">12</div>
                     <span tagx="fpage">pdb top91</span>
                     <span class="pub-id'">[10.1101/pdb.top91]</span>
                     <span class="pub-id'">[21123432]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR19"></a>
                  <span class="label">19.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Rust</span>
                           <span tagx="given-names">MJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Bates</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Zhuang</span>
                           <span tagx="given-names">X</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Sub-diffraction-limit imaging by stochastic optical reconstruction microscopy (STORM)</span>
                     <span tagx="source">Nat Methods</span>
                     <span tagx="year">2006</span>
                     <span tagx="volume">3</span>
                     <div tagxxx="issue">10</div>
                     <span tagx="fpage">793</span>
                     <span tagx="lpage">5</span>
                     <span class="pub-id'">[10.1038/nmeth929]</span>
                     <span class="pub-id'">[16896339]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR20"></a>
                  <span class="label">20.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Hell</span>
                           <span tagx="given-names">SW</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Wichmann</span>
                           <span tagx="given-names">J</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Breaking the diffraction resolution limit by stimulated emission: stimulated-emission-depletion
                        fluorescence microscopy
                     </span>
                     <span tagx="source">Opt Lett</span>
                     <span tagx="year">1994</span>
                     <span tagx="volume">19</span>
                     <div tagxxx="issue">11</div>
                     <span tagx="fpage">780</span>
                     <span tagx="lpage">2</span>
                     <span class="pub-id'">[10.1364/OL.19.000780]</span>
                     <span class="pub-id'">[19844443]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR21"></a>
                  <span class="label">21.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Markaki</span>
                           <span tagx="given-names">Y</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Smeets</span>
                           <span tagx="given-names">D</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Fiedler</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Schmid</span>
                           <span tagx="given-names">VJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Schermelleh</span>
                           <span tagx="given-names">L</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Cremer</span>
                           <span tagx="given-names">T</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">The potential of 3D-FISH and super-resolution structured illumination microscopy for
                        studies of 3D nuclear architecture: 3D structured illumination microscopy of defined
                        chromosomal structures visualized by 3D (immuno)-FISH opens new perspectives for studies
                        of nuclear architecture
                     </span>
                     <span tagx="source">Bioessays</span>
                     <span tagx="year">2012</span>
                     <span tagx="volume">34</span>
                     <div tagxxx="issue">5</div>
                     <span tagx="fpage">412</span>
                     <span tagx="lpage">26</span>
                     <span class="pub-id'">[10.1002/bies.201100176]</span>
                     <span class="pub-id'">[22508100]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR22"></a>
                  <span class="label">22.</span>
                  <div tagxxx="mixed-citation">Huebner B, Cremer T, Neumann J. Correlative Microscopy of Individual Cells: Sequential
                     Application of Microscopic Systems with Increasing Resolution to Study the Nuclear
                     Landscape. Methods in Molecular Biology. 2013;1042(Yaron Shav-Tal (ed.), Imaging Gene
                     Expression: Methods and Protocols):299 - 336. doi:DOI 10.1007/978-1-62703-526-2_21.
                  </div>
               </li>
               <li tag="ref">
                  <a name="CR23"></a>
                  <span class="label">23.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Markaki</span>
                           <span tagx="given-names">Y</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gunkel</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Schermelleh</span>
                           <span tagx="given-names">L</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Beichmanis</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Neumann</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Heidemann</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Functional nuclear organization of transcription and DNA replication: a topographical
                        marriage between chromatin domains and the interchromatin compartment
                     </span>
                     <span tagx="source">Cold Spring Harb Symp Quant Biol</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">75</span>
                     <span tagx="fpage">475</span>
                     <span tagx="lpage">92</span>
                     <span class="pub-id'">[10.1101/sqb.2010.75.042]</span>
                     <span class="pub-id'">[21467142]</span>
                  </span>
               </li>
            </ul>
         </div>
      </div>
   </div>
</html>