<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <div xmlns="" tagx="article">
      <head>
         <style type="text/css"> table, tr, td {border : thin;}</style>
         <span class="citation_publisher">BioMed Central</span>
         <h2 class="citation-title">One-step real-time RT-PCR assays for serotyping dengue virus in clinical samples</h2>
         <span class="doi">10.1186/s12879-015-1226-z</span>
         <div class="contrib-group">contrib: 
            	  CONT
            	  
            <span class="citation_author">Erik Alm</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Gunnel Lindegren</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Kerstin Ingrid Falk</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Nina Lagerqvist</span>
            <span class="citation_author_institution_ref"></span>
            	AFF
            	  
            <span class="citation_author_institution">[ ]</span>
            	AFF
            	  
            <span class="citation_author_institution">[ ]</span>
         </div>
         <span class="email">[]</span>
         <span class="conflict">[]</span>
         <span class="contribs">[]</span>
         <span class="citation_pubdate">[2015-11-2]</span>
         <span class="citation_copyright">[Copyright : ]</span>
         <span class="citation_licence">[Licence Open AccessThis article is distributed under the terms of the Creative Commons
            Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/),
            which permits unrestricted use, distribution, and reproduction in any medium, provided
            you give appropriate credit to the original author(s) and the source, provide a link
            to the Creative Commons license, and indicate if changes were made. The Creative Commons
            Public Domain Dedication waiver (http://creativecommons.org/publicdomain/zero/1.0/)
            applies to the data made available in this article, unless otherwise stated.]
         </span>
         <span class="citation_funding">[]</span>
         <span class="citation_pagecount">[]</span>
         <span class="citation_data">[]</span>
      </head>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <div id="">
            <h2>Background</h2>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Dengue is one of the leading causes of morbidity in tropical and subtropical regions
               and infection with any of the four dengue virus serotypes (DENV1-4) result in a wide
               range of clinical manifestations. Given the geographic expansion of DENV1-4, assays
               for serotyping are needed to be able to perform surveillance and epidemiological studies.
               In this study, we describe the design and validation of one-step real-time serotype-specific
               DENV RT-PCR assays.
            </p>
         </div>
         <div id="">
            <h2>Methods</h2>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The DENV1, DENV2, DENV3, and DENV4 RT-PCR assays were designed using all available
               whole genome DENV sequences in the NCBI nucleotide collection. Because of the high
               mutation rates of RNA viruses, the assays were performed in singleplex format to enable
               quick modifications to the primer and probe sequences when new genetic variants emerge.
               The analytical performance of the RT-PCR assays were evaluated using in vitro transcribed
               RNA and their specificity was determined by testing 24 DENV isolates, external DENV
               control panels and RNA preparation of non-DENV flaviviruses and non-dengue clinical
               samples. Additionally, the clinical performance of the serotype-specific DENV RT-PCR
               were compared to that of the CDC DENV-1-4 RT-PCR using 85 clinical samples collected
               from patients presenting with acute dengue.
            </p>
         </div>
         <div id="">
            <h2>Results</h2>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The RT-PCR assays were found to be specific for their respective serotype and did
               not cross-react with other flaviviruses or human mRNA. All assays had a linear dynamic
               range of 10
               <sup>2</sup> to 10
               <sup>6</sup> copies/reaction with detection limits between 12 and 44 copies/reaction. When testing
               sera from 85 confirmed acute dengue cases, the serotype-specific DENV RT-PCR assays
               had 100 % positive agreement with the FDA-approved CDC DENV-1-4 RT-PCR assay performed
               in a singleplex format. Additionally 15 samples that tested negative in the CDC DENV-1-4-
               RT-PCR assay were found positive using the serotype-specific DENV RT-PCR assays.
            </p>
         </div>
         <div id="">
            <h2>Conclusions</h2>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Our results suggest that these RT-PCR assays are useful alternatives to existing methods
               for serotyping DENV in clinical sera.
            </p>
         </div>
         <div id="">
            <h2>Electronic supplementary material</h2>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The online version of this article (doi:10.1186/s12879-015-1226-z) contains supplementary
               material, which is available to authorized users.
            </p>
         </div>
      </div>
      <div id="Sec1">
         <h2>Background</h2>
         <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Dengue is the most widespread mosquito-borne viral disease with an estimated 390 million
            infections yearly [
            <span ref-type="bibr" rid="CR1">
               <sup>
                  <a href="#CR1">1</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="CR2">
               <sup>
                  <a href="#CR2">2</a>
               </sup>
            </span>]. In the last decades, the global distribution of dengue virus (DENV) has expanded
            to include more geographic areas and all four serotypes (DENV1-4) are now present
            in Asia, Africa, and the Americas [
            <span ref-type="bibr" rid="CR3">
               <sup>
                  <a href="#CR3">3</a>
               </sup>
            </span>]. Not only is dengue a public health concern in endemic regions, dengue has also
            been increasingly reported in returning travelers [
            <span ref-type="bibr" rid="CR4">
               <sup>
                  <a href="#CR4">4</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="CR5">
               <sup>
                  <a href="#CR5">5</a>
               </sup>
            </span>]. Infection with any of the DENV serotypes is commonly asymptomatic or presented
            as a feverish illness accompanied by severe headache, muscle and joint pains, and
            vomiting [
            <span ref-type="bibr" rid="CR6">
               <sup>
                  <a href="#CR6">6</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="CR7">
               <sup>
                  <a href="#CR7">7</a>
               </sup>
            </span>]. Approximately 500 000 cases annually require hospitalization [
            <span ref-type="bibr" rid="CR3">
               <sup>
                  <a href="#CR3">3</a>
               </sup>
            </span>]. Severe dengue is characterized by abdominal pain, bleeding, fatigue, and persistent
            vomiting, and can lead to deadly complications [
            <span ref-type="bibr" rid="CR8">
               <sup>
                  <a href="#CR8">8</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="CR9">
               <sup>
                  <a href="#CR9">9</a>
               </sup>
            </span>].
         </p>
         <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Acute dengue can be diagnosed by detecting the DENV genome. A number of real-time
            RT-PCR assays have previously been developed [
            <span ref-type="bibr" rid="CR10">
               <sup>
                  <a href="#CR10">10</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="CR14">
               <sup>
                  <a href="#CR14">14</a>
               </sup>
            </span>], including a universal DENV real-time RT-PCR designed and validated at our laboratory
            [
            <span ref-type="bibr" rid="CR15">
               <sup>
                  <a href="#CR15">15</a>
               </sup>
            </span>]. Methods detecting DENV genomes are recommended by the World Health Organization
            to be used for laboratory confirmation of dengue during the first five to six days
            after symptomatic onset [
            <span ref-type="bibr" rid="CR16">
               <sup>
                  <a href="#CR16">16</a>
               </sup>
            </span>] and PCR-based techniques are at present the only methods for determining the infecting
            serotype during acute disease.
         </p>
         <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Here we present the development and clinical validation of four DENV serotype-specific
            real-time RT-PCR assays useful in situations requiring serotyping. Like many RNA viruses,
            DENV displays considerable genetic diversity [
            <span ref-type="bibr" rid="CR17">
               <sup>
                  <a href="#CR17">17</a>
               </sup>
            </span>]. Consequently, these serotype-specific RT-PCR assays were set up in singleplex format,
            allowing easy modification of the individual assays when new genetic variants emerge.
            The DENV1, DENV2, DENV3, and DENV4 real-time RT-PCR assays were designed using all
            DENV1 (
            <i>n</i> = 1532), DENV2 (
            <i>n</i> = 1117), DENV3 (
            <i>n</i> = 832), and DENV4 (
            <i>n</i> = 145) whole genome sequences available in the NCBI nucleotide collection at the
            design stage. The performance and specificity of the assays were evaluated by analyzing
            in vitro transcribed RNA, DENV isolates, and external control panels. Using 85 serum
            samples obtained from travelers returning from the tropics presenting with acute dengue,
            the DENV1, DENV2, DENV3 and DENV4 RT-PCR assays were compared to the CDC DENV-1-4
            RT-PCR assay [
            <span ref-type="bibr" rid="CR10">
               <sup>
                  <a href="#CR10">10</a>
               </sup>
            </span>]. The theoretical and clinical validation of these DENV serotype-specific RT-PCR
            assays suggest that they are good alternatives to existing methods for serotyping
            DENV in clinical samples.
         </p>
      </div>
      <div id="Sec2">
         <h2>Methods</h2>
         <div id="Sec3">
            <h3>Primer and probe design</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">To be able to find serotype-specific conserved regions, all whole genome sequences
               of DENV1-4 available at the design stage (2014–10–31) were downloaded from NCBI and
               used for assay design. Multiple sequence alignments containing the genomic sequences
               of the individual serotypes and alignments containing all DENV genomic sequences were
               created using CLC Genomics Workbench 7.5 (
               <div tagxxx="ext-link">www.clcbio.com</div>). Primers and probes were constructed using in-house software. Melting temperatures
               (Tm) were verified using Primer Express® v3.0 (Applied Biosystems®). Theoretical specificity
               of the systems was investigated using BLAST against the NCBI nucleotide database with
               very loose match criteria (word-size = 7, E-cutoff = 1000, match/mismatch cost +1/-1,
               gap cost 5/2).
            </p>
         </div>
         <div id="Sec4">
            <h3>RNA extraction</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Viral RNA was extracted from 140 μL supernatant of infected cells or from patient
               sera using QIAamp® viral RNA extraction kit (Qiagen), following the protocol from
               the manufacturer. RNA was eluted in 60 μL elution buffer and stored at –80 °C pending
               analysis. Infectious cell supernatant was subjected to Trizol LS®/chloroform treatment
               following the manufacturer’s instruction (Invitrogen™) prior to RNA extraction. To
               confirm the integrity of the extraction reagents and the successful recovery of RNA
               from clinical samples, the presence of human beta-actin mRNA was analyzed using a
               commercial TaqMan® probe-based RT-PCR assay (Applied Biosystems®).
            </p>
         </div>
         <div id="Sec5">
            <h3>One-step real-time RT-PCR assays</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The DENV1, DENV2, DENV3, and DENV4 RT-PCR assays were carried out in 25 μL reaction
               mixtures containing 5 μL template RNA, TaqMan® Fast Virus 1-step mastermix (Applied
               Biosystems®), UltraPure™ DNase/RNase-Free Distilled Water (Invitrogen™), 0.9 μM of
               each primer, and 0.2 μM probe (Table 
               <span ref-type="table" rid="Tab1">
                  <a href="#Tab1">1</a>
               </span>). The MGB-probes were labeled with FAM reporter dye and a non-fluorescent quencher.
               Primers and probes were purchased from Life Technologies. Amplification and detection
               were performed in a StepOne Plus real-time PCR system (Applied Biosystems®). Thermocycling
               parameters were as follows: reverse transcription at 50 °C for 5 min, inactivation
               at 95 °C for 20 s, followed by 45 cycles of fluorescence detection at 95 °C for 3
               s, and annealing at 60 °C for 30 s. The baseline and threshold were set using the
               auto-baseline and threshold feature in StepOne Software v2.2.2 (Applied Biosystems®).
               Samples were considered positive if target amplification was recorded within 40 cycles.
               <div id="Tab1">
                  <span class="label">Table 1</span>
                  <caption>
                     <p>Primer and probe sequences</p>
                  </caption>
                  <table>
                     <thead>
                        <tr>
                           <th>Assay</th>
                           <th>Primer/probe</th>
                           <th>Amplicon size (nt)</th>
                           <th>Target region</th>
                           <th>Sequences (5’-3’)</th>
                           <th>Position</th>
                           <th>Tm</th>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>DENV1</td>
                           <td>DENV1_F</td>
                           <td>71</td>
                           <td>NS5</td>
                           <td>CAATGGATGACAACAGAAGAYATG</td>
                           <td>9974–9997</td>
                           <td>56.6</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>DENV1_R</td>
                           <td></td>
                           <td></td>
                           <td>TCCATCCATGGGTTTTCCTCTAT</td>
                           <td>10022–10044</td>
                           <td>59.5</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>DENV1_P</td>
                           <td></td>
                           <td></td>
                           <td>TCAGTGTGGAATAGGGTTT</td>
                           <td>10001–10019</td>
                           <td>70.0</td>
                        </tr>
                        <tr>
                           <td>DENV2</td>
                           <td>DENV2_F</td>
                           <td>199</td>
                           <td>E protein</td>
                           <td>GCAGAAACACAACATGGAACRATAGT</td>
                           <td>1873–1898</td>
                           <td>56.6</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>DENV2_R</td>
                           <td></td>
                           <td></td>
                           <td>TGATGTAGCTGTCTCCRAATGG</td>
                           <td>2050–2071</td>
                           <td>59.8</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>DENV2_P</td>
                           <td></td>
                           <td></td>
                           <td>TCAACATAGAAGCAGAACC</td>
                           <td>2030–2048</td>
                           <td>68.0</td>
                        </tr>
                        <tr>
                           <td>DENV3</td>
                           <td>DENV3_F</td>
                           <td>167</td>
                           <td>NS1</td>
                           <td>ATGGAATGTGTGGGAGGTGG</td>
                           <td>2860–2879</td>
                           <td>59.1</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>DENV3_R</td>
                           <td></td>
                           <td></td>
                           <td>GGCTTTCTATCCARTAGCCCATG</td>
                           <td>3004–3026</td>
                           <td>59.8</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>DENV3_P</td>
                           <td></td>
                           <td></td>
                           <td>TATGGCTGAAACTCCGAG</td>
                           <td>2913–2930</td>
                           <td>68.0</td>
                        </tr>
                        <tr>
                           <td>DENV4</td>
                           <td>DENV4_F</td>
                           <td>114</td>
                           <td>5’ UTR/</td>
                           <td>GCAGATCTCTGGAAAAATGAACCA</td>
                           <td>86–109</td>
                           <td>60.4</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>DENV4_R</td>
                           <td></td>
                           <td>capsid</td>
                           <td>GAGAATCTCTTCACCAACCCYTG</td>
                           <td>177–199</td>
                           <td>59.8</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>DENV4_P</td>
                           <td></td>
                           <td>protein</td>
                           <td>TCAATATGCTGAAACGC</td>
                           <td>136–152</td>
                           <td>68.0</td>
                        </tr>
                     </div>
                  </table>
                  <table-wrap-foot>
                     <p>Degenerated nucleotides (nt): R: A/G, Y: G/A/C. The MGB-modified probes were labelled
                        with FAM and a non-fluorescence quencher (NFQ)
                     </p>
                     <p>Genome positions are given according to the NCBI reference sequence for DENV1 [GenBank:NC_001477],
                        DENV2 [GenBank:NC_001474], DENV3 [GenBank:NC_001475], and DENV4 [GenBank:NC_002640]
                     </p>
                     <p>The mean melting temperatures (Tm) are shown for primers with degenerated nucleotides</p>
                  </table-wrap-foot>
               </div>
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The CDC DENV-1-4 Real-Time RT-PCR Assay [
               <span ref-type="bibr" rid="CR10">
                  <sup>
                     <a href="#CR10">10</a>
                  </sup>
               </span>] was performed in singleplex reactions following the manufacturer’s instructions
               (Centers for Disease Control and Prevention) in 25 μL volumes using the SuperScript®
               III Platinum® One-Step qRT-PCR Kit (Invitrogen™). Amplification and detection were
               performed in a 7500 Fast DX Real-time PCR instrument (Applied Biosystems®). The data
               analysis were performed as described in the manufacturer’s guidelines. In short, the
               threshold was adjusted to fall within the PCR exponential phase in the linear view.
               The manufacturer’s instructions specifies that a specimen is considered positive for
               either DENV1, 2, 3, or 4 if the amplification curve crosses the threshold line within
               37 cycles (Cq &lt; 37).
            </p>
         </div>
         <div id="Sec6">
            <h3>Amplification efficiency, linear dynamic range, and limit of detection</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The efficiency of amplification and the dynamic range were determined by testing triplicates
               of 10-fold serial dilutions of in vitro transcribed RNA (BioSynthesis Inc.). The RNA
               transcripts were 158, 235, 189, and 133 nt in length and were based on the sequences
               of DENV1 [GenBank:GU131828], DENV2 [GenBank:EU482743], DENV3 [GenBank:AY679147], and
               DENV4 [GenBank:GQ199881], respectively. The RNA was diluted in RNase/DNase-free water
               (Life Technologies) and stored at –80°C. The RNA concentration was determined by the
               manufacturer and verified using a Nano Drop ND-1000 spectrophotometer (Thermo Scientific).
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The limit of detection (LoD) was determined by testing two-fold serially diluted in
               vitro transcribed RNA close to the detection limit of the assay. The transcript RNA
               was tested in eight replicates in three separate experiments and LoD was defined as
               the last dilution in which a fluorescence signal could be detected in all 24 replicates.
               Dilutions and other pipetting procedures were performed using a robotic QIAgility™
               workstation (Qiagen).
            </p>
         </div>
         <div id="Sec7">
            <h3>Specificity study</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">DENV used in this study was DENV1 (8356/10, Hawaii, and West Pac.), DENV2 (4397/11
               and NewGuinea C), DENV3 (3140/09 and H-87), DENV4 (3274/09 and H-241) and 15 partly
               sequenced DENV isolates: DENV1 (
               <i>n</i> = 8), DENV2 (
               <i>n</i> = 3), and DENV3 (
               <i>n</i> = 4). The following 12 non-DENV agents were selected for the specificity validation:
               Andes virus (ANDV), chikungunya virus (CHIKV), hantaan virus (HTNV), Lassa virus (LASV),
               Rift Valley fever virus (RVFV), Seoul virus (SEOV), Dobrava-Belgrade virus (DOBV),
               tick-borne encephalitis virus (TBEV), Usutu virus (USUV), West Nile virus (WNV), yellow
               fever virus (YFV), and Zika virus (ZIKV). Hantaviruses were propagated in Vero E6
               cells (ATCC: 1586) in Minimal Essential Medium supplemented with 5 % fetal bovine
               serum, 1 mM HEPES, and 100 U/mL penicillin, and 100 μg/mL streptomycin. CHIKV, DENV,
               LASV, RVFV, TBEV, USUV, WNV, YFV, and ZIKV were propagated in Vero cells (ATCC: CCL-81)
               in Medium 199 supplemented with 5 % fetal bovine serum, 1 mM HEPES, and 100 U/mL penicillin,
               and 100 μg/mL streptomycin. All reagents were from Gibco, Life Technologies. Cells
               were maintained at 37 °C in a humidified atmosphere containing 5 % CO2. All handling
               of infectious material was performed in bio-safety level (BSL) 3, and for LASV, BSL
               4 containment laboratories. External DENV control panels were obtained from Quality
               Control for Molecular Diagnostics (QCMD) in the years of 2011, 2012, 2013, and 2014.
            </p>
         </div>
         <div id="Sec8">
            <h3>Clinical samples</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Serum samples (
               <i>n</i> = 85) collected during 2014–2015 from Swedish residents returning from travel with
               confirmed dengue were obtained from the biorepository of the Public Health Agency
               of Sweden as stipulated in the regulations for use of such material in diagnostic
               development and quality assessment. The Public Health Agency of Sweden performs all
               dengue diagnostics in Sweden and the biobank deposited samples used in this study
               had previously been confirmed DENV positive by using a universal one-step real-time
               DENV RT-PCR [
               <span ref-type="bibr" rid="CR15">
                  <sup>
                     <a href="#CR15">15</a>
                  </sup>
               </span>] and the SD Bioline Dengue NS1 Ag Rapid Test (Standard Diagnostics, Inc.). Additionally,
               all patients had been confirmed as having dengue using Panbio® Dengue IgM capture
               ELISA (Alere™) and in a few cases an in-house IFA [
               <span ref-type="bibr" rid="CR18">
                  <sup>
                     <a href="#CR18">18</a>
                  </sup>
               </span>]. Serum samples (
               <i>n</i> = 40) collected during 2014 from non-dengue cases were used for testing possible
               unspecific reactions of the DENV1, DENV2, DENV3, and DENV4 real-time RT-PCR assays
               with human RNA. No informed consent or ethical permit was required since The Swedish
               Ethical Review Act (2003:16), Ethical Review of Research Involving Humans (
               <div tagxxx="ext-link">http://www.epn.se/media/1205/the_ethical_review_act.pdf</div>) is not applicable for diagnostic development and quality assessment.
            </p>
         </div>
      </div>
      <div id="Sec9">
         <h2>Results and discussion</h2>
         <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The primers and probes were designed using 1532 DENV1, 1117 DENV2, 832 DENV3, and
            145 DENV4 complete genome sequences. To minimize the possibility of unspecific detection
            of non-targeted serotypes, the individual RT-PCR assays were designed to have a minimum
            number of mismatches to the targeted DENV serotype (Additional file 
            <span ref-type="media" rid="MOESM1">1</span>) while a maximum number of mismatches in the 3’ region of the primers and evenly
            distributed mismatches in the probe in respect to the sequences of non-targeted serotypes
            (Additional file 
            <span ref-type="media" rid="MOESM2">2</span>). The sequences of the primers and probes can be found in Table 
            <span ref-type="table" rid="Tab1">
               <a href="#Tab1">1</a>
            </span>. BLAST against all non-DENV sequences in the NCBI Nucleotide Collection indicated
            no risk of cross-reaction with other species.
         </p>
         <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">To determine the amplification efficiency and the linear dynamic range, in vitro transcribed
            RNA was 10-fold serially diluted and tested in triplicates in the individual assays.
            All assays had a linear dynamic range from 10
            <sup>2</sup> to 10
            <sup>8</sup> copies/reaction (Fig. 
            <span ref-type="fig" rid="Fig1">1</span>). Over this interval the amplification efficiency was 96 % (R
            <sup>2</sup> = 0.999, y-intercept = 37.4), 97 % (R
            <sup>2</sup> = 1, y-intercept = 38.8), 97 % (R
            <sup>2</sup> = 0.999, y-intercept = 37.7), and 102 % (R
            <sup>2</sup> = 0.999, y-intercept = 38.1) for the DENV1, DENV2, DENV3, and DENV4 RT-PCR assay,
            respectively (Fig. 
            <span ref-type="fig" rid="Fig1">1</span>).
            <div tagxxx="fig">
               <span class="label">Fig. 1</span>
               <caption>
                  <p>Linear dynamic range. The linear dynamic ranges of the RT-PCR assays were determined
                     by testing triplicates of 10-fold serially diluted in vitro transcribed RNA. Panel
                     (
                     <b>a</b>) represents the DENV1 assay, and (
                     <b>b</b>), (
                     <b>c</b>), and (
                     <b>d</b>) the DENV2, DENV3, and DENV4 RT-PCR assays, respectively. Each dot represents the
                     mean Cq-value from three replicates, the error bars indicate 95 % confidence intervals,
                     and the lines represents the best fitting lin-log regression models
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
         </p>
         <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">LoD was determined by testing two-fold serial dilutions of in vitro transcribed RNA
            in 24 replicates (Fig. 
            <span ref-type="fig" rid="Fig2">2</span>). The detection limits for the one-step real-time DENV1, DENV2, DENV3, and DENV4
            RT-PCR assays were 37 copies/reaction (Fig. 
            <span ref-type="fig" rid="Fig2">2a</span>), 20 copies/reaction (Fig. 
            <span ref-type="fig" rid="Fig2">2b</span>), 12 copies/reaction (Fig. 
            <span ref-type="fig" rid="Fig2">2c</span>), and 44 copies/reaction (Fig. 
            <span ref-type="fig" rid="Fig2">2d</span>), respectively.
            <div tagxxx="fig">
               <span class="label">Fig. 2</span>
               <caption>
                  <p>Limit of detection. Limit of detection was determined by assaying eight replicates
                     of twofold serially diluted RNA transcripts in three separate experiments. The number
                     of positives per total number of replicates tested is shown for (
                     <b>a</b>) the DENV1 RT-PCR assay, (
                     <b>b</b>) the DENV2 RT-PCR assay, (
                     <b>c</b>) the DENV3 RT-PCR assay, and (
                     <b>d</b>) the DENV4 RT-PCR assay. Horizontal lines indicate mean values, boxes denote the
                     25th to 75th percentiles, whiskers the 5–95 % percentiles, and dots represent outliers
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
         </p>
         <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The assays were evaluated for their specificity by testing 24 DENV isolates and four
            external QCMD control panels, each including 10 samples containing different concentrations
            of DENV and two negative controls (Table 
            <span ref-type="table" rid="Tab2">
               <a href="#Tab2">2</a>
            </span>). The four serotype-specific RT-PCR assays correctly classified all samples with
            no observed cross-reactions (Table 
            <span ref-type="table" rid="Tab2">
               <a href="#Tab2">2</a>
            </span>). To confirm that the primers and probes of the DENV1, DENV2, DENV3, and DENV4 RT-PCR
            assays did not react with human RNA, RNA preparations of 40 serum samples obtained
            from patient with non-dengue diagnosis were analyzed. All of these samples contained
            mRNA of human beta-actin, indicating adequate RNA extraction. No signal was observed
            in the DENV1, DENV2, DENV3, and DENV4 RT-PCR assays.
            <div id="Tab2">
               <span class="label">Table 2</span>
               <caption>
                  <p>Specificity of the DENV1, DENV2, DENV3, and DENV4 real-time RT-PCR assays</p>
               </caption>
               <table>
                  <thead>
                     <tr>
                        <th>Sample</th>
                        <th>No. of positive samples/no. tested</th>
                     </tr>
                     <tr>
                        <th></th>
                        <th>DENV1 RT-PCR</th>
                        <th>DENV2 RT-PCR</th>
                        <th>DENV3 RT-PCR</th>
                        <th>DENV4 RT-PCR</th>
                     </tr>
                  </thead>
                  <div tagxxx="tbody">
                     <tr>
                        <td>DENV1 isolates</td>
                        <td>11/11</td>
                        <td>0/11</td>
                        <td>0/11</td>
                        <td>0/11</td>
                     </tr>
                     <tr>
                        <td>DENV2 isolates</td>
                        <td>0/5</td>
                        <td>5/5</td>
                        <td>0/5</td>
                        <td>0/5</td>
                     </tr>
                     <tr>
                        <td>DENV3 isolates</td>
                        <td>0/6</td>
                        <td>0/6</td>
                        <td>6/6</td>
                        <td>0/6</td>
                     </tr>
                     <tr>
                        <td>DENV4 isolates</td>
                        <td>0/2</td>
                        <td>0/2</td>
                        <td>0/2</td>
                        <td>2/2</td>
                     </tr>
                     <tr>
                        <td>QCMD panel, DENV1</td>
                        <td>17/17</td>
                        <td>0/17</td>
                        <td>0/17</td>
                        <td>0/17</td>
                     </tr>
                     <tr>
                        <td>QCMD panel, DENV2</td>
                        <td>0/11</td>
                        <td>11/11</td>
                        <td>0/11</td>
                        <td>0/11</td>
                     </tr>
                     <tr>
                        <td>QCMD panel, DENV3</td>
                        <td>0/8</td>
                        <td>0/8</td>
                        <td>8/8</td>
                        <td>0/8</td>
                     </tr>
                     <tr>
                        <td>QCMD panel, DENV4</td>
                        <td>0/4</td>
                        <td>0/4</td>
                        <td>0/4</td>
                        <td>4/4</td>
                     </tr>
                     <tr>
                        <td>QCMD negative controls</td>
                        <td>0/8</td>
                        <td>0/8</td>
                        <td>0/8</td>
                        <td>0/8</td>
                     </tr>
                  </div>
               </table>
               <table-wrap-foot>
                  <p>RNA preparations from DENV1, DENV2, DENV3, and DENV4 isolates obtained Cq-values between
                     16 and 25 when analyzed using their respective serotype-specific RT-PCR assay
                  </p>
                  <p>The concentrations of DENV1, DENV2, and DENV3 included in the QCMD panels were 10
                     <sup>3</sup> to 10
                     <sup>6</sup>, 10
                     <sup>2</sup> to 10
                     <sup>5</sup>, and 10
                     <sup>3</sup> to 10
                     <sup>5</sup> copies/mL, respectively. The concentration of DENV4 was 10
                     <sup>5</sup> copies/mL in all four QCMD panel samples
                  </p>
               </table-wrap-foot>
            </div>
         </p>
         <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The differential diagnosis of dengue can be challenging due to the wide range of clinical
            manifestations [
            <span ref-type="bibr" rid="CR6">
               <sup>
                  <a href="#CR6">6</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="CR9">
               <sup>
                  <a href="#CR9">9</a>
               </sup>
            </span>]. For this reason, the specificity of the RT-PCR assays were further evaluated by
            testing RNA extracted from high-titred solutions of other flaviviruses and viruses
            important for dengue differential diagnosis. No unspecific reactions were observed
            for RNA extracted from preparations of other flaviviruses (TBEV, USUV, WNV, YFV, and
            ZIKV) or non-flaviviruses (ANDV, CHIKV, DOBV, HTNV, RVFV, LASV, and SEOV).
         </p>
         <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The serotype-specific DENV RT-PCR assays were evaluated for their clinical performance
            by retrospectively testing 85 serum samples obtained from patients presenting with
            acute dengue (Table 
            <span ref-type="table" rid="Tab3">
               <a href="#Tab3">3</a>
            </span>). These 85 serum samples were also tested using the FDA-approved CDC DENV-1-4 RT-PCR
            (reference method) performed in singleplex format [
            <span ref-type="bibr" rid="CR10">
               <sup>
                  <a href="#CR10">10</a>
               </sup>
            </span>] (Table 
            <span ref-type="table" rid="Tab3">
               <a href="#Tab3">3</a>
            </span>). The DENV1, DENV2, DENV3, and DENV4 RT-PCR assay (test 
            <span ref-type="sec" rid="Sec2">methods</span>) results were all in 100 % positive agreement [
            <span ref-type="bibr" rid="CR19">
               <sup>
                  <a href="#CR19">19</a>
               </sup>
            </span>] to the CDC DENV-1-4 RT-PCR results , i.e. all samples that were positive in the
            reference method were also positive in the test methods (Table 
            <span ref-type="table" rid="Tab3">
               <a href="#Tab3">3</a>
            </span> and Additional file 
            <span ref-type="media" rid="MOESM3">3</span>). An additional 15 samples not detected by the CDC DENV RT-PCR assay were classified
            as DENV1 (
            <i>n</i> = 6), DENV2 (
            <i>n</i> = 6), DENV3 (
            <i>n</i> = 2), and DENV4 (
            <i>n</i> = 1) using the DENV1, DENV2, DENV3, and DENV RT-PCR assays, respectively (Table 
            <span ref-type="table" rid="Tab3">
               <a href="#Tab3">3</a>
            </span> and Additional file 
            <span ref-type="media" rid="MOESM3">3</span>). Four of these 15 samples were collected between days 4 and 5 after symptomatic
            onset, the remaining 11 samples were collected days 6–9. The DENV1 and DENV2 assays
            were found to be more sensitive than the CDC DENV1 and DENV2 assays (Chi-square test,
            
            <i>p</i> = 0.009 and 
            <i>p</i> = 0.003, respectively). The DENV3 and DENV4 assays were not found to be significantly
            more sensitive than the CDC DENV3 and DENV4 assays (Chi-square test, 
            <i>p</i> = 0.11 and 
            <i>p</i> = 0.28, respectively). The CDC DENV-1-4 RT-PCR assay has previously been shown to
            be less sensitive than an in-house developed RT-PCR assay especially for serum samples
            collected more than 5 days after symptomatic onset [
            <span ref-type="bibr" rid="CR20">
               <sup>
                  <a href="#CR20">20</a>
               </sup>
            </span>].
            <div id="Tab3">
               <span class="label">Table 3</span>
               <caption>
                  <p>Comparable analysis between the serotype-specific RT-PCR assays and the CDC DENV1-4
                     RT-PCR
                  </p>
               </caption>
               <table>
                  <div tagxxx="tbody">
                     <tr>
                        <td></td>
                        <td></td>
                        <td>CDC DENV1 RT-PCR</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td></td>
                        <td>negative</td>
                        <td>Positive</td>
                     </tr>
                     <tr>
                        <td>DENV1 RT-PCR</td>
                        <td>negative</td>
                        <td>36</td>
                        <td>0</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>positive</td>
                        <td>6</td>
                        <td>43</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td></td>
                        <td>CDC DENV2 RT-PCR</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td></td>
                        <td>negative</td>
                        <td>positive</td>
                     </tr>
                     <tr>
                        <td>DENV2 RT-PCR</td>
                        <td>negative</td>
                        <td>67</td>
                        <td>0</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>positive</td>
                        <td>6</td>
                        <td>12</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td></td>
                        <td>CDC DENV3 RT-PCR</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td></td>
                        <td>negative</td>
                        <td>positive</td>
                     </tr>
                     <tr>
                        <td>DENV3 RT-PCR</td>
                        <td>negative</td>
                        <td>76</td>
                        <td>0</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>positive</td>
                        <td>2</td>
                        <td>7</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td></td>
                        <td>CDC DENV4 RT-PCR</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td></td>
                        <td>negative</td>
                        <td>positive</td>
                     </tr>
                     <tr>
                        <td>DENV4 RT-PCR</td>
                        <td>negative</td>
                        <td>78</td>
                        <td>0</td>
                     </tr>
                     <tr>
                        <td></td>
                        <td>positive</td>
                        <td>1</td>
                        <td>6</td>
                     </tr>
                  </div>
               </table>
               <table-wrap-foot>
                  <p>A sample was classified as positive in the CDC DENV-1-4 real-time RT-PCR assays if
                     the amplification curve crossed the threshold line within 37 cycles
                  </p>
                  <p>Five of the six DENV1 RT-PCR positive samples, two of the six DENV2 RT-PCR positive
                     samples, both the DENV3 RT-PCR positive samples, and the DENV4 RT-PCR positive sample
                     that tested negative in respective CDC DENV-1-4 RT-PCR singleplex assay crossed the
                     threshold line between 37 and 40 cycles in the CDC assays
                  </p>
               </table-wrap-foot>
            </div>
         </p>
      </div>
      <div id="Sec10">
         <h2>Conclusions</h2>
         <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The four DENV serotype-specific RT-PCR assays presented here were able to successfully
            serotype dengue in clinical samples. RNA preparations of 40 serum samples obtained
            from patient with non-dengue diagnosis did not show unspecific reactions, however,
            it should be noted that the assay have only been evaluated using selected samples
            i.e. samples already confirmed as DENV positive. No unspecific reactions were observed
            when high-titred preparations of non-targeted DENV serotypes were analyzed, this may
            be useful especially considering that co-circulations of several serotypes as well
            as co-infections with two serotypes have been reported [
            <span ref-type="bibr" rid="CR21">
               <sup>
                  <a href="#CR21">21</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="CR23">
               <sup>
                  <a href="#CR23">23</a>
               </sup>
            </span>]. The DENV1 and DENV2 RT-PCR assays were found to be more sensitive than the corresponding
            FDA-approved CDC DENV assays and the DENV3 and DENV4 were found to be at least as
            sensitive as the corresponding CDC DENV assays when serum samples from 85 acute dengue
            cases were analyzed. This indicates that the DENV1, DENV2, DENV3 and DENV4 assays
            are useful alternatives to other methods for serotyping DENV.
         </p>
      </div>
      <div tagxxx="back">
         <div tagxxx="app-group">
            <div tagxxx="app">
               <div id="Sec11">
                  <h2>Additional files</h2>
                  <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
                     <div tagxxx="media">
                        <span class="label">Additional file 1:</span>
                        <caption>
                           <p>
                              <b>Overview of the primers and probes.</b> Vertical bars and percentages show the fraction of sequences with nucleotides deviating
                              from the consensus of (A) DENV1, (B) DENV2, (C) DENV3, and (D) DENV4 serotypes. Values
                              below 1 % are not shown. Numbers indicate genomic positions. (PDF 82 kb)
                           </p>
                        </caption>
                     </div>
                     <div tagxxx="media">
                        <span class="label">Additional file 2:</span>
                        <caption>
                           <p>
                              <b>Overview of the theoretical specificity towards non-targeted serotypes.</b> The sequences of the primers and probe in the (A) DENV1, (B) DENV2, (C) DENV3, and
                              (D) DENV4 RT-PCR plotted against their respective non-targeted serotypes. Numbers
                              above the primers and probe indicate genomic positions. (PDF 87 kb)
                           </p>
                        </caption>
                     </div>
                     <div tagxxx="media">
                        <span class="label">Additional file 3:</span>
                        <caption>
                           <p>
                              <b>Cq-value comparison between the test methods and reference method.</b> The table shows the Cq-values obtained for the 85 dengue positive samples tested
                              in the different serotype-specific RT-PCR assays. Information on days after symptomatic
                              onset were included for samples for which this data was available. (PDF 253 kb)
                           </p>
                        </caption>
                     </div>
                  </p>
               </div>
            </div>
         </div>
         <div tagxxx="glossary">
            <div tagxxx="title">Abbreviations</div>
            <div tagxxx="def-list">
               <div tagxxx="def-item">
                  <div tagxxx="term">ANDV</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Andes virus</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">BLAST</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Basic local alignment search tool</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">BSL</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Biosafety level</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">CDC</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Center of Disease Control</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">CHIKV</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Chikungunya virus</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">Cq</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Quantification cycle</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">DENV</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Dengue virus</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">DENV1-4</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Dengue virus serotype 1-4</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">DOBV</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Dobrava-Belgrade virus</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">FAM</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">6-carboxyfluorescein</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">FDA</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Food and Drug Administration</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">HTNV</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Hantaan virus</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">LASV</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Lassa virus</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">LoD</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Limit of detection</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">MGB</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Minor groove binder</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">NCBI</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">National Center for Biotechnology Information</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">NFQ</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Non-fluorescence quencher</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">nt</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Nucleotide</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">QCMD</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Quality Control for Molecular Diagnostics</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">RT-PCR</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Reverse transcription polymerase chain reaction</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">RVFV</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Rift Valley fever virus</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">SEOV</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Seoul virus</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">TBEV</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Tick-borne encephalitis virus</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">Tm</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Melting temperature</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">USUV</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Usutu virus</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">WNV</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">West Nile virus</p>
                  </div>
               </div>
               <div tagxxx="def-item">
                  <div tagxxx="term">ZIKV</div>
                  <div tagxxx="def">
                     <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">Zika virus</p>
                  </div>
               </div>
            </div>
         </div>
         <div tagxxx="fn-group">
            <div class="fn" id="">
               <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
                  <b>Competing interest</b>
               </p>
               <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The authors declare that they have no competing interest.</p>
            </div>
            <div class="fn" id="">
               <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">
                  <b>Authors’ contributions</b>
               </p>
               <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">EA carried out the sequence analysis, participated in the design of the study and
                  drafted the manuscript. GL carried out some of the laboratory work. KF participated
                  in the design of the study and had valuable comments to the manuscript. NL conceived
                  the study, participated in its design, drafted the manuscript, and performed most
                  of the laboratory work. All authors read and approved the final manuscript.
               </p>
            </div>
         </div>
         <div tagxxx="ack">
            <div tagxxx="title">Acknowledgement</div>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink">The authors thank the European Virus Archive (EVA) for providing DENV (strains 8356/10,
               4397/11, 3140/09, and 3274/09), LASV, and ZIKV.
            </p>
         </div>
         <div tag="ref-list">
            <ul>
               <div tagxxx="title">References</div>
               <li tag="ref">
                  <a name="CR1"></a>
                  <span class="label">1.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Gubler</span>
                           <span tagx="given-names">DJ</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Epidemic dengue/dengue hemorrhagic fever as a public health, social and economic problem
                        in the 21st century
                     </span>
                     <span tagx="source">Trends Microbiol</span>
                     <span tagx="year">2002</span>
                     <span tagx="volume">10</span>
                     <div tagxxx="issue">2</div>
                     <span tagx="fpage">100</span>
                     <span tagx="lpage">3</span>
                     <span class="pub-id'">[10.1016/S0966-842X(01)02288-0]</span>
                     <span class="pub-id'">[11827812]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR2"></a>
                  <span class="label">2.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Bhatt</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gething</span>
                           <span tagx="given-names">PW</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Brady</span>
                           <span tagx="given-names">OJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Messina</span>
                           <span tagx="given-names">JP</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Farlow</span>
                           <span tagx="given-names">AW</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Moyes</span>
                           <span tagx="given-names">CL</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">The global distribution and burden of dengue</span>
                     <span tagx="source">Nature</span>
                     <span tagx="year">2013</span>
                     <span tagx="volume">496</span>
                     <div tagxxx="issue">7446</div>
                     <span tagx="fpage">504</span>
                     <span tagx="lpage">7</span>
                     <span class="pub-id'">[10.1038/nature12060]</span>
                     <span class="pub-id'">[23563266]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR3"></a>
                  <span class="label">3.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Guzman</span>
                           <span tagx="given-names">MG</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Halstead</span>
                           <span tagx="given-names">SB</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Artsob</span>
                           <span tagx="given-names">H</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Buchy</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Farrar</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gubler</span>
                           <span tagx="given-names">DJ</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Dengue: a continuing global threat</span>
                     <span tagx="source">Nat Rev Microbiol</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">8</span>
                     <div tagxxx="issue">12 Suppl</div>
                     <span tagx="fpage">S7</span>
                     <span tagx="lpage">16</span>
                     <span class="pub-id'">[10.1038/nrmicro2460]</span>
                     <span class="pub-id'">[21079655]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR4"></a>
                  <span class="label">4.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Wilder-Smith</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Schwartz</span>
                           <span tagx="given-names">E</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Dengue in travelers</span>
                     <span tagx="source">N Engl J Med</span>
                     <span tagx="year">2005</span>
                     <span tagx="volume">353</span>
                     <div tagxxx="issue">9</div>
                     <span tagx="fpage">924</span>
                     <span tagx="lpage">32</span>
                     <span class="pub-id'">[10.1056/NEJMra041927]</span>
                     <span class="pub-id'">[16135837]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR5"></a>
                  <span class="label">5.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Schlagenhauf</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Weld</span>
                           <span tagx="given-names">L</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Goorhuis</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gautret</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Weber</span>
                           <span tagx="given-names">R</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">von Sonnenburg</span>
                           <span tagx="given-names">F</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Travel-associated infection presenting in Europe (2008-12): an analysis of EuroTravNet
                        longitudinal, surveillance data, and evaluation of the effect of the pre-travel consultation
                     </span>
                     <span tagx="source">Lancet Infect Dis</span>
                     <span tagx="year">2014</span>
                     <span class="pub-id'">[25477022]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR6"></a>
                  <span class="label">6.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Low</span>
                           <span tagx="given-names">JG</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Ong</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Tan</span>
                           <span tagx="given-names">LK</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Chaterji</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Chow</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Lim</span>
                           <span tagx="given-names">WY</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">The early clinical features of dengue in adults: challenges for early clinical diagnosis</span>
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">5</span>
                     <div tagxxx="issue">5</div>
                     <span class="pub-id'">[10.1371/journal.pntd.0001191]</span>
                     <span class="pub-id'">[21655307]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR7"></a>
                  <span class="label">7.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Endy</span>
                           <span tagx="given-names">TP</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Anderson</span>
                           <span tagx="given-names">KB</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Nisalak</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Yoon</span>
                           <span tagx="given-names">IK</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Green</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Rothman</span>
                           <span tagx="given-names">AL</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Determinants of inapparent and symptomatic dengue infection in a prospective study
                        of primary school children in Kamphaeng Phet, Thailand
                     </span>
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">5</span>
                     <div tagxxx="issue">3</div>
                     <span class="pub-id'">[10.1371/journal.pntd.0000975]</span>
                     <span class="pub-id'">[21390158]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR8"></a>
                  <span class="label">8.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Huy</span>
                           <span tagx="given-names">NT</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Van Giang</span>
                           <span tagx="given-names">T</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Thuy</span>
                           <span tagx="given-names">DH</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Kikuchi</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Hien</span>
                           <span tagx="given-names">TT</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Zamora</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Factors associated with dengue shock syndrome: a systematic review and meta-analysis</span>
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="year">2013</span>
                     <span tagx="volume">7</span>
                     <div tagxxx="issue">9</div>
                     <span class="pub-id'">[10.1371/journal.pntd.0002412]</span>
                     <span class="pub-id'">[24086778]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR9"></a>
                  <span class="label">9.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Thomas</span>
                           <span tagx="given-names">L</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Moravie</span>
                           <span tagx="given-names">V</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Besnier</span>
                           <span tagx="given-names">F</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Valentino</span>
                           <span tagx="given-names">R</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Kaidomar</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Coquet</span>
                           <span tagx="given-names">LV</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Clinical presentation of dengue among patients admitted to the adult emergency department
                        of a tertiary care hospital in Martinique: implications for triage, management, and
                        reporting
                     </span>
                     <span tagx="source">Ann Emerg Med</span>
                     <span tagx="year">2012</span>
                     <span tagx="volume">59</span>
                     <div tagxxx="issue">1</div>
                     <span tagx="fpage">42</span>
                     <span tagx="lpage">50</span>
                     <span class="pub-id'">[10.1016/j.annemergmed.2011.08.010]</span>
                     <span class="pub-id'">[21903297]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR10"></a>
                  <span class="label">10.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Santiago</span>
                           <span tagx="given-names">GA</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Vergne</span>
                           <span tagx="given-names">E</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Quiles</span>
                           <span tagx="given-names">Y</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Cosme</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Vazquez</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Medina</span>
                           <span tagx="given-names">JF</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Analytical and clinical performance of the CDC real time RT-PCR assay for detection
                        and typing of dengue virus
                     </span>
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="year">2013</span>
                     <span tagx="volume">7</span>
                     <div tagxxx="issue">7</div>
                     <span class="pub-id'">[10.1371/journal.pntd.0002311]</span>
                     <span class="pub-id'">[23875046]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR11"></a>
                  <span class="label">11.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Waggoner</span>
                           <span tagx="given-names">JJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Abeynayake</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Sahoo</span>
                           <span tagx="given-names">MK</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gresh</span>
                           <span tagx="given-names">L</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Tellez</span>
                           <span tagx="given-names">Y</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gonzalez</span>
                           <span tagx="given-names">K</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Development of an internally controlled real-time reverse transcriptase PCR assay
                        for pan-dengue virus detection and comparison of four molecular dengue virus detection
                        assays
                     </span>
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="year">2013</span>
                     <span tagx="volume">51</span>
                     <div tagxxx="issue">7</div>
                     <span tagx="fpage">2172</span>
                     <span tagx="lpage">81</span>
                     <span class="pub-id'">[10.1128/JCM.00548-13]</span>
                     <span class="pub-id'">[23637298]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR12"></a>
                  <span class="label">12.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Waggoner</span>
                           <span tagx="given-names">JJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Abeynayake</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Sahoo</span>
                           <span tagx="given-names">MK</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gresh</span>
                           <span tagx="given-names">L</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Tellez</span>
                           <span tagx="given-names">Y</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gonzalez</span>
                           <span tagx="given-names">K</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Single-reaction, multiplex, real-time rt-PCR for the detection, quantitation, and
                        serotyping of dengue viruses
                     </span>
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="year">2013</span>
                     <span tagx="volume">7</span>
                     <div tagxxx="issue">4</div>
                     <span class="pub-id'">[10.1371/journal.pntd.0002116]</span>
                     <span class="pub-id'">[23638191]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR13"></a>
                  <span class="label">13.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Cecilia</span>
                           <span tagx="given-names">D</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Kakade</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Alagarasu</span>
                           <span tagx="given-names">K</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Patil</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Salunke</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Parashar</span>
                           <span tagx="given-names">D</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Development of a multiplex real-time RT-PCR assay for simultaneous detection of dengue
                        and chikungunya viruses
                     </span>
                     <span tagx="source">Arch Virol</span>
                     <span tagx="year">2015</span>
                     <span tagx="volume">160</span>
                     <div tagxxx="issue">1</div>
                     <span tagx="fpage">323</span>
                     <span tagx="lpage">7</span>
                     <span class="pub-id'">[10.1007/s00705-014-2217-x]</span>
                     <span class="pub-id'">[25233940]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR14"></a>
                  <span class="label">14.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Najioullah</span>
                           <span tagx="given-names">F</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Viron</span>
                           <span tagx="given-names">F</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Cesaire</span>
                           <span tagx="given-names">R</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Evaluation of four commercial real-time RT-PCR kits for the detection of dengue viruses
                        in clinical samples
                     </span>
                     <span tagx="source">Virol J</span>
                     <span tagx="year">2014</span>
                     <span tagx="volume">11</span>
                     <span tagx="fpage">164</span>
                     <span class="pub-id'">[10.1186/1743-422X-11-164]</span>
                     <span class="pub-id'">[25219286]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR15"></a>
                  <span class="label">15.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Alm</span>
                           <span tagx="given-names">E</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Lesko</span>
                           <span tagx="given-names">B</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Lindegren</span>
                           <span tagx="given-names">G</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Ahlm</span>
                           <span tagx="given-names">C</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Soderholm</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Falk</span>
                           <span tagx="given-names">KI</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Universal single-probe rt-PCR assay for diagnosis of dengue virus infections</span>
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="year">2014</span>
                     <span tagx="volume">8</span>
                     <div tagxxx="issue">12</div>
                     <span class="pub-id'">[10.1371/journal.pntd.0003416]</span>
                     <span class="pub-id'">[25522325]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR16"></a>
                  <span class="label">16.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="collab'">[WHO]</span>
                     </span>
                     <span tagx="source">Dengue: Guidlines for Diagnosis, Treatment, Prevention adn Control., vol New edn</span>
                     <span tagx="year">2009</span>
                     <span class="publisher-loc'">[Geneva]</span>
                     <span class="publisher-name'">[WHO]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR17"></a>
                  <span class="label">17.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Holmes</span>
                           <span tagx="given-names">EC</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Burch</span>
                           <span tagx="given-names">SS</span>,
                        </span>
                     </span>
                     <span tagx="article-title">The causes and consequences of genetic variation in dengue virus</span>
                     <span tagx="source">Trends Microbiol</span>
                     <span tagx="year">2000</span>
                     <span tagx="volume">8</span>
                     <div tagxxx="issue">2</div>
                     <span tagx="fpage">74</span>
                     <span tagx="lpage">7</span>
                     <span class="pub-id'">[10.1016/S0966-842X(99)01669-8]</span>
                     <span class="pub-id'">[10664600]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR18"></a>
                  <span class="label">18.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Vene</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Mangiafico</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Niklasson</span>
                           <span tagx="given-names">B</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Indirect immunofluorescence for serological diagnosis of dengue virus infections in
                        Swedish patients
                     </span>
                     <span tagx="source">Clin Diagn Virol</span>
                     <span tagx="year">1995</span>
                     <span tagx="volume">4</span>
                     <div tagxxx="issue">1</div>
                     <span tagx="fpage">43</span>
                     <span tagx="lpage">50</span>
                     <span class="pub-id'">[10.1016/0928-0197(94)00060-8]</span>
                     <span class="pub-id'">[15566826]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR19"></a>
                  <span class="label">19.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Cicchetti</span>
                           <span tagx="given-names">DV</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Feinstein</span>
                           <span tagx="given-names">AR</span>,
                        </span>
                     </span>
                     <span tagx="article-title">High agreement but low kappa: II. Resolving the paradoxes</span>
                     <span tagx="source">J Clin Epidemiol</span>
                     <span tagx="year">1990</span>
                     <span tagx="volume">43</span>
                     <div tagxxx="issue">6</div>
                     <span tagx="fpage">551</span>
                     <span tagx="lpage">8</span>
                     <span class="pub-id'">[10.1016/0895-4356(90)90159-M]</span>
                     <span class="pub-id'">[2189948]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR20"></a>
                  <span class="label">20.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Waggoner</span>
                           <span tagx="given-names">JJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Abeynayake</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Sahoo</span>
                           <span tagx="given-names">MK</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gresh</span>
                           <span tagx="given-names">L</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Tellez</span>
                           <span tagx="given-names">Y</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gonzalez</span>
                           <span tagx="given-names">K</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Comparison of the FDA-approved CDC DENV-1-4 real-time reverse transcription-PCR with
                        a laboratory-developed assay for dengue virus detection and serotyping
                     </span>
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="year">2013</span>
                     <span tagx="volume">51</span>
                     <div tagxxx="issue">10</div>
                     <span tagx="fpage">3418</span>
                     <span tagx="lpage">20</span>
                     <span class="pub-id'">[10.1128/JCM.01359-13]</span>
                     <span class="pub-id'">[23903549]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR21"></a>
                  <span class="label">21.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Cnops</span>
                           <span tagx="given-names">L</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Domingo</span>
                           <span tagx="given-names">C</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Van den Bossche</span>
                           <span tagx="given-names">D</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Vekens</span>
                           <span tagx="given-names">E</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Brigou</span>
                           <span tagx="given-names">E</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Van Esbroeck</span>
                           <span tagx="given-names">M</span>,
                        </span>
                     </span>
                     <span tagx="article-title">First dengue co-infection in a Belgian traveler returning from Thailand, July 2013</span>
                     <span tagx="source">J Clin Virol</span>
                     <span tagx="year">2014</span>
                     <span tagx="volume">61</span>
                     <div tagxxx="issue">4</div>
                     <span tagx="fpage">597</span>
                     <span tagx="lpage">9</span>
                     <span class="pub-id'">[10.1016/j.jcv.2014.10.008]</span>
                     <span class="pub-id'">[25453334]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR22"></a>
                  <span class="label">22.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dar</span>
                           <span tagx="given-names">L</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gupta</span>
                           <span tagx="given-names">E</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Narang</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Broor</span>
                           <span tagx="given-names">S</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Cocirculation of dengue serotypes, Delhi, India, 2003</span>
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="year">2006</span>
                     <span tagx="volume">12</span>
                     <div tagxxx="issue">2</div>
                     <span tagx="fpage">352</span>
                     <span tagx="lpage">3</span>
                     <span class="pub-id'">[10.3201/eid1202.050767]</span>
                     <span class="pub-id'">[17080580]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="CR23"></a>
                  <span class="label">23.</span>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Vinodkumar</span>
                           <span tagx="given-names">CS</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Kalapannavar</span>
                           <span tagx="given-names">NK</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Basavarajappa</span>
                           <span tagx="given-names">KG</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Sanjay</span>
                           <span tagx="given-names">D</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gowli</span>
                           <span tagx="given-names">C</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Nadig</span>
                           <span tagx="given-names">NG</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Episode of coexisting infections with multiple dengue virus serotypes in central Karnataka,
                        India
                     </span>
                     <span tagx="source">J Infect Public Health</span>
                     <span tagx="year">2013</span>
                     <span tagx="volume">6</span>
                     <div tagxxx="issue">4</div>
                     <span tagx="fpage">302</span>
                     <span tagx="lpage">6</span>
                     <span class="pub-id'">[10.1016/j.jiph.2013.01.004]</span>
                     <span class="pub-id'">[23806706]</span>
                  </span>
               </li>
            </ul>
         </div>
      </div>
   </div>
</html>