<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <div xmlns="" tagx="article">
      <head>
         <style type="text/css"> table, tr, td {border : thin;}</style>
         <span class="citation_publisher">Oxford University Press</span>
         <h2 class="citation-title">Rapid Evolutionary Dynamics of Structural Disorder as a Potential Driving Force for
            Biological Divergence in Flaviviruses
         </h2>
         <span class="doi">10.1093/gbe/evt026</span>
         <div class="contrib-group">contrib: 
            	  CONT
            	  
            <span class="citation_author">Juan F. Ortiz</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Madolyn L. MacDonald</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Patrick Masterson</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Vladimir N. Uversky</span>
            <span class="citation_author_institution_ref">3</span>
            <span class="citation_author">Jessica Siltberg-Liberles</span>
            <span class="citation_author_institution_ref">1</span>
         </div>
         <span class="email">[jliberle@uwyo.edu]</span>
         <span class="conflict">[]</span>
         <span class="contribs">[]</span>
         <span class="citation_pubdate">[2013-2-15]</span>
         <span class="citation_copyright">[Copyright 2013: ]</span>
         <span class="citation_licence">[Licence This is an Open Access article distributed under the terms of the Creative
            Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/),
            which permits unrestricted non-commercial use, distribution, and reproduction in any
            medium, provided the original work is properly cited.]
         </span>
         <span class="citation_funding">[]</span>
         <span class="citation_pagecount">[]</span>
         <span class="citation_data">[]</span>
      </head>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Protein structure is commonly regarded to be conserved and to dictate function. Most
            proteins rely on conformational flexibility to some degree. Are regions that convey
            conformational flexibility conserved over evolutionary time? Can changes in conformational
            flexibility alter protein function? Here, the evolutionary dynamics of structurally
            ordered and disordered (flexible) regions are investigated genome-wide in flaviviruses,
            revealing that the amount and location of structural disorder fluctuates highly among
            related proteins. Some regions are prone to shift between structured and flexible
            states. Increased evolutionary dynamics of structural disorder is observed for some
            lineages but not in others. Lineage-specific transitions of this kind could alter
            the conformational ensemble accessible to the same protein in different species, causing
            a functional change, even if the predominant function remains conserved. Thus, rapid
            evolutionary dynamics of structural disorder is a potential driving force for phenotypic
            divergence among flaviviruses.
         </p>
      </div>
      <div id="">
         <h2>Introduction</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Two central tenets of molecular biology are that protein structure is more conserved
            than protein sequence and protein structure is crucial for protein function. However,
            it is important to note that proteins are dynamic and found as conformational ensembles
            to various extents (
            <span ref-type="bibr" rid="evt026-B17">
               <sup>
                  <a href="#evt026-B17">Gunasekaran et al. 2004</a>
               </sup>
            </span>). A large portion of any proteome is occupied by biologically active proteins without
            unique 3D structures (
            <span ref-type="bibr" rid="evt026-B38">
               <sup>
                  <a href="#evt026-B38">Romero et al. 1998</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B12">
               <sup>
                  <a href="#evt026-B12">Dunker et al. 2000</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B49">
               <sup>
                  <a href="#evt026-B49">Uversky et al. 2000</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B54">
               <sup>
                  <a href="#evt026-B54">Ward et al. 2004</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B59">
               <sup>
                  <a href="#evt026-B59">Xue et al. 2012</a>
               </sup>
            </span>). These proteins possess numerous intriguing properties, are intimately involved
            in various cellular processes (
            <span ref-type="bibr" rid="evt026-B55">
               <sup>
                  <a href="#evt026-B55">Wright and Dyson 1999</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B9">
               <sup>
                  <a href="#evt026-B9">Dunker et al. 2001</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B10">
               <sup>
                  <a href="#evt026-B10">Dunker et al. 2002</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B21">
               <sup>
                  <a href="#evt026-B21">Iakoucheva et al. 2002</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B45">
               <sup>
                  <a href="#evt026-B45">Tompa 2002</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B48">
               <sup>
                  <a href="#evt026-B48">Uversky 2002</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B11">
               <sup>
                  <a href="#evt026-B11">Dunker et al. 2005</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B13">
               <sup>
                  <a href="#evt026-B13">Dyson and Wright 2005</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B50">
               <sup>
                  <a href="#evt026-B50">Uversky et al. 2005</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B53">
               <sup>
                  <a href="#evt026-B53">Vucetic et al. 2007</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B57">
               <sup>
                  <a href="#evt026-B57">Xie et al. 2007</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B24">
               <sup>
                  <a href="#evt026-B24">Kim et al. 2008</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B37">
               <sup>
                  <a href="#evt026-B37">Oldfield et al. 2008</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B29">
               <sup>
                  <a href="#evt026-B29">Liu et al. 2009</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B56">
               <sup>
                  <a href="#evt026-B56">Wright and Dyson 2009</a>
               </sup>
            </span>), and are commonly found to be related to the pathogenesis of various diseases (
            <span ref-type="bibr" rid="evt026-B51">
               <sup>
                  <a href="#evt026-B51">Uversky et al. 2008</a>
               </sup>
            </span>). Frequently involved in complex protein–protein, protein–nucleic acid, and protein–small
            molecule interactions, some of these interactions can induce a disorder-to-order transition
            in the entire protein or in its part (
            <span ref-type="bibr" rid="evt026-B55">
               <sup>
                  <a href="#evt026-B55">Wright and Dyson 1999</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B49">
               <sup>
                  <a href="#evt026-B49">Uversky et al. 2000</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B9">
               <sup>
                  <a href="#evt026-B9">Dunker et al. 2001</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B45">
               <sup>
                  <a href="#evt026-B45">Tompa 2002</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B48">
               <sup>
                  <a href="#evt026-B48">Uversky 2002</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B13">
               <sup>
                  <a href="#evt026-B13">Dyson and Wright 2005</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B36">
               <sup>
                  <a href="#evt026-B36">Oldfield et al. 2005</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B33">
               <sup>
                  <a href="#evt026-B33">Mohan et al. 2006</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B52">
               <sup>
                  <a href="#evt026-B52">Vacic et al. 2007</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B8">
               <sup>
                  <a href="#evt026-B8">Dosztányi et al. 2009</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B32">
               <sup>
                  <a href="#evt026-B32">Mészáros et al. 2009</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B56">
               <sup>
                  <a href="#evt026-B56">Wright and Dyson 2009</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B47">
               <sup>
                  <a href="#evt026-B47">Uversky 2011</a>
               </sup>
            </span>). Furthermore, confomationally flexible proteins opens a unique capability for one
            protein to be involved in interaction with several unrelated binding partners and
            to gain different bound structures (
            <span ref-type="bibr" rid="evt026-B37">
               <sup>
                  <a href="#evt026-B37">Oldfield et al. 2008</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B20">
               <sup>
                  <a href="#evt026-B20">Hsu et al. 2012</a>
               </sup>
            </span>). This means that the same sequence can adopt multiple conformations. Thus, proteins
            that are found as conformational ensembles may have more than one functional conformation
            and multiple functions.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The equilibrium of a protein’s conformational ensemble is controlled by various external
            signals, such as pH, temperature, or binding partners (
            <span ref-type="bibr" rid="evt026-B6">
               <sup>
                  <a href="#evt026-B6">del Sol et al. 2009</a>
               </sup>
            </span>). From an evolutionary viewpoint, sequences that adopt a conformational ensemble
            can propagate biological divergence through multifaceted selective pressures. Sequence
            changes altering the equilibrium of the conformational ensemble, and causing a loss
            of a subset of promiscuous functions, may be better tolerated than sequence changes
            that cause a protein to lose its entire function due to misfolding. On evolutionary
            time scales, changes in the equilibrium of a conformational ensemble may result in
            highly different conformational ensembles among homologous proteins (
            <span ref-type="bibr" rid="evt026-B41">
               <sup>
                  <a href="#evt026-B41">Siltberg-Liberles et al. 2011</a>
               </sup>
            </span>).
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Proteins that have intrinsically disordered regions are present as conformational
            ensembles. Structurally disordered regions act as dynamic switches (
            <span ref-type="bibr" rid="evt026-B42">
               <sup>
                  <a href="#evt026-B42">Smock and Gierasch 2009</a>
               </sup>
            </span>). These proteins have (large or small) regions that rapidly sample multiple conformations
            due to a shallow rugged energy landscape (
            <span ref-type="bibr" rid="evt026-B46">
               <sup>
                  <a href="#evt026-B46">Tsai et al. 2001</a>
               </sup>
            </span>). As conformations become more ordered or stabilized in response to an external signal,
            the conformational ensemble endures a population shift (
            <span ref-type="bibr" rid="evt026-B31">
               <sup>
                  <a href="#evt026-B31">Ma et al. 1999</a>
               </sup>
            </span>). In accordance with the extended conformational selection processes, binding events
            ranging from lock and key to induced fit are plausible (
            <span ref-type="bibr" rid="evt026-B3">
               <sup>
                  <a href="#evt026-B3">Csermely et al. 2010</a>
               </sup>
            </span>). Intuitively, one can hypothesize that the conformational ensemble endures population
            shifts in response to sequence divergence. Consequently, this will ultimately impact
            processes of conformational selection in response to different stimuli in a lineage-specific
            manner, as a mutation-driven conformational selection (
            <span ref-type="bibr" rid="evt026-B44">
               <sup>
                  <a href="#evt026-B44">Tokuriki and Tawfik 2009</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B41">
               <sup>
                  <a href="#evt026-B41">Siltberg-Liberles et al. 2011</a>
               </sup>
            </span>).
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We hypothesize that subtle changes in conformational flexibility can drive biological
            divergence among proteins that seem structurally and functionally conserved, such
            as orthologous proteins in closely related species. To test this hypothesis, we have
            investigated how structural disorder, as an approximation for conformational flexibility,
            changes among Dengue viruses and other flaviviruses, e.g., yellow fever virus, Japanese
            encephalitis virus, and West Nile virus. According to the World Health Organization,
            the mosquito-borne Dengue virus alone infects 50 million people worldwide per year,
            resulting in 22,000 fatalities. While vaccines are present for some flaviviruses (e.g.,
            yellow fever virus, Japanese encephalitis virus, and tick-borne encephalitis virus),
            Dengue virus vaccines have proven a challenge due to the presence of different serotypes
            and complex antibody binding affinities (reviewed in 
            <span ref-type="bibr" rid="evt026-B19">
               <sup>
                  <a href="#evt026-B19">Heinz and Stiasny (2012)</a>
               </sup>
            </span>). Importantly, flaviviruses depend on conformational flexibility for their life cycle
            (
            <span ref-type="bibr" rid="evt026-B19">
               <sup>
                  <a href="#evt026-B19">Heinz and Stiasny 2012</a>
               </sup>
            </span>) and encode a small RNA genome, which is expressed as a polyprotein (approximately
            3,400 residues in Dengue virus) containing 11 separate protein chains. Therefore,
            the flaviviruses were subjected to a genome-wide investigation of the evolutionary
            dynamics of structural disorder with implications for phenotypic divergence.
         </p>
      </div>
      <div id="">
         <h2>Methods</h2>
         <div id="">
            <h3>Polyprotein Composition</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The 11 proteins encoded by the polyprotein in DENV-1 are capsid protein (C), membrane
               glycoprotein precursor (Mp), envelope protein (E), nonstructural protein 1 (NS1),
               nonstructural protein 2A (NS2A), nonstructural protein 2B (NS2B), nonstructural protein
               3 (NS3), nonstructural protein 4A (NS4A), 2K protein (2K), nonstructural protein 4B
               (NS4B), and nonstructural protein 5 (NS5). Protein 2K is only 23 residues long and
               therefore excluded from this study.
            </p>
         </div>
         <div id="">
            <h3>Protein Phylogenies</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">NCBI BLAST (
               <span ref-type="bibr" rid="evt026-B1">
                  <sup>
                     <a href="#evt026-B1">Altschul et al. 1990</a>
                  </sup>
               </span>) was performed for each protein chain in the Dengue 1 virus’ polyprotein independently
               against the genus flavivirus in the refseq and nr databases. Sequences (
               <div tagxxx="ext-link">supplementary table S1</div>, 
               <div tagxxx="ext-link">Supplementary Material</div> online) were aligned with MAFFT (
               <span ref-type="bibr" rid="evt026-B23">
                  <sup>
                     <a href="#evt026-B23">Katoh et al. 2002</a>
                  </sup>
               </span>). Model testing was performed using ProtTest (
               <span ref-type="bibr" rid="evt026-B4">
                  <sup>
                     <a href="#evt026-B4">Darriba et al. 2011</a>
                  </sup>
               </span>). The best models (
               <div tagxxx="ext-link">supplementary table S1</div>, 
               <div tagxxx="ext-link">Supplementary Material</div> online) from ProtTest were used to build the protein trees using PhyML (
               <span ref-type="bibr" rid="evt026-B16">
                  <sup>
                     <a href="#evt026-B16">Guindon et al. 2005</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="evt026-B15">
                  <sup>
                     <a href="#evt026-B15">2009</a>
                  </sup>
               </span>) with 1,000 bootstraps.
            </p>
         </div>
         <div id="">
            <h3>Evolutionary Amino Acid Substitution Rate per Site</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">For each site in the different alignments, the evolutionary rate of amino acid substitutions
               was calculated using MEGA5 (
               <span ref-type="bibr" rid="evt026-B43">
                  <sup>
                     <a href="#evt026-B43">Tamura et al. 2011</a>
                  </sup>
               </span>) based on the PhyML trees. Mean (relative) evolutionary rate are scaled such that
               the average evolutionary rate across all sites is 1. This means that sites showing
               a rate &lt;1 are evolving slower than average, and those with a rate &gt;1 are evolving
               faster than average. These relative rates were estimated under the Jones–Taylor–Thornton
               (
               <span ref-type="bibr" rid="evt026-B22">
                  <sup>
                     <a href="#evt026-B22">1992</a>
                  </sup>
               </span>) model (
               <span ref-type="bibr" rid="evt026-B22">
                  <sup>
                     <a href="#evt026-B22">Jones et al. 1992</a>
                  </sup>
               </span>) including a discrete five-category Gamma distribution.
            </p>
         </div>
         <div id="">
            <h3>Prediction of Structural Disorder</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">For every sequence in the multiple sequence alignments (MSAs) used for protein phylogenies,
               structural disorder was predicted using IUPred (
               <span ref-type="bibr" rid="evt026-B7">
                  <sup>
                     <a href="#evt026-B7">Dosztányi et al. 2005</a>
                  </sup>
               </span>) and PONDR-FIT (
               <span ref-type="bibr" rid="evt026-B58">
                  <sup>
                     <a href="#evt026-B58">Xue, Dunbrack, et al. 2010</a>
                  </sup>
               </span>). All predictions were run on unaligned, ungapped sequences.
            </p>
         </div>
         <div id="">
            <h3>Analysis</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To analyze the evolutionary dynamics of structural disorder, this feature must be
               analyzed in an evolutionary context. For every protein phylogeny, the value of structural
               disorder predicted for every residue in all sequences in a phylogeny was projected
               onto the MSAs, in order to line up comparable sites. These were visualized in heatmaps
               using iTOL (
               <span ref-type="bibr" rid="evt026-B27">
                  <sup>
                     <a href="#evt026-B27">Letunic and Bork 2007</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="evt026-B28">
                  <sup>
                     <a href="#evt026-B28">2011</a>
                  </sup>
               </span>). For estimating evolutionary dynamics, the structural disorder predictions were
               reduced to two states, ordered and disordered. All sites with IUPred prediction values
               &lt;0.4 were assigned order and all sites ≥0.4 were assigned disorder. Similarly, for
               PONDR-FIT, 0.5 was used as the cutoff for order and disorder. For every protein phylogeny
               and site in the corresponding MSA, the change of state (order vs. disorder) was evaluated
               using parsimony as implemented in GLOOME (
               <span ref-type="bibr" rid="evt026-B2">
                  <sup>
                     <a href="#evt026-B2">Cohen et al. 2010</a>
                  </sup>
               </span>) and normalized by the number of nodes in each phylogeny. If a gap was present for
               a species at a specific position, that species was excluded for the analysis of that
               site. Thus, all sites with data were analyzed. Furthermore, to visualize the amount
               of disorder to order transition (DOT) in a phylogenic context, all branches with DOTs
               in at least 5% of the sites per row length of the MSA were identified as branches
               showing rapid evolutionary dynamics of DOT.
            </p>
         </div>
      </div>
      <div id="">
         <h2>Results</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To investigate the evolutionary dynamics of conformational flexibility in proteins
            encoded by Dengue virus and other flaviviruses, protein phylogenies were constructed
            for all homologs of the 11 proteins in Dengue virus serotype 1 (DENV-1). Protein structure
            disorder was predicted as a proxy for conformational flexibility using IUPred (
            <span ref-type="bibr" rid="evt026-B7">
               <sup>
                  <a href="#evt026-B7">Dosztányi et al. 2005</a>
               </sup>
            </span>) and PONDR-FIT (
            <span ref-type="bibr" rid="evt026-B58">
               <sup>
                  <a href="#evt026-B58">Xue, Dunbrack, et al. 2010</a>
               </sup>
            </span>). While the IUPred predictions are continuous, the cutoff of 0.4 was found to match
            the disordered regions in the Disprot database of disordered proteins (
            <span ref-type="bibr" rid="evt026-B14">
               <sup>
                  <a href="#evt026-B14">Fuxreiter et al. 2007</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B39">
               <sup>
                  <a href="#evt026-B39">Sickmeier et al. 2007</a>
               </sup>
            </span>). Here, a cutoff of 0.4 was used to establish disorder or order. For PONDR-FIT, the
            boundary for order and disorder is 0.5 (
            <span ref-type="bibr" rid="evt026-B58">
               <sup>
                  <a href="#evt026-B58">Xue, Dunbrack, et al. 2010</a>
               </sup>
            </span>), and thus, the 0.5 cutoff was used to denote order and disorder. The two discrete
            states (disorder and order) for the two different prediction methods were analyzed
            in a phylogenetic context using parsimony.
         </p>
         <div id="">
            <h3>Phylogenies</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The phylogeny for the envelope protein (
               <span ref-type="fig" rid="evt026-F1">fig. 1</span>) was rooted with Tamana bat virus (TABV) as the outgroup, in accordance with it being
               a remote flavivirus species (
               <span ref-type="bibr" rid="evt026-B5">
                  <sup>
                     <a href="#evt026-B5">de Lamballerie et al. 2002</a>
                  </sup>
               </span>). After TABV, the Kamiti River virus (KRV) clade, including KRV, Cell Fusing Agent
               virus (CFAV), Culex flavivirus (CxFV), Aedes flavivirus (AeFV), branches off. Quang
               Binh virus (QBV) is frequently found in this clade too. At the next node junction,
               the upper branch further splits into two main clades. The upper main clade of the
               upper branch has the Rio Bravo virus (RBV) clade (Montana myotis leukoencephalitis
               virus [MMLV], Modoc virus [MODV], Apoi virus [APOIV], and RBV) and the Tick-borne
               encephalitis virus (TBEV) clade (Louping ill virus [LIV], Omsk hemorrhagic fever virus
               [OHFV], Langat virus [LGTV], Alkhurma hemorrhagic fever virus [AHFV], Karshi virus
               [KARV], Powasson virus [POWV], and TBEV). On the lower main clade of the upper branch,
               the yellow fever virus (YFV) clade (Sepik virus [SEPV], Wesselbron virus [WESSV],
               and YFV) and the Entebbe bat virus (ENTV) clade (Yokose virus [YOKV] and ENTV) are
               found. On the lower branch, the upper Dengue virus (DENV) clade consists of the four
               different serotypes of DENV (DENV-1–4).
               
               <div tagxxx="fig">
                  <span class="label">Fig. 1.—</span>
                  <caption>
                     <p>The envelope protein phylogeny. The phylogeny of the envelope protein represents the
                        evolutionary relationships between the different flaviviruses. The different clades
                        are highlighted and the clade identifier is shown in italic. In most phylogenies for
                        the separate protein chains, the clades are reconstructed, with the exception of the
                        ZIKV clade and to a lesser extent the YFV clade. This phylogeny is consistent with
                        a recent flavivirus phylogeny (
                        <span ref-type="bibr" rid="evt026-B30">
                           <sup>
                              <a href="#evt026-B30">Lobo et al. 2009</a>
                           </sup>
                        </span>), where the following hosts were indicated: 
                        <sup>1</sup>no known arthropod vector virus, 
                        <sup>2</sup>insect only virus, 
                        <sup>3</sup>tick-borne virus, and 
                        <sup>4</sup>mosquito-borne virus.
                     </p>
                  </caption>
                  <div tagxxx="graphic"></div>
               </div>
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The lower clade of the lower branch has a monophyletic West Nile virus (WNV) clade
               (Bagaza virus [BAGV], Ilheus virus [ILHV], St. Louis Encephalitis virus [SLEV], West
               nile virus p [WNVp], Murray Valley Encephalitis virus [MVEV], Usutu virus [USUV],
               Japanese Encephalitis virus [JEV], and WNV). Kedougou virus (KEDV), Zika virus (ZIKV),
               Kokobera virus (KOKV) and Aroa virus (AROAV) are rarely found in consistent clades
               but tend to end up close to the DENV and the WNV clades.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Full-length homologs for all DENV-1 proteins are only found in the ZIKV and WNV clades.
               Only E, NS3, and NS5 are found across all flavivirus clades in this study. The final
               phylogenies for all protein families follow similar clade topologies as in the envelope
               protein. There are small variations within the clades, mostly due to the different
               species composition for the different proteins (
               <div tagxxx="ext-link">supplementary fig. S1</div>, 
               <div tagxxx="ext-link">Supplementary Material</div> online).
            </p>
         </div>
         <div id="">
            <h3>Disorder Prediction</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Structural disorder was predicted for all proteins using IUPred (
               <span ref-type="bibr" rid="evt026-B7">
                  <sup>
                     <a href="#evt026-B7">Dosztányi et al. 2005</a>
                  </sup>
               </span>) and PONDR-FIT (
               <span ref-type="bibr" rid="evt026-B58">
                  <sup>
                     <a href="#evt026-B58">Xue, Dunbrack, et al. 2010</a>
                  </sup>
               </span>). Comparing the disorder predictions from IUPred (
               <div tagxxx="ext-link">supplementary fig. S2</div>, 
               <div tagxxx="ext-link">Supplementary Material</div> online) and PONDR-FIT (
               <div tagxxx="ext-link">supplementary fig. S3</div>, 
               <div tagxxx="ext-link">Supplementary Material</div> online) reveals similar results, but PONDR-FIT overpredicts disorder in the N- and
               C-termini (
               <div tagxxx="ext-link">supplementary fig. S4</div>, 
               <div tagxxx="ext-link">Supplementary Material</div> online). For the clarity of this study, we will focus on the IUPred results.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The percentage of disordered sites per protein and the distribution of proteins in
               the different viruses show high variation (
               <span ref-type="fig" rid="evt026-F2">fig. 2</span>). NS2A and NS4A show very little disorder and therefore less emphasis will be put
               on these two proteins. Most viruses have 20–30% of the residues in C in the disordered
               state, but in ZIKV only 1% of the sites are predicted to be disordered. In Mp, the
               percentages of disordered sites are in the range of 0–31%. The envelope protein, E,
               is 14% disordered in the outgroup TABV, but the first clades to branch off, KRV and
               RBV, are much less disordered than the younger clades (DENV, ZIKV, and WNV). NS1,
               NS3, and NS5 have the highest amount of disordered sites, with NS3 being more disordered
               than NS5, and NS1 is the least disordered. Most NS2B and NS4B proteins have little
               disorder. In addition, even if the percentage of disorder is the same, it cannot be
               assumed that the sites displaying disorder are conserved. Subsequently, the disorder
               predictions were mapped onto the MSAs used for the phylogeny reconstructions and visualized
               as heatmaps following the same order as the sequences in the specific phylogeny for
               each protein (
               <span ref-type="fig" rid="evt026-F3">fig. 3</span>A). While the heatmap shows the variation in disorder prediction for all sites in
               the alignment, it does not quantify how disorder–order states actually change over
               the phylogeny. Thus, the disorder predictions for each site in the alignment were
               discretized into two states (ordered or disordered). The discrete states were analyzed
               site by site across the phylogeny, allowing us to capture two measures of evolutionary
               dynamics: 1) DOT per site per node (
               <span ref-type="fig" rid="evt026-F3">fig. 3</span>B) and 2) DOTs per branch (
               <span ref-type="fig" rid="evt026-F4">fig. 4</span> and 
               <div tagxxx="ext-link">supplementary fig. S2</div>, 
               <div tagxxx="ext-link">Supplementary Material</div> online) for each protein phylogeny and heatmap pair.
               
               <div tagxxx="fig">
                  <span class="label">Fig. 2.—</span>
                  <caption>
                     <p>Disorder content per protein per species. The percentage of sites per protein chain
                        per virus that are predicted to be structurally disordered. The different viruses
                        occur in the graph in the order given below the graph from left to right.
                     </p>
                  </caption>
                  <div tagxxx="graphic"></div>
               </div>
               <div tagxxx="fig">
                  <span class="label">Fig. 3.—</span>
                  <caption>
                     <p>Changes in disorder across sites. (
                        <i>A</i>) The combined output for the capsid is shown. The heatmap represent order (blue to
                        white) and disorder (white to red) prediction by IUPred per site in the multiple sequence
                        alignment. The phylogeny is shown on the right. The parsimony analysis of DOT across
                        the phylogentic tree is shown on top of the heat map as changes per site per node.
                        (
                        <i>B</i>) The parsimony analysis of DOT over the phylogentic tree as changes per site per
                        node. (Combined outputs for all proteins and details about each phylogenetic reconstruction,
                        see 
                        <div tagxxx="ext-link">Supplementary table S1</div> and 
                        <div tagxxx="ext-link">Supplementary figs</div>. 
                        <div tagxxx="ext-link">S1</div> and 
                        <div tagxxx="ext-link">S2</div>, 
                        <div tagxxx="ext-link">Supplementary Material</div> online.)
                     </p>
                  </caption>
                  <div tagxxx="graphic"></div>
               </div>
               <div tagxxx="fig">
                  <span class="label">Fig. 4.—</span>
                  <caption>
                     <p>Changes in disorder across branches. The result of the parsimony analysis of disorder
                        to order transitions or vice versa over the phylogentic tree as changes per branch.
                        Branches with more than 5% of sites in DOT are shown with wider lines: (
                        <i>A</i>) capsid, (
                        <i>B</i>) membrane glycoprotein precursor, (
                        <i>C</i>) envelope, (
                        <i>D</i>) NS1, (
                        <i>E</i>) NS2B, (
                        <i>F</i>) NS3, (
                        <i>G</i>) NS5, and (
                        <i>H</i>) NS4B. (For complete output, including NS2A, 2K, and NS4A, which have no branches
                        with more than 5% of sites in DOT, see 
                        <div tagxxx="ext-link">Supplementary fig. S2</div>, 
                        <div tagxxx="ext-link">Supplementary Material</div> online.)
                     </p>
                  </caption>
                  <div tagxxx="graphic"></div>
                  <div tagxxx="graphic"></div>
               </div>
            </p>
         </div>
         <div id="">
            <h3>DOTs per Site per Node</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">All proteins showing disordered sites display rapid evolutionary dynamics; the DOTs
               are frequent (
               <span ref-type="fig" rid="evt026-F3">fig. 3</span>B). Very few sites are disordered across all taxa, while the ordered sites show much
               higher conservation. All proteins analyzed here have ordered, highly conserved regions,
               but most also have disordered regions that are showing fast evolutionary dynamics.
               Some disordered regions appear to expand from one clade to another. Consequently,
               the conformational ensembles may be changing quickly between closely related orthologs.
               This pattern also allows for the conserved, predominant functional conformations to
               form, but with differences in response to various stimuli.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To evaluate whether the sites that show rapid evolutionary dynamics of DOT also are
               showing high evolutionary rate of amino acid substitutions, we compared the DOT per
               site versus evolutionary rate per site (
               <span ref-type="fig" rid="evt026-F5">fig. 5</span>). Sites with rapid DOT do not necessarily have an elevated amino acid substitution
               rate, and there is no clear correlation between DOT and evolutionary rate of amino
               acid substitution.
               
               <div tagxxx="fig">
                  <span class="label">Fig. 5.—</span>
                  <caption>
                     <p>DOT versus evolutionary rate of amino acid substitutions. Plots showing the disorder
                        to order transitions versus the evolutionary rate of amino acid substitutions: (
                        <i>A</i>) capsid, (
                        <i>B</i>) membrane glycoprotein precursor, (
                        <i>C</i>) envelope, (
                        <i>D</i>) NS1, (
                        <i>E</i>) NS2A, (
                        <i>F</i>) NS2A, (
                        <i>G</i>) NS4A, (
                        <i>H</i>) NS4B, (
                        <i>I</i>) NS3, and (
                        <i>J</i>) NS5.
                     </p>
                  </caption>
                  <div tagxxx="graphic"></div>
               </div>
            </p>
         </div>
         <div id="">
            <h3>DOTs per Branch</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Observing the branches where at least 5% of the sites in the alignment are changing
               states (
               <span ref-type="fig" rid="evt026-F4">fig. 4</span>), it appears that DOTs are more frequent among the DENV, ZIKV, and WNV clades than
               among the KRV, APOI, and YFV clades. Protein E, NS3, and NS5, are especially enriched
               in branches with rapid evolutionary dynamics of DOTs.
            </p>
         </div>
      </div>
      <div id="">
         <h2>Discussion</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Structurally ordered regions show a higher degree of conservation than structurally
            disordered regions, and many regions prone to be structurally disordered can rapidly
            transition between order and disorder. While conserved structurally ordered regions
            imply structural conservation, conserved structurally disordered regions can hide
            changes in conformational properties such as secondary structure propensity (
            <span ref-type="bibr" rid="evt026-B40">
               <sup>
                  <a href="#evt026-B40">Siltberg-Liberles 2011</a>
               </sup>
            </span>). Comparing the percentage of average structural disorder per protein in orthologous
            proteins from different viruses reveals high variation levels among more distantly
            related clades. Conversely, within clades and between more closely related clades,
            there is less variation of the average structural disorder per protein. These results
            imply that structural disorder is changing among the sequences under investigation
            here. Indeed, the parsimony analysis of how structural disorder and order vary at
            the same site for different species across the alignment in the phylogenetic context
            confirms the fluctuation between disorder and order. The parsimony analysis identifies
            sites and regions that are rapidly changing between order and disorder as well as
            constantly ordered regions. The parsimony analysis across the phylogeny illustrating
            where the different changes occur provides a lineage-specific perspective. It reveals
            that, among these flaviviruses, the changes per lineage are not evenly distributed
            nor does it seem to correlate with branch lengths. Some lineages are undergoing far
            more DOT than others. High lineage-specific DOT is likely to be of biological importance
            as a route to lineage-specific specialization. Subtle or not, changes in the conformational
            flexibility can alter the conformational ensemble. Regions that have experienced high
            DOT are likely to be less important to the primary function of the protein, but they
            may be important in providing a diverse set of promiscuous functions (such as interactions
            and regulation). These promiscuous functions are likely to change as DOTs occur.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Here, we have compared a broad set of flavivirus proteomes, but the DENV, ZIKV, and
            WNV clades have the most similar proteomes. These viruses are phenotypically different,
            governed only by changes in orthologous sequences. These three clades have experienced
            frequent lineage-specific DOT and while the primary functions are likely to remain,
            the set of promiscuous functions and interactions might have changed among these proteins.
            Direct evidence to these points is hard to derive but a few circumstantial indications
            add credibility of these points. First, antibody dependent enhancement is the proposed
            mechanism for increased severity in secondary Dengue virus infections. Antibodies
            targeted for the envelope protein are supposed to neutralize the virion by preventing
            it from binding to Fc receptors. If the affinity of the antibody from a primary infection
            is too low (or is present in too low concentration) for a different serotype of Dengue
            virus, the virion is not neutralized and the secondary infection is enhanced (
            <span ref-type="bibr" rid="evt026-B18">
               <sup>
                  <a href="#evt026-B18">Guzman and Vazquez 2010</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="evt026-B19">
               <sup>
                  <a href="#evt026-B19">Heinz and Stiasny 2012</a>
               </sup>
            </span>). The DENV clade is enriched in lineage-specific DOT.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Second, we note that especially NS3 and NS5 have undergone high lineage-specific DOT.
            In a recent study, NS3 and NS5 from six different flaviviruses (DENV1, AHFV, WNV,
            JEV, TBEV, and Kunjin virus) were used to determine the human host–flavivirus protein–protein
            interaction network by comparing their interactions with 120 human cellular target
            proteins. Most interactions between NS3 and/or NS5 from the different flaviviruses
            are species-specific. Only two of the human target proteins interacted with four of
            the six flaviviruses, and 82 of the human target proteins interacted with only one
            of the six flaviviruses (
            <span ref-type="bibr" rid="evt026-B26">
               <sup>
                  <a href="#evt026-B26">Le Breton et al. 2011</a>
               </sup>
            </span>). This supports the hypothesis that phenotypic divergence can result as the conformational
            flexibility changes, here rewiring protein–protein interaction networks. Further support
            is provided by a recent study that found interactomes to be depleted in conserved
            interactions mediated by disordered proteins as compared with ordered proteins networks
            (
            <span ref-type="bibr" rid="evt026-B34">
               <sup>
                  <a href="#evt026-B34">Mosca et al. 2012</a>
               </sup>
            </span>).
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Third, experimental structural biology suggests that the main conformations of the
            envelope protein are fairly conserved across many viruses and also that there are
            differences in the surface accessible loops among DENV and TBEV (
            <span ref-type="bibr" rid="evt026-B61">
               <sup>
                  <a href="#evt026-B61">Zhang et al. 2004</a>
               </sup>
            </span>). Mapping the locations of the structurally divergent surface loops onto the envelope
            heatmap shows that these locations often correspond to regions of altered disorder
            or order (
            <div tagxxx="ext-link">supplementary fig. S5</div>, 
            <div tagxxx="ext-link">Supplementary Material</div> online).
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">These are important observations suggesting that while reporting merely the average
            structural disorder per protein or per proteome was sufficient as a first estimate
            of the prevalence of structural disorder, it is not enough to infer the evolutionary
            importance that disordered regions may have in a protein or in a proteome. Here, we
            have shown rapid evolutionary dynamics of DOT along several branches in this part
            of the flavivirus phylogeny. This is the first comparative genomic study of structural
            DOT and it shows that regions of conserved order are intermixed with regions displaying
            rapid evolutionary dynamics of DOT. The major trends identified here are 1) the amount
            and location of structural disorder fluctuates among orthologs and 2) lineage-specific
            (structural and, thus, functional) dynamic fluctuations are frequent and could be
            a major driving force for phenotypic divergence. These trends are from a group of
            related viruses. It has been proposed that viruses use structural disorder to function,
            because viruses must quickly adapt to their changing environments and for their pathogenicity
            (
            <span ref-type="bibr" rid="evt026-B60">
               <sup>
                  <a href="#evt026-B60">Xue, Williams, et al. 2010</a>
               </sup>
            </span>). Our results support and expand that hypothesis. The rapid change in disorder offers
            not only the possibility of antibody-dependent enhancement but also novel compositions
            of host target protein interactions. These trends may not apply to all proteins in
            a general sense, but nevertheless provide a first view of how structural disorder
            contributes to biodiversity on a genome-wide level.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Infections caused by Dengue virus are emerging. If we cannot prevent infections by
            vaccines, antivirals offer an alternative strategy. The results presented here are
            valuable for identifying virus-specific functionally important regions that can be
            targeted for both vaccine and antiviral designs.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Extended conformational selection is not only at play through conformational selection
            but also includes induced fit as a mechanism. It is important to note that both mechanisms
            depend on conformational flexibility. This work is focused on the evolutionary dynamics
            of structural disorder and order in homologous sites and shows that in addition to
            the previously identified mechanism of adding long disordered segments as a mean to
            alter disorder content during protein evolution (
            <span ref-type="bibr" rid="evt026-B35">
               <sup>
                  <a href="#evt026-B35">Nido et al. 2012</a>
               </sup>
            </span>), both direct and indirect amino acid substitutions offer another mechanism for altering
            disorder content. Thus, the results presented here support the view that mutation-driven
            extended conformational selection is a potential mechanism for biological divergence.
         </p>
      </div>
      <div id="">
         <h2>Supplementary Material</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
            <div tagxxx="ext-link">Supplementary figures S1–S5</div> and 
            <div tagxxx="ext-link">table S1</div> are available at 
            <i>Genome Biology and Evolution</i> online (
            <div tagxxx="ext-link">http://www.gbe.oxfordjournals.org/</div>).
         </p>
         <div tagxxx="supplementary-material">
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Supplementary Data</div>
            </caption>
            <div tagxxx="media"></div>
            <div tagxxx="media"></div>
         </div>
      </div>
      <div tagxxx="back">
         <div tagxxx="ack">
            <div tagxxx="title">Acknowledgments</div>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The authors thank Virginia Schmit for helpful discussions and David Liberles and Virginia
               Schmit for careful reading of the manuscript. This project was supported by grants
               from the 
               <div tagxxx="funding-source">National Center for Research Resources</div> (
               <div tagxxx="award-id">5P20RR016474-12</div>) and the 
               <div tagxxx="funding-source">National Institute of General Medical Sciences</div> (
               <div tagxxx="award-id">8 P20 GM103432-12</div>) from the 
               <div tagxxx="funding-source">National Institutes of Health</div>. The content is solely the responsibility of the author and does not necessarily
               represent the official views of the National Center for Research Resources or the
               National Institutes of Health.
            </p>
         </div>
         <div tag="ref-list">
            <ul>
               <div tagxxx="title">Literature Cited</div>
               <li tag="ref">
                  <a name="evt026-B1"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Altschul</span>
                           <span tagx="given-names">SF</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gish</span>
                           <span tagx="given-names">W</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Miller</span>
                           <span tagx="given-names">W</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Myers</span>
                           <span tagx="given-names">EW</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Lipman</span>
                           <span tagx="given-names">DJ</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Basic local alignment search tool</span>
                     <span tagx="source">J Mol Biol.</span>
                     <span tagx="year">1990</span>
                     <span tagx="volume">215</span>
                     <span tagx="fpage">403</span>
                     <span tagx="lpage">410</span>
                     <span class="pub-id'">[2231712]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B2"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Cohen</span>
                           <span tagx="given-names">O</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Ashkenazy</span>
                           <span tagx="given-names">H</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Belinky</span>
                           <span tagx="given-names">F</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Huchon</span>
                           <span tagx="given-names">D</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Pupko</span>
                           <span tagx="given-names">T</span>,
                        </span>
                     </span>
                     <span tagx="article-title">GLOOME: gain loss mapping engine</span>
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">26</span>
                     <span tagx="fpage">2914</span>
                     <span tagx="lpage">2915</span>
                     <span class="pub-id'">[20876605]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B3"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Csermely</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Palotai</span>
                           <span tagx="given-names">R</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Nussinov</span>
                           <span tagx="given-names">R</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Induced fit, conformational selection and independent dynamic segments: an extended
                        view of binding events
                     </span>
                     <span tagx="source">Trends Biochem Sci.</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">35</span>
                     <span tagx="fpage">539</span>
                     <span tagx="lpage">546</span>
                     <span class="pub-id'">[20541943]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B4"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Darriba</span>
                           <span tagx="given-names">D</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Taboada</span>
                           <span tagx="given-names">GL</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Doallo</span>
                           <span tagx="given-names">R</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Posada</span>
                           <span tagx="given-names">D</span>,
                        </span>
                     </span>
                     <span tagx="article-title">ProtTest 3: fast selection of best-fit models of protein evolution</span>
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">27</span>
                     <span tagx="fpage">1164</span>
                     <span tagx="lpage">1165</span>
                     <span class="pub-id'">[21335321]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B5"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">de Lamballerie</span>
                           <span tagx="given-names">X</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Genome sequence analysis of Tamana bat virus and its relationship with the genus Flavivirus</span>
                     <span tagx="source">J Gen Virol.</span>
                     <span tagx="year">2002</span>
                     <span tagx="volume">83</span>
                     <span tagx="fpage">2443</span>
                     <span tagx="lpage">2454</span>
                     <span class="pub-id'">[12237426]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B6"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">del Sol</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Tsai</span>
                           <span tagx="given-names">C-J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Ma</span>
                           <span tagx="given-names">B</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Nussinov</span>
                           <span tagx="given-names">R</span>,
                        </span>
                     </span>
                     <span tagx="article-title">The origin of allosteric functional modulation: multiple pre-existing pathways</span>
                     <span tagx="source">Structure</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">17</span>
                     <span tagx="fpage">1042</span>
                     <span tagx="lpage">1050</span>
                     <span class="pub-id'">[19679084]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B7"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dosztányi</span>
                           <span tagx="given-names">Z</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Csizmok</span>
                           <span tagx="given-names">V</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Tompa</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Simon</span>
                           <span tagx="given-names">I</span>,
                        </span>
                     </span>
                     <span tagx="article-title">IUPred: web server for the prediction of intrinsically unstructured regions of proteins
                        based on estimated energy content
                     </span>
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="year">2005</span>
                     <span tagx="volume">21</span>
                     <span tagx="fpage">3433</span>
                     <span tagx="lpage">3434</span>
                     <span class="pub-id'">[15955779]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B8"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dosztányi</span>
                           <span tagx="given-names">Z</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Mészáros</span>
                           <span tagx="given-names">B</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Simon</span>
                           <span tagx="given-names">I</span>,
                        </span>
                     </span>
                     <span tagx="article-title">ANCHOR: web server for predicting protein binding regions in disordered proteins</span>
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">25</span>
                     <span tagx="fpage">2745</span>
                     <span tagx="lpage">2746</span>
                     <span class="pub-id'">[19717576]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B9"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dunker</span>
                           <span tagx="given-names">AK</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Brown</span>
                           <span tagx="given-names">CJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Lawson</span>
                           <span tagx="given-names">JD</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Iakoucheva</span>
                           <span tagx="given-names">LM</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Obradović</span>
                           <span tagx="given-names">Z</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Intrinsic disorder and protein function</span>
                     <span tagx="source">Biochemistry</span>
                     <span tagx="year">2002</span>
                     <span tagx="volume">41</span>
                     <span tagx="fpage">6573</span>
                     <span tagx="lpage">6582</span>
                     <span class="pub-id'">[12022860]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B10"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dunker</span>
                           <span tagx="given-names">AK</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Cortese</span>
                           <span tagx="given-names">MS</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Romero</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Iakoucheva</span>
                           <span tagx="given-names">LM</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Uversky</span>
                           <span tagx="given-names">VN</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Flexible nets. The roles of intrinsic disorder in protein interaction networks</span>
                     <span tagx="source">FEBS J.</span>
                     <span tagx="year">2005</span>
                     <span tagx="volume">272</span>
                     <span tagx="fpage">5129</span>
                     <span tagx="lpage">5148</span>
                     <span class="pub-id'">[16218947]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B11"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dunker</span>
                           <span tagx="given-names">AK</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Intrinsically disordered protein</span>
                     <span tagx="source">J Mol Graph Model.</span>
                     <span tagx="year">2001</span>
                     <span tagx="volume">19</span>
                     <span tagx="fpage">26</span>
                     <span tagx="lpage">59</span>
                     <span class="pub-id'">[11381529]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B12"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dunker</span>
                           <span tagx="given-names">AK</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Obradovic</span>
                           <span tagx="given-names">Z</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Romero</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Garner</span>
                           <span tagx="given-names">EC</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Brown</span>
                           <span tagx="given-names">CJ</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Intrinsic protein disorder in complete genomes</span>
                     <span tagx="source">Genome Inform.</span>
                     <span tagx="year">2000</span>
                     <span tagx="volume">11</span>
                     <span tagx="fpage">161</span>
                     <span tagx="lpage">171</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B13"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dyson</span>
                           <span tagx="given-names">HJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Wright</span>
                           <span tagx="given-names">PE</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Intrinsically unstructured proteins and their functions</span>
                     <span tagx="source">Nat Rev Mol Cell Biol.</span>
                     <span tagx="year">2005</span>
                     <span tagx="volume">6</span>
                     <span tagx="fpage">197</span>
                     <span tagx="lpage">208</span>
                     <span class="pub-id'">[15738986]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B14"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Fuxreiter</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Tompa</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Simon</span>
                           <span tagx="given-names">I</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Local structural disorder imparts plasticity on linear motifs</span>
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="year">2007</span>
                     <span tagx="volume">23</span>
                     <span tagx="fpage">950</span>
                     <span tagx="lpage">956</span>
                     <span class="pub-id'">[17387114]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B15"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Guindon</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Delsuc</span>
                           <span tagx="given-names">F</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Dufayard</span>
                           <span tagx="given-names">J-F</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gascuel</span>
                           <span tagx="given-names">O</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Estimating maximum likelihood phylogenies with PhyML</span>
                     <span tagx="source">Methods Mol Biol.</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">537</span>
                     <span tagx="fpage">113</span>
                     <span tagx="lpage">137</span>
                     <span class="pub-id'">[19378142]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B16"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Guindon</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Lethiec</span>
                           <span tagx="given-names">F</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Duroux</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gascuel</span>
                           <span tagx="given-names">O</span>,
                        </span>
                     </span>
                     <span tagx="article-title">PHYML Online—a web server for fast maximum likelihood-based phylogenetic inference</span>
                     <span tagx="source">Nucleic Acids Res.</span>
                     <span tagx="year">2005</span>
                     <span tagx="volume">33</span>
                     <span tagx="fpage">W557</span>
                     <span tagx="lpage">W559</span>
                     <span class="pub-id'">[15980534]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B17"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Gunasekaran</span>
                           <span tagx="given-names">K</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Ma</span>
                           <span tagx="given-names">B</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Nussinov</span>
                           <span tagx="given-names">R</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Is allostery an intrinsic property of all dynamic proteins?</span>
                     <span tagx="source">Proteins</span>
                     <span tagx="year">2004</span>
                     <span tagx="volume">57</span>
                     <span tagx="fpage">433</span>
                     <span tagx="lpage">443</span>
                     <span class="pub-id'">[15382234]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B18"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Guzman</span>
                           <span tagx="given-names">MG</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Vazquez</span>
                           <span tagx="given-names">S</span>,
                        </span>
                     </span>
                     <span tagx="article-title">The complexity of antibody-dependent enhancement of dengue virus infection</span>
                     <span tagx="source">Viruses</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">2</span>
                     <span tagx="fpage">2649</span>
                     <span tagx="lpage">2662</span>
                     <span class="pub-id'">[21994635]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B19"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Heinz</span>
                           <span tagx="given-names">FX</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Stiasny</span>
                           <span tagx="given-names">K</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Flaviviruses and flavivirus vaccines</span>
                     <span tagx="source">Vaccine</span>
                     <span tagx="year">2012</span>
                     <span tagx="volume">30</span>
                     <span tagx="fpage">4301</span>
                     <span tagx="lpage">4306</span>
                     <span class="pub-id'">[22682286]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B20"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Hsu</span>
                           <span tagx="given-names">W-L</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Intrinsic protein disorder and protein-protein interactions</span>
                     <span tagx="source">Pac Symp Biocomput</span>
                     <span tagx="year">2012</span>
                     <span tagx="fpage">116</span>
                     <span tagx="lpage">127</span>
                     <span class="pub-id'">[22174268]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B21"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Iakoucheva</span>
                           <span tagx="given-names">LM</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Brown</span>
                           <span tagx="given-names">CJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Lawson</span>
                           <span tagx="given-names">JD</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Obradović</span>
                           <span tagx="given-names">Z</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Dunker</span>
                           <span tagx="given-names">AK</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Intrinsic disorder in cell-signaling and cancer-associated proteins</span>
                     <span tagx="source">J Mol Biol.</span>
                     <span tagx="year">2002</span>
                     <span tagx="volume">323</span>
                     <span tagx="fpage">573</span>
                     <span tagx="lpage">584</span>
                     <span class="pub-id'">[12381310]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B22"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Jones</span>
                           <span tagx="given-names">DT</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Taylor</span>
                           <span tagx="given-names">WR</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Thornton</span>
                           <span tagx="given-names">JM</span>,
                        </span>
                     </span>
                     <span tagx="article-title">The rapid generation of mutation data matrices from protein sequences</span>
                     <span tagx="source">Comput Appl Biosci.</span>
                     <span tagx="year">1992</span>
                     <span tagx="volume">8</span>
                     <span tagx="fpage">275</span>
                     <span tagx="lpage">282</span>
                     <span class="pub-id'">[1633570]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B23"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Katoh</span>
                           <span tagx="given-names">K</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Misawa</span>
                           <span tagx="given-names">K</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Kuma</span>
                           <span tagx="given-names">K</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Miyata</span>
                           <span tagx="given-names">T</span>,
                        </span>
                     </span>
                     <span tagx="article-title">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier
                        transform
                     </span>
                     <span tagx="source">Nucleic Acids Res.</span>
                     <span tagx="year">2002</span>
                     <span tagx="volume">30</span>
                     <span tagx="fpage">3059</span>
                     <span tagx="lpage">3066</span>
                     <span class="pub-id'">[12136088]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B24"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Kim</span>
                           <span tagx="given-names">PM</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Sboner</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Xia</span>
                           <span tagx="given-names">Y</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gerstein</span>
                           <span tagx="given-names">M</span>,
                        </span>
                     </span>
                     <span tagx="article-title">The role of disorder in interaction networks: a structural analysis</span>
                     <span tagx="source">Mol Syst Biol.</span>
                     <span tagx="year">2008</span>
                     <span tagx="volume">4</span>
                     <span tagx="fpage">179</span>
                     <span class="pub-id'">[18364713]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B26"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Le Breton</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Flavivirus NS3 and NS5 proteins interaction network: a high-throughput yeast two-hybrid
                        screen
                     </span>
                     <span tagx="source">BMC Microbiol.</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">11</span>
                     <span tagx="fpage">234</span>
                     <span class="pub-id'">[22014111]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B27"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Letunic</span>
                           <span tagx="given-names">I</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Bork</span>
                           <span tagx="given-names">P</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Interactive Tree Of Life (iTOL): an online tool for phylogenetic tree display and
                        annotation
                     </span>
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="year">2007</span>
                     <span tagx="volume">23</span>
                     <span tagx="fpage">127</span>
                     <span tagx="lpage">128</span>
                     <span class="pub-id'">[17050570]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B28"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Letunic</span>
                           <span tagx="given-names">I</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Bork</span>
                           <span tagx="given-names">P</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Interactive Tree Of Life v2: online annotation and display of phylogenetic trees made
                        easy
                     </span>
                     <span tagx="source">Nucleic Acids Res.</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">39</span>
                     <span tagx="fpage">W475</span>
                     <span tagx="lpage">W478</span>
                     <span class="pub-id'">[21470960]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B29"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Liu</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Faeder</span>
                           <span tagx="given-names">JR</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Camacho</span>
                           <span tagx="given-names">CJ</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Toward a quantitative theory of intrinsically disordered proteins and their function</span>
                     <span tagx="source">Proc Natl Acad Sci U S A.</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">106</span>
                     <span tagx="fpage">19819</span>
                     <span tagx="lpage">19823</span>
                     <span class="pub-id'">[19903882]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B30"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Lobo</span>
                           <span tagx="given-names">FP</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Virus-host coevolution: common patterns of nucleotide motif usage in Flaviviridae
                        and their hosts
                     </span>
                     <span tagx="source">PloS One</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">4</span>
                     <span tagx="fpage">e6282</span>
                     <span class="pub-id'">[19617912]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B31"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Ma</span>
                           <span tagx="given-names">B</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Kumar</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Tsai</span>
                           <span tagx="given-names">CJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Nussinov</span>
                           <span tagx="given-names">R</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Folding funnels and binding mechanisms</span>
                     <span tagx="source">Protein Eng.</span>
                     <span tagx="year">1999</span>
                     <span tagx="volume">12</span>
                     <span tagx="fpage">713</span>
                     <span tagx="lpage">720</span>
                     <span class="pub-id'">[10506280]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B32"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Mészáros</span>
                           <span tagx="given-names">B</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Simon</span>
                           <span tagx="given-names">I</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Dosztányi</span>
                           <span tagx="given-names">Z</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Prediction of protein binding regions in disordered proteins</span>
                     <span tagx="source">PLoS Comput Biol.</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">5</span>
                     <span tagx="fpage">e1000376</span>
                     <span class="pub-id'">[19412530]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B33"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Mohan</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Analysis of molecular recognition features (MoRFs)</span>
                     <span tagx="source">J Mol Biol.</span>
                     <span tagx="year">2006</span>
                     <span tagx="volume">362</span>
                     <span tagx="fpage">1043</span>
                     <span tagx="lpage">1059</span>
                     <span class="pub-id'">[16935303]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B34"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Mosca</span>
                           <span tagx="given-names">R</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Pache</span>
                           <span tagx="given-names">RA</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Aloy</span>
                           <span tagx="given-names">P</span>,
                        </span>
                     </span>
                     <span tagx="article-title">The role of structural disorder in the rewiring of protein interactions through evolution</span>
                     <span tagx="source">Mol Cell Proteomics</span>
                     <span tagx="year">2012</span>
                     <span tagx="volume">11</span>
                     <span tagx="fpage">M111.014969</span>
                     <span class="pub-id'">[22389433]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B35"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Nido</span>
                           <span tagx="given-names">GS</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Méndez</span>
                           <span tagx="given-names">R</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Pascual-García</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Abia</span>
                           <span tagx="given-names">D</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Bastolla</span>
                           <span tagx="given-names">U</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Protein disorder in the centrosome correlates with complexity in cell types number</span>
                     <span tagx="source">Mol Biosyst.</span>
                     <span tagx="year">2012</span>
                     <span tagx="volume">8</span>
                     <span tagx="fpage">353</span>
                     <span tagx="lpage">367</span>
                     <span class="pub-id'">[22076659]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B36"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Oldfield</span>
                           <span tagx="given-names">CJ</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Coupled folding and binding with alpha-helix-forming molecular recognition elements</span>
                     <span tagx="source">Biochemistry</span>
                     <span tagx="year">2005</span>
                     <span tagx="volume">44</span>
                     <span tagx="fpage">12454</span>
                     <span tagx="lpage">12470</span>
                     <span class="pub-id'">[16156658]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B37"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Oldfield</span>
                           <span tagx="given-names">CJ</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Flexible nets: disorder and induced fit in the associations of p53 and 14-3-3 with
                        their partners
                     </span>
                     <span tagx="source">BMC Genomics</span>
                     <span tagx="year">2008</span>
                     <span tagx="volume">9</span>
                     <div tagxxx="issue">Suppl 1</div>
                     <span tagx="fpage">S1</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B38"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Romero</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Thousands of proteins likely to have long disordered regions</span>
                     <span tagx="source">Pac Symp Biocomput</span>
                     <span tagx="year">1998</span>
                     <span tagx="fpage">437</span>
                     <span tagx="lpage">448</span>
                     <span class="pub-id'">[9697202]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B39"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Sickmeier</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">DisProt: the Database of Disordered Proteins</span>
                     <span tagx="source">Nucleic Acids Res.</span>
                     <span tagx="year">2007</span>
                     <span tagx="volume">35</span>
                     <span tagx="fpage">D786</span>
                     <span tagx="lpage">D793</span>
                     <span class="pub-id'">[17145717]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B40"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Siltberg-Liberles</span>
                           <span tagx="given-names">J</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Evolution of structurally disordered proteins promotes neostructuralization</span>
                     <span tagx="source">Mol Biol Evol.</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">28</span>
                     <span tagx="fpage">59</span>
                     <span tagx="lpage">62</span>
                     <span class="pub-id'">[21037204]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B41"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Siltberg-Liberles</span>
                           <span tagx="given-names">J</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Grahnen</span>
                           <span tagx="given-names">JA</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Liberles</span>
                           <span tagx="given-names">DA</span>,
                        </span>
                     </span>
                     <span tagx="article-title">The evolution of protein structures and structural ensembles under functional constraint</span>
                     <span tagx="source">Genes</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">2</span>
                     <span tagx="fpage">748</span>
                     <span tagx="lpage">762</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B42"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Smock</span>
                           <span tagx="given-names">RG</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gierasch</span>
                           <span tagx="given-names">LM</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Sending signals dynamically</span>
                     <span tagx="source">Science</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">324</span>
                     <span tagx="fpage">198</span>
                     <span tagx="lpage">203</span>
                     <span class="pub-id'">[19359576]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B43"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Tamura</span>
                           <span tagx="given-names">K</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary
                        distance, and maximum parsimony methods
                     </span>
                     <span tagx="source">Mol Biol Evol.</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">28</span>
                     <span tagx="fpage">2731</span>
                     <span tagx="lpage">2739</span>
                     <span class="pub-id'">[21546353]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B44"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Tokuriki</span>
                           <span tagx="given-names">N</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Tawfik</span>
                           <span tagx="given-names">DS</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Protein dynamism and evolvability</span>
                     <span tagx="source">Science</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">324</span>
                     <span tagx="fpage">203</span>
                     <span tagx="lpage">207</span>
                     <span class="pub-id'">[19359577]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B45"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Tompa</span>
                           <span tagx="given-names">P</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Intrinsically unstructured proteins</span>
                     <span tagx="source">Trends Biochem Sci.</span>
                     <span tagx="year">2002</span>
                     <span tagx="volume">27</span>
                     <span tagx="fpage">527</span>
                     <span tagx="lpage">533</span>
                     <span class="pub-id'">[12368089]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B46"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Tsai</span>
                           <span tagx="given-names">CJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Ma</span>
                           <span tagx="given-names">B</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Sham</span>
                           <span tagx="given-names">YY</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Kumar</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Nussinov</span>
                           <span tagx="given-names">R</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Structured disorder and conformational selection</span>
                     <span tagx="source">Proteins</span>
                     <span tagx="year">2001</span>
                     <span tagx="volume">44</span>
                     <span tagx="fpage">418</span>
                     <span tagx="lpage">427</span>
                     <span class="pub-id'">[11484219]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B47"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Uversky</span>
                           <span tagx="given-names">VN</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Multitude of binding modes attainable by intrinsically disordered proteins: a portrait
                        gallery of disorder-based complexes
                     </span>
                     <span tagx="source">Chem Soc Rev.</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">40</span>
                     <span tagx="fpage">1623</span>
                     <span tagx="lpage">34</span>
                     <span class="pub-id'">[21049125]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B48"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Uversky</span>
                           <span tagx="given-names">VN</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Natively unfolded proteins: a point where biology waits for physics</span>
                     <span tagx="source">Protein Sci.</span>
                     <span tagx="year">2002</span>
                     <span tagx="volume">11</span>
                     <span tagx="fpage">739</span>
                     <span tagx="lpage">756</span>
                     <span class="pub-id'">[11910019]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B49"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Uversky</span>
                           <span tagx="given-names">VN</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Gillespie</span>
                           <span tagx="given-names">JR</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Fink</span>
                           <span tagx="given-names">AL</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Why are “natively unfolded” proteins unstructured under physiologic conditions?</span>
                     <span tagx="source">Proteins</span>
                     <span tagx="year">2000</span>
                     <span tagx="volume">41</span>
                     <span tagx="fpage">415</span>
                     <span tagx="lpage">427</span>
                     <span class="pub-id'">[11025552]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B50"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Uversky</span>
                           <span tagx="given-names">VN</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Oldfield</span>
                           <span tagx="given-names">CJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Dunker</span>
                           <span tagx="given-names">AK</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Showing your ID: intrinsic disorder as an ID for recognition, regulation and cell
                        signaling
                     </span>
                     <span tagx="source">J Mol Recognit.</span>
                     <span tagx="year">2005</span>
                     <span tagx="volume">18</span>
                     <span tagx="fpage">343</span>
                     <span tagx="lpage">384</span>
                     <span class="pub-id'">[16094605]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B51"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Uversky</span>
                           <span tagx="given-names">VN</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Oldfield</span>
                           <span tagx="given-names">CJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Dunker</span>
                           <span tagx="given-names">AK</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Intrinsically disordered proteins in human diseases: introducing the D2 concept</span>
                     <span tagx="source">Ann Rev Biophys.</span>
                     <span tagx="year">2008</span>
                     <span tagx="volume">37</span>
                     <span tagx="fpage">215</span>
                     <span tagx="lpage">246</span>
                     <span class="pub-id'">[18573080]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B52"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Vacic</span>
                           <span tagx="given-names">V</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Characterization of molecular recognition features, MoRFs, and their binding partners</span>
                     <span tagx="source">J Proteome Res.</span>
                     <span tagx="year">2007</span>
                     <span tagx="volume">6</span>
                     <span tagx="fpage">2351</span>
                     <span tagx="lpage">2366</span>
                     <span class="pub-id'">[17488107]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B53"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Vucetic</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Functional anthology of intrinsic disorder. 2. Cellular components, domains, technical
                        terms, developmental processes, and coding sequence diversities correlated with long
                        disordered regions
                     </span>
                     <span tagx="source">J Proteome Res.</span>
                     <span tagx="year">2007</span>
                     <span tagx="volume">6</span>
                     <span tagx="fpage">1899</span>
                     <span tagx="lpage">1916</span>
                     <span class="pub-id'">[17391015]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B54"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Ward</span>
                           <span tagx="given-names">JJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Sodhi</span>
                           <span tagx="given-names">JS</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">McGuffin</span>
                           <span tagx="given-names">LJ</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Buxton</span>
                           <span tagx="given-names">BF</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Jones</span>
                           <span tagx="given-names">DT</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Prediction and functional analysis of native disorder in proteins from the three kingdoms
                        of life
                     </span>
                     <span tagx="source">J Mol Biol.</span>
                     <span tagx="year">2004</span>
                     <span tagx="volume">337</span>
                     <span tagx="fpage">635</span>
                     <span tagx="lpage">645</span>
                     <span class="pub-id'">[15019783]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B55"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Wright</span>
                           <span tagx="given-names">PE</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Dyson</span>
                           <span tagx="given-names">HJ</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Intrinsically unstructured proteins: re-assessing the protein structure-function paradigm</span>
                     <span tagx="source">J Mol Biol.</span>
                     <span tagx="year">1999</span>
                     <span tagx="volume">293</span>
                     <span tagx="fpage">321</span>
                     <span tagx="lpage">331</span>
                     <span class="pub-id'">[10550212]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B56"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Wright</span>
                           <span tagx="given-names">PE</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Dyson</span>
                           <span tagx="given-names">HJ</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Linking folding and binding</span>
                     <span tagx="source">Curr Opin Struct Biol.</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">19</span>
                     <span tagx="fpage">31</span>
                     <span tagx="lpage">38</span>
                     <span class="pub-id'">[19157855]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B57"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Xie</span>
                           <span tagx="given-names">H</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Functional anthology of intrinsic disorder. 1. Biological processes and functions
                        of proteins with long disordered regions
                     </span>
                     <span tagx="source">J Proteome Res.</span>
                     <span tagx="year">2007</span>
                     <span tagx="volume">6</span>
                     <span tagx="fpage">1882</span>
                     <span tagx="lpage">1898</span>
                     <span class="pub-id'">[17391014]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B58"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Xue</span>
                           <span tagx="given-names">B</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Dunbrack</span>
                           <span tagx="given-names">RL</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Williams</span>
                           <span tagx="given-names">RW</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Dunker</span>
                           <span tagx="given-names">AK</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Uversky</span>
                           <span tagx="given-names">VN</span>,
                        </span>
                     </span>
                     <span tagx="article-title">PONDR-FIT: a meta-predictor of intrinsically disordered amino acids</span>
                     <span tagx="source">Biochim Biophys Acta.</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">1804</span>
                     <span tagx="fpage">996</span>
                     <span tagx="lpage">1010</span>
                     <span class="pub-id'">[20100603]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B59"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Xue</span>
                           <span tagx="given-names">B</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Dunker</span>
                           <span tagx="given-names">AK</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Uversky</span>
                           <span tagx="given-names">VN</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Orderly order in protein intrinsic disorder distribution: disorder in 3500 proteomes
                        from viruses and the three domains of life
                     </span>
                     <span tagx="source">J Biomol Struct Dynam.</span>
                     <span tagx="year">2012</span>
                     <span tagx="volume">30</span>
                     <span tagx="fpage">137</span>
                     <span tagx="lpage">149</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B60"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Xue</span>
                           <span tagx="given-names">B</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Williams</span>
                           <span tagx="given-names">RW</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Viral disorder or disordered viruses: do viral proteins possess unique features?</span>
                     <span tagx="source">Protein Pept Lett.</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">17</span>
                     <span tagx="fpage">932</span>
                     <span tagx="lpage">951</span>
                     <span class="pub-id'">[20450483]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="evt026-B61"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Zhang</span>
                           <span tagx="given-names">Y</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Conformational changes of the flavivirus E glycoprotein</span>
                     <span tagx="source">Structure</span>
                     <span tagx="year">2004</span>
                     <span tagx="volume">12</span>
                     <span tagx="fpage">1607</span>
                     <span tagx="lpage">1618</span>
                     <span class="pub-id'">[15341726]</span>
                  </span>
               </li>
            </ul>
         </div>
      </div>
   </div>
</html>