<?xml version="1.0" encoding="UTF-8"?>
<html>
 <body>
  <ul>
   <li> elements and the epigenetic mechanisms that modulate induced and constitutive expression of these  <a href="http://en.wikipedia.org/wiki/MHC">MHC</a>  genes. "    </li>
   <li>sine residues act as molecular switches for CIITA-mediated transcriptional activation/silencing of  <a href="http://en.wikipedia.org/wiki/MHC">MHC</a>  genes. In this respect, it was found that HDAC1 and HDAC2 interfere in the transcriptional transac"    </li>
   <li>Transcriptional regulation of  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a> , the gene encoding CIITA, is mediated through the activity of four independent promoter units (CII"    </li>
   <li>2002 ). The promoter function of CIITA-PII is still ill-defined. The various  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a>  promoters each transcribe a unique first exon and are located within a region of approximately 14 "    </li>
   <li> Schematic overview of the  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a>  multi-promoter region . Shown are the factors and elements governing transactivation of CIITA-PI i"    </li>
   <li>Over the past years, several regulatory elements in  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a>  promoters and interacting factors that are important for transcriptional activation have been iden"    </li>
   <li>ssion of CIITA-PIV (Chen et al., 2007 ). Transcriptional repression of  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a>  mediated by Blimp-1/PRDI-BF1 is also associated with histone deacetylase and lysine methyltransfer"    </li>
   <li> found also to exist at CIITA-PI, CIITA-PII and CIITA-PIV, revealing the involvement of the entire  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a>  multi-promoter region. As a consequence of the repressive histone marks and resulting chromatin in"    </li>
   <li>In dendritic cell maturation chromatin remodeling also plays an important role in the control of  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a>  transcription (Landmann et al., 2001 ; Choi et al., &lt;a href=&quot;#"    </li>
   <li>nsported MHC-II molecules at the cell surface is accompanied by rapid transcriptional silencing of  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a>  in matured dendritic cells (Landmann et al., 2001 ). Like in plasma B"    </li>
   <li>ck the increase in histone H3 and H4 acetylation during differentiation resulting in inhibition of  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a>  transcription (Choi et al., 2009 ). Recently the involvement of the tr"    </li>
   <li>elements and chromatin remodeling also play an essential role in the transcriptional regulation of  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a>  (Ni et al., 2008 ). Transcriptional activation of CIITA-PIV by IFNγ r"    </li>
   <li>TA-mediated activation of MHC-II genes, but also controls the transcriptional activation of  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a>  itself through long-range chromatin and promoter interactions. "    </li>
   <li>sion coincides with lack of detectable levels of CIITA. In cancer the transcriptional silencing of  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a>  and resulting extinction of MHC-II gene expression is frequently noted (Murphy and Tomasi, "    </li>
   <li>53&quot;&gt;2008 ). Besides CpG dinucleotide methylation, the lack of IFNγ-induced transcription of  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a>  is also associated with histone deacetylase activities (Magner et al., 2000 "    </li>
   <li>l line it was demonstrated for the first time that histone methylation played an important role in  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a>  transcriptional silencing (Holling et al., 2007 ). The strongly reduc"    </li>
   <li>ilencing and maintenance of cellular identity. Consistent with the transcriptional silent state of  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a>  was the lack of RNA polymerase II recruitment into CIITA-PIV chromatin after IFNγ-induction (Holli"    </li>
   <li>ma (Holling et al., 2007 ). Notably, the transcriptional silencing of  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a>  by histone methylation in the absence of CpG dinucleotide methylation is in line with the observat"    </li>
   <li> for de novo methylation of DNA, and that this reflects an intermediate epigenetic state of  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a>  in the complete shut down of MHC-II mediated antigen presentation functions. More recently, lack o"    </li>
   <li>se repressive histone marks were also found to be associated with the other CIITA promoters in the  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a>  multi-promoter region, while the opposite was noted for T lymphoma cells (Van Eggermond et al., &lt;"    </li>
   <li>s provide a clear link with expression of Polycomb Group proteins and transcriptional silencing of  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a>  and resulting lack of expression of MHC-II molecules in cancer. "    </li>
   <li>n additional level of transcriptional control for MHC-II gene expression is at the level of  <a href="http://en.wikipedia.org/wiki/MHC2TA">MHC2TA</a> , the gene encoding CIITA, which is also tightly regulated by both genetic and epigenetic mechanism"    </li>
   <li> mechanisms. Because of the involvement of epigenetic mechanisms in the transcriptional control of  <a href="http://en.wikipedia.org/wiki/MHC">MHC</a>  genes, deviations in these epigenetic mechanisms as observed under pathological conditions such as"   </li>
  </ul>
 </body>
 <body>
  <ul>
   <li>however, technically difficult, and more than ten years since our first demonstration of targeting  <a href="http://en.wikipedia.org/wiki/COL1A1">COL1A1</a>  in sheep [3] , there are still few other examples:"  </li>
   <li>in sheep [3] , there are still few other examples:  <a href="http://en.wikipedia.org/wiki/PRNP">PRNP</a>  in sheep, cattle and goats [4] – &lt;a href=&quot;#p"  </li>
   <li>sup&gt; [4] – [6] ,  <a href="http://en.wikipedia.org/wiki/GGTA1">GGTA1</a>  in pigs [7] , [8] &lt;"    </li>
   <li> [7] , [8] ,  <a href="http://en.wikipedia.org/wiki/IGH">IGH</a>  in cattle and pigs [5] , &lt;a href=&quot;#pone.0021"  </li>
   <li>href=&quot;#pone.0021045-Kuroiwa2&quot;&gt;[9] , [10] ,  <a href="http://en.wikipedia.org/wiki/IGKC">IGKC</a>  in pigs [11] and CFTR in pigs &lt;a "  </li>
   <li>no1&quot;&gt;[10] , IGKC in pigs [11] and  <a href="http://en.wikipedia.org/wiki/CFTR">CFTR</a>  in pigs [12] . " </li>
  </ul>
 </body>
 <body>
  <ul>
   <li>  <a href="http://en.wikipedia.org/wiki/DENV">DENV</a> : Dengue virus, JEV: Japanese encephalitis virus, RSSEV: Russian spring-summer enceph"    </li>
   <li> DENV: Dengue virus,  <a href="http://en.wikipedia.org/wiki/JEV">JEV</a> : Japanese encephalitis virus, RSSEV: Russian spring-summer encephalitis virus, WNV: "    </li>
   <li> DENV: Dengue virus, JEV: Japanese encephalitis virus,  <a href="http://en.wikipedia.org/wiki/RSSEV">RSSEV</a> : Russian spring-summer encephalitis virus, WNV: West Nile virus, TBEV: Tick-borne en"    </li>
   <li>, JEV: Japanese encephalitis virus, RSSEV: Russian spring-summer encephalitis virus,  <a href="http://en.wikipedia.org/wiki/WNV">WNV</a> : West Nile virus, TBEV: Tick-borne encephalitis virus, SLEV: Saint Louis encephaliti"    </li>
   <li>alitis virus, RSSEV: Russian spring-summer encephalitis virus, WNV: West Nile virus,  <a href="http://en.wikipedia.org/wiki/TBEV">TBEV</a> : Tick-borne encephalitis virus, SLEV: Saint Louis encephalitis virus, RVFV: Rift val"    </li>
   <li>ummer encephalitis virus, WNV: West Nile virus, TBEV: Tick-borne encephalitis virus,  <a href="http://en.wikipedia.org/wiki/SLEV">SLEV</a> : Saint Louis encephalitis virus, RVFV: Rift valley fever virus, YFV: Yellow fever vi"    </li>
   <li>le virus, TBEV: Tick-borne encephalitis virus, SLEV: Saint Louis encephalitis virus,  <a href="http://en.wikipedia.org/wiki/RVFV">RVFV</a> : Rift valley fever virus, YFV: Yellow fever virus. "    </li>
   <li>phalitis virus, SLEV: Saint Louis encephalitis virus, RVFV: Rift valley fever virus,  <a href="http://en.wikipedia.org/wiki/YFV">YFV</a> : Yellow fever virus. "    </li>
   <li>  <a href="http://en.wikipedia.org/wiki/DENV">DENV</a> : Dengue virus, JEV: Japanese encephalitis virus, RSSEV: Russian spring-summer enceph"    </li>
   <li> DENV: Dengue virus,  <a href="http://en.wikipedia.org/wiki/JEV">JEV</a> : Japanese encephalitis virus, RSSEV: Russian spring-summer encephalitis virus, WNV: "    </li>
   <li> DENV: Dengue virus, JEV: Japanese encephalitis virus,  <a href="http://en.wikipedia.org/wiki/RSSEV">RSSEV</a> : Russian spring-summer encephalitis virus, WNV: West Nile virus, TBEV: Tick-borne en"    </li>
   <li>, JEV: Japanese encephalitis virus, RSSEV: Russian spring-summer encephalitis virus,  <a href="http://en.wikipedia.org/wiki/WNV">WNV</a> : West Nile virus, TBEV: Tick-borne encephalitis virus, YFV: Yellow fever virus. "    </li>
   <li>alitis virus, RSSEV: Russian spring-summer encephalitis virus, WNV: West Nile virus,  <a href="http://en.wikipedia.org/wiki/TBEV">TBEV</a> : Tick-borne encephalitis virus, YFV: Yellow fever virus. "    </li>
   <li>ummer encephalitis virus, WNV: West Nile virus, TBEV: Tick-borne encephalitis virus,  <a href="http://en.wikipedia.org/wiki/YFV">YFV</a> : Yellow fever virus. "    </li>
   <li>  <a href="http://en.wikipedia.org/wiki/YFV">YFV</a> : Yellow fever virus, TBE: Tick-borne encephalitis virus, DENV: Dengue virus. "    </li>
   <li> YFV: Yellow fever virus,  <a href="http://en.wikipedia.org/wiki/TBE">TBE</a> : Tick-borne encephalitis virus, DENV: Dengue virus. "    </li>
   <li> YFV: Yellow fever virus, TBE: Tick-borne encephalitis virus,  <a href="http://en.wikipedia.org/wiki/DENV">DENV</a> : Dengue virus. "   </li>
  </ul>
 </body>
 <body>
  <ul>
   <li>FAM, 6-carboxyfluorescein;  <a href="http://en.wikipedia.org/wiki/BBQ">BBQ</a> , Black Berry Quencher. Y = T or C, R = A or G. "   </li>
  </ul>
 </body>
 <body>
  <ul>
   <li>ional biotic or abiotic factors. Analysis of mtDNA sequences revealed four haplotypes each for the  <a href="http://en.wikipedia.org/wiki/COI">COI</a>  and ND5 genes, with a dominant haplotype shared by all Cameroonian samples [ &lt;a href"    </li>
   <li> abiotic factors. Analysis of mtDNA sequences revealed four haplotypes each for the COI and  <a href="http://en.wikipedia.org/wiki/ND5">ND5</a>  genes, with a dominant haplotype shared by all Cameroonian samples [ 42 &lt;/s"    </li>
   <li> all Cameroonian samples [ 42 ]. Phylogeographical analysis based on  <a href="http://en.wikipedia.org/wiki/COI">COI</a>  polymorphism indicated that Ae. albopictus populations in Cameroon are related to tropical "    </li>
   <li>ted by Armbruster et al. [ 44 ]. Phylogeographical analysis based on  <a href="http://en.wikipedia.org/wiki/COI">COI</a>  polymorphism indicated that the Ae. albopictus haplotype in the CAR population segregated i"   </li>
  </ul>
 </body>
 <body>
  <ul>
   <li> quantified by real time RT-PCR in separated embryoblast (16.03±1.43 molecules per 10 3  <a href="http://en.wikipedia.org/wiki/GAPDH">GAPDH</a>  molecules) and trophoblast (15.31±1.63 molecules per 10 3 GAPDH molecules). Bcat"  </li>
   <li> 10 3 GAPDH molecules) and trophoblast (15.31±1.63 molecules per 10 3  <a href="http://en.wikipedia.org/wiki/GAPDH">GAPDH</a>  molecules). Bcat2 transcripts were equal in both cell lineages. During gastrulation at day 6, blas" </li>
  </ul>
 </body>
 <body>
  <ul>
   <li>pone.0133602.ref079&quot;&gt;79 ]. Of note, barcoding involves comparison of short sequences of the  <a href="http://en.wikipedia.org/wiki/COI">COI</a>  mitochondrial gene, producing distance measures that are only useful for delineating and different"    </li>
   <li>ineating and differentiating species-level taxa, not for estimating their relationships. Combining  <a href="http://en.wikipedia.org/wiki/COI">COI</a>  and COII mitochondrial genes from 17 aedine species in western Africa [ &lt;a href=&quot;#po"    </li>
   <li>fferentiating species-level taxa, not for estimating their relationships. Combining COI and  <a href="http://en.wikipedia.org/wiki/COII">COII</a>  mitochondrial genes from 17 aedine species in western Africa [ &lt;a href=&quot;#pone.0133602.ref08"   </li>
  </ul>
 </body>
 <body>
  <ul>
   <li>y metabolic resistance mechanisms. Cytochrome P450 genes were implicated in conferring resistance.  <a href="http://en.wikipedia.org/wiki/CYP6BB2">CYP6BB2</a> , CYP6M11, CYP6N12, CYP9J9, CYP9J10 and CCE3 genes were upregula"    </li>
   <li>stance mechanisms. Cytochrome P450 genes were implicated in conferring resistance. CYP6BB2,  <a href="http://en.wikipedia.org/wiki/CYP6M11">CYP6M11</a> , CYP6N12, CYP9J9, CYP9J10 and CCE3 genes were upregulated in the resis"    </li>
   <li>s. Cytochrome P450 genes were implicated in conferring resistance. CYP6BB2, CYP6M11,  <a href="http://en.wikipedia.org/wiki/CYP6N12">CYP6N12</a> , CYP9J9, CYP9J10 and CCE3 genes were upregulated in the resistant populations"    </li>
   <li>50 genes were implicated in conferring resistance. CYP6BB2, CYP6M11, CYP6N12,  <a href="http://en.wikipedia.org/wiki/CYP9J9">CYP9J9</a> , CYP9J10 and CCE3 genes were upregulated in the resistant populations and were commo"    </li>
   <li>mplicated in conferring resistance. CYP6BB2, CYP6M11, CYP6N12, CYP9J9,  <a href="http://en.wikipedia.org/wiki/CYP9J10">CYP9J10</a>  and CCE3 genes were upregulated in the resistant populations and were common to other popul"    </li>
   <li>ring resistance. CYP6BB2, CYP6M11, CYP6N12, CYP9J9, CYP9J10 and  <a href="http://en.wikipedia.org/wiki/CCE3">CCE3</a>  genes were upregulated in the resistant populations and were common to other populations at a regi"    </li>
   <li>t mosquitoes was extracted with the Purelink Genomic DNA extraction kit (Invitrogen, Carlsbad, CA,  <a href="http://en.wikipedia.org/wiki/USA">USA</a> ) according to the manufacturer's instructions, with the following modification: cell lysis was per"    </li>
   <li>throids in mosquitoes. Among the genes common to the three studied populations, one was from CCE (  <a href="http://en.wikipedia.org/wiki/CCE3">CCE3</a> ) and four were CYP450 ( CYP12F7, CYP9J10 and CYP9J27, CYP6BB2). CY6"    </li>
   <li>es common to the three studied populations, one was from CCE ( CCE3) and four were CYP450 (  <a href="http://en.wikipedia.org/wiki/CYP12F7">CYP12F7</a> , CYP9J10 and CYP9J27, CYP6BB2). CY6BB2 is noticeable with up-regulatio"    </li>
   <li> three studied populations, one was from CCE ( CCE3) and four were CYP450 ( CYP12F7,  <a href="http://en.wikipedia.org/wiki/CYP9J10">CYP9J10</a>  and CYP9J27, CYP6BB2). CY6BB2 is noticeable with up-regulations respectively "    </li>
   <li>lations, one was from CCE ( CCE3) and four were CYP450 ( CYP12F7, CYP9J10 and  <a href="http://en.wikipedia.org/wiki/CYP9J27">CYP9J27</a> , CYP6BB2). CY6BB2 is noticeable with up-regulations respectively of 33.83, 13.47 and"    </li>
   <li> from CCE ( CCE3) and four were CYP450 ( CYP12F7, CYP9J10 and CYP9J27,  <a href="http://en.wikipedia.org/wiki/CYP6BB2">CYP6BB2</a> ). CY6BB2 is noticeable with up-regulations respectively of 33.83, 13.47 and 14.19 fold v"    </li>
   <li>E3) and four were CYP450 ( CYP12F7, CYP9J10 and CYP9J27, CYP6BB2).  <a href="http://en.wikipedia.org/wiki/CY6BB2">CY6BB2</a>  is noticeable with up-regulations respectively of 33.83, 13.47 and 14.19 fold vs the NO pop"    </li>
   <li>ur others are differentially expressed with only one common to GUY and none to the GUA population.  <a href="http://en.wikipedia.org/wiki/CYP9J28">CYP9J28</a> , CYP9J27 and CYP6M9 were also up-regulated up to 12.78 to 20.48 fold compared to the"    </li>
   <li>fferentially expressed with only one common to GUY and none to the GUA population. CYP9J28,  <a href="http://en.wikipedia.org/wiki/CYP9J27">CYP9J27</a>  and CYP6M9 were also up-regulated up to 12.78 to 20.48 fold compared to the NO strain. GUY "    </li>
   <li>sed with only one common to GUY and none to the GUA population. CYP9J28, CYP9J27 and  <a href="http://en.wikipedia.org/wiki/CYP6M9">CYP6M9</a>  were also up-regulated up to 12.78 to 20.48 fold compared to the NO strain. GUY and GUA population"    </li>
   <li>f038&quot;&gt;38 ]. CCE and CYP9J enzymes are able to degrade the insecticide whereas CYP6Z such as  <a href="http://en.wikipedia.org/wiki/CYP6Z8">CYP6Z8</a>  seems to degrade subsequent products [ 38 ]. In the p"    </li>
   <li>up&gt; , 42 ] ( Table 2 ).  <a href="http://en.wikipedia.org/wiki/CYP6BB2">CYP6BB2</a> , CYP6Z6, CYP6M11, CYP9J23, CYP9J9v1, CYP9J9v2, CYP9J28, "    </li>
   <li>href=&quot;#pntd.0004226.ref042&quot;&gt;42 ] ( Table 2 ). CYP6BB2,  <a href="http://en.wikipedia.org/wiki/CYP6Z6">CYP6Z6</a> , CYP6M11, CYP9J23, CYP9J9v1, CYP9J9v2, CYP9J28, CYP6CB1,"    </li>
   <li>4226.ref042&quot;&gt;42 ] ( Table 2 ). CYP6BB2, CYP6Z6,  <a href="http://en.wikipedia.org/wiki/CYP6M11">CYP6M11</a> , CYP9J23, CYP9J9v1, CYP9J9v2, CYP9J28, CYP6CB1, CYP9J27,"    </li>
   <li> ] ( Table 2 ). CYP6BB2, CYP6Z6, CYP6M11,  <a href="http://en.wikipedia.org/wiki/CYP9J23">CYP9J23</a> , CYP9J9v1, CYP9J9v2, CYP9J28, CYP6CB1, CYP9J27, CYP9J10,"    </li>
   <li>. CYP6BB2, CYP6Z6, CYP6M11, CYP9J23, CYP9J9v1, CYP9J9v2,  <a href="http://en.wikipedia.org/wiki/CYP9J28">CYP9J28</a> , CYP6CB1, CYP9J27, CYP9J10, GSTE4 were found in six to nine Ae. aeg"    </li>
   <li>, CYP6Z6, CYP6M11, CYP9J23, CYP9J9v1, CYP9J9v2, CYP9J28,  <a href="http://en.wikipedia.org/wiki/CYP6CB1">CYP6CB1</a> , CYP9J27, CYP9J10, GSTE4 were found in six to nine Ae. aegypti populat"    </li>
   <li> CYP6M11, CYP9J23, CYP9J9v1, CYP9J9v2, CYP9J28, CYP6CB1,  <a href="http://en.wikipedia.org/wiki/CYP9J27">CYP9J27</a> , CYP9J10, GSTE4 were found in six to nine Ae. aegypti populations in South Am"    </li>
   <li> CYP9J23, CYP9J9v1, CYP9J9v2, CYP9J28, CYP6CB1, CYP9J27,  <a href="http://en.wikipedia.org/wiki/CYP9J10">CYP9J10</a> , GSTE4 were found in six to nine Ae. aegypti populations in South America and Asia, "    </li>
   <li> CYP9J9v1, CYP9J9v2, CYP9J28, CYP6CB1, CYP9J27, CYP9J10,  <a href="http://en.wikipedia.org/wiki/GSTE4">GSTE4</a>  were found in six to nine Ae. aegypti populations in South America and Asia, enhancing thei"    </li>
   <li>sup&gt; 41 , 43 ].  <a href="http://en.wikipedia.org/wiki/CYP6BB2">CYP6BB2</a>  and CYP9M6 were identified as degrading permethrin in Kasai et al. [ &lt;a href=&quot;#pntd."    </li>
   <li>.0004226.ref041&quot;&gt;41 , 43 ]. CYP6BB2 and  <a href="http://en.wikipedia.org/wiki/CYP9M6">CYP9M6</a>  were identified as degrading permethrin in Kasai et al. [ 41"    </li>
   <li> 41 ]. However, none of our studied population exhibited the  <a href="http://en.wikipedia.org/wiki/CYP9J32">CYP9J32</a>  up-regulated gene found in Mexico, Thailand and Vietnam. CYP9J32 along with CYP9J24,"    </li>
   <li>d population exhibited the CYP9J32 up-regulated gene found in Mexico, Thailand and Vietnam.  <a href="http://en.wikipedia.org/wiki/CYP9J32">CYP9J32</a>  along with CYP9J24, CYP9J26, and CYP9J28 are the four enzymes for which the r"    </li>
   <li> CYP9J32 up-regulated gene found in Mexico, Thailand and Vietnam. CYP9J32 along with  <a href="http://en.wikipedia.org/wiki/CYP9J24">CYP9J24</a> , CYP9J26, and CYP9J28 are the four enzymes for which the role in degrading permethri"    </li>
   <li>up-regulated gene found in Mexico, Thailand and Vietnam. CYP9J32 along with CYP9J24,  <a href="http://en.wikipedia.org/wiki/CYP9J26">CYP9J26</a> , and CYP9J28 are the four enzymes for which the role in degrading permethrin and deltamethr"    </li>
   <li>und in Mexico, Thailand and Vietnam. CYP9J32 along with CYP9J24, CYP9J26, and  <a href="http://en.wikipedia.org/wiki/CYP9J28">CYP9J28</a>  are the four enzymes for which the role in degrading permethrin and deltamethrin was demonstrated "    </li>
   <li>fter 24h observation. Among nine of the differentially expressed genes in this population, three (  <a href="http://en.wikipedia.org/wiki/CYP6BB2">CYP6BB2</a> , CYP9M6, CYP9J28) have been proved to degrade pyrethroids. We can assume that those "    </li>
   <li>tion. Among nine of the differentially expressed genes in this population, three ( CYP6BB2,  <a href="http://en.wikipedia.org/wiki/CYP9M6">CYP9M6</a> , CYP9J28) have been proved to degrade pyrethroids. We can assume that those particular gene"    </li>
   <li>e of the differentially expressed genes in this population, three ( CYP6BB2, CYP9M6,  <a href="http://en.wikipedia.org/wiki/CYP9J28">CYP9J28</a> ) have been proved to degrade pyrethroids. We can assume that those particular genes play a key rol"   </li>
  </ul>
 </body>
 <body>
  <ul>
   <li>, Tanzania and Uganda for two nuclear genes apolipophorin 2 ( apoLp2) and cytochrome p450 (  <a href="http://en.wikipedia.org/wiki/CYPJ92">CYPJ92</a> ), the ribosomal internal transcribed spacer region ( ITS) and the mitochondrial cytochrome "    </li>
   <li>oLp2) and cytochrome p450 ( CYPJ92), the ribosomal internal transcribed spacer region (  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a> ) and the mitochondrial cytochrome c oxidase ( COI) barcoding region. Nuclear genes apoLp"    </li>
   <li>omal internal transcribed spacer region ( ITS) and the mitochondrial cytochrome c oxidase (  <a href="http://en.wikipedia.org/wiki/COI">COI</a> ) barcoding region. Nuclear genes apoLp2 and CYPJ92 were unable to differentiate betw"    </li>
   <li>mitochondrial cytochrome c oxidase ( COI) barcoding region. Nuclear genes apoLp2 and  <a href="http://en.wikipedia.org/wiki/CYPJ92">CYPJ92</a>  were unable to differentiate between species Ae. bromeliae and Ae. lilii due to ance"    </li>
   <li> between species Ae. bromeliae and Ae. lilii due to ancestral lineage sorting, while  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  sequence data provided clear topological separation on a phylogeny. The standard COI barcod"    </li>
   <li> while ITS sequence data provided clear topological separation on a phylogeny. The standard  <a href="http://en.wikipedia.org/wiki/COI">COI</a>  barcoding region was shown to be subject to species introgression and unable to clearly distinguis"    </li>
   <li> isomorphic, sympatric and non-biomedically important sister taxon, Ae. lilii, based on the  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  region. Using molecular species verification, we describe novel immature habitats for Ae. lilii"    </li>
   <li>ey perform at distinguishing these species. We found that the mitochondrial cytochrome c oxidase (  <a href="http://en.wikipedia.org/wiki/COI">COI</a> ) barcoding region and nuclear apolipophorin 2 ( apoLp2) and cytochrome p450 ( CYPJ92"    </li>
   <li> ( COI) barcoding region and nuclear apolipophorin 2 ( apoLp2) and cytochrome p450 (  <a href="http://en.wikipedia.org/wiki/CYPJ92">CYPJ92</a> ) were unable to reliably distinguish these species. Conversely, genetic variation at the internal "    </li>
   <li>stinguish these species. Conversely, genetic variation at the internal transcribed spacer region (  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a> ) was able to confirm the vector Ae. bromeliae and non-vector Ae. lilii as distinct s"    </li>
   <li>firm the vector Ae. bromeliae and non-vector Ae. lilii as distinct species. Based on  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  sequence differences, we developed a robust molecular method to identify the vector Ae. bromeli"    </li>
   <li>pophilic populations from Uganda and Nigeria based on the non-coding internal transcribed spacer (  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a> ) region of ribosomal DNA. In that study, species were designated according to both blood feeding p"    </li>
   <li>as they used different molecular markers; two nuclear genes, apolipophorin 2 and cytochrome p450 (  <a href="http://en.wikipedia.org/wiki/CYPJ92">CYPJ92</a> ). "    </li>
   <li>A 658 bp region of the cytochrome c oxidase (  <a href="http://en.wikipedia.org/wiki/COI">COI</a> ) gene has been widely adopted as a DNA barcoding standard for species identification because it sh"    </li>
   <li>4250.ref034&quot;&gt;34 ]. Another barcoding candidate is the nuclear internal transcribed spacer (  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a> ) regions of the ribosomal gene cistron that comprises the 18S, 5.8S, and 28S genes, an external sp"    </li>
   <li> comprises the 18S, 5.8S, and 28S genes, an external spacer region and two internal spacer regions  <a href="http://en.wikipedia.org/wiki/ITS1">ITS1</a>  and ITS2 [ 35 ]. The ITS regions evolve"    </li>
   <li>8S, 5.8S, and 28S genes, an external spacer region and two internal spacer regions ITS1 and  <a href="http://en.wikipedia.org/wiki/ITS2">ITS2</a>  [ 35 ]. The ITS regions evolve at a rapid rate"    </li>
   <li>pacer regions ITS1 and ITS2 [ 35 ]. The  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  regions evolve at a rapid rate in the absence of functional constraint [ &lt;a href=&quot;#pntd.000"    </li>
   <li> 37 ]. Because they are tandemly repeated in the genome, the  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  spacer regions are also subject to concerted evolution whereby paralogues are homogenised by genet"    </li>
   <li> [ 37 ]. Although not useful in all taxonomic groups,  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  can differentiate between sister species in a large number of cases and has been widely used to de"    </li>
   <li>this by sequencing the same mosquito samples at previously used molecular markers ( apoLp2,  <a href="http://en.wikipedia.org/wiki/CYPJ92">CYPJ92</a>  and ITS). In addition, we were able to evaluate the utility of various molecular markers, i"    </li>
   <li> the same mosquito samples at previously used molecular markers ( apoLp2, CYPJ92 and  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a> ). In addition, we were able to evaluate the utility of various molecular markers, including the &lt;i"    </li>
   <li>i&gt;). In addition, we were able to evaluate the utility of various molecular markers, including the  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  and COI regions, in determining species bounds. We apply these findings to develop a molecu"    </li>
   <li>n, we were able to evaluate the utility of various molecular markers, including the ITS and  <a href="http://en.wikipedia.org/wiki/COI">COI</a>  regions, in determining species bounds. We apply these findings to develop a molecular identificat"    </li>
   <li>ds. We apply these findings to develop a molecular identification method based on variation in the  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  region to distinguish the disease vector Ae. bromeliae from its non-vector sister species &lt;"    </li>
   <li>individuals of the Simpsoni Complex were amplified and sequenced for a region of the mitochondrial  <a href="http://en.wikipedia.org/wiki/COI">COI</a>  gene with universal primers LCO1490 (5’GGTCAACAAATCATAAAGATATTGG’3) and HCO2198 (5’TAAACTTCAGGGTGA"    </li>
   <li>individuals from Tanzania that were selected to represent the genetic diversity we observed at the  <a href="http://en.wikipedia.org/wiki/COI">COI</a>  gene. This was achieved with the 18SFHIN and CP16 primers (5’-GTAAGCTTCCTTTGTACACACCGCCCGT-3’ and "    </li>
   <li>Thirteen Individuals, including eight of the same individuals sequenced at the  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  region, were also sequenced for regions of the nuclear genes apolipophorin 2 ( apoLp2) and "    </li>
   <li> 28 ]. Another marker, short-chain dehydrogenase-reductase (  <a href="http://en.wikipedia.org/wiki/SDR">SDR</a> ) also used by Walter et al. [ 28 ] in the Simp"    </li>
   <li>  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  products were cloned using the P-GEM cloning kit (Promega, UK) as per instructions. Transformants "    </li>
   <li>GAAACAGCTATGAC-3’) [ 48 ] were used to amplify cloned  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  products in the following 13μl reaction; 1.25 units of BIOTAQ DNA Polymerase (BioLine, UK), 1 X NH"    </li>
   <li>Cloned  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  sequences were aligned with Mukwaya et al.’s [ 27 &lt;/s"    </li>
   <li>g was a feature of all primers tested which we suspect may result from variable indel length among  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  copies within an individual. Although PCR protocols were optimised to reduce non-specific banding,"    </li>
   <li>ith Geneious v5.4.7 [ 49 ]. For the apoLp2 and  <a href="http://en.wikipedia.org/wiki/CYPJ92">CYPJ92</a>  datasets, files were prepared using seqPHASE [ 50 ] a"    </li>
   <li>7 ] were used as outgroups in conjunction with data generated during the present study. The  <a href="http://en.wikipedia.org/wiki/COI">COI</a>  dataset was aligned with two COI sequences of Ae. bromeliae from the Indian Ocean is"    </li>
   <li>junction with data generated during the present study. The COI dataset was aligned with two  <a href="http://en.wikipedia.org/wiki/COI">COI</a>  sequences of Ae. bromeliae from the Indian Ocean island of Mayotte available from GenBank ("    </li>
   <li>4 ]. The Tamura-Nei model with uniform rates among sites was used to construct NJ trees for  <a href="http://en.wikipedia.org/wiki/COI">COI</a> . The Kimura 2 parameter model (K80) with uniform rates among sites was used for ITS sequenc"    </li>
   <li>ees for COI. The Kimura 2 parameter model (K80) with uniform rates among sites was used for  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  sequences and K80 with 0.8 gamma-distributed sites was used to construct trees for apoLp2 a"    </li>
   <li>quences and K80 with 0.8 gamma-distributed sites was used to construct trees for apoLp2 and  <a href="http://en.wikipedia.org/wiki/CYPJ92">CYPJ92</a> . For all genetic markers, missing data including indels were excluded from analysis. Topological s"    </li>
   <li>xcluded from analysis. Topological support was determined through 1000 bootstrap replications. The  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  sequence tree was constructed without an outgroup because the high level of divergence between the"    </li>
   <li>ergence between the outgroup and the ingroups presents a challenge for sequence alignment. The two  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  sequences of Ae. bromeliae from Mayotte [ 43 ]"    </li>
   <li>A hierarchical AMOVA was performed on the  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  sequence data generated in this study together with Mukwaya et al.’s sequences [ &lt;a "    </li>
   <li>250.ref055&quot;&gt;55 ]. A hierarchical AMOVA was also performed on the apoLp2 (n = 13) and  <a href="http://en.wikipedia.org/wiki/CYPJ92">CYPJ92</a>  (n = 13) sequences generated in this study. "    </li>
   <li>utions and eight indels of varying length were observed between species in the first 300 bp of the  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  sequence alignment. The neighbour joining tree of ITS sequences revealed two major clades w"    </li>
   <li>en species in the first 300 bp of the ITS sequence alignment. The neighbour joining tree of  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  sequences revealed two major clades with a bootstrap support of 98% ( Fig 2). As reported previous"    </li>
   <li> with reference to Mukwaya et al.’s [ 27 ]  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a> -based species designation. "    </li>
   <li>itoes collected from Kanyawara, Bundibugyo (Bwamba region) and Najjembe in Uganda represented both  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  clades. In comparison, individuals collected from Tanzania strictly aggregated into the ITS"    </li>
   <li>ITS clades. In comparison, individuals collected from Tanzania strictly aggregated into the  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  lineage of Ae. bromeliae. AMOVA revealed that almost all sequence differences can be explai"    </li>
   <li>Sequencing of individuals at both  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  and nuclear genes apoLp2 and CYPJ92 allowed us to compare the ITS-based speci"    </li>
   <li>Sequencing of individuals at both ITS and nuclear genes apoLp2 and  <a href="http://en.wikipedia.org/wiki/CYPJ92">CYPJ92</a>  allowed us to compare the ITS-based species designation of Mukwaya et al. [ &lt;"    </li>
   <li>als at both ITS and nuclear genes apoLp2 and CYPJ92 allowed us to compare the  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a> -based species designation of Mukwaya et al. [ 27 &lt;/su"    </li>
   <li>&gt; ]. Two well-supported lineages were observed in NJ trees for the nuclear genes apoLp2 and  <a href="http://en.wikipedia.org/wiki/CYPJ92">CYPJ92</a>  with 100% bootstrap support (Figs 3 and 4). Individuals confirmed as Ae. bromeliae or Ae"    </li>
   <li>upport (Figs 3 and 4). Individuals confirmed as Ae. bromeliae or Ae. lilii using the  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  region with reference to Mukwaya et al. [ 27 ]"    </li>
   <li>ref027&quot;&gt;27 ], all fall into a single clade for each of the nuclear genes, apoLp2 and  <a href="http://en.wikipedia.org/wiki/CYPJ92">CYPJ92</a> . For both gene trees, the other lineage comprises only the sequences from the forest species in Wa"    </li>
   <li>eferred to as Ae. lilii. A hierarchical AMOVA revealed that nuclear genes apoLp2 and  <a href="http://en.wikipedia.org/wiki/CYPJ92">CYPJ92</a>  could not effectively distinguish between Ae. bromeliae and Ae. lilii with only 0.86"    </li>
   <li>port (90%, 90%) and two with weak bootstrap support (56% and 43%) in the NJ tree for mitochondrial  <a href="http://en.wikipedia.org/wiki/COI">COI</a>  with individuals tending to cluster according to geographic origin ( Fig 5). The sequences from Ma"    </li>
   <li>/i&gt; from Tanzania. Individuals designated as Ae. bromeliae or Ae. lilii according to  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  sequence variation tend to group according to species within the same clades, but there are three "    </li>
   <li>within the same clades, but there are three exceptions. One individual from Uganda, exhibiting the  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  sequence of Ae. bromeliae clusters within a grouping of Ae. lilii while two individu"    </li>
   <li>tic variation within (42.83%) and between populations (55.18%) than between species (1.99%) at the  <a href="http://en.wikipedia.org/wiki/COI">COI</a>  gene. "    </li>
   <li>Based on fixed differences between species in the  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  sequences, species specific primers were designed to amplify PCR products in Ae. bromeliae "    </li>
   <li>hod for the YFV vector Ae. bromeliae and non-vector Ae. lilii using variation at the  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  region allowed us to make for the first time reliable inferences about mosquito ecology and distri"    </li>
   <li>the Ae. simpsoni s.l. mosquitoes tested fell into one or other of two genetically divergent  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  lineages. Based on sequence similarity these correspond directly to the two lineages found by Mukw"    </li>
   <li>rm the existence of Ae. bromeliae and Ae. lilii as two distinct species since, using  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  sequence variation, they remain genetically distinct in sympatry at several locations in Uganda. "    </li>
   <li> Kenya at three nuclear genes. Our phylogenetic analysis reveals that the genetic variation in the  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  region does not correspond to the two distinct lineages found in Walter et al. [ &lt;a "    </li>
   <li>lii fall into a single phylogenetic clade at nuclear genes apoLp2 (apolipophorin 2) and  <a href="http://en.wikipedia.org/wiki/CYPJ92">CYPJ92</a>  (cytochrome p450). This is consistent with these more slowly evolving nuclear genes being unable t"    </li>
   <li>250.ref056&quot;&gt;56 ]. The forest lineage from Kenya detected by nuclear genes apoLp2 and  <a href="http://en.wikipedia.org/wiki/CYPJ92">CYPJ92</a>  is clearly distinct and has no shared ancestral polymorphism with either Ae. bromeliae or &lt;"    </li>
   <li>The mitochondrial  <a href="http://en.wikipedia.org/wiki/COI">COI</a>  species barcode marker was also unable to distinguish these taxa with most genetic variation occur"    </li>
   <li>ion occurring between geographic populations rather than between species. Whilst this inability of  <a href="http://en.wikipedia.org/wiki/COI">COI</a>  to distinguish Ae. bromeliae and Ae. lilii could be due to incomplete lineage sortin"    </li>
   <li>uish species in this genus without prior confirmation from an additional marker. We found that the  <a href="http://en.wikipedia.org/wiki/ITS">ITS</a>  region was the only marker able to reliably separate species of the Simpsoni Complex and therefore"   </li>
  </ul>
 </body>
 <body>
  <ul>
   <li>the efficiency of virus replication in the salivary gland. This study has shown that modulation of  <a href="http://en.wikipedia.org/wiki/OBP10">OBP10</a>  and OBP22 genes expression as well as DENV infection-responsive odorant-binding protein gen"    </li>
   <li> virus replication in the salivary gland. This study has shown that modulation of OBP10 and  <a href="http://en.wikipedia.org/wiki/OBP22">OBP22</a>  genes expression as well as DENV infection-responsive odorant-binding protein genes increase the t"    </li>
   <li>vector’s genome. Single point mutation in the voltage-gated sodium channel gene at position 1534 (  <a href="http://en.wikipedia.org/wiki/F1534C">F1534C</a> ) resulting in phenylalanine to cysteine substitution in Ae. aegypti confers resistance to p"    </li>
   <li>ease [ 45 – 47 ]. Polymorphisms of the  <a href="http://en.wikipedia.org/wiki/TAP1">TAP1</a>  and TAP2 genes could be directly associated with the risk of developing dengue disease amon"  </li>
   <li> href=&quot;#CR45&quot;&gt;45 – 47 ]. Polymorphisms of the TAP1 and  <a href="http://en.wikipedia.org/wiki/TAP2">TAP2</a>  genes could be directly associated with the risk of developing dengue disease among the primary-in"  </li>
   <li>g dengue disease among the primary-infected individuals [ 48 ]. Both  <a href="http://en.wikipedia.org/wiki/TAP1">TAP1</a>  and TAP2 are located within the MHC class II region and homozygosity of the TAP1&lt;"  </li>
   <li> among the primary-infected individuals [ 48 ]. Both TAP1 and  <a href="http://en.wikipedia.org/wiki/TAP2">TAP2</a>  are located within the MHC class II region and homozygosity of the TAP1 at position "  </li>
   <li>TAP1 and TAP2 are located within the MHC class II region and homozygosity of the  <a href="http://en.wikipedia.org/wiki/TAP1">TAP1</a>  at position 1333 and 1637 and for TAP2 at position 2379, respectively, was found to protect"  </li>
   <li>e MHC class II region and homozygosity of the TAP1 at position 1333 and 1637 and for  <a href="http://en.wikipedia.org/wiki/TAP2">TAP2</a>  at position 2379, respectively, was found to protect against developing severe forms of dengue [ "  </li>
   <li>-alpha, FcGRIIA, vitamin D receptor, transporters associated with antigen presentation, and  <a href="http://en.wikipedia.org/wiki/JAK1">JAK1</a>  has also been proposed [ 50 ]. The importance of Vit-D in DEN"  </li>
   <li>infection against 2018 population control cases in Vietnam. The data showed that SNPs at two loci,  <a href="http://en.wikipedia.org/wiki/MICB">MICB</a>  and PLCE1, significantly increased the likelihood of developing DSS in children. This findi"  </li>
   <li>t 2018 population control cases in Vietnam. The data showed that SNPs at two loci, MICB and  <a href="http://en.wikipedia.org/wiki/PLCE1">PLCE1</a> , significantly increased the likelihood of developing DSS in children. This finding was further va"  </li>
   <li>ependent cohort of 1737 cases and 2934 controls [ 53 ]. A SNP in the  <a href="http://en.wikipedia.org/wiki/MICB">MICB</a>  gene coding for the MHC class I polypeptide-related sequence B, an inducible activating ligand for"  </li>
   <li>ection [ 54 , 55 ]. On the other hand,  <a href="http://en.wikipedia.org/wiki/PLCE1">PLCE1</a>  plays a primordial role in maintaining intact vascular endothelial cell barrier function, hence, p"  </li>
   <li> role in maintaining intact vascular endothelial cell barrier function, hence, polymorphism of the  <a href="http://en.wikipedia.org/wiki/PLCE1">PLCE1</a>  gene may lead to blood vessels leakage and circulatory hypovolemia during DSS [ &lt;a href=&quot;#C"  </li>
   <li>neration approach is being used to develop a vaccine containing recombinant subunit E domain III (  <a href="http://en.wikipedia.org/wiki/ED3">ED3</a> ) and the results of laboratory tests have shown the development of potent neutralizing antibodies "   </li>
  </ul>
 </body>
</html>
