<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <div xmlns="" tagx="article">
      <head>
         <style type="text/css"> table, tr, td {border : thin;}</style>
         <span class="citation_publisher">BioMed Central</span>
         <h2 class="citation-title">Development of one-step quantitative reverse transcription PCR for the rapid detection
            of flaviviruses
         </h2>
         <span class="doi">10.1186/1743-422X-10-58</span>
         <div class="contrib-group">contrib: 
            	  CONT
            	  
            <span class="citation_author">Pranav Patel</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Olfert Landt</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Marco Kaiser</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Oumar Faye</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Tanja Koppe</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Ulrich Lass</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Amadou A Sall</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Matthias Niedrig</span>
            <span class="citation_author_institution_ref"></span>
         </div>
         <span class="email">[]</span>
         <span class="conflict">[]</span>
         <span class="contribs">[]</span>
         <span class="citation_pubdate">[2013-2-14]</span>
         <span class="citation_copyright">[Copyright 2013: Patel et al.; licensee BioMed Central Ltd.]</span>
         <span class="citation_licence">[Licence This is an Open Access article distributed under the terms of the Creative
            Commons Attribution License (http://creativecommons.org/licenses/by/2.0), which permits
            unrestricted use, distribution, and reproduction in any medium, provided the original
            work is properly cited.]
         </span>
         <span class="citation_funding">[]</span>
         <span class="citation_pagecount">[]</span>
         <span class="citation_data">[]</span>
      </head>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <div id="">
            <h2>Background</h2>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The genus 
               <i>Flavivirus</i> includes several pathogenic agents that cause severe illness in humans. Re-emergence
               of West Nile virus in Europe and continuous spread of certain flaviviruses such as
               dengue, yellow fever and Japanese encephalitis viruses represent a global danger to
               public health. Therefore, a rapid and accurate molecular method is required for diagnostics
               and epidemiological surveillance of flaviviruses.
            </p>
         </div>
         <div id="">
            <h2>Methods</h2>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A Pan-Flavi quantitative RT-PCR assay using a Locked-Nucleic Acid probe targeting
               the flavivirus NS5 gene was developed and optimized to detect a wide range of flaviviruses
               simultaneously. The specificity and sensitivity of the Pan-Flavi assay were tested
               using RNA of different flaviviruses and non-flaviviruses. Furthermore, the assay was
               compared directly to flavivirus species-specific assays for the ability to detect
               flaviviruses sensitively.
            </p>
         </div>
         <div id="">
            <h2>Results</h2>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Two degenerate primers and one Locked-Nucleic Acids probe were designed to amplify
               most of the flaviviruses. To increase the specificity and fluorescence signal of the
               Pan-Flavi assay for detection of yellow fever virus and dengue virus 4, additional
               primers and probes were included. Viral RNA of thirty different flaviviruses was detected,
               verifying the broad range specificity. The testing of this assay was successful, using
               standard plasmid and RNA dilutions of yellow fever virus vaccine strain, dengue virus
               1 and tick-borne encephalitis virus, with a sensitivity limit of 10–100 genome copies/reaction.
               Also comparatively good results were achieved for detecting different flaviviruses
               by the Pan-Flavi assay when compared to the flavivirus species-specific assays.
            </p>
         </div>
         <div id="">
            <h2>Conclusion</h2>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The assay is rapid, broad-range flavivirus-specific and highly sensitive making it
               a valuable tool for rapid detection of flaviviruses in livestock samples, epidemiological
               studies or as useful complement to single flavivirus-specific assays for clinical
               diagnosis.
            </p>
         </div>
      </div>
      <div id="">
         <h2>Background</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The genus 
            <i>Flavivirus</i> of the family 
            <i>Flaviviridae</i> consists of more than 70 virus species including many arthropod-borne viruses. It
            contains highly pathogenic agents such as the name-giving member yellow fever virus
            (YFV), West Nile virus (WNV), Japanese encephalitis virus (JEV), tick-borne encephalitis
            virus (TBEV) and dengue virus (DENV), which can cause encephalitis or hemorrhagic
            fever and pose some of the most challenging emerging diseases in humans [
            <span ref-type="bibr" rid="B1">
               <sup>
                  <a href="#B1">1</a>
               </sup>
            </span>].
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Flaviviruses comprise three epidemiologically distinct groups; the mosquito-borne
            group, the tick-borne group and the unknown vector viruses [
            <span ref-type="bibr" rid="B2">
               <sup>
                  <a href="#B2">2</a>
               </sup>
            </span>]. They are enveloped positive-stranded RNA viruses with a genome of approximately
            11 kb. The viral genome encodes a large single polyprotein from which three structural
            proteins, Capsid (C), Envelope (E) and Membrane (M) and seven non-structural proteins,
            NS1, NS2A, NS2B, NS3, NS4A, NS4B and NS5, are produced [
            <span ref-type="bibr" rid="B3">
               <sup>
                  <a href="#B3">3</a>
               </sup>
            </span>].
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Flaviviruses pose a threat to public health worldwide, especially in developing countries.
            In recent years, there has been a dramatic increase in cases of DENV infections globally.
            The disease is endemic in more than 100 countries in Africa, the Americas, the Eastern
            Mediterranean, South-east Asia, and the Western Pacific. About 50–100 million cases
            of DENV infection are reported worldwide every year, resulting in 500,000 cases of
            hemorrhagic fever requiring hospitalization and 25,000 fatalities [
            <span ref-type="bibr" rid="B4">
               <sup>
                  <a href="#B4">4</a>
               </sup>
            </span>]. Yellow Fever is endemic in 45 countries throughout Africa and Latin America, with
            approximately 200,000 cases of human infections worldwide, and 30,000 mortalities
            [
            <span ref-type="bibr" rid="B5">
               <sup>
                  <a href="#B5">5</a>
               </sup>
            </span>]. WNV is one of the most widespread flaviviruses. WNV lineage 1 is endemic in the
            Americas, India, and Australia. Both lineages 1 and 2 are found in Africa and Europe
            [
            <span ref-type="bibr" rid="B6">
               <sup>
                  <a href="#B6">6</a>
               </sup>
            </span>-
            <span ref-type="bibr" rid="B8">
               <sup>
                  <a href="#B8">8</a>
               </sup>
            </span>]. The recent WNV outbreak reported in August 2010 in Greece was the first large outbreak
            in humans in Europe since the Romanian outbreak in 1996–1997 [
            <span ref-type="bibr" rid="B9">
               <sup>
                  <a href="#B9">9</a>
               </sup>
            </span>]. JEV is the leading cause of viral encephalitis in Asia with 30,000–50,000 clinical
            cases reported annually and 10,000 deaths [
            <span ref-type="bibr" rid="B10">
               <sup>
                  <a href="#B10">10</a>
               </sup>
            </span>]. TBEV, virus-causing infection of the central nervous system, has spread extensively
            in some European regions. According to ECDC [
            <span ref-type="bibr" rid="B11">
               <sup>
                  <a href="#B11">11</a>
               </sup>
            </span>], human cases of TEBV infections have increased in the past 30 years dramatically
            posing a danger to public health in European countries.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A number of flaviviruses have repeatedly demonstrated their ability to expand their
            geographical range. Some current examples are the introduction of Usutu virus and
            Bagaza virus into central Europe [
            <span ref-type="bibr" rid="B12">
               <sup>
                  <a href="#B12">12</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="B13">
               <sup>
                  <a href="#B13">13</a>
               </sup>
            </span>], or Zika virus into Yap island and Cambodia [
            <span ref-type="bibr" rid="B14">
               <sup>
                  <a href="#B14">14</a>
               </sup>
            </span>]. In recent years, also a high number of mosquito-only flaviviruses including 
            <i>Culex</i> and 
            <i>Aedes</i> flaviviruses were discovered throughout the world [
            <span ref-type="bibr" rid="B15">
               <sup>
                  <a href="#B15">15</a>
               </sup>
            </span>-
            <span ref-type="bibr" rid="B18">
               <sup>
                  <a href="#B18">18</a>
               </sup>
            </span>]. Apart from the human pathogenic and mosquito-only flaviviruses, several novel flaviviruses
            were latterly discovered: Nounane in Cote d’Ivoire [
            <span ref-type="bibr" rid="B19">
               <sup>
                  <a href="#B19">19</a>
               </sup>
            </span>], T’Ho virus in Mexico [
            <span ref-type="bibr" rid="B20">
               <sup>
                  <a href="#B20">20</a>
               </sup>
            </span>], Lammi virus in Finnland [
            <span ref-type="bibr" rid="B21">
               <sup>
                  <a href="#B21">21</a>
               </sup>
            </span>], and Duck tembusu virus in China [
            <span ref-type="bibr" rid="B22">
               <sup>
                  <a href="#B22">22</a>
               </sup>
            </span>]. In contrast to mosquito-only flaviviruses, these novel flaviviruses were shown
            to be phylogenetically related to human pathogenic flaviviruses.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The clinical diagnosis of flavivirus infections is not unambiguous due to unspecific
            symptoms varying from mild, febrile illness to viral hemorrhagic fever. Many of these
            viruses have a common transmission vector and can co-circulate in the same area. All
            these factors make a precise identification of the pathogen difficult.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Today most diagnostic laboratories use serological assays for flavivirus testing.
            These tests are commonly based on the enzyme-linked immunosorbent assay (ELISA) and
            detect virus-specific IgM and IgG antibodies. Antibodies are undetectable prior to
            5–7 days after onset of infection which hampers the usefulness of the serological
            methods. Molecular techniques, in contrast, can be used already in the acute phase
            and are known to be rapid, highly specific and sensitive.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Since the early 1990s, several group-specific and generic molecular assays for detection
            of flaviviruses have been developed as reviewed previously [
            <span ref-type="bibr" rid="B23">
               <sup>
                  <a href="#B23">23</a>
               </sup>
            </span>-
            <span ref-type="bibr" rid="B25">
               <sup>
                  <a href="#B25">25</a>
               </sup>
            </span>]. A number of attempts to detect several flaviviruses in a single reverse transcription-polymerase
            chain reaction (RT-PCR) assay have been made [
            <span ref-type="bibr" rid="B26">
               <sup>
                  <a href="#B26">26</a>
               </sup>
            </span>]. These assays vary in the selected target region, assay format and detection method.
            The highly conserved NS3 and NS5 genes have been used predominantly for flavivirus
            testing, mainly as nested, hemi-nested RT-PCR or SYBR green real-time PCR, without
            using a probe for sequence verification.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The aim of this study was to develop a rapid, sensitive and reliable TaqMan probe-based
            quantitative RT-PCR (qRT-PCR) assay for simultaneous detection of several flaviviruses,
            using the conserved NS5 gene region. The introduction of Locked-Nucleic Acid (LNA)
            bases in the probe increases robustness, specificity and sensitivity of the assay
            and has been shown to allow quantification of the viral load.
         </p>
      </div>
      <div id="">
         <h2>Results</h2>
         <div id="">
            <h3>Design and optimization of Pan-Flavi assay</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Primers from publication [
               <span ref-type="bibr" rid="B26">
                  <sup>
                     <a href="#B26">26</a>
                  </sup>
               </span>-
               <span ref-type="bibr" rid="B28">
                  <sup>
                     <a href="#B28">28</a>
                  </sup>
               </span>] were aligned against genomic sequences of all flaviviruses deposited in the NCBI
               database to verify their flavivirus-generic amplification capacity. Degenerate primers
               were selected based on these published sequences, but modified in order to increase
               the spectrum of flaviviruses included. As alternative approach individual specific
               flavivirus species-specific primers were combined to avoid unspecificity due to base
               combinations contained in the degenerate primers not matching exisiting viral sequences.
               In particular we used 17 forward and 15 reverse primers and another four reverse primers
               targeting different locations. Testing of the primer mix with viral RNA revealed that
               the performance of the primer mixes was not improved compared to the results obtained
               with the degenerate primers. Also for the probes we tested 22 individual flavivirus-specific
               sequences covering all known sequence variants at the binding site and established
               mixtures yielding comparable threshold cycle (Ct) values for all virus species tested,
               but the signal level was significant lower than for degenerate LNA-probes (data not
               shown). In conclusion, neither the set of individual primers nor the set of probes
               improved the detection limit nor the range of viruses detected and we continued the
               evaluation with the degenerate set of primers combined with one of the three LNA-probes.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In total, six combinations assembled from two degenerate primer sets (Flavi all S
               &amp; Flavi all AS1 and Flavi all S &amp; Flavi all AS2) and three probes (Flavi all probe
               1, Flavi all probe 2 and Flavi all probe 3) were tested using RNA from flaviviruses
               and other viruses to check the performance of these Pan-Flavi assays. Combinations
               with Flavi all probe 3 showed earlier Ct values (Table 
               <span ref-type="table" rid="T1">
                  <a href="#T1">1</a>
               </span>). Best results among these six combinations were obtained with Flavi all S, Flavi
               all AS2 primers and Flavi all probe 3, but detection of DENV type 4 was insufficient.
               Analysis by gel electrophoresis revealed missing amplification, except a faint band
               for primer combination Flavi all S and AS1, correlating to a low signal and very late
               Ct value (Table 
               <span ref-type="table" rid="T1">
                  <a href="#T1">1</a>
               </span>).
            </p>
            <div id="T1">
               <span class="label">Table 1</span>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Comparison of designed two degenerate primer sets and three probes containing wobble
                     and LNA bases in six combinations for detection of the different flavivirus RNA
                  </p>
               </caption>
               <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <colgroup>
                     <col></col>
                     <col></col>
                     <col></col>
                     <col></col>
                     <col></col>
                     <col></col>
                     <col></col>
                     <col></col>
                  </colgroup>
                  <thead>
                     <tr>
                        <th>
                           <b>
                              <i>Sample No</i>
                           </b>
                        </th>
                        <th>
                           <b>
                              <i>Virus</i>
                           </b>
                        </th>
                        <th>
                           <b>
                              <i>Ct value obtained through testing the combination of Flavi all degenerate primers
                                 and probe
                              </i>
                           </b>
                           <hr></hr>
                        </th>
                     </tr>
                     <tr>
                        <th>
                           <b>S/AS1/probe1</b>
                        </th>
                        <th>
                           <b>S/AS1/probe2</b>
                        </th>
                        <th>
                           <b>S/AS1/probe3</b>
                        </th>
                        <th>
                           <b>S/AS2/probe1</b>
                        </th>
                        <th>
                           <b>S/AS2/probe2</b>
                        </th>
                        <th>
                           <b>S/AS2/probe3</b>
                        </th>
                     </tr>
                  </thead>
                  <div tagxxx="tbody">
                     <tr>
                        <td>
                           <b>1</b>
                           <hr></hr>
                        </td>
                        <td>DENV-1
                           <hr></hr>
                        </td>
                        <td>31.50
                           <hr></hr>
                        </td>
                        <td>17.57
                           <hr></hr>
                        </td>
                        <td>15.66
                           <hr></hr>
                        </td>
                        <td>27.95
                           <hr></hr>
                        </td>
                        <td>17.84
                           <hr></hr>
                        </td>
                        <td>15.61
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>2</b>
                           <hr></hr>
                        </td>
                        <td>DENV-2
                           <hr></hr>
                        </td>
                        <td>22.69
                           <hr></hr>
                        </td>
                        <td>22.21
                           <hr></hr>
                        </td>
                        <td>19.76
                           <hr></hr>
                        </td>
                        <td>22.27
                           <hr></hr>
                        </td>
                        <td>22.19
                           <hr></hr>
                        </td>
                        <td>19.92
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>3</b>
                           <hr></hr>
                        </td>
                        <td>DENV-3
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>21.25
                           <hr></hr>
                        </td>
                        <td>19.14
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>20.63
                           <hr></hr>
                        </td>
                        <td>18.57
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>4</b>
                           <hr></hr>
                        </td>
                        <td>DENV-4
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>35.50
                           <hr></hr>
                        </td>
                        <td>35.48
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>5</b>
                           <hr></hr>
                        </td>
                        <td>JEV
                           <hr></hr>
                        </td>
                        <td>20.40
                           <hr></hr>
                        </td>
                        <td>21.63
                           <hr></hr>
                        </td>
                        <td>20.93
                           <hr></hr>
                        </td>
                        <td>18.99
                           <hr></hr>
                        </td>
                        <td>19.98
                           <hr></hr>
                        </td>
                        <td>19.27
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>6</b>
                           <hr></hr>
                        </td>
                        <td>RSSEV
                           <hr></hr>
                        </td>
                        <td>27.69
                           <hr></hr>
                        </td>
                        <td>29.38
                           <hr></hr>
                        </td>
                        <td>27.31
                           <hr></hr>
                        </td>
                        <td>24.67
                           <hr></hr>
                        </td>
                        <td>26.13
                           <hr></hr>
                        </td>
                        <td>23.81
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>7</b>
                           <hr></hr>
                        </td>
                        <td>TBEV K23
                           <hr></hr>
                        </td>
                        <td>17.68
                           <hr></hr>
                        </td>
                        <td>21.81
                           <hr></hr>
                        </td>
                        <td>16.99
                           <hr></hr>
                        </td>
                        <td>16.73
                           <hr></hr>
                        </td>
                        <td>18.63
                           <hr></hr>
                        </td>
                        <td>16.32
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>8</b>
                           <hr></hr>
                        </td>
                        <td>TBEV Louping ill
                           <hr></hr>
                        </td>
                        <td>26.00
                           <hr></hr>
                        </td>
                        <td>28.75
                           <hr></hr>
                        </td>
                        <td>23.78
                           <hr></hr>
                        </td>
                        <td>22.13
                           <hr></hr>
                        </td>
                        <td>24.8
                           <hr></hr>
                        </td>
                        <td>21.22
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>9</b>
                           <hr></hr>
                        </td>
                        <td>Usutu virus
                           <hr></hr>
                        </td>
                        <td>25.63
                           <hr></hr>
                        </td>
                        <td>33.96
                           <hr></hr>
                        </td>
                        <td>25.91
                           <hr></hr>
                        </td>
                        <td>22.96
                           <hr></hr>
                        </td>
                        <td>30.3
                           <hr></hr>
                        </td>
                        <td>23.48
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>10</b>
                           <hr></hr>
                        </td>
                        <td>WNV Uganda
                           <hr></hr>
                        </td>
                        <td>30.22
                           <hr></hr>
                        </td>
                        <td>30.27
                           <hr></hr>
                        </td>
                        <td>30.68
                           <hr></hr>
                        </td>
                        <td>28.46
                           <hr></hr>
                        </td>
                        <td>28.45
                           <hr></hr>
                        </td>
                        <td>28.26
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>11</b>
                           <hr></hr>
                        </td>
                        <td>WNV Israel
                           <hr></hr>
                        </td>
                        <td>21.76
                           <hr></hr>
                        </td>
                        <td>21.83
                           <hr></hr>
                        </td>
                        <td>20.78
                           <hr></hr>
                        </td>
                        <td>20.06
                           <hr></hr>
                        </td>
                        <td>21.68
                           <hr></hr>
                        </td>
                        <td>19.8
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>12</b>
                           <hr></hr>
                        </td>
                        <td>SLEV
                           <hr></hr>
                        </td>
                        <td>25.85
                           <hr></hr>
                        </td>
                        <td>26.95
                           <hr></hr>
                        </td>
                        <td>25.57
                           <hr></hr>
                        </td>
                        <td>24.41
                           <hr></hr>
                        </td>
                        <td>24.90
                           <hr></hr>
                        </td>
                        <td>23.49
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>13</b>
                           <hr></hr>
                        </td>
                        <td>YFV 17D
                           <hr></hr>
                        </td>
                        <td>19.82
                           <hr></hr>
                        </td>
                        <td>22.48
                           <hr></hr>
                        </td>
                        <td>18.41
                           <hr></hr>
                        </td>
                        <td>20.03
                           <hr></hr>
                        </td>
                        <td>22.12
                           <hr></hr>
                        </td>
                        <td>18.92
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>14</b>
                           <hr></hr>
                        </td>
                        <td>YFV Asibi
                           <hr></hr>
                        </td>
                        <td>23.71
                           <hr></hr>
                        </td>
                        <td>26.98
                           <hr></hr>
                        </td>
                        <td>22.82
                           <hr></hr>
                        </td>
                        <td>24.17
                           <hr></hr>
                        </td>
                        <td>26.83
                           <hr></hr>
                        </td>
                        <td>23.53
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>15</b>
                           <hr></hr>
                        </td>
                        <td>YFV Brasil
                           <hr></hr>
                        </td>
                        <td>21.35
                           <hr></hr>
                        </td>
                        <td>23.73
                           <hr></hr>
                        </td>
                        <td>20.48
                           <hr></hr>
                        </td>
                        <td>21.75
                           <hr></hr>
                        </td>
                        <td>23.82
                           <hr></hr>
                        </td>
                        <td>21.43
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>16</b>
                           <hr></hr>
                        </td>
                        <td>YFV Ivory Coast
                           <hr></hr>
                        </td>
                        <td>21.35
                           <hr></hr>
                        </td>
                        <td>23.98
                           <hr></hr>
                        </td>
                        <td>20.36
                           <hr></hr>
                        </td>
                        <td>21.47
                           <hr></hr>
                        </td>
                        <td>23.94
                           <hr></hr>
                        </td>
                        <td>21.44
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>18</b>
                           <hr></hr>
                        </td>
                        <td>Chikungunya virus
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>19</b>
                           <hr></hr>
                        </td>
                        <td>RVFV
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>20</b>
                           <hr></hr>
                        </td>
                        <td>Sindbis virus
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>21</b>
                           <hr></hr>
                        </td>
                        <td>H1N1 Influenza A
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>22</b>
                           <hr></hr>
                        </td>
                        <td>H5N1 Influenza A
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                        <td>No Ct
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>17</b>
                        </td>
                        <td>Negative control</td>
                        <td>No Ct</td>
                        <td>No Ct</td>
                        <td>No Ct</td>
                        <td>No Ct</td>
                        <td>No Ct</td>
                        <td>No Ct</td>
                     </tr>
                  </div>
               </table>
               <table-wrap-foot xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>
                     <i>DENV</i>: Dengue virus, 
                     <i>JEV</i>: Japanese encephalitis virus, 
                     <i>RSSEV</i>: Russian spring-summer encephalitis virus, 
                     <i>WNV</i>: West Nile virus, 
                     <i>TBEV</i>: Tick-borne encephalitis virus, 
                     <i>SLEV</i>: Saint Louis encephalitis virus, 
                     <i>RVFV</i>: Rift valley fever virus, 
                     <i>YFV</i>: Yellow fever virus.
                  </p>
                  <p>S: Flavi all S, AS1: Falvi all AS1, AS2: Flavi all AS2, probe1: Flavi all probe 1,
                     probe 2: Flavi all probe 2, probe 3: Flavi all probe 3.
                  </p>
               </table-wrap-foot>
            </div>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To overcome this deficit the combination Flavi all S/AS2/probe 3 was supplemented
               with an additional DEN4 F primer and all tests were repeated. DENV type 4 was now
               detectable with Ct values similar to the specific assay. The additional DEN4 F primer
               had no influence on the detection of all other flaviviruses tested (Table 
               <span ref-type="table" rid="T2">
                  <a href="#T2">2</a>
               </span>). Evaluation of the Pan-Flavi assay was performed using primers Flavi all S/DEN4
               F and Flavi all AS2. Changing primers to Flavi all S/Flavi all S2/ Flavi all AS4 allowed
               to detect also Nounane virus (Table 
               <span ref-type="table" rid="T3">
                  <a href="#T3">3</a>
               </span>), without changing specificity and sensitivity of Pan-Flavi assay for the other viruses
               (data not shown).
            </p>
            <div id="T2">
               <span class="label">Table 2</span>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Representative results of the Pan-Flavi assay to detect different flaviviruses, and
                     its comparison with flavivirus species-specific assays
                  </p>
               </caption>
               <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <colgroup>
                     <col></col>
                     <col></col>
                     <col></col>
                     <col></col>
                     <col></col>
                     <col></col>
                     <col></col>
                     <col></col>
                     <col></col>
                  </colgroup>
                  <thead>
                     <tr>
                        <th>
                           <b>Virus</b>
                        </th>
                        <th>
                           <b>Strain/ Acc. No.</b>
                        </th>
                        <th>
                           <b>Pan-flavi assay*</b>
                           <sup>
                              <b>,#</b>
                           </sup>
                           <hr></hr>
                        </th>
                        <th>
                           <b>Virus-specific assay</b>
                           <sup>
                              <b>#</b>
                           </sup>
                           <hr></hr>
                        </th>
                        <td> </td>
                     </tr>
                     <tr>
                        <th>
                           <b>Mean of Ct value</b>
                        </th>
                        <th>
                           <b>SD of Ct value</b>
                        </th>
                        <th>
                           <b>Mean of GC/ml</b>
                        </th>
                        <th>
                           <b>Mean of Ct value</b>
                        </th>
                        <th>
                           <b>SD of Ct value</b>
                        </th>
                        <th>
                           <b>Mean of GC/ml</b>
                        </th>
                        <th>
                           <b>Reference</b>
                        </th>
                     </tr>
                  </thead>
                  <div tagxxx="tbody">
                     <tr>
                        <td>
                           <b>
                              <i>Flavivirus</i>
                           </b>
                           <hr></hr>
                        </td>
                        <td> </td>
                        <td> </td>
                        <td> </td>
                        <td> </td>
                        <td> </td>
                        <td> </td>
                        <td> </td>
                        <td> </td>
                     </tr>
                     <tr>
                        <td>DENV-1
                           <hr></hr>
                        </td>
                        <td>ATCC VR-344
                           <hr></hr>
                        </td>
                        <td>15.64
                           <hr></hr>
                        </td>
                        <td>0.02
                           <hr></hr>
                        </td>
                        <td>2.0 × 10
                           <sup>9</sup>
                           <hr></hr>
                        </td>
                        <td>15.93
                           <hr></hr>
                        </td>
                        <td>0.03
                           <hr></hr>
                        </td>
                        <td>4.6 × 10
                           <sup>9</sup>
                           <hr></hr>
                        </td>
                        <td>[
                           <span ref-type="bibr" rid="B29">
                              <sup>
                                 <a href="#B29">29</a>
                              </sup>
                           </span>]
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>DENV-2
                           <hr></hr>
                        </td>
                        <td>ATCC VR-345
                           <hr></hr>
                        </td>
                        <td>20.76
                           <hr></hr>
                        </td>
                        <td>0.20
                           <hr></hr>
                        </td>
                        <td>6.6 × 10
                           <sup>7</sup>
                           <hr></hr>
                        </td>
                        <td>18.77
                           <hr></hr>
                        </td>
                        <td>0.11
                           <hr></hr>
                        </td>
                        <td>6.7 × 10
                           <sup>8</sup>
                           <hr></hr>
                        </td>
                        <td> 
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>DENV-3
                           <hr></hr>
                        </td>
                        <td>ATCC VR-1256
                           <hr></hr>
                        </td>
                        <td>19.36
                           <hr></hr>
                        </td>
                        <td>0.01
                           <hr></hr>
                        </td>
                        <td>1.7 × 10
                           <sup>8</sup>
                           <hr></hr>
                        </td>
                        <td>20.30
                           <hr></hr>
                        </td>
                        <td>0.01
                           <hr></hr>
                        </td>
                        <td>2.4 × 10
                           <sup>8</sup>
                           <hr></hr>
                        </td>
                        <td> 
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>DENV-4
                           <hr></hr>
                        </td>
                        <td>ATCC VR-1257
                           <hr></hr>
                        </td>
                        <td>16.33
                           <hr></hr>
                        </td>
                        <td>0.19
                           <hr></hr>
                        </td>
                        <td>1.3 × 10
                           <sup>9</sup>
                           <hr></hr>
                        </td>
                        <td>16.18
                           <hr></hr>
                        </td>
                        <td>0.29
                           <hr></hr>
                        </td>
                        <td>3.9 × 10
                           <sup>9</sup>
                           <hr></hr>
                        </td>
                        <td> 
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>YFV 17D
                           <hr></hr>
                        </td>
                        <td>X03700
                           <hr></hr>
                        </td>
                        <td>19.17
                           <hr></hr>
                        </td>
                        <td>0.17
                           <hr></hr>
                        </td>
                        <td>1.9 × 10
                           <sup>8</sup>
                           <hr></hr>
                        </td>
                        <td>20.51
                           <hr></hr>
                        </td>
                        <td>0.17
                           <hr></hr>
                        </td>
                        <td>5.6 × 10
                           <sup>8</sup>
                           <hr></hr>
                        </td>
                        <td>[
                           <span ref-type="bibr" rid="B30">
                              <sup>
                                 <a href="#B30">30</a>
                              </sup>
                           </span>]
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>YFV ASIBI
                           <hr></hr>
                        </td>
                        <td>RKI reference strain
                           <hr></hr>
                        </td>
                        <td>23.73
                           <hr></hr>
                        </td>
                        <td>0.05
                           <hr></hr>
                        </td>
                        <td>9.1 × 10
                           <sup>6</sup>
                           <hr></hr>
                        </td>
                        <td>23.72
                           <hr></hr>
                        </td>
                        <td>0.28
                           <hr></hr>
                        </td>
                        <td>4.7 × 10
                           <sup>7</sup>
                           <hr></hr>
                        </td>
                        <td> 
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>YFV Brazil
                           <hr></hr>
                        </td>
                        <td>RKI reference strain
                           <hr></hr>
                        </td>
                        <td>21.18
                           <hr></hr>
                        </td>
                        <td>0.07
                           <hr></hr>
                        </td>
                        <td>5.0 × 10
                           <sup>7</sup>
                           <hr></hr>
                        </td>
                        <td>22.50
                           <hr></hr>
                        </td>
                        <td>0.10
                           <hr></hr>
                        </td>
                        <td>1.2 × 10
                           <sup>8</sup>
                           <hr></hr>
                        </td>
                        <td> 
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>YFV Ivory Cost
                           <hr></hr>
                        </td>
                        <td>RKI reference strain
                           <hr></hr>
                        </td>
                        <td>21.44
                           <hr></hr>
                        </td>
                        <td>0.21
                           <hr></hr>
                        </td>
                        <td>4.2 × 10
                           <sup>7</sup>
                           <hr></hr>
                        </td>
                        <td>21.29
                           <hr></hr>
                        </td>
                        <td>0.05
                           <hr></hr>
                        </td>
                        <td>3.1 × 10
                           <sup>8</sup>
                           <hr></hr>
                        </td>
                        <td> 
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>WNV Uganda
                           <hr></hr>
                        </td>
                        <td>AY532665, lineage 1
                           <hr></hr>
                        </td>
                        <td>28.74
                           <hr></hr>
                        </td>
                        <td>0.06
                           <hr></hr>
                        </td>
                        <td>3.2 × 10
                           <sup>5</sup>
                           <hr></hr>
                        </td>
                        <td>26.83
                           <hr></hr>
                        </td>
                        <td>0.27
                           <hr></hr>
                        </td>
                        <td>3.4 × 10
                           <sup>6</sup>
                           <hr></hr>
                        </td>
                        <td>[
                           <span ref-type="bibr" rid="B31">
                              <sup>
                                 <a href="#B31">31</a>
                              </sup>
                           </span>]
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>WNV Israel
                           <hr></hr>
                        </td>
                        <td>lineage 2
                           <hr></hr>
                        </td>
                        <td>19.81
                           <hr></hr>
                        </td>
                        <td>0.15
                           <hr></hr>
                        </td>
                        <td>1.2 × 10
                           <sup>8</sup>
                           <hr></hr>
                        </td>
                        <td>19.86
                           <hr></hr>
                        </td>
                        <td>0.01
                           <hr></hr>
                        </td>
                        <td>4.0 × 10
                           <sup>8</sup>
                           <hr></hr>
                        </td>
                        <td> 
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>TBEV K23
                           <hr></hr>
                        </td>
                        <td>AF091010
                           <hr></hr>
                        </td>
                        <td>16.33
                           <hr></hr>
                        </td>
                        <td>0.03
                           <hr></hr>
                        </td>
                        <td>1.3 × 10
                           <sup>9</sup>
                           <hr></hr>
                        </td>
                        <td>16.78
                           <hr></hr>
                        </td>
                        <td>0.14
                           <hr></hr>
                        </td>
                        <td>2.1 × 10
                           <sup>8</sup>
                           <hr></hr>
                        </td>
                        <td>[
                           <span ref-type="bibr" rid="B32">
                              <sup>
                                 <a href="#B32">32</a>
                              </sup>
                           </span>]
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>RSSEV
                           <hr></hr>
                        </td>
                        <td>RKI reference strain
                           <hr></hr>
                        </td>
                        <td>24.20
                           <hr></hr>
                        </td>
                        <td>0.32
                           <hr></hr>
                        </td>
                        <td>6.7 × 10
                           <sup>6</sup>
                           <hr></hr>
                        </td>
                        <td>19.35
                           <hr></hr>
                        </td>
                        <td>0.01
                           <hr></hr>
                        </td>
                        <td>4.1 × 10
                           <sup>6</sup>
                           <hr></hr>
                        </td>
                        <td> 
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>TBEV Louping ill
                           <hr></hr>
                        </td>
                        <td>RKI reference strain
                           <hr></hr>
                        </td>
                        <td>20.59
                           <hr></hr>
                        </td>
                        <td>0.22
                           <hr></hr>
                        </td>
                        <td>7.4 × 10
                           <sup>7</sup>
                           <hr></hr>
                        </td>
                        <td>20.43
                           <hr></hr>
                        </td>
                        <td>0.06
                           <hr></hr>
                        </td>
                        <td>2.0 × 10
                           <sup>7</sup>
                           <hr></hr>
                        </td>
                        <td> 
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>JEV</td>
                        <td>ATCC SA14-14-2</td>
                        <td>19.36</td>
                        <td>0.01</td>
                        <td>2.4 × 10
                           <sup>8</sup>
                        </td>
                        <td>16.73</td>
                        <td>0.01</td>
                        <td>7.7 × 10
                           <sup>8</sup>
                        </td>
                        <td>In house assay</td>
                     </tr>
                  </div>
               </table>
               <table-wrap-foot xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>
                     <i>GC</i>: genome copies, 
                     <i>Ct</i>: threshold cycle, 
                     <i>SD</i>: standard deviation.
                  </p>
                  <p>
                     <i>DENV</i>: Dengue virus, 
                     <i>JEV</i>: Japanese encephalitis virus, 
                     <i>RSSEV</i>: Russian spring-summer encephalitis virus, 
                     <i>WNV</i>: West Nile virus, 
                     <i>TBEV</i>: Tick-borne encephalitis virus, 
                     <i>YFV</i>: Yellow fever virus.
                  </p>
                  <p>
                     <sup>*</sup>Pan-Flavi assay was performed using Flavi-all S, Flavi all AS2, DEN4 F primer and
                     Flavi all probe3 mix.
                  </p>
                  <p>
                     <sup>#</sup>The same amount of RNA (5 μl) was applied to both pan-Flavi assay and virus species-specific
                     one-step qRT-PCR assays.
                  </p>
               </table-wrap-foot>
            </div>
            <div id="T3">
               <span class="label">Table 3</span>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Performance of the Pan-Flavi assay to detect different flaviviruses. Five μl of viral
                     RNA extracted from virus cell culture supernatant were applied to both pan-Flavi assay
                     and RT-PCR
                  </p>
               </caption>
               <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <colgroup>
                     <col></col>
                     <col></col>
                     <col></col>
                     <col></col>
                     <col></col>
                  </colgroup>
                  <thead>
                     <tr>
                        <th>
                           <b>Virus</b>
                        </th>
                        <th>
                           <b>Virus abbreviation</b>
                        </th>
                        <th>
                           <b>Strain/Reference</b>
                           <hr></hr>
                        </th>
                        <th>
                           <b>Pan-flavi assay</b>
                           <hr></hr>
                        </th>
                        <th>
                           <b>RT-PCR *</b>
                           <hr></hr>
                        </th>
                     </tr>
                     <tr>
                        <th> </th>
                        <th>
                           <b>Ct value</b>
                        </th>
                        <th>
                           <b>Confirmation by gel</b>
                        </th>
                     </tr>
                  </thead>
                  <div tagxxx="tbody">
                     <tr>
                        <td>
                           <b>
                              <i>Flaviviridae</i>
                           </b>
                           <hr></hr>
                        </td>
                        <td> </td>
                        <td> </td>
                        <td> </td>
                        <td> </td>
                     </tr>
                     <tr>
                        <td> St. Louis Encephalitis virus
                           <hr></hr>
                        </td>
                        <td>SLEV
                           <hr></hr>
                        </td>
                        <td>ATCC VR-1265
                           <hr></hr>
                        </td>
                        <td>22.19
                           <hr></hr>
                        </td>
                        <td>N.d.
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td> Usutu virus
                           <hr></hr>
                        </td>
                        <td>USUV
                           <hr></hr>
                        </td>
                        <td>AY453411
                           <hr></hr>
                        </td>
                        <td>22.48
                           <hr></hr>
                        </td>
                        <td>N.d.
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td> Kunjin virus
                           <hr></hr>
                        </td>
                        <td>KUNV
                           <hr></hr>
                        </td>
                        <td>PI.France
                           <hr></hr>
                        </td>
                        <td>24.97
                           <hr></hr>
                        </td>
                        <td>N.d.
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td> Kedougou virus
                           <hr></hr>
                        </td>
                        <td>KEDV
                           <hr></hr>
                        </td>
                        <td>ArD 14701
                           <hr></hr>
                        </td>
                        <td>29.62
                           <hr></hr>
                        </td>
                        <td>Positive
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td> Koutango virus
                           <hr></hr>
                        </td>
                        <td>KOUV
                           <hr></hr>
                        </td>
                        <td>AnD 5443
                           <hr></hr>
                        </td>
                        <td>30.29
                           <hr></hr>
                        </td>
                        <td>Positive
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td> Spondweni virus
                           <hr></hr>
                        </td>
                        <td>SPOV
                           <hr></hr>
                        </td>
                        <td>SA Ar 94
                           <hr></hr>
                        </td>
                        <td>26.02
                           <hr></hr>
                        </td>
                        <td>Negative
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td> Uganda S virus
                           <hr></hr>
                        </td>
                        <td>UGSV
                           <hr></hr>
                        </td>
                        <td>ArD 109325
                           <hr></hr>
                        </td>
                        <td>29.48
                           <hr></hr>
                        </td>
                        <td>Positive
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td> Wesselsbron
                           <hr></hr>
                        </td>
                        <td>WESSV
                           <hr></hr>
                        </td>
                        <td>ArB 4177
                           <hr></hr>
                        </td>
                        <td>24.73
                           <hr></hr>
                        </td>
                        <td>Positive
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td> Zika virus
                           <hr></hr>
                        </td>
                        <td>ZIKV
                           <hr></hr>
                        </td>
                        <td>PI, Senegal
                           <hr></hr>
                        </td>
                        <td>22.45
                           <hr></hr>
                        </td>
                        <td>Positive
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td> Sepik virus
                           <hr></hr>
                        </td>
                        <td>SEPV
                           <hr></hr>
                        </td>
                        <td>MK 7148
                           <hr></hr>
                        </td>
                        <td>16.77
                           <hr></hr>
                        </td>
                        <td>Positive
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td> Dakar bat virus
                           <hr></hr>
                        </td>
                        <td>DBV
                           <hr></hr>
                        </td>
                        <td>AnD 249
                           <hr></hr>
                        </td>
                        <td>32.57
                           <hr></hr>
                        </td>
                        <td>Positive
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td> Yaounade virus
                           <hr></hr>
                        </td>
                        <td>YAOV
                           <hr></hr>
                        </td>
                        <td>ArY 276/68
                           <hr></hr>
                        </td>
                        <td>17.52
                           <hr></hr>
                        </td>
                        <td>Positive
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td> Bouboui virus
                           <hr></hr>
                        </td>
                        <td>BOUV
                           <hr></hr>
                        </td>
                        <td>ArB 490
                           <hr></hr>
                        </td>
                        <td>31.88
                           <hr></hr>
                        </td>
                        <td>Negative
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td> Saboya virus
                           <hr></hr>
                        </td>
                        <td>SABV
                           <hr></hr>
                        </td>
                        <td>AnD 4600
                           <hr></hr>
                        </td>
                        <td>27.50
                           <hr></hr>
                        </td>
                        <td>Negative
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td> Nounane virus
                           <sup>#</sup>
                           <hr></hr>
                        </td>
                        <td>NOUV
                           <hr></hr>
                        </td>
                        <td>B3 isolate
                           <hr></hr>
                        </td>
                        <td>No ct
                           <hr></hr>
                        </td>
                        <td>N.d.
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>
                              <i>Bunya viridae</i>
                           </b>
                           <hr></hr>
                        </td>
                        <td> </td>
                        <td> </td>
                        <td> </td>
                        <td> </td>
                     </tr>
                     <tr>
                        <td> Rift valley fever virus
                           <hr></hr>
                        </td>
                        <td>RVFV
                           <hr></hr>
                        </td>
                        <td>RKI
                           <hr></hr>
                        </td>
                        <td>No ct
                           <hr></hr>
                        </td>
                        <td>N.d.
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>
                              <i>Togaviridae</i>
                           </b>
                           <hr></hr>
                        </td>
                        <td> </td>
                        <td> </td>
                        <td> </td>
                        <td> </td>
                     </tr>
                     <tr>
                        <td> Chikungunya virus
                           <hr></hr>
                        </td>
                        <td> 
                           <hr></hr>
                        </td>
                        <td>LR 2006
                           <hr></hr>
                        </td>
                        <td>No ct
                           <hr></hr>
                        </td>
                        <td>N.d.
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td> Chikungunya virus
                           <hr></hr>
                        </td>
                        <td> 
                           <hr></hr>
                        </td>
                        <td>ST05,African isolate
                           <hr></hr>
                        </td>
                        <td>No ct
                           <hr></hr>
                        </td>
                        <td>N.d.
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td> Sindbis virus
                           <hr></hr>
                        </td>
                        <td> 
                           <hr></hr>
                        </td>
                        <td>RKI
                           <hr></hr>
                        </td>
                        <td>No ct
                           <hr></hr>
                        </td>
                        <td>N.d.
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>
                           <b>
                              <i>Influneza virus</i>
                           </b>
                           <hr></hr>
                        </td>
                        <td> </td>
                        <td> </td>
                        <td> </td>
                        <td> </td>
                     </tr>
                     <tr>
                        <td> H1N1 Hamburg
                           <hr></hr>
                        </td>
                        <td> 
                           <hr></hr>
                        </td>
                        <td>A/Hamburg/04/2009 H1N1
                           <hr></hr>
                        </td>
                        <td>No ct
                           <hr></hr>
                        </td>
                        <td>N.d.
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td> H1N1 California
                           <hr></hr>
                        </td>
                        <td> 
                           <hr></hr>
                        </td>
                        <td>A/California/04/2009 H1N1
                           <hr></hr>
                        </td>
                        <td>No ct
                           <hr></hr>
                        </td>
                        <td>N.d.
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td> H5N1</td>
                        <td> </td>
                        <td>A/dk/Germany R603/06 H5N1</td>
                        <td>No ct</td>
                        <td>N.d.</td>
                     </tr>
                  </div>
               </table>
               <table-wrap-foot xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>
                     <i>N.d</i>.: not done, 
                     <i>Ct</i>: threshold cycle, * RNA of African flaviviruses were detected using reference method
                     described previously [
                     <span ref-type="bibr" rid="B33">
                        <sup>
                           <a href="#B33">33</a>
                        </sup>
                     </span>].
                     <sup>#</sup> Nounane virus was detected using final optimized primer sets Flavi all S/Flavi all
                     S2/Flavi all AS4 and Flavi all probe 3 mix.
                  </p>
               </table-wrap-foot>
            </div>
         </div>
         <div id="">
            <h3>Specificity evaluation of the Pan-Flavi assay</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The specificity of the final version of the Pan-Flavi assay was tested using a panel
               of mosquito- and tick-borne flaviviruses, one insect-only novel flavivirus and one
               flavivirus with unknown vector, one virus of the 
               <i>Bunyaviridae</i> family, three of the 
               <i>Togaviridae</i> family and three of the 
               <i>Orthomyxoviridae</i> family
               <i>.</i> Viral RNA of thirty different flaviviruses was detected, verifying the broad range
               specificity. However, one insect-only novel flavivirus was not detected. Neither non-flaviviruses
               nor negative controls resulted in signals, demonstrating the absence of cross-reactivity
               of the Pan-Flavi assay with non-flaviviruses (Tables 
               <span ref-type="table" rid="T2">
                  <a href="#T2">2</a>
               </span> and 
               <span ref-type="table" rid="T3">
                  <a href="#T3">3</a>
               </span>).
            </p>
         </div>
         <div id="">
            <h3>Sensitivity evaluation of the Pan-Flavi assay</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To determine the linearity of the assay, 10-fold serial dilutions of the standard
               plasmid were tested. The Pan-Flavi assay was shown to be linear over a range of 5
               log
               <div tagxxx="sub">10</div> copies. Limit of detection is at least 10 copies/reaction of standard plasmid (Figure 
               <span ref-type="fig" rid="F1">1</span>).
            </p>
            <div tagxxx="fig">
               <span class="label">Figure 1</span>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>
                     <b>Standard curve constructed using ten-fold serial dilution of standard plasmid containing
                        NS5 gene of yellow fever virus.
                     </b> The detection thresholds (Ct values) are plotted against the log of known standard
                     plasmid (pYFV) copy numbers.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The Pan-Flavi assay was performed as a one-step qRT-PCR. The sensitivity of the Pan-Flavi
               assay to detect viral RNA was compared to three virus species-specific one-step qRT-PCR
               assays for DENV, TBEV and YFV, all assays could be thus compared in one run. The RNA
               stocks of DENV-1, YFV strain 17D and TBEV strain K23 were quantified by virus species-specific
               qRT-PCR as described previously [
               <span ref-type="bibr" rid="B29">
                  <sup>
                     <a href="#B29">29</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="B30">
                  <sup>
                     <a href="#B30">30</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="B32">
                  <sup>
                     <a href="#B32">32</a>
                  </sup>
               </span>]. Then, 10-fold serial dilution of the RNA was prepared from quantified stocks to
               cover a range of 10
               <sup>1</sup> to 10
               <sup>7</sup> genome copies/reaction. The detection limit of Pan-Flavi assay was determined using
               these pre-quantified RNA dilutions. Detection limit of the Pan-Flavi assay was 100
               genome copies for YFV 17D, 10 genome copies for DENV-1 and 10 genome copies for TBEV
               K23 (Table 
               <span ref-type="table" rid="T4">
                  <a href="#T4">4</a>
               </span>).
            </p>
            <div id="T4">
               <span class="label">Table 4</span>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Test of sensitivity of Pan-Flavi assay with pre-quantified viral RNA dilutions</p>
               </caption>
               <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <colgroup>
                     <col></col>
                     <col></col>
                     <col></col>
                     <col></col>
                  </colgroup>
                  <thead>
                     <tr>
                        <th>
                           <b>Quantity (Genome copies/reaction)</b>
                        </th>
                        <th>
                           <b>Ct values for DENV-1</b>
                        </th>
                        <th>
                           <b>Ct values for TBEV strain K23</b>
                        </th>
                        <th>
                           <b>Ct values for YFV strain 17D</b>
                        </th>
                     </tr>
                  </thead>
                  <div tagxxx="tbody">
                     <tr>
                        <td>1 × 10
                           <sup>7</sup>
                           <hr></hr>
                        </td>
                        <td>16.8
                           <hr></hr>
                        </td>
                        <td>16.6
                           <hr></hr>
                        </td>
                        <td>-
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>1 × 10
                           <sup>6</sup>
                           <hr></hr>
                        </td>
                        <td>20.5
                           <hr></hr>
                        </td>
                        <td>19.5
                           <hr></hr>
                        </td>
                        <td>20.8
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>1 × 10
                           <sup>5</sup>
                           <hr></hr>
                        </td>
                        <td>23.4
                           <hr></hr>
                        </td>
                        <td>23.0
                           <hr></hr>
                        </td>
                        <td>24.2
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>1 × 10
                           <sup>4</sup>
                           <hr></hr>
                        </td>
                        <td>27.4
                           <hr></hr>
                        </td>
                        <td>26.2
                           <hr></hr>
                        </td>
                        <td>27.6
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>1 × 10
                           <sup>3</sup>
                           <hr></hr>
                        </td>
                        <td>30.2
                           <hr></hr>
                        </td>
                        <td>29.4
                           <hr></hr>
                        </td>
                        <td>30.1
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>1 × 10
                           <sup>2</sup>
                           <hr></hr>
                        </td>
                        <td>37.1
                           <hr></hr>
                        </td>
                        <td>31.7
                           <hr></hr>
                        </td>
                        <td>36.4
                           <hr></hr>
                        </td>
                     </tr>
                     <tr>
                        <td>1 × 10
                           <sup>1</sup>
                        </td>
                        <td>39.7</td>
                        <td>33.0</td>
                        <td>No Ct</td>
                     </tr>
                  </div>
               </table>
               <table-wrap-foot xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>
                     <i>GC</i>: genome copies, 
                     <i>Ct</i>: threshold cycle.
                  </p>
                  <p>
                     <i>YFV</i>: Yellow fever virus, 
                     <i>TBE</i>: Tick-borne encephalitis virus, 
                     <i>DENV</i>: Dengue virus.
                  </p>
               </table-wrap-foot>
            </div>
         </div>
         <div id="">
            <h3>Comparison of the Pan-Flavi assay with virus species-specific qRT-PCR</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The same amounts of RNA (5μl) from DENV, WNV, YFV, TBEV and JEV were detected quantitatively
               by using virus species-specific qRT-PCR assays, and the results were compared with
               those obtained by the Pan-Flavi assay for these viruses (Table 
               <span ref-type="table" rid="T2">
                  <a href="#T2">2</a>
               </span>). All four serotypes of DENVs were detected by the Pan-Flavi assay, with almost identical
               Ct values and GC/ml; although for DENV the Pan-Flavi assay showed delayed Ct values
               corresponding to one log dilution. For all YFV isolates tested the results were similar
               to those with the YFV-specific assay. Also WNV lineage 2 gave the same results as
               the WNV-specific assay, while the Ct values for lineage 1 virus were delayed. All
               three subtypes of TBEV were detected by both the Pan-Flavi assay and a TBEV-specific
               assay. The Ct values and genome copies/ml determined by both methods were identical
               except for RSSEV. For the detection of JEV, the Pan-Flavi assay had half a log lower
               sensitivity compared to the JEV-specific assay. WNV- and JEV-specific assays were
               performed in a two-step format which could explain their higher sensitivity compared
               to the Pan-Flavi assay.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In summary, the performance of the Pan-Flavi assay was comparable to virus species-specific
               assays except for minor differences for some strains of flaviviruses.
            </p>
         </div>
      </div>
      <div id="">
         <h2>Discussion</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The ongoing emergence of flaviviruses poses major public health concerns worldwide.
            Techniques used for molecular testing and surveillance of flaviviruses should be able
            to detect and identify a wide range of flavivirus species rapidly, with a high level
            of specificity and sensitivity. For this purpose, an LNA probe-based qRT-PCR (Pan-Flavi
            assay) for detection of RNA of various flaviviruses was developed. We tested several
            important human-pathogenic flaviviruses to demonstrate the ability to detect relevant
            members of the flavivirus genus. The Pan-Flavi assay was also tested with other viruses,
            in particular phleboviruses, alphaviruses and influenza viruses, to determine absence
            of cross-reactivity.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In recent years, generic approaches targeting highly conserved NS3 and NS5 genes have
            been described for the detection of flaviviruses as summarized previously [
            <span ref-type="bibr" rid="B24">
               <sup>
                  <a href="#B24">24</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="B26">
               <sup>
                  <a href="#B26">26</a>
               </sup>
            </span>]. These methods were predominantly in the format of nested RT-PCR or SYBR green qRT-PCR
            which required post-amplification methods such as sequencing or melting temperature
            analysis to verify the result. In general, these methods have shown variable or lower
            sensitivity than species-specific approaches when evaluated during EQA exercises for
            the molecular diagnosis of different flavivirus [
            <span ref-type="bibr" rid="B29">
               <sup>
                  <a href="#B29">29</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="B34">
               <sup>
                  <a href="#B34">34</a>
               </sup>
            </span>]. Only in some cases assays were comparable to those from the species-specific methods
            [
            <span ref-type="bibr" rid="B27">
               <sup>
                  <a href="#B27">27</a>
               </sup>
            </span>].
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In the past 10 years, the primer sets targeting a conserved region of NS5 have been
            developed and optimized for a broad detection of flaviviruses. This approach was not
            only useful to detect known flaviviruses, but has also led to discovery of novel flaviviruses
            [
            <span ref-type="bibr" rid="B18">
               <sup>
                  <a href="#B18">18</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="B35">
               <sup>
                  <a href="#B35">35</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="B36">
               <sup>
                  <a href="#B36">36</a>
               </sup>
            </span>].
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">For the novel Pan-Flavi assay, the degenerate primer sequences share the conserved
            NS5 region (Figure 
            <span ref-type="fig" rid="F2">2</span>) targeted by the previous published primers (Figure 
            <span ref-type="fig" rid="F3">3</span>). These primers exhibited good efficiency in detecting a broad range of common and
            exotic flaviviruses (Tables 
            <span ref-type="table" rid="T2">
               <a href="#T2">2</a>
            </span> and 
            <span ref-type="table" rid="T3">
               <a href="#T3">3</a>
            </span>) but failed to detect the RNA of DENV type 4 or Nounane virus. The alignment for
            the Flavi all S primer revealed mismatches at 6th and 8th position from the 3′-end
            which would have been rated as non-critical (Figure 
            <span ref-type="fig" rid="F2">2</span>). However, only addition of DEN4 F primer improved the assay for DENV type 4 (Tables 
            <span ref-type="table" rid="T1">
               <a href="#T1">1</a>
            </span> and 
            <span ref-type="table" rid="T2">
               <a href="#T2">2</a>
            </span>). Remarkably, the 3rd position from 3′-end mismatch in the reverse primer Flavi all
            AS2 for YFV, DENV type 2 and Usutu virus did not have any impact as these three viruses
            were detected well by Pan-Flavi assay (Figure 
            <span ref-type="fig" rid="F2">2</span>). These results underline that the 
            <i>in-silico</i> analysis provides only a limited prediction for the primer specificity, which still
            requires to be confirmed by laboratory testing. Designing of a single probe for detection
            of all flaviviruses is challenging because of the genetic diversity and the restricted
            length of conserved regions. One approach is the use of oligonucleotide mixtures covering
            many target sequence variants, which was applied for detection of DENV and Crimean-Congo
            Hemorrhagic fever virus (CCHF) [
            <span ref-type="bibr" rid="B37">
               <sup>
                  <a href="#B37">37</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="B38">
               <sup>
                  <a href="#B38">38</a>
               </sup>
            </span>]. Other approaches to design broad range probes are the use of a short conserved
            sequence in combination of minor groove binding (MGB) or LNA modifications [
            <span ref-type="bibr" rid="B39">
               <sup>
                  <a href="#B39">39</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="B40">
               <sup>
                  <a href="#B40">40</a>
               </sup>
            </span>]. First, we used 22 specific probes for flaviviruses generic detection, but the amplification
            signals were insufficient. To overcome this problem, probes with LNA bases were designed
            for the Pan-Flavi assay. Major advantages of LNA probes in comparison to other real-time
            PCR formats are the shorter probe length, stability, increased sensitivity, specificity
            and quenching efficiency which have been demonstrated by many studies [
            <span ref-type="bibr" rid="B30">
               <sup>
                  <a href="#B30">30</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="B41">
               <sup>
                  <a href="#B41">41</a>
               </sup>
            </span>-
            <span ref-type="bibr" rid="B45">
               <sup>
                  <a href="#B45">45</a>
               </sup>
            </span>]. One of three designed and tested LNA-based probes (Flavi all probe) successfully
            targeted a broad spectrum of different flaviviruses. The signal for detection of YFV
            and DENV type 4 were optimized by adding small fractions of specific probes (Table 
            <span ref-type="table" rid="T5">
               <a href="#T5">5</a>
            </span>).
         </p>
         <div tagxxx="fig">
            <span class="label">Figure 2</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>Sequence alignment of primers and probe with 26 flaviviruses conserved region of NS5
                     gene.
                  </b> + Symbol under oligonucleotides indicates position of locked nucleotides.
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <span class="label">Figure 3</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>The published and Flavi-all primer sites in the Yellow fever 17D genome (NC 002031)
                     with locations of genes.
                  </b> In dark green: forward primer, in light green: reverse primer, in red: site of probe
                  used in this study. The figure is prepared using Geneious pro version (5.4.3) created
                  by Biomatters.
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div id="T5">
            <span class="label">Table 5</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>Oligonucleotide sequence of primers and probes used in Pan-Flavi qRT-PCR assay</p>
            </caption>
            <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <colgroup>
                  <col></col>
                  <col></col>
                  <col></col>
                  <col></col>
                  <col></col>
                  <col></col>
               </colgroup>
               <thead>
                  <tr>
                     <th>
                        <b>Primer &amp; Probe</b>
                     </th>
                     <th>
                        <b>Sequence</b>
                     </th>
                     <th>
                        <b>Concentration</b>
                     </th>
                     <th>
                        <b>Orientation</b>
                        <sup>
                           <b>a</b>
                        </sup>
                     </th>
                     <th>
                        <b>Position</b>
                        <sup>
                           <b>b</b>
                        </sup>
                     </th>
                     <th>
                        <b>Tm (°C)</b>
                     </th>
                  </tr>
               </thead>
               <div tagxxx="tbody">
                  <tr>
                     <td>
                        <b>Flavi all S</b>
                        <hr></hr>
                     </td>
                     <td>TACAACATgATggggAARAgAgARAA
                        <hr></hr>
                     </td>
                     <td>10 pmol
                        <hr></hr>
                     </td>
                     <td>S
                        <hr></hr>
                     </td>
                     <td>8993–9019
                        <hr></hr>
                     </td>
                     <td>54.7
                        <hr></hr>
                     </td>
                  </tr>
                  <tr>
                     <td>
                        <b>DEN4 F</b>
                        <hr></hr>
                     </td>
                     <td>TACAACATgATgggRAAACgTgAGAA
                        <hr></hr>
                     </td>
                     <td>10 pmol
                        <hr></hr>
                     </td>
                     <td>S
                        <hr></hr>
                     </td>
                     <td>8996–9019
                        <hr></hr>
                     </td>
                     <td>60.18
                        <hr></hr>
                     </td>
                  </tr>
                  <tr>
                     <td>
                        <b>Flavi all AS 1</b>
                        <hr></hr>
                     </td>
                     <td>gTCCCANCCDgCKgTRTC
                        <hr></hr>
                     </td>
                     <td>10 pmol
                        <hr></hr>
                     </td>
                     <td>AS
                        <hr></hr>
                     </td>
                     <td>9236–9253
                        <hr></hr>
                     </td>
                     <td>52.5
                        <hr></hr>
                     </td>
                  </tr>
                  <tr>
                     <td> 
                        <hr></hr>
                     </td>
                     <td>gTCCCATCCAgCKgTRTCATC
                        <hr></hr>
                     </td>
                     <td>5 pmol
                        <hr></hr>
                     </td>
                     <td>AS
                        <hr></hr>
                     </td>
                     <td>9236–9256
                        <hr></hr>
                     </td>
                     <td>57.1
                        <hr></hr>
                     </td>
                  </tr>
                  <tr>
                     <td>
                        <b>Flavi all AS 2</b>
                        <hr></hr>
                     </td>
                     <td>gTgTCCCAgCCNgCKgTgTCATCWgC
                        <hr></hr>
                     </td>
                     <td>10 pmol
                        <hr></hr>
                     </td>
                     <td>AS
                        <hr></hr>
                     </td>
                     <td>9232–9260
                        <hr></hr>
                     </td>
                     <td>69.6
                        <hr></hr>
                     </td>
                  </tr>
                  <tr>
                     <td>
                        <b>Flavi all probe 1</b>
                        <hr></hr>
                     </td>
                     <td>FAM-AARggHAgYMgNgCCA 
                        <b>+ T</b>H 
                        <b>+ T + g + g + T</b>-BBQ
                        <hr></hr>
                     </td>
                     <td>5 pmol
                        <hr></hr>
                     </td>
                     <td>S
                        <hr></hr>
                     </td>
                     <td>9044–9065
                        <hr></hr>
                     </td>
                     <td>69–83
                        <hr></hr>
                     </td>
                  </tr>
                  <tr>
                     <td>
                        <b>Flavi all probe 2</b>
                        <hr></hr>
                     </td>
                     <td>FAM-Tg 
                        <b>+ g</b>TWYATgTggYTNg 
                        <b>+ g</b>RgC--BBQ
                        <hr></hr>
                     </td>
                     <td>5 pmol
                        <hr></hr>
                     </td>
                     <td>S
                        <hr></hr>
                     </td>
                     <td>9062–9082
                        <hr></hr>
                     </td>
                     <td>62–75
                        <hr></hr>
                     </td>
                  </tr>
                  <tr>
                     <td>
                        <b>Flavi all probe 3 mix</b>
                        <hr></hr>
                     </td>
                     <td>FAM-Tg 
                        <b>+ g</b>TWYATgT 
                        <b>+ g</b>gYTNg 
                        <b>+ g</b>RgC—BBQ
                        <sup>c</sup>
                        <hr></hr>
                     </td>
                     <td>5 pmol
                        <hr></hr>
                     </td>
                     <td>S
                        <hr></hr>
                     </td>
                     <td>9062–9082
                        <hr></hr>
                     </td>
                     <td>66-79
                        <hr></hr>
                     </td>
                  </tr>
                  <tr>
                     <td> 
                        <hr></hr>
                     </td>
                     <td>FAM-CCgTgCCATATggTATATgTggCTgggAgC-BBQ
                        <sup>d</sup>
                        <hr></hr>
                     </td>
                     <td>0.5 pmol
                        <hr></hr>
                     </td>
                     <td>S
                        <hr></hr>
                     </td>
                     <td>9052–9081
                        <hr></hr>
                     </td>
                     <td>74.7
                        <hr></hr>
                     </td>
                  </tr>
                  <tr>
                     <td> 
                        <hr></hr>
                     </td>
                     <td>FAM-TTTCTggAATTTgAAgCCCTgggTTT-BBQ
                        <sup>e</sup>
                        <hr></hr>
                     </td>
                     <td>0.5 pmol
                        <hr></hr>
                     </td>
                     <td>S
                        <hr></hr>
                     </td>
                     <td>9086–9012
                        <hr></hr>
                     </td>
                     <td>68
                        <hr></hr>
                     </td>
                  </tr>
                  <tr>
                     <td>
                        <b>Flavi all S2</b>
                        <hr></hr>
                     </td>
                     <td>TACAACATgATgggMAAACgYgARAA
                        <hr></hr>
                     </td>
                     <td>10 pmol
                        <hr></hr>
                     </td>
                     <td>S
                        <hr></hr>
                     </td>
                     <td>8996–9019
                        <hr></hr>
                     </td>
                     <td>58.2
                        <hr></hr>
                     </td>
                  </tr>
                  <tr>
                     <td>
                        <b>Flavi all AS4</b>
                     </td>
                     <td>gTgTCCCAGCCNgCKgTRTCRTC</td>
                     <td>10 pmol</td>
                     <td>S</td>
                     <td>9235–9260</td>
                     <td>64.1</td>
                  </tr>
               </div>
            </table>
            <table-wrap-foot xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>Tm: temperature, Locked-Nucleic Acid bases are written as ‘ 
                  <b>+ _</b>’, e.g., 
                  <b>+A.</b>
               </p>
               <p>Degenerate bases: R = (A/G), W = (A/T), K = (T/G), Y = (C/T), N = (A/G/T/C).</p>
               <p>
                  <sup>
                     <i>a</i>
                  </sup> S: sense orientation, AS: antisense orientation.
               </p>
               <p>
                  <sup>
                     <i>b</i>
                  </sup> YFV strain; accession no: NC 002031.
               </p>
               <p>
                  <sup>
                     <i>c</i>
                  </sup> Flavi all probe, 
                  <sup>
                     <i>d</i>
                  </sup> Flavi probe YFV, 
                  <sup>
                     <i>e</i>
                  </sup> Flavi probe DEN4.
               </p>
               <p>Final Pan-Flavi assay comprise of Flavi all S, Flavi all S2 &amp; Flavi all AS4 primer
                  and Flavi all probe 3 mix.
               </p>
            </table-wrap-foot>
         </div>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">At present, RT-PCR in nested or hemi-nested format is used most frequently. It requires
            sequencing for the identification of viruses and needs approximately one day for experimentation.
            In addition, this carries a high risk of contamination caused by open handling of
            PCR products, increasing the potential for false positives. In contrast, the Pan-Flavi
            assay presented here requires only some 50 minutes for specific and sensitive detection
            of several flaviviruses in one reaction. The detection sensitivity of YFV, DENV, TBEV,
            JEV and WNV, using the described Pan-Flavi assay, was close to or as good as the species-specific
            qRT-PCR assays. These differences may have resulted from deviant reactions conditions,
            enzyme kits and instruments, which were used as published or in-house established,
            and it is to mention, that WNV and JEV assays were performed in a two-step PCR which
            is known to be more sensitive than the one-step procedure [
            <span ref-type="bibr" rid="B46">
               <sup>
                  <a href="#B46">46</a>
               </sup>
            </span>].
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The sensitivity of the Pan-Flavi assay to detect RNA was demonstrated by using 10-fold
            serial dilutions of viral RNA from DENV-1, YFV and TBEV. The assay was able to detect
            10–100 viral RNA copies per reaction, depending on the respective flavivirus. The
            assay allows the quantification of flavivirus RNA by use of standard plasmids or standardised
            in-vitro RNA, providing information regarding viral load and can be easily established
            in other laboratories. Therefore, the Pan-Flavi assay could be used to confirm the
            presence of flaviviruses in a sample without post amplification steps or complement
            species-specific flavivirus assays. The amplicon length is about 260 bp, which is
            sufficient to distinguish and identify different the virus by (pyro) sequencing or
            using for example an array.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The Pan-Flavi assay allows rapid detection of a broad range of flaviviruses including
            local-endemic and exotic species with a high level of sensitivity and is cost-effective,
            making it suitable for screening of livestock samples, monitoring viral activity,
            outbreak investigation as well as surveillance studies of flaviviruses, and might
            help finding and identifying novel flaviviruses. We expect that the assay can be also
            helpful for testing clinical samples from patients with symptoms especially in an
            early phase of infection and without knowledge about locally prevalent flaviviruses
            or travel history in order to avoid parallel testing on several flaviviruses, although
            a negative test will of course not prove absence of a flavivirus infection.
         </p>
      </div>
      <div id="">
         <h2>Conclusions</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Re-emergence of West Nile virus in Europe and continuous spread of certain flaviviruses
            such as dengue, yellow fever and Japanese encephalitis viruses represent a global
            danger to public health. The Pan-Flavi assay presented here has been developed and
            evaluated in comparison with flavivirus species-specific qPCR assays to detect a broad
            range of flaviviruses simultaneously in a single-tube. This assay represents a valuable
            tool for the detection of flaviviruses in livestock samples, and will support molecular
            diagnosis as well as surveillance studies of various vectors and non-human hosts.
         </p>
      </div>
      <div id="">
         <h2>Methods</h2>
         <div id="">
            <h3>Viruses and isolation of viral RNA</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">All flavivirus and non-flavivirus strains listed below were derived from cell culture
               and provided by the Robert Koch Institute, Berlin, Germany. The following inactivated
               and stable virus preparations were used in this study: DENV-1 VR344 (Thai 1958); DENV-2
               VR345 (TH-36 strain); DENV-3 VR216 (H87 strain); DENV-4 VR217 (H241 strain); WNV Uganda
               strain (AY532665); WNV Israel (H. Bin, Sheba Medical Center, Israel), kunjin virus
               (Institute Pasteur, France), usutu virus (AY453411); JEV (ATCC SA14-14-2); Saint Louis
               encephalitis virus (SLEV) (ATCC VR-1265); TBEV strain K23 (AF091010); YFV strain 17D
               (X03700); YFV strain ASIBI (AY640589); YFV strain Brazil; YFV strain Ivory Coast;
               Russian Spring Summer Encephalitis virus (RSSEV); chikungunya virus (LR 2006); chikungunya
               virus African isolate; sindbis virus; Rift Valley Fever virus and influenza A virus
               subtype H5N1 (A/dk/Germany R603/06 H5N1).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Two different standard preparations of H1N1 influenza viruses (A/California/04/2009
               and A/Hamburg/04/2009) used in this study were provided by the European Network for
               Diagnostics of Imported Viral Diseases (ENIVD). RNA of Nounane virus was kindly provided
               by Dr. Fabian Leendertz from NG2, Robert Koch Institute, Berlin. RNA of eleven African
               flaviviruses (Table 
               <span ref-type="table" rid="T3">
                  <a href="#T3">3</a>
               </span>) were kindly provided by Dr. Amadou A Sall and testing of these viruses was done
               at Institut Pastuer Dakar, Senegal.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Viral RNA from above described viruses was isolated from 140 μl aliquots of cell culture
               supernatants, using the QIAamp Viral Mini Kit (Qiagen, Hilden, Germany) according
               to the manufacturer’s instructions. RNA was eluted in 100 μl of elution buffer and
               stored at -80°C until further use.
            </p>
         </div>
         <div id="">
            <h3>Primers and TaqMan probe</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The flaviviral NS5 gene contains conserved regions often used for PCR testing. Our
               laboratory results but also the literature [
               <span ref-type="bibr" rid="B27">
                  <sup>
                     <a href="#B27">27</a>
                  </sup>
               </span>] revealed that the commonly used primers [
               <span ref-type="bibr" rid="B26">
                  <sup>
                     <a href="#B26">26</a>
                  </sup>
               </span>-
               <span ref-type="bibr" rid="B28">
                  <sup>
                     <a href="#B28">28</a>
                  </sup>
               </span>] exhibit a low sensitivity and we failed to detect rare flaviviruses such as Nounane
               virus. The primer binding region were aligned against all Genbank (NCBI) deposited
               flavivirus sequences including partial entries for the NS5 region only using the MEGA5
               software. Based on the alignment, 32 primers and 22 probes representing all known
               variants of flavivirus sequences were selected to be used as mixture in one reaction
               (data not shown). In an alternative approach two degenerate generic primer sets and
               three degenerate probes containing LNA nucleotides were designed to match ‘all’ flaviviruses.
               LNA nucleotides contain a 2
               <sup>′</sup>–5
               <sup>′</sup> bridge, increasing the duplex stability, which in turn allows the use of shorter
               sequence probes matching the conserved sequence motif.
            </p>
         </div>
         <div id="">
            <h3>Pan-Flavi qRT-PCR assay</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The Pan-Flavi qRT-PCR assay (Pan-Flavi assay) was run with the Transcriptor One-Step
               RT-PCR Kit (Roche Diagnostics Mannheim, Germany) according to the manufacturer’s instructions.
               In brief, 5 μl of RNA, 10 pmol of primers Flavi all sense, Flavi all antisense 2 and
               DEN4 F, and a mixture of three probes (Table 
               <span ref-type="table" rid="T5">
                  <a href="#T5">5</a>
               </span>) were added to the master mix. Optimized run conditions were as follows: reverse
               transcription at 55°C for 2 min, denaturation at 95°C for 30 s and 45 cycles of 95°C
               for 10 s and 60°C for 25 s. The amplification was performed on a Roche LightCycler
               480 instrument, software version 1.5. The spectrum of the assay can be expanded by
               using Flavi all S2 instead of DEN4 F primer (Table 
               <span ref-type="table" rid="T5">
                  <a href="#T5">5</a>
               </span>).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Serial dilutions of a standard (10–10
               <sup>6</sup> copies/μl) were run in duplicate for quantification. Standard plasmid was prepared
               by cloning the PCR product obtained from YFV using a TOPO TA Cloning Kit (Invitrogen,
               Karlsruhe, Germany) according to the manufacturer’s instructions and quantified by
               spectrophotometric method.
            </p>
         </div>
         <div id="">
            <h3>Specific qRT-PCR for different flaviviruses</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Except for DENV and JEV we used published group-specific assays for comparison.</p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">For TBEV, we used an assay targeting the NS1 gene as described previously [
               <span ref-type="bibr" rid="B32">
                  <sup>
                     <a href="#B32">32</a>
                  </sup>
               </span>]. qRT- PCR was carried out in one-step format on an ABI 7500 cycler (Applied Biosystems,
               California, USA), using the Superscript III Platinum OneStep qRT-PCR Kit (Invitrogen).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A WNV-specific qRT-PCR assay for detection of lineages 1 and 2 was used to detect
               and quantify genome copies (GC) of WNV as described previously [
               <span ref-type="bibr" rid="B31">
                  <sup>
                     <a href="#B31">31</a>
                  </sup>
               </span>]. qRT-PCR was performed in two-step format on a Stratagene MX 3000 cycler (Agilent
               Technologies, Santa Clara, USA).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">YFV-specific primer combination YFV FP/RP and probe YFV LNA2 were used to detect and
               quantify genomic RNA of YFV as described previously [
               <span ref-type="bibr" rid="B30">
                  <sup>
                     <a href="#B30">30</a>
                  </sup>
               </span>]. The assay was performed in one-step format on the LightCycler 480 instrument using
               the QuantiTect Virus Kit (Qiagen, Hilden, Germany).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">DENV-genomic RNA was tested and quantified by an in-house qRT-PCR as described previously
               [
               <span ref-type="bibr" rid="B29">
                  <sup>
                     <a href="#B29">29</a>
                  </sup>
               </span>]. The assay is a DENV group-specific assay and was carried out in one-step format
               on an ABI 7500 real-time PCR system using the AgPath-ID One-Step RT-PCR Kit. Plasmid
               standards were used for the quantification of the DENV GE.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A JEV-specific in-house qPCR assay was used for detection and quantification of JEV
               GE. qPCR was performed in two-step format on a Stratagene MX 3000 cycler. cDNA was
               generated using SuperScript II (Invitrogen) and was stored at -20°C until use. The
               test was performed using Platinum Taq polymerase (Invitrogen) in a total volume of
               25 μl containing 5 μl of cDNA, 2 μl of 10x Reaction Mix, 4 mM MgCl
               <div tagxxx="sub">2</div>, 300 nM of each primer (JEV9250 F: CgTCCAAAAgCTgggATACAT and JEV9334 R: gTCCCAYCCggCggTRTC),
               100 nM of probe (FAM-TCCgTgAYATAgCAggRAAgCAAgg –BBQ) and 2.5 μM of dNTPs, using a
               Stratagene Mx3000 instrument and the following conditions: 15 min at 95°C; 45 cycles
               of 15 s at 95°C and 30 s at 60°C. Ten-fold serial dilutions of the standard (plasmid
               10–10
               <sup>6</sup> copies/reaction) were included in order to determine the limit of detection.
            </p>
         </div>
         <div id="">
            <h3>Final protocol for extended range specificity</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The pan-flavi assay was evaluated using forward primers Flavi all S and DEN4 F, reverse
               primer Flavi all AS2 and Flavi all probe 3 mix. Based on an alignment of latterly
               available NS5 gene flavivirus sequences, we substituted primer DEN4F by Flavi all
               S2 and Flavi all AS2 by Flavi all AS4 to include some novel flaviviruses (Table 
               <span ref-type="table" rid="T3">
                  <a href="#T3">3</a>
               </span>).
            </p>
         </div>
      </div>
      <div id="">
         <h2>Abbreviations</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">RT: Reverse transcription; Tm: Melting temperature; Ct: Threshold cycle; GC: Genome
            copies; DENV: Dengue virus; YFV: Yellow fever virus; TBEV: Tick-borne encephalitis
            virus; WNV: West nile virus; JEV: Japanese encephalitis virus; LNA: Locked-nucleic
            acids.
         </p>
      </div>
      <div id="">
         <h2>Competing interests</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The authors declare that they have no competing interests.</p>
      </div>
      <div id="">
         <h2>Authors’ contributions</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">PP, MK conceived and designed the experiments; OL, MK, PP participated in the sequence
            alignment/assay design; PP, TK, UL, OF performed the Experiments; PP, OL, MK analysed
            the data; AS, OL, MN contributed reagents/materials/analysis tool, PP wrote paper.
            All authors have read and approved this manuscript.
         </p>
      </div>
      <div id="">
         <h2>Authors’ information</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Pranav Patel is working as a scientist at Department of Centre for Biological Security
            1, Robert Koch Institute, Berlin, Germany and has vast experience on developing rapid
            diagnostic assays and platforms for detection of emerging pathogens.
         </p>
      </div>
      <div tagxxx="back">
         <div id="">
            <h2>Acknowledgments</h2>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Oligonucleotides were kindly provided by the Tib-Molbiol, Berlin. The authors are
               grateful to Anette Teichmann and Patricia Bußmann for technical support. We also thank
               Eduardo Thuroff and Regina Schädler for their critical review of the manuscript and
               Ursula Erikli for copy editing.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">This work was supported by the consultant laboratory for tick-borne encephalitis virus
               at Center for Biological Security 1 (ZBS1), Robert Koch institute as a part of research
               activities.
            </p>
         </div>
         <div tag="ref-list">
            <ul>
               <li tag="ref">
                  <a name="B1"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Mackenzie</span>
                        <span tagx="given-names">JS</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Gubler</span>
                        <span tagx="given-names">DJ</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Petersen</span>
                        <span tagx="given-names">LR</span>,
                     </span>
                     <span tagx="article-title">Emerging flaviviruses: the spread and resurgence of Japanese encephalitis, West Nile
                        and dengue viruses
                     </span>
                     <span tagx="source">Nat Med</span>
                     <span tagx="year">2004</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">S98</span>
                     <span tagx="lpage">S109</span>
                     <span class="pub-id'">[10.1038/nm1144]</span>
                     <span class="pub-id'">[15577938]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B2"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Porterfield</span>
                        <span tagx="given-names">JS</span>,
                     </span>
                     <span tagx="article-title">Arboviruses - structure and classification (author’s transl)</span>
                     <span tagx="source">Med Trop (Mars)</span>
                     <span tagx="year">1980</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">493</span>
                     <span tagx="lpage">498</span>
                     <span class="pub-id'">[7442509]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B3"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Heinz</span>
                        <span tagx="given-names">FX</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Allison</span>
                        <span tagx="given-names">SL</span>,
                     </span>
                     <span tagx="article-title">Structures and mechanisms in flavivirus fusion</span>
                     <span tagx="source">Adv Virus Res</span>
                     <span tagx="year">2000</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">231</span>
                     <span tagx="lpage">269</span>
                     <span class="pub-id'">[11050944]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B4"></a>
                  <div tagxxx="mixed-citation">
                     <span class="collab'">[World Health Organization]</span>
                     <span tagx="source">Dengue and dengue haemorrhagic fever Fact sheet No. 117</span>
                     <span tagx="year">2009</span>
                     <div tagxxx="comment">Available from: [
                        <div tagxxx="ext-link">http://www.who.int/mediacentre/factsheets/fs117/en/</div>]. Accessed on 18 October 2011
                     </div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B5"></a>
                  <div tagxxx="mixed-citation">
                     <span class="collab'">[World Health Organisation]</span>
                     <span tagx="source">Yellow fever Fact sheet N 100</span>
                     <span tagx="year">2009</span>
                     <div tagxxx="comment">Available from: [
                        <div tagxxx="ext-link">http://www.who.int/mediacentre/factsheets/fs100/en/</div>]. Accessed on 15 October 2011
                     </div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B6"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Bakonyi</span>
                        <span tagx="given-names">T</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Hubalek</span>
                        <span tagx="given-names">Z</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Rudolf</span>
                        <span tagx="given-names">I</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Nowotny</span>
                        <span tagx="given-names">N</span>,
                     </span>
                     <span tagx="article-title">Novel flavivirus or new lineage of West Nile virus, central Europe</span>
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="year">2005</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">225</span>
                     <span tagx="lpage">231</span>
                     <span class="pub-id'">[10.3201/eid1102.041028]</span>
                     <span class="pub-id'">[15752439]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B7"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Bakonyi</span>
                        <span tagx="given-names">T</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Ivanics</span>
                        <span tagx="given-names">E</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Erdelyi</span>
                        <span tagx="given-names">K</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Ursu</span>
                        <span tagx="given-names">K</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Ferenczi</span>
                        <span tagx="given-names">E</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Weissenbock</span>
                        <span tagx="given-names">H</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Nowotny</span>
                        <span tagx="given-names">N</span>,
                     </span>
                     <span tagx="article-title">Lineage 1 and 2 strains of encephalitic West Nile virus, central Europe</span>
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="year">2006</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">618</span>
                     <span tagx="lpage">623</span>
                     <span class="pub-id'">[10.3201/eid1204.051379]</span>
                     <span class="pub-id'">[16704810]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B8"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Venter</span>
                        <span tagx="given-names">M</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Human</span>
                        <span tagx="given-names">S</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Zaayman</span>
                        <span tagx="given-names">D</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Gerdes</span>
                        <span tagx="given-names">GH</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Williams</span>
                        <span tagx="given-names">J</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Steyl</span>
                        <span tagx="given-names">J</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Leman</span>
                        <span tagx="given-names">PA</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Paweska</span>
                        <span tagx="given-names">JT</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Setzkorn</span>
                        <span tagx="given-names">H</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Rous</span>
                        <span tagx="given-names">G</span>,
                     </span>
                     <span tagx="article-title">Lineage 2 west nile virus as cause of fatal neurologic disease in horses, South Africa</span>
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">877</span>
                     <span tagx="lpage">884</span>
                     <span class="pub-id'">[10.3201/eid1506.081515]</span>
                     <span class="pub-id'">[19523285]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B9"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Papa</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Danis</span>
                        <span tagx="given-names">K</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Baka</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Bakas</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Dougas</span>
                        <span tagx="given-names">G</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Lytras</span>
                        <span tagx="given-names">T</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Theocharopoulos</span>
                        <span tagx="given-names">G</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Chrysagis</span>
                        <span tagx="given-names">D</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Vassiliadou</span>
                        <span tagx="given-names">E</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Kamaria</span>
                        <span tagx="given-names">F</span>,
                     </span>
                     <span tagx="article-title">Ongoing outbreak of West Nile virus infections in humans in Greece, July-August 2010</span>
                     <span tagx="source">Euro Surveill</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">pii:19644</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B10"></a>
                  <div tagxxx="mixed-citation">
                     <span class="collab'">[Centers for Disease Control and Prevention]</span>
                     <span tagx="source">Japanese Encephalitis Fact Sheet</span>
                     <span class="publisher-name'">[Atlanta: CDC]</span>
                     <div tagxxx="comment">Available from: [
                        <div tagxxx="ext-link">http://www.cdc.gov/japaneseencephalitis/qa/index.html</div>]. Accessed on 10.02.2013.
                     </div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B11"></a>
                  <div tagxxx="mixed-citation">
                     <span class="collab'">[Tick-borne-diseases]</span>
                     <span tagx="article-title">Factsheet for health professionals</span>
                     <div tagxxx="comment">[
                        <div tagxxx="ext-link">http://ecdc.europa.eu/en/healthtopics/tick_borne_diseases/tick_borne_encephalitis/basic_facts/Pages/factsheet_health_professionals.aspx</div>]
                     </div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B12"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Aguero</span>
                        <span tagx="given-names">M</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Fernandez-Pinero</span>
                        <span tagx="given-names">J</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Buitrago</span>
                        <span tagx="given-names">D</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Sanchez</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Elizalde</span>
                        <span tagx="given-names">M</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">San Miguel</span>
                        <span tagx="given-names">E</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Villalba</span>
                        <span tagx="given-names">R</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Llorente</span>
                        <span tagx="given-names">F</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Jimenez-Clavero</span>
                        <span tagx="given-names">MA</span>,
                     </span>
                     <span tagx="article-title">Bagaza virus in partridges and pheasants, Spain, 2010</span>
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">1498</span>
                     <span tagx="lpage">1501</span>
                     <span class="pub-id'">[21801633]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B13"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Weissenbock</span>
                        <span tagx="given-names">H</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Kolodziejek</span>
                        <span tagx="given-names">J</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Url</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Lussy</span>
                        <span tagx="given-names">H</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Rebel-Bauder</span>
                        <span tagx="given-names">B</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Nowotny</span>
                        <span tagx="given-names">N</span>,
                     </span>
                     <span tagx="article-title">Emergence of Usutu virus, an African mosquito-borne flavivirus of the Japanese encephalitis
                        virus group, central Europe
                     </span>
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="year">2002</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">652</span>
                     <span tagx="lpage">656</span>
                     <span class="pub-id'">[10.3201/eid0807.020094]</span>
                     <span class="pub-id'">[12095429]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B14"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Haddow</span>
                        <span tagx="given-names">AD</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Schuh</span>
                        <span tagx="given-names">AJ</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Yasuda</span>
                        <span tagx="given-names">CY</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Kasper</span>
                        <span tagx="given-names">MR</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Heang</span>
                        <span tagx="given-names">V</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Huy</span>
                        <span tagx="given-names">R</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Guzman</span>
                        <span tagx="given-names">H</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Tesh</span>
                        <span tagx="given-names">RB</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Weaver</span>
                        <span tagx="given-names">SC</span>,
                     </span>
                     <span tagx="article-title">Genetic characterization of Zika virus strains: geographic expansion of the Asian
                        lineage
                     </span>
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="year">2012</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">e1477</span>
                     <span class="pub-id'">[10.1371/journal.pntd.0001477]</span>
                     <span class="pub-id'">[22389730]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B15"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Calzolari</span>
                        <span tagx="given-names">M</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Ze-Ze</span>
                        <span tagx="given-names">L</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Ruzek</span>
                        <span tagx="given-names">D</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Vazquez</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Jeffries</span>
                        <span tagx="given-names">C</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Defilippo</span>
                        <span tagx="given-names">F</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Osorio</span>
                        <span tagx="given-names">HC</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Kilian</span>
                        <span tagx="given-names">P</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Ruiz</span>
                        <span tagx="given-names">S</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Fooks</span>
                        <span tagx="given-names">AR</span>,
                     </span>
                     <span tagx="article-title">Detection of mosquito-only flaviviruses in Europe</span>
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="year">2012</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">1215</span>
                     <span tagx="lpage">1225</span>
                     <span class="pub-id'">[10.1099/vir.0.040485-0]</span>
                     <span class="pub-id'">[22377581]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B16"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Hoshino</span>
                        <span tagx="given-names">K</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Isawa</span>
                        <span tagx="given-names">H</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Tsuda</span>
                        <span tagx="given-names">Y</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Sawabe</span>
                        <span tagx="given-names">K</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Kobayashi</span>
                        <span tagx="given-names">M</span>,
                     </span>
                     <span tagx="article-title">Isolation and characterization of a new insect flavivirus from Aedes albopictus and
                        Aedes flavopictus mosquitoes in Japan
                     </span>
                     <span tagx="source">Virology</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">119</span>
                     <span tagx="lpage">129</span>
                     <span class="pub-id'">[10.1016/j.virol.2009.06.025]</span>
                     <span class="pub-id'">[19580982]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B17"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Tyler</span>
                        <span tagx="given-names">S</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Bolling</span>
                        <span tagx="given-names">BG</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Blair</span>
                        <span tagx="given-names">CD</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Brault</span>
                        <span tagx="given-names">AC</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Pabbaraju</span>
                        <span tagx="given-names">K</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Armijos</span>
                        <span tagx="given-names">MV</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Clark</span>
                        <span tagx="given-names">DC</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Calisher</span>
                        <span tagx="given-names">CH</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Drebot</span>
                        <span tagx="given-names">MA</span>,
                     </span>
                     <span tagx="article-title">Distribution and phylogenetic comparisons of a novel mosquito flavivirus sequence
                        present in Culex tarsalis Mosquitoes from western Canada with viruses isolated in
                        California and Colorado
                     </span>
                     <span tagx="source">Am J Trop Med Hyg</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">162</span>
                     <span tagx="lpage">168</span>
                     <span class="pub-id'">[10.4269/ajtmh.2011.10-0469]</span>
                     <span class="pub-id'">[21734143]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B18"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Vazquez</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Sanchez-Seco</span>
                        <span tagx="given-names">MP</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Palacios</span>
                        <span tagx="given-names">G</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Molero</span>
                        <span tagx="given-names">F</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Reyes</span>
                        <span tagx="given-names">N</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Ruiz</span>
                        <span tagx="given-names">S</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Aranda</span>
                        <span tagx="given-names">C</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Marques</span>
                        <span tagx="given-names">E</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Escosa</span>
                        <span tagx="given-names">R</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Moreno</span>
                        <span tagx="given-names">J</span>,
                     </span>
                     <span tagx="article-title">Novel flaviviruses detected in different species of mosquitoes in Spain</span>
                     <span tagx="source">Vector Borne Zoonotic Dis</span>
                     <span tagx="year">2012</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">223</span>
                     <span tagx="lpage">229</span>
                     <span class="pub-id'">[10.1089/vbz.2011.0687]</span>
                     <span class="pub-id'">[22022811]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B19"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Junglen</span>
                        <span tagx="given-names">S</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Kopp</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Kurth</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Pauli</span>
                        <span tagx="given-names">G</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Ellerbrok</span>
                        <span tagx="given-names">H</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Leendertz</span>
                        <span tagx="given-names">FH</span>,
                     </span>
                     <span tagx="article-title">A new flavivirus and a new vector: characterization of a novel flavivirus isolated
                        from uranotaenia mosquitoes from a tropical rain forest
                     </span>
                     <span tagx="source">J Virol</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">4462</span>
                     <span tagx="lpage">4468</span>
                     <span class="pub-id'">[10.1128/JVI.00014-09]</span>
                     <span class="pub-id'">[19224998]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B20"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Farfan-Ale</span>
                        <span tagx="given-names">JA</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Lorono-Pino</span>
                        <span tagx="given-names">MA</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Garcia-Rejon</span>
                        <span tagx="given-names">JE</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Hovav</span>
                        <span tagx="given-names">E</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Powers</span>
                        <span tagx="given-names">AM</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Lin</span>
                        <span tagx="given-names">M</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Dorman</span>
                        <span tagx="given-names">KS</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Platt</span>
                        <span tagx="given-names">KB</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Bartholomay</span>
                        <span tagx="given-names">LC</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Soto</span>
                        <span tagx="given-names">V</span>,
                     </span>
                     <span tagx="article-title">Detection of RNA from a novel West Nile-like virus and high prevalence of an insect-specific
                        flavivirus in mosquitoes in the Yucatan Peninsula of Mexico
                     </span>
                     <span tagx="source">Am J Trop Med Hyg</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">85</span>
                     <span tagx="lpage">95</span>
                     <span class="pub-id'">[19141845]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B21"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Huhtamo</span>
                        <span tagx="given-names">E</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Putkuri</span>
                        <span tagx="given-names">N</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Kurkela</span>
                        <span tagx="given-names">S</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Manni</span>
                        <span tagx="given-names">T</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Vaheri</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Vapalahti</span>
                        <span tagx="given-names">O</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Uzcategui</span>
                        <span tagx="given-names">NY</span>,
                     </span>
                     <span tagx="article-title">Characterization of a novel flavivirus from mosquitoes in northern europe that is
                        related to mosquito-borne flaviviruses of the tropics
                     </span>
                     <span tagx="source">J Virol</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">9532</span>
                     <span tagx="lpage">9540</span>
                     <span class="pub-id'">[10.1128/JVI.00529-09]</span>
                     <span class="pub-id'">[19570865]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B22"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Yun</span>
                        <span tagx="given-names">T</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Zhang</span>
                        <span tagx="given-names">D</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Ma</span>
                        <span tagx="given-names">X</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Cao</span>
                        <span tagx="given-names">Z</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Chen</span>
                        <span tagx="given-names">L</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Ni</span>
                        <span tagx="given-names">Z</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Ye</span>
                        <span tagx="given-names">W</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Yu</span>
                        <span tagx="given-names">B</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Hua</span>
                        <span tagx="given-names">J</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Zhang</span>
                        <span tagx="given-names">Y</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Zhang</span>
                        <span tagx="given-names">C</span>,
                     </span>
                     <span tagx="article-title">Complete genome sequence of a novel flavivirus, duck tembusu virus, isolated from
                        ducks and geese in china
                     </span>
                     <span tagx="source">J Virol</span>
                     <span tagx="year">2012</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">3406</span>
                     <span tagx="lpage">3407</span>
                     <span class="pub-id'">[10.1128/JVI.07132-11]</span>
                     <span class="pub-id'">[22354945]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B23"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Lanciotti</span>
                        <span tagx="given-names">RS</span>,
                     </span>
                     <span tagx="article-title">Molecular amplification assays for the detection of flaviviruses</span>
                     <span tagx="source">Adv Virus Res</span>
                     <span tagx="year">2003</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">67</span>
                     <span tagx="lpage">99</span>
                     <span class="pub-id'">[14714430]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B24"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Domingo</span>
                        <span tagx="given-names">C</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Patel</span>
                        <span tagx="given-names">P</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Linke</span>
                        <span tagx="given-names">S</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Achazi</span>
                        <span tagx="given-names">K</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Niedrig</span>
                        <span tagx="given-names">M</span>,
                     </span>
                     <span tagx="article-title">Molecular diagnosis of flaviviruses</span>
                     <span tagx="source">Future Virology</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">1059</span>
                     <span tagx="lpage">1074</span>
                     <span class="pub-id'">[10.2217/fvl.11.77]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B25"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Sekaran</span>
                        <span tagx="given-names">SD</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Artsob</span>
                        <span tagx="given-names">H</span>,
                     </span>
                     <span tagx="article-title">Molecular diagnostics for the detection of human flavivirus infections</span>
                     <span tagx="source">Expert Opinion on Medical Diagnostics</span>
                     <span tagx="year">2007</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">521</span>
                     <span tagx="lpage">530</span>
                     <span class="pub-id'">[10.1517/17530059.1.4.521]</span>
                     <span class="pub-id'">[23496358]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B26"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Johnson</span>
                        <span tagx="given-names">N</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Wakeley</span>
                        <span tagx="given-names">PR</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Mansfield</span>
                        <span tagx="given-names">KL</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">McCracken</span>
                        <span tagx="given-names">F</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Haxton</span>
                        <span tagx="given-names">B</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Phipps</span>
                        <span tagx="given-names">LP</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Fooks</span>
                        <span tagx="given-names">AR</span>,
                     </span>
                     <span tagx="article-title">Assessment of a novel real-time pan-flavivirus RT-polymerase chain reaction</span>
                     <span tagx="source">Vector Borne Zoonotic Dis</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">665</span>
                     <span tagx="lpage">671</span>
                     <span class="pub-id'">[10.1089/vbz.2009.0210]</span>
                     <span class="pub-id'">[20854019]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B27"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Scaramozzino</span>
                        <span tagx="given-names">N</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Crance</span>
                        <span tagx="given-names">JM</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Jouan</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">DeBriel</span>
                        <span tagx="given-names">DA</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Stoll</span>
                        <span tagx="given-names">F</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Garin</span>
                        <span tagx="given-names">D</span>,
                     </span>
                     <span tagx="article-title">Comparison of flavivirus universal primer pairs and development of a rapid, highly
                        sensitive heminested reverse transcription-PCR assay for detection of flaviviruses
                        targeted to a conserved region of the NS5 gene sequences
                     </span>
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="year">2001</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">1922</span>
                     <span tagx="lpage">1927</span>
                     <span class="pub-id'">[10.1128/JCM.39.5.1922-1927.2001]</span>
                     <span class="pub-id'">[11326014]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B28"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Chien</span>
                        <span tagx="given-names">LJ</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Liao</span>
                        <span tagx="given-names">TL</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Shu</span>
                        <span tagx="given-names">PY</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Huang</span>
                        <span tagx="given-names">JH</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Gubler</span>
                        <span tagx="given-names">DJ</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Chang</span>
                        <span tagx="given-names">GJ</span>,
                     </span>
                     <span tagx="article-title">Development of real-time reverse transcriptase PCR assays to detect and serotype dengue
                        viruses
                     </span>
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="year">2006</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">1295</span>
                     <span tagx="lpage">1304</span>
                     <span class="pub-id'">[10.1128/JCM.44.4.1295-1304.2006]</span>
                     <span class="pub-id'">[16597854]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B29"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Domingo</span>
                        <span tagx="given-names">C</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Niedrig</span>
                        <span tagx="given-names">M</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Teichmann</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Kaiser</span>
                        <span tagx="given-names">M</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Rumer</span>
                        <span tagx="given-names">L</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Jarman</span>
                        <span tagx="given-names">RG</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Donoso-Mantke</span>
                        <span tagx="given-names">O</span>,
                     </span>
                     <span tagx="article-title">2nd International external quality control assessment for the molecular diagnosis
                        of dengue infections
                     </span>
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">e833</span>
                     <span class="pub-id'">[10.1371/journal.pntd.0000833]</span>
                     <span class="pub-id'">[20957194]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B30"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Weidmann</span>
                        <span tagx="given-names">M</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Faye</span>
                        <span tagx="given-names">O</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Kranaster</span>
                        <span tagx="given-names">R</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Marx</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Nunes</span>
                        <span tagx="given-names">MR</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Vasconcelos</span>
                        <span tagx="given-names">PF</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Hufert</span>
                        <span tagx="given-names">FT</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Sall</span>
                        <span tagx="given-names">AA</span>,
                     </span>
                     <span tagx="article-title">Improved LNA probe-based assay for the detection of African and South American yellow
                        fever virus strains
                     </span>
                     <span tagx="source">J Clin Virol</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">187</span>
                     <span tagx="lpage">192</span>
                     <span class="pub-id'">[10.1016/j.jcv.2010.04.013]</span>
                     <span class="pub-id'">[20556888]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B31"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Linke</span>
                        <span tagx="given-names">S</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Ellerbrok</span>
                        <span tagx="given-names">H</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Niedrig</span>
                        <span tagx="given-names">M</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Nitsche</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Pauli</span>
                        <span tagx="given-names">G</span>,
                     </span>
                     <span tagx="article-title">Detection of West Nile virus lineages 1 and 2 by real-time PCR</span>
                     <span tagx="source">J Virol Methods</span>
                     <span tagx="year">2007</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">355</span>
                     <span tagx="lpage">358</span>
                     <span class="pub-id'">[10.1016/j.jviromet.2007.05.021]</span>
                     <span class="pub-id'">[17604132]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B32"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Achazi</span>
                        <span tagx="given-names">K</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Nitsche</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Patel</span>
                        <span tagx="given-names">P</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Radonic</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Donoso Mantke</span>
                        <span tagx="given-names">O</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Niedrig</span>
                        <span tagx="given-names">M</span>,
                     </span>
                     <span tagx="article-title">Detection and differentiation of tick-borne encephalitis virus subtypes by a reverse
                        transcription quantitative real-time PCR and pyrosequencing
                     </span>
                     <span tagx="source">J Virol Methods</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">34</span>
                     <span tagx="lpage">39</span>
                     <span class="pub-id'">[10.1016/j.jviromet.2010.09.026]</span>
                     <span class="pub-id'">[20933016]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B33"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Pierre</span>
                        <span tagx="given-names">V</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Drouet</span>
                        <span tagx="given-names">MT</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Deubel</span>
                        <span tagx="given-names">V</span>,
                     </span>
                     <span tagx="article-title">Identification of mosquito-borne flavivirus sequences using universal primers and
                        reverse transcription/polymerase chain reaction
                     </span>
                     <span tagx="source">Res Virol</span>
                     <span tagx="year">1994</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">93</span>
                     <span tagx="lpage">104</span>
                     <span class="pub-id'">[7520190]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B34"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Domingo</span>
                        <span tagx="given-names">C</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Escadafal</span>
                        <span tagx="given-names">C</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Rumer</span>
                        <span tagx="given-names">L</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Mendez</span>
                        <span tagx="given-names">JA</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Garcia</span>
                        <span tagx="given-names">P</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Sall</span>
                        <span tagx="given-names">AA</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Teichmann</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Donoso-Mantke</span>
                        <span tagx="given-names">O</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Niedrig</span>
                        <span tagx="given-names">M</span>,
                     </span>
                     <span tagx="article-title">First international external quality assessment study on molecular and serological
                        methods for yellow fever diagnosis
                     </span>
                     <span tagx="source">PLoS One</span>
                     <span tagx="year">2012</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">e36291</span>
                     <span class="pub-id'">[10.1371/journal.pone.0036291]</span>
                     <span class="pub-id'">[22570700]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B35"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Figueiredo</span>
                        <span tagx="given-names">LT</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Batista</span>
                        <span tagx="given-names">WC</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Kashima</span>
                        <span tagx="given-names">S</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Nassar</span>
                        <span tagx="given-names">ES</span>,
                     </span>
                     <span tagx="article-title">Identification of Brazilian flaviviruses by a simplified reverse transcription-polymerase
                        chain reaction method using Flavivirus universal primers
                     </span>
                     <span tagx="source">Am J Trop Med Hyg</span>
                     <span tagx="year">1998</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">357</span>
                     <span tagx="lpage">362</span>
                     <span class="pub-id'">[9749625]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B36"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Roiz</span>
                        <span tagx="given-names">D</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Vazquez</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Seco</span>
                        <span tagx="given-names">MP</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Tenorio</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Rizzoli</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span tagx="article-title">Detection of novel insect flavivirus sequences integrated in Aedes albopictus (Diptera:
                        Culicidae) in Northern Italy
                     </span>
                     <span tagx="source">Virol J</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">93</span>
                     <span class="pub-id'">[10.1186/1743-422X-6-93]</span>
                     <span class="pub-id'">[19575816]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B37"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gijavanekar</span>
                        <span tagx="given-names">C</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Anez-Lingerfelt</span>
                        <span tagx="given-names">M</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Feng</span>
                        <span tagx="given-names">C</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Putonti</span>
                        <span tagx="given-names">C</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Fox</span>
                        <span tagx="given-names">GE</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Sabo</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Fofanov</span>
                        <span tagx="given-names">Y</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Willson</span>
                        <span tagx="given-names">RC</span>,
                     </span>
                     <span tagx="article-title">PCR detection of nearly any dengue virus strain using a highly sensitive primer ‘cocktail’</span>
                     <span tagx="source">FEBS J</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">1676</span>
                     <span tagx="lpage">1687</span>
                     <span class="pub-id'">[10.1111/j.1742-4658.2011.08091.x]</span>
                     <span class="pub-id'">[21410643]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B38"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Wolfel</span>
                        <span tagx="given-names">R</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Paweska</span>
                        <span tagx="given-names">JT</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Petersen</span>
                        <span tagx="given-names">N</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Grobbelaar</span>
                        <span tagx="given-names">AA</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Leman</span>
                        <span tagx="given-names">PA</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Hewson</span>
                        <span tagx="given-names">R</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Georges-Courbot</span>
                        <span tagx="given-names">MC</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Papa</span>
                        <span tagx="given-names">A</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Gunther</span>
                        <span tagx="given-names">S</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Drosten</span>
                        <span tagx="given-names">C</span>,
                     </span>
                     <span tagx="article-title">Virus detection and monitoring of viral load in Crimean-Congo hemorrhagic fever virus
                        patients
                     </span>
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="year">2007</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">1097</span>
                     <span tagx="lpage">1100</span>
                     <span class="pub-id'">[10.3201/eid1307.070068]</span>
                     <span class="pub-id'">[18214191]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B39"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Tolstrup</span>
                        <span tagx="given-names">N</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Nielsen</span>
                        <span tagx="given-names">PS</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Kolberg</span>
                        <span tagx="given-names">JG</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Frankel</span>
                        <span tagx="given-names">AM</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Vissing</span>
                        <span tagx="given-names">H</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Kauppinen</span>
                        <span tagx="given-names">S</span>,
                     </span>
                     <span tagx="article-title">OligoDesign: Optimal design of LNA (locked nucleic acid) oligonucleotide capture probes
                        for gene expression profiling
                     </span>
                     <span tagx="source">Nucleic Acids Res</span>
                     <span tagx="year">2003</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">3758</span>
                     <span tagx="lpage">3762</span>
                     <span class="pub-id'">[10.1093/nar/gkg580]</span>
                     <span class="pub-id'">[12824412]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B40"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Yao</span>
                        <span tagx="given-names">Y</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Nellaker</span>
                        <span tagx="given-names">C</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Karlsson</span>
                        <span tagx="given-names">H</span>,
                     </span>
                     <span tagx="article-title">Evaluation of minor groove binding probe and Taqman probe PCR assays: Influence of
                        mismatches and template complexity on quantification
                     </span>
                     <span tagx="source">Mol Cell Probes</span>
                     <span tagx="year">2006</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">311</span>
                     <span tagx="lpage">316</span>
                     <span class="pub-id'">[10.1016/j.mcp.2006.03.003]</span>
                     <span class="pub-id'">[16704921]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B41"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Frieden</span>
                        <span tagx="given-names">M</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Hansen</span>
                        <span tagx="given-names">HF</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Koch</span>
                        <span tagx="given-names">T</span>,
                     </span>
                     <span tagx="article-title">Nuclease stability of LNA oligonucleotides and LNA-DNA chimeras</span>
                     <span tagx="source">Nucleosides Nucleotides Nucleic Acids</span>
                     <span tagx="year">2003</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">1041</span>
                     <span tagx="lpage">1043</span>
                     <span class="pub-id'">[10.1081/NCN-120022731]</span>
                     <span class="pub-id'">[14565339]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B42"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Johnson</span>
                        <span tagx="given-names">MP</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Haupt</span>
                        <span tagx="given-names">LM</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Griffiths</span>
                        <span tagx="given-names">LR</span>,
                     </span>
                     <span tagx="article-title">Locked nucleic acid (LNA) single nucleotide polymorphism (SNP) genotype analysis and
                        validation using real-time PCR
                     </span>
                     <span tagx="source">Nucleic Acids Res</span>
                     <span tagx="year">2004</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">e55</span>
                     <span class="pub-id'">[10.1093/nar/gnh046]</span>
                     <span class="pub-id'">[15047860]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B43"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Letertre</span>
                        <span tagx="given-names">C</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Perelle</span>
                        <span tagx="given-names">S</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Dilasser</span>
                        <span tagx="given-names">F</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Arar</span>
                        <span tagx="given-names">K</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Fach</span>
                        <span tagx="given-names">P</span>,
                     </span>
                     <span tagx="article-title">Evaluation of the performance of LNA and MGB probes in 5′-nuclease PCR assays</span>
                     <span tagx="source">Mol Cell Probes</span>
                     <span tagx="year">2003</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">307</span>
                     <span tagx="lpage">311</span>
                     <span class="pub-id'">[10.1016/j.mcp.2003.08.004]</span>
                     <span class="pub-id'">[14602482]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B44"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Reynisson</span>
                        <span tagx="given-names">E</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Josefsen</span>
                        <span tagx="given-names">MH</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Krause</span>
                        <span tagx="given-names">M</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Hoorfar</span>
                        <span tagx="given-names">J</span>,
                     </span>
                     <span tagx="article-title">Evaluation of probe chemistries and platforms to improve the detection limit of real-time
                        PCR
                     </span>
                     <span tagx="source">J Microbiol Methods</span>
                     <span tagx="year">2006</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">206</span>
                     <span tagx="lpage">216</span>
                     <span class="pub-id'">[10.1016/j.mimet.2005.11.006]</span>
                     <span class="pub-id'">[16364478]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B45"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Ugozzoli</span>
                        <span tagx="given-names">LA</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Latorra</span>
                        <span tagx="given-names">D</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Puckett</span>
                        <span tagx="given-names">R</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Arar</span>
                        <span tagx="given-names">K</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Hamby</span>
                        <span tagx="given-names">K</span>,
                     </span>
                     <span tagx="article-title">Real-time genotyping with oligonucleotide probes containing locked nucleic acids</span>
                     <span tagx="source">Anal Biochem</span>
                     <span tagx="year">2004</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">143</span>
                     <span tagx="lpage">152</span>
                     <span class="pub-id'">[10.1016/j.ab.2003.09.003]</span>
                     <span class="pub-id'">[14654057]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="B46"></a>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Wacker</span>
                        <span tagx="given-names">MJ</span>,
                     </span>
                     <span class="name'">
                        <span tagx="surname">Godard</span>
                        <span tagx="given-names">MP</span>,
                     </span>
                     <span tagx="article-title">Analysis of one-step and two-step real-time RT-PCR using SuperScript III</span>
                     <span tagx="source">J Biomol Tech</span>
                     <span tagx="year">2005</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">266</span>
                     <span tagx="lpage">271</span>
                     <span class="pub-id'">[16461951]</span>
                  </div>
               </li>
            </ul>
         </div>
      </div>
   </div>
</html>