<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <div xmlns="" tagx="article">
      <head>
         <style type="text/css"> table, tr, td {border : thin;}</style>
         <span class="citation_publisher">Oxford University Press</span>
         <h2 class="citation-title">A new way to protect privacy in large-scale genome-wide association studies</h2>
         <span class="doi">10.1093/bioinformatics/btt066</span>
         <div class="contrib-group">contrib: 
            	  CONT
            	  
            <span class="citation_author">Liina Kamm</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Dan Bogdanov</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Sven Laur</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Jaak Vilo</span>
            <span class="citation_author_institution_ref">1</span>
         </div>
         <span class="email">[]</span>
         <span class="conflict">[]</span>
         <span class="contribs">[]</span>
         <span class="citation_pubdate">[2013-2-14]</span>
         <span class="citation_copyright">[Copyright 2013: ]</span>
         <span class="citation_licence">[Licence This is an Open Access article distributed under the terms of the Creative
            Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which permits
            unrestricted reuse, distribution, and reproduction in any medium, provided the original
            work is properly cited.]
         </span>
         <span class="citation_funding">[]</span>
         <span class="citation_pagecount">[]</span>
         <span class="citation_data">[]</span>
      </head>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
            <b>Motivation:</b> Increased availability of various genotyping techniques has initiated a race for
            finding genetic markers that can be used in diagnostics and personalized medicine.
            Although many genetic risk factors are known, key causes of common diseases with complex
            heritage patterns are still unknown. Identification of such complex traits requires
            a targeted study over a large collection of data. Ideally, such studies bring together
            data from many biobanks. However, data aggregation on such a large scale raises many
            privacy issues.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
            <b>Results:</b> We show how to conduct such studies without violating privacy of individual donors
            and without leaking the data to third parties. The presented solution has provable
            security guarantees.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
            <b>Contact:</b>
            <div tagxxx="email">jaak.vilo@ut.ee</div>
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
            <b>Supplementary information:</b>
            <div tagxxx="ext-link">Supplementary data</div> are available at 
            <i>Bioinformatics</i> online.
         </p>
      </div>
      <div id="SEC1">
         <h2>1 INTRODUCTION</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Genome-wide association studies (GWAS) are one of the driving reasons behind the formation
            of nationwide and privately funded gene banks. Many chronic diseases and various cancer
            types are known to have genetic disposition factors (
            <span ref-type="bibr" rid="btt066-B7">
               <sup>
                  <a href="#btt066-B7">Chakravarti and Little, 2003</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="btt066-B20">
               <sup>
                  <a href="#btt066-B20">Lander and Schork, 1994</a>
               </sup>
            </span>). Although many underlying genetic signatures have been successfully identified for
            Mendelian disorders (
            <span ref-type="bibr" rid="btt066-B15">
               <sup>
                  <a href="#btt066-B15">Hamosh et al., 2005</a>
               </sup>
            </span>), not many genetic risk factors for complex diseases have been discovered and confirmed.
            GWAS have identified some risk factors for type II diabetes (
            <span ref-type="bibr" rid="btt066-B29">
               <sup>
                  <a href="#btt066-B29">Prokopenko et al., 2008</a>
               </sup>
            </span>) and for a few other common diseases (
            <span ref-type="bibr" rid="btt066-B25">
               <sup>
                  <a href="#btt066-B25">Manolio et al., 2008</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="btt066-B38">
               <sup>
                  <a href="#btt066-B38">Wellcome Trust Case Control Consortium, 2007</a>
               </sup>
            </span>). GWAS have been modestly successful in pharmacogenetics (
            <span ref-type="bibr" rid="btt066-B13">
               <sup>
                  <a href="#btt066-B13">Grant and Hakonarson, 2007</a>
               </sup>
            </span>) and cancer research (
            <span ref-type="bibr" rid="btt066-B35">
               <sup>
                  <a href="#btt066-B35">Varghese and Easton, 2010</a>
               </sup>
            </span>). The size and structure of a study cohort are the main limiting factors in such
            studies, as the individual impact of genomic differences is usually small. Larger
            sample sizes increase the sensitivity of statistical tests and make it possible to
            apply a wide range of data-mining methods (
            <span ref-type="bibr" rid="btt066-B26">
               <sup>
                  <a href="#btt066-B26">Moore et al., 2010</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="btt066-B34">
               <sup>
                  <a href="#btt066-B34">Szymczak et al., 2009</a>
               </sup>
            </span>).
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Ideally, studies should use nationwide and continent-wide patient cohorts. Formation
            of such cohorts is becoming feasible as genotyping costs are rapidly decreasing (
            <span ref-type="bibr" rid="btt066-B16">
               <sup>
                  <a href="#btt066-B16">Hayden, 2010</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="btt066-B28">
               <sup>
                  <a href="#btt066-B28">Pettersson et al., 2009</a>
               </sup>
            </span>). In addition to nationwide biobanks, e.g. the UK Biobank, several personal genomics
            companies, such as 23andMe and Navigenics, already possess large and diverse patient
            cohorts. Biobanks are also forming large collaboration networks, such as P3G and HuGENet,
            to combine their patient cohorts and improve study quality. Privacy of individual
            gene donors is one of the biggest concerns in such projects. In many countries, genotype
            data are classified as sensitive data that can be handled by complying with specific
            restrictions, e.g. HIPAA in the USA and the Data Protection Directive in the European
            Union. These restrictions are justified, as a leak of genetic information can cause
            genome-based discrimination when more health-related patterns have been discovered.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Standard anonymization methods are not applicable to genotype data, as the data themselves
            are an ultimate identity code. Only 30–80 out of 30 million single-nucleotide polymorphisms
            (SNPs) are needed to uniquely identify a person (
            <span ref-type="bibr" rid="btt066-B21">
               <sup>
                  <a href="#btt066-B21">Lin et al., 2004</a>
               </sup>
            </span>). Moreover, the size of online genotype databases for genealogy studies, such as
            SGMF and YHRD, has made re-identification of anonymized genotype data a real threat
            (
            <span ref-type="bibr" rid="btt066-B14">
               <sup>
                  <a href="#btt066-B14">Gymrek et al., 2013</a>
               </sup>
            </span>). Re-identification attacks (
            <span ref-type="bibr" rid="btt066-B23">
               <sup>
                  <a href="#btt066-B23">Malin and Sweeney, 2000</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="btt066-B24">
               <sup>
                  <a href="#btt066-B24">2002</a>
               </sup>
            </span>) based on combining inferred phenotypes with public data become practical, as the
            list of known associations between genotype and phenotypic traits (
            <span ref-type="bibr" rid="btt066-B17">
               <sup>
                  <a href="#btt066-B17">Hindorff et al., 2009</a>
               </sup>
            </span>) evolves. Finally, 
            <span ref-type="bibr" rid="btt066-B18">
               <sup>
                  <a href="#btt066-B18">Homer et al. (2008)</a>
               </sup>
            </span> showed that even aggregated pools of genomic data can leak private information. Although
            follow-up studies (
            <span ref-type="bibr" rid="btt066-B37">
               <sup>
                  <a href="#btt066-B37">Visscher and Hill, 2009</a>
               </sup>
            </span>) softened initial claims, the threat remains.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">These findings created a debate whether one can promise privacy of genotype data in
            consent forms at all (
            <span ref-type="bibr" rid="btt066-B27">
               <sup>
                  <a href="#btt066-B27">P3G Consortium et al., 2009</a>
               </sup>
            </span>). In the following, we show how to set up an infrastructure where the genotype data
            can be stored and processed so that none of the peers involved in the process can
            reconstruct the data, and thus the risk of accidental leaks and malicious data abuse
            is greatly reduced. The data analysis algorithms are executed in an oblivious manner
            so that only the desired outcome is revealed to the user and nothing else. Differently
            from well-known data perturbation and masking techniques (
            <span ref-type="bibr" rid="btt066-B22">
               <sup>
                  <a href="#btt066-B22">Machanavajjhala et al., 2007</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="btt066-B33">
               <sup>
                  <a href="#btt066-B33">Sweeney, 2002</a>
               </sup>
            </span>), security guarantees are cryptographic. These guarantees depend on the computational
            complexity of well-established mathematical problems and not on the background knowledge
            of potential attackers. As such, the presented methodology is applicable to protecting
            biobanks and other medical databases.
         </p>
      </div>
      <div id="SEC2">
         <h2>2 APPLICATION SCENARIOS</h2>
         <div id="SEC2.1">
            <h3>2.1 Privacy threats in medical studies</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">There are four principal groups of stakeholders in a typical medical study: data donors,
               data collectors, data analysts and supervisory organizations. Data donors consent
               to give their tissue samples and record other types of medical data, e.g. various
               questionnaires covering health issues. Data collectors are responsible for gathering
               and storing the data and keeping the donors’ confidentiality. The baseline requirements
               are forced by the laws, e.g. by the Genetic Information Nondiscrimination Act of 2008
               in the USA. However, more stringent privacy guarantees can reduce data donors’ fears
               about data abuse and increase the participation rate.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Data collectors and supervisory organizations must guarantee that the data analysts
               (researchers) meet privacy restrictions. The confidentiality problem is somewhat smaller
               if the data analysts are from the same institution as the data collectors. If the
               analysts are not part of the organization, usually confidentiality agreements are
               signed between the parties. In most cases, collected data are stored in several databases
               so that direct personal information is not accessible to researchers. Instead, each
               patient gets a (pseudo) random barcode that links different databases together (
               <span ref-type="bibr" rid="btt066-B10">
                  <sup>
                     <a href="#btt066-B10">De Moor et al., 2003</a>
                  </sup>
               </span>). On rare occasions, databases are merged either to identify specific persons or
               to form datasets needed for medical research.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">All such solutions provide only partial security guarantees for data donors. Even
               if genetic data are stored separately and are not directly accessible, they must be
               (partially) released before the actual data analysis is carried out. Hence, a single
               fault by a data analyst or an insider attack might effectively obliterate all privacy
               guarantees. Therefore, compromises in confidentiality are needed for the creation
               of worldwide data banks for the scientific community.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">These privacy issues are often alleviated with data perturbation and agglomeration
               techniques. For instance, the National Institutes of Health in the USA published the
               ratio of SNP alleles of various case–control studies (
               <span ref-type="bibr" rid="btt066-B8">
                  <sup>
                     <a href="#btt066-B8">Couzin, 2008</a>
                  </sup>
               </span>) because it is essentially impossible to split a DNA mixture back to individual genotypes.
               However, it turned out that it is possible to detect whether a specific person is
               in the mixture or not (
               <span ref-type="bibr" rid="btt066-B18">
                  <sup>
                     <a href="#btt066-B18">Homer et al., 2008</a>
                  </sup>
               </span>). As case–control groups are often based on sensitive information, the data had to
               be removed.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Such unexpected security breaches are common for 
               <i>ad hoc</i> perturbation or agglomeration methods because one often overlooks the effect of potential
               background knowledge to security. Although for certain problems perturbation and agglomeration
               methods can provide provable security, their overall applicability is limited and
               there are many known impossibility results (
               <span ref-type="bibr" rid="btt066-B11">
                  <sup>
                     <a href="#btt066-B11">Dwork, 2011</a>
                  </sup>
               </span>).
            </p>
         </div>
         <div id="SEC2.2">
            <h3>2.2 A novel solution based on distributed storage</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In this work, we propose a data collection system where sensitive data are secret
               shared among several independent entities. In brief, secret sharing assures that each
               party gets completely random-looking data. However, when all parties pool the data
               together sensitive values can be restored (see 
               <span ref-type="fig" rid="btt066-F1">Fig. 1</span> for an example of how secret sharing works). Depending on the exact nature of the
               used secret sharing scheme, the privacy of shared values is preserved even if some
               of the parties holding shares collude (discussed further in the methods section).
               Hence, data can be shared without the fear of unexpected disclosure. Moreover, with
               the use of specific multi-party computation techniques, computations on secret shared
               data can be carried out without leaking any information (
               <span ref-type="bibr" rid="btt066-B3">
                  <sup>
                     <a href="#btt066-B3">Bogdanov et al., 2008</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="btt066-B9">
                  <sup>
                     <a href="#btt066-B9">Damgård et al., 2009</a>
                  </sup>
               </span>). As a result, one can deploy a distributed computation environment that securely
               collects the data, does oblivious computations and returns the desired end results.
               Such systems have been successfully used for securing auctions (
               <span ref-type="bibr" rid="btt066-B5">
                  <sup>
                     <a href="#btt066-B5">Bogetoft et al., 2009</a>
                  </sup>
               </span>) and analyzing financial data (
               <span ref-type="bibr" rid="btt066-B4">
                  <sup>
                     <a href="#btt066-B4">Bogdanov et al., 2012</a>
                  </sup>
               </span>).
               
               <div tagxxx="fig">
                  <span class="label">Fig. 1.</span>
                  <caption>
                     <p>This figure illustrates how players A and B use a 
                        <i>3-out-of-3</i> additive secret sharing scheme to distribute two 32-bit integer values 
                        <i>x</i> and 
                        <i>y</i> to shares. The shares are sent to three servers that use the homomorphic property
                        of the scheme to securely compute the sum of 
                        <i>x</i> and 
                        <i>y</i>. The shares of the sum are sent to player C, who reconstructs the result
                     </p>
                  </caption>
                  <div tagxxx="graphic"></div>
               </div>
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">There are other cryptographically secure computation techniques, e.g. (fully) homomorphic
               encryption. However, these techniques are significantly slower and less feasible on
               the large genome databases.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <span ref-type="fig" rid="btt066-F2">Figure 2</span> depicts the overall workflow of secure GWAS. The core of such a system consists of
               three or more dedicated data centers (
               <i>hosts</i>) that are assumed to be independent organizations. For a worldwide study, these can
               be biobanks of different countries, regulatory authorities and patient interest groups.
               None of them have to be unconditionally trusted as long as too many of them do not
               collude with others. In particular, a successful attack against one or two of the
               data centers leaks no information and there even exists a recovery procedure from
               such attacks.
               
               <div tagxxx="fig">
                  <span class="label">Fig. 2.</span>
                  <caption>
                     <p>Secure GWAS consists of three major stages: data acquisition, formation of case–control
                        groups and statistical testing. Panel (
                        <b>A</b>) depicts how these three stages are linked. Data are gathered and sent in securely
                        coded shares to be stored. For statistical tests, case and control info is securely
                        coded and applied to the securely stored data so that statistical analyses can be
                        carried out. Panel (
                        <b>B</b>) describes two alternative scenarios that can be used for secure storage of genotype
                        and phenotype data. Scenario 1 depicts a situation where genotype data are entered
                        into secure storage by the wetlab and phenotype data are entered by the donors themselves.
                        Scenario 2 depicts a case where different gene banks send selected genotype and phenotype
                        data to secure storage so they can make joint analyses on more data. Panel (
                        <b>C</b>) describes how case and control groups can be formed. In the simplest case, researches
                        have unrestricted access to phenotype data and can thus form case and control groups
                        by themselves. In more complex settings, researchers do not have rights to access
                        phenotype data, and hosts must use secure multi-party computations to construct case
                        and control groups based on inclusion criteria
                     </p>
                  </caption>
                  <div tagxxx="graphic"></div>
               </div>
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Genomic data are entered into the system by primary data collectors, e.g. wetlabs
               or different biobanks who collect and process the biological samples and who want
               to make joint analyses on shared data. For this operation, standard clinical protocols
               for genotyping are sufficient for security. At the end of the stage, the data are
               secret shared and transferred securely to the hosts.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Clinical data can be entered into the system either by the data donors themselves
               or by primary data collectors. The data are sent to the secure phenotype database
               in secret shared form to ensure confidentiality. In the analysis stage, the data analyst
               specifies the algorithm to be run on the secret shared data and waits for the results.
               For GWAS, the analyst first forms case and control groups. Next, the algorithm computes
               the necessary statistics and, finally, it releases the loci that show a statistically
               significant differentiation according to the specified case–control groups. If the
               analyst wants to further examine a secured input value, all parties hosting the system
               must agree to disclose the respective shares.
            </p>
         </div>
         <div id="SEC2.3">
            <h3>2.3 Potential advantages and drawbacks</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We acknowledge that standard security measures are sufficient when the data are collected
               and analyzed by a single organization. Still, long-term projects can benefit from
               distributed data storage. First, a break-in into a data center yields no usable information.
               Second, splitting the data among independent organizations gives additional guarantees
               for the data donors, e.g. participants of commercial studies have no way of knowing
               what happens to their data if the project goes bankrupt. If one core center belongs
               to the state, participants know with greater certainty that their data cannot be abused.
               Third, other sources of private data can be incorporated into the analysis without
               privacy leaks. In particular, medical institutions can use their patient records to
               enhance analysis. The proposed solution provides a way to conduct the analysis so
               that neither the gene bank nor the medical institution releases their data.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The benefits of our approach are evident in collaborative studies between independent
               biobanks. As nothing beyond the desired test results are revealed during the computation,
               the solution provides superior privacy guarantees compared with alternatives based
               on meta-analysis techniques (
               <span ref-type="bibr" rid="btt066-B39">
                  <sup>
                     <a href="#btt066-B39">Wolfson et al., 2010</a>
                  </sup>
               </span>), where each biobank first computes local summaries that are collaboratively merged
               into a final result. As a result, only a few summary values are disclosed. However,
               it is impossible to tell what exactly can be inferred about concealed values. Moreover,
               leakages of individual studies can cumulate as in DNA pools, where aggregation of
               minuscule effects on SNP frequencies allows us to make strong conclusions.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The biggest technical drawback of our solution is computational efficiency. As the
               data are secret shared between core centers and each oblivious computation step requires
               network communication, secure algorithms are several magnitudes slower than their
               insecure counterparts. Hence, we set up a controlled experiment to show that GWAS
               are feasible in our setting.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Finally, note that legal restrictions can form a major obstacle, as secret sharing
               is not as widely used as data encryption. However, legal issues are out of our scope
               as we analyze only the technological feasibility and potential advantages.
            </p>
         </div>
      </div>
      <div id="SEC3">
         <h2>3 METHODS</h2>
         <div id="SEC3.1">
            <h3>3.1 Single-point analysis</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The first step in GWAS is splitting individuals into case and control groups. These
               groups are formed based on phenotypic traits, such as presence and severity of disease.
               Although all four nucleotides can be located in a SNP site, it is common to consider
               two alleles, where the first one corresponds to the reference sequence and the other
               represents potential mutations. 
               <span ref-type="table" rid="btt066-T1">
                  <a href="#btt066-T1">Table 1</a>
               </span> depicts the 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> contingency table for allele counts in case and control groups, where an allele is
               counted twice if it is present in both DNA strands (is homozygous). The test statistic
               for the standard 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> test is expressed as
               
               <div tagxxx="disp-formula">
                  <span class="label">(1)</span>
                  <div tagxxx="graphic"></div>
               </div>
               and the test statistic for equiproportionality of the allele A in both groups is
               
               <div tagxxx="disp-formula">
                  <span class="label">(2)</span>
                  <div tagxxx="graphic"></div>
               </div>
               For reasonable sample sizes, both test statistics are distributed according to 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> distribution with one degree of freedom. See the work of 
               <span ref-type="bibr" rid="btt066-B36">
                  <sup>
                     <a href="#btt066-B36">Visscher and Hellard (2003)</a>
                  </sup>
               </span> for further details.
               
               <div id="btt066-T1">
                  <span class="label">Table 1.</span>
                  <caption>
                     <p>Contingency table for the standard 
                        <div tagxxx="inline-formula">
                           <div tagxxx="inline-graphic"></div>
                        </div> test
                     </p>
                  </caption>
                  <table>
                     <thead>
                        <tr>
                           <th>Group</th>
                           <th>Allele A</th>
                           <th>Allele B</th>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>Cases</td>
                           <td>
                              <i>a</i>
                           </td>
                           <td>
                              <i>c</i>
                           </td>
                        </tr>
                        <tr>
                           <td>Controls</td>
                           <td>
                              <i>b</i>
                           </td>
                           <td>
                              <i>d</i>
                           </td>
                        </tr>
                     </div>
                  </table>
               </div>
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">These tests are accurate if the Hardy–Weinberg equilibrium condition is satisfied
               for a particular SNP, whereas the Cochran–Armitage test for trend can be used without
               this assumption. First, one must assemble the 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> contingency table depicted in 
               <span ref-type="table" rid="btt066-T2">
                  <a href="#btt066-T2">Table 2</a>
               </span> and then compute the tests statistic as
               
               <div tagxxx="disp-formula">
                  <span class="label">(3)</span>
                  <div tagxxx="graphic"></div>
               </div>
               where the weights 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> are chosen according to the suspected influence mechanism (usually 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> is an appropriate choice). As before, 
               <i>T</i>
               <div tagxxx="sub">3</div> is approximately distributed according to 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> distribution with one degree of freedom (
               <span ref-type="bibr" rid="btt066-B1">
                  <sup>
                     <a href="#btt066-B1">Armitage, 1955</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="btt066-B30">
                  <sup>
                     <a href="#btt066-B30">Sasieni, 1997</a>
                  </sup>
               </span>).
               
               <div id="btt066-T2">
                  <span class="label">Table 2.</span>
                  <caption>
                     <p>Contingency table for the Cochran–Armitage test for trend</p>
                  </caption>
                  <table>
                     <thead>
                        <tr>
                           <th>Group</th>
                           <th>Allele AA</th>
                           <th>Allele AB</th>
                           <th>Allele BB</th>
                           <th>Total</th>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>Cases</td>
                           <td>
                              <i>r</i>
                              <div tagxxx="sub">0</div>
                           </td>
                           <td>
                              <i>r</i>
                              <div tagxxx="sub">1</div>
                           </td>
                           <td>
                              <i>r</i>
                              <div tagxxx="sub">2</div>
                           </td>
                           <td>
                              <i>m</i>
                              <div tagxxx="sub">1</div>
                           </td>
                        </tr>
                        <tr>
                           <td>Controls</td>
                           <td>
                              <i>s</i>
                              <div tagxxx="sub">0</div>
                           </td>
                           <td>
                              <i>s</i>
                              <div tagxxx="sub">1</div>
                           </td>
                           <td>
                              <i>s</i>
                              <div tagxxx="sub">2</div>
                           </td>
                           <td>
                              <i>m</i>
                              <div tagxxx="sub">2</div>
                           </td>
                        </tr>
                        <tr>
                           <td>Total</td>
                           <td>
                              <i>n</i>
                              <div tagxxx="sub">0</div>
                           </td>
                           <td>
                              <i>n</i>
                              <div tagxxx="sub">1</div>
                           </td>
                           <td>
                              <i>n</i>
                              <div tagxxx="sub">2</div>
                           </td>
                           <td>
                              <i>N</i>
                           </td>
                        </tr>
                     </div>
                  </table>
               </div>
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Transmission disequilibrium test (TDT) is applicable only if the data consist of parent–child
               trios. The test measures whether one homozygous genotype is more over-represented
               than the other among affected children with heterozygous parents. For that we must
               first select trios where both parents have heterozygous genotype. Let 
               <i>u</i> be the count of AA and 
               <i>v</i> be the count of BB genotypes among children. Then the corresponding statistic
               
               <div tagxxx="disp-formula">
                  <span class="label">(4)</span>
                  <div tagxxx="graphic"></div>
               </div>
               is again approximately distributed according to 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> distribution with one degree of freedom (
               <span ref-type="bibr" rid="btt066-B31">
                  <sup>
                     <a href="#btt066-B31">Spielman et al., 1993</a>
                  </sup>
               </span>). Compared with previous tests, TDT is less sensitive to sampling artefacts but it
               also requires more structured data.
            </p>
         </div>
         <div id="SEC3.2">
            <h3>3.2 Essentials of secure computing using secret sharing</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A secure computation program is similar to a standard computer program. The difference
               between the two is in how the data are stored and processed. In a standard program,
               all values are processed publicly, whereas in a secure computation program, it is
               possible to specify which values are publicly visible and which are stored using techniques
               like secret sharing. These values can be used in computations without leaking their
               contents.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In our proposed solution, the data are secret shared between three or more hosts.
               The hosts themselves are not able to understand the values stored in their databases
               because each value looks like random noise owing to secret sharing. However, it is
               not trivial to perform computations on secret shared values as special secure multi-party
               computation protocols are required. The secure computation protocols used by the hosts
               preserve the privacy of the data during computation. The genotype data remain secure
               as long as the hosts do not share their databases of shares with each other. 
               <span ref-type="fig" rid="btt066-F1">Figure 1</span> shows how secure multi-party computation works with secret sharing.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Secure computation can be used to perform most data processing operations. However,
               current solutions have some important differences compared with standard programming:
               (i) floating-point operations are significantly slower; (ii) comparison operations
               are slower than multiplication and addition; (iii) parallel execution of several operations
               is faster than sequential execution. Further details can be found in the 
               <div tagxxx="ext-link">Supplementary Data</div> and in the articles by 
               <span ref-type="bibr" rid="btt066-B3">
                  <sup>
                     <a href="#btt066-B3">Bogdanov et al. (2008)</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="btt066-B9">
                  <sup>
                     <a href="#btt066-B9">Damgård et al. (2009)</a>
                  </sup>
               </span> and 
               <span ref-type="bibr" rid="btt066-B12">
                  <sup>
                     <a href="#btt066-B12">Geisler (2010)</a>
                  </sup>
               </span>.
            </p>
         </div>
         <div id="SEC3.3">
            <h3>3.3 Secure storage of genotype data</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Allele-level descriptions of genotypes are commonly stored as sequences of pairs where
               each pair is encoded as AA, AB, BB or NN, corresponding to a specific SNP. In GWAS,
               such data are converted into contingency tables as described in 
               <span ref-type="table" rid="btt066-T1">
                  <a href="#btt066-T1">Tables 1</a>
               </span> and 
               <span ref-type="table" rid="btt066-T2">
                  <a href="#btt066-T2">2</a>
               </span> depending on the analysis method being used.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">This kind of counting, however, requires the use of string comparison operations that
               we would like to avoid, as they tend to be relatively slow in the case of share computing.
               Therefore, we represent each SNP as a pair of integers 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div>, where 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> counts the occurrences of the first allele and 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> the occurrences of the second allele in that SNP. That is, pairs AA, AB, BB, NN are
               encoded as (2,0), (1,1), (0,2), (0,0), respectively.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">During the data collection phase, data donors, wetlabs or gene banks must first convert
               genotypes into the form described above and then secret share them between data hosts.
               As a result, we get a secret shared database where each column corresponds to an individual
               donor and there are two rows for each SNP (one for 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> and one for 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div>). For each donor, we also store an ID value that uniquely identifies them.
            </p>
         </div>
         <div id="SEC3.4">
            <h3>3.4 Secure formation of case–control groups</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To hide the identities of case and control group members, secret shared index vectors
               are used to specify the groups. An index vector 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> is a zero-one vector, where 
               <i>x
                  <div tagxxx="sub">i</div>
               </i> corresponds to the 
               <i>i</i>-th column in the database and 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> if and only if the 
               <i>i</i>-th person is a member of the group that vector represents.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The two principal ways of how to construct case–control groups can be seen in 
               <span ref-type="fig" rid="btt066-F2">Figure 2</span> Panel C. In the first scenario, analysts have direct access to clinical data for
               a set of donors and, thus, are able to select case and control groups and construct
               index vectors. In the second scenario, phenotype data are also stored in secret shared
               form. In this case, there exists a secret shared database of phenotype attributes
               that consists of boolean attributes (e.g. has diabetes) and integers (e.g. age, blood
               pressure, height). To construct case and control groups, the analyst has to specify
               a logical expression based on which the hosts perform the necessary comparisons and
               output the two index vectors in secret shared form. For simple inclusion criteria,
               such as the one we used for experiments 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div>, the corresponding secure comparison protocol is rather efficient.
            </p>
         </div>
         <div id="SEC3.5">
            <h3>3.5 Secure assembly of contingency groups</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Let 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> and 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> be index vectors for case and control groups, respectively. Let 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> and 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> be database columns for a particular SNP. Then the allele counts needed for tests
               (1) and (2) can be expressed as
               
               <div tagxxx="disp-formula">
                  <span class="label">(5)</span>
                  <div tagxxx="graphic"></div>
               </div>
               Data hosts can securely compute shares of 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> for each SNP.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">For the second contingency table, 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> for allele combinations AB, BB and NN. Similarly, 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> for AA, AB and NN and 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> for AA, BB and NN. We can express counts as
               
               <div tagxxx="disp-formula">
                  <span class="label">(6)</span>
                  <div tagxxx="graphic"></div>
               </div>
               For TDT, we need to construct an index vector 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> for detecting heterozygous parents, i.e. 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> if at least one parent is homozygous and 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> otherwise. As 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> for homozygous allele combinations, we compute
               
               <div tagxxx="disp-formula">
                  <span class="label">(7)</span>
                  <div tagxxx="graphic"></div>
               </div>
               To get shares of 
               <i>u</i> and 
               <i>v</i>, we can combine 
               <i>z</i> with counts of AA and BB.
            </p>
         </div>
         <div id="SEC3.6">
            <h3>3.6 Secure statistical testing</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To determine whether a SNP is significant, one must check whether a 
               <i>P</i>-value that is associated with a test statistic is below a pre-described significance
               level. Let 
               <i>α</i> be the desired significance level and let 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> be such that 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> when 
               <i>T</i> is distributed according to 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> distribution with one degree of freedom. A SNP is significant only if the corresponding
               test statistic is 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div>.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Direct evaluation of formulae (1)–(4) requires floating-point arithmetic, which we
               would like to avoid. Hence, we must rewrite the equation 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> in terms of integer operations. Let 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> be represented as a fraction 
               <i>p</i>/
               <i>q</i> and the test statistic as a fraction 
               <i>m</i>/
               <i>n</i>. Then the condition 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div> is equivalent to the condition 
               <div tagxxx="inline-formula">
                  <div tagxxx="inline-graphic"></div>
               </div>. Both sides of this inequality can be securely computed and the inequality can be
               evaluated by using secure comparison operation, after which we can publish the comparison
               results to find out which SNPs are significant.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The significance level must be determined considering the multiple testing issue.
               The simplest way is to use Bonferroni correction, which is a conservative measure.
               It is also possible to perform privacy-preserving FDR correction. We estimate that
               the secure version of the standard FDR procedure takes about 10 min to complete for
               262 264 SNPs. However, there are alternatives to the original algorithm that are significantly
               faster; see the 
               <div tagxxx="ext-link">Supplementary Data</div> for further details.
            </p>
         </div>
      </div>
      <div id="SEC4">
         <h2>4 RESULTS</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To demonstrate the feasibility of our approach, we used the S
            <div tagxxx="sc">haremind</div> multi-party computation platform (
            <span ref-type="bibr" rid="btt066-B3">
               <sup>
                  <a href="#btt066-B3">Bogdanov et al., 2008</a>
               </sup>
            </span>) to implement core algorithms for GWAS. Our choice was mainly motivated by the efficiency
            and ease of use of the S
            <div tagxxx="sc">haremind</div> platform. Alternative platforms [Viff (
            <span ref-type="bibr" rid="btt066-B9">
               <sup>
                  <a href="#btt066-B9">Damgård et al., 2009</a>
               </sup>
            </span>) and F
            <div tagxxx="sc">airplay</div>MP (
            <span ref-type="bibr" rid="btt066-B2">
               <sup>
                  <a href="#btt066-B2">Ben-David et al., 2008</a>
               </sup>
            </span>)] should give similar results.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We used 270 genotypes from the H
            <div tagxxx="sc">ap</div>M
            <div tagxxx="sc">ap</div> project (
            <span ref-type="bibr" rid="btt066-B19">
               <sup>
                  <a href="#btt066-B19">International HapMap Consortium, 2003</a>
               </sup>
            </span>) measured with the Affymetrix Mapping 500K Array as the main data source. In each
            experiment, we divided the data randomly into case and control groups and performed
            genome-wide search for highly differentiated SNPs. For that we used cryptographically
            secure counterparts of standard statistical tests used in GWAS: two 
            <div tagxxx="inline-formula">
               <div tagxxx="inline-graphic"></div>
            </div> tests for independence (
            <span ref-type="bibr" rid="btt066-B36">
               <sup>
                  <a href="#btt066-B36">Visscher and Hellard, 2003</a>
               </sup>
            </span>), Cochran–Armitage test for trend (
            <span ref-type="bibr" rid="btt066-B30">
               <sup>
                  <a href="#btt066-B30">Sasieni, 1997</a>
               </sup>
            </span>) and TDT (
            <span ref-type="bibr" rid="btt066-B31">
               <sup>
                  <a href="#btt066-B31">Spielman et al., 1993</a>
               </sup>
            </span>). As our algorithms return exactly the same outputs as original algorithms, we report
            only performance results for various sub-tasks. To show the variability of running
            times, we report the mean and standard deviation of four independent runs.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Each of the donors has 262 264 measured SNPs. First, we ran the algorithm on the data
            of 270 donors, and then we went on to test the data of 540, 810 and 1080 donors. We
            performed the experiments on three servers running S
            <div tagxxx="sc">haremind</div>. Each server was an off-the-shelf server-grade machine with 48 GB RAM of which less
            was used, twelve 2.93 GHz Intel Xeon (Westmere) cores of which two were used and a
            1 Gb/s local area network (LAN) connection. At the moment, the network connection
            is the bottleneck in terms of algorithm running time; however, S
            <div tagxxx="sc">haremind</div> has been successfully used in real applications (
            <span ref-type="bibr" rid="btt066-B4">
               <sup>
                  <a href="#btt066-B4">Bogdanov et al., 2012</a>
               </sup>
            </span>).
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The time spent on data acquisition and secure storage does not depend on the statistical
            test used later on. It depends only on the number of SNPs and the number of gene donors.
            The average time it takes to encode and share the SNPs for the described case can
            be seen in 
            <span ref-type="table" rid="btt066-T3">
               <a href="#btt066-T3">Table 3</a>
            </span>. Note that secret sharing and uploading data are done only once for each dataset;
            hence, this is a single-time cost.
            
            <div id="btt066-T3">
               <span class="label">Table 3.</span>
               <caption>
                  <p>Performance results for data upload and filtering</p>
               </caption>
               <table>
                  <thead>
                     <tr>
                        <th>Number of donors</th>
                        <th>Upload</th>
                        <th>Filtering</th>
                     </tr>
                  </thead>
                  <div tagxxx="tbody">
                     <tr>
                        <td>270 donors</td>
                        <td>12.0 min</td>
                        <td>0.51 s</td>
                     </tr>
                     <tr>
                        <td>540 donors</td>
                        <td>17.3 min</td>
                        <td>0.59 s</td>
                     </tr>
                     <tr>
                        <td>810 donors</td>
                        <td>23.2 min</td>
                        <td>0.63 s</td>
                     </tr>
                     <tr>
                        <td>1080 donors</td>
                        <td>29.4 min</td>
                        <td>0.68 s</td>
                     </tr>
                  </div>
               </table>
            </div>
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The time needed to form case–control groups depends on the application scenario. When
            the analyst has direct access to phenotype data and can form case and control groups
            by herself/himself, then there is no computational overhead. In more involved cases,
            the case and control groups must be constructed based on secret shared phenotype attributes.
            In this case, the overhead depends on the complexity of inclusion criteria for case
            and control groups. Filtering results presented in 
            <span ref-type="table" rid="btt066-T3">
               <a href="#btt066-T3">Table 3</a>
            </span> show that formation of such groups can be done in seconds for typical inclusion criteria
            that consist of simple comparison operations mixed with logical conjunctives.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The time needed to perform the statistical test depends on the test, but in all cases
            it can be broken down into counting allele frequencies and evaluating test statistic.
            As 
            <span ref-type="table" rid="btt066-T4">
               <a href="#btt066-T4">Tables 4</a>
            </span> and 
            <span ref-type="table" rid="btt066-T5">
               <a href="#btt066-T5">5</a>
            </span> clearly show, the main performance bottleneck is frequency counting, which scales
            linearly w.r.t. the total number of SNP measurements. As our encoding is optimized
            for 
            <div tagxxx="inline-formula">
               <div tagxxx="inline-graphic"></div>
            </div> test, a better encoding will enhance the performance of the Cochran–Armitage tests
            but not beyond 
            <div tagxxx="inline-formula">
               <div tagxxx="inline-graphic"></div>
            </div> results. The total duration of the analysis is the sum of the frequency analysis
            and evaluation, as the other parts have a negligible duration.
            
            <div id="btt066-T4">
               <span class="label">Table 4.</span>
               <caption>
                  <p>Performance results for three different frequency analyses</p>
               </caption>
               <table>
                  <thead>
                     <tr>
                        <th>Number of donors</th>
                        <th>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> tests
                        </th>
                        <th>Cochran–Armitage</th>
                        <th>TDT</th>
                     </tr>
                  </thead>
                  <div tagxxx="tbody">
                     <tr>
                        <td>270 donors</td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> min
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> min
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> min
                        </td>
                     </tr>
                     <tr>
                        <td>540 donors</td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> min
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> min
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> min
                        </td>
                     </tr>
                     <tr>
                        <td>810 donors</td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> min
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> min
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> min
                        </td>
                     </tr>
                     <tr>
                        <td>1080 donors</td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> min
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> min
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> min
                        </td>
                     </tr>
                     <tr>
                        <td>1080 donors (non-secure)</td>
                        <td>14 s</td>
                        <td>35 s</td>
                        <td>11 s</td>
                     </tr>
                  </div>
               </table>
            </div>
            <div id="btt066-T5">
               <span class="label">Table 5.</span>
               <caption>
                  <p>Performance results for four test evaluation methods</p>
               </caption>
               <table>
                  <thead>
                     <tr>
                        <th>Number of donors</th>
                        <th>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> test 1
                        </th>
                        <th>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> test 2
                        </th>
                        <th>Cochran– Armitage</th>
                        <th>TDT</th>
                     </tr>
                  </thead>
                  <div tagxxx="tbody">
                     <tr>
                        <td>270 donors</td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> s
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> s
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> s
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> s
                        </td>
                     </tr>
                     <tr>
                        <td>540 donors</td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> s
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> s
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> s
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> s
                        </td>
                     </tr>
                     <tr>
                        <td>810 donors</td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> s
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> s
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> s
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> s
                        </td>
                     </tr>
                     <tr>
                        <td>1080 donors</td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> s
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> s
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> s
                        </td>
                        <td>
                           <div tagxxx="inline-formula">
                              <div tagxxx="inline-graphic"></div>
                           </div> s
                        </td>
                     </tr>
                     <tr>
                        <td>1080 donors (non-secure)</td>
                        <td>21 ms</td>
                        <td>20 ms</td>
                        <td>49 ms</td>
                        <td>11 ms</td>
                     </tr>
                  </div>
               </table>
            </div>
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The presented results clearly show that cryptographically secure evaluation of statistical
            tests on genome-wide scale is practically feasible. The expected running time is hours
            instead of a few minutes when computed non-securely. However, the latter is not a
            significant slowdown compared with the time needed to acquire the data if the secure
            analysis method is not used.
         </p>
      </div>
      <div id="SEC5">
         <h2>5 DISCUSSION</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Although the results prove the practical feasibility of cryptographically secure GWAS,
            the solution is also notably more resource demanding than the alternatives. We have
            to analyze further whether potential benefits outweigh costs. We consider three potential
            application scenarios and contrast our approach with the alternatives.
         </p>
         <div id="SEC5.1">
            <h3>5.1 Collaboration between hospitals and biobanks</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">By now many countries have national or state-funded biobanks. There are more than
               40 state-governed biobanks in Europe (
               <span ref-type="bibr" rid="btt066-B40">
                  <sup>
                     <a href="#btt066-B40">Zika et al., 2010</a>
                  </sup>
               </span>) and many others in Asia and America (
               <span ref-type="bibr" rid="btt066-B32">
                  <sup>
                     <a href="#btt066-B32">Swede et al., 2007</a>
                  </sup>
               </span>). Thus, there is a huge potential for collaborative studies between biobanks, hospitals
               and pharmaceutical companies. For example, if a clinical study indicates that a treatment
               is ineffective for a certain group of people, a genome-wide association study can
               indicate whether a particular set of SNPs can be used to predict efficacy of a treatment.
               In particular, GWAS have shown that certain SNP mutations influence efficacy of treatments
               for asthma, inflammatory bowel disease, coronary heart disease and cancer (
               <span ref-type="bibr" rid="btt066-B13">
                  <sup>
                     <a href="#btt066-B13">Grant and Hakonarson, 2007</a>
                  </sup>
               </span>).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Privacy issues are the major obstacle in such studies: neither biobanks nor research
               institutions can give out data without explicit consent from the patients or explicit
               decision by a relevant ethics committee. In such scenarios, secure GWAS can be used
               as a pilot study for assessing potential benefits of combined studies. In particular,
               there is no reason to merge the data for further analysis if a secure GWAS reveals
               no differentially expressed SNPs. Because such a study can be conducted in a few hours,
               it can significantly speed up pharmacogenetic studies and the results can be used
               by ethics committees to make more seasoned decisions.
            </p>
         </div>
         <div id="SEC5.2">
            <h3>5.2 Servicing study data without privacy breaches</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In many state-funded studies, collected data must be made accessible for public use
               by submitting them to a central repository. The NIH example shows a direct publication
               of GWAS data may cause unintended consequences even if the data are presented to the
               public in aggregated form (
               <span ref-type="bibr" rid="btt066-B8">
                  <sup>
                     <a href="#btt066-B8">Couzin, 2008</a>
                  </sup>
               </span>). One potential solution is to set up an online web service for conducting GWAS:
               a researcher just posts inclusion criteria for case and control groups and all computations
               are done by the host. In most cases, such a solution is adequate without cryptographic
               countermeasures. Privacy issues emerge only if researchers want to pool together data
               from several different repositories to detect weak associations or the inclusion criteria
               must remain private. In these cases, secure GWAS methodology can be applied by combining
               methods for international consortium studies with phenotype-based filtering (see 
               <span ref-type="fig" rid="btt066-F2">Fig. 2</span>).
            </p>
         </div>
         <div id="SEC5.3">
            <h3>5.3 Faster and more secure consortium studies</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">One of the main hurdles in GWAS is the sample size. For rare diseases, there are not
               enough genotyped patients to form a big enough case group. Also, over- and under-representation
               of sub-populations can cause spurious associations. Larger studies involving several
               international biobanks can diminish the impact of such problems.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Two competing alternatives to our solution in this setting are federated database
               systems with additional security mechanisms and hierarchical aggregation of the data.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A federated database system offers middleware that automatically binds together several
               sources and allows users to make various queries without knowing how the data are
               organized. As such, a federated database system does not solve privacy issues. Hence,
               one needs an honest broker—a dedicated server with strengthened security measures
               that assembles the data and processes all queries (
               <span ref-type="bibr" rid="btt066-B6">
                  <sup>
                     <a href="#btt066-B6">Boyd et al., 2007</a>
                  </sup>
               </span>). The latter introduces a single point of failure—nothing can be done if the security
               of the honest broker is breached. Also, as the data are sent directly to the honest
               broker, all biobanks must have explicit clearance for releasing the data.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Hierarchical data aggregation is applicable when results of individual studies can
               be combined with some meta-analysis technique (e.g. 
               <span ref-type="bibr" rid="btt066-B39">
                  <sup>
                     <a href="#btt066-B39">Wolfson et al., 2010</a>
                  </sup>
               </span>). In such cases, the honest broker must access only aggregated summaries of individual
               data sources to produce the desired result. Consequently, we get stronger privacy
               guarantees, as the broker receives only a limited amount of information. However,
               unexpected privacy breaches can still occur because aggregation methods provide no
               explicit security guarantees and it is extremely difficult to assess how much information
               is leaked through summaries. Also, the approach cannot be used when members of case
               and control groups must be kept secret.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In a nutshell, while the alternatives are faster than our solution, they are also
               much more vulnerable to various attacks, and thus, privacy concerns can prevent their
               usage or considerably delay the initial setup time. Moreover, hierarchical data aggregation
               techniques can be combined with our solution. Namely, our solution can be used to
               replace the honest broker—biobanks secret share the aggregated results, and thus,
               fewer operations must be done on shares. For the analysis part of GWAS, the resulting
               hybrid algorithm will only take the time given in 
               <span ref-type="table" rid="btt066-T5">
                  <a href="#btt066-T5">Table 5</a>
               </span> plus a little overhead, making the computation time ∼30 s to 1 min as the filtering
               of case and control groups and computation of contingency tables are done locally.
            </p>
         </div>
         <div id="SEC5.4">
            <h3>5.4 Long-term security in a personal genomic project</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The rapid decrease of genotyping costs and moderate success in genetic diagnostics
               have sparkled interest in personal genomics. Companies, such as 23andMe, deCODEme
               and Navigenics, offer personalized genotyping services. Although participants have
               a right to withdraw their data at any moment, this right is enforced only by physical
               and organizational methods. As a consequence, a single successful outsider or insider
               attack can obsolete all privacy guarantees. Numerous data leakages in other areas
               have shown that this is an irreversible procedure. Once data have leaked, there is
               no way to recall them. On a shorter timescale, such events are highly improbable.
               However, such projects need privacy guarantees that last more than 100 years to protect
               participants and their offspring. In such settings, distributed storage based on secret
               sharing is one of the best cryptographic alternatives, as a successful breach of security
               of a single facility yields no information and it is possible to recover from such
               events.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <i>Funding</i>: The research was supported by the 
               <div tagxxx="funding-source">European Regional Development Fund</div> through 
               <div tagxxx="funding-source">Estonian Centre of Excellence in Computer Science</div> and 
               <div tagxxx="funding-source">Software Technology and Applications Competence Center</div>.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <i>Conflict of Interest</i>: none declared.
            </p>
         </div>
      </div>
      <div id="">
         <h2>Supplementary Material</h2>
         <div tagxxx="supplementary-material">
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Supplementary Data</div>
            </caption>
            <div tagxxx="media"></div>
            <div tagxxx="media"></div>
         </div>
      </div>
      <div tagxxx="back">
         <div tag="ref-list">
            <ul>
               <div tagxxx="title">REFERENCES</div>
               <li tag="ref">
                  <a name="btt066-B1"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Armitage</span>
                           <span tagx="given-names">P</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Tests for linear trends in proportions and frequencies</span>
                     <span tagx="source">Biometrics</span>
                     <span tagx="year">1955</span>
                     <span tagx="volume">11</span>
                     <span tagx="fpage">375</span>
                     <span tagx="lpage">386</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B2"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Ben-David</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">FairplayMP: a system for secure multi-party computation</span>
                     <span tagx="source">Proceedings of ACM CCS’08</span>
                     <span tagx="year">2008</span>
                     <span class="publisher-loc'">[New York]</span>
                     <span class="publisher-name'">[ACM]</span>
                     <span tagx="fpage">257</span>
                     <span tagx="lpage">266</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B3"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Bogdanov</span>
                           <span tagx="given-names">D</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Sharemind: a framework for fast privacy-preserving computations</span>
                     <span tagx="source">Proceedings of ESORICS’08. Vol. 5283 of Lecture Notes in Computer Science</span>
                     <span tagx="year">2008</span>
                     <span class="publisher-loc'">[Heidelberg]</span>
                     <span class="publisher-name'">[Springer]</span>
                     <span tagx="fpage">192</span>
                     <span tagx="lpage">206</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B4"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Bogdanov</span>
                           <span tagx="given-names">D</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Deploying secure multi-party computation for financial data analysis</span>
                     <span tagx="source">Proceedings of Financial Cryptography 2012. Vol. 7397 of Lecture Notes in Computer
                        Science
                     </span>
                     <span tagx="year">2012</span>
                     <span class="publisher-loc'">[Heidelberg]</span>
                     <span class="publisher-name'">[Springer]</span>
                     <span tagx="fpage">57</span>
                     <span tagx="lpage">64</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B5"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Bogetoft</span>
                           <span tagx="given-names">P</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dingledine</span>
                           <span tagx="given-names">R</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Golle</span>
                           <span tagx="given-names">P</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Secure multiparty computation goes live</span>
                     <span tagx="source">Proceedings of Financial Cryptography 2009. Vol. 5628 of Lecture Notes in Computer
                        Science
                     </span>
                     <span tagx="year">2009</span>
                     <span class="publisher-loc'">[Heidelberg]</span>
                     <span class="publisher-name'">[Springer]</span>
                     <span tagx="fpage">325</span>
                     <span tagx="lpage">343</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B6"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Boyd</span>
                           <span tagx="given-names">AD</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">An ‘Honest Broker’ mechanism to maintain privacy for patient care and academic medical
                        research
                     </span>
                     <span tagx="source">Int. J. Med. Inform.</span>
                     <span tagx="year">2007</span>
                     <span tagx="volume">76</span>
                     <span tagx="fpage">407</span>
                     <span tagx="lpage">411</span>
                     <span class="pub-id'">[17081800]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B7"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Chakravarti</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Little</span>
                           <span tagx="given-names">P</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Nature, nurture and human disease</span>
                     <span tagx="source">Nature</span>
                     <span tagx="year">2003</span>
                     <span tagx="volume">421</span>
                     <span tagx="fpage">412</span>
                     <span tagx="lpage">414</span>
                     <span class="pub-id'">[12540911]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B8"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Couzin</span>
                           <span tagx="given-names">J</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Genetic privacy. Whole-genome data not anonymous, challenging assumptions</span>
                     <span tagx="source">Science</span>
                     <span tagx="year">2008</span>
                     <span tagx="volume">321</span>
                     <span tagx="fpage">1278</span>
                     <span class="pub-id'">[18772401]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B9"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Damgård</span>
                           <span tagx="given-names">I</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Jarecki</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Tsudik</span>
                           <span tagx="given-names">G</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Asynchronous multiparty computation: theory and implementation</span>
                     <span tagx="source">Proceedings of Public Key Cryptography 2009. Vol. 5443 of Lecture Notes in Computer
                        Science
                     </span>
                     <span tagx="year">2009</span>
                     <span class="publisher-loc'">[Heidelberg]</span>
                     <span class="publisher-name'">[Springer]</span>
                     <span tagx="fpage">160</span>
                     <span tagx="lpage">179</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B10"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">De Moor</span>
                           <span tagx="given-names">GJE</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Privacy enhancing techniques—the key to secure communication and management of clinical
                        and genomic data
                     </span>
                     <span tagx="source">Methods Inf. Med.</span>
                     <span tagx="year">2003</span>
                     <span tagx="volume">42</span>
                     <span tagx="fpage">148</span>
                     <span tagx="lpage">153</span>
                     <span class="pub-id'">[12743651]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B11"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Dwork</span>
                           <span tagx="given-names">C</span>,
                        </span>
                     </span>
                     <span tagx="article-title">A firm foundation for private data analysis</span>
                     <span tagx="source">Commun. ACM</span>
                     <span tagx="year">2011</span>
                     <span tagx="volume">54</span>
                     <span tagx="fpage">86</span>
                     <span tagx="lpage">95</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B12"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Geisler</span>
                           <span tagx="given-names">M</span>,
                        </span>
                     </span>
                     <span tagx="source">Cryptographic protocols: theory and implementation</span>
                     <span tagx="year">2010</span>
                     <span class="publisher-name'">[PhD Thesis, Aarhus University]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B13"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Grant</span>
                           <span tagx="given-names">SFA</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Hakonarson</span>
                           <span tagx="given-names">H</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Recent development in pharmacogenomics: from candidate genes to genome-wide association
                        studies
                     </span>
                     <span tagx="source">Expert. Rev. Mol. Diagn.</span>
                     <span tagx="year">2007</span>
                     <span tagx="volume">7</span>
                     <span tagx="fpage">371</span>
                     <span tagx="lpage">393</span>
                     <span class="pub-id'">[17620046]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B14"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Gymrek</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Identifying personal genomes by surname inference</span>
                     <span tagx="source">Science</span>
                     <span tagx="year">2013</span>
                     <span tagx="volume">339</span>
                     <span tagx="fpage">321</span>
                     <span tagx="lpage">324</span>
                     <span class="pub-id'">[23329047]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B15"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Hamosh</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Online Mendelian Inheritance in Man (OMIM), a knowledgebase of human genes and genetic
                        disorders
                     </span>
                     <span tagx="source">Nucleic Acids Res.</span>
                     <span tagx="year">2005</span>
                     <span tagx="volume">33</span>
                     <span tagx="fpage">D514</span>
                     <span tagx="lpage">D517</span>
                     <span class="pub-id'">[15608251]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B16"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Hayden</span>
                           <span tagx="given-names">EC</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Genomics firms turn to other markets</span>
                     <span tagx="source">Nature</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">463</span>
                     <span tagx="fpage">859</span>
                     <span class="pub-id'">[20164890]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B17"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Hindorff</span>
                           <span tagx="given-names">LA</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Potential etiologic and functional implications of genome-wide association loci for
                        human diseases and traits
                     </span>
                     <span tagx="source">Proc. Natl Acad. Sci. USA</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">106</span>
                     <span tagx="fpage">9362</span>
                     <span tagx="lpage">9367</span>
                     <span class="pub-id'">[19474294]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B18"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Homer</span>
                           <span tagx="given-names">N</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Resolving individuals contributing trace amounts of DNA to highly complex mixtures
                        using high-density SNP genotyping microarrays
                     </span>
                     <span tagx="source">PLoS Genet.</span>
                     <span tagx="year">2008</span>
                     <span tagx="volume">4</span>
                     <span tagx="fpage">e1000167</span>
                     <span class="pub-id'">[18769715]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B19"></a>
                  <span class="element-citation'">
                     <span class="collab'">[International HapMap Consortium]</span>
                     <span tagx="article-title">The International HapMap Project</span>
                     <span tagx="source">Nature</span>
                     <span tagx="year">2003</span>
                     <span tagx="volume">426</span>
                     <span tagx="fpage">789</span>
                     <span tagx="lpage">796</span>
                     <span class="pub-id'">[14685227]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B20"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Lander</span>
                           <span tagx="given-names">ES</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Schork</span>
                           <span tagx="given-names">NJ</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Genetic dissection of complex traits</span>
                     <span tagx="source">Science</span>
                     <span tagx="year">1994</span>
                     <span tagx="volume">265</span>
                     <span tagx="fpage">2037</span>
                     <span tagx="lpage">2048</span>
                     <span class="pub-id'">[8091226]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B21"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Lin</span>
                           <span tagx="given-names">Z</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Genetics. Genomic research and human subject privacy</span>
                     <span tagx="source">Science</span>
                     <span tagx="year">2004</span>
                     <span tagx="volume">305</span>
                     <span tagx="fpage">183</span>
                     <span class="pub-id'">[15247459]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B22"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Machanavajjhala</span>
                           <span tagx="given-names">A</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">L-diversity: privacy beyond k-anonymity</span>
                     <span tagx="source">ACM Trans. Knowl. Discov. Data</span>
                     <span tagx="year">2007</span>
                     <span tagx="volume">1</span>
                     <span tagx="fpage">3</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B23"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Malin</span>
                           <span tagx="given-names">B</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Sweeney</span>
                           <span tagx="given-names">L</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Determining the identifiability of DNA database entries</span>
                     <span tagx="source">Proc. AMIA Symp.</span>
                     <span tagx="year">2000</span>
                     <span tagx="fpage">537</span>
                     <span tagx="lpage">541</span>
                     <span class="pub-id'">[11079941]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B24"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Malin</span>
                           <span tagx="given-names">BA</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Sweeney</span>
                           <span tagx="given-names">LA</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Inferring genotype from clinical phenotype through a knowledge based algorithm</span>
                     <span tagx="source">Pac. Symp. Biocomput.</span>
                     <span tagx="year">2002</span>
                     <span tagx="fpage">41</span>
                     <span tagx="lpage">52</span>
                     <span class="pub-id'">[11928494]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B25"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Manolio</span>
                           <span tagx="given-names">TA</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">A HapMap harvest of insights into the genetics of common disease</span>
                     <span tagx="source">J Clin Invest</span>
                     <span tagx="year">2008</span>
                     <span tagx="volume">118</span>
                     <span tagx="fpage">1590</span>
                     <span tagx="lpage">1605</span>
                     <span class="pub-id'">[18451988]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B26"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Moore</span>
                           <span tagx="given-names">JH</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Bioinformatics challenges for genome-wide association studies</span>
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">26</span>
                     <span tagx="fpage">445</span>
                     <span tagx="lpage">455</span>
                     <span class="pub-id'">[20053841]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B27"></a>
                  <span class="element-citation'">
                     <span class="collab'">[P3G Consortium et al.]</span>
                     <span tagx="article-title">Public access to genome-wide data: five views on balancing research with privacy and
                        protection
                     </span>
                     <span tagx="source">PLoS Genet.</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">5</span>
                     <span tagx="fpage">e1000665</span>
                     <span class="pub-id'">[19798440]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B28"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Pettersson</span>
                           <span tagx="given-names">E</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Generations of sequencing technologies</span>
                     <span tagx="source">Genomics</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">93</span>
                     <span tagx="fpage">105</span>
                     <span tagx="lpage">111</span>
                     <span class="pub-id'">[18992322]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B29"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Prokopenko</span>
                           <span tagx="given-names">I</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Type 2 diabetes: new genes, new understanding</span>
                     <span tagx="source">Trends Genet.</span>
                     <span tagx="year">2008</span>
                     <span tagx="volume">24</span>
                     <span tagx="fpage">613</span>
                     <span tagx="lpage">621</span>
                     <span class="pub-id'">[18952314]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B30"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Sasieni</span>
                           <span tagx="given-names">PD</span>,
                        </span>
                     </span>
                     <span tagx="article-title">From genotypes to genes: doubling the sample size</span>
                     <span tagx="source">Biometrics</span>
                     <span tagx="year">1997</span>
                     <span tagx="volume">53</span>
                     <span tagx="fpage">1253</span>
                     <span tagx="lpage">1261</span>
                     <span class="pub-id'">[9423247]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B31"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Spielman</span>
                           <span tagx="given-names">RS</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Transmission test for linkage disequilibrium: the insulin gene region and insulin-dependent
                        diabetes mellitus (IDDM)
                     </span>
                     <span tagx="source">Am. J. Hum. Genet.</span>
                     <span tagx="year">1993</span>
                     <span tagx="volume">52</span>
                     <span tagx="fpage">506</span>
                     <span tagx="lpage">516</span>
                     <span class="pub-id'">[8447318]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B32"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Swede</span>
                           <span tagx="given-names">H</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">National population-based biobanks for genetic research</span>
                     <span tagx="source">Genet. Med.</span>
                     <span tagx="year">2007</span>
                     <span tagx="volume">9</span>
                     <span tagx="fpage">141</span>
                     <span tagx="lpage">149</span>
                     <span class="pub-id'">[17413418]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B33"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Sweeney</span>
                           <span tagx="given-names">L</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Achieving k-anonymity privacy protection using generalization and suppression</span>
                     <span tagx="source">Int. J. Uncertain. Fuzz.</span>
                     <span tagx="year">2002</span>
                     <span tagx="volume">10</span>
                     <span tagx="fpage">571</span>
                     <span tagx="lpage">588</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B34"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Szymczak</span>
                           <span tagx="given-names">S</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Machine learning in genome-wide association studies</span>
                     <span tagx="source">Genet. Epidemiol.</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">33</span>
                     <div tagxxx="issue">Suppl. 1</div>
                     <span tagx="fpage">S51</span>
                     <span tagx="lpage">S57</span>
                     <span class="pub-id'">[19924717]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B35"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Varghese</span>
                           <span tagx="given-names">JS</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Easton</span>
                           <span tagx="given-names">DF</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Genome-wide association studies in common cancers—what have we learnt?</span>
                     <span tagx="source">Curr. Opin. Genet. Dev.</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">20</span>
                     <span tagx="fpage">201</span>
                     <span tagx="lpage">209</span>
                     <span class="pub-id'">[20418093]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B36"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Visscher</span>
                           <span tagx="given-names">PM</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Hellard</span>
                           <span tagx="given-names">SL</span>,
                        </span>
                     </span>
                     <span tagx="article-title">Simple method to analyze SNP-based association studies using DNA pools</span>
                     <span tagx="source">Genet. Epidemiol.</span>
                     <span tagx="year">2003</span>
                     <span tagx="volume">24</span>
                     <span tagx="fpage">291</span>
                     <span tagx="lpage">296</span>
                     <span class="pub-id'">[12687646]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B37"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Visscher</span>
                           <span tagx="given-names">PM</span>,
                        </span>
                        <span class="name'">
                           <span tagx="surname">Hill</span>
                           <span tagx="given-names">WG</span>,
                        </span>
                     </span>
                     <span tagx="article-title">The limits of individual identification from sample allele frequencies: theory and
                        statistical analysis
                     </span>
                     <span tagx="source">PLoS Genet.</span>
                     <span tagx="year">2009</span>
                     <span tagx="volume">5</span>
                     <span tagx="fpage">e1000628</span>
                     <span class="pub-id'">[19798439]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B38"></a>
                  <span class="element-citation'">
                     <span class="collab'">[Wellcome Trust Case Control Consortium]</span>
                     <span tagx="article-title">Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared
                        controls
                     </span>
                     <span tagx="source">Nature</span>
                     <span tagx="year">2007</span>
                     <span tagx="volume">447</span>
                     <span tagx="fpage">661</span>
                     <span tagx="lpage">678</span>
                     <span class="pub-id'">[17554300]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B39"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Wolfson</span>
                           <span tagx="given-names">M</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">Datashield: resolving a conflict in contemporary bioscience—performing a pooled analysis
                        of individual-level data without sharing the data
                     </span>
                     <span tagx="source">Int. J. Epidemiol.</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">39</span>
                     <span tagx="fpage">1372</span>
                     <span tagx="lpage">1382</span>
                     <span class="pub-id'">[20630989]</span>
                  </span>
               </li>
               <li tag="ref">
                  <a name="btt066-B40"></a>
                  <span class="element-citation'">
                     <span class="person-group'">
                        <span class="name'">
                           <span tagx="surname">Zika</span>
                           <span tagx="given-names">E</span>,
                        </span>
                        <i>et al.</i>
                     </span>
                     <span tagx="article-title">A European survey on biobanks: trends and issues</span>
                     <span tagx="source">Public Health Genomics</span>
                     <span tagx="year">2010</span>
                     <span tagx="volume">14</span>
                     <span tagx="fpage">96</span>
                     <span tagx="lpage">103</span>
                     <span class="pub-id'">[20395653]</span>
                  </span>
               </li>
            </ul>
         </div>
      </div>
   </div>
</html>