<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <div xmlns="" tagxxx="article">
      <head>
         <meta content="Nature Pub. Group" name="citation_publisher"></meta>
         <meta content="10.1038/ncomms4985" name="citation_doi"></meta>
         <meta content="Evidence of natural Wolbachia infections in field populations of Anopheles gambiae" name="citation_title"></meta>
         <title>Evidence of natural Wolbachia infections in field populations of Anopheles gambiae</title>
         <div tagxxx="contrib-group">
            <meta content="Francesco Baldini" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="Nicola Segata" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="Julien Pompon" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="Perrine Marcenac" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="W. Robert Shaw" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="Roch K. Dabiré" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="Abdoulaye Diabaté" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="Elena A. Levashina" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="Flaminia Catteruccia" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="1 " name="citation_author_institution"></meta>
            <meta content="2 " name="citation_author_institution"></meta>
            <meta content="3 " name="citation_author_institution"></meta>
            <meta content="4 " name="citation_author_institution"></meta>
            <meta content="5 " name="citation_author_institution"></meta>
            <meta content="6 " name="citation_author_institution"></meta>
            <meta content="7 " name="citation_author_institution"></meta>
            <meta content="8 " name="citation_author_institution"></meta>
            <meta content="9 " name="citation_author_institution"></meta>
         </div>
         <meta content="levashina@mpiib-berlin.mpg.de" name="email"></meta>
         <meta content="" name="conflict"></meta>
         <meta content="" name="contribs"></meta>
         <meta content="2014-06-06" name="citation_pubdate"></meta>
         <meta content="Copyright 2014: Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved." name="citation_copyright"></meta>
         <meta content="Licence This work is licensed under a Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported License. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. To view a copy of this license, visit http://creativecommons.org/licenses/by-nc-nd/3.0/" name="citation_licence"></meta>
         <meta content="" name="citation_funding"></meta>
         <meta content="" name="citation_pagecount"></meta>
         <meta content="" name="citation_data"></meta>
      </head>
      <div id="abstract" tag="abstract">
         <p>
            <i>Wolbachia</i> are maternally transmitted intracellular bacteria that invade insect populations
            by manipulating their reproduction and immunity and thus limiting the spread of numerous
            human pathogens. Experimental 
            <i>Wolbachia</i> infections can reduce 
            <i>Plasmodium</i> numbers in 
            <i>Anopheles</i> mosquitoes in the laboratory, however, natural 
            <i>Wolbachia</i> infections in field anophelines have never been reported. Here we show evidence of
            
            <i>Wolbachia</i> infections in 
            <i>Anopheles gambiae</i> in Burkina Faso, West Africa. Sequencing of the 
            <i>16S</i> rRNA gene identified 
            <i>Wolbachia</i> sequences in both female and male germlines across two seasons, and determined that
            these sequences are vertically transmitted from mother to offspring. Whole-genome
            sequencing of positive samples suggests that the genetic material identified in 
            <i>An. gambiae</i> belongs to a novel 
            <i>Wolbachia</i> strain, related to but distinct from strains infecting other arthropods. The evidence
            of 
            <i>Wolbachia</i> infections in natural 
            <i>Anopheles</i> populations promotes further investigations on the possible use of natural 
            <i>Wolbachia</i>–
            <i>Anopheles</i> associations to limit malaria transmission.
         </p>
      </div>
      <div id="abstract" tag="abstract">
         <p>
            <div tagxxx="inline-graphic"></div>
            <i>Wolbachia</i> bacteria live within the cells of many insect species, manipulating their hosts’
            reproduction and immune responses. Here, the authors show that these microbes also
            infect wild populations of malaria-spreading 
            <i>Anopheles</i> mosquitoes, supporting a potential use of 
            <i>Wolbachia</i> to limit malaria transmission.
         </p>
      </div>
      <p>Malaria-transmitting 
         <i>Anopheles</i> mosquitoes are the deadliest animals on this planet, causing the death of more than
         600,000 people each year and endangering the lives of half of the world’s population
         <span ref-type="bibr" rid="b1">1</span>. Current insecticide-based control strategies to stop malaria transmission by targeting
         the mosquito vector are limited by the rapid spread of insecticide resistance
         <span ref-type="bibr" rid="b2">2</span>. In addition, these interventions target only indoor feeding and resting populations,
         with the use of insecticide-treated bednets and the application of indoor residual
         sprays, respectively. For decades, the use of 
         <i>Wolbachia</i> endosymbionts has been proposed as an alternative to chemical strategies because
         of the ability of these bacteria to rapidly invade insect populations via cytoplasmic
         incompatibility
         <span ref-type="bibr" rid="b3">3</span>, and successful 
         <i>Wolbachia</i> invasions in field settings have been demonstrated in the case of the dengue and
         yellow fever vector 
         <i>Aedes aegypti</i>
         <span ref-type="bibr" rid="b4">4</span>. Recent proof that 
         <i>Wolbachia</i> infections of 
         <i>Anopheles</i> vectors limit the development of the 
         <i>Plasmodium</i> parasites that cause malaria
         <span ref-type="bibr" rid="b5">5</span>
         <span ref-type="bibr" rid="b6">6</span>
         <span ref-type="bibr" rid="b7">7</span>
         <span ref-type="bibr" rid="b8">8</span> makes these bacteria a particularly attractive tool for the control of both endo-
         and exophagic populations of malaria-transmitting anophelines. Long-standing limitations
         concerning the introduction of 
         <i>Wolbachia</i> into laboratory colonies of 
         <i>Anopheles</i> mosquitoes have been recently overcome
         <span ref-type="bibr" rid="b5">5</span>; however, the usefulness of this system for the control of 
         <i>Anopheles</i> populations has been undermined by the apparent absence of natural infections. Indeed,
         while 
         <i>Wolbachia</i> strains have been detected in many insects
         <span ref-type="bibr" rid="b9">9</span>
         <span ref-type="bibr" rid="b10">10</span>, attempts to identify these bacteria in field 
         <i>Anopheles</i> have failed, promoting the belief that these mosquitoes are not natural hosts for
         
         <i>Wolbachia</i>
         <span ref-type="bibr" rid="b11">11</span>
         <span ref-type="bibr" rid="b12">12</span>
         <span ref-type="bibr" rid="b13">13</span>.
      </p>
      <p>In this study, we report evidence of natural 
         <i>Wolbachia</i> infections in two incipient species of the major malaria vector, 
         <i>Anopheles gambiae</i>. We isolate 
         <i>Wolbachia</i>-specific 
         <i>16S</i> rRNA sequences from the reproductive organs and carcasses of adult mosquitoes and
         from larval carcasses in three different villages in Burkina Faso, West Africa. Whole-genome
         shotgun sequencing of two positive samples reveals a previously uncharacterized 
         <i>Wolbachia</i> strain that is maternally transmitted in laboratory settings. These results open
         new avenues for exploiting 
         <i>Wolbachia</i> infections for field applications targeting the malaria mosquito.
      </p>
      <div id="">
         <h2>Results</h2>
         <div id="">
            <h3>
               <i>Wolbachia</i> sequences detected in field 
               <i>An. gambiae</i> populations
            </h3>
            <p>We collected mating couples from natural 
               <i>An. gambiae</i> mating swarms in Burkina Faso, West Africa, to identify the microbial populations
               of the male and female reproductive tracts. We analysed two reproductively isolated
               populations of 
               <i>An. gambiae</i>, the M and S molecular forms, which do not interbreed in the field
               <span ref-type="bibr" rid="b14">14</span> and are now classified as two separate species (
               <i>An. coluzzii</i> and 
               <i>An. gambiae</i>, respectively
               <span ref-type="bibr" rid="b15">15</span>). For simplicity, in the text and in the tables, we will refer to these two species
               as M and S forms. Our collections included 81 couples captured in three different
               villages: two villages in Vallée du Kou (VK5 and VK7), where the M form is prevalent,
               and one village in Soumousso, primarily populated with S form (
               <span ref-type="fig" rid="f1">Fig. 1</span>, 
               <span ref-type="supplementary-material" rid="S1">Supplementary Table 1</span>).
            </p>
            <p>An initial high-throughput sequencing of the 
               <i>16S</i> rRNA gene (variable region V4, average 23,589, s.d. 16,070 assembled reads per sample)
               amplified from ovaries and testes dissected from 30 mating couples produced a molecular
               fingerprint of the bacterial population of these reproductive tissues. Surprisingly,
               this analysis identified one sample, derived from the testes of an S form male collected
               in Soumousso, infected with 
               <i>Wolbachia</i>. This sample contained 21.8% of reads (5,412 out of 24,800) matching the V4 regions
               of the 
               <i>Wolbachia 16S</i> rRNA gene, with a percentage identity ranging between 95.3 and 97.6%. This percentage
               is fully consistent with the overall diversity of known 
               <i>Wolbachia</i> V4 sequences (average identity 95.9%, s.d. 1.75%, median 96.0% as estimated from
               available sequences, see Methods), while it is incompatible with any other sequenced
               bacterial 
               <i>16S</i> gene (closest matches at &lt;89% identity). Phylogenetic analysis rooted the 
               <i>Wolbachia</i> sequences into one of the two main subtrees of the genus, further confirming their
               taxonomic placement (
               <span ref-type="fig" rid="f2">Fig. 2</span>).
            </p>
            <p>The identification of 
               <i>Wolbachia</i> sequences in the testes of an 
               <i>An. gambiae</i> male prompted us to analyse the remaining mosquito specimens collected from the same
               mating swarms in the three villages. To this aim, a more sensitive PCR-based amplification
               and sequencing of a 
               <i>16S</i> rRNA gene segment comprising three variable regions (V6, V7 and V8) was utilized
               to specifically detect 
               <i>Wolbachia</i> in ovaries and testes dissected from the remaining 51 mating couples, using previously
               validated primers
               <span ref-type="bibr" rid="b16">16</span>. The mosquito carcasses were also examined using the same method. Out of the 102
               mosquitoes analysed, 11 were positive for 
               <i>Wolbachia 16S</i> rRNA, leading to a frequency of infection of 10.8% similarly distributed between
               males and females (5 males and 6 females) (
               <span ref-type="supplementary-material" rid="S1">Supplementary Table 1</span>). Interestingly, 
               <i>Wolbachia</i> sequences were PCR amplified from either the reproductive tissues or from the carcasses,
               but never from both, with the exception of one male, where 
               <i>16S</i> sequences were identified by high-throughput sequencing in the testes and by PCR
               in the carcass. Although sequences corresponding to these endosymbionts were more
               prevalent in M mosquitoes, the difference in frequency between the two species was
               not statistically significant (12.8% in M samples compared with 4.2% in S samples).
               We also amplified DNA from fourth instar larvae collected from the same three villages
               (100 specimens from Soumousso, 80 specimens from VK5 and 70 specimens from VK7) using
               the same primer sets described above. 
               <i>Wolbachia</i> sequences were found in five larval samples (three from Soumousso, one from VK5 and
               one from VK7), suggesting occurrence of maternal transmission (
               <span ref-type="supplementary-material" rid="S1">Supplementary Table 2</span>).
            </p>
         </div>
         <div id="">
            <h3>
               <i>Wolbachia</i> sequences group into two different clusters
            </h3>
            <p>Analysis of the amplified regions determined the presence of at least two distinct
               clusters of 
               <i>Wolbachia</i> sequences (
               <span ref-type="supplementary-material" rid="S1">Supplementary Fig. 1</span>). One cluster was identical to the reference strains 
               <i>w</i>AlbB isolated from 
               <i>A. albopictus</i>
               <span ref-type="bibr" rid="b17">17</span>, while the second was closely related to several 
               <i>w</i>Pip strains isolated from 
               <i>Culex</i> mosquitoes and 
               <i>Drosophila</i> species
               <span ref-type="bibr" rid="b18">18</span>. The 
               <i>w</i>AlbB-like cluster was detected only in reproductive tissues (ovaries and testes),
               while 
               <i>w</i>Pip-like sequences were more widespread and were found in ovaries, carcasses and whole
               larvae (
               <span ref-type="supplementary-material" rid="S1">Supplementary Fig. 1</span>). Although these 
               <i>Wolbachia</i> ribosomal sequences are not host specific, the identification of two distinct clusters
               suggests the occurrence of independent 
               <i>Wolbachia</i> infections in 
               <i>An. gambiae</i>, as observed in other hosts
               <span ref-type="bibr" rid="b19">19</span>.
            </p>
            <p>We next analysed the distribution of the 
               <i>Wolbachia</i> sequences among the different villages. A larger number of positive individuals were
               isolated from mating swarms in VK5, where 7 out of the 36 collected mosquitoes showed
               evidence of infection (19.4%). The other 4 positive samples were found in VK7 (3 samples
               out of 42, 7.1%) and Soumousso (1 sample out of 24, 4.2%) (
               <span ref-type="supplementary-material" rid="S1">Supplementary Table 1</span>). Strikingly, six of the seven positive samples isolated from VK5 had been collected
               in just two of the seven mating swarms analysed in that village (
               <span ref-type="supplementary-material" rid="S1">Supplementary Table 1</span>). If confirmed, such clustering of 
               <i>Wolbachia</i>-positive individuals in specific swarms would suggest that ecological and environmental
               factors might play a key role in the establishment of 
               <i>Wolbachia</i> infections in the 
               <i>An. gambiae</i> host.
            </p>
         </div>
         <div id="">
            <h3>The 
               <i>Wolbachia</i> strain belongs to a new phylogenetic group
            </h3>
            <p>To expand the 
               <i>16S</i> rRNA-based analysis and better characterize the 
               <i>Wolbachia</i> strain found in 
               <i>An. gambiae</i>, we performed whole-genome shotgun (WGS) metagenomic sequencing of two of the 
               <i>Wolbachia</i>-positive samples from mosquito ovaries (
               <span ref-type="supplementary-material" rid="S1">Supplementary Table 3</span>). Sequences corresponding to the 
               <i>An. gambiae</i> genome were screened out, and a pipeline for detecting 
               <i>Wolbachia</i>-specific sequences based on the unique marker approach was applied
               <span ref-type="bibr" rid="b20">20</span> (see Methods). A total of 571 reads uniquely attributable to 
               <i>Wolbachia</i> were detected (
               <span ref-type="fig" rid="f3">Fig. 3a,b</span>, 
               <span ref-type="supplementary-material" rid="S1">Supplementary Fig. 3a,b</span>) with 86±3.9 and 87.0±4.4% average identity for the two samples analysed, which is
               in line with the sequence divergence observed for 
               <i>Wolbachia</i> strains in different supergroups (
               <span ref-type="supplementary-material" rid="S1">Supplementary Fig. 2</span>). These sequences matched 134 
               <i>Wolbachia</i> genes belonging to different functional categories (
               <span ref-type="fig" rid="f3">Fig. 3c</span>, 
               <span ref-type="supplementary-material" rid="S1">Supplementary Fig. 3c</span>). The majority of reads (32.2% on average across the two samples) matched genes from
               metabolic pathways, while a relevant number of reads (13.3%) corresponded to 
               <i>Wolbachia</i>-specific transposases; this is in line with the observation that transposases abundantly
               populate the genome of 
               <i>Wolbachia</i> strains from many insects including mosquitoes (for example, transposases correspond
               to 8.32% of all genes annotated in the 
               <i>w</i>Pip genome
               <span ref-type="bibr" rid="b18">18</span>). Alignment of our reads to eight fully or partially sequenced 
               <i>Wolbachia</i> reference genomes (
               <span ref-type="supplementary-material" rid="S1">Supplementary Table 4</span>) demonstrated that the strain identified here belongs to a potential 
               <i>Anopheles</i>-specific phylogenetic supergroup, distinct from the arthropod-associated and evolutionarily
               related supergroups A and B (
               <span ref-type="fig" rid="f3">Fig. 3d</span>, 
               <span ref-type="supplementary-material" rid="S1">Supplementary Fig. 3D</span>). We henceforth call this strain 
               <i>w</i>Anga.
            </p>
         </div>
         <div id="">
            <h3>
               <i>w</i>Anga is maternally transmitted
            </h3>
            <p>The presence of 
               <i>Wolbachia</i> DNA in the reproductive tissues of female adults prompted the question of whether
               these bacteria are inherited from mother to offspring, an essential prerequisite for
               their spread through a population. To obtain irrefutable proof of vertical transmission
               and estimate its efficiency, semi-gravid females were collected from houses in VK5
               two seasons after the initial collections. The same sets of 
               <i>Wolbachia 16S</i> fragments were identified at a frequency of 21% (19 out of 91 females), and the progenies
               of the 5 
               <i>Wolbachia</i>-positive females that laid eggs were then analysed for infection. Occurrence of maternal
               transmission was detected in all progenies, with an average transmission frequency
               of 68% (ranging from 56 to 100%) (
               <span ref-type="fig" rid="f4">Fig. 4</span>). Taken together, these data confirm the presence of 
               <i>Wolbachia</i> sequences over the course of 2 years and indicate occurrence of vertical transmission
               from mother to offspring, as normally observed in 
               <i>Wolbachia</i> infections.
            </p>
         </div>
      </div>
      <div id="">
         <h2>Discussion</h2>
         <p>The identification of genomic sequences from a novel 
            <i>Wolbachia</i> strain in two incipient species of 
            <i>An. gambiae</i> over the course of different seasons suggests that anopheline mosquitoes naturally
            harbour these bacteria, prompting renewed efforts to exploit 
            <i>Wolbachia</i> to block malaria transmission. Past attempts to identify 
            <i>Wolbachia</i> in these mosquitoes may have failed due to possible methodological limitations in
            the detection systems used, including non-optimal DNA amplification and extraction
            methods, and size of the sampled mosquito population
            <span ref-type="bibr" rid="b11">11</span>
            <span ref-type="bibr" rid="b12">12</span>
            <span ref-type="bibr" rid="b13">13</span>. In addition, the newly identified 
            <i>w</i>Anga strain appears to be highly divergent from 
            <i>Wolbachia</i> strains isolated in other insects. Indeed, our attempts to amplify by PCR two 
            <i>Wolbachia</i>-specific genes commonly used in phylogenetic analyses, the 
            <i>wolbachia surface protein wsp</i> and the 
            <i>fructose-biphosphate aldolase fbpA</i>, were unsuccessful despite numerous attempts (see 
            <span ref-type="supplementary-material" rid="S1">Supplementary Table 5</span> for primer sets used), suggesting a low degree of sequence conservation. Interestingly,
            in some 
            <i>Wolbachia</i> strains infecting 
            <i>C. pipiens</i>, the 
            <i>wsp</i>B gene is disrupted by the insertion of an IS256 transposon
            <span ref-type="bibr" rid="b21">21</span>, which belongs to the same family of transposons identified in 
            <i>w</i>Anga. Similar transposon insertions into 
            <i>wsp</i> may have occurred in the 
            <i>Wolbachia</i> strain infecting 
            <i>An. gambiae</i>, compromising the amplification of this gene.
         </p>
         <p>Although examples of horizontal gene transfer (HGT) between 
            <i>Wolbachia</i> and insect hosts are widespread
            <span ref-type="bibr" rid="b22">22</span>
            <span ref-type="bibr" rid="b23">23</span>, two major observations strongly argue against the possibility of HGT of 
            <i>Wolbachia</i> sequences into the 
            <i>An. gambiae</i> genome: (1) with one exception, evidence of infection was identified in reproductive
            tissues (ovaries and testes) from nine females and males but not in the carcasses
            dissected from the same individuals, or vice versa, ruling against a possible transfer
            into the mosquito chromosomes; (2) the average coverage of 
            <i>Wolbachia</i> in our WGS samples was lower than 0.05 × , which is incompatible with the three orders
            of magnitude higher coverage of the 
            <i>An. gambiae</i> genome in the same samples (&gt;75 × ). Alternatively, HGT may have occurred into a
            bacterium or eukaryotic microorganism that infects the 
            <i>Anopheles</i> germline and is maternally inherited from mother to progeny (based on the evidence
            of vertical transmission of 
            <i>16S</i> sequences). Regardless of their origin, the 
            <i>Wolbachia</i> sequences identified here may still be sufficient to induce 
            <i>Wolbachia-</i>like reproductive phenotypes, such as bidirectional cytoplasmic incompatibility, that
            would impact future field deployments of experimental 
            <i>Wolbachia</i> infections
            <span ref-type="bibr" rid="b4">4</span>.
         </p>
         <p>The unexpected discovery in the mosquito germline of maternally inherited 
            <i>Wolbachia</i> organisms will prompt further studies of the ecological, environmental and genetic
            determinants of susceptibility of 
            <i>Anopheles</i> mosquitoes to 
            <i>Wolbachia</i> infections. It will also spark critical investigation into whether natural 
            <i>Wolbachia–Anopheles</i> associations limit the development of 
            <i>Plasmodium</i> parasites in the mosquito host, thus aiding the design of novel effective bacterial
            infection strategies to control malaria transmission.
         </p>
      </div>
      <div id="">
         <h2>Methods</h2>
         <div id="">
            <h3>Mosquito collections</h3>
            <p>Mosquito samples were initially collected during August–September 2011 in three different
               sites near Bobo-Dioulasso, Burkina Faso. The village of Soumousso (11°00′N; 4°02′W)
               is located 55 km North-East of Bobo-Dioulasso. It is characterized by wooded savannah
               and by temporary breeding sites, more favourable to S form (
               <i>An. gambiae</i>)
               <span ref-type="bibr" rid="b14">14</span>. The two other collection sites are located in Vallée du Kou, a large rice-growing
               area situated 30 km North-West of Bobo-Dioulasso. The village of VK5 (11°23′N; 4°24′W)
               is completely surrounded by rice fields, while VK7 (11°24′N; 04°24′W) is characterized
               by rice fields to the South and by Savannah to the North. Because of the irrigation
               system, rice fields form permanent mosquito breeding sites in which the M form (
               <i>An. coluzzii</i>) thrives
               <span ref-type="bibr" rid="b14">14</span>. Nonetheless, few transient breeding sites could be found in depressions and ponds
               within the villages. Male and female adult mosquitoes were collected 
               <i>in copula</i>
               <span ref-type="bibr" rid="b24">24</span> from an average of six mating swarms per collection day. Fourth instar larvae were
               also collected from temporary and permanent water pools from each site. A schematic
               representation of the villages and swarm locations is provided in 
               <span ref-type="fig" rid="f1">Fig. 1</span>. In August 2013, blood-fed 
               <i>An. gambiae</i> females were collected inside the houses in VK5 and allowed to individually oviposit
               in the insectary.
            </p>
         </div>
         <div id="">
            <h3>DNA extraction and species genotyping</h3>
            <p>Genomic DNA was extracted from dissected reproductive tissues (testes and ovaries)
               using DNeasy kit (Qiagen), and from carcasses using NucleoSpin 96 Tissue kit (Macherey-Nagel).
               In 2013, to estimate 
               <i>Wolbachia</i> maternal transmission, DNA was extracted using DNeasy kit (Qiagen) from whole females
               that were allowed to oviposit, and from their progenies. For M and S genotyping, DNA
               was extracted from a leg using a fast extraction method. In brief, individual legs
               were incubated in 40 μl of grinding buffer (10 mM Tris-HCl pH 8.2, 1 mM EDTA, 25 mM
               NaCl) with 0.2 mg ml
               <sup>−1</sup> proteinase K for 45 min at 37 °C, then 5 min at 95 °C to inactivate the enzyme. DNA
               extracts (1 μl) were then subjected to PCR amplification targeting the locus S200
               × 6.1 using specific primers (FWD: 5′-TCGCCTTAGACCTTGCGTTA-3′; and REV: 5′-CGCTTCAAGAATTCGAGATAC-3′)
               <span ref-type="bibr" rid="b25">25</span>. M and S genotyping was also used on larvae and adult carcasses. Larval DNA was used
               to determine the sex using Y-specific primers (F: 5′-CAAAACGACAGCAGTTCC-3′; and R:
               5′-TAAACCAAGTCCGTCGCT-3′).
            </p>
         </div>
         <div id="">
            <h3>
               <i>16S</i> rRNA profiling and sequencing
            </h3>
            <p>The 
               <i>16S</i> rRNA gene data set consisted of Illumina MiSeq sequences targeting the V4 variable
               region. Detailed protocols used for 
               <i>16S</i> amplification and sequencing were previously described
               <span ref-type="bibr" rid="b26">26</span>. In brief, genomic DNA from testes and ovaries was subjected to 
               <i>16S</i> rRNA amplifications using primers incorporating the Illumina adapters and a sample
               barcode sequence, allowing directional sequencing covering the variable region V4
               (15F: 5′-GTGCCAGCMGCCGCGGTAA-3′; and 806R: 5′-GGACTACHVGGGTWTCTAAT-3′). PCR mixtures
               contained 10 μl of diluted template (1:50), 10 μl of HotMasterMix with the HotMaster
               Taq DNA Polymerase (5 Prime) and 5 μl of primer mix (2 μM of each primer). The cycling
               conditions consisted of an initial denaturation at 94 °C for 3 min, followed by 30
               cycles of denaturation at 94 °C for 45 s, annealing at 50 °C for 60 s, extension at
               72 °C for 5 min and a final extension at 72 °C for 10 min. Amplicons were quantified
               on the Caliper LabChipGX (PerkinElmer, Waltham, MA), pooled in equimolar concentrations,
               size selected (375–425 bp) on the Pippin Prep (Sage Sciences, Beverly, MA) to reduce
               nonspecific amplification products from host DNA and a final library size and quantification
               was done on an Agilent Bioanalyzer 2100 DNA 1000 chips (Agilent Technologies, Santa
               Clara, CA). Sequencing was performed on the Illumina MiSeq v2 platform, according
               to the manufacturer’s specifications with addition of 5% PhiX, generating paired-end
               reads of 175 bp in length in each direction. The overlapping paired-end reads were
               stitched together (~\n97 bp overlap) and size selected to reduce nonspecific amplification
               products from host DNA (225–275 bp).
            </p>
         </div>
         <div id="">
            <h3>High-throughput 
               <i>16S</i> rRNA screening
            </h3>
            <p>We first applied the QIIME pipeline version 1.6 (ref. 
               <span ref-type="bibr" rid="b27">27</span>) to the 
               <i>16S</i> rRNA data set that detected in the testis sample G23656 (male SMS5.1 in 
               <span ref-type="supplementary-material" rid="S1">Supplementary Fig. 1</span>) a total of 19.9% of reads assigned to 
               <i>Wolbachia</i>. To specifically investigate and validate the prediction about the presence of 
               <i>Wolbachia</i> in the sample, we implemented an additional pipeline. We first integrated all 
               <i>16S</i> rRNA sequences assigned to 
               <i>Wolbachia</i> included in the SILVA
               <span ref-type="bibr" rid="b28">28</span> and NCBI repositories; we manually inspected those sequences with a &gt;3% nucleotide
               divergence from any other 
               <i>Wolbachia 16S</i>, which let us exclude five sequences that were mislabeled with close endosymbionts
               such as 
               <i>Bartonella</i>, 
               <i>Rickettsia</i> and 
               <i>Francisella</i>. We obtained a set of 2,064 
               <i>Wolbachia 16S</i> sequences from which the 253-nt-long V4 region was extracted and clustered at 99%
               identity, generating 115 operational taxonomic units (OTUs) covering the diversity
               in the 
               <i>Wolbachia</i> genus. The all-versus-all mapping of the 115 OTU sequence representatives provided
               a lower-bound estimate of the genus’ total diversity (average identity of 95.9%, s.d.
               1.75%, median 96.0%).
            </p>
            <p>We then performed the mapping of the 24,800 reads of sample G23656 against the full
               SILVA database, retaining only those sequences with full-length percentage identity
               of at least 95% with a 
               <i>Wolbachia 16S</i>. The resulting 5,412 reads have a best hit other than 
               <i>Wolbachia</i> at &lt;89% identity, confirming that a fifth of the sample (21.82%) consists of V4 fragments
               from 
               <i>Wolbachia 16S</i> rRNA. The percentage identity was in the 95.3–97.6% interval, which is fully consistent
               with the observed diversity in the V4 hypervariable region of the 
               <i>Wolbachia 16S</i> rRNA genes (average 95.9%, s.d. 1.75% as reported above).
            </p>
            <p>The V4 sequences from sample G23656 were then clustered into OTUs at 99% identity,
               discarding those OTUs with &lt;50 sequences. The resulting OTUs were merged with the
               
               <i>Wolbachia</i> OTUs from the SILVA and NCBI repository (generated as described above) and aligned
               with MUSCLE version 3.8.31 (ref. 
               <span ref-type="bibr" rid="b29">29</span>). A V4 sequence representative of the Rickettsia 
               <i>16S</i> was added as outgroup. A phylogenetic tree was then built using RAxML version 7.4.2
               (ref. 
               <span ref-type="bibr" rid="b30">30</span>) with the GTRGAMMA model, bootstrapping (1,000 replicates), best maximum likelihood
               tree inference, and displayed with GraPhlAn ( 
               <div tagxxx="ext-link">https://bitbucket.org/nsegata/graphlan</div>) representing the cardinality of the OTUs as circular barplots.
            </p>
         </div>
         <div id="">
            <h3>
               <i>Wolbachia</i>-specific PCR detection and sequencing
            </h3>
            <p>DNA from testes and ovaries was rehydrated from the 96-well Qiasafe plate (Qiagen)
               using 30 μl of water. A total of 2 μl of DNA was used for 
               <i>Wolbachia</i> PCR detection using primers specific for 
               <i>Wolbachia 16S</i> rDNA (W-Specf 5′-CATACCTATTCGAAGGGATAG-3′, W-Specr 5′-AGCTTCGAGTGAAACCAATTC-3′) following
               standard procedures
               <span ref-type="bibr" rid="b16">16</span>. Positive samples showed a 438-bp band that was purified with QIAquick Gel Extraction
               kit (Qiagen) and sequenced (Eurofins MWG Operon, Ebersberg, Germany). Sample DNA quality
               was assessed with PCR using primers for the 
               <i>RpS7</i> (AGAP010592) 
               <i>An. gambiae</i> gene (FWD 5′-GGCGATCATCATCTACGTGC-3′; and REV 5′-GTAGCTGCTGCAAACTTCGG-3′). Similarly,
               a total of 2 μl of DNA was used for PCR detection of 
               <i>wsp</i> and 
               <i>fbpA</i> genes following standard procedures (see 
               <span ref-type="supplementary-material" rid="S1">Supplementary Table 5</span> for primer sets used).
            </p>
         </div>
         <div id="">
            <h3>WGS sequencing and analysis pipeline</h3>
            <p>Shotgun metagenomic sequencing was performed on two mosquito samples from infected
               ovaries using the remainder of the DNA available after 
               <i>Wolbachia-</i>specific PCR detection (1–2 ng). Due to this limiting condition, libraries were prepared
               with 1 ng DNA according to the Nextera XT protocol (Version Oct 2012). Briefly, the
               DNA was fragmented in 5 μl of Amplicon Tagment Mix and 10 μl of Tagment DNA buffer
               (Illumina, San Diego, CA, USA). Tagmentation reactions were completed by incubation
               for 5 min at 55 °C followed by neutralization with 5 μl of Neutralise Tagment Buffer
               for 5 min. Tagmented DNA was used as the template in a 50-μl limited-cycle PCR (12
               cycles) and processed as described in the Nextera XT protocol. Amplified DNA was purified
               with AMPure XP beads and then normalized to 2nM. Sequencing was performed on a HiSeq2000
               (Illumina, San Diego, CA, USA) employing one full lane per library with 101 bp paired-end
               reads.
            </p>
            <p>Raw WGS results consisting of &gt;400 M 101-nt-long paired-end reads (
               <span ref-type="supplementary-material" rid="S1">Supplementary Table 3</span>) were subject to quality control and sliding window trimming with a minimum resulting
               read length of 80 bp, and to sequencing artefact removal using PRINSEQ version 0.20.3
               (ref. 
               <span ref-type="bibr" rid="b31">31</span>). As expected, 
               <i>An. gambiae</i> DNA was quantitatively dominant in the read pool, and was removed by BowTie2 mapping
               <span ref-type="bibr" rid="b32">32</span> using the ‘very-sensitive’ preset option against the 
               <i>An. gambiae</i> PEST reference genome ( 
               <div tagxxx="ext-link">http://www.vectorbase.org/</div>). 
               <span ref-type="supplementary-material" rid="S1">Supplementary Table S2</span> reports the number of reads that were retained after each pre-processing step.
            </p>
            <p>The resulting read set was mapped against the seven available 
               <i>Wolbachia</i> genomes and the high-quality draft assembly of the 
               <i>Wolbachia</i> strain 
               <i>w</i>AlbB (
               <span ref-type="supplementary-material" rid="S1">Supplementary Table 4</span>). To quantify the expected sequence divergence between 
               <i>Wolbachia</i> strains and supergroups, we performed all-versus-all sequence mapping (with BLASTN)
               with all open reading frames (ORFs), considering as pairwise common ORFs those sequences
               with &gt;80% identity over 50% of the ORF length (
               <span ref-type="supplementary-material" rid="S1">Supplementary Fig. 2</span>). Reads were uniquely attributable to 
               <i>Wolbachia</i> on the basis of the concept of unique marker sequences
               <span ref-type="bibr" rid="b20">20</span>, performed in four steps. First step: BowTie2 mapping against the eight 
               <i>Wolbachia</i> reference genomes was performed to identify 
               <i>Wolbachia</i> candidate reads. The mapping was performed with enhanced sensitivity (score-min L,-1.0,-1.0
               -D 25 -R 5 -N 1 -L 12 -i S,2,0.25) to capture 
               <i>Wolbachia</i> sequence divergence and host specificity as assessed by ORFs’ sequence comparison
               among available 
               <i>Wolbachia</i> genomes (
               <span ref-type="supplementary-material" rid="S1">Supplementary Fig. 3</span>). The matches were also confirmed by BLASTN with word size of length 7. Second step:
               candidate reads coming from the small and large ribosomal units (
               <i>16S</i> rRNA and 
               <i>23S</i> rRNA genes) were screened out by sequence mapping against the comprehensive ribosomal
               sequences in the SILVA database release 111. Third step: the remaining ribosomal-free
               candidate reads were mapped against the full RefSeq genomic database version 60 to
               identify any non-
               <i>Wolbachia</i>-specific hits using BLASTN with word size of length 7. Fourth step: on the basis
               of the mapping results of steps 1 and 3, the final set of reads from the 
               <i>Wolbachia</i> strain identified in 
               <i>An. gambiae</i> (
               <i>w</i>Anga) was compiled selecting all reads showing &gt;80% identity over &gt;95 nt to at least
               one 
               <i>Wolbachia</i> strain, and no hits longer than 80 nt to other organisms. Reads hitting non-
               <i>Wolbachia</i> genomes with identities below 80% and at least one 
               <i>Wolbachia</i> genome at &gt;90% were also retained. As control, the same procedure was also applied
               to the two genera closest to 
               <i>Wolbachia</i> according to the PhyloPhlAn tree of life
               <span ref-type="bibr" rid="b33">33</span>, namely, 
               <i>Anaplasma</i> (six reference genomes) and 
               <i>Ehrlichia</i> (five reference genomes). No uniquely attributable reads were found in this analysis.
            </p>
            <p>
               <i>w</i>Anga reads mapping to all seven 
               <i>Wolbachia</i> genomes were then retained for phylogenetic analysis. The homologous sequences were
               extracted and aligned to each 
               <i>w</i>Anga reads with MUSCLE version 3.8.31 (ref. 
               <span ref-type="bibr" rid="b29">29</span>) and the alignments edited to remove leading and ending gaps. Sequence-specific phylogenetic
               trees were built using RAxML version 7.4.2 (ref. 
               <span ref-type="bibr" rid="b30">30</span>) with the GTRGAMMA model, bootstrapping (1,000 replicates) and best maximum likelihood
               tree inference. Sequence-specific phylogenetic distances were computed inferring the
               patristic distances within each tree, and reported with box plots in 
               <span ref-type="fig" rid="f3">Fig. 3b</span> and 
               <span ref-type="supplementary-material" rid="S1">Supplementary Fig. 3B</span>. A final phylogenetic tree was also built using RAxML (GTRGAMMA model, 1,000 bootstrapping
               replicates) on the concatenated alignments (
               <span ref-type="fig" rid="f3">Fig. 3d</span>, 
               <span ref-type="supplementary-material" rid="S1">Supplementary Fig. 3D</span>).
            </p>
         </div>
      </div>
      <div id="">
         <h2>Author contributions</h2>
         <p>F.C., E.A.L., F.B., J.P., N.S. and A.D. designed the experiments. F.B., J.P., W.R.S.
            and P.M. performed the experiments. F.C., N.S., F.B., J.P. and E.A.L. analysed the
            data. N.S. performed the sequencing analysis. R.K.D. and A.D. provided samples for
            the analysis. F.C., N.S., F.B., J.P., E.A.L. and A.D. wrote the manuscript. F.B.,
            N.S. and J.P. contributed equally to this study.
         </p>
      </div>
      <div id="">
         <h2>Additional information</h2>
         <p>
            <b>Accession codes:</b> Nucleotide sequences of PCR-amplified fragments of 
            <i>Wolbachia</i> 16S rRNA genes have been deposited in the GenBank nucleotide database under accession
            codes 
            <div tagxxx="ext-link">KJ728739</div> to 
            <div tagxxx="ext-link">KJ728755</div>. Sequence reads for the same 16S-rRNA amplicon sequences have been deposited in the
            NCBI Sequence Read Archive (SRA) under accession code 
            <div tagxxx="ext-link">SRR610826</div>. Sequence reads for the two whole-genome shotgun sequencing samples have been deposited
            in the NCBI Sequence Read Archive (SRA) under accession codes 
            <div tagxxx="ext-link">SRR1238105</div> to 
            <div tagxxx="ext-link">SRR1238106</div>.
         </p>
         <p>
            <b>How to cite this article:</b> Baldini, F. 
            <i>et al.</i> Evidence of natural 
            <i>Wolbachia</i> infections in field populations of 
            <i>Anopheles gambiae</i>. 
            <i>Nat. Commun.</i> 5:3985 doi: 10.1038/ncomms4985 (2014).
         </p>
      </div>
      <div id="S1">
         <h2>Supplementary Material</h2>
         <div tagxxx="supplementary-material">
            <caption>
               <div tagxxx="title">Supplementary Information</div>
               <p>Supplementary Figures 1-3, Supplementary Tables 1-5 and Supplementary References</p>
            </caption>
            <div tagxxx="media"></div>
         </div>
      </div>
      <div tagxxx="back">
         <div tagxxx="ack">
            <p>We thank S. Ilboudo, H. Guel, D. D. Soma, S. Sawadogo and E. Bilgo for help with sample
               collection and dissection, V. De Sanctis and R. Bertorelli (NGS Facility at the Centre
               for Integrative Biology and LaBSSAH, University of Trento) for WGS sequencing and
               D.E. Neafsey and L. Waldron for helpful discussions. We thank the Broad sequencing
               platform for 
               <i>16S</i> sequencing. This work was funded by a European Research Council FP7 ERC Starting
               Grant project ‘Anorep’ (grant ID: 260897), a Biotechnology and Biological Sciences
               Research Council grant (grant BB/I002898/1) and a Wellcome Trust grant (grant ID:
               093553) to F.C., by the European Union FP7 grant (PCIG13-618833) and startup funding
               at the Centre for Integrative Biology (University of Trento) to N.S., by the EC FP7
               Networks of Excellence EVIMalaR (242095) to E.A.L. and F.C., and by an ANR-11-BSV7-009-01
               GIME to E.A.L. F.C. was supported by a NIH grant (grant ID: NIH 1R01AI104956-01A1).
               J.P. was supported by fellowships from FRM and the Fondation Des Treilles.
            </p>
         </div>
         <div tag="ref-list">
            <ul>
               <li tag="ref">
                  <div tagxxx="mixed-citation">World Health Organization. 
                     <span tagx="source">World Malaria Report: 2013</span> World Health Organization (
                     <span tagx="year">2013</span>).
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Ranson</span>
                        <span tagx="given-names">H.</span>
                     </div>
                     <i>et al.</i>
                     <span tagx="article-title">Pyrethroid resistance in African anopheline mosquitoes: what are the implications
                        for malaria control?
                     </span>
                     <span tagx="source">Trends Parasitol.</span>
                     <span tagx="volume">27</span>, 
                     <span tagx="fpage">91</span>–
                     <span tagx="lpage">98</span> (
                     <span tagx="year">2011</span>).
                     <div tagxxx="pub-id">20843745</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Walker</span>
                        <span tagx="given-names">T.</span>
                     </div> &amp; 
                     <div tagxxx="name">
                        <span tagx="surname">Moreira</span>
                        <span tagx="given-names">L. A.</span>
                     </div>
                     <span tagx="article-title">Can 
                        <i>Wolbachia</i> be used to control malaria?
                     </span>
                     <span tagx="source">Mem. Inst. Oswaldo Cruz</span>
                     <span tagx="volume">106</span>, 
                     <span tagx="fpage">212</span>–
                     <span tagx="lpage">217</span> (
                     <span tagx="year">2011</span>).
                     <div tagxxx="pub-id">21881776</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Hoffmann</span>
                        <span tagx="given-names">A.</span>
                     </div>
                     <i>et al.</i>
                     <span tagx="article-title">Successful establishment of 
                        <i>Wolbachia</i> in 
                        <i>Aedes</i> populations to suppress dengue transmission
                     </span>. 
                     <span tagx="source">Nature</span>
                     <span tagx="volume">476</span>, 
                     <span tagx="fpage">454</span>–
                     <span tagx="lpage">457</span> (
                     <span tagx="year">2011</span>).
                     <div tagxxx="pub-id">21866160</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Bian</span>
                        <span tagx="given-names">G.</span>
                     </div>
                     <i>et al.</i>
                     <span tagx="article-title">
                        <i>Wolbachia</i> invades 
                        <i>Anopheles stephensi</i> populations and induces refractoriness to 
                        <i>Plasmodium</i> infection
                     </span>. 
                     <span tagx="source">Science</span>
                     <span tagx="volume">340</span>, 
                     <span tagx="fpage">748</span>–
                     <span tagx="lpage">751</span> (
                     <span tagx="year">2013</span>).
                     <div tagxxx="pub-id">23661760</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Hughes</span>
                        <span tagx="given-names">G. L.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Koga</span>
                        <span tagx="given-names">R.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Xue</span>
                        <span tagx="given-names">P.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Fukatsu</span>
                        <span tagx="given-names">T.</span>
                     </div> &amp; 
                     <div tagxxx="name">
                        <span tagx="surname">Rasgon</span>
                        <span tagx="given-names">J. L.</span>
                     </div>
                     <span tagx="article-title">
                        <i>Wolbachia</i> infections are virulent and inhibit the human malaria parasite 
                        <i>Plasmodium falciparum in Anopheles gambiae</i>
                     </span>. 
                     <span tagx="source">PLoS Pathog.</span>
                     <span tagx="volume">7</span>, 
                     <span tagx="fpage">e1002043</span> (
                     <span tagx="year">2011</span>).
                     <div tagxxx="pub-id">21625582</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Kambris</span>
                        <span tagx="given-names">Z.</span>
                     </div>
                     <i>et al.</i>
                     <span tagx="article-title">
                        <i>Wolbachia</i> stimulates immune gene expression and inhibits 
                        <i>Plasmodium</i> development in 
                        <i>Anopheles gambiae</i>
                     </span>. 
                     <span tagx="source">PLoS Pathog.</span>
                     <span tagx="volume">6</span>, 
                     <span tagx="fpage">e1001143</span> (
                     <span tagx="year">2010</span>).
                     <div tagxxx="pub-id">20949079</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Murdock</span>
                        <span tagx="given-names">C. C.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Blanford</span>
                        <span tagx="given-names">S.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Hughes</span>
                        <span tagx="given-names">G. L.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Rasgon</span>
                        <span tagx="given-names">J. L.</span>
                     </div> &amp; 
                     <div tagxxx="name">
                        <span tagx="surname">Thomas</span>
                        <span tagx="given-names">M. B.</span>
                     </div>
                     <span tagx="article-title">Temperature alters 
                        <i>Plasmodium</i> blocking by 
                        <i>Wolbachia</i>
                     </span>. 
                     <span tagx="source">Sci. Rep.</span>
                     <span tagx="volume">4</span>, 
                     <span tagx="fpage">3932</span> (
                     <span tagx="year">2014</span>).
                     <div tagxxx="pub-id">24488176</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Jeyaprakash</span>
                        <span tagx="given-names">A.</span>
                     </div> &amp; 
                     <div tagxxx="name">
                        <span tagx="surname">Hoy</span>
                        <span tagx="given-names">M.</span>
                     </div>
                     <span tagx="article-title">Long PCR improves 
                        <i>Wolbachia</i> DNA amplification: wsp sequences found in 76% of sixty‐three arthropod species
                     </span>. 
                     <span tagx="source">Insect Mol. Biol.</span>
                     <span tagx="volume">9</span>, 
                     <span tagx="fpage">393</span>–
                     <span tagx="lpage">405</span> (
                     <span tagx="year">2000</span>).
                     <div tagxxx="pub-id">10971717</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Hilgenboecker</span>
                        <span tagx="given-names">K.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Hammerstein</span>
                        <span tagx="given-names">P.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Schlattmann</span>
                        <span tagx="given-names">P.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Telschow</span>
                        <span tagx="given-names">A.</span>
                     </div> &amp; 
                     <div tagxxx="name">
                        <span tagx="surname">Werren</span>
                        <span tagx="given-names">J. H.</span>
                     </div>
                     <span tagx="article-title">How many species are infected with 
                        <i>Wolbachia</i>?–a statistical analysis of current data
                     </span>. 
                     <span tagx="source">FEMS Microbiol. Lett.</span>
                     <span tagx="volume">281</span>, 
                     <span tagx="fpage">215</span>–
                     <span tagx="lpage">220</span> (
                     <span tagx="year">2008</span>).
                     <div tagxxx="pub-id">18312577</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Kittayapong</span>
                        <span tagx="given-names">P.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Baisley</span>
                        <span tagx="given-names">K. J.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Baimai</span>
                        <span tagx="given-names">V.</span>
                     </div> &amp; 
                     <div tagxxx="name">
                        <span tagx="surname">O'Neill</span>
                        <span tagx="given-names">S. L.</span>
                     </div>
                     <span tagx="article-title">Distribution and diversity of 
                        <i>Wolbachia</i> infections in Southeast Asian mosquitoes (Diptera: Culicidae)
                     </span>. 
                     <span tagx="source">J. Med. Entomol.</span>
                     <span tagx="volume">37</span>, 
                     <span tagx="fpage">340</span>–
                     <span tagx="lpage">345</span> (
                     <span tagx="year">2000</span>).
                     <div tagxxx="pub-id">15535575</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Ricci</span>
                        <span tagx="given-names">I.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Cancrini</span>
                        <span tagx="given-names">G.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Gabrielli</span>
                        <span tagx="given-names">S.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">D'amelio</span>
                        <span tagx="given-names">S.</span>
                     </div> &amp; 
                     <div tagxxx="name">
                        <span tagx="surname">Favia</span>
                        <span tagx="given-names">G.</span>
                     </div>
                     <span tagx="article-title">Searching for 
                        <i>Wolbachia</i> (Rickettsiales: Rickettsiaceae) in mosquitoes (Diptera: Culicidae): large polymerase
                        chain reaction survey and new identifications
                     </span>. 
                     <span tagx="source">J. Med. Entomol.</span>
                     <span tagx="volume">39</span>, 
                     <span tagx="fpage">562</span>–
                     <span tagx="lpage">567</span> (
                     <span tagx="year">2002</span>).
                     <div tagxxx="pub-id">12144285</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Rasgon</span>
                        <span tagx="given-names">J. L.</span>
                     </div> &amp; 
                     <div tagxxx="name">
                        <span tagx="surname">Scott</span>
                        <span tagx="given-names">T. W.</span>
                     </div>
                     <span tagx="article-title">An initial survey for 
                        <i>Wolbachia</i> (Rickettsiales: Rickettsiaceae) infections in selected California mosquitoes (Diptera:
                        Culicidae)
                     </span>. 
                     <span tagx="source">J. Med. Entomol.</span>
                     <span tagx="volume">41</span>, 
                     <span tagx="fpage">255</span>–
                     <span tagx="lpage">257</span> (
                     <span tagx="year">2004</span>).
                     <div tagxxx="pub-id">15061286</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">della Torre</span>
                        <span tagx="given-names">A.</span>
                     </div>
                     <i>et al.</i>
                     <span tagx="article-title">Speciation within 
                        <i>Anopheles gambiae</i>—the glass is half full
                     </span>. 
                     <span tagx="source">Science</span>
                     <span tagx="volume">298</span>, 
                     <span tagx="fpage">115</span>–
                     <span tagx="lpage">117</span> (
                     <span tagx="year">2002</span>).
                     <div tagxxx="pub-id">12364784</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Coetzee</span>
                        <span tagx="given-names">M.</span>
                     </div>
                     <i>et al.</i>
                     <span tagx="article-title">
                        <i>Anopheles coluzzii</i> and 
                        <i>Anopheles amharicus</i>, new members of the 
                        <i>Anopheles gambiae</i> complex
                     </span>. 
                     <span tagx="source">Zootaxa</span>
                     <span tagx="volume">3619</span>, 
                     <span tagx="fpage">246</span>–
                     <span tagx="lpage">274</span> (
                     <span tagx="year">2013</span>).
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Werren</span>
                        <span tagx="given-names">J. H.</span>
                     </div> &amp; 
                     <div tagxxx="name">
                        <span tagx="surname">Windsor</span>
                        <span tagx="given-names">D. M.</span>
                     </div>
                     <span tagx="article-title">
                        <i>Wolbachia</i> infection frequencies in insects: evidence of a global equilibrium?
                     </span>
                     <span tagx="source">Proc. Biol. Sci.</span>
                     <span tagx="volume">267</span>, 
                     <span tagx="fpage">1277</span>–
                     <span tagx="lpage">1285</span> (
                     <span tagx="year">2000</span>).
                     <div tagxxx="pub-id">10972121</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Mavingui</span>
                        <span tagx="given-names">P.</span>
                     </div>
                     <i>et al.</i>
                     <span tagx="article-title">Whole-genome sequence of 
                        <i>Wolbachia</i> strain 
                        <i>w</i>AlbB, an endosymbiont of tiger mosquito vector 
                        <i>Aedes albopictus</i>
                     </span>. 
                     <span tagx="source">J. Bacteriol.</span>
                     <span tagx="volume">194</span>, 
                     <span tagx="fpage">1840</span>–
                     <span tagx="lpage">1840</span> (
                     <span tagx="year">2012</span>).
                     <div tagxxx="pub-id">22408242</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Klasson</span>
                        <span tagx="given-names">L.</span>
                     </div>
                     <i>et al.</i>
                     <span tagx="article-title">Genome evolution of 
                        <i>Wolbachia</i> strain 
                        <i>w</i>Pip from the 
                        <i>Culex pipiens</i> group
                     </span>. 
                     <span tagx="source">Mol. Biol. Evol.</span>
                     <span tagx="volume">25</span>, 
                     <span tagx="fpage">1877</span>–
                     <span tagx="lpage">1887</span> (
                     <span tagx="year">2008</span>).
                     <div tagxxx="pub-id">18550617</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Ellegaard</span>
                        <span tagx="given-names">K. M.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Klasson</span>
                        <span tagx="given-names">L.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Näslund</span>
                        <span tagx="given-names">K.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Bourtzis</span>
                        <span tagx="given-names">K.</span>
                     </div> &amp; 
                     <div tagxxx="name">
                        <span tagx="surname">Andersson</span>
                        <span tagx="given-names">S. G.</span>
                     </div>
                     <span tagx="article-title">Comparative genomics of 
                        <i>Wolbachia</i> and the bacterial species concept
                     </span>. 
                     <span tagx="source">PLoS Genet.</span>
                     <span tagx="volume">9</span>, 
                     <span tagx="fpage">e1003381</span> (
                     <span tagx="year">2013</span>).
                     <div tagxxx="pub-id">23593012</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Segata</span>
                        <span tagx="given-names">N.</span>
                     </div>
                     <i>et al.</i>
                     <span tagx="article-title">Metagenomic microbial community profiling using unique clade-specific marker genes</span>. 
                     <span tagx="source">Nat. Methods</span>
                     <span tagx="volume">9</span>, 
                     <span tagx="fpage">811</span>–
                     <span tagx="lpage">814</span> (
                     <span tagx="year">2012</span>).
                     <div tagxxx="pub-id">22688413</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Sanogo</span>
                        <span tagx="given-names">Y.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Dobson</span>
                        <span tagx="given-names">S.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Bordenstein</span>
                        <span tagx="given-names">S.</span>
                     </div> &amp; 
                     <div tagxxx="name">
                        <span tagx="surname">Novak</span>
                        <span tagx="given-names">R.</span>
                     </div>
                     <span tagx="article-title">Disruption of the 
                        <i>Wolbachia</i> surface protein gene 
                        <i>wspB</i> by a transposable element in mosquitoes of the 
                        <i>Culex pipiens</i> complex (Diptera, Culicidae)
                     </span>. 
                     <span tagx="source">Insect Mol. Biol.</span>
                     <span tagx="volume">16</span>, 
                     <span tagx="fpage">143</span>–
                     <span tagx="lpage">154</span> (
                     <span tagx="year">2007</span>).
                     <div tagxxx="pub-id">17298560</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Hotopp</span>
                        <span tagx="given-names">J. C. D.</span>
                     </div>
                     <i>et al.</i>
                     <span tagx="article-title">Widespread lateral gene transfer from intracellular bacteria to multicellular eukaryotes</span>. 
                     <span tagx="source">Science</span>
                     <span tagx="volume">317</span>, 
                     <span tagx="fpage">1753</span>–
                     <span tagx="lpage">1756</span> (
                     <span tagx="year">2007</span>).
                     <div tagxxx="pub-id">17761848</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Robinson</span>
                        <span tagx="given-names">K. M.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Sieber</span>
                        <span tagx="given-names">K. B.</span>
                     </div> &amp; 
                     <div tagxxx="name">
                        <span tagx="surname">Hotopp</span>
                        <span tagx="given-names">J. C. D.</span>
                     </div>
                     <span tagx="article-title">A review of bacteria-animal lateral gene transfer may inform our understanding of
                        diseases like cancer
                     </span>. 
                     <span tagx="source">PLoS Genet.</span>
                     <span tagx="volume">9</span>, 
                     <span tagx="fpage">e1003877</span> (
                     <span tagx="year">2013</span>).
                     <div tagxxx="pub-id">24146634</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Diabate</span>
                        <span tagx="given-names">A.</span>
                     </div>
                     <i>et al.</i>
                     <span tagx="article-title">Mixed swarms of the molecular M and S forms of 
                        <i>Anopheles gambiae</i> (Diptera: Culicidae) in sympatric area from Burkina Faso
                     </span>. 
                     <span tagx="source">J. Med. Entomol.</span>
                     <span tagx="volume">43</span>, 
                     <span tagx="fpage">480</span>–
                     <span tagx="lpage">483</span> (
                     <span tagx="year">2006</span>).
                     <div tagxxx="pub-id">16739404</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Santolamazza</span>
                        <span tagx="given-names">F.</span>
                     </div>
                     <i>et al.</i>
                     <span tagx="article-title">Insertion polymorphisms of SINE200 retrotransposons within speciation islands of 
                        <i>Anopheles gambiae</i> molecular forms
                     </span>. 
                     <span tagx="source">Malar. J.</span>
                     <span tagx="volume">7</span>, 
                     <span tagx="fpage">163</span> (
                     <span tagx="year">2008</span>).
                     <div tagxxx="pub-id">18724871</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Caporaso</span>
                        <span tagx="given-names">J. G.</span>
                     </div>
                     <i>et al.</i>
                     <span tagx="article-title">Ultra-high-throughput microbial community analysis on the Illumina HiSeq and MiSeq
                        platforms
                     </span>. 
                     <span tagx="source">ISME J.</span>
                     <span tagx="volume">6</span>, 
                     <span tagx="fpage">1621</span>–
                     <span tagx="lpage">1624</span> (
                     <span tagx="year">2012</span>).
                     <div tagxxx="pub-id">22402401</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Caporaso</span>
                        <span tagx="given-names">J. G.</span>
                     </div>
                     <i>et al.</i>
                     <span tagx="article-title">QIIME allows analysis of high-throughput community sequencing data</span>. 
                     <span tagx="source">Nat. Methods</span>
                     <span tagx="volume">7</span>, 
                     <span tagx="fpage">335</span>–
                     <span tagx="lpage">336</span> (
                     <span tagx="year">2010</span>).
                     <div tagxxx="pub-id">20383131</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Quast</span>
                        <span tagx="given-names">C.</span>
                     </div>
                     <i>et al.</i>
                     <span tagx="article-title">The SILVA ribosomal RNA gene database project: improved data processing and web-based
                        tools
                     </span>. 
                     <span tagx="source">Nucleic Acids Res.</span>
                     <span tagx="volume">41</span>, 
                     <span tagx="fpage">D590</span>–
                     <span tagx="lpage">D596</span> (
                     <span tagx="year">2013</span>).
                     <div tagxxx="pub-id">23193283</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Edgar</span>
                        <span tagx="given-names">R. C.</span>
                     </div>
                     <span tagx="article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. 
                     <span tagx="source">Nucleic Acids Res.</span>
                     <span tagx="volume">32</span>, 
                     <span tagx="fpage">1792</span>–
                     <span tagx="lpage">1797</span> (
                     <span tagx="year">2004</span>).
                     <div tagxxx="pub-id">15034147</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Stamatakis</span>
                        <span tagx="given-names">A.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Ludwig</span>
                        <span tagx="given-names">T.</span>
                     </div> &amp; 
                     <div tagxxx="name">
                        <span tagx="surname">Meier</span>
                        <span tagx="given-names">H.</span>
                     </div>
                     <span tagx="article-title">RAxML-III: a fast program for maximum likelihood-based inference of large phylogenetic
                        trees
                     </span>. 
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="volume">21</span>, 
                     <span tagx="fpage">456</span>–
                     <span tagx="lpage">463</span> (
                     <span tagx="year">2005</span>).
                     <div tagxxx="pub-id">15608047</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Schmieder</span>
                        <span tagx="given-names">R.</span>
                     </div> &amp; 
                     <div tagxxx="name">
                        <span tagx="surname">Edwards</span>
                        <span tagx="given-names">R.</span>
                     </div>
                     <span tagx="article-title">Quality control and preprocessing of metagenomic datasets</span>. 
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="volume">27</span>, 
                     <span tagx="fpage">863</span>–
                     <span tagx="lpage">864</span> (
                     <span tagx="year">2011</span>).
                     <div tagxxx="pub-id">21278185</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Langmead</span>
                        <span tagx="given-names">B.</span>
                     </div> &amp; 
                     <div tagxxx="name">
                        <span tagx="surname">Salzberg</span>
                        <span tagx="given-names">S. L.</span>
                     </div>
                     <span tagx="article-title">Fast gapped-read alignment with Bowtie 2</span>. 
                     <span tagx="source">Nat. Methods</span>
                     <span tagx="volume">9</span>, 
                     <span tagx="fpage">357</span>–
                     <span tagx="lpage">359</span> (
                     <span tagx="year">2012</span>).
                     <div tagxxx="pub-id">22388286</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Segata</span>
                        <span tagx="given-names">N.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Bornigen</span>
                        <span tagx="given-names">D.</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Morgan</span>
                        <span tagx="given-names">X. C.</span>
                     </div> &amp; 
                     <div tagxxx="name">
                        <span tagx="surname">Huttenhower</span>
                        <span tagx="given-names">C.</span>
                     </div>
                     <span tagx="article-title">PhyloPhlAn is a new method for improved phylogenetic and taxonomic placement of microbes</span>. 
                     <span tagx="source">Nat. Commun.</span>
                     <span tagx="volume">4</span>, 
                     <span tagx="fpage">2304</span> (
                     <span tagx="year">2013</span>).
                     <div tagxxx="pub-id">23942190</div>
                  </div>
               </li>
            </ul>
         </div>
      </div>
      <div tagxxx="floats-group">
         <div tagxxx="fig">
            <div tagxxx="label">Figure 1</div>
            <caption>
               <div tagxxx="title">Mosquito collection sites and distribution of 
                  <i>Wolbachia</i>-infected individuals.
               </div>
               <p>The maps describe the three villages (Soumousso, VK5 and VK7) where 
                  <i>An. gambiae</i> couples from different mating swarms (indicated by circles) and larvae from different
                  breeding sites (indicated by squares) were collected. Swarm sites and larval breeding
                  grounds are identified by numbers and letters, respectively (see 
                  <span ref-type="supplementary-material" rid="S1">Supplementary Table 1 and 2</span>). Sites where 
                  <i>Wolbachia</i>-positive mosquitoes were found are highlighted in red. Maps were adapted with permission
                  from originals courtesy of A. A. Millogo, IRSS/Centre Muraz, Bobo-Dioulasso, Burkina
                  Faso.
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <div tagxxx="label">Figure 2</div>
            <caption>
               <div tagxxx="title">Phylogenetic tree for 
                  <i>Wolbachia 16S</i> rRNA sequences.
               </div>
               <p>The tree was built using V4 
                  <i>16S</i> rRNA fragments from 
                  <i>Wolbachia</i> reference sequences available in public repositories (NCBI and SILVA) and from the
                  sequences obtained in this study. The sequences are clustered at 99% identity and
                  the cardinality of each OTU obtained is reported in a logarithmic scale as a bar chart
                  external to the tree (green for reference sequences and red for the new sequences).
                  Shadings highlight subtrees with high bootstrapping support (&gt;90%) and host organisms
                  are reported for those OTUs in which at least one-third of the sequences are consistently
                  associated to the same host.
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <div tagxxx="label">Figure 3</div>
            <caption>
               <div tagxxx="title">Whole-genome shotgun sequencing of a 
                  <i>Wolbachia</i>-positive sample identifies a new 
                  <i>Wolbachia</i> strain in 
                  <i>An. gambiae</i>.
               </div>
               <p>Ovaries from a 
                  <i>Wolbachia</i>-positive female (sample S1) were sequenced using WGS. (
                  <b>a</b>) Percentage (Perc.) identity of 395 short sequences uniquely attributable to 
                  <i>Wolbachia</i> from sample S1 versus eight sequenced strains. The 
                  <i>Wolbachia</i> reference strains and the supergroups are indicated with the corresponding percentage
                  of assigned sequences. (
                  <b>b</b>) Distribution of phylogenetic distances between different 
                  <i>Wolbachia</i> supergroups, from phylogenies reconstructed separately on each of the short sequences
                  universally conserved within 
                  <i>Wolbachia</i> genomes (light blue inset in 
                  <b>a</b>). (
                  <b>c</b>) Functional classification of 
                  <i>Wolbachia</i> loci identified by alignment to read sequences, based on NCBI annotated gene functions.
                  (
                  <b>d</b>) 
                  <i>Wolbachia</i> phylogeny, comprising the new 
                  <i>Wolbachia</i> strain 
                  <i>w</i>Anga isolated in 
                  <i>An. gambiae</i>, reconstructed from the concatenated sequences of 
                  <b>b</b>. These analyses show that 
                  <i>w</i>Anga is different from all other strains sequenced so far. The same analyses for another
                  PCR-positive 
                  <i>Wolbachia</i> sample are available in 
                  <span ref-type="supplementary-material" rid="S1">Supplementary Fig. 3</span>.
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <div tagxxx="label">Figure 4</div>
            <caption>
               <div tagxxx="title">Vertical transmission of 
                  <i>Wolbachia</i> from mother to offspring.
               </div>
               <p>A total of 14 
                  <i>Wolbachia</i>-positive blood-fed 
                  <i>An. gambiae</i> females collected from houses in VK5 were allowed to lay eggs individually. The progeny
                  of the five 
                  <i>Wolbachia</i>-positive females (W+) that laid eggs was screened for the presence of 
                  <i>Wolbachia</i> at the fourth larval developmental stage (L4). Red indicates 
                  <i>Wolbachia</i>-positive larvae and adults. The number of larvae screened is shown for each female
                  (
                  <i>n</i>).
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
      </div>
   </div>
</html>