<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <div xmlns="" tagxxx="article">
      <head>
         <meta content="Public Library of Science" name="citation_publisher"></meta>
         <meta content="10.1371/journal.pntd.0002349" name="citation_doi"></meta>
         <meta content="Multi-Gene Detection and Identification of Mosquito-Borne RNA Viruses Using an Oligonucleotide Microarray" name="citation_title"></meta>
         <title>Multi-Gene Detection and Identification of Mosquito-Borne RNA Viruses Using an Oligonucleotide
            Microarray
         </title>
         <div tagxxx="contrib-group">
            <meta content="Nathan D. Grubaugh" name="citation_author"></meta>
            <meta content="*" name="citation_author_institution_ref"></meta>
            <meta content="Scott S. McMenamy" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="Michael J. Turell" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="John S. Lee" name="citation_author"></meta>
            <meta content="¤b" name="citation_author_institution_ref"></meta>
         </div>
         <meta content="nathan.grubaugh@colostate.edu" name="email"></meta>
         <meta content="The authors have declared that no competing interests exist." name="conflict"></meta>
         <meta content="Conceived and designed the experiments: NDG MJT JSL. Performed the experiments: NDG SSM. Analyzed the data: NDG SSM JSL. Contributed reagents/materials/analysis tools: NDG SSM MJT. Wrote the paper: NDG SSM MJT JSL. Performed the field experiments: NDG SSM JSL." name="contribs"></meta>
         <meta content="2013-8-15" name="citation_pubdate"></meta>
         <meta content="Copyright 2013: " name="citation_copyright"></meta>
         <meta content="Licence This is an open-access article, free of all copyright, and may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose. The work is made available under the Creative Commons CC0 public domain dedication." name="citation_licence"></meta>
         <meta content="This research was funded by the Military Infectious Disease Research Program project # U0176_09_RD, U0321_12_RD (JSL), and supported in part by an appointment to the Postgraduate Research Participation Program at the U.S. Army Medical Research Institute of Infectious Disease administered by the Oak Ridge Institute for Science and Education through an interagency agreement between the U.S. Department of Energy and USAMRMC (NDG). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript." name="citation_funding"></meta>
         <meta content="" name="citation_pagecount"></meta>
         <meta content="" name="citation_data"></meta>
      </head>
      <div id="abstract" tag="abstract">
         <div id="">
            <h2>Background</h2>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Arthropod-borne viruses are important emerging pathogens world-wide. Viruses transmitted
               by mosquitoes, such as dengue, yellow fever, and Japanese encephalitis viruses, infect
               hundreds of millions of people and animals each year. Global surveillance of these
               viruses in mosquito vectors using molecular based assays is critical for prevention
               and control of the associated diseases. Here, we report an oligonucleotide DNA microarray
               design, termed ArboChip5.1, for multi-gene detection and identification of mosquito-borne
               RNA viruses from the genera 
               <i>Flavivirus</i> (family 
               <i>Flaviviridae</i>), 
               <i>Alphavirus</i> (
               <i>Togaviridae</i>), 
               <i>Orthobunyavirus</i> (
               <i>Bunyaviridae</i>), and 
               <i>Phlebovirus</i> (
               <i>Bunyaviridae</i>).
            </p>
         </div>
         <div id="">
            <h2>Methodology/Principal Findings</h2>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The assay utilizes targeted PCR amplification of three genes from each virus genus
               for electrochemical detection on a portable, field-tested microarray platform. Fifty-two
               viruses propagated in cell-culture were used to evaluate the specificity of the PCR
               primer sets and the ArboChip5.1 microarray capture probes. The microarray detected
               all of the tested viruses and differentiated between many closely related viruses
               such as members of the dengue, Japanese encephalitis, and Semliki Forest virus clades.
               Laboratory infected mosquitoes were used to simulate field samples and to determine
               the limits of detection. Additionally, we identified dengue virus type 3, Japanese
               encephalitis virus, Tembusu virus, 
               <i>Culex</i> flavivirus, and a Quang Binh-like virus from mosquitoes collected in Thailand in
               2011 and 2012.
            </p>
         </div>
         <div id="">
            <h2>Conclusions/Significance</h2>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We demonstrated that the described assay can be utilized in a comprehensive field
               surveillance program by the broad-range amplification and specific identification
               of arboviruses from infected mosquitoes. Furthermore, the microarray platform can
               be deployed in the field and viral RNA extraction to data analysis can occur in as
               little as 12 h. The information derived from the ArboChip5.1 microarray can help to
               establish public health priorities, detect disease outbreaks, and evaluate control
               programs.
            </p>
         </div>
      </div>
      <div id="abstract" tag="abstract">
         <div tagxxx="title">Author Summary</div>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Approximately half of the world's population is at risk of viral, mosquito-borne illness
            such as dengue, yellow fever, Japanese encephalitis, Rift Valley fever, and chikungunya.
            In the past, these viruses have been regarded as pathogens of the tropics; however,
            they are emerging as global causes of illness. Very few effective drugs and vaccines
            have been developed for mosquito-borne viral infections and even less are available
            to people in resource-limited countries. An important aspect of disease prevention
            is mosquito surveillance to determine geographical range and seasonal prevalence of
            the associated viruses. However, there are hundreds of viruses transmitted by mosquitoes
            that are pathogenic to humans and animals. Using a portable microarray, we developed
            an assay with the ability to detect most of the known medically important viruses
            transmitted by mosquitoes. This assay was designed for use in conjunction with broad-range
            screening tools as a cost effective, rapid method to determine the identity of viruses
            from infected mosquitoes. To our knowledge, this is the most comprehensive assay to
            date for field surveillance of mosquito-borne viruses.
         </p>
      </div>
      <div id="s1">
         <h2>Introduction</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Arthropod-borne viruses (arboviruses) are important human and veterinary pathogens
            that are biologically transmitted to vertebrates by hematophagous (blood feeding)
            arthropod vectors, such as female mosquitoes. The diverse group of mosquito-borne
            RNA viruses primarily includes flaviviruses (
            <i>Flaviviridae</i>: 
            <i>Flavivirus</i>), alphaviruses (
            <i>Togaviridae</i>: 
            <i>Alphavirus</i>), orthobunyaviruses (
            <i>Bunyaviridae</i>: 
            <i>Orthobunyavirus</i>), and phleboviruses (
            <i>Bunyaviridae</i>: 
            <i>Phlebovirus</i>) 
            <span ref-type="bibr" rid="pntd.0002349-Weaver1">[1]</span>. Flaviviruses contain a positive-sense, single-stranded RNA genome of approximately
            10.9 kilo bases (kb) in length with a single open reading frame encoding three structural
            and seven nonstructural proteins 
            <span ref-type="bibr" rid="pntd.0002349-Lindenbach1">[2]</span>. Mosquito-borne flaviviruses are phylogenetically divided into two groups: those
            transmitted by 
            <i>Aedes</i> species mosquitoes, such as dengue virus (DENV) and yellow fever virus (YFV), and
            those transmitted by 
            <i>Culex</i> species mosquitoes, such as Japanese encephalitis virus (JEV) and West Nile virus
            (WNV). Additionally, flaviviruses continue to be isolated from mosquitoes without
            a known vertebrate host, termed arthropod-specific viruses 
            <span ref-type="bibr" rid="pntd.0002349-Stollar1">[3]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Crabtree1">[4]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Sang1">[5]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Cook1">[6]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Hoshino1">[7]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Crabtree2">[8]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Bolling1">[9]</span>. Alphavirus genomes are also positive-sense, single-stranded RNA molecules, approximately
            11.7 kb in length that translate into four nonstructural and three structural proteins
            
            <span ref-type="bibr" rid="pntd.0002349-Strauss1">[10]</span>. They are classified in two geographically isolated groups, Old World alphaviruses,
            such as chikungunya virus (CHIKV) that is found in parts of Africa, Asia, and Europe,
            and New World alphaviruses, such as eastern equine encephalitis and Venezuelan equine
            encephalitis (VEEV) viruses that circulate in the Americas. Orthobunyaviruses and
            phleboviruses have tripartite, single-stranded, negative-sense RNA genomes. The RNA
            segments, designated by their distinct sizes, small (S), medium (M), and large (L),
            encode for the nonstructural proteins, structural proteins, and RNA-dependent RNA
            polymerase, respectively 
            <span ref-type="bibr" rid="pntd.0002349-Elliott1">[11]</span>. Orthobunyaviruses are primarily mosquito-borne and are distributed globally. The
            majority of phleboviruses use phlebotomine sand flies as their primary vectors, but
            many can also be transmitted by mosquitoes. One notable example is Rift Valley fever
            virus (RVFV), which is transmitted by a number of different mosquito species in nature
            
            <span ref-type="bibr" rid="pntd.0002349-Meegan1">[12]</span>.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Arboviruses represent nearly 30% of all emerging infectious disease in the last 50
            years 
            <span ref-type="bibr" rid="pntd.0002349-Jones1">[13]</span>. Emergence and re-emergence of arboviral pathogens can be attributed to many factors,
            such as globalization, altering weather patterns, increased production of livestock,
            and tropical urbanization 
            <span ref-type="bibr" rid="pntd.0002349-Weaver1">[1]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Hollidge1">[14]</span>. The majority of emerging arboviruses are known pathogens with high epidemic and
            epizootic potentials when introduced into new populations, as evident by several examples:
            RVFV in Africa 
            <span ref-type="bibr" rid="pntd.0002349-Meegan2">[15]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Nguku1">[16]</span>, WNV in North America 
            <span ref-type="bibr" rid="pntd.0002349-Briese1">[17]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Brinton1">[18]</span>, and CHIKV in areas near the Indian Ocean 
            <span ref-type="bibr" rid="pntd.0002349-KariukiNjenga1">[19]</span>.With limited antiviral drugs and vaccines available, global surveillance of newly
            emerging and re-emerging arboviruses is critical for early detection and prevention
            of arboviral diseases.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">An active surveillance program should include the monitoring of levels of virus activity
            in vector populations and in vertebrate hosts 
            <span ref-type="bibr" rid="pntd.0002349-CDC1">[20]</span>. This can be challenging for a comprehensive surveillance program because mosquito-borne
            viruses are taxonomically diverse and expansive. Traditional polymerase chain reaction
            (PCR) based assays can be too limited in scope to detect unexpected circulating viruses.
            Microarrays, on the other hand, can detect hundreds or thousands of viral agents using
            oligonucleotide DNA probes 
            <span ref-type="bibr" rid="pntd.0002349-Wang1">[21]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Wang2">[22]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Palacios1">[23]</span>. Microarray assays have been developed to detect a wide range of viruses, including
            some mosquito-borne viruses 
            <span ref-type="bibr" rid="pntd.0002349-Wang1">[21]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Wang2">[22]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Palacios1">[23]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Korimbocus1">[24]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Nordstrom1">[25]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Chou1">[26]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Putonti1">[27]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-XiaoPing1">[28]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Berthet1">[29]</span>. These assays have been shown to be valuable tools for the detection of viral RNA
            in clinical samples. However, none of the assays have been designed or analyzed for
            use with infected mosquitoes. Most of the assays included an unbiased, random amplification
            method for microarray detection. We have previously reported that such techniques
            are efficient for the amplification viral nucleic acids isolated from cultured cells,
            but not from a complex matrix of RNA from mosquito homogenates 
            <span ref-type="bibr" rid="pntd.0002349-Grubaugh1">[30]</span>. Additionally, most of the microarray designs are not wide-ranging and only cover
            a few of the most concerning viral pathogens. Lastly, none have been evaluated for
            field-use.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In this report, we describe an oligonucleotide DNA microarray, the ArboChip5.1, which
            targets multiple genes from 144 mosquito-borne RNA viruses from the genera 
            <i>Flavivirus</i>, 
            <i>Alphavirus</i>, 
            <i>Orthobunyavirus</i>, and 
            <i>Phlebovirus</i>. For each genus, at least three sets of consensus gene-specific primers (GSPs) were
            designed for efficient PCR amplification of the viral nucleic acids isolated from
            infected mosquitoes and virus-specific microarray capture probes were designed to
            differentiate between the PCR amplicons. We previously demonstrated that a portable
            microarray platform, the ElectraSense 4×2K (CustomArray, Inc., Bothell, WA), is practical
            for field diagnostics 
            <span ref-type="bibr" rid="pntd.0002349-Grubaugh1">[30]</span>. The platform included a small and rugged microarray reader that was able to analyze
            four samples simultaneously against 2,240 oligonucleotide DNA probes using electrochemical
            detection (ECD) 
            <span ref-type="bibr" rid="pntd.0002349-Roth1">[31]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Ghindilis1">[32]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Lodes1">[33]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Lodes2">[34]</span>. The assay, as described here, was able to detect and identify RNA belonging to several
            arboviruses of medical and veterinary importance, including DENV type 3 (DENV-3),
            JEV, Tembusu virus (TMUV), 
            <i>Culex</i> flavivirus (CxFV), and a Quang Binh-like virus from field-collected mosquitoes from
            Thailand during 2011 and 2012.
         </p>
      </div>
      <div id="s2">
         <h2>Materials and Methods</h2>
         <div id="s2a">
            <h3>Ethics statement</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Entomological collections from private land and residences were conducted with the
               owners/residents permission.
            </p>
         </div>
         <div id="s2b">
            <h3>Assay overview and method substitutions</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The assay workflow is displayed in 
               <span ref-type="fig" rid="pntd-0002349-g001">Figure 1</span>. Most of the assays were completed in 12 h or less. The methods for mosquito collection,
               RNA extraction, and virus screening can be substituted with a group's own methods.
               Virus screening methods should typically include PCR assays for genus-level detection
               using consensus primers. Complementary DNA (cDNA) from mosquito pools that tested
               positive using the screening assays should continue with the assay procedures, starting
               at the “asymmetric PCR amplification with GSPs and biotinylation” step.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pntd.0002349.g001</div>
               <div tagxxx="label">Figure 1</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Microarray assay workflow.</div>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
         </div>
         <div id="s2c">
            <h3>Gene-specific primer design</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Consensus GSPs for PCR amplification of microarray targets can be found in 
               <span ref-type="table" rid="pntd-0002349-t001">Table 1</span>. The GSPs were modified from published assays 
               <span ref-type="bibr" rid="pntd.0002349-MaherSturgess1">[35]</span>, 
               <span ref-type="bibr" rid="pntd.0002349-Eshoo1">[36]</span> or designed by aligning all microarray target sequences, according to virus genus,
               using the Megalign/Clustal W software (DNASTAR, Inc., Madison, WI). The GSPs were
               selected from conserved regions meeting the following criteria: 17–32 base pairs (bp)
               in length, melting temperatures between 50 and 65°C, and guanine-cytosine content
               (GC-content) between 45 and 65%.
            </p>
            <div tagxxx="table-wrap">
               <div tagxxx="object-id">10.1371/journal.pntd.0002349.t001</div>
               <div tagxxx="label">Table 1</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Gene-specific primers (GSPs) for PCR amplification of microarray targets.</div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">hsidesgroups
                     <colgroup>1
                        <col>left1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                     </colgroup>
                     <thead>
                        <tr>
                           <td>left11Virus genus</td>
                           <td>left11Gene/segment</td>
                           <td>left11Primer name</td>
                           <td>left11GSP sequence (5′ to 3′)</td>
                           <td>left11Product (bp)</td>
                           <td>left11GenBank ID</td>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>left11
                              <i>Flavivirus</i>
                           </td>
                           <td>left11E</td>
                           <td>left11F1269-F</td>
                           <td>left11
                              <div tagxxx="named-content">GAGGCTGGGGAAATGGCTG</div>
                           </td>
                           <td>left11969</td>
                           <td>left11NC_002031</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11</td>
                           <td>left11F2225-R</td>
                           <td>left11
                              <div tagxxx="named-content">CCTCCAACTGATCCAAAGTCCCA</div>
                           </td>
                           <td>left11</td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11NS3</td>
                           <td>left11F5015-F</td>
                           <td>left11
                              <div tagxxx="named-content">GTGGTTGGNCTGTATGGNAA</div>
                           </td>
                           <td>left11812</td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11</td>
                           <td>left11F5807-R</td>
                           <td>left11
                              <div tagxxx="named-content">CCCATTTCTGAGATGTCAGT</div>
                           </td>
                           <td>left11</td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11NS5</td>
                           <td>left11F8276d-F
                              <span ref-type="table-fn" rid="nt104">c</span>
                           </td>
                           <td>left11
                              <div tagxxx="named-content">AAYTCNACNCANGARATGTAY</div>
                           </td>
                           <td>left11804</td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11</td>
                           <td>left11F9063d-R
                              <span ref-type="table-fn" rid="nt104">c</span>
                           </td>
                           <td>left11
                              <div tagxxx="named-content">CCNARCCACATRWACCA</div>
                           </td>
                           <td>left11</td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11
                              <i>Alphavirus</i>
                           </td>
                           <td>left11nsP1</td>
                           <td>left11A183d-F
                              <span ref-type="table-fn" rid="nt105">d</span>
                           </td>
                           <td>left11
                              <div tagxxx="named-content">TCCATGCHMAYGCBAGRGCDTTYTCGCA</div>
                           </td>
                           <td>left11937</td>
                           <td>left11NC_004162</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11</td>
                           <td>left11A1095d-R</td>
                           <td>left11
                              <div tagxxx="named-content">GTNGCHADDATNCCNGTCATYTGRT</div>
                           </td>
                           <td>left11</td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11nsP4</td>
                           <td>left11A6493d-F</td>
                           <td>left11
                              <div tagxxx="named-content">CCRYTDCANSAVRTACCNATGGA</div>
                           </td>
                           <td>left111009</td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11</td>
                           <td>left11A7482d-R</td>
                           <td>left11
                              <div tagxxx="named-content">CHATYYAGGWYMRCCGTASA</div>
                           </td>
                           <td>left11</td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11E1</td>
                           <td>left11A10240d-F</td>
                           <td>left11
                              <div tagxxx="named-content">GGBGTNTACCCNTTYATGTGGGG</div>
                           </td>
                           <td>left11914</td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11</td>
                           <td>left11A11135d-R</td>
                           <td>left11
                              <div tagxxx="named-content">ATGTGGTCYKYHGGDGGNT</div>
                           </td>
                           <td>left11</td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11
                              <i>Phlebovirus</i>
                           </td>
                           <td>left11S</td>
                           <td>left11PR-S1192d-F</td>
                           <td>left11
                              <div tagxxx="named-content">TCAATRAKRCCAGCAAAGCTRGGATGCATC</div>
                           </td>
                           <td>left11391</td>
                           <td>left11NC_002045</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11</td>
                           <td>left11PR-S1558d-R</td>
                           <td>left11
                              <div tagxxx="named-content">GGGTSMAWGAGTTTGCTTAYCAAGG</div>
                           </td>
                           <td>left11</td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11M</td>
                           <td>left11P-M2292d-F</td>
                           <td>left11
                              <div tagxxx="named-content">TGYAGRGARGGNMMKAGYTTYTGGAC</div>
                           </td>
                           <td>left11689</td>
                           <td>left11NC_002044</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11</td>
                           <td>left11P-M2955d-R</td>
                           <td>left11
                              <div tagxxx="named-content">CCATTCCTVRCYTGWGGYARAGANCC</div>
                           </td>
                           <td>left11</td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11L</td>
                           <td>left11P-L1992d-F</td>
                           <td>left11
                              <div tagxxx="named-content">GGARGAYAARGCWRYMACWGAAG</div>
                           </td>
                           <td>left111068</td>
                           <td>left11NC_002043</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11</td>
                           <td>left11P-L3039d-R</td>
                           <td>left11
                              <div tagxxx="named-content">GGAWGDGTGAAYTCRCAYARC</div>
                           </td>
                           <td>left11</td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11
                              <i>Orthobunyavirus</i>
                           </td>
                           <td>left11S BUNV
                              <span ref-type="table-fn" rid="nt102">a</span>
                           </td>
                           <td>left11OB-S121-F</td>
                           <td>left11
                              <div tagxxx="named-content">CACCAGCAGTACTTTTGACCCAGA</div>
                           </td>
                           <td>left11649</td>
                           <td>left11NC_001927</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11</td>
                           <td>left11OB-S746d-R</td>
                           <td>left11
                              <div tagxxx="named-content">TTTAGCHARGAABTCYCTRGCWGC</div>
                           </td>
                           <td>left11</td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11S CEV
                              <span ref-type="table-fn" rid="nt103">b</span>
                           </td>
                           <td>left11OC-S127d-F</td>
                           <td>left11
                              <div tagxxx="named-content">TTTRAYCCNGAKGCAGGGTWTGTGG</div>
                           </td>
                           <td>left11484</td>
                           <td>left11U12797</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11</td>
                           <td>left11OC-S594d-R</td>
                           <td>left11
                              <div tagxxx="named-content">GCYTTCTTCAGGWACTKDGSRTCCATC</div>
                           </td>
                           <td>left11</td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11M</td>
                           <td>left11O-M3266d-F</td>
                           <td>left11
                              <div tagxxx="named-content">GGNTGTGTWTWTGGNTCWTGYCARGA</div>
                           </td>
                           <td>left11906</td>
                           <td>left11NC_001926</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11</td>
                           <td>left11O-M4147d-R</td>
                           <td>left11
                              <div tagxxx="named-content">CCTTCATCYYKNACYCTRCAYADCC</div>
                           </td>
                           <td>left11</td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11L</td>
                           <td>left11O-L3156-F</td>
                           <td>left11
                              <div tagxxx="named-content">GCAGATATGTCAAAATGGAGTGCTC</div>
                           </td>
                           <td>left111088</td>
                           <td>left11NC_001925</td>
                        </tr>
                        <tr>
                           <td>left11</td>
                           <td>left11</td>
                           <td>left11O-L4223-R</td>
                           <td>left11
                              <div tagxxx="named-content">CTCATGTCACTTGTTTCACCC</div>
                           </td>
                           <td>left11</td>
                           <td>left11</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <div tagxxx="table-wrap-foot">
                  <div tagxxx="fn">
                     <div tagxxx="label"></div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The primer start position, based on the GenBank source, is listed in the primer name.
                        The “d” in the primer name designates that degenerate nucleotides were included. “F”
                        and “R” at the end of the primer name designates forward and reverse, respectively.
                        BUNV, Bunyamwera virus; CEV, California encephalitis virus.
                     </p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">a</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">S segment primers for BUNV and Wyeomyia clade targets.</p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">b</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">S segment primers for CEV, Bwamba, and Simbu clade targets.</p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">c</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Previously published as Flav100F and Flav200R 
                        <span ref-type="bibr" rid="pntd.0002349-MaherSturgess1">[35]</span>.
                     </p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">d</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Modified version of VIR966-F 
                        <span ref-type="bibr" rid="pntd.0002349-Eshoo1">[36]</span>.
                     </p>
                  </div>
               </div>
            </div>
         </div>
         <div id="s2d">
            <h3>Microarray capture probe design</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Probes were designed to target unique viral sequences, 30 to 45 nucleotides in length,
               between the GSPs using methods previously described 
               <span ref-type="bibr" rid="pntd.0002349-Grubaugh1">[30]</span>. A total of 2,097 oligonucleotide probes were selected for inclusion on the ArboChip5.1.
               The probe set included: 802 targeting flaviviruses, 307 targeting alphaviruses, 572
               targeting orthobunyaviruses, and 381 targeting phleboviruses. Positive hybridization
               (n = 10) and negative background control (n = 25) probes were added to the design
               as previously reported 
               <span ref-type="bibr" rid="pntd.0002349-Wojciechowski1">[37]</span>. Multiple copies of the control probes were added to fill the 2,240 probe sites present
               on the sectored microarray chip. 
               <span ref-type="supplementary-material" rid="pntd.0002349.s003">Table S1</span> displays all the targeted viruses and the number of probes for each virus selected
               for inclusion on the ArboChip5.1. The complete list of capture probe sequences can
               be found in 
               <span ref-type="supplementary-material" rid="pntd.0002349.s004">Table S2</span>. All of the probes included on the ArboChip5.1 design were evaluated with earlier
               design versions and found to not hybridize with unintended targets, i.e., cDNA from
               uninfected cultured cells, uninfected mosquitoes, and viruses from different genera
               (unpublished data). Due to a lack of sequence information at the time of probe design,
               probes for all three gene targets were not included for some viruses. Probes targeting
               viruses not tested in this publication should be considered investigational until
               evaluated. The oligonucleotide probes were synthesized directly on the ElectraSense
               4×2K sectored microarray by CustomArray, Inc.
            </p>
         </div>
         <div id="s2e">
            <h3>Microarray probe grouping</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The virus-targeted capture probes were sorted into 12 groups, three groups for each
               genus based on the gene target. Each group was further sorted into subgroups based
               on the virus clade and probe specificity. For example, a probe specific for the DENV-3
               NS5 gene would be in the 
               <i>Flavivirus</i> NS5 group and the DENV clade, DENV-3 subgroup (DENV_DENV3). Probes that hybridized
               to multiple, but related viruses were classified as non-virus-specific and sorted
               into genus (i.e. flavivirus_generic) or clade-specific (i.e. DENV_clade) subgroups.
               Some very closely related viruses, such as O'nyong-nyong virus (ONNV) and Igbo Ora
               virus (IOV) of the Semliki Forest virus (SFV) clade, could not be differentiated.
               Probes for virus targets that could not be differentiated were sorted into subgroups
               containing virus complexes (i.e. SFV_ONNV/IOV).
            </p>
         </div>
         <div id="s2f">
            <h3>Laboratory viruses and mosquito infections</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The laboratory viruses used in this study are listed in 
               <span ref-type="table" rid="pntd-0002349-t002">Tables 2</span>–
               <span ref-type="table" rid="pntd-0002349-t005">5</span>. Viruses were propagated primarily in Vero (African green monkey kidney) or C6/36
               (
               <i>Aedes albopictus</i>) cell cultures. Mosquitoes were inoculated intrathoracically (0.3 µL/mosquito) with
               selected viruses at approximately 10
               <sup>5</sup> plaque forming units (PFU)/mL 
               <span ref-type="bibr" rid="pntd.0002349-Rosen1">[38]</span>. Inoculated mosquitoes were held for at least 7 days at 26°C to allow for virus replication.
               Virus-inoculated mosquitoes were triturated individually in diluent (Eagle's minimal
               essential medium containing 10% heat-inactivated fetal bovine serum, 0.075% NaHCO
               <div tagxxx="sub">3</div>, and 100 units of penicillin and 100 µg of streptomycin per mL). All virus preparations
               were tested in duplicate using the microarray assay.
            </p>
            <div tagxxx="table-wrap">
               <div tagxxx="object-id">10.1371/journal.pntd.0002349.t002</div>
               <div tagxxx="label">Table 2</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Microarray reproducibility.</div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">hsidesgroups
                     <colgroup>1
                        <col>left1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                     </colgroup>
                     <thead>
                        <tr>
                           <td>left11Virus</td>
                           <td>left11Run 1</td>
                           <td>left11Run 2</td>
                           <td>left11Run 3</td>
                           <td>left11Run 4</td>
                           <td>left11Mean</td>
                           <td>left11SD</td>
                           <td>left1195% CI</td>
                           <td>left11CV</td>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>left11West Nile L1</td>
                           <td>left1114.29</td>
                           <td>left1114.95</td>
                           <td>left1114.91</td>
                           <td>left1114.89</td>
                           <td>left1114.76</td>
                           <td>left110.27</td>
                           <td>left1114.32, 15.19</td>
                           <td>left111.8%</td>
                        </tr>
                        <tr>
                           <td>left11Chikungunya</td>
                           <td>left1116.59</td>
                           <td>left1119.84</td>
                           <td>left1121.11</td>
                           <td>left1123.23</td>
                           <td>left1120.19</td>
                           <td>left112.41</td>
                           <td>left1116.36, 24.02</td>
                           <td>left1111.9%</td>
                        </tr>
                        <tr>
                           <td>left11Rift Valley fever</td>
                           <td>left1126.78</td>
                           <td>left1125.53</td>
                           <td>left1129.96</td>
                           <td>left1129.54</td>
                           <td>left1127.95</td>
                           <td>left111.85</td>
                           <td>left1125.0, 30.9</td>
                           <td>left116.6%</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <div tagxxx="table-wrap-foot">
                  <div tagxxx="fn">
                     <div tagxxx="label"></div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Microarray analysis using cell-cultured derived viruses were performed in four replicates.
                        The z-scores of the virus-specific probes were averaged and listed for each run. The
                        mean and standard deviation (SD) was calculated to determine the 95% confidence interval
                        (CI) and coefficient of variance (CV) for each virus.
                     </p>
                  </div>
               </div>
            </div>
            <div tagxxx="table-wrap">
               <div tagxxx="object-id">10.1371/journal.pntd.0002349.t003</div>
               <div tagxxx="label">Table 3</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">PCR amplification and microarray detection of flaviviruses.</div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">hsidesgroups
                     <colgroup>1
                        <col>left1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                     </colgroup>
                     <thead>
                        <tr>
                           <td>left11</td>
                           <td>left11</td>
                           <td>3left1PCR amplification/microarray detection
                              <span ref-type="table-fn" rid="nt108">a</span>
                           </td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11Virus</td>
                           <td>left11Strain</td>
                           <td>left11E</td>
                           <td>left11NS3</td>
                           <td>left11NS5</td>
                           <td>left11Mosquito species
                              <span ref-type="table-fn" rid="nt109">b</span>
                           </td>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>left11Bussuquara</td>
                           <td>left11RV270</td>
                           <td>left11POS
                              <span ref-type="table-fn" rid="nt111">d</span>/yes
                           </td>
                           <td>left11POS
                              <span ref-type="table-fn" rid="nt110">c</span>/yes
                           </td>
                           <td>left11POS
                              <span ref-type="table-fn" rid="nt110">c</span>/yes
                           </td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Chaoyang</td>
                           <td>left11ROK144</td>
                           <td>left11POS/yes
                              <span ref-type="table-fn" rid="nt111">d</span>
                              <span ref-type="table-fn" rid="nt112">e</span>
                           </td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Dengue type 1</td>
                           <td>left11HAW</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11
                              <i>Ae. aegypti</i>
                           </td>
                        </tr>
                        <tr>
                           <td>left11Dengue type 2</td>
                           <td>left11S16803</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11
                              <i>Ae. aegypti</i>
                           </td>
                        </tr>
                        <tr>
                           <td>left11Dengue type 3</td>
                           <td>left11Thai 1987</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11
                              <i>Ae. aegypti</i>
                           </td>
                        </tr>
                        <tr>
                           <td>left11Dengue type 4</td>
                           <td>left11CAR 341750</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11
                              <i>Ae. albopictus</i>
                           </td>
                        </tr>
                        <tr>
                           <td>left11JE</td>
                           <td>left11Th9-0175</td>
                           <td>left11POS/no</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11
                              <i>Cx. pipiens</i>
                           </td>
                        </tr>
                        <tr>
                           <td>left11Kunjin</td>
                           <td>left11R4336a</td>
                           <td>left11NEG</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11MVE</td>
                           <td>left11RV241</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Quang Binh</td>
                           <td>left11Th5-0215</td>
                           <td>left11POS/no</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Rocio</td>
                           <td>left11SP H34675</td>
                           <td>left11NEG</td>
                           <td>left11POS
                              <span ref-type="table-fn" rid="nt110">c</span>/yes
                           </td>
                           <td>left11POS
                              <span ref-type="table-fn" rid="nt110">c</span>/yes
                           </td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11SLE</td>
                           <td>left11Ft. Washington</td>
                           <td>left11POS
                              <span ref-type="table-fn" rid="nt110">c</span>/yes
                           </td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Tembusu</td>
                           <td>left11Th3-0385</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11
                              <i>Cx. tarsalis</i>
                           </td>
                        </tr>
                        <tr>
                           <td>left11Tembusu</td>
                           <td>left11Th6-0381</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11West Nile L1</td>
                           <td>left11EG101</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11West Nile L1</td>
                           <td>left11NY397-99</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11
                              <i>Cx. pipiens</i>
                           </td>
                        </tr>
                        <tr>
                           <td>left11West Nile L2</td>
                           <td>left11KLF 76</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11West Nile L2</td>
                           <td>left11KLF 146</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Yellow fever</td>
                           <td>left1117D</td>
                           <td>left11POS
                              <span ref-type="table-fn" rid="nt110">c</span>/yes
                           </td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11
                              <i>Ae. aegypti</i>
                           </td>
                        </tr>
                        <tr>
                           <td>left11Zika</td>
                           <td>left1130306</td>
                           <td>left11NEG</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <div tagxxx="table-wrap-foot">
                  <div tagxxx="fn">
                     <div tagxxx="label"></div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Viral RNA derived from cell culture or infected mosquitoes were PCR amplified using
                        GSPs. Amplicons were analyzed using the ArboChip5.1 microarray. JE, Japanese encephalitis;
                        MVE, Murray Valley encephalitis; SLE, St. Louis encephalitis; L1, lineage 1; L2, lineage
                        2; POS, positive PCR amplification; NEG, negative PCR amplification; yes, detected
                        by microarray; no, not detected by microarray; n.t., not tested.
                     </p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">a</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Viruses propagated in cell culture and identified by microarray to species unless
                        otherwise noted.
                     </p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">b</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Virus-infected mosquito used for microarray evaluations, microarray detected at least
                        one target for each virus.
                     </p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">c</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">PCR amplification produced a weak visible band.</p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">d</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">ArboChip5.1 does not include probes specific to the target.</p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">e</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Microarray detection with genus-level probes only.</p>
                  </div>
               </div>
            </div>
            <div tagxxx="table-wrap">
               <div tagxxx="object-id">10.1371/journal.pntd.0002349.t004</div>
               <div tagxxx="label">Table 4</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">PCR amplification and microarray detection of alphaviruses.</div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">hsidesgroups
                     <colgroup>1
                        <col>left1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                     </colgroup>
                     <thead>
                        <tr>
                           <td>left11</td>
                           <td>left11</td>
                           <td>3left1PCR amplification/microarray detection
                              <span ref-type="table-fn" rid="nt114">a</span>
                           </td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11Virus</td>
                           <td>left11Strain</td>
                           <td>left11nsP1</td>
                           <td>left11nsP4</td>
                           <td>left11E1</td>
                           <td>left11Mosquito species
                              <span ref-type="table-fn" rid="nt115">b</span>
                           </td>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>left11Aura</td>
                           <td>left11RIID 1990</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Babanki</td>
                           <td>left11Ken07-46A-49</td>
                           <td>left11POS/yes
                              <span ref-type="table-fn" rid="nt119">f</span>
                           </td>
                           <td>left11POS/yes
                              <span ref-type="table-fn" rid="nt119">f</span>
                           </td>
                           <td>left11POS/yes
                              <span ref-type="table-fn" rid="nt119">f</span>
                           </td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Chikungunya</td>
                           <td>left11INDO23574</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11
                              <i>Ae. taeniorhynchus</i>
                           </td>
                        </tr>
                        <tr>
                           <td>left11Getah</td>
                           <td>left11ROK-2.0017</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS
                              <span ref-type="table-fn" rid="nt116">c</span>/yes
                           </td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Mayaro</td>
                           <td>left11TR467</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Ndumu</td>
                           <td>left11Ken07-332</td>
                           <td>left11POS/yes
                              <span ref-type="table-fn" rid="nt117">d</span>
                              <span ref-type="table-fn" rid="nt118">e</span>
                           </td>
                           <td>left11POS/yes</td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt117">d</span>
                           </td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Ockelbo</td>
                           <td>left11ISL-44</td>
                           <td>left11POS/yes
                              <span ref-type="table-fn" rid="nt119">f</span>
                           </td>
                           <td>left11POS/yes
                              <span ref-type="table-fn" rid="nt119">f</span>
                           </td>
                           <td>left11POS/yes
                              <span ref-type="table-fn" rid="nt119">f</span>
                           </td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11O'nyong nyong</td>
                           <td>left11Gulu</td>
                           <td>left11POS/yes
                              <span ref-type="table-fn" rid="nt120">g</span>
                           </td>
                           <td>left11POS/yes
                              <span ref-type="table-fn" rid="nt120">g</span>
                           </td>
                           <td>left11POS/yes
                              <span ref-type="table-fn" rid="nt120">g</span>
                           </td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Ross River</td>
                           <td>left11T-49</td>
                           <td>left11POS
                              <span ref-type="table-fn" rid="nt116">c</span>/yes
                           </td>
                           <td>left11POS
                              <span ref-type="table-fn" rid="nt116">c</span>/yes
                           </td>
                           <td>left11NEG</td>
                           <td>left11
                              <i>Ae. albopictus</i>
                           </td>
                        </tr>
                        <tr>
                           <td>left11Semliki Fortest</td>
                           <td>left11Ken07-586</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Sindbis</td>
                           <td>left11Ken07-611</td>
                           <td>left11POS/yes
                              <span ref-type="table-fn" rid="nt119">f</span>
                           </td>
                           <td>left11POS/yes
                              <span ref-type="table-fn" rid="nt119">f</span>
                           </td>
                           <td>left11POS/yes
                              <span ref-type="table-fn" rid="nt119">f</span>
                           </td>
                           <td>left11
                              <i>Ae. aegypti</i>
                           </td>
                        </tr>
                        <tr>
                           <td>left11Una</td>
                           <td>left11PE-1.0800</td>
                           <td>left11POS/no
                              <span ref-type="table-fn" rid="nt117">d</span>
                           </td>
                           <td>left11POS/yes</td>
                           <td>left11POS/no
                              <span ref-type="table-fn" rid="nt117">d</span>
                           </td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11VEE</td>
                           <td>left11TC83</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11WEE</td>
                           <td>left11McMillan</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/no</td>
                           <td>left11
                              <i>Ae. albopictus</i>
                           </td>
                        </tr>
                     </div>
                  </table>
               </div>
               <div tagxxx="table-wrap-foot">
                  <div tagxxx="fn">
                     <div tagxxx="label"></div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Viral RNA derived from cell culture or infected mosquitoes were PCR amplified using
                        GSPs. Amplicons were analyzed using the ArboChip5.1 microarray. VEE, Venezuelan equine
                        encephalitis; WEE, western equine encephalomyelitis; POS, positive PCR amplification;
                        NEG, negative PCR amplification; yes, detected by microarray; no, not detected by
                        microarray; n.t., not tested.
                     </p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">a</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Viruses propagated in cell culture and identified by microarray to species unless
                        otherwise noted.
                     </p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">b</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Virus-infected mosquito used for microarray evaluations, microarray detected at least
                        one target for each virus.
                     </p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">c</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">PCR amplification produced a weak visible band.</p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">d</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">ArboChip5.1 does not include probes specific to the target.</p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">e</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Microarray detection with genus-level probes only.</p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">f</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Target was detected but could not be differentiated between BBKV, OCKV, and SINV.</p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">g</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Target was detected but could not be differentiated between ONNV and IOV.</p>
                  </div>
               </div>
            </div>
            <div tagxxx="table-wrap">
               <div tagxxx="object-id">10.1371/journal.pntd.0002349.t005</div>
               <div tagxxx="label">Table 5</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">PCR amplification and microarray detection of phleboviruses.</div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">hsidesgroups
                     <colgroup>1
                        <col>left1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                     </colgroup>
                     <thead>
                        <tr>
                           <td>left11</td>
                           <td>left11</td>
                           <td>3left1PCR amplification/microarray detection
                              <span ref-type="table-fn" rid="nt122">a</span>
                           </td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11Virus</td>
                           <td>left11Strain</td>
                           <td>left11S</td>
                           <td>left11M</td>
                           <td>left11L</td>
                           <td>left11Mosquito species
                              <span ref-type="table-fn" rid="nt123">b</span>
                           </td>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>left11Rift Valley fever</td>
                           <td>left11ZH501</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POSe/yes</td>
                           <td>left11
                              <i>Cx. pipiens</i>
                           </td>
                        </tr>
                        <tr>
                           <td>left11Rift Valley fever</td>
                           <td>left11ZH548</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POSe/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Sandfly fever Naples</td>
                           <td>left1185-055</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/no
                              <span ref-type="table-fn" rid="nt124">c</span>
                           </td>
                           <td>left11POS/yes
                              <span ref-type="table-fn" rid="nt124">c</span>
                              <span ref-type="table-fn" rid="nt125">d</span>
                           </td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Toscana</td>
                           <td>left11ISS PHI3</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Candiru</td>
                           <td>left11BeH2251</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Chagres
                              <span ref-type="table-fn" rid="nt126">e</span>
                           </td>
                           <td>left11LW10</td>
                           <td>left11POS/no
                              <span ref-type="table-fn" rid="nt124">c</span>
                           </td>
                           <td>left11POS/no
                              <span ref-type="table-fn" rid="nt124">c</span>
                           </td>
                           <td>left11POS/yes
                              <span ref-type="table-fn" rid="nt124">c</span>
                              <span ref-type="table-fn" rid="nt125">d</span>
                           </td>
                           <td>left11n.t.</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <div tagxxx="table-wrap-foot">
                  <div tagxxx="fn">
                     <div tagxxx="label"></div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Viral RNA derived from cell culture or infected mosquitoes were PCR amplified using
                        GSPs. Amplicons were analyzed using the ArboChip5.1 microarray. POS, positive PCR
                        amplification; NEG, negative PCR amplification; yes, detected by microarray; no, not
                        detected by microarray; n.t., not tested.
                     </p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">a</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Viruses propagated in cell culture and identified by microarray to species unless
                        otherwise noted.
                     </p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">b</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Virus-infected mosquito used for microarray evaluations, microarray detected at least
                        one target for each virus.
                     </p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">c</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">ArboChip5.1 does not include probes specific to the target.</p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">d</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Microarray detection with genus-level probes only.</p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">e</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Not a target virus on ArboChip5.1.</p>
                  </div>
               </div>
            </div>
         </div>
         <div id="s2g">
            <h3>Isolation of RNA and synthesis of cDNA</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">TRIzol-LS (Invitrogen Inc., Carlsbad, CA) extraction of RNA and cDNA synthesis using
               random hexamers and SuperscriptII (Invitrogen Inc.) was completed as previously described,
               except that the RNA was not subjected to a second round of purification before cDNA
               synthesis 
               <span ref-type="bibr" rid="pntd.0002349-OGuinn1">[39]</span>.
            </p>
         </div>
         <div id="s2h">
            <h3>Asymmetric PCR amplification and biotin labelling of microarray targets</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The sequences of the GSPs are listed in 
               <span ref-type="table" rid="pntd-0002349-t001">Table 1</span>. Each sample was amplified using the appropriate virus genus GSP set as determined
               by virus screening. Asymmetric PCR amplification and biotin labelling of the microarray
               targets was accomplished as follows. For a 25 µL reaction, 2 µL of cDNA, 1 µL of forward
               primer (10 µM), 5 µL of reverse primer (10 µM), 3 µL of biotin-14-dCTP (0.4 mM) (Invitrogen
               Inc.), and 14 µL of nuclease-free water was added to each PCR tube containing one
               puRe Taq Ready-To-Go™ PCR bead (Amersham Biosciences, Corp., Piscataway, NJ). The
               thermocycling conditions were set as follows: 95°C for two min; 8 cycles of: 94°C
               for 15 sec, 56–42°C (starting at 56°C, reducing 2°C each cycle) for 30 sec, and 72°C
               for 60 sec; 32 cycles of: 94°C for 15 sec, 40°C for 30 sec, and 72°C for 60 sec; followed
               by 72°C for 7 min and a final hold at 4°C. Amplicons were visualized using 2% Agarose
               E-Gel gels that contained ethidium bromide (Invitrogen Inc.), as previously described
               
               <span ref-type="bibr" rid="pntd.0002349-OGuinn1">[39]</span>. Expected product sizes are listed in 
               <span ref-type="table" rid="pntd-0002349-t001">Table 1</span>.
            </p>
         </div>
         <div id="s2i">
            <h3>Preparation of positive controls for hybridization</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Ten microarray probes (
               <span ref-type="supplementary-material" rid="pntd.0002349.s004">Table S2</span>) and reverse compliment oligonucleotides were used as positive controls for hybridization.
               The 10 positive oligonucleotide controls (100 µM) were combined equally resulting
               in a final concentration of 10 µM for each oligonucleotide. Ten microliters of the
               positive control pool was biotin labelled using the Label IT® μArray Biotin Labelling
               Kit (Mirus Bio LLC, Madison, WI) following the vendor's instructions. The biotin-labelled
               positive control pool was purified using the MinElute PCR Purification kit (QIAGEN
               Inc., Valencia, CA) following the manufacturer's instructions and eluted from the
               purification columns using 20 µL of nuclease-free water and 1 µL was used to spike
               each sample before hybridization (see below).
            </p>
         </div>
         <div id="s2j">
            <h3>Microarray hybridization and electrochemical detection</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The ArboChip5.1 microarray chips were hydrated using phosphate-buffered saline (PBS)
               (pH 7.4) for 10 min at 65°C and then pre-hybridized for 5 min at 50°C in pre-hybridization
               buffer (6× SSPE [0.9 M NaCl/60 mM sodium phosphate/6 mM EDTA], 0.05% Tween-20, 14
               mM EDTA, 5× Denhardt's solution, 0.05% sodium dodecyl sulfate (SDS)) with rotation
               using a UVP HB-500 Minidizer hybridization incubator (Ultra-Violet Products, LLC,
               Upland, CA) that was modified with microarray clamps fixed onto the rotisserie wheel.
               Preparation of the DNA samples for microarray hybridization was completed as follows.
               In a PCR tube, 15 µL of asymmetric biotin-labelled PCR amplicons (5 µL of each of
               the three gene PCR amplicons, including products without a visible band of the expected
               size) and 1 µL of biotin-labelled positive hybridization control oligonucleotides
               were mixed with 15 µL of 2× hybridization buffer (12× SSPE, 0.1% Tween-20, 28 mM EDTA,
               0.1% SDS). The samples were denatured for 1 min at 95°C, cooled for 1 min at 4°C,
               and then added to the microarray chambers. The microarray samples were hybridized
               at 50°C for 2 h with rotation. The microarray chambers were rinsed with 2× PBST (2×
               phosphate-buffered saline pH 7.4, 0.1% Tween-20), re-incubated with the same solution
               at 50°C for 5 min with rotation, and washed two more times with 2× PBST. The chambers
               were blocked with ElectraSense Blocking Buffer for 15 min at room temperature (RT)
               and labelled with ElectraSense Biotin Labeling Solution for 15 min at RT. The chambers
               were washed twice with ElectraSense Biotin Wash Solution, incubated for 10 min at
               RT, washed a third time with the Biotin Wash Solution, and rinsed with ElectraSense
               TMB Rinse Solution. The samples were developed using ElectraSense TMB Substrate and
               the ECD signals were measured in picoamps using the ElectraSense Reader within 1 min
               of adding the substrate. Data were recorded using the ElectraSense application software.
               All ElectraSense products were purchased from CustomArray, Inc. and the hybridization
               and detection methods were based on their recommendations, except that the 16 h hybridization
               was reduced to 2 h.
            </p>
         </div>
         <div id="s2k">
            <h3>Microarray data analysis</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The data were transformed into text files and transferred to Microsoft Excel (Microsoft
               Corp., Redmond, WA) for analysis. Each capture probe was sorted into one group, consisting
               of the virus genus and gene or segment, and one subgroup, consisting of the virus
               clade and specific target. The virus clades were based on observed phylogenetic analysis
               and did not always correspond to other reported clades or serogroups. The specific
               targets were single viruses or groups of related viruses.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The ECD signals were transformed into standard scores (z-scores) by subtracting the
               average signal of the negative background controls from each measured probe and dividing
               the difference by the standard deviation of the negative controls. Aforementioned,
               the probes were sorted into groups and subgroups for analysis. Subgroups with average
               z-scores greater than 10 were considered positive and used for viral RNA identification.
               Z-scores greater than 10 indicate that the measured probe values were greater than
               10 standard deviations above the background, therefore significant. Graphs for each
               group expressing the subgroups' average z-scores, maximum individual probe z-scores,
               and the positive cut-off (10) were created for visual analysis. Example data analysis
               is shown in 
               <span ref-type="supplementary-material" rid="pntd.0002349.s001">Dataset S1</span>.
            </p>
         </div>
         <div id="s2l">
            <h3>Microarray stripping</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In order for the microarray chips to be re-hybridized with new PCR amplicons, the
               previous amplicon:probe hybrids were denatured and then the amplicons were washed
               off. This was accomplished by washing the chip with Stripping Solution (CustomArray,
               Inc.), incubating the chip in the same solution at 65°C for 1 h, and then washing
               the chips with 95% EtOH and nuclease-free water. The microarray chambers were filled
               with PBS until re-use. The microarray chips were not used more than five times.
            </p>
         </div>
         <div id="s2m">
            <h3>Determination of the lower limits of detection and comparison to real-time and conventional
               PCR
            </h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The microarray lower limit of detection (LLOD) for virus infected mosquitoes was evaluated
               by using 10-fold serial dilutions of RNA extracted from one infected mosquito pooled
               with 24 uninfected mosquitoes. Isolation of RNA and synthesis of cDNA was completed
               using the methods described. The mosquito pool dilutions were tested using the described
               microarray amplification and detection methods and compared to results obtained using
               corresponding real-time PCR (qPCR) and convectional PCR assays. Real-time PCR was
               completed as follows: a 20-µL reaction contained 10 µL of SYBR Premix 
               <i>Ex Taq</i> DNA Polymerase (Takara Bio Inc., Otsu, Japan), 0.4 µL of 10 µM forward primer, 0.4
               µL of 10 µM reverse primer, 7.2 µL of nuclease-free water, and 2 µL of cDNA template
               using the following primer sets: mFU1 and cFD2 primers for the detection of flaviviruses
               
               <span ref-type="bibr" rid="pntd.0002349-Chao1">[40]</span>, VIR2052 primers for the detection of alphaviruses 
               <span ref-type="bibr" rid="pntd.0002349-Eshoo1">[36]</span>, and RVS and RVAs primers for the detection of RVFV 
               <span ref-type="bibr" rid="pntd.0002349-Drosten1">[41]</span>. The cycling conditions were as follows: 95°C for 30 sec, then 40 cycles of 95°C
               for 5 sec and 60°C for 20 sec. Fluorescence was read at the end of the 60°C annealing-extension
               step. Conventional PCR was completed as previously described 
               <span ref-type="bibr" rid="pntd.0002349-OGuinn1">[39]</span> using the following broad-range primer sets: MA and cFD2 primers (260 bp amplicon)
               for the detection of flaviviruses 
               <span ref-type="bibr" rid="pntd.0002349-Kuno1">[42]</span>, VIR2052 primers (120 bp amplicon) for the detection of alphaviruses 
               <span ref-type="bibr" rid="pntd.0002349-Eshoo1">[36]</span>, and Phlebo forward 1, forward 2, and reverse primers (370 bp amplicons) for of the
               detection of phleboviruses 
               <span ref-type="bibr" rid="pntd.0002349-Lambert1">[43]</span>.
            </p>
         </div>
         <div id="s2n">
            <h3>Mosquito field-collections and virus screening</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Mosquitoes were collected from locations near Lopburi and Kamphaeng Phet, Thailand,
               during March and April of 2011 and near Kamphaeng Phet and Ranyong, Thailand, during
               April and May of 2012. The mosquitoes were identified, processed, tested for the presence
               of viral RNA using the broad-range conventional PCR assays for flaviviruses, alphaviruses,
               and phleboviruses described above. In addition, BSC82V and BSC332V primers (250 bp
               amplicons) were used to screen for the Bunyamwera virus (BUNV) group 
               <span ref-type="bibr" rid="pntd.0002349-Kuno2">[44]</span>. The ABI 3100 genetic analyzer and Big Dye 3.1 (PE Biosystems, Inc., Foster City,
               CA) was used to sequence the amplicons.
            </p>
         </div>
      </div>
      <div id="s3">
         <h2>Results</h2>
         <div id="s3a">
            <h3>Assay specificity and reproducibility</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The GSP sets did not PCR amplify viruses from a different genus, uninfected cells
               (C6/36, BHK, and Vero cells), or uninfected mosquitoes (using species listed in 
               <span ref-type="table" rid="pntd-0002349-t003">Tables 3</span>–
               <span ref-type="table" rid="pntd-0002349-t006">6</span>). The negative PCR reactions were still tested on the microarray and none were detected.
               Fifty-two virus strains from 46 different species were evaluated for PCR amplification
               and probe specificity, representing 46/144 (32%) of the total targeted viruses. Each
               microarray probe was evaluated for specificity to its intended target(s). Some PCR
               amplicons cross-hybridized to non-predicted probes (i.e. probes specific for a different
               virus or group of viruses); however, in each case there was no more than one such
               probe per subgroup and the average z-scores were never greater than 10. No probes
               cross-hybridized with viruses from a different genus.
            </p>
            <div tagxxx="table-wrap">
               <div tagxxx="object-id">10.1371/journal.pntd.0002349.t006</div>
               <div tagxxx="label">Table 6</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">PCR amplification and microarray detection of orthobunyaviruses.</div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">hsidesgroups
                     <colgroup>1
                        <col>left1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                     </colgroup>
                     <thead>
                        <tr>
                           <td>left11</td>
                           <td>left11</td>
                           <td>4left1PCR amplification/microarray detection
                              <span ref-type="table-fn" rid="nt128">a</span>
                           </td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11Virus</td>
                           <td>left11Strain</td>
                           <td>left11S BUNV</td>
                           <td>left11S CEV</td>
                           <td>left11M</td>
                           <td>left11L</td>
                           <td>left11Mosquito species
                              <span ref-type="table-fn" rid="nt129">b</span>
                           </td>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>left11Bunyamwera</td>
                           <td>left11330</td>
                           <td>left11POS/no
                              <span ref-type="table-fn" rid="nt130">c</span>
                           </td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt131">d</span>
                           </td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Bunyamwera</td>
                           <td>left11131B-06</td>
                           <td>left11POS/no
                              <span ref-type="table-fn" rid="nt130">c</span>
                           </td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt131">d</span>
                           </td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Bunyamwera</td>
                           <td>left11460</td>
                           <td>left11POS/no
                              <span ref-type="table-fn" rid="nt130">c</span>
                           </td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt131">d</span>
                           </td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Bwamba</td>
                           <td>left116502</td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt132">e</span>
                           </td>
                           <td>left11POSd/yes</td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt130">c</span>
                           </td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt130">c</span>
                           </td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Cache Valley</td>
                           <td>left11RV257</td>
                           <td>left11POS/yes</td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt131">d</span>
                           </td>
                           <td>left11POS/yes</td>
                           <td>left11POS/no
                              <span ref-type="table-fn" rid="nt130">c</span>
                           </td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Germiston</td>
                           <td>left11SAAr1050</td>
                           <td>left11POS/no</td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt131">d</span>
                           </td>
                           <td>left11POS/yes</td>
                           <td>left11POS/no
                              <span ref-type="table-fn" rid="nt130">c</span>
                           </td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Ilesha</td>
                           <td>left11Zika-54</td>
                           <td>left11NEG</td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt131">d</span>
                           </td>
                           <td>left11POS/yes</td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt130">c</span>
                           </td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11La Crosse</td>
                           <td>left1197-WV-131</td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt131">d</span>
                           </td>
                           <td>left11POS/yes
                              <span ref-type="table-fn" rid="nt130">c</span>
                              <span ref-type="table-fn" rid="nt132">e</span>
                           </td>
                           <td>left11POS/yes</td>
                           <td>left11POS/yes</td>
                           <td>left11
                              <i>Ae. aegypti</i>
                           </td>
                        </tr>
                        <tr>
                           <td>left11Maguari</td>
                           <td>left11MSP-18</td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt130">c</span>
                           </td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt131">d</span>
                           </td>
                           <td>left11NEG</td>
                           <td>left11POS/no
                              <span ref-type="table-fn" rid="nt130">c</span>
                           </td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Oropouche</td>
                           <td>left11TR9760</td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt131">d</span>
                           </td>
                           <td>left11POS/yes</td>
                           <td>left11NEG</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Pongola</td>
                           <td>left11SA Tar</td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt131">d</span>
                           </td>
                           <td>left11POS/yes</td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt130">c</span>
                           </td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt130">c</span>
                           </td>
                           <td>left11n.t.</td>
                        </tr>
                        <tr>
                           <td>left11Tahyna</td>
                           <td>left11RV285</td>
                           <td>left11NEG
                              <span ref-type="table-fn" rid="nt131">d</span>
                           </td>
                           <td>left11POS/yes</td>
                           <td>left11POS/no</td>
                           <td>left11POS/yes</td>
                           <td>left11n.t.</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <div tagxxx="table-wrap-foot">
                  <div tagxxx="fn">
                     <div tagxxx="label"></div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Viral RNA derived from cell culture or infected mosquitoes were PCR amplified using
                        GSPs. Amplicons were analyzed using the ArboChip5.1 microarray. POS, positive PCR
                        amplification; NEG, negative PCR amplification; yes, detected by microarray; no, not
                        detected by microarray; n.t., not tested.
                     </p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">a</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Viruses propagated in cell culture and identified by microarray to species unless
                        otherwise noted.
                     </p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">b</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Virus-infected mosquito used for microarray evaluations, microarray detected at least
                        one target for each virus.
                     </p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">c</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">ArboChip5.1 does not include probes specific to the target.</p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">d</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">GSP set not specific for this clade.</p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">e</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">CEV clade detection only.</p>
                  </div>
               </div>
            </div>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The measured variability between replicates of the same samples was used to determine
               if the assay results were reproducible. Cell-culture derived WNV lineage 1 (WNVL1,
               strain NY397-99), CHIKV (strain INDO23574), and RVFV (strain ZH501) were PCR amplified
               using the GSP sets and analyzed by the microarray four times independently. The z-scores
               from the virus-specific probes were averaged for each replicate. The mean, standard
               deviation, 95% confidence interval and coefficient of variance was calculated for
               each virus. The results are summarized in 
               <span ref-type="table" rid="pntd-0002349-t002">Table 2</span>.
            </p>
         </div>
         <div id="s3b">
            <h3>Amplification, detection, and identification of flaviviruses</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Microarray identification of PCR amplicons using the consensus flavivirus GSPs was
               evaluated using 20 flaviviruses propagated in cell culture (
               <span ref-type="table" rid="pntd-0002349-t003">Table 3</span>). A visual analysis example of WNVL1 (strain NY397-99) is shown in 
               <span ref-type="fig" rid="pntd-0002349-g002">Figure 2</span>. All of the flaviviruses tested were identified to species via the NS3 and NS5 gene
               targets and 15/20 flaviviruses (75%) were detected by all three gene targets. The
               Kunjin virus (KUNV), Rocio virus (ROCV), and Zika virus (ZIKV) strains tested did
               not PCR amplify using the flavivirus E gene GSPs. The JEV and Quang Binh virus (QBV)
               strains tested did amplify using the flavivirus E gene GSPs but were not detected
               by the microarray E gene probes. Multiple strains of TMUV, WNVL1, and WNVL2 were used
               to evaluate microarray identification of virus targets containing slight nucleotide
               differences. The 17 different flaviviruses evaluated represents 17/34 (50%) of the
               total flaviviruses targeted by the ArboChip5.1.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pntd.0002349.g002</div>
               <div tagxxx="label">Figure 2</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Visual analysis of cell-culture derived WNVL1 strain NY397-99.</div>
                  <p>
                     <i>Flavivirus</i> gene-specific PCR amplicons were identified using oligonucleotide microarray probes
                     grouped by virus genus and gene, (A) 
                     <i>Flavivirus</i> E gene, (B) 
                     <i>Flavivirus</i> NS3 gene, and (C) 
                     <i>Flavivirus</i> NS5 gene, and sorted into subgroups based on phylogenetic clade and target virus(es).
                     ECD signals were converted into z-scores. Subgroups with average z-scores greater
                     than 10 were considered positive and used for virus identification. The plotted maximum
                     z-scores represent the greatest individual probe z-score within a subgroup and were
                     used to determine cross-hybridization. The PCR amplicons hybridized with probes in
                     the 
                     <i>Flavivirus</i> generic, JEV clade, WNVL1/KUNV, and WNVL1 subgroups for all three gene targets. WNVL1
                     was differentiated from KUNV by the less than 10 z-scores for the KUNV subgroups.
                     The virus abbreviations are defined in 
                     <span ref-type="supplementary-material" rid="pntd.0002349.s003">Table S1</span>.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Single mosquitoes infected with DENV types 1–4, TMUV, JEV, WNVL1, and YFV were evaluated
               (virus strains and mosquito species are listed in 
               <span ref-type="table" rid="pntd-0002349-t003">Table 3</span>). Overall, the results were similar to those obtained using the viruses propagated
               in cell culture, with the following exceptions. The PCR bands for DENV-4 infected
               
               <i>Ae. albopictus</i> were faint but they were still accurately identified by all three genes on the microarray.
               The E gene was not PCR amplified from either stain of TMUV infected 
               <i>Culex tarsalis</i> and could not be detected on the microarray. To mimic field samples, single mosquitoes
               infected with DENV-1, TMUV, and WNVL1 were pooled with 24 negative mosquitoes. Compared
               to single infected mosquitoes, the identification results were the same but had overall
               lower z-scores. A WNVL1-infected and a JEV-infected 
               <i>Cx. pipiens</i> were pooled with 23 negative mosquitoes to assess the amplification and detection
               of two distinct viruses that could be found in a field collected pool of mosquitoes.
               All three gene targets were amplified using the flavivirus GSP sets but the E genes
               of JEV and WNVL1 and the NS5 gene of WNVL1 were not detected using the microarray.
               Three of the five JEV-specific NS5 gene probes had z-scores greater than 10, but the
               average z-score for the subgroup was less than 10 (9.13). However, all of the JEV-
               and WNVL1-specific (out of 5 and 8, respectively) and two of the four WNVL1/KUNV NS3
               probes had z-scores greater than 10 (
               <span ref-type="supplementary-material" rid="pntd.0002349.s002">Figure S1</span>). The JEV-specific probes and the WNVL1-specific probes did not hybridize to NS3
               amplicons from the other virus when evaluated individually; therefore this is an example
               of microarray detecting the RNA from two related but distinct flaviviruses in the
               same mosquito pool.
            </p>
         </div>
         <div id="s3c">
            <h3>Amplification, detection, and identification of alphaviruses</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Microarray identification of PCR amplicons using the consensus alphavirus GSPs was
               evaluated using 14 alphaviruses propagated in cell culture (
               <span ref-type="table" rid="pntd-0002349-t004">Table 4</span>). A visual analysis example of CHIKV (strain INDO23574) is shown in 
               <span ref-type="fig" rid="pntd-0002349-g003">Figure 3</span>. All of the alphaviruses were identified to species via detection by the nsP1 and
               nsP4 gene targets and 10/14 (71%) were identified by all three gene targets. However,
               Ndumu virus (NDUV) and Ross River virus (RRV) were not PCR amplified using the alphavirus
               E1 GSP set and the Una virus (UNAV) nsP1 and E1 amplicons were not detected by the
               microarray probes. Subspecies of Sindbis virus (SINV), Ockelbo virus (OCKV) and Babanki
               virus (BBKV), could not be differentiated from SINV and the probes for all three viruses
               were placed in the “SINV_clade” subgroup. Likewise, ONNV and IOV, a subspecies of
               ONNV, could not be differentiated and the probes were placed into the same subgroup.
               The 14 alphaviruses evaluated represents 14/23 (61%) of the total alphaviruses targeted
               on the microarray design.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pntd.0002349.g003</div>
               <div tagxxx="label">Figure 3</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Visual analysis of cell-culture derived CHIKV strain INDO23574.</div>
                  <p>
                     <i>Alphavirus</i> gene-specific PCR amplicons were identified using oligonucleotide microarray probes
                     grouped by virus genus and gene, (A) 
                     <i>Alphavirus</i> E1 gene, (B) 
                     <i>Alphavirus</i> nsP1 gene, and (C) 
                     <i>Alphavirus</i> nsP4 gene, and sorted into subgroups based on phylogenetic clade and target virus(es).
                     ECD signals were converted into z-scores. Subgroups with average z-scores greater
                     than 10 were considered positive and used for virus identification. The plotted maximum
                     z-scores represent the greatest individual probe z-score within a subgroup and were
                     used to determine cross-hybridization. The PCR amplicons hybridized with probes in
                     the CHIKV subgroups for all three gene targets and probes in the 
                     <i>Alphavirus</i> generic and SFV clade subgroups for the nsP1 and nsP4 genes. The virus abbreviations
                     are defined in 
                     <span ref-type="supplementary-material" rid="pntd.0002349.s003">Table S1</span>.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To mimic field samples, 
               <i>Aedes</i> species mosquitoes infected with CHIKV, RRV, SINV, and western equine encephalomyelitis
               virus (WEEV) were evaluated individually and pooled with 24 uninfected mosquitoes
               (virus strains and mosquito species are listed in 
               <span ref-type="table" rid="pntd-0002349-t004">Table 4</span>). As expected, the z-scores were the greatest when using PCR amplicons generated
               from cell culture propagated virus and were approximately twofold greater compared
               to using PCR amplicons from individual infected mosquitoes. Single infected mosquitoes
               pooled with 24 uninfected mosquitoes produced the lowest z-scores; however, the z-scores
               were still greater than 10 resulting in positive virus identification.
            </p>
         </div>
         <div id="s3d">
            <h3>Amplification, detection, and identification of phleboviruses</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Microarray identification of phlebovirus GSP PCR amplicons were evaluated using six
               phleboviruses propagated in cell culture (
               <span ref-type="table" rid="pntd-0002349-t005">Table 5</span>). A visual analysis example of RVFV (strain ZH548) is shown in 
               <span ref-type="fig" rid="pntd-0002349-g004">Figure 4</span>. All of the phleboviruses that were tested were PCR positive using the phlebovirus
               GSPs and 4/6 (67%) were identified to species via all three gene targets. However,
               sandfly fever Naples virus (SFNV) was not detected by the M segment probes. The five
               targeted virus species evaluated represents 5/23 (22%) of the total phleboviruses
               targeted by the ArboChip5.1. The microarray did not contain Chagres virus-specific
               probes but it was included to evaluate the detection of non-targeted viruses. Chagres
               virus was detected by the L segment 
               <i>Phlebovirus</i> generic probes, showing the detection of non-targeted but related viruses is possible,
               but virus-specific identification is not.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pntd.0002349.g004</div>
               <div tagxxx="label">Figure 4</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Visual analysis of cell-culture derived RVFV strain ZH548.</div>
                  <p>
                     <i>Phlebovirus</i> gene-specific PCR amplicons were identified using oligonucleotide microarray probes
                     grouped by virus genus and segment, (A) 
                     <i>Phlebovirus</i> S segment, (B) 
                     <i>Phlebovirus</i> M segment, and (C) 
                     <i>Phlebovirus</i> L segment, and sorted into subgroups based on phylogenetic clade and target virus(es).
                     ECD signals were converted into z-scores. Subgroups with average z-scores greater
                     than 10 were considered positive and used for virus identification. The plotted maximum
                     z-scores represent the greatest individual probe z-score within a subgroup and were
                     used to determine cross-hybridization. The PCR amplicons hybridized with probes in
                     the RVFV subgroups for all three segment targets and 
                     <i>Phlebovirus</i> generic subgroup for L segment. The virus abbreviations are defined in 
                     <span ref-type="supplementary-material" rid="pntd.0002349.s003">Table S1</span>.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <i>Culex pipiens</i> infected with RVFV strain ZH501 were evaluated alone and in pools of 24 uninfected
               mosquitoes to simulate field conditions. The microarray identified RVFV via all three
               gene targets for both preparations.
            </p>
         </div>
         <div id="s3e">
            <h3>Amplification, detection, and identification of orthobunyaviruses</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Twelve orthobunyaviruses propagated in cell culture were used to evaluate microarray
               identification of phlebovirus GSP PCR amplicons (
               <span ref-type="table" rid="pntd-0002349-t006">Table 6</span>). Example visual analysis of BUNV (strain 131B-06) is shown in 
               <span ref-type="fig" rid="pntd-0002349-g005">Figure 5</span>. The orthobunyaviruses tested could not be PCR amplified using a single pair of S
               segment GSPs. Therefore two sets of consensus S segments GSPs were created targeting
               the two major orthobunyavirus groups, the BUNV and California encephalitis virus (CEV)
               serogroups. Due to a lack of whole genome sequence information publically available
               and high rates of cross-hybridization of related orthobunyaviruses, Oropouche virus
               (OROV) and Tahyna virus (TAHV) were the only viruses evaluated that had virus-specific
               probes designed for all three genome segments. Seven of the 12 (58%) orthobunyaviruses
               were PCR amplified with three GSP sets. The microarray only detected all three segments
               from La Crosse virus (LACV); however, the M segment was only identified as belonging
               to the CEV clade (no M segment specific probes for LACV due to cross-reactivity with
               other CEV clade viruses). All of the orthobunyavirus tested were PCR amplified, detected,
               and identified to virus species by at least one target, except for Maguari virus,
               which was not detected by the microarray due to a lack of PCR amplification of the
               S and M segments and a lack of virus specific probes for the L segment. Multiple strains
               of BUNV were used to evaluate microarray identification of virus targets with slight
               nucleotide differences. The 10 different orthobunyavirus species evaluated represents
               10/60 (17%) of the total orthobunyaviruses targeted on the microarray design.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pntd.0002349.g005</div>
               <div tagxxx="label">Figure 5</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Visual analysis of cell-culture derived BUNV strain 131B-06.</div>
                  <p>
                     <i>Orthobunyavirus</i> gene-specific PCR amplicons were identified using oligonucleotide microarray probes
                     grouped by virus genus and segment, (A) 
                     <i>Orthobunyavirus</i> S segment, (B) 
                     <i>Orthobunyavirus</i> M segment, and (C) 
                     <i>Orthobunyavirus</i> L segment, and sorted into subgroups based on phylogenetic clade and target virus(es).
                     ECD signals were converted into z-scores. Subgroups with average z-scores greater
                     than 10 were considered positive and used for virus identification. The plotted maximum
                     z-scores represent the greatest individual probe z-score within a subgroup and were
                     used to determine cross-hybridization. The PCR amplicons hybridized with probes in
                     the BUNV subgroups for the M and L segment targets but virus-specific probes could
                     not be designed for the S segment. The BUNV amplicons did have z-scores greater than
                     10 for seven of the 42 S segment BUNV clade probes, but the average subgroup z-score
                     was less than 10. The virus abbreviations are defined in 
                     <span ref-type="supplementary-material" rid="pntd.0002349.s003">Table S1</span>.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <i>Ae. aegypti</i>-infected with LACV was used to evaluate orthobunyavirus detection in mosquitoes.
               All three segments were PCR amplified and detected by the microarray from single infected
               mosquitoes.
            </p>
         </div>
         <div id="s3f">
            <h3>Lower limits of detection</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Ten-fold serial dilutions of RNA extracted from one infected mosquito pooled with
               24 uninfected mosquitoes were used to determine the LLOD for WNVL1 (strain NY397-99),
               SINV (strain Ken07-611), and RVFV (strain ZH501). Pools of 
               <i>Cx. pipiens</i> were used for WNVL1 and RVFV and pools of 
               <i>Ae. aegypti</i> were used for SINV. The infected mosquito titers were as follows: WNVL1-infected
               and RVFV-infected 
               <i>Cx. pipiens</i> were 10
               <sup>4.6</sup> PFU/mL and 10
               <sup>6.4</sup> PFU/mL, respectively, and SINV-infected 
               <i>Ae. Aegypti</i> was 10
               <sup>4.3</sup> PFU/mL. The LLOD of cDNA by the using the microarray, which includes asymmetric PCR
               amplification and probe hybridization, was compared to qPCR and conventional PCR.
               The LLOD data is summarized in 
               <span ref-type="table" rid="pntd-0002349-t007">Table 7</span>.
            </p>
            <div tagxxx="table-wrap">
               <div tagxxx="object-id">10.1371/journal.pntd.0002349.t007</div>
               <div tagxxx="label">Table 7</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Lower limits of detection from virus-infected mosquito pools.</div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">hsidesgroups
                     <colgroup>1
                        <col>left1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                     </colgroup>
                     <thead>
                        <tr>
                           <td>left11</td>
                           <td>3left1Lower limits of detection (PFUe)</td>
                        </tr>
                        <tr>
                           <td>left11Virus</td>
                           <td>left11qPCR</td>
                           <td>left11Conv. PCR</td>
                           <td>left11Microarray</td>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>left11West Nile</td>
                           <td>left11200</td>
                           <td>left11200</td>
                           <td>left11200</td>
                        </tr>
                        <tr>
                           <td>left11Sindbis</td>
                           <td>left1110</td>
                           <td>left1110</td>
                           <td>left11100</td>
                        </tr>
                        <tr>
                           <td>left11Rift Valley fever</td>
                           <td>left11125</td>
                           <td>left11125</td>
                           <td>left111,250</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <div tagxxx="table-wrap-foot">
                  <div tagxxx="fn">
                     <div tagxxx="label"></div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The lower limits of detection determined for infected mosquitoes were evaluated by
                        using 10-fold serial dilutions of RNA extracted from one infected mosquito pooled
                        with 24 uninfected mosquitoes. PFUe, plaque forming unit equivalents; qPCR, real-time
                        PCR, Conv. PCR, conventional PCR.
                     </p>
                  </div>
               </div>
            </div>
         </div>
         <div id="s3g">
            <h3>Evaluation with field-collected mosquitoes</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Mosquitoes collected in Thailand were used to determine the applicability of the ArboChip5.1
               microarray for identifying virus RNA present in mosquito pools (n = 1–25). From1,445
               mosquito pools (642 pools were collected in 2011 and 803 pools were collected in 2012)
               that were screened for the presence of flaviviruses by conventional PCR, 13 pools
               were confirmed as containing flavivirus RNA. Selected medically important mosquito
               pools, such as those containing 
               <i>Ae. aegypti</i>, were screened for the presence of alphaviruses, phleboviruses, and BUNV group viruses,
               however, no positive pools were detected. The flavivirus RNA positive pools were further
               analyzed using the microarray (
               <span ref-type="table" rid="pntd-0002349-t008">Table 8</span>). Visual analysis of the microarray data for sample Th9-0122 is shown in 
               <span ref-type="fig" rid="pntd-0002349-g006">Figure 6</span> and the raw data analysis are shown in 
               <span ref-type="supplementary-material" rid="pntd.0002349.s001">Dataset S1</span>. Nine of the 13 pools were correctly identified by at least one gene targeted by
               the microarray. RNA sequences related to QBV were found in three pools not identified
               by the microarray. The other pool not identified by the microarray contained RNA related
               to Wang Thong virus, a virus not included in the ArboChip5.1 design due to a lack
               of sequence information in the targeted gene regions. The results were verified by
               virus isolation and sequencing of the PCR amplicons. Pool number Th9-0032 was identified
               in the field as QBV, but partial sequencing of the NS5 gene showed that while QBV
               was its closest match, it was only 78.3% identical. There was less than a 70 bp overlap
               between the targeted NS5 region that identified Th9-0032 as QBV and the partial sequence.
               Further analysis on the other genes will need to be completed to determine the identity
               of the QBV-like virus.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pntd.0002349.g006</div>
               <div tagxxx="label">Figure 6</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Visual analysis of 
                     <i>Ae. aegypti</i> mosquito pool Th9-0122 collected in Thailand.
                  </div>
                  <p>Screening methods indicated that the mosquito pool contained flavivirus RNA and cDNA
                     from the pool was PCR amplified with 
                     <i>Flavivirus</i> (A) E, (B) NS3, and (C) NS5 GSPs. The PCR amplicons hybridized only with probes in
                     the DENV3 subgroups for the (A) E and (C) NS5 gene targets. For the (B) NS3 gene,
                     the amplicons hybridized with probes in the DENV clade and DENV2/3 (hybridizes with
                     DENV2 and DENV3) subgroups, but did not hybridize with the DENV2-specific subgroup.
                     The mosquito pool was identified as containing DENV-3 RNA.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <div tagxxx="table-wrap">
               <div tagxxx="object-id">10.1371/journal.pntd.0002349.t008</div>
               <div tagxxx="label">Table 8</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Microarray identification of flavivirus RNA from Thailand mosquitoes (2011–2012).</div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">hsidesgroups
                     <colgroup>1
                        <col>left1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                        <col>center1</col>
                     </colgroup>
                     <thead>
                        <tr>
                           <td>left11</td>
                           <td>left11</td>
                           <td>3left1PCR result/microarray identification</td>
                           <td>left11</td>
                        </tr>
                        <tr>
                           <td>left11Pool No.
                              <span ref-type="table-fn" rid="nt135">a</span>
                           </td>
                           <td>left11Mosquito species (pool size)</td>
                           <td>left11E</td>
                           <td>left11NS3</td>
                           <td>left11NS5</td>
                           <td>left11Sequencing result
                              <span ref-type="table-fn" rid="nt136">b</span>
                           </td>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>left11Th9-0011</td>
                           <td>left11
                              <i>Cx. quinquefasciatus</i> (1)
                           </td>
                           <td>left11NEG</td>
                           <td>left11NEG</td>
                           <td>left11POS/CxFV</td>
                           <td>left11CxFV</td>
                        </tr>
                        <tr>
                           <td>left11Th9-0024</td>
                           <td>left11
                              <i>Cx. vishnui</i> (2)
                           </td>
                           <td>left11POS/n.d.</td>
                           <td>left11NEG</td>
                           <td>left11NEG</td>
                           <td>left11QBV</td>
                        </tr>
                        <tr>
                           <td>left11Th9-0032</td>
                           <td>left11
                              <i>Cx. quinquefasciatus</i> (6)
                           </td>
                           <td>left11POS/n.d.</td>
                           <td>left11POS
                              <span ref-type="table-fn" rid="nt137">c</span>/n.d.
                           </td>
                           <td>left11POS
                              <span ref-type="table-fn" rid="nt137">c</span>/QBV
                           </td>
                           <td>left11QBV-like
                              <span ref-type="table-fn" rid="nt138">d</span>
                           </td>
                        </tr>
                        <tr>
                           <td>left11Th9-0122</td>
                           <td>left11
                              <i>Ae. aegypti</i> (1)
                           </td>
                           <td>left11POS/DENV3</td>
                           <td>left11POS/DENV3</td>
                           <td>left11POS/DENV3</td>
                           <td>left11DENV3</td>
                        </tr>
                        <tr>
                           <td>left11Th9-0164</td>
                           <td>left11
                              <i>Cx. tritaeniorhynchus</i> (25)
                           </td>
                           <td>left11POS/n.d.</td>
                           <td>left11POS/JEV</td>
                           <td>left11POS/JEV</td>
                           <td>left11JEV</td>
                        </tr>
                        <tr>
                           <td>left11Th9-0175</td>
                           <td>left11
                              <i>Cx. tritaeniorhynchus</i> (25)
                           </td>
                           <td>left11POS/n.d.</td>
                           <td>left11POS/JEV</td>
                           <td>left11POS/JEV</td>
                           <td>left11JEV</td>
                        </tr>
                        <tr>
                           <td>left11Th10-0040</td>
                           <td>left11
                              <i>Cx. vishnui</i> (20)
                           </td>
                           <td>left11NEG</td>
                           <td>left11POS/TMUV</td>
                           <td>left11POS/n.d.</td>
                           <td>left11TMUV</td>
                        </tr>
                        <tr>
                           <td>left11Th10-0121</td>
                           <td>left11
                              <i>Cx. vishnui</i> (20)
                           </td>
                           <td>left11NEG</td>
                           <td>left11POS/TMUV</td>
                           <td>left11POS/TMUV</td>
                           <td>left11TMUV</td>
                        </tr>
                        <tr>
                           <td>left11Th10-0217</td>
                           <td>left11
                              <i>Cx. vishnui</i> (20)
                           </td>
                           <td>left11NEG</td>
                           <td>left11POS/TMUV</td>
                           <td>left11POS/TMUV</td>
                           <td>left11TMUV</td>
                        </tr>
                        <tr>
                           <td>left11Th10-0552</td>
                           <td>left11
                              <i>Cx. vishnui</i> (20)
                           </td>
                           <td>left11NEG</td>
                           <td>left11POS/TMUV</td>
                           <td>left11POS/TMUV</td>
                           <td>left11TMUV</td>
                        </tr>
                        <tr>
                           <td>left11Th10-0578</td>
                           <td>left11
                              <i>Cx. fuscocephala</i> (16)
                           </td>
                           <td>left11NEG</td>
                           <td>left11POS/n.d.</td>
                           <td>left11NEG</td>
                           <td>left11WTV
                              <span ref-type="table-fn" rid="nt139">e</span>
                           </td>
                        </tr>
                        <tr>
                           <td>left11Th10-0591</td>
                           <td>left11
                              <i>Cx. gelidus</i> (20)
                           </td>
                           <td>left11NEG</td>
                           <td>left11NEG</td>
                           <td>left11POS
                              <span ref-type="table-fn" rid="nt137">c</span>/n.d.
                           </td>
                           <td>left11QBV</td>
                        </tr>
                        <tr>
                           <td>left11Th10-0767</td>
                           <td>left11
                              <i>Cx. gelidus</i> (2)
                           </td>
                           <td>left11NEG</td>
                           <td>left11NEG</td>
                           <td>left11POS
                              <span ref-type="table-fn" rid="nt137">c</span>/n.d.
                           </td>
                           <td>left11QBV</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <div tagxxx="table-wrap-foot">
                  <div tagxxx="fn">
                     <div tagxxx="label"></div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">cDNAs from mosquito pools that were found to contain viral RNA by flavivirus screening
                        methods were PCR amplified using GSPs and analyzed using the ArboChip5.1 microarray
                        for virus identification. POS, positive PCR amplification; NEG, negative PCR amplification;
                        n.d., no detection; CxFV, 
                        <i>Culex</i> flavivirus; QBV, Quang Binh virus; DENV3, dengue virus type 3; JEV, Japanese encephalitis
                        virus; TMUV, tembusu virus; WTV, Wang Thong virus.
                     </p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">a</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Mosquito pool No. beginning with Th9 were collected in 2011, Th10 were collected in
                        2012.
                     </p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">b</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Identifcations based on greater than 90% identity using partial NS5 sequences.</p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">c</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">PCR amplification produces a weak visible band.</p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">d</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Partial NS5 sequence 78.3% identical to QBV.</p>
                  </div>
                  <div tagxxx="fn">
                     <div tagxxx="label">e</div>
                     <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">WTV targeted probes not included on ArboChip5.1.</p>
                  </div>
               </div>
            </div>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Virus-negative pools of 
               <i>Cx. vishnui</i>, 
               <i>Cx. tritaeniorhynchus</i>, 
               <i>Cx. quinquefasciatus</i>, 
               <i>Cx. gelidus</i>, 
               <i>Cx. terzii</i>, 
               <i>Ae. albopictus</i>, 
               <i>Ae. vexans</i>, 
               <i>Mansonia uniformis</i>, 
               <i>Armigeres subalbatus</i>, and 
               <i>Anopheles peditaeniatus</i> (n = 19–25 mosquitoes/pool) were processed and used to confirm that the microarray
               identifications were not from cross-hybridizing mosquito RNA or DNA. All mosquito
               pools that tested negative by PCR also produced negative microarray results.
            </p>
         </div>
      </div>
      <div id="s4">
         <h2>Discussion</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The ArboChip5.1 consists of 2,097 oligonucleotide probes targeting 144 different mosquito-borne
            RNA viruses across the genera 
            <i>Flavivirus</i>, 
            <i>Alphavirus</i>, 
            <i>Orthobunyavirus</i>, and 
            <i>Phlebovirus</i>. For most of the selected viruses, capture probes were designed to multiple genes
            to increase the odds of detection, identification, and confirmation. The assay also
            utilized consensus GSPs for broad-range PCR amplification of viral RNA, even in the
            presence of an abundance amount of mosquito nucleic acids. The microarray platform
            described here, and as we previously discussed, is portable and rugged enough to endure
            travel and field conditions 
            <span ref-type="bibr" rid="pntd.0002349-Grubaugh1">[30]</span>. The microarray was able to identify viral RNA extracted from cell culture, laboratory-infected
            mosquitoes, and field-collected mosquitoes, thus making it versatile and practical.
            To our knowledge, this is the most comprehensive microarray assay specifically designed
            for arbovirus surveillance in mosquito vectors.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Previously developed and newly defined assays were created or adapted to advance and
            to streamline the process for using microarrays in field surveillance. RNA extraction
            and the PCR field-based procedures previously described 
            <span ref-type="bibr" rid="pntd.0002349-OGuinn1">[39]</span> were modified slightly to fit the needs of the microarray. Asymmetric PCR amplification
            was utilized to produce a higher ratio of reverse compliment amplicons to hybridize
            with the microarray probes. In addition, biotinylation of the amplicons was incorporated
            into the PCR to improve the assay efficiency and ease of use. In a previous study,
            we evaluated using direct detection of viral RNA without amplification, random primed
            PCR amplification, and gene-specific PCR amplification and determined that gene-specific
            PCR amplification was needed for microarray detection of flavivirus RNA in infected
            mosquitoes 
            <span ref-type="bibr" rid="pntd.0002349-Grubaugh1">[30]</span>. Gene-specific multiplexed PCR was found to reduce the sensitivity of microarray
            detection (data not shown); therefore, the reactions were performed individually with
            each GSP set. For a given virus genus, the amplified gene targets were pooled after
            PCR to reduce the number of microarrays to be tested. Though some level of variability
            was observed, the outcomes of the microarrays were found to be reproducible. However,
            the microarray chips have been found to fail on occasion if they were not stored or
            stripped properly.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The limiting factors for creating the viral diagnostic DNA microarray presented here
            were the availability of viral sequences present in publically available databases
            and the sequence diversity in the targeted nucleotide regions. Flaviviruses and alphaviruses
            are generally well characterized, and their entire genomic sequences are available
            for most known species. As a result, unique probes targeting all three genes of diagnostic
            interest for most flaviviruses and alphaviruses were created. We were able to detect
            and identify all of the flaviviruses and alphaviruses tested from both cell culture
            and infected mosquitoes, except for a few closely related alphaviruses as discussed
            below. Closely related flaviviruses, such as the four serotypes of DENV and WNVL1,
            WNVL2, and KUNV were clearly differentiated using the microarray. Additionally, JEV
            and WNVL1, both JEV clade flaviviruses, were detected from a pool of laboratory infected
            mosquitoes, showcasing the ability of the microarray to differentiate between closely
            related viruses that might be found in the same mosquito pool. Though we were able
            to differentiate between many closely related alphaviruses, we were not able to differentiate
            between SINV, OCKV, and BBKV and between ONNV and IOV. This was because there was
            not enough sequence diversity between the listed alphaviruses in the targeted nucleotide
            regions to create probes that would not cross-hybridize using the described assay
            conditions. Instead, the viruses were identified as either part of the SINV or ONNV
            clades and other data, such as the mosquito species and collection location, could
            be used to distinguish the specific alphavirus.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Some bunyaviruses, such as BUNV (
            <i>Orthobunyavirus</i>), LACV (
            <i>Orthobunyavirus</i>), and RVFV (
            <i>Phlebovirus</i>), are very well studied and genetically characterized. However, the 
            <i>Bunyaviridae</i> family is expansive and diverse, and despite many characterization efforts 
            <span ref-type="bibr" rid="pntd.0002349-Lambert1">[43]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Lambert2">[45]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Mores1">[46]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Chowdhary1">[47]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Hang1">[48]</span>, many bunyavirus genomes have not been fully sequenced. Sequence information is especially
            lacking for the L segment for many of the orthobunyaviruses, as evident by our nominal
            L segment probes and was the reason why we could not design probes to all three segments
            for most of the evaluated orthobunyaviruses. Even in some cases where complete sequence
            information was available, virus-specific probes could not be designed for all of
            the segments due to sequence similarities. For instance, the S segment of orthobunyaviruses
            share close sequence identity 
            <span ref-type="bibr" rid="pntd.0002349-Lambert2">[45]</span>, making microarray differentiation of this segment difficult. Moreover, bunyaviruses,
            like many other segmented viruses, can use natural genome reassortment as a driving
            force for evolution 
            <span ref-type="bibr" rid="pntd.0002349-Turell1">[49]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Briese2">[50]</span>. For example, Ngari virus is the reassortment of two other orthobunyaviruses: the
            S and L segments from BUNV and the M segment from Batai virus 
            <span ref-type="bibr" rid="pntd.0002349-Briese3">[51]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Yanase1">[52]</span>. These factors combined made it difficult to design virus-specific probes to all
            three segments for many of these viruses. However, there was enough sequence data
            available and nucleotide diversity to design probes to make microarray identification
            of at least one segment for the orthobunyaviruses and phleboviruses tested.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Field-collected mosquitoes have varied arbovirus titers based on the virus strain,
            mosquito species, environmental conditions, and infection status (nondisseminated
            limited to the midgut or disseminated throughout the mosquito's body) 
            <span ref-type="bibr" rid="pntd.0002349-Nasci1">[53]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Turell2">[54]</span>. Dilutions of mosquito pools containing a single mosquito with a known disseminated
            infection with WNV, SINV, or RVFV and 24 uninfected mosquitoes were made to mimic
            the virus titer variations. The LLODs were determined for viral RNA present in pools
            of laboratory infected mosquitoes and were compared to modified qPCR and conventional
            PCR assays. The microarray assay was expected to be less sensitive than the PCR assays
            because the microarray has an additional detection bottleneck at the point of probe
            hybridization. Yet the microarray LLODs compared to the PCR assays were at most only
            a log greater, making it suitable by comparison for diagnostic use. If greater sensitivities
            are desired, longer hybridization times (data not shown) and post-hybridization signal
            amplification 
            <span ref-type="bibr" rid="pntd.0002349-Wojciechowski1">[37]</span> would help increase detection of low virus titers. However, the methods described
            here were optimized to reduce the assay time while maintaining a sensitivity level
            adequate for analyzing field-collected samples. This was evaluated in Thailand in
            the springs of 2011 and 2012 using various numbers of pooled mosquitoes. From the
            field collections, RNA from medically relevant viruses, including DENV-3 and JEV,
            and TMUV, a virus known to cause severe disease in ducks 
            <span ref-type="bibr" rid="pntd.0002349-Tang1">[55]</span>, was identified. Viral RNA from two arthropod-specific flaviviruses, CxFV and a QBV-like
            virus, were also identified, though RNA isolated from three other flavivirus PCR positive
            pools could not be identified. After passage of the PCR positive samples in cultured
            cells, QBV RNA was identified using the microarray, suggesting that the level of QBV
            RNA in the mosquito pools was below the microarray's LLOD. Bolling 
            <i>et al.</i> noticed a bimodal distribution of CxFV in a naturally infected 
            <i>Cx. pipiens</i> laboratory colony 
            <span ref-type="bibr" rid="pntd.0002349-Bolling2">[56]</span> which, along with the other factors mentioned above for varied titers, may explain
            why some arthropod-specific flaviviruses from field samples were below microarray's
            LLOD.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The intended use of the ArboChip5.1 is for arbovirus surveillance studies targeting
            mosquitoes. The concept is to use generic screening assays to identify mosquito pools
            potentially containing arboviral RNA. The screening assays should be based on the
            field or laboratory team's capabilities and can include the conventional and real-time
            assays cited in this paper or other published assays 
            <span ref-type="bibr" rid="pntd.0002349-Eshoo1">[36]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Chao1">[40]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Drosten1">[41]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Kuno1">[42]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Lambert1">[43]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Kuno2">[44]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Grywna1">[57]</span>, 
            <span ref-type="bibr" rid="pntd.0002349-Yang1">[58]</span>. Viral RNA positive mosquito pools would then be PCR amplified using the GSPs for
            the genus (or genera) of interest and analyzed using the microarray platform. These
            PCR assays should not be used alone for screening mosquito pools for arbovirus RNA
            because 1) asymmetric PCR amplification can reduce sensitivity, 2) biotin would be
            wasted on many negative pools, increasing the costs of surveillance and 3) the degenerate
            GSP sets were designed to increase assay sensitivity, but at the cost of specificity,
            meaning additional non-specific bands are produced which can confound interpretation
            of the results. The microarray assay could be used to analyze viral RNA isolated from
            animal tissue or serum, or from sand flies, because many phleboviruses are included
            in the design. However, viral RNA isolated from these sources has not been evaluated
            using the microarray and associated methods.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Multiplexed-PCR methods for viral RNA identification can be efficient for a small
            sub-set of viruses, yet when there is a need to discern between a larger set of mosquito-borne
            viruses, PCR approaches for virus identification can become laborious. The microarray,
            as described here, is a time and cost effective method for the detection of numerous
            viral RNA species potentially present within an infected mosquito with limited bias.
            Each ElectraSense 4×2K microarray has four wells to analyze four samples at a time,
            and when using the rotisserie hybridization incubator described here, holding four
            microarray chips at a time, up to 16 samples can be processed in as little as 12 hours.
            The major limitations of this assay are that only known viral sequences can be used
            for probe design, meaning that it cannot identify novel viruses, and it cannot discern
            between slight nucleotide changes, meaning that it cannot anticipate viral evolution.
            Even though previously unknown or non-targeted viral RNA maybe detected because of
            some sequence homology in one of the targeted regions, the microarray would not be
            able to identify novel viruses. On the other hand, next-generation sequencing (NGS)
            has the potential to provide the most amount of nucleic acid information, but they
            are not portable and require more time and labor to create sequencing libraries and
            analyze the results. Moreover, NGS systems are much more expensive to operate, generally
            costing over $1000 per run 
            <span ref-type="bibr" rid="pntd.0002349-Liu1">[59]</span>. To compare, ElectraSense 4×2K microarray chips costs approximately $500 each and
            can be used four to five times, thus costing $25 to $31.25 per sample.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In summary, the ArboChip5.1 microarray can identify multiple genes from a wide range
            of mosquito-borne RNA viruses through broad-range PCR amplification and detection
            with virus-specific probes. It is vastly more multiplexed than PCR assays alone, more
            specific than universal microarrays, and more cost effective than most sequencing
            platforms. The microarray reader is small and rugged, making it field-portable. This
            system is an ideal tool for active surveillance or monitoring programs in regions
            where little is known about the circulating mosquito-borne viruses. We have demonstrated
            that it detects many viruses of medical importance, such as DENV, YFV, JEV, and CHIKV.
            In addition, the microarray targets many viruses that are also major causes of animal
            disease, such as RVFV, VEEV, and WNV. The ultimate goal is to provide researchers,
            veterinarians, and clinicians with a diagnostic tool that will allow them to recognize
            previously known pathogens that are emerging or re-emerging before they become major
            health issues.
         </p>
      </div>
      <div id="s5">
         <h2>Supporting Information</h2>
         <div tagxxx="supplementary-material">
            <div tagxxx="label">Dataset S1</div>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>Example data analysis.</b> Data analysis worksheet with example data from the field collected mosquito pool
                  Th9-0122 (
                  <span ref-type="fig" rid="pntd-0002349-g006">Figure 6</span>, 
                  <span ref-type="table" rid="pntd-0002349-t008">Table 8</span>). Individual ECD signals and calculated z-scores are shown for each probe. The data
                  summary shows the average z-scores and maximum z-scores from each subgroup. Visual
                  analysis of the subgroups is provided for each group (virus genus and gene target).
               </p>
               <p>(XLSX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <div tagxxx="label">Figure S1</div>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>Detection of JEV and WNVL1 from a dual-infected pool of </b>
                  <b>
                     <i>Cx. pipiens</i>
                  </b>
                  <b> mosquitoes.</b> A mosquito pool (n = 25) containing two infected 
                  <i>Cx. pipiens</i>, one each with JEV and WNVL1, and 23 uninfected mosquitoes were processed together
                  and PCR amplified using the 
                  <i>Flavivirus</i> GSP sets. The NS3 PCR amplicons hybridized with probes in the JEV-specific, WNVL1/KUNV,
                  and WNVL1-specific subgroups. The JEV and WNVL1 probes have not been observed to cross-hybridize
                  to the other virus when analyzed individually, thus showing the detection of RNA from
                  two related but distinct flaviviruses in a single mosquito pool. The virus abbreviations
                  are defined in 
                  <span ref-type="supplementary-material" rid="pntd.0002349.s003">Table S1</span>.
               </p>
               <p>(TIF)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <div tagxxx="label">Table S1</div>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>Target viruses.</b> List of the ArboChip5.1 targeted mosquito-borne RNA viruses, sorted by genera and
                  clades (as determined by phylogenetic comparisons of the gene target sequences and
                  not necessarily represented by taxonomic clades), with the number of virus-specific
                  and non-specific probes.
               </p>
               <p>(XLSX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <div tagxxx="label">Table S2</div>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>Probe list.</b> List of the ArboChip5.1 oligonucleotide probes created for the multi-gene detection
                  of RNA from mosquito-borne viruses. For each of 2,097 probes, the probe name, group
                  (genus and gene target), subgroup (clade and virus), sequence source (GenBank accession
                  number), start position, design source, and sequence is listed.
               </p>
               <p>(XLSX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
      </div>
      <div tagxxx="back">
         <div tagxxx="ack">
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We thank Sarah Pisarcik, Jennifer Groebner, Collin Fitzpatrick, Susana Padilla, and
               Steven Barnes, Virology Division, USAMRIID, for laboratory support, Cindy Rossi, Diagnostic
               Systems Division, USAMRIID, for providing viruses, Dr. Leonard Wasieloski, Diagnostic
               Systems Division, USAMRIID, for probe design and data analysis counselling, Dr. Brian
               Evans, Boonsong Jaichapor, Prasan Kankaew, Dr. Alongkot Ponlawat, Ampornpan Kengluecha,
               and Arissara Pomgsiri, Department of Entomology, AFRIMS, Thailand, for field support,
               and Dr. Dacri Smith and Douglas Brackney, College of Veterinary Medicine and Biological
               Sciences, Colorado State University, for their critical review of the manuscript.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The mention of trade names or commercial products does not constitute endorsement
               or recommendation for use by the Department of the Army or the Department of Defense.
               The opinions and assertions contained herein are those of the authors and are not
               to be construed as official or reflecting the views of the Department of the Army
               or the Department of Defense.
            </p>
         </div>
         <div tag="ref-list">
            <ul>
               <div tagxxx="title">References</div>
               <li tag="ref">
                  <div tagxxx="label">1</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Weaver</span>
                        <span tagx="given-names">SC</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Reisen</span>
                        <span tagx="given-names">WK</span>
                     </div> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">Present and future arboviral threats</span>. 
                     <span tagx="source">Antiviral Res</span>
                     <span tagx="volume">82</span>: 
                     <span tagx="fpage">328</span>
                     <div tagxxx="pub-id">19857523</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">2</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Lindenbach</span>
                        <span tagx="given-names">BD</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Rice</span>
                        <span tagx="given-names">CM</span>
                     </div> (
                     <span tagx="year">2003</span>) 
                     <span tagx="article-title">Molecular biology of flaviviruses</span>. 
                     <span tagx="source">Adv Virus Res</span>
                     <span tagx="volume">59</span>: 
                     <span tagx="fpage">23</span>–
                     <span tagx="lpage">61</span>
                     <div tagxxx="pub-id">14696326</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">3</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Stollar</span>
                        <span tagx="given-names">V</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Thomas</span>
                        <span tagx="given-names">VL</span>
                     </div> (
                     <span tagx="year">1975</span>) 
                     <span tagx="article-title">An agent in the 
                        <i>Aedes aegypti</i> cell line (Peleg) which causes fusion of 
                        <i>Aedes albopictus</i> cells
                     </span>. 
                     <span tagx="source">Virology</span>
                     <span tagx="volume">64</span>: 
                     <span tagx="fpage">367</span>–
                     <span tagx="lpage">377</span>
                     <div tagxxx="pub-id">806166</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">4</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Crabtree</span>
                        <span tagx="given-names">MB</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Sang</span>
                        <span tagx="given-names">RC</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Stollar</span>
                        <span tagx="given-names">V</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Dunster</span>
                        <span tagx="given-names">LM</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Miller</span>
                        <span tagx="given-names">BR</span>
                     </div> (
                     <span tagx="year">2003</span>) 
                     <span tagx="article-title">Genetic and phemotypic characterization of the newly described insect flavivirus,
                        Kamiti River virus
                     </span>. 
                     <span tagx="source">Arch Virol</span>
                     <span tagx="volume">148</span>: 
                     <span tagx="fpage">1095</span>–
                     <span tagx="lpage">118</span>
                     <div tagxxx="pub-id">12756617</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">5</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Sang</span>
                        <span tagx="given-names">RC</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Gichogo</span>
                        <span tagx="given-names">A</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Gachoya</span>
                        <span tagx="given-names">J</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Dunster</span>
                        <span tagx="given-names">MD</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Ofula</span>
                        <span tagx="given-names">V</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2003</span>) 
                     <span tagx="article-title">Isolation of a new flavivirus related to cell fusing agent virus (CFAV) from field-collected
                        flood-water 
                        <i>Aedes</i> mosquitoes sampled from a dambo in central Kenya
                     </span>. 
                     <span tagx="source">Arch Virol</span>
                     <span tagx="volume">148</span>: 
                     <span tagx="fpage">1085</span>–
                     <span tagx="lpage">1093</span>
                     <div tagxxx="pub-id">12756616</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">6</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Cook</span>
                        <span tagx="given-names">S</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Bennet</span>
                        <span tagx="given-names">SN</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">EC</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">De Chesse</span>
                        <span tagx="given-names">R</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Moureau</span>
                        <span tagx="given-names">G</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">Isolation of a new strain of the flavivirus cell fusing agent virus in a natural mosquito
                        population from Puerto Rico
                     </span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">87</span>: 
                     <span tagx="fpage">735</span>–
                     <span tagx="lpage">748</span>
                     <div tagxxx="pub-id">16528021</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">7</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Hoshino</span>
                        <span tagx="given-names">K</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Isawa</span>
                        <span tagx="given-names">H</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Tsuda</span>
                        <span tagx="given-names">Y</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Yano</span>
                        <span tagx="given-names">K</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Sasaki</span>
                        <span tagx="given-names">T</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">Genetic characterization of a new insect flavivirus isolated from 
                        <i>Culex pipiens</i> mosquito in Japan
                     </span>. 
                     <span tagx="source">Virology</span>
                     <span tagx="volume">359</span>: 
                     <span tagx="fpage">405</span>–
                     <span tagx="lpage">414</span>
                     <div tagxxx="pub-id">17070886</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">8</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Crabtree</span>
                        <span tagx="given-names">MB</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Nga</span>
                        <span tagx="given-names">PT</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Miller</span>
                        <span tagx="given-names">BR</span>
                     </div> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Isolation and characterization of a new mosquito flavivirus, Quang Binh virus, from
                        Vietnam
                     </span>. 
                     <span tagx="source">Arch Virol</span>
                     <span tagx="volume">154</span>: 
                     <span tagx="fpage">857</span>–
                     <span tagx="lpage">860</span>
                     <div tagxxx="pub-id">19347244</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">9</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Bolling</span>
                        <span tagx="given-names">BG</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Eisen</span>
                        <span tagx="given-names">L</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Moore</span>
                        <span tagx="given-names">CG</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Blair</span>
                        <span tagx="given-names">CD</span>
                     </div> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">Insect-specific flaviviruses from Culex mosquitoes in Colorado, with evidence of vertical
                        transmission
                     </span>. 
                     <span tagx="source">Am J Trop Med Hyg</span>
                     <span tagx="volume">85</span>: 
                     <span tagx="fpage">169</span>–
                     <span tagx="lpage">177</span>
                     <div tagxxx="pub-id">21734144</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">10</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Strauss</span>
                        <span tagx="given-names">JH</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Strauss</span>
                        <span tagx="given-names">EG</span>
                     </div> (
                     <span tagx="year">1994</span>) 
                     <span tagx="article-title">The alphaviruses: gene expression, replication, and evolution</span>. 
                     <span tagx="source">Microbiol Rev</span>
                     <span tagx="volume">58</span>: 
                     <span tagx="fpage">491</span>–
                     <span tagx="lpage">562</span>
                     <div tagxxx="pub-id">7968923</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">11</div>
                  <div tagxxx="mixed-citation">Elliott RM, Bouloy M, Calisher CH, Goldbach R, Moyer JT, 
                     <div tagxxx="etal">et al</div>. (2000) Family Bunyaviridae. In: van Regenmortel MHV, Fauguet CM, Bishop DHL editors.
                     Virus Taxonomy: Seventh Report of the International Committee on Taxonomy of Viruses.
                     San Diego: Academic Press. pp. 599–621.
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">12</div>
                  <div tagxxx="mixed-citation">Meegan JM, Bailey CL (1988) Rift Valley fever. In: Monath TP editor. The Arboviruses:
                     Epidemiology and Ecology, Vol. IV. Boca Raton, FL: CRC Press. pp. 61–76.
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">13</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Jones</span>
                        <span tagx="given-names">KE</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Patel</span>
                        <span tagx="given-names">NG</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Levy</span>
                        <span tagx="given-names">MA</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Storeygard</span>
                        <span tagx="given-names">A</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Balk</span>
                        <span tagx="given-names">D</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2008</span>) 
                     <span tagx="article-title">Global trends in emerging infectious diseases</span>. 
                     <span tagx="source">Nature</span>
                     <span tagx="volume">451</span>: 
                     <span tagx="fpage">990</span>–
                     <span tagx="lpage">994</span>
                     <div tagxxx="pub-id">18288193</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">14</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Hollidge</span>
                        <span tagx="given-names">BS</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">González-Scarano</span>
                        <span tagx="given-names">F</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Soldan</span>
                        <span tagx="given-names">SS</span>
                     </div> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">Arboviral encephalitides: transmission, emergence, and pathogenesis</span>. 
                     <span tagx="source">J Neuroimmune Pharmacol</span>
                     <span tagx="volume">5</span>: 
                     <span tagx="fpage">428</span>–
                     <span tagx="lpage">442</span>
                     <div tagxxx="pub-id">20652430</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">15</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Meegan</span>
                        <span tagx="given-names">JM</span>
                     </div> (
                     <span tagx="year">1979</span>) 
                     <span tagx="article-title">Rift Valley fever in Egypt: an overview of the epizootics in 1977 and 1978</span>. 
                     <span tagx="source">Contrib Epidemiol Biostat</span>
                     <span tagx="volume">3</span>: 
                     <span tagx="fpage">100</span>–
                     <span tagx="lpage">113</span>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">16</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Nguku</span>
                        <span tagx="given-names">PM</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Sharif</span>
                        <span tagx="given-names">SK</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Mutonga</span>
                        <span tagx="given-names">D</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Amwayi</span>
                        <span tagx="given-names">S</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Omolo</span>
                        <span tagx="given-names">J</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">An investigation of a major outbreak of Rift Valley fever in Kenya: 2006–2007</span>. 
                     <span tagx="source">Am J Trop Med Hyg</span>
                     <span tagx="volume">88</span>: 
                     <span tagx="fpage">5</span>–
                     <span tagx="lpage">13</span>
                     <div tagxxx="pub-id">20682900</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">17</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Briese</span>
                        <span tagx="given-names">T</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Jia</span>
                        <span tagx="given-names">XY</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Huang</span>
                        <span tagx="given-names">C</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Grady</span>
                        <span tagx="given-names">LJ</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Lipkin</span>
                        <span tagx="given-names">WI</span>
                     </div> (
                     <span tagx="year">1999</span>) 
                     <span tagx="article-title">Identification of a Kunjin/West Nile-like flavivirus in brains of patients with New
                        York encephalitis
                     </span>. 
                     <span tagx="source">Lancet</span>
                     <span tagx="volume">354</span>: 
                     <span tagx="fpage">1650</span>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">18</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Brinton</span>
                        <span tagx="given-names">MA</span>
                     </div> (
                     <span tagx="year">2002</span>) 
                     <span tagx="article-title">The molecular biology of West Nile virus: a new invader of the western hemisphere</span>. 
                     <span tagx="source">Annu Rev Microbiol</span>
                     <span tagx="volume">56</span>: 
                     <span tagx="fpage">371</span>–
                     <span tagx="lpage">402</span>
                     <div tagxxx="pub-id">12142476</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">19</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Kariuki Njenga</span>
                        <span tagx="given-names">M</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Nderitu</span>
                        <span tagx="given-names">L</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Ledermann</span>
                        <span tagx="given-names">JP</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Ndirangu</span>
                        <span tagx="given-names">A</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Logue</span>
                        <span tagx="given-names">CH</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2008</span>) 
                     <span tagx="article-title">Tracking epidemic Chikungunya virus into the Indian Ocean from East Africa</span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">89</span>: 
                     <span tagx="fpage">2754</span>–
                     <span tagx="lpage">2760</span>
                     <div tagxxx="pub-id">18931072</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">20</div>
                  <div tagxxx="mixed-citation">CDC Guidelines for Arbovirus Surveillance in the US, CDC Division of Vector-Borne
                     Infectious Diseases (DVBID). 
                     <div tagxxx="ext-link">http://www.cdc.gov/ncidod/dvbid/Arbor/arboguid.htm</div> (Accessed 05 January 2013).
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">21</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Wang</span>
                        <span tagx="given-names">D</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Coscoy</span>
                        <span tagx="given-names">L</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Zylberberg</span>
                        <span tagx="given-names">M</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Avila</span>
                        <span tagx="given-names">PC</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Boushey</span>
                        <span tagx="given-names">HA</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2002</span>) 
                     <span tagx="article-title">Microarray-based detection and genotyping of viral pathogens</span>. 
                     <span tagx="source">PNAS</span>
                     <span tagx="volume">99</span>: 
                     <span tagx="fpage">15687</span>–
                     <span tagx="lpage">15692</span>
                     <div tagxxx="pub-id">12429852</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">22</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Wang</span>
                        <span tagx="given-names">D</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Urisman</span>
                        <span tagx="given-names">A</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Liu</span>
                        <span tagx="given-names">YT</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Springer</span>
                        <span tagx="given-names">M</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Ksiazek</span>
                        <span tagx="given-names">TG</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2003</span>) 
                     <span tagx="article-title">Viral discovery an sequence recovery using DNA microarrays</span>. 
                     <span tagx="source">PLoS Bio</span>
                     <span tagx="volume">1</span>: 
                     <span tagx="fpage">257</span>–
                     <span tagx="lpage">260</span>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">23</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Palacios</span>
                        <span tagx="given-names">G</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Quan</span>
                        <span tagx="given-names">PL</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Hirschberg</span>
                        <span tagx="given-names">DL</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Liu</span>
                        <span tagx="given-names">Y</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Zhai</span>
                        <span tagx="given-names">J</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">Panmicrobial oligonucleotide array for diagnosis of infectious diseases</span>. 
                     <span tagx="source">EID</span>
                     <span tagx="volume">13</span>: 
                     <span tagx="fpage">73</span>–
                     <span tagx="lpage">81</span>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">24</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Korimbocus</span>
                        <span tagx="given-names">J</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Scaramozzino</span>
                        <span tagx="given-names">N</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Lacroix</span>
                        <span tagx="given-names">B</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Crance</span>
                        <span tagx="given-names">JM</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Garin</span>
                        <span tagx="given-names">D</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2005</span>) 
                     <span tagx="article-title">DNA probe array for the simultaneous identification of herpesviruses, enteroviruses,
                        and flaviviruses
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">43</span>: 
                     <span tagx="fpage">3779</span>–
                     <span tagx="lpage">3787</span>
                     <div tagxxx="pub-id">16081910</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">25</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Nordstrom</span>
                        <span tagx="given-names">H</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Falk</span>
                        <span tagx="given-names">KI</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Lindegren</span>
                        <span tagx="given-names">G</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Mouzavi-Jazi</span>
                        <span tagx="given-names">M</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Walden</span>
                        <span tagx="given-names">A</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2005</span>) 
                     <span tagx="article-title">DNA microarray technique for detection and identification of seven flaviviruses pathogenic
                        for man
                     </span>. 
                     <span tagx="source">J Med Virol</span>
                     <span tagx="volume">77</span>: 
                     <span tagx="fpage">528</span>–
                     <span tagx="lpage">540</span>
                     <div tagxxx="pub-id">16254977</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">26</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Chou</span>
                        <span tagx="given-names">C</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Lee</span>
                        <span tagx="given-names">T</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Chen</span>
                        <span tagx="given-names">C</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Hsiao</span>
                        <span tagx="given-names">H</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Lin</span>
                        <span tagx="given-names">Y</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">Design of microarray probes for virus identification and detection of emerging viruses
                        at the genus level
                     </span>. 
                     <span tagx="source">BMC Bioinfo</span>
                     <span tagx="volume">7</span>: 
                     <span tagx="fpage">232</span>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">27</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Putonti</span>
                        <span tagx="given-names">C</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Chumakov</span>
                        <span tagx="given-names">S</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Mitra</span>
                        <span tagx="given-names">R</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Fox</span>
                        <span tagx="given-names">GE</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Wilson</span>
                        <span tagx="given-names">RC</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">Human-blind probes and primers for dengue virus identification</span>. 
                     <span tagx="source">FEBS J</span>
                     <span tagx="volume">273</span>: 
                     <span tagx="fpage">398</span>–
                     <span tagx="lpage">408</span>
                     <div tagxxx="pub-id">16403026</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">28</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Xiao-Ping</span>
                        <span tagx="given-names">K</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Yong-Qiang</span>
                        <span tagx="given-names">L</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Qing-Ge</span>
                        <span tagx="given-names">S</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Hong</span>
                        <span tagx="given-names">L</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Quing-Yu</span>
                        <span tagx="given-names">Z</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Development of a consensus microarray method for identification of some highly pathogenic
                        viruses
                     </span>. 
                     <span tagx="source">J Med Virol</span>
                     <span tagx="volume">81</span>: 
                     <span tagx="fpage">1945</span>–
                     <span tagx="lpage">1950</span>
                     <div tagxxx="pub-id">19774692</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">29</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Berthet</span>
                        <span tagx="given-names">N</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Paulous</span>
                        <span tagx="given-names">S</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Coffey</span>
                        <span tagx="given-names">LL</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Frenkiel</span>
                        <span tagx="given-names">MP</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Moltini</span>
                        <span tagx="given-names">I</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Resequencing microarray method for molecular diagnosis of human arboviral diseases</span>. 
                     <span tagx="source">J Clin Virol</span>
                     <span tagx="volume">56</span>: 
                     <span tagx="fpage">322</span>–
                     <span tagx="lpage">327</span>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">30</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Grubaugh</span>
                        <span tagx="given-names">ND</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Petz</span>
                        <span tagx="given-names">LN</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Melanson</span>
                        <span tagx="given-names">VR</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">McMenamy</span>
                        <span tagx="given-names">SS</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Turell</span>
                        <span tagx="given-names">MJ</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Evaluation of a field-portable DNA microarray platform and nucleic acid amplification
                        strategies for the detection of arboviruses, arthropods, and bloodmeals
                     </span>. 
                     <span tagx="source">Am J Trop Med Hyg</span> [Epub ahead of print].
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">31</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Roth</span>
                        <span tagx="given-names">KM</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Peyvan</span>
                        <span tagx="given-names">K</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Schwarzkopf</span>
                        <span tagx="given-names">KR</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Ghindilis</span>
                        <span tagx="given-names">AL</span>
                     </div> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">Elecetrochemical detection of short DNA oligomer hybridization using the CombiMatrix
                        microarray reader
                     </span>. 
                     <span tagx="source">Electroanalysis</span>
                     <span tagx="volume">18</span>: 
                     <span tagx="fpage">1982</span>–
                     <span tagx="lpage">1988</span>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">32</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Ghindilis</span>
                        <span tagx="given-names">AL</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Smith</span>
                        <span tagx="given-names">MW</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Schwarzkopf</span>
                        <span tagx="given-names">KR</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Roth</span>
                        <span tagx="given-names">KM</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Peyvan</span>
                        <span tagx="given-names">K</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">CombiMatrix oligonucleotide arrays: genotyping and gene expression assays employing
                        electrochemical detection
                     </span>. 
                     <span tagx="source">Biosens Bioelectron</span>
                     <span tagx="volume">22</span>: 
                     <span tagx="fpage">1853</span>–
                     <span tagx="lpage">1860</span>
                     <div tagxxx="pub-id">16891109</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">33</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Lodes</span>
                        <span tagx="given-names">MJ</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Suciu</span>
                        <span tagx="given-names">D</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Elliot</span>
                        <span tagx="given-names">M</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Stover</span>
                        <span tagx="given-names">AG</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Ross</span>
                        <span tagx="given-names">M</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">Use of a semiconductor-based oligonucleotide microarray for Influenza A virus subtype
                        identification and sequencing
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">44</span>: 
                     <span tagx="fpage">1209</span>–
                     <span tagx="lpage">1218</span>
                     <div tagxxx="pub-id">16597840</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">34</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Lodes</span>
                        <span tagx="given-names">MJ</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Suciu</span>
                        <span tagx="given-names">D</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Willmoth</span>
                        <span tagx="given-names">JL</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Ross</span>
                        <span tagx="given-names">M</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Munro</span>
                        <span tagx="given-names">S</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">Identification of upper respiratory tract pathogens using electrochemical detection
                        on an oligonucleotide microarray
                     </span>. 
                     <span tagx="source">PLoS One</span>
                     <span tagx="volume">2</span>: 
                     <span tagx="fpage">e924</span>
                     <div tagxxx="pub-id">17895966</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">35</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Maher-Sturgess</span>
                        <span tagx="given-names">SL</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Forrester</span>
                        <span tagx="given-names">NL</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Wayper</span>
                        <span tagx="given-names">PJ</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Hall</span>
                        <span tagx="given-names">RA</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2008</span>) 
                     <span tagx="article-title">Universal primers that amplify RNA from all three flavivirus subgroups</span>. 
                     <span tagx="source">Virol J</span>
                     <span tagx="volume">5</span>: 
                     <span tagx="fpage">16</span>
                     <div tagxxx="pub-id">18218114</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">36</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Eshoo</span>
                        <span tagx="given-names">MW</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Whitehouse</span>
                        <span tagx="given-names">CA</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Zoll</span>
                        <span tagx="given-names">ST</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Massire</span>
                        <span tagx="given-names">C</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Pennella</span>
                        <span tagx="given-names">TD</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">Direct broad-range detection of alphaviruses in mosquito extracts</span>. 
                     <span tagx="source">Virology</span>
                     <span tagx="volume">368</span>: 
                     <span tagx="fpage">286</span>–
                     <span tagx="lpage">295</span>
                     <div tagxxx="pub-id">17655905</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">37</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Wojciechowski</span>
                        <span tagx="given-names">J</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Chase-Baldwin</span>
                        <span tagx="given-names">K</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Wasieloski</span>
                        <span tagx="given-names">LP</span>
                        <div tagxxx="suffix">Jr</div>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Padilla</span>
                        <span tagx="given-names">S</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Vora</span>
                        <span tagx="given-names">GJ</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">Enhancement of deoxyribonucleic acid microarray performance using post-hybridization
                        signal amplification. Anal. Chim
                     </span>. 
                     <span tagx="source">Acta</span>
                     <span tagx="volume">679</span>: 
                     <span tagx="fpage">85</span>–
                     <span tagx="lpage">90</span>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">38</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Rosen</span>
                        <span tagx="given-names">L</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Gubler</span>
                        <span tagx="given-names">D</span>
                     </div> (
                     <span tagx="year">1974</span>) 
                     <span tagx="article-title">The use of mosquitoes to detect and propagate dengue viruses</span>. 
                     <span tagx="source">Am J Trop Med Hyg</span>
                     <span tagx="volume">23</span>: 
                     <span tagx="fpage">1153</span>–
                     <span tagx="lpage">1160</span>
                     <div tagxxx="pub-id">4429185</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">39</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">O'Guinn</span>
                        <span tagx="given-names">ML</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Lee</span>
                        <span tagx="given-names">JS</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Kondig</span>
                        <span tagx="given-names">JP</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Fernandez</span>
                        <span tagx="given-names">R</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Carbajal</span>
                        <span tagx="given-names">F</span>
                     </div> (
                     <span tagx="year">2004</span>) 
                     <span tagx="article-title">Field detection of Eastern equine encephalitis virus in the Amazon Basin region of
                        Peru using reverse transcription-polymerase chain reaction adapted for field identification
                        of arthropod-borne pathogens
                     </span>. 
                     <span tagx="source">Am J Trop Med Hyg</span>
                     <span tagx="volume">70</span>: 
                     <span tagx="fpage">164</span>–
                     <span tagx="lpage">171</span>
                     <div tagxxx="pub-id">14993628</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">40</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Chao</span>
                        <span tagx="given-names">D</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Davis</span>
                        <span tagx="given-names">BS</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Chang</span>
                        <span tagx="given-names">GJ</span>
                     </div> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">Development of multiplex real-time reverse transcriptase PCR assays for detecting
                        eight medically important flaviviruses in mosquitoes
                     </span>. 
                     <span tagx="source">J Clin Micro</span>
                     <span tagx="volume">45</span>: 
                     <span tagx="fpage">584</span>–
                     <span tagx="lpage">589</span>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">41</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Drosten</span>
                        <span tagx="given-names">C</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Göttig</span>
                        <span tagx="given-names">S</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Schilling</span>
                        <span tagx="given-names">S</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Asper</span>
                        <span tagx="given-names">M</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Panning</span>
                        <span tagx="given-names">M</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2002</span>) 
                     <span tagx="article-title">Rapid detection and quantification of RNA of Ebola and Marbug viruses, Lassa virus,
                        Crimean-Congo hemorrhagic fever virus, Rift Valley fever virus, dengue virus, and
                        yellow fever virus by real-time reverse transcription-PCR
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">40</span>: 
                     <span tagx="fpage">2323</span>–
                     <span tagx="lpage">2330</span>
                     <div tagxxx="pub-id">12089242</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">42</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Kuno</span>
                        <span tagx="given-names">G</span>
                     </div> (
                     <span tagx="year">1998</span>) 
                     <span tagx="article-title">Universal diagnostic RT-PCR protocol for arboviruses</span>. 
                     <span tagx="source">J Virol Meth</span>
                     <span tagx="volume">72</span>: 
                     <span tagx="fpage">27</span>–
                     <span tagx="lpage">41</span>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">43</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Lambert</span>
                        <span tagx="given-names">AJ</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Lanciotti</span>
                        <span tagx="given-names">RS</span>
                     </div> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Consensus amplification and novel multiplex sequencing method for S segment species
                        identification of 47 viruses of the Orthobunyavirus, Phlebovirus, and Nairovirus genera
                        of the family Bunyaviridae
                     </span>. 
                     <span tagx="source">J Clin Micro</span>
                     <span tagx="volume">47</span>: 
                     <span tagx="fpage">2398</span>–
                     <span tagx="lpage">2404</span>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">44</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Kuno</span>
                        <span tagx="given-names">G</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Mitchel</span>
                        <span tagx="given-names">C</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Chang</span>
                        <span tagx="given-names">GJ</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Smith</span>
                        <span tagx="given-names">G</span>
                     </div> (
                     <span tagx="year">1996</span>) 
                     <span tagx="article-title">Detecting bunyaviruses of the Bunyamwera and California serogroups by a PCR technique</span>. 
                     <span tagx="source">J Clin Micro</span>
                     <span tagx="volume">34</span>: 
                     <span tagx="fpage">1184</span>–
                     <span tagx="lpage">1188</span>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">45</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Lambert</span>
                        <span tagx="given-names">AJ</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Lanciotti</span>
                        <span tagx="given-names">RS</span>
                     </div> (
                     <span tagx="year">2008</span>) 
                     <span tagx="article-title">Molecular characterization of medically important viruses of the genus Orthobunyavirus</span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">89</span>: 
                     <span tagx="fpage">2580</span>–
                     <span tagx="lpage">2585</span>
                     <div tagxxx="pub-id">18796727</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">46</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Mores</span>
                        <span tagx="given-names">CN</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Turell</span>
                        <span tagx="given-names">MJ</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Dyer</span>
                        <span tagx="given-names">J</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Rossi</span>
                        <span tagx="given-names">CA</span>
                     </div> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Phylogenetic relationships among orthobunyaviruses isolated from mosquitoes captured
                        in Peru
                     </span>. 
                     <span tagx="source">Vector Borne Zoonotic Dis</span>
                     <span tagx="volume">9</span>: 
                     <span tagx="fpage">25</span>–
                     <span tagx="lpage">32</span>
                     <div tagxxx="pub-id">18759638</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">47</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Chowdhary</span>
                        <span tagx="given-names">R</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Street</span>
                        <span tagx="given-names">C</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Travassos da Rossa</span>
                        <span tagx="given-names">A</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Nunes</span>
                        <span tagx="given-names">MR</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Tee</span>
                        <span tagx="given-names">KK</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Genetic characterization of the Wyeomyia group of orthobunyaviruses and their phylogenetic
                        relationships
                     </span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">93</span>: 
                     <span tagx="fpage">1023</span>–
                     <span tagx="lpage">1034</span>
                     <div tagxxx="pub-id">22278828</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">48</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Hang</span>
                        <span tagx="given-names">J</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Forshey</span>
                        <span tagx="given-names">BM</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Kochel</span>
                        <span tagx="given-names">TJ</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Li</span>
                        <span tagx="given-names">T</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Solórzano</span>
                        <span tagx="given-names">VF</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Random amplification and pyrosequencing for identification of novel viral genome sequences</span>. 
                     <span tagx="source">J Biomol Tech</span>
                     <span tagx="volume">23</span>: 
                     <span tagx="fpage">4</span>–
                     <span tagx="lpage">10</span>
                     <div tagxxx="pub-id">22468136</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">49</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Turell</span>
                        <span tagx="given-names">MJ</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Saluzzo</span>
                        <span tagx="given-names">J-F</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Tammariello</span>
                        <span tagx="given-names">RF</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Smith</span>
                        <span tagx="given-names">JF</span>
                     </div> (
                     <span tagx="year">1990</span>) 
                     <span tagx="article-title">Generation and transmission of Rift Valley fever viral reassortants by the mosquito
                        
                        <i>Culex pipiens</i>
                     </span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">71</span>: 
                     <span tagx="fpage">2307</span>–
                     <span tagx="lpage">2312</span>
                     <div tagxxx="pub-id">2230736</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">50</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Briese</span>
                        <span tagx="given-names">T</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Kapoor</span>
                        <span tagx="given-names">V</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Lipkin</span>
                        <span tagx="given-names">WI</span>
                     </div> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">Natural M-segment reassortment in Potosi and Main Drain viruses: implications for
                        the evolution of orthobunyaviruses
                     </span>. 
                     <span tagx="source">Arch Virol</span>
                     <span tagx="volume">152</span>: 
                     <span tagx="fpage">2237</span>–
                     <span tagx="lpage">2247</span>
                     <div tagxxx="pub-id">17891328</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">51</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Briese</span>
                        <span tagx="given-names">T</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Bird</span>
                        <span tagx="given-names">B</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Kapoor</span>
                        <span tagx="given-names">V</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Nichol</span>
                        <span tagx="given-names">ST</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Lipkin</span>
                        <span tagx="given-names">WI</span>
                     </div> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">Batai and Ngari viruses: M segment reassortment and association with severe febrile
                        disease outbreaks in East Africia
                     </span>. 
                     <span tagx="source">J Virol</span>
                     <span tagx="volume">80</span>: 
                     <span tagx="fpage">5627</span>–
                     <span tagx="lpage">5630</span>
                     <div tagxxx="pub-id">16699043</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">52</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Yanase</span>
                        <span tagx="given-names">T</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Kato</span>
                        <span tagx="given-names">T</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Yamakawa</span>
                        <span tagx="given-names">M</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Takayoshi</span>
                        <span tagx="given-names">K</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Nakamura</span>
                        <span tagx="given-names">K</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">Genetic characterization of Batai virus indicates a genomic reassortment between orthobunyaviruses
                        in nature
                     </span>. 
                     <span tagx="source">Arch Virol</span>
                     <span tagx="volume">151</span>: 
                     <span tagx="fpage">2253</span>–
                     <span tagx="lpage">2260</span>
                     <div tagxxx="pub-id">16820982</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">53</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Nasci</span>
                        <span tagx="given-names">RS</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Mitchell</span>
                        <span tagx="given-names">CJ</span>
                     </div> (
                     <span tagx="year">1996</span>) 
                     <span tagx="article-title">Arbovirus titer variation in field-collected mosquitoes</span>. 
                     <span tagx="source">J Am Mosq Control Assoc</span>
                     <span tagx="volume">12</span>: 
                     <span tagx="fpage">167</span>–
                     <span tagx="lpage">171</span>
                     <div tagxxx="pub-id">8827588</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">54</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Turell</span>
                        <span tagx="given-names">MJ</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Gargan</span>
                        <span tagx="given-names">TPII</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Bailey</span>
                        <span tagx="given-names">CL</span>
                     </div> (
                     <span tagx="year">1984</span>) 
                     <span tagx="article-title">Dissemination and replication of Rift Valley fever virus in 
                        <i>Culex pipiens</i>
                     </span>. 
                     <span tagx="source">Am J Trop Med Hyg</span>
                     <span tagx="volume">33</span>: 
                     <span tagx="fpage">176</span>–
                     <span tagx="lpage">181</span>
                     <div tagxxx="pub-id">6696176</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">55</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Tang</span>
                        <span tagx="given-names">Y</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Diao</span>
                        <span tagx="given-names">Y</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Gao</span>
                        <span tagx="given-names">X</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Yu</span>
                        <span tagx="given-names">C</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Chen</span>
                        <span tagx="given-names">L</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Analysis of the complete genome of Tembusu virus, a flavivirus isolate from ducks
                        in China
                     </span>. 
                     <span tagx="source">Transbound Emerg Dis</span>
                     <span tagx="volume">59</span>: 
                     <span tagx="fpage">336</span>–
                     <span tagx="lpage">343</span>
                     <div tagxxx="pub-id">22103739</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">56</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Bolling</span>
                        <span tagx="given-names">BG</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Olea-Popelka</span>
                        <span tagx="given-names">FJ</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Eisen</span>
                        <span tagx="given-names">L</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Moore</span>
                        <span tagx="given-names">CG</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Blair</span>
                        <span tagx="given-names">CD</span>
                     </div> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Transmission dynamics of an insect-specific flavivirus in a naturally infected 
                        <i>Culex pipiens</i> laboratory colony and effects of co-infection on vector competence for West Nile
                        virus
                     </span>. 
                     <span tagx="source">Virology</span>
                     <span tagx="volume">427</span>: 
                     <span tagx="fpage">90</span>–
                     <span tagx="lpage">97</span>
                     <div tagxxx="pub-id">22425062</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">57</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Grywna</span>
                        <span tagx="given-names">K</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Kupfer</span>
                        <span tagx="given-names">B</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Panning</span>
                        <span tagx="given-names">M</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Drexler</span>
                        <span tagx="given-names">JF</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Emmerich</span>
                        <span tagx="given-names">P</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">Detection of all species of the genus Alphavirus by reverse transcription-PCR with
                        diagnostic sensitivity
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">48</span>: 
                     <span tagx="fpage">2286</span>–
                     <span tagx="lpage">3387</span>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">58</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Yang</span>
                        <span tagx="given-names">CF</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Chen</span>
                        <span tagx="given-names">CF</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Su</span>
                        <span tagx="given-names">CL</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Teng</span>
                        <span tagx="given-names">HJ</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Lu</span>
                        <span tagx="given-names">LC</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">Screening of mosquitoes using SYBR Green I-based real-time RT-PCR with group-specific
                        primers for detection of Flaviviruses and Alphaviruses in Taiwan
                     </span>. 
                     <span tagx="source">J Virol Methods</span>
                     <span tagx="volume">168</span>: 
                     <span tagx="fpage">147</span>–
                     <span tagx="lpage">151</span>
                     <div tagxxx="pub-id">20471427</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">59</div>
                  <div tagxxx="mixed-citation">
                     <div tagxxx="name">
                        <span tagx="surname">Liu</span>
                        <span tagx="given-names">L</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Li</span>
                        <span tagx="given-names">Y</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Hu</span>
                        <span tagx="given-names">N</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">He</span>
                        <span tagx="given-names">Y</span>
                     </div>, 
                     <div tagxxx="name">
                        <span tagx="surname">Pong</span>
                        <span tagx="given-names">R</span>
                     </div>, 
                     <div tagxxx="etal">et al</div> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Comparison of next-generation sequencing systems</span>. 
                     <span tagx="source">J Biomed Biotechnol</span>
                     <span tagx="volume">2012</span>: 
                     <span tagx="fpage">251364</span>
                     <div tagxxx="pub-id">22829749</div>
                  </div>
               </li>
            </ul>
         </div>
      </div>
   </div>
</html>