<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <div xmlns="" tagx="article">
      <head>
         <style type="text/css"> table, tr, td {border : thin;}</style>
         <span class="citation_publisher">Nature Publishing Group</span>
         <h2 class="citation-title">Ecophysiology of an uncultivated lineage of Aigarchaeota from an oxic, hot spring
            filamentous ‘streamer' community
         </h2>
         <span class="doi">10.1038/ismej.2015.83</span>
         <div class="contrib-group">contrib: 
            	  CONT
            	  
            <span class="citation_author">Jacob P Beam</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Zackary J Jay</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Markus C Schmid</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Douglas B Rusch</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Margaret F Romine</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Ryan de M Jennings</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Mark A Kozubal</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Susannah G Tringe</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">Michael Wagner</span>
            <span class="citation_author_institution_ref"></span>
            <span class="citation_author">William P Inskeep</span>
            <span class="citation_author_institution_ref"></span>
            	AFF
            	  
            <span class="citation_author_institution">[1 ]</span>
            	AFF
            	  
            <span class="citation_author_institution">[2 ]</span>
            	AFF
            	  
            <span class="citation_author_institution">[3 ]</span>
            	AFF
            	  
            <span class="citation_author_institution">[4 ]</span>
            	AFF
            	  
            <span class="citation_author_institution">[5 ]</span>
            	AFF
            	  
            <span class="citation_author_institution">[6 ]</span>
         </div>
         <span class="email">[binskeep@montana.edu]</span>
         <span class="conflict">[]</span>
         <span class="contribs">[]</span>
         <span class="citation_pubdate">[2015-07-03]</span>
         <span class="citation_copyright">[Copyright 2016: International Society for Microbial Ecology]</span>
         <span class="citation_licence">[Licence This work is licensed under a Creative Commons Attribution-NonCommercial-ShareAlike
            4.0 International License. The images or other third party material in this article
            are included in the article's Creative Commons license, unless indicated otherwise
            in the credit line; if the material is not included under the Creative Commons license,
            users will need to obtain permission from the license holder to reproduce the material.
            To view a copy of this license, visit http://creativecommons.org/licenses/by-nc-sa/4.0/]
         </span>
         <span class="citation_funding">[]</span>
         <span class="citation_pagecount">[]</span>
         <span class="citation_data">[]</span>
      </head>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <p xmlns:mml="http://www.w3.org/1998/Math/MathML">The candidate archaeal phylum ‘Aigarchaeota' contains microorganisms from terrestrial
            and subsurface geothermal ecosystems. The phylogeny and metabolic potential of Aigarchaeota
            has been deduced from several recent single-cell amplified genomes; however, a detailed
            description of their metabolic potential and 
            <i>in situ</i> transcriptional activity is absent. Here, we report a comprehensive metatranscriptome-based
            reconstruction of the 
            <i>in situ</i> metabolism of Aigarchaeota in an oxic, hot spring filamentous ‘streamer' community.
            Fluorescence 
            <i>in situ</i> hybridization showed that these newly discovered Aigarchaeota are filamentous, which
            is consistent with the presence and transcription of an actin-encoding gene. Aigarchaeota
            filaments are intricately associated with other community members, which include both
            bacteria (for example, filamentous 
            <i>Thermocrinis</i> spp.) and archaea. Metabolic reconstruction of genomic and metatranscriptomic data
            suggests that this aigarchaeon is an aerobic, chemoorganoheterotroph with autotrophic
            potential. A heme copper oxidase complex was identified in the environmental genome
            assembly and highly transcribed 
            <i>in situ</i>. Potential electron donors include acetate, fatty acids, amino acids, sugars and
            aromatic compounds, which may originate from extracellular polymeric substances produced
            by other microorganisms shown to exist in close proximity and/or autochthonous dissolved
            organic carbon (OC). Transcripts related to genes specific to each of these potential
            electron donors were identified, indicating that this aigarchaeon likely utilizes
            several OC substrates. Characterized members of this lineage cannot synthesize heme,
            and other cofactors and vitamins 
            <i>de novo</i>, which suggests auxotrophy. We propose the name 
            <i>Candidatus</i> ‘Calditenuis aerorheumensis' for this aigarchaeon, which describes its filamentous
            morphology and its primary electron acceptor, oxygen.
         </p>
      </div>
      <div id="">
         <h2>Introduction</h2>
         <p xmlns:mml="http://www.w3.org/1998/Math/MathML">Members of the candidate archaeal phylum ‘Aigarchaeota' were first identified in a
            subsurface hydrothermal ecosystem (
            <span ref-type="bibr" rid="bib28">
               <sup>
                  <a href="#bib28">Hirayama et al., 2005</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib46">
               <sup>
                  <a href="#bib46">Nunoura et al., 2005</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib47">
               <sup>
                  <a href="#bib47">Nunoura et al., 2011</a>
               </sup>
            </span>) and recently in a terrestrial hot spring as part of the Genome Encyclopedia of Bacteria
            and Archaea (GEBA) project utilizing single-cell genomics (
            <span ref-type="bibr" rid="bib52">
               <sup>
                  <a href="#bib52">Rinke et al., 2013</a>
               </sup>
            </span>). Aigarchaeota-related 16S rRNA genes have been detected in numerous terrestrial,
            subsurface, and marine hydrothermal environments (
            <span ref-type="bibr" rid="bib28">
               <sup>
                  <a href="#bib28">Hirayama et al., 2005</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib46">
               <sup>
                  <a href="#bib46">Nunoura et al., 2005</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib16">
               <sup>
                  <a href="#bib16">Costa et al., 2009</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib61">
               <sup>
                  <a href="#bib61">Vick et al., 2010</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib44">
               <sup>
                  <a href="#bib44">Meyer-Dombard et al., 2011</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib14">
               <sup>
                  <a href="#bib14">Cole et al., 2013</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib18">
               <sup>
                  <a href="#bib18">De Leon et al., 2013</a>
               </sup>
            </span>
            <span ref-type="bibr" rid="bib14">
               <sup>
                  <a href="#bib14"></a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib59">
               <sup>
                  <a href="#bib59">Takacs-Vesbach et al., 2013</a>
               </sup>
            </span>). Phylogenetic placement of Aigarchaeota remains ambiguous, but recent phylogenomic
            studies reveal a close relationship with the Thaumarchaeota and possibly Crenarchaeota
            (
            <span ref-type="bibr" rid="bib12">
               <sup>
                  <a href="#bib12">Brochier-Armanet et al., 2011</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib47">
               <sup>
                  <a href="#bib47">Nunoura et al., 2011</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib50">
               <sup>
                  <a href="#bib50">Rayman et al., 2014</a>
               </sup>
            </span>). These reports have provided important insight into the phylogeny and evolution
            of the Aigarchaeota, yet little is known about the metabolic potential or 
            <i>in situ</i> activity of members of this phylum. Near-complete genome sequences have been generated
            for several members of the Aigarchaeota using cultivation-independent methods including
            fosmids (
            <span ref-type="bibr" rid="bib47">
               <sup>
                  <a href="#bib47">Nunoura et al., 2011</a>
               </sup>
            </span>), single-cell amplified genomes (
            <span ref-type="bibr" rid="bib52">
               <sup>
                  <a href="#bib52">Rinke et al., 2013</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib1">
               <sup>
                  <a href="#bib1">Alba et al., 2014</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib26">
               <sup>
                  <a href="#bib26">Hedlund et al., 2014</a>
               </sup>
            </span>) and metagenomics (
            <span ref-type="bibr" rid="bib1">
               <sup>
                  <a href="#bib1">Alba et al., 2014</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib26">
               <sup>
                  <a href="#bib26">Hedlund et al., 2014</a>
               </sup>
            </span>). However, limited genome characterization and metabolic activity analysis has been
            completed on any members of this group (
            <span ref-type="bibr" rid="bib47">
               <sup>
                  <a href="#bib47">Nunoura et al., 2011</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib52">
               <sup>
                  <a href="#bib52">Rinke et al., 2013</a>
               </sup>
            </span>), especially those Aigarchaeota important in geothermal ecosystems of Yellowstone
            National Park (YNP). Moreover, no attempts have been made to link members of the Aigarchaeota
            with microbial activity (for example, metatranscriptomics) in relation to other community
            members and their physicochemical environment.
         </p>
         <p xmlns:mml="http://www.w3.org/1998/Math/MathML">The environmental genome of 
            <i>Candidatus</i> ‘
            <i>Caldiarchaeum subterraneum</i>' was recovered from a mildly acidic (pH=5.1), oxic (~10 μ
            <div tagxxx="sc">m</div> dissolved oxygen (DO)) subsurface filamentous ‘streamer' community (
            <span ref-type="bibr" rid="bib28">
               <sup>
                  <a href="#bib28">Hirayama et al., 2005</a>
               </sup>
            </span>; depth ~320 m) at a temperature of 70 °C, and provided the first insight into the
            evolution and potential metabolism of members of the Aigarchaeota (
            <span ref-type="bibr" rid="bib47">
               <sup>
                  <a href="#bib47">Nunoura et al., 2011</a>
               </sup>
            </span>). Available genomic information from members of this deeply-rooted archaeal lineage
            suggests a possible evolutionary linkage with Eukarya (for example, an ubiquitin protein
            modification system; 
            <span ref-type="bibr" rid="bib47">
               <sup>
                  <a href="#bib47">Nunoura et al., 2011</a>
               </sup>
            </span>). Populations of 
            <i>C. subterraneum</i> might grow via the oxidation of hydrogen or carbon monoxide (CO) coupled to oxygen
            reduction by a heme copper (terminal) oxidase (HCO) complex. It is possible that these
            organisms can fix carbon dioxide via the 3-hydroxypropionate/4-hydroxybutyrate or
            dicarboxylate/4-hydroxybutyrate cycle (
            <span ref-type="bibr" rid="bib47">
               <sup>
                  <a href="#bib47">Nunoura et al., 2011</a>
               </sup>
            </span>). However, 
            <i>C. subterraneum</i> lacked the key enzyme (4-hydroxybutyryl-CoA dehydratase) of these pathways (
            <span ref-type="bibr" rid="bib8">
               <sup>
                  <a href="#bib8">Berg et al., 2010</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="bib47">
               <sup>
                  <a href="#bib47">Nunoura et al., 2011</a>
               </sup>
            </span>). Genome sequence of two different Aigarchaeota single-cells were obtained from oxic
            (DO=42 μ
            <div tagxxx="sc">m</div>), high temperature (81°C), circumneutral (pH=7.1) sediments at Great Boiling Spring
            (
            <span ref-type="bibr" rid="bib52">
               <sup>
                  <a href="#bib52">Rinke et al., 2013</a>
               </sup>
            </span>). Although the metabolic reconstruction of these Aigarchaeota was not a major focus
            of that study, the use of oxygen as an electron acceptor appears to be a consistent
            trait based on the presence of HCOs (
            <span ref-type="bibr" rid="bib52">
               <sup>
                  <a href="#bib52">Rinke et al., 2013</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib1">
               <sup>
                  <a href="#bib1">Alba et al., 2014</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib26">
               <sup>
                  <a href="#bib26">Hedlund et al., 2014</a>
               </sup>
            </span>). In addition, it has been speculated that some Aigarchaeota may also utilize other
            electron acceptors, such as oxidized sulfur or nitrogen compounds, but experimental
            data supporting these traits are unavailable (
            <span ref-type="bibr" rid="bib47">
               <sup>
                  <a href="#bib47">Nunoura et al., 2011</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib1">
               <sup>
                  <a href="#bib1">Alba et al., 2014</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib26">
               <sup>
                  <a href="#bib26">Hedlund et al., 2014</a>
               </sup>
            </span>).
         </p>
         <p xmlns:mml="http://www.w3.org/1998/Math/MathML">The filamentous ‘pink streamer' communities at Octopus Spring (OS) (Lower Geyser Basin,
            YNP) have been observed, studied, and intrigued scientists for more than 100 years
            (
            <span ref-type="bibr" rid="bib55">
               <sup>
                  <a href="#bib55">Setchell, 1903</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib13">
               <sup>
                  <a href="#bib13">Brock, 1967</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib4">
               <sup>
                  <a href="#bib4">Bauman and Simmonds, 1969</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib58">
               <sup>
                  <a href="#bib58">Stahl et al., 1985</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib51">
               <sup>
                  <a href="#bib51">Reysenbach et al., 1994</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib32">
               <sup>
                  <a href="#bib32">Jahnke et al., 2001</a>
               </sup>
            </span>; 
            <span ref-type="bibr" rid="bib10">
               <sup>
                  <a href="#bib10">Blank et al., 2002</a>
               </sup>
            </span>). A recent study of the three predominant lineages of Aquificales in YNP (
            <span ref-type="bibr" rid="bib59">
               <sup>
                  <a href="#bib59">Takacs-Vesbach et al., 2013</a>
               </sup>
            </span>) showed that the high temperature (~82 °C) OS streamer community contained several
            uncharacterized and uncultured taxa, including a novel Aigarchaeota population. The
            presence of Aigarchaeota in OS presents an excellent opportunity to understand the
            metabolic capabilities and the activity of this population 
            <i>in situ</i>. Consequently, the objectives of this study were to (i) obtain a detailed genome-based
            metabolic reconstruction of the predominant Aigarchaeota population present in OS,
            YNP, (ii) identify the morphology and spatial arrangement of Aigarchaeota and adjacent
            
            <i>Thermocrinis</i> spp. using fluorescence 
            <i>in situ</i> hybridization (FISH), and (iii) determine the 
            <i>in situ</i> transcriptional activity of this Aigarchaeota population to elucidate metabolic processes
            and potential community interactions.
         </p>
      </div>
      <div id="">
         <h2>Materials and methods</h2>
         <div id="">
            <h3>Site description and geochemistry</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">OS (
               <span ref-type="fig" rid="fig1">Figure 1</span>; Thermal Inventory ID LWCG138) is an alkaline siliceous geothermal spring located
               in the White Creek Area of the Lower Geyser Basin Region (lat 44.534083611, long −110.79788944)
               of YNP, WY, USA. The sample site discussed herein is classified as a ‘pink' filamentous
               ‘streamer' community (
               <span ref-type="fig" rid="fig1">Figure 1</span>), which has been the subject of many prior studies (for example, 
               <span ref-type="bibr" rid="bib4">
                  <sup>
                     <a href="#bib4">Bauman and Simmonds, 1969</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib58">
                  <sup>
                     <a href="#bib58">Stahl et al., 1985</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib51">
                  <sup>
                     <a href="#bib51">Reysenbach et al., 1994</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib32">
                  <sup>
                     <a href="#bib32">Jahnke et al., 2001</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib10">
                  <sup>
                     <a href="#bib10">Blank et al., 2002</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib59">
                  <sup>
                     <a href="#bib59">Takacs-Vesbach et al., 2013</a>
                  </sup>
               </span>). The metagenome sample discussed here was part of a 20-site study in YNP that included
               Aquificales-dominated ecosystems (
               <span ref-type="bibr" rid="bib59">
                  <sup>
                     <a href="#bib59">Takacs-Vesbach et al., 2013</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib31">
                  <sup>
                     <a href="#bib31">Inskeep et al., 2013</a>
                  </sup>
               </span>). These streamer communities colonize habitats across a temperature range of 78–84 °C,
               pH~8 and DO ~40 μ
               <div tagxxx="sc">m</div> (
               <span ref-type="supplementary-material" rid="sup1">Supplementary Table S1</span>). The streamer communities are considered oxic because the DO concentration is well
               above 0.2 μ
               <div tagxxx="sc">m</div>, the (K
               <div tagxxx="sub">m</div>) of microbial HCO complexes (
               <span ref-type="bibr" rid="bib45">
                  <sup>
                     <a href="#bib45">Morris and Schmidt, 2013</a>
                  </sup>
               </span>) that were previously identified in the OS streamer community (
               <span ref-type="bibr" rid="bib59">
                  <sup>
                     <a href="#bib59">Takacs-Vesbach et al., 2013</a>
                  </sup>
               </span>). Microbial streamers present in the outflow channel exhibit a ‘streaming' or oscillating
               motion in the high-velocity (~0.5 m s
               <sup>−1</sup>) outflow channel of OS (
               <span ref-type="supplementary-material" rid="sup1">Supplementary Movie S1</span>). Details on aqueous geochemical and physical sampling are discussed elsewhere (
               <span ref-type="bibr" rid="bib31">
                  <sup>
                     <a href="#bib31">Inskeep et al., 2013</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib59">
                  <sup>
                     <a href="#bib59">Takacs-Vesbach et al., 2013</a>
                  </sup>
               </span>).
            </p>
         </div>
         <div id="">
            <h3>DNA extraction and metagenome sequencing</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">Community DNA was extracted from OS ‘pink streamer' biomass (
               <span ref-type="fig" rid="fig1">Figure 1</span>) sampled on 23 August 2007 by enzymatic and bead-beating lysis, pooled and sequenced
               using Sanger technology at the Department of Energy-Joint Genome Institute, Walnut
               Creek, CA, USA (see 
               <span ref-type="bibr" rid="bib59">
                  <sup>
                     <a href="#bib59">Takacs-Vesbach et al., 2013</a>
                  </sup>
               </span> for details on sequencing and assembly). In addition, Illumina sequencing was performed
               at Department of Energy-Joint Genome Institute to increase sequencing depth and coverage
               on two samples collected on 13 October 2011 and 5 October, 2012 from DNA extracted
               with the FastDNA Soil DNA extraction kit and protocol (MP Biomedicals LLC, Solon,
               OH, USA). The metagenomic sequence discussed here is publically available on the Integrated
               Microbial Genomes with Microbiome Samples under the IMG submission IDs 885, 10389
               and 14458 for the 2007, 2011 and 2012 samples, respectively. Metagenome statistics
               from these samples are summarized in 
               <span ref-type="supplementary-material" rid="sup1">Supplementary Table S2</span>.
            </p>
         </div>
         <div id="">
            <h3>
               <i>De novo</i> Aigarchaeota assembly analysis and annotation
            </h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">Nucleotide word frequency-principal components analysis (NWF-PCA) of assembled metagenome
               sequence (
               <span ref-type="bibr" rid="bib59">
                  <sup>
                     <a href="#bib59">Takacs-Vesbach et al., 2013</a>
                  </sup>
               </span>) from OS revealed the presence of a novel Aigarchaeota lineage and several new bacterial
               taxa distantly related to the Thermotogae (
               <span ref-type="bibr" rid="bib59">
                  <sup>
                     <a href="#bib59">Takacs-Vesbach et al., 2013</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib15">
                  <sup>
                     <a href="#bib15">Colman et al., 2015</a>
                  </sup>
               </span>). Other community members in OS include 
               <i>Thermocrinis</i> spp., 
               <i>Pyrobaculum</i> spp. and a novel Firmicutes population (
               <span ref-type="bibr" rid="bib59">
                  <sup>
                     <a href="#bib59">Takacs-Vesbach et al., 2013</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib34">
                  <sup>
                     <a href="#bib34">Jay et al., 2014</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib15">
                  <sup>
                     <a href="#bib15">Coleman et al., 2015)</a>
                  </sup>
               </span>. A total of 1.16 Mb of sequence was recovered for the novel Aigarchaeota population
               with an average G+C content of 60.2% (
               <span ref-type="fig" rid="fig2">Figure 2</span>; 
               <span ref-type="supplementary-material" rid="sup1">Supplementary Table S3</span>). The 
               <i>de novo</i> aigarchaeon Sanger assembly (that is, 
               <i>Calditenuis aerorheumensis</i>) was further analyzed with NWF-PCA (online web server, 
               <div tagxxx="ext-link">http://doug1dev.cgb.indiana.edu/uploadScatterplotForm3.html</div>) and compared with other available Aigarchaeota sequence assemblies (&gt;1 Mb), 
               <i>Caldiarchaeum subterraneum</i> (
               <span ref-type="bibr" rid="bib47">
                  <sup>
                     <a href="#bib47">Nunoura et al., 2011</a>
                  </sup>
               </span>) and two single-cell amplified genomes from Great Boiling Spring (that is, aigarchaeota
               J15 and B22; 
               <span ref-type="bibr" rid="bib52">
                  <sup>
                     <a href="#bib52">Rinke et al., 2013</a>
                  </sup>
               </span>). The word size was set to four, minimum FASTA size set to 5000 bp, normalize GC
               only, normalize values on and chop sequences to 5000 bp. Celera assembled sequence
               for the Aigarchaeota population from OS was annotated with Integrated Microbial Genome/Expert
               Review with Microbiome Samples (
               <span ref-type="bibr" rid="bib42">
                  <sup>
                     <a href="#bib42">Markowitz et al., 2012</a>
                  </sup>
               </span>) using Prodigal (
               <span ref-type="bibr" rid="bib30">
                  <sup>
                     <a href="#bib30">Hyatt et al., 2010</a>
                  </sup>
               </span>) to identify the open reading frames. Manual annotation of all genes and pathways
               was necessary because the annotation pipeline missed a number of archaeal central
               carbon pathways. Thus, BLASTp of the 
               <i>Calditenuis aerorheumensis de novo</i> assembly with annotated genes of archaeal central carbon pathways was utilized (for
               example, 
               <span ref-type="bibr" rid="bib7">
                  <sup>
                     <a href="#bib7">Berg et al., 2007</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib29">
                  <sup>
                     <a href="#bib29">Huber et al., 2008</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib11">
                  <sup>
                     <a href="#bib11">Bräsen et al., 2014</a>
                  </sup>
               </span>). Moreover, simple annotation of a gene based on a Cluster of Orthologous Gene assignment
               is misleading. For example, canonical aerobic CO dehydrogenases have been ‘identified'
               in Aigarchaeota (
               <span ref-type="bibr" rid="bib47">
                  <sup>
                     <a href="#bib47">Nunoura et al., 2011</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib26">
                  <sup>
                     <a href="#bib26">Hedlund et al., 2014</a>
                  </sup>
               </span>); however, this assignment is dubious given no evidence of necessary cofactor binding
               sites in the large subunit of this enzyme complex (
               <span ref-type="bibr" rid="bib21">
                  <sup>
                     <a href="#bib21">Dobbek et al., 2002</a>
                  </sup>
               </span>). Thus, it was necessary to align the deduced protein sequences of these genes (and
               others) with ClustalW (default alignment settings) to identify conserved cofactor
               binding and active sites with of known function and structure. The curated 
               <i>Calditenuis aerorheumensis</i> assembly can be found on IMG/Mer under the name Aigarchaeota archaeon str. OS1 (
               <i>Calditenuis aerorheumensis</i>) and IMG submission ID 21516, and also under the NCBI BioProject ID PRJNA279181.
            </p>
         </div>
         <div id="">
            <h3>RNA extraction, sequencing and annotation</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">RNA was extracted from a matched sample of the OS ‘pink' streamer community (13 October
               2011) by modifying a FastRNA Pro-Soil-Direct Kit (MP Biomedicals) extraction kit and
               method. RNA from the streamer community was preserved in the field by placing streamer
               biomass directly into RNALater (Life Technologies, Carlsbad, CA, USA) and shaking
               vigorously for ~15 s. Streamer biomass (~0.5 g, 
               <i>n</i>=3) was removed from the RNALater solution in a sterile, RNase-free hood, added to
               0.5 ml of RNApro soil lysis solution in a lysing E matrix tube and vortexed for 15 min.
               The sample lysate was then centrifuged at 16 000 
               <i>g</i> for 10 min. The supernatant was transferred into a new 2 ml microcentrifuge tube
               (~500–800 μl), to which 1 ml of TriReagent was added and incubated for 5 min at room
               temperature (RT). Next, 200 μl of chloroform was added to the TriReagent/lysate mixture
               and incubated for 15 min at room temperature, followed by centrifugation at 16 000 
               <i>g</i> at 4 °C for 15 min. The triplicate supernatants containing RNA were pooled, and 500 μl
               of ice cold isopropanol (100%) was added and incubated overnight at −20 °C to facilitate
               RNA precipitation. RNA was pelleted by centrifugation at 16 000 
               <i>g</i> at 4 °C for 15 min and the pellet was washed once in ice cold 70% EtOH and centrifuged
               as before. The pellet was allowed to dry at room temperature for ~30 min, and immediately
               resuspended in 100 μl of TE buffer (pH=7, Life Technologies) and quantified with a
               Qubit 2.0 fluoremeter and Qubit RNA Broad Range Assay kit (Life Technologies). Total
               RNA quality was checked with a Bioanalyzer 2100 (Agilent Technologies, Santa Clara,
               CA, USA). DNA contamination was analyzed by PCR with universal archaeal and bacterial
               16S rRNA genes primers and ethidium bromide agarose gel electrophoresis. When DNA
               was present it was removed by DNase I treatment (New England Biolabs, Ipswich, MA,
               USA) for 30 min at 37 °C, followed by lithium chloride/EtOH precipitation overnight
               at −20 °C (
               <span ref-type="bibr" rid="bib5">
                  <sup>
                     <a href="#bib5">Beam et al., 2014</a>
                  </sup>
               </span>).
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">Double-stranded complementary DNA (cDNA) was synthesized at the Department of Energy-Joint
               Genome Institute from the OS total RNA extracts. In all, 2 μg of total RNA was subjected
               to rRNA depletion using Ribozero rRNA removal kit—Bacteria (Epicentre, Madison, WI,
               USA). Ribosomal RNA-depleted RNA was purified using Ampure XP beads (Agencourt, Beckman
               Coulter, Indianapolis, IN, USA), then fragmented using RNA Fragmentation Reagents
               (Life Technologies) at 70 °C for 2 min, targeting fragments ranging from 200 to 300 bp,
               then purified using Ampure XP beads. Reverse transcription was performed using SuperScript
               II Reverse Transcriptase (Life Technologies) with an initial annealing of random hexamers
               (Fermentas, Thermo Fisher Scientific, Pittsburgh, PA, USA) at 65 °C for 5 min, follow
               by an incubation of 42 °C for 50 min and an inactivation step at 70 °C for 10 min.
               Complementary DNA (cDNA) was then purified with Ampure XP beads, followed by second
               strand synthesis at 16 °C for 1 hour using a dNTP mix where dTTP is replaced by dUTP.
               Double-stranded cDNA fragments were purified and selected for targeted fragments (200–300 bp)
               using Ampure XP beads. The dscDNA was then blunt-ended, A-tailed and ligated with
               library adaptors using Kapa Library Amplification Kit (Kapa Biosystems, Wilmington,
               MA, USA) and purified using Ampure XP beads. Digestion of dUTP was then performed
               using AmpErase UNG (Applied Biosystems, Life Technologies) to remove second strand
               cDNA. Digested cDNA was purified with Ampure SPRI beads, followed by 10 cycles of
               PCR amplification using Kapa Library Amplification Kit. The final library was cleaned
               with Ampure SPRI beads and sequenced on an Illumina HiSeq 2000 platform generating
               paired-end reads of 150 bp.
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">Ribosomal RNA reads were removed 
               <i>in silico</i> using the Kmer-based tool DUK (
               <span ref-type="bibr" rid="bib40">
                  <sup>
                     <a href="#bib40">Li et al., 2011</a>
                  </sup>
               </span>) and messenger RNA (mRNA)-enriched reads were mapped to the 
               <i>Calditenuis aerorheumensis</i> assembly from OS utilizing Rockhopper, which uses a Bowtie2-like alignment algorithm
               (
               <span ref-type="bibr" rid="bib43">
                  <sup>
                     <a href="#bib43">McClure et al., 2013</a>
                  </sup>
               </span>). Mapped mRNA reads were standardized by the calculation reads per kilobase of transcript
               per million reads mapped (RPKM), which allows for the direct comparison of the level
               of transcription of all genes in the 
               <i>de novo</i> assembly (
               <span ref-type="supplementary-material" rid="sup1">Supplementary Table S4</span>). RPKM is calculated in Rockhopper as: RPKM=((number of mapped reads/length of transcript
               (gene) in kilobase)/million mapped reads); 
               <span ref-type="bibr" rid="bib43">
                  <sup>
                     <a href="#bib43">McClure et al., 2013</a>
                  </sup>
               </span>). Generally, if more reads map to a transcript of length x, and less map to another
               transcript with the same length x in the same sample, the former will have a higher
               RPKM value, and thus represent more 
               <i>in vivo</i> transcriptional activity.
            </p>
         </div>
         <div id="">
            <h3>Phylogeny construction</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">A detailed 16S rRNA gene tree was constructed that contained a broad representation
               of members of the ‘Aigarchaeota' in context with the Thaumarchaeota and Crenarchaeota.
               Aigarchaeota-related 16S rRNA genes were retrieved by BLAST homology from NCBI 
               <i>nr</i> database including environmental sequences. 
               <i>Calditenuis aerorheumensis</i> was also put into context by including 16S rRNA genes from recent Aigarchaeota single-cell
               genomes (
               <span ref-type="bibr" rid="bib52">
                  <sup>
                     <a href="#bib52">Rinke et al., 2013</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib1">
                  <sup>
                     <a href="#bib1">Alba et al., 2014</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib26">
                  <sup>
                     <a href="#bib26">Hedlund et al., 2014</a>
                  </sup>
               </span>). Sequences were aligned with ClustalW 1.6 implemented in MEGA 5.2.2 (
               <span ref-type="bibr" rid="bib60">
                  <sup>
                     <a href="#bib60">Tamura et al., 2011</a>
                  </sup>
               </span>) with default alignment settings. Alignments were manually inspected for obvious
               misaligned regions and introns, and removed from the final alignment (for example,
               most Thermoproteales contain 16S rRNA gene introns). A maximum-likelihood phylogenetic
               tree was created from the final alignment (1328 nucleotide positions) with the general
               time reversible model, estimated alpha parameter and four discrete gamma categories
               in MEGA 5.5.2 (100 bootstraps) (
               <span ref-type="bibr" rid="bib60">
                  <sup>
                     <a href="#bib60">Tamura et al., 2011</a>
                  </sup>
               </span>). Physical and aqueous geochemical data mapped onto the 16S rRNA gene tree is from
               Great Boiling Spring (
               <span ref-type="bibr" rid="bib16">
                  <sup>
                     <a href="#bib16">Costa et al., 2009</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib52">
                  <sup>
                     <a href="#bib52">Rinke et al., 2013</a>
                  </sup>
               </span>), Little Hot Creek (
               <span ref-type="bibr" rid="bib61">
                  <sup>
                     <a href="#bib61">Vick et al., 2010</a>
                  </sup>
               </span>), Yellowstone Lake (
               <span ref-type="bibr" rid="bib35">
                  <sup>
                     <a href="#bib35">Kan et al., 2011</a>
                  </sup>
               </span>) and other YNP hot spring environments (
               <span ref-type="bibr" rid="bib44">
                  <sup>
                     <a href="#bib44">Meyer-Dombard et al., 2011</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib59">
                  <sup>
                     <a href="#bib59">Takacs-Vesbach et al., 2013</a>
                  </sup>
               </span>).
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">A concatenated ribosomal protein phylogeny was constructed with 16 ribosomal proteins
               that were identified in all Aigarchaeota sequence assemblies (&gt;1 Mb, 
               <i>n</i>=4) shared with Thaumarchaeota, Crenarchaeota and 
               <i>Eukarya</i>. The deduced proteins identified that fulfilled the above criteria were: L5, L10,
               L14, L15, L18, L19E, L24E, L24P, L44E, S6E, S8, S11P, S19, S27E and S28E. The deduced
               protein sequences were aligned individually with ClustalW 1.6 with default alignment
               parameters in MEGA 5.5.2 (
               <span ref-type="bibr" rid="bib60">
                  <sup>
                     <a href="#bib60">Tamura et al., 2011</a>
                  </sup>
               </span>). The resulting alignments were concatenated with a custom perl script (
               <div tagxxx="ext-link">http://raven.iab.alaska.edu/~ntakebay/teaching/programming/perl-scripts/fastaConcat.pl</div>). The concatenated aligned ribosomal proteins were manually inspected for regions
               where homology was doubtful and removed for final analysis. A maximum-likelihood phylogenetic
               tree was generated from the concatenated ribosomal proteins (2249 unambiguously aligned
               amino acids) using the WAG model and estimated alpha parameter (100 bootstraps) in
               MEGA 5.5.2 (
               <span ref-type="bibr" rid="bib60">
                  <sup>
                     <a href="#bib60">Tamura et al., 2011</a>
                  </sup>
               </span>).
            </p>
         </div>
         <div id="">
            <h3>FISH</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">Hot spring microbial filaments from OS (~82–84 °C) were collected on 28 August 2013
               with a sterile scalpel and immediately fixed in 1% paraformaldehyde (final concentration)
               for 5 min at 4 °C. The fixative was then decanted and the sample was washed once in
               a 1:1 solution of 1 × phosphate-buffered saline:100% ethanol (EtOH) mixture (~10 ml)
               and stored in the 1:1 solution of 1 × phosphate-buffered saline: 100% EtOH at −20 °C
               until FISH. A FISH probe specific for all Group 1 A aigarchaeota (
               <span ref-type="fig" rid="fig3">Figure 3</span>; Aig800, Cy3-5′-GGC CCG TAG CCG CCC CGA CA-3′-Cy3; 
               <span ref-type="supplementary-material" rid="sup1">Supplementary Table S5</span>) was manually designed from 16S rRNA gene alignments of this group around 16S rRNA
               sites accessible to probes (
               <span ref-type="bibr" rid="bib6">
                  <sup>
                     <a href="#bib6">Behrens et al., 2003</a>
                  </sup>
               </span>) and dual labeled with Cy3 (Integrated DNA Technologies, Coralsville, IA, USA) (
               <span ref-type="bibr" rid="bib57">
                  <sup>
                     <a href="#bib57">Stoecker et al., 2010</a>
                  </sup>
               </span>). Individual masses of microbial filaments were removed from the storage solution
               and rinsed three times in 0.22 μm filtered double-distilled water (ddH
               <div tagxxx="sub">2</div>O). Next, the ddH
               <div tagxxx="sub">2</div>O rinsed filaments were embedded in Tissue-Tek O.C.T (Sakura Finetek, Torrance, CA,
               USA) tissue media on a dry-ice block for cryosectioning and sectioned on a CM1800
               cryostat (Leica Microsystems, Buffalo Grove, IL, USA). Thin sections were transferred
               to poly-lysine-coated glass microscope slides and stored at 4 °C until hybridization.
               Initial SYBR gold staining of thin sections revealed copious amounts of extracellular
               DNA present in the streamer community. Consequently, all thin sections utilized for
               FISH were pretreated with DNase I (New England Biolabs) for 30 min at 37 °C to degrade
               extracellular DNA, thus preventing non-specific probe binding. The DNase I reaction
               buffer was removed from the thin sections by rinsing in ddH
               <div tagxxx="sub">2</div>O (0.22 μm filtered) for ~1 min. Microscope slides containing thin sections (5 μm,
               three per slide) were dehydrated in an increasing EtOH series for 3 min each in 50,
               80 and 100% EtOH, then air dried. Hybridization buffer (30 μl) containing 40% formamide,
               0.9 M NaCl, 20 m
               <div tagxxx="sc">m</div> Tris HCL and 0.1% sodium dodecyl sulfate (added last to avoid precipitation) was
               added to each thin section and 1 μl of each probe (working solutions were 30 ng μl
               <sup>−1</sup> for Cy3/Cy5 and 50 ng μl
               <sup>−1</sup> for 6-FAM) to the buffer-coated thin sections. The slides were then placed in a 50 ml
               tube containing tissue paper soaked in the remaining hybridization buffer (~1 ml),
               sealed with the cap and placed in a 46 °C hybridization oven for 1.5 h. After hybridization,
               the slide was removed and immediately placed in 50 ml of pre-warmed wash buffer containing
               46 m
               <div tagxxx="sc">m</div> NaCl, 20 m
               <div tagxxx="sc">m</div> Tris HCl and 5 m
               <div tagxxx="sc">m</div> EDTA for exactly 10 min in a 48 °C water bath. The slide was removed from the wash
               buffer, rinsed in ddH
               <div tagxxx="sub">2</div>O and dried with laboratory air. The slide was immediately visualized by confocal
               microscopy (Leica SP5 inverted confocal scanning laser microscope at the Confocal
               Microscope Facility at the Center for Biofilm Engineering, Montana State University,
               Bozeman, MT, USA) or stored at −20 °C for up to 3 days without loss of fluorescence.
               Non-specific probe binding to streamer thin sections was not observed using a probe
               that is not complementary to any known 16S rRNA (nonEub338). All FISH probes used
               in this study can be found in 
               <span ref-type="supplementary-material" rid="sup1">Supplementary Table S5</span>.
            </p>
         </div>
         <div id="">
            <h3>Scanning electron microscopy</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">Filamentous ‘pink streamer' biomass was collected from OS on 12 October 2011 and fixed
               with 1% (final concentration) filter-sterilized glutaraldehyde. Streamer biomass was
               placed on a 0.22-μm filter and coated with iridium to reduce charging, and imaged
               with a Zeiss SUPRA 55VP field emission scanning electron microscope (Image and Chemical
               Analysis Laboratory, Montana State University) at low voltage (1 keV).
            </p>
         </div>
      </div>
      <div id="">
         <h2>Results and Discussion</h2>
         <div id="">
            <h3>
               <i>De novo</i> Aigarchaeota assembly
            </h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">The sequence assembly of the Aigarchaeota population from OS was compared with all
               available Aigarchaeota sequence assemblies (&gt;1 Mb total sequence) using tetra-NWF-PCA
               (
               <span ref-type="fig" rid="fig2">Figure 2a</span>). The aigarchaeon from OS (
               <i>Candidatus</i> ‘
               <i>Calditenuis aerorheumensis</i>', hereafter referred to as 
               <i>Calditenuis aerorheumensis</i>) exhibits highly similar codon usage and G+C content to aigarchaeon B22 from Great
               Boiling Spring (GBS) (
               <span ref-type="fig" rid="fig2">Figure 2a</span>). Further G+C content analysis of assembled sequence confirmed the close relationship
               of 
               <i>C. aerorheumensis</i> to aigarchaeon B22 with average G+C contents of 60.2 and 61.3%, respectively (
               <span ref-type="fig" rid="fig2">Figure 2b</span>). These two lineages have an average nucleotide identity of 87% among shared assembled
               sequence (~0.7 Mb), which suggests that they likely belong to the same novel candidate
               genus, 
               <i>Calditenuis</i>. 
               <i>Calditenuis aerorheumensis</i> and aigarchaeon B22 share a large amount of deduced protein sequences (~800) &gt;80%
               amino-acid identity (
               <span ref-type="fig" rid="fig2">Figure 2c</span>), and also supports their inclusion in the same genus. Aigarchaeon J15 from GBS and
               
               <i>Caldiarchaeum subterraneum</i> each form separate clusters in NWF-PCA due to differences in codon usage bias and
               G+C content (
               <span ref-type="fig" rid="fig2">Figure 2a and b</span>). The relative abundance of 
               <i>C. aerorheumensis</i> populations has been relatively stable over three sampling years at OS and represent
               5.2±2.5% of all the random-sequence reads (
               <span ref-type="supplementary-material" rid="sup1">Supplementary Table S1</span>). A total of 1422 protein-encoding genes were predicted (
               <span ref-type="supplementary-material" rid="sup1">Supplementary Table S3</span>), which corresponds to a coding density of ~1.2 protein-encoding genes per kb genome,
               similar to the protein-encoding density in aigarchaeon B22 of 1.13 protein-encoding
               genes per kb genome. Genome completeness estimates from conserved archaeal single-copy
               genes (
               <span ref-type="bibr" rid="bib52">
                  <sup>
                     <a href="#bib52">Rinke et al., 2013</a>
                  </sup>
               </span>) suggest that the 
               <i>C. aerorheumensis</i> assembly is ~80% complete, which equates to an estimated genome size of ~1.45 Mb.
               The high protein-coding density taken together with the small-predicted genome size
               suggests that this organism may have undergone extensive genome streamlining (
               <span ref-type="bibr" rid="bib24">
                  <sup>
                     <a href="#bib24">Giovannoni et al., 2005</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="bib25">
                  <sup>
                     <a href="#bib25">2014</a>
                  </sup>
               </span>).
            </p>
         </div>
         <div id="">
            <h3>FISH identification</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">All known Aigarchaeota encode for actin, which suggests that they are rods or filaments
               (
               <span ref-type="bibr" rid="bib23">
                  <sup>
                     <a href="#bib23">Ettema et al., 2011</a>
                  </sup>
               </span>); however, no definitive morphological observations have been made for any member
               of the Aigarchaeota. FISH with probe Aig800 designed to target terrestrial Group 1 A
               Aigarchaeota (including aigarchaeon B22) revealed that 
               <i>Calditenuis aerorheumensis</i> populations in OS are filamentous (
               <span ref-type="fig" rid="fig3">Figure 3a and b</span>) ranging from 0.5 μm (diameter) by up to 20 μm (length), but may be longer due to
               damage during cryosectioning or sample dehydration. The broad-coverage archaeal FISH
               probe Arch915 also hybridized to these filaments (
               <span ref-type="fig" rid="fig3">Figure 3a and b</span>). 
               <i>Calditenuis aerorheumensis</i> filaments were often found in close contact or vicinity of 
               <i>Thermocrinis</i> spp. (
               <span ref-type="fig" rid="fig3">Figure 3a and b</span>), which were specifically detected using the newly designed probe Aqi338 (
               <span ref-type="bibr" rid="bib39">
                  <sup>
                     <a href="#bib39">Kubo et al., 2011</a>
                  </sup>
               </span>), and might be related to metabolite sharing and/or auxotrophic requirements of 
               <i>C. aerorheumensis</i>. FISH probes targeting 
               <i>Thermocrinis</i> spp., all bacteria, and all archaea revealed a close association of all microorganisms
               in the OS streamer community (
               <span ref-type="fig" rid="fig3">Figure 3c and d</span>). Archaea and 
               <i>Thermocrinis</i> spp. are often found in helical bundled filaments, and 
               <i>Thermocrinis</i> spp. can also be found in contact with other bacteria (
               <span ref-type="fig" rid="fig3">Figure 3c and d</span>). Scanning electron micrographs of the OS streamer community (
               <span ref-type="fig" rid="fig3">Figure 3e and f</span>) also confirm the filamentous habit of several population types, which range in diameters
               of ~0.2–0.5 μm up to ~0.7–1 μm. Small (&lt;20 nm), extracellular polymeric substances
               (EPS) were evident in SEM images and represent a significant volume of the OS streamer
               community biomass (
               <span ref-type="fig" rid="fig3">Figure 3e and f</span>).
            </p>
         </div>
         <div id="">
            <h3>Phylogeny and distribution</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">Phylogenetic analysis of 16S rRNA genes from the candidate phylum Aigarchaeota across
               numerous terrestrial, subsurface and freshwater geothermal environments revealed eight
               genus-level lineages (
               <span ref-type="fig" rid="fig4">Figure 4</span>). The 
               <i>C. aerorheumensis</i> population groups with several YNP 16S rRNA gene clones from YNP Bison Pool ‘streamers',
               as well as single-cell sequence (aigarchaeon B22) from GBS (
               <span ref-type="fig" rid="fig4">Figure 4</span>) at temperatures ranging from ~74 °C to 86 °C and pH values of 7.2–7.9. Group 1A
               Aigarchaeota are found in oxic hot spring ecosystems (DO ~10–53 μ
               <div tagxxx="sc">m</div>), which is consistent across all the Aigarchaeota lineages currently identified by
               16S rRNA genes. Aigarchaeota are distributed in geothermal environments at high temperature
               (68–87 °C) and moderately acidic to alkaline pH values (~5–9). The phylogenetic position
               of 
               <i>C. aerorheumensis</i> was also confirmed using a conserved set of single-copy ribosomal proteins. The concatenated
               phylogenetic tree shows that 
               <i>C. aerorheumensis</i> and aigarchaeon B22 are closely related, and form a separate group compared to 
               <i>C. subterraneum</i> and aigarchaeon J15 (
               <span ref-type="fig" rid="fig5">Figure 5</span>).
            </p>
         </div>
         <div id="">
            <h3>
               <i>In situ</i> metatranscriptome
            </h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">The 
               <i>C. aerorheumensis</i> assembly was utilized to align metatranscriptome reads from OS (~18 million mRNA
               enriched reads). A total of 544 290 mRNA reads were aligned to 
               <i>C. aerorheumensis</i>, which constitutes ~3% of the total mRNA reads from the streamer community. Analysis
               of all mRNA reads using Cluster of Orthologous Genes showed that the transcription
               of a number of cellular processes were higher than their relative abundance in the
               genome (
               <span ref-type="supplementary-material" rid="sup1">Supplementary Figure S1</span>). Notably, energy production and conservation, and amino-acid transport and catabolism
               were over-transcribed compared with the relative abundance in the genome (
               <span ref-type="supplementary-material" rid="sup1">Supplementary Figure S1</span>). This finding suggests that amino-acid transport and catabolism is an important
               energy, carbon and nitrogen source for 
               <i>C. aerorheumensis</i> in the streamer community.
            </p>
         </div>
         <div id="">
            <h3>Respiration</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">A detailed metabolic model was constructed for 
               <i>C. aerorheumensis</i> using curated genome sequence and mRNA transcripts mapped to individual genes in
               each pathway (
               <span ref-type="fig" rid="fig6">Figure 6</span>). Pathways and metabolic functions that exhibited increased transcription levels
               were emphasized to reflect 
               <i>in situ</i> activity (
               <span ref-type="fig" rid="fig6">Figure 6</span>). The streamer community present in OS (
               <span ref-type="fig" rid="fig1">Figure 1</span>) oscillates in the high-velocity channel (
               <span ref-type="supplementary-material" rid="sup1">Supplementary Movie S1</span>), which has aqueous O
               <div tagxxx="sub">2</div> concentrations ranging from ~20 to 40 μ
               <div tagxxx="sc">m</div> (
               <span ref-type="supplementary-material" rid="sup1">Supplementary Table S1</span>). Gradients in O
               <div tagxxx="sub">2</div> are likely to occur within the streamer community that could create microaerobic
               or hypoxic microenvironments (for example, 
               <span ref-type="bibr" rid="bib9">
                  <sup>
                     <a href="#bib9">Bernstein et al., 2013</a>
                  </sup>
               </span>). A single HCO complex (_00543–_00544) was identified in 
               <i>C. aerorheumensis</i> that recruited numerous RNA reads from the metatranscriptome (RPKM ~70 000; 
               <span ref-type="fig" rid="fig6">Figure 6</span>), and represents the dominant terminal electron acceptor utilized by 
               <i>C. aerorheumensis</i> in OS. Similar HCO complexes were also identified in 60% of available Aigarchaeota
               sequence assemblies in IMG/Mer (
               <span ref-type="supplementary-material" rid="sup1">Supplementary Table S7</span>), which suggests that numerous members of this phylum are aerobic, and that O
               <div tagxxx="sub">2</div> is an important niche-defining parameter for many members of this phylum. The importance
               of oxygen to the Aigarchaeota is also exemplified by the presence of genes that code
               for proteins responsible for degradation of reactive oxygen species in 
               <i>C. aerorheumensis</i>, as well as 65% of Aigarchaeota genomes (
               <span ref-type="supplementary-material" rid="sup1">Supplementary Table S7</span>). An Fe-Mn superoxide dismutase was identified and transcription of this gene was
               high (RPKM ~800 000; 
               <span ref-type="fig" rid="fig6">Figure 6</span>). A 2-Cys peroxiredoxin (_00775) and a 1-Cys peroxiredoxin (_00485) were identified
               and these enzymes are involved in the intra and extracellular degradation of hydrogen
               peroxide, respectively. Intracellular hydrogen peroxide is formed primarily by superoxide
               dismutase, and is hazardous to the cell by participating in Fenton reactions with
               Fe(II) that generate the DNA-damaging hydroxyl radical (•OH; 
               <span ref-type="bibr" rid="bib27">
                  <sup>
                     <a href="#bib27">Henle and Linn, 1997</a>
                  </sup>
               </span>). Thus, the transcription of the 2-Cys peroxiredoxin is very high (RPKM ~130 000)
               and is comparable to the transcription levels of superoxide dismutase (RPKM ~800 000).
               Degradation of extracellular hydrogen peroxide by 1-Cys peroxiredoxin (RPKM ~15 000),
               which can form by the interaction of UV light and dissolved OC with O
               <div tagxxx="sub">2</div> (
               <span ref-type="bibr" rid="bib22">
                  <sup>
                     <a href="#bib22">Draper and Crosby, 1983</a>
                  </sup>
               </span>), may be necessary for survival in these environments and/or an important ecosystem
               service that 
               <i>C. aerorheumensis</i> performs for the OS streamer community, similar to 
               <i>Vibrio pelagius</i> and 
               <i>Synechococcus</i> spp. in the open ocean (
               <span ref-type="bibr" rid="bib48">
                  <sup>
                     <a href="#bib48">Petasne and Zika, 1997</a>
                  </sup>
               </span>).
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">The 
               <i>C. aerorheumensis</i> population in OS also contains an operon encoding a putative tetrathionate reductase
               (TTR) system (_00420–_00424). To date, archaeal TTRs have only been identified in
               
               <i>Pyrobaculum</i> spp. and 
               <i>Archaeoglobus fulgidus</i> (
               <span ref-type="bibr" rid="bib17">
                  <sup>
                     <a href="#bib17">Cozen et al., 2009</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="bib41">
                  <sup>
                     <a href="#bib41">Liebensteiner et al., 2013</a>
                  </sup>
               </span>). The TTR operon was transcribed at low abundance (RPKM ~9000; 
               <span ref-type="fig" rid="fig6">Figure 6</span>) and suggests that reduction of other terminal electron acceptors may provide an
               alternative to oxygen. Tetrathionate (S
               <div tagxxx="sub">4</div>O
               <div tagxxx="sub">6</div>
               <sup>2−</sup>) has not been detected in OS; however, 
               <i>Thermocrinis</i> spp. contain genes necessary for the oxidation of reduced sulfur compounds (
               <span ref-type="bibr" rid="bib59">
                  <sup>
                     <a href="#bib59">Takacs-Vesbach et al., 2013</a>
                  </sup>
               </span>) and may produce S
               <div tagxxx="sub">4</div>O
               <div tagxxx="sub">6</div>
               <sup>2−</sup> as a byproduct, a feature that has been commonly observed in marine bacteria (
               <span ref-type="bibr" rid="bib49">
                  <sup>
                     <a href="#bib49">Podgorsek and Imhoff, 1999</a>
                  </sup>
               </span>).
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">Reducing equivalents (for example, NADH) generated from the oxidation of OC substrates
               (see below) are fed into the electron transport chain at the NADH dehydrogenase complex
               (
               <span ref-type="fig" rid="fig6">Figure 6</span>; complex I), which is not encoded in an operon. The membrane-bound succinate dehydrogenase
               complex (
               <span ref-type="fig" rid="fig6">Figure 6</span>; complex II) also feeds the electron transport chain. The 
               <i>C. aerorheumensis</i> assembly does not encode a typical cytochrome C complex III for electron transport
               to complex IV (HCO); however, several membrane-bound blue copper proteins may serve
               as electron carries to the HCO or TTR complexes (
               <span ref-type="fig" rid="fig6">Figure 6</span>; _00503–_00505). The 
               <i>C. aerorheumensis</i> population from OS has a complete electron transport chain, predominantly coupled
               to the reduction of oxygen, and adenosine triphosphate (ATP) synthesis driven by an
               archaeal V-type ATP synthase (
               <span ref-type="fig" rid="fig6">Figure 6</span>).
            </p>
         </div>
         <div id="">
            <h3>Energy metabolism</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <i>Calditenuis aerorheumensis</i> contains genes necessary for the degradation of different organic carbon (OC) compounds
               coupled with adenosine triphosphate (organotrophy) and biomass (heterotrophy) synthesis.
               Major carbon substrates utilized for growth include acetate, fatty acids, amino acids
               and sugars. These OC constituents may originate from EPS produced by other thermophilic
               bacteria and archaea in OS and/or from autochthonous dissolved OC (
               <span ref-type="supplementary-material" rid="sup1">Supplementary Table S1</span>). Acetate is transported by a putative citrate/acetate antiporter (_00932; RPKM ~20 000),
               and utilized by an acetyl-CoA synthetase (_00897) to produce acetyl-CoA (
               <span ref-type="fig" rid="fig6">Figure 6</span>), which can then be fed into the tricarboxylic acid (TCA) cycle or gluconeogenesis.
               Acetate utilization is further supported by the presence of a functional glyoxylate
               bypass (
               <span ref-type="fig" rid="fig6">Figure 6</span>). The presence of isocitrate lyase (_00945; RPKM ~29 000) and malate synthase (_01089;
               RPKM ~70 000; 
               <span ref-type="fig" rid="fig6">Figure 6</span>) provides a mechanism to incorporate carbon from acetate into biomass without the
               loss of CO
               <div tagxxx="sub">2</div> through the TCA cycle. 
               <i>C. aerorheumensis</i> also has the capacity to utilize fatty acids produced by thermophilic bacteria in
               OS as carbon and energy sources. This β-oxidation pathway contains an acyl-CoA synthetase
               (_01439; RPKM ~34 000) that converts fatty acids to acyl-CoA derivatives utilized
               by three different acyl-CoA dehydrogenases (_01040; _01032; _00259), which are likely
               specific for fatty acids of certain chain lengths (for example, odd or even). Transcription
               of one acyl-CoA dehydrogenase was considerably greater than several others (_01032;
               RPKM ~120 000) and likely corresponds to a preferred fatty acid utilized 
               <i>in situ</i> by 
               <i>C. aerorheumensis</i>. Although the metabolism of fatty acids appears to be distributed across most archaea
               (
               <span ref-type="bibr" rid="bib19">
                  <sup>
                     <a href="#bib19">Dibrova et al., 2014</a>
                  </sup>
               </span>), no enzymatic characterization of the different acyl-CoA dehydrogenases has been
               conducted. Physiological studies have shown that 
               <i>Archaeoglobus fulgidus</i> can grow on either short- and/or long-chain fatty acids (
               <span ref-type="bibr" rid="bib36">
                  <sup>
                     <a href="#bib36">Khelifi et al., 2010</a>
                  </sup>
               </span>).
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <i>C. aerorheumensis</i> also exhibits the metabolic capacity to utilize numerous amino acids and oligopeptides
               as carbon and energy sources. Transporters for amino acids were identified in the
               
               <i>de novo</i> assembly (
               <span ref-type="fig" rid="fig6">Figure 6</span>), and branched-chain amino acids are transported by a high-affinity ABC transporter
               that was highly transcribed (
               <span ref-type="fig" rid="fig6">Figure 6</span>; RPKM&gt;100 000). A single copy of a dihydroxy acid dehydratase (
               <i>ivlD</i>; _01261) was identified, which may catalyze the conversion of branched-chain amino
               acids to TCA intermediates. Other amino acids are catabolized by a 2-oxoisovalerate
               oxidoreductase complex (_00164+_00741), which was also highly transcribed (RPKM&gt;70 000),
               then incorporated into the TCA cycle (
               <span ref-type="fig" rid="fig6">Figure 6</span>).
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">A set of genes containing LigA and LigB homologs were identified in a single operon
               (_01094–_01099) potentially involved in the degradation of lignin or other aromatic
               compounds. Transcription levels in this pathway were increased for a hypothetical
               protein (_01096; RPKM ~200 000) and a ferredoxin ring-hydroxylating dioxygenase (_01095;
               RPKM ~70 000). The aromatic compound utilized by this pathway could not be identified,
               but these genes are also found in other Aigarchaeota (
               <span ref-type="bibr" rid="bib26">
                  <sup>
                     <a href="#bib26">Hedlund et al., 2014</a>
                  </sup>
               </span>). The source pool and outflow channel of OS contain significant concentrations of
               dissolved OC (~60 μ
               <div tagxxx="sc">m</div>; 
               <span ref-type="supplementary-material" rid="sup1">Supplementary Table S1</span>), which would be sufficient to support heterotrophic populations of 
               <i>C. aerorheumensis</i>.
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">It has been suggested that some members of the Aigarchaeota are capable of using CO
               as an energy source (
               <span ref-type="bibr" rid="bib47">
                  <sup>
                     <a href="#bib47">Nunoura et al., 2011</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib52">
                  <sup>
                     <a href="#bib52">Rinke et al., 2013</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib26">
                  <sup>
                     <a href="#bib26">Hedlund et al., 2014</a>
                  </sup>
               </span>), but physiological evidence is lacking. The 
               <i>C. aerorheumensis</i> assembly contains five copies of putative large-subunit aerobic CO dehydrogenases
               (
               <i>coxL</i>) (COG1529; _00394; _00730; _01206; _01316; 01360). However, all putative CoxL homologs
               from Aigarchaeota lack the conserved active site residues (VAYRCSFR) identified in
               canonical aerobic CO dehydrogenases (
               <span ref-type="supplementary-material" rid="sup1">Supplementary Figure S2</span>; 
               <span ref-type="bibr" rid="bib21">
                  <sup>
                     <a href="#bib21">Dobbek et al., 2002</a>
                  </sup>
               </span>). The gene order of the middle and small subunits (_00406; _00407) is identical to
               the aerobic CO oxidizer, 
               <i>Oligotropha carboxidovorans</i>, and both contain necessary cofactor binding sites (
               <span ref-type="bibr" rid="bib20">
                  <sup>
                     <a href="#bib20">Dobbek et al., 1999</a>
                  </sup>
               </span>). The transcription of 
               <i>coxSML</i> genes were elevated (
               <span ref-type="fig" rid="fig6">Figure 6</span>; RPKM&gt;150 000), so the function of these enzymes is important to determine and may
               aid in interpreting 
               <i>in situ</i> physiology. Dissolved CO concentrations measured in OS are below detection (~10 n
               <div tagxxx="sc">m</div>) and CO concentrations in other geothermal waters range from ~30 n
               <div tagxxx="sc">m</div> to below detection levels (
               <span ref-type="bibr" rid="bib37">
                  <sup>
                     <a href="#bib37">Kochetkova et al., 2011</a>
                  </sup>
               </span>). Consequently, 
               <i>C. aerorheumensis</i> populations are not likely utilizing CO as a principal electron donor given the low
               concentrations of CO relative to other electron donors such as reduced OC compounds
               (for example, dissolved OC ~60 μ
               <div tagxxx="sc">m</div>; 
               <span ref-type="supplementary-material" rid="sup1">Supplementary Table S1</span>).
            </p>
         </div>
         <div id="">
            <h3>Central carbon metabolism</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <i>C. aerorheumensis</i> exhibits two routes for the conversion of glucose to pyruvate via either Embden–Meyerhof–Parnas
               glycolysis or Entner–Doudoroff pathways (
               <span ref-type="fig" rid="fig6">Figure 6</span>). A putative glucose ABC transporter operon was identified (_00266–_00269) and transcription
               was low except for the periplasmic subunit (_00269; 
               <span ref-type="fig" rid="fig6">Figure 6</span>). Gluconeogenesis is also possible and the key enzyme fructose 1,6-bisphosphate aldolase/phosphatase
               is present, which has the conserved residues (GKDDP) implicated in catalysis (
               <span ref-type="bibr" rid="bib53">
                  <sup>
                     <a href="#bib53">Say and Fuchs, 2010</a>
                  </sup>
               </span>). All genes required for a complete TCA cycle were also identified with the exception
               of fumarase (
               <span ref-type="fig" rid="fig6">Figure 6</span>). Transcription of TCA cycle genes were high (RPKM &gt;10 000), which is consistent
               with apparent primary sources of OC that 
               <i>C. aerorheumensis</i> populations utilize 
               <i>in situ</i> (that is, acetate, fatty acids and amino acids). A complete oxidative pentose phosphate
               pathway was identified in 
               <i>C. aerorheumensis</i> with the exception of gluconolactonase, which is also absent in other thermophilic
               Thaumarchaeota (
               <span ref-type="bibr" rid="bib56">
                  <sup>
                     <a href="#bib56">Spang et al., 2012</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib5">
                  <sup>
                     <a href="#bib5">Beam et al., 2014</a>
                  </sup>
               </span>).
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">Autotrophic Crenarchaeota and Thaumarchaeota fix inorganic carbon dioxide via the
               3-hydroxypropionate/4-hydroxybutyrate (HP/HB) or dicarboxylate/4-hydroxybuyrate (DC/HB)
               cycles, which share a key marker enzyme, 4-hydroxybutyryl-CoA dehydratase (4-BUDH;
               
               <span ref-type="bibr" rid="bib8">
                  <sup>
                     <a href="#bib8">Berg et al., 2010</a>
                  </sup>
               </span>). A gene for a Type-2 4-BUDH (_01438) was identified in the 
               <i>C. aerorheumensis</i> assembly (
               <span ref-type="fig" rid="fig7">Figure 7</span>; 
               <span ref-type="supplementary-material" rid="sup1">Supplementary Figure S3</span>); however, Type-2 4-BUDHs have yet to be biochemically characterized, and all share
               a conserved His-292, but lack conserved cysteine residues required for (4Fe-4S) cluster
               binding (
               <span ref-type="bibr" rid="bib7">
                  <sup>
                     <a href="#bib7">Berg et al., 2007</a>
                  </sup>
               </span>). The transcription of the Type-2 4-BUDH was very high (RPKM&gt;100 000) in 
               <i>C. aerorheumensis</i> (
               <span ref-type="fig" rid="fig7">Figure 7</span>). No other function for this gene can be assigned other than the conversion of 4-hydroxybutyryl-CoA
               to crotonyl-CoA, which suggests that the Aigarchaeota Type-2 4-BUDHs may be functional
               in the HP/HB cycle. Moreover, 
               <i>C. aerorheumensis</i> encodes for every gene of the HP/HB pathway, with the exception of NADPH-dependent
               malonate semialdehyde reductase and methylmalonyl-CoA epimerase (
               <span ref-type="fig" rid="fig7">Figure 7</span>; 
               <span ref-type="supplementary-material" rid="sup1">Supplementary Table S8</span>). This is the first activity-based measurement that suggests members of the Aigarchaeota
               may fix carbon dioxide via the HP/HB pathway.
            </p>
         </div>
         <div id="">
            <h3>Cofactor, vitamin and amino-acid biosynthesis</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">Pathways responsible for the synthesis of all amino acids, F
               <div tagxxx="sub">420</div>, pantothenate (B
               <div tagxxx="sub">5</div>), pyroxidine (B
               <div tagxxx="sub">6</div>) and cobalamin salvage were identified in 
               <i>C. aerorheumensis</i> (
               <span ref-type="table" rid="tbl1">
                  <a href="#tbl1">Table 1</a>
               </span>). Coenzyme F
               <div tagxxx="sub">420</div> is utilized in the glucose-6-phosphate dehydrogenase enzyme in the oxidative pentose
               phosphate pathway in 
               <i>C. aerorheumensis.</i> This cofactor was also identified in the Thaumarchaeota, methanogenic archaea, and
               ‘Geoarchaeota' (
               <span ref-type="bibr" rid="bib38">
                  <sup>
                     <a href="#bib38">Kozubal et al., 2012</a>
                  </sup>
               </span>; 
               <span ref-type="bibr" rid="bib56">
                  <sup>
                     <a href="#bib56">Spang et al., 2012</a>
                  </sup>
               </span>). On the basis of the available sequence data, 
               <i>C. aerorheumensis</i> cannot synthesize heme, thiamin, riboflavin, niacin and biotin; however, putative
               transporters were identified for these essential cofactors and vitamins (
               <span ref-type="table" rid="tbl1">
                  <a href="#tbl1">Table 1</a>
               </span>). The absence of vitamin and cofactor biosynthesis pathways appears common within
               this group of archaea (
               <span ref-type="supplementary-material" rid="sup1">Supplementary Table S9</span>); however, there are only a few complete genomes available for this candidate phylum.
               The absence of heme biosynthetic genes in 
               <i>C. aerorheumensis</i> and single-cell genomes from the Aigarchaeota (
               <span ref-type="supplementary-material" rid="sup1">Supplementary Table S9</span>) is surprising because heme is the essential cofactor for oxygen binding in HCOs,
               and these genes were highly transcribed (
               <span ref-type="fig" rid="fig6">Figure 6</span>). The putative heme transporter was found at low transcriptional abundance (
               <span ref-type="fig" rid="fig6">Figure 6</span>), which might suggest strict heme recycling 
               <i>in vivo</i>, as high intracellular heme concentrations are often hazardous and/or lethal (
               <span ref-type="bibr" rid="bib3">
                  <sup>
                     <a href="#bib3">Anzaldi and Skaar, 2010</a>
                  </sup>
               </span>). Low transcription levels of transporters for other cofactors and vitamins (
               <span ref-type="fig" rid="fig6">Figure 6</span>; except for thiamin and niacin) suggests conservative 
               <i>in vivo</i> control of vitamin and cofactor recycling. Widespread auxotrophy in all known members
               of Aigarchaeota (
               <span ref-type="supplementary-material" rid="sup1">Supplementary Table S9</span>) suggest that they rely on other community members to supply them with essential
               compounds for growth and reproduction. The energetic cost of maintaining complete
               vitamin and cofactor biosynthesis pathways may have outweighed the cost of acquiring
               them from external sources (
               <span ref-type="bibr" rid="bib24">
                  <sup>
                     <a href="#bib24">Giovannoni et al., 2005</a>
                  </sup>
               </span>). 
               <i>Calditenuis aerorheumensis</i> encodes the capacity for natural DNA competence (
               <i>dprA</i>) also present in some Thermoproteales (
               <span ref-type="bibr" rid="bib54">
                  <sup>
                     <a href="#bib54">Siebers et al., 2011</a>
                  </sup>
               </span>), and may be able to re-acquire complete cofactor and biosynthesis pathways under
               selection pressure.
            </p>
         </div>
         <div id="">
            <h3>Motility and attachment</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">Archaeal flagella are more similar to type IV secretion systems found in bacteria
               and have nothing in common with bacterial flagella other than the use for motion or
               taxis (
               <span ref-type="bibr" rid="bib33">
                  <sup>
                     <a href="#bib33">Jarrell and Albers, 2012</a>
                  </sup>
               </span>). All genes within an operon for an archaeal flagellum were found in 
               <i>C. aerorheumensis</i> (_00011–_00018), and were transcribed at low levels (
               <span ref-type="fig" rid="fig6">Figure 6</span>; RPKM&lt;1500). Flagellar synthesis may be more important during streamer formation
               and could explain the downregulation of these genes in this ‘mature' community. A
               pilus system (_01412–_01414) was also identified in 
               <i>C. aerorheumensis</i>. All pilus genes were highly transcribed (
               <span ref-type="fig" rid="fig6">Figure 6</span>; RPKM&gt;150 000) and may be involved in cell–cell contact and association of 
               <i>C. aerorheumensis</i> with other community members observed in FISH images (
               <span ref-type="fig" rid="fig3">Figure 3a and b</span>). The pilus system in 
               <i>C. aerorheumensis</i> may be involved in the formation of EPS, which are abundant and represent a substantial
               fraction of the community biomass (
               <span ref-type="fig" rid="fig3">Figure 3e and f</span>). Pili have been identified in other archaea as well, and are involved in cell–cell
               adherence and/or biofilm formation (
               <span ref-type="bibr" rid="bib2">
                  <sup>
                     <a href="#bib2">Albers and Meyer, 2011</a>
                  </sup>
               </span>).
            </p>
         </div>
         <div id="">
            <h3>Etymology</h3>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">The provisional taxonomic assignment for this group of Aigarchaeota is ‘
               <i>Candidatus</i>: Calditenuis aerorheumensis' alluding to its thermal habitat, filamentous shape and
               aerobic catabolism.
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">Calditenuis gen. nov.</p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">Calditenuis aerorheumensis sp. nov.</p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">Genus
               <i>: Caldi</i> (L. adj.): warm, 
               <i>tenuis</i> (L. adj.): thin or slender. Species: 
               <i>aero</i> (Gr. noun): air or atmosphere, 
               <i>rheumensis</i> (Gr. noun); a stream, current, or that which flows. The Genus name describes the
               organism's thermophilic nature and thin, filamentous shape. The species name alludes
               to the organism's principal terminal electron acceptor, oxygen and filamentous ‘streamer'
               habitat.
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">Locality: oxic (~30 μ
               <div tagxxx="sc">m</div>), high temperature (~75–85 °C) and pH ~7.5–8.5 thermal springs.
            </p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">Diagnosis: a thin, filamentous, aerobic chemoorganohetero(auto)troph from the Aigarchaeota.</p>
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">Comment: Future efforts to isolate members of this genus and all other Aigarchaeota
               (with the exception of aigarchaeon J15) might consider adding heme to vitamin solutions,
               which is not common in Wolfe's vitamin solution (
               <span ref-type="bibr" rid="bib62">
                  <sup>
                     <a href="#bib62">Wolin et al., 1963</a>
                  </sup>
               </span>) typically used in archaeal enrichment cultures. Moreover, 
               <i>C. aerorheumensis</i> is likely oligotrophic and may not respond to high concentrations of C sources and/or
               vitamins.
            </p>
         </div>
      </div>
      <div tagxxx="back">
         <div tagxxx="ack">
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">This work was supported by the Department of Energy (DOE)—Pacific Northwest National
               Laboratory Foundational Science Focus Area in Biological Interactions (Subcontract
               112443), the DOE—Joint Genome Institute Community Sequencing Projects (CSP 787081
               and 787701), the National Science Foundation—Integrative Graduate Education and Traineeship
               Program (JPB, ZJJ and R de MJ; DGE 0654336), and an Advanced Grant of the European
               Research Council to MW (MCS and MW; NITRICARE 294343). The work conducted by the Department
               of Energy-Joint Genome Institute, a DOE Office of Science User Facility, was supported
               by the Office of Science of the US DOE (contract no. DE-AC02-05CH11231). We appreciate
               Brian Hedlund and Tim Alba for data sharing, Christie Hendrix and Stacey Gunther (YNP
               Center for Resources) for research permitting in YNP (permits YELL-SCI-5068 and -5686)
               and Betsey Pitts for cryosection instruction and helpful comments on confocal microscopy.
               FISH images were taken at microscope facilities at the Center for Biofilm Engineering
               (Montana State University) funded by the NSF-MRI Program and the M.J. Murdock Charitable
               Trust. The open access fees were generously provided by the Library's Author Fund
               at Montana State University.
            </p>
         </div>
         <div tagxxx="fn-group">
            <div class="fn" id="">
               <p xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <span ref-type="supplementary-material" rid="sup1">Supplementary Information</span> accompanies this paper on The ISME Journal website (http://www.nature.com/ismej)
               </p>
            </div>
         </div>
         <div tagxxx="notes">
            <p xmlns:mml="http://www.w3.org/1998/Math/MathML">The authors declare no conflict of interest.</p>
         </div>
         <div tag="ref-list">
            <ul>
               <li tag="ref">
                  <a name="bib1"></a>
                  <div tagxxx="mixed-citation">Alba TW, Murugapiran SK, Blainey PC, Dodsworth JA, Thomas S, Woyke T et al. (
                     <span tagx="year">2014</span>). 
                     <span tagx="article-title">Insights into the global diversity and physiology of 'Aigarchaeota' through synergistic
                        analysis of single-cell genomes and metagenomes
                     </span>. 
                     <span tagx="source">Ninth Annual DOE Joint Genome Institute User Meeting</span>. Walnut Creek, CA, USA.
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib2"></a>
                  <div tagxxx="mixed-citation">Albers S-V, Meyer BH. (
                     <span tagx="year">2011</span>). 
                     <span tagx="article-title">The archaeal cell envelope</span>. 
                     <span tagx="source">Nat Rev Microbiol</span>
                     <span tagx="volume">9</span>: 
                     <span tagx="fpage">414</span>–426.
                     <span class="pub-id'">[21572458]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib3"></a>
                  <div tagxxx="mixed-citation">Anzaldi L, Skaar E. (
                     <span tagx="year">2010</span>). 
                     <span tagx="article-title">Overcoming the heme paradox: heme toxicity and tolerance in bacterial pathogens</span>. 
                     <span tagx="source">Infect Immun</span>
                     <span tagx="volume">78</span>: 
                     <span tagx="fpage">4977</span>–4989.
                     <span class="pub-id'">[20679437]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib4"></a>
                  <div tagxxx="mixed-citation">Bauman AJ, Simmonds PG. (
                     <span tagx="year">1969</span>). 
                     <span tagx="article-title">Fatty acids and polar lipids of extremely thermophilic bacterial masses from two Yellowstone
                        hot springs
                     </span>. 
                     <span tagx="source">J Bacteriol</span>
                     <span tagx="volume">98</span>: 
                     <span tagx="fpage">531</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib5"></a>
                  <div tagxxx="mixed-citation">Beam JP, Jay ZJ, Kozubal MA, Inskeep WP. (
                     <span tagx="year">2014</span>). 
                     <span tagx="article-title">Niche specialization of novel Thaumarchaeota to oxic and hypoxic acidic geothermal
                        springs of Yellowstone National Park
                     </span>. 
                     <span tagx="source">ISME J</span>
                     <span tagx="volume">8</span>: 
                     <span tagx="fpage">938</span>–951.
                     <span class="pub-id'">[24196321]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib6"></a>
                  <div tagxxx="mixed-citation">Behrens S, Rühland C, Inácio J, Huber H, Fonseca Á, Spencer-Martins I et al. (
                     <span tagx="year">2003</span>). 
                     <span tagx="article-title">
                        <i>In situ</i> accessibility of small-subunit rRNA of members of the domains 
                        <i>Bacteria, Archaea, and Eucarya</i> to Cy3-labeled oligonucleotide probes
                     </span>. 
                     <span tagx="source">Appl Environ Microbiol</span>
                     <span tagx="volume">69</span>: 
                     <span tagx="fpage">1748</span>–1758.
                     <span class="pub-id'">[12620867]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib7"></a>
                  <div tagxxx="mixed-citation">Berg I, Kockelkorn D, Buckel W, Fuchs G. (
                     <span tagx="year">2007</span>). 
                     <span tagx="article-title">A 3-hydroxypropionate/4-hydroxybutyrate autotrophic carbon dioxide assimilation pathway
                        in archaea
                     </span>. 
                     <span tagx="source">Science</span>
                     <span tagx="volume">318</span>: 
                     <span tagx="fpage">1782</span>–1786.
                     <span class="pub-id'">[18079405]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib8"></a>
                  <div tagxxx="mixed-citation">Berg I, Kockelkorn D, Ramos-Vera W, Say R, Zarzycki J, Hugler M et al. (
                     <span tagx="year">2010</span>). 
                     <span tagx="article-title">Autotrophic carbon fixation in archaea</span>. 
                     <span tagx="source">Nature Rev Microbiol</span>
                     <span tagx="volume">8</span>: 
                     <span tagx="fpage">447</span>–460.
                     <span class="pub-id'">[20453874]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib9"></a>
                  <div tagxxx="mixed-citation">Bernstein HC, Beam JP, Kozubal MA, Carlson RP, Inskeep WP. (
                     <span tagx="year">2013</span>). 
                     <span tagx="article-title">
                        <i>In situ</i> analysis of oxygen consumption and diffusive transport in high-temperature acidic
                        iron-oxide microbial mats
                     </span>. 
                     <span tagx="source">Environ Microbiol</span>
                     <span tagx="volume">15</span>: 
                     <span tagx="fpage">2360</span>–2370.
                     <span class="pub-id'">[23516993]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib10"></a>
                  <div tagxxx="mixed-citation">Blank CE, Cady SL, Pace NR. (
                     <span tagx="year">2002</span>). 
                     <span tagx="article-title">Microbial composition of near-boiling silica-depositing thermal springs throughout
                        Yellowstone national park
                     </span>. 
                     <span tagx="source">Appl Environ Microbiol</span>
                     <span tagx="volume">68</span>: 
                     <span tagx="fpage">5123</span>–5135.
                     <span class="pub-id'">[12324363]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib11"></a>
                  <div tagxxx="mixed-citation">Bräsen C, Esser D, Rauch B, Siebers B. (
                     <span tagx="year">2014</span>). 
                     <span tagx="article-title">Carbohydrate metabolism in Archaea: current insights into unusual enzymes and pathways
                        and their regulation
                     </span>. 
                     <span tagx="source">Microbiol Mol Biol Rev</span>
                     <span tagx="volume">78</span>: 
                     <span tagx="fpage">89</span>–175.
                     <span class="pub-id'">[24600042]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib12"></a>
                  <div tagxxx="mixed-citation">Brochier-Armanet C, Forterre P, Gribaldo S. (
                     <span tagx="year">2011</span>). 
                     <span tagx="article-title">Phylogeny and evolution of the Archaea: one hundred genomes later</span>. 
                     <span tagx="source">Curr Opin Microbiol</span>
                     <span tagx="volume">14</span>: 
                     <span tagx="fpage">274</span>–281.
                     <span class="pub-id'">[21632276]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib13"></a>
                  <div tagxxx="mixed-citation">Brock TD. (
                     <span tagx="year">1967</span>). 
                     <span tagx="article-title">Life at high temperatures</span>. 
                     <span tagx="source">Science</span>
                     <span tagx="volume">158</span>: 
                     <span tagx="fpage">1012</span>–1019.
                     <span class="pub-id'">[4861476]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib14"></a>
                  <div tagxxx="mixed-citation">Cole JK, Peacock JP, Dodsworth JA, Williams JA, Thompson DB, Dong H et al. (
                     <span tagx="year">2013</span>). 
                     <span tagx="article-title">Sediment microbial communities in Great Boiling Spring are controlled by temperature
                        and distinct from water communities
                     </span>. 
                     <span tagx="source">ISME J</span>
                     <span tagx="volume">7</span>: 
                     <span tagx="fpage">718</span>–729.
                     <span class="pub-id'">[23235293]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib15"></a>
                  <div tagxxx="mixed-citation">Colman DR, Jay ZJ, Inskeep WP, Rusch DB, Jennings RdeM, Maas KR et al. (
                     <span tagx="year">2015</span>). 
                     <span tagx="article-title">Characterization of novel, deeply-branching bacterial populations recovered from YNP
                        thermal metagenomes
                     </span>. 
                     <span tagx="source">ISME J</span> (in review).
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib16"></a>
                  <div tagxxx="mixed-citation">Costa KC, Navarro JB, Shock EL, Zhang CL, Soukup D, Hedlund BP. (
                     <span tagx="year">2009</span>). 
                     <span tagx="article-title">Microbiology and geochemistry of great boiling and mud hot springs in the United States
                        Great Basin
                     </span>. 
                     <span tagx="source">Extremophiles</span>
                     <span tagx="volume">13</span>: 
                     <span tagx="fpage">447</span>–459.
                     <span class="pub-id'">[19247786]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib17"></a>
                  <div tagxxx="mixed-citation">Cozen A, Weirauch M, Pollard K, Bernick D, Stuart J, Lowe T. (
                     <span tagx="year">2009</span>). 
                     <span tagx="article-title">Transcriptional map of respiratory versatility in the hyperthermophilic crenarchaeon
                        
                        <i>Pyrobaculum aerophilum</i>
                     </span>. 
                     <span tagx="source">J Bacteriol</span>
                     <span tagx="volume">191</span>: 
                     <span tagx="fpage">782</span>–794.
                     <span class="pub-id'">[19047344]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib18"></a>
                  <div tagxxx="mixed-citation">De Leon KB, Gerlach R, Peyton BM, Fields MW. (
                     <span tagx="year">2013</span>). 
                     <span tagx="article-title">Archaeal and bacterial communities in three alkaline hot springs in Heart Lake Geyser
                        Basin, Yellowstone National Park
                     </span>. 
                     <span tagx="source">Front Microbiol</span>
                     <span tagx="volume">4</span>: 
                     <span tagx="fpage">330</span>.
                     <span class="pub-id'">[24282404]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib19"></a>
                  <div tagxxx="mixed-citation">Dibrova D, Galperin M, Mulkidjanian A. (
                     <span tagx="year">2014</span>). 
                     <span tagx="article-title">Phylogenomic reconstruction of archaeal fatty acid metabolism</span>. 
                     <span tagx="source">Environ Microbiol</span>
                     <span tagx="volume">16</span>: 
                     <span tagx="fpage">907</span>–918.
                     <span class="pub-id'">[24818264]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib20"></a>
                  <div tagxxx="mixed-citation">Dobbek H, Gremer L, Meyer O, Huber H. (
                     <span tagx="year">1999</span>). 
                     <span tagx="article-title">Crystal structure and mechanism of CO dehydrogenase, a molybdo iron-sulfur flavoprotein
                        containing S-selanylcysteine
                     </span>. 
                     <span tagx="source">Proc Natl Acad Sci USA</span>
                     <span tagx="volume">96</span>: 
                     <span tagx="fpage">8884</span>–8889.
                     <span class="pub-id'">[10430865]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib21"></a>
                  <div tagxxx="mixed-citation">Dobbek H, Gremer L, Kiefersauer R, Huber R, Meyer O. (
                     <span tagx="year">2002</span>). 
                     <span tagx="article-title">Catalysis at a dinuclear [CuSMo(==O)OH] cluster in a CO dehydrogenase resolved at
                        1.1-Å resolution
                     </span>. 
                     <span tagx="source">Proc Natl Acad Sci USA</span>
                     <span tagx="volume">99</span>: 
                     <span tagx="fpage">15971</span>–15976.
                     <span class="pub-id'">[12475995]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib22"></a>
                  <div tagxxx="mixed-citation">Draper W, Crosby D. (
                     <span tagx="year">1983</span>). 
                     <span tagx="article-title">The photochemical generation of hydrogen-peroxide in natural-waters</span>. 
                     <span tagx="source">Arch Environ Contam Toxicol</span>
                     <span tagx="volume">12</span>: 
                     <span tagx="fpage">121</span>–126.
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib23"></a>
                  <div tagxxx="mixed-citation">Ettema T, Lindas A, Bernander R. (
                     <span tagx="year">2011</span>). 
                     <span tagx="article-title">An actin-based cytoskeleton in archaea</span>. 
                     <span tagx="source">Mol Microbiol</span>
                     <span tagx="volume">80</span>: 
                     <span tagx="fpage">1052</span>–1061.
                     <span class="pub-id'">[21414041]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib24"></a>
                  <div tagxxx="mixed-citation">Giovannoni SJ, Tripp HJ, Givan S, Podar M, Vargin KL, Baptista D et al. (
                     <span tagx="year">2005</span>). 
                     <span tagx="article-title">Genome streamlining in a cosmopolitan oceanic bacterium</span>. 
                     <span tagx="source">Science</span>
                     <span tagx="volume">309</span>: 
                     <span tagx="fpage">1242</span>–1245.
                     <span class="pub-id'">[16109880]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib25"></a>
                  <div tagxxx="mixed-citation">Giovannoni SJ, Thrash JC, Temperton B. (
                     <span tagx="year">2014</span>). 
                     <span tagx="article-title">Implications of streamlining theory for microbial ecology</span>. 
                     <span tagx="source">ISME J</span>
                     <span tagx="volume">8</span>: 
                     <span tagx="fpage">1553</span>–1565.
                     <span class="pub-id'">[24739623]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib26"></a>
                  <div tagxxx="mixed-citation">Hedlund BP, Dodsworth JA, Murugapiran SK, Rinke C, Woyke T. (
                     <span tagx="year">2014</span>). 
                     <span tagx="article-title">Impact of single-cell genomics and metagenomics on the emerging view of extremophile
                        “microbial dark matter”
                     </span>. 
                     <span tagx="source">Extremophiles</span>
                     <span tagx="volume">5</span>: 
                     <span tagx="fpage">865</span>–875.
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib27"></a>
                  <div tagxxx="mixed-citation">Henle ES, Linn S. (
                     <span tagx="year">1997</span>). 
                     <span tagx="article-title">Formation, prevention, and repair of DNA damage by iron/hydrogen peroxide</span>. 
                     <span tagx="source">J Biol Chem</span>
                     <span tagx="volume">272</span>: 
                     <span tagx="fpage">19095</span>–19098.
                     <span class="pub-id'">[9235895]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib28"></a>
                  <div tagxxx="mixed-citation">Hirayama H, Takai K, Inagaki F, Yamato Y, Suzuki M, Nealson K et al. (
                     <span tagx="year">2005</span>). 
                     <span tagx="article-title">Bacterial community shift along a subsurface geothermal water stream in a Japanese
                        gold mine
                     </span>. 
                     <span tagx="source">Extremophiles</span>
                     <span tagx="volume">9</span>: 
                     <span tagx="fpage">169</span>–184.
                     <span class="pub-id'">[15776216]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib29"></a>
                  <div tagxxx="mixed-citation">Huber H, Gallenberger M, Jahn U, Eylert E, Berg I, Kockelkorn D et al. (
                     <span tagx="year">2008</span>). 
                     <span tagx="article-title">A dicarboxylate/4-hydroxybutyrate autotrophic carbon assimilation cycle in the hyperthermophilic
                        Archaeum 
                        <i>Ignicoccus hospitalis</i>
                     </span>. 
                     <span tagx="source">Proc Natl Acad Sci USA</span>
                     <span tagx="volume">105</span>: 
                     <span tagx="fpage">7851</span>–7856.
                     <span class="pub-id'">[18511565]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib30"></a>
                  <div tagxxx="mixed-citation">Hyatt D, Chen G, LoCascio P, Land M, Larimer F, Hauser L. (
                     <span tagx="year">2010</span>). 
                     <span tagx="article-title">Prodigal: prokaryotic gene recognition and translation initiation site identification</span>. 
                     <span tagx="source">BMC Bioinformat</span>
                     <span tagx="volume">11</span>: 
                     <span tagx="fpage">119</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib31"></a>
                  <div tagxxx="mixed-citation">Inskeep WP, Jay Z J, Tringe SG, Herrgard M, Rusch D B et al. (
                     <span tagx="year">2013</span>). 
                     <span tagx="article-title">The YNP Metagenome Project: environmental parameters responsible for microbial distribution
                        in the Yellowstone geothermal ecosystem
                     </span>. 
                     <span tagx="source">Front Microbiol</span>
                     <span tagx="volume">4</span>: 
                     <span tagx="fpage">67</span>.
                     <span class="pub-id'">[23653623]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib32"></a>
                  <div tagxxx="mixed-citation">Jahnke LL, Eder W, Huber R, Hope JM, Hinrichs K-U, Hayes JM et al. (
                     <span tagx="year">2001</span>). 
                     <span tagx="article-title">Signature lipids and stable carbon isotope analyses of Octopus Spring hyperthermophilic
                        communities compared with those of aquificales representatives
                     </span>. 
                     <span tagx="source">Appl Environ Microbiol</span>
                     <span tagx="volume">67</span>: 
                     <span tagx="fpage">5179</span>–5189.
                     <span class="pub-id'">[11679343]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib33"></a>
                  <div tagxxx="mixed-citation">Jarrell K, Albers S. (
                     <span tagx="year">2012</span>). 
                     <span tagx="article-title">The archaellum: an old motility structure with a new name</span>. 
                     <span tagx="source">Trends Microbiol</span>
                     <span tagx="volume">20</span>: 
                     <span tagx="fpage">307</span>–312.
                     <span class="pub-id'">[22613456]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib34"></a>
                  <div tagxxx="mixed-citation">Jay ZJ, Rusch DB, Romine MF, Beam JP, Inskeep WP. (
                     <span tagx="year">2014</span>). 
                     <span tagx="article-title">Geochemical constraints on the distribution and function of thermoproteales populations
                        in Yellowstone national park
                     </span>. 
                     <span tagx="source">American Geophysical Union Conference</span>. San Francisco, CA, USA.
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib35"></a>
                  <div tagxxx="mixed-citation">Kan J, Clingenpeel S, Macur RE, Inskeep WP, Lovalvo S, Varley J et al. (
                     <span tagx="year">2011</span>). 
                     <span tagx="article-title">Archaea in Yellowstone Lake</span>. 
                     <span tagx="source">ISME J</span>
                     <span tagx="volume">5</span>: 
                     <span tagx="fpage">1784</span>–1795.
                     <span class="pub-id'">[21544103]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib36"></a>
                  <div tagxxx="mixed-citation">Khelifi N, Grossi V, Hamdi M, Dolla A, Tholozan J, Ollivier B et al. (
                     <span tagx="year">2010</span>). 
                     <span tagx="article-title">Anaerobic oxidation of fatty acids and alkenes by the hyperthermophilic sulfate-reducing
                        archaeon 
                        <i>Archaeoglobus fulgidus</i>
                     </span>. 
                     <span tagx="source">Appl Environ Microbiol</span>
                     <span tagx="volume">76</span>: 
                     <span tagx="fpage">3057</span>–3060.
                     <span class="pub-id'">[20305028]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib37"></a>
                  <div tagxxx="mixed-citation">Kochetkova T, Rusanov I, Pimenov N, Kolganova T, Lebedinsky A, Bonch-Osmolovskaya
                     E et al. (
                     <span tagx="year">2011</span>). 
                     <span tagx="article-title">Anaerobic transformation of carbon monoxide by microbial communities of Kamchatka
                        hot springs
                     </span>. 
                     <span tagx="source">Extremophiles</span>
                     <span tagx="volume">15</span>: 
                     <span tagx="fpage">319</span>–325.
                     <span class="pub-id'">[21387195]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib38"></a>
                  <div tagxxx="mixed-citation">Kozubal MA, Romine M, Jennings RdeM, Jay ZJ, Tringe SG, Rusch DB et al. (
                     <span tagx="year">2012</span>). 
                     <span tagx="article-title">Geoarchaeota: a new candidate phylum in the Archaea from high-temperature acidic iron
                        mats in Yellowstone National Park
                     </span>. 
                     <span tagx="source">ISME J</span>
                     <span tagx="volume">7</span>: 
                     <span tagx="fpage">622</span>–634.
                     <span class="pub-id'">[23151644]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib39"></a>
                  <div tagxxx="mixed-citation">Kubo K, Knittel K, Amann R, Fukui M, Matsuura K. (
                     <span tagx="year">2011</span>). 
                     <span tagx="article-title">Sulfur-metabolizing bacterial populations in microbial mats of the Nakabusa hot spring,
                        Japan
                     </span>. 
                     <span tagx="source">Syst Appl Microbiol</span>
                     <span tagx="volume">34</span>: 
                     <span tagx="fpage">293</span>–302.
                     <span class="pub-id'">[21353426]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib40"></a>
                  <div tagxxx="mixed-citation">Li M, Copeland A, Han J. (
                     <span tagx="year">2011</span>). 
                     <span tagx="article-title">DUK - A fast ad Efficient Kmer Based Sequence Matching Tool. Lawrence Berkeley National
                        Laboratory
                     </span>. LBNL Paper LBNL-4516E-Poster p.
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib41"></a>
                  <div tagxxx="mixed-citation">Liebensteiner M, Pinkse M, Schaap P, Stams A, Lomans B. (
                     <span tagx="year">2013</span>). 
                     <span tagx="article-title">Archaeal (Per)Chlorate reduction at high temperature: an interplay of biotic and abiotic
                        reactions
                     </span>. 
                     <span tagx="source">Science</span>
                     <span tagx="volume">340</span>: 
                     <span tagx="fpage">85</span>–87.
                     <span class="pub-id'">[23559251]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib42"></a>
                  <div tagxxx="mixed-citation">Markowitz V, Chen I-M, Palaniappan K, Chu K, Szeto E, Grechkin Y et al. (
                     <span tagx="year">2012</span>). 
                     <span tagx="article-title">IMG: the integrated microbial genomes database and comparative analysis system</span>. 
                     <span tagx="source">Nucleic Acids Res</span>
                     <span tagx="volume">40</span>: 
                     <span tagx="fpage">D115</span>–D122.
                     <span class="pub-id'">[22194640]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib43"></a>
                  <div tagxxx="mixed-citation">McClure R, Balasubramanian D, Sun Y, Bobrovskyy M, Sumby P, Genco C et al. (
                     <span tagx="year">2013</span>). 
                     <span tagx="article-title">Computational analysis of bacterial RNA-Seq data</span>. 
                     <span tagx="source">Nucleic Acids Res</span>
                     <span tagx="volume">41</span>: 
                     <span tagx="fpage">e140</span>.
                     <span class="pub-id'">[23716638]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib44"></a>
                  <div tagxxx="mixed-citation">Meyer-Dombard DR, Swingley W, Raymond J, Havig J, Shock EL, Summons RE. (
                     <span tagx="year">2011</span>). 
                     <span tagx="article-title">Hydrothermal ecotones and streamer biofilm communities in the Lower Geyser Basin,
                        Yellowstone National Park
                     </span>. 
                     <span tagx="source">Environ Microbiol</span>
                     <span tagx="volume">13</span>: 
                     <span tagx="fpage">2216</span>–2231.
                     <span class="pub-id'">[21453405]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib45"></a>
                  <div tagxxx="mixed-citation">Morris R, Schmidt T. (
                     <span tagx="year">2013</span>). 
                     <span tagx="article-title">Shallow breathing: bacterial life at low O
                        <div tagxxx="sub">2</div>
                     </span>. 
                     <span tagx="source">Nat Rev Microbiol</span>
                     <span tagx="volume">11</span>: 
                     <span tagx="fpage">205</span>–212.
                     <span class="pub-id'">[23411864]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib46"></a>
                  <div tagxxx="mixed-citation">Nunoura T, Hirayama H, Takami H, Oida H, Nishi S, Shimamura S et al. (
                     <span tagx="year">2005</span>). 
                     <span tagx="article-title">Genetic and functional properties of uncultivated thermophilic crenarchaeotes from
                        a subsurface gold mine as revealed by analysis of genome fragments
                     </span>. 
                     <span tagx="source">Environ Microbiol</span>
                     <span tagx="volume">7</span>: 
                     <span tagx="fpage">1967</span>–1984.
                     <span class="pub-id'">[16309394]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib47"></a>
                  <div tagxxx="mixed-citation">Nunoura T, Takaki Y, Kakuta J, Nishi S, Sugahara J, Kazama H et al. (
                     <span tagx="year">2011</span>). 
                     <span tagx="article-title">Insights into the evolution of Archaea and eukaryotic protein modifier systems revealed
                        by the genome of a novel archaeal group
                     </span>. 
                     <span tagx="source">Nucleic Acids Res</span>
                     <span tagx="volume">39</span>: 
                     <span tagx="fpage">3204</span>–3223.
                     <span class="pub-id'">[21169198]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib48"></a>
                  <div tagxxx="mixed-citation">Petasne R, Zika R. (
                     <span tagx="year">1997</span>). 
                     <span tagx="article-title">Hydrogen peroxide lifetimes in south Florida coastal and offshore waters</span>. 
                     <span tagx="source">Mar Chem</span>
                     <span tagx="volume">56</span>: 
                     <span tagx="fpage">215</span>–225.
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib49"></a>
                  <div tagxxx="mixed-citation">Podgorsek L, Imhoff JF. (
                     <span tagx="year">1999</span>). 
                     <span tagx="article-title">Tetrathionate production by sulfur oxidizing bacteria and the role of tetrathionate
                        in the sulfur cycle of Baltic Sea sediments
                     </span>. 
                     <span tagx="source">Aquat Microb Ecol</span>
                     <span tagx="volume">17</span>: 
                     <span tagx="fpage">255</span>–265.
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib50"></a>
                  <div tagxxx="mixed-citation">Rayman K, Forterre P, Brochier-Armanet C, Gribaldo S. (
                     <span tagx="year">2014</span>). 
                     <span tagx="article-title">Global phylogenomic analysis disentangles the complex evolutionary history of DNA
                        replication in archaea
                     </span>. 
                     <span tagx="source">Genome Biol Evol</span>
                     <span tagx="volume">6</span>: 
                     <span tagx="fpage">192</span>–212.
                     <span class="pub-id'">[24398374]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib51"></a>
                  <div tagxxx="mixed-citation">Reysenbach A-L, Wickham GS, Pace NR. (
                     <span tagx="year">1994</span>). 
                     <span tagx="article-title">Phylogenetic analysis of the hyperthermophilic pink filament community in Octopus
                        Spring, Yellowstone National Park
                     </span>. 
                     <span tagx="source">Appl Environ Microbiol</span>
                     <span tagx="volume">60</span>: 
                     <span tagx="fpage">2113</span>–2119.
                     <span class="pub-id'">[7518219]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib52"></a>
                  <div tagxxx="mixed-citation">Rinke C, Schwientek P, Sczyrba A, Ivanova N, Anderson I, Cheng J et al. (
                     <span tagx="year">2013</span>). 
                     <span tagx="article-title">Insights into the phylogeny and coding potential of microbial dark matter</span>. 
                     <span tagx="source">Nature</span>
                     <span tagx="volume">499</span>: 
                     <span tagx="fpage">431</span>–437.
                     <span class="pub-id'">[23851394]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib53"></a>
                  <div tagxxx="mixed-citation">Say RF, Fuchs G. (
                     <span tagx="year">2010</span>). 
                     <span tagx="article-title">Fructose 1,6-bisphosphate aldolase/phosphatase may be an ancestral gluconeogenic enzyme</span>. 
                     <span tagx="source">Nature</span>
                     <span tagx="volume">464</span>: 
                     <span tagx="fpage">1077</span>–1081.
                     <span class="pub-id'">[20348906]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib54"></a>
                  <div tagxxx="mixed-citation">Siebers B, Zaparty M, Raddatz G, Tjaden B, Albers S-V, Bell SD et al. (
                     <span tagx="year">2011</span>). 
                     <span tagx="article-title">The complete genome sequence of 
                        <i>Thermoproteus tenax</i>: a physiologically versatile member of the 
                        <i>Crenarchaeota</i>
                     </span>. 
                     <span tagx="source">PLoS One</span>
                     <span tagx="volume">6</span>: 
                     <span tagx="fpage">e24222</span>.
                     <span class="pub-id'">[22003381]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib55"></a>
                  <div tagxxx="mixed-citation">Setchell WA. (
                     <span tagx="year">1903</span>). 
                     <span tagx="article-title">The upper temperature limits of life</span>. 
                     <span tagx="source">Science</span>
                     <span tagx="volume">17</span>: 
                     <span tagx="fpage">934</span>–937.
                     <span class="pub-id'">[17844131]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib56"></a>
                  <div tagxxx="mixed-citation">Spang A, Poehlein A, Offre P, Zumbrägel S, Haider S, Rychlik N et al. (
                     <span tagx="year">2012</span>). 
                     <span tagx="article-title">The genome of the ammonia-oxidizing 
                        <i>Candidatus</i>
                        <i>Nitrososphaera gargensis</i>: insights into metabolic versatility and environmental adaptations
                     </span>. 
                     <span tagx="source">Environ Microbiol</span>
                     <span tagx="volume">14</span>: 
                     <span tagx="fpage">3122</span>–3145.
                     <span class="pub-id'">[23057602]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib57"></a>
                  <div tagxxx="mixed-citation">Stoecker K, Dorninger C, Daims H, Wagner M. (
                     <span tagx="year">2010</span>). 
                     <span tagx="article-title">Double labeling of oligonucleotide probes for fluorescence 
                        <i>in situ</i> hybridization (DOPE-FISH) improves signal intensity and increases rRNA accessibility
                     </span>. 
                     <span tagx="source">Appl Environ Microbiol</span>
                     <span tagx="volume">76</span>: 
                     <span tagx="fpage">922</span>–926.
                     <span class="pub-id'">[19966029]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib58"></a>
                  <div tagxxx="mixed-citation">Stahl DA, Lane DJ, Olsen GJ, Pace NR. (
                     <span tagx="year">1985</span>). 
                     <span tagx="article-title">Characterization of a Yellowstone hot spring microbial community by 5S rRNA seqences</span>. 
                     <span tagx="source">Appl Environ Microbiol</span>
                     <span tagx="volume">49</span>: 
                     <span tagx="fpage">1379</span>–1384.
                     <span class="pub-id'">[2409920]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib59"></a>
                  <div tagxxx="mixed-citation">Takacs-Vesbach C, Inskeep W, Jay Z, Herrgard M, Rusch D, Tringe S et al. (
                     <span tagx="year">2013</span>). 
                     <span tagx="article-title">Metagenome sequence analysis of filamentous microbial communities obtained from geochemically
                        distinct geothermal channels reveals specialization of three Aquificales lineages
                     </span>. 
                     <span tagx="source">Front Microbiol</span>
                     <span tagx="volume">4</span>: 
                     <span tagx="fpage">84</span>.
                     <span class="pub-id'">[23755042]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib60"></a>
                  <div tagxxx="mixed-citation">Tamura K, Peterson D, Peterson N, Stecher G, Nei M, Kumar S. (
                     <span tagx="year">2011</span>). 
                     <span tagx="article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary
                        distance, and maximum parsimony methods
                     </span>. 
                     <span tagx="source">Mol Biol Evol</span>
                     <span tagx="volume">28</span>: 
                     <span tagx="fpage">2731</span>–2739.
                     <span class="pub-id'">[21546353]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib61"></a>
                  <div tagxxx="mixed-citation">Vick TJ, Dodsworth JA, Costa KC, Shock EL, Hedlund BP. (
                     <span tagx="year">2010</span>). 
                     <span tagx="article-title">Microbiology and geochemistry of Little Hot Creek, a hot spring environment in the
                        Long Valley Caldera
                     </span>. 
                     <span tagx="source">Geobiology</span>
                     <span tagx="volume">8</span>: 
                     <span tagx="fpage">140</span>–154.
                     <span class="pub-id'">[20002204]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="bib62"></a>
                  <div tagxxx="mixed-citation">Wolin E, Wolin M, Wolfe R. (
                     <span tagx="year">1963</span>). 
                     <span tagx="article-title">Formation of methane by bacterial extracts</span>. 
                     <span tagx="source">J Biol Chem</span>
                     <span tagx="volume">238</span>: 
                     <span tagx="fpage">2882</span>–2886.
                     <span class="pub-id'">[14063318]</span>
                  </div>
               </li>
            </ul>
         </div>
         <div id="sup1">
            <h2>Supplementary Material</h2>
            <div tagxxx="supplementary-material">
               <span class="label">Supplementary Movie S1</span>
               <div tagxxx="media">
                  <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div tagxxx="supplementary-material">
               <span class="label">Supplementary Figure S1</span>
               <div tagxxx="media">
                  <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div tagxxx="supplementary-material">
               <span class="label">Supplementary Figure S2</span>
               <div tagxxx="media">
                  <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div tagxxx="supplementary-material">
               <span class="label">Supplementary Figure S3</span>
               <div tagxxx="media">
                  <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div tagxxx="supplementary-material">
               <span class="label">Supplementary Table S1</span>
               <div tagxxx="media">
                  <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div tagxxx="supplementary-material">
               <span class="label">Supplementary Table S2</span>
               <div tagxxx="media">
                  <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div tagxxx="supplementary-material">
               <span class="label">Supplementary Table S3</span>
               <div tagxxx="media">
                  <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div tagxxx="supplementary-material">
               <span class="label">Supplementary Table S4</span>
               <div tagxxx="media">
                  <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div tagxxx="supplementary-material">
               <span class="label">Supplementary Table S5</span>
               <div tagxxx="media">
                  <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div tagxxx="supplementary-material">
               <span class="label">Supplementary Table S6</span>
               <div tagxxx="media">
                  <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div tagxxx="supplementary-material">
               <span class="label">Supplementary Table S7</span>
               <div tagxxx="media">
                  <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div tagxxx="supplementary-material">
               <span class="label">Supplementary Table S8</span>
               <div tagxxx="media">
                  <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div tagxxx="supplementary-material">
               <span class="label">Supplementary Table S9</span>
               <div tagxxx="media">
                  <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
            <div tagxxx="supplementary-material">
               <span class="label">Supplementary Movie S1 Legend</span>
               <div tagxxx="media">
                  <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <p>Click here for additional data file.</p>
                  </caption>
               </div>
            </div>
         </div>
      </div>
      <div tagxxx="floats-group">
         <div tagxxx="fig">
            <span class="label">Figure 1</span>
            <caption xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>Photograph of a pink filamentous ‘streamer' community found in the outflow channel
                  of OS (24 July 2014).
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <span class="label">Figure 2</span>
            <caption xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>NWF-PCA (
                  <b>a</b>) of the aigarchaeon 
                  <i>de novo</i> assembly (
                  <i>Calditenuis aerorheumensis</i>, red) from OS compared with 
                  <i>Candidatus</i> ‘
                  <i>Caldiarchaeum subterraneum</i>' (yellow), and single-cell Aigarchaeota genomes from Great Boiling Spring (GBS):
                  aigarchaeon B22 (blue) and aigarchaeon J15 (purple). The G+C content (%) of scaffolds
                  (
                  <b>b</b>) from 
                  <i>C. aerorheumensis</i> (red), GBS aigarchaeon B22 (blue) and GBS aigarchaeon J15 (purple) plotted as a function
                  of decreasing scaffold length. The cumulative scaffold length is on the secondary
                  
                  <i>y</i>-axis (red, blue and purple lines represent 
                  <i>C. aerorheumensi</i>, GBS aigarchaeon B22 and GBS aigarchaeon J15, respectively). Deduced protein sequence
                  comparison (
                  <b>c</b>) of 
                  <i>Calditenuis aerorheumensis</i> versus aigarchaeon B22 (blue), aigarchaeon J15 (purple) and 
                  <i>Caldiarchaeum subterraneum</i> (yellow).
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <span class="label">Figure 3</span>
            <caption xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>FISH composite images (
                  <b>a</b>, 
                  <b>b</b>) of OS filamentous streamer thin sections (5 μm) hybridized with the following DOPE
                  probes: Cy3-Aig800 targeting Group 1A aigarchaeota (purple), Cy5-Arch915 (blue) and
                  6-FAM-Aqi338 (green). The aigarchaeota filaments are purple due to the overlapping
                  emission from the Cy3 (red) and Cy5 (blue)-labeled probes. FISH composite image (
                  <b>c</b>, 
                  <b>d</b>) with general probes Aqi338 (green), Eub338 (red) and Arch915 (blue). Scanning electron
                  micrographs (
                  <b>e</b>, 
                  <b>f</b>) of OS streamer community, showing filamentous organisms and condensed EPS and/or
                  cellular appendages.
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <span class="label">Figure 4</span>
            <caption xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>Maximum likelihood 16S rRNA gene phylogenetic tree (1328 unambiguously aligned nucleotide
                  positions) of the candidate phylum ‘Aigarchaeota', Thaumarchaeota and Crenarchaeota
                  rooted with 
                  <i>Thermus thermophilus</i>. Bootstrap (BS) values from 100 replications are included at the nodes; BS values
                  &lt;50% were not included. Scale bar represents the number of nucleotide substitutions
                  per site.
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <span class="label">Figure 5</span>
            <caption xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>Maximum likelihood phylogenetic tree of 16 concatenated ribosomal proteins (2249 amino-acid
                  positions) from 
                  <i>Calditenuis aerorheumensis</i> compared with other Aigarchaeota, Thaumarchaeota, Crenarchaeota and 
                  <i>Eukarya</i> included as an outgroup. Numbers at the nodes represent bootstrap values from 100
                  replications and the scale bar represents the estimated amino-acid substitutions per
                  site.
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <span class="label">Figure 6</span>
            <caption xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>Detailed metabolic reconstruction of 
                  <i>Calditenuis aerorheumensis</i> from OS based on environmental genome sequence and standardized mRNA reads (reported
                  as RPKM; 
                  <span ref-type="bibr" rid="bib43">
                     <sup>
                        <a href="#bib43">McClure et al., 2013</a>
                     </sup>
                  </span>). Numbers in colored boxes correspond to enzymes in specific pathways (
                  <span ref-type="supplementary-material" rid="sup1">Supplementary Table S6</span>). ABC, adenosine triphosphate binding cassette; BC, blue copper protein (membrane
                  bound); CO?, potential aerobic CO dehydrogenase complex; DOP, dissolved organic phosphorous;
                  ED, Entner–Doudoroff; EMP, Embden–Meyerhof–Parnas glycolysis; GS, glyoxylate shunt/bypass;
                  HCO, HCO complex (subunits I+III, II); II, succinate dehydrogenase complex; NUO, NADH
                  dehydrogenase complex; PPP, oxidative pentose phosphate pathway; PRX, 1 and 2-cysteine
                  peroxiredoxin; SOD, superoxide dismutase; TCA, tricarboxylic acid cycle; TTR, tetrathionate
                  reductase complex.
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div tagxxx="fig">
            <span class="label">Figure 7</span>
            <caption xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>The 3-hydroxypropionate/4-hydroxybutyrate CO
                  <div tagxxx="sub">2</div> fixation cycle that is potentially operative in 
                  <i>Calditenuis aerorheumensis</i> (modified from 
                  <span ref-type="bibr" rid="bib7">
                     <sup>
                        <a href="#bib7">Berg et al., 2007</a>
                     </sup>
                  </span>). mRNA reads from OS mapped to this pathway are reported as RPKM to take into account
                  varying transcript lengths of the corresponding genes. Enzymes in the cycle: 
                  <b>1</b>, acetyl-CoA carboxylase; 
                  <b>2</b>, malonyl-CoA reductase; 
                  <b>3</b>, malonate semialdehyde reductase; 
                  <b>4</b>, 3-hydroxypropionyl-CoA synthetase; 
                  <b>5</b>, 3-hydroxypropionyl-CoA dehydratase; 
                  <b>6</b>, acryloyl-CoA reductase; 
                  <b>7</b>, propionyl-CoA carboxylase; 
                  <b>8</b>, methylmalonyl-CoA epimerase; 
                  <b>9</b>, methylmalonyl-CoA mutase; 
                  <b>10</b>, succinyl-CoA reductase; 
                  <b>11</b>, succinate semialdehyde reductase; 
                  <b>12</b>, 4-hydroxybutyryl-CoA synthetase; 
                  <b>13</b>, 4-hydroxybutyryl-CoA dehydratase; 
                  <b>14</b>, crotonyl-CoA hydratase; 
                  <b>15</b>, 3-hydroxybutyryl-CoA dehydrogenase; 
                  <b>16</b>, acetoacetyl-CoA β-ketothiolase. Note: 
                  <b>1</b>, missing the biotin carboxylase subunit of bifunctional acetyl/propionyl-CoA carboxylase.
                  Genes and corresponding RPKM values are presented in 
                  <span ref-type="supplementary-material" rid="sup1">Supplementary Table S8</span>.
               </p>
            </caption>
            <div tagxxx="graphic"></div>
         </div>
         <div id="tbl1">
            <span class="label">Table 1</span>
            <caption xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Amino acid, cofactor and vitamin biosynthesis pathways identified in 
                  <i>Calditenuis aerorheumensis</i>
               </div>
            </caption>
            <table xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <colgroup>
                  <col></col>
                  <col></col>
                  <col></col>
               </colgroup>
               <thead>
                  <tr>
                     <th>
                        <i>Pathway</i>
                     </th>
                     <th>
                        <i>Auxotroph</i>
                     </th>
                     <th>
                        <i>Transporter locus</i>
                     </th>
                  </tr>
               </thead>
               <div tagxxx="tbody">
                  <tr>
                     <td>Aromatic AA</td>
                     <td>N</td>
                     <td>—</td>
                  </tr>
                  <tr>
                     <td>Branched-chain AA</td>
                     <td>N</td>
                     <td>Many</td>
                  </tr>
                  <tr>
                     <td>Heme</td>
                     <td>Y</td>
                     <td>_00183–_00186</td>
                  </tr>
                  <tr>
                     <td>F
                        <div tagxxx="sub">420</div>
                     </td>
                     <td>N</td>
                     <td>—</td>
                  </tr>
                  <tr>
                     <td>Thiamin (B
                        <div tagxxx="sub">1</div>)
                     </td>
                     <td>Y</td>
                     <td>_00602</td>
                  </tr>
                  <tr>
                     <td>Riboflavin (B
                        <div tagxxx="sub">2</div>)
                     </td>
                     <td>Y</td>
                     <td>_00369–_00370</td>
                  </tr>
                  <tr>
                     <td>Niacin (B
                        <div tagxxx="sub">3</div>)
                     </td>
                     <td>Y</td>
                     <td>_01243–_01244</td>
                  </tr>
                  <tr>
                     <td>Pantothenate (B
                        <div tagxxx="sub">5</div>)
                     </td>
                     <td>N</td>
                     <td>—</td>
                  </tr>
                  <tr>
                     <td>Pyroxidine (B
                        <div tagxxx="sub">6</div>)
                     </td>
                     <td>N</td>
                     <td>—</td>
                  </tr>
                  <tr>
                     <td>Biotin (B
                        <div tagxxx="sub">7</div>)
                     </td>
                     <td>Y</td>
                     <td>_00296–_00298</td>
                  </tr>
                  <tr>
                     <td>Cobalamin (B
                        <div tagxxx="sub">12</div>)
                     </td>
                     <td>Y</td>
                     <td>Salvage pathway</td>
                  </tr>
               </div>
            </table>
            <table-wrap-foot xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div class="fn" id="t1-fn1">
                  <p>Abbreviations: AA, amino acid; N, no; Y, yes. '—' indicates no transporter.</p>
               </div>
            </table-wrap-foot>
         </div>
      </div>
   </div>
</html>