<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <div xmlns="" tagx="article">
      <head>
         <style type="text/css"> table, tr, td {border : thin;}</style>
         <span class="citation_publisher">Public Library of Science</span>
         <h2 class="citation-title">Rapid Molecular Assays for the Detection of Yellow Fever Virus in Low-Resource Settings</h2>
         <span class="doi">10.1371/journal.pntd.0002730</span>
         <div class="contrib-group">contrib: 
            	  CONT
            	  
            <span class="citation_author">Camille Escadafal</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Oumar Faye</span>
            <span class="citation_author_institution_ref">2</span>
            <span class="citation_author">Amadou Alpha Sall</span>
            <span class="citation_author_institution_ref">2</span>
            <span class="citation_author">Ousmane Faye</span>
            <span class="citation_author_institution_ref">2</span>
            <span class="citation_author">Manfred Weidmann</span>
            <span class="citation_author_institution_ref">3</span>
            <span class="citation_author">Oliver Strohmeier</span>
            <span class="citation_author_institution_ref">5</span>
            <span class="citation_author">Felix von Stetten</span>
            <span class="citation_author_institution_ref">5</span>
            <span class="citation_author">Josef Drexler</span>
            <span class="citation_author_institution_ref">7</span>
            <span class="citation_author">Michael Eberhard</span>
            <span class="citation_author_institution_ref">7</span>
            <span class="citation_author">Matthias Niedrig</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Pranav Patel</span>
            <span class="citation_author_institution_ref">1</span>
            	  CONT
            	  
            <div tagxxx="contrib">
               <span class="name'">
                  <span tagx="surname">Charrel</span>
                  <span tagx="given-names">Remi</span>,
               </span>
               <div tagxxx="role">Editor</div>
               <span ref-type="aff" rid="edit1"></span>
            </div>
         </div>
         <span class="email">[escadafal@rki.de]</span>
         <span class="conflict">[I have read the journal's policy and have the following conflicts: Josef Drexler and
            Michael Eberhard are employees of the commercial company QIAGEN Lake Constance GmbH.
            The affiliations to this company do not alter our adherence to all PLOS NTDs policies
            on sharing data and materials. The authors have declared that no other competing interests
            exist.]
         </span>
         <span class="contribs">[Conceived and designed the experiments: CE MW PP. Performed the experiments: CE PP.
            Analyzed the data: CE MN PP. Contributed reagents/materials/analysis tools: OumF AAS
            OusF MW OS FvS JD ME. Wrote the paper: CE AAS PP. Designed the software used in analysis:
            JD ME.]
         </span>
         <span class="citation_pubdate">[2014-3-6]</span>
         <span class="citation_copyright">[Copyright 2014: Escadafal et al]</span>
         <span class="citation_licence">[Licence This is an open-access article distributed under the terms of the Creative
            Commons Attribution License, which permits unrestricted use, distribution, and reproduction
            in any medium, provided the original author and source are credited.]
         </span>
         <span class="citation_funding">[This work was funded by the Federal Ministry of Education and Research (BMBF) under
            the research program for civil security of the German Federal Government, as part
            of the high-tech strategy for “Szenario-Orientierte Notfall-Diagnostik für den Feldeinsatz”
            (S.O.N.D.E.) (Project No: 13N10117), the Deutsche Gesellschaft für Internationale
            Zusammenarbeit (GIZ) Pandemic Preparedness Initiative (VN 81140270) and National Institutes
            of Health (NIH) (Grant Number 5R01A 1069145). The funders had no role in study design,
            data collection and analysis, decision to publish, or preparation of the manuscript.]
         </span>
         <span class="citation_pagecount">[]</span>
         <span class="citation_data">[]</span>
      </head>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <div id="">
            <h2>Background</h2>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Yellow fever (YF) is an acute viral hemorrhagic disease transmitted by 
               <i>Aedes</i> mosquitoes. The causative agent, the yellow fever virus (YFV), is found in tropical
               and subtropical areas of South America and Africa. Although a vaccine is available
               since the 1930s, YF still causes thousands of deaths and several outbreaks have recently
               occurred in Africa. Therefore, rapid and reliable diagnostic methods easy to perform
               in low-resources settings could have a major impact on early detection of outbreaks
               and implementation of appropriate response strategies such as vaccination and/or vector
               control.
            </p>
         </div>
         <div id="">
            <h2>Methodology</h2>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The aim of this study was to develop a YFV nucleic acid detection method applicable
               in outbreak investigations and surveillance studies in low-resource and field settings.
               The method should be simple, robust, rapid and reliable. Therefore, we adopted an
               isothermal approach and developed a recombinase polymerase amplification (RPA) assay
               which can be performed with a small portable instrument and easy-to-use lyophilized
               reagents. The assay was developed in three different formats (real-time with or without
               microfluidic semi-automated system and lateral-flow assay) to evaluate their application
               for different purposes. Analytical specificity and sensitivity were evaluated with
               a wide panel of viruses and serial dilutions of YFV RNA. Mosquito pools and spiked
               human plasma samples were also tested for assay validation. Finally, real-time RPA
               in portable format was tested under field conditions in Senegal.
            </p>
         </div>
         <div id="">
            <h2>Conclusion/Significance</h2>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The assay was able to detect 20 different YFV strains and demonstrated no cross-reactions
               with closely related viruses. The RPA assay proved to be a robust, portable method
               with a low detection limit (&lt;21 genome equivalent copies per reaction) and rapid processing
               time (&lt;20 min). Results from real-time RPA field testing were comparable to results
               obtained in the laboratory, thus confirming our method is suitable for YFV detection
               in low-resource settings.
            </p>
         </div>
      </div>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <div tagxxx="title">Author Summary</div>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Despite the use of a safe and effective vaccine, yellow fever virus is still causing
            hundreds of thousands of infections and tens of thousands of deaths every year. The
            disease is widespread in South America and Africa where several outbreaks have occurred
            in the past years. As the disease is difficult to distinguish from other illnesses
            during its early stage, it is necessary to develop reliable, rapid and simple diagnostic
            methods to confirm YF cases to be able to respond effectively to outbreaks through
            vaccination and vector control. In this study, we describe the development a diagnostic
            method for YFV, using an isothermal technology called recombinase polymerase amplification
            which allows detection of the virus within 20 minutes, using a portable and easy-to-use
            device. The YFV RPA assay proved to be a specific and sensitive detection method during
            testing in the laboratory and under field conditions in Senegal.
         </p>
      </div>
      <div id="s1">
         <h2>Introduction</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Yellow fever (YF) has been one of the most feared diseases during the past centuries,
            its historical impact ranking next to plague and smallpox. Unfortunately, unlike smallpox,
            YF virus (YFV) cannot be eradicated as its transmission by mosquitoes includes a sylvatic
            cycle. Despite the use of an effective vaccine since the 1930s, the World Health Organization
            (WHO) estimates that the disease affects more than 200,000 persons causing 30,000
            deaths per year 
            <span ref-type="bibr" rid="pntd.0002730-1">
               <sup>
                  <a href="#pntd.0002730-1">[1]</a>
               </sup>
            </span>. YF remains an important public health problem for the populations of 44 countries,
            33 in Africa and 11 in Central and South America, where altogether almost 900 million
            people are at risk. In recent years, the number of YF cases has increased 
            <span ref-type="bibr" rid="pntd.0002730-2">
               <sup>
                  <a href="#pntd.0002730-2">[2]</a>
               </sup>
            </span>, and there is great concern that the disease might be introduced into new areas 
            <span ref-type="bibr" rid="pntd.0002730-Weaver1">
               <sup>
                  <a href="#pntd.0002730-Weaver1">[3]</a>
               </sup>
            </span>. Recently, severe outbreaks have occurred in regions of Africa that have long been
            free of the virus, such as Darfur in Sudan or South Omo in Ethiopia which experienced
            the worst YF outbreak in Africa in 20 years in 2012 
            <span ref-type="bibr" rid="pntd.0002730-Yuill1">
               <sup>
                  <a href="#pntd.0002730-Yuill1">[4]</a>
               </sup>
            </span>.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">YFV is the prototype of the genus Flavivirus (family Flaviviridae) which comprises
            more than 80 positive-sense, single-stranded RNA viruses, including other human pathogens
            such as dengue, West Nile virus, Usutu virus, Zika virus, Japanese encephalitis virus
            and Tick-borne encephalitis virus 
            <span ref-type="bibr" rid="pntd.0002730-Gould1">
               <sup>
                  <a href="#pntd.0002730-Gould1">[5]</a>
               </sup>
            </span>.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Diagnosis of YFV infection is very challenging as the early symptoms caused by YFV
            are not specific. Laboratory confirmation is therefore essential for the differential
            diagnosis of YF with leptospirosis, malaria, viral hepatitis and other hemorrhagic
            diseases. Laboratory testing is also challenged by the short duration of the YF viremia
            in humans, the low-level laboratory infrastructure in most endemic areas and cross-reactions
            when using serological methods which lack specificity 
            <span ref-type="bibr" rid="pntd.0002730-HoughtonTrivio1">
               <sup>
                  <a href="#pntd.0002730-HoughtonTrivio1">[6]</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pntd.0002730-Koraka1">
               <sup>
                  <a href="#pntd.0002730-Koraka1">[8]</a>
               </sup>
            </span>.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Alternatively, molecular diagnostic methods represent essential tools for early diagnostics
            as they are able to detect infections during the viremic phase. Early detection of
            cases is crucial to provide efficient patient management, rapid outbreak response
            and emergency vaccination measures. For this reason, considerable efforts are made
            to develop accessible direct detection methods based on molecular detection which
            allow a rapid and highly sensitive detection of YFV. Several molecular methods for
            YFV detection based on polymerase chain reaction (PCR), such as real-time RT-PCR,
            have been established, but these methods require the use of complex instruments and
            well-equipped laboratories 
            <span ref-type="bibr" rid="pntd.0002730-Weidmann1">
               <sup>
                  <a href="#pntd.0002730-Weidmann1">[9]</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pntd.0002730-Bae1">
               <sup>
                  <a href="#pntd.0002730-Bae1">[13]</a>
               </sup>
            </span>. However, in the case of direct detection methods for YFV, it is essential to be
            able to provide a portable, simple and robust method suitable for low-resource settings
            and field diagnosis, especially for outbreak response. For this reason, new molecular
            methods based on isothermal amplification have been developed for YFV detection, such
            as real-time reverse-transcription loop-mediated isothermal amplification (RT-LAMP)
            
            <span ref-type="bibr" rid="pntd.0002730-Kwallah1">
               <sup>
                  <a href="#pntd.0002730-Kwallah1">[14]</a>
               </sup>
            </span> and helicase-dependent amplification assays (HDA) 
            <span ref-type="bibr" rid="pntd.0002730-Domingo1">
               <sup>
                  <a href="#pntd.0002730-Domingo1">[15]</a>
               </sup>
            </span>.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In this study, we describe the establishment of a reverse-transcriptase recombinase
            polymerase amplification (RPA) assay for YFV detection. During RPA reaction, YFV RNA
            is first transcripted to DNA by a reverse-transcriptase. Secondarily, a phage derivated
            recombinase forms a nucleoprotein complex with the oligonucleotide primers which is
            able to scan for homologous sequences in the DNA template. RPA reaction can be performed
            between 25 and 42°C since denaturation of the DNA template is not required. If the
            target is present, the oligonucleotides are extended by strand displacing polymerases
            
            <span ref-type="bibr" rid="pntd.0002730-Piepenburg1">
               <sup>
                  <a href="#pntd.0002730-Piepenburg1">[16]</a>
               </sup>
            </span>. Real-time signal detection of the amplification can be performed within 15 minutes
            by using TwistAmp™ Exo probes (TwistDx, Cambridge, UK) and a ESEQuant Tube Scanner
            (QIAGEN Lake Constance GmbH, Stockach, Germany), a small easy-to-use fluorescence
            detection system which can perform eight measurements simultaneously. In low-resource
            settings where no power supply is available, the Tube Scanner device can be powered
            by a car adaptor, a small rechargeable battery or a battery charged by solar panels
            
            <span ref-type="bibr" rid="pntd.0002730-AbdElWahed1">
               <sup>
                  <a href="#pntd.0002730-AbdElWahed1">[17]</a>
               </sup>
            </span>. The RPA assay can also be integrated into a semi-automated system, using a GeneSlice
            microfluidic cartridge (HSG-IMIT, Freiburg, Germany) installed in a “SONDE” player
            device. As an alternative to real-time measurement, RPA results may be visualized
            after amplification on lateral-flow stripes (LFS) by using a different probe, TwistAmp™
            Nfo (TwistDx, Cambridge, UK), during the RPA reaction. The reaction system can be
            stabilized in a dried formulation transportable without a cold chain.
         </p>
      </div>
      <div id="s2">
         <h2>Materials and Methods</h2>
         <div id="s2a">
            <h3>Viruses and mosquito pools</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Virus strains used were provided by the Robert Koch Institute in Berlin, the Bernhard-Nocht-Institute
               in Hamburg in Germany, and the Pasteur Institute of Dakar in Senegal. All virus strains
               were derived from cell culture, inactivated and stabilized. YFV strains are listed
               in 
               <span ref-type="table" rid="pntd-0002730-t001">
                  <a href="#pntd-0002730-t001">Table 1</a>
               </span> and other viral strains in 
               <span ref-type="table" rid="pntd-0002730-t002">
                  <a href="#pntd-0002730-t002">Table 2</a>
               </span>.
            </p>
            <div id="pntd-0002730-t001">
               <div tagxxx="object-id">10.1371/journal.pntd.0002730.t001</div>
               <span class="label">Table 1</span>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Yellow fever viral strains used for analytical specificity testing.</div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <colgroup>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                     </colgroup>
                     <thead>
                        <tr>
                           <td>Virus description</td>
                           <td>Accession No.</td>
                           <td>Origin</td>
                           <td>Date</td>
                           <td>Lineage</td>
                           <td>real-time RT-PCR</td>
                           <td>RT-RPA</td>
                        </tr>
                        <tr>
                           <td>YFV virus strains</td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td>Ct values</td>
                           <td>real-time Tt [min]</td>
                           <td>LFS</td>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>ArD 24553</td>
                           <td>_</td>
                           <td>Senegal</td>
                           <td>1976</td>
                           <td>_</td>
                           <td>24,6</td>
                           <td>3,3</td>
                           <td>n.d.</td>
                        </tr>
                        <tr>
                           <td>ArD 408/78</td>
                           <td>_</td>
                           <td>Burkina Faso</td>
                           <td>1978</td>
                           <td>_</td>
                           <td>23,9</td>
                           <td>3,0</td>
                           <td>n.d.</td>
                        </tr>
                        <tr>
                           <td>HD 117294</td>
                           <td>JX898868</td>
                           <td>Senegal</td>
                           <td>1995</td>
                           <td>6</td>
                           <td>16,5</td>
                           <td>2,3</td>
                           <td>n.d.</td>
                        </tr>
                        <tr>
                           <td>ArD 114891</td>
                           <td>_</td>
                           <td>Senegal</td>
                           <td>1995</td>
                           <td>6</td>
                           <td>16,0</td>
                           <td>1,6</td>
                           <td>n.d.</td>
                        </tr>
                        <tr>
                           <td>ArD 99740</td>
                           <td>_</td>
                           <td>Senegal</td>
                           <td>1993</td>
                           <td>3</td>
                           <td>25,0</td>
                           <td>5,1</td>
                           <td>n.d.</td>
                        </tr>
                        <tr>
                           <td>ArD 114991</td>
                           <td>_</td>
                           <td>Senegal</td>
                           <td>1995</td>
                           <td>_</td>
                           <td>24,3</td>
                           <td>3,4</td>
                           <td>n.d.</td>
                        </tr>
                        <tr>
                           <td>HD 122030</td>
                           <td>_</td>
                           <td>Senegal</td>
                           <td>1996</td>
                           <td>6</td>
                           <td>19,4</td>
                           <td>2,4</td>
                           <td>n.d.</td>
                        </tr>
                        <tr>
                           <td>ArD 122522</td>
                           <td>_</td>
                           <td>Senegal</td>
                           <td>1996</td>
                           <td>6</td>
                           <td>21,3</td>
                           <td>3,3</td>
                           <td>n.d.</td>
                        </tr>
                        <tr>
                           <td>HA 016/97</td>
                           <td>_</td>
                           <td>Liberia</td>
                           <td>1997</td>
                           <td>_</td>
                           <td>20,0</td>
                           <td>1,6</td>
                           <td>n.d.</td>
                        </tr>
                        <tr>
                           <td>HD 47471</td>
                           <td>_</td>
                           <td>Mauritania</td>
                           <td>1987</td>
                           <td>_</td>
                           <td>28,5</td>
                           <td>5,9</td>
                           <td>n.d.</td>
                        </tr>
                        <tr>
                           <td>ArD D X</td>
                           <td>_</td>
                           <td>Senegal</td>
                           <td>2000</td>
                           <td>5</td>
                           <td>21,4</td>
                           <td>2,4</td>
                           <td>n.d.</td>
                        </tr>
                        <tr>
                           <td>Asibi</td>
                           <td>AY640589.1</td>
                           <td>Ghana</td>
                           <td>1927</td>
                           <td>_</td>
                           <td>20,6</td>
                           <td>3,2</td>
                           <td>pos</td>
                        </tr>
                        <tr>
                           <td>ArD 114896</td>
                           <td>JX898871</td>
                           <td>Senegal</td>
                           <td>1995</td>
                           <td>3</td>
                           <td>20,3</td>
                           <td>3,1</td>
                           <td>pos</td>
                        </tr>
                        <tr>
                           <td>ArD 156468</td>
                           <td>JX898876</td>
                           <td>Senegal</td>
                           <td>2001</td>
                           <td>4</td>
                           <td>16,8</td>
                           <td>2,4</td>
                           <td>pos</td>
                        </tr>
                        <tr>
                           <td>DakArAmt7</td>
                           <td>JX898869</td>
                           <td>Ivory Coast</td>
                           <td>1973</td>
                           <td>1</td>
                           <td>15,4</td>
                           <td>2,1</td>
                           <td>pos</td>
                        </tr>
                        <tr>
                           <td>ArD 121040</td>
                           <td>JX898870</td>
                           <td>Senegal</td>
                           <td>1996</td>
                           <td>6</td>
                           <td>16,4</td>
                           <td>2,3</td>
                           <td>pos</td>
                        </tr>
                        <tr>
                           <td>ArD 149214</td>
                           <td>JX898873</td>
                           <td>Senegal</td>
                           <td>2000</td>
                           <td>5</td>
                           <td>15,5</td>
                           <td>2,2</td>
                           <td>pos</td>
                        </tr>
                        <tr>
                           <td>Ivory C 1999</td>
                           <td>AY603338.1</td>
                           <td>Ivory Coast</td>
                           <td>1999</td>
                           <td>6</td>
                           <td>19,1</td>
                           <td>2,5</td>
                           <td>pos</td>
                        </tr>
                        <tr>
                           <td>Trinidad 79A 788379</td>
                           <td>AF094612.1</td>
                           <td>Brazil</td>
                           <td>1979</td>
                           <td>3</td>
                           <td>20,1</td>
                           <td>2,5</td>
                           <td>pos</td>
                        </tr>
                        <tr>
                           <td>17D RKI #142/94/1</td>
                           <td>Vaccine strain</td>
                           <td>RKI</td>
                           <td>_</td>
                           <td>_</td>
                           <td>20,0</td>
                           <td>2,1</td>
                           <td>pos</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <table-wrap-foot xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div class="fn" id="nt101">
                     <span class="label"></span>
                     <p>n.d.: not determined; pos: positive; neg: negative; Tt: time threshold.</p>
                  </div>
               </table-wrap-foot>
            </div>
            <div id="pntd-0002730-t002">
               <div tagxxx="object-id">10.1371/journal.pntd.0002730.t002</div>
               <span class="label">Table 2</span>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Viral strains other than YFV used for analytical specificity testing.</div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <colgroup>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                     </colgroup>
                     <thead>
                        <tr>
                           <td>Virus family</td>
                           <td>Virus specie</td>
                           <td>Virus strain</td>
                           <td>Real-time RT-PCR</td>
                           <td>RT-RPA result</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td>reference</td>
                           <td>result (Ct)</td>
                           <td>real-time/LFS</td>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>
                              <b>
                                 <i>Flaviviridae other than</i>
                              </b>
                           </td>
                           <td>Dengue virus serotype 1</td>
                           <td>VR344 (Thai 1958 strain)</td>
                           <td></td>
                           <td>15.9</td>
                           <td>neg</td>
                        </tr>
                        <tr>
                           <td>
                              <b>
                                 <i>YFV</i>
                              </b>
                           </td>
                           <td>Dengue virus serotype 2</td>
                           <td>VR345 (TH-36 strain)</td>
                           <td>
                              <span ref-type="bibr" rid="pntd.0002730-Domingo5">
                                 <sup>
                                    <a href="#pntd.0002730-Domingo5">[28]</a>
                                 </sup>
                              </span> ih
                           </td>
                           <td>18.8</td>
                           <td>neg</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>Dengue virus serotype 3</td>
                           <td>VR216 (H87 strain)</td>
                           <td></td>
                           <td>20.3</td>
                           <td>neg</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>Dengue virus serotype 4</td>
                           <td>VR217 (H241 strain)</td>
                           <td></td>
                           <td>16.2</td>
                           <td>neg</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>West Nile virus lineage 1</td>
                           <td>Israel</td>
                           <td>
                              <span ref-type="bibr" rid="pntd.0002730-Linke1">
                                 <sup>
                                    <a href="#pntd.0002730-Linke1">[29]</a>
                                 </sup>
                              </span>
                           </td>
                           <td>19.9</td>
                           <td>neg</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>West Nile virus lineage 2</td>
                           <td>Uganda</td>
                           <td></td>
                           <td>26.8</td>
                           <td>neg</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>TB Encephalitis virus</td>
                           <td>K23 strain</td>
                           <td>
                              <span ref-type="bibr" rid="pntd.0002730-Achazi1">
                                 <sup>
                                    <a href="#pntd.0002730-Achazi1">[30]</a>
                                 </sup>
                              </span>
                           </td>
                           <td>16.3</td>
                           <td>neg</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>RSS Encephalitis virus</td>
                           <td>Far eastern subtype</td>
                           <td></td>
                           <td>24.2</td>
                           <td>neg</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>Japanese Encephalitis virus</td>
                           <td>ATCC SA14-14-2</td>
                           <td>
                              <span ref-type="bibr" rid="pntd.0002730-Patel1">
                                 <sup>
                                    <a href="#pntd.0002730-Patel1">[31]</a>
                                 </sup>
                              </span>
                           </td>
                           <td>19.4</td>
                           <td>neg</td>
                        </tr>
                        <tr>
                           <td>
                              <b>
                                 <i>Bunyaviridae</i>
                              </b>
                           </td>
                           <td>Rift Valley Fever virus</td>
                           <td>strain ZH548</td>
                           <td>
                              <span ref-type="bibr" rid="pntd.0002730-Weidmann2">
                                 <sup>
                                    <a href="#pntd.0002730-Weidmann2">[32]</a>
                                 </sup>
                              </span>
                           </td>
                           <td>26.2</td>
                           <td>neg</td>
                        </tr>
                        <tr>
                           <td>
                              <b>
                                 <i>Filoviridae</i>
                              </b>
                           </td>
                           <td>Ebola virus</td>
                           <td>Zaire strain</td>
                           <td>
                              <span ref-type="bibr" rid="pntd.0002730-Weidmann3">
                                 <sup>
                                    <a href="#pntd.0002730-Weidmann3">[33]</a>
                                 </sup>
                              </span>
                           </td>
                           <td>24.7</td>
                           <td>neg</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>Marburg virus</td>
                           <td>Musoke strain</td>
                           <td></td>
                           <td>24.4</td>
                           <td>neg</td>
                        </tr>
                        <tr>
                           <td>
                              <b>
                                 <i>Alphaviridae</i>
                              </b>
                           </td>
                           <td>Chikungunya virus</td>
                           <td>African isolate</td>
                           <td>ih</td>
                           <td>17.5</td>
                           <td>neg</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <table-wrap-foot xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div class="fn" id="nt102">
                     <span class="label"></span>
                     <p>TB: Tick-borne; RSS: Russian Spring Summer; neg: negative; ih: in-house assay.</p>
                  </div>
               </table-wrap-foot>
            </div>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Pools of mosquitoes, some of them infected with YFV, were provided by the Pasteur
               Institute of Dakar. The mosquito sampling protocol was extensively described by Diallo
               and colleagues 
               <span ref-type="bibr" rid="pntd.0002730-Diallo1">
                  <sup>
                     <a href="#pntd.0002730-Diallo1">[18]</a>
                  </sup>
               </span>.
            </p>
         </div>
         <div id="s2b">
            <h3>RNA extraction and sample preparation</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Viral RNA was isolated from 140-µl aliquots of cell culture supernatants or 100-µl
               aliquots of mosquito pools, using the QIAamp Viral Mini Kit (QIAGEN Lake Constance
               GmbH, Stockach, Germany) according to the manufacturer's instructions. RNA was eluted
               in 100 µl of elution buffer and stored at −80°C until further use.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In order to use an energy-free method in the field trial, RNA extraction was performed
               with the innuPREP MP Basic Kit A (Jena Analytik, Jena, Germany) with a magnetic bead
               separation rack combined with proteinase K treatment according to the manufacturer's
               instructions. The nucleic acids were eluted in 100 µl of nuclease-free distilled water,
               and 5 µl were subjected to PCR or RPA, respectively.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">RNA was extracted from 10-fold serial dilutions of YFV preparation and stored in aliquots
               at −80°C until use to assess the sensitivity of the extraction method. Human plasma
               samples spiked with low concentrations of YFV were used as a model for assay validation
               with clinical samples.
            </p>
         </div>
         <div id="s2c">
            <h3>Primer and probe design for RPA</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">All of the 79 YFV full-length sequences covering the 5′-UTR region available in the
               database (NCBI) were aligned using Geneious 5.0 software. According to Piepenburg
               and colleagues, primers of 30 nt to 35 nt in length are recommended for RPA 
               <span ref-type="bibr" rid="pntd.0002730-Piepenburg1">
                  <sup>
                     <a href="#pntd.0002730-Piepenburg1">[16]</a>
                  </sup>
               </span>. One set of degenerate generic primers (YFV RF/RR) was designed according to the
               alignment for amplification of different YFV strains (
               <span ref-type="table" rid="pntd-0002730-t001">
                  <a href="#pntd-0002730-t001">Table 1</a>
               </span>). The primer sequences were identical for both lateral-flow strip RPA (LFS RT-RPA)
               and real-time RT-RPA primers, except for an additional biotinylation at the 5′ end
               of the LFS reverse primer. RPA exo probe for fluorogenic detection and RPA nfo probe
               for detection of dual-labeled amplicon were designed according to RPA guidelines from
               TwistDx (Cambridge, United Kingdom) and synthesized by TIB MOLBIOL (Berlin, Germany).
            </p>
         </div>
         <div id="s2d">
            <h3>Real-time RT-PCR</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">YFV-specific primer YFV FP/RP and probe YFV LNA2 were used to detect and quantify
               genomic RNA of YFV as described previously 
               <span ref-type="bibr" rid="pntd.0002730-Weidmann1">
                  <sup>
                     <a href="#pntd.0002730-Weidmann1">[9]</a>
                  </sup>
               </span>. The assay was performed in a one-step format on the ABI 7500 instrument using the
               QuantiTect Virus Kit (QIAGEN Lake Constance GmbH, Stockach, Germany).
            </p>
         </div>
         <div id="s2e">
            <h3>Lateral-flow strip RT-RPA assay</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">LFS-RPA assay was performed using the TwistAmp™ nfo RT kit from TwistDx (Cambridge,
               United Kingdom) according to the manufacturer's instructions. Briefly, 29.5 µl of
               rehydration solution were mixed with 7.2 µl of PCR water, 2.1 µl of each primer (10
               µM) and 0.6 µl of the target-specific RPA nfo probe (10 µM). Then 5 µl of RNA template
               was added to the 41.5 µl master mix. The template/master mix solution was added to
               the dry reagent pellet and mixed by pipetting up and down. Finally, the reaction was
               triggered by adding 3.5 µl of magnesium acetate (Mg(OAc)2, 280 mM) to the 46.5 µl
               reaction mix. The reaction mix was placed into the heating block at 39°C for 20 min,
               with brief mixing and centrifugation after 3–4 min of incubation. After amplification
               at 39°C for 20 min, 2 µl of amplification product was diluted in 100 µl of PBST buffer,
               and 10 µl of diluted amplicon was dropped on the sample pad of a HybriDetect lateral
               flow stripe (LFS) (Milenia Biotec, Giessen, Germany). Strips were then placed into
               tubes containing 100 µl of PBST buffer. The final result was read visually after 5
               min of incubation. A test was considered positive when the detection line as well
               as the control line was visible. A test was considered negative when only the control
               line was visible.
            </p>
         </div>
         <div id="s2f">
            <h3>Real-time RT-RPA assay</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Real-time RT-RPA assay was performed using the TwistAmp™ exo RT kit according to the
               manufacturer's instructions. The TwistAmp™ exo RT kit contains an additional RT-enzyme
               enabling the DNA amplification of RNA targets. Briefly, 37.7 µl of rehydration solution
               were mixed with 2.1 µl of each primer (10 µM) and 0.6 µl of the target-specific RPA
               exo probe (10 µM). Then 5 µl of RNA template was added to the 42.5 µl master mix.
               The template/master mix solution was added to the dry reagent pellet and mixed by
               pipetting up and down. Finally, the reaction was triggered by adding 3.5 µl of Mg(OAc)2
               (280 mM) to the 47.5 µl reaction mix. The reaction tubes were mixed, centrifuged and
               then placed into the ESE Quant Tube Scanner for real-time monitoring of fluorescence.
               Reaction was performed at 39°C for 20 min, with brief mixing and centrifugation of
               reaction tubes after 3–4 min of incubation. This reaction temperature was determined
               optimal in terms of sensitivity. For data analysis, the Tube Scanner requires to be
               connected to a computer installed with the ESEQuant Tube Scanner software Version
               1.0. Threshold values were determined by slope validation, i.e. slope (mV/min) values
               were compared in order to distinguish positive results from negative results. Further
               development and standardization of the method would allow using the device on its
               own with a direct display of positive or negative results for each sample.
            </p>
         </div>
         <div id="s2g">
            <h3>Analytical specificity of real-time and LFS RT-RPA</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To test whether our assay is able to detect a wide variety of YFV strains, we utilized
               a panel of 20 different YFV strains described in 
               <span ref-type="table" rid="pntd-0002730-t001">
                  <a href="#pntd-0002730-t001">Table 1</a>
               </span>. The analytical specificity was tested with a panel of 13 arboviruses and hemorrhagic
               fever viruses of which 9 are flaviviruses genetically related to YFV (
               <span ref-type="table" rid="pntd-0002730-t002">
                  <a href="#pntd-0002730-t002">Table 2</a>
               </span>).
            </p>
         </div>
         <div id="s2h">
            <h3>Analytical sensitivity of real-time and LFS RT-RPA</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">RT-RPA analytical sensitivity was evaluated by testing RNA extracts from 10-fold serial
               dilutions of YFV preparations comparatively to real-time RT-PCR used as the reference
               method. RNA was extracted from the YFV Asibi strain and RNA concentrations ranged
               from 2×10
               <sup>5</sup> to 8 genome equivalent copies per reaction (GC/rxn). Repeatability of the method
               was assessed by testing each dilution 10 times with real-time RT-RPA and 5 times with
               LFS-RT-RPA.
            </p>
         </div>
         <div id="s2i">
            <h3>Centrifugal microfluidic cartridge</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Centrifugal microfluidic cartridges 
               <span ref-type="bibr" rid="pntd.0002730-Mark1">
                  <sup>
                     <a href="#pntd.0002730-Mark1">[19]</a>
                  </sup>
               </span>, termed GeneSlice (HSG-IMIT, Lab on a Chip Design- and Foundry Service, Freiburg,
               Germany), were used to demonstrate process automation of real-time RT-RPA in a small
               and portable processing device, the “SONDE” player, that may be used in the field
               with minimum manual interaction (
               <span ref-type="fig" rid="pntd-0002730-g001">Fig. 1B</span>). The GeneSlice contains a microfluidic channel network that allows to aliquot an
               initial reaction mixture into 8 subvolumes by applying centrifugal forces. Each subvolume
               is then transferred into a separated amplification chamber (
               <span ref-type="fig" rid="pntd-0002730-g001">Fig. 1A</span>) 
               <span ref-type="bibr" rid="pntd.0002730-Mark2">
                  <sup>
                     <a href="#pntd.0002730-Mark2">[20]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pntd.0002730-Focke1">
                  <sup>
                     <a href="#pntd.0002730-Focke1">[21]</a>
                  </sup>
               </span>.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pntd.0002730.g001</div>
               <span class="label">Figure 1</span>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Centrifugal microfluidic platform.</div>
                  <p>
                     <b>A:</b> GeneSlice cartridge contains the microfluidic structure for aliquoting the reaction
                     mix into eight 10 µl subvolumes; 
                     <b>B:</b> Prototype device for processing the GeneSlices (“SONDE player”) featuring defined
                     rotation, acceleration and deceleration, heating and fluorescence detection (QIAGEN
                     Lake Constance GmbH, Stockach, Germany).
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The reaction mixture was composed of 73 µl rehydration solution, 4.2 µl forward/reverse
               primer (10 µM each), 1.2 µl probe (10 µM), 7 µl of Mg(OAc)2 (280 mM) and 10 µl of
               the DNA/RNA template. Three lyophilized pellets from the TwistAmp™ exo RT kit were
               resuspended in the 90 µl-reaction mixture. The reaction mixture is aliquoted into
               eight 10 µl volumes and transferred into amplification chamber by centrifuge force.
               Excess mixture is collected into a waste chamber. The “SONDE” player heats the samples
               at 41°C and RPA reaction is initiated in each amplification chamber. The fluorescence
               signal produced by the amplification is monitored for 20 minutes by the integrated
               detection unit. Amplification results were analyzed using IsoAmp Software (QIAGEN
               Lake Constance GmbH, Stockach, Germany).
            </p>
         </div>
         <div id="s2j">
            <h3>Field trial of real-time RT-RPA</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Real-time RT-RPA assay combined with a magnetic bead-based extraction method was tested
               under field conditions in Senegal. Inactivated YFV virus and YFV RNA controls were
               prepared in dry-stabilized format using DNAstable Blood and RNAstable reagents (Biomatrica,
               San Diego, USA), respectively. These controls were stored at ambient temperature until
               further use in the field.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">For the field trial of real-time RT-RPA, all reagents and instruments required were
               packed and transported by car from Dakar (14°43′12″N 17°28′48″W) to Mbour (14°25′19″N
               16°57′51″W) at ambient temperature. At the Mbour city health center, the RPA setup
               was deployed and RNA was extracted from dry-stabilized virus controls using innuPrep
               MP basic kit. Subsequently, the extracted RNAs were tested with real-time RT-RPA for
               YFV. In order to reproduce field conditions where no power supply is available, the
               Tube Scanner was powered by a battery charged by solar panels.
            </p>
         </div>
      </div>
      <div id="s3">
         <h2>Results</h2>
         <div id="s3a">
            <h3>Primer and probe design for RPA</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">By analyzing the alignment of all available full genome sequences of YFV, the conserved
               5′-non-coding region (NCR) of the YFV genome was chosen for primer and probe design
               (
               <span ref-type="table" rid="pntd-0002730-t003">
                  <a href="#pntd-0002730-t003">Table 3</a>
               </span>). The primer set YFV RF/RR efficiently amplified YFV RNA in LFS RT-RPA and real-time
               RT-RPA assays.
            </p>
            <div id="pntd-0002730-t003">
               <div tagxxx="object-id">10.1371/journal.pntd.0002730.t003</div>
               <span class="label">Table 3</span>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">List of primers and probe for the lateral-flow stripe and real-time RPA assay based
                     on the YFV strain accession n° NC00203.
                  </div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <colgroup>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                     </colgroup>
                     <thead>
                        <tr>
                           <td>Assay format</td>
                           <td>Oligo name</td>
                           <td>Sequence 5′→3′</td>
                           <td>Direction</td>
                           <td>Position</td>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>
                              <b>Lateral-flow stripe RT-RPA</b>
                           </td>
                           <td>
                              <b>YFV RF</b>
                           </td>
                           <td>
                              <div tagxxx="named-content">AAATCCTGTGTGCTAATTGAGGTGYATTGG</div>
                           </td>
                           <td>sense</td>
                           <td>4 to 33</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>
                              <b>YFV RR-Bio</b>
                           </td>
                           <td>
                              <b>Biotin-</b>
                              <div tagxxx="named-content">ACATDWTCTGGTCARTTCTCTGCTAATCGC</div>
                           </td>
                           <td>antisense</td>
                           <td>93 to 122</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>
                              <b>YFV Rprobe nfo</b>
                           </td>
                           <td>
                              <b>FAM-</b>
                              <div tagxxx="named-content">CTGCAAATCGAGTTGCTAGGCAATAAACAC</div>
                              <b>[THF]</b>
                              <div tagxxx="named-content">TTTGGATT-AATTTTRATCGTT</div>
                              <b>-Ph</b>
                           </td>
                           <td>sense</td>
                           <td>35 to 86</td>
                        </tr>
                        <tr>
                           <td>
                              <b>Real-time RT-RPA</b>
                           </td>
                           <td>
                              <b>YFV RF</b>
                           </td>
                           <td>
                              <div tagxxx="named-content">AAATCCTGKGTGCTAATTGAGGTGYATTGG</div>
                           </td>
                           <td>sense</td>
                           <td>4 to 33</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>
                              <b>YFV RR</b>
                           </td>
                           <td>
                              <div tagxxx="named-content">ACATDWTCTGGTCARTTCTCTGCTAATCGC</div>
                           </td>
                           <td>antisense</td>
                           <td>93 to 122</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>
                              <b>YFV Rprobe exo</b>
                           </td>
                           <td>
                              <div tagxxx="named-content">gCAAATCgAgTTgCTAggCAATAAACACATT</div>
                              <b>[BHQdT]g[THF]A[FAMdT]</b>
                              <div tagxxx="named-content">TAATTTTRATCgTTC</div>
                              <b>-Ph</b>
                           </td>
                           <td>sense</td>
                           <td>37 to 87</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <table-wrap-foot xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div class="fn" id="nt103">
                     <span class="label"></span>
                     <p>
                        <b>FAM</b>: 6-Carboxyfluorescein; 
                        <b>THF</b>: tetrahydrofuran; 
                        <b>Ph</b>: 3′phosphate to block elongation; 
                        <b>BHQ</b>: black hole quencher.
                     </p>
                  </div>
               </table-wrap-foot>
            </div>
         </div>
         <div id="s3b">
            <h3>Analytical specificity of real-time and LFS RT-RPA</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The analytical specificity testing revealed that all the 20 different YFV strains
               were detected by both LFS and real-time RT-RPA assays. The testing results of the
               panel of 13 viruses other than YFV showed no cross-reactions, as all results were
               negative for both assays (
               <span ref-type="table" rid="pntd-0002730-t002">
                  <a href="#pntd-0002730-t002">Table 2</a>
               </span>). However, concerns with specificity were encountered with the LFS RT-RPA assay,
               as a faint band was observed in the negative controls when running time exceeded 5
               minutes, thus potentially generating false-positive results.
            </p>
         </div>
         <div id="s3c">
            <h3>Analytical sensitivity of real-time and LFS RT-RPA</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The analytical sensitivity of RT-RPA assays was evaluated by testing the RNA extracts
               from 10-fold serial dilutions of YFV preparations and by comparing real-time RT-RPA
               and real-time RT-PCR test results. The real-time RT-PCR showed linear results for
               the quantification of RNA standards over a range of 10 to 10
               <sup>6</sup> genome copies. Real-time RT-PCR detected as low as 8 GC/rxn while real-time and LFS
               RT-RPA assays could detect as low as 44 GC/rxn in YFV RNA extracts and 21 GC/rxn for
               the testing of YFV-spiked human plasma samples (
               <span ref-type="fig" rid="pntd-0002730-g002">Figure 2</span>). The amplification curves of the YFV RNA extracts from 10-fold serial dilutions
               are shown in 
               <span ref-type="fig" rid="pntd-0002730-g003">Figure 3</span>-A for real-time RT-RPA results and 
               <span ref-type="fig" rid="pntd-0002730-g003">Figure 3</span>-B for real-time RT-PCR results.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pntd.0002730.g002</div>
               <span class="label">Figure 2</span>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Sensitivity testing of the real-time and LFS RT-RPA with YFV cell supernatant and
                     human plasma spiked with YFV in comparison with real-time RT-PCR results.
                  </div>
                  <p>Ct: cycle threshold; Tt: time threshold; Neg.: Negative; Pos.: Positive; Undet.: Undetermined.</p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pntd.0002730.g003</div>
               <span class="label">Figure 3</span>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Amplification plots of real-time measurements for extracted RNA from 10-fold serial
                     dilutions of YFV; A: RT-PCR results; B: RT-RPA results.
                  </div>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
         </div>
         <div id="s3d">
            <h3>Testing of mosquito pools with real-time RT-RPA on the Tube Scanner</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Thirty-four samples of monospecific pools of wild-caught mosquitoes collected from
               Kedougou, southern Senegal were included in this study. The RNA extracts from these
               samples were tested in parallel with real-time RT-PCR and RT-RPA. Fourteen mosquito
               samples out of 34 (41.2%) resulted negative in real-time RT-PCR and 20 were positive
               (58.8%) with Ct values ranging from 24.65 to 35.51 (data not shown). Of the 20 samples
               detected positive in real-time RT-PCR, 16 were tested positive by real-time RT-RPA
               assay, providing a sensitivity of 80% (95% CI: 56.3% to 94.1%). Of the 14 samples
               tested negative in real-time RT-PCR, all were also tested negative by real-time RT-RPA
               assay, providing a specificity of 100% (95% CI: 76.7% to 100%). The overall agreement
               between the two assays was 88.4% (30/34) (
               <span ref-type="table" rid="pntd-0002730-t004">
                  <a href="#pntd-0002730-t004">Table 4</a>
               </span>).
            </p>
            <div id="pntd-0002730-t004">
               <div tagxxx="object-id">10.1371/journal.pntd.0002730.t004</div>
               <span class="label">Table 4</span>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Performance of the real-time RT-RPA assay using the Tube Scanner or the GeneSlice
                     cartridge in comparison to the reference method, real-time RT-PCR, for detecting YFV
                     in mosquito pools.
                  </div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <colgroup>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                     </colgroup>
                     <thead>
                        <tr>
                           <td></td>
                           <td></td>
                           <td>Real-time RT-PCR</td>
                           <td>Performance characteristics (%)</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td></td>
                           <td>Positive</td>
                           <td>Negative</td>
                           <td>Sensitivity</td>
                           <td>Specificity</td>
                           <td>PPV</td>
                           <td>NPV</td>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>
                              <b>RT-RPA on Tubescanner</b>
                           </td>
                           <td>Positive</td>
                           <td>16</td>
                           <td>0</td>
                           <td>80%</td>
                           <td>100%</td>
                           <td>100%</td>
                           <td>77.8%</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>Negative</td>
                           <td>4</td>
                           <td>14</td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>Total (n = 34)</td>
                           <td>20</td>
                           <td>14</td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>
                              <b>RT-RPA on</b>
                           </td>
                           <td>Positive</td>
                           <td>10</td>
                           <td>0</td>
                           <td>71.4%</td>
                           <td>100%</td>
                           <td>100%</td>
                           <td>76.5%</td>
                        </tr>
                        <tr>
                           <td>
                              <b>GeneSlice</b>
                           </td>
                           <td>Negative</td>
                           <td>4</td>
                           <td>13</td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>Total (n = 27)</td>
                           <td>14</td>
                           <td>13</td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                     </div>
                  </table>
               </div>
               <table-wrap-foot xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div class="fn" id="nt104">
                     <span class="label"></span>
                     <p>PPV: positive predictive value; NPV: negative predictive value.</p>
                  </div>
               </table-wrap-foot>
            </div>
         </div>
         <div id="s3e">
            <h3>Testing of mosquito pools with real-time RT-RPA on the microfluidic platform</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Twenty-seven RNA samples of mosquito pools were included in this part of the study.
               Thirteen mosquito samples out of 27 (48.1%) had negative results in real-time RT-PCR
               and 14 were positive (51.9%), with Ct values ranging from 27 to 35.5 (data not shown).
               Of the 14 samples tested positive with real-time RT-PCR, 10 were tested positive by
               real-time RT-RPA, providing a sensitivity of 71.4% (95%CI: 41.9% to 91.4%). All of
               the 13 samples that tested negative in real-time RT-PCR were also tested negative
               by real-time RT-RPA assay, providing a specificity of 100% (95% CI: 75% to 100%).
               The overall agreement between the two assays was 85.2% (23/27) (
               <span ref-type="table" rid="pntd-0002730-t004">
                  <a href="#pntd-0002730-t004">Table 4</a>
               </span>).
            </p>
         </div>
         <div id="s3f">
            <h3>Field testing of real-time RT-RPA</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The virus and RNA controls were stabilized with DNAstable Blood and RNAstable reagents
               and tested in the laboratory using real-time RT-PCR and real-time RT-RPA. These results
               were compared to the testing results of the same amount of control samples without
               stabilizer, stored at −20°C. Results were comparable and proved the stabilization
               process to be effective. The average cycle threshold (Ct) and time threshold (Tt)
               values for all samples were 31.08 (SD = 0.74) and 5.5 (SD = 0.22), respectively. When
               stabilized controls and non-stabilized controls at −20°C were tested on real-time
               RPA during the field trial, the mean of Tt values of these samples was 5.3. These
               results are comparable to the values detected previously in the laboratory, indicating
               good reproducibility of the complete experimental workflow in the field.
            </p>
         </div>
      </div>
      <div id="s4">
         <h2>Discussion</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In this study, we describe the development of a RT-RPA assay for YFV detection which
            can be performed without complex equipment in a basic laboratory setting, a rural
            health care center or an outbreak field investigation. We designed a set of primers
            and probe and developed a real-time methodology which enables to detect down to 21
            GC/rxn. This detection limit is slightly higher than the 8 GC/rxn detected by real-time
            PCR 
            <span ref-type="bibr" rid="pntd.0002730-Weidmann1">
               <sup>
                  <a href="#pntd.0002730-Weidmann1">[9]</a>
               </sup>
            </span>. Nonetheless, this level of sensitivity is sufficient to detect wild-type YFV in
            natural infections or serious adverse events (SAEs) following YFV immunization which
            produce viremia levels up to 10
            <sup>8</sup> PFU/ml 
            <span ref-type="bibr" rid="pntd.0002730-Barrett1">
               <sup>
                  <a href="#pntd.0002730-Barrett1">[22]</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pntd.0002730-Domingo3">
               <sup>
                  <a href="#pntd.0002730-Domingo3">[24]</a>
               </sup>
            </span>.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Test results for spiked human plasma samples indicated that serum does not affect
            significantly the assay sensitivity. Therefore, we can assume that the test can be
            applied for laboratory case confirmation of suspected YFV cases. However, there is
            further need to validate intensively the assay using YF clinical samples from various
            endemic countries and from patients at different stages of the disease.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The LFS-RPA assay experienced specificity problems, as a faint nonspecific band appeared
            in the negative controls when running time exceeded 5 minutes. Such faint bands have
            not been observed neither for the very low dilutions of YFV RNA nor during testing
            of other viruses. Therefore, these false-positive results are not due to contamination
            but rather to the clotting of proteins or primers which could not bind to any template.
            Unequivocal interpretation of LFS may be provided by an ESEQuant Lateral Flow Reader
            (QIAGEN Lake Constance GmbH, Stockach, Germany). However, at this point, we recommend
            particular caution during LFS operation and interpretation and further optimization
            of the assay before use under field conditions.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Real-time RT-RPA results demonstrated an optimal specificity. Testing results of the
            mosquito pools demonstrated an analytical specificity of 100% on both the Tube Scanner
            and the microfluidic GeneSlice cartridge. Real-time RT-RPA on the microfluidic GeneSlice
            cartridge showed a statistically similar sensitivity (71.4% and 80% respectively)
            as the confidence intervals of both sensitivity values overlap. The lower sensitivity
            value of the GeneSlice method might be due to the complexity of the microfluidic unit
            operations which comprises release of liquid reagents, reconstitution of lyophilized
            reagents, aliquoting the sample into eight independent reaction cavities and mixing
            of reagents with the RNA samples. Nevertheless, the performance of the GeneSlice is
            satisfying, and no cross-contamination between wells was observed. Moreover, this
            semi-automated and downscaled system leads to a significant reduction in costs, manual
            work and waste, making it an attractive method for point-of-care applications such
            as the screening for hemorrhagic fevers in Africa. However, the real-time RT-RPA on
            the Tube Scanner was used for further evaluation including in field conditions because
            of its higher sensitivity.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">During the field study, real-time RT-RPA has demonstrated similar performance to that
            during previous testing under laboratory conditions. Based on our results, the assay
            proves to have great potential as a point-of-care molecular diagnostic method for
            various reasons: all reagents are lyophilized with the main RPA reagents provided
            in a single dried pellet, which simplifies assay preparation and allows long-term
            storage at room temperature; amplification is performed at constant temperature; ESEQuant
            Tube Scanner device is significantly lighter, smaller and cheaper than all other available
            mobile PCR cyclers or turbidimeter devices for LAMP assays; the assay has a low energy
            consumption; reaction times are short and the system is simple, robust and portable.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The cost is approximately 4 euros per test for real-time RPA and 5 euros per test
            for real-time RT-PCR in lyophilized form. At this stage, costs per sample for both
            techniques are comparable. However RT-RPA is a newly developed technique and prices
            are likely to decrease in the future while availability and throughput will increase.
            Furthermore, the detection device for real-time RPA is approximately 10 times cheaper
            than a real-time PCR machine.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">An external quality assessment study on diagnostic methods for YFV infections launched
            in 2011 revealed that the main weakness observed for molecular methods was the inability
            of some assays to detect the YFV genome of wild-type strains, whereas the vaccine
            strain was always detected 
            <span ref-type="bibr" rid="pntd.0002730-Domingo4">
               <sup>
                  <a href="#pntd.0002730-Domingo4">[25]</a>
               </sup>
            </span>. This specificity problem has not been observed for the YFV RT-RPA assay, as all
            YFV strains were detected. Furthermore, our assay revealed no cross-reactions with
            other closely related viruses.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Recently, another isothermal amplification method for YFV detection was developed
            based on reverse transcription loop-mediated isothermal amplification (RT-LAMP) technology
            
            <span ref-type="bibr" rid="pntd.0002730-Kwallah1">
               <sup>
                  <a href="#pntd.0002730-Kwallah1">[14]</a>
               </sup>
            </span>. In contrast to RPA, LAMP requires a larger set of six primers, a higher temperature
            (62°C) and a longer run time. Sensitivity is not comparable, as results of RT-LAMP
            were expressed as PFU instead of GC detected, but RT-LAMP usually presents equal or
            lower sensitivity than RPA 
            <span ref-type="bibr" rid="pntd.0002730-Euler1">
               <sup>
                  <a href="#pntd.0002730-Euler1">[26]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pntd.0002730-Escadafal1">
               <sup>
                  <a href="#pntd.0002730-Escadafal1">[27]</a>
               </sup>
            </span>. In fact LAMP uses nonspecific intercalating fluorophores for detection while RPA
            uses specific detection probes.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In summary, we have developed a very rapid and sensitive isothermal RPA assay in real-time
            and lateral-flow stripe format for the detection of YFV. Both of these assays can
            be easily applied in low-resource settings as an alternative to traditional laboratory-based
            molecular diagnostic assays. However, the LFS format needs further optimization to
            exclude all risks of false-positive results. The real-time RT-RPA assay, using the
            transportable Tube Scanner device combined with the RNA extraction method based on
            magnetic beads, and the use of lyophilized reagents which can be stored at ambient
            temperature allowed us to apply our RPA assay under field conditions in Senegal with
            performance similar to that of cutting-edge laboratory settings.
         </p>
      </div>
      <div tagxxx="back">
         <div tagxxx="ack">
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We thank Diawo Diallo and Mawlouth Diallo from Institut Pasteur de Dakar for providing
               the mosquito pools tested in this study. We are grateful to the health workers of
               the healthcare center in Mbour and the Ministry of Health in Senegal for their assistance
               during the field testing. We are grateful to TwistDx for providing the TwistAmpnfo
               + RT kit for this study.
            </p>
         </div>
         <div tag="ref-list">
            <ul>
               <div tagxxx="title">References</div>
               <li tag="ref">
                  <a name="pntd.0002730-1"></a>
                  <span class="label">1</span>
                  <div tagxxx="mixed-citation">
                     <span tagx="article-title">Yellow fever fact sheet</span>. 
                     <span tagx="source">Wkly Epidemiol Rec</span>
                     <span tagx="volume">85</span>: 
                     <span tagx="fpage">33</span>–
                     <span tagx="lpage">36</span>
                     <span class="pub-id'">[20151492]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-2"></a>
                  <span class="label">2</span>
                  <div tagxxx="mixed-citation">
                     <span tagx="article-title">Yellow fever in the WHO African and American Regions, 2010</span>. 
                     <span tagx="source">Wkly Epidemiol Rec</span>
                     <span tagx="volume">86</span>: 
                     <span tagx="fpage">370</span>–
                     <span tagx="lpage">376</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Weaver1"></a>
                  <span class="label">3</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Weaver</span>
                        <span tagx="given-names">SC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Reisen</span>
                        <span tagx="given-names">WK</span>,
                     </span> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">Present and future arboviral threats</span>. 
                     <span tagx="source">Antiviral Res</span>
                     <span tagx="volume">85</span>: 
                     <span tagx="fpage">328</span>–
                     <span tagx="lpage">345</span>
                     <span class="pub-id'">[19857523]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Yuill1"></a>
                  <span class="label">4</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Yuill</span>
                        <span tagx="given-names">TM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Woodall</span>
                        <span tagx="given-names">JP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Baekeland</span>
                        <span tagx="given-names">S</span>,
                     </span> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">Latest outbreak news from ProMED-mail. Yellow fever outbreak-Darfur Sudan and Chad</span>. 
                     <span tagx="source">Int J Infect Dis</span>
                     <span tagx="volume">17</span>: 
                     <span tagx="fpage">e476</span>–
                     <span tagx="lpage">478</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Gould1"></a>
                  <span class="label">5</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Solomon</span>
                        <span tagx="given-names">T</span>,
                     </span> (
                     <span tagx="year">2008</span>) 
                     <span tagx="article-title">Pathogenic flaviviruses</span>. 
                     <span tagx="source">Lancet</span>
                     <span tagx="volume">371</span>: 
                     <span tagx="fpage">500</span>–
                     <span tagx="lpage">509</span>
                     <span class="pub-id'">[18262042]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-HoughtonTrivio1"></a>
                  <span class="label">6</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Houghton-Triviño</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Montaña</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Castellanos</span>
                        <span tagx="given-names">J</span>,
                     </span> (
                     <span tagx="year">2008</span>) 
                     <span tagx="article-title">Dengue-yellow fever sera cross-reactivity; challenges for diagnosis</span>. 
                     <span tagx="source">Rev Salud Publica (Bogota)</span>
                     <span tagx="volume">10</span>: 
                     <span tagx="fpage">299</span>–
                     <span tagx="lpage">307</span>
                     <span class="pub-id'">[19039426]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Mansfield1"></a>
                  <span class="label">7</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Mansfield</span>
                        <span tagx="given-names">KL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Horton</span>
                        <span tagx="given-names">DL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Johnson</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Li</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Barrett</span>
                        <span tagx="given-names">AD</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">Flavivirus-induced antibody cross-reactivity</span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">92</span>: 
                     <span tagx="fpage">2821</span>–
                     <span tagx="lpage">2829</span>
                     <span class="pub-id'">[21900425]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Koraka1"></a>
                  <span class="label">8</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Koraka</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Zeller</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Niedrig</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Osterhaus</span>
                        <span tagx="given-names">AD</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Groen</span>
                        <span tagx="given-names">J</span>,
                     </span> (
                     <span tagx="year">2002</span>) 
                     <span tagx="article-title">Reactivity of serum samples from patients with a flavivirus infection measured by
                        immunofluorescence assay and ELISA
                     </span>. 
                     <span tagx="source">Microbes Infect</span>
                     <span tagx="volume">4</span>: 
                     <span tagx="fpage">1209</span>–
                     <span tagx="lpage">1215</span>
                     <span class="pub-id'">[12467761]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Weidmann1"></a>
                  <span class="label">9</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Weidmann</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Faye</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Faye</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kranaster</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Marx</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">Improved LNA probe-based assay for the detection of African and South American yellow
                        fever virus strains
                     </span>. 
                     <span tagx="source">J Clin Virol</span>
                     <span tagx="volume">48</span>: 
                     <span tagx="fpage">187</span>–
                     <span tagx="lpage">192</span>
                     <span class="pub-id'">[20556888]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Nunes1"></a>
                  <span class="label">10</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Nunes</span>
                        <span tagx="given-names">MR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Palacios</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nunes</span>
                        <span tagx="given-names">KN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Casseb</span>
                        <span tagx="given-names">SM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Martins</span>
                        <span tagx="given-names">LC</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">Evaluation of two molecular methods for the detection of Yellow fever virus genome</span>. 
                     <span tagx="source">J Virol Methods</span>
                     <span tagx="volume">174</span>: 
                     <span tagx="fpage">29</span>–
                     <span tagx="lpage">34</span>
                     <span class="pub-id'">[21419803]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Dash1"></a>
                  <span class="label">11</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Dash</span>
                        <span tagx="given-names">PK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Boutonnier</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Prina</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sharma</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Reiter</span>
                        <span tagx="given-names">P</span>,
                     </span> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Development of a SYBR green I based RT-PCR assay for yellow fever virus: application
                        in assessment of YFV infection in Aedes aegypti
                     </span>. 
                     <span tagx="source">Virol J</span>
                     <span tagx="volume">9</span>: 
                     <span tagx="fpage">27</span>
                     <span class="pub-id'">[22264275]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Drosten1"></a>
                  <span class="label">12</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Drosten</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gottig</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Schilling</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Asper</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Panning</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2002</span>) 
                     <span tagx="article-title">Rapid detection and quantification of RNA of Ebola and Marburg viruses, Lassa virus,
                        Crimean-Congo hemorrhagic fever virus, Rift Valley fever virus, dengue virus, and
                        yellow fever virus by real-time reverse transcription-PCR
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">40</span>: 
                     <span tagx="fpage">2323</span>–
                     <span tagx="lpage">2330</span>
                     <span class="pub-id'">[12089242]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Bae1"></a>
                  <span class="label">13</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Bae</span>
                        <span tagx="given-names">HG</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nitsche</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Teichmann</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Biel</span>
                        <span tagx="given-names">SS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Niedrig</span>
                        <span tagx="given-names">M</span>,
                     </span> (
                     <span tagx="year">2003</span>) 
                     <span tagx="article-title">Detection of yellow fever virus: a comparison of quantitative real-time PCR and plaque
                        assay
                     </span>. 
                     <span tagx="source">J Virol Methods</span>
                     <span tagx="volume">110</span>: 
                     <span tagx="fpage">185</span>–
                     <span tagx="lpage">191</span>
                     <span class="pub-id'">[12798247]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Kwallah1"></a>
                  <span class="label">14</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Kwallah</span>
                        <span tagx="given-names">AO</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Inoue</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Muigai</span>
                        <span tagx="given-names">AW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kubo</span>
                        <span tagx="given-names">T</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sang</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">A Real-Time Reverse Transcription Loop-Mediated Isothermal Amplification Assay For
                        The Rapid Detection Of Yellow Fever Virus
                     </span>. 
                     <span tagx="source">J Virol Methods</span>
                     <span tagx="volume">193</span>: 
                     <span tagx="fpage">23</span>–
                     <span tagx="lpage">27</span>
                     <span class="pub-id'">[23692685]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Domingo1"></a>
                  <span class="label">15</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Domingo</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Patel</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Yillah</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Weidmann</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Mendez</span>
                        <span tagx="given-names">JA</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Advanced yellow fever virus genome detection in point-of-care facilities and reference
                        laboratories
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">50</span>: 
                     <span tagx="fpage">4054</span>–
                     <span tagx="lpage">4060</span>
                     <span class="pub-id'">[23052311]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Piepenburg1"></a>
                  <span class="label">16</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Piepenburg</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Williams</span>
                        <span tagx="given-names">CH</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Stemple</span>
                        <span tagx="given-names">DL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Armes</span>
                        <span tagx="given-names">NA</span>,
                     </span> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">DNA detection using recombination proteins</span>. 
                     <span tagx="source">PLoS Biol</span>
                     <span tagx="volume">4</span>: 
                     <span tagx="fpage">e204</span>
                     <span class="pub-id'">[16756388]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-AbdElWahed1"></a>
                  <span class="label">17</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Abd El Wahed</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">El-Deeb</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">El-Tholoth</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Abd El Kader</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ahmed</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">A Portable Reverse Transcription Recombinase Polymerase Amplification Assay for Rapid
                        Detection of Foot-and-Mouth Disease Virus
                     </span>. 
                     <span tagx="source">PLoS One</span>
                     <span tagx="volume">8</span>: 
                     <span tagx="fpage">e71642</span>
                     <span class="pub-id'">[23977101]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Diallo1"></a>
                  <span class="label">18</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Diallo</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sall</span>
                        <span tagx="given-names">AA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Buenemann</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chen</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Faye</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Landscape ecology of sylvatic chikungunya virus and mosquito vectors in southeastern
                        Senegal
                     </span>. 
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="volume">6</span>: 
                     <span tagx="fpage">e1649</span>
                     <span class="pub-id'">[22720097]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Mark1"></a>
                  <span class="label">19</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Mark</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Haeberle</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Roth</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">von Stetten</span>
                        <span tagx="given-names">F</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Zengerle</span>
                        <span tagx="given-names">R</span>,
                     </span> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">Microfluidic lab-on-a-chip platforms: requirements, characteristics and applications</span>. 
                     <span tagx="source">Chem Soc Rev</span>
                     <span tagx="volume">39</span>: 
                     <span tagx="fpage">1153</span>–
                     <span tagx="lpage">1182</span>
                     <span class="pub-id'">[20179830]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Mark2"></a>
                  <span class="label">20</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Mark</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Weber</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lutz</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Focke</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Zengerle</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">Aliquoting on the centrifugal microfluidic platform based on centrifugo-pneumatic
                        valves
                     </span>. 
                     <span tagx="source">Microfluidics and Nanofluidics</span>
                     <span tagx="volume">10</span>: 
                     <span tagx="fpage">1279</span>–
                     <span tagx="lpage">1288</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Focke1"></a>
                  <span class="label">21</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Focke</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Stumpf</span>
                        <span tagx="given-names">F</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Faltin</span>
                        <span tagx="given-names">B</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Reith</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Bamarni</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">Microstructuring of polymer films for sensitive genotyping by real-time PCR on a centrifugal
                        microfluidic platform
                     </span>. 
                     <span tagx="source">Lab Chip</span>
                     <span tagx="volume">10</span>: 
                     <span tagx="fpage">2519</span>–
                     <span tagx="lpage">2526</span>
                     <span class="pub-id'">[20607174]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Barrett1"></a>
                  <span class="label">22</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Barrett</span>
                        <span tagx="given-names">AD</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Teuwen</span>
                        <span tagx="given-names">DE</span>,
                     </span> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Yellow fever vaccine - how does it work and why do rare cases of serious adverse events
                        take place?
                     </span>
                     <span tagx="source">Curr Opin Immunol</span>
                     <span tagx="volume">21</span>: 
                     <span tagx="fpage">308</span>–
                     <span tagx="lpage">313</span>
                     <span class="pub-id'">[19520559]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Domingo2"></a>
                  <span class="label">23</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Domingo</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Niedrig</span>
                        <span tagx="given-names">M</span>,
                     </span> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Safety of 17D derived yellow fever vaccines</span>. 
                     <span tagx="source">Expert Opin Drug Saf</span>
                     <span tagx="volume">8</span>: 
                     <span tagx="fpage">211</span>–
                     <span tagx="lpage">221</span>
                     <span class="pub-id'">[19309249]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Domingo3"></a>
                  <span class="label">24</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Domingo</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Yactayo</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Agbenu</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Demanou</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Schulz</span>
                        <span tagx="given-names">AR</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">Detection of yellow fever 17D genome in urine</span>. 
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">49</span>: 
                     <span tagx="fpage">760</span>–
                     <span tagx="lpage">762</span>
                     <span class="pub-id'">[21106799]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Domingo4"></a>
                  <span class="label">25</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Domingo</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Escadafal</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rumer</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Mendez</span>
                        <span tagx="given-names">JA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Garcia</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">First international external quality assessment study on molecular and serological
                        methods for yellow fever diagnosis
                     </span>. 
                     <span tagx="source">PLoS One</span>
                     <span tagx="volume">7</span>: 
                     <span tagx="fpage">e36291</span>
                     <span class="pub-id'">[22570700]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Euler1"></a>
                  <span class="label">26</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Euler</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Wang</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Heidenreich</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Patel</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Strohmeier</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">Development of a panel of recombinase polymerase amplification assays for detection
                        of biothreat agents
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="volume">51</span>: 
                     <span tagx="fpage">1110</span>–
                     <span tagx="lpage">1117</span>
                     <span class="pub-id'">[23345286]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Escadafal1"></a>
                  <span class="label">27</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Escadafal</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Paweska</span>
                        <span tagx="given-names">JT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Grobbelaar</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">le Roux</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Bouloy</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">International external quality assessment of molecular detection of rift valley Fever
                        virus
                     </span>. 
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="volume">7</span>: 
                     <span tagx="fpage">e2244</span>
                     <span class="pub-id'">[23717706]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Domingo5"></a>
                  <span class="label">28</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Domingo</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Niedrig</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Teichmann</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kaiser</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rumer</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">2nd International external quality control assessment for the molecular diagnosis
                        of dengue infections
                     </span>. 
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="volume">4</span>: 
                     <span tagx="fpage">0000833</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Linke1"></a>
                  <span class="label">29</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Linke</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ellerbrok</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Niedrig</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nitsche</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Pauli</span>
                        <span tagx="given-names">G</span>,
                     </span> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">Detection of West Nile virus lineages 1 and 2 by real-time PCR</span>. 
                     <span tagx="source">J Virol Methods</span>
                     <span tagx="volume">146</span>: 
                     <span tagx="fpage">355</span>–
                     <span tagx="lpage">358</span>
                     <span class="pub-id'">[17604132]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Achazi1"></a>
                  <span class="label">30</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Achazi</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nitsche</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Patel</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Radonic</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Donoso Mantke</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">Detection and differentiation of tick-borne encephalitis virus subtypes by a reverse
                        transcription quantitative real-time PCR and pyrosequencing
                     </span>. 
                     <span tagx="source">J Virol Methods</span>
                     <span tagx="volume">171</span>: 
                     <span tagx="fpage">34</span>–
                     <span tagx="lpage">39</span>
                     <span class="pub-id'">[20933016]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Patel1"></a>
                  <span class="label">31</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Patel</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Landt</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kaiser</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Faye</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Koppe</span>
                        <span tagx="given-names">T</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2013</span>) 
                     <span tagx="article-title">Development of one-step quantitative reverse transcription PCR for the rapid detection
                        of flaviviruses
                     </span>. 
                     <span tagx="source">Virol J</span>
                     <span tagx="volume">10</span>: 
                     <span tagx="fpage">58</span>
                     <span class="pub-id'">[23410000]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Weidmann2"></a>
                  <span class="label">32</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Weidmann</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sanchez-Seco</span>
                        <span tagx="given-names">MP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sall</span>
                        <span tagx="given-names">AA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ly</span>
                        <span tagx="given-names">PO</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Thiongane</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2008</span>) 
                     <span tagx="article-title">Rapid detection of important human pathogenic Phleboviruses</span>. 
                     <span tagx="source">J Clin Virol</span>
                     <span tagx="volume">41</span>: 
                     <span tagx="fpage">138</span>–
                     <span tagx="lpage">142</span>
                     <span class="pub-id'">[18006376]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002730-Weidmann3"></a>
                  <span class="label">33</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Weidmann</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Muhlberger</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hufert</span>
                        <span tagx="given-names">FT</span>,
                     </span> (
                     <span tagx="year">2004</span>) 
                     <span tagx="article-title">Rapid detection protocol for filoviruses</span>. 
                     <span tagx="source">J Clin Virol</span>
                     <span tagx="volume">30</span>: 
                     <span tagx="fpage">94</span>–
                     <span tagx="lpage">99</span>
                     <span class="pub-id'">[15072761]</span>
                  </div>
               </li>
            </ul>
         </div>
      </div>
   </div>
</html>