<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <div xmlns="" tagxxx="article">
      <head>
         <meta content="Centers for Disease Control and Prevention" name="citation_publisher"></meta>
         <meta content="10.3201/eid1408.080287" name="citation_doi"></meta>
         <meta content="Genetic and Serologic Properties of Zika Virus Associated with an Epidemic, Yap State, Micronesia, 2007 " name="citation_title"></meta>
         <title>Genetic and Serologic Properties of Zika Virus Associated with an Epidemic, Yap State,
            Micronesia, 2007 
         </title>
         <div tagxxx="contrib-group">
            <meta content="Robert S. Lanciotti" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="Olga L. Kosoy" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="Janeen J. Laven" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="Jason O. Velez" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="Amy J. Lambert" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="Alison J. Johnson" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="Stephanie M. Stanfield" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="Mark R. Duffy" name="citation_author"></meta>
            <meta content="" name="citation_author_institution_ref"></meta>
            <meta content="* " name="citation_author_institution"></meta>
         </div>
         <meta content="rsl2@cdc.gov" name="email"></meta>
         <meta content="" name="conflict"></meta>
         <meta content="" name="contribs"></meta>
         <meta content="--" name="citation_pubdate"></meta>
         <meta content="Copyright : " name="citation_copyright"></meta>
         <meta content="Licence " name="citation_licence"></meta>
         <meta content="" name="citation_funding"></meta>
         <meta content="" name="citation_pagecount"></meta>
         <meta content="" name="citation_data"></meta>
      </head>
      <div id="abstract" tag="abstract">
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">One-sentence summary for table of contents: The full coding region nucleic acid sequence
            and serologic properties of the virus were identified.
         </p>
      </div>
      <div id="abstract" tag="abstract">
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Zika virus (ZIKV) is a mosquito-borne flavivirus first isolated in Uganda from a sentinel
            monkey in 1947. Mosquito and sentinel animal surveillance studies have demonstrated
            that ZIKV is endemic to Africa and Southeast Asia, yet reported human cases are rare,
            with &lt;10 cases reported in the literature. In June 2007, an epidemic of fever and
            rash associated with ZIKV was detected in Yap State, Federated States of Micronesia.
            We report the genetic and serologic properties of the ZIKV associated with this epidemic.
         </p>
      </div>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Zika virus (ZIKV) is a mosquito-transmitted virus in the family 
         <i>Flaviviridae</i> and genus 
         <i>Flavivirus</i>. It was initially isolated in 1947 from blood of a febrile sentinel rhesus monkey
         during a yellow fever study in the Zika forest of Uganda (
         <span ref-type="bibr" rid="R1">
            <i>1</i>
         </span>). The virus was subsequently isolated from a pool of 
         <i>Aedes africanus</i> mosquitoes collected in 1948 from the same region of the Zika forest; a serologic
         survey conducted at that time showed that 6.1% of the residents in nearby regions
         of Uganda had specific antibodies to ZIKV (
         <span ref-type="bibr" rid="R1">
            <i>1</i>
         </span>,
         <span ref-type="bibr" rid="R2">
            <i>2</i>
         </span>).
      </p>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Over the next 20 years, several ZIKV isolates were obtained from 
         <i>Aedes</i> spp. in Africa (
         <i>Ae</i>. 
         <i>africanus</i>) and Malaysia (
         <i>Ae</i>. 
         <i>aegypti</i>), implicating these species as likely epidemic or enzootic vectors (
         <span ref-type="bibr" rid="R3">
            <i>3</i>
         </span>–
         <span ref-type="bibr" rid="R5">
            <i>5</i>
         </span>). Several ZIKV human isolates were also obtained in the 1960s and 1970s from East
         and West Africa during routine arbovirus surveillance studies in the absence of epidemics
         (
         <span ref-type="bibr" rid="R6">
            <i>6</i>
         </span>–
         <span ref-type="bibr" rid="R8">
            <i>8</i>
         </span>). Additional serologic studies in the 1950s and 1960s detected ZIKV infections among
         humans in Egypt, Nigeria, Uganda, India, Malaysia, Indonesia, Pakistan, Thailand,
         North Vietnam, and the Philippines (
         <span ref-type="bibr" rid="R5">
            <i>5</i>
         </span>). These data strongly suggest widespread occurrence of ZIKV from Africa to Southeast
         Asia west and north of the Wallace line.
      </p>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In 1977, ZIKV infection was confirmed among 7 patients in central Java, Indonesia,
         during an acute fever study (
         <span ref-type="bibr" rid="R9">
            <i>9</i>
         </span>). Data on these 7 ZIKV cases and several previously reported human infections indicated
         that clinical characteristics of infection with ZIKV included fever, headache, malaise,
         stomach ache, dizziness, anorexia, and maculopapular rash; in all cases infection
         appeared relatively mild, self-limiting, and nonlethal (
         <span ref-type="bibr" rid="R6">
            <i>6</i>
         </span>,
         <span ref-type="bibr" rid="R8">
            <i>8</i>
         </span>–
         <span ref-type="bibr" rid="R10">
            <i>10</i>
         </span>).
      </p>
      <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In April 2007, an epidemic of rash, conjunctivitis, and arthralgia was noted by physicians
         in Yap State, Federated States of Micronesia (
         <span ref-type="bibr" rid="R11">
            <i>11</i>
         </span>). Laboratory testing with a rapid assay suggested that a dengue virus (DENV) was
         the causative agent. In June 2007, samples were sent for confirmatory testing to the
         Arbovirus Diagnostic Laboratory at the Centers for Disease Control and Prevention
         (CDC, Fort Collins, CO, USA). Serologic testing by immunoglobulin (Ig) M–capture ELISA
         with DENV antigen confirmed recent flavivirus infection in several patients. Testing
         by reverse transcription–PCR (RT-PCR) with flavivirus consensus primers generated
         DNA fragments, which when subjected to nucleic acid sequencing, demonstrated ≈90%
         nucleotide identity with ZIKV. These findings indicated that ZIKV was the causative
         agent of the Yap epidemic. We report serologic parameters of the immune response among
         ZIKV-infected humans, data on estimated levels of viremia, and the complete coding
         region nucleic acid sequence of ZIKV associated with this epidemic.
      </p>
      <div id="">
         <h2>Methods</h2>
         <div id="">
            <h3>Analysis of Patient Samples</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Details of the epidemic, including clinical and laboratory findings for all patients,
               will be reported elsewhere (M.R. Duffy et al., unpub. data). A subset of ZIKV-infected
               patients for whom acute- and convalescent-phase paired serum specimens had been collected
               was analyzed by using several serologic assays to evaluate the extent of cross-reactivity
               to several related flaviviruses. Patients were classified as primary flavivirus/ZIKV
               infected or secondary flavivirus/ZIKV probable infected. Primary flavivirus/ZIKV–infected
               patients were those in whom acute-phase serum specimens (&lt;10 days) had no detectible
               antibodies (by IgG ELISA and plaque-reduction neutralization test [PRNT]) to any of
               the heterologous flaviviruses tested (
               <span ref-type="table" rid="T1">Tables 1</span>, 
               <span ref-type="table" rid="T2">2</span>) and were either IgM-positive in their acute-phase specimen or IgM and IgG positive
               for ZIKV in a convalescent-phase specimen (seroconversion). Secondary flavivirus/ZIKV
               probable–infected patients were those who had detectable antibodies to 
               <u>&gt;</u>1 heterologous flaviviruses in their acute-phase specimen and were also IgM positive
               for ZIKV in their acute-phase specimen, or IgM and IgG positive for ZIKV in their
               convalescent-phase specimen. The designation “ZIKV probable” was used because secondary
               flavivirus infections demonstrate extensive cross-reactivity with other flaviviruses,
               and in some cases, higher serologic reactivity to the original infecting flavivirus
               (“original antigenic sin” phenomenon). Thus, in secondary flavivirus infections shown
               in 
               <span ref-type="table" rid="T1">Tables 1</span> and 
               <span ref-type="table" rid="T2">2</span>, serologic data alone is insufficient to confirm ZIKV as the recently infecting flavivirus.
               However, these secondary flavivirus/ZIKV probable infections were likely recent ZIKV
               infections because ZIKV was the only virus detected during the epidemic in Yap, a
               relatively small and isolated island (
               <span ref-type="bibr" rid="R11">
                  <i>11</i>
               </span>).
            </p>
            <div tagxxx="table-wrap">
               <div tagxxx="label">Table 1</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">IgG and IgM testing with heterologous flaviviruses of patients infected with ZIKV,
                     Yap State, Micronesia, 2007*
                  </div>
               </caption>
               <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">hsidesgroups
                  <col>531</col>
                  <col>701</col>
                  <col>541</col>
                  <col>91</col>
                  <col>491</col>
                  <col>491</col>
                  <col>491</col>
                  <col>491</col>
                  <col>491</col>
                  <col>491</col>
                  <thead>
                     <tr>
                        <div tagxxx="th">Patient</div>
                        <div tagxxx="th">Days after onset</div>
                        <div tagxxx="th"> IgG
                           <div tagxxx="hr"></div>
                        </div>
                        <div tagxxx="th"></div>
                        <div tagxxx="th">IgM
                           <div tagxxx="hr"></div>
                        </div>
                     </tr>
                     <tr>
                        <div tagxxx="th">ZIKV</div>
                        <div tagxxx="th">ZIKV</div>
                        <div tagxxx="th">DENV</div>
                        <div tagxxx="th">YFV</div>
                        <div tagxxx="th">JEV</div>
                        <div tagxxx="th">MVEV</div>
                        <div tagxxx="th">WNV</div>
                     </tr>
                  </thead>
                  <div tagxxx="tbody">
                     <tr>
                        <td>2topleftcol1Primary flavivirus ZIKV</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                     </tr>
                     <tr>
                        <td>topleftrow11822a</td>
                        <td>topcenter115</td>
                        <td>topcenter111.5</td>
                        <td>topleft11</td>
                        <td>topcenter1123.2</td>
                        <td>topcenter111.3</td>
                        <td>topcenter111.4</td>
                        <td>topcenter111.7</td>
                        <td>topcenter111.1</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11822b</td>
                        <td>topcenter1110</td>
                        <td>topcenter111.2</td>
                        <td>topleft11</td>
                        <td>topcenter1139.5</td>
                        <td>topcenter111.2</td>
                        <td>topcenter111.0</td>
                        <td>topcenter112.4</td>
                        <td>topcenter111.2</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11822c</td>
                        <td>topcenter1124</td>
                        <td>topcenter113.3</td>
                        <td>topleft11</td>
                        <td>topcenter1113.1</td>
                        <td>topcenter112.7</td>
                        <td>topcenter110.63</td>
                        <td>topcenter111.8</td>
                        <td>topcenter111.3</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11830a</td>
                        <td>topcenter112</td>
                        <td>topcenter111.1</td>
                        <td>topleft11</td>
                        <td>topcenter111.3</td>
                        <td>topcenter114.4</td>
                        <td>topcenter110.48</td>
                        <td>topcenter114.4</td>
                        <td>topcenter112.9</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11830b</td>
                        <td>topcenter1121</td>
                        <td>topcenter111.8</td>
                        <td>topleft11</td>
                        <td>topcenter1116.3</td>
                        <td>topcenter111.9</td>
                        <td>topcenter110.63</td>
                        <td>topcenter111.3</td>
                        <td>topcenter111.6</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11849a</td>
                        <td>topcenter113</td>
                        <td>topcenter111.5</td>
                        <td>topleft11</td>
                        <td>topcenter114.5</td>
                        <td>topcenter110.92</td>
                        <td>topcenter110.95</td>
                        <td>topcenter111.2</td>
                        <td>topcenter110.66</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11849b</td>
                        <td>topcenter1118</td>
                        <td>topcenter113.0</td>
                        <td>topleft11</td>
                        <td>topcenter1118.2</td>
                        <td>topcenter112.2</td>
                        <td>topcenter111.0</td>
                        <td>topcenter112.7</td>
                        <td>topcenter111.5</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11862a</td>
                        <td>topcenter116</td>
                        <td>topcenter111.9</td>
                        <td>topleft11</td>
                        <td>topcenter1125.4</td>
                        <td>topcenter111.7</td>
                        <td>topcenter111.1</td>
                        <td>topcenter111.8</td>
                        <td>topcenter111.0</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11862b
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter1120
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter112.6
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topleft11
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter1115.4
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter112
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter111.1
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter112.3
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter111.1
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter11Eq
                           <div tagxxx="hr"></div>
                        </td>
                     </tr>
                     <tr>
                        <td>3topleftcol1Secondary flavivirus ZIKV (probable)</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                     </tr>
                     <tr>
                        <td>topleftrow11817a</td>
                        <td>topcenter111</td>
                        <td>topcenter115.9</td>
                        <td>topleft11</td>
                        <td>topcenter111.4</td>
                        <td>topcenter111.7</td>
                        <td>topcenter110.8</td>
                        <td>topcenter111.7</td>
                        <td>topcenter110.7</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11817b</td>
                        <td>topcenter1119</td>
                        <td>topcenter115.7</td>
                        <td>topleft11</td>
                        <td>topcenter118.1</td>
                        <td>topcenter115.1</td>
                        <td>topcenter112.1</td>
                        <td>topcenter111.7</td>
                        <td>topcenter111.0</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11833a</td>
                        <td>topcenter111</td>
                        <td>topcenter113.4</td>
                        <td>topleft11</td>
                        <td>topcenter111.7</td>
                        <td>topcenter113.7</td>
                        <td>topcenter111.0</td>
                        <td>topcenter112.8</td>
                        <td>topcenter111.3</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11833b</td>
                        <td>topcenter1119</td>
                        <td>topcenter118.2</td>
                        <td>topleft11</td>
                        <td>topcenter113.1</td>
                        <td>topcenter112.3</td>
                        <td>topcenter110.9</td>
                        <td>topcenter112.5</td>
                        <td>topcenter111.3</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11844a</td>
                        <td>topcenter112</td>
                        <td>topcenter113.8</td>
                        <td>topleft11</td>
                        <td>topcenter113.8</td>
                        <td>topcenter116.8</td>
                        <td>topcenter112.0</td>
                        <td>topcenter1121.5</td>
                        <td>topcenter110.7</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11844b</td>
                        <td>topcenter1116</td>
                        <td>topcenter118.5</td>
                        <td>topleft11</td>
                        <td>topcenter1112.7</td>
                        <td>topcenter1114.9</td>
                        <td>topcenter117.0</td>
                        <td>topcenter1142.9</td>
                        <td>topcenter111.6</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11955a</td>
                        <td>topcenter111</td>
                        <td>topcenter115.0</td>
                        <td>topleft11</td>
                        <td>topcenter111.8</td>
                        <td>topcenter113.7</td>
                        <td>topcenter111.0</td>
                        <td>topcenter113.4</td>
                        <td>topcenter112.4</td>
                        <td>topcenter11Eq</td>
                     </tr>
                     <tr>
                        <td>topleftrow11955b</td>
                        <td>topcenter1114</td>
                        <td>topcenter1126.6</td>
                        <td>topleft11</td>
                        <td>topcenter1110.9</td>
                        <td>topcenter113.4</td>
                        <td>topcenter110.8</td>
                        <td>topcenter111.7</td>
                        <td>topcenter114.0</td>
                        <td>topcenter11Eq</td>
                     </tr>
                     <tr>
                        <td>topleftrow11968a</td>
                        <td>topcenter111</td>
                        <td>topcenter114.0</td>
                        <td>topleft11</td>
                        <td>topcenter111.7</td>
                        <td>topcenter111.3</td>
                        <td>topcenter110.6</td>
                        <td>topcenter111.2</td>
                        <td>topcenter111.2</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11968b</td>
                        <td>topcenter113</td>
                        <td>topcenter1112.3</td>
                        <td>topleft11</td>
                        <td>topcenter1120.4</td>
                        <td>topcenter112.9</td>
                        <td>topcenter110.8</td>
                        <td>topcenter110.9</td>
                        <td>topcenter112.0</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11839a</td>
                        <td>topcenter113</td>
                        <td>topcenter111</td>
                        <td>topleft11</td>
                        <td>topcenter110.92</td>
                        <td>topcenter113.4</td>
                        <td>topcenter110.7</td>
                        <td>topcenter112.7</td>
                        <td>topcenter112.1</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11839b</td>
                        <td>topcenter1120</td>
                        <td>topcenter114.9</td>
                        <td>topleft11</td>
                        <td>topcenter1117.2</td>
                        <td>topcenter112.2</td>
                        <td>topcenter112.1</td>
                        <td>topcenter111.9</td>
                        <td>topcenter111.8</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11847a</td>
                        <td>topcenter115</td>
                        <td>topcenter110.9</td>
                        <td>topleft11</td>
                        <td>topcenter110.94</td>
                        <td>topcenter114.1</td>
                        <td>topcenter114.1</td>
                        <td>topcenter112.3</td>
                        <td>topcenter111.3</td>
                        <td>topcenter11–</td>
                     </tr>
                     <tr>
                        <td>topleftrow11847b</td>
                        <td>topcenter118</td>
                        <td>topcenter1114.1</td>
                        <td>topleft11</td>
                        <td>topcenter1121.5</td>
                        <td>topcenter111.4</td>
                        <td>topcenter113.3</td>
                        <td>topcenter111.1</td>
                        <td>topcenter112.6</td>
                        <td>topcenter11–</td>
                     </tr>
                  </div>
               </table>
               <div tagxxx="table-wrap-foot">
                  <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">*Ig, immunoglobulin; ZIKV, Zika virus; DENV, dengue virus type 1–4 mixture; YFV, yellow
                     fever virus; JEV, Japanese encephalitis virus; MVEV, Murray Valley encephalitis virus;
                     WNV, West Nile virus; –, negative. Eq, result in equivocal range of the assay. IgG
                     and IgM testing was conducted by ELISA except for WNV, which was tested by microsphere
                     assay; ELISA values are patient optical densities divided by negative control optical
                     densities; &lt;2, negative; 2–3 equivocal; &gt;3 positive.
                  </p>
               </div>
            </div>
            <div tagxxx="table-wrap">
               <div tagxxx="label">Table 2</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Neutralization testing with heterologous flaviviruses of patients infected with ZIKV,
                     Yap State, Micronesia, 2007*
                  </div>
               </caption>
               <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">hsidesgroups
                  <col>521</col>
                  <col>451</col>
                  <col>411</col>
                  <col>411</col>
                  <col>411</col>
                  <col>411</col>
                  <col>411</col>
                  <col>351</col>
                  <col>351</col>
                  <col>351</col>
                  <col>351</col>
                  <col>351</col>
                  <thead>
                     <tr>
                        <div tagxxx="th">Patient</div>
                        <div tagxxx="th">Days after onset</div>
                        <div tagxxx="th">PRNT
                           <div tagxxx="sub">90</div> titer
                           <div tagxxx="hr"></div>
                        </div>
                     </tr>
                     <tr>
                        <div tagxxx="th">ZIKV</div>
                        <div tagxxx="th">DENV1</div>
                        <div tagxxx="th">DENV2</div>
                        <div tagxxx="th">DENV3</div>
                        <div tagxxx="th">DENV4</div>
                        <div tagxxx="th">JEV</div>
                        <div tagxxx="th">YFV</div>
                        <div tagxxx="th">WNV</div>
                        <div tagxxx="th">SLEV</div>
                        <div tagxxx="th">MVEV</div>
                     </tr>
                  </thead>
                  <div tagxxx="tbody">
                     <tr>
                        <td>2topleftcol1Primary flavivirus ZIKV</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                     </tr>
                     <tr>
                        <td>topleftrow11822a</td>
                        <td>topcenter115</td>
                        <td>topcenter11320</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                     </tr>
                     <tr>
                        <td>topleftrow11822b</td>
                        <td>topcenter1110</td>
                        <td>topcenter112,560</td>
                        <td>topcenter1110</td>
                        <td>topcenter1110</td>
                        <td>topcenter1110</td>
                        <td>topcenter1110</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                     </tr>
                     <tr>
                        <td>topleftrow11822c</td>
                        <td>topcenter1124</td>
                        <td>topcenter115,120</td>
                        <td>topcenter1110</td>
                        <td>topcenter1110</td>
                        <td>topcenter1110</td>
                        <td>topcenter1110</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                     </tr>
                     <tr>
                        <td>topleftrow11830a</td>
                        <td>topcenter112</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11NT‡</td>
                        <td>topcenter11NT</td>
                        <td>topcenter11NT</td>
                        <td>topcenter11NT</td>
                        <td>topcenter11NT</td>
                        <td>topcenter11NT</td>
                        <td>topcenter11NT</td>
                        <td>topcenter11NT</td>
                     </tr>
                     <tr>
                        <td>topleftrow11830b</td>
                        <td>topcenter1121</td>
                        <td>topcenter112,560</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                     </tr>
                     <tr>
                        <td>topleftrow11849a</td>
                        <td>topcenter113</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                     </tr>
                     <tr>
                        <td>topleftrow11849b</td>
                        <td>topcenter1118</td>
                        <td>topcenter1110,240</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter1120</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                     </tr>
                     <tr>
                        <td>topleftrow11862a</td>
                        <td>topcenter116</td>
                        <td>topcenter11320</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                     </tr>
                     <tr>
                        <td>topleftrow11862b
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter1120
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter112,560
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter1110
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter1110
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter11&lt;10
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter11&lt;10
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter11&lt;10
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter11&lt;10
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter11&lt;10
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter1110
                           <div tagxxx="hr"></div>
                        </td>
                        <td>topcenter11&lt;10
                           <div tagxxx="hr"></div>
                        </td>
                     </tr>
                     <tr>
                        <td>3topleftcol1Secondary flavivirus ZIKV (probable)</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                     </tr>
                     <tr>
                        <td>topleftrow11817a</td>
                        <td>topcenter111</td>
                        <td>topcenter1180</td>
                        <td>topcenter1180</td>
                        <td>topcenter11160</td>
                        <td>topcenter11320</td>
                        <td>topcenter11160</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter1140</td>
                        <td>topcenter1140</td>
                     </tr>
                     <tr>
                        <td>topleftrow11817b</td>
                        <td>topcenter1119</td>
                        <td>topcenter1110,240</td>
                        <td>topcenter112,560</td>
                        <td>topcenter1120,480</td>
                        <td>topcenter115,120</td>
                        <td>topcenter115,120</td>
                        <td>topcenter1120</td>
                        <td>topcenter11320</td>
                        <td>topcenter11160</td>
                        <td>topcenter111,280</td>
                        <td>topcenter11640</td>
                     </tr>
                     <tr>
                        <td>topleftrow11833a</td>
                        <td>topcenter111</td>
                        <td>topcenter11160</td>
                        <td>topcenter11320</td>
                        <td>topcenter1180</td>
                        <td>topcenter1140</td>
                        <td>topcenter1120</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                     </tr>
                     <tr>
                        <td>topleftrow11833b</td>
                        <td>topcenter1119</td>
                        <td>topcenter1181,920</td>
                        <td>topcenter1120,480</td>
                        <td>topcenter115,120</td>
                        <td>topcenter115,120</td>
                        <td>topcenter111,280</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter1180</td>
                        <td>topcenter11320</td>
                        <td>topcenter11320</td>
                     </tr>
                     <tr>
                        <td>topleftrow11844a</td>
                        <td>topcenter112</td>
                        <td>topcenter1120</td>
                        <td>topcenter111,280</td>
                        <td>topcenter11640</td>
                        <td>topcenter11320</td>
                        <td>topcenter11160</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter115</td>
                        <td>topcenter1120</td>
                        <td>topcenter1120</td>
                     </tr>
                     <tr>
                        <td>topleftrow11844b</td>
                        <td>topcenter1116</td>
                        <td>topcenter1110,240</td>
                        <td>topcenter1140,980</td>
                        <td>topcenter1110,240</td>
                        <td>topcenter115,120</td>
                        <td>topcenter111,280</td>
                        <td>topcenter115</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11160</td>
                        <td>topcenter11640</td>
                        <td>topcenter11640</td>
                     </tr>
                     <tr>
                        <td>topleftrow11955a</td>
                        <td>topcenter111</td>
                        <td>topcenter1140</td>
                        <td>topcenter111,280</td>
                        <td>topcenter11640</td>
                        <td>topcenter11160</td>
                        <td>topcenter11320</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter1120</td>
                        <td>topcenter1120</td>
                     </tr>
                     <tr>
                        <td>topleftrow11955b</td>
                        <td>topcenter1114</td>
                        <td>topcenter11163,840</td>
                        <td>topcenter1181,920</td>
                        <td>topcenter1120,480</td>
                        <td>topcenter1110,240</td>
                        <td>topcenter115,120</td>
                        <td>topcenter1110</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11640</td>
                        <td>topcenter112,560</td>
                        <td>topcenter111,280</td>
                     </tr>
                     <tr>
                        <td>topleftrow11968a</td>
                        <td>topcenter111</td>
                        <td>topcenter1180</td>
                        <td>topcenter11320</td>
                        <td>topcenter11320</td>
                        <td>topcenter1180</td>
                        <td>topcenter1140</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter1140</td>
                        <td>topcenter1120</td>
                     </tr>
                     <tr>
                        <td>topleftrow11968b</td>
                        <td>topcenter113</td>
                        <td>topcenter1110,240</td>
                        <td>topcenter11640</td>
                        <td>topcenter11640</td>
                        <td>topcenter11160</td>
                        <td>topcenter11160</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter1110</td>
                        <td>topcenter1140</td>
                        <td>topcenter1120</td>
                     </tr>
                     <tr>
                        <td>topleftrow11839a</td>
                        <td>topcenter113</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter1110</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter1140</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                     </tr>
                     <tr>
                        <td>topleftrow11839b</td>
                        <td>topcenter1120</td>
                        <td>topcenter1110,240</td>
                        <td>topcenter1140</td>
                        <td>topcenter11320</td>
                        <td>topcenter1180</td>
                        <td>topcenter1180</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11640</td>
                        <td>topcenter1140</td>
                        <td>topcenter1180</td>
                        <td>topcenter1180</td>
                     </tr>
                     <tr>
                        <td>topleftrow11847a</td>
                        <td>topcenter115</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11640</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter11&lt;10</td>
                     </tr>
                     <tr>
                        <td>topleftrow11847b</td>
                        <td>topcenter118</td>
                        <td>topcenter112,560</td>
                        <td>topcenter1140</td>
                        <td>topcenter11320</td>
                        <td>topcenter11160</td>
                        <td>topcenter1140</td>
                        <td>topcenter11&lt;10</td>
                        <td>topcenter111,280</td>
                        <td>topcenter1180</td>
                        <td>topcenter11320</td>
                        <td>topcenter11320</td>
                     </tr>
                  </div>
               </table>
               <div tagxxx="table-wrap-foot">
                  <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">*PRNT
                     <div tagxxx="sub">90</div> titer, 90% plaque-reduction neutralization test titer; ZIKV, Zika virus; DENV, dengue
                     virus; JEV, Japanese encephalitis virus; YFV, yellow fever virus; WNV, West Nile virus;
                     SLEV, St. Louis encephalitis virus; MVEV, Murray Valley encephalitis virus; NT, not
                     tested (sample depleted).
                  </p>
               </div>
            </div>
         </div>
         <div id="">
            <h3>Serologic Testing</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Acute- and convalescent-phase serum samples were tested by IgG ELISA with ZIKV antigen
               as described for detection of IgG to arboviruses (
               <span ref-type="bibr" rid="R12">
                  <i>12</i>
               </span>). Samples were also tested by IgM ELISA as described with the following viral antigens:
               ZIKV, DENV 1–4 mixture, yellow fever virus (YFV), Japanese encephalitis virus, and
               Murray Valley encephalitis virus (
               <span ref-type="bibr" rid="R13">
                  <i>13</i>
               </span>). Testing for IgM to West Nile virus (WNV) and St. Louis encephalitis virus was performed
               by using a microsphere immunoassay (
               <span ref-type="bibr" rid="R14">
                  <i>14</i>
               </span>). Ratios of patient optical density values to negative control values (P/Ns) were
               calculated for IgG and IgM ELISAs. Values &gt;3 were considered positive, and values
               2–3 were considered equivocal. Neutralizing antibody titers were determined by using
               a PRNT with a 90% cut-off value (
               <span ref-type="bibr" rid="R15">
                  <i>15</i>
               </span>).
            </p>
         </div>
         <div id="">
            <h3>Real-Time RT-PCR</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Two real-time primer/probe sets specific for the ZIKV 2007 strain were designed by
               using ZIKV 2007 nucleotide sequence data in the PrimerExpress software package (Applied
               Biosystems, Foster City, CA, USA). Primers were synthesized by Operon Biotechnologies
               (Huntsville, AL, USA) with 5-FAM as the reporter dye for the probe (
               <span ref-type="table" rid="T3">Table 3</span>). All real-time assays were performed by using the QuantiTect Probe RT-PCR Kit (QIAGEN,
               Valencia, CA, USA) with amplification in the iCycler instrument (Bio-Rad, Hercules,
               CA, USA) following the manufacturer’s protocol. Specificity of the ZIKV primers was
               evaluated by testing the following viral RNAs, all of which yielded negative results:
               DENV-1, DENV-2, DENV-3, DENV-4, WNV, St. Louis encephalitis virus, YFV, Powassan virus,
               Semliki Forest virus, o’nyong-nyong virus, chikungunya virus, and Spondweni virus
               (SPOV).
            </p>
            <div tagxxx="table-wrap">
               <div tagxxx="label">Table 3</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Description and performance characteristics of Zika virus real-time RT-PCR primer/probe
                     sets*
                  </div>
               </caption>
               <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">hsidesgroups
                  <col>811</col>
                  <col>691</col>
                  <col>2001</col>
                  <col>721</col>
                  <col>581</col>
                  <thead>
                     <tr>
                        <div tagxxx="th">Primer</div>
                        <div tagxxx="th">Genome position†</div>
                        <div tagxxx="th">Sequence (5′ → 3′)</div>
                        <div tagxxx="th">Sensitivity, no. copies</div>
                        <div tagxxx="th">Specificity‡</div>
                     </tr>
                  </thead>
                  <div tagxxx="tbody">
                     <tr>
                        <td>topleftrow11ZIKV 835</td>
                        <td>topcenter11835–857</td>
                        <td>topcenter11TTGGTCATGATACTGCTGATTGC</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                     </tr>
                     <tr>
                        <td>topleftrow11ZIKV 911c</td>
                        <td>topcenter11911–890</td>
                        <td>topcenter11CCTTCCACAAAGTCCCTATTGC</td>
                        <td>topcenter11100</td>
                        <td>topcenter11ZIKV</td>
                     </tr>
                     <tr>
                        <td>topleftrow11ZIKV 860-FAM</td>
                        <td>topcenter11860–886</td>
                        <td>topcenter11CGGCATACAGCATCAGGTGCATAGGAG</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                     </tr>
                     <tr>
                        <td>topleftrow11ZIKV 1086</td>
                        <td>topcenter111086–1102</td>
                        <td>topcenter11CCGCTGCCCAACACAAG</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                     </tr>
                     <tr>
                        <td>topleftrow11ZIKV 1162c</td>
                        <td>topcenter111162–1139</td>
                        <td>topcenter11CCACTAACGTTCTTTTGCAGACAT</td>
                        <td>topcenter1125</td>
                        <td>topcenter11ZIKV</td>
                     </tr>
                     <tr>
                        <td>topleftrow11ZIKV 1107-FAM</td>
                        <td>topcenter111107–1137</td>
                        <td>topcenter11AGCCTACCTTGACAAGCAGTCAGACACTCAA</td>
                        <td>topleft11</td>
                        <td>topleft11</td>
                     </tr>
                  </div>
               </table>
               <div tagxxx="table-wrap-foot">
                  <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">*RT-PCR, reverse transcription–PCR; ZIKV, Zika virus.&#x2028;†Based on ZIKV MR 766 GenBank
                     accession no. AY632535.&#x2028;‡ZIKV specificity indicates a positive result with ZIKV only
                     and no reactivity with dengue virus-1 (DENV-1), DENV-2, DENV-3, DENV-4, West Nile
                     virus, St. Louis encephalitis virus, yellow fever virus, Powassan virus, Semliki Forest
                     virus, o’nyong-nyong virus, chikungunya virus, and Spondweni virus.
                  </p>
               </div>
            </div>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Sensitivity of the ZIKV real-time assay was evaluated by testing dilutions of known
               copy numbers of an RNA transcript copy of the ZIKV 2007 sequence. Copy numbers of
               RNA were determined by using the Ribogreen RNA-specific Quantitiation Kit (Invitrogen)
               and the TBE-380 mini-fluorometer (Turner Biosystems, Sunnyvale, CA, USA). RNA transcripts
               ranging from 16,000 to 0.2 copies were tested in quadruplicate to determine the sensitivity
               limit and to construct a standard curve for estimating the genome copy number of ZIKV
               in patient samples. All serum samples obtained during the epidemic were tested for
               ZIKV RNA by using this newly designed real-time RT-PCR. Concentration of viral RNA
               (copies/milliliter) was estimated in ZIKV-positive patients by using the standard
               curve calculated by the iCycler instrument (
               <span ref-type="table" rid="T4">Table 4</span>). All RT-PCR–positive specimens were placed on monolayers of Vero, LLC-MK2, and C6/36
               cells to isolate virus; no specimens showed virus replication.
            </p>
            <div tagxxx="table-wrap">
               <div tagxxx="label">Table 4</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Results of quantitative real-time RT-PCR of samples from ZIKV-positive patients, Yap
                     State, Micronesia, 2007*
                  </div>
               </caption>
               <table xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">hsidesgroups
                  <col>351</col>
                  <col>301</col>
                  <col>451</col>
                  <col>451</col>
                  <col>301</col>
                  <col>511</col>
                  <thead>
                     <tr>
                        <div tagxxx="th">Patient</div>
                        <div tagxxx="th">Days after onset</div>
                        <div tagxxx="th">ZIKV real-time RT-PCR
                           <div tagxxx="hr"></div>
                        </div>
                     </tr>
                     <tr>
                        <div tagxxx="th">Ct-860†</div>
                        <div tagxxx="th">Ct-1107†</div>
                        <div tagxxx="th">Result</div>
                        <div tagxxx="th">Estimated copies/mL‡</div>
                     </tr>
                  </thead>
                  <div tagxxx="tbody">
                     <tr>
                        <td>topleftrow11824</td>
                        <td>topcenter111</td>
                        <td>topcenter1134.3</td>
                        <td>topcenter1134.7</td>
                        <td>topcenter11+</td>
                        <td>topcenter1111,647</td>
                     </tr>
                     <tr>
                        <td>topleftrow11939</td>
                        <td>topcenter112</td>
                        <td>topcenter1132.0</td>
                        <td>topcenter1132.4</td>
                        <td>topcenter11+</td>
                        <td>topcenter1167,817</td>
                     </tr>
                     <tr>
                        <td>topleftrow11947</td>
                        <td>topcenter112</td>
                        <td>topcenter1134.3</td>
                        <td>topcenter1133.9</td>
                        <td>topcenter11+</td>
                        <td>topcenter1121,495</td>
                     </tr>
                     <tr>
                        <td>topleftrow11949</td>
                        <td>topcenter112</td>
                        <td>topcenter1135.1</td>
                        <td>topcenter1135.1</td>
                        <td>topcenter11+</td>
                        <td>topcenter118,573</td>
                     </tr>
                     <tr>
                        <td>topleftrow11969</td>
                        <td>topcenter111</td>
                        <td>topcenter1129.4</td>
                        <td>topcenter1129.3</td>
                        <td>topcenter11+</td>
                        <td>topcenter11728,800</td>
                     </tr>
                     <tr>
                        <td>topleftrow11037</td>
                        <td>topcenter111</td>
                        <td>topcenter1132.1</td>
                        <td>topcenter1132.5</td>
                        <td>topcenter11+</td>
                        <td>topcenter1162,816</td>
                     </tr>
                     <tr>
                        <td>topleftrow11830a</td>
                        <td>topcenter112</td>
                        <td>topcenter1130.7</td>
                        <td>topcenter1130.0</td>
                        <td>topcenter11+</td>
                        <td>topcenter11426,325</td>
                     </tr>
                     <tr>
                        <td>topleftrow11847a</td>
                        <td>topcenter115</td>
                        <td>topcenter1134.8</td>
                        <td>topcenter1134.7</td>
                        <td>topcenter11+</td>
                        <td>topcenter1111,647</td>
                     </tr>
                     <tr>
                        <td>topleftrow11950a</td>
                        <td>topcenter110</td>
                        <td>topcenter1132.2</td>
                        <td>topcenter1132.7</td>
                        <td>topcenter11+</td>
                        <td>topcenter1153,894</td>
                     </tr>
                     <tr>
                        <td>topleftrow11943</td>
                        <td>topcenter113</td>
                        <td>topcenter1137.6</td>
                        <td>topcenter1135.6</td>
                        <td>topcenter11+</td>
                        <td>topcenter115,845</td>
                     </tr>
                     <tr>
                        <td>topleftrow11952</td>
                        <td>topcenter111</td>
                        <td>topcenter1129.3</td>
                        <td>topcenter1129.5</td>
                        <td>topcenter11+</td>
                        <td>topcenter11625,280</td>
                     </tr>
                     <tr>
                        <td>topleftrow11958</td>
                        <td>topcenter1111</td>
                        <td>topcenter1129.9</td>
                        <td>topcenter1130.3</td>
                        <td>topcenter11+</td>
                        <td>topcenter11338,797</td>
                     </tr>
                     <tr>
                        <td>topleftrow11970</td>
                        <td>topcenter111</td>
                        <td>topcenter1135.5</td>
                        <td>topcenter1134.8</td>
                        <td>topcenter11+</td>
                        <td>topcenter1110,788</td>
                     </tr>
                     <tr>
                        <td>topleftrow1142</td>
                        <td>topcenter110</td>
                        <td>topcenter1132.9</td>
                        <td>topcenter1133.6</td>
                        <td>topcenter11+</td>
                        <td>topcenter1127,048</td>
                     </tr>
                     <tr>
                        <td>topleftrow11941</td>
                        <td>topcenter113</td>
                        <td>topcenter1131.1</td>
                        <td>topcenter1138.0</td>
                        <td>topcenter11+</td>
                        <td>topcenter11930</td>
                     </tr>
                     <tr>
                        <td>topleftrow11964</td>
                        <td>topcenter110</td>
                        <td>topcenter1138.3</td>
                        <td>topcenter1137.6</td>
                        <td>topcenter11+</td>
                        <td>topcenter111,263</td>
                     </tr>
                     <tr>
                        <td>topleftrow11063a</td>
                        <td>topcenter112</td>
                        <td>topcenter1137.5</td>
                        <td>topcenter1138.0</td>
                        <td>topcenter11+</td>
                        <td>topcenter11930</td>
                     </tr>
                  </div>
               </table>
               <div tagxxx="table-wrap-foot">
                  <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">*RT-PCR, reverse transcription­–PCR; ZIKV, Zika virus; Ct, crossing threshold; +,
                     positive.&#x2028;†Ct values with primer set 835/911c/860-FAM or 1086/1162c/1107-FAM. Values
                     &lt;38.5 are positive.&#x2028;‡Estimated by testing quantitated dilutions of ZIKV RNA transcripts
                     and standard curve calculation generated by the iCycler instrument (Bio-Rad, Hercules,
                     CA, USA; see Methods).
                  </p>
               </div>
            </div>
         </div>
         <div id="">
            <h3>Nucleic Acid Sequencing and Phylogenetic Analysis</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">RNA was extracted from patient samples that demonstrated the highest concentration
               of ZIKV RNA determined by the real-time assay, and for which sufficient sample volume
               was available (patients 824, 037, 830a, and 958). Briefly, RNA was extracted from
               150 μL of serum by using the QIAamp Viral RNA Mini Kit (QIAGEN), and RNA was eluted
               with 75 μL of RNase-free water. A series of RT-PCRs was performed with each RNA preparation
               by using primer pairs designed to generate overlapping DNA fragments that spanned
               the entire polyprotein coding region of the virus. Primers were designed by using
               the ZIKV MR 766 prototype virus coding region sequence (GenBank accession no. AY632535)
               and the PrimerSelect software module of the LaserGene package (DNASTAR Inc., Madison,
               WI, USA). Several primers initially failed to amplify because of sequence mismatches
               between ZIKV MR 766 and ZIKV Yap 2007. Therefore, primers were redesigned by using
               newly generated DNA sequence data, and a “genome walking” approach was used to derive
               complete coding region sequence data. The complete list of amplification and sequencing
               primers is available upon request.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">All RT-PCRs were performed with 10 μL of RNA by using the OneStep RT-PCR Kit (QIAGEN)
               following the manufacturer’s protocol. DNAs were analyzed by 2% agarose gel electrophoresis,
               and bands of the predicted size were excised from the gel and purified by using the
               QIAquick Gel Extraction Kit (QIAGEN). Purified DNAs were subjected to nucleic acid
               sequence analysis with sequencing primers spaced ≈500 bases apart on both strands
               of the DNA fragments by using the ABI BigDye Terminator V3.1 Ready Reaction Cycle
               Sequencing Mixture (Applied Biosystems). Nucleotide sequence was determined by capillary
               electrophoresis by using the ABI 3130 genetic analyzer (Applied Biosystems) following
               the manufacturer’s protcol. Raw sequence data were aligned and edited by using the
               SeqMan module of LaserGene (DNASTAR Inc.). Because of insufficient sample volume,
               no patient RNA was sufficient to generate DNA that included the entire coding region.
               Therefore, DNA data obtained from 4 patients was combined to generate a consensus
               sequence heretofore designated the ZIKV 2007 epidemic consensus (EC) sequence (GenBank
               accession no. EU545988).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The complete coding region of ZIKV 2007 EC or the nonstructural protein 5 (NS5) gene
               subregion was aligned with all available flavivirus sequences in GenBank by using
               the Clustal W algorithm within the MEGA version 4 software package (
               <div tagxxx="ext-link">www.megasoftware.net</div>). Phylogenetic trees were constructed by using either the complete coding region
               or the NS5 region because a large number of NS5 sequences were available in GenBank
               and trees for the NS5 region have been constructed (
               <span ref-type="bibr" rid="R16">
                  <i>16</i>
               </span>). Additional ZIKV strains from the CDC/World Health Organization reference collection
               (strains 41662, 41524, and 41525) isolated from 
               <i>Aedes</i> spp. mosquitoes collected in Senegal in 1984 were also amplified by RT-PCR in the
               NS5 region and subjected to nucleic acid sequencing as described above and included
               in the NS5 region analysis. Trees were constructed from coding region data or from
               NS5 data by MEGA 4 from aligned nucleotide sequences. We used maximum parsimony, neighbor-joining,
               or minimum evolution algorithms with 2,000 replicates for bootstrap support of tree
               groupings. All trees generated nearly identical topology; only the neighbor-joining
               NS5 tree is shown (
               <span ref-type="fig" rid="F1">Figure 1</span>).
            </p>
            <div tagxxx="fig">
               <div tagxxx="label">Figure 1</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Phylogenetic tree constructed from nucleic acid data from flavivirus nonstructural
                     protein 5 region by the neighbor-joining algorithm in MEGA (
                     <div tagxxx="ext-link">www.megasoftware.net</div>). Numbers to the left of the nodes are bootstrap percentages (2,000 replications)
                     for clades. Clade numbers correspond to clades identified by Kuno et al. (
                     <span ref-type="bibr" rid="R16">
                        <i>16</i>
                     </span>). Enc, encephalitis; ME, meningoencephalitis.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
         </div>
      </div>
      <div id="">
         <h2>Results</h2>
         <div id="">
            <h3>Serologic Analysis</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <span ref-type="table" rid="T1">Tables 1</span> and 
               <span ref-type="table" rid="T2">2</span> show results of analysis for IgG and IgM and PRNTs of all acute- and convalescent-phase
               paired specimens obtained during the epidemic. Specimens were divided into primary
               and secondary infections on the basis of antibody testing results of acute-phase specimens.
               IgM antibody response in primary flavivirus/ZIKV–infected patients was specific for
               ZIKV. However, all of these patients showed some limited degree of cross-reactivity
               with heterologous flaviviruses. Patient 830a showed IgM-positive results with DENV
               and Japanese encephalitis virus, whereas all patients showed equivocal results (P/N
               2–3) with several of the flaviviruses tested, suggesting low levels of cross-reactivity.
               PRNT
               <div tagxxx="sub">90</div> results also showed that the neutralizing antibody response among primary flavivirus/ZIKV–infected
               patients was highly specific. Most convalescent-phase PRNT titers for heterologous
               flaviviruses were negative and rarely exceeded 10 (20 in 1 instance; patient 849b).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Most patient specimens from the Yap epidemic tested were secondary flavirius infections
               as determined by criteria described for antibody to flavivirus in acute-phase specimens.
               A subset of these patients for whom acute- and convalescent-phase specimens were available
               was tested for reactivity against heterologous flaviviruses; results are shown in
               
               <span ref-type="table" rid="T1">Tables 1</span> and 
               <span ref-type="table" rid="T2">2</span>. In contrast to primary flavivirus/ZIKV–infected patients, secondary flavivirus–infected
               patients showed a high degree of serologic cross-reactivity with other flaviviruses.
               Six of 7 patients were positive for IgM against 
               <u>&gt;</u>1 of the heterologous flaviviruses tested, and all demonstrated low levels of cross-reactive
               IgM as shown by a P/N value in the equivocal range. PRNT
               <div tagxxx="sub">90</div> results showed that among secondary flavivirus/ZIKV–probable patients, the neutralizing
               antibody response was higher to ZIKV and more cross-reactive, a finding commonly observed
               among secondary flavivirus infections. A 
               <u>&gt;</u>4-fold PRNT
               <div tagxxx="sub">90</div> titer between ZIKV and heterologous flaviviruses was observed in only 3 of the 7
               patients. In all other cases, the PRNT difference between ZIKV and other flaviviruses
               tested was 
               <u>&lt;</u>2-fold; in 2 patients (817b and 844b) the PRNT titer was higher for 1 of the heterologous
               flaviviruses. The PRNT result for the acute-phase specimen from patient 847 suggests
               previous vaccination with YFV. The convalescent-phase specimen from patient 847 showed
               a high titer to YFV, a demonstration of the previously described “original antigenic
               sin” phenomenon observed among flaviviruses (
               <span ref-type="bibr" rid="R17">
                  <i>17</i>
               </span>).
            </p>
         </div>
         <div id="">
            <h3>Real-Time RT-PCR</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A real-time RT-PCR was developed by using newly derived sequence data obtained from
               several ZIKV-infected patients. All acute-phase specimens obtained during the Yap
               epidemic (n = 157) were tested in this assay with 2 unique primer/probe sets. Seventeen
               samples were positive, 10 were equivocal, and 130 were negative (data not shown).
               The equivocal designation indicates that a particular sample was positive by only
               1 of the 2 primer sets or showed crossing thresholds &gt;38.5, which suggests either
               a false-positive result or a sample with low levels of ZIKV RNA below the defined
               cut-off of the assay. 
               <span ref-type="table" rid="T4">Table 4</span> shows estimated viral concentrations of the 17 ZIKV-positive specimens. The viral
               RNA concentrations were ≈900–729,000 copies/mL. Most (15 of 17) of the ZIKV-positive
               samples were from specimens collected 
               <u>&lt;</u>3 days after onset; however, 1 specimen (patient 958) collected on day 11 after onset
               was positive with an estimated titer of ≈339,000 copies/mL.
            </p>
         </div>
         <div id="">
            <h3>Nucleic Acid Sequence and Phylogenetic Analysis</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Several RT-PCR–positive serum specimens were selected, and RNA was amplified by RT-PCR
               to generate DNA sequence data for the complete coding region. Because of limited specimen
               volume, the complete coding region genome sequence was only obtainable by combining
               sequence data from DNA fragments generated from 4 patients. Thus, the designation
               EC sequence is used to indicate that the sequence was derived from multiple patients
               during the epidemic. The exact contribution of sequence data from each patient is
               available upon request. However, the following points should be noted. Approximately
               96% of the complete coding region was obtained from 3 patients; sequence data from
               the fourth patient was used primarily to fill in short gaps in the data. Second, ≈50%
               of the coding region data was derived from a complete overlap of data from 
               <u>&gt;</u>2 patients; in these overlap regions the sequence identity between different patients
               was ≈100%. Only 2-nt differences between patients were noted within the overlapping
               regions, strongly suggesting that 1 ZIKV strain circulated during the epidemic.
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Percentage identity over the entire coding region of ZIKV 2007 EC sequence, when compared
               with the prototype ZIKV (MR 766, isolated in 1947), was 88.9% and 96.5% at the nucleotide
               and amino acid levels, respectively. Phylogenetic trees constructed from the complete
               coding region of all available flaviviruses generated by a variety of methods (neighbor-joining,
               maximum-parsimony, or minimum-evolution) showed the same overall topology, with the
               ZIKV prototype and 2007 EC virus placed in a unique clade (clade 10) within the mosquito-borne
               flavivirus cluster previously described by Kuno et al. (
               <span ref-type="bibr" rid="R16">
                  <i>16</i>
               </span>). Alignment with phylogenetic tree construction by neighbor-joining, maximum-parsimony,
               or minimum-evolution algorithms was also performed for the NS5 region of all available
               flaviviruses because extensive sequencing and phylogenetic analysis have been conducted
               for this region (
               <span ref-type="bibr" rid="R16">
                  <i>16</i>
               </span>).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Three additional ZIKV strains isolated from Senegal in 1984 and sequenced in this
               study were also included in a tree. This NS5 tree demonstrated similar topology to
               the complete coding region tree, with all ZIKVs placed within a unique clade (clade
               10) along with SPOV. 
               <span ref-type="fig" rid="F1">Figure 1</span> shows the NS5 tree with only mosquito-borne flaviviruses (cluster) displayed. This
               NS5 tree also shows that within the Zika/Spondweni clade there appear to be 3 branches
               among ZIKVs: Nigerian ZIKVs, prototype MR766, and 2007 Yap virus. Percentage identity
               among these ZIKVs confirms the tree topology, in which ZIKV 2007 EC is most distally
               related to East and West African ZIKV strains (data not shown).
            </p>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The predicted amino acid sequence of ZIKV 2007 EC contains the Asn-X-Ser/Thr glycosylation
               motif at position 154 in the envelope glycoprotein, found in many flaviviruses, yet
               absent by deletion in the prototype ZIKV MR 766. This region of the prototype virus,
               along with 3 ZIKVs isolated from Senegal in 1984, was sequenced (
               <span ref-type="fig" rid="F2">Figure 2</span>). Included in this alignment is a ZIKV isolate from GenBank (accession no. AF372422).
               Sequencing confirmed that prototype ZIKV MR766 has a 4-aa (12-nt) deletion when compared
               with ZIKV 2007 EC virus and ZIKVs from Senegal.
            </p>
            <div tagxxx="fig">
               <div tagxxx="label">Figure 2</div>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Alignment of nucleotide and amino acid sequences adjacent to the envelope (ENV)–154
                     glycosylation site of Zika virus strains. Dashes indicate deletions. EC, epidemic
                     consensus.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
         </div>
      </div>
      <div id="">
         <h2>Discussion</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Historically, ZIKV has rarely been associated with human disease, with only 1 small
            cluster of human cases in Indonesia reported (
            <span ref-type="bibr" rid="R9">
               <i>9</i>
            </span>). We report a widespread epidemic of human disease associated with ZIKV in Yap State
            in 2007. ZIKV epidemics may have occurred but been misdiagnosed as dengue because
            of similar clinical symptoms and serologic cross-reactivity with DENVs. Our serologic
            data indicate that ZIKV-infected patients can be positive in an IgM assay for DENVs,
            particularly if ZIKV is a secondary flavivirus infection. If ZIKV is the first flavivirus
            encountered, our data indicate that cross-reactivity is minimal. However, when ZIKV
            infection occurs after a flavivirus infection, our data indicate that the extent of
            cross-reactivity in the IgM assay is greater. Therefore, if ZIKV infections occur
            in a population with DENV (or other flavivirus) background immunity, our data suggest
            that extensive cross-reactivity in the dengue IgM assay will occur, which could lead
            to the erroneous conclusion that dengue caused the epidemic. Whether this cross-reactivity
            has occurred is open to speculation. However, reexamination of specimens from dengue
            epidemics may provide an answer. In addition, use of virus isolation or RT-PCR for
            laboratory diagnosis of dengue infections would also prevent this misinterpretation.
            Therefore, use of virus detection assays in dengue epidemics should be a component
            of laboratory testing algorithms.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Levels of viremia among ZIKV-infected patients were relatively low. Unfortunately,
            measurement of concentration of infectious ZIKV was not possible because a virus isolate
            was not obtained from any patient during the epidemic. Absence of a ZIKV 2007 isolate
            also precluded use of a ZIKV 2007 isolate to generate a standard curve in the RT-PCR,
            which in turn could have estimated the concentration of infectious virus within patients.
            An estimation of the number of genome copies circulating in ZIKV-infected patients
            was calculated by using an RNA transcript and provides some indication of infectious
            virus concentration in ZIKV-infected patients. If one assumes a ratio range of 200–500
            genome copies per infectious virus particle, a range reported for several flaviviruses,
            then the copies/milliliter values in 
            <span ref-type="table" rid="T4">Table 4</span> would be in the range of ≈2–3,500 infectious virus particles/mL, with only 4 specimens
            in which ZIKV exceeded 1,000 infectious units/mL (
            <span ref-type="bibr" rid="R18">
               <i>18</i>
            </span>,
            <span ref-type="bibr" rid="R19">
               <i>19</i>
            </span>). These findings may partially explain why ZIKV was not isolated, especially if one
            considers that shipping samples to our laboratory took ≈1 week, and shipping conditions
            were not conducive to virus isolation. These concentration estimates are also consistent
            with those of a study in which a ZIKV-infected human volunteer showed low viremia;
            virus was isolated only on day 4, and the volunteer was unable to infect 
            <i>Ae</i>. 
            <i>aegypti</i> mosquitoes that fed on the patient during the acute stage of disease (
            <span ref-type="bibr" rid="R10">
               <i>10</i>
            </span>).
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Although generation of a complete coding region nucleic acid sequence by using a combination
            of patient samples from the epidemic is an unconventional approach, it was performed
            out of necessity because of limited volumes of patient samples. However, the extent
            of agreement among overlapping regions confirms that the sequence obtained accurately
            represents the virus associated with the epidemic. Nucleic acid sequence of ZIKV 2007
            showed divergence (11%) from the prototype strain (MR766) isolated in 1947. However,
            the predicted amino acid sequence is fairly conserved (96%), which is likely the result
            of the selective pressure maintained on the virus because replication occurs in vertebrate
            hosts and arthropod vectors.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Phylogenetic trees based on the complete coding region or the NS5 region confirm results
            of a study in which ZIKV was classified in a unique clade among the mosquito-borne
            flaviviruses and most closely related to SPOV (
            <span ref-type="bibr" rid="R16">
               <i>16</i>
            </span>). The NS5 mosquito-borne flavivirus tree (
            <span ref-type="fig" rid="F1">Figure 1</span>), which includes additional ZIKV isolates, confirms these relationships and suggests
            that there are 3 subclades among ZIKV isolates that reflect geographic origin. Senegal
            ZIKVs and prototype virus from Uganda may represent West and East African lineages,
            respectively. The 2007 ZIKV is distantly related to these 2 African subclades and
            may represent divergence from a common ancestor with spread throughout Southeast Asia
            and the Pacific. Human ZIKV cases were detected in peninsular Malaysia in 1980, which
            confirms that ZIKV was active in this region before 2007 (
            <span ref-type="bibr" rid="R9">
               <i>9</i>
            </span>). Additional sequence analysis of other temporally and geographically distinct ZIKV
            strains is needed to further elucidate relationships among these viruses.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Of particular interest is an additional 12 nt in the envelope gene (corresponding
            to 4 aa) in our ZIKV isolate that were not present in the ZIKV prototype virus (
            <span ref-type="fig" rid="F2">Figure 2</span>). This difference is noteworthy because these 4 aa correspond to the envelope protein
            154 glycosylation motif found in many flaviviruses and associated in some instances
            with virulence. This glycosylation motif is also absent because of a 6-aa deletion
            in the ZIKV isolate obtained from GenBank (accession no. AF372422); however, the geographic
            and temporal origins of this virus were not available. Loss of the envelope protein
            154 glycoslyation site has been observed in some flaviviruses, and in the case of
            Kunjin virus has been shown to occur during passage. However, with Kunjin virus, the
            glycosylation site motif was lost because of a 1-base mutation, rather than a deletion,
            that altered the N-X-S/T sequon (
            <span ref-type="bibr" rid="R20">
               <i>20</i>
            </span>). Loss of this glycosylation site by a 4-aa deletion has also been observed in several
            lineage-2 WNV strains when compared with all other WNV strains (
            <span ref-type="bibr" rid="R21">
               <i>21</i>
            </span>).
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The glycoslyation motif in WNV may be lost during extensive mouse brain passage; however,
            no direct evidence exists to support this hypothesis (
            <span ref-type="bibr" rid="R21">
               <i>21</i>
            </span>). This process may occur in ZIKV; the glycoslyation motif in MR 766 may have been
            present in earlier passages of prototype MR766 and lost during extensive mouse brain
            passage. However, earlier passage strains of MR766 were not available for investigating
            this hypothesis. Alternatively, the presence or absence of this glycosylation motif
            may represent an ancient evolutionary event with subsequent divergence of 2 ZIKV types
            with or without the E-154 glycosylation site amino acids. Sequence data derived from
            3 additional ZIKV isolates from Senegal showed that glycosylation is intact in these
            isolates, which suggests evolutionary divergence. More extensive sequence analysis
            of available ZIKV strains of various temporal, geographic, and passage histories may
            provide some insight into this issue.
         </p>
      </div>
      <div tagxxx="back">
         <div tagxxx="fn-group">
            <div tagxxx="fn">
               <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <i>Suggested citation for this article</i>: Lanciotti RS, Kosoy OL, Laven JJ, Velez JO, Lambert AJ, Johnson AJ, et al. Genetic
                  and serologic properties of Zika virus associated with an epidemic, Yap State, Micronesia,
                  2007. Emerg Infect Dis [serial on the Internet]. 2008 Aug [
                  <i>date cited</i>]. Available from 
                  <div tagxxx="ext-link">http://www.cdc.gov/EID/content/14/8/1232.htm</div>
               </p>
            </div>
         </div>
         <div tagxxx="ack">
            <div tagxxx="title">Acknowledgments</div>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We thank Amanda Panella and Roselyn Hochbein for assistance in processing and manipulating
               data for patient samples from the epidemic.
            </p>
         </div>
         <div tagxxx="bio">
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Dr Lanciotti is chief of the Diagnostic and Reference Laboratory within the Arbovirus
               Diseases Branch at CDC. His primary research interests are laboratory diagnosis of
               arbovirus infections and diagnostic test development and support for public health
               laboratories worldwide.
            </p>
         </div>
         <div tag="ref-list">
            <ul>
               <div tagxxx="title">References</div>
               <li tag="ref">
                  <div tagxxx="label">1. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Dick</span>
                        <span tagx="given-names">GW</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Kitchen</span>
                        <span tagx="given-names">SF</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Haddow</span>
                        <span tagx="given-names">AJ</span>
                     </div>
                     <span tagx="article-title">Zika virus isolations and serological specificity.</span>
                     <span tagx="source">Trans R Soc Trop Med Hyg</span>
                     <span tagx="year">1952</span>;
                     <span tagx="volume">46</span>:
                     <span tagx="fpage">509</span>–
                     <span tagx="lpage">20</span>
                     <div tagxxx="pub-id">10.1016/0035-9203(52)90042-4</div>
                     <div tagxxx="pub-id">12995440</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">2. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Dick</span>
                        <span tagx="given-names">GW</span>
                     </div>
                     <span tagx="article-title">Zika virus pathogenicity and physical properties.</span>
                     <span tagx="source">Trans R Soc Trop Med Hyg</span>
                     <span tagx="year">1952</span>;
                     <span tagx="volume">46</span>:
                     <span tagx="fpage">521</span>–
                     <span tagx="lpage">34</span>
                     <div tagxxx="pub-id">10.1016/0035-9203(52)90043-6</div>
                     <div tagxxx="pub-id">12995441</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">3. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Weinbren</span>
                        <span tagx="given-names">MP</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Williams</span>
                        <span tagx="given-names">MC</span>
                     </div>
                     <span tagx="article-title">Zika virus:further isolations in the Zika area and some studies on the strains isolated.</span>
                     <span tagx="source">Trans R Soc Trop Med Hyg</span>
                     <span tagx="year">1958</span>;
                     <span tagx="volume">52</span>:
                     <span tagx="fpage">263</span>–
                     <span tagx="lpage">8</span>
                     <div tagxxx="pub-id">10.1016/0035-9203(58)90085-3</div>
                     <div tagxxx="pub-id">13556872</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">4. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Haddow</span>
                        <span tagx="given-names">AJ</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Williams</span>
                        <span tagx="given-names">MC</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Woodall</span>
                        <span tagx="given-names">JP</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Simpson</span>
                        <span tagx="given-names">DIH</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Goma</span>
                        <span tagx="given-names">LK</span>
                     </div>
                     <span tagx="article-title">Twelve isolations of Zika virus from 
                        <i>Aedes</i> (
                        <i>Stegomyia</i>) 
                        <i>africanus</i> (Theobald) taken in and above a Ugandan forest.
                     </span>
                     <span tagx="source">Bull World Health Organ</span>
                     <span tagx="year">1964</span>;
                     <span tagx="volume">31</span>:
                     <span tagx="fpage">57</span>–
                     <span tagx="lpage">69</span>
                     <div tagxxx="pub-id">14230895</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">5. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Marchette</span>
                        <span tagx="given-names">NJ</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Garcia</span>
                        <span tagx="given-names">R</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Rudnick</span>
                        <span tagx="given-names">A</span>
                     </div>
                     <span tagx="article-title">Isolation of Zika virus from 
                        <i>Aedes aegypti</i> mosquitoes in Malaysia.
                     </span>
                     <span tagx="source">Am J Trop Med Hyg</span>
                     <span tagx="year">1969</span>;
                     <span tagx="volume">18</span>:
                     <span tagx="fpage">411</span>–
                     <span tagx="lpage">5</span>
                     <div tagxxx="pub-id">4976739</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">6. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Simpson</span>
                        <span tagx="given-names">DIH</span>
                     </div>
                     <span tagx="article-title">Zika virus infection in man.</span>
                     <span tagx="source">Trans R Soc Trop Med Hyg</span>
                     <span tagx="year">1964</span>;
                     <span tagx="volume">58</span>:
                     <span tagx="fpage">335</span>–
                     <span tagx="lpage">8</span>
                     <div tagxxx="pub-id">10.1016/0035-9203(64)90201-9</div>
                     <div tagxxx="pub-id">14175744</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">7. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Moore</span>
                        <span tagx="given-names">DL</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Causey</span>
                        <span tagx="given-names">OR</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Carey</span>
                        <span tagx="given-names">DE</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Reddy</span>
                        <span tagx="given-names">S</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Cooke</span>
                        <span tagx="given-names">AR</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Akinkugbe</span>
                        <span tagx="given-names">FM</span>
                     </div>, 
                     <div tagxxx="etal"></div>
                     <span tagx="article-title">Arthropod-borne viral infection of man in Nigeria, 1964–1970.</span>
                     <span tagx="source">Ann Trop Med Parasitol</span>
                     <span tagx="year">1975</span>;
                     <span tagx="volume">69</span>:
                     <span tagx="fpage">49</span>–
                     <span tagx="lpage">64</span>
                     <div tagxxx="pub-id">1124969</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">8. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Fagbami</span>
                        <span tagx="given-names">AH</span>
                     </div>
                     <span tagx="article-title">Zika virus infections in Nigeria: virological and seroepidemiological investigations
                        in Oyo State.
                     </span>
                     <span tagx="source">J Hyg (Lond)</span>
                     <span tagx="year">1979</span>;
                     <span tagx="volume">83</span>:
                     <span tagx="fpage">213</span>–
                     <span tagx="lpage">9</span>
                     <div tagxxx="pub-id">489960</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">9. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Olson</span>
                        <span tagx="given-names">JG</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Ksiazek</span>
                        <span tagx="given-names">TG</span>
                     </div>
                     <span tagx="article-title">Suhandiman, Triwibowo. Zika virus, a cause of fever in Central Java, Indonesia.</span>
                     <span tagx="source">Trans R Soc Trop Med Hyg</span>
                     <span tagx="year">1981</span>;
                     <span tagx="volume">75</span>:
                     <span tagx="fpage">389</span>–
                     <span tagx="lpage">93</span>
                     <div tagxxx="pub-id">10.1016/0035-9203(81)90100-0</div>
                     <div tagxxx="pub-id">6275577</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">10. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Bearcroft</span>
                        <span tagx="given-names">WG</span>
                     </div>
                     <span tagx="article-title">Zika virus infection experimentally induced in a human volunteer.</span>
                     <span tagx="source">Trans R Soc Trop Med Hyg</span>
                     <span tagx="year">1956</span>;
                     <span tagx="volume">50</span>:
                     <span tagx="fpage">442</span>–
                     <span tagx="lpage">8</span>
                     <div tagxxx="pub-id">10.1016/0035-9203(56)90090-6</div>
                     <div tagxxx="pub-id">13380987</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">11. </div>
                  <div tagxxx="citation">Zika virus outbreak—Micronesia (Yap). Suspected. ProMed June 27, 
                     <span tagx="year">2007</span> [cited 2008 May 8]. Available from 
                     <div tagxxx="ext-link">http://www.promedmail.org</div>, archive no.: 20070627.2065.
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">12. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Johnson</span>
                        <span tagx="given-names">AJ</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Martin</span>
                        <span tagx="given-names">DA</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Karabatsos</span>
                        <span tagx="given-names">N</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Roehrig</span>
                        <span tagx="given-names">JT</span>
                     </div>
                     <span tagx="article-title">Detection of anti-arboviral immunoglobulin G by using a monoclonal antibody-based
                        capture ELISA.
                     </span>
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="year">2000</span>;
                     <span tagx="volume">38</span>:
                     <span tagx="fpage">1827</span>–
                     <span tagx="lpage">31</span>
                     <div tagxxx="pub-id">10790108</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">13. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Martin</span>
                        <span tagx="given-names">DA</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Muth</span>
                        <span tagx="given-names">DA</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Brown</span>
                        <span tagx="given-names">T</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Johnson</span>
                        <span tagx="given-names">AJ</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Karabatsos</span>
                        <span tagx="given-names">N</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Roehrig</span>
                        <span tagx="given-names">JT</span>
                     </div>
                     <span tagx="article-title">Standardization of immunoglobulin M capture enzyme-linked immunosorbent assays for
                        routine diagnosis of arboviral infections.
                     </span>
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="year">2000</span>;
                     <span tagx="volume">38</span>:
                     <span tagx="fpage">1823</span>–
                     <span tagx="lpage">6</span>
                     <div tagxxx="pub-id">10790107</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">14. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Johnson</span>
                        <span tagx="given-names">AJ</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Noga</span>
                        <span tagx="given-names">AJ</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Kosoy</span>
                        <span tagx="given-names">O</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Lanciotti</span>
                        <span tagx="given-names">RS</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Johnson</span>
                        <span tagx="given-names">AA</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Biggerstaff</span>
                        <span tagx="given-names">BJ</span>
                     </div>
                     <span tagx="article-title">Duplex microsphere-based immunoassay for detection of anti-West Nile virus and anti-St.
                        Louis encephalitis virus immunoglobulin M antibodies.
                     </span>
                     <span tagx="source">Clin Diagn Lab Immunol</span>
                     <span tagx="year">2005</span>;
                     <span tagx="volume">12</span>:
                     <span tagx="fpage">566</span>–
                     <span tagx="lpage">74</span>
                     <div tagxxx="pub-id">10.1128/CDLI.12.5.566-574.2005</div>
                     <div tagxxx="pub-id">15879016</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">15. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Calisher</span>
                        <span tagx="given-names">CH</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Karabatsos</span>
                        <span tagx="given-names">N</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Dalrymple</span>
                        <span tagx="given-names">JM</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Shope</span>
                        <span tagx="given-names">RE</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Porterfield</span>
                        <span tagx="given-names">JS</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Westaway</span>
                        <span tagx="given-names">EG</span>
                     </div>, 
                     <div tagxxx="etal"></div>
                     <span tagx="article-title">Antigenic relationships between flaviviruses as determined by cross-neutralization
                        tests with polyclonal antisera.
                     </span>
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="year">1989</span>;
                     <span tagx="volume">70</span>:
                     <span tagx="fpage">37</span>–
                     <span tagx="lpage">43</span>
                     <div tagxxx="pub-id">10.1099/0022-1317-70-1-37</div>
                     <div tagxxx="pub-id">2543738</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">16. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Kuno</span>
                        <span tagx="given-names">G</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Chang</span>
                        <span tagx="given-names">GJ</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Tsuchiya</span>
                        <span tagx="given-names">R</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Karabatsos</span>
                        <span tagx="given-names">N</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Cropp</span>
                        <span tagx="given-names">BC</span>
                     </div>
                     <span tagx="article-title">Phylogeny of the genus 
                        <i>Flavivirus.</i>
                     </span>
                     <span tagx="source">J Virol</span>
                     <span tagx="year">1998</span>;
                     <span tagx="volume">72</span>:
                     <span tagx="fpage">73</span>–
                     <span tagx="lpage">83</span>
                     <div tagxxx="pub-id">9420202</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">17. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Halstead</span>
                        <span tagx="given-names">SB</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Rojanasuphot</span>
                        <span tagx="given-names">S</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Sangkawibha</span>
                        <span tagx="given-names">N</span>
                     </div>
                     <span tagx="article-title">Original antigenic sin in dengue.</span>
                     <span tagx="source">Am J Trop Med Hyg</span>
                     <span tagx="year">1983</span>;
                     <span tagx="volume">32</span>:
                     <span tagx="fpage">154</span>–
                     <span tagx="lpage">6</span>
                     <div tagxxx="pub-id">6824120</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">18. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Wang</span>
                        <span tagx="given-names">WK</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Lee</span>
                        <span tagx="given-names">CN</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Kao</span>
                        <span tagx="given-names">CL</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Lin</span>
                        <span tagx="given-names">YL</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">King</span>
                        <span tagx="given-names">CC</span>
                     </div>
                     <span tagx="article-title">Quantitative competitive reverse transcription-PCR for quantification of dengue virus
                        RNA.
                     </span>
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="year">2000</span>;
                     <span tagx="volume">38</span>:
                     <span tagx="fpage">3306</span>–
                     <span tagx="lpage">10</span>
                     <div tagxxx="pub-id">10970375</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">19. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Shi</span>
                        <span tagx="given-names">PY</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Kauffman</span>
                        <span tagx="given-names">EB</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Ren</span>
                        <span tagx="given-names">P</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Felton</span>
                        <span tagx="given-names">A</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Tai</span>
                        <span tagx="given-names">JH</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Dupuis</span>
                        <span tagx="given-names">AP</span>
                     </div>, 
                     <div tagxxx="etal"></div>
                     <span tagx="article-title">High-throughput detection of West Nile virus RNA.</span>
                     <span tagx="source">J Clin Microbiol</span>
                     <span tagx="year">2001</span>;
                     <span tagx="volume">39</span>:
                     <span tagx="fpage">1264</span>–
                     <span tagx="lpage">71</span>
                     <div tagxxx="pub-id">10.1128/JCM.39.4.1264-1271.2001</div>
                     <div tagxxx="pub-id">11283039</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">20. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Adams</span>
                        <span tagx="given-names">SC</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Broom</span>
                        <span tagx="given-names">AK</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Sammels</span>
                        <span tagx="given-names">LM</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Hartnett</span>
                        <span tagx="given-names">AC</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Howard</span>
                        <span tagx="given-names">MJ</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Coelen</span>
                        <span tagx="given-names">RJ</span>
                     </div>, 
                     <div tagxxx="etal"></div>
                     <span tagx="article-title">Glycosylation and antigenic variation among Kunjin virus isolates.</span>
                     <span tagx="source">Virology</span>
                     <span tagx="year">1995</span>;
                     <span tagx="volume">206</span>:
                     <span tagx="fpage">49</span>–
                     <span tagx="lpage">56</span>
                     <div tagxxx="pub-id">10.1016/S0042-6822(95)80018-2</div>
                     <div tagxxx="pub-id">7530394</div>
                  </div>
               </li>
               <li tag="ref">
                  <div tagxxx="label">21. </div>
                  <div tagxxx="citation">
                     <div tagxxx="string-name">
                        <span tagx="surname">Berthet</span>
                        <span tagx="given-names">FX</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Zeller</span>
                        <span tagx="given-names">HG</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Drouet</span>
                        <span tagx="given-names">MT</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Rauzier</span>
                        <span tagx="given-names">J</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Digoutte</span>
                        <span tagx="given-names">JP</span>
                     </div>, 
                     <div tagxxx="string-name">
                        <span tagx="surname">Deubel</span>
                        <span tagx="given-names">V</span>
                     </div>
                     <span tagx="article-title">Extensive nucleotide changes and deletions within the envelope glycoprotein gene of
                        Euro-African West Nile viruses.
                     </span>
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="year">1997</span>;
                     <span tagx="volume">78</span>:
                     <span tagx="fpage">2293</span>–
                     <span tagx="lpage">7</span>
                     <div tagxxx="pub-id">9292017</div>
                  </div>
               </li>
            </ul>
         </div>
      </div>
   </div>
</html>