<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <div xmlns="" tagx="article">
      <head>
         <style type="text/css"> table, tr, td {border : thin;}</style>
         <span class="citation_publisher">Public Library of Science</span>
         <h2 class="citation-title">Molecular Characterisation of Chikungunya Virus Infections in Trinidad and Comparison
            of Clinical and Laboratory Features with Dengue and Other Acute Febrile Cases
         </h2>
         <span class="doi">10.1371/journal.pntd.0004199</span>
         <div class="contrib-group">contrib: 
            	  CONT
            	  
            <span class="citation_author">Nikita Sahadeo</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Hamish Mohammed</span>
            <span class="citation_author_institution_ref">2</span>
            <span class="citation_author">Orchid M. Allicock</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Albert J. Auguste</span>
            <span class="citation_author_institution_ref">3</span>
            <span class="citation_author">Steven G. Widen</span>
            <span class="citation_author_institution_ref">4</span>
            <span class="citation_author">Kimberly Badal</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Krishna Pulchan</span>
            <span class="citation_author_institution_ref">5</span>
            <span class="citation_author">Jerome E. Foster</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Scott C. Weaver</span>
            <span class="citation_author_institution_ref">3</span>
            <span class="citation_author">Christine V. F. Carrington</span>
            <span class="citation_author_institution_ref">1</span>
            	  CONT
            	  
            <div tagxxx="contrib">
               <span class="name'">
                  <span tagx="surname">Lopes da Fonseca</span>
                  <span tagx="given-names">Benedito Antonio</span>,
               </span>
               <div tagxxx="role">Editor</div>
               <span ref-type="aff" rid="edit1"></span>
            </div>
         </div>
         <span class="email">[christine.carrington@sta.uwi.edu]</span>
         <span class="conflict">[The authors have declared that no competing interests exist.]</span>
         <span class="contribs">[Conceived and designed the experiments: CVFC NS. Performed the experiments: NS OMA
            KB AJA. Analyzed the data: NS HM OMA SGW CVFC JEF. Wrote the paper: NS HM OMA AJA
            SGW KB KP JEF SCW CVFC. Facilitated hospital surveillance: KP.]
         </span>
         <span class="citation_pubdate">[2015-11-18]</span>
         <span class="citation_copyright">[Copyright 2015: Sahadeo et al]</span>
         <span class="citation_licence">[Licence This is an open access article distributed under the terms of the Creative
            Commons Attribution License, which permits unrestricted use, distribution, and reproduction
            in any medium, provided the original author and source are credited]
         </span>
         <span class="citation_funding">[Funding for this study was provided by grants from The University of the West Indies
            (UWI) Research Development Impact Fund (http://sta.uwi.edu/rdifund/) to CVFC, the
            International Society for Infectious Diseases to NS and NIH grant AI093491 to SCW.
            AJA is supported by the James W. McLaughlin Endowment fund. The funders had no role
            in study design, data collection and analysis, decision to publish, or preparation
            of the manuscript.]
         </span>
         <span class="citation_pagecount">[]</span>
         <span class="citation_data">[Data AvailabilityThe 8 newly generated CHIKV whole genome sequences are available
            from Genbank, with accession numbers: KR046227, KR046228, KR046229, KR046230, KR046231,
            KR046232, KR046233 and KR046234.]
         </span>
         <div tagxxx="notes">
            <div tagxxx="title">Data Availability</div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The 8 newly generated CHIKV whole genome sequences are available from Genbank, with
               accession numbers: KR046227, KR046228, KR046229, KR046230, KR046231, KR046232, KR046233
               and KR046234.
            </p>
         </div>
      </head>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Local transmission of Chikungunya virus (CHIKV) was first documented in Trinidad and
            Tobago (T&amp;T) in July 2014 preceding a large epidemic. At initial presentation, it
            is difficult to distinguish chikungunya fever (CHIKF) from other acute undifferentiated
            febrile illnesses (AUFIs), including life-threatening dengue disease. We characterised
            and compared dengue virus (DENV) and CHIKV infections in 158 patients presenting with
            suspected dengue fever (DF) and CHIKF at a major hospital in T&amp;T, and performed phylogenetic
            analyses on CHIKV genomic sequences recovered from 8 individuals. The characteristics
            of patients with and without PCR-confirmed CHIKV were compared using Pearson’s χ
            <sup>
               <i>2</i>
            </sup> and student’s t-tests, and adjusted odds ratios (aORs) and 95% confidence intervals
            (CIs) were determined using logistic regression. We then compared signs and symptoms
            of people with RT-qPCR-confirmed CHIKV and DENV infections using the Mann-Whitney
            U, Pearson’s χ
            <sup>
               <i>2</i>
            </sup> and Fisher’s exact tests. Among the 158 persons there were 8 (6%) RT-qPCR-confirmed
            DENV and 30 (22%) RT-qPCR-confirmed CHIKV infections. Phylogenetic analyses showed
            that the CHIKV strains belonged to the Asian genotype and were most closely related
            to a British Virgin Islands strain isolated at the beginning of the 2013/14 outbreak
            in the Americas. Compared to persons who were RT-qPCR-negative for CHIKV, RT-qPCR-positive
            individuals were significantly more likely to have joint pain (aOR: 4.52 [95% CI:
            1.28–16.00]), less likely to be interviewed at a later stage of illness (days post
            onset of fever—aOR: 0.56 [0.40–0.78]) and had a lower white blood cell count (aOR:
            0.83 [0.71–0.96]). Among the 38 patients with RT-qPCR-confirmed CHIKV or DENV, there
            were no significant differences in symptomatic presentation. However when individuals
            with serological evidence of recent DENV or CHIKV infection were included in the analyses,
            there were key differences in clinical presentation between CHIKF and other AUFIs
            including DF, which can be used to triage patients for appropriate care in the clinical
            setting.
         </p>
      </div>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <div tagxxx="title">Author Summary</div>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Chikungunya virus (CHIKV) recently emerged in the Americas and caused a major epidemic
            of chikungunya fever (CHIKF). While not usually life threatening, CHIKF is a debilitating
            and often chronic illness resulting in major morbidity and economic losses. It is
            difficult to distinguish CHIKF from other viral illnesses that cause acute fevers
            including dengue fever (DF), an important consideration since DF can be life-threatening
            and early identification and treatment of cases is key to reducing mortality. In this
            study we investigated individuals presenting to a major hospital in Trinidad and Tobago
            (T&amp;T) with suspected DF or CHIKF, and identified signs and symptoms that distinguish
            these two illnesses. We also recovered complete genome sequences for CHIKV and show
            that the etiologic strain in T&amp;T is a closely related descendent of the strain first
            isolated from the British Virgin Islands at the beginning of the outbreak in the Americas.
         </p>
      </div>
      <div id="sec001">
         <h2>Introduction</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Chikungunya virus (CHIKV; Family 
            <i>Togaviridae</i>, genus 
            <i>Alphavirus</i>) is the aetiological agent of chikungunya fever (CHIKF), which presents as an acute
            onset high fever with headache, back pain, muscle and joint pain. The joint pain can
            vary in intensity but is often very intense, predominates at the ankles, wrists and
            phalanges and is coupled with swelling. CHIKF is usually a self-limiting disease,
            and serious outcomes (e.g. neurological complications) and fatalities appear to be
            rare [
            <span ref-type="bibr" rid="pntd.0004199.ref001">
               <sup>
                  <a href="#pntd.0004199.ref001">1</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pntd.0004199.ref002">
               <sup>
                  <a href="#pntd.0004199.ref002">2</a>
               </sup>
            </span>]. However up to 60% of infections are followed by chronic arthritic conditions, with
            recurrent debilitating joint pain several years post-infection [
            <span ref-type="bibr" rid="pntd.0004199.ref003">
               <sup>
                  <a href="#pntd.0004199.ref003">3</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pntd.0004199.ref004">
               <sup>
                  <a href="#pntd.0004199.ref004">4</a>
               </sup>
            </span>]
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">CHIKV exists as a single serotype thought to confer life-long immunity in recovered
            individuals. There is, however, sufficient variation to discern three genotypes, namely
            the enzootic West African (WAf), and East /Central/South African (ECSA), and the epidemic
            Asian genotypes. The ECSA recently gave rise to the Indian Ocean Lineage (IOL) responsible
            for epidemics in the Indian Ocean islands, mainland India and Europe beginning in
            2004 [
            <span ref-type="bibr" rid="pntd.0004199.ref005">
               <sup>
                  <a href="#pntd.0004199.ref005">5</a>
               </sup>
            </span>]. The expansion of this lineage has been attributed to adaptive mutations in the
            E1 and E2 envelope glycoproteins that provide a fitness advantage in 
            <i>Aedes albopictus</i> without reducing fitness in 
            <i>A</i>. 
            <i>aegypti</i> [
            <span ref-type="bibr" rid="pntd.0004199.ref006">
               <sup>
                  <a href="#pntd.0004199.ref006">6</a>
               </sup>
            </span>] and permit rapid lineage diversification [
            <span ref-type="bibr" rid="pntd.0004199.ref007">
               <sup>
                  <a href="#pntd.0004199.ref007">7</a>
               </sup>
            </span>] [
            <span ref-type="bibr" rid="pntd.0004199.ref008">
               <sup>
                  <a href="#pntd.0004199.ref008">8</a>
               </sup>
            </span>].
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In light of the dramatic re-emergence of CHIKV in Asia, the intensity of global travel
            and the widespread prevalence of 
            <i>A</i>. 
            <i>aegypti</i> and 
            <i>A</i>. 
            <i>albopictus</i>, the emergence of CHIKV in the Americas was long anticipated [
            <span ref-type="bibr" rid="pntd.0004199.ref009">
               <sup>
                  <a href="#pntd.0004199.ref009">9</a>
               </sup>
            </span>]. As expected, numerous imported (travel-related) cases were subsequently documented
            in the Americas, none of which resulted in local transmission until December 2013
            when autochthonous transmission was documented in the Caribbean island of St. Martin
            [
            <span ref-type="bibr" rid="pntd.0004199.ref010">
               <sup>
                  <a href="#pntd.0004199.ref010">10</a>
               </sup>
            </span>]. The etiologic virus, which belongs to the Asian genotype rather than the IOL [
            <span ref-type="bibr" rid="pntd.0004199.ref011">
               <sup>
                  <a href="#pntd.0004199.ref011">11</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pntd.0004199.ref012">
               <sup>
                  <a href="#pntd.0004199.ref012">12</a>
               </sup>
            </span>], spread rapidly first amongst the islands of the Caribbean archipelago then on to
            the mainland Americas [
            <span ref-type="bibr" rid="pntd.0004199.ref013">
               <sup>
                  <a href="#pntd.0004199.ref013">13</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pntd.0004199.ref015">
               <sup>
                  <a href="#pntd.0004199.ref015">15</a>
               </sup>
            </span>]. As of 7 August 2015 CHIKF cases were confirmed in 50 countries / territories in
            the Caribbean and mainland Americas with approximately 1.7 million suspected / confirmed
            cases since October 2013 [
            <span ref-type="bibr" rid="pntd.0004199.ref016">
               <sup>
                  <a href="#pntd.0004199.ref016">16</a>
               </sup>
            </span>]. CHIKV belonging to the ECSA genotype was also recently documented in Brazil but
            has not been confirmed elsewhere in the Americas [
            <span ref-type="bibr" rid="pntd.0004199.ref017">
               <sup>
                  <a href="#pntd.0004199.ref017">17</a>
               </sup>
            </span>].
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Although CHIKF is rarely life threatening, the symptoms can be severely incapacitating,
            rendering patients unable to perform normal tasks or go to work. As a result, particularly
            in immunologically naïve populations where attack rates can be as high as 30–50% and
            the vast majority of infections are symptomatic [
            <span ref-type="bibr" rid="pntd.0004199.ref018">
               <sup>
                  <a href="#pntd.0004199.ref018">18</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pntd.0004199.ref019">
               <sup>
                  <a href="#pntd.0004199.ref019">19</a>
               </sup>
            </span>] [
            <span ref-type="bibr" rid="pntd.0004199.ref020">
               <sup>
                  <a href="#pntd.0004199.ref020">20</a>
               </sup>
            </span>], the disease burden, potential to overwhelm public health systems and indirect costs
            can be very significant. Clinical overlap and confusion with other acute undifferentiated
            fevers e.g. dengue disease (which can be fatal), are additional concerns in affected
            regions.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The Caribbean twin-island Republic of Trinidad and Tobago (T&amp;T) reported its first
            cases of CHIKV infection in July 2014 (in Trinidad) [
            <span ref-type="bibr" rid="pntd.0004199.ref021">
               <sup>
                  <a href="#pntd.0004199.ref021">21</a>
               </sup>
            </span>]. In this study we report the genetic characterisation of CHIKV and compare clinical,
            laboratory and epidemiological characteristics of patients with and without confirmed
            CHIKV or dengue virus (DENV) infections who presented at a major hospital in Trinidad,
            during an acute undifferentiated febrile illness (AUFI) surveillance study.
         </p>
      </div>
      <div id="sec002">
         <h2>Methods</h2>
         <div id="sec003">
            <h3>Study population, recruitment and sample collection</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Study participants were enrolled at the Adult Primary Care Facility (APCF) of the
               Eric Williams Medical Sciences Complex (EWMSC) during the period of December 25
               <sup>th</sup> 2013 to November 5
               <sup>th</sup> 2014. Individuals were recruited, on average, 3 days a week throughout this period,
               with the exception of October 2014, when no sample collection was undertaken. Individuals
               presenting at the APCF on study days with suspected dengue fever (DF) or CHIKF were
               eligible for enrollment. Specifically, patients were eligible for enrollment if they
               were febrile (having an axillary temperature of ≥38°C) or had a reported history of
               fever within the last 7 days, accompanied by one or more of the following symptoms
               commonly associated with DF and CHIKF: headache, nausea, vomiting, diarrhoea, aches
               and pains, eye pain, evidence of haemorrhagic manifestation, rash, joint pain and
               swollen joints. Individuals who had a readily identifiable focus of infection were
               excluded from the study.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">At presentation and enrollment, whole blood was collected in non-additive and/or ethylenediaminetetraacetic
               acid (EDTA) containing blood collection tubes and a questionnaire eliciting relevant
               demographic and clinical information, medical and recent travel history, and risk
               factors for exposure to mosquito bites was administered. Whole blood was stored at
               4°C and centrifuged at 1500g for 10 minutes at 4°C within 24 hours. Sera and plasma
               samples were stored at -80°C until use.
            </p>
         </div>
         <div id="sec004">
            <h3>Reverse transcription-quantitative polymerase chain reaction (RT-qPCR) detection of
               CHIKV and DENV viral RNA
            </h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Nucleic acid extraction was performed using QIAamp Viral RNA Mini Kit (Qiagen; Valencia,
               CA) and carried out according to the manufacturer’s instructions. Extractions were
               performed using 140μl of serum, or plasma where serum was unavailable. Nucleic acid
               samples were screened and serotyped for DENV-1-4 using the DENV multiplex RT-qPCR
               assay previously described by Waggoner [
               <span ref-type="bibr" rid="pntd.0004199.ref022">
                  <sup>
                     <a href="#pntd.0004199.ref022">22</a>
                  </sup>
               </span>]. Screening for CHIKV was carried out using the RT-qPCR general assay previously
               described by Panning [
               <span ref-type="bibr" rid="pntd.0004199.ref023">
                  <sup>
                     <a href="#pntd.0004199.ref023">23</a>
                  </sup>
               </span>]. All RT-qPCR screening was done using the SuperScript III Platinum One-Step qRT-PCR
               kit (Invitrogen; Carlsbad, CA) with final reaction volumes amended to 25μl instead
               of the manufacturer’s recommended 50μl.
            </p>
         </div>
         <div id="sec005">
            <h3>Illumina sequencing</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">For ten samples with low Ct values in RT-qPCR assays, viral RNA was extracted then
               fragmented by incubation at 94°C for eight (8) minutes in 19.5 μl of fragmentation
               buffer (Illumina Inc., San Diego, CA). First and second strand synthesis, adapter
               ligation and amplification of the library were performed using the Illumina TruSeq
               RNA Sample Preparation kit v2 under conditions prescribed by the manufacturer (Illumina
               Inc., San Diego, CA). Cluster formation of the library DNA templates was performed
               using the TruSeq PE Cluster Kit v3 (Illumina Inc., San Diego, CA) and the Illumina
               cBot workstation using conditions recommended by the manufacturer. Paired end 50 base
               sequencing by synthesis was performed using TruSeq SBS kit v3 (Illumina Inc., San
               Diego, CA) on an Illumina HiSeq 1000 using protocols defined by the manufacturer.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Cluster density per lane was 820–940 k/mm
               <sup>2</sup> and post-filter reads ranged from 148–218 million per lane. Base call conversion
               to sequence reads was performed using CASAVA-1.8.2. Reads were filtered for quality
               and adapter sequences were removed, then viral contigs were assembled 
               <i>de novo</i> using AbySS software [
               <span ref-type="bibr" rid="pntd.0004199.ref024">
                  <sup>
                     <a href="#pntd.0004199.ref024">24</a>
                  </sup>
               </span>]. Assembled contigs were checked using bowtie2 to align reads to the contigs [
               <span ref-type="bibr" rid="pntd.0004199.ref025">
                  <sup>
                     <a href="#pntd.0004199.ref025">25</a>
                  </sup>
               </span>] followed by visualization using the integrative genomics viewer [
               <span ref-type="bibr" rid="pntd.0004199.ref026">
                  <sup>
                     <a href="#pntd.0004199.ref026">26</a>
                  </sup>
               </span>].
            </p>
         </div>
         <div id="sec006">
            <h3>Phylogenetic analyses</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Newly generated CHIKV genomic sequences along with sequences from Genbank representing
               all three genotypes were aligned using the ClustalW alignment tool [
               <span ref-type="bibr" rid="pntd.0004199.ref027">
                  <sup>
                     <a href="#pntd.0004199.ref027">27</a>
                  </sup>
               </span>] within Geneious Version 7.1.7 (
               <div tagxxx="ext-link">www.geneious.com</div>; [
               <span ref-type="bibr" rid="pntd.0004199.ref028">
                  <sup>
                     <a href="#pntd.0004199.ref028">28</a>
                  </sup>
               </span>]). Sequences were trimmed to 11,259 nucleotides at the boundaries of the open reading
               frames (ORFs) so that ambiguous alignment of the 5’ and 3’ untranslated regions was
               excluded from analyses. The final data set comprised of 74 of these complete coding
               sequences from 23 countries isolated during 1953–2014 (
               <span ref-type="supplementary-material" rid="pntd.0004199.s002">S1 Table</span>). Nucleotide and amino acid position numberings are based on NCBI reference sequence
               NC_004162.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To rule out the presence of recombination within the data set, which could affect
               the phylogenetic structure, all genomic sequences were screened using SBP and GARD
               [
               <span ref-type="bibr" rid="pntd.0004199.ref029">
                  <sup>
                     <a href="#pntd.0004199.ref029">29</a>
                  </sup>
               </span>] from the HyPhy online package [
               <span ref-type="bibr" rid="pntd.0004199.ref030">
                  <sup>
                     <a href="#pntd.0004199.ref030">30</a>
                  </sup>
               </span>]. The best-fit model for subsequent analyses (GTR+G
               <div tagxxx="sub">4</div>) was selected using JMODELTEST 2.1.7 [
               <span ref-type="bibr" rid="pntd.0004199.ref031">
                  <sup>
                     <a href="#pntd.0004199.ref031">31</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pntd.0004199.ref032">
                  <sup>
                     <a href="#pntd.0004199.ref032">32</a>
                  </sup>
               </span>]. Phylogenetic trees were inferred under this model using the maximum-likelihood
               (ML) method in the PhyML program online [
               <span ref-type="bibr" rid="pntd.0004199.ref033">
                  <sup>
                     <a href="#pntd.0004199.ref033">33</a>
                  </sup>
               </span>] and the maximum clade credibility (MCC) from BEAST version 1.8.1 [
               <span ref-type="bibr" rid="pntd.0004199.ref034">
                  <sup>
                     <a href="#pntd.0004199.ref034">34</a>
                  </sup>
               </span>].
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Substitution rates and times to the most recent common ancestor (T
               <div tagxxx="sub">MRCA</div>) were jointly estimated using BEAST, with the GTR+G
               <div tagxxx="sub">4</div> model of nucleotide substitution, along with a relaxed lognormal molecular clock
               model [
               <span ref-type="bibr" rid="pntd.0004199.ref035">
                  <sup>
                     <a href="#pntd.0004199.ref035">35</a>
                  </sup>
               </span>] and a GMRF skyline tree prior [
               <span ref-type="bibr" rid="pntd.0004199.ref036">
                  <sup>
                     <a href="#pntd.0004199.ref036">36</a>
                  </sup>
               </span>]. The analysis ran in duplicate for 50 million generations with 10% removed as burnin,
               and the convergence of parameters were assessed by calculating the effective sample
               size (ESS &gt;200) using TRACER v1.6 [
               <span ref-type="bibr" rid="pntd.0004199.ref037">
                  <sup>
                     <a href="#pntd.0004199.ref037">37</a>
                  </sup>
               </span>].
            </p>
         </div>
         <div id="sec007">
            <h3>Dengue-specific IgG and IgM Enzyme-linked immunosorbent assay (ELISA)</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Panbio Dengue IgM and IgG Capture ELISA (Brisbane, Australia) were used to screen
               sera for the presence of DENV-specific antibodies using manufacturer’s instructions,
               and all samples were screened in duplicate. Samples were designated as positive for
               dengue-specific IgM if both duplicates returned a value of &gt;11 Panbio Units and as
               negative at &lt;9 Panbio Units. Samples were designated as positive for dengue-specific
               IgG if both duplicates returned a value of &gt;22 Panbio Units and as negative at &lt;18
               Panbio Units. For samples returning equivocal results (9 to 11 and 18 to 22 Panbio
               Units) or where the duplicates fell into different categories (i.e. one weakly positive
               or negative and the other equivocal), the test was repeated. Samples that remained
               equivocal were designated as such and for those with that remained with the duplicates
               falling in different categories, the mean value was taken and the sample classified
               accordingly.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">According to the manufacturer (due to the detection thresholds for the two assays),
               the presence of detectable levels of IgM (in the absence of detectable IgG) indicates
               a probable recent primary infection with DENV, while the presence of detectable levels
               of IgG indicates probable recent secondary dengue virus infection. The manufacturers
               note that IgM levels in a secondary infection may be undetectable by the assay. Thus
               IgM+/IgG- result was interpreted as evidence of a probable recent primary DENV infection,
               and both IgM+/IgG+ and IgM-/IgG+ results were interpreted as probable recent secondary
               DENV infections.
            </p>
         </div>
         <div id="sec008">
            <h3>Chikungunya-specific IgM Enzyme-linked immunosorbent assay (ELISA)</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Euroimmum Chikungunya IgM Capture ELISA (Leubeck, Germany) was used to screen sera
               and plasma for the presence of CHIKV-specific IgM antibodies. ELISAs were carried
               out in accordance with the manufacturer’s instructions and all samples were screened
               in duplicate. Samples were designated as positive for CHIKV-specific IgM if both duplicates
               returned a value of ≥ 1.1 and as negative at &lt; 0.8. No sample tested returned an equivocal
               result (≥ 0.8 to &lt; 1.1).
            </p>
         </div>
         <div id="sec009">
            <h3>Statistical analysis</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The frequencies of patient demographic and clinical characteristics were determined
               by questionnaire. The outcome for analysis was having a confirmed infection with CHIKV,
               defined as a positive result on RT-qPCR. Bivariable associations with a confirmed
               CHIKV infection were determined using the student’s t-test for continuous variables
               and the Pearson’s chi-square or Fisher’s exact test for categorical variables. All
               variables with a 
               <i>p</i>-value less than or equal to 0.10 were considered in the final multivariable model,
               which was determined using binary logistic regression using a forward stepwise selection
               procedure (entry: 5%, exit: 10%). Adjusted odds ratios (aORs) and 95% confidence intervals
               (CIs) are reported.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To compare the symptomatic presentation for RT-qPCR positive DENV and CHIKV, a bivariable
               analysis was performed using the Mann-Whitney U test for continuous variables and
               the Pearson’s chi-square or Fisher’s exact test for categorical variables. Data were
               entered using Microsoft Excel 2007 (Microsoft Corporation, Redmond, WA, USA), then
               transferred to IBM SPSS Statistics v20 (IBM Corporation, Armonk, NY, USA) for analysis.
               All 
               <i>p</i>-values &lt;0.05 were considered significant.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Positive predictive values (PPVs), negative predictive values (NPVs), and the sensitivity,
               specificity and likelihood ratios of different combinations of clinical features used
               to distinguish confirmed and probable DF and CHIKF from other AUFIs were also calculated.
               Combinations used to differentiate DF from CHIKF were also compared. For both CHIKF
               and DF these values were calculated using confirmed cases (RT-PCR positive for virus)
               and probable recent cases (ELISA positive for IgM/IgG antibodies).
            </p>
         </div>
         <div id="sec010">
            <h3>Accession numbers</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The 8 newly generated CHIKV whole genome sequences are available from Genbank. Accession
               numbers: KR046227, KR046228, KR046229, KR046230, KR046231, KR046232, KR046233 and
               KR046234.
            </p>
         </div>
         <div id="sec011">
            <h3>Ethical approval</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The study protocols were approved by the Ethics Committee of the University of the
               West Indies, St. Augustine and the Trinidad and Tobago Ministry of Health. Permission
               to carry out the study at the EWMSC was granted by the North Central Regional Health
               Authority (NCRHA). Written informed consent was obtained from all participants.
            </p>
         </div>
      </div>
      <div id="sec012">
         <h2>Results</h2>
         <div id="sec013">
            <h3>Frequencies</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">There were 158 patients who met the inclusion criteria for AUFI and agreed to participate
               in the study. Of these, the median age was 32, ranging from 16–88 years. Half (50%)
               of the patients were male and, of those with complete data on ethnicity, the majority
               were Afro- (33%), Indo-(31%) Trinidadian or Mixed (26%) (
               <span ref-type="supplementary-material" rid="pntd.0004199.s003">S2 Table</span>). Just under a third (30%) had received greater than a secondary level of education,
               and most (69%) were employed. Most patients (79%) were nationals of Trinidad &amp; Tobago,
               with relatively few (7%) reporting a travel history outside of Trinidad in the two
               weeks prior to interview. Twenty-two percent reported having had laboratory-confirmed
               DF previously, 25% reported that others in their household had been febrile in the
               two weeks prior to their interview, 55% reported storing water at home and, while
               15% reported having screened windows at home, most (80%) reported a history of mosquito
               bites at their place of residence.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">At interview, the most commonly reported symptoms were headache (83%), weakness (67%),
               joint pain (65%) and muscle pain (65%). Seventeen percent reported some degree of
               haemorrhagic manifestations and 22% reported a rash (
               <span ref-type="supplementary-material" rid="pntd.0004199.s003">S2 Table</span>). Thirty (19%) of the 158 individuals were positive for CHIKV on RT-qPCR. The earliest
               case was an individual sampled on August 5
               <sup>th</sup>, 2014, about three weeks after the first CHIKV case was confirmed in Trinidad [
               <span ref-type="bibr" rid="pntd.0004199.ref038">
                  <sup>
                     <a href="#pntd.0004199.ref038">38</a>
                  </sup>
               </span>]. The number of RT-qPCR confirmed cases peaked in week 38 when the number of individuals
               enrolled was highest (
               <span ref-type="fig" rid="pntd.0004199.g001">Fig 1</span>). Cases were concentrated along the urbanised “East-West corridor” in the north of
               Trinidad primarily within the boundaries of the North Central Regional Health Authority
               served by the EWMSC. Additionally, 27 (17%) of the 158 individuals were positive by
               ELISA for anti-CHIKV IgM antibodies, including 6 of those persons positive by RT-qPCR
               for CHIKV.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pntd.0004199.g001</div>
               <span class="label">Fig 1</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Total number of patients enrolled and number of CHIKV and DENV RT-qPCR positive cases
                     by week.
                  </div>
                  <p>The green line graph indicates the number of patients enrolled per week. Histogram
                     bars indicate the number of individuals confirmed as RT-qPCR positive for CHIKV (blue)
                     or DENV (red) by RT-qPCR. Weeks 41–43 in 2014 are excluded as no patient enrolment
                     was attempted during this period.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Eight of the 158 individuals (5.1%) were confirmed as DENV-positive by RT-qPCR (
               <span ref-type="fig" rid="pntd.0004199.g001">Fig 1</span>). Of these, 6 were DENV-1 and one each DENV-3 and DENV-4. For 125 individuals (including
               6 of the aforementioned RT-qPCR positive individuals), there was sufficient serum
               available for both dengue-specific IgM and IgG ELISAs (
               <span ref-type="supplementary-material" rid="pntd.0004199.s004">S3 Table</span>). Using the combined RT-qPCR and ELISA results, 4.8% (n = 6) were designated as “confirmed
               current DENV infections” of which three were primary infections (i.e. IgM+/IgG-) and
               three were secondary infections (IgM+/IgG+), 18.4% (n = 23) were “probable recent
               primary DENV infections”, 42.4% (n = 53) were “probable recent secondary DENV infections”
               and 3.2% (n = 4) were equivocal. In total 66% (n = 82) had evidence of a current or
               probable recent DENV infection (
               <span ref-type="supplementary-material" rid="pntd.0004199.s004">S3 Table</span>).
            </p>
         </div>
         <div id="sec014">
            <h3>Bivariable analyses</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">On bivariable analyses (
               <span ref-type="table" rid="pntd.0004199.t001">
                  <a href="#pntd.0004199.t001">Table 1</a>
               </span> &amp; 
               <span ref-type="supplementary-material" rid="pntd.0004199.s005">S4 Table</span>), patients with RT-qPCR confirmed CHIKV infection (CHIKV+; n = 30) were significantly
               more likely to report joint pain (83%, 
               <i>p</i> = 0.020) than individuals who were RT-qPCR-negative for CHIKV (not CHIKV+) but were
               less likely to have travelled outside of Trinidad in the 2 weeks prior to interview
               (17%, 
               <i>p</i> = 0.044), or to have reported laboratory-confirmed dengue previously (7%, 
               <i>p</i> = 0.026). They were also more likely to have presented to the APCF earlier (mean
               days post onset of fever: 2.39 [CHIKV+] vs. 3.33 [not CHIKV+], 
               <i>p</i> = 0.021) and had lower mean white blood cell counts (6.52 [CHIKV+] vs. 8.36x10
               <sup>3</sup>/μl [not CHIKV+], 
               <i>p</i> = 0.016).
            </p>
            <div id="pntd.0004199.t001">
               <div tagxxx="object-id">10.1371/journal.pntd.0004199.t001</div>
               <span class="label">Table 1</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Bivariable associations between select characteristics of patients of the APCF of
                     the EWMSC, T&amp;T (Dec 2013 –Nov 2014) and having a confirmed infection
                     <span ref-type="table-fn" rid="t001fn002">
                        <sup>
                           <a href="#t001fn002">ⱡ</a>
                        </sup>
                     </span> with CHIKV.
                  </div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <colgroup>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                     </colgroup>
                     <thead>
                        <tr>
                           <th>Parameters</th>
                           <th>Not confirmed</th>
                           <th>Confirmed</th>
                           <th>p-value</th>
                        </tr>
                        <tr>
                           <th></th>
                           <th>n</th>
                           <th>(%)</th>
                           <th>n</th>
                           <th>(%)</th>
                           <th></th>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>Demographics</td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>Age, years</td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>    &lt;32</td>
                           <td>64</td>
                           <td>(50.0)</td>
                           <td>11</td>
                           <td>(36.7)</td>
                           <td>0.188</td>
                        </tr>
                        <tr>
                           <td>    &gt;32</td>
                           <td>64</td>
                           <td>(50.0)</td>
                           <td>19</td>
                           <td>(63.3)</td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>Sex</td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>    Male</td>
                           <td>68</td>
                           <td>(53.1)</td>
                           <td>11</td>
                           <td>(36.7)</td>
                           <td>0.105</td>
                        </tr>
                        <tr>
                           <td>    Female</td>
                           <td>60</td>
                           <td>(46.9)</td>
                           <td>19</td>
                           <td>(63.3)</td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>Marital status</td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>    Not married
                              <span ref-type="table-fn" rid="t001fn007">
                                 <sup>
                                    <a href="#t001fn007">a</a>
                                 </sup>
                              </span>
                           </td>
                           <td>58</td>
                           <td>(47.5)</td>
                           <td>7</td>
                           <td>(26.9)</td>
                           <td>0.054
                              <span ref-type="table-fn" rid="t001fn003">
                                 <sup>
                                    <a href="#t001fn003">†</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                        <tr>
                           <td>    Married (including common-law)</td>
                           <td>64</td>
                           <td>(52.5)</td>
                           <td>19</td>
                           <td>(73.1)</td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>Nationality</td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>    Not Trinidadian or Tobagonian</td>
                           <td>24</td>
                           <td>(18.8)</td>
                           <td>10</td>
                           <td>(33.3)</td>
                           <td>0.080
                              <span ref-type="table-fn" rid="t001fn003">
                                 <sup>
                                    <a href="#t001fn003">†</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                        <tr>
                           <td>    Trinidadian or Tobagonian</td>
                           <td>104</td>
                           <td>(81.3)</td>
                           <td>20</td>
                           <td>(66.7)</td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>Risk factors</td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>Travelled outside of Trinidad in the 2 weeks prior to interview</td>
                           <td>6</td>
                           <td>(5.0)</td>
                           <td>5</td>
                           <td>(16.7)</td>
                           <td>0.044
                              <span ref-type="table-fn" rid="t001fn004">
                                 <sup>
                                    <a href="#t001fn004">*</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                        <tr>
                           <td>Previous dengue</td>
                           <td>32</td>
                           <td>(26.0)</td>
                           <td>2</td>
                           <td>(6.9)</td>
                           <td>0.026
                              <span ref-type="table-fn" rid="t001fn004">
                                 <sup>
                                    <a href="#t001fn004">*</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                        <tr>
                           <td>Clinical factors</td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>Headache</td>
                           <td>108</td>
                           <td>(84.4)</td>
                           <td>23</td>
                           <td>(76.7)</td>
                           <td>0.313</td>
                        </tr>
                        <tr>
                           <td>Muscle pain</td>
                           <td>81</td>
                           <td>(63.3)</td>
                           <td>21</td>
                           <td>(70.0)</td>
                           <td>0.489</td>
                        </tr>
                        <tr>
                           <td>Joint pain</td>
                           <td>78</td>
                           <td>(60.9)</td>
                           <td>25</td>
                           <td>(83.3)</td>
                           <td>0.020
                              <span ref-type="table-fn" rid="t001fn004">
                                 <sup>
                                    <a href="#t001fn004">*</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                        <tr>
                           <td>Rash</td>
                           <td>24</td>
                           <td>(18.8)</td>
                           <td>10</td>
                           <td>(33.3)</td>
                           <td>0.080
                              <span ref-type="table-fn" rid="t001fn003">
                                 <sup>
                                    <a href="#t001fn003">†</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                        <tr>
                           <td>Sore throat</td>
                           <td>29</td>
                           <td>(22.7)</td>
                           <td>1</td>
                           <td>(3.3)</td>
                           <td>0.015
                              <span ref-type="table-fn" rid="t001fn004">
                                 <sup>
                                    <a href="#t001fn004">*</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                        <tr>
                           <td>Abdominal pain</td>
                           <td>40</td>
                           <td>(31.2)</td>
                           <td>2</td>
                           <td>(6.7)</td>
                           <td>0.006
                              <span ref-type="table-fn" rid="t001fn005">
                                 <sup>
                                    <a href="#t001fn005">**</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                        <tr>
                           <td>Any haemorrhagic manifestation
                              <span ref-type="table-fn" rid="t001fn008">
                                 <sup>
                                    <a href="#t001fn008">b</a>
                                 </sup>
                              </span>
                           </td>
                           <td>23</td>
                           <td>(18.0)</td>
                           <td>3</td>
                           <td>(10.0)</td>
                           <td>0.414</td>
                        </tr>
                        <tr>
                           <td></td>
                           <td>Mean</td>
                           <td>(s.d.)
                              <span ref-type="table-fn" rid="t001fn006">
                                 <sup>
                                    <a href="#t001fn006">‡</a>
                                 </sup>
                              </span>
                           </td>
                           <td>Mean</td>
                           <td>(s.d.)
                              <span ref-type="table-fn" rid="t001fn006">
                                 <sup>
                                    <a href="#t001fn006">‡</a>
                                 </sup>
                              </span>
                           </td>
                           <td>
                              <i>p-value</i>
                           </td>
                        </tr>
                        <tr>
                           <td>Days post onset of fever</td>
                           <td>3.33</td>
                           <td>(1.96)</td>
                           <td>2.39</td>
                           <td>(1.77)</td>
                           <td>0.021
                              <span ref-type="table-fn" rid="t001fn004">
                                 <sup>
                                    <a href="#t001fn004">*</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                        <tr>
                           <td>White blood cell count (10
                              <sup>3</sup>/ml)
                           </td>
                           <td>8.36</td>
                           <td>(4.85)</td>
                           <td>6.52</td>
                           <td>(3.25)</td>
                           <td>0.016
                              <span ref-type="table-fn" rid="t001fn004">
                                 <sup>
                                    <a href="#t001fn004">*</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                        <tr>
                           <td>Haematocrit (%)</td>
                           <td>43.81</td>
                           <td>(23.7)</td>
                           <td>39</td>
                           <td>(7.99)</td>
                           <td>0.276</td>
                        </tr>
                        <tr>
                           <td>Platelet count (10
                              <sup>3</sup>/ml)
                           </td>
                           <td>213.2</td>
                           <td>-100.86</td>
                           <td>234.6</td>
                           <td>(72.48)</td>
                           <td>0.277</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <table-wrap-foot xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div class="fn" id="t001fn001">
                     <p>A more comprehensive list of variables is presented in 
                        <span ref-type="supplementary-material" rid="pntd.0004199.s003">S2 Table</span>.
                     </p>
                  </div>
                  <div class="fn" id="t001fn002">
                     <p>
                        <sup>ⱡ</sup> Defined as a positive result by RT-qPCR
                     </p>
                  </div>
                  <div class="fn" id="t001fn003">
                     <p>
                        <sup>†</sup>
                        <i>p</i>&lt;0.10
                     </p>
                  </div>
                  <div class="fn" id="t001fn004">
                     <p>*
                        <i>p</i>&lt;0.05
                     </p>
                  </div>
                  <div class="fn" id="t001fn005">
                     <p>**
                        <i>p</i>&lt;0.01
                     </p>
                  </div>
                  <div class="fn" id="t001fn006">
                     <p>
                        <sup>‡</sup> Standard deviation
                     </p>
                  </div>
                  <div class="fn" id="t001fn007">
                     <p>
                        <sup>a</sup> Includes: single, divorced and widower
                     </p>
                  </div>
                  <div class="fn" id="t001fn008">
                     <p>
                        <sup>b</sup> Includes: nose bleed, gum bleed, abnormal vaginal bleeding, blood in urine, bruising
                        or blood in stool
                     </p>
                  </div>
               </table-wrap-foot>
            </div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In addition to the above variables, the following also met the criteria for inclusion
               in the multivariable model: marital status (
               <i>p</i> = 0.054), nationality (
               <i>p</i> = 0.080), number of days post onset of illness (
               <i>p</i> = 0.032), rash (
               <i>p</i> = 0.084), sore throat (
               <i>p</i> = 0.015), and abdominal pain (
               <i>p</i> = 0.006). Due to a highly significant negative correlation between nationality and
               travel history, only travel history was included in the model. Similarly, due to a
               strong, positive and highly significant correlation between number of days post onset
               of fever and days post onset of illness, only the former was considered in the model.
               Lastly, due to small cell sizes (n ≤ 2), having had dengue previously, sore throat
               and abdominal pain were not considered in the final model.
            </p>
         </div>
         <div id="sec015">
            <h3>Multivariable analyses</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The following variables were considered in the final multivariable model: marital
               status, joint pain, rash, days post onset of fever and white blood cell count. As
               shown in
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <span ref-type="table" rid="pntd.0004199.t002">
                  <a href="#pntd.0004199.t002">Table 2</a>
               </span>, those who had confirmed CHIKV infection were significantly more likely to have reported
               joint pain (aOR: 4.52, 95% CI: 1.28–16.00), and significantly less likely to be interviewed
               at a later stage of illness (days post onset of fever—aOR: 0.56, 95% CI: 0.40–0.78)
               and have a lower white blood cell count (aOR: 0.83, 95% CI: 0.71–0.96).
            </p>
            <div id="pntd.0004199.t002">
               <div tagxxx="object-id">10.1371/journal.pntd.0004199.t002</div>
               <span class="label">Table 2</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Multivariable associations with having a confirmed infection
                     <span ref-type="table-fn" rid="t002fn001">
                        <sup>
                           <a href="#t002fn001">ⱡ</a>
                        </sup>
                     </span> with CHIKV in patients of the APCF of the EWMSC, T&amp;T (Dec 2013 –Nov 2014).
                  </div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <colgroup>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                     </colgroup>
                     <thead>
                        <tr>
                           <th> </th>
                           <th>Adjusted odds ratio</th>
                           <th>(95% confidence interval)</th>
                           <th>
                              <i>p</i>-value
                           </th>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>Joint pain</td>
                           <td> </td>
                           <td> </td>
                           <td> </td>
                        </tr>
                        <tr>
                           <td>    No</td>
                           <td>1</td>
                           <td>Reference</td>
                           <td> </td>
                        </tr>
                        <tr>
                           <td>    Yes</td>
                           <td>4.52</td>
                           <td>(1.28,16.00)</td>
                           <td>0.019
                              <span ref-type="table-fn" rid="t002fn002">
                                 <sup>
                                    <a href="#t002fn002">*</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                        <tr>
                           <td>Days post onset of fever</td>
                           <td>0.56</td>
                           <td>(0.40,0.78)</td>
                           <td>0.001
                              <span ref-type="table-fn" rid="t002fn003">
                                 <sup>
                                    <a href="#t002fn003">**</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                        <tr>
                           <td>White blood cell count (10
                              <sup>3</sup>/μl)
                           </td>
                           <td>0.83</td>
                           <td>(0.71,0.96)</td>
                           <td>0.014
                              <span ref-type="table-fn" rid="t002fn002">
                                 <sup>
                                    <a href="#t002fn002">*</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                     </div>
                  </table>
               </div>
               <table-wrap-foot xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div class="fn" id="t002fn001">
                     <p>
                        <sup>ⱡ</sup> Defined as a positive result by RT-qPCR
                     </p>
                  </div>
                  <div class="fn" id="t002fn002">
                     <p>*
                        <i>p</i>&lt;0.05
                     </p>
                  </div>
                  <div class="fn" id="t002fn003">
                     <p>**
                        <i>p</i>&lt;0.01
                     </p>
                  </div>
               </table-wrap-foot>
            </div>
         </div>
         <div id="sec016">
            <h3>Comparison of patients positive by RT-qPCR for DENV and CHIKV</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Among the 38 patients positive by RT-qPCR for CHIKV (n = 30) or DENV (n = 8), there
               were no significant differences in clinical presentations (p &gt; 0.05) between those
               with CHIKV and those with DENV (
               <span ref-type="table" rid="pntd.0004199.t003">
                  <a href="#pntd.0004199.t003">Table 3</a>
               </span> and 
               <span ref-type="supplementary-material" rid="pntd.0004199.s006">S5 Table</span>). However, when compared to DENV, persons with CHIKV more often reported joint pain
               (83% vs. 50%, 
               <i>p</i> = 0.071) and rash (33% vs. 0%, 
               <i>p</i> = 0.082), and less often reported abdominal pain (7% vs. 38%, 
               <i>p</i> = 0.053). Patients with DENV had significantly lower median white blood cell count
               (3.50 vs. 6.00 x10
               <sup>3</sup>/μl, 
               <i>p</i> = 0.028) and platelet (PLT) counts (1.47 vs. 2.39x10
               <sup>5</sup>/μl, 
               <i>p</i> = 0.022).
            </p>
            <div id="pntd.0004199.t003">
               <div tagxxx="object-id">10.1371/journal.pntd.0004199.t003</div>
               <span class="label">Table 3</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Bivariable associations between symptoms and clinical characteristics of patients
                     of the APCF of the EWMSC, T&amp;T (Dec 2013 –Nov 2014) and having a confirmed infection
                     <span ref-type="table-fn" rid="t003fn002">
                        <sup>
                           <a href="#t003fn002">ⱡ</a>
                        </sup>
                     </span> with either DENV or CHIKV.
                  </div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <colgroup>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                     </colgroup>
                     <thead>
                        <tr>
                           <th> </th>
                           <th>Status</th>
                           <th> </th>
                           <th> </th>
                           <th> </th>
                           <th> </th>
                        </tr>
                        <tr>
                           <th> </th>
                           <th>DENV</th>
                           <th> </th>
                           <th>CHIKV</th>
                           <th> </th>
                           <th>p-value</th>
                        </tr>
                        <tr>
                           <th> </th>
                           <th>n</th>
                           <th>(%)</th>
                           <th>n</th>
                           <th>(%)</th>
                           <th> </th>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>Symptoms</td>
                           <td> </td>
                           <td> </td>
                           <td> </td>
                           <td> </td>
                           <td> </td>
                        </tr>
                        <tr>
                           <td>Headache</td>
                           <td>7</td>
                           <td>(87.5)</td>
                           <td>23</td>
                           <td>76.7)</td>
                           <td>0.66</td>
                        </tr>
                        <tr>
                           <td>Muscle pain</td>
                           <td>5</td>
                           <td>(62.5)</td>
                           <td>21</td>
                           <td>(70.0)</td>
                           <td>0.689</td>
                        </tr>
                        <tr>
                           <td>Joint pain</td>
                           <td>4</td>
                           <td>(50.0)</td>
                           <td>5</td>
                           <td>(83.3)</td>
                           <td>0.071
                              <span ref-type="table-fn" rid="t003fn003">
                                 <sup>
                                    <a href="#t003fn003">†</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                        <tr>
                           <td>Back pain</td>
                           <td>1</td>
                           <td>(12.5)</td>
                           <td>0</td>
                           <td>0</td>
                           <td>0.211</td>
                        </tr>
                        <tr>
                           <td>Rash</td>
                           <td>0</td>
                           <td>0</td>
                           <td>10</td>
                           <td>(33.3)</td>
                           <td>0.082
                              <span ref-type="table-fn" rid="t003fn003">
                                 <sup>
                                    <a href="#t003fn003">†</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                        <tr>
                           <td>Fatigue</td>
                           <td>1</td>
                           <td>(12.5)</td>
                           <td>15</td>
                           <td>(50.0)</td>
                           <td>0.106</td>
                        </tr>
                        <tr>
                           <td>Eye pain</td>
                           <td>5</td>
                           <td>(62.5)</td>
                           <td>12</td>
                           <td>(40.0)</td>
                           <td>0.426</td>
                        </tr>
                        <tr>
                           <td>Cough</td>
                           <td>3</td>
                           <td>(37.5)</td>
                           <td>7</td>
                           <td>(23.3)</td>
                           <td>0.411</td>
                        </tr>
                        <tr>
                           <td>Nausea</td>
                           <td>4</td>
                           <td>(50.0)</td>
                           <td>6</td>
                           <td>(20.0)</td>
                           <td>0.17</td>
                        </tr>
                        <tr>
                           <td>Vomiting</td>
                           <td>4</td>
                           <td>(50.0)</td>
                           <td>8</td>
                           <td>(26.7)</td>
                           <td>0.232</td>
                        </tr>
                        <tr>
                           <td>Diarrhoea</td>
                           <td>2</td>
                           <td>(25.0)</td>
                           <td>5</td>
                           <td>(16.7</td>
                           <td>0.624</td>
                        </tr>
                        <tr>
                           <td>Sore throat</td>
                           <td>1</td>
                           <td>(12.5)</td>
                           <td>1</td>
                           <td>(3.3)</td>
                           <td>0.381</td>
                        </tr>
                        <tr>
                           <td>Weakness</td>
                           <td>5</td>
                           <td>(62.5)</td>
                           <td>21</td>
                           <td>(70.0)</td>
                           <td>0.689</td>
                        </tr>
                        <tr>
                           <td>Stiff neck</td>
                           <td>1</td>
                           <td>(12.5)</td>
                           <td>4</td>
                           <td>(13.3)</td>
                           <td>1</td>
                        </tr>
                        <tr>
                           <td>Dizziness</td>
                           <td>2</td>
                           <td>(25.0)</td>
                           <td>13</td>
                           <td>(43.3)</td>
                           <td>0.44</td>
                        </tr>
                        <tr>
                           <td>Disorientation</td>
                           <td>0</td>
                           <td>0</td>
                           <td>3</td>
                           <td>(10.0)</td>
                           <td>1</td>
                        </tr>
                        <tr>
                           <td>Abdominal pain</td>
                           <td>3</td>
                           <td>(37.5)</td>
                           <td>2</td>
                           <td>(6.7)</td>
                           <td>0.053
                              <span ref-type="table-fn" rid="t003fn003">
                                 <sup>
                                    <a href="#t003fn003">†</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                        <tr>
                           <td>Any haemorrhagic manifestation
                              <sup>a</sup>
                           </td>
                           <td>0</td>
                           <td>0</td>
                           <td>3</td>
                           <td>(10.0)</td>
                           <td>1</td>
                        </tr>
                        <tr>
                           <td> </td>
                           <td>Median</td>
                           <td>Min-Max
                              <span ref-type="table-fn" rid="t003fn005">
                                 <sup>
                                    <a href="#t003fn005">‡</a>
                                 </sup>
                              </span>
                           </td>
                           <td>Median</td>
                           <td>Min-Max
                              <span ref-type="table-fn" rid="t003fn005">
                                 <sup>
                                    <a href="#t003fn005">‡</a>
                                 </sup>
                              </span>
                           </td>
                           <td>p-value</td>
                        </tr>
                        <tr>
                           <td>Days post onset of fever</td>
                           <td>2.5</td>
                           <td>1.0–5.0</td>
                           <td>2</td>
                           <td>0.0–6.0</td>
                           <td>0.484</td>
                        </tr>
                        <tr>
                           <td>Temperature (°C)</td>
                           <td>37</td>
                           <td>36.7–39.3</td>
                           <td>38</td>
                           <td>37.0–39.0</td>
                           <td>0.288</td>
                        </tr>
                        <tr>
                           <td>White blood cell count (10
                              <sup>3</sup>/ml)
                           </td>
                           <td>3.5</td>
                           <td>1.7–6.3</td>
                           <td>6</td>
                           <td>2.7–17.6</td>
                           <td>0.028
                              <span ref-type="table-fn" rid="t003fn004">
                                 <sup>
                                    <a href="#t003fn004">*</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                        <tr>
                           <td>Haematocrit (%)</td>
                           <td>43.1</td>
                           <td>37.9–48.8</td>
                           <td>39.85</td>
                           <td>12.9–51.4</td>
                           <td>0.092
                              <span ref-type="table-fn" rid="t003fn003">
                                 <sup>
                                    <a href="#t003fn003">†</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                        <tr>
                           <td>Platelet count (10
                              <sup>3</sup>/ml)
                           </td>
                           <td>147</td>
                           <td>51.0–294.0</td>
                           <td>239</td>
                           <td>83.0–375.0</td>
                           <td>0.022
                              <span ref-type="table-fn" rid="t003fn004">
                                 <sup>
                                    <a href="#t003fn004">*</a>
                                 </sup>
                              </span>
                           </td>
                        </tr>
                     </div>
                  </table>
               </div>
               <table-wrap-foot xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div class="fn" id="t003fn001">
                     <p>A more comprehensive list of variables is presented in 
                        <span ref-type="supplementary-material" rid="pntd.0004199.s006">S5 Table</span>.
                     </p>
                  </div>
                  <div class="fn" id="t003fn002">
                     <p>
                        <sup>ⱡ</sup> Defined as a positive result by RT-qPCR
                     </p>
                  </div>
                  <div class="fn" id="t003fn003">
                     <p>
                        <sup>†</sup>
                        <i>p</i>&lt;0.10
                     </p>
                  </div>
                  <div class="fn" id="t003fn004">
                     <p>*
                        <i>p</i>&lt;0.05
                     </p>
                  </div>
                  <div class="fn" id="t003fn005">
                     <p>
                        <sup>‡</sup> Minimum-Maximum
                     </p>
                  </div>
                  <div class="fn" id="t003fn006">
                     <p>
                        <sup>a</sup> Includes: nose bleed, gum bleed, abnormal vaginal bleeding, blood in urine, bruising
                        or blood in stool.
                     </p>
                  </div>
               </table-wrap-foot>
            </div>
         </div>
         <div id="sec017">
            <h3>Distinguishing confirmed and probable recent DF and CHIKF from other AUFIs using clinical
               features
            </h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The clinical features joint pain and white blood cell count &lt;7 x10
               <sup>3</sup>/μl were selected as they were significant on bivariable analysis. For DF, PLT count
               &lt;150 x10
               <sup>3</sup>/ μl and abdominal pain were also selected as patients with confirmed DENV infection
               were more likely to display/report these symptoms than persons with confirmed CHIKV
               infection. Similarly for CHIKF rash was included as patients with confirmed CHIKV
               infection were more likely to report this sign than persons with confirmed DENV infection
               (
               <span ref-type="table" rid="pntd.0004199.t004">
                  <a href="#pntd.0004199.t004">Table 4</a>
               </span>).
            </p>
            <div id="pntd.0004199.t004">
               <div tagxxx="object-id">10.1371/journal.pntd.0004199.t004</div>
               <span class="label">Table 4</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Positive predictive values (PPVs), negative predictive values (NPVs), sensitivity,
                     specificity and likelihood ratios of clinical features used to distinguish patients
                     of the APCF of the EWMSC, T&amp;T (Dec 2013 –Nov 2014) with DF 
                     <span ref-type="table-fn" rid="t004fn001">
                        <sup>
                           <a href="#t004fn001">ⱡ</a>
                        </sup>
                     </span> and CHIKF 
                     <span ref-type="table-fn" rid="t004fn001">
                        <sup>
                           <a href="#t004fn001">ⱡ</a>
                        </sup>
                     </span> from patients with other AUFIs.
                  </div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <colgroup>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                     </colgroup>
                     <thead>
                        <tr>
                           <th>Parameters</th>
                           <th>PPV (%)</th>
                           <th>NPV (%)</th>
                           <th>Sens (%)</th>
                           <th>Spec (%)</th>
                           <th>Likelihood Ratio (+)</th>
                           <th>Likelihood Ratio (-)</th>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>CHIKV</td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>Joint pain</td>
                           <td>43.7</td>
                           <td>89.1</td>
                           <td>88.2</td>
                           <td>45.8</td>
                           <td>1.6</td>
                           <td>0.3</td>
                        </tr>
                        <tr>
                           <td>Rash</td>
                           <td>70.6</td>
                           <td>78.1</td>
                           <td>47.1</td>
                           <td>90.6</td>
                           <td>5.0</td>
                           <td>0.6</td>
                        </tr>
                        <tr>
                           <td>White blood cell count (&lt;7x10
                              <sup>3</sup>/μl)
                           </td>
                           <td>48.8</td>
                           <td>83.6</td>
                           <td>78.0</td>
                           <td>57.7</td>
                           <td>1.8</td>
                           <td>0.4</td>
                        </tr>
                        <tr>
                           <td>Rash &amp; joint pain</td>
                           <td>72.4</td>
                           <td>94.0</td>
                           <td>87.5</td>
                           <td>85.5</td>
                           <td>6.0</td>
                           <td>0.2</td>
                        </tr>
                        <tr>
                           <td>Rash &amp; white blood cell count (&lt;7 x10
                              <sup>3</sup>/μl)
                           </td>
                           <td>74.1</td>
                           <td>86.9</td>
                           <td>71.4</td>
                           <td>88.3</td>
                           <td>1.9</td>
                           <td>0.3</td>
                        </tr>
                        <tr>
                           <td>Joint pain &amp; white blood cell count (&lt;7x10
                              <sup>3</sup>/μl)
                           </td>
                           <td>20.0</td>
                           <td>96.3</td>
                           <td>83.3</td>
                           <td>56.5</td>
                           <td>2.5</td>
                           <td>0.5</td>
                        </tr>
                        <tr>
                           <td>Rash, joint pain &amp; white blood cell count (&lt;7x10
                              <sup>3</sup>/μl)
                           </td>
                           <td>77.3</td>
                           <td>96.3</td>
                           <td>94.4</td>
                           <td>83.9</td>
                           <td>5.9</td>
                           <td>0.1</td>
                        </tr>
                        <tr>
                           <td>DENV</td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                           <td></td>
                        </tr>
                        <tr>
                           <td>Platelet count (&lt;150x10
                              <sup>3</sup>/μl)
                           </td>
                           <td>84.6</td>
                           <td>34.9</td>
                           <td>28.2</td>
                           <td>88.2</td>
                           <td>2.4</td>
                           <td>0.8</td>
                        </tr>
                        <tr>
                           <td>Abdominal pain</td>
                           <td>77.4</td>
                           <td>34.8</td>
                           <td>28.6</td>
                           <td>82.1</td>
                           <td>1.6</td>
                           <td>0.9</td>
                        </tr>
                        <tr>
                           <td>Abdominal pain &amp; platelet count (&lt;150 x10
                              <sup>3</sup>/μl)
                           </td>
                           <td>83.3</td>
                           <td>48.2</td>
                           <td>15.2</td>
                           <td>96.3</td>
                           <td>4.1</td>
                           <td>0.9</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <table-wrap-foot xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div class="fn" id="t004fn001">
                     <p>
                        <sup>ⱡ</sup> Defined as a positive result by RT-qPCR and/or positive for IgM/IgG antibodies by
                        ELISA
                     </p>
                  </div>
               </table-wrap-foot>
            </div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Individually none of the clinical features was able to efficiently differentiate DF
               or CHIKF from other AUFIs. The results are given in 
               <span ref-type="table" rid="pntd.0004199.t004">
                  <a href="#pntd.0004199.t004">Table 4</a>
               </span>. However the combination of rash, joint pain and white blood cell count &lt;7 x10
               <sup>3</sup>/μl was most efficiently able to differentiate CHIKF from other AUFIs. This combination
               had a PPV and NPV of 77% and 96% respectively. The three parameters combined had a
               sensitivity of 94% and a specificity of 84%. Furthermore the positive and negative
               likelihood ratios were 5.9 and 0.1 respectively (
               <span ref-type="table" rid="pntd.0004199.t004">
                  <a href="#pntd.0004199.t004">Table 4</a>
               </span>).
            </p>
         </div>
         <div id="sec018">
            <h3>Distinguishing DF from CHIKF using clinical features</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">None of the combinations tested was overall able to efficiently differentiate between
               DENV and CHIKV. The combination that performed the best was rash, joint pain and white
               blood cell count &lt;7 x10
               <sup>3</sup>/μl with a PPV and NPV of 58% and 95% respectively as well as a sensitivity of 88%
               and a specificity of 78% (
               <span ref-type="table" rid="pntd.0004199.t005">
                  <a href="#pntd.0004199.t005">Table 5</a>
               </span>).
            </p>
            <div id="pntd.0004199.t005">
               <div tagxxx="object-id">10.1371/journal.pntd.0004199.t005</div>
               <span class="label">Table 5</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Positive predictive values (PPVs), negative predictive values (NPVs), sensitivity,
                     specificity and likelihood ratios of clinical features used to distinguish patients
                     of the APCF of the EWMSC, T&amp;T (Dec 2013 –Nov 2014) with DF 
                     <span ref-type="table-fn" rid="t005fn001">
                        <sup>
                           <a href="#t005fn001">ⱡ</a>
                        </sup>
                     </span> infection from patients with CHIKF
                     <span ref-type="table-fn" rid="t005fn001">
                        <sup>
                           <a href="#t005fn001">ⱡ</a>
                        </sup>
                     </span>.
                  </div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <colgroup>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                     </colgroup>
                     <thead>
                        <tr>
                           <th>Parameters</th>
                           <th>Sens (%)</th>
                           <th>Spec (%)</th>
                           <th>PPV (%)</th>
                           <th>NPV (%)</th>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>Platelet count (&lt;150x10
                              <sup>3</sup>/μl), abdominal pain
                           </td>
                           <td>7.3</td>
                           <td>96.3</td>
                           <td>75.0</td>
                           <td>40.6</td>
                        </tr>
                        <tr>
                           <td>Joint pain, rash</td>
                           <td>72.7</td>
                           <td>82.1</td>
                           <td>53.3</td>
                           <td>91.4</td>
                        </tr>
                        <tr>
                           <td>Joint pain, white blood cell count (&lt;7x10
                              <sup>3</sup>/μl)
                           </td>
                           <td>91.7</td>
                           <td>50.0</td>
                           <td>37.9</td>
                           <td>94.7</td>
                        </tr>
                        <tr>
                           <td>White blood cell count (&lt;7x10
                              <sup>3</sup>/μl), rash
                           </td>
                           <td>50.0</td>
                           <td>84.4</td>
                           <td>53.3</td>
                           <td>82.6</td>
                        </tr>
                        <tr>
                           <td>White blood cell count (&lt;7x10
                              <sup>3</sup>/μl), rash, joint pain
                           </td>
                           <td>87.5</td>
                           <td>78.3</td>
                           <td>58.3</td>
                           <td>94.7</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <table-wrap-foot xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div class="fn" id="t005fn001">
                     <p>
                        <sup>ⱡ</sup> Defined as a positive result by RT-qPCR and/or positive for IgM/IgG antibodies by
                        ELISA
                     </p>
                  </div>
               </table-wrap-foot>
            </div>
         </div>
         <div id="sec019">
            <h3>Molecular characterisation and phylogeny of CHIKV</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Using an Illumina platform, complete genome sequences for CHIKV were determined directly
               from RNA isolated from the sera of 8 individuals. The overall alignment rate of the
               reads varied for each sample, ranging from 0.45–17.5%. Nucleotide identity amongst
               the consensus sequences from these individuals was 99.9–100% and 99.8–100% at the
               amino acid level (
               <span ref-type="supplementary-material" rid="pntd.0004199.s007">S6 Table</span>). Non-synonymous substitutions occurred at nucleotide positions 774 (T→A), 1509 (C→T),
               1781 (C→T), 3563 (G→A), 8822 (C→T) and 10,059 (G→A), resulting in changes at aminoacid
               residues 233 (L→Q), 478 (A→V), 569 (R→C) and 1163 (V→I) in the non-structural polypeptide
               and at residues 419 (T→I) and 832 (G→R) in the structural polypeptide. The latter
               two being in the E2 and E1 proteins respectively. Both MCC (
               <span ref-type="fig" rid="pntd.0004199.g002">Fig 2</span>) and ML phylogenies (
               <span ref-type="supplementary-material" rid="pntd.0004199.s001">S1 Fig</span>) estimated from the data set showed that the Trinidad sequences clustered together
               with the BVI sequence (Accession no. KJ451624) isolated in 2014 near the beginning
               of the Caribbean outbreak [
               <span ref-type="bibr" rid="pntd.0004199.ref039">
                  <sup>
                     <a href="#pntd.0004199.ref039">39</a>
                  </sup>
               </span>]. The most closely related Asian sequences circulated between 2012 and 2013 in China,
               the Philippines and Micronesia. The clade containing the Trinidad and BVI sequences
               was defined by the aforementioned amino acid changes and was estimated to be evolving
               at a rate of 5 x 10
               <sup>−3</sup> substitutions per site per year (95% HPD: 1 x 10
               <sup>−4–</sup>1 x 10
               <sup>−3</sup> substitutions per site per year) with the most recent common ancestor (MRCA) having
               arisen about 1 year prior to November 2014 `(0.98 years ago [95% higher probability
               density (HPD): 0.49–1.63]).
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pntd.0004199.g002</div>
               <span class="label">Fig 2</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Maximum clade credibility (MCC) phylogeny based on the complete coding region of 74
                     CHIKV sequences.
                  </div>
                  <p>The three major CHIKV genotypes are labelled, as well as the IOL within the ECSA clade.
                     Sequences generated from this study (Genbank accession numbers (KR046227 –KR046234)
                     are labelled in blue. Nodes with clade credibilities (posterior probabilities) ≥ 0.95
                     are labelled accordingly. The clade credibility for the node supporting the Trinidad
                     and BVI sequences is shown in the insert together with the mean estimated time to
                     most recent common ancestor (tMRCA) in years (with 95% HPD in brackets).
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">For three individuals, sequencing was of sufficient depth to allow reliable reporting
               of within-host variation at individual nucleotide positions within the non-structural
               and structural polypeptide genes. Within the non-structural polypeptide ORF, for one
               individual, at nucleotide position 1228 (i.e. residue 384 of nsp1), 1.1% of reads
               had A instead of the majority T (p = 2.16 x 10
               <sup>−7</sup>) both of which encode Leu, and at nucleotide position 3276, 1.6% of reads had T instead
               of G (p = 1.77 x 10
               <sup>−13</sup>) which would result in a Ser to Ile substitution at residue 1067 of the nonstructural
               polypeptide (within nsp2). For the structural polypeptide gene, variation was detected
               at nucleotide position 9377 (residue 604 i.e. residue 279 in E2) where the minority
               species was A instead of G. This variant, which was detected in three individuals
               at frequencies of 2.03, 2.08 and 2.69% (p = 1.48 x 10
               <sup>−23</sup>, 1.15 x 10
               <sup>−4</sup>, 5.59 x 10
               <sup>−5</sup> respectively), would result in a Gly-to-Glu change. One individual also had a minority
               variant at nucleotide position 9037 such that 1.18% had T instead of G (p &lt; 6.19 x
               10
               <sup>−2</sup>), which would result in replacement of Glu at amino acid 491 of the structural polypeptide
               (residue 165 in E2) with a stop codon. None of the four single adaptive mutations
               in E1 (A226V) and E2 (K252Q; K233E; L210Q) or the synergistic adaptive mutations in
               E2/E3 (R198Q/S18F) that confer a fitness advantage for IOL strains in 
               <i>A</i>. 
               <i>albopictus</i> [
               <span ref-type="bibr" rid="pntd.0004199.ref006">
                  <sup>
                     <a href="#pntd.0004199.ref006">6</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pntd.0004199.ref008">
                  <sup>
                     <a href="#pntd.0004199.ref008">8</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pntd.0004199.ref040">
                  <sup>
                     <a href="#pntd.0004199.ref040">40</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pntd.0004199.ref041">
                  <sup>
                     <a href="#pntd.0004199.ref041">41</a>
                  </sup>
               </span>] were detected.
            </p>
         </div>
      </div>
      <div id="sec020">
         <h2>Discussion</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Of the 158 individuals enrolled in the study CHIKV was detected by RT-qPCR in 19%
            (n = 30). Anti-CHIKV IgM antibodies were detected in 6 of the 30 individuals and in
            21 individuals who tested negative for CHIKV by RT-qPCR. Therefore 51 individuals
            (32.3%) had evidence of infection with CHIKV. CHIKV accounted for 7.7%, 4,5%, 36.6%,
            and 13.3% of cases enrolled in July, August, September and November, respectively.
            As of August, 2015, 303 laboratory-confirmed cases had been reported in T&amp;T by the
            Caribbean Public Health Agency [
            <span ref-type="bibr" rid="pntd.0004199.ref042">
               <sup>
                  <a href="#pntd.0004199.ref042">42</a>
               </sup>
            </span>]. Data on the total number of suspected and probable cases are not available, but
            it may be presumed to be much higher. Without a reliable estimate of total CHIKV incidence
            in the country it is difficult to interpret the observed trend in the number of cases
            detected in this study. The initial increase in number of observed CHIKV cases may
            be a reflection of the rapid expansion of the virus in Trinidad while the moderate
            decline in the proportion of CHIKV cases in November compared to September may reflect
            a decline in incidence or may simply be a result of people becoming less likely to
            seek hospital care as knowledge about the virus and the nonlethal consequences of
            infection became more widespread. The Ministry of Health, for example, launched a
            public education campaign including a “Chikungunya Virus Bus Tour” in August 2014.
            Because of the overlap with Dengue symptomology the general advice was that individuals
            showing symptoms should immediately seek healthcare, however most would probably have
            visited a primary care facility as opposed to hospital emergency care department such
            as the one where this study was conducted.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Our results confirm that the etiologic CHIKV strain in Trinidad belongs to the Asian
            genotype and is nearly a direct descendent of the BVI strain isolated in January 2014
            [
            <span ref-type="bibr" rid="pntd.0004199.ref011">
               <sup>
                  <a href="#pntd.0004199.ref011">11</a>
               </sup>
            </span>], with which it shares over 99.9% nucleotide sequence identity across the two ORFs.
            At the time of writing, no other genomic CHIKV nucleotide sequences from the Americas
            were available so, although likely, a common origin for CHIKV throughout the Caribbean
            cannot be confirmed. Additional sequencing would be required to confirm whether other
            lineages or genotypes have been circulating in the Caribbean as has been the case
            in Brazil (REF). Analysis of intrahost variation in three individuals indicated the
            existence of variants with non-synonymous substitutions at low frequency (&lt;3%). None
            of these corresponded to known adaptive mutations that confer a fitness advantage
            for IOL strains in 
            <i>A</i>. 
            <i>albopictus</i> [
            <span ref-type="bibr" rid="pntd.0004199.ref006">
               <sup>
                  <a href="#pntd.0004199.ref006">6</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pntd.0004199.ref008">
               <sup>
                  <a href="#pntd.0004199.ref008">8</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pntd.0004199.ref040">
               <sup>
                  <a href="#pntd.0004199.ref040">40</a>
               </sup>
            </span>,
            <span ref-type="bibr" rid="pntd.0004199.ref041">
               <sup>
                  <a href="#pntd.0004199.ref041">41</a>
               </sup>
            </span>].
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The data presented arose from a febrile surveillance study for which the initial focus
            was DENV, however with the emergence of CHIKV in the Americas, specific screening
            for CHIKV using RT-qPCR was included. Amongst the 158 participants, only 8 were RT-qPCR
            positive for DENV. However based on the subset of samples (n = 125) that underwent
            both RT-qPCR and complete ELISA testing, a total of 82 (66%) had evidence of a current
            or recent DENV infections. Most of the 8 RT-qPCR confirmed DENV infections were DENV-1,
            which is consistent with the results of PCR-based serotyping of DENV-positive sera
            derived from the Trinidad Public Health Laboratories from 2011–2014 which showed that
            DENV-1 and -4 were the dominant serotypes during this period, with DENV-1 being the
            more common (N. Sahadeo et al., 
            <i>manuscript in preparation</i>).
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Our findings suggest that persons with confirmed CHIKV infection (i.e. RT-qPCR-positive
            for CHIKV) visited the APFC roughly one day earlier than those without RT-qPCR evidence
            of CHIKV. Others have reported significantly earlier presentation of CHIKV infected
            individuals at health care institutions compared to DENV infected individuals [
            <span ref-type="bibr" rid="pntd.0004199.ref043">
               <sup>
                  <a href="#pntd.0004199.ref043">43</a>
               </sup>
            </span>]. This may be a consequence of the more severe joint pain associated with CHIKV infection,
            which in the current study was more commonly reported in individuals with confirmed
            CHIKV infection than in individuals negative for CHIKV and DENV by RT-qPCR, although
            in the case of DENV the difference was not significant.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Several studies have reported leukocytosis as more likely to be present in chikungunya
            cases, and leucopenia and thrombocytopenia being more likely to be present in dengue
            cases [
            <span ref-type="bibr" rid="pntd.0004199.ref043">
               <sup>
                  <a href="#pntd.0004199.ref043">43</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pntd.0004199.ref050">
               <sup>
                  <a href="#pntd.0004199.ref050">50</a>
               </sup>
            </span>]. In particular, thrombocytopenia is considered a hallmark of DENV infection [
            <span ref-type="bibr" rid="pntd.0004199.ref051">
               <sup>
                  <a href="#pntd.0004199.ref051">51</a>
               </sup>
            </span>]. In the current study, bivariable analysis indicated lower mean white blood cell
            counts in individuals with confirmed CHIKV infection compared to individuals with
            AUFI that were negative for CHIKV by RT-qPCR. However the extent of leucopenia in
            the those confirmed CHIKV cases was less than observed in the 88 individuals with
            confirmed DENV infection (i.e. positive for DENV by RT-qPCR), who had significantly
            lower white blood cell and platelet counts than individuals with confirmed CHIKV infection.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">There have been concerns that the arrival of CHIKV would result in life-threatening
            DENV cases being overlooked, as the two are clinically indistinguishable during the
            early stages. This concern was critical because timely confirmatory laboratory testing
            is largely unavailable and rapid “point-of-care” approaches can be costly and unreliable.
            Our data suggest that platelet counts, which are routinely performed at the EWMSC
            and other major healthcare institutions in T&amp;T, are useful for identifying individuals
            who are more likely to have DENV than CHIKV. Our data indicate that in addition to
            being lower than in individuals with confirmed CHIKV infection, the mean platelet
            count in the 8 individuals with confirmed DENV infection was at a level (&lt;150 x 10
            <sup>3</sup>/μl) at which the EWMSC would normally recommend further monitored. Additionally,
            out of those 8 individuals, five had platelet counts below this threshold.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Furthermore, we investigated the feasibility of using combinations of clinical features
            as a means of differentiating between individuals with confirmed and probable CHIKF
            (i.e. with evidence of CHIKV infection by PCR and / or ELISA) and individuals with
            confirmed and recent probable DF. The combination rash, joint pain and white blood
            cell count &lt;7 x10
            <sup>3</sup>/μl was most efficient with a PPV and NPV of 77% and 96% respectively and, a sensitivity
            and specificity of 94% and 84% respectively. Rash, joint pain and white blood cell
            count &lt;7 x10
            <sup>3</sup>/μl together were also most efficient at differentiating CHIKF from other AUFIs. In
            this case the combination had a PPV and NPV of 58% and 95% respectively and, a sensitivity
            and specificity of 88% and 78% respectively. Considering the mean PLT count was lower
            (&lt;150 x 10
            <sup>3</sup>/μl) in individuals positive for DENV by RT-qPCR, a profile using these 4 clinical
            features might be suitable as a means of differentiating between those with CHIKV
            infection and those with DENV infection.
         </p>
         <div id="sec021">
            <h3>Strengths and limitations</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In this study participants were solicited three days per week from amongst individuals
               identified by attending physicians as meeting the criteria of having been febrile
               within the last seven days. We cannot rule out selection bias at this interface and
               so cannot generalise our findings to those who were not interviewed. However as it
               was a study of AUFI and was not restricted to persons with a specific infection, selection
               bias among those who thought they had DENV or CHIKV may be less of a concern. Three
               interviewers were involved in the administration of questionnaires to participants
               of the study. In order to ensure consistency, they were trained in the delivery of
               questions and recording of responses but the possibility of interviewer bias cannot
               be completely excluded.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">This study was performed at a tertiary level public healthcare facility, and the distribution
               of risk factors may be different among those who sought care at public primary care
               facilities (i.e. local health centres) or at other private healthcare facilities.
               Importantly, despite having a small sample size for the comparison of persons with
               and without CHIKV, there was sufficient power to detect several meaningful associations
               on bivariable and multivariable analysis. For the sub-analysis comparing the symptoms
               of those with CHIKV or DENV, the sample size was restricted to 38 and thus analyses
               were underpowered. In spite of this, there appeared to be differences in the symptomatic
               presentation of persons infected with either.
            </p>
         </div>
         <div id="sec022">
            <h3>Conclusion</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In summary, our study confirms the presence of Asian genotype CHIKV in Trinidad and
               identifies symptoms that distinguish individuals with acute fevers who are viraemic
               for CHIKV from those who are CHIKV negative by RT-qPCR, and identifies laboratory
               and clinical features that distinguish between those who are viraemic for CHIKV versus
               DENV. 51 (32.3%) individuals had evidence of CHIKV infection (positive for CHIKV by
               RT-qPCR and / or positive for anti-CHIKV IgM antibodies by ELISA). Of these, only
               30 were positive for CHIKV by RT-qPCR. While all individuals reported being within
               7 days of onset of their symptoms (when viraemia is often detectable) it is possible
               that there were individuals in whom viraemia was already cleared or too low to be
               detected. Additionally, future studies aimed at determining population-level seroprevalence
               will be essential in order to get an accurate estimate of the impact (economic and
               otherwise) of CHIKV on the Trinidad population and to predict the potential impact
               of future outbreaks. It will also be important to monitor the genotypes in circulation
               given the potential for accumulation of adaptive mutations demonstrated by some strains
               [
               <span ref-type="bibr" rid="pntd.0004199.ref006">
                  <sup>
                     <a href="#pntd.0004199.ref006">6</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pntd.0004199.ref008">
                  <sup>
                     <a href="#pntd.0004199.ref008">8</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pntd.0004199.ref040">
                  <sup>
                     <a href="#pntd.0004199.ref040">40</a>
                  </sup>
               </span>,
               <span ref-type="bibr" rid="pntd.0004199.ref041">
                  <sup>
                     <a href="#pntd.0004199.ref041">41</a>
                  </sup>
               </span>].
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The rapid spread of the newly emerged CHIKV in the Caribbean in 2014 and the resulting
               epidemic demonstrate the dramatic effect of the introduction of a “novel” virus on
               a naïve population, especially where vectors are endemic. Public health authorities
               in the Americas are, as of August 2015, on alert for the emergence of Zika virus,
               a flavivirus known to cause widespread outbreaks of dengue-like illness in Africa
               and Asia, after being recently detected in the region for the first time. As commercial
               air travel expands this will continue to facilitate the movement of viruses and ongoing
               monitoring of individuals with AUFIs is essential to rapidly detect the emergence
               of viruses to non-endemic areas.
            </p>
         </div>
      </div>
      <div id="sec023">
         <h2>Supporting Information</h2>
         <div tagxxx="supplementary-material">
            <span class="label">S1 Fig</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Maximum Likelihood phylogeny of CHIKV complete coding region of 74 CHIKV sequences.</div>
               <p>The three major CHIKV genotypes are labelled, as well as the IOL within the ECSA clade.
                  Sequences generated from this study (Genbank accession numbers KR046227 –KR046234)
                  are labelled in in grey. Nodes supported by bootstrap values ≥ 0.95 are labelled accordingly.
               </p>
               <p>(TIF)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S1 Table</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Sequences included in data set for phylogenetic analysis.</div>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S2 Table</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Frequencies of characteristics of the patients of the Adult Primary Care Facility
                  of the Eric Williams Medical Sciences Complex, Trinidad &amp; Tobago (Dec 2013–Nov 2014).
               </div>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S3 Table</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">DENV status based on DENV specific IgM ELISA, IgG ELISA and RT-qPCR for 125 individuals
                  with sufficient testing.
               </div>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S4 Table</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Bivariable associations of patients of the Adult Primary Care Facility of the Eric
                  Williams Medical Sciences Complex, Trinidad &amp; Tobago, (Dec 2013–Nov 2014) with having
                  a confirmed infection
                  <sup>ⱡ</sup> with Chikungunya virus (CHIKV).
               </div>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S5 Table</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Bivariable associations of patients of the Adult Primary Care Facility of the Eric
                  Williams Medical Sciences Complex, Trinidad &amp; Tobago, (Dec 2013–Nov 2014) with having
                  a confirmed infection
                  <sup>ⱡ</sup> with either any Dengue virus serotype (DENV) or Chikungunya virus (CHIKV).
               </div>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S6 Table</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Percentage nucleotide and amino acid identities across the open reading frame of CHIKV
                  sequences from Trinidad and the British Virgin Islands.
               </div>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
      </div>
      <div tagxxx="back">
         <div tagxxx="ack">
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We thank Marlon Knights, Kendall Braithwaite, Janelle Francis, Kristina Eccles, Steve
               Badan and Ria Maharaj-Sooknanan for assistance with phlebotomy, and Giselle Yard for
               organising a location for study participant recruitment and private interviews at
               the APCF.
            </p>
         </div>
         <div tag="ref-list">
            <ul>
               <div tagxxx="title">References</div>
               <li tag="ref">
                  <a name="pntd.0004199.ref001"></a>
                  <span class="label">1</span>
                  <div tagxxx="mixed-citation">Griffin DE. Alphaviruses. In: Knipe D, Howley P, editors. Fields Virology. Philidelphia;
                     2001. pp. 917–962.
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref002"></a>
                  <span class="label">2</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Rampal</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sharda</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Meena</span>
                        <span tagx="given-names">H</span>,
                     </span>. 
                     <span tagx="article-title">Neurological complications in Chikungunya fever</span>. 
                     <span tagx="source">J Assoc Physicians India</span>. 
                     <span tagx="year">2007</span>;
                     <span tagx="volume">55</span>: 
                     <span tagx="fpage">765</span>–
                     <span tagx="lpage">769</span>. 
                     <span class="pub-id'">[18290551]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref003"></a>
                  <span class="label">3</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Brighton</span>
                        <span tagx="given-names">SW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Prozesky</span>
                        <span tagx="given-names">OW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">la Harpe de</span>
                        <span tagx="given-names">AL</span>,
                     </span>. 
                     <span tagx="article-title">Chikungunya virus infection. A retrospective study of 107 cases</span>. 
                     <span tagx="source">S Afr Med J</span>. 
                     <span tagx="year">1983</span>;
                     <span tagx="volume">63</span>: 
                     <span tagx="fpage">313</span>–
                     <span tagx="lpage">315</span>. 
                     <span class="pub-id'">[6298956]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref004"></a>
                  <span class="label">4</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Schwartz</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Albert</span>
                        <span tagx="given-names">ML</span>,
                     </span>. 
                     <span tagx="article-title">Biology and pathogenesis of chikungunya virus</span>. 
                     <span tagx="source">Nat Rev Micro</span>. 
                     <span tagx="year">2010</span>;
                     <span tagx="volume">8</span>: 
                     <span tagx="fpage">491</span>–
                     <span tagx="lpage">500</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1038/nrmicro2368</div>
                     </div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref005"></a>
                  <span class="label">5</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Volk</span>
                        <span tagx="given-names">SM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chen</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tsetsarkin</span>
                        <span tagx="given-names">KA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Adams</span>
                        <span tagx="given-names">AP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Garcia</span>
                        <span tagx="given-names">TI</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sall</span>
                        <span tagx="given-names">AA</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Genome-scale phylogenetic analyses of chikungunya virus reveal independent emergences
                        of recent epidemics and various evolutionary rates
                     </span>. 
                     <span tagx="source">J Virol</span>. 
                     <span tagx="year">2010</span>;
                     <span tagx="volume">84</span>: 
                     <span tagx="fpage">6497</span>–
                     <span tagx="lpage">6504</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1128/JVI.01603-09</div>
                     </div>
                     <span class="pub-id'">[20410280]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref006"></a>
                  <span class="label">6</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Tsetsarkin</span>
                        <span tagx="given-names">KA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vanlandingham</span>
                        <span tagx="given-names">DL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">McGee</span>
                        <span tagx="given-names">CE</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Higgs</span>
                        <span tagx="given-names">S</span>,
                     </span>. 
                     <span tagx="article-title">A single mutation in chikungunya virus affects vector specificity and epidemic potential</span>. 
                     <span tagx="source">PLoS Pathog</span>. 
                     <span tagx="year">2007</span>;
                     <span tagx="volume">3</span>: 
                     <span tagx="fpage">e201</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1371/journal.ppat.0030201</div>
                     </div>
                     <span class="pub-id'">[18069894]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref007"></a>
                  <span class="label">7</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Tsetsarkin</span>
                        <span tagx="given-names">KA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Weaver</span>
                        <span tagx="given-names">SC</span>,
                     </span>. 
                     <span tagx="article-title">Sequential adaptive mutations enhance efficient vector switching by Chikungunya virus
                        and its epidemic emergence
                     </span>. 
                     <span class="name'">
                        <span tagx="surname">Heise</span>
                        <span tagx="given-names">M</span>,
                     </span>, editor. 
                     <span tagx="source">PLoS Pathog</span>. 
                     <span tagx="year">2011</span>;
                     <span tagx="volume">7</span>: 
                     <span tagx="fpage">e1002412</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1371/journal.ppat.1002412</div>
                     </div>
                     <span class="pub-id'">[22174678]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref008"></a>
                  <span class="label">8</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Tsetsarkin</span>
                        <span tagx="given-names">KA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chen</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Yun</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rossi</span>
                        <span tagx="given-names">SL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Plante</span>
                        <span tagx="given-names">KS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Guerbois</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Multi-peaked adaptive landscape for chikungunya virus evolution predicts continued
                        fitness optimization in Aedes albopictus mosquitoes
                     </span>. 
                     <span tagx="source">Nature Communications</span>. 
                     <span tagx="year">2014</span>;
                     <span tagx="volume">5</span>: 
                     <span tagx="fpage">4084</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1038/ncomms5084</div>
                     </div>
                     <span class="pub-id'">[24933611]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref009"></a>
                  <span class="label">9</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Weaver</span>
                        <span tagx="given-names">SC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Reisen</span>
                        <span tagx="given-names">WK</span>,
                     </span>. 
                     <span tagx="article-title">Present and future arboviral threats</span>. 
                     <span tagx="source">Antiviral Res</span>. 
                     <span tagx="year">2010</span>;
                     <span tagx="volume">85</span>: 
                     <span tagx="fpage">328</span>–
                     <span tagx="lpage">345</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/j.antiviral.2009.10.008</div>
                     </div>
                     <span class="pub-id'">[19857523]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref010"></a>
                  <span class="label">10</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Cassadou</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Boucau</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Petit-Sinturel</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Huc</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Leparc-Goffart</span>
                        <span tagx="given-names">I</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ledrans</span>
                        <span tagx="given-names">M</span>,
                     </span>. 
                     <span tagx="article-title">Emergence of chikungunya fever on the French side of Saint Martin island, October
                        to December 2013
                     </span>. 
                     <span tagx="source">Euro Surveill</span>. 
                     <span tagx="year">2014</span>;
                     <span tagx="volume">19</span>.
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref011"></a>
                  <span class="label">11</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Lanciotti</span>
                        <span tagx="given-names">RS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Valadere</span>
                        <span tagx="given-names">AM</span>,
                     </span>. 
                     <span tagx="article-title">Transcontinental movement of Asian genotype chikungunya virus</span>. 
                     <span tagx="source">Emerging Infect Dis</span>. 
                     <span tagx="year">2014</span>;
                     <span tagx="volume">20</span>: 
                     <span tagx="fpage">1400</span>–
                     <span tagx="lpage">1402</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.3201/eid2008.140268</div>
                     </div>
                     <span class="pub-id'">[25076384]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref012"></a>
                  <span class="label">12</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Añez</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Heisey</span>
                        <span tagx="given-names">DA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rios</span>
                        <span tagx="given-names">M</span>,
                     </span>. 
                     <span tagx="article-title">Complete Coding Region Sequence of a Chikungunya Virus Strain Used for Formulation
                        of CBER/FDA RNA Reference Reagents for Nucleic Acid Testing
                     </span>. 
                     <span tagx="source">Genome Announc</span>. American Society for Microbiology; 
                     <span tagx="year">2014</span>;
                     <span tagx="volume">2</span>: e00587–14–e00587–14. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1128/genomeA.00587-14</div>
                     </div>
                     <span class="pub-id'">[25081254]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref013"></a>
                  <span class="label">13</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Fischer</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Staples</span>
                        <span tagx="given-names">JE</span>,
                     </span>, 
                     <span tagx="article-title">Arboviral Diseases Branch, National Center for Emerging and Zoonotic Infectious Diseases,
                        CDC. Notes from the field: chikungunya virus spreads in the Americas—Caribbean and
                        South America, 2013–2014
                     </span>. 
                     <span tagx="source">MMWR Morb Mortal Wkly Rep</span>. 
                     <span tagx="year">2014</span>;
                     <span tagx="volume">63</span>: 
                     <span tagx="fpage">500</span>–
                     <span tagx="lpage">501</span>. 
                     <span class="pub-id'">[24898168]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref014"></a>
                  <span class="label">14</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Johansson</span>
                        <span tagx="given-names">MA</span>,
                     </span>. 
                     <span tagx="article-title">Chikungunya on the move</span>. 
                     <span tagx="source">Trends Parasitol</span>. 
                     <span tagx="year">2015</span>;
                     <span tagx="volume">31</span>: 
                     <span tagx="fpage">43</span>–
                     <span tagx="lpage">45</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/j.pt.2014.12.008</div>
                     </div>
                     <span class="pub-id'">[25649340]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref015"></a>
                  <span class="label">15</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Weaver</span>
                        <span tagx="given-names">SC</span>,
                     </span>. 
                     <span tagx="article-title">PLOS Neglected Tropical Diseases: Arrival of Chikungunya Virus in the New World: Prospects
                        for Spread and Impact on Public Health
                     </span>. 
                     <span tagx="source">PLoS Negl Trop Dis</span>. 
                     <span tagx="year">2014</span>;
                     <span tagx="volume">8</span>: 
                     <span tagx="fpage">e2921</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1371/journal.pntd.0002921</div>
                     </div>
                     <span class="pub-id'">[24967777]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref016"></a>
                  <span class="label">16</span>
                  <div tagxxx="mixed-citation">Pan American Health Organisation. Number of reported cases of Chikungunya fever in
                     the Americas—EW 31 (August 7, 2015). 2015. [Internet].
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref017"></a>
                  <span class="label">17</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Nunes</span>
                        <span tagx="given-names">MRT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Faria</span>
                        <span tagx="given-names">NR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">de Vasconcelos</span>
                        <span tagx="given-names">JM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Golding</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kraemer</span>
                        <span tagx="given-names">MUG</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">de Oliveira</span>
                        <span tagx="given-names">LF</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Emergence and potential for spread of Chikungunya virus in Brazil</span>. 
                     <span tagx="source">BMC Medicine</span>. BioMed Central Ltd; 
                     <span tagx="year">2015</span>;
                     <span tagx="volume">13</span>: 
                     <span tagx="fpage">102</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1186/s12916-015-0348-x</div>
                     </div>
                     <span class="pub-id'">[25976325]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref018"></a>
                  <span class="label">18</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gérardin</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Guernier</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Perrau</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Fianu</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Le Roux</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Grivard</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Estimating Chikungunya prevalence in La Réunion Island outbreak by serosurveys: two
                        methods for two critical times of the epidemic
                     </span>. 
                     <span tagx="source">BMC Infect Dis</span>. BioMed Central Ltd; 
                     <span tagx="year">2008</span>;
                     <span tagx="volume">8</span>: 
                     <span tagx="fpage">99</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1186/1471-2334-8-99</div>
                     </div>
                     <span class="pub-id'">[18662384]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref019"></a>
                  <span class="label">19</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Vijayakumar</span>
                        <span tagx="given-names">KP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Anish</span>
                        <span tagx="given-names">TSN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">George</span>
                        <span tagx="given-names">B</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lawrence</span>
                        <span tagx="given-names">T</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Muthukkutty</span>
                        <span tagx="given-names">SC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ramachandran</span>
                        <span tagx="given-names">R</span>,
                     </span>. 
                     <span tagx="article-title">Clinical Profile of Chikungunya Patients during the Epidemic of 2007 in Kerala, India</span>. 
                     <span tagx="source">Journal of Global Infectious Diseases</span>. Medknow Publications; 
                     <span tagx="year">2011</span>;
                     <span tagx="volume">3</span>: 
                     <span tagx="fpage">221</span>–
                     <span tagx="lpage">226</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.4103/0974-777X.83526</div>
                     </div>
                     <span class="pub-id'">[21887052]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref020"></a>
                  <span class="label">20</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Weaver</span>
                        <span tagx="given-names">SC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lecuit</span>
                        <span tagx="given-names">M</span>,
                     </span>. 
                     <span tagx="article-title">Chikungunya virus and the global spread of a mosquito-borne disease</span>. 
                     <span tagx="source">N Engl J Med</span>. 
                     <span tagx="year">2015</span>;
                     <span tagx="volume">372</span>: 
                     <span tagx="fpage">1231</span>–
                     <span tagx="lpage">1239</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1056/NEJMra1406035</div>
                     </div>
                     <span class="pub-id'">[25806915]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref021"></a>
                  <span class="label">21</span>
                  <div tagxxx="mixed-citation">Pan American Health Organisation. Number of reported cases of Chikungunya Fever in
                     the Americas—EW 30 (July 25, 2014) [Internet].
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref022"></a>
                  <span class="label">22</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Waggoner</span>
                        <span tagx="given-names">JJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Abeynayake</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sahoo</span>
                        <span tagx="given-names">MK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gresh</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tellez</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gonzalez</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Single-reaction, multiplex, real-time rt-PCR for the detection, quantitation, and
                        serotyping of dengue viruses
                     </span>. 
                     <span tagx="source">PLoS Negl Trop Dis</span>. 
                     <span tagx="year">2013</span>;
                     <span tagx="volume">7</span>: 
                     <span tagx="fpage">e2116</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1371/journal.pntd.0002116</div>
                     </div>
                     <span class="pub-id'">[23638191]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref023"></a>
                  <span class="label">23</span>
                  <div tagxxx="mixed-citation">
                     <span tagx="article-title">Chikungunya fever in travelers returning to Europe from the Indian Ocean region, 2006</span>. 
                     <span tagx="year">2008</span>;
                     <span tagx="volume">14</span>: 
                     <span tagx="fpage">416</span>–
                     <span tagx="lpage">422</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.3201/eid1403.070906</div>
                     </div>
                     <span class="pub-id'">[18325256]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref024"></a>
                  <span class="label">24</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Simpson</span>
                        <span tagx="given-names">JT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Wong</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Jackman</span>
                        <span tagx="given-names">SD</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Schein</span>
                        <span tagx="given-names">JE</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Jones</span>
                        <span tagx="given-names">SJM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Birol</span>
                        <span tagx="given-names">I</span>,
                     </span>. 
                     <span tagx="article-title">ABySS: a parallel assembler for short read sequence data</span>. 
                     <span tagx="source">Genome Res</span>. Cold Spring Harbor Lab; 
                     <span tagx="year">2009</span>;
                     <span tagx="volume">19</span>: 
                     <span tagx="fpage">1117</span>–
                     <span tagx="lpage">1123</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1101/gr.089532.108</div>
                     </div>
                     <span class="pub-id'">[19251739]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref025"></a>
                  <span class="label">25</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Langmead</span>
                        <span tagx="given-names">B</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Salzberg</span>
                        <span tagx="given-names">SL</span>,
                     </span>. 
                     <span tagx="article-title">Fast gapped-read alignment with Bowtie 2</span>. 
                     <span tagx="source">Nat Methods</span>. 
                     <span tagx="year">2012</span>;
                     <span tagx="volume">9</span>: 
                     <span tagx="fpage">357</span>–
                     <span tagx="lpage">359</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1038/nmeth.1923</div>
                     </div>
                     <span class="pub-id'">[22388286]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref026"></a>
                  <span class="label">26</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Robinson</span>
                        <span tagx="given-names">JT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Thorvaldsdóttir</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Winckler</span>
                        <span tagx="given-names">W</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Guttman</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lander</span>
                        <span tagx="given-names">ES</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Getz</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Integrative genomics viewer</span>. 
                     <span tagx="source">Nat Biotechnol</span>. Nature Publishing Group; 
                     <span tagx="year">2011</span>;
                     <span tagx="volume">29</span>: 
                     <span tagx="fpage">24</span>–
                     <span tagx="lpage">26</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1038/nbt.1754</div>
                     </div>
                     <span class="pub-id'">[21221095]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref027"></a>
                  <span class="label">27</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Larkin</span>
                        <span tagx="given-names">MA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Blackshields</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Brown</span>
                        <span tagx="given-names">NP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chenna</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">McGettigan</span>
                        <span tagx="given-names">PA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">McWilliam</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Clustal W and Clustal X version 2.0</span>. 
                     <span tagx="source">Bioinformatics</span>. Oxford University Press; 
                     <span tagx="year">2007</span>;
                     <span tagx="volume">23</span>: 
                     <span tagx="fpage">2947</span>–
                     <span tagx="lpage">2948</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1093/bioinformatics/btm404</div>
                     </div>
                     <span class="pub-id'">[17846036]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref028"></a>
                  <span class="label">28</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Kearse</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Moir</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Wilson</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Stones-Havas</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Cheung</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sturrock</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Geneious Basic: an integrated and extendable desktop software platform for the organization
                        and analysis of sequence data
                     </span>. 
                     <span tagx="source">Bioinformatics</span>. Oxford University Press; 
                     <span tagx="year">2012</span>;
                     <span tagx="volume">28</span>: 
                     <span tagx="fpage">1647</span>–
                     <span tagx="lpage">1649</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1093/bioinformatics/bts199</div>
                     </div>
                     <span class="pub-id'">[22543367]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref029"></a>
                  <span class="label">29</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Kosakovsky Pond</span>
                        <span tagx="given-names">SL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Posada</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gravenor</span>
                        <span tagx="given-names">MB</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Woelk</span>
                        <span tagx="given-names">CH</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Frost</span>
                        <span tagx="given-names">SDW</span>,
                     </span>. 
                     <span tagx="article-title">GARD: a genetic algorithm for recombination detection</span>. 
                     <span tagx="source">Bioinformatics</span>. Oxford University Press; 
                     <span tagx="year">2006</span>;
                     <span tagx="volume">22</span>: 
                     <span tagx="fpage">3096</span>–
                     <span tagx="lpage">3098</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1093/bioinformatics/btl474</div>
                     </div>
                     <span class="pub-id'">[17110367]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref030"></a>
                  <span class="label">30</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Pond</span>
                        <span tagx="given-names">SLK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Frost</span>
                        <span tagx="given-names">SDW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Muse</span>
                        <span tagx="given-names">SV</span>,
                     </span>. 
                     <span tagx="article-title">HyPhy: hypothesis testing using phylogenies</span>. 
                     <span tagx="source">Bioinformatics</span>. Oxford University Press; 
                     <span tagx="year">2005</span>;
                     <span tagx="volume">21</span>: 
                     <span tagx="fpage">676</span>–
                     <span tagx="lpage">679</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1093/bioinformatics/bti079</div>
                     </div>
                     <span class="pub-id'">[15509596]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref031"></a>
                  <span class="label">31</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Guindon</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gascuel</span>
                        <span tagx="given-names">O</span>,
                     </span>. 
                     <span tagx="article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood</span>. 
                     <span tagx="source">Syst Biol</span>. 
                     <span tagx="year">2003</span>;
                     <span tagx="volume">52</span>: 
                     <span tagx="fpage">696</span>–
                     <span tagx="lpage">704</span>. 
                     <span class="pub-id'">[14530136]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref032"></a>
                  <span class="label">32</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Darriba</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Taboada</span>
                        <span tagx="given-names">GL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Doallo</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Posada</span>
                        <span tagx="given-names">D</span>,
                     </span>. 
                     <span tagx="article-title">jModelTest 2: more models, new heuristics and parallel computing</span>. 
                     <span tagx="source">Nat Methods</span>. 
                     <span tagx="year">2012</span>;
                     <span tagx="volume">9</span>: 
                     <span tagx="fpage">772</span>–
                     <span tagx="lpage">772</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1038/nmeth.2109</div>
                     </div>
                     <span class="pub-id'">[22847109]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref033"></a>
                  <span class="label">33</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Guindon</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Dufayard</span>
                        <span tagx="given-names">J-F</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lefort</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Anisimova</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hordijk</span>
                        <span tagx="given-names">W</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gascuel</span>
                        <span tagx="given-names">O</span>,
                     </span>. 
                     <span tagx="article-title">New Algorithms and Methods to Estimate Maximum-Likelihood Phylogenies: Assessing the
                        Performance of PhyML 3.0
                     </span>. 
                     <span tagx="source">Syst Biol</span>. Oxford University Press; 
                     <span tagx="year">2010</span>;
                     <span tagx="volume">59</span>: 
                     <span tagx="fpage">307</span>–
                     <span tagx="lpage">321</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1093/sysbio/syq010</div>
                     </div>
                     <span class="pub-id'">[20525638]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref034"></a>
                  <span class="label">34</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Drummond</span>
                        <span tagx="given-names">AJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Suchard</span>
                        <span tagx="given-names">MA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Xie</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rambaut</span>
                        <span tagx="given-names">A</span>,
                     </span>. 
                     <span tagx="article-title">Bayesian phylogenetics with BEAUti and the BEAST 1.7</span>. 
                     <span tagx="source">Mol Biol Evol</span>. 
                     <span tagx="year">2012</span>;: 
                     <span tagx="fpage">3</span>–
                     <span tagx="lpage">6</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1093/molbev/mss075</div>
                     </div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref035"></a>
                  <span class="label">35</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Drummond</span>
                        <span tagx="given-names">AJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ho</span>
                        <span tagx="given-names">SYW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Phillips</span>
                        <span tagx="given-names">MJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rambaut</span>
                        <span tagx="given-names">A</span>,
                     </span>. 
                     <span tagx="article-title">Relaxed phylogenetics and dating with confidence</span>. 
                     <span tagx="source">PLoS Biol</span>. 
                     <span tagx="year">2006</span>;
                     <span tagx="volume">4</span>: 
                     <span tagx="fpage">e88</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1371/journal.pbio.0040088</div>
                     </div>
                     <span class="pub-id'">[16683862]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref036"></a>
                  <span class="label">36</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Drummond</span>
                        <span tagx="given-names">AJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rambaut</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Shapiro</span>
                        <span tagx="given-names">B</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Pybus</span>
                        <span tagx="given-names">OG</span>,
                     </span>. 
                     <span tagx="article-title">Bayesian coalescent inference of past population dynamics from molecular sequences</span>. 
                     <span tagx="source">Mol Biol Evol</span>. Oxford University Press; 
                     <span tagx="year">2005</span>;
                     <span tagx="volume">22</span>: 
                     <span tagx="fpage">1185</span>–
                     <span tagx="lpage">1192</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1093/molbev/msi103</div>
                     </div>
                     <span class="pub-id'">[15703244]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref037"></a>
                  <span class="label">37</span>
                  <div tagxxx="mixed-citation">Rambaut A, Suchard MA, Xie D, Drummond AJ. Tracer v1.6 [Internet]. Available: 
                     <div tagxxx="ext-link">http://beast.bio.ed.ac.uk/Tracer</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref038"></a>
                  <span class="label">38</span>
                  <div tagxxx="mixed-citation">Caribbean Public Health Agency. Chikungunya Update #25 July 21 2014 [Internet]. 2014
                     Jul pp. 1–2. Available: 
                     <div tagxxx="ext-link">http://carpha.org/What-We-Do/Public-Health-Activitihttp://carpha.org/DesktopModules/Bring2mind/DMX/Download.aspx?EntryId=1303&amp;PortalId=0&amp;DownloadMethod=attachmentes/Chikungunya</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref039"></a>
                  <span class="label">39</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Leparc-Goffart</span>
                        <span tagx="given-names">I</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nougairede</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Cassadou</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Prat</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">de Lamballerie</span>
                        <span tagx="given-names">X</span>,
                     </span>. 
                     <span tagx="article-title">Chikungunya in the Americas</span>. 
                     <span tagx="source">Lancet</span>. Centers for Disease Control and Prevention; 
                     <span tagx="year">2014</span>;
                     <span tagx="volume">383</span>: 
                     <span tagx="fpage">514</span> EP––1402. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.3201/eid2008.140268</div>
                     </div>
                     <span class="pub-id'">[24506907]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref040"></a>
                  <span class="label">40</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Vazeille</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Moutailler</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Coudrier</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rousseaux</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Khun</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Huerre</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Two Chikungunya isolates from the outbreak of La Reunion (Indian Ocean) exhibit different
                        patterns of infection in the mosquito, Aedes albopictus
                     </span>. 
                     <span tagx="source">PLoS ONE</span>. 
                     <span tagx="year">2007</span>;
                     <span tagx="volume">2</span>: 
                     <span tagx="fpage">e1168</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1371/journal.pone.0001168</div>
                     </div>
                     <span class="pub-id'">[18000540]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref041"></a>
                  <span class="label">41</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Tsetsarkin</span>
                        <span tagx="given-names">KA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chen</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Leal</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Forrester</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Higgs</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Huang</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Chikungunya virus emergence is constrained in Asia by lineage-specific adaptive landscapes</span>. 
                     <span tagx="source">Proc Natl Acad Sci USA</span>. 
                     <span tagx="year">2011</span>;
                     <span tagx="volume">108</span>: 
                     <span tagx="fpage">7872</span>–
                     <span tagx="lpage">7877</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1073/pnas.1018344108</div>
                     </div>
                     <span class="pub-id'">[21518887]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref042"></a>
                  <span class="label">42</span>
                  <div tagxxx="mixed-citation">Caribbean Public Health Agency. Chikungunya Update #54 February 18 2015–4 [Internet].
                     18 Feb 2015 [cited 22 Feb 2015] pp. 1–2. Available: 
                     <div tagxxx="ext-link">http://carpha.org/DesktopModules/Bring2mind/DMX/Download.aspx?EntryId=1457&amp;PortalId=0&amp;DownloadMethod=attachment</div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref043"></a>
                  <span class="label">43</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Reller</span>
                        <span tagx="given-names">ME</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Akoroda</span>
                        <span tagx="given-names">U</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nagahawatte</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Devasiri</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kodikaarachchi</span>
                        <span tagx="given-names">W</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Strouse</span>
                        <span tagx="given-names">JJ</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Chikungunya as a cause of acute febrile illness in southern Sri Lanka</span>. 
                     <span class="name'">
                        <span tagx="surname">Coffey</span>
                        <span tagx="given-names">LL</span>,
                     </span>, editor. 
                     <span tagx="source">PLoS ONE</span>. 
                     <span tagx="year">2013</span>;
                     <span tagx="volume">8</span>: 
                     <span tagx="fpage">e82259</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1371/journal.pone.0082259</div>
                     </div>
                     <span class="pub-id'">[24312651]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref044"></a>
                  <span class="label">44</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Lee</span>
                        <span tagx="given-names">VJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chow</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Zheng</span>
                        <span tagx="given-names">X</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Carrasco</span>
                        <span tagx="given-names">LR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Cook</span>
                        <span tagx="given-names">AR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lye</span>
                        <span tagx="given-names">DC</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Simple clinical and laboratory predictors of Chikungunya versus dengue infections
                        in adults
                     </span>. 
                     <span class="name'">
                        <span tagx="surname">Powers</span>
                        <span tagx="given-names">AM</span>,
                     </span>, editor. 
                     <span tagx="source">PLoS Negl Trop Dis</span>. 
                     <span tagx="year">2012</span>;
                     <span tagx="volume">6</span>: 
                     <span tagx="fpage">e1786</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1371/journal.pntd.0001786</div>
                     </div>
                     <span class="pub-id'">[23029573]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref045"></a>
                  <span class="label">45</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Halstead</span>
                        <span tagx="given-names">SB</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Udomsakdi</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Singharaj</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nisalak</span>
                        <span tagx="given-names">A</span>,
                     </span>. 
                     <span tagx="article-title">Dengue chikungunya virus infection in man in Thailand, 1962–1964. 3. Clinical, epidemiologic,
                        and virologic observations on disease in non-indigenous white persons
                     </span>. 
                     <span tagx="source">Am J Trop Med Hyg</span>. 
                     <span tagx="year">1969</span>;
                     <span tagx="volume">18</span>: 
                     <span tagx="fpage">984</span>–
                     <span tagx="lpage">996</span>. 
                     <span class="pub-id'">[5355244]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref046"></a>
                  <span class="label">46</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Halstead</span>
                        <span tagx="given-names">SB</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nimmannitya</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Margiotta</span>
                        <span tagx="given-names">MR</span>,
                     </span>. 
                     <span tagx="article-title">Dengue d chikungunya virus infection in man in Thailand, 1962–1964. II. Observations
                        on disease in outpatients
                     </span>. 
                     <span tagx="source">Am J Trop Med Hyg</span>. 
                     <span tagx="year">1969</span>;
                     <span tagx="volume">18</span>: 
                     <span tagx="fpage">972</span>–
                     <span tagx="lpage">983</span>. 
                     <span class="pub-id'">[5355243]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref047"></a>
                  <span class="label">47</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Nimmannitya</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Halstead</span>
                        <span tagx="given-names">SB</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Cohen</span>
                        <span tagx="given-names">SN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Margiotta</span>
                        <span tagx="given-names">MR</span>,
                     </span>. 
                     <span tagx="article-title">Dengue and chikungunya virus infection in man in Thailand, 1962–1964. I. Observations
                        on hospitalized patients with hemorrhagic fever
                     </span>. 
                     <span tagx="source">Am J Trop Med Hyg</span>. 
                     <span tagx="year">1969</span>;
                     <span tagx="volume">18</span>: 
                     <span tagx="fpage">954</span>–
                     <span tagx="lpage">971</span>. 
                     <span class="pub-id'">[5355242]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref048"></a>
                  <span class="label">48</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Hochedez</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Canestri</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Guihot</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Brichler</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Bricaire</span>
                        <span tagx="given-names">F</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Caumes</span>
                        <span tagx="given-names">E</span>,
                     </span>. 
                     <span tagx="article-title">Management of travelers with fever and exanthema, notably dengue and chikungunya infections</span>. 
                     <span tagx="source">Am J Trop Med Hyg</span>. 
                     <span tagx="year">2008</span>;
                     <span tagx="volume">78</span>: 
                     <span tagx="fpage">710</span>–
                     <span tagx="lpage">713</span>. 
                     <span class="pub-id'">[18458301]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref049"></a>
                  <span class="label">49</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Kularatne</span>
                        <span tagx="given-names">SAM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gihan</span>
                        <span tagx="given-names">MC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Weerasinghe</span>
                        <span tagx="given-names">SC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gunasena</span>
                        <span tagx="given-names">S</span>,
                     </span>. 
                     <span tagx="article-title">Concurrent outbreaks of Chikungunya and Dengue fever in Kandy, Sri Lanka, 2006–07:
                        a comparative analysis of clinical and laboratory features
                     </span>. 
                     <span tagx="source">Postgrad Med J</span>. The Fellowship of Postgraduate Medicine; 
                     <span tagx="year">2009</span>;
                     <span tagx="volume">85</span>: 
                     <span tagx="fpage">342</span>–
                     <span tagx="lpage">346</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1136/pgmj.2007.066746</div>
                     </div>
                     <span class="pub-id'">[19581242]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref050"></a>
                  <span class="label">50</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Laoprasopwattana</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kaewjungwad</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Jarumanokul</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Geater</span>
                        <span tagx="given-names">A</span>,
                     </span>. 
                     <span tagx="article-title">Differential diagnosis of Chikungunya, dengue viral infection and other acute febrile
                        illnesses in children
                     </span>. 
                     <span tagx="source">Pediatr Infect Dis J</span>. 
                     <span tagx="year">2012</span>;
                     <span tagx="volume">31</span>: 
                     <span tagx="fpage">459</span>–
                     <span tagx="lpage">463</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1097/INF.0b013e31824bb06d</div>
                     </div>
                     <span class="pub-id'">[22301475]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0004199.ref051"></a>
                  <span class="label">51</span>
                  <div tagxxx="mixed-citation">World Health Organisation. Dengue: Guidelines for diagnosis, treatment, prevention
                     and control—New edition. 2009. pp. 1–160.
                  </div>
               </li>
            </ul>
         </div>
      </div>
   </div>
</html>