<?xml version="1.0" encoding="UTF-8"?><div tagx="article">
   <head>
      <meta content="Public Library of Science" name="citation_publisher"></meta>
      <meta content="10.1371/journal.pone.0116215" name="citation_doi"></meta>
      <meta content="In Silico Analysis of Usher Encoding Genes in Klebsiella pneumoniae and Characterization of Their Role in Adhesion and Colonization" name="citation_title"></meta>
      <title>In Silico Analysis of Usher Encoding Genes in Klebsiella pneumoniae and Characterization
         of Their Role in Adhesion and Colonization
      </title>
      <div tagx="contrib-group">
         <meta content="Fida Khater" name="citation_author"></meta>
         <meta content="1" name="citation_author_institution_ref"></meta>
         <meta content="Damien Balestrino" name="citation_author"></meta>
         <meta content="1" name="citation_author_institution_ref"></meta>
         <meta content="Nicolas Charbonnel" name="citation_author"></meta>
         <meta content="1" name="citation_author_institution_ref"></meta>
         <meta content="Jean François Dufayard" name="citation_author"></meta>
         <meta content="2" name="citation_author_institution_ref"></meta>
         <meta content="Sylvain Brisse" name="citation_author"></meta>
         <meta content="3" name="citation_author_institution_ref"></meta>
         <meta content="Christiane Forestier" name="citation_author"></meta>
         <meta content="1" name="citation_author_institution_ref"></meta>
      </div>
      <div tagx="contrib-group"></div>
      <meta content="Christiane.forestier@udamail.fr" name="email"></meta>
      <meta content="Competing Interests: The authors would like to add that Christiane Forestier is a PLOS ONE Editorial Board member, but this does not alter their adherence to PLOS ONE Editorial policies and criteria." name="conflict"></meta>
      <meta content="Conceived and designed the experiments: FK DB JFD SB CF. Performed the experiments: FK NC. Analyzed the data: FK DB JFD CF. Contributed reagents/materials/analysis tools: FK SB. Wrote the paper: FK DB JFD SB CF." name="contribs"></meta>
      <meta content="2015-3-9" name="citation_pubdate"></meta>
      <meta content="Copyright 2015: Khater et al" name="citation_copyright"></meta>
      <meta content="Licence This is an open access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited" name="citation_licence"></meta>
      <meta content="The authors have no support or funding to report." name="citation_funding"></meta>
      <meta content="24" name="citation_pagecount"></meta>
      <meta content="Data AvailabilityThe data may be found in the EMBL database, the INSDC databases (Assembly_name: KPLM21; Study_ID: PRJEB7075; Sample_ID: ERS537769 | Contigs: CCVM01000001-CCVM01000154)." name="citation_data"></meta>
   </head>
   <div id="abstract" tag="abstract">
      <p>Chaperone/usher (CU) assembly pathway is used by a wide range of 
         <i>Enterobacteriaceae</i> to assemble adhesive surface structures called pili or fimbriae that play a role
         in bacteria-host cell interactions. 
         <i>In silico</i> analysis revealed that the genome of 
         <i>Klebsiella pneumoniae</i> LM21 harbors eight chromosomal CU loci belonging to γκп and ϭ clusters. Of these,
         only two correspond to previously described operons, namely type 1 and type 3-encoding
         operons. Isogenic usher deletion mutants of 
         <i>K</i>. 
         <i>pneumoniae</i> LM21 were constructed for each locus and their role in adhesion to animal (Intestine
         407) and plant (
         <i>Arabidopsis thaliana</i>) cells, biofilm formation and murine intestinal colonization was investigated. Type
         3 pili usher deleted mutant was impaired in all assays, whereas type 1 pili usher
         deleted mutant only showed attenuation in adhesion to plant cells and in intestinal
         colonization. The LM21Δ
         <i>kpjC</i> mutant was impaired in its capacity to adhere to 
         <i>Arabidopsis</i> cells and to colonize the murine intestine, either alone or in co-inoculation experiments.
         Deletion of LM21
         <i>kpgC</i> induced a significant decrease in biofilm formation, in adhesion to animal cells
         and in colonization of the mice intestine. The LM21∆
         <i>kpaC</i> and LM21∆
         <i>kpeC</i> mutants were only attenuated in biofilm formation and the adhesion abilities to 
         <i>Arabidopsis</i> cells, respectively. No clear 
         <i>in vitro</i> or 
         <i>in vivo</i> effect was observed for LM21∆
         <i>kpbC</i> and LM21∆
         <i>kpdC</i> mutants. The multiplicity of CU loci in 
         <i>K</i>. 
         <i>pneumoniae</i> genome and their specific adhesion pattern probably reflect the ability of the bacteria
         to adhere to different substrates in its diverse ecological niches.
      </p>
   </div>
   <div id="sec001">
      <h2>Introduction</h2>
      <p>Bacterial adhesion to epithelial cells and abiotic surfaces is frequently mediated
         by diverse surface proteinaceous appendages referred to as adhesins. Gram-negative
         bacteria develop several fimbrial adhesins that are proteinaceous non-flagellar filaments
         with thin hair-like extension on the bacterial cell surface [
         <span ref-type="bibr" rid="pone.0116215.ref001">1</span>]. They are assembled by dedicated secretion systems and are composed primarily of
         the major repeating subunit protein with minor-subunit proteins including the adhesion
         subunits that enable the bacteria to specifically target a cell component or a surface
         [
         <span ref-type="bibr" rid="pone.0116215.ref002">2</span>]. Studies on the biochemistry and genetics of fimbrial biosynthesis have given rise
         to a nomenclature that distinguishes between fimbriae on the basis of their assembly
         mechanism [
         <span ref-type="bibr" rid="pone.0116215.ref003">3</span>]. The chaperone/usher (CU)-dependent pathway represents the most abundant secretory
         pathway among Gram-negative bacteria [
         <span ref-type="bibr" rid="pone.0116215.ref003">3</span>]. It is composed of a chaperone periplasmic protein and an outer membrane protein
         named the usher. The chaperone prevents self-aggregation of the fimbriae subunits
         and directs them to the usher, which in turn is involved in the secretion and correct
         assembly of the external fimbrial subunits [
         <span ref-type="bibr" rid="pone.0116215.ref003">3</span>]. Genes encoding the CU pathway are located on both chromosomal and plasmid molecules
         and are clustered with similar organization among different bacteria: an upstream
         region containing regulatory genes and a single downstream operon containing the required
         structural and assembly components. In addition, multiple CU pathways may be present
         in a single bacterial genome, which presumably confers the ability to adhere to a
         variety of different receptors and surfaces [
         <span ref-type="bibr" rid="pone.0116215.ref004">4</span>–
         <span ref-type="bibr" rid="pone.0116215.ref006">6</span>]. Expression of the CU gene clusters is typically highly regulated, subject to phase
         variation and responsive to environmental cues [
         <span ref-type="bibr" rid="pone.0116215.ref007">7</span>]. This cross-talk probably ensures that each bacterium does not express all pilus
         types at a given time, enabling the control of adhesive specificity.
      </p>
      <p>The CU family has been described among members of the beta-proteobacteria and in cyanobacteria
         [
         <span ref-type="bibr" rid="pone.0116215.ref003">3</span>,
         <span ref-type="bibr" rid="pone.0116215.ref008">8</span>], but is mostly prevalent among enteric members of the γ-proteobacteria, including
         
         <i>Escherichia coli</i>, 
         <i>Salmonella</i> and 
         <i>Yersinia</i>. 
         <i>Klebsiella pneumoniae</i> is a pathogenic 
         <i>Enterobacteriaceae</i> that commonly causes nosocomial, increasingly multidrug resistant infections and
         is also emerging as a community pathogen [
         <span ref-type="bibr" rid="pone.0116215.ref002">2</span>,
         <span ref-type="bibr" rid="pone.0116215.ref009">9</span>,
         <span ref-type="bibr" rid="pone.0116215.ref010">10</span>]. Two classical CU fimbriae have been described in 
         <i>Klebsiella</i>, type 1 and type 3-pili. By binding to mannosylated glycoproteins, type 1 pili allow
         bacterial adhesion to uroepithelial cells and the development of cystitis [
         <span ref-type="bibr" rid="pone.0116215.ref011">11</span>–
         <span ref-type="bibr" rid="pone.0116215.ref013">13</span>]. Type 3 fimbriae mediate adhesion to several cell types 
         <i>in vitro</i> such as tracheal epithelial cells, renal tubular cells, extracellular matrix proteins
         and the membrane of human lung tissue [
         <span ref-type="bibr" rid="pone.0116215.ref014">14</span>–
         <span ref-type="bibr" rid="pone.0116215.ref016">16</span>]. In addition, both fimbriae have been associated with biofilm formation in 
         <i>K</i>. 
         <i>pneumoniae</i> [
         <span ref-type="bibr" rid="pone.0116215.ref002">2</span>,
         <span ref-type="bibr" rid="pone.0116215.ref017">17</span>]. Recently Wu et al. [
         <span ref-type="bibr" rid="pone.0116215.ref018">18</span>] described nine fimbrial loci found in 
         <i>K</i>. 
         <i>pneumoniae</i> NTUH- K2044: 
         <i>fim</i>, 
         <i>mrk</i>, and seven other fimbrial loci called 
         <i>kpa</i> to 
         <i>kpg</i>. Apart from the NTUH-K2044 
         <i>kpc</i> locus, no studies have investigated the 
         <i>in vitro</i> and/or 
         <i>in vivo</i> role of the other 
         <i>K</i>. 
         <i>pneumoniae</i> accessory fimbrial potential operons [
         <span ref-type="bibr" rid="pone.0116215.ref018">18</span>].
      </p>
      <p>Given the recent increase in 
         <i>K</i>. 
         <i>pneumoniae</i> clinical importance and its well recognized diversity of niches, we assessed the
         presence of CU-like genes in this pathogen by searching for genes encoding putative
         fimbrial usher proteins. We analyzed their distribution, genetic conservation and
         genetic location. We used a reverse genetics approach to further investigate adhesion
         and colonization phenotype associated with eight LM21 usher operon candidates and
         
         <i>in vitro</i> and 
         <i>in vivo</i> models.
      </p>
   </div>
   <div id="sec002">
      <h2>Material and Methods</h2>
      <div id="sec003">
         <h3>Identification of Chaperone-Usher loci in 
            <i>K</i>. 
            <i>pneumoniae</i> LM21
         </h3>
         <p>
            <i>K</i>. 
            <i>pneumoniae LM21</i> was isolated from a cutaneous wound of a patient hospitalized in a intensive care
            unit of the teaching hospital of Clermont-Ferrand. Its sequencing was performed by
            GATC Biotech (Konstanz, Germany) using Illumina technology and a 2×100 nucleotide
            (nt) paired-end strategy. All reads were pre-processed to remove low quality or artefactual
            nucleotides. First, all nucleotides occurring at 5' and 3' ends and supported by a
            Phred quality score &lt; 30 were trimmed off using Sickle (
            <div tagx="ext-link">http://www.github.com/najoshi/sickle</div>). Second, contaminant oligonucleotides (i.e. library adaptors) were detected and
            trimmed off using AlienTrimmer [
            <span ref-type="bibr" rid="pone.0116215.ref019">19</span>]. Third, reads shorter than 60 nt after the above cleaning steps were discarded,
            as well as those containing more than 20% nucleotides with Phred score &lt; 30. Resulting
            reads were assembled using clc_assembler from the CLC Genomics Workbench analysis
            package (
            <div tagx="ext-link">http://www.clcbio.com/products/clc-genomics-workbench/</div>). Contigs were reordered and reoriented, using as reference the genomic sequence
            of strain MGH 78578, with Mauve Contig Mover [
            <span ref-type="bibr" rid="pone.0116215.ref020">20</span>]. Obvious contaminant contigs were discarded. 154 remaining contigs (total genome
            size: 5,509,751 nt; N50, 104,592 nt) were subsequently imported into the Microscope
            database system [
            <span ref-type="bibr" rid="pone.0116215.ref021">21</span>]. The genome was annotated automatically within the MicroScope platform and manually
            visualized using the Magnifying Genomes (MaGe) web interface [
            <span ref-type="bibr" rid="pone.0116215.ref022">22</span>,
            <span ref-type="bibr" rid="pone.0116215.ref023">23</span>]. The LM21 genome sequence was deposited in the EMBL database and is available from
            the INSDC databases (Assembly_name: KPLM21; Study_ID: PRJEB7075; Sample_ID: ERS537769
            | Contigs: CCVM01000001-CCVM01000154).
         </p>
         <p>The NCBI Blast 2.28+ program was used to look for the presence of usher-like sequences
            in 
            <i>K</i>. 
            <i>pneumoniae</i> strain LM21. Usher amino acid sequences annotated in NCBI were used as initial BLASTp
            queries to look for the presence of CU in 
            <i>K</i>. 
            <i>pneumoniae</i> LM21. BLASTp searches were performed using the BLOSUM62 matrix and an E-value cut-off
            score of 0.1. Newly identified proteins with a reported E-value of 0 were retained,
            whereas hits with an E-value &gt;0 were screened for the presence of an usher protein
            family domain (PFAM00577) and/or flanking chaperone (PF00345, PF02753 or COG3121)
            for encoding genes before they were added to the usher query list. The NCBI Conserved
            Domain Database (CDD) component of the NCBI Entrez query retrieval system was used
            to examine amino-acid sequences for conserved domain [
            <span ref-type="bibr" rid="pone.0116215.ref024">24</span>].
         </p>
         <p>After each BLASTp run, the updated usher query database was used to re-probe the genome
            sequences until no new sequences were found. All putative usher amino-acid sequences
            encoded by LM21 
            <i>K</i>. 
            <i>pneumoniae</i> genome were downloaded from MaGe database. They are listed in 
            <span ref-type="supplementary-material" rid="pone.0116215.s001">S1 Table</span>. The presence of the eight usher encoded genes predicted by 
            <i>in silico</i> analysis was verified by PCRs using primers listed in 
            <span ref-type="table" rid="pone.0116215.t001">Table 1</span> (P1 to P16).
         </p>
         <div tagx="table-wrap">
            <div tagx="object-id">10.1371/journal.pone.0116215.t001</div>
            <div tagx="label">Table 1</div>
            <caption>
               <div tagx="title">Primers used in this study.</div>
            </caption>
            <div tagx="alternatives">
               <div tagx="graphic"></div>
               <table frame="hsides" rules="groups">
                  <colgroup span="1">
                     <col align="left" valign="middle" span="1"></col>
                     <col align="left" valign="middle" span="1"></col>
                     <col align="left" valign="middle" span="1"></col>
                     <col align="left" valign="middle" span="1"></col>
                     <col align="left" valign="middle" span="1"></col>
                  </colgroup>
                  <thead>
                     <tr>
                        <div tagx="th">
                           <i>Num</i>
                        </div>
                        <div tagx="th">
                           <i>Primers Name</i>
                        </div>
                        <div tagx="th">Oligonucleotide sequence (5'- 3')</div>
                        <div tagx="th">PCR product size (bp)</div>
                        <div tagx="th">Use</div>
                     </tr>
                  </thead>
                  <div tagx="tbody">
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P1</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpaC</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">CCGGGCACCTATCACCTG</td>
                        <td rowspan="2" align="left" colspan="1">607 pb</td>
                        <td rowspan="2" align="left" colspan="1">Primers used to check the presence of usher LM21
                           <i>kpaC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P2</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpaC</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">GCTTAACCGGCACGCTGG</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P3</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpbC</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GTCCTGGCGAGTTCGTTACGG</td>
                        <td rowspan="2" align="left" colspan="1">658 pb</td>
                        <td rowspan="2" align="left" colspan="1">Primers used to check the presence of usher LM21
                           <i>kpbC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P4</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpbC</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CGTTGGCGCCGTAGTCGATACG</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P5</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpd</i>C-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GAGATCGCCTGCAGCGGCAG</td>
                        <td rowspan="2" align="left" colspan="1">568 pb</td>
                        <td rowspan="2" align="left" colspan="1">Primers used to check the presence of usher LM21
                           <i>kpdC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P6</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpdC</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CAGCGCTGTCCACCGTCGC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P7</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpeC</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">CCGCCAACAGCGATGATAAGCC</td>
                        <td rowspan="2" align="left" colspan="1">509 pb</td>
                        <td rowspan="2" align="left" colspan="1">Primers used to check the presence of usher LM21
                           <i>kpeC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P8</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpeC</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">GAACCCGCGCATCAGCGAC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P9</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpgC</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GGTATTGACCGGCAGCGTG</td>
                        <td rowspan="2" align="left" colspan="1">574 pb</td>
                        <td rowspan="2" align="left" colspan="1">Primers used to check the presence of usher LM21
                           <i>kpgC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P10</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpgC</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">GCGGGCAAAGCATCGACG</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P11</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpj</i>C-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">TCTCTTTGCGAAGCCAAC</td>
                        <td rowspan="2" align="left" colspan="1">582 pb</td>
                        <td rowspan="2" align="left" colspan="1">Primers used to check the presence of usher LM21
                           <i>kpjC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P12</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpj</i>C-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CATGGATCGGGAATGGAT</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P13</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>mrkC</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GCCGCAAAGCGGTTGGTG</td>
                        <td rowspan="2" align="left" colspan="1">592 pb</td>
                        <td rowspan="2" align="left" colspan="1">Primers used to check the presence of usher LM21
                           <i>mrkC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P14</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>mrkC-</i>Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CGACGCCGTTCTACATTACC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P15</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>fimC</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GCTGACCCCGTAATAGAG</td>
                        <td rowspan="2" align="left" colspan="1">475 pb</td>
                        <td rowspan="2" align="left" colspan="1">Primers used to check the presence of usher LM21
                           <i>fimC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P16</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>fimC</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CGCGCCACCCTCTATCTC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P17</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpa-Kana-</i>Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">AGCCATGACGCAGGCGCGACTGCTTCTCCCTGCCTGTCCGCGGAGTATCTGAATTCATGGGTAGGCTGGAGCTGCTTCG</td>
                        <td rowspan="2" align="left" colspan="1">1595 pb</td>
                        <td rowspan="2" align="left" colspan="1">LM21Δ
                           <i>kpa</i> isogenic mutant construction
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P18</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpa-Kana-</i>Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CATATCAGCCCCTCTTCCATGTTGAAAACCTGCATCCATTACCGACATTCGGCCTGATAGCATATGAATATCCTCCTTAGTTC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P19</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpb-Kana</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">AGCATAGTAGGGATAAAAAAAGAAGTATTTTTCTCATTCTTTATTGTTCCTTGGTCAATAGTAGGCTGGAGCTGCTTCG</td>
                        <td rowspan="2" align="left" colspan="1">1595 pb</td>
                        <td rowspan="2" align="left" colspan="1">LM21Δ
                           <i>kpb</i> isogenic mutant construction
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P20</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpb-Kana-Rv</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">GATGTGTCTATGCCAAATAAGAATAATGGATTATTCAAACTATCCACCATCTTCTTTGCACATATGAATATCCTCCTTAGTTC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P21</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpd-Kana-</i>Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">TCAGGGGTACCTCTTCTGCTGGCCGGCGGGTACTGCGCTTCGCTGTCGGCAGCGTGCCAGGTAGGCTGGAGCTGCTTCG</td>
                        <td rowspan="2" align="left" colspan="1">1595 pb</td>
                        <td rowspan="2" align="left" colspan="1">LM21Δ
                           <i>kpd</i> isogenic mutant construction
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P22</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpd-Kana-</i>Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">ACAGGAAGTTGATGTTTCCTTTACTGGCGCATGCCTGACGGCGCTGTGCCGCCAGGCAGACATATGAATATCCTCCTTAGTTC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P23</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpe-Kana-</i>Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GCGTCAGGGCGACCCGACGCAGGAGAGTTAAAAAGGACATGATAACGTCTGCTGTAAGATGTAGGCTGGAGCTGCTTCG</td>
                        <td rowspan="2" align="left" colspan="1">1595 pb</td>
                        <td rowspan="2" align="left" colspan="1">LM21Δ
                           <i>kpe</i> isogenic mutant construction
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P24</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpe-Kana-</i>Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">TTTGGCGCTGTGCAGACGTTTAGCGCCCCCGTTTCGGCGCACTAAGCGTCTGATGTCGCGCATATGAATATCCTCCTTAGTTC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P25</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpg-Kana-</i>Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GCGCTCAGCCCCAGCAGAAAGAGGAAAGGAAGTAGACGAAAAACAGATTTCATAGCCGCCGTAGGCTGGAGCTGCTTCG</td>
                        <td rowspan="2" align="left" colspan="1">1595 pb</td>
                        <td rowspan="2" align="left" colspan="1">LM21Δ
                           <i>kpg</i> isogenic mutant construction
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P26</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpg-Kana-</i>Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">AGAATAACAACAGCCAGGGTGCGGGAGTATGCCTTGTTGCCCGTTACCGCCGGGCGGGCGCATATGAATATCCTCCTTAGTTC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P27</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpj-Kana</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">ATTAATGGGTATGGAATTTTGGGTCAGTTCAATTTCAATATGAAATATGCATGCCTCTTGGTAGGCTGGAGCTGCTTCG</td>
                        <td rowspan="2" align="left" colspan="1">1595 pb</td>
                        <td rowspan="2" align="left" colspan="1">LM21Δ
                           <i>kpj</i> isogenic mutant construction
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P28</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpj-Kana-</i>Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">ACCACTTCATTTTATTCTTCCTCGTAAGGACGGCTATTTACTTTGCTCAAGGGCGCAGAGCATATGAATATCCTCCTTAGTTC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P29</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>mrk-Kana-</i>Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">ATTAAGACGATAAAAAGCGTTAGTAATTTCCTCAGCGACATACGCTATCCTTTGTTGTTTGTAGGCTGGAGCTGCTTCG</td>
                        <td rowspan="2" align="left" colspan="1">1595 pb</td>
                        <td rowspan="2" align="left" colspan="1">LM21Δ
                           <i>mrk</i> isogenic mutant construction
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P30</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>mrk-Kana-</i>Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CATTAATGACTATGGCGCCGATGTCGCAGTGAAAGTGGCCGTTAAATAAGGGAAAAGCTACATATGAATATCCTCCTTAGTTC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P31</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>fim-Kana-</i>Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GGGACACGCTGCGCATGACCAGAAGCAATGACAGCCGGGCAACGTCGGACCGGAGGGATAGTAGGCTGGAGCTGCTTCG</td>
                        <td rowspan="2" align="left" colspan="1">1595 pb</td>
                        <td rowspan="2" align="left" colspan="1">LM21Δ
                           <i>fim</i> isogenic mutant construction
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P32</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>fim-Kana-</i>Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">GGTGCGCCCAGGGTGAACAGGGCGCCCAGCAGATACTGCAGTGTTCTCATCATGATCTCC
                           <u>CATATGAATATCCTCCTTAGTTC</u>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P33</td>
                        <td align="left" rowspan="1" colspan="1">
                           <i>kana-</i>Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GTAGGCTGGAGCTGCTTCG</td>
                        <td rowspan="2" align="left" colspan="1">1475 pb</td>
                        <td rowspan="2" align="left" colspan="1">Primers used to check the presence of kanamycin cassette</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P34</td>
                        <td align="left" rowspan="1" colspan="1">
                           <i>kana-</i>Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CATATGAATATCCTCCTTAGTTC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P35</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpa</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GTACGCCAACGTCTTCCGC</td>
                        <td rowspan="4" align="left" colspan="1">2451 pb</td>
                        <td rowspan="4" align="left" colspan="1">Two-step PCR amplification to generate fragment containing 
                           <i>kpa</i> operon promoter fused to LM21
                           <i>kpaC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P36</td>
                        <td align="left" rowspan="1" colspan="1">LM21_P_
                           <i>kpa</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">TTCCCTGACATACCATAATCATCCTGGCCGCGTTCGATAC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P37</td>
                        <td align="left" rowspan="1" colspan="1">LM21_P_
                           <i>kpa-</i>Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GTATCGAACGCGGCCAGGATGATTATGGTATGTCAGGGAA</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P38</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpa</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">TTACCGACATTCGGCCTG</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P39</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpb</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GGTGAAAAAGCATCTGAGC</td>
                        <td rowspan="4" align="left" colspan="1">2661 bp</td>
                        <td rowspan="4" align="left" colspan="1">Two-step PCR amplification to generate fragment containing 
                           <i>kpb</i> operon promoter fused to LM21
                           <i>kpbC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P40</td>
                        <td align="left" rowspan="1" colspan="1">LM21_P_
                           <i>kpb</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CCATTATTCTTATTTGGCATCTACCGGTTTAGCACGACC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P41</td>
                        <td align="left" rowspan="1" colspan="1">LM21_P_
                           <i>kpb-</i>Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GGTCGTGCTAAACCGGTAGATGCCAAATAAGAATAATGG</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P42</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpb</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">TTAGCGACAAACCGTTTCG</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P43</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpd</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GAAAGCGCAGGGAGTATG</td>
                        <td rowspan="4" align="left" colspan="1">2454 bp</td>
                        <td rowspan="4" align="left" colspan="1">Two-step PCR amplification to generate fragment containing 
                           <i>kpd</i> operon promoter fused to LM21
                           <i>kpdC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P44</td>
                        <td align="left" rowspan="1" colspan="1">LM21_P_
                           <i>kpd</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CTTGATCTCCCTGAGTTTCATTCAACATAATGACCTCCTGGC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P45</td>
                        <td align="left" rowspan="1" colspan="1">LM21_P_
                           <i>kpd-</i>Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GCCAGGAGGTCATTATGTTGAATGAAACTCAGGGAGATCAAG</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P46</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpd</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">TTACTTCTCCTTTATTCCCGG</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P47</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpe</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">CTATGATGAATCCTATGCCG</td>
                        <td rowspan="4" align="left" colspan="1">2762 bp</td>
                        <td rowspan="4" align="left" colspan="1">Two-step PCR amplification to generate fragment containing 
                           <i>kpe</i> operon promoter fused to LM21
                           <i>kpe</i>C
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P48</td>
                        <td align="left" rowspan="1" colspan="1">LM21_P_
                           <i>kpe</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CCGGGCCGTGGCGCATGATCGAGCTGCTTTTTCAGC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P49</td>
                        <td align="left" rowspan="1" colspan="1">LM21_P_
                           <i>kpe-</i>Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GCTGAAAAAGCAGCTCGATCATGCGCCACGGCCCGG</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P50</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpe-</i>Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">TTAAAAAGGACATGATAACGTC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P51</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpg</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">CTAATATCCTGCGGAAGG</td>
                        <td rowspan="4" align="left" colspan="1">2802 pb</td>
                        <td rowspan="4" align="left" colspan="1">Two-step PCR amplification to generate fragment containing 
                           <i>kpg</i> operon promoter fused to LM21
                           <i>kpgC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P52</td>
                        <td align="left" rowspan="1" colspan="1">LM21_P_
                           <i>kpg</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CGCCGTCCGGTAAGCTCATTCAGGATTGATAGTGTTGGC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P53</td>
                        <td align="left" rowspan="1" colspan="1">LM21_P_
                           <i>kpg-</i>Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GCCAACACTATCAATCCTGAATGAGCTTACCGGACGGCG</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P54</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpg</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">TCAGCGTGTCTGGCACAG</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P55</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpj</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GCTTTAACTCGCAGAAAGG</td>
                        <td rowspan="4" align="left" colspan="1">2927pb</td>
                        <td rowspan="4" align="left" colspan="1">Two-step PCR amplification to generate fragment containing 
                           <i>kpj</i> operon promoter fused to LM21
                           <i>kpjC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P56</td>
                        <td align="left" rowspan="1" colspan="1">LM21_P_
                           <i>kpj</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">AATCCATCGTTGAGGCATTCAGCTGGTCCAGGTGG</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P57</td>
                        <td align="left" rowspan="1" colspan="1">LM21_P_
                           <i>kpj</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">CCACCTGGACCAGCTGAATGCCTCAACGATGGATT</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P58</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpj</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">TTAACTTTCATTCTCTTCTCCTGAC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P59</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>mrk</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GCTGACCAACAAAATCATCC</td>
                        <td rowspan="4" align="left" colspan="1">2794 pb</td>
                        <td rowspan="4" align="left" colspan="1">Two-step PCR amplification to generate fragment containing 
                           <i>mrk</i> operon promoter fused to LM21
                           <i>mrkC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P60</td>
                        <td align="left" rowspan="1" colspan="1">LM21_P_
                           <i>mrk</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CAGAATGACCTCTGCTTCATGCTGACCTCAGAATAAAAATACA</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P61</td>
                        <td align="left" rowspan="1" colspan="1">LM21_P_
                           <i>mrk</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">TGTATTTTTATTCTGAGGTCAGCATGAAGCAGAGGTCATTCTG</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P62</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>mrk</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">GCCATTAAGACGATAAAAAGC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P63</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>fim</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">CAAATTTTATGTTAACGACGC</td>
                        <td rowspan="4" align="left" colspan="1">2893 pb</td>
                        <td rowspan="4" align="left" colspan="1">Two-step PCR amplification to generate fragment containing 
                           <i>fim</i> operon promoter fused to LM21
                           <i>fim</i>C
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P64</td>
                        <td align="left" rowspan="1" colspan="1">LM21_P_
                           <i>fim</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CCATATTTCCGATGTGACATTCATGCGCGAATATCGTC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P65</td>
                        <td align="left" rowspan="1" colspan="1">LM21_P_
                           <i>fim-</i>Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GACGATATTCGCGCATGAATGTCACATCGGAAATATGG</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P66</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>fim</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">GACGTTATCATGTCCTTTTTAA</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P67</td>
                        <td align="left" rowspan="1" colspan="1">LM21-
                           <i>kpa-</i>RT-qPCR-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GCTCCGACGATAAAGAATATAAC</td>
                        <td rowspan="2" align="left" colspan="1">113 pb</td>
                        <td rowspan="2" align="left" colspan="1">Primers used for qPCR to verify the expression of 
                           <i>kpaC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P68</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpa-</i>RT-qPCR-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CAGGTTGGTGTATTTATCTTCC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P69</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpb-</i>RT-qPCR-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GGTAATTATACCTCTCGCTCAC</td>
                        <td rowspan="2" align="left" colspan="1">114 pb</td>
                        <td rowspan="2" align="left" colspan="1">Primers used for qPCR to verify the expression of 
                           <i>kpbC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P70</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpb-</i>RT-qPCR-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">GTCCCAGTAATCTTCAATCAAC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P71</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpd-</i>RT-qPCR-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">CTGTCATTTAACGGAATCTGG</td>
                        <td rowspan="2" align="left" colspan="1">112 pb</td>
                        <td rowspan="2" align="left" colspan="1">Primers used for qPCR to verify the expression of 
                           <i>kpdC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P72</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpd-</i>RT-qPCR-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">GCTGGGCATACAGATAATAATTG</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P73</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpe-</i>RT-qPCR-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">CAATACCATAACCTCTCGGTC</td>
                        <td rowspan="2" align="left" colspan="1">149 pb</td>
                        <td rowspan="2" align="left" colspan="1">Primers used for qPCR to verify the expression of 
                           <i>kpeC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P74</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpe-</i>RT-qPCR-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CGATCAAAACCTTTGCTGTAC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P75</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpg-</i>RT-qPCR-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GATGTCTGACGATGAAATGC</td>
                        <td rowspan="2" align="left" colspan="1">125 pb</td>
                        <td rowspan="2" align="left" colspan="1">Primers used for qPCR to verify the expression of 
                           <i>kpgC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P76</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpg-</i>RT-qPCR-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CAAAGGTCTCGTAGATCACATAC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P77</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpj-</i>RT-qPCR-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">CTGATATCACGTTTGTTGACG</td>
                        <td rowspan="2" align="left" colspan="1">121 pb</td>
                        <td rowspan="2" align="left" colspan="1">Primers used for qPCR to verify the expression of 
                           <i>kpj</i>C
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P78</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>kpj-</i>RT-qPCR-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CGGGAATAAACTTCTCGATC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P79</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>mrk-</i>RT-qPCR-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">CTGTGGTTTGGCGATAAC</td>
                        <td rowspan="2" align="left" colspan="1">125 pb</td>
                        <td rowspan="2" align="left" colspan="1">Primers used for qPCR to verify the expression of 
                           <i>mrkC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P80</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>mrk-</i>RT-qPCR-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CCGTAGTTAAGGTTGTTTTCAC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P81</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>fim-</i>RT-qPCR-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">GAAAAACATGGGCTATTTCG</td>
                        <td rowspan="2" align="left" colspan="1">115 pb</td>
                        <td rowspan="2" align="left" colspan="1">Primers used for qPCR to verify the expression of 
                           <i>fimC</i>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P82</td>
                        <td align="left" rowspan="1" colspan="1">LM21_
                           <i>fim-</i>RT-qPCR-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">GGATTTGTTATAGAGAAAACGC</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P83</td>
                        <td align="left" rowspan="1" colspan="1">RT-
                           <i>gyrA</i>-Fw
                        </td>
                        <td align="left" rowspan="1" colspan="1">ATTGGGCAATGACTGGAACAA</td>
                        <td rowspan="2" align="left" colspan="1">149 pb</td>
                        <td rowspan="2" align="left" colspan="1">Primers used for qPCR to verify the expression of 
                           <i>gyrA</i> in LM21 wild type
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">P84</td>
                        <td align="left" rowspan="1" colspan="1">RT-
                           <i>gyrA</i>-Rv
                        </td>
                        <td align="left" rowspan="1" colspan="1">CCACCAGCATGTAACGCAG</td>
                     </tr>
                  </div>
               </table>
            </div>
         </div>
      </div>
      <div id="sec004">
         <h3>Locus structure prediction and genetic analysis</h3>
         <p>To define potential operon genetic organization, we visualized flanking regions of
            usher nucleotide sequences in xBase2 [
            <span ref-type="bibr" rid="pone.0116215.ref025">25</span>]. Fimbrial encoding genes were identified using conserved protein domain searches
            [
            <span ref-type="bibr" rid="pone.0116215.ref024">24</span>] and sequence homology to annotated genes. Intergenic regions &gt;200pb were investigated
            for the presence of protein encoding sequences with conserved fimbrial domains or
            significant sequence identity to fimbrial subunits.
         </p>
      </div>
      <div id="sec005">
         <h3>Multiple sequence alignment and phylogenetics</h3>
         <p>Publicly available 
            <i>K</i>. 
            <i>pneumoniae</i> genomes used for this study were those from strains HS11286, MGH 78578, NTUH-K2044,
            342 (reassigned to 
            <i>K</i>. 
            <i>variicola</i> since genome sequencing), KCTC 2242, 1084, JM45, CG43, Kp13, 30684/NJST258_2 and
            30660/NJST258_1.
         </p>
         <p>Full-length usher amino acid sequences from intact fimbrial operons were used to infer
            evolutionary relationships. Sequences were aligned in MAFFT v6.617b [
            <span ref-type="bibr" rid="pone.0116215.ref026">26</span>,
            <span ref-type="bibr" rid="pone.0116215.ref027">27</span>], using iterative global-pair refinement method with default gap penalties. The alignment
            was cleaned with trimal [
            <span ref-type="bibr" rid="pone.0116215.ref028">28</span>] so that every site with more than 30% of gaps or with an average similarity inferior
            to 0.0001 was removed.
         </p>
         <p>Phylogenetic analyses were performed with PhyML 3.1 [
            <span ref-type="bibr" rid="pone.0116215.ref029">29</span>], using LG+gamma model with a 4 categories gamma law. To estimate the confidence
            in the tree topology, statistical aLRT-SH-like supports were computed [
            <span ref-type="bibr" rid="pone.0116215.ref030">30</span>]. Alignments and phylogenetic tree were repeated with usher sequences of previous
            usher phylograms [
            <span ref-type="bibr" rid="pone.0116215.ref003">3</span>] to check tree validity (data not shown).
         </p>
      </div>
      <div id="sec006">
         <h3>Bacterial strains, plasmids and growth conditions</h3>
         <p>The bacterial strains and plasmids used in the study are shown in 
            <span ref-type="table" rid="pone.0116215.t002">Table 2</span>. All bacterial strains were stored at −80°C in Lysogeny Broth (LB) medium containing
            20% glycerol. When appropriate, antibiotics were added to the media at the following
            concentrations: ampicillin (50μg/ml), kanamycin (50μg/ml), tetracycline (20μg/ml),
            streptomycin (50μg/ml) and spectinomycin (50μg/ml). LB media, 0.4% glucose M63B1 minimal
            medium (M63B1–0.4% Glu) and Dulbecco’s modified Eagle’s medium (DMEM) were used for
            experiments. Bacterial growth was monitored by measuring the optical density at 620
            nm (OD
            <div tagx="sub">620</div>) and plating dilution on agar plates to determine colony forming units.
         </p>
         <div tagx="table-wrap">
            <div tagx="object-id">10.1371/journal.pone.0116215.t002</div>
            <div tagx="label">Table 2</div>
            <caption>
               <div tagx="title">Strains and plasmids used.</div>
            </caption>
            <div tagx="alternatives">
               <div tagx="graphic"></div>
               <table frame="hsides" rules="groups">
                  <colgroup span="1">
                     <col align="left" valign="middle" span="1"></col>
                     <col align="left" valign="middle" span="1"></col>
                     <col align="left" valign="middle" span="1"></col>
                     <col align="left" valign="middle" span="1"></col>
                  </colgroup>
                  <thead>
                     <tr>
                        <div tagx="th">Num</div>
                        <div tagx="th">Strains or Plasmids</div>
                        <div tagx="th">Decription</div>
                        <div tagx="th">Source and/or reference</div>
                     </tr>
                  </thead>
                  <div tagx="tbody">
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S1</td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 strain SHV-1::
                           <i>aadA7</i>-
                           <i>gfp</i>mut3; serogroup O25, K35, Sp
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">[
                           <span ref-type="bibr" rid="pone.0116215.ref064">64</span>]
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S2</td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i>/pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 gfp/pSTAB; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S3</td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i>-
                           <i>Strepto</i>/pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i>/pSTAB; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>, St
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S4</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpaC</i>-
                           <i>kana</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpaC</i>::KmFRT; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S5</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpbC</i>-
                           <i>kana</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpbC</i>::KmFRT; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S6</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpdC</i>-
                           <i>kana</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpdC</i>::KmFRT; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S7</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpeC-kana</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpeC</i>::KmFRT; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S8</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δk
                           <i>pgC</i>-
                           <i>kana</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpgC</i>::KmFRT; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S9</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpjC</i>-
                           <i>kana</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpjC</i>::KmFRT; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S10</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>mrkC</i>-
                           <i>kana</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>mrk</i>C::KmFRT; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S11</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>fimC</i>-
                           <i>kana</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>fim</i>C::KmFRT; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S12</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpaC-kana</i>/pSTAB-
                           <i>kpaC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpa</i>C::KmFRT/pSTAB_
                           <i>kpaC</i>; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S13</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpbC-kana</i>/pSTAB-
                           <i>kpbC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpbC</i>::KmFRT/pSTAB_
                           <i>kpbC</i>; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S14</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpdC-kana</i>/pSTAB-
                           <i>kpdC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpd</i>C::KmFRT/pSTAB_
                           <i>kpdC</i>; Sp
                           <sup>r,</sup> Ap
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S15</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpeC-kana</i>/pSTAB-
                           <i>kpeC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpeC</i>::KmFRT/pSTAB_
                           <i>kpeC;</i> Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S16</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpgC-kana</i>/pSTAB-
                           <i>kpgC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpgC</i>::KmFRT/pSTAB_
                           <i>kpgC</i>; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S17</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpjC-kana</i>/pSTAB-
                           <i>kpjC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpjC</i>::KmFRT/pSTAB_
                           <i>kpjC</i>; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S18</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>mrkC-kana</i>/pSTAB-
                           <i>mrkC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>mrkC</i>::KmFRT/pSTAB_
                           <i>mrkC</i>; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S19</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>fimC-kana</i>/pSTAB-
                           <i>fim</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>fimC</i>::KmFRT/ pSTAB_
                           <i>fim</i>C, Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S20</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpaC-kana/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpaC</i>::KmFRT/pSTAB; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S21</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpbC-kana/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpbC</i>::KmFRT/pSTAB; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S22</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpdC-kana/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpdC</i>::KmFRT/pSTAB; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S23</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpeC</i>-
                           <i>kana</i>/pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpeC</i>::KmFRT/pSTAB; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S24</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpgC-kana/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpgC</i>::KmFRT/pSTAB; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S25</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpjC-kana/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpjC</i>::KmFRT/pSTAB; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S26</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>mrkC-kana/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>mrkC</i>::KmFRT/pSTAB; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S27</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>fimC-kana/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>fimC</i>::KmFRT/pSTAB; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, Km
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S28</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpaC-kana</i>-
                           <i>Strepto</i>/pSTAB-
                           <i>kpaC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpaC</i>::KmFRT/pSTAB_
                           <i>kpaC</i>; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>,Ap
                           <sup>r</sup>, St
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S29</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpbC-kana</i>-
                           <i>Strepto</i>/pSTAB-
                           <i>kpbC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpbC</i>::
                           <i>KmFRT</i>/pSTAB_
                           <i>kpbC</i>; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, St
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S30</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpdC-kana</i>-
                           <i>Strepto</i>/pSTAB-
                           <i>kpdC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpdC</i>::KmFRT/pSTAB_
                           <i>kpdC</i>; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, St
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S31</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpeC-kana</i>-
                           <i>Strepto</i>/pSTAB-
                           <i>kpeC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpeC</i>::
                           <i>KmFRT</i>/pSTAB_
                           <i>kpeC</i>; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, St
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S32</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpgC-kana</i>-
                           <i>Strepto</i>/pSTAB-
                           <i>kpgC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpgC</i>::
                           <i>KmFRT</i>/pSTAB_
                           <i>kpgC</i>; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, St
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S33</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpjC-kana</i>-
                           <i>Strepto</i>/pSTAB-
                           <i>kpjC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpjC</i>::
                           <i>KmFRT</i>/pSTAB_
                           <i>kpjC</i>; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, St
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S34</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>mrkC-kana</i>-
                           <i>Strepto</i>/pSTAB-
                           <i>mrkC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>mrkC</i>::
                           <i>KmFRT</i>/pSTAB_
                           <i>mrkC</i>; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, St
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S35</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>fimC-kana</i>-
                           <i>Strepto</i>/pSTAB-
                           <i>fimC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>fimC</i>::
                           <i>KmFRT</i>/pSTAB_
                           <i>fimC</i>; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, St
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S36</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpaC-kana</i>-
                           <i>Strepto/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpaC</i>::KmFRT/pSTAB; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, St
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S37</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpbC-kana</i>-
                           <i>Strepto/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpbC</i>::KmFRT/pSTAB; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, St
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S38</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpdC-kana</i>-
                           <i>Strepto/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpdC</i>::KmFRT/pSTAB; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, St
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S39</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpeC-kana</i>-
                           <i>Strepto</i>/pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpeC</i>::KmFRT/pSTAB; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, St
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S40</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpgC-kana</i>-
                           <i>Strepto/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpgC</i>::KmFRT/pSTAB; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, St
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S41</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpjC-kana</i>-
                           <i>Strepto/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpj</i>C::KmFRT/pSTAB; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, St
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S42</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>mrkC-kana</i>-
                           <i>Strepto/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>mrkC</i>::KmFRT/pSTAB; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, St
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S43</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>fimC-kana</i>-
                           <i>Strepto/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>fimC</i>::KmFRT/pSTAB; Sp
                           <sup>r</sup>, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, St
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S44</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpaC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpaC</i>; Sp
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S45</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpbC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpbC</i>; Sp
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S46</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpdC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpdC;</i> Sp
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S47</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpe</i>C
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpeC</i>, Sp
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S48</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpgC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpgC;</i> Sp
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S49</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpjC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpj</i>C; Sp
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S50</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>mrkC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>mrkC;</i> Sp
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S51</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>fimC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>fimC</i>, Sp
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S52</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpaC</i>/pSTAB-
                           <i>kpaC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpaC</i>/pSTAB_
                           <i>kpaC</i>; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S53</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpbC</i>/pSTAB-
                           <i>kpbC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpbC</i>/pSTAB_
                           <i>kpbC</i>; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S54</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpdC</i>/pSTAB-
                           <i>kpdC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpd</i>C/pSTAB_
                           <i>kpdC</i>; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S55</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpeC</i>/pSTAB-
                           <i>kpeC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpeC</i>/pSTAB_
                           <i>kpeC</i>; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S56</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpgC</i>/pSTAB-
                           <i>kpgC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpgC</i>/pSTAB_
                           <i>kpgC</i>; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S57</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpjC</i>/pSTAB-
                           <i>kpjC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpjC</i>/pSTAB_
                           <i>kpjC</i>; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S58</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>mrkC</i>/pSTAB-
                           <i>mrkC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>mrkC</i>/pSTAB_
                           <i>mrkC</i>; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S59</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>fimC</i>/pSTAB-
                           <i>fimC</i>
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>fimC</i>/pSTAB_
                           <i>fimC</i>; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S60</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpaC/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpaC</i>/pSTAB; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S61</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpbC/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpbC</i>/pSTAB; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S62</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpdC/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpdC</i>/pSTAB; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S63</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpeC/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpeC</i>/pSTAB; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S64</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpgC/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kpgC</i>/pSTAB; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S65</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>kpjC/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>kp</i>C/pSTAB; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S66</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>mrkC</i>/pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>mrkC</i>/pSTAB; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S67</td>
                        <td align="left" rowspan="1" colspan="1">LM21Δ
                           <i>fimC/</i>pSTAB
                        </td>
                        <td align="left" rowspan="1" colspan="1">LM21 
                           <i>gfp</i> Δ
                           <i>fimC</i>/pSTAB; Sp
                           <sup>r</sup>, Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">This study</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">S68</td>
                        <td align="left" rowspan="1" colspan="1">TOP10</td>
                        <td align="left" rowspan="1" colspan="1">
                           <i>E</i>.
                           <i>coli</i> from invitrogen
                        </td>
                        <td align="left" rowspan="1" colspan="1">Invitrogen</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">
                           <b>Plasmids</b>
                        </td>
                        <td colspan="3" align="left" rowspan="1">
                           <b> </b>
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">V1</td>
                        <td align="left" rowspan="1" colspan="1">pKOBEG199</td>
                        <td align="left" rowspan="1" colspan="1">Plasmid encoding Lambda Red recombinase protein</td>
                        <td align="left" rowspan="1" colspan="1">[
                           <span ref-type="bibr" rid="pone.0116215.ref031">31</span>]
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">V2</td>
                        <td align="left" rowspan="1" colspan="1">pKD4</td>
                        <td align="left" rowspan="1" colspan="1">Plasmid with FRT-flanked kanamycin-resistance cassette used for kanamycin casette
                           amplification, Km
                           <sup>r</sup>, Ap
                           <sup>r</sup>, Km
                           <sup>r</sup>,
                        </td>
                        <td align="left" rowspan="1" colspan="1">[
                           <span ref-type="bibr" rid="pone.0116215.ref033">33</span>]
                        </td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">V3</td>
                        <td align="left" rowspan="1" colspan="1">pBlunt</td>
                        <td align="left" rowspan="1" colspan="1">Used for PCR product cloning</td>
                        <td align="left" rowspan="1" colspan="1">Invitrogen</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">V4</td>
                        <td align="left" rowspan="1" colspan="1">pSTAB</td>
                        <td align="left" rowspan="1" colspan="1">pZE derivative plamsid. Contains the 
                           <i>flm</i> toxin-antitoxin system from F plasmid; Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">Gift from J.M. Ghigo</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">V5</td>
                        <td align="left" rowspan="1" colspan="1">pCP20</td>
                        <td align="left" rowspan="1" colspan="1">pCP20 carries the yeast recombinase gene (FLP, aka exo), chloramphenicol and ampicillin
                           resistant gene and temperature sensitive replication. Ap
                           <sup>r</sup>
                        </td>
                        <td align="left" rowspan="1" colspan="1">[
                           <span ref-type="bibr" rid="pone.0116215.ref032">32</span>]
                        </td>
                     </tr>
                  </div>
               </table>
            </div>
            <div tagx="table-wrap-foot">
               <div tagx="fn">
                  <p>Abbreviations: Ap, ampicillin; Km, kanamycin; St, streptomycin; Sp, spectinomycin</p>
               </div>
            </div>
         </div>
      </div>
      <div id="sec007">
         <h3>Cloning of the 
            <i>K</i>. 
            <i>pneumoniae</i> LM21 usher-like genes, construction of isogenic and transcomplementation mutants
         </h3>
         <p>Primers were designed on the basis of 
            <i>K</i>. 
            <i>pneumoniae</i> LM21 genome sequence information. All primers used are listed in 
            <span ref-type="table" rid="pone.0116215.t001">Table 1</span>. Chromosomal DNA extraction was performed by NucleoSpin tissue kit (Macherey-Nagel)
            according to the manufacturer’s recommendations.
         </p>
         <p>The usher-defective mutants were created by allelic exchange after replacement of
            the Usher encoding gene by the selectable kanamycin resistance gene according to Chaveroche
            et 
            <i>al</i>. [
            <span ref-type="bibr" rid="pone.0116215.ref031">31</span>]. The kanamycin cassette flanked by 60-bp fragments, which correspond to the encoding
            upstream and downstream regions of the usher encoding gene, was generated using pKD4
            plasmid as template and primers P17 to P32 (
            <span ref-type="table" rid="pone.0116215.t001">Table 1</span>). They were electroporated in the 
            <i>K</i>. 
            <i>pneumoniae</i> LM21 strain harboring the lambda-red protein-encoding plasmid pKOBEG199 under the
            control of a promoter induced by 
            <div tagx="sc">l</div>-arabinose. Mutants were selected onto LB agar containing kanamycin. The loss of the
            pKOBEG199 plasmid was then checked on LB containing tetracycline. The substitution
            of the encoding usher gene by the kanamycin cassette was further checked by PCR performed
            with primers P33 and P34 (
            <span ref-type="table" rid="pone.0116215.t001">Table 1</span>). In a second step, the antibiotic resistant encoding gene was excised from the mutant’s
            genome using the pCP20 plasmid, a temperature-sensitive replication plasmid with thermal
            induction of flippase recombinase (FLP) synthesis [
            <span ref-type="bibr" rid="pone.0116215.ref032">32</span>], which gave rise to nonpolar mutants as previously described in Datsenko et 
            <i>al</i>. [
            <span ref-type="bibr" rid="pone.0116215.ref033">33</span>]. For 
            <i>in vivo</i> assays, spontaneous mutants of Δ
            <i>usher-kana</i> (designated LM21Δ
            <i>usher</i>-
            <i>kana</i> in 
            <span ref-type="table" rid="pone.0116215.t002">Table 2</span>) were selected for streptomycin resistance (designated LM21Δ
            <i>usher</i>-
            <i>kana-strepto</i> in 
            <span ref-type="table" rid="pone.0116215.t002">Table 2</span>)
         </p>
         <p>PCRs were performed using a Biorad T100 Thermal cycler. Restriction enzymes, Phusion
            high-Fidelity DNA polymerase and TaKaRa LA Taq were purchased from New England Biolabs,
            Thermo Scientific and Takara Biotechnology Inc, respectively, and used according to
            the manufacturers’ recommendations.
         </p>
         <p>For transcomplementation assays, fragments containing the entire 
            <i>usher-like</i> genes and their own putative promoters, as detected by sequence analysis, were amplified
            from 
            <i>K</i>. 
            <i>pneumoniae</i> LM21 genomic DNA by an overlapping extension PCR using primers listed in 
            <span ref-type="table" rid="pone.0116215.t001">Table 1</span> (P35 to P66). The resulting fragment was cloned using Zero Blunt PCR cloning kit
            (Invitrogen) and subcloned into the 
            <i>Eco</i>R1 or 
            <i>Bam</i>H1 digested pSTAB vector. Resulting recombinant vectors (pSTAB-
            <i>usher</i>) were then introduced by electroporation into the isogenic mutants. In parallel,
            the LM21 wild type strain and the isogenic mutants were transformed with the empty
            pSTAB plasmid vector (
            <span ref-type="table" rid="pone.0116215.t002">Table 2</span>).
         </p>
      </div>
      <div id="sec008">
         <h3>RNA manipulations, real-time RT-PCR</h3>
         <p>Total RNA was extracted from bacteria grown in 3 different media conditions (LB, M63B1
            and DMEM), using Trizol reagent (Invitrogen) according to the method described by
            Toledo-Araba et 
            <i>al</i>. [
            <span ref-type="bibr" rid="pone.0116215.ref034">34</span>] after lysing bacteria in the Precellys tissue homogenizer (Bertin Technologie).
            Reverse transcription was performed with the iScript cDNA synthesis kit (Biorad) using
            1 μg of RNA on the T100 Thermal Cycler (Biorad) according to the manufacturers’ recommendations,
            and quantification of cDNA levels was done using the SsoAdvanced Universal SYBR Green
            Supermix (Biorad) on a C1000 thermal Cycler- CFX96 (Biorad) with primers P67 to P82
            (
            <span ref-type="table" rid="pone.0116215.t001">Table 1</span>). As internal controls, the 
            <i>rpoB</i> gene was amplified with primers P83/P84. Amplification of a single expected product
            was confirmed by melting curve analysis. The amplification efficiency (E) of the reactions
            for the target genes was determined and used to compare relative gene expression.
            The ratio = (E
            <div tagx="sub">sample</div>)
            <sup>ΔCPsample</sup> / (E
            <div tagx="sub">reference</div>)
            <sup>ΔCPreference</sup> was calculated (CP: crossing point).
         </p>
      </div>
      <div id="sec009">
         <h3>Biofilm assays</h3>
         <p>The ability of bacteria to form biofilm was assessed in both a static microtiter plate
            and a dynamic microfermentor biofilm model. All experiments were performed in biological
            and technical triplicates. For the microtiter experiment measuring the early biofilm
            formation capacity, 4.10
            <sup>6</sup> CFU/mL of an overnight culture were inoculated into 100μl of M63B1–0.4% Glu in a
            96-well PVC microtiter plate (Falcon). After 4h incubation at 37°C, bacterial cells
            were stained for 15 minutes at room temperature by adding 50 μl of 0.5% (wt/vol) aqueous
            solution of crystal violet in each well. After five washes with distilled water, the
            bound dye was released from stained cells using 95% ethanol and measured by absorbance
            at 570 nm.
         </p>
         <p>Biofilm formation was performed in 60 ml aerated microfermentors to assess mature
            biofilm formation as described by Ghigo et 
            <i>al</i>. [
            <span ref-type="bibr" rid="pone.0116215.ref035">35</span>]. Continuous flow of 100ml/h of M63B1–0.4% Glu medium and constant aeration with
            sterile pressed air (0.3 bar) were used. After 24h of incubation, mature biofilms
            formed on the removable glass slide were dislodged by vortexing and sonication and
            resuspended in saline. Bacterial biomass was quantified by determining the number
            of CFUs.
         </p>
      </div>
      <div id="sec010">
         <h3>Cell line growth and adhesion assays</h3>
         <p>Intestine 407 (Int-407) cells derived from human embryonic intestinal epithelium were
            purchased from American Type Culture Collection (ATCC strain CCL-6). Cells were grown
            in a humidified incubator at 37°C under 5% CO
            <div tagx="sub">2</div>. Cells were cultured in Dulbecco’s Modifed Eagle Medium (DMEM) (PAA-laboratories
            GmbH—Dominique Dutscher) containing 10% (v/v) heat-inactivated fetal bovine serum
            (PAA-laboratories GmbH—Dominique Dutscher), 50 U/mL penicillin, and 50 μg/mL streptomycin.
            Adhesion assays were conducted in biological and technical triplicate over three to
            five successive passages of Int-407 cells. Briefly, monolayers were seeded with 4
            × 10
            <sup>5</sup> cells per well in 24-well tissue culture plates (Polylabo, Strasbourg, France) and
            incubated for 24 h. Monolayers were then infected at a multiplicity of infection (MOI)
            of 100 bacteria per cell in 1 ml of the cell culture medium without antibiotics and
            with heat-inactivated fetal calf serum (FCS) (PAA-laboratories GmbH—Dominique Dutscher).
            After incubation for 4h at 37°C in an atmosphere of 5% CO
            <div tagx="sub">2</div>, unattached bacteria were removed by washing the cell monolayers four times with
            sterile PBS. After detachment of the cells by addition of 1mL of Triton 1% (in PBS)
            per well, the suspension was transferred into a 1.5 mL reaction tube and the number
            of adhering bacteria was determined by quantification of CFUs by plating dilution
            onto LB agar plates.
         </p>
      </div>
      <div id="sec011">
         <h3>
            <i>In vitro</i> plant adhesion assays
         </h3>
         <p>Seeds of wild type 
            <i>Arabidopsis thaliana</i> (Col-O) were obtained from the Nottingham 
            <i>Arabidopsis</i> Stock Center (NASC). For 
            <i>in vitro</i> cultures, seeds were sterilized in 70% EtOH with 0.05% SDS followed by washing in
            95% EtOH, dried and sown on germination medium containing 0.8% w/v agar, 1% w/v sucrose
            and half-strength Murashige &amp; Skoog salts (M0255; Duchefa Biochemie, Netherlands).
            After 2 days of stratification at 4°C in darkness, plants were grown under long-day
            conditions (6h light/8h dark cycles) at 23°C. Fifteen-day-old 
            <i>Arabidopsis</i> plants were used for the 
            <i>in vitro</i> adhesion assays.
         </p>
         <p>Bacterial adhesion was tested as described in Haahtela 
            <i>et al</i>. [
            <span ref-type="bibr" rid="pone.0116215.ref036">36</span>]. Briefly, 2.5 x 10
            <sup>6</sup> bacteria were incubated with whole plants in 5 ml of PBS at room temperature in a
            controlled environment incubator shaker set at 30 rpm for 4 hours. A non-infected
            plant control was maintained under the same conditions. The plants were then washed
            twice for 15 min with 10mL of saline (0.9 [w/s] sodium chloride) and homogenized in
            1mL of saline with a tissue grinder (Kontes, size C), and the suspension was serially
            diluted and the number of adhering bacteria was determined by quantification of CFUs
            by plating the dilution onto LB agar plates. Each experiment was conducted in biological
            and technical triplicate.
         </p>
      </div>
      <div id="sec012">
         <h3>Mice Intestinal colonisation</h3>
         <p>Female specific-pathogen mice (OF1 Swiss; 3 to 5 weeks old, 22g, Charles River Swiss)
            were used. The models and protocols used in this study were all approved by the ethics
            committee of Auvergne (Comité Régional d'Ethique en Matière d'Expérimentation Animale
            Auvergne, CEMEEA C2EA-02) in compliance with the European Community guiding in the
            care and use of laboratory animals (86/609/CEE). A total of 50 animals were used in
            this experiment. Animals were housed five to a cage in a temperature-controlled room
            with a 12 h light/12 h dark cycle and were fed with Rodent Diet (A04-Safe, Epinay/Orge,
            France) ad libitum throughout the experiments. Cages were cleaned each two days. One
            day prior to infection, water was withdrawn and replaced with sterile water containing
            5g of streptomycin per liter throughout the experiment. After 1 week of acclimation,
            200 μl of bacterial suspension in sterile water (10
            <sup>7</sup> CFU) were given intragastrically to each mouse. For competition assays, each of the
            8 LM21Δ
            <i>usher</i>-
            <i>kana-strepto</i> mutants was mixed with wild type strain LM21 in equal amounts in 200μl sterile water
            and administered intragastrically to 5 mice per group. The mutant strains showing
            a deficiency in colonization were then tested individually (5 mice per strain) and
            in competition with their respective trans-complemented strain by the same procedure.
            After 1 day and subsequently every day for 10 days, feces were collected and homogenized
            in 1mL saline, and serial dilutions were plated onto selective media. As described
            above, the removed feces were plated onto streptomycin-containing LB plates to measure
            the total number of CFUs and onto streptomycin-kanamycin-containing LB plates to measure
            the number of Δ
            <i>usher</i> mutant CFUs. From these numbers, the exact ratio of mutant to wild type or transcomplement
            was calculated for inoculum and feces contents. Mice were euthanized by cervical dislocation
            on day 12. Each experiment was conducted in biological and technical triplicate.
         </p>
      </div>
      <div id="sec013">
         <h3>Western blot analysis of type 3 fimbriae expression</h3>
         <p>Overnight grown bacteria (8.10
            <sup>9</sup>) were mechanically disrupted by sonication in 2mL of 40mM Tris-buffer, and lysates
            were centrifuged for 10 min at 20,000 g to pellet unbroken cells. Supernatants were
            ultracentrifuged for 1h at 100,000g, and pellets suspended in 0.1 mL of sarkosyl 0.5%
            were then incubated for 30 min on ice to solubilize inner membrane. Samples were then
            ultracentrifuged 1h at 100,000g, and pellets were suspended in 0.05 mL of tris-buffer.
            Protein quantification was performed using Bradford Protein Assay (Biorad) reagent
            according the manufacturers’ recommendations. Five μg of protein samples were then
            analyzed by western blotting using an anti-MrkA polyclonal antibody (generous gift
            from Steven Clegg).
         </p>
      </div>
      <div id="sec014">
         <h3>Statistical analysis</h3>
         <p>For analysis of the significance of differences, Student’s t-test was used to compare
            data from the two groups. All experiments were made at least three times. A P-value
            of ≤ 0.05 was considered to be statistically significant. RT-PCR statistical analysis
            was performed using the non-parametric One-way ANOVA. P values of ≤ 0.05 were considered
            to be statistically significant.
         </p>
      </div>
      <div id="sec015">
         <h3>Accession Numbers</h3>
         <p>The INSDC accession numbers of usher proteins used in this study are listed below.
            
            <i>Klebsiella pneumoniae</i> HS11286 (AEW59011, AEW59085, AEW60001, AEW61304, AEW6304); 
            <i>Klebsiella pneumoniae</i> MGH 78578 (ABR75716, ABR75717, ABR75730, ABR75944, ABR77102, ABR78394,ABR78675, ABR78797,
            ABR79828, ABR79894); 
            <i>Klebsiella pneumoniae</i> NTUH-K2044 (BAH61229, BAH61295,BAH61460, BAH62173, BAH63378, BAH64776, BAH64779,
            BAH65059, BAH65073); 
            <i>Klebsiella variicola</i> 342 (initially identified as 
            <i>K</i>. 
            <i>pneumoniae</i>: ACI08730, ACI09976, ACI10676, ACI09288, ACI10013, ACI08659, ACI06599, ACI08146,
            ACI08479); 
            <i>Klebsiella pneumoniae</i> KCTC 2242 (AEJ96010, AEJ96079, AEJ96970, AEJ98151, AEJ99466, AEJ99752, AEJ99817,
            AEJ99818, AEJ99901); Klebsiella 
            <i>pneumoniae</i> JM45 (AGT22732, AGT22745, AGT23011, AGT24313, AGT25490, AGT25491, AGT26343, AGT26409);
            
            <i>Klebsiella pneumoniae</i> CG43 (AGX39043, AGX39045, AGX41005, AGX40456); 
            <i>Klebsiella pneumoniae</i> 1084 (AFQ64205, AFQ67915, AFQ67856, AFQ67011, AFQ65856, AFQ67240, AFQ64480, AFQ64205);
            
            <i>Klebsiella pneumoniae</i> Kp13 (AHE42797, AHE42813,AHE43103, AHE43106, AHE44628, AHE45921, AHE46162, AHE46764,
            AHE46837); 
            <i>Klebsiella pneumoniae</i> 30684/NJST258_2 (AHM77757, AHM77758, AHM77775, AHM78064, AHM78067, AHM79571, AHM80854,
            AHM80923, AHM81143, AHM81806, AHM81807, AHM81882); 
            <i>Klebsiella pneumoniae</i> 30660/NJST258_1 (AHM83349, AHM83350, AHM83368, AHM83664, AHM83667, AHM85224, AHM86458,
            AHM86526, AHM86817, AHM87489, AHM87490, AHM87517); 
            <i>Klebsiella pneumoniae</i>
            <div tagx="sc">LM21 (KPLM21_90123, KPLM21_160040, KPLM21_610081, KPLM21_1040039, KPLM21_200008, KPLM21
               _1000123, KPLM21_90135, KPLM21_220012).
            </div>
         </p>
      </div>
   </div>
   <div id="sec016">
      <h2>Results</h2>
      <div id="sec017">
         <h3>Identification and characterization of CU fimbrial loci in 
            <i>K</i>. 
            <i>pneumoniae</i> LM21 genome
         </h3>
         <p>In the 
            <i>K</i>. 
            <i>pneumoniae</i> LM21 genome, a total of eight potential CU fimbrial operons defined as polycistronic
            gene clusters containing at least one usher and one chaperone encoding sequence and
            flanked by one or more genes encoding fimbrial subunits were identified. The genetic
            organization of these CU fimbrial gene clusters (
            <span ref-type="fig" rid="pone.0116215.g001">Fig. 1</span>) was predicted by inspecting individual genes for conserved fimbrial protein domains.
            The usher proteins are members of a classical chaperone/usher family and share conserved
            domains (PFAM00577 and/or COG3188). The presence of usher encoding genes was confirmed
            by PCR using primers P1 to P16 (
            <span ref-type="table" rid="pone.0116215.t001">Table 1</span>). The LM21 
            <i>K</i>. 
            <i>pneumoniae</i> CU loci comprised type 1 and type 3 fimbrial gene clusters 
            <i>fim</i> and 
            <i>mrk</i> that are divergently clustered and transcribed. Five of the other six LM21 CU loci
            showed a strong similarity toward operons found out by Wu 
            <i>et al</i>. [
            <span ref-type="bibr" rid="pone.0116215.ref018">18</span>] in strain NTUH-K2044. Together with type 1 and type 3, they showed a minimum of
            93% of identity (coverage length &gt;98%) when compared to NTUH-K2044 operon subunit,
            with an identity value between 95 and 100% for usher encoding genes. The common CU
            operons between LM21 and NTUH-K2044 strains were 
            <i>kpa</i>, 
            <i>kpb</i>, 
            <i>kpd</i>, 
            <i>kpe</i>, 
            <i>kpg</i>, 
            <i>mrk</i> and 
            <i>fim</i> accordingly to the nomenclature proposed by Wu 
            <i>et al</i>. [
            <span ref-type="bibr" rid="pone.0116215.ref037">37</span>].
            <div tagx="sub">.</div> The eighth LM21 CU operon detected, which corresponds to ORFs KPLM21_200008.KPLM21_200011,
            has not been described so far; as 
            <i>kph</i> and 
            <i>kpi</i> were proposed recently for novel CU clusters in 
            <i>K</i>. 
            <i>pneumoniae</i> strains BJ1-GA and SA1 [
            <span ref-type="bibr" rid="pone.0116215.ref038">38</span>], the novel cluster from LM21 was called 
            <i>kpj</i>. Using these data as well as the sequences of the CU fimbriae usher detected in the
            twelve NCBI sequenced 
            <i>K</i>. 
            <i>pneumoniae</i> genomes (S1), which contain an average of 9 CU operons/genome (minimum 5 and maximum
            12), a circular phylogram was constructed to display the evolutionary relationship
            of these amino acid sequences (See 
            <span ref-type="supplementary-material" rid="pone.0116215.s001">S1 Table</span>). The circular phylogram of 
            <i>K</i>. 
            <i>pneumoniae</i> usher sequences demonstrated that the 
            <i>K</i>. 
            <i>pneumoniae</i> species contains representatives of the six clades defined by Nuccio &amp; Baumler [
            <span ref-type="bibr" rid="pone.0116215.ref003">3</span>] (
            <span ref-type="fig" rid="pone.0116215.g002">Fig. 2</span>). The γ clade was the largest and encompassed 50 CU fimbrial types across five subclades.
         </p>
         <div tagx="fig">
            <div tagx="object-id">10.1371/journal.pone.0116215.g001</div>
            <div tagx="label">Fig 1</div>
            <caption>
               <div tagx="title">Genetic organization of CU fimbrial types identified in 
                  <i>Klebsiella pneumoniae</i> LM21.
               </div>
               <p>The genetic organization of the different fimbrial types is depicted diagrammatically.
                  The designation of putative fimbrial genes and the locus tag of ORFs annotated in
                  the 
                  <i>K</i>. 
                  <i>pneumoniae</i> LM21 genome are indicated. A total of eight fimbrial gene clusters and genes encoding
                  putative regulators are shown. Each of these fimbrial loci is underlined. Fimbriae
                  are grouped according to the Nuccio cladding scheme (Nuccio and Baümler, 2007). Genes
                  are color-coded according to predicted function of the corresponding protein product,
                  with associated Pfam and COG domains indicated (CGO and PF). The scale represents
                  DNA length in kilo base pair. Reference locus tags for individual fimbrial types are
                  displayed under the locus.
               </p>
            </caption>
            <div tagx="graphic"></div>
         </div>
         <div tagx="fig">
            <div tagx="object-id">10.1371/journal.pone.0116215.g002</div>
            <div tagx="label">Fig 2</div>
            <caption>
               <div tagx="title">Circular phylogram of fimbrial usher proteins identified in 
                  <i>K</i>. 
                  <i>pneumoniae</i>.
               </div>
               <p>A total of 90 amino acid sequences deduced from the 7 to 9 CU loci of the twelve 
                  <i>K</i>. 
                  <i>pneumoniae</i> genomes available in the NCBI data bank were used to infer the evolutionary relationship
                  of usher protein. Fimbrial gene clusters were grouped according to the Nuccio subclade
                  system (α, β, ϭ, п, κ, γ) and highlighted in color. 
                  <i>K pneumoniae</i> LM21 usher proteins are leaf labeled in red.
               </p>
            </caption>
            <div tagx="graphic"></div>
         </div>
         <p>The classification of the fimbrial gene clusters into CU clades based on the usher
            sequences was previously shown to correlate with the gene arrangement within clusters
            [
            <span ref-type="bibr" rid="pone.0116215.ref039">39</span>]. The eight LM21 usher loci belong to γκп (for seven of them) and ϭ clade (one of
            them, LM21
            <i>kpd</i>) (
            <span ref-type="fig" rid="pone.0116215.g001">Fig. 1</span>) and are characterized by a common pilus subunit homology domain (PFAM00419). Seven
            CU loci share the same gene organization encoding first the major subunit then a chaperone
            followed by an usher (MCU organization: M for major subunit, C for chaperone and U
            for usher) (
            <span ref-type="fig" rid="pone.0116215.g001">Fig. 1</span>). They are split into different subclades: LM21
            <i>kpa</i>, LM21
            <i>kpe</i>, LM21
            <i>kpg</i>, LM21
            <i>kpj and</i> LM21
            <i>fim</i> are MCUT (T for Tip adhesin)-organized and belong to γ1 subclade. They share, like
            all γ clade members, the PFAM00419 subunit domain and a COG3539 domain, which are
            also found in subunits of γ1 and γ2 fimbriae. The LM21
            <i>mrk</i> operon, which is MCUT-organized, belongs to the γ
            <div tagx="sub">4</div>—fimbriae. Unlike other members of the γ-fimbriae, the LM21
            <i>kpb</i> operon belongs to the γ3-fimbriae with a MCUM (M for Major subunit) organization
            that does not contain the conserved domains PFAM00419 and COG3539 but PFAM04619 subunit
            domain and PFAM02753 chaperone domain (
            <span ref-type="fig" rid="pone.0116215.g001">Fig. 1</span>). Finally, the LM21
            <i>kpd</i> operon belongs to the ϭ cluster with an MCUT core operon structure and has significant
            homology to COG5430 domains in its subunits. Chaperones of the LM21
            <i>kpd</i> operon contain either a COG00345 domain only or a COG3121 domain and a PFAM00345
            domain. No tip adhesin was present in this locus.
         </p>
         <p>Real time RT-PCR analysis performed using RNA samples from bacteria grown in rich
            (LB, DMEM) or minimal (M63B1) media used for phenotypical characterization showed
            that the eight LM21 usher encoding genes were all expressed, whatever the growth conditions.
            Compared to the 
            <i>rpoB</i> housekeeping gene, there was no variation of usher gene expression between the different
            media, excepted for 
            <i>kpgC</i> which was expressed at higher level in DMEM compared to LB (p&lt;0.05; ANOVA) (
            <span ref-type="table" rid="pone.0116215.t003">Table 3</span>).
         </p>
         <div tagx="table-wrap">
            <div tagx="object-id">10.1371/journal.pone.0116215.t003</div>
            <div tagx="label">Table 3</div>
            <caption>
               <div tagx="title">RT-PCR analysis of usher genes expression in M63B1, LB and DMEM media.</div>
            </caption>
            <div tagx="alternatives">
               <div tagx="graphic"></div>
               <table frame="hsides" rules="groups">
                  <colgroup span="1">
                     <col align="left" valign="middle" span="1"></col>
                     <col align="left" valign="middle" span="1"></col>
                     <col align="left" valign="middle" span="1"></col>
                     <col align="left" valign="middle" span="1"></col>
                  </colgroup>
                  <thead>
                     <tr>
                        <div tagx="th"></div>
                        <div tagx="th">Culture medium</div>
                     </tr>
                     <tr>
                        <div tagx="th">target gene</div>
                        <div tagx="th">M63B1</div>
                        <div tagx="th">LB</div>
                        <div tagx="th">DMEM</div>
                     </tr>
                  </thead>
                  <div tagx="tbody">
                     <tr>
                        <td align="left" rowspan="1" colspan="1">
                           <b>
                              <i>kpaC</i>
                           </b>
                        </td>
                        <td align="left" rowspan="1" colspan="1">46.6 ± 10.7</td>
                        <td align="left" rowspan="1" colspan="1">51.6 ± 8.2</td>
                        <td align="left" rowspan="1" colspan="1">49.6 ± 21.1</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">
                           <b>
                              <i>kpbC</i>
                           </b>
                        </td>
                        <td align="left" rowspan="1" colspan="1">7.1 ± 1.2</td>
                        <td align="left" rowspan="1" colspan="1">7.5 ± 0.8</td>
                        <td align="left" rowspan="1" colspan="1">7.1 ± 0.4</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">
                           <b>
                              <i>kpdC</i>
                           </b>
                        </td>
                        <td align="left" rowspan="1" colspan="1">52.0 ± 2.2</td>
                        <td align="left" rowspan="1" colspan="1">40.9 ± 3.2</td>
                        <td align="left" rowspan="1" colspan="1">55.1 ± 11.0</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">
                           <b>
                              <i>kpeC</i>
                           </b>
                        </td>
                        <td align="left" rowspan="1" colspan="1">441.3 ± 81.7</td>
                        <td align="left" rowspan="1" colspan="1">382.4 ± 83.1</td>
                        <td align="left" rowspan="1" colspan="1">344.4 ± 58.5</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">
                           <b>
                              <i>kpgC</i>
                           </b>
                        </td>
                        <td align="left" rowspan="1" colspan="1">93.6 ± 4.2</td>
                        <td align="left" rowspan="1" colspan="1">111.6 ± 20.4</td>
                        <td align="left" rowspan="1" colspan="1">75.3 ± 8.5</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">
                           <b>
                              <i>kpjC</i>
                           </b>
                        </td>
                        <td align="left" rowspan="1" colspan="1">94.6 ± 9.9</td>
                        <td align="left" rowspan="1" colspan="1">81.4 ± 20.3</td>
                        <td align="left" rowspan="1" colspan="1">92.6 ± 5.7</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">
                           <b>
                              <i>mrkC</i>
                           </b>
                        </td>
                        <td align="left" rowspan="1" colspan="1">16.6 ± 0.6</td>
                        <td align="left" rowspan="1" colspan="1">17.2 ± 2.9</td>
                        <td align="left" rowspan="1" colspan="1">14.7 ± 1.5</td>
                     </tr>
                     <tr>
                        <td align="left" rowspan="1" colspan="1">
                           <b>
                              <i>fimC</i>
                           </b>
                        </td>
                        <td align="left" rowspan="1" colspan="1">30.5 ± 1.4</td>
                        <td align="left" rowspan="1" colspan="1">34.1 ± 1.5</td>
                        <td align="left" rowspan="1" colspan="1">28.2 ± 4.9</td>
                     </tr>
                  </div>
               </table>
            </div>
            <div tagx="table-wrap-foot">
               <div tagx="fn">
                  <p>Results were expressed in fold-decreased expression (±SD) compared to 
                     <i>rpoB</i> housekeeping gene expression level. Expression levels were compared by nonparametric
                     one-way ANOVA, comparing expression of each usher gene in the three different media.
                     Only 
                     <i>kpgC</i> expression was higher in DMEM compared to LB; p &lt; 0.05 (ANOVA).
                  </p>
               </div>
            </div>
         </div>
         <p>To investigate the role of the LM21 
            <i>K</i>. 
            <i>pneumoniae</i> CU loci, isogenic deletion mutants were created by allelic replacement of each of
            the eight potential encoding usher genes. The growth rate of each mutant was similar
            to that of the wild type (
            <span ref-type="supplementary-material" rid="pone.0116215.s003">S1 Fig.</span>). Western blot analysis performed with MrkA specific antibodies showed that all usher
            deleted mutants, except Δ
            <i>mrkC</i>, expressed type 3 pili at their cell surface (
            <span ref-type="supplementary-material" rid="pone.0116215.s004">S2 Fig.</span>). In addition, assessment of the expression of the 7 usher genes in Δ
            <i>mrkC</i> mutant by RT-PCR indicated that there was no significant variation compared to the
            wild-type strain (
            <span ref-type="supplementary-material" rid="pone.0116215.s002">S2 Table</span>).
         </p>
      </div>
      <div id="sec018">
         <h3>Adhesion and colonization phenotype of usher deletion mutants</h3>
         <p>Determination of biofilm biomass using CV staining in microtiter plates and biomass
            determination in the microfermentor device indicated that three mutants (LM21Δ
            <i>kpaC</i>, LM21Δ
            <i>kpgC</i> and LM21Δ
            <i>mrkC</i>) were impaired in their biofilm formation ability compared to the wild type 
            <i>K</i>. 
            <i>pneumoniae</i> LM21 strain (
            <span ref-type="fig" rid="pone.0116215.g003">Fig. 3</span>). The ability to form biofilm was partially restored by transcomplementation with
            the wild type usher encoding gene (up to 80% of the wild type level), whatever the
            biofilm formation model used (
            <span ref-type="fig" rid="pone.0116215.g003">Fig. 3A and 3B</span>).
         </p>
         <div tagx="fig">
            <div tagx="object-id">10.1371/journal.pone.0116215.g003</div>
            <div tagx="label">Fig 3</div>
            <caption>
               <div tagx="title">Biofilm formation capacity of the 
                  <i>K</i>. 
                  <i>pneumoniae</i> LM21
                  <i>Δusher</i> mutants strains and, for three of them, their transcomplemented strains.
               </div>
               <p>Biofilms developed were quantified (A) by crystal violet staining on microtiter plates
                  after 4 hours of incubation and (B) by CFU determination after 24 hours of incubation
                  in the microfermentor model, as described in experimental procedures. Data are means
                  of measurement made in triplicate. The biofilm formation ability of the mutant strains
                  is expressed as a percentage of LM21 wild type biofilm, set to 100% (OD
                  <div tagx="sub">600</div> and CFU values for the 
                  <i>K</i>. 
                  <i>pneumoniae</i> LM21 wild type are respectively 0.52 and 1.24x10
                  <sup>9</sup>). The error bars represent standard errors of the means. Significant differences
                  are indicated by * and ** for p &lt; 0.05 and p&lt;0.01, respectively (Student’s t-test).
               </p>
            </caption>
            <div tagx="graphic"></div>
         </div>
         <p>The eight usher-deleted mutants were also tested for their ability to adhere to human
            intestinal Int-407 cells. Two of the mutants (LM21Δ
            <i>kpgC</i> and LM21Δ
            <i>mrkC</i>) adhered less than the wild type strain LM21, (75% and 20%, respectively) whereas
            the six other mutants did not show any difference compared to the parental strain.
            The adhesion phenotype was partially restored by the trans-complemented strains (up
            to 80% of the wild type level) (
            <span ref-type="fig" rid="pone.0116215.g004">Fig. 4</span>).
         </p>
         <div tagx="fig">
            <div tagx="object-id">10.1371/journal.pone.0116215.g004</div>
            <div tagx="label">Fig 4</div>
            <caption>
               <div tagx="title">Adhesion assays to Int-407 cells of the LM21
                  <i>Δusher</i> mutants strains and, ² for two of them, their transcomplemented mutants.
               </div>
               <p>Results are expressed as the percentages of LM21 wild type adhesion, set to 100% (CFU
                  value for 
                  <i>K</i>. 
                  <i>pneumoniae</i> LM21 wild type was 1.93x10
                  <sup>9</sup>). Data are the means of measurements made in biological and technical triplicate.
                  Significant differences are indicated by * p &lt; 0.05 and ** for p&lt;0.01 (Student’s t-test).
               </p>
            </caption>
            <div tagx="graphic"></div>
         </div>
         <p>Adhesion assays performed with whole 15-day-old 
            <i>A</i>. 
            <i>thaliana</i> specimens showed that four mutants, LM21Δ
            <i>kpeC</i>, LM21Δ
            <i>kpjC</i>, LM21Δ
            <i>fimC</i> and LM21Δ
            <i>mrkC</i>, adhered at significantly lower levels than those of the wild type strain (P&lt;0.01
            with LM21Δ
            <i>kpjC</i> and P&lt;0.05 for LM21Δ
            <i>kpeC</i>, LM21Δ
            <i>fimC)</i> (
            <span ref-type="fig" rid="pone.0116215.g005">Fig. 5</span>).
         </p>
         <div tagx="fig">
            <div tagx="object-id">10.1371/journal.pone.0116215.g005</div>
            <div tagx="label">Fig 5</div>
            <caption>
               <div tagx="title">Adhesion assays to 
                  <i>Arabidopsis thaliana</i> whole seedlings of the 
                  <i>K</i>. 
                  <i>pneumoniae</i> LM21
                  <i>Δusher</i> mutants strains and, for four of them, their transcomplemented mutants.
               </div>
               <p>Results are expressed as the percentages of LM21 wild type adhesion, set to 100% (CFU
                  value for 
                  <i>K</i>. 
                  <i>pneumoniae</i> LM21 wild type was 1.47.10
                  <sup>7</sup>). Data are the means of measurements made in biological and technical triplicate.
                  Significant differences are indicated by * and ** for p &lt; 0.05 and p&lt;0.01 respectively
                  (Student’s t-test).
               </p>
            </caption>
            <div tagx="graphic"></div>
         </div>
         <p>Determination of the ability of each usher mutant to colonize mice intestinal tract
            concurrently with the parental wild type strain indicated that four mutants, LM21Δ
            <i>kpgC</i>, LM21Δ
            <i>kpjC</i>, LM21Δ
            <i>fimC</i> and LM21Δ
            <i>mrkC</i>, were significantly impaired compared to the levels of the wild type strain (
            <span ref-type="fig" rid="pone.0116215.g006">Fig. 6A</span>) (P&lt;0.01 by the student’s test for LM21Δ
            <i>kpgC and</i> LM21
            <i>mrkC</i> P&lt;0.05 for 
            <i>fimC</i> and P&lt;0.01 for LM21Δ
            <i>kpjC</i>). Further colonization assays performed with the highest attenuated mutant, LM21Δ
            <i>kpjC</i>, showed that this mutant alone was also unable to colonize the intestinal tract and
            was outcompeted by its trans-complemented LM21Δ
            <i>kpjC/</i>pSTAB-
            <i>kpjC</i> mutant (
            <span ref-type="fig" rid="pone.0116215.g006">Fig. 6B</span>).
         </p>
         <div tagx="fig">
            <div tagx="object-id">10.1371/journal.pone.0116215.g006</div>
            <div tagx="label">Fig 6</div>
            <caption>
               <div tagx="title">Colonization assays in the murine model of 
                  <i>K. pneumoniae</i> LM21
                  <i>Δusher</i> mutants strains and trans-complemented mutants.
               </div>
               <p>(A) The colonization properties of the strains are shown as the competition between
                  the wild type and the eight isogenic mutants. (B) For mutants showing an highly attenuated
                  phenotype, individual assays involving the mutant alone and in competition assays
                  with its trans-complemented strain were conducted. Data are means of measurement made
                  with 5 mice per group. Significant differences are indicated by * and ** for p &lt; 0.05
                  and p&lt;0.01, respectively (Student’s t-test).
               </p>
            </caption>
            <div tagx="graphic"></div>
         </div>
      </div>
   </div>
   <div id="sec019">
      <h2>Discussion</h2>
      <p>To detect CU clusters, the choice of the usher as target gene was conditioned by the
         fact that one usher encoding gene is ubiquitously associated with all CU gene clusters
         and is present in a single copy in all CU gene clusters so far described. A total
         of eight potential usher-encoding genes were detected in 
         <i>K</i>. 
         <i>pneumoniae</i> LM21 genome, all of them harboring adjacent cognate chaperone-encoding and fimbrial
         subunit-encoding genes. Whereas genes for CU pathways have been shown to be encoded
         on both chromosomal and plasmid locations [
         <span ref-type="bibr" rid="pone.0116215.ref040">40</span>,
         <span ref-type="bibr" rid="pone.0116215.ref041">41</span>], the eight CU loci detected in the genome of 
         <i>K</i>. 
         <i>pneumoniae</i> LM21 were located on the chromosome and most of them belonged to the γ
         <div tagx="sub">1</div> subclade. This result highlights the main role of this subclade in bacterial survival
         and pathogeny processes as previously suggested [
         <span ref-type="bibr" rid="pone.0116215.ref041">41</span>]. In addition, analysis of the surrounding regions of each CU operon did not reveal
         potential mobile DNA sequences (data not shown), suggesting that these operons have
         not been acquired through recent lateral gene transfers [
         <span ref-type="bibr" rid="pone.0116215.ref042">42</span>].
      </p>
      <p>Of the eight CU-loci detected in 
         <i>K</i>. 
         <i>pneumoniae</i> LM21 genome, two corresponded to previously well-characterized CU operons. The locus
         with ORFs KPLM21_90133 to KPLM21_90137 was identified as the type 3-encoding 
         <i>mrk</i> operon previously described in all members of the family 
         <i>Enterobacteriaceae</i>. While first identified and characterized in 
         <i>Klebsiella</i>, type 3 pili are commonly found in other 
         <i>Enterobacteriaceae</i>, and 
         <i>mrk</i> gene clusters are mostly chromosome-borne [
         <span ref-type="bibr" rid="pone.0116215.ref007">7</span>,
         <span ref-type="bibr" rid="pone.0116215.ref043">43</span>,
         <span ref-type="bibr" rid="pone.0116215.ref044">44</span>]. m
         <i>rk</i> operon belongs to γ
         <div tagx="sub">4</div> clade and shares the core operon structure present in all members of the γ
         <div tagx="sub">4</div> clade [
         <span ref-type="bibr" rid="pone.0116215.ref003">3</span>]. The type 3 fimbriae are characterized by their ability to agglutinate erythrocytes
         treated with tannic acid 
         <i>in vitro</i>, and this phenotype has been referred to as the mannose-resistant 
         <i>Klebsiella-</i>like hemagglutination (MR/K) reaction [
         <span ref-type="bibr" rid="pone.0116215.ref045">45</span>]. Type 3 fimbriae have also been shown to mediate attachment to endothelial and bladder
         epithelial cell lines and to play a role in biofilm formation on abiotic surfaces
         and surfaces coated with host-derived materials [
         <span ref-type="bibr" rid="pone.0116215.ref002">2</span>,
         <span ref-type="bibr" rid="pone.0116215.ref046">46</span>–
         <span ref-type="bibr" rid="pone.0116215.ref051">51</span>]. Previous studies have also reported that type 3 fimbriae are efficient in promoting
         enterobacterial adherence to the roots of various grasses and cereals [
         <span ref-type="bibr" rid="pone.0116215.ref036">36</span>]. In our study, deletion of the usher-encoding ORF of this locus impaired all tested
         phenotypes, including murine intestinal colonization contrary to the results obtained
         after deletion of the whole 
         <i>mrk</i> operon by Struve 
         <i>et al</i>. [
         <span ref-type="bibr" rid="pone.0116215.ref047">47</span>]. It has been previously demonstrated that type 3 fimbriae deletion down-regulates
         type 1 fimbrial expression [
         <span ref-type="bibr" rid="pone.0116215.ref002">2</span>]. In this study, deletion of 
         <i>mrkC</i> did not significantly modify the expression of the 7 other usher genes in strain
         LM21 (
         <span ref-type="supplementary-material" rid="pone.0116215.s002">S2 Table</span>), indicating no cross-regulation. Altogether, these results suggest that LM21 type
         3 pili is a major actor in bacterial adherence both 
         <i>in vitro</i> and 
         <i>in vivo</i> by means of a large array of targets and tissue tropisms.
      </p>
      <p>Type 1 fimbriae-encoding operon was the second previously well-known CU operon detected
         in our study. These fimbriae are found in virtually all members of the family 
         <i>Enterobacteriaceae</i> [
         <span ref-type="bibr" rid="pone.0116215.ref012">12</span>]. The genetic organization of the 
         <i>K</i>. 
         <i>pneumoniae fim</i> operon resembles that of 
         <i>E</i>. 
         <i>coli fim</i> and contains homologs of all nine 
         <i>fim</i> genes described previously [
         <span ref-type="bibr" rid="pone.0116215.ref003">3</span>,
         <span ref-type="bibr" rid="pone.0116215.ref037">37</span>]. 
         <i>fimC</i> gene knock-out experiment in 
         <i>K</i>. 
         <i>pneumoniae</i> LM21 did not impair its biofilm formation capacity, as previously described for other
         
         <i>K</i>. 
         <i>pneumoniae</i> strains [
         <span ref-type="bibr" rid="pone.0116215.ref051">51</span>–
         <span ref-type="bibr" rid="pone.0116215.ref054">54</span>]. However, these results are at variance with those obtained in the closely model
         
         <i>E</i>. 
         <i>coli</i>, in which type 1 fimbriae have been shown to promote biofilm formation [
         <span ref-type="bibr" rid="pone.0116215.ref053">53</span>]. This intriguing difference could be related to the characteristic production of
         copious amounts of capsular material by 
         <i>K</i>. 
         <i>pneumoniae</i> that impedes type 1 functionality during biofilm formation due to the shortness of
         these pili [
         <span ref-type="bibr" rid="pone.0116215.ref055">55</span>,
         <span ref-type="bibr" rid="pone.0116215.ref056">56</span>].
      </p>
      <p>In agreement with van Aarsten et 
         <i>al</i>.,[
         <span ref-type="bibr" rid="pone.0116215.ref057">57</span>], we observed no significant difference between the LM21Δ
         <i>fimC</i> mutant and its parental strains in the 
         <i>in vitro</i> animal cell adhesion assay. However, this mutant was impaired in its ability to adhere
         to 
         <i>Arabidopsis</i> seedlings (
         <span ref-type="fig" rid="pone.0116215.g005">Fig. 5</span>), suggesting that the type 1 fimbriae are involved in 
         <i>K</i>. 
         <i>pneumoniae</i> plant adhesion, as in the grass model of Haahtela 
         <i>et al</i>. [
         <span ref-type="bibr" rid="pone.0116215.ref036">36</span>]. In addition, and in contrast with previous reports [
         <span ref-type="bibr" rid="pone.0116215.ref054">54</span>,
         <span ref-type="bibr" rid="pone.0116215.ref057">57</span>], murine intestinal co-colonization assay performed with LM21Δ
         <i>fimC</i> mutant showed that the mutant had a lower colonization capacity than the wild type.
      </p>
      <p>In addition to type 3 and type 1-encoding operons, 
         <i>K</i>. 
         <i>pneumoniae</i> LM21 genome harbored six other CU fimbriae systems. The newly identified 
         <i>kpj</i> cluster was shown to be involved in adhesion to 
         <i>Arabidopsis</i> tissues and in murine colonization. Determination of the ability of each mutant to
         colonize the murine intestinal tract was initially performed in co-colonization assays
         including the wild-type strain. To avoid misinterpretation due to concurrent colonization
         processes that potentially influence the host immune status and thus modify the capacity
         of both microorganisms to establish, the LM21Δ 
         <i>kpj</i> mutant was assessed individually in the murine model; the bacterial load in the animals
         feces rapidly declined, indicating the mutant was also impaired in its colonization
         capacity when given alone (
         <span ref-type="fig" rid="pone.0116215.g006">Fig. 6B</span>). Interestingly, the LM21Δ
         <i>kpjC</i> mutant was not impaired in its capacity to adhere to Int-407 cells, suggesting the
         adhesin potentially encoded by this operon does not recognize receptors at the surface
         of intestinal cells but rather interacts with other gastro-intestinal components.
      </p>
      <p>Deletion of the LM21
         <i>kpjC</i> usher-encoding gene significantly decreased its ability to form biofilm and its ability
         to adhere 
         <i>in vitro</i> to Int-407 and plant cells, whereas deletion of LM21
         <i>kpaC</i> usher gene was only associated with a decreased ability to form biofilm in both early
         and late stages. These results suggest that the different fimbrial adhesins harbored
         by 
         <i>K</i>. 
         <i>pneumoniae</i> have specific role allowing the bacteria to adhere to different receptors present
         in different niche and environnement. It has previously been demonstrated that type
         1 fimbrial expression is up-regulated in wild type 
         <i>K</i>. 
         <i>pneumoniae</i> infecting the bladder, but is down-regulated in cells colonizing the intestinal tract
         or infecting the lungs [
         <span ref-type="bibr" rid="pone.0116215.ref054">54</span>].
      </p>
      <p>Regarding the loci LM21
         <i>kpb</i>, LM21
         <i>kpd</i> and LM21
         <i>kpe</i>, no clear 
         <i>in vitro</i> or 
         <i>in vivo</i> role was identified whatever the phenotype investigated, except for LM21
         <i>kpe</i>, for which deletion induced a slightly decreased adhesion in the plant model. We
         demonstrated in this study that the 8 LM21 CU usher genes were expressed 
         <i>in vitro</i> even though specific demonstration of the presence of corresponding fimbriae at the
         bacterial cell surface still requires further investigations. Numerous cryptic fimbrial
         operons have been detected in the genomes of 
         <i>E</i>. 
         <i>coli</i> K-12, 
         <i>E</i>. 
         <i>coli</i> O157:H7 and 
         <i>Salmonella</i>, whose expression is subject to phase variation in response to environment cues [
         <span ref-type="bibr" rid="pone.0116215.ref058">58</span>–
         <span ref-type="bibr" rid="pone.0116215.ref060">60</span>]. Each bacterium likely expresses a few pili type at a given time according to its
         growth or virulence requirements. Besides, by analogy with other CU systems, the upregulation
         of expression and biosynthesis of fimbriae could involve a complex interplay of multiple
         transcriptional regulator, invertible promoter switchs or DNA methylation-based systems,
         in addition to a potential regulation by the levels of expression of other surface
         components [
         <span ref-type="bibr" rid="pone.0116215.ref005">5</span>,
         <span ref-type="bibr" rid="pone.0116215.ref061">61</span>–
         <span ref-type="bibr" rid="pone.0116215.ref063">63</span>].
      </p>
      <p>In conclusion, we report the characterization of eight CU loci on the genome of 
         <i>K</i>. 
         <i>pneumoniae</i> LM21. Using several 
         <i>in vitro</i> and 
         <i>in vivo</i> experimental models, we were able to show the involvement of five of them in bacterial
         adhesion or colonization processes, demonstrating therefore the large adhesion capacity
         of this species. However, complete demonstration of its adhesion capacities still
         requires the identification of the substrates recognized by the potential adhesins
         harbored by the non-yet phenotypically characterized CU operons. In addition, epidemiological
         studies assessing the presence and the expression of these loci in a large panel of
         
         <i>K</i>. 
         <i>pneumoniae</i> isolates from different sources will complete the elucidation of their respective
         role.
      </p>
   </div>
   <div id="sec020">
      <h2>Supporting Information</h2>
      <div tagx="supplementary-material">
         <div tagx="label">S1 Table</div>
         <caption>
            <div tagx="title">Strains labels and accession numbers of gene encoding usher identified in 
               <i>Klebsiella pneumoniae</i> strains genomes present in NCBI and in this study.
            </div>
            <p>(PDF)</p>
         </caption>
         <div tagx="media">
            <caption>
               <p>Click here for additional data file.</p>
            </caption>
         </div>
      </div>
      <div tagx="supplementary-material">
         <div tagx="label">S2 Table</div>
         <caption>
            <div tagx="title">RT-PCR analysis of the expression of usher genes in the 
               <i>mrkC</i>-deleted mutant (type-3 pili usher) and the LM21 wild-type strain.
            </div>
            <p>(PDF)</p>
         </caption>
         <div tagx="media">
            <caption>
               <p>Click here for additional data file.</p>
            </caption>
         </div>
      </div>
      <div tagx="supplementary-material">
         <div tagx="label">S1 Fig</div>
         <caption>
            <div tagx="title">Growth curves of LM21 wild-type strain and its usher-deleted mutants.</div>
            <p>Bacterial cells were collected every 30 min for 8.5 hours and plated on media. Results
               are expressed as the number of CFU/ml.
            </p>
            <p>(TIF)</p>
         </caption>
         <div tagx="media">
            <caption>
               <p>Click here for additional data file.</p>
            </caption>
         </div>
      </div>
      <div tagx="supplementary-material">
         <div tagx="label">S2 Fig</div>
         <caption>
            <div tagx="title">Western blot of bacterial surface extracts of the usher-deleted mutants and the LM21
               wild-type strain.
            </div>
            <p>The figure shows an immunoblot of a gel on which 5 μg of extract from each bacterial
               strain has been loaded. The gel was immunostained with an antibody that recognizes
               the major subunit of type 3 pili (MrkA). MW, molecular weight size marker.
            </p>
            <p>(TIF)</p>
         </caption>
         <div tagx="media">
            <caption>
               <p>Click here for additional data file.</p>
            </caption>
         </div>
      </div>
   </div>
   <div tagx="back">
      <div tagx="ack">
         <p>We thank LABGeM team and the National Infrastructure « France Genomique » for the
            annotation of 
            <i>K</i>. 
            <i>pneumoniae LM21</i> genome. We thank Marie Claude Espagnol, Aline V. Probst and Christophe Tatout for
            generously providing the Arabidopsis thaliana plants used in our study. We also thank
            Michael Picard and Anne Pracros for their technical assistance.
         </p>
      </div>
      <div tag="ref-list">
         <ul>
            <div tagx="title">References</div>
            <li tag="ref">
               <div tagx="label">1</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Thanassi</span>
                     <span tagx="given-names">DG</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Bliska</span>
                     <span tagx="given-names">JB</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Christie</span>
                     <span tagx="given-names">PJ</span>
                  </span> (
                  <span tagx="year">2012</span>) 
                  <span tagx="article-title">Surface organelles assembled by secretion systems of Gram-negative bacteria: diversity
                     in structure and function
                  </span>. 
                  <span tagx="source">FEMS Microbiol Rev</span>
                  <span tagx="volume">36</span>: 
                  <span tagx="fpage">1046</span>–
                  <span tagx="lpage">1082</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1111/j.1574-6976.2012.00342.x</div>
                  </div>
                  <div tagx="object-id">22545799</div>
                  <div tagx="pub-id">22545799</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">2</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Schroll</span>
                     <span tagx="given-names">C</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Barken</span>
                     <span tagx="given-names">KB</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Krogfelt</span>
                     <span tagx="given-names">KA</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Struve</span>
                     <span tagx="given-names">C</span>
                  </span> (
                  <span tagx="year">2010</span>) 
                  <span tagx="article-title">Role of type 1 and type 3 fimbriae in Klebsiella pneumoniae biofilm formation</span>. 
                  <span tagx="source">BMC Microbiol</span>
                  <span tagx="volume">10</span>: 
                  <span tagx="fpage">179</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1186/1471-2180-10-179</div>
                  </div>
                  <div tagx="object-id">20573190</div>
                  <div tagx="pub-id">20573190</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">3</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Nuccio</span>
                     <span tagx="given-names">S-P</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Bäumler</span>
                     <span tagx="given-names">AJ</span>
                  </span> (
                  <span tagx="year">2007</span>) 
                  <span tagx="article-title">Evolution of the chaperone/usher assembly pathway: fimbrial classification goes Greek</span>. 
                  <span tagx="source">Microbiol Mol Biol Rev MMBR</span>
                  <span tagx="volume">71</span>: 
                  <span tagx="fpage">551</span>–
                  <span tagx="lpage">575</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1128/MMBR.00014-07</div>
                  </div>
                  <div tagx="object-id">18063717</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">4</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Felek</span>
                     <span tagx="given-names">S</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Jeong</span>
                     <span tagx="given-names">JJ</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Runco</span>
                     <span tagx="given-names">LM</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Murray</span>
                     <span tagx="given-names">S</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Thanassi</span>
                     <span tagx="given-names">DG</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">2011</span>) 
                  <span tagx="article-title">Contributions of chaperone/usher systems to cell binding, biofilm formation and Yersinia
                     pestis virulence
                  </span>. 
                  <span tagx="source">Microbiol Read Engl</span>
                  <span tagx="volume">157</span>: 
                  <span tagx="fpage">805</span>–
                  <span tagx="lpage">818</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1099/mic.0.044826-0</div>
                  </div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">5</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Korea</span>
                     <span tagx="given-names">C-G</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Badouraly</span>
                     <span tagx="given-names">R</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Prevost</span>
                     <span tagx="given-names">M-C</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Ghigo</span>
                     <span tagx="given-names">J-M</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Beloin</span>
                     <span tagx="given-names">C</span>
                  </span> (
                  <span tagx="year">2010</span>) 
                  <span tagx="article-title">Escherichia coli K-12 possesses multiple cryptic but functional chaperone-usher fimbriae
                     with distinct surface specificities
                  </span>. 
                  <span tagx="source">Environ Microbiol</span>
                  <span tagx="volume">12</span>: 
                  <span tagx="fpage">1957</span>–
                  <span tagx="lpage">1977</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1111/j.1462-2920.2010.02202.x</div>
                  </div>
                  <div tagx="object-id">20345943</div>
                  <div tagx="pub-id">20345943</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">6</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Van der Velden</span>
                     <span tagx="given-names">AW</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Bäumler</span>
                     <span tagx="given-names">AJ</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Tsolis</span>
                     <span tagx="given-names">RM</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Heffron</span>
                     <span tagx="given-names">F</span>
                  </span> (
                  <span tagx="year">1998</span>) 
                  <span tagx="article-title">Multiple fimbrial adhesins are required for full virulence of Salmonella typhimurium
                     in mice
                  </span>. 
                  <span tagx="source">Infect Immun</span>
                  <span tagx="volume">66</span>: 
                  <span tagx="fpage">2803</span>–
                  <span tagx="lpage">2808</span>
                  <div tagx="object-id">9596751</div>
                  <div tagx="pub-id">9596751</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">7</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Burmølle</span>
                     <span tagx="given-names">M</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Bahl</span>
                     <span tagx="given-names">MI</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Jensen</span>
                     <span tagx="given-names">LB</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Sørensen</span>
                     <span tagx="given-names">SJ</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Hansen</span>
                     <span tagx="given-names">LH</span>
                  </span> (
                  <span tagx="year">2008</span>) 
                  <span tagx="article-title">Type 3 fimbriae, encoded by the conjugative plasmid pOLA52, enhance biofilm formation
                     and transfer frequencies in Enterobacteriaceae strains
                  </span>. 
                  <span tagx="source">Microbiol Read Engl</span>
                  <span tagx="volume">154</span>: 
                  <span tagx="fpage">187</span>–
                  <span tagx="lpage">195</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1099/mic.0.2007/010454-0</div>
                  </div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">8</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Yen</span>
                     <span tagx="given-names">MR</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Peabody</span>
                     <span tagx="given-names">CR</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Partovi</span>
                     <span tagx="given-names">SM</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Zhai</span>
                     <span tagx="given-names">Y</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Tseng</span>
                     <span tagx="given-names">YH</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">2002</span>) 
                  <span tagx="article-title">Protein-translocating outer membrane porins of Gram-negative bacteria</span>. 
                  <span tagx="source">Biochim Biophys Acta</span>
                  <span tagx="volume">1562</span>: 
                  <span tagx="fpage">6</span>–
                  <span tagx="lpage">31</span>
                  <div tagx="object-id">11988218</div>
                  <div tagx="pub-id">11988218</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">9</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Podschun</span>
                     <span tagx="given-names">R</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Ullmann</span>
                     <span tagx="given-names">U</span>
                  </span> (
                  <span tagx="year">1998</span>) 
                  <span tagx="article-title">Klebsiella spp. as nosocomial pathogens: epidemiology, taxonomy, typing methods, and
                     pathogenicity factors
                  </span>. 
                  <span tagx="source">Clin Microbiol Rev</span>
                  <span tagx="volume">11</span>: 
                  <span tagx="fpage">589</span>–
                  <span tagx="lpage">603</span>
                  <div tagx="object-id">9767057</div>
                  <div tagx="pub-id">9767057</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">10</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Sahly</span>
                     <span tagx="given-names">H</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Podschun</span>
                     <span tagx="given-names">R</span>
                  </span> (
                  <span tagx="year">1997</span>) 
                  <span tagx="article-title">Clinical, bacteriological, and serological aspects of Klebsiella infections and their
                     spondylarthropathic sequelae
                  </span>. 
                  <span tagx="source">Clin Diagn Lab Immunol</span>
                  <span tagx="volume">4</span>: 
                  <span tagx="fpage">393</span>–
                  <span tagx="lpage">399</span>
                  <div tagx="object-id">9220153</div>
                  <div tagx="pub-id">9220153</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">11</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Abraham</span>
                     <span tagx="given-names">SN</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Sun</span>
                     <span tagx="given-names">D</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Dale</span>
                     <span tagx="given-names">JB</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Beachey</span>
                     <span tagx="given-names">EH</span>
                  </span> (
                  <span tagx="year">1988</span>) 
                  <span tagx="article-title">Conservation of the D-mannose-adhesion protein among type 1 fimbriated members of
                     the family Enterobacteriaceae
                  </span>. 
                  <span tagx="source">Nature</span>
                  <span tagx="volume">336</span>: 
                  <span tagx="fpage">682</span>–
                  <span tagx="lpage">684</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1038/336682a0</div>
                  </div>
                  <div tagx="object-id">2904657</div>
                  <div tagx="pub-id">2904657</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">12</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Klemm</span>
                     <span tagx="given-names">P</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Schembri</span>
                     <span tagx="given-names">MA</span>
                  </span> (
                  <span tagx="year">2000</span>) 
                  <span tagx="article-title">Bacterial adhesins: function and structure</span>. 
                  <span tagx="source">Int J Med Microbiol IJMM</span>
                  <span tagx="volume">290</span>: 
                  <span tagx="fpage">27</span>–
                  <span tagx="lpage">35</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1016/S1438-4221(00)80102-2</div>
                  </div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">13</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Mulvey</span>
                     <span tagx="given-names">MA</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Lopez-Boado</span>
                     <span tagx="given-names">YS</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Wilson</span>
                     <span tagx="given-names">CL</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Roth</span>
                     <span tagx="given-names">R</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Parks</span>
                     <span tagx="given-names">WC</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">1998</span>) 
                  <span tagx="article-title">Induction and evasion of host defenses by type 1-piliated uropathogenic Escherichia
                     coli
                  </span>. 
                  <span tagx="source">Science</span>
                  <span tagx="volume">282</span>: 
                  <span tagx="fpage">1494</span>–
                  <span tagx="lpage">1497</span>
                  <div tagx="object-id">9822381</div>
                  <div tagx="pub-id">9822381</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">14</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Hornick</span>
                     <span tagx="given-names">DB</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Allen</span>
                     <span tagx="given-names">BL</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Horn</span>
                     <span tagx="given-names">MA</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Clegg</span>
                     <span tagx="given-names">S</span>
                  </span> (
                  <span tagx="year">1992</span>) 
                  <span tagx="article-title">Adherence to respiratory epithelia by recombinant Escherichia coli expressing Klebsiella
                     pneumoniae type 3 fimbrial gene products
                  </span>. 
                  <span tagx="source">Infect Immun</span>
                  <span tagx="volume">60</span>: 
                  <span tagx="fpage">1577</span>–
                  <span tagx="lpage">1588</span>
                  <div tagx="object-id">1312518</div>
                  <div tagx="pub-id">1312518</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">15</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Schurtz</span>
                     <span tagx="given-names">TA</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Hornick</span>
                     <span tagx="given-names">DB</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Korhonen</span>
                     <span tagx="given-names">TK</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Clegg</span>
                     <span tagx="given-names">S</span>
                  </span> (
                  <span tagx="year">1994</span>) 
                  <span tagx="article-title">The type 3 fimbrial adhesin gene (mrkD) of Klebsiella species is not conserved among
                     all fimbriate strains
                  </span>. 
                  <span tagx="source">Infect Immun</span>
                  <span tagx="volume">62</span>: 
                  <span tagx="fpage">4186</span>–
                  <span tagx="lpage">4191</span>
                  <div tagx="object-id">7927674</div>
                  <div tagx="pub-id">7927674</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">16</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Tarkkanen</span>
                     <span tagx="given-names">AM</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Virkola</span>
                     <span tagx="given-names">R</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Clegg</span>
                     <span tagx="given-names">S</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Korhonen</span>
                     <span tagx="given-names">TK</span>
                  </span> (
                  <span tagx="year">1997</span>) 
                  <span tagx="article-title">Binding of the type 3 fimbriae of Klebsiella pneumoniae to human endothelial and urinary
                     bladder cells
                  </span>. 
                  <span tagx="source">Infect Immun</span>
                  <span tagx="volume">65</span>: 
                  <span tagx="fpage">1546</span>–
                  <span tagx="lpage">1549</span>
                  <div tagx="object-id">9119502</div>
                  <div tagx="pub-id">9119502</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">17</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Murphy</span>
                     <span tagx="given-names">CN</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Mortensen</span>
                     <span tagx="given-names">MS</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Krogfelt</span>
                     <span tagx="given-names">KA</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Clegg</span>
                     <span tagx="given-names">S</span>
                  </span> (
                  <span tagx="year">2013</span>) 
                  <span tagx="article-title">Role of Klebsiella pneumoniae type 1 and type 3 fimbriae in colonizing silicone tubes
                     implanted into the bladders of mice as a model of catheter-associated urinary tract
                     infections
                  </span>. 
                  <span tagx="source">Infect Immun</span>
                  <span tagx="volume">81</span>: 
                  <span tagx="fpage">3009</span>–
                  <span tagx="lpage">3017</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1128/IAI.00348-13</div>
                  </div>
                  <div tagx="object-id">23753626</div>
                  <div tagx="pub-id">23753626</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">18</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Wu</span>
                     <span tagx="given-names">C-C</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Huang</span>
                     <span tagx="given-names">Y-J</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Fung</span>
                     <span tagx="given-names">C-P</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Peng</span>
                     <span tagx="given-names">H-L</span>
                  </span> (
                  <span tagx="year">2010</span>) 
                  <span tagx="article-title">Regulation of the Klebsiella pneumoniae Kpc fimbriae by the site-specific recombinase
                     KpcI
                  </span>. 
                  <span tagx="source">Microbiol Read Engl</span>
                  <span tagx="volume">156</span>: 
                  <span tagx="fpage">1983</span>–
                  <span tagx="lpage">1992</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1099/mic.0.038158-0</div>
                  </div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">19</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Criscuolo</span>
                     <span tagx="given-names">A</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Brisse</span>
                     <span tagx="given-names">S</span>
                  </span> (
                  <span tagx="year">2013</span>) 
                  <span tagx="article-title">AlienTrimmer: a tool to quickly and accurately trim off multiple short contaminant
                     sequences from high-throughput sequencing reads
                  </span>. 
                  <span tagx="source">Genomics</span>
                  <span tagx="volume">102</span>: 
                  <span tagx="fpage">500</span>–
                  <span tagx="lpage">506</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1016/j.ygeno.2013.07.011</div>
                  </div>
                  <div tagx="object-id">23912058</div>
                  <div tagx="pub-id">23912058</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">20</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Rissman</span>
                     <span tagx="given-names">AI</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Mau</span>
                     <span tagx="given-names">B</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Biehl</span>
                     <span tagx="given-names">BS</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Darling</span>
                     <span tagx="given-names">AE</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Glasner</span>
                     <span tagx="given-names">JD</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">2009</span>) 
                  <span tagx="article-title">Reordering contigs of draft genomes using the Mauve aligner</span>. 
                  <span tagx="source">Bioinforma Oxf Engl</span>
                  <span tagx="volume">25</span>: 
                  <span tagx="fpage">2071</span>–
                  <span tagx="lpage">2073</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1093/bioinformatics/btp356</div>
                  </div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">21</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Vallenet</span>
                     <span tagx="given-names">D</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Belda</span>
                     <span tagx="given-names">E</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Calteau</span>
                     <span tagx="given-names">A</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Cruveiller</span>
                     <span tagx="given-names">S</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Engelen</span>
                     <span tagx="given-names">S</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">2013</span>) 
                  <span tagx="article-title">MicroScope—an integrated microbial resource for the curation and comparative analysis
                     of genomic and metabolic data
                  </span>. 
                  <span tagx="source">Nucleic Acids Res</span>
                  <span tagx="volume">41</span>: 
                  <span tagx="fpage">D636</span>–
                  <span tagx="lpage">D647</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1093/nar/gks1194</div>
                  </div>
                  <div tagx="object-id">23193269</div>
                  <div tagx="pub-id">23193269</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">22</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Vallenet</span>
                     <span tagx="given-names">D</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Labarre</span>
                     <span tagx="given-names">L</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Rouy</span>
                     <span tagx="given-names">Z</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Barbe</span>
                     <span tagx="given-names">V</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Bocs</span>
                     <span tagx="given-names">S</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">2006</span>) 
                  <span tagx="article-title">MaGe: a microbial genome annotation system supported by synteny results</span>. 
                  <span tagx="source">Nucleic Acids Res</span>
                  <span tagx="volume">34</span>: 
                  <span tagx="fpage">53</span>–
                  <span tagx="lpage">65</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1093/nar/gkj406</div>
                  </div>
                  <div tagx="object-id">16407324</div>
                  <div tagx="pub-id">16407324</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">23</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Bocs</span>
                     <span tagx="given-names">S</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Cruveiller</span>
                     <span tagx="given-names">S</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Vallenet</span>
                     <span tagx="given-names">D</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Nuel</span>
                     <span tagx="given-names">G</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Médigue</span>
                     <span tagx="given-names">C</span>
                  </span> (
                  <span tagx="year">2003</span>) 
                  <span tagx="article-title">AMIGene: Annotation of MIcrobial Genes</span>. 
                  <span tagx="source">Nucleic Acids Res</span>
                  <span tagx="volume">31</span>: 
                  <span tagx="fpage">3723</span>–
                  <span tagx="lpage">3726</span>
                  <div tagx="object-id">12824403</div>
                  <div tagx="pub-id">12824403</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">24</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Marchler-Bauer</span>
                     <span tagx="given-names">A</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Lu</span>
                     <span tagx="given-names">S</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Anderson</span>
                     <span tagx="given-names">JB</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Chitsaz</span>
                     <span tagx="given-names">F</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Derbyshire</span>
                     <span tagx="given-names">MK</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">2011</span>) 
                  <span tagx="article-title">CDD: a Conserved Domain Database for the functional annotation of proteins</span>. 
                  <span tagx="source">Nucleic Acids Res</span>
                  <span tagx="volume">39</span>: 
                  <span tagx="fpage">D225</span>–
                  <span tagx="lpage">D229</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1093/nar/gkq1189</div>
                  </div>
                  <div tagx="object-id">21109532</div>
                  <div tagx="pub-id">21109532</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">25</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Chaudhuri</span>
                     <span tagx="given-names">RR</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Loman</span>
                     <span tagx="given-names">NJ</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Snyder</span>
                     <span tagx="given-names">LAS</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Bailey</span>
                     <span tagx="given-names">CM</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Stekel</span>
                     <span tagx="given-names">DJ</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">2008</span>) 
                  <span tagx="article-title">xBASE2: a comprehensive resource for comparative bacterial genomics</span>. 
                  <span tagx="source">Nucleic Acids Res</span>
                  <span tagx="volume">36</span>: 
                  <span tagx="fpage">D543</span>–
                  <span tagx="lpage">D546</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1093/nar/gkm928</div>
                  </div>
                  <div tagx="object-id">17984072</div>
                  <div tagx="pub-id">17984072</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">26</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Anderson</span>
                     <span tagx="given-names">GG</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Palermo</span>
                     <span tagx="given-names">JJ</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Schilling</span>
                     <span tagx="given-names">JD</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Roth</span>
                     <span tagx="given-names">R</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Heuser</span>
                     <span tagx="given-names">J</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">2003</span>) 
                  <span tagx="article-title">Intracellular bacterial biofilm-like pods in urinary tract infections</span>. 
                  <span tagx="source">Science</span>
                  <span tagx="volume">301</span>: 
                  <span tagx="fpage">105</span>–
                  <span tagx="lpage">107</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1126/science.1084550</div>
                  </div>
                  <div tagx="object-id">12843396</div>
                  <div tagx="pub-id">12843396</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">27</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Katoh</span>
                     <span tagx="given-names">K</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Misawa</span>
                     <span tagx="given-names">K</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Kuma</span>
                     <span tagx="given-names">K</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Miyata</span>
                     <span tagx="given-names">T</span>
                  </span> (
                  <span tagx="year">2002</span>) 
                  <span tagx="article-title">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier
                     transform
                  </span>. 
                  <span tagx="source">Nucleic Acids Res</span>
                  <span tagx="volume">30</span>: 
                  <span tagx="fpage">3059</span>–
                  <span tagx="lpage">3066</span>
                  <div tagx="object-id">12136088</div>
                  <div tagx="pub-id">12136088</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">28</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Capella-Gutiérrez</span>
                     <span tagx="given-names">S</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Silla-Martínez</span>
                     <span tagx="given-names">JM</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Gabaldón</span>
                     <span tagx="given-names">T</span>
                  </span> (
                  <span tagx="year">2009</span>) 
                  <span tagx="article-title">trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses</span>. 
                  <span tagx="source">Bioinforma Oxf Engl</span>
                  <span tagx="volume">25</span>: 
                  <span tagx="fpage">1972</span>–
                  <span tagx="lpage">1973</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1093/bioinformatics/btp348</div>
                  </div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">29</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Guindon</span>
                     <span tagx="given-names">S</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Dufayard</span>
                     <span tagx="given-names">J-F</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Lefort</span>
                     <span tagx="given-names">V</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Anisimova</span>
                     <span tagx="given-names">M</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Hordijk</span>
                     <span tagx="given-names">W</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">2010</span>) 
                  <span tagx="article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the
                     performance of PhyML 3.0
                  </span>. 
                  <span tagx="source">Syst Biol</span>
                  <span tagx="volume">59</span>: 
                  <span tagx="fpage">307</span>–
                  <span tagx="lpage">321</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1093/sysbio/syq010</div>
                  </div>
                  <div tagx="object-id">20525638</div>
                  <div tagx="pub-id">20525638</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">30</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Anisimova</span>
                     <span tagx="given-names">M</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Gil</span>
                     <span tagx="given-names">M</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Dufayard</span>
                     <span tagx="given-names">J-F</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Dessimoz</span>
                     <span tagx="given-names">C</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Gascuel</span>
                     <span tagx="given-names">O</span>
                  </span> (
                  <span tagx="year">2011</span>) 
                  <span tagx="article-title">Survey of branch support methods demonstrates accuracy, power, and robustness of fast
                     likelihood-based approximation schemes
                  </span>. 
                  <span tagx="source">Syst Biol</span>
                  <span tagx="volume">60</span>: 
                  <span tagx="fpage">685</span>–
                  <span tagx="lpage">699</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1093/sysbio/syr041</div>
                  </div>
                  <div tagx="object-id">21540409</div>
                  <div tagx="pub-id">21540409</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">31</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Chaveroche</span>
                     <span tagx="given-names">MK</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Ghigo</span>
                     <span tagx="given-names">JM</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">d’ Enfert</span>
                     <span tagx="given-names">C</span>
                  </span> (
                  <span tagx="year">2000</span>) 
                  <span tagx="article-title">A rapid method for efficient gene replacement in the filamentous fungus Aspergillus
                     nidulans
                  </span>. 
                  <span tagx="source">Nucleic Acids Res</span>
                  <span tagx="volume">28</span>: 
                  <span tagx="fpage">E97</span>
                  <div tagx="object-id">11071951</div>
                  <div tagx="pub-id">11071951</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">32</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Cherepanov</span>
                     <span tagx="given-names">PP</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Wackernagel</span>
                     <span tagx="given-names">W</span>
                  </span> (
                  <span tagx="year">1995</span>) 
                  <span tagx="article-title">Gene disruption in Escherichia coli: TcR and KmR cassettes with the option of Flp-catalyzed
                     excision of the antibiotic-resistance determinant
                  </span>. 
                  <span tagx="source">Gene</span>
                  <span tagx="volume">158</span>: 
                  <span tagx="fpage">9</span>–
                  <span tagx="lpage">14</span>
                  <div tagx="object-id">7789817</div>
                  <div tagx="pub-id">7789817</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">33</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Datsenko</span>
                     <span tagx="given-names">KA</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Wanner</span>
                     <span tagx="given-names">BL</span>
                  </span> (
                  <span tagx="year">2000</span>) 
                  <span tagx="article-title">One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products</span>. 
                  <span tagx="source">Proc Natl Acad Sci U S A</span>
                  <span tagx="volume">97</span>: 
                  <span tagx="fpage">6640</span>–
                  <span tagx="lpage">6645</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1073/pnas.120163297</div>
                  </div>
                  <div tagx="object-id">10829079</div>
                  <div tagx="pub-id">10829079</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">34</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Toledo-Arana</span>
                     <span tagx="given-names">A</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Dussurget</span>
                     <span tagx="given-names">O</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Nikitas</span>
                     <span tagx="given-names">G</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Sesto</span>
                     <span tagx="given-names">N</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Guet-Revillet</span>
                     <span tagx="given-names">H</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">2009</span>) 
                  <span tagx="article-title">The Listeria transcriptional landscape from saprophytism to virulence</span>. 
                  <span tagx="source">Nature</span>
                  <span tagx="volume">459</span>: 
                  <span tagx="fpage">950</span>–
                  <span tagx="lpage">956</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1038/nature08080</div>
                  </div>
                  <div tagx="object-id">19448609</div>
                  <div tagx="pub-id">19448609</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">35</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Ghigo</span>
                     <span tagx="given-names">JM</span>
                  </span> (
                  <span tagx="year">2001</span>) 
                  <span tagx="article-title">Natural conjugative plasmids induce bacterial biofilm development</span>. 
                  <span tagx="source">Nature</span>
                  <span tagx="volume">412</span>: 
                  <span tagx="fpage">442</span>–
                  <span tagx="lpage">445</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1038/35086581</div>
                  </div>
                  <div tagx="object-id">11473319</div>
                  <div tagx="pub-id">11473319</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">36</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Haahtela</span>
                     <span tagx="given-names">K</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Laakso</span>
                     <span tagx="given-names">T</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Korhonen</span>
                     <span tagx="given-names">TK</span>
                  </span> (
                  <span tagx="year">1986</span>) 
                  <span tagx="article-title">Associative Nitrogen Fixation by Klebsiella spp.: Adhesion Sites and Inoculation Effects
                     on Grass Roots
                  </span>. 
                  <span tagx="source">Appl Environ Microbiol</span>
                  <span tagx="volume">52</span>: 
                  <span tagx="fpage">1074</span>–
                  <span tagx="lpage">1079</span>
                  <div tagx="object-id">16347205</div>
                  <div tagx="pub-id">16347205</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">37</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Wu</span>
                     <span tagx="given-names">K-M</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Li</span>
                     <span tagx="given-names">L-H</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Yan</span>
                     <span tagx="given-names">J-J</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Tsao</span>
                     <span tagx="given-names">N</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Liao</span>
                     <span tagx="given-names">T-L</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">2009</span>) 
                  <span tagx="article-title">Genome sequencing and comparative analysis of Klebsiella pneumoniae NTUH-K2044, a
                     strain causing liver abscess and meningitis
                  </span>. 
                  <span tagx="source">J Bacteriol</span>
                  <span tagx="volume">191</span>: 
                  <span tagx="fpage">4492</span>–
                  <span tagx="lpage">4501</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1128/JB.00315-09</div>
                  </div>
                  <div tagx="object-id">19447910</div>
                  <div tagx="pub-id">19447910</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">38</div>
               <div tagx="mixed-citation">Bialek-Davenet S, Criscuolo A, Ailloud F, Passet V, Delannoy-Vieillard A, et al. (2014)
                  Genomic definition of Klebsiella pneumoniae clonal groups reveals emergence of dual-risk
                  hypervirulent and multiresistant strains. Emerg Infect Dis; In press
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">39</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Busch</span>
                     <span tagx="given-names">A</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Waksman</span>
                     <span tagx="given-names">G</span>
                  </span> (
                  <span tagx="year">2012</span>) 
                  <span tagx="article-title">Chaperone-usher pathways: diversity and pilus assembly mechanism</span>. 
                  <span tagx="source">Philos Trans R Soc Lond B Biol Sci</span>
                  <span tagx="volume">367</span>: 
                  <span tagx="fpage">1112</span>–
                  <span tagx="lpage">1122</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1098/rstb.2011.0206</div>
                  </div>
                  <div tagx="object-id">22411982</div>
                  <div tagx="pub-id">22411982</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">40</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Wurpel</span>
                     <span tagx="given-names">DJ</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Beatson</span>
                     <span tagx="given-names">SA</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Totsika</span>
                     <span tagx="given-names">M</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Petty</span>
                     <span tagx="given-names">NK</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Schembri</span>
                     <span tagx="given-names">MA</span>
                  </span> (
                  <span tagx="year">2013</span>) 
                  <span tagx="article-title">Chaperone-usher fimbriae of Escherichia coli</span>. 
                  <span tagx="source">PloS One</span>
                  <span tagx="volume">8</span>: 
                  <span tagx="fpage">e52835</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1371/journal.pone.0052835</div>
                  </div>
                  <div tagx="object-id">23382825</div>
                  <div tagx="pub-id">23382825</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">41</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Yue</span>
                     <span tagx="given-names">M</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Rankin</span>
                     <span tagx="given-names">SC</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Blanchet</span>
                     <span tagx="given-names">RT</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Nulton</span>
                     <span tagx="given-names">JD</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Edwards</span>
                     <span tagx="given-names">RA</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">2012</span>) 
                  <span tagx="article-title">Diversification of the Salmonella fimbriae: a model of macro- and microevolution</span>. 
                  <span tagx="source">PloS One</span>
                  <span tagx="volume">7</span>: 
                  <span tagx="fpage">e38596</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1371/journal.pone.0038596</div>
                  </div>
                  <div tagx="object-id">22701679</div>
                  <div tagx="pub-id">22701679</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">42</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Brisse</span>
                     <span tagx="given-names">S</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Fevre</span>
                     <span tagx="given-names">C</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Passet</span>
                     <span tagx="given-names">V</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Issenhuth-Jeanjean</span>
                     <span tagx="given-names">S</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Tournebize</span>
                     <span tagx="given-names">R</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">2009</span>) 
                  <span tagx="article-title">Virulent clones of Klebsiella pneumoniae: identification and evolutionary scenario
                     based on genomic and phenotypic characterization
                  </span>. 
                  <span tagx="source">PloS One</span>
                  <span tagx="volume">4</span>: 
                  <span tagx="fpage">e4982</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1371/journal.pone.0004982</div>
                  </div>
                  <div tagx="object-id">19319196</div>
                  <div tagx="pub-id">19319196</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">43</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Ong</span>
                     <span tagx="given-names">C-LY</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Ulett</span>
                     <span tagx="given-names">GC</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Mabbett</span>
                     <span tagx="given-names">AN</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Beatson</span>
                     <span tagx="given-names">SA</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Webb</span>
                     <span tagx="given-names">RI</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">2008</span>) 
                  <span tagx="article-title">Identification of type 3 fimbriae in uropathogenic Escherichia coli reveals a role
                     in biofilm formation
                  </span>. 
                  <span tagx="source">J Bacteriol</span>
                  <span tagx="volume">190</span>: 
                  <span tagx="fpage">1054</span>–
                  <span tagx="lpage">1063</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1128/JB.01523-07</div>
                  </div>
                  <div tagx="object-id">18055599</div>
                  <div tagx="pub-id">18055599</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">44</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Ong</span>
                     <span tagx="given-names">CY</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Beatson</span>
                     <span tagx="given-names">SA</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Totsika</span>
                     <span tagx="given-names">M</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Forestier</span>
                     <span tagx="given-names">C</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">McEwan</span>
                     <span tagx="given-names">AG</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">2010</span>) 
                  <span tagx="article-title">Molecular analysis of type 3 fimbrial genes from Escherichia coli, Klebsiella and
                     Citrobacter species
                  </span>. 
                  <span tagx="source">BMC Microbiol</span>
                  <span tagx="volume">10</span>: 
                  <span tagx="fpage">183</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1186/1471-2180-10-183</div>
                  </div>
                  <div tagx="object-id">20576143</div>
                  <div tagx="pub-id">20576143</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">45</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Duguid</span>
                     <span tagx="given-names">JP</span>
                  </span> (
                  <span tagx="year">1959</span>) 
                  <span tagx="article-title">Fimbriae and adhesive properties in Klebsiella strains</span>. 
                  <span tagx="source">J Gen Microbiol</span>
                  <span tagx="volume">21</span>: 
                  <span tagx="fpage">271</span>–
                  <span tagx="lpage">286</span>
                  <div tagx="object-id">13818511</div>
                  <div tagx="pub-id">13818511</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">46</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Boddicker</span>
                     <span tagx="given-names">JD</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Anderson</span>
                     <span tagx="given-names">RA</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Jagnow</span>
                     <span tagx="given-names">J</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Clegg</span>
                     <span tagx="given-names">S</span>
                  </span> (
                  <span tagx="year">2006</span>) 
                  <span tagx="article-title">Signature-tagged mutagenesis of Klebsiella pneumoniae to identify genes that influence
                     biofilm formation on extracellular matrix material
                  </span>. 
                  <span tagx="source">Infect Immun</span>
                  <span tagx="volume">74</span>: 
                  <span tagx="fpage">4590</span>–
                  <span tagx="lpage">4597</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1128/IAI.00129-06</div>
                  </div>
                  <div tagx="object-id">16861646</div>
                  <div tagx="pub-id">16861646</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">47</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Jagnow</span>
                     <span tagx="given-names">J</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Clegg</span>
                     <span tagx="given-names">S</span>
                  </span> (
                  <span tagx="year">2003</span>) 
                  <span tagx="article-title">Klebsiella pneumoniae MrkD-mediated biofilm formation on extracellular matrix- and
                     collagen-coated surfaces
                  </span>. 
                  <span tagx="source">Microbiol Read Engl</span>
                  <span tagx="volume">149</span>: 
                  <span tagx="fpage">2397</span>–
                  <span tagx="lpage">2405</span>
                  <div tagx="object-id">12949165</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">48</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Langstraat</span>
                     <span tagx="given-names">J</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Bohse</span>
                     <span tagx="given-names">M</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Clegg</span>
                     <span tagx="given-names">S</span>
                  </span> (
                  <span tagx="year">2001</span>) 
                  <span tagx="article-title">Type 3 fimbrial shaft (MrkA) of Klebsiella pneumoniae, but not the fimbrial adhesin
                     (MrkD), facilitates biofilm formation
                  </span>. 
                  <span tagx="source">Infect Immun</span>
                  <span tagx="volume">69</span>: 
                  <span tagx="fpage">5805</span>–
                  <span tagx="lpage">5812</span>
                  <div tagx="object-id">11500458</div>
                  <div tagx="pub-id">11500458</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">49</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Lavender</span>
                     <span tagx="given-names">HF</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Jagnow</span>
                     <span tagx="given-names">JR</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Clegg</span>
                     <span tagx="given-names">S</span>
                  </span> (
                  <span tagx="year">2004</span>) 
                  <span tagx="article-title">Biofilm formation in vitro and virulence in vivo of mutants of Klebsiella pneumoniae</span>. 
                  <span tagx="source">Infect Immun</span>
                  <span tagx="volume">72</span>: 
                  <span tagx="fpage">4888</span>–
                  <span tagx="lpage">4890</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1128/IAI.72.8.4888-4890.2004</div>
                  </div>
                  <div tagx="object-id">15271955</div>
                  <div tagx="pub-id">15271955</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">50</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Di Martino</span>
                     <span tagx="given-names">P</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Cafferini</span>
                     <span tagx="given-names">N</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Joly</span>
                     <span tagx="given-names">B</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Darfeuille-Michaud</span>
                     <span tagx="given-names">A</span>
                  </span> (
                  <span tagx="year">2003</span>) 
                  <span tagx="article-title">Klebsiella pneumoniae type 3 pili facilitate adherence and biofilm formation on abiotic
                     surfaces
                  </span>. 
                  <span tagx="source">Res Microbiol</span>
                  <span tagx="volume">154</span>: 
                  <span tagx="fpage">9</span>–
                  <span tagx="lpage">16</span>
                  <div tagx="object-id">12576153</div>
                  <div tagx="pub-id">12576153</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">51</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Struve</span>
                     <span tagx="given-names">C</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Bojer</span>
                     <span tagx="given-names">M</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Krogfelt</span>
                     <span tagx="given-names">KA</span>
                  </span> (
                  <span tagx="year">2009</span>) 
                  <span tagx="article-title">Identification of a conserved chromosomal region encoding Klebsiella pneumoniae type
                     1 and type 3 fimbriae and assessment of the role of fimbriae in pathogenicity
                  </span>. 
                  <span tagx="source">Infect Immun</span>
                  <span tagx="volume">77</span>: 
                  <span tagx="fpage">5016</span>–
                  <span tagx="lpage">5024</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1128/IAI.00585-09</div>
                  </div>
                  <div tagx="object-id">19703972</div>
                  <div tagx="pub-id">19703972</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">52</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Pratt</span>
                     <span tagx="given-names">LA</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Kolter</span>
                     <span tagx="given-names">R</span>
                  </span> (
                  <span tagx="year">1998</span>) 
                  <span tagx="article-title">Genetic analysis of Escherichia coli biofilm formation: roles of flagella, motility,
                     chemotaxis and type I pili
                  </span>. 
                  <span tagx="source">Mol Microbiol</span>
                  <span tagx="volume">30</span>: 
                  <span tagx="fpage">285</span>–
                  <span tagx="lpage">293</span>
                  <div tagx="object-id">9791174</div>
                  <div tagx="pub-id">9791174</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">53</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Schembri</span>
                     <span tagx="given-names">MA</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Christiansen</span>
                     <span tagx="given-names">G</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Klemm</span>
                     <span tagx="given-names">P</span>
                  </span> (
                  <span tagx="year">2001</span>) 
                  <span tagx="article-title">FimH-mediated autoaggregation of Escherichia coli</span>. 
                  <span tagx="source">Mol Microbiol</span>
                  <span tagx="volume">41</span>: 
                  <span tagx="fpage">1419</span>–
                  <span tagx="lpage">1430</span>
                  <div tagx="object-id">11580845</div>
                  <div tagx="pub-id">11580845</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">54</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Struve</span>
                     <span tagx="given-names">C</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Bojer</span>
                     <span tagx="given-names">M</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Krogfelt</span>
                     <span tagx="given-names">KA</span>
                  </span> (
                  <span tagx="year">2008</span>) 
                  <span tagx="article-title">Characterization of Klebsiella pneumoniae type 1 fimbriae by detection of phase variation
                     during colonization and infection and impact on virulence
                  </span>. 
                  <span tagx="source">Infect Immun</span>
                  <span tagx="volume">76</span>: 
                  <span tagx="fpage">4055</span>–
                  <span tagx="lpage">4065</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1128/IAI.00494-08</div>
                  </div>
                  <div tagx="object-id">18559432</div>
                  <div tagx="pub-id">18559432</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">55</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Hahn</span>
                     <span tagx="given-names">E</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Wild</span>
                     <span tagx="given-names">P</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Hermanns</span>
                     <span tagx="given-names">U</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Sebbel</span>
                     <span tagx="given-names">P</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Glockshuber</span>
                     <span tagx="given-names">R</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">2002</span>) 
                  <span tagx="article-title">Exploring the 3D molecular architecture of Escherichia coli type 1 pili</span>. 
                  <span tagx="source">J Mol Biol</span>
                  <span tagx="volume">323</span>: 
                  <span tagx="fpage">845</span>–
                  <span tagx="lpage">857</span>
                  <div tagx="object-id">12417198</div>
                  <div tagx="pub-id">12417198</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">56</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Jones</span>
                     <span tagx="given-names">CH</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Pinkner</span>
                     <span tagx="given-names">JS</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Roth</span>
                     <span tagx="given-names">R</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Heuser</span>
                     <span tagx="given-names">J</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Nicholes</span>
                     <span tagx="given-names">AV</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">1995</span>) 
                  <span tagx="article-title">FimH adhesin of type 1 pili is assembled into a fibrillar tip structure in the Enterobacteriaceae</span>. 
                  <span tagx="source">Proc Natl Acad Sci U S A</span>
                  <span tagx="volume">92</span>: 
                  <span tagx="fpage">2081</span>–
                  <span tagx="lpage">2085</span>
                  <div tagx="object-id">7892228</div>
                  <div tagx="pub-id">7892228</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">57</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Van Aartsen</span>
                     <span tagx="given-names">JJ</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Stahlhut</span>
                     <span tagx="given-names">SG</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Harrison</span>
                     <span tagx="given-names">EM</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Crosatti</span>
                     <span tagx="given-names">M</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Ou</span>
                     <span tagx="given-names">H-Y</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">2012</span>) 
                  <span tagx="article-title">Characterization of a novel chaperone/usher fimbrial operon present on KpGI-5, a methionine
                     tRNA gene-associated genomic island in Klebsiella pneumoniae
                  </span>. 
                  <span tagx="source">BMC Microbiol</span>
                  <span tagx="volume">12</span>: 
                  <span tagx="fpage">59</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1186/1471-2180-12-59</div>
                  </div>
                  <div tagx="object-id">22520965</div>
                  <div tagx="pub-id">22520965</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">58</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Blomfield</span>
                     <span tagx="given-names">IC</span>
                  </span> (
                  <span tagx="year">2001</span>) 
                  <span tagx="article-title">The regulation of pap and type 1 fimbriation in Escherichia coli</span>. 
                  <span tagx="source">Adv Microb Physiol</span>
                  <span tagx="volume">45</span>: 
                  <span tagx="fpage">1</span>–
                  <span tagx="lpage">49</span>
                  <div tagx="object-id">11450107</div>
                  <div tagx="pub-id">11450107</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">59</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Chen</span>
                     <span tagx="given-names">TH</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Elberg</span>
                     <span tagx="given-names">SS</span>
                  </span> (
                  <span tagx="year">1977</span>) 
                  <span tagx="article-title">Scanning electron microscopic study of virulent Yersinia pestis and Yersinia pseudotuberculosis
                     type 1
                  </span>. 
                  <span tagx="source">Infect Immun</span>
                  <span tagx="volume">15</span>: 
                  <span tagx="fpage">972</span>–
                  <span tagx="lpage">977</span>
                  <div tagx="object-id">858647</div>
                  <div tagx="pub-id">858647</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">60</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Van der Woude</span>
                     <span tagx="given-names">M</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Braaten</span>
                     <span tagx="given-names">B</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Low</span>
                     <span tagx="given-names">D</span>
                  </span> (
                  <span tagx="year">1996</span>) 
                  <span tagx="article-title">Epigenetic phase variation of the pap operon in Escherichia coli</span>. 
                  <span tagx="source">Trends Microbiol</span>
                  <span tagx="volume">4</span>: 
                  <span tagx="fpage">5</span>–
                  <span tagx="lpage">9</span>
                  <div tagx="object-id">8824788</div>
                  <div tagx="pub-id">8824788</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">61</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Clegg</span>
                     <span tagx="given-names">S</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Wilson</span>
                     <span tagx="given-names">J</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Johnson</span>
                     <span tagx="given-names">J</span>
                  </span> (
                  <span tagx="year">2011</span>) 
                  <span tagx="article-title">More than one way to control hair growth: regulatory mechanisms in enterobacteria
                     that affect fimbriae assembled by the chaperone/usher pathway
                  </span>. 
                  <span tagx="source">J Bacteriol</span>
                  <span tagx="volume">193</span>: 
                  <span tagx="fpage">2081</span>–
                  <span tagx="lpage">2088</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1128/JB.00071-11</div>
                  </div>
                  <div tagx="object-id">21398554</div>
                  <div tagx="pub-id">21398554</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">62</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Rosen</span>
                     <span tagx="given-names">DA</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Pinkner</span>
                     <span tagx="given-names">JS</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Jones</span>
                     <span tagx="given-names">JM</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Walker</span>
                     <span tagx="given-names">JN</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Clegg</span>
                     <span tagx="given-names">S</span>
                  </span>, 
                  <div tagx="etal">et al</div> (
                  <span tagx="year">2008</span>) 
                  <span tagx="article-title">Utilization of an intracellular bacterial community pathway in Klebsiella pneumoniae
                     urinary tract infection and the effects of FimK on type 1 pilus expression
                  </span>. 
                  <span tagx="source">Infect Immun</span>
                  <span tagx="volume">76</span>: 
                  <span tagx="fpage">3337</span>–
                  <span tagx="lpage">3345</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1128/IAI.00090-08</div>
                  </div>
                  <div tagx="object-id">18411285</div>
                  <div tagx="pub-id">18411285</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">63</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Schembri</span>
                     <span tagx="given-names">MA</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Blom</span>
                     <span tagx="given-names">J</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Krogfelt</span>
                     <span tagx="given-names">KA</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Klemm</span>
                     <span tagx="given-names">P</span>
                  </span> (
                  <span tagx="year">2005</span>) 
                  <span tagx="article-title">Capsule and fimbria interaction in Klebsiella pneumoniae</span>. 
                  <span tagx="source">Infect Immun</span>
                  <span tagx="volume">73</span>: 
                  <span tagx="fpage">4626</span>–
                  <span tagx="lpage">4633</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1128/IAI.73.8.4626-4633.2005</div>
                  </div>
                  <div tagx="object-id">16040975</div>
                  <div tagx="pub-id">16040975</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="label">64</div>
               <div tagx="mixed-citation">
                  <span tagx="name">
                     <span tagx="surname">Balestrino</span>
                     <span tagx="given-names">D</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Ghigo</span>
                     <span tagx="given-names">J-M</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Charbonnel</span>
                     <span tagx="given-names">N</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Haagensen</span>
                     <span tagx="given-names">JAJ</span>
                  </span>, 
                  <span tagx="name">
                     <span tagx="surname">Forestier</span>
                     <span tagx="given-names">C</span>
                  </span> (
                  <span tagx="year">2008</span>) 
                  <span tagx="article-title">The characterization of functions involved in the establishment and maturation of
                     Klebsiella pneumoniae in vitro biofilm reveals dual roles for surface exopolysaccharides
                  </span>. 
                  <span tagx="source">Environ Microbiol</span>
                  <span tagx="volume">10</span>: 
                  <span tagx="fpage">685</span>–
                  <span tagx="lpage">701</span>
                  <div tagx="comment">doi:
                     <div tagx="ext-link">10.1111/j.1462-2920.2007.01491.x</div>
                  </div>
                  <div tagx="object-id">18237304</div>
                  <div tagx="pub-id">18237304</div>
               </div>
            </li>
         </ul>
      </div>
   </div>
</div>