<?xml version="1.0" encoding="UTF-8"?><div tagx="article">
   <head>
      <meta content="eLife Sciences Publications Limited" name="citation_publisher"></meta>
      <meta content="10.7554/eLife.04407" name="citation_doi"></meta>
      <meta content="&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;Genetic transformation of structural and functional circuitry&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;rewires the&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;Drosophila&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;brain&#xA;&#x9;&#x9;&#x9;&#x9;" name="citation_title"></meta>
      <title>
         					Genetic transformation of structural and functional circuitry
         					rewires the
         					Drosophila
         					brain
         				
      </title>
      <div tagx="contrib-group">
         <meta content="Sonia Sen" name="citation_author"></meta>
         <meta content="" name="citation_author_institution_ref"></meta>
         <meta content="Deshou Cao" name="citation_author"></meta>
         <meta content="" name="citation_author_institution_ref"></meta>
         <meta content="Ramveer Choudhary" name="citation_author"></meta>
         <meta content="" name="citation_author_institution_ref"></meta>
         <meta content="Silvia Biagini" name="citation_author"></meta>
         <meta content="" name="citation_author_institution_ref"></meta>
         <meta content="Jing W Wang" name="citation_author"></meta>
         <meta content="" name="citation_author_institution_ref"></meta>
         <meta content="Heinrich Reichert" name="citation_author"></meta>
         <meta content="" name="citation_author_institution_ref"></meta>
         <meta content="K VijayRaghavan" name="citation_author"></meta>
         <meta content="" name="citation_author_institution_ref"></meta>
         <meta content="1 &#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Bangalore&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;" name="citation_author_institution"></meta>
         <meta content="2 &#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Manipal&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;" name="citation_author_institution"></meta>
         <meta content="3 &#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;San Diego&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;" name="citation_author_institution"></meta>
         <meta content="4 &#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Milan&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;" name="citation_author_institution"></meta>
         <meta content="5 &#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Basel&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;" name="citation_author_institution"></meta>
      </div>
      <div tagx="contrib-group"></div>
      <meta content="vijay@ncbs.res.in" name="email"></meta>
      <meta content="" name="conflict"></meta>
      <meta content="" name="contribs"></meta>
      <meta content="2014-12-29" name="citation_pubdate"></meta>
      <meta content="Copyright 2014: Sen et al" name="citation_copyright"></meta>
      <meta content="Licence &#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;This article is distributed under the terms of the&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Creative Commons Attribution License&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;, which permits unrestricted use and redistribution provided that&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;the original author and source are credited.&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;" name="citation_licence"></meta>
      <meta content="&#xA;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;http://dx.doi.org/10.13039/501100001405&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Tata Insitute of Fundamental&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;research&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;National Centre for Biological Sciences&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;VijayRaghavan&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;K&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;http://dx.doi.org/10.13039/501100001703&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;École Polytechnique Fédérale de&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Lausanne&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;Indo-Swiss Bilateral Research Initiative&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Reichert&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Heinrich&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;VijayRaghavan&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;K&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;http://dx.doi.org/10.13039/501100001711&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Swiss National Science Foundation&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;31003A 140607&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Reichert&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Heinrich&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;http://dx.doi.org/10.13039/100000002&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;National&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Institutes of Health&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;DK092640&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Wang&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Jing W&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;http://dx.doi.org/10.13039/501100000522&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Company of Biologists&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Sen&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Sonia&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;http://dx.doi.org/10.13039/100002491&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Bristol-Myers Squibb&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Sen&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Sonia&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Department of Science and&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;Technology, India&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;JC Bose Fellowship&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;VijayRaghavan&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;K&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;&#xA;&#x9;&#x9;&#x9;&#x9;The funders had no role&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;in study design, data collection and interpretation, or the&#xA;&#x9;&#x9;&#x9;&#x9;&#x9;decision to submit the work for publication.&#xA;&#x9;&#x9;&#x9;" name="citation_funding"></meta>
      <meta content="" name="citation_pagecount"></meta>
      <meta content="" name="citation_data"></meta>
   </head>
   <div id="abstract" tag="abstract">
      <div tagx="object-id">abstract-1</div>
      <div tagx="object-id">10.7554/eLife.04407.001</div>
      <p id="p-2">
         					Acquisition of distinct neuronal identities during development is
         					critical for the assembly of diverse functional neural circuits in
         					the brain. In both vertebrates and invertebrates, intrinsic
         					determinants are thought to act in neural progenitors to specify
         					their identity and the identity of their neuronal progeny. However,
         					the extent to which individual factors can contribute to this is
         					poorly understood. We investigate the role of
         					
         <i>orthodenticle</i>
         					in the specification of an identified neuroblast (neuronal
         					progenitor) lineage in the
         					
         <i>Drosophila</i>
         					brain. Loss of
         					
         <i>orthodenticle</i>
         					from this neuroblast affects molecular properties, neuroanatomical
         					features, and functional inputs of progeny neurons, such that an
         					entire central complex lineage transforms into a functional
         					olfactory projection neuron lineage. This ability to change
         					functional macrocircuitry of the brain through changes in gene
         					expression in a single neuroblast reveals a surprising capacity for
         					novel circuit formation in the brain and provides a paradigm for
         					large-scale evolutionary modification of circuitry.
         				
      </p>
      <p id="p-3">
         <b>DOI:</b>
         <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.001</div>
      </p>
   </div>
   <div id="abstract" tag="abstract">
      <div tagx="object-id">abstract-2</div>
      <div tagx="object-id">10.7554/eLife.04407.002</div>
      <div tagx="title">eLife digest</div>
      <p id="p-4">The cells in the brain—including the neurons that
         					transmit information—work together in groups called neural
         					circuits. These cells develop from precursor cells called
         					neuroblasts. Each neuroblast can produce many cells, and it is
         					likely that cells that develop from the same neuroblast work
         					together in the adult brain in the same neural circuit. How the
         					adult cells develop into their final form plays an important role
         					in creating a neural circuit, but this process is not fully
         					understood.
      </p>
      <p id="p-5">
         					In many animals, the complexity of their brain makes it difficult
         					to follow how each individual neuroblast develops. However, all of
         					the neuroblasts in the relatively simple brain of the fruit fly
         					
         <i>Drosophila</i>
         					have been identified. Furthermore, the genes responsible for
         					establishing the initial identity of each neuroblast in the
         					
         <i>Drosophila</i>
         					brain are known. These genes may also determine which adult neurons
         					develop from the neuroblast, and when each type of neuron is
         					produced. However, the extent to which a single gene can influence
         					the identity of neurons is unclear.
         				
      </p>
      <p id="p-6">
         					Sen et al. focused on two types of neuroblasts, each of which,
         					although found next to each other in the developing
         					
         <i>Drosophila</i>
         					brain, produces neurons for different neural circuits. One of the
         					neuroblasts generates the olfactory neurons responsible for
         					detecting smells; the other innervates the ‘central complex’ that
         					has a number of roles, including controlling the fly's movements. A
         					gene called
         					
         <i>orthodenticle</i>
         					is expressed by the central complex neuroblast, but not by the
         					olfactory neuroblast, and helps to separate the two neural circuits
         					into different regions of the fly brain.
         				
      </p>
      <p id="p-7">
         					Sen et al. found that deleting the
         					
         <i>orthodenticle</i>
         					gene from the central complex neuroblast causes it to develop into
         					olfactory neurons instead of central complex neurons. Tests showed
         					that the modified neurons are completely transformed; they not only
         					work like olfactory neurons, but they also have the same structure
         					and molecular properties. Sen et al. have therefore demonstrated
         					that it is possible to drastically alter the circuitry of the fruit
         					fly brain by changing how one gene is expressed in one neuroblast.
         					This suggests that new neural circuits can form relatively easily,
         					and so could help us to understand how different brain structures
         					and neural circuits evolved.
         				
      </p>
      <p id="p-8">
         <b>DOI:</b>
         <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.002</div>
      </p>
   </div>
   <div id="s1">
      <h2>Introduction</h2>
      <p id="p-9">
         				Animals display a wide repertoire of complex and adaptable
         				behaviours executed by equally complex nervous systems.
         				Understanding how the vast number of diverse cell types is assembled
         				into functional neural circuits in complex brains during development
         				is a major challenge. Studies of lineage tracing and circuit mapping
         				reveal that heterogeneous pools of neural progenitors sequentially
         				generate series of neuronal progeny, and that such lineally related
         				neurons with shared developmental histories often share functional
         				connectivity in the brain. In consequence, neural lineages can be
         				considered to form neuroanatomical units of projection that
         				represent the developmental basis of the functional circuitry of the
         				brain (
         				
         <span ref-type="bibr" rid="bib26">Pearson and Doe, 2004</span>
         				;
         				
         <span ref-type="bibr" rid="bib1">Cardona et al., 2010</span>
         				;
         				
         <span ref-type="bibr" rid="bib28">Pereanu et al., 2010</span>
         				;
         				
         <span ref-type="bibr" rid="bib3">Custo Greig et al., 2013</span>
         				;
         				
         <span ref-type="bibr" rid="bib9">Franco and Muller, 2013</span>
         				;
         				
         <span ref-type="bibr" rid="bib10">Gao et al., 2013</span>
         				;
         				
         <span ref-type="bibr" rid="bib16">Kohwi and Doe, 2013</span>
         				).
         			
      </p>
      <p id="p-10">
         				This is exemplified in
         				
         <i>Drosophila</i>
         				where the tens of thousands of neurons that comprise the adult brain
         				are generated during development by a set of approximately 100 pairs
         				of individually identifiable neural stem cells called neuroblasts (
         				
         <span ref-type="bibr" rid="bib42">Truman and Bate, 1988</span>
         				;
         				
         <span ref-type="bibr" rid="bib43">Urbach et al., 2003</span>
         				;
         				
         <span ref-type="bibr" rid="bib44">Urbach and Technau, 2003a</span>
         				,
         				
         <span ref-type="bibr" rid="bib46">2003b</span>
         				;
         				
         <span ref-type="bibr" rid="bib40">Technau et al., 2006</span>
         				). Each neuroblast gives rise to a specific, invariant lineage of
         				post-mitotic neural cells in a highly stereotyped manner and in many
         				cases, lineally related neurons share functional connectivity and
         				many neuroanatomical features such as innervation of common
         				neuropiles in the brain and common axon tract projection patterns (
         				
         <span ref-type="bibr" rid="bib27">Pereanu and Hartenstein, 2006</span>
         				;
         				
         <span ref-type="bibr" rid="bib13">Ito et al., 2013</span>
         				;
         				
         <span ref-type="bibr" rid="bib24">Lovick et al., 2013</span>
         				;
         				
         <span ref-type="bibr" rid="bib48">Wong et al., 2013</span>
         				;
         				
         <span ref-type="bibr" rid="bib49">Yu et al., 2013</span>
         				). Examples of this are the four neuroblast lineages that give rise
         				to the intrinsic cells of the mushroom body, a neuropile compartment
         				involved in learning and memory, or the five neuroblast lineages
         				that innervate the antennal lobe, the primary olfactory processing
         				centre in the fly brain (
         				
         <span ref-type="bibr" rid="bib12">Ito and Hotta, 1992</span>
         				;
         				
         <span ref-type="bibr" rid="bib11">Ito et al., 1997</span>
         				;
         				
         <span ref-type="bibr" rid="bib39">Stocker et al., 1997</span>
         				;
         				
         <span ref-type="bibr" rid="bib14">Jefferis et al., 2001</span>
         				;
         				
         <span ref-type="bibr" rid="bib7">Das et al., 2008</span>
         				,
         				
         <span ref-type="bibr" rid="bib5">2011</span>
         				,
         				
         <span ref-type="bibr" rid="bib6">2013</span>
         				;
         				
         <span ref-type="bibr" rid="bib17">Lai et al., 2008</span>
         				;
         				
         <span ref-type="bibr" rid="bib2">Chou et al., 2010</span>
         				). Thus, neuroblast lineages are considered to form neuroanatomical
         				units of projection that represent the developmental basis of the
         				functional macrocircuitry of the fly brain (
         				
         <span ref-type="bibr" rid="bib27">Pereanu and Hartenstein, 2006</span>
         				;
         				
         <span ref-type="bibr" rid="bib1">Cardona et al., 2010</span>
         				;
         				
         <span ref-type="bibr" rid="bib24">Lovick et al., 2013</span>
         				;
         				
         <span ref-type="bibr" rid="bib48">Wong et al., 2013</span>
         				).
         			
      </p>
      <p id="p-11">
         				Comparable principles are manifested in the developing cerebral
         				cortex of vertebrates, which consists of diverse neurons organized
         				into six distinct layers, each of which is laid in place
         				sequentially during development. Neural progenitors in the cortex
         				are known to be multipotent, capable of generating neurons that
         				populate each of the layers. Lineage tracing experiments in mice
         				suggest that lineally related neurons occupy columns spanning across
         				the layers of the cerebral cortex, as proposed in the ‘radial unit
         				hypothesis’ (
         				
         <span ref-type="bibr" rid="bib30">Rakic, 1988</span>
         				;
         				
         <span ref-type="bibr" rid="bib51">Yu et al., 2009b</span>
         				;
         				
         <span ref-type="bibr" rid="bib20">Li et al., 2012</span>
         				;
         				
         <span ref-type="bibr" rid="bib25">Ohtsuki et al., 2012</span>
         				). Furthermore, lineally related neurons also show a propensity to
         				interconnect and have functional similarity, for example, similar
         				orientation preferences in the visual cortex (
         				
         <span ref-type="bibr" rid="bib20">Li et al., 2012</span>
         				;
         				
         <span ref-type="bibr" rid="bib25">Ohtsuki et al., 2012</span>
         				). Thus in vertebrates as in invertebrates, developmental history
         				and lineage relationships govern the assembly of functional
         				circuits.
         			
      </p>
      <p id="p-12">
         				In order to understand how lineally specified circuits develop in
         				the brain, it is critical to understand the molecular mechanisms
         				that confer unique identities to neural progenitors and their
         				lineages. Studies on the molecular genetics of brain development
         				indicate that neural progenitors emerge from the embryonic
         				neuroectoderm where unique spatial information represented by unique
         				combinations of gene expression specifies unique identities to the
         				progenitors. For example, in
         				
         <i>Drosophila</i>
         				, the embryonic neuroectoderm becomes spatially regionalized due to
         				the action of embryonic patterning genes, which define the
         				anterior-posterior and dorsal-ventral body axes. Their combined
         				expression creates a Cartesian coordinate-like gene expression
         				system in the neuroectoderm, resulting in unique domains of
         				expression of developmental control genes along the neuroectoderm (
         				
         <span ref-type="bibr" rid="bib37">Skeath and Thor, 2003</span>
         				;
         				
         <span ref-type="bibr" rid="bib45">Urbach and Technau, 2004</span>
         				;
         				
         <span ref-type="bibr" rid="bib40">Technau et al., 2006</span>
         				). Changes in the combinatorial expression pattern of these genes in
         				specific domains can lead to changes in the identities of the
         				neuroblasts that delaminate from the neuroectoderm during
         				embryogenesis (e.g.,
         				
         <span ref-type="bibr" rid="bib8">Deshpande et al., 2001</span>
         				). The process of spatial regionalization of the embryonic
         				neuroectoderm is very similar in vertebrates. Homologous embryonic
         				patterning genes result in unique domains of combinatorial gene
         				expression along the vertebrate neuroectoderm (
         				
         <span ref-type="bibr" rid="bib33">Reichert and Simeone, 2001</span>
         				;
         				
         <span ref-type="bibr" rid="bib22">Lichtneckert and Reichert, 2005</span>
         				,
         				
         <span ref-type="bibr" rid="bib21">2008</span>
         				;
         				
         <span ref-type="bibr" rid="bib31">Reichert, 2009</span>
         				). Thus, in both vertebrates and invertebrates, the cells of the
         				neuroectoderm acquire unique spatial information in the form of a
         				combinatorial code of gene expression, which is conferred by
         				embryonic patterning genes. It is noteworthy that neural progenitors
         				also use temporal information (typically a series of sequentially
         				expressed transcription factors) to generate neuronal diversity
         				within lineages (
         				
         <span ref-type="bibr" rid="bib26">Pearson and Doe, 2004</span>
         				;
         				
         <span ref-type="bibr" rid="bib23">Lin and Lee, 2012</span>
         				;
         				
         <span ref-type="bibr" rid="bib16">Kohwi and Doe, 2013</span>
         				). While spatial cues convert a homogenous pool of progenitors into
         				heterogeneous populations, temporal cues result in the ordered
         				production of different neural subtypes from each progenitor.
         			
      </p>
      <p id="p-13">Given that spatial information in the neuroectoderm, in
         				the form of embryonic patterning genes, imparts heterogeneity to
         				neural progenitor populations, it is likely that these genes might
         				also act as intrinsic determinants in the progenitor to give
         				lineages their unique identities and hence determine their place in
         				neural circuitry. This implies that spatially encoded intrinsic
         				factors determine the identity of the progenitor and, as a
         				consequence, the unique circuit features of its neural lineage.
         				According to this, removal or addition of one or more of these genes
         				could lead to a change in neuroblast identity resulting in
         				transformation of the neuronal lineage and the lineal circuitry that
         				derives from it. However, the extent to which individual
         				transcription factors can contribute to this specification of
         				neuroblast identity is not well understood.
      </p>
      <p id="p-14">
         				In order to test this, it is important to be able to uniquely
         				identify individual neural progenitors and their lineages in the
         				brain. The complexity of the vertebrate brain makes it difficult to
         				conduct such an analysis at the resolution of single progenitors and
         				single lineages. However, each of the neuroblasts in the
         				
         <i>Drosophila</i>
         				brain has been identified and their lineages characterized in the
         				larval and adult brains (
         				
         <span ref-type="bibr" rid="bib13">Ito et al., 2013</span>
         				;
         				
         <span ref-type="bibr" rid="bib24">Lovick et al., 2013</span>
         				;
         				
         <span ref-type="bibr" rid="bib48">Wong et al., 2013</span>
         				;
         				
         <span ref-type="bibr" rid="bib49">Yu et al., 2013</span>
         				). Furthermore, each of these neuroblasts has also been
         				characterized by the expression of a specific combination of spatial
         				genes, which could act as cell intrinsic determinants in the
         				specification of unique neuroblast identity and hence control
         				lineage-specific neuronal cell fate (
         				
         <span ref-type="bibr" rid="bib37">Skeath and Thor, 2003</span>
         				;
         				
         <span ref-type="bibr" rid="bib45">Urbach and Technau, 2004</span>
         				;
         				
         <span ref-type="bibr" rid="bib40">Technau et al., 2006</span>
         				). This allows an investigation of the role of putative intrinsic
         				determinants by changing their expression in identified stem cells
         				and assessing its effect at the lineage level in an otherwise normal
         				brain.
         			
      </p>
      <p id="p-15">
         				Here, we focus on two identified neuroblast lineages in the
         				
         <i>Drosophila</i>
         				brain, LALv1 and ALad1, which develop in close spatial proximity to
         				each other in the larval brain but become spatially segregated in
         				the adult brain. While the ALad1 neuroblast generates olfactory
         				projection interneurons that innervate the antennal lobe, the LALv1
         				neuroblast generates wide-field interneurons that innervate the
         				central complex. We show that
         				
         <i>orthodenticle</i>
         				(
         				
         <i>otd</i>
         				), an embryonic patterning gene involved in specifying the
         				anterior-most regions of the neuroectoderm and embryonic brain (
         				
         <span ref-type="bibr" rid="bib21">Lichtneckert and Reichert, 2008</span>
         				;
         				
         <span ref-type="bibr" rid="bib32">Reichert and Bello, 2010</span>
         				), is expressed during development in the LALv1 neuroblast lineage
         				but not in the ALad1 neuroblast lineage. Remarkably, loss of
         				
         <i>otd</i>
         				from the LALv1 neuroblast results in a complete transformation in
         				the identity of the neurons that derive from this lineage. The
         				
         <i>otd</i>
         				null LALv1 neurons transform into antennal lobe projection
         				interneurons similar to the ALad1 lineage, and this transformation
         				includes a complete change in the neuroanatomy of the neurons, a
         				change in their molecular properties as well as in their functional
         				connectivity. This remarkably complete respecification of a
         				neuroblast lineage upon the mutation of a single gene in the brain
         				demonstrates that intrinsic determinants acting in the neuroblast
         				during development specify the identity of its neural progeny and
         				the macrocircuitry that these progeny establish. This large-scale
         				modification of functional circuits in the brain by a single
         				transcription factor in a single stem cell is unprecedented and
         				reveals a surprising capacity for novel neural circuit formation in
         				the developing brain, which may provide a paradigm for large-scale
         				evolutionary modification of brain connectivity.
         			
      </p>
   </div>
   <div id="s2">
      <h2>Results</h2>
      <div id="s2-1">
         <h3>Development, morphogenesis, and differential
            					expression of Otd in two identified central brain neuroblast
            					lineages, LALv1 and ALad1
         </h3>
         <p id="p-16">
            					We focused our analysis on two identified neuroblast lineages
            					referred to as LALv1 and ALad1 (
            					
            <span ref-type="bibr" rid="bib27">Pereanu and Hartenstein, 2006</span>
            					;
            					
            <span ref-type="bibr" rid="bib24">Lovick et al., 2013</span>
            					) (see ‘Materials and methods’ for lineage nomenclature). During
            					postembryonic development in the larval brain, the adult-specific
            					(postembryonically generated) neural progeny of these lineages have
            					their cell bodies clustered close to each other, dorsal to the
            					larval antennal lobe (
            					
            <span ref-type="fig" rid="fig1">Figure 1A,B</span>
            					). Although their cell body clusters are closely apposed, the two
            					lineages can be easily identified based on their distinct and
            					unique axon tracts that project to different brain regions (
            					
            <span ref-type="bibr" rid="bib28">Pereanu et al., 2010</span>
            					;
            					
            <span ref-type="bibr" rid="bib6">Das et al., 2013</span>
            					;
            					
            <span ref-type="bibr" rid="bib24">Lovick et al., 2013</span>
            					).
            					
            <div tagx="fig">
               <div tagx="object-id">elife;3/0/e04407/FIG1</div>
               <div tagx="object-id">F1</div>
               <div tagx="object-id">fig1</div>
               <div tagx="object-id">10.7554/eLife.04407.003</div>
               <div tagx="label">Figure 1.</div>
               <caption id="caption-1">
                  <div tagx="title">Development, morphogenesis, and differential
                     								Otd expression in two identified central brain neuroblast
                     								lineages, LALv1 and ALad1.
                  </div>
                  <p id="p-17">
                     								(
                     								
                     <b>A</b>
                     								and
                     								
                     <b>B</b>
                     								) show anterior and lateral views of 3D reconstructions of the
                     								LALv1 (green) and the ALad1 (magenta) lineages in the larval
                     								brain. (
                     								
                     <b>A</b>
                     								) shows that the cell bodies are closely apposed to each other
                     								and lie above the larval antennal lobe (AL, yellow), (
                     								
                     <b>B</b>
                     								) shows their tracts diverge—the ALad1 tract (magenta) projects
                     								dorsally and the LALv1 tract (green) projects posteriorly behind
                     								the AL and splits. (
                     								
                     <b>C</b>
                     								and
                     								
                     <b>D</b>
                     								) show WT MARCM clones of the larval ALad1 and LALv1 lineages,
                     								respectively. Their cell bodies are outlined by white dotted
                     								lines and the white arrows trace their tracts. Insets in
                     								
                     <b>C</b>
                     								and
                     								
                     <b>D</b>
                     								show that while LALv1 cells (cyan arrow in
                     								
                     <b>D</b>
                     								) express Otd, ALad1 cells (cyan arrow in
                     								
                     <b>C</b>
                     								) do not. (
                     								
                     <b>E</b>
                     								–
                     								
                     <b>L</b>
                     								) is a third larval instar brain (CS) immunolabelled with
                     								neurotactin (green, to identify lineages), Otd (red) and TOPRO-3
                     								(to label nuclei). The LALv1 lineage is documented in
                     								
                     <b>E</b>
                     								–
                     								
                     <b>H</b>
                     								, and the ALad1 lineage is documented in
                     								
                     <b>I</b>
                     								–
                     								
                     <b>L</b>
                     								. The neuroblasts are marked with yellow dotted lines and the
                     								lineages are marked with white dotted lines. The LALv1
                     								neuroblast expresses Otd (yellow arrow in
                     								
                     <b>F</b>
                     								) and the ALad1 neuroblast does not (yellow arrow in
                     								
                     <b>J</b>
                     								). (
                     								
                     <b>M</b>
                     								and
                     								
                     <b>N</b>
                     								) show anterior and lateral views of 3D reconstructions of the
                     								LALv1 (green) and the ALad1 (magenta) lineages in the adult
                     								brain. Note that the adult antennal lobe (AL, yellow in
                     								
                     <b>M</b>
                     								and
                     								
                     <b>N</b>
                     								) is situated between the ALad1 lineage (antero-dorsal to AL)
                     								and the LALv1 lineage (ventral to AL) and the cell bodies of
                     								these lineages are not closely apposed anymore. The arrows in
                     								
                     <b>M</b>
                     								and
                     								
                     <b>N</b>
                     								indicate the ALad1 tract (magenta), which projects dorsally
                     								towards the protocerebrum and the LALv1 tract (green), which
                     								projects posterior to the AL. (
                     								
                     <b>O</b>
                     								and
                     								
                     <b>P</b>
                     								) show WT clones of the adult LALv1 and ALad1 lineages,
                     								respectively. Their cell bodies are outlined by white dotted
                     								lines and the AL is outlined by yellow dotted lines. White
                     								arrows trace the tracts of these lineages. The LALv1 lineage
                     								innervates the lateral accessory lobe (LAL) and the central
                     								complex (CC). The ALad1 lineage innervates the calyx of the
                     								mushroom body (MB) and lateral horn (LH). The midline is
                     								represented by a yellow line in all images. Scale bars in
                     								
                     <b>C</b>
                     								(applicable to
                     								
                     <b>D</b>
                     								) and in
                     								
                     <b>L</b>
                     								(applicable to
                     								
                     <b>E</b>
                     								–
                     								
                     <b>L</b>
                     								) are 20 µm. Scale bar in
                     								
                     <b>P</b>
                     								(applicable to
                     								
                     <b>O</b>
                     								) is 50 µm. Genotypes in
                     								
                     <b>C</b>
                     								and
                     								
                     <b>D</b>
                     								:
                     								
                     <i>FRT19A/FRT19A,Tub-Gal80,hsFLP;
                        									Tub-Gal4,UAS-mCD8::GFP/+
                     </i>
                     								. Genotype in
                     								
                     <b>O</b>
                     								:
                     								
                     <i>FRT19A/FRT19A,Tub-Gal80,hsFLP;
                        									Per-Gal4,UAS-mCD8::GFP/+
                     </i>
                     								. Genotype in
                     								
                     <b>P</b>
                     								:
                     								
                     <i>FRT19A/FRT19A,Tub-Gal80,hsFLP;
                        									GH146-Gal4,UAS-mCD8::GFP/+
                     </i>
                     								.
                     							
                  </p>
                  <p id="p-18">
                     <b>DOI:</b>
                     <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.003</div>
                  </p>
               </caption>
               <div tagx="graphic"></div>
            </div>
         </p>
         <p id="p-19">
            					The anatomical features of these two wild-type lineages can be
            					visualized by MARCM clonal labelling (randomly induced neuroblast
            					clones; ubiquitous
            					
            <i>Tub-Gal4</i>
            					driver). The ALad1 lineage initially projects its axon tract
            					medially, dorsal to the larval antennal lobe, then turns
            					posteriorly and projects towards the protocerebrum via the medial
            					antennal lobe tract (
            					
            <span ref-type="fig" rid="fig1">Figure 1B,C</span>
            					) (
            					
            <span ref-type="bibr" rid="bib6">Das et al., 2013</span>
            					;
            					
            <span ref-type="bibr" rid="bib24">Lovick et al., 2013</span>
            					). The LALv1 lineage initially projects its axon tract
            					ventro-medially, posterior to the larval antennal lobe, then loops
            					dorsally and splits into two secondary axon tracts (
            					
            <span ref-type="fig" rid="fig1">Figure 1B,D</span>
            					) (
            					
            <span ref-type="bibr" rid="bib38">Spindler and Hartenstein, 2011</span>
            					;
            					
            <span ref-type="bibr" rid="bib24">Lovick et al., 2013</span>
            					).
            				
         </p>
         <p id="p-20">
            					In addition to the differences in axonal trajectories, we found
            					that these two lineages also differed in their expression of the
            					transcription factor Otd. Co-immunolabelling for the homeodomain
            					transcription factor Otd and for Neurotactin (to identify
            					lineage-specific axon tracts) shows that the LALv1 neuroblast (
            					
            <span ref-type="fig" rid="fig1">Figure 1F</span>
            					) and all of its lineal progeny (white dotted lines in
            					
            <span ref-type="fig" rid="fig1">Figure 1F</span>
            					and inset in
            					
            <span ref-type="fig" rid="fig1">Figure 1D</span>
            					) express Otd. In contrast, neither the ALad1 neuroblast nor its
            					lineal progeny are found to express Otd (
            					
            <span ref-type="fig" rid="fig1">Figure 1I–L</span>
            					and inset in
            					
            <span ref-type="fig" rid="fig1">Figure 1C</span>
            					).
            				
         </p>
         <p id="p-21">
            					In the adult brain, the neural progeny of the ALad1 lineage are
            					olfactory projection neurons, which innervate the glomeruli of the
            					antennal lobe and the neural progeny of the LALv1 lineage are
            					wide-field interneurons that innervate the central complex, a
            					sensorimotor integration centre in the fly brain (
            					
            <span ref-type="bibr" rid="bib13">Ito et al., 2013</span>
            					;
            					
            <span ref-type="bibr" rid="bib48">Wong et al., 2013</span>
            					;
            					
            <span ref-type="bibr" rid="bib49">Yu et al., 2013</span>
            					). To study the neuroanatomical features of the two lineages in the
            					mature brain, we took advantage of the fact that they are
            					differentially labelled by four enhancer-Gal4 driver lines in the
            					adult brain. Thus, the adult LALv1 lineage is labelled by the
            					
            <i>OK371-</i>
            					Gal4 (a glutamatergic neuron label) and
            					
            <i>Per</i>
            					-Gal4 driver lines (
            					
            <span ref-type="fig" rid="fig1">Figure 1O</span>
            					,
            					
            <span ref-type="table" rid="tbl1">Table 1</span>
            					), while the ALad1 lineage is not. Conversely, the adult ALad1
            					lineage is labelled by the
            					
            <i>Cha7.4-</i>
            					Gal4 (a cholinergic neuron label) and
            					
            <i>GH146</i>
            					-Gal4 lines, while the adult LALv1 lineage is not (
            					
            <span ref-type="fig" rid="fig1">Figure 1P</span>
            					,
            					
            <span ref-type="table" rid="tbl1">Table 1</span>
            					).
            					
            <div tagx="table-wrap">
               <div tagx="object-id">elife;3/0/e04407/TBL1</div>
               <div tagx="object-id">T1</div>
               <div tagx="object-id">tbl1</div>
               <div tagx="object-id">10.7554/eLife.04407.004</div>
               <div tagx="label">Table 1.</div>
               <caption id="caption-2">
                  <p id="p-22">Summary of the specific molecular changes in the
                     								LALv1 and Alad1 lineages
                  </p>
                  <p id="p-23">
                     <b>DOI:</b>
                     <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.004</div>
                  </p>
               </caption>
               <table frame="hsides" rules="groups" id="table-1">
                  <thead id="thead-1">
                     <tr id="tr-1">
                        <div tagx="th"></div>
                        <div tagx="th">WT LALv1</div>
                        <div tagx="th">
                           										otd
                           										
                           <sup>
                              <i>−/−</i>
                           </sup>
                           										LALv1
                           									
                        </div>
                        <div tagx="th">WT ALad1</div>
                     </tr>
                  </thead>
                  <div tagx="tbody">
                     <tr id="tr-2">
                        <td rowspan="1" colspan="1" id="td-1">OK371-Gal4</td>
                        <td rowspan="1" colspan="1" id="td-2">+</td>
                        <td rowspan="1" colspan="1" id="td-3">−</td>
                        <td rowspan="1" colspan="1" id="td-4">−</td>
                     </tr>
                     <tr id="tr-3">
                        <td rowspan="1" colspan="1" id="td-5">Per-Gal4</td>
                        <td rowspan="1" colspan="1" id="td-6">+</td>
                        <td rowspan="1" colspan="1" id="td-7">−</td>
                        <td rowspan="1" colspan="1" id="td-8">−</td>
                     </tr>
                     <tr id="tr-4">
                        <td rowspan="1" colspan="1" id="td-9">Lim</td>
                        <td rowspan="1" colspan="1" id="td-10">+</td>
                        <td rowspan="1" colspan="1" id="td-11">−</td>
                        <td rowspan="1" colspan="1" id="td-12">−</td>
                     </tr>
                     <tr id="tr-5">
                        <td rowspan="1" colspan="1" id="td-13">Cha7.4-Gal4</td>
                        <td rowspan="1" colspan="1" id="td-14">−</td>
                        <td rowspan="1" colspan="1" id="td-15">+</td>
                        <td rowspan="1" colspan="1" id="td-16">+</td>
                     </tr>
                     <tr id="tr-6">
                        <td rowspan="1" colspan="1" id="td-17">GH146-Gal4</td>
                        <td rowspan="1" colspan="1" id="td-18">−</td>
                        <td rowspan="1" colspan="1" id="td-19">+</td>
                        <td rowspan="1" colspan="1" id="td-20">+</td>
                     </tr>
                     <tr id="tr-7">
                        <td rowspan="1" colspan="1" id="td-21">LN1-Gal4</td>
                        <td rowspan="1" colspan="1" id="td-22">−</td>
                        <td rowspan="1" colspan="1" id="td-23">−</td>
                        <td rowspan="1" colspan="1" id="td-24">−</td>
                     </tr>
                     <tr id="tr-8">
                        <td rowspan="1" colspan="1" id="td-25">Acj6</td>
                        <td rowspan="1" colspan="1" id="td-26">+</td>
                        <td rowspan="1" colspan="1" id="td-27">+</td>
                        <td rowspan="1" colspan="1" id="td-28">+</td>
                     </tr>
                  </div>
               </table>
               <div tagx="table-wrap-foot">
                  <div tagx="fn">
                     <p id="p-24">
                        									Note that the molecular signature of the
                        									
                        <i>otd</i>
                        <sup>
                           <i>−/−</i>
                        </sup>
                        									LALv1 lineage is similar to that of the wild-type ALad1
                        									lineage.
                        								
                     </p>
                  </div>
               </div>
            </div>
         </p>
         <p id="p-25">
            					MARCM clonal labelling of the neurons in the adult ALad1 lineage
            					using
            					
            <i>GH146-Gal4</i>
            					or
            					
            <i>Cha7.4</i>
            					-Gal4 drivers shows that their cell bodies are positioned dorsal to
            					the adult antennal lobe, their dendrites innervate the antennal
            					lobe glomeruli and their axons exit the lobe via the medial
            					antennal lobe tract (
            					
            <span ref-type="fig" rid="fig1">Figure 1M,N,P</span>
            					). The axons then project dorso-posteriorly to innervate the calyx
            					of the mushroom body and the lateral horn (
            					
            <span ref-type="other" rid="video1">Video 1</span>
            					) (
            					
            <span ref-type="bibr" rid="bib13">Ito et al., 2013</span>
            					;
            					
            <span ref-type="bibr" rid="bib48">Wong et al., 2013</span>
            					;
            					
            <span ref-type="bibr" rid="bib49">Yu et al., 2013</span>
            					).
            					
            <div tagx="media">
               <div tagx="object-id">media-1</div>
               <div tagx="object-id">video1</div>
               <div tagx="object-id">10.7554/eLife.04407.014</div>
               <div tagx="label">Video 1.</div>
               <caption id="caption-3">
                  <div tagx="title">Projection pattern of the wild-type ALad1
                     								lineage.
                  </div>
                  <p id="p-26">
                     <b>DOI:</b>
                     <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.014</div>
                  </p>
               </caption>
            </div>
         </p>
         <p id="p-27">
            					MARCM clonal labelling of the neurons in the adult LALv1 lineage
            					using
            					
            <i>Per-Gal4</i>
            					or
            					
            <i>OK371-Gal4</i>
            					drivers shows that their cell bodies are positioned ventral to the
            					adult antennal lobe and their axons project into the loVM tract,
            					which courses posteriorly behind the adult antennal lobe, then
            					loops dorsally creating the prominent LEp fascicle, which
            					innervates the central complex neuropiles and the lateral accessory
            					lobe (
            					
            <span ref-type="other" rid="video2">Video 2</span>
            					) (
            					
            <span ref-type="fig" rid="fig1">Figure 1O</span>
            					) (
            					
            <span ref-type="bibr" rid="bib38">Spindler and Hartenstein, 2011</span>
            					;
            					
            <span ref-type="bibr" rid="bib13">Ito et al., 2013</span>
            					;
            					
            <span ref-type="bibr" rid="bib24">Lovick et al., 2013</span>
            					;
            					
            <span ref-type="bibr" rid="bib48">Wong et al., 2013</span>
            					;
            					
            <span ref-type="bibr" rid="bib49">Yu et al., 2013</span>
            					). As in the corresponding larval lineages, the adult LALv1
            					neuroblast lineage expresses Otd while the adult ALad1 neuroblast
            					lineage does not (insets in
            					
            <span ref-type="fig" rid="fig1">Figure 1O,P</span>
            					). It is noteworthy that in the adult brain the cell bodies of the
            					LALv1 neurons are located ventral to the adult antennal lobe,
            					whereas in the larval brain the position of the cell bodies is
            					dorsal to the larval antennal lobe (compare
            					
            <span ref-type="fig" rid="fig1">Figure 1A,M</span>
            					). This change in cell body position of the LALv1 lineage occurs as
            					a consequence of the morphogenetic changes associated with the de
            					novo development of the adult antennal lobe (
            					
            <span ref-type="bibr" rid="bib15">Jefferis et al., 2004</span>
            					;
            					
            <span ref-type="bibr" rid="bib38">Spindler and Hartenstein, 2011</span>
            					;
            					
            <span ref-type="bibr" rid="bib24">Lovick et al., 2013</span>
            					;
            					
            <span ref-type="bibr" rid="bib48">Wong et al., 2013</span>
            					).
            					
            <div tagx="media">
               <div tagx="object-id">media-2</div>
               <div tagx="object-id">video2</div>
               <div tagx="object-id">10.7554/eLife.04407.013</div>
               <div tagx="label">Video 2.</div>
               <caption id="caption-4">
                  <div tagx="title">Projection pattern of the wild-type LAlv1
                     								lineage.
                  </div>
                  <p id="p-28">
                     <b>DOI:</b>
                     <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.013</div>
                  </p>
               </caption>
            </div>
         </p>
         <p id="p-29">
            					In summary, the LALv1 neuroblast and its progeny, which innervate
            					the central complex, express
            					
            <i>otd</i>
            					throughout brain development as well as in the adult brain, while
            					the ALad1 neuroblast and its progeny neurons, which innervate the
            					antennal lobe, do not.
            				
         </p>
      </div>
      <div id="s2-2">
         <h3>Loss of Otd from the LALv1 neuroblast results in
            					lineage identity transformation
         </h3>
         <p id="p-30">
            					As
            					
            <i>otd</i>
            					is expressed in all of the cells of the central complex lineage—the
            					neuroblast and the postmitotic neurons—we tested its possible
            					function in both these cell types. In order to do this, we used
            					MARCM-based clonal mutational methods to generate GFP-labelled
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					clones in the LALv1 lineage in an otherwise heterozygous background
            					(
            					
            <span ref-type="bibr" rid="bib19">Lee and Luo, 2001</span>
            					). Using this technique, it is possible to genetically inactivate
            					
            <i>otd</i>
            					in the postmitotic neurons, the GMC, or the neuroblast, thus
            					allowing us to assess its role in each of these cell types (see
            					schematic in
            					
            <span ref-type="fig" rid="fig2">Figure 2A</span>
            					). We generated such
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					clones early in larval development and analyzed them in the adult
            					brain.
            					
            <div tagx="fig">
               <div tagx="object-id">elife;3/0/e04407/FIG2</div>
               <div tagx="object-id">F2</div>
               <div tagx="object-id">fig2</div>
               <div tagx="object-id">10.7554/eLife.04407.005</div>
               <div tagx="label">Figure 2.</div>
               <caption id="caption-5">
                  <div tagx="title">
                     <i>otd</i>
                     <sup>
                        <i>−/−</i>
                     </sup>
                     								postmitotic clones of the LALv1 lineage.
                     							
                  </div>
                  <p id="p-31">
                     								(
                     								
                     <b>A</b>
                     								) shows a schematic of clone generation by the MARCM method. (
                     								
                     <b>B</b>
                     								and
                     								
                     <b>C</b>
                     								) document two wild-type single cell MARCM clones of the LALv1
                     								lineage. These neurons skirt around the antennal lobe (white
                     								dotted lines) and innervate the LAL. (
                     								
                     <b>D</b>
                     								–
                     								
                     <b>F</b>
                     								) show that these wild-type single cell clones express Otd. (
                     								
                     <b>G</b>
                     								and
                     								
                     <b>H</b>
                     								) document two single cell
                     								
                     <i>otd</i>
                     <sup>
                        <i>−/−</i>
                     </sup>
                     								MARCM clones of the LALv1 lineage. (
                     								
                     <b>I</b>
                     								–
                     								
                     <b>K</b>
                     								) shows one such cell (
                     								
                     <b>I</b>
                     								), which does not express Otd (
                     								
                     <b>J</b>
                     								). Note that the
                     								
                     <i>otd</i>
                     <sup>
                        <i>−/−</i>
                     </sup>
                     								single cell clones of the LALv1 lineage shown in
                     								
                     <b>G</b>
                     								and
                     								
                     <b>H</b>
                     								skirt around the antennal lobe and innervate the LAL and one of
                     								the central complex nodule (yellow arrowhead in
                     								
                     <b>G</b>
                     								and
                     								
                     <b>H</b>
                     								), like the wild-type single cell clones shown in
                     								
                     <b>B</b>
                     								and
                     								
                     <b>C</b>
                     								. Genotypes in
                     								
                     <b>B</b>
                     								–
                     								
                     <b>F</b>
                     								:
                     								
                     <i>FRT19A/FRT19A,Tub-Gal80,hsFLP; Per-Gal4 or
                        									OK371-Gal4,UAS-mCD8::GFP/+
                     </i>
                     								. Genotypes in
                     								
                     <b>G</b>
                     								–
                     								
                     <b>K</b>
                     								:
                     								
                     <i>FRT19A,oc</i>
                     <sup>
                        <i>2</i>
                     </sup>
                     <i>/FRT19A,Tub-Gal80,hsFLP; Per-Gal4 or
                        									OK371-Gal4,UAS-mCD8::GFP/+
                     </i>
                     								. Scale bar is 50 µm. Yellow line represents the midline.
                     							
                  </p>
                  <p id="p-32">
                     <b>DOI:</b>
                     <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.005</div>
                  </p>
               </caption>
               <div tagx="graphic"></div>
            </div>
         </p>
         <p id="p-33">
            					Using this technique, we first investigated a possible requirement
            					of
            					
            <i>otd</i>
            					in the postmitotic neurons of the central complex lineage. In these
            					experiments, in which we used the
            					
            <i>OK371-Gal4</i>
            					and
            					
            <i>Per-Gal4</i>
            					driver lines to label the MARCM clones, we obtained a total of
            					seven wild-type single cell clones and 11
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					single cell clones. Although we have not dated the birth of these
            					clones precisely (matched the time of clone generation), the
            					wild-type single cell clones that we obtained in our experiments
            					were very similar to those described previously (
            					
            <span ref-type="bibr" rid="bib50">Yu et al., 2009a</span>
            					). Six of these single cell wild-type clones consisted of neurons
            					that innervated both the lateral accessory lobe as well as one of
            					the noduli of the central complex (
            					
            <span ref-type="fig" rid="fig2">Figure 2B</span>
            					) and one clone only innervated the lateral accessory lobe (
            					
            <span ref-type="fig" rid="fig2">Figure 2C</span>
            					). All of the 11
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					single cell clones we recovered also displayed a similar
            					neuroanatomy. Their cell bodies were located ventral to the
            					antennal lobe, their axons coursed through the loVM and LEp tracts
            					and they all innervated the lateral accessory lobe and one of the
            					noduli of the central complex (
            					
            <span ref-type="fig" rid="fig2">Figure 2G,H</span>
            					). Thus, loss of
            					
            <i>otd</i>
            					function from the postmitotic neurons did not result in any gross
            					defects in the neuroanatomy of these neurons. It is however
            					possible, that there are fine-scale changes in the arborisation of
            					these neurons within the lateral accessory lobe and the central
            					complex that we were unable to identify. It is also possible that
            					
            <i>otd</i>
            					function is required in the GMC for the targeting of the
            					postmitotic neurons (see schematic in
            					
            <span ref-type="fig" rid="fig2">Figure 2A</span>
            					). However, in our experiments, we never obtained two cell GMC
            					clones in order to be able to address this possibility.
            				
         </p>
         <p id="p-34">
            					We then asked if
            					
            <i>otd</i>
            					might be required in the neuroblast of the LALv1 lineage for its
            					proper development. In order to test this, we inactivated
            					
            <i>otd</i>
            					in the neuroblast during early larval development and analyzed the
            					neuroanatomy of the resultant labelled wild-type and
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					mutant neurons in the adult brain. In these experiments, in which
            					we used the
            					
            <i>OK371-Gal4</i>
            					and
            					
            <i>Per-Gal4 driver</i>
            					lines to label the MARCM clones, we recovered 6 and 14 wild-type
            					clones, respectively. As expected, all the wild-type neuroblast
            					clones displayed the neuroanatomy of the central complex lineage as
            					described above (ventrally position cell bodies, axon projection
            					via the loVM and LEp tracts and innervations in the lateral
            					accessory lobe and central complex.
            					
            <span ref-type="fig" rid="fig3">Figure 3B–E,J–M</span>
            					). However, when we generated
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					neuroblast clones in the LALv1 lineage (identifiable by the loss of
            					Otd staining in the corresponding cell cluster ventral to the
            					antennal lobe; white arrowhead in
            					
            <span ref-type="fig" rid="fig3">Figure 3G,O</span>
            					), neither of these drivers labelled the mutant LALv1 lineage (
            					
            <span ref-type="fig" rid="fig3">Figure 3F–I,N–Q</span>
            					).
            					
            <div tagx="fig">
               <div tagx="object-id">elife;3/0/e04407/FIG3</div>
               <div tagx="object-id">F3</div>
               <div tagx="object-id">fig3</div>
               <div tagx="object-id">10.7554/eLife.04407.006</div>
               <div tagx="label">Figure 3.</div>
               <caption id="caption-6">
                  <div tagx="title">
                     								Loss of
                     								
                     <i>otd</i>
                     								from the LALv1 lineage results in the suppression of the OK371
                     								and Per enhancers.
                     							
                  </div>
                  <p id="p-35">
                     								(
                     								
                     <b>A</b>
                     								) schematises the experimental logic. In neuroblast clones of
                     								the LALv1 lineage-specific enhancer Gal4s label the wild-type
                     								LALv1 neurons because they are active in the lineage (for
                     								example, OK371 and Per). Inability to label the LALv1 neurons in
                     								
                     <i>otd</i>
                     <sup>
                        <i>−/−</i>
                     </sup>
                     								neuroblast clones of the LALv1 lineage (detectable by absence of
                     								Otd immunolabelling) will suggest that the enhancers become
                     								suppressed in the mutant neurons. The brains in
                     								
                     <b>B</b>
                     								–
                     								
                     <b>E</b>
                     								and
                     								
                     <b>J</b>
                     								–
                     								
                     <b>M</b>
                     								show wild-type MARCM clones of the LALv1 lineage labelled by
                     								
                     <i>OK371</i>
                     								-
                     								
                     <i>Gal4</i>
                     								and
                     								
                     <i>Per</i>
                     								-
                     								
                     <i>Gal4</i>
                     								, respectively. The insets in
                     								
                     <b>B</b>
                     								–
                     								
                     <b>E</b>
                     								and
                     								
                     <b>J</b>
                     								–
                     								
                     <b>M</b>
                     								show that these cells express Otd. The brains in
                     								
                     <b>F</b>
                     								–
                     								
                     <b>I</b>
                     								and
                     								
                     <b>N</b>
                     								–
                     								
                     <b>Q</b>
                     								show
                     								
                     <i>otd</i>
                     <sup>
                        <i>−/−</i>
                     </sup>
                     								clones of the LALv1 lineage. Otd expression is lost in one
                     								hemisphere in these brains (white arrowhead in
                     								
                     <b>G</b>
                     								and
                     								
                     <b>O</b>
                     								; compare with the Otd expression within white dotted lines in
                     								the other brain hemisphere). Neither the
                     								
                     <i>OK371</i>
                     								-
                     								
                     <i>Gal4</i>
                     								(
                     								
                     <b>F</b>
                     								) nor the
                     								
                     <i>Per</i>
                     								-
                     								
                     <i>Gal4</i>
                     								(
                     								
                     <b>N</b>
                     								) enhancers drive the expression of UAS-
                     								
                     <i>mCD8</i>
                     								::
                     								
                     <i>GFP</i>
                     								in these cells and the transformed lineage is not labelled in
                     								these experiments. Genotype in
                     								
                     <b>B</b>
                     								–
                     								
                     <b>E</b>
                     								:
                     								
                     <i>FRT19A/FRT19A,Tub-Gal80,hsFLP;
                        									OK371-Gal4,UAS-mCD8::GFP/+
                     </i>
                     								. Genotype in
                     								
                     <b>F</b>
                     								–
                     								
                     <b>I</b>
                     								:
                     								
                     <i>FRT19A, otd</i>
                     <sup>
                        <i>YH13</i>
                     </sup>
                     <i>/FRT19A,Tub-Gal80,hsFLP;
                        									OK371-Gal4,UAS-mCD8::GFP/+
                     </i>
                     								. Genotype in
                     								
                     <b>J</b>
                     								–
                     								
                     <b>M</b>
                     								:
                     								
                     <i>FRT19A/FRT19A,Tub-Gal80,hsFLP;
                        									Per-Gal4,UAS-mCD8::GFP/+
                     </i>
                     								. Genotype in
                     								
                     <b>N</b>
                     								–
                     								
                     <b>Q</b>
                     								:
                     								
                     <i>FRT19A, otd</i>
                     <sup>
                        <i>YH13</i>
                     </sup>
                     <i>/FRT19A,Tub-Gal80,hsFLP;
                        									Per-Gal4,UAS-mCD8::GFP/+
                     </i>
                     								. Midline is represented by a yellow line. Scale bar is 50 µm.
                     							
                  </p>
                  <p id="p-36">
                     <b>DOI:</b>
                     <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.006</div>
                  </p>
               </caption>
               <div tagx="graphic"></div>
            </div>
         </p>
         <p id="p-37">
            					In order to investigate the neuroanatomy of the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 lineage further, we utilized the ubiquitously expressed
            					
            <i>Tub-Gal4</i>
            					driver to label neuroblast clones and recovered 19 WT and 37
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					neuroblast MARCM clones in the LALv1 lineage. While the wild-type
            					neurons displayed all the features of the LALv1 lineage described
            					above (
            					
            <span ref-type="fig" rid="fig4s1">Figure 4—figure supplement 1A</span>
            					), the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 neuroblast clones had dramatic changes in its neuroanatomy (
            					
            <span ref-type="fig" rid="fig4s1">Figure 4—figure supplement 1B–D</span>
            					). Mutant neurons no longer innervated the central complex or
            					lateral accessory lobe neuropiles; instead they innervated the
            					antennal lobe neuropile (asterisk in
            					
            <span ref-type="fig" rid="fig4s1">Figure 4—figure supplement 1B–D</span>
            					) and sent projections via the medial antennal lobe tract towards
            					the protocerebrum (arrowhead in
            					
            <span ref-type="fig" rid="fig4s1">Figure 4—figure supplement 1B–D</span>
            					). These changes in dendritic and axonal innervation patterns were
            					reversed by targeted expression of the full-length
            					
            <i>otd</i>
            					coding sequence in mutant neuroblast clones using the
            					
            <i>Tub-Gal4</i>
            					(
            					
            <span ref-type="fig" rid="fig4s2">Figure 4—figure supplement 2</span>
            					).
            				
         </p>
         <p id="p-38">Taken together, these findings indicate that the mutant
            					LALv1 neurons have acquired a transformed identity. Moreover these
            					data suggest that this transformed identity has features
            					characteristic of antennal lobe projection neurons.
         </p>
      </div>
      <div id="s2-3">
         <h3>Determining the lineage identity of the
            					transformed neurons
         </h3>
         <p id="p-39">
            					As the neuroanatomy of the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 lineage shows such a dramatic transformation, we wanted to
            					confirm that the transformed neurons did indeed belong to the LALv1
            					lineage. We used three different approaches to determine that it
            					was indeed the LALv1 lineage that transformed into an antennal
            					lobe-like lineage upon the loss of
            					
            <i>otd</i>
            					from its neuroblast. First, we showed that the appearance of the
            					transformed
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 lineage corresponds to the appearance of an extra antennal
            					lobe lineage. Second, we showed that the transformed
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 lineage results in the corresponding loss of the LEp tract
            					specific to the wild-type LALv1 lineage. Third, we used an
            					independent molecular marker to unambiguously identify the
            					wild-type and
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 lineage.
            				
         </p>
         <div id="s2-3-1">
            <h3>An extra antennal lobe lineage</h3>
            <p id="p-40">
               						In the wild-type adult brain, the
               						
               <i>GH146-Gal4</i>
               						driver specifically labels a subset of the antennal lobe
               						projection interneurons that derive from three identified
               						neuroblast lineages, ALad1, ALl1, and ALv1 (
               						
               <span ref-type="bibr" rid="bib39">Stocker et al., 1997</span>
               						;
               						
               <span ref-type="bibr" rid="bib14">Jefferis et al., 2001</span>
               						). If the transformed identity of the
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						LALv1 lineage does indeed correspond to that of an antennal lobe
               						lineage, it should also express the antennal lobe specific
               						enhancer Gal4 driver line,
               						
               <i>GH146.</i>
               						To investigate this, we generated wild-type and
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						MARCM clones in the LALv1 lineage and used the
               						
               <i>GH146-Gal4</i>
               						enhancer line to label the recovered clones. As expected, the only
               						wild-type neuroblast clones recovered corresponded to the ALad1,
               						ALl1, and ALv1 lineages;
               						
               <i>GH146-Gal4</i>
               						labelled wild-type neuroblast clones of the LALv1 lineage were
               						never recovered in these experiments. However, when
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						neuroblast clones were induced in the LALv1 lineage, we recovered
               						39 examples of a fourth type of
               						
               <i>GH146-Gal4</i>
               						-labelled neuroblast clone. As the expression of
               						
               <i>GH146-Gal4</i>
               						is much more restricted that
               						
               <i>Tubulin-Gal4</i>
               						, this also provided us with the opportunity to describe the
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						LALv1 lineage in more detail. The neurons in this type of mutant
               						clone all exhibited extensive dendritic innervation of the
               						antennal lobe (39/39;
               						
               <span ref-type="fig" rid="fig4">Figure 4D–G</span>
               						). This innervation was largely multiglomerular (
               						
               <span ref-type="fig" rid="fig4">Figure 4D–G</span>
               						), although not all glomeruli were always innervated (yellow
               						arrowhead in
               						
               <span ref-type="fig" rid="fig4">Figure 4E</span>
               						) and innervation tended to be more intense in the posterior parts
               						of antennal lobe (
               						
               <span ref-type="fig" rid="fig4">Figure 4G</span>
               						). Furthermore, the axons of this clone projected via the medial
               						antennal lobe tract towards the protocerebrum (37/39; white arrow
               						in
               						
               <span ref-type="fig" rid="fig4">Figure 4B,G,H,K</span>
               						), where they innervated the calyx of the mushroom bodies (39/39;
               						
               <span ref-type="fig" rid="fig4">Figure 4I,L</span>
               						) and then turned laterally to innervate the lateral horn (39/39;
               						
               <span ref-type="fig" rid="fig4">Figure 4J,M</span>
               						).
               						
               <div tagx="fig-group">
                  <div tagx="fig">
                     <div tagx="object-id">elife;3/0/e04407/FIG4</div>
                     <div tagx="object-id">F4</div>
                     <div tagx="object-id">fig4</div>
                     <div tagx="object-id">10.7554/eLife.04407.007</div>
                     <div tagx="label">Figure 4.</div>
                     <caption id="caption-7">
                        <div tagx="title">
                           										Loss of
                           										
                           <i>otd</i>
                           										from LALv1 lineage results in activation of the antennal
                           										lobe-specific enhancer, GH146-Gal4, thus allowing detailed
                           										study of the neuroanatomy of the transformed
                           										
                           <i>otd</i>
                           <sup>
                              <i>-−/−</i>
                           </sup>
                           										LALv1 lineage.
                           									
                        </div>
                        <p id="p-41">
                           										(
                           										
                           <b>A</b>
                           										and
                           										
                           <b>B</b>
                           										) show WT LALv1 and ALad1 lineages, respectively, and (
                           										
                           <b>C</b>
                           										) shows an
                           										
                           <i>otd</i>
                           <sup>
                              <i>−/−</i>
                           </sup>
                           										LALv1 lineage. Note that the neuroanatomy of the
                           										
                           <i>otd</i>
                           <sup>
                              <i>−/−</i>
                           </sup>
                           										LALv1 lineage (
                           										
                           <b>C</b>
                           										) is completely unlike the WT LALv1 lineage (
                           										
                           <b>A</b>
                           										) and is strikingly similar to the WT ALad1 lineage (
                           										
                           <b>B</b>
                           										). (
                           										
                           <b>D</b>
                           										–
                           										
                           <b>M</b>
                           										) document different brains that contain the
                           										
                           <i>otd</i>
                           <sup>
                              <i>−/−</i>
                           </sup>
                           										LALv1 lineage. They have multiglomerular dendritic
                           										innervations in the AL (
                           										
                           <b>D</b>
                           										–
                           										
                           <b>G</b>
                           										), but not all glomeruli are always innervated (yellow
                           										arrowhead in
                           										
                           <b>E</b>
                           										). There are more innervations in the more posterior parts of
                           										the antennal lobe (
                           										
                           <b>F</b>
                           										and
                           										
                           <b>G</b>
                           										). Their axon tracts project via the medial antennal lobe
                           										tract (arrowhead in
                           										
                           <b>C</b>
                           										,
                           										
                           <b>G</b>
                           										,
                           										
                           <b>H</b>
                           										,
                           										
                           <b>K</b>
                           										) to innervate the calyx of the mushroom body (MB;
                           										
                           <b>I</b>
                           										and
                           										
                           <b>L</b>
                           										) and the lateral horn (LH;
                           										
                           <b>J</b>
                           										and
                           										
                           <b>M</b>
                           										). Genotype in
                           										
                           <b>A</b>
                           										:
                           										
                           <i>FRT19A/FRT19A,Tub-Gal80,hsFLP;
                              											Per-Gal4,UAS-mCD8::GFP/+
                           </i>
                           										. Genotype in
                           										
                           <b>B</b>
                           										:
                           										
                           <i>FRT19A/FRT19A,Tub-Gal80,hsFLP;
                              											GH146-Gal4,UAS-mCD8::GFP/+
                           </i>
                           										. Genotype in
                           										
                           <b>C</b>
                           										–
                           										
                           <b>M</b>
                           										:
                           										
                           <i>FRT19A,oc</i>
                           <sup>
                              <i>2</i>
                           </sup>
                           <i>or FRT19A,otd</i>
                           <sup>
                              <i>YH13</i>
                           </sup>
                           <i>/FRT19A,Tub-Gal80,hsFLP;
                              											GH146-Gal4,UAS-mCD8::GFP/+
                           </i>
                           										. Scale bars are 50 µm (scale bar in
                           										
                           <b>A</b>
                           										is applicable to
                           										
                           <b>B</b>
                           										and
                           										
                           <b>C</b>
                           										; the one in
                           										
                           <b>D</b>
                           										is applicable through to
                           										
                           <b>G</b>
                           										; the one in
                           										
                           <b>H</b>
                           										is applicable to
                           										
                           <b>K</b>
                           										; the one in
                           										
                           <b>I</b>
                           										is applicable to
                           										
                           <b>L</b>
                           										,
                           										
                           <b>J</b>
                           										,
                           										
                           <b>M</b>
                           										). Yellow line represents the midline.
                           									
                        </p>
                        <p id="p-42">
                           <b>DOI:</b>
                           <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.007</div>
                        </p>
                     </caption>
                     <div tagx="graphic"></div>
                  </div>
                  <div tagx="fig">
                     <div tagx="object-id">elife;3/0/e04407/FIG4S1</div>
                     <div tagx="object-id">F5</div>
                     <div tagx="object-id">fig4s1</div>
                     <div tagx="object-id">10.7554/eLife.04407.008</div>
                     <div tagx="label">Figure 4—figure supplement 1.</div>
                     <caption id="caption-8">
                        <div tagx="title">
                           										Loss of
                           										
                           <i>otd</i>
                           										from the LALv1 neuroblast lineage, transforms it into an
                           										antennal lobe PN lineage.
                           									
                        </div>
                        <p id="p-43">
                           										(
                           										
                           <b>A</b>
                           										and
                           										
                           <b>B</b>
                           										) are Tubulin-Gal4 labelled adult, wild-type clones of the
                           										LALv1 lineage. (
                           										
                           <b>A</b>
                           										) shows the anterior sections and (
                           										
                           <b>B</b>
                           										) shows more posterior sections of the same clone. (
                           										
                           <b>A</b>
                           										) The cell bodies of LALv1 lie ventral to the antennal lobe
                           										(AL) and express the transcription factor otd (inset in
                           										
                           <b>A</b>
                           										). The axon tract of this lineage skirts around the AL
                           										(magenta arrows in
                           										
                           <b>A</b>
                           										and
                           										
                           <b>B</b>
                           										) and innervates the lateral accessory lobe (LAL, magenta
                           										dotted lines in
                           										
                           <b>A</b>
                           										) and the central complex (CC, magenta dotted lines in
                           										
                           <b>B</b>
                           										). Note that there are no innervation from this lineage in the
                           										AL. (
                           										
                           <b>C</b>
                           										and
                           										
                           <b>D</b>
                           										) are tubulin labelled adult, otd null clones of the LALv1 NB
                           										lineage. (
                           										
                           <b>C</b>
                           										) shows the anterior sections, and (
                           										
                           <b>D</b>
                           										) shows more posterior sections of the same clone. The otd
                           										
                           <sup>
                              <i>−/−</i>
                           </sup>
                           										LALv1 lineage innervates the AL (magenta dotted lines in
                           										
                           <b>C</b>
                           										and
                           										
                           <b>D</b>
                           										) and sends its axon tracts towards the protocerebrum (magenta
                           										arrows in
                           										
                           <b>D</b>
                           										).
                           									
                        </p>
                        <p id="p-44">
                           <b>DOI:</b>
                           <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.008</div>
                        </p>
                     </caption>
                     <div tagx="graphic"></div>
                  </div>
                  <div tagx="fig">
                     <div tagx="object-id">elife;3/0/e04407/FIG4S2</div>
                     <div tagx="object-id">F6</div>
                     <div tagx="object-id">fig4s2</div>
                     <div tagx="object-id">10.7554/eLife.04407.009</div>
                     <div tagx="label">Figure 4—figure supplement 2.</div>
                     <caption id="caption-9">
                        <div tagx="title">
                           										Expressing full-length
                           										
                           <i>otd</i>
                           										in the
                           										
                           <i>otd</i>
                           <sup>
                              <i>−/−</i>
                           </sup>
                           										LALv1 lineage rescues the transformation phenotype.
                           									
                        </div>
                        <p id="p-45">
                           										(
                           										
                           <b>A</b>
                           										–
                           										
                           <b>D</b>
                           										) is a MARCM labelled
                           										
                           <i>otd</i>
                           <sup>
                              <i>−/−</i>
                           </sup>
                           										LALv1 lineage that also expresses
                           										
                           <i>UAS</i>
                           										-
                           										
                           <i>otd</i>
                           										. The cell bodies of the
                           										
                           <i>otd</i>
                           <sup>
                              <i>−/−</i>
                           </sup>
                           										LALv1 lineage shown in
                           										
                           <b>A</b>
                           										(magenta dotted lines) express the transgenic
                           										
                           <i>UAS-otd</i>
                           										, shown in
                           										
                           <b>B</b>
                           										(magenta dotted lines). The neuroanatomy of this rescued
                           										lineage resembles that of the WT LALv1 lineage, which skirts
                           										around the antennal lobe (yellow dotted lines) and innervates
                           										the lateral accessory lobe (white arrowheads). Genotype:
                           										
                           <i>FRT19A,otd</i>
                           <sup>
                              <i>YH13</i>
                           </sup>
                           <i>or oc</i>
                           <sup>
                              <i>2</i>
                           </sup>
                           <i>/FRT19A,Tub-Gal80,hsFLP;
                              											Tub-Gal4,UAS-mCD8::GFP/UAS-otd
                           </i>
                           										).
                           									
                        </p>
                        <p id="p-46">
                           <b>DOI:</b>
                           <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.009</div>
                        </p>
                     </caption>
                     <div tagx="graphic"></div>
                  </div>
               </div>
            </p>
            <p id="p-47">
               						The cell body position of this type of
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						clone as well as the entry point of its tract into the antennal
               						lobe (both ventro-medial) did not correspond to any of the known
               						
               <i>GH146-Gal4</i>
               						labelled antennal lobe lineages (
               						
               <span ref-type="bibr" rid="bib13">Ito et al., 2013</span>
               						;
               						
               <span ref-type="bibr" rid="bib49">Yu et al., 2013</span>
               						). Importantly, this is also true for the ALv1 lineage, whose cell
               						bodies are also located ventral to the antennal lobe; despite the
               						ventral cell body position of the LALv1 and ALv1 lineages, their
               						overall neuroanatomy is very different from each other. The
               						neurites of the
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						LALv1 lineage enter the lobe medially (magenta asterisk in
               						
               <span ref-type="fig" rid="fig5">Figure 5A,D,H</span>
               						) while the neurites of the ALv1 (as well as the
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						ALv1) enter it laterally (yellow asterisk in
               						
               <span ref-type="fig" rid="fig5">Figure 5A,D,H</span>
               						). Moreover, while the
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						LALv1 lineage uses the medial antennal lobe tract (magenta arrow
               						in
               						
               <span ref-type="fig" rid="fig5">Figure 5A,D,H</span>
               						), the ALv1 (as well as the
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						ALv1) uses the mediolateral antennal lobe tract (yellow arrow in
               						
               <span ref-type="fig" rid="fig5">Figure 5A,C</span>
               						). Finally, while the
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						LALv1 lineage first innervates the calyx of the mushroom body and
               						then the lateral horn, the GH146-labelled neurons of ALv1 (as well
               						as the
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						ALv1) largely innervate only the lateral horn. The
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						LALv1 lineage was also recovered along with each of the three
               						antennal lobe lineages, and in one case all three known
               						
               <i>GH146-Gal4</i>
               						labelled antennal lobe lineages (ALad1, ALl1 and ALv1) were
               						recovered along with it, resulting in four distinct
               						
               <i>GH146-Gal4</i>
               						labelled lineages in the brain (
               						
               <span ref-type="fig" rid="fig5">Figure 5A–C</span>
               						).
               						
               <div tagx="fig">
                  <div tagx="object-id">elife;3/0/e04407/FIG5</div>
                  <div tagx="object-id">F7</div>
                  <div tagx="object-id">fig5</div>
                  <div tagx="object-id">10.7554/eLife.04407.010</div>
                  <div tagx="label">Figure 5.</div>
                  <caption id="caption-10">
                     <div tagx="title">
                        									Loss of
                        									
                        <i>otd</i>
                        									from the LALv1 lineage results in an extra, fourth antennal
                        									lobe lineage labelled by the
                        									
                        <i>GH146</i>
                        									enhancer.
                        								
                     </div>
                     <p id="p-48">
                        									(
                        									
                        <b>A</b>
                        									–
                        									
                        <b>C</b>
                        									) document a MARCM clonal brain, in which the clones null for
                        									
                        <i>otd</i>
                        									function and are labelled by the
                        									
                        <i>GH146-Gal4</i>
                        									. In this brain, the three known antennal lobe lineages
                        									normally labelled by
                        									
                        <i>GH146-Gal4</i>
                        									(ALad1, ALl1, and Alv1) have been recovered, along with an
                        									additional fourth neuroblast lineage, LALv1. (
                        									
                        <b>D</b>
                        									–
                        									
                        <b>G</b>
                        									) document a control brain where Otd is not down regulated in
                        									the LALv1 lineage (yellow dotted lines in
                        									
                        <b>E</b>
                        									). In this brain, there are three clusters of antennal lobe
                        									projection neurons labelled by
                        									
                        <i>GH146-QF</i>
                        									corresponding to the ALad1, ALl1 and ALv1 lineages, magenta
                        									dotted lines in
                        									
                        <b>D</b>
                        									and
                        									
                        <b>G</b>
                        									). (
                        									
                        <b>H</b>
                        									–
                        									
                        <b>K</b>
                        									) show a brain in which there has been efficient knock down of
                        									Otd in the LALv1 lineage (note loss of Otd immunolabelling
                        									ventral to the antennal lobe; yellow dotted lines in
                        									
                        <b>I</b>
                        									). In this brain, apart from the ALad1, ALl1, and ALv1
                        									projection neurons, an additional, fourth cluster of cells is
                        									seen innervating the antennal lobe (‘4’, LALv1). The yellow
                        									asterisks in
                        									
                        <b>A</b>
                        									,
                        									
                        <b>D</b>
                        									,
                        									
                        <b>H</b>
                        									indicate the point of entry of the ALv1 lineage into the
                        									antennal lobe, and the yellow arrow indicates its axon tract.
                        									The magenta asterisks in
                        									
                        <b>A</b>
                        									,
                        									
                        <b>D</b>
                        									,
                        									
                        <b>H</b>
                        									indicate the point of entry of the LALv1 lineage into the
                        									antennal lobe, and the magenta arrow indicates its axon tract.
                        									Note that these are distinct from each other. Genotype in
                        									
                        <b>A</b>
                        									–
                        									
                        <b>C</b>
                        									:
                        									
                        <i>FRT19A,otd</i>
                        <sup>
                           <i>YH13</i>
                        </sup>
                        <i>/FRT19A,Tub-Gal80,hsFLP;
                           										GH146-Gal4,UAS-mCD8::GFP/+
                        </i>
                        									. Genotype in
                        									
                        <b>D</b>
                        									–
                        									
                        <b>K</b>
                        									:
                        									
                        <i>UASdicer/+;Insc-Gal4/UAS-miRNA-otd-1;GH146-QF,QUAS-mtdTomato-HA/+</i>
                        									). Scale bars are 50 µm. The one in
                        									
                        <b>A</b>
                        									is applicable to
                        									
                        <b>B</b>
                        									and
                        									
                        <b>C</b>
                        									and the one in in
                        									
                        <b>D</b>
                        									is applicable to
                        									
                        <b>D</b>
                        									–
                        									
                        <b>K</b>
                        									. Yellow line represents the midline.
                        								
                     </p>
                     <p id="p-49">
                        <b>DOI:</b>
                        <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.010</div>
                     </p>
                  </caption>
                  <div tagx="graphic"></div>
               </div>
            </p>
            <p id="p-50">
               						We further confirmed this using the
               						
               <i>GH146-QF</i>
               						driver, which, like the
               						
               <i>GH146-Gal4</i>
               						driver labels the ALad1, ALl1, and the ALv1 lineages (
               						
               <span ref-type="bibr" rid="bib29">Potter et al., 2010</span>
               						). In this background, we used the pan-neuroblast-specific
               						
               <i>Insc-Gal4</i>
               						driver to down regulate Otd expression early in all neuroblasts.
               						As expected, in control brains, the
               						
               <i>GH146</i>
               						-
               						
               <i>QF</i>
               						driver labelled a total of three antennal lobe lineages ALad1,
               						ALl1, and ALv1 (magenta dotted lines in
               						
               <span ref-type="fig" rid="fig5">Figure 5D–G</span>
               						). In contrast, in brains where
               						
               <i>otd</i>
               						was efficiently down regulated in all neuroblasts, the
               						
               <i>GH146-QF</i>
               						driver labelled an additional fourth projection interneuron
               						lineage in addition to the three clusters normally seen (
               						
               <span ref-type="fig" rid="fig5">Figure 5H–K</span>
               						). This confirms that the transformed
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						LALv1 lineage is distinct from the other antennal lobe lineages
               						and results in the addition of an extra projection interneuron
               						lineage in the antennal lobe. Taken together, these data suggest
               						that the loss of
               						
               <i>otd</i>
               						from the LALv1 neuroblast results in the addition of an extra
               						antennal lobe lineage.
               					
            </p>
         </div>
         <div id="s2-3-2">
            <h3>Loss of the LEp tract</h3>
            <p id="p-51">
               						If the loss of
               						
               <i>otd</i>
               						from the neuroblast of the LALv1 lineage does indeed result in its
               						neuroanatomical transformation into a lineage of a different fate,
               						then this should correspond to the loss of the LALv1-specific axon
               						tract (LEp) in the brain. To investigate this, we first
               						characterized the axon tract of the wild-type LALv1 lineage, which
               						is readily identifiable in the adult brain based on Neuroglian
               						immunolabelling patterns (
               						
               <span ref-type="bibr" rid="bib28">Pereanu et al., 2010</span>
               						). In wild-type brains, Neuroglian immunolabelling shows the loVM
               						tract (cyan arrow, left hemisphere in
               						
               <span ref-type="fig" rid="fig6">Figure 6C,D</span>
               						) and the characteristic LEp tract of the LALv1 lineage, which
               						projects around the antennal lobes and towards the central complex
               						(cyan arrowhead, left hemisphere in
               						
               <span ref-type="fig" rid="fig6">Figure 6C,D</span>
               						). In brains in which one LALv1 neuroblast clone is mutant, the
               						brain hemisphere that contains the
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						LALv1 neuroblast clone (identified by the loss of Otd
               						immunolabelling; yellow dotted lines in
               						
               <span ref-type="fig" rid="fig6">Figure 6B′</span>
               						), still shows the loVM tract (cyan arrow, right hemisphere in
               						
               <span ref-type="fig" rid="fig6">Figure 6C,D</span>
               						), which is shared by other lineages. However, the LALv1-specific
               						LEp tract, which projects towards the central complex is entirely
               						missing (cyan arrowhead, right hemisphere in
               						
               <span ref-type="fig" rid="fig6">Figure 6C,D</span>
               						). This shows that loss of
               						
               <i>otd</i>
               						from the LALv1 neuroblast corresponds to the loss of the LEp tract
               						of the wild-type LALv1 lineage, providing support that loss of
               						
               <i>otd</i>
               						from the LALv1 neuroblast transforms it into a lineage of
               						different identity.
               						
               <div tagx="fig">
                  <div tagx="object-id">elife;3/0/e04407/FIG6</div>
                  <div tagx="object-id">F8</div>
                  <div tagx="object-id">fig6</div>
                  <div tagx="object-id">10.7554/eLife.04407.011</div>
                  <div tagx="label">Figure 6.</div>
                  <caption id="caption-11">
                     <div tagx="title">
                        									Loss of
                        									
                        <i>otd</i>
                        									from the LALv1 lineage results in the absence of the LEp tract.
                        								
                     </div>
                     <p id="p-52">
                        									(
                        									
                        <b>A</b>
                        									–
                        									
                        <b>D</b>
                        									) and (
                        									
                        <b>A′</b>
                        									–
                        									
                        <b>D′</b>
                        									) document a brain with a single MARCM labelled
                        									
                        <i>otd</i>
                        <sup>
                           <i>−/−</i>
                        </sup>
                        									LALv1 lineage. (
                        									
                        <b>A</b>
                        									–
                        									
                        <b>D</b>
                        									) show more posterior sections that contain the tracts of the
                        									LALv1 lineage; (
                        									
                        <b>A′</b>
                        									–
                        									
                        <b>D′</b>
                        									) show more anterior sections of the same brain that documents
                        									its cell bodies. The right brain hemisphere contains an
                        									
                        <i>otd</i>
                        <sup>
                           <i>−/−</i>
                        </sup>
                        									LALv1 lineage as seen by the loss of Otd immunolabelling
                        									ventral to the antennal lobe (yellow dotted lines in
                        									
                        <b>B′</b>
                        									), while the left hemisphere contains a wild-type LALv1 lineage
                        									(hence not labelled by MARCM) as seen by the presence of Otd
                        									immunolabelling (magenta dotted lines in
                        									
                        <b>B′</b>
                        									). In the left brain hemisphere, which contains the wild-type
                        									LALv1 lineage, the loVM (cyan arrow on the left in
                        									
                        <b>C</b>
                        									) and the LALv1 specific LEp tracts (cyan arrowhead on the left
                        									in
                        									
                        <b>C</b>
                        									) that are identifiable by Neuroglian immunolabelling
                        									(highlighted in magenta). In the right brain hemisphere, which
                        									contains the
                        									
                        <i>otd</i>
                        <sup>
                           <i>−/−</i>
                        </sup>
                        									LALv1 lineage, the loVM tract (taken by other lineages) is
                        									still present (cyan arrow on the right in
                        									
                        <b>C</b>
                        									). The LALv1 specific LEp tract (cyan arrowhead on the right in
                        									
                        <b>C</b>
                        									) that is exclusively made by the LALv1 lineage, is entirely
                        									missing in the right brain hemisphere, which contains the
                        									
                        <i>otd</i>
                        <sup>
                           <i>−/−</i>
                        </sup>
                        									LALv1 lineage (cyan arrow on the right in
                        									
                        <b>C</b>
                        									). The yellow arrowheads in
                        									
                        <b>A</b>
                        									,
                        									
                        <b>C</b>
                        									,
                        									
                        <b>D</b>
                        									point to the new tract of the
                        									
                        <i>otd</i>
                        <sup>
                           <i>−/−</i>
                        </sup>
                        									LALv1 lineage innervating the antennal lobe (magenta asterisk).
                        									Genotype:
                        									
                        <i>FRT19A, otd</i>
                        <sup>
                           <i>YH13</i>
                        </sup>
                        <i>/FRT19A,Tub-Gal80,hsFLP;
                           										GH146-Gal4,UAS-mCD8::GFP/+
                        </i>
                        									. The midline is represented by a yellow line in all images.
                        								
                     </p>
                     <p id="p-53">
                        <b>DOI:</b>
                        <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.011</div>
                     </p>
                  </caption>
                  <div tagx="graphic"></div>
               </div>
            </p>
         </div>
         <div id="s2-3-3">
            <h3>An independent molecular marker</h3>
            <p id="p-54">
               						To investigate the identity of the mutant LALv1 lineage further,
               						we identified Acj6 as a molecular marker that could unambiguously
               						identify the LALv1 lineage in wild-type and
               						
               <i>otd</i>
               						mutants. Acj6 is a POU transcription factor that is known to be
               						expressed in the ALad1 lineage and in a subset of the ALl1-derived
               						projection interneurons of the wild-type brain (
               						
               <span ref-type="fig" rid="fig7">Figure 7B</span>
               						). In addition to these two cell clusters, we observed a third
               						Acj6 positive cell cluster ventral to the antennal lobe of the
               						wild-type brain (cyan arrowhead in
               						
               <span ref-type="fig" rid="fig7">Figure 7B</span>
               						). MARCM clonal labelling using the
               						
               <i>Per-Gal4</i>
               						enhancer line together with anti-Acj6 and anti-Otd antibodies
               						unambiguously identified this cluster as the LALv1 lineage (cyan
               						arrowhead in
               						
               <span ref-type="fig" rid="fig7">Figure 7A–D</span>
               						). Importantly, this cell cluster continues to express Acj6
               						immunoreactivity following mutational inactivation of
               						
               <i>otd</i>
               						in the LALv1 lineage (
               						
               <span ref-type="fig" rid="fig7">Figure 7F,G</span>
               						). Thus, Acj6 provides a molecular marker for the identification
               						of the LALv1 lineage independent of Otd expression in wild-type
               						and mutant clones. The analysis of
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						LALv1 MARCM clones identified by Acj6 immunolabelling and
               						co-labelled by
               						
               <i>GH146-Gal4</i>
               						shows that the fourth
               						
               <i>GH146-</i>
               						positive neuroblast clone described above does indeed correspond
               						to the mutant LALv1 lineage (
               						
               <span ref-type="fig" rid="fig7">Figure 7E,H</span>
               						). This confirms that upon the loss of
               						
               <i>otd</i>
               						from the neuroblast, the neural progeny of the LALv1 lineage
               						transform into an antennal lobe fate.
               						
               <div tagx="fig">
                  <div tagx="object-id">elife;3/0/e04407/FIG7</div>
                  <div tagx="object-id">F9</div>
                  <div tagx="object-id">fig7</div>
                  <div tagx="object-id">10.7554/eLife.04407.012</div>
                  <div tagx="label">Figure 7.</div>
                  <caption id="caption-12">
                     <div tagx="title">
                        									The Acj6-positive cell cluster ventral to the AL is the LALv1
                        									lineage and when null for
                        									
                        <i>otd</i>
                        									, innervates the AL.
                        								
                     </div>
                     <p id="p-55">
                        									(
                        									
                        <b>A</b>
                        									–
                        									
                        <b>D</b>
                        									) document a WT MARCM clone of the LALv1 lineage labelled with
                        									
                        <i>Per-Gal4</i>
                        									. This brain has been immunolabelled with Acj6 (
                        									
                        <b>B</b>
                        									) and Otd (
                        									
                        <b>C</b>
                        									). In
                        									
                        <b>B</b>
                        									, only three clusters of Acj6 positive cells (yellow dotted
                        									lines) are seen around each of the ALs (yellow dotted lines).
                        									The clusters that are ventral to AL (cyan arrowhead) belong to
                        									the LALv1 lineage as seen by the WT MARCM clone shown in
                        									
                        <b>A</b>
                        									. As expected, these cells are also Otd positive (cyan
                        									arrowhead in
                        									
                        <b>C</b>
                        									). (
                        									
                        <b>E</b>
                        									–
                        									
                        <b>H</b>
                        									) show an
                        									
                        <i>otd</i>
                        <sup>
                           <i>−/−</i>
                        </sup>
                        									LALv1 MARCM clone. This brain is also immunolabelled with Acj6
                        									(
                        									
                        <b>F</b>
                        									) and Otd (
                        									
                        <b>G</b>
                        									). Note that the
                        									
                        <i>otd</i>
                        <sup>
                           <i>−/−</i>
                        </sup>
                        									LALv1 cells, which have a transformed neuroanatomy (cyan
                        									arrowhead in
                        									
                        <b>E</b>
                        									), are null for
                        									
                        <i>otd</i>
                        									(cyan arrowhead in
                        									
                        <b>G</b>
                        									) but are identifiable as the ventral cluster of Acj6 positive
                        									cells (cyan arrowhead in
                        									
                        <b>F</b>
                        									). Genotype in
                        									
                        <b>A</b>
                        									–
                        									
                        <b>D</b>
                        									:
                        									
                        <i>FRT19A/FRT19A,Tub-Gal80,hsFLP;
                           										Per-Gal4,UAS-mCD8::GFP/+
                        </i>
                        									. Genotype in
                        									
                        <b>E</b>
                        									–
                        									
                        <b>H</b>
                        									:
                        									
                        <i>FRT19A, otd</i>
                        <sup>
                           <i>YH13</i>
                        </sup>
                        <i>/FRT19A,Tub-Gal80,hsFLP;
                           										GH146-Gal4,UAS-mCD8::GFP/+
                        </i>
                        									. Scale bar is 50 µm.
                        								
                     </p>
                     <p id="p-56">
                        <b>DOI:</b>
                        <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.012</div>
                     </p>
                  </caption>
                  <div tagx="graphic"></div>
               </div>
            </p>
            <p id="p-57">
               						Interestingly, the cell body position of the transformed
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						LALv1 neurons varied somewhat in the 76
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						neuroblast clones obtained (from both the
               						
               <i>Tub-Gal4</i>
               						and
               						
               <i>GH146-Gal4</i>
               						experiments). In most cases (54/76) the cell body position of the
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						LALv1 neuroblast clones remained ventral to the antennal lobe,
               						similar to the cell body position of the wild-type central complex
               						lineage. Thus, in most cases, in terms of cell body position the
               						neuroanatomy did not transform towards the antennal lobe lineage
               						position (antero-dorsal to the adult antennal lobe). Occasionally,
               						however, the cell bodies were shifted closer to the midline
               						(17/76), and in a few rare cases the
               						
               <i>otd</i>
               <sup>
                  <i>−/−</i>
               </sup>
               						LALv1 neuronal cell bodies were located antero-dorsal to the
               						antennal lobe (5/76), a position similar to the wild-type antennal
               						lobe lineage (see
               						
               <span ref-type="other" rid="video3">Video 3</span>
               						). This suggests that loss of
               						
               <i>otd</i>
               						in LALv1 neurons not only consistently transform their axonal and
               						dendritic terminals towards the antennal lobe lineage neuroanatomy
               						but also relocate their cell bodies in some cases to resemble the
               						ALad1 antennal lobe projection neuron lineage.
               						
               <div tagx="media">
                  <div tagx="object-id">media-3</div>
                  <div tagx="object-id">video3</div>
                  <div tagx="object-id">10.7554/eLife.04407.015</div>
                  <div tagx="label">Video 3.</div>
                  <caption id="caption-13">
                     <div tagx="title">
                        									Projection pattern of the otd
                        									
                        <sup>−/−</sup>
                        									LAlv1 lineage.
                        								
                     </div>
                     <p id="p-58">
                        <b>DOI:</b>
                        <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.015</div>
                     </p>
                  </caption>
               </div>
            </p>
            <p id="p-59">
               						In summary, loss of
               						
               <i>otd</i>
               						from the LALv1 neuroblast results in a transformation of its
               						progeny neurons, from a wild-type central complex identity to an
               						antennal lobe projection neuron identity.
               					
            </p>
         </div>
      </div>
      <div id="s2-4">
         <h3>Overexpression of Otd in the antennal lobe
            					lineage results in a partial reciprocal transformation
         </h3>
         <p id="p-60">
            					We next asked whether
            					
            <i>otd</i>
            					gain-of-function in the antennal lobe lineage, ALad1, might result
            					in a reciprocal anatomical transformation of this lineage into one
            					resembling the wild-type central complex lineage, LALv1. We used
            					the MARCM system to misexpress the full-length
            					
            <i>otd</i>
            					coding sequence in the antennal lobe neuroblast clones using a
            					
            <i>Tub-Gal4</i>
            					driver. In all Otd misexpression clones of the antennal lobe
            					lineage (15/15), we found a partial transformation of this lineage
            					towards the central complex identity (
            					
            <span ref-type="fig" rid="fig8">Figure 8</span>
            					). All 15 clones comprised a few cells that retained
            					neuroanatomical features of the wild-type antennal lobe lineage
            					such as antero-dorsal cell body position, innervation of the
            					antennal lobe, and axonal projections via the medial antennal lobe
            					tract (yellow asterisk and arrowhead in
            					
            <span ref-type="fig" rid="fig8">Figure 8A–C</span>
            					). However, most of the cells in the clones displayed
            					neuroanatomical features of the central complex lineage. These cell
            					bodies were positioned ventral to the adult antennal lobe, they
            					projected their axons via the loVM and LEp tracts and they
            					innervated the lateral accessory lobe (magenta asterisk and white
            					arrowheads in
            					
            <span ref-type="fig" rid="fig8">Figure 8A–C</span>
            					). Thus,
            					
            <i>otd</i>
            					gain-of-function was able to cause a partial, albeit incomplete
            					transformation, of the antennal lobe lineage into a central
            					complex-like lineage.
            					
            <div tagx="fig">
               <div tagx="object-id">elife;3/0/e04407/FIG8</div>
               <div tagx="object-id">F10</div>
               <div tagx="object-id">fig8</div>
               <div tagx="object-id">10.7554/eLife.04407.016</div>
               <div tagx="label">Figure 8.</div>
               <caption id="caption-14">
                  <div tagx="title">Overexpression of Otd in the ALad1 lineage
                     								results in a partial reciprocal transformation.
                  </div>
                  <p id="p-61">
                     								(
                     								
                     <b>A</b>
                     								–
                     								
                     <b>C</b>
                     								) ALad1 neuroblast clone misexpressing Otd. (
                     								
                     <b>B</b>
                     								and
                     								
                     <b>C</b>
                     								) are the anterior and lateral views of the 3D reconstructions
                     								of the clone in
                     								
                     <b>A</b>
                     								. Few of the Otd misexpressing cells (yellow asterisk in
                     								
                     <b>A</b>
                     								) retain the wild type neuroanatomy of the ALad1 lineage
                     								(magenta cells in the reconstructions)—they have innervations in
                     								the AL glomeruli (yellow arrowheads in
                     								
                     <b>A</b>
                     								–
                     								
                     <b>C</b>
                     								) and project via the antennal-cerebral tract (yellow arrowhead
                     								in
                     								
                     <b>C</b>
                     								). Most of the Otd misexpressing ALad1 neurons are seen ventral
                     								to the AL (magenta asterisk in
                     								
                     <b>A</b>
                     								; green cells in the reconstructions in
                     								
                     <b>B</b>
                     								and
                     								
                     <b>C</b>
                     								). They do not innervate the AL and instead project towards the
                     								LAL (white arrowheads in
                     								
                     <b>A</b>
                     								–
                     								
                     <b>C</b>
                     								). Genotype in
                     								
                     <b>A</b>
                     								–
                     								
                     <b>C</b>
                     								:
                     								
                     <i>FRT19A/FRT19A,Tub-Gal80,hsFLP;
                        									Tub-Gal4,UAS-mCD8::GFP/UAS-otd
                     </i>
                     								. Midline is represented by a yellow line. Scale bar is 50 µm.
                     							
                  </p>
                  <p id="p-62">
                     <b>DOI:</b>
                     <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.016</div>
                  </p>
               </caption>
               <div tagx="graphic"></div>
            </div>
         </p>
      </div>
      <div id="s2-5">
         <h3>
            					Specific molecular changes occur in the transformed
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 lineage
            				
         </h3>
         <p id="p-63">
            					The innervation pattern of the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 neuroblast lineage strikingly resembled that of the antennal
            					lobe lineage, ALad1. Furthermore, the projection neuron-specific
            					
            <i>GH146</i>
            					driver, which does not label the wild-type central complex lineage,
            					labelled the transformed
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 neurons. This ectopic activation of the
            					
            <i>GH146</i>
            					driver in the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					neurons indicates that the anatomical transformation of the mutant
            					neurons might be accompanied by corresponding molecular
            					transformations that reflect a transformation to the antennal lobe
            					lineage identity. To test if this is indeed the case, we analyzed
            					the activity of select molecular markers (the Gal4 driver lines
            					described above) in the central complex lineage (summarized in
            					
            <span ref-type="table" rid="tbl1">Table 1</span>
            					). In order to test the activity of these enhancers in the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 lineage, we generated
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					MARCM clones and used these selected
            					
            <i>Gal4</i>
            					lines to label the mutant lineage (see schematic in
            					
            <span ref-type="fig" rid="fig3">Figure 3A</span>
            					). Finally, we also tested the expression of the homeodomain
            					transcription factor LIM1, which is expressed in the wild-type
            					LALv1 lineage and is not expressed in the wild-type ALad1 lineage.
            				
         </p>
         <p id="p-64">
            					As described above, when we generated wild-type MARCM clones and
            					used either
            					
            <i>OK371</i>
            					-
            					
            <i>Gal4</i>
            					or
            					
            <i>Per</i>
            					-
            					
            <i>Gal4</i>
            					driver lines to label the clones, we found that both lines were
            					able to drive reporter expression in the wild-type LALv1 lineage (
            					
            <span ref-type="fig" rid="fig3">Figure 3B–E,J–M</span>
            					). In contrast, neither of these driver lines was able to drive
            					reporter expression in the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 lineage (
            					
            <span ref-type="fig" rid="fig3">Figure 3F–I,N–Q</span>
            					). This suggests that the
            					
            <i>OK371</i>
            					-
            					
            <i>Gal4</i>
            					and
            					
            <i>Per</i>
            					-
            					
            <i>Gal4</i>
            					driver lines are suppressed in the transformed
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 neurons.
            				
         </p>
         <p id="p-65">
            					However, in these experiments, the transformed neurons were not
            					labelled at all because the activity of the drivers was suppressed
            					in the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 lineage. In order to confirm this finding, we decided to
            					positively label the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 neurons and in this background assay the activity of the
            					
            <i>Gal4</i>
            					drivers. In order to do this, we utilized the dual MARCM method (
            					
            <span ref-type="bibr" rid="bib18">Lai and Lee, 2006</span>
            					), which uses two independent binary expression systems (Gal4-UAS
            					and LexA–LexA operator) to label the MARCM clones. In these
            					experiments, we used the
            					
            <i>GH146-LexA</i>
            					driver to label the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 neurons positively and combined it with
            					
            <i>OK371</i>
            					-
            					
            <i>Gal4</i>
            					and
            					
            <i>Per</i>
            					-
            					
            <i>Gal4</i>
            					driver lines to assay their activities. We first tested if the
            					
            <i>GH146-LexA</i>
            					driver, like the
            					
            <i>GH146-Gal4</i>
            					and the
            					
            <i>GH146-QF</i>
            					driver, was active in the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 neurons. Thus, in the first set of dual MARCM experiments, we
            					combined the
            					
            <i>GH146-LexA</i>
            					with
            					
            <i>Tubulin-Gal4</i>
            					(
            					
            <span ref-type="fig" rid="fig9">Figure 9A–D</span>
            					). Under these conditions, when we generated
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					neuroblast clones in the LALv1 lineage, we found that the
            					transformed neurons were labelled by both the
            					
            <i>Tubulin-Gal4</i>
            					(magenta dotted lines and inset in
            					
            <span ref-type="fig" rid="fig9">Figure 9A</span>
            					) and
            					
            <i>GH146-LexA</i>
            					(magenta dotted lines and inset in
            					
            <span ref-type="fig" rid="fig9">Figure 9B</span>
            					) drivers, confirming that
            					
            <i>GH146-LexA</i>
            					, like
            					
            <i>GH146-Gal4</i>
            					and the
            					
            <i>GH146-QF</i>
            					, is active in the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 neurons and thus able to label it. In the following dual
            					MARCM experiments, we used the
            					
            <i>GH146-LexA</i>
            					to positively label the transformed
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 neurons and combined it with either
            					
            <i>OK371</i>
            					-
            					
            <i>Gal4</i>
            					or
            					
            <i>Per</i>
            					-
            					
            <i>Gal4</i>
            					driver lines to assay for their activity in the transformed
            					neurons. In both cases, while the
            					
            <i>GH146-LexA</i>
            					positively labelled the transformed
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 neurons (magenta dotted lines and insets in
            					
            <span ref-type="fig" rid="fig9">Figure 9E,I</span>
            					), neither of the Gal4 driver lines was able to drive reporter
            					expression in these mutant neurons (magenta dotted lines and insets
            					in
            					
            <span ref-type="fig" rid="fig9">Figure 9F,J</span>
            					). This indicates that enhancers that are normally active in the
            					wild-type central complex lineage and inactive in the antennal lobe
            					lineage become suppressed in the transformed
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 lineage.
            					
            <div tagx="fig">
               <div tagx="object-id">elife;3/0/e04407/FIG9</div>
               <div tagx="object-id">F11</div>
               <div tagx="object-id">fig9</div>
               <div tagx="object-id">10.7554/eLife.04407.017</div>
               <div tagx="label">Figure 9.</div>
               <caption id="caption-15">
                  <div tagx="title">
                     								Specific molecular changes occur in the transformed
                     								
                     <i>otd</i>
                     <sup>
                        <i>−/−</i>
                     </sup>
                     								LALv1 lineage.
                     							
                  </div>
                  <p id="p-66">
                     								(
                     								
                     <b>A</b>
                     								–
                     								
                     <b>D</b>
                     								) documents an
                     								
                     <i>otd</i>
                     <sup>
                        <i>−/−</i>
                     </sup>
                     								LALv1 neuroblast clone that has been dual labelled with
                     								
                     <i>Tubulin-Gal4</i>
                     								(
                     								
                     <b>A</b>
                     								) and
                     								
                     <i>GH146-LexA</i>
                     								(
                     								
                     <b>B</b>
                     								) using the dual MARCM technique. (
                     								
                     <b>E</b>
                     								–
                     								
                     <b>H</b>
                     								and
                     								
                     <b>I</b>
                     								–
                     								
                     <b>L</b>
                     								) document
                     								
                     <i>otd</i>
                     <sup>
                        <i>−/−</i>
                     </sup>
                     								LALv1 neuroblast clones that have been dual labelled with
                     								
                     <i>GH146-LexA</i>
                     								(
                     								
                     <b>E</b>
                     								and
                     								
                     <b>I</b>
                     								) to positively label the mutant neurons and either
                     								
                     <i>OK371-Gal4</i>
                     								(
                     								
                     <b>E</b>
                     								–
                     								
                     <b>H</b>
                     								) or Per-Gal4 (
                     								
                     <b>I</b>
                     								–
                     								
                     <b>L</b>
                     								) using the dual MARCM technique. In these clones, while the
                     								
                     <i>GH146-LexA</i>
                     								driver labels the transformed
                     								
                     <i>otd</i>
                     <sup>
                        <i>−/−</i>
                     </sup>
                     								LALv1 neurons (
                     								
                     <b>E</b>
                     								and
                     								
                     <b>I</b>
                     								), neither the
                     								
                     <i>OK371-Gal4</i>
                     								(
                     								
                     <b>F</b>
                     								) nor the
                     								
                     <i>Per-Gal4</i>
                     								(
                     								
                     <b>J</b>
                     								) do. (
                     								
                     <b>M</b>
                     								–
                     								
                     <b>P</b>
                     								) documents a transformed
                     								
                     <i>otd</i>
                     <sup>
                        <i>−/−</i>
                     </sup>
                     								LALv1 lineage (inset and magenta dotted lines in
                     								
                     <b>N</b>
                     								) labelled with
                     								
                     <i>Cha7.4-Gal4</i>
                     								. While
                     								
                     <i>Cha7.4-Gal4</i>
                     								is not active in the wild-type LALv1 lineage (not shown), it
                     								becomes activated in the
                     								
                     <i>otd</i>
                     <sup>
                        <i>−/−</i>
                     </sup>
                     								LALv1 lineage (inset and magenta dotted lines in
                     								
                     <b>M</b>
                     								–
                     								
                     <b>P</b>
                     								). Genotypes in
                     								
                     <b>A</b>
                     								–
                     								
                     <b>D</b>
                     								:
                     								
                     <i>FRT19A, otd</i>
                     <sup>
                        <i>YH13</i>
                     </sup>
                     <i>/FRT19A,Tub-Gal80,hsFLP;
                        									Tubulin-Gal4,UAS-mCD8::GFP/FRTG13,GH146-LexA::GAD,LexAop::rCD2::GFP
                     </i>
                     								. Genotype in
                     								
                     <b>E</b>
                     								–
                     								
                     <b>H</b>
                     								:
                     								
                     <i>FRT19A,otd</i>
                     <sup>
                        <i>YH13</i>
                     </sup>
                     <i>/FRT19A,Tub-Gal80,hsFLP;
                        									OK371-Gal4,UAS-mCD8::GFP/FRTG13,GH146-LexA::GAD,LexAop::rCD2::GFP
                     </i>
                     								. Genotype in
                     								
                     <b>I</b>
                     								–
                     								
                     <b>L</b>
                     								:
                     								
                     <i>FRT19A,otd</i>
                     <sup>
                        <i>YH13</i>
                     </sup>
                     <i>/FRT19A,Tub-Gal80,hsFLP;Per-Gal4,UAS-mCD8::GFP/FRTG13,GH146-LexA::GAD,LexAop::rCD2::GFP</i>
                     								. Genotype in
                     								
                     <b>H</b>
                     								:
                     								
                     <i>FRT19A,otd</i>
                     <sup>
                        <i>YH13</i>
                     </sup>
                     <i>/FRT19A,Tub-Gal80,hsFLP;
                        									Cha7.4-Gal4,UAS-mCD8::GFP/+
                     </i>
                     								. Midline is represented by a yellow line.
                     							
                  </p>
                  <p id="p-67">
                     <b>DOI:</b>
                     <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.017</div>
                  </p>
               </caption>
               <div tagx="graphic"></div>
            </div>
         </p>
         <p id="p-68">
            					Might the converse be true; do enhancers that are normally inactive
            					in the wild-type central complex lineage and active in the antennal
            					lobe lineage become activated in the transformed
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 lineage? To test this, we used the
            					
            <i>Cha7</i>
            					.
            					
            <i>4-Gal4</i>
            					in MARCM clonal experiments. As expected, we never recovered
            					
            <i>Cha7</i>
            					.
            					
            <i>4-Gal4</i>
            					labelled LALv1 neuroblast clones in wild-type MARCM experiments
            					(data not shown). In contrast, when we generated
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					neuroblast clones in the LALv1 lineage (identifiable by the loss of
            					Otd staining ventral to the antennal lobe; magenta dotted lines in
            					
            <span ref-type="fig" rid="fig9">Figure 9N</span>
            					) the
            					
            <i>Cha7.4-Gal4</i>
            					driver robustly drove reporter expression in the transformed
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 lineage (magenta dotted lines and insets in
            					
            <span ref-type="fig" rid="fig9">Figure 9M–P</span>
            					). This suggests that the
            					
            <i>Cha7.4-Gal4</i>
            					becomes activated in the transformed
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 lineage. Furthermore, the concomitant loss of the
            					
            <i>OK371</i>
            					-
            					
            <i>Gal4</i>
            					driver (a putative glutamatergic label) and ectopic activation of
            					the
            					
            <i>Cha7.4-Gal4</i>
            					driver (a putative cholinergic label) suggest that there might be a
            					change in neurotransmitter identity of the LALv1 lineage from
            					wild-type glutamatergic to transformed cholinergic identities,
            					similar to the cholinergic identity of wild-type antennal lobe
            					neurons.
            				
         </p>
         <p id="p-69">
            					Finally, immunolabelling with an anti-LIM antibody indicates that
            					the wild-type central complex lineage expresses the homeodomain
            					transcription factor LIM (
            					
            <span ref-type="fig" rid="fig10">Figure 10G–I</span>
            					) while the transformed
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 lineage, like the wild-type antennal lobe lineage, does not (
            					
            <span ref-type="fig" rid="fig10">Figure 10J–L</span>
            					). Interestingly, the
            					
            <i>LN1-Gal4</i>
            					driver, which is inactive in both the wild-type LALv1 and ALad1
            					lineages, remains inactive in the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 lineage (data not shown).
            					
            <div tagx="fig">
               <div tagx="object-id">elife;3/0/e04407/FIG10</div>
               <div tagx="object-id">F12</div>
               <div tagx="object-id">fig10</div>
               <div tagx="object-id">10.7554/eLife.04407.018</div>
               <div tagx="label">Figure 10.</div>
               <caption id="caption-16">
                  <div tagx="title">
                     								Specific molecular changes occur in the transformed
                     								
                     <i>otd</i>
                     <sup>
                        <i>−/−</i>
                     </sup>
                     								LALv1 lineage.
                     							
                  </div>
                  <p id="p-70">
                     								(
                     								
                     <b>A</b>
                     								–
                     								
                     <b>C</b>
                     								and
                     								
                     <b>G</b>
                     								–
                     								
                     <b>I</b>
                     								) document neuronal cell bodies of the wild-type LALv1 lineage
                     								immunolabelled with Acj6 (
                     								
                     <b>A</b>
                     								–
                     								
                     <b>C</b>
                     								) and Lim1 (
                     								
                     <b>G</b>
                     								–
                     								
                     <b>I</b>
                     								). The WT LALv1 neurons express both these molecular markers. (
                     								
                     <b>D</b>
                     								–
                     								
                     <b>F</b>
                     								and
                     								
                     <b>J</b>
                     								–
                     								
                     <b>L</b>
                     								) document neuronal cell bodies of the
                     								
                     <i>otd</i>
                     <sup>
                        <i>−/−</i>
                     </sup>
                     								LALv1 lineage immunolabelled with Acj6 (
                     								
                     <b>D</b>
                     								–
                     								
                     <b>F</b>
                     								) and Lim1 (
                     								
                     <b>J</b>
                     								–
                     								
                     <b>L</b>
                     								). While
                     								
                     <i>otd</i>
                     <sup>
                        <i>−/−</i>
                     </sup>
                     								LALv1 neurons continue to express Acj6 (
                     								
                     <b>E</b>
                     								), they downregulate Lim1 expression (
                     								
                     <b>K</b>
                     								).
                     							
                  </p>
                  <p id="p-71">
                     <b>DOI:</b>
                     <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.018</div>
                  </p>
               </caption>
               <div tagx="graphic"></div>
            </div>
         </p>
         <p id="p-72">
            					Taken together, these findings indicate that the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 lineage acquires the molecular signature of a wild-type
            					antennal lobe lineage (see
            					
            <span ref-type="table" rid="tbl1">Table 1</span>
            					) implying that
            					
            <i>otd</i>
            					loss-of-function in the LALv1 neuroblast lineage results in a
            					molecular as well as an anatomical transformation of this lineage
            					into one resembling the ALad1 lineage.
            				
         </p>
      </div>
      <div id="s2-6">
         <h3>
            					The
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					transformed LALv1 lineage establishes functional connectivity in
            					the antennal lobe and can respond to odour stimulation
            				
         </h3>
         <p id="p-73">
            					Given the extent of the anatomical and molecular transformations
            					seen in the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 neuroblast lineage, might the neurons in the transformed
            					lineage receive functional input from olfactory sensory neurons? To
            					address this question, we specifically expressed the calcium sensor
            					G-CaMP3 in the transformed
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 lineage by MARCM clonal labeling (
            					
            <span ref-type="bibr" rid="bib47">Wang et al., 2003</span>
            					) and used two-photon microscopy to monitor calcium activity in the
            					dendrites of these transformed neurons in the antennal lobe.
            				
         </p>
         <p id="p-74">
            					We first tested if the transformed
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 neurons established functional connectivity with other
            					neurons in the antennal lobe. Typically, olfactory sensory neurons
            					bring odour information to the antennal lobe via the antennal
            					nerve. Here, they make synaptic connections with projection neurons
            					and local interneurons; projection neurons, take the odour
            					information to higher brain centres (mushroom body and lateral
            					horn) and local interneurons process the odour information locally
            					in the antennal lobe. We reasoned that if the transformed neurons
            					did made functional connections within the antennal lobe they would
            					be postsynaptic to the olfactory sensory neurons and would be
            					activated upon the activation of the antennal nerve. We therefore
            					electrically stimulated the antennal nerve while simultaneously
            					monitoring calcium activity from the transformed neurons. We found
            					that electrical stimulation of the antennal nerve, which contains
            					the axons of the olfactory sensory neurons, resulted in an increase
            					in calcium activity in the dendrites of the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 lineage. Moreover, a greater number of electrical stimulus
            					pulses applied to the antennal nerve resulted in a corresponding
            					increase in the amplitude of the calcium signal recorded in the
            					mutant transformed neurons (
            					
            <span ref-type="fig" rid="fig11">Figure 11A–C</span>
            					). These results demonstrate that the transformed
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 neurons were able to make functional connections in the
            					antennal lobe and were able to receive functional input from
            					sensory afferents.
            					
            <div tagx="fig">
               <div tagx="object-id">elife;3/0/e04407/FIG11</div>
               <div tagx="object-id">F13</div>
               <div tagx="object-id">fig11</div>
               <div tagx="object-id">10.7554/eLife.04407.019</div>
               <div tagx="label">Figure 11.</div>
               <caption id="caption-17">
                  <div tagx="title">
                     								The
                     								
                     <i>otd</i>
                     <sup>
                        <i>−/−</i>
                     </sup>
                     								transformed LALv1 lineage has functional synapses in the AL and
                     								can respond to odour stimulation.
                     							
                  </div>
                  <p id="p-75">
                     								(
                     								
                     <b>A</b>
                     								) Two-photon calcium imaging from the
                     								
                     <i>otd</i>
                     <sup>
                        <i>−/−</i>
                     </sup>
                     								LALv1 neurons in response to electrical stimulation of the
                     								antennal nerve. Gray-scale image shows average pre-stimulation
                     								fluorescence of one transformed lineage. Colour images show
                     								fluorescence changes in response to different numbers of
                     								electrical stimulation (duration, 1 ms; amplitude, 10 V;
                     								frequency, 100 Hz). Graph in
                     								
                     <b>B</b>
                     								shows peak amplitude of ΔF/F against the number of stimulations.
                     								Error bars represent S.E.M. n = 5. (
                     								
                     <b>C</b>
                     								) Average traces that plot ΔF/F over time at different stimulus
                     								intensities. Shaded region in each trace represents S.E.M. (
                     								
                     <b>D</b>
                     								) Two-photon calcium imaging from the
                     								
                     <i>otd</i>
                     <sup>
                        <i>−/−</i>
                     </sup>
                     								LALv1 neurons in response to odour stimulation of the antenna.
                     								Gray-scale image shows the structure of the transformed lineage
                     								and three identifiable glomeruli. The colour images show
                     								glomerular activation patterns evoked by isoamyl acetate (IAA),
                     								ethyl butyrate (EB), 3-octanol (3-O) and 3-heptanol (3-H). IAA
                     								activated VM2, DM2 and DM3. EB activated both VM2 and DM2. 3-O
                     								and 3-H activated DM2 and VM2, respectively. (
                     								
                     <b>E</b>
                     								) Average traces that plot ΔF/F over time in VM2, DM2 and DM3.
                     								The shaded region in each trace represents S.E.M. n = 3–5. The
                     								false colour scales (ΔF) and scale bars are shown at the right
                     								of each panel. Genotype:
                     								
                     <i>FRT19A, otd</i>
                     <sup>
                        <i>YH13</i>
                     </sup>
                     <i>/FRT19A,Tub-Gal80,hsFLP;GH146-Gal4,UAS-GCaMP3/GH146-Gal4,UAS-GCaMP3</i>
                     								.
                     							
                  </p>
                  <p id="p-76">
                     <b>DOI:</b>
                     <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.019</div>
                  </p>
               </caption>
               <div tagx="graphic"></div>
            </div>
         </p>
         <p id="p-77">
            					We further investigated if the transformed
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 neurons could respond to specific odour stimuli. To do this,
            					we performed calcium imaging experiments similar to those described
            					above but replaced the electrical stimulation of the antennal nerve
            					with odour stimulation of the intact antenna (olfactory sensory
            					neurons). Four different odorants (isoamyl acetate, ethyl butyrate,
            					3-octonal, 3-heptanol) were selected based on their ability to
            					excite all or some of the VM2, DM2 and DM3 glomeruli (
            					
            <span ref-type="bibr" rid="bib4">Dacks et al., 2009</span>
            					;
            					
            <span ref-type="bibr" rid="bib35">Semmelhack and Wang, 2009</span>
            					) in the antennal lobe, which are innervated by the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 neurons. Imaging calcium activity from the dendrites of the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 neurons in these glomeruli in response to the selected odours
            					show that each of the four odorants evoked a unique pattern of
            					glomerular activity. Isoamyl acetate excited all three glomeruli,
            					whereas ethyl butyrate excited only the VM2 and DM2 glomeruli.
            					3-octanol and 3-heptanol, however, excited just the DM2 and VM2
            					glomeruli, respectively (
            					
            <span ref-type="fig" rid="fig11">Figure 11D,E</span>
            					). These patterns of glomerular activation in the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 neurons are strikingly similar to that of wild-type antennal
            					lobe olfactory projection neurons (
            					
            <span ref-type="bibr" rid="bib47">Wang et al., 2003</span>
            					;
            					
            <span ref-type="bibr" rid="bib4">Dacks et al., 2009</span>
            					;
            					
            <span ref-type="bibr" rid="bib35">Semmelhack and Wang, 2009</span>
            					).
            				
         </p>
         <p id="p-78">
            					Taken together, these functional studies indicate that the
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					LALv1 neurons receive specific input from olfactory sensory neurons
            					that results in glomerulus-specific activation patterns to
            					different odorants. This in turn implies that
            					
            <i>otd</i>
            					loss-of-function in a single neuroblast leads to a remarkably
            					extensive reconfiguration of the macrocircuitry in the brain, which
            					includes anatomical, molecular as well as functional transformation
            					of neurons in the central complex lineage into neurons with
            					properties of olfactory projection neurons.
            				
         </p>
      </div>
   </div>
   <div id="s3">
      <h2>Discussion</h2>
      <p id="p-79">
         				During neuronal development in both vertebrates and invertebrates
         				neural progenitors use spatial and temporal information to generate
         				diverse neuronal subtypes. For example, in
         				
         <i>Drosophila</i>
         				, unique spatial information imparts heterogeneity to the neuroblast
         				pool and then temporal cues acting in the neuroblasts generate
         				further diversity. In this way diverse neuronal subtypes are
         				produced by the neuroblast lineages, which consequently create the
         				diverse functional macrocircuitry of the brain. In addition, the
         				neuroblasts in the central brain of
         				
         <i>Drosophila</i>
         				are characterized by the expression of a specific combination of
         				cell intrinsic determinants (
         				
         <span ref-type="bibr" rid="bib44">Urbach and Technau, 2003a</span>
         				) that are thought to act in the specification of unique neuroblast
         				identity and hence control lineage-specific neuronal cell fate.
         			
      </p>
      <p id="p-80">
         				In this study, we show that such intrinsic determinants present in
         				the neuroblast are essential for the proper specification of the
         				entire lineage that derives from the neuroblast. Our data
         				demonstrate that a remarkable rewiring of the functional
         				macrocircuitry of the brain occurs due to the manipulation of one
         				intrinsic factor,
         				
         <i>otd</i>
         				, acting in an identified neuroblast during development. This
         				transformation affects molecular properties, anatomical projection
         				patterns (dendritic and axonal), and functional inputs in all of the
         				neurons in the lineage (summarized in
         				
         <span ref-type="fig" rid="fig12">Figure 12</span>
         				) such that a central complex lineage is transformed into a
         				functional olfactory projection neuron lineage. This
         				
         <i>otd</i>
         				-dependent, lineage-specific respecification of interneurons has
         				implications for our understanding of the development and evolution
         				of the circuitry in the brain.
         				
         <div tagx="fig">
            <div tagx="object-id">elife;3/0/e04407/FIG12</div>
            <div tagx="object-id">F14</div>
            <div tagx="object-id">fig12</div>
            <div tagx="object-id">10.7554/eLife.04407.020</div>
            <div tagx="label">Figure 12.</div>
            <caption id="caption-18">
               <div tagx="title">Transformation of the LALv1 lineage.</div>
               <p id="p-81">
                  							(
                  							
                  <b>A</b>
                  							) The WT LALv1 is a wide-field central complex lineage that
                  							expresses Otd (orange). (
                  							
                  <b>B</b>
                  							) The loss of
                  							
                  <i>otd</i>
                  							function from the LALv1 neuroblast transforms these neurons into
                  							antennal lobe projection neurons. This neuroanatomical
                  							transformation is accompanied by molecular changes—markers that
                  							are active in the WT LALv1 (green box in
                  							
                  <b>A</b>
                  							) are suppressed in the mutant (
                  							
                  <b>B</b>
                  							; with the exception of Acj6); those that are suppressed in the
                  							WT (red box in
                  							
                  <b>A</b>
                  							), are activated in the mutant (
                  							
                  <b>B</b>
                  							; with the exception of LN1-Gal4). The exceptions are in
                  							agreement with a transformation to antennal lobe projection
                  							neuron fate (see
                  							
                  <span ref-type="table" rid="tbl1">Table 1</span>
                  							).
                  						
               </p>
               <p id="p-82">
                  <b>DOI:</b>
                  <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.020</div>
               </p>
            </caption>
            <div tagx="graphic"></div>
         </div>
      </p>
      <p id="p-83">
         				The ability of a neuroblast lineage to transform completely into
         				another upon the loss of a single intrinsic determinant suggests
         				that many of the other putative members of a potential neural
         				identity code might be shared between these lineages. The
         				observation that the neuroblasts of the central complex and antennal
         				lobe lineages develop in such close spatial proximity to each other
         				during early development suggests that these two neuroblasts may
         				experience similar spatial cues as they develop on the
         				neuroectoderm. If this is the case, then by manipulating a single
         				differentially expressed factor,
         				
         <i>otd</i>
         				, we might have been able to uncover the underlying similarity in
         				the intrinsic spatial code between the two neuroblast lineages.
         				Importantly, this neuroblast-specific transformation of lineage
         				identity resulted in an alteration of the brain's circuitry such
         				that an entire neuroanatomical unit of projection to the central
         				complex was lacking while a novel and functional ectopic unit of
         				projection was added to the antennal lobe.
         			
      </p>
      <p id="p-84">
         				Implicit in these findings are the notions of ‘coded’ and ‘soft’
         				properties of circuit assembly. On the one hand, the neuroanatomical
         				and molecular transformation described above demonstrates that
         				circuitry in the brain is ‘hard-wired’ or ‘coded’ by the spatially
         				encoded intrinsic factors—the presence or absence of
         				
         <i>otd</i>
         				from the central complex neuroblast determines the identity of the
         				resultant neurons. On the other hand, the resulting functional
         				transformation suggests that circuit assembly involves substantial
         				‘soft-wiring’—the olfactory sensory neurons and interneurons
         				indigenous to the antennal lobe are able to make functional
         				connections with the extraneous transformed
         				
         <i>otd</i>
         <sup>
            <i>−/−</i>
         </sup>
         				LALv1 neurons, which they are normally not ‘hard-wired’ to connect
         				with. Thus, while genetically encoded properties might ‘lock’
         				lineages into particular circuit states (central complex or antennal
         				lobe) it is their ‘soft’ properties (developmental plasticity) that
         				allow circuits to functionally incorporate changes as dramatic as
         				extraneous neurons. As both these wiring strategies operate
         				simultaneously in the brain, it bestows upon the brain the huge
         				potential of evolvability of functional circuits.
         			
      </p>
      <p id="p-85">
         				Many interesting questions emerge as a result of our findings. How
         				does the developmental plasticity of a functional circuit support
         				these large-scale rearrangements? Do developing circuits acquire a
         				propensity for exuberant connectivities, or do they try and maintain
         				a homeostasis in their connections and therefore make compensatory
         				changes in the number of synapses with their normal partners? It has
         				been shown in some cases that neuronal activity can mediate such
         				‘soft’ properties of synaptic connections (
         				
         <span ref-type="bibr" rid="bib41">Tripodi et al., 2008</span>
         				;
         				
         <span ref-type="bibr" rid="bib36">Singh et al., 2010</span>
         				). It will be interesting to test if this is also the case for the
         				transformed neurons and the olfactory circuit. Finally, do all parts
         				of the brain display such striking developmental plasticity such
         				that they can be remodelled to this extent and incorporate
         				extraneous neurons into existing circuitry?
         			
      </p>
      <p id="p-86">
         				The ability to change the functional macrocircuitry of the brain
         				through changes in the expression of a single transcription factor
         				in a single neuroblast lineage may provide a simple paradigm for
         				large-scale modification of brain connectivity during evolution. The
         				
         <i>otd</i>
         <sup>
            <i>−/−</i>
         </sup>
         				transformed LALv1 lineage functionally integrates into the antennal
         				lobe circuitry and participates in olfactory information processing.
         				This suggests that a functional rewiring of the olfactory circuitry
         				can occur due to the addition of an entire neuroblast lineage to the
         				normal olfactory circuit. In more general terms, this type of
         				lineage-specific rewiring might fuel the evolutionary modification
         				of neural circuitry in the brain. It provides an elegant and simple
         				solution to the evolution of complex circuitry in that a
         				‘microevolutionary’ molecular change (changing the expression of one
         				gene in one cell) can have ‘macroevolutionary’ consequences on the
         				brain's circuitry (changing an entire macrocircuit or an information
         				processing module). This simple strategy suggests that large-scale
         				changes in the brain's wiring do not need to come about through many
         				minor, sequentially accumulating changes at the cellular level.
         				Instead, large-scale wiring changes can occur in response to
         				remarkably simple changes in gene expression in single cells. In
         				this context, our data may have provided evidence for the
         				mechanistic ease with which circuitry in the brain can evolve.
         			
      </p>
   </div>
   <div id="s4">
      <h2>Materials and methods</h2>
      <div id="s4-1">
         <h3>Lineage nomenclature</h3>
         <p id="p-87">
            					The brain lineages have been named by various groups in the past.
            					Here, we list the various names by which the two lineages, which
            					are the focus of our study, have been called. ALad1 (
            					
            <span ref-type="bibr" rid="bib13">Ito et al., 2013</span>
            					;
            					
            <span ref-type="bibr" rid="bib49">Yu et al., 2013</span>
            					)/BAmv3 (
            					
            <span ref-type="bibr" rid="bib27">Pereanu and Hartenstein, 2006</span>
            					;
            					
            <span ref-type="bibr" rid="bib24">Lovick et al., 2013</span>
            					;
            					
            <span ref-type="bibr" rid="bib48">Wong et al., 2013</span>
            					)/adNB (
            					
            <span ref-type="bibr" rid="bib14">Jefferis et al., 2001</span>
            					). LALv1 (
            					
            <span ref-type="bibr" rid="bib13">Ito et al., 2013</span>
            					;
            					
            <span ref-type="bibr" rid="bib49">Yu et al., 2013</span>
            					)/BAmv1 (
            					
            <span ref-type="bibr" rid="bib27">Pereanu and Hartenstein, 2006</span>
            					;
            					
            <span ref-type="bibr" rid="bib24">Lovick et al., 2013</span>
            					;
            					
            <span ref-type="bibr" rid="bib48">Wong et al., 2013</span>
            					).
            				
         </p>
      </div>
      <div id="s4-2">
         <h3>Fly strains and genetics</h3>
         <p id="p-88">Fly stocks were obtained from the Bloomington Stock
            					Centre (IN, USA) and, unless otherwise stated, were grown on
            					cornmeal media, at 25°C. UAS-miRNA otd-1 was kindly provided by
            					Henry Sun, Taiwan.
         </p>
         <p id="p-89">
            					To generate WT MARCM clones females of the genotype
            					
            <i>FRT19A</i>
            					or
            					
            <i>FRT19A;UAS-mCD8::GFP</i>
            					were crossed to males of the following genotypes:
            					
            <i>FR19A,Tub-Gal80,hsFLP;
               						Tub-Gal4,UAS-mCD8::GFP/CyO
            </i>
            					or
            					
            <i>FR19A,Tub-Gal80,hsFLP;
               						OK371-Gal4,UAS-mCD8::GFP/CyO
            </i>
            					or
            					
            <i>FR19A,Tub-Gal80,hsFLP;
               						Per-Gal4,UAS-mCD8::GFP/CyO
            </i>
            					or
            					
            <i>FR19A,Tub-Gal80,hsFLP;
               						GH146-Gal4,UAS-mCD8::GFP/CyO
            </i>
            					or
            					
            <i>FR19A,Tub-Gal80,hsFLP;
               						LN1-Gal4,UAS-mCD8::GFP/CyO
            </i>
            					or
            					
            <i>FR19A,Tub-Gal80,hsFLP;
               						Cha7.4-Gal4,UAS-GFP/CyO.
            </i>
            					To generate
            					
            <i>otd</i>
            <sup>
               <i>−/−</i>
            </sup>
            					MARCM clones, females of the genotype
            					
            <i>FRT19A,otd</i>
            <sup>
               <i>YH13</i>
            </sup>
            <i>/FM7c</i>
            					or
            					
            <i>FRT19A,oc</i>
            <sup>
               <i>2</i>
            </sup>
            <i>/FM7c</i>
            					or
            					
            <i>FRT19A, otd</i>
            <sup>
               <i>YH13</i>
            </sup>
            <i>;UAS-mCD8::GFP</i>
            					or
            					
            <i>FRT19A,oc</i>
            <sup>
               <i>2</i>
            </sup>
            <i>;UAS-mCD8::GFP</i>
            					were crossed to males of the above-mentioned genotypes.
            				
         </p>
         <p id="p-90">
            					To knockdown
            					
            <i>otd</i>
            					in neuroblasts using RNAi and simultaneously visualise
            					GH146-labelled cells, females of the genotype
            					
            <i>UAS-dicer; Insc-Gal4/CyO</i>
            					were crossed to males of the genotype
            					
            <i>UAS-miRNA otd-1/CyO;
               						GH146-QF,QUAS-mtdTomato-HA/TM6B
            </i>
            					and the crosses were grown at 25°C.
            				
         </p>
         <p id="p-91">
            					To generate Dual MARCM clones, females of the genotype
            					
            <i>FRT19A,otd</i>
            <sup>
               <i>YH13</i>
            </sup>
            <i>/FM7a; FRTG13,
               						GH146-LexA::GAD,LexAop-rCD2::GFP/CyO
            </i>
            					were crossed to males of the following genotypes:
            					
            <i>FRT19A,Tub-Gal80,hsFLP;
               						Tub-Gal4,UAS-mCD8::GFP/CyO or FRT19A,Tub-Gal80,hsFLP;
               						OK371-Gal4,UAS-mCD8::GFP/CyO
            </i>
            					or
            					
            <i>FRT19A,Tub-Gal80,hsFLP;
               						Per-Gal4,UAS-mCD8::GFP/CyO.
            </i>
         </p>
         <p id="p-92">
            					To generate Otd overexpression clones, females of the genotype
            					
            <i>FRT19A</i>
            					;
            					
            <i>UAS-otd</i>
            					were crossed to males of the genotype
            					
            <i>FR19A,Tub-Gal80,hsFLP;
               						Tub-Gal4,UAS-mCD8::GFP/CyO.
            </i>
            					To generate rescue clones, females of the genotype
            					
            <i>FRT19A otd</i>
            <sup>
               <i>YH13</i>
            </sup>
            <i>/FM7a</i>
            					;
            					
            <i>UAS-otd</i>
            					were crossed to males of the genotype
            					
            <i>FR19A,Tub-Gal80,hsFLP;
               						Tub-Gal4,UAS-mCD8::GFP/CyO.
            </i>
         </p>
         <p id="p-93">
            					To generate clones for functional imaging, females of the genotype
            					
            <i>FRT19A,otd</i>
            <sup>
               <i>YH13</i>
            </sup>
            <i>/FM7a;GH146-Gal4,UAS-GCaMP3/CyO</i>
            					were crossed to males of the genotype
            					
            <i>FR19A,Tub-Gal80,hsFLP;
               						GH146-Gal4,UAS-GCaMP3/CyO.
            </i>
         </p>
         <p id="p-94">Embryos were collected from the progeny of all the
            					clonal crosses and these were given a heat shock for 1 hr at 37°C
            					at either the embryonic stage, 0–4 hr after larval hatching or 24
            					hr after larval hatching.
         </p>
         <p id="p-95">
            					To knock down Otd in neuroblasts using RNA interference, females of
            					the genotype
            					
            <i>UASdicer; Insc-Gal4</i>
            					were crossed to males of the genotype
            					
            <i>UAS-miRNA-otd-1;GH146-QF,QUAS-mtdTomato-HA/TM6B</i>
            					and grown at 25°C.
            				
         </p>
         <div id="s4-2-1">
            <h3>Generation of UAS-Otd</h3>
            <p id="p-96">
               						A full-length
               						
               <i>otd</i>
               						-cDNA for expression under the control of the UAS promoter was
               						made in the KVR lab by cloning a full-length
               						
               <i>otd</i>
               						cDNA including the 5′ and 3′ UTRs into the pJRFC-MUH vector. The
               						full-length o
               						
               <i>td</i>
               						cDNA clone RE-10280 (pBluscript backbone) was purchased from DGRC.
               						Confirmed plasmid DNA (pJFRC-10xUAS-Otd FL with an attB
               						integration site) was microinjected (2 μg/μl in 1× microinjection
               						buffer) into
               						
               <i>Drosophila</i>
               						embryos that contain PhiC31 integrase and selected attP docking
               						site on the second chromosome. Further crossing of G0 flies,
               						screening of the transformants and balancing of insertions
               						performed at the Transgenic Fly Facility at C-CAMP facility at
               						NCBS campus, Bangalore, India.
               					
            </p>
         </div>
      </div>
      <div id="s4-3">
         <h3>Immunohistochemistry and imaging</h3>
         <p id="p-97">Brains were dissected in 1× PBS and fixed in freshly
            					prepared 4% PFA for 30 min at room temperature. The fixative was
            					removed, and the brains were washed with blocking solution (1× PBS
            					with 0.3% TritonX and 0.1% BSA). Primary antibodies were diluted in
            					blocking solution. The samples were incubated in a moist chamber on
            					horizontal shaker at 4°C for 24 hr. Samples were then washed with
            					0.3% PTX (1× PBS with 0.3% TritonX) and secondary antibody diluted
            					in 0.3% PTX was added. The samples were incubated in this at 4°C in
            					a moist chamber on horizontal shaker overnight, after which they
            					were washed and mounted in vectashield on glass slides.
         </p>
         <p id="p-98">Primary antibodies used were: rabbit anti-GFP (1:10,000;
            					Molecular Probes, Invitrogen, Delhi, India); chick anti-GFP
            					(1:10,000, AbCam, Cambridge, UK); mouse anti-Bruchpilot (mAbnc82,
            					1:20; DSHB, Iowa, USA); Rb anti-Otd (1:1,500, gift from Henry Sun,
            					IMB, Sinica, Taiwan); guinea pig anti-Otd (1:7,500, gift from
            					Tiffany Cook, Cincinnati, OH); mouse anti-Acj6 (1:25, DSHB, Iowa,
            					USA); rabbit anti-dLim1 (1:1000; a gift from J Botas); rat
            					anti-mCD8 (1:100, Caltag, Burlingname, Californina); mouse
            					anti-rCD2 (AbD Serotec, Raleigh, NC, USA); mouse anti-Neurotactin
            					(BP106, 1:10, DSHB, Iowa, USA); mouse anti-Neuroglian (BP102, 1:10,
            					DSHB, Iowa, USA); rabbit anti-HA (1:100, AbCam). Secondary
            					antibodies used were Alexa Fluor-488-, Alexa Fluor-568- and Alexa
            					Fluor-647-coupled antibodies generated in goat (1:400; Molecular
            					Probes, Invitrogen, Delhi, India) and TO-PRO -3 iodide-661 (Thermo
            					Fischer Scientific, MA,USA).
         </p>
         <p id="p-99">All samples were imaged on Olympus Fluoview (FV1000)
            					laser scanning confocal microscope. Optical sections were acquired
            					at 1-µm intervals with a picture size of 512 × 512 pixels. Images
            					were digitally processed using Adobe Photoshop CS3. 3-D
            					reconstructions were made using Amira.
         </p>
      </div>
      <div id="s4-4">
         <h3>Functional imaging</h3>
         <p id="p-100">
            					Brains of clonal animals were dissected in Ca
            					
            <sup>2+</sup>
            					-free AHL saline, which contains 108 mM NaCl, 5 mM KCl, 4 mM NaHCO
            					
            <div tagx="sub">3</div>
            					, 1 mM NaH
            					
            <div tagx="sub">2</div>
            					PO
            					
            <div tagx="sub">4</div>
            					, 8.2 mM MgCl
            					
            <div tagx="sub">2</div>
            					, 5 mM HEPES, 5 mM trehalose, and 10 mM sucrose, with pH adjusted
            					to 7.4. Live brains that contain only the
            					
            <i>otd</i>
            					null transformed neurons were selected for imaging experiments.
            				
         </p>
         <div id="s4-4-1">
            <h3>Two-photon calcium imaging and antennal nerve
               						stimulation (electrical)
            </h3>
            <p id="p-101">
               						Two-photon calcium imaging was performed as described previously (
               						
               <span ref-type="bibr" rid="bib34">Root et al., 2008</span>
               						). The antennal nerves were cut from the base of the antennae. The
               						brain preparation was then pinned down on a Sylgard-coated petri
               						dish with AHL saline containing 2 mM CaCl
               						
               <div tagx="sub">2</div>
               						. The antennal nerve was stimulated electrically with a glass
               						suction electrode, at 1 ms in duration, 10 V in amplitude and 100
               						Hz in frequency (S48 Grass stimulator). The response of the
               						transformed neurons was monitored by a custom-built two-photon
               						microscope. Excitation wavelength was 930 nm. Images were captured
               						at 4 frames/s.
               					
            </p>
         </div>
         <div id="s4-4-2">
            <h3>Two-photon calcium imaging and odour
               						stimulation
            </h3>
            <p id="p-102">
               						For odour stimulation, the fly brain was dissected leaving the
               						antennae intact and embedded in agarose containing AHL saline with
               						2 mM CaCl
               						
               <div tagx="sub">2</div>
               						. The agarose gel was removed from the antennae and a piece of
               						Kimwipes was used to dry the antennae. A glass coverslip was
               						placed on top of the brain preparation for imaging. Odour delivery
               						was controlled by solenoid valves described previously (
               						
               <span ref-type="bibr" rid="bib34">Root et al., 2008</span>
               						). Odour vapour was obtained by placing a filter paper containing
               						10 μl of an odorant in a 100-ml bottle. Mixing an air stream with
               						an odour stream at different flow ratios was used to deliver
               						odorants at a specific concentration. Isoamyl acetate and ethyl
               						butyrate were delivered at 1% (odorant at 10 ml/min and air at 990
               						ml/min), whereas 3-heptanol and 3-octanol were delivered at 2.5%
               						saturated vapor pressure. Each odorant was applied for a duration
               						of 2 s. Images were acquired for 20 s at a rate of 4 frames/s and
               						a resolution of 128 × 128 pixels. At the end of each experiment,
               						an image stack was collected at a resolution of 512 × 512 pixels
               						for glomerulus identification. The data were analyzed and plotted
               						using Igor Pro 6.2 (Wavemetrics). The peak response of stimulation
               						(ΔF/F) was shown as mean ± S.E.M.
               					
            </p>
         </div>
      </div>
   </div>
   <div tagx="back">
      <div tagx="ack">
         <div tagx="title">Acknowledgements</div>
         <p id="p-103">We dedicate this work to the memory of our late friend
            				and colleague Veronica Rodrigues (1953–2010). We acknowledge DST,
            				Govt. of India–Centre for Nanotechnology (no. SR/S5/NM-36/2005) for
            				CIFF at NCBS. We are grateful to Henry Sun and Tiffany Cook for
            				their generous gift of reagents.
         </p>
      </div>
      <div id="">
         <h2>Additional information</h2>
         <div tagx="fn-group">
            <div tagx="title">Competing interests</div>
            <div tagx="fn">
               <p id="p-104">
                  						KVR: Senior editor,
                  						
                  <i>eLife.</i>
               </p>
            </div>
            <div tagx="fn">
               <p id="p-105">The other authors declare that no competing interests
                  						exist.
               </p>
            </div>
         </div>
         <div tagx="fn-group">
            <div tagx="title">Author contributions</div>
            <div tagx="fn">
               <p id="p-106">SS, Conception and design, Acquisition of data,
                  						Analysis and interpretation of data, Drafting or revising the
                  						article
               </p>
            </div>
            <div tagx="fn">
               <p id="p-107">DC, Conception and design, Acquisition of data,
                  						Analysis and interpretation of data, Drafting or revising the
                  						article
               </p>
            </div>
            <div tagx="fn">
               <p id="p-108">RC, Conception and design, Acquisition of data,
                  						Analysis and interpretation of data, Drafting or revising the
                  						article
               </p>
            </div>
            <div tagx="fn">
               <p id="p-109">SB, Conception and design, Acquisition of data,
                  						Analysis and interpretation of data, Drafting or revising the
                  						article
               </p>
            </div>
            <div tagx="fn">
               <p id="p-110">JWW, Conception and design, Analysis and interpretation
                  						of data, Drafting or revising the article
               </p>
            </div>
            <div tagx="fn">
               <p id="p-111">HR, Conception and design, Analysis and interpretation
                  						of data, Drafting or revising the article
               </p>
            </div>
            <div tagx="fn">
               <p id="p-112">KVR, Conception and design, Analysis and interpretation
                  						of data, Drafting or revising the article
               </p>
            </div>
         </div>
      </div>
      <div tag="ref-list">
         <ul>
            <div tagx="title">References</div>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Cardona</span>
                        <span tagx="given-names">A</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Saalfeld</span>
                        <span tagx="given-names">S</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Arganda</span>
                        <span tagx="given-names">I</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Pereanu</span>
                        <span tagx="given-names">W</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Schindelin</span>
                        <span tagx="given-names">J</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Hartenstein</span>
                        <span tagx="given-names">V</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2010</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Identifying neuronal lineages of
                     						
                     <i>Drosophila</i>
                     						by sequence analysis of axon tracts
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">The Journal of Neuroscience</span>
                  <span tagx="volume">30</span>
                  					:
                  					
                  <span tagx="fpage">7538</span>
                  					–
                  					
                  <span tagx="lpage">7553</span>
                  					,
                  					
                  <div tagx="pub-id">10.1523/JNEUROSCI.0186-10.2010</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Chou</span>
                        <span tagx="given-names">YH</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Spletter</span>
                        <span tagx="given-names">ML</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Yaksi</span>
                        <span tagx="given-names">E</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Leong</span>
                        <span tagx="given-names">JC</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Wilson</span>
                        <span tagx="given-names">RI</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Luo</span>
                        <span tagx="given-names">L</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2010</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Diversity and wiring variability of olfactory local interneurons
                     						in the
                     						
                     <i>Drosophila</i>
                     						antennal lobe
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Nature Neuroscience</span>
                  <span tagx="volume">13</span>
                  					:
                  					
                  <span tagx="fpage">439</span>
                  					–
                  					
                  <span tagx="lpage">449</span>
                  					,
                  					
                  <div tagx="pub-id">10.1038/nn.2489</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Custo Greig</span>
                        <span tagx="given-names">L</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Woodworth</span>
                        <span tagx="given-names">M</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Galazo</span>
                        <span tagx="given-names">M</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Padmanabhan</span>
                        <span tagx="given-names">H</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Macklis</span>
                        <span tagx="given-names">J</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2013</span>
                  					)
                  					
                  <span tagx="article-title">Molecular logic of neocortical
                     						projection neuron specification, development and diversity
                  </span>
                  					.
                  					
                  <span tagx="source">Nature Reviews Neuroscience</span>
                  <span tagx="volume">14</span>
                  					:
                  					
                  <span tagx="fpage">755</span>
                  					–
                  					
                  <span tagx="lpage">769</span>
                  					,
                  					
                  <div tagx="pub-id">10.1038/nrn3586</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Dacks</span>
                        <span tagx="given-names">AM</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Green</span>
                        <span tagx="given-names">DS</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Root</span>
                        <span tagx="given-names">CM</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Nighorn</span>
                        <span tagx="given-names">AJ</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Wang</span>
                        <span tagx="given-names">JW</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2009</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Serotonin modulates olfactory processing in the antennal lobe of
                     						
                     <i>Drosophila</i>
                  </span>
                  					.
                  					
                  <span tagx="source">Journal of Neurogenetics</span>
                  <span tagx="volume">23</span>
                  					:
                  					
                  <span tagx="fpage">366</span>
                  					–
                  					
                  <span tagx="lpage">377</span>
                  					,
                  					
                  <div tagx="pub-id">10.3109/01677060903085722</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Das</span>
                        <span tagx="given-names">A</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Chiang</span>
                        <span tagx="given-names">A</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Davla</span>
                        <span tagx="given-names">S</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Priya</span>
                        <span tagx="given-names">R</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Reichert</span>
                        <span tagx="given-names">H</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Vijayraghavan</span>
                        <span tagx="given-names">K</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Rodrigues</span>
                        <span tagx="given-names">V</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2011</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Identification and analysis of a glutamatergic local interneuron
                     						lineage in the adult
                     						
                     <i>Drosophila</i>
                     						olfactory system
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Neural Systems &amp; Circuits</span>
                  <span tagx="volume">1</span>
                  					:
                  					
                  <span tagx="fpage">4</span>
                  					,
                  					
                  <div tagx="pub-id">10.1186/2042-1001-1-4</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Das</span>
                        <span tagx="given-names">A</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Gupta</span>
                        <span tagx="given-names">T</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Davla</span>
                        <span tagx="given-names">S</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Prieto-Godino</span>
                        <span tagx="given-names">LL</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Diegelmann</span>
                        <span tagx="given-names">S</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Reddy</span>
                        <span tagx="given-names">OV</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Raghavan</span>
                        <span tagx="given-names">KV</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Reichert</span>
                        <span tagx="given-names">H</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Lovick</span>
                        <span tagx="given-names">J</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Hartenstein</span>
                        <span tagx="given-names">V</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2013</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Neuroblast lineage-specific origin of the neurons of the
                     						
                     <i>Drosophila</i>
                     						larval olfactory system
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Developmental Biology</span>
                  <span tagx="volume">373</span>
                  					:
                  					
                  <span tagx="fpage">322</span>
                  					–
                  					
                  <span tagx="lpage">337</span>
                  					,
                  					
                  <div tagx="pub-id">10.1016/j.ydbio.2012.11.003</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Das</span>
                        <span tagx="given-names">A</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Sen</span>
                        <span tagx="given-names">S</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Lichtneckert</span>
                        <span tagx="given-names">R</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Okada</span>
                        <span tagx="given-names">R</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Ito</span>
                        <span tagx="given-names">K</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Rodrigues</span>
                        <span tagx="given-names">V</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Reichert</span>
                        <span tagx="given-names">H</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2008</span>
                  					)
                  					
                  <span tagx="article-title">
                     <i>Drosophila</i>
                     						olfactory local interneurons and projection neurons derive from a
                     						common neuroblast lineage specified by the empty spiracles gene
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Neural Development</span>
                  <span tagx="volume">3</span>
                  					:
                  					
                  <span tagx="fpage">33</span>
                  					,
                  					
                  <div tagx="pub-id">10.1186/1749-8104-3-33</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Deshpande</span>
                        <span tagx="given-names">N</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Dittrich</span>
                        <span tagx="given-names">R</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Technau</span>
                        <span tagx="given-names">GM</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Urban</span>
                        <span tagx="given-names">J</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2001</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Successive specification of
                     						
                     <i>Drosophila</i>
                     						neuroblasts NB 6-4 and NB 7-3 depends on interaction of the
                     						segment polarity genes wingless, gooseberry and naked cuticle
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Development</span>
                  <span tagx="volume">128</span>
                  					:
                  					
                  <span tagx="fpage">3253</span>
                  					–
                  					
                  <span tagx="lpage">3261</span>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Franco</span>
                        <span tagx="given-names">SJ</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Muller</span>
                        <span tagx="given-names">U</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2013</span>
                  					)
                  					
                  <span tagx="article-title">Shaping our Minds: stem and
                     						progenitor cell diversity in the Mammalian neocortex
                  </span>
                  					.
                  					
                  <span tagx="source">Neuron</span>
                  <span tagx="volume">77</span>
                  					:
                  					
                  <span tagx="fpage">19</span>
                  					–
                  					
                  <span tagx="lpage">34</span>
                  					,
                  					
                  <div tagx="pub-id">10.1016/j.neuron.2012.12.022</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Gao</span>
                        <span tagx="given-names">P</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Sultan</span>
                        <span tagx="given-names">KT</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Zhang</span>
                        <span tagx="given-names">XJ</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Shi</span>
                        <span tagx="given-names">SH</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2013</span>
                  					)
                  					
                  <span tagx="article-title">Lineage-dependent circuit
                     						assembly in the neocortex
                  </span>
                  					.
                  					
                  <span tagx="source">Development</span>
                  <span tagx="volume">140</span>
                  					:
                  					
                  <span tagx="fpage">2645</span>
                  					–
                  					
                  <span tagx="lpage">2655</span>
                  					,
                  					
                  <div tagx="pub-id">10.1242/dev.087668</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Ito</span>
                        <span tagx="given-names">K</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Awano</span>
                        <span tagx="given-names">W</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Suzuki</span>
                        <span tagx="given-names">K</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Hiromi</span>
                        <span tagx="given-names">Y</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Yamamoto</span>
                        <span tagx="given-names">D</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">1997</span>
                  					)
                  					
                  <span tagx="article-title">
                     						The
                     						
                     <i>Drosophila</i>
                     						mushroom body is a quadruple structure of clonal units each of
                     						which contains a virtually identical set of neurones and glial
                     						cells
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Development</span>
                  <span tagx="volume">124</span>
                  					:
                  					
                  <span tagx="fpage">761</span>
                  					–
                  					
                  <span tagx="lpage">771</span>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Ito</span>
                        <span tagx="given-names">K</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Hotta</span>
                        <span tagx="given-names">Y</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">1992</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Proliferation pattern of postembryonic neuroblasts in the brain of
                     						
                     <i>Drosophila melanogaster</i>
                  </span>
                  					.
                  					
                  <span tagx="source">Developmental Biology</span>
                  <span tagx="volume">149</span>
                  					:
                  					
                  <span tagx="fpage">134</span>
                  					–
                  					
                  <span tagx="lpage">148</span>
                  					,
                  					
                  <div tagx="pub-id">10.1016/0012-1606(92)90270-Q</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Ito</span>
                        <span tagx="given-names">M</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Masuda</span>
                        <span tagx="given-names">N</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Shinomiya</span>
                        <span tagx="given-names">K</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Endo</span>
                        <span tagx="given-names">K</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Ito</span>
                        <span tagx="given-names">K</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2013</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Systematic analysis of neural projections reveals clonal
                     						composition of the
                     						
                     <i>Drosophila</i>
                     						brain
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Current Biology</span>
                  <span tagx="volume">23</span>
                  					:
                  					
                  <span tagx="fpage">644</span>
                  					–
                  					
                  <span tagx="lpage">655</span>
                  					,
                  					
                  <div tagx="pub-id">10.1016/j.cub.2013.03.015</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Jefferis</span>
                        <span tagx="given-names">GS</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Marin</span>
                        <span tagx="given-names">EC</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Stocker</span>
                        <span tagx="given-names">RF</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Luo</span>
                        <span tagx="given-names">L</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2001</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Target neuron prespecification in the olfactory map of
                     						
                     <i>Drosophila</i>
                  </span>
                  					.
                  					
                  <span tagx="source">Nature</span>
                  <span tagx="volume">414</span>
                  					:
                  					
                  <span tagx="fpage">204</span>
                  					–
                  					
                  <span tagx="lpage">208</span>
                  					,
                  					
                  <div tagx="pub-id">10.1038/35102574</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Jefferis</span>
                        <span tagx="given-names">GS</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Vyas</span>
                        <span tagx="given-names">RM</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Berdnik</span>
                        <span tagx="given-names">D</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Ramaekers</span>
                        <span tagx="given-names">A</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Stocker</span>
                        <span tagx="given-names">RF</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Tanaka</span>
                        <span tagx="given-names">NK</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Ito</span>
                        <span tagx="given-names">K</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Luo</span>
                        <span tagx="given-names">L</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2004</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Developmental origin of wiring specificity in the olfactory system
                     						of
                     						
                     <i>Drosophila</i>
                  </span>
                  					.
                  					
                  <span tagx="source">Development</span>
                  <span tagx="volume">131</span>
                  					:
                  					
                  <span tagx="fpage">117</span>
                  					–
                  					
                  <span tagx="lpage">130</span>
                  					,
                  					
                  <div tagx="pub-id">10.1242/dev.00896</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Kohwi</span>
                        <span tagx="given-names">M</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Doe</span>
                        <span tagx="given-names">CQ</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2013</span>
                  					)
                  					
                  <span tagx="article-title">Temporal fate specification and
                     						neural progenitor competence during development
                  </span>
                  					.
                  					
                  <span tagx="source">Nature Reviews Neuroscience</span>
                  <span tagx="volume">14</span>
                  					:
                  					
                  <span tagx="fpage">823</span>
                  					–
                  					
                  <span tagx="lpage">838</span>
                  					,
                  					
                  <div tagx="pub-id">10.1038/nrn3618</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Lai</span>
                        <span tagx="given-names">SL</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Awasaki</span>
                        <span tagx="given-names">T</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Ito</span>
                        <span tagx="given-names">K</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Lee</span>
                        <span tagx="given-names">T</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2008</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Clonal analysis of
                     						
                     <i>Drosophila</i>
                     						antennal lobe neurons: diverse neuronal architectures in the
                     						lateral neuroblast lineage
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Development</span>
                  <span tagx="volume">135</span>
                  					:
                  					
                  <span tagx="fpage">2883</span>
                  					–
                  					
                  <span tagx="lpage">2893</span>
                  					,
                  					
                  <div tagx="pub-id">10.1242/dev.024380</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Lai</span>
                        <span tagx="given-names">SL</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Lee</span>
                        <span tagx="given-names">T</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2006</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Genetic mosaic with dual binary transcriptional systems in
                     						
                     <i>Drosophila</i>
                  </span>
                  					.
                  					
                  <span tagx="source">Nature Neuroscience</span>
                  <span tagx="volume">9</span>
                  					:
                  					
                  <span tagx="fpage">703</span>
                  					–
                  					
                  <span tagx="lpage">709</span>
                  					,
                  					
                  <div tagx="pub-id">10.1038/nn1681</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Lee</span>
                        <span tagx="given-names">T</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Luo</span>
                        <span tagx="given-names">L</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2001</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Mosaic analysis with a repressible cell marker (MARCM) for
                     						
                     <i>Drosophila</i>
                     						neural development
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Trends in Neurosciences</span>
                  <span tagx="volume">24</span>
                  					:
                  					
                  <span tagx="fpage">251</span>
                  					–
                  					
                  <span tagx="lpage">254</span>
                  					,
                  					
                  <div tagx="pub-id">10.1016/S0166-2236(00)01791-4</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Li</span>
                        <span tagx="given-names">Y</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Lu</span>
                        <span tagx="given-names">H</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Cheng</span>
                        <span tagx="given-names">P</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Ge</span>
                        <span tagx="given-names">S</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Xu</span>
                        <span tagx="given-names">H</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Shi</span>
                        <span tagx="given-names">SH</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Dan</span>
                        <span tagx="given-names">Y</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2012</span>
                  					)
                  					
                  <span tagx="article-title">Clonally related visual
                     						cortical neurons show similar stimulus feature selectivity
                  </span>
                  					.
                  					
                  <span tagx="source">Nature</span>
                  <span tagx="volume">486</span>
                  					:
                  					
                  <span tagx="fpage">118</span>
                  					–
                  					
                  <span tagx="lpage">121</span>
                  					,
                  					
                  <div tagx="pub-id">10.1038/nature11110</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Lichtneckert</span>
                        <span tagx="given-names">R</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Reichert</span>
                        <span tagx="given-names">H</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2008</span>
                  					)
                  					
                  <span tagx="article-title">Anteroposterior regionalization
                     						of the brain: genetic and comparative aspects
                  </span>
                  					.
                  					
                  <span tagx="source">Advances in Experimental Medicine and Biology</span>
                  <span tagx="volume">628</span>
                  					:
                  					
                  <span tagx="fpage">32</span>
                  					–
                  					
                  <span tagx="lpage">41</span>
                  					,
                  					
                  <div tagx="pub-id">10.1007/978-0-387-78261-4_2</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Lichtneckert</span>
                        <span tagx="given-names">R</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Reichert</span>
                        <span tagx="given-names">H</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2005</span>
                  					)
                  					
                  <span tagx="article-title">Insights into the urbilaterian
                     						brain: conserved genetic patterning mechanisms in insect and
                     						vertebrate brain development
                  </span>
                  					.
                  					
                  <span tagx="source">Heredity</span>
                  <span tagx="volume">94</span>
                  					:
                  					
                  <span tagx="fpage">465</span>
                  					–
                  					
                  <span tagx="lpage">477</span>
                  					,
                  					
                  <div tagx="pub-id">10.1038/sj.hdy.6800664</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Lin</span>
                        <span tagx="given-names">S</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Lee</span>
                        <span tagx="given-names">T</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2012</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Generating neuronal diversity in the
                     						
                     <i>Drosophila</i>
                     						central nervous system
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Developmental Dynamics</span>
                  <span tagx="volume">241</span>
                  					:
                  					
                  <span tagx="fpage">57</span>
                  					–
                  					
                  <span tagx="lpage">68</span>
                  					,
                  					
                  <div tagx="pub-id">10.1002/dvdy.22739</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Lovick</span>
                        <span tagx="given-names">JK</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Ngo</span>
                        <span tagx="given-names">KT</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Omoto</span>
                        <span tagx="given-names">JJ</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Wong</span>
                        <span tagx="given-names">DC</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Nguyen</span>
                        <span tagx="given-names">JD</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Hartenstein</span>
                        <span tagx="given-names">V</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2013</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Postembryonic lineages of the
                     						
                     <i>Drosophila</i>
                     						brain: I. Development of the lineage-associated fiber tracts
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Developmental Biology</span>
                  <span tagx="volume">384</span>
                  					:
                  					
                  <span tagx="fpage">228</span>
                  					–
                  					
                  <span tagx="lpage">257</span>
                  					,
                  					
                  <div tagx="pub-id">10.1016/j.ydbio.2013.07.008</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Ohtsuki</span>
                        <span tagx="given-names">G</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Nishiyama</span>
                        <span tagx="given-names">M</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Yoshida</span>
                        <span tagx="given-names">T</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Murakami</span>
                        <span tagx="given-names">T</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Histed</span>
                        <span tagx="given-names">M</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Lois</span>
                        <span tagx="given-names">C</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Ohki</span>
                        <span tagx="given-names">K</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2012</span>
                  					)
                  					
                  <span tagx="article-title">Similarity of visual
                     						selectivity among clonally related neurons in visual cortex
                  </span>
                  					.
                  					
                  <span tagx="source">Neuron</span>
                  <span tagx="volume">75</span>
                  					:
                  					
                  <span tagx="fpage">65</span>
                  					–
                  					
                  <span tagx="lpage">72</span>
                  					,
                  					
                  <div tagx="pub-id">10.1016/j.neuron.2012.05.023</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Pearson</span>
                        <span tagx="given-names">BJ</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Doe</span>
                        <span tagx="given-names">CQ</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2004</span>
                  					)
                  					
                  <span tagx="article-title">Specification of temporal
                     						identity in the developing nervous system
                  </span>
                  					.
                  					
                  <span tagx="source">Annual Review of Cell and Developmental
                     						Biology
                  </span>
                  <span tagx="volume">20</span>
                  					:
                  					
                  <span tagx="fpage">619</span>
                  					–
                  					
                  <span tagx="lpage">647</span>
                  					,
                  					
                  <div tagx="pub-id">10.1146/annurev.cellbio.19.111301.115142</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Pereanu</span>
                        <span tagx="given-names">W</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Hartenstein</span>
                        <span tagx="given-names">V</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2006</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Neural lineages of the
                     						
                     <i>Drosophila</i>
                     						brain: a three-dimensional digital atlas of the pattern of lineage
                     						location and projection at the late larval stage
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">The Journal of Neuroscience</span>
                  <span tagx="volume">26</span>
                  					:
                  					
                  <span tagx="fpage">5534</span>
                  					–
                  					
                  <span tagx="lpage">5553</span>
                  					,
                  					
                  <div tagx="pub-id">10.1523/JNEUROSCI.4708-05.2006</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Pereanu</span>
                        <span tagx="given-names">W</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Kumar</span>
                        <span tagx="given-names">A</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Jennett</span>
                        <span tagx="given-names">A</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Reichert</span>
                        <span tagx="given-names">H</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Hartenstein</span>
                        <span tagx="given-names">V</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2010</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Development-based compartmentalization of the
                     						
                     <i>Drosophila</i>
                     						central brain
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">The Journal of Comparative Neurology</span>
                  <span tagx="volume">518</span>
                  					:
                  					
                  <span tagx="fpage">2996</span>
                  					–
                  					
                  <span tagx="lpage">3023</span>
                  					,
                  					
                  <div tagx="pub-id">10.1002/cne.22376</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Potter</span>
                        <span tagx="given-names">CJ</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Tasic</span>
                        <span tagx="given-names">B</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Russler</span>
                        <span tagx="given-names">EV</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Liang</span>
                        <span tagx="given-names">L</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Luo</span>
                        <span tagx="given-names">L</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2010</span>
                  					)
                  					
                  <span tagx="article-title">The Q system: a repressible
                     						binary system for transgene expression, lineage tracing and mosaic
                     						analysis
                  </span>
                  					.
                  					
                  <span tagx="source">Cell</span>
                  <span tagx="volume">141</span>
                  					:
                  					
                  <span tagx="fpage">536</span>
                  					–
                  					
                  <span tagx="lpage">548</span>
                  					,
                  					
                  <div tagx="pub-id">10.1016/j.cell.2010.02.025</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Rakic</span>
                        <span tagx="given-names">P</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">1988</span>
                  					)
                  					
                  <span tagx="article-title">Specification of cerebral
                     						cortical areas
                  </span>
                  					.
                  					
                  <span tagx="source">Science</span>
                  <span tagx="volume">241</span>
                  					:
                  					
                  <span tagx="fpage">170</span>
                  					–
                  					
                  <span tagx="lpage">176</span>
                  					,
                  					
                  <div tagx="pub-id">10.1126/science.3291116</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Reichert</span>
                        <span tagx="given-names">H</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2009</span>
                  					)
                  					
                  <span tagx="article-title">Evolutionary conservation of
                     						mechanisms for neural regionalization, proliferation and
                     						interconnection in brain development
                  </span>
                  					.
                  					
                  <span tagx="source">Biology Letters</span>
                  <span tagx="volume">5</span>
                  					:
                  					
                  <span tagx="fpage">112</span>
                  					–
                  					
                  <span tagx="lpage">116</span>
                  					,
                  					
                  <div tagx="pub-id">10.1098/rsbl.2008.0337</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Reichert</span>
                        <span tagx="given-names">H</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Bello</span>
                        <span tagx="given-names">B</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2010</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Hox genes and brain development in
                     						
                     <i>Drosophila</i>
                  </span>
                  					.
                  					
                  <span tagx="source">Advances in Experimental Medicine and Biology</span>
                  <span tagx="volume">689</span>
                  					:
                  					
                  <span tagx="fpage">145</span>
                  					–
                  					
                  <span tagx="lpage">153</span>
                  					,
                  					
                  <div tagx="pub-id">10.1007/978-1-4419-6673-5_11</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Reichert</span>
                        <span tagx="given-names">H</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Simeone</span>
                        <span tagx="given-names">A</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2001</span>
                  					)
                  					
                  <span tagx="article-title">Developmental genetic evidence
                     						for a monophyletic origin of the bilaterian brain
                  </span>
                  					.
                  					
                  <span tagx="source">Philosophical Transactions of the Royal
                     						Society of London. Series B, Biological Sciences
                  </span>
                  <span tagx="volume">356</span>
                  					:
                  					
                  <span tagx="fpage">1533</span>
                  					–
                  					
                  <span tagx="lpage">1544</span>
                  					,
                  					
                  <div tagx="pub-id">10.1098/rstb.2001.0972</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Root</span>
                        <span tagx="given-names">CM</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Masuyama</span>
                        <span tagx="given-names">K</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Green</span>
                        <span tagx="given-names">DS</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Enell</span>
                        <span tagx="given-names">LE</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Nässel</span>
                        <span tagx="given-names">DR</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Lee</span>
                        <span tagx="given-names">CH</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Wang</span>
                        <span tagx="given-names">JW</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2008</span>
                  					)
                  					
                  <span tagx="article-title">A presynaptic gain control
                     						mechanism fine-tunes olfactory behavior
                  </span>
                  					.
                  					
                  <span tagx="source">Neuron</span>
                  <span tagx="volume">59</span>
                  					:
                  					
                  <span tagx="fpage">311</span>
                  					–
                  					
                  <span tagx="lpage">321</span>
                  					,
                  					
                  <div tagx="pub-id">10.1016/j.neuron.2008.07.003</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Semmelhack</span>
                        <span tagx="given-names">JL</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Wang</span>
                        <span tagx="given-names">JW</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2009</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Select
                     						
                     <i>Drosophila</i>
                     						glomeruli mediate innate olfactory attraction and aversion
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Nature</span>
                  <span tagx="volume">459</span>
                  					:
                  					
                  <span tagx="fpage">218</span>
                  					–
                  					
                  <span tagx="lpage">223</span>
                  					,
                  					
                  <div tagx="pub-id">10.1038/nature07983</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Singh</span>
                        <span tagx="given-names">AP</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">VijayRaghavan</span>
                        <span tagx="given-names">K</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Rodrigues</span>
                        <span tagx="given-names">V</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2010</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Dendritic refinement of an identified neuron in the
                     						
                     <i>Drosophila</i>
                     						CNS is regulated by neuronal activity and Wnt signaling
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Development</span>
                  <span tagx="volume">137</span>
                  					:
                  					
                  <span tagx="fpage">1351</span>
                  					–
                  					
                  <span tagx="lpage">1360</span>
                  					,
                  					
                  <div tagx="pub-id">10.1242/dev.044131</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Skeath</span>
                        <span tagx="given-names">JB</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Thor</span>
                        <span tagx="given-names">S</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2003</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Genetic control of
                     						
                     <i>Drosophila</i>
                     						nerve cord development
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Current Opinion in Neurobiology</span>
                  <span tagx="volume">13</span>
                  					:
                  					
                  <span tagx="fpage">8</span>
                  					–
                  					
                  <span tagx="lpage">15</span>
                  					,
                  					
                  <div tagx="pub-id">10.1016/S0959-4388(03)00007-2</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Spindler</span>
                        <span tagx="given-names">SR</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Hartenstein</span>
                        <span tagx="given-names">V</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2011</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Bazooka mediates secondary axon morphology in
                     						
                     <i>Drosophila</i>
                     						brain lineages
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Neural Development</span>
                  <span tagx="volume">6</span>
                  					:
                  					
                  <span tagx="fpage">16</span>
                  					,
                  					
                  <div tagx="pub-id">10.1186/1749-8104-6-16</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Stocker</span>
                        <span tagx="given-names">RF</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Heimbeck</span>
                        <span tagx="given-names">G</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Gendre</span>
                        <span tagx="given-names">N</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">de Belle</span>
                        <span tagx="given-names">JS</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">1997</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Neuroblast ablation in
                     						
                     <i>Drosophila</i>
                     						P[GAL4] lines reveals origins of olfactory interneurons
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Journal of Neurobiology</span>
                  <span tagx="volume">32</span>
                  					:
                  					
                  <span tagx="fpage">443</span>
                  					–
                  					
                  <span tagx="lpage">456</span>
                  					,
                  					
                  <div tagx="pub-id">10.1002/(SICI)1097-4695(199705)32:53.0.CO;2-5</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Technau</span>
                        <span tagx="given-names">GM</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Berger</span>
                        <span tagx="given-names">C</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Urbach</span>
                        <span tagx="given-names">R</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2006</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Generation of cell diversity and segmental pattern in the
                     						embryonic central nervous system of
                     						
                     <i>Drosophila</i>
                  </span>
                  					.
                  					
                  <span tagx="source">Developmental Dynamics</span>
                  <span tagx="volume">235</span>
                  					:
                  					
                  <span tagx="fpage">861</span>
                  					–
                  					
                  <span tagx="lpage">869</span>
                  					,
                  					
                  <div tagx="pub-id">10.1002/dvdy.20566</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Tripodi</span>
                        <span tagx="given-names">M</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Evers</span>
                        <span tagx="given-names">JF</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Mauss</span>
                        <span tagx="given-names">A</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Bate</span>
                        <span tagx="given-names">M</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Landgraf</span>
                        <span tagx="given-names">M</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2008</span>
                  					)
                  					
                  <span tagx="article-title">Structural homeostasis:
                     						compensatory adjustments of dendritic arbor geometry in response
                     						to variations of synaptic input
                  </span>
                  					.
                  					
                  <span tagx="source">PLOS Biology</span>
                  <span tagx="volume">6</span>
                  					:
                  					
                  <span tagx="fpage">e260</span>
                  					,
                  					
                  <div tagx="pub-id">10.1371/journal.pbio.0060260</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Truman</span>
                        <span tagx="given-names">JW</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Bate</span>
                        <span tagx="given-names">M</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">1988</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Spatial and temporal patterns of neurogenesis in the central
                     						nervous system of
                     						
                     <i>Drosophila melanogaster</i>
                  </span>
                  					.
                  					
                  <span tagx="source">Developmental Biology</span>
                  <span tagx="volume">125</span>
                  					:
                  					
                  <span tagx="fpage">145</span>
                  					–
                  					
                  <span tagx="lpage">157</span>
                  					,
                  					
                  <div tagx="pub-id">10.1016/0012-1606(88)90067-X</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Urbach</span>
                        <span tagx="given-names">R</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Schnabel</span>
                        <span tagx="given-names">R</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Technau</span>
                        <span tagx="given-names">GM</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2003</span>
                  					)
                  					
                  <span tagx="article-title">
                     						The pattern of neuroblast formation, mitotic domains and proneural
                     						gene expression during early brain development in
                     						
                     <i>Drosophila</i>
                  </span>
                  					.
                  					
                  <span tagx="source">Development</span>
                  <span tagx="volume">130</span>
                  					:
                  					
                  <span tagx="fpage">3589</span>
                  					–
                  					
                  <span tagx="lpage">3606</span>
                  					,
                  					
                  <div tagx="pub-id">10.1242/dev.00528</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Urbach</span>
                        <span tagx="given-names">R</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Technau</span>
                        <span tagx="given-names">GM</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2003a</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Molecular markers for identified neuroblasts in the developing
                     						brain of
                     						
                     <i>Drosophila</i>
                  </span>
                  					.
                  					
                  <span tagx="source">Development</span>
                  <span tagx="volume">130</span>
                  					:
                  					
                  <span tagx="fpage">3621</span>
                  					–
                  					
                  <span tagx="lpage">3637</span>
                  					,
                  					
                  <div tagx="pub-id">10.1242/dev.00533</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Urbach</span>
                        <span tagx="given-names">R</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Technau</span>
                        <span tagx="given-names">GM</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2004</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Neuroblast formation and patterning during early brain development
                     						in
                     						
                     <i>Drosophila</i>
                  </span>
                  					.
                  					
                  <span tagx="source">Bioessays</span>
                  <span tagx="volume">26</span>
                  					:
                  					
                  <span tagx="fpage">739</span>
                  					–
                  					
                  <span tagx="lpage">751</span>
                  					,
                  					
                  <div tagx="pub-id">10.1002/bies.20062</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Urbach</span>
                        <span tagx="given-names">R</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Technau</span>
                        <span tagx="given-names">GM</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2003b</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Segment polarity and DV patterning gene expression reveals
                     						segmental organization of the
                     						
                     <i>Drosophila</i>
                     						brain
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Development</span>
                  <span tagx="volume">130</span>
                  					:
                  					
                  <span tagx="fpage">3607</span>
                  					–
                  					
                  <span tagx="lpage">3620</span>
                  					,
                  					
                  <div tagx="pub-id">10.1242/dev.00532</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Wang</span>
                        <span tagx="given-names">JW</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Wong</span>
                        <span tagx="given-names">AM</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Flores</span>
                        <span tagx="given-names">J</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Vosshall</span>
                        <span tagx="given-names">LB</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Axel</span>
                        <span tagx="given-names">R</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2003</span>
                  					)
                  					
                  <span tagx="article-title">Two-photon calcium imaging
                     						reveals an odor-evoked map of activity in the fly brain
                  </span>
                  					.
                  					
                  <span tagx="source">Cell</span>
                  <span tagx="volume">112</span>
                  					:
                  					
                  <span tagx="fpage">271</span>
                  					–
                  					
                  <span tagx="lpage">282</span>
                  					,
                  					
                  <div tagx="pub-id">10.1016/S0092-8674(03)00004-7</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Wong</span>
                        <span tagx="given-names">DC</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Lovick</span>
                        <span tagx="given-names">JK</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Ngo</span>
                        <span tagx="given-names">KT</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Borisuthirattana</span>
                        <span tagx="given-names">W</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Omoto</span>
                        <span tagx="given-names">JJ</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Hartenstein</span>
                        <span tagx="given-names">V</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2013</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Postembryonic lineages of the
                     						
                     <i>Drosophila</i>
                     						brain: II. Identification of lineage projection patterns based on
                     						MARCM clones
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Developmental Biology</span>
                  <span tagx="volume">384</span>
                  					:
                  					
                  <span tagx="fpage">258</span>
                  					–
                  					
                  <span tagx="lpage">289</span>
                  					,
                  					
                  <div tagx="pub-id">10.1016/j.ydbio.2013.07.009</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Yu</span>
                        <span tagx="given-names">HH</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Awasaki</span>
                        <span tagx="given-names">T</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Schroeder</span>
                        <span tagx="given-names">MD</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Long</span>
                        <span tagx="given-names">F</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Yang</span>
                        <span tagx="given-names">JS</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">He</span>
                        <span tagx="given-names">Y</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Ding</span>
                        <span tagx="given-names">P</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Kao</span>
                        <span tagx="given-names">JC</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Wu</span>
                        <span tagx="given-names">GY</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Peng</span>
                        <span tagx="given-names">H</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Myers</span>
                        <span tagx="given-names">G</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Lee</span>
                        <span tagx="given-names">T</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2013</span>
                  					)
                  					
                  <span tagx="article-title">
                     						Clonal development and organization of the adult
                     						
                     <i>Drosophila</i>
                     						central brain
                     					
                  </span>
                  					.
                  					
                  <span tagx="source">Current Biology</span>
                  <span tagx="volume">23</span>
                  					:
                  					
                  <span tagx="fpage">633</span>
                  					–
                  					
                  <span tagx="lpage">643</span>
                  					,
                  					
                  <div tagx="pub-id">10.1016/j.cub.2013.02.057</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Yu</span>
                        <span tagx="given-names">HH</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Chen</span>
                        <span tagx="given-names">CH</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Shi</span>
                        <span tagx="given-names">L</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Huang</span>
                        <span tagx="given-names">Y</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Lee</span>
                        <span tagx="given-names">T</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2009a</span>
                  					)
                  					
                  <span tagx="article-title">Twin-spot MARCM to reveal the
                     						developmental origin and identity of neurons
                  </span>
                  					.
                  					
                  <span tagx="source">Nature Neuroscience</span>
                  <span tagx="volume">12</span>
                  					:
                  					
                  <span tagx="fpage">947</span>
                  					–
                  					
                  <span tagx="lpage">953</span>
                  					,
                  					
                  <div tagx="pub-id">10.1038/nn.2345</div>
               </div>
            </li>
            <li tag="ref">
               <div tagx="citation">
                  <div tagx="person-group">
                     <span tagx="name">
                        <span tagx="surname">Yu</span>
                        <span tagx="given-names">YC</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Bultje</span>
                        <span tagx="given-names">RS</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Wang</span>
                        <span tagx="given-names">X</span>
                     </span>
                     <span tagx="name">
                        <span tagx="surname">Shi</span>
                        <span tagx="given-names">SH</span>
                     </span>
                  </div>
                  					(
                  					
                  <span tagx="year">2009b</span>
                  					)
                  					
                  <span tagx="article-title">Specific synapses develop
                     						preferentially among sister excitatory neurons in the neocortex
                  </span>
                  					.
                  					
                  <span tagx="source">Nature</span>
                  <span tagx="volume">458</span>
                  					:
                  					
                  <span tagx="fpage">501</span>
                  					–
                  					
                  <span tagx="lpage">504</span>
                  					,
                  					
                  <div tagx="pub-id">10.1038/nature07722</div>
               </div>
            </li>
         </ul>
      </div>
   </div>
   <div tagx="sub-article">
      <div tagx="front-stub">
         <div tagx="article-id">elife;3/0/e04407</div>
         <div tagx="article-id">elife;eLife.04407</div>
         <div tagx="article-id">eLife.04407.021</div>
         <div tagx="article-id">10.7554/eLife.04407.021</div>
         <div tagx="article-id">1</div>
         <div tagx="title-group">
            <span tagx="article-title">Decision letter</span>
         </div>
         <div tagx="contrib-group"></div>
         <div tagx="self-uri"></div>
      </div>
      <div id="abstract" tag="abstract">
         <div tagx="object-id">abstract-1</div>
         <div tagx="object-id">10.7554/eLife.04407.001</div>
         <p id="p-2">
            					Acquisition of distinct neuronal identities during development is
            					critical for the assembly of diverse functional neural circuits in
            					the brain. In both vertebrates and invertebrates, intrinsic
            					determinants are thought to act in neural progenitors to specify
            					their identity and the identity of their neuronal progeny. However,
            					the extent to which individual factors can contribute to this is
            					poorly understood. We investigate the role of
            					
            <i>orthodenticle</i>
            					in the specification of an identified neuroblast (neuronal
            					progenitor) lineage in the
            					
            <i>Drosophila</i>
            					brain. Loss of
            					
            <i>orthodenticle</i>
            					from this neuroblast affects molecular properties, neuroanatomical
            					features, and functional inputs of progeny neurons, such that an
            					entire central complex lineage transforms into a functional
            					olfactory projection neuron lineage. This ability to change
            					functional macrocircuitry of the brain through changes in gene
            					expression in a single neuroblast reveals a surprising capacity for
            					novel circuit formation in the brain and provides a paradigm for
            					large-scale evolutionary modification of circuitry.
            				
         </p>
         <p id="p-3">
            <b>DOI:</b>
            <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.001</div>
         </p>
      </div>
      <div id="abstract" tag="abstract">
         <div tagx="object-id">abstract-2</div>
         <div tagx="object-id">10.7554/eLife.04407.002</div>
         <div tagx="title">eLife digest</div>
         <p id="p-4">The cells in the brain—including the neurons that
            					transmit information—work together in groups called neural
            					circuits. These cells develop from precursor cells called
            					neuroblasts. Each neuroblast can produce many cells, and it is
            					likely that cells that develop from the same neuroblast work
            					together in the adult brain in the same neural circuit. How the
            					adult cells develop into their final form plays an important role
            					in creating a neural circuit, but this process is not fully
            					understood.
         </p>
         <p id="p-5">
            					In many animals, the complexity of their brain makes it difficult
            					to follow how each individual neuroblast develops. However, all of
            					the neuroblasts in the relatively simple brain of the fruit fly
            					
            <i>Drosophila</i>
            					have been identified. Furthermore, the genes responsible for
            					establishing the initial identity of each neuroblast in the
            					
            <i>Drosophila</i>
            					brain are known. These genes may also determine which adult neurons
            					develop from the neuroblast, and when each type of neuron is
            					produced. However, the extent to which a single gene can influence
            					the identity of neurons is unclear.
            				
         </p>
         <p id="p-6">
            					Sen et al. focused on two types of neuroblasts, each of which,
            					although found next to each other in the developing
            					
            <i>Drosophila</i>
            					brain, produces neurons for different neural circuits. One of the
            					neuroblasts generates the olfactory neurons responsible for
            					detecting smells; the other innervates the ‘central complex’ that
            					has a number of roles, including controlling the fly's movements. A
            					gene called
            					
            <i>orthodenticle</i>
            					is expressed by the central complex neuroblast, but not by the
            					olfactory neuroblast, and helps to separate the two neural circuits
            					into different regions of the fly brain.
            				
         </p>
         <p id="p-7">
            					Sen et al. found that deleting the
            					
            <i>orthodenticle</i>
            					gene from the central complex neuroblast causes it to develop into
            					olfactory neurons instead of central complex neurons. Tests showed
            					that the modified neurons are completely transformed; they not only
            					work like olfactory neurons, but they also have the same structure
            					and molecular properties. Sen et al. have therefore demonstrated
            					that it is possible to drastically alter the circuitry of the fruit
            					fly brain by changing how one gene is expressed in one neuroblast.
            					This suggests that new neural circuits can form relatively easily,
            					and so could help us to understand how different brain structures
            					and neural circuits evolved.
            				
         </p>
         <p id="p-8">
            <b>DOI:</b>
            <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.002</div>
         </p>
      </div>
      <div tagx="boxed-text">
         <p id="p-113">
            					eLife posts the editorial decision letter and author response on a
            					selection of the published articles (subject to the approval of the
            					authors). An edited version of the letter sent to the authors after
            					peer review is shown, indicating the substantive concerns or
            					comments; minor concerns are not usually shown. Reviewers have the
            					opportunity to discuss the decision before the letter is sent (see
            					
            <div tagx="ext-link">review
               						process
            </div>
            					). Similarly, the author response typically shows only responses to
            					the major concerns raised by the reviewers.
            				
         </p>
      </div>
      <p id="p-114">
         				Thank you for sending your work entitled “Rewiring the brain:
         				Genetic transformation of structural and functional circuitry in the
         				
         <i>Drosophila</i>
         				brain” for consideration at
         				
         <i>eLife</i>
         				. Your article has been favorably evaluated by Fiona Watt (Senior
         				editor), a Reviewing editor, and 2 reviewers.
         			
      </p>
      <p id="p-115">This paper provides a comprehensive analysis of a genetic
         				manipulation, consisting in removing or ectopically expressing the
         				gene orthodenticle (otd), which resulted in the fate transformation
         				of a neural lineage in the Drosophila brain. The authors show
         				convincingly that manipulating otd at the stage of the stem cell
         				(neuroblast) determines the fate of the lineage; loss of otd in a
         				lineage that normally innervates the central complex (LALv1) results
         				in the transformation of its neurons into a lineage that normally
         				develops at a close by position, associated with the antennal lobe).
         				The data are clearly presented and documented. The findings are of
         				great significance for the field. They underline the early global
         				function of transcription factors in fate specification. They also
         				shed light on the interplay between intrinsic (“hard wired”) factors
         				and plasticity; for example, otd-mutant axons adopting the antennal
         				projection fate are able to branch and properly innervate the
         				antennal lobe, even though they start out at a different location
         				(the location of the LALv1 lineage).
      </p>
      <p id="p-116">Substantial concerns:</p>
      <p id="p-117">
         				1) The interpretations depend on the ability to recognize the same
         				lineage after it has been profoundly transformed. It was difficult
         				for me to be assured of this because of how the paper was organized.
         				The study involves the use of a number of different driver lines
         				that are employed in various contexts throughout the paper. In most
         				cases we are not told why a given driver line was selected for a
         				particular experiment or, importantly, the normal expression pattern
         				for each line with respect to LALv1 and ALad1 and other lineages
         				with which they might be confused. For example, for the RNAi
         				experiments, why select insc-GAL4? What is the selectivity of the
         				line? Also, the Per-Gal4 line was used when examining the single
         				neuron otd-/- clones of LALv1 but GH146-Gal4 was used for the
         				neuroblast clones of the lineage. It is only when we get to
         				
         <span ref-type="fig" rid="fig6">Figure 6</span>
         				do we see that two drivers that express in wild-type LALv1 cells
         				(Per-Gal4 and OK371-Gal4) no longer express in these neurons after
         				they are transformed by otd removal. This information needs to be
         				brought up earlier in the paper.
         			
      </p>
      <p id="p-118">2) Comparison of 4B and F shows a single cluster of otd+
         				cells that coincide with LALv1 in “B”, but in “F” part of the
         				weakened otd+ cluster is now ascribed to two different clusters:
         				part to LALv1 [yellow outline] and the remainder to ALv1 [which
         				should not express otd?]. I am not convinced that we are seeing 2
         				distinct clusters rather than a split version of ALv1. It would be
         				good to have a more convincing example of the appearance of a
         				distinct GH146 cluster with the reduction in otd.
      </p>
      <p id="p-119">3) The ALv1 lineage is a potential complication because
         				it is a ventral lineage of antennal lobe projection neurons that is
         				situated next to LALv1 in the adult. These cells have an anatomy
         				that is similar to the cells of the transformed LALv1 and the GH146
         				driver expresses in these cells. The authors should have recovered
         				many ALv1 clones in their experiments. It is important to know how
         				they distinguished ALv1 clones from the transformed LALv1.
      </p>
      <p id="p-120">4) There are other pieces of information that would help
         				establish the correspondence of the LALv1 cells to the LALv1otd-
         				neurons: 1) The observation that the LEp tract is missing when the
         				lineage is transformed is a compelling piece of information. In
         				addition to the reconstruction [or instead of it], it would be good
         				to show an appropriate confocal section showing the loss of the
         				tract on the transformed side and its presence on the control side.
         				2) If they have them, it would be good to show images of the
         				transformed clones in the larval stage. The position of the
         				neuroblast and its cell cluster should be in its normal position but
         				its axon bundle would then be taking the transformed trajectory.
      </p>
   </div>
   <div tagx="sub-article">
      <div tagx="front-stub">
         <div tagx="article-id">elife;3/0/e04407</div>
         <div tagx="article-id">elife;eLife.04407</div>
         <div tagx="article-id">eLife.04407.022</div>
         <div tagx="article-id">10.7554/eLife.04407.022</div>
         <div tagx="article-id">2</div>
         <div tagx="title-group">
            <span tagx="article-title">Author response</span>
         </div>
         <div tagx="self-uri"></div>
         <div tagx="self-uri"></div>
      </div>
      <div id="abstract" tag="abstract">
         <div tagx="object-id">abstract-1</div>
         <div tagx="object-id">10.7554/eLife.04407.001</div>
         <p id="p-2">
            					Acquisition of distinct neuronal identities during development is
            					critical for the assembly of diverse functional neural circuits in
            					the brain. In both vertebrates and invertebrates, intrinsic
            					determinants are thought to act in neural progenitors to specify
            					their identity and the identity of their neuronal progeny. However,
            					the extent to which individual factors can contribute to this is
            					poorly understood. We investigate the role of
            					
            <i>orthodenticle</i>
            					in the specification of an identified neuroblast (neuronal
            					progenitor) lineage in the
            					
            <i>Drosophila</i>
            					brain. Loss of
            					
            <i>orthodenticle</i>
            					from this neuroblast affects molecular properties, neuroanatomical
            					features, and functional inputs of progeny neurons, such that an
            					entire central complex lineage transforms into a functional
            					olfactory projection neuron lineage. This ability to change
            					functional macrocircuitry of the brain through changes in gene
            					expression in a single neuroblast reveals a surprising capacity for
            					novel circuit formation in the brain and provides a paradigm for
            					large-scale evolutionary modification of circuitry.
            				
         </p>
         <p id="p-3">
            <b>DOI:</b>
            <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.001</div>
         </p>
      </div>
      <div id="abstract" tag="abstract">
         <div tagx="object-id">abstract-2</div>
         <div tagx="object-id">10.7554/eLife.04407.002</div>
         <div tagx="title">eLife digest</div>
         <p id="p-4">The cells in the brain—including the neurons that
            					transmit information—work together in groups called neural
            					circuits. These cells develop from precursor cells called
            					neuroblasts. Each neuroblast can produce many cells, and it is
            					likely that cells that develop from the same neuroblast work
            					together in the adult brain in the same neural circuit. How the
            					adult cells develop into their final form plays an important role
            					in creating a neural circuit, but this process is not fully
            					understood.
         </p>
         <p id="p-5">
            					In many animals, the complexity of their brain makes it difficult
            					to follow how each individual neuroblast develops. However, all of
            					the neuroblasts in the relatively simple brain of the fruit fly
            					
            <i>Drosophila</i>
            					have been identified. Furthermore, the genes responsible for
            					establishing the initial identity of each neuroblast in the
            					
            <i>Drosophila</i>
            					brain are known. These genes may also determine which adult neurons
            					develop from the neuroblast, and when each type of neuron is
            					produced. However, the extent to which a single gene can influence
            					the identity of neurons is unclear.
            				
         </p>
         <p id="p-6">
            					Sen et al. focused on two types of neuroblasts, each of which,
            					although found next to each other in the developing
            					
            <i>Drosophila</i>
            					brain, produces neurons for different neural circuits. One of the
            					neuroblasts generates the olfactory neurons responsible for
            					detecting smells; the other innervates the ‘central complex’ that
            					has a number of roles, including controlling the fly's movements. A
            					gene called
            					
            <i>orthodenticle</i>
            					is expressed by the central complex neuroblast, but not by the
            					olfactory neuroblast, and helps to separate the two neural circuits
            					into different regions of the fly brain.
            				
         </p>
         <p id="p-7">
            					Sen et al. found that deleting the
            					
            <i>orthodenticle</i>
            					gene from the central complex neuroblast causes it to develop into
            					olfactory neurons instead of central complex neurons. Tests showed
            					that the modified neurons are completely transformed; they not only
            					work like olfactory neurons, but they also have the same structure
            					and molecular properties. Sen et al. have therefore demonstrated
            					that it is possible to drastically alter the circuitry of the fruit
            					fly brain by changing how one gene is expressed in one neuroblast.
            					This suggests that new neural circuits can form relatively easily,
            					and so could help us to understand how different brain structures
            					and neural circuits evolved.
            				
         </p>
         <p id="p-8">
            <b>DOI:</b>
            <div tagx="ext-link">http://dx.doi.org/10.7554/eLife.04407.002</div>
         </p>
      </div>
      <p id="p-121">
         <i>1) The interpretations depend on the ability
            					to recognize the same lineage after it has been profoundly
            					transformed. It was difficult for me to be assured of this because
            					of how the paper was organized. The study involves the use of a
            					number of different driver lines that are employed in various
            					contexts throughout the paper. In most cases we are not told why a
            					given driver line was selected for a particular experiment or,
            					importantly, the normal expression pattern for each line with
            					respect to LALv1 and ALad1 and other lineages with which they might
            					be confused. For example, for the RNAi experiments, why select
            					insc-GAL4? What is the selectivity of the line? Also, the Per-Gal4
            					line was used when examining the single neuron otd-/- clones of
            					LALv1 but GH146-Gal4 was used for the neuroblast clones of the
            					lineage. It is only when we get to
         </i>
         <span ref-type="fig" rid="fig6">
            <i>Figure 6</i>
         </span>
         <i>do we see that two drivers that express in
            					wild-type LALv1 cells (Per-Gal4 and OK371-Gal4) no longer express
            					in these neurons after they are transformed by otd removal. This
            					information needs to be brought up earlier in the paper
         </i>
         				.
         			
      </p>
      <p id="p-122">
         				We thank the reviewers for bringing this to our attention. We have
         				now reorganized the manuscript so as to first mention the different
         				Gal4 drivers used (see paragraph 4 under the sub heading,
         				‘Development, morphogenesis and differential expression of Otd in
         				two identified central brain neuroblast lineages, LALv1 and ALad1’).
         				We have also taken care to mention the Gal4 drivers used for each of
         				the experiments when we describe them. For example, see second
         				sentence in paragraph 2 and third sentence in paragraph 3 under the
         				sub heading,
         				
         <i>‘</i>
         				Loss of Otd from the LALv1 neuroblast results in lineage identity
         				transformation.
         				
         <i>’</i>
      </p>
      <p id="p-123">
         				The use of
         				
         <i>Insc-Gal4</i>
         				was necessary because MARCM experiments suggested that Otd is
         				required in the neuroblast to maintain the central complex identity
         				of the LALv1 lineage. None of the other drivers used in the study
         				were able to drive RNAi expression as early. We have now mentioned
         				this in the text (see second sentence in paragraph 4 under the sub
         				heading, ‘Determining the lineage identity of the transformed
         				neurons’).
         			
      </p>
      <p id="p-124">
         <i>2) Comparison of 4B and F shows a single
            					cluster of otd+ cells that coincide with LALv1 in “B”, but in “F”
            					part of the weakened otd+ cluster is now ascribed to two different
            					clusters: part to LALv1 [yellow outline] and the remainder to ALv1
            					[which should not express otd?]. I am not convinced that we are
            					seeing 2 distinct clusters rather than a split version of ALv1. It
            					would be good to have a more convincing example of the appearance
            					of a distinct GH146 cluster with the reduction in otd
         </i>
         				.
         			
      </p>
      <p id="p-125">
         				In this experiment we used the
         				
         <i>Insc-Gal4</i>
         				to drive and Otd RNAi in all neuroblasts and used
         				
         <i>GH146-QF&gt;QUAS-Tomato::HA</i>
         				to visualize the neurons. However, as RNAi is not always completely
         				penetrant, we often did not observe a complete loss of Otd in the
         				LALv1 lineage. The brain shown in the previous
         				
         <span ref-type="fig" rid="fig4">Figure 4</span>
         				(current
         				
         <span ref-type="fig" rid="fig5">Figure 5</span>
         				) shows the strongest reduction in Otd in the LALv1 lineage. We have
         				now additionally shown an
         				
         <i>otd</i>
         <sup>
            <i>-/-</i>
         </sup>
         				MARCM clonal brain in which all four neuroblasts lineages have been
         				recovered. Please see
         				
         <span ref-type="fig" rid="fig5">Figure 5 A-C</span>
         				.
         			
      </p>
      <p id="p-126">Regarding splitting of the cell clusters, please see
         				response 3 below.
      </p>
      <p id="p-127">
         <i>3) The ALv1 lineage is a potential
            					complication because it is a ventral lineage of antennal lobe
            					projection neurons that is situated next to LALv1 in the adult.
            					These cells have an anatomy that is similar to the cells of the
            					transformed LALv1 and the GH146 driver expresses in these cells.
            					The authors should have recovered many ALv1 clones in their
            					experiments. It is important to know how they distinguished ALv1
            					clones from the transformed LALv1
         </i>
         				.
         			
      </p>
      <p id="p-128">
         				We thank the reviewers for pointing out this potential complication.
         				The GH146 driver labels only about 6-10 cells of the ALv1 lineage.
         				These neurons are indeed antennal lobe projection interneurons,
         				however, their neuroanatomy is very different from the
         				
         <i>otd</i>
         <sup>
            <i>-/-</i>
         </sup>
         				LALv1 lineage. The entry point of the axons of these two lineages
         				into the antennal lobe is diametrically opposite to each other;
         				while the
         				
         <i>otd</i>
         <sup>
            <i>-/-</i>
         </sup>
         				LALv1 lineage enters the lobe medially, ALv1 (as well as the
         				
         <i>otd</i>
         <sup>
            <i>-/-</i>
         </sup>
         				ALv1) enters it laterally (see asterisks in
         				
         <span ref-type="fig" rid="fig13">Author response image 1A,B</span>
         				). Most strikingly, their axons exit the lobe towards the
         				protocerebrum using different tracts; while the
         				
         <i>otd</i>
         <sup>
            <i>-/-</i>
         </sup>
         				LALv1 lineage uses the medial antennal lobe tract, the ALv1 (as well
         				as the
         				
         <i>otd</i>
         <sup>
            <i>-/-</i>
         </sup>
         				ALv1) use the mediolateral antennal lobe tract (see arrow in Author
         				response 1A,B). Both these tracts are visible in
         				
         <span ref-type="fig" rid="fig5">Figure 5 A-C</span>
         				. Finally, while the
         				
         <i>otd</i>
         <sup>
            <i>-/-</i>
         </sup>
         				LALv1 lineage first innervates the calyx of the mushroom body and
         				then the lateral horn, the GH146-labelled neurons of ALv1 (as well
         				as the
         				
         <i>otd</i>
         <sup>
            <i>-/-</i>
         </sup>
         				ALv1) largely innervate only the lateral horn (see
         				
         <span ref-type="fig" rid="fig13">Author response image 1A,B</span>
         				). We have brought out these points in the main text as well. Please
         				see paragraph 3 under the subheading,
         				
         <i>‘</i>
         				Determining the lineage identity of the transformed neurons’ (also
         				see
         				
         <span ref-type="fig" rid="fig13">Author response image 1</span>
         				).
         			
      </p>
      <p id="p-129">
         				The ALv1 lineage does not express Otd and, as expected, we do not
         				see any gross defects in any of the
         				
         <i>otd</i>
         <sup>
            <i>-/-</i>
         </sup>
         				ALv1 clones we have recovered (see
         				
         <span ref-type="fig" rid="fig13">Author response image 1</span>
         				). Moreover, although we have recovered the
         				
         <i>otd</i>
         <sup>
            <i>-/-</i>
         </sup>
         				LALv1 lineage along with other neuroblast clones of the antennal
         				lobe including the ALv1 lineage, we frequently recover it with no
         				other clones in the brain. In these clones, we observe no splitting
         				of the cell cluster and the mutant lineage never uses the
         				mediolateral antennal lobe tract.
         			
      </p>
      <p id="p-130">
         				Given these data we conclude that the
         				
         <i>otd</i>
         <sup>
            <i>-/-</i>
         </sup>
         				LALv1 lineage and the
         				
         <i>otd</i>
         <sup>
            <i>-/-</i>
         </sup>
         				ALv1 lineage have two distinct clonal origins (are two distinct
         				neuroblast lineages) and are not a result of splitting of the
         				
         <i>otd</i>
         <sup>
            <i>-/-</i>
         </sup>
         				LALv1 lineage.
         				
         <div tagx="fig">
            <div tagx="object-id">elife;3/0/e04407/FIG13</div>
            <div tagx="object-id">F15</div>
            <div tagx="object-id">fig13</div>
            <div tagx="label">Author response image 1.</div>
            <caption id="caption-19">
               <p id="p-131">
                  							WT (A) and
                  							
                  <i>otd</i>
                  <sup>
                     <i>-/-</i>
                  </sup>
                  							(B) ALv1 lineage. Note the position of the cell body (yellow
                  							dotted lines), the lateral point of entry of axon tract into the
                  							antennal lobe (indicated by the asterisk), the use of the
                  							mediolateral antennal lobe tract (indicated by the arrow) and the
                  							predominant innervation of the lateral horn of both the WT (A)
                  							and the
                  							
                  <i>otd</i>
                  <sup>
                     <i>-/-</i>
                  </sup>
                  							ALv1 lineage (B). Midline is represented by the yellow line.
                  							Genotype in A:
                  							
                  <i>FRT19A/FRT19A,Tub-Gal80,hsFLP;
                     								GH146-Gal4,UAS-mCD8::GFP/+.
                  </i>
                  							Genotype in B
                  							
                  <i>: FRT19A, oc</i>
                  <sup>
                     <i>2</i>
                  </sup>
                  <i>/FRT19A,Tub-Gal80,hsFLP;
                     								GH146-Gal4,UAS-mCD8::GFP/+.
                  </i>
               </p>
            </caption>
            <div tagx="graphic"></div>
         </div>
      </p>
      <p id="p-132">
         <i>4) There are other pieces of information that
            					would help establish the correspondence of the LALv1 cells to the
            					LALv1otd- neurons: 1) The observation that the LEp tract is missing
            					when the lineage is transformed is a compelling piece of
            					information. In addition to the reconstruction [or instead of it],
            					it would be good to show an appropriate confocal section showing
            					the loss of the tract on the transformed side and its presence on
            					the control side. 2) If they have them, it would be good to show
            					images of the transformed clones in the larval stage. The position
            					of the neuroblast and its cell cluster should be in its normal
            					position but its axon bundle would then be taking the transformed
            					trajectory
         </i>
         				.
         			
      </p>
      <p id="p-133">
         				We thank the reviewers for pointing this out. We have reorganized
         				the text and added an entire new results section under the sub
         				heading, ‘Determining the lineage identity of the transformed
         				neurons’ in which we systematically list out three different ways by
         				which it is possible to infer correspondence between the LALv1 and
         				the transformed LALv1 neurons. 1) The appearance of an extra
         				antennal lobe lineage (see
         				
         <span ref-type="fig" rid="fig5">Figure 5</span>
         				). 2) The loss of the LEp tract (we have shown the confocal sections
         				instead of the 3D reconstruction; see
         				
         <span ref-type="fig" rid="fig6">Figure 6</span>
         				). 3) The use of a molecular marker, Acj6 that allows the
         				unambiguous identification of the LALv1 lineage in both the WT and
         				mutant conditions (see
         				
         <span ref-type="fig" rid="fig7">Figure 7</span>
         				).
         			
      </p>
      <p id="p-134">
         				We have also attempted to trace the
         				
         <i>otd</i>
         <sup>
            <i>-/-</i>
         </sup>
         				LALv1 lineage into the larval brain. In these experiments we
         				recovered
         				
         <i>GH146-Gal4</i>
         				labelled
         				
         <i>otd</i>
         <sup>
            <i>-/-</i>
         </sup>
         				LALv1 clones, however these clones were largely indistinguishable
         				from the ALad1 lineage (except for the loss of Otd immunolabelling
         				corresponding to the larval LALv1 lineage). This is because in the
         				larval brain, the cell bodies of LALv1 and ALad1 lie closely apposed
         				to each other and when the LALv1 lineage is transformed, they also
         				share similar trajectories. However, in a rare clone, the
         				
         <i>otd</i>
         <sup>
            <i>-/-</i>
         </sup>
         				LALv1 lineage retained a few projections into the loVM tract,
         				allowing us to unambiguously identify it as the LALv1 lineage. In
         				this brain preparation we observed that the
         				
         <i>otd</i>
         <sup>
            <i>-/-</i>
         </sup>
         				LALv1 lineage sent a majority of its projections via the medial
         				antennal lobe tract (see
         				
         <span ref-type="fig" rid="fig14">Author response image 2</span>
         				), confirming that the
         				
         <i>otd</i>
         <sup>
            <i>-/-</i>
         </sup>
         				LALv1 lineage indeed transforms to an ALad1 neuroblast lineage like
         				fate.
         				
         <div tagx="fig">
            <div tagx="object-id">elife;3/0/e04407/FIG14</div>
            <div tagx="object-id">F16</div>
            <div tagx="object-id">fig14</div>
            <div tagx="label">Author response image 2.</div>
            <caption id="caption-20">
               <p id="p-135">
                  							Note the larval
                  							
                  <i>otd</i>
                  <sup>
                     <i>-/-</i>
                  </sup>
                  							LALv1 lineage on the right hemisphere, which shows a loss of Otd
                  							immunolabelling in B. A few neurites of this lineage enter the
                  							loVM tract (cyan arrow in A,C), while most take the medial
                  							antennal lobe tract towards the protocerebrum (magenta arrow in
                  							A,C). Midline is represented by the yellow line. Genotype:
                  							
                  <i>FRT19A, otd</i>
                  <sup>
                     <i>YH13</i>
                  </sup>
                  <i>/FRT19A,Tub-Gal80,hsFLP;
                     								GH146-Gal4,UAS-mCD8::GFP/+./+.
                  </i>
               </p>
            </caption>
            <div tagx="graphic"></div>
         </div>
      </p>
   </div>
</div>