<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <div xmlns="" tagx="article">
      <head>
         <style type="text/css"> table, tr, td {border : thin;}</style>
         <span class="citation_publisher">Public Library of Science</span>
         <h2 class="citation-title">Molecular Evolution of Zika Virus during Its Emergence in the 20th Century</h2>
         <span class="doi">10.1371/journal.pntd.0002636</span>
         <div class="contrib-group">contrib: 
            	  CONT
            	  
            <span class="citation_author">Oumar Faye</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Caio C. M. Freire</span>
            <span class="citation_author_institution_ref">2</span>
            <span class="citation_author">Atila Iamarino</span>
            <span class="citation_author_institution_ref">2</span>
            <span class="citation_author">Ousmane Faye</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Juliana Velasco C. de Oliveira</span>
            <span class="citation_author_institution_ref">2</span>
            <span class="citation_author">Mawlouth Diallo</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Paolo M. A. Zanotto</span>
            <span class="citation_author_institution_ref">2</span>
            <span class="citation_author">Amadou Alpha Sall</span>
            <span class="citation_author_institution_ref">1</span>
            	  CONT
            	  
            <div tagxxx="contrib">
               <span class="name'">
                  <span tagx="surname">Bird</span>
                  <span tagx="given-names">Brian</span>,
               </span>
               <div tagxxx="role">Editor</div>
               <span ref-type="aff" rid="edit1"></span>
            </div>
         </div>
         <span class="email">[asall@pasteur.sn]</span>
         <span class="conflict">[The authors have declared that no competing interests exist.]</span>
         <span class="contribs">[Conceived and designed the experiments: AAS OumF. Performed the experiments: OumF
            JVCdO. Analyzed the data: PMAZ CCMF AAS OumF AI. Contributed reagents/materials/analysis
            tools: AAS OumF PMAZ CCMF OusF AI JVCdO. Wrote the paper: OumF CCMF AAS PMAZ OusF
            MD AI JVCdO.]
         </span>
         <span class="citation_pubdate">[2014-1-9]</span>
         <span class="citation_copyright">[Copyright 2014: Faye et al]</span>
         <span class="citation_licence">[Licence This is an open-access article distributed under the terms of the Creative
            Commons Attribution License, which permits unrestricted use, distribution, and reproduction
            in any medium, provided the original author and source are credited.]
         </span>
         <span class="citation_funding">[This study was made possible by the Institute Pasteur of Dakar in Senegal, FAPESP
            (Fundação de Amparo a Pesquisa do Estado de Sao Paulo, Brazil) projects #00/04205-6
            (VGDN program), #08/17013-6 and #10/19341-4. CCMF has a CAPES studentship, AI (project
            #12/04818-5) and JVCdO received FAPESP scholarships and PMAZ holds a CNPq-PQ grant.
            Funding was also provided by NIH grant R01-AI069145. The funders had no role in study
            design, data collection and analysis, decision to publish, or preparation of the manuscript.]
         </span>
         <span class="citation_pagecount">[]</span>
         <span class="citation_data">[]</span>
      </head>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Zika virus (ZIKV) is a mosquito-borne flavivirus first isolated in Uganda in 1947.
            Although entomological and virologic surveillance have reported ZIKV enzootic activity
            in diverse countries of Africa and Asia, few human cases were reported until 2007,
            when a Zika fever epidemic took place in Micronesia. In the context of West Africa,
            the WHO Collaborating Centre for Arboviruses and Hemorrhagic Fever at Institut Pasteur
            of Dakar (
            <div tagxxx="ext-link">http://www.pasteur.fr/recherche/banques/CRORA/</div>) reports the periodic circulation of ZIKV since 1968. Despite several reports on
            ZIKV, the genetic relationships among viral strains from West Africa remain poorly
            understood. To evaluate the viral spread and its molecular epidemiology, we investigated
            37 ZIKV isolates collected from 1968 to 2002 in six localities in Senegal and Côte
            d'Ivoire. In addition, we included strains from six other countries. Our results suggested
            that these two countries in West Africa experienced at least two independent introductions
            of ZIKV during the 20
            <sup>th</sup> century, and that apparently these viral lineages were not restricted by mosquito
            vector species. Moreover, we present evidence that ZIKV has possibly undergone recombination
            in nature and that a loss of the N154 glycosylation site in the envelope protein was
            a possible adaptive response to the 
            <i>Aedes dalzieli</i> vector.
         </p>
      </div>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <div tagxxx="title">Author Summary</div>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Zika fever is a mosquito-borne illness caused by a flavivirus. Human infections with
            Zika virus (ZIKV) could cause fever, malaise and cutaneous rash. Despite several ZIKV
            reports since 1947 when it was first isolated at Zika forest in Uganda, molecular
            evolution of ZIKV as an emerging agent remains poorly understood. Moreover, despite
            several ZIKV reports from Africa and Asia, few human cases were notified until 2007
            when an epidemic took place in Micronesia. In West Africa, surveillance programs have
            reported periodic circulation of the virus since 1968. To help fill the gap in understanding
            ZIKV evolution, 43 ZIKV samples were analyzed. We focused on: (
            <i>i</i>) adaptive genetic changes including protein glycosylation patterns, (
            <i>ii</i>) phylogenetic relationship among isolates and their spatiotemporal patterns of spread
            across Africa and Asia and, (
            <i>iii</i>) dispersion among vertebrate reservoirs and invertebrate vector species. Our results
            indicated that ZIKV may have experienced recombination in nature and that, after it
            emerged from Uganda in the early of the 20
            <sup>th</sup> century, it moved to West Africa and Asia in the first half of the century, without
            any clear preference for host and vector species.
         </p>
      </div>
      <div id="s1">
         <h2>Introduction</h2>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Zika virus (ZIKV) is a mosquito-borne flavivirus, a member of the Spondweni serocomplex,
            whose natural transmission cycle involves mainly vectors from the 
            <i>Aedes</i> genus (
            <i>A. furcifer</i>, 
            <i>A. taylori</i>, 
            <i>A. luteocephalus</i> and 
            <i>A. africanus</i>) and monkeys 
            <span ref-type="bibr" rid="pntd.0002636-Hayes1">
               <sup>
                  <a href="#pntd.0002636-Hayes1">[1]</a>
               </sup>
            </span>, while humans are occasional hosts. Clinical pictures range from asymptomatic cases
            to an influenza-like syndrome associated to fever, headache, malaise and cutaneous
            rash 
            <span ref-type="bibr" rid="pntd.0002636-Simpson1">
               <sup>
                  <a href="#pntd.0002636-Simpson1">[2]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pntd.0002636-Bearcroft1">
               <sup>
                  <a href="#pntd.0002636-Bearcroft1">[3]</a>
               </sup>
            </span>. Likewise, direct contact is also considered a potential route of transmission among
            humans, probably during sexual intercourse 
            <span ref-type="bibr" rid="pntd.0002636-Foy1">
               <sup>
                  <a href="#pntd.0002636-Foy1">[4]</a>
               </sup>
            </span>. The first isolation of ZIKV was in 1947 from the blood of a sentinel Rhesus monkey
            No. 766, stationed in the Zika forest, near the Lake Victoria in Uganda, and in 1948
            ZIKV was also isolated in the same forest from a pool of 
            <i>A. africanus</i> mosquitoes 
            <span ref-type="bibr" rid="pntd.0002636-Dick1">
               <sup>
                  <a href="#pntd.0002636-Dick1">[5]</a>
               </sup>
            </span>. Thereafter, serological and entomological data indicated ZIKV infections in the
            African continent in Nigeria in 1971 and 1975 
            <span ref-type="bibr" rid="pntd.0002636-Fagbami1">
               <sup>
                  <a href="#pntd.0002636-Fagbami1">[6]</a>
               </sup>
            </span>, Sierra Leone in 1972 
            <span ref-type="bibr" rid="pntd.0002636-Robin1">
               <sup>
                  <a href="#pntd.0002636-Robin1">[7]</a>
               </sup>
            </span>, Gabon in 1975 
            <span ref-type="bibr" rid="pntd.0002636-Jan1">
               <sup>
                  <a href="#pntd.0002636-Jan1">[8]</a>
               </sup>
            </span>, Uganda in 1969 and 1970 
            <span ref-type="bibr" rid="pntd.0002636-McCrae1">
               <sup>
                  <a href="#pntd.0002636-McCrae1">[9]</a>
               </sup>
            </span>, Central African Republic in 1979 
            <span ref-type="bibr" rid="pntd.0002636-Saluzzo1">
               <sup>
                  <a href="#pntd.0002636-Saluzzo1">[10]</a>
               </sup>
            </span>, Senegal from 1988 to 1991 
            <span ref-type="bibr" rid="pntd.0002636-Monlun1">
               <sup>
                  <a href="#pntd.0002636-Monlun1">[11]</a>
               </sup>
            </span> and Côte d'Ivoire in 1999 
            <span ref-type="bibr" rid="pntd.0002636-AkouaKoffi1">
               <sup>
                  <a href="#pntd.0002636-AkouaKoffi1">[12]</a>
               </sup>
            </span>. Recently, ZIKV was detected in Senegal in 2011 and 2012 (unpublished data). In addition,
            ZIKV infections in Asia were reported in Pakistan 
            <span ref-type="bibr" rid="pntd.0002636-Darwish1">
               <sup>
                  <a href="#pntd.0002636-Darwish1">[13]</a>
               </sup>
            </span>, Malaysia 
            <span ref-type="bibr" rid="pntd.0002636-Marchette1">
               <sup>
                  <a href="#pntd.0002636-Marchette1">[14]</a>
               </sup>
            </span>, Indonesia in 1977 and 1978 
            <span ref-type="bibr" rid="pntd.0002636-Olson1">
               <sup>
                  <a href="#pntd.0002636-Olson1">[15]</a>
               </sup>
            </span>, Micronesia in 2007 
            <span ref-type="bibr" rid="pntd.0002636-Duffy1">
               <sup>
                  <a href="#pntd.0002636-Duffy1">[16]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pntd.0002636-Lanciotti1">
               <sup>
                  <a href="#pntd.0002636-Lanciotti1">[17]</a>
               </sup>
            </span> and Cambodia in 2010 
            <span ref-type="bibr" rid="pntd.0002636-Heang1">
               <sup>
                  <a href="#pntd.0002636-Heang1">[18]</a>
               </sup>
            </span>. Although ZIKV was repeatedly isolated, only 14 human cases were reported before
            April 2007, when a Zika fever (ZF) epidemic occurred in Yap island in Micronesia,
            where 49 confirmed cases and 73% of the residents older than 3 years provided serologic
            evidence for recent ZIKV infection 
            <span ref-type="bibr" rid="pntd.0002636-Duffy1">
               <sup>
                  <a href="#pntd.0002636-Duffy1">[16]</a>
               </sup>
            </span>. This outbreak showcased the potential of ZF as an emerging disease, which could
            be misdiagnosed as dengue fever, as happened during the beginning of the Micronesian
            outbreak 
            <span ref-type="bibr" rid="pntd.0002636-Duffy1">
               <sup>
                  <a href="#pntd.0002636-Duffy1">[16]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pntd.0002636-Lanciotti1">
               <sup>
                  <a href="#pntd.0002636-Lanciotti1">[17]</a>
               </sup>
            </span>.
         </p>
         <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The ZIKV genome consists of a single-stranded positive sense RNA molecule with 10794
            kb of length with 2 flanking non-coding regions (5′ and 3′ NCR) and a single long
            open reading frame encoding a polyprotein: 5′-C-prM-E-NS1-NS2A-NS2B-NS3-NS4A-NS4B-NS5-3′,
            that is cleaved into capsid (C), precursor of membrane (prM), envelope (E) and seven
            non-structural proteins (NS) 
            <span ref-type="bibr" rid="pntd.0002636-Chambers1">
               <sup>
                  <a href="#pntd.0002636-Chambers1">[19]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pntd.0002636-Kuno1">
               <sup>
                  <a href="#pntd.0002636-Kuno1">[20]</a>
               </sup>
            </span>. The E protein (≈53 kDa) is the major virion surface protein. E is involved in various
            aspects of the viral cycle, mediating binding and membrane fusion 
            <span ref-type="bibr" rid="pntd.0002636-Lindenbach1">
               <sup>
                  <a href="#pntd.0002636-Lindenbach1">[21]</a>
               </sup>
            </span>. The NS5 protein (≈103 kDa) is the largest viral protein whose C-terminal portion
            has RNA-dependent RNA polymerase (RdRP) activity and the N-terminus is involved in
            RNA capping by virtue of its processing due to methyl transferase activity 
            <span ref-type="bibr" rid="pntd.0002636-Lindenbach1">
               <sup>
                  <a href="#pntd.0002636-Lindenbach1">[21]</a>
               </sup>
            </span>. The 3′NCR of the ZIKV genome contains about 428 nucleotides, including 27 folding
            patterns 
            <span ref-type="bibr" rid="pntd.0002636-Kuno1">
               <sup>
                  <a href="#pntd.0002636-Kuno1">[20]</a>
               </sup>
            </span> that may be involved in the recognition by cellular or viral factors, translation,
            genome stabilization, RNA packaging, or cyclization 
            <span ref-type="bibr" rid="pntd.0002636-Lindenbach1">
               <sup>
                  <a href="#pntd.0002636-Lindenbach1">[21]</a>
               </sup>
            </span>. Although diverse studies have contributed greatly to our understanding of the evolutionary
            biology of flaviviruses in general 
            <span ref-type="bibr" rid="pntd.0002636-Kuno2">
               <sup>
                  <a href="#pntd.0002636-Kuno2">[22]</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pntd.0002636-Zanotto1">
               <sup>
                  <a href="#pntd.0002636-Zanotto1">[25]</a>
               </sup>
            </span>, few studies have addressed ZIKV evolutionary biology 
            <span ref-type="bibr" rid="pntd.0002636-Lanciotti1">
               <sup>
                  <a href="#pntd.0002636-Lanciotti1">[17]</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pntd.0002636-Haddow1">
               <sup>
                  <a href="#pntd.0002636-Haddow1">[26]</a>
               </sup>
            </span>. Those studies report three main ZIKV lineages, one from Asia and two from Africa.
            Aiming to fill this gap and gain better insights ZIKV molecular evolution in the 20
            <sup>th</sup> century, we investigated 43 ZIKV strains, sampled from 1947 to 2007 in Africa and
            Asia, to describe phylogenetic relationships, selective influences, recombination
            events, phylodynamics, phylogeography, host correlations with viral lineages and glycosylation
            patterns.
         </p>
      </div>
      <div id="s2">
         <h2>Methods</h2>
         <div id="s2a">
            <h3>Ethical statements</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Samples used in this study are part of the Institute Pasteur in Dakar collection (WHO
               Collaborating Centre for Arboviruses and/or Hemorrhagic Fever Reference and Research).
               Monkey and human strains (AnD 30332 and HD 78788) were obtained respectively in 1979
               and 1991 in Senegal during routine surveillance. None of the data was directly derived
               from human or animal samples but rather from cell culture supernatant. Therefore all
               the samples were anonymous and only reference numbers were used during the analysis
               that originated this study.
            </p>
         </div>
         <div id="s2b">
            <h3>Virus isolates</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">ZIKV strains were provided by CRORA at the Institute Pasteur of Dakar. The strains
               were obtained from mosquitoes, humans and other mammals isolated in Burkina Faso,
               Central African Republic, Côte d'Ivoire and Senegal in West Africa (
               <span ref-type="supplementary-material" rid="pntd.0002636.s005">Table S1</span>). Viral stocks were prepared by inoculating viral strains into 
               <i>Aedes pseudoscutellaris</i> clone 61 monolayer in Leibovitz 15 (L-15) growth medium (GibcoBRL, Grand Island,
               NY, USA) supplemented with 5% fetal bovine serum (FBS) (GibcoBRL, Grand Island, NY,
               USA), 10% Tryptose Phosphate and antibiotics (Sigma, Gmbh, Germany). Viral infection
               was confirmed after seven days of propagation by an indirect immunofluorescence assay
               (IFA) using specific hyper-immune mouse ascitic fluid, as described previously 
               <span ref-type="bibr" rid="pntd.0002636-Digoutte1">
                  <sup>
                     <a href="#pntd.0002636-Digoutte1">[27]</a>
                  </sup>
               </span>. Cultures supernatants were collected for virus RNA isolation.
            </p>
         </div>
         <div id="s2c">
            <h3>RNA extraction</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">RNA was extracted from ZIKV stocks using the QIAamp RNA Viral Kit (Qiagen, Hilden,
               Germany) according to the manufacturer's recommendations. RNA was eluted in 50 µl
               of AVE buffer and stored at −80°C until use.
            </p>
         </div>
         <div id="s2d">
            <h3>RT-PCR amplification</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">For cDNA synthesis, 10 µl of viral RNA was mixed with 1 µl of each of a reverse primer
               (2 pmol), 1 µl of deoxynucleotide triphospahte (dNTP) (10 mM each dNTP and the mixture
               was heated at 65°C for 5 min. Reverse transcription was performed in 20 µl mixture
               containing mixed of 2.5 U RNasin (Promega, Madison, USA) 5 U of Superscript III reverse
               transcriptase (Invitrogen, Carlsbad, USA) and incubated at 55°C for 50 min, followed
               by 70°C for 15 min. PCR products were generated independently using the primers Unifor/Unirev,
               FD3/FU1 and VD8/EMFI to amplify partial E, NS5 and NS5/3′NC region respectively 
               <span ref-type="bibr" rid="pntd.0002636-Faye1">
                  <sup>
                     <a href="#pntd.0002636-Faye1">[28]</a>
                  </sup>
               </span>. Five microliters of cDNA were mixed with 10× buffer, 5 µl of each primer, 5 µl of
               dNTPs 10 mM, 3 µl of MgCl
               <div tagxxx="sub">2</div>, and 0.5 µl of Taq polymerase (Promega, Madison, USA).
            </p>
         </div>
         <div id="s2e">
            <h3>Nucleotide sequencing</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">PCR products of the expected size were purified from agarose gels with the QiaQuick
               Gel Extraction Kit (Qiagen, Hilden, Germany) as specified by the manufacturer. Both
               strands of each PCR product were sequenced directly with the ABI Prism BigDye Terminator
               Cycle Sequencing Ready Reaction Kit V3.1 on an Applied Biosystems 3100 DNA Analyzer
               (Applied Bisoystem, Foster City, CA, USA) at the Laboratory of Molecular Evolution
               and Bioinformatics, Biomedical Sciences Institute, University of Sao Paulo, Brazil.
               We deposited thirty two 753 bp-long sequences from the E gene (Accession numbers:
               KF383015-KF383046), thirty one of NS5 (708 bp) (Accession numbers: KF38304-KF383114),
               thirty seven of 3′NCR (537 bp) (Accession numbers: KF383047-KF383083) and six genomes
               (10274 bp) (Accession numbers: KF383115–KF383120) in GenBank (
               <div tagxxx="ext-link">www.ncbi.nlm.nih.gov/genbank/</div>) from thirty eight viral strains (
               <span ref-type="supplementary-material" rid="pntd.0002636.s005">Table S1</span>). Additional sequences representing strains from Kedougou in Senegal, Nigeria, Malaysia,
               the Ugandan prototype MR766, the strain related to Micronesian outbreak in 2007 and
               the Spondweni virus were obtained from GenBank, with the following accession numbers,
               respectively: HQ234501, HQ234500, HQ234499, NC_012532, EU545988 and DQ859064.1 (
               <span ref-type="supplementary-material" rid="pntd.0002636.s005">Table S1</span>).
            </p>
         </div>
         <div id="s2f">
            <h3>Recombination detection</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Prior to the analyses, all sequences were aligned with MUSCLE v3.7 
               <span ref-type="bibr" rid="pntd.0002636-Edgar1">
                  <sup>
                     <a href="#pntd.0002636-Edgar1">[29]</a>
                  </sup>
               </span> and manually edited with SeaView v4.3.3 
               <span ref-type="bibr" rid="pntd.0002636-Gouy1">
                  <sup>
                     <a href="#pntd.0002636-Gouy1">[30]</a>
                  </sup>
               </span>. To prevent potential biases during phylogenetic inference due to recombination,
               we first analyzed the sequences of available ZIKV genomes with RDP v4.4.8 program
               
               <span ref-type="bibr" rid="pntd.0002636-Martin1">
                  <sup>
                     <a href="#pntd.0002636-Martin1">[31]</a>
                  </sup>
               </span> that incorporates RDP 
               <span ref-type="bibr" rid="pntd.0002636-Martin2">
                  <sup>
                     <a href="#pntd.0002636-Martin2">[32]</a>
                  </sup>
               </span>, GENECONV 
               <span ref-type="bibr" rid="pntd.0002636-Padidam1">
                  <sup>
                     <a href="#pntd.0002636-Padidam1">[33]</a>
                  </sup>
               </span>, Chimaera 
               <span ref-type="bibr" rid="pntd.0002636-Posada1">
                  <sup>
                     <a href="#pntd.0002636-Posada1">[34]</a>
                  </sup>
               </span>, MaxChi 
               <span ref-type="bibr" rid="pntd.0002636-Smith1">
                  <sup>
                     <a href="#pntd.0002636-Smith1">[35]</a>
                  </sup>
               </span>, Bootscan 
               <span ref-type="bibr" rid="pntd.0002636-Martin3">
                  <sup>
                     <a href="#pntd.0002636-Martin3">[36]</a>
                  </sup>
               </span>, SiScan 
               <span ref-type="bibr" rid="pntd.0002636-Gibbs1">
                  <sup>
                     <a href="#pntd.0002636-Gibbs1">[37]</a>
                  </sup>
               </span> and 3Seq 
               <span ref-type="bibr" rid="pntd.0002636-Boni1">
                  <sup>
                     <a href="#pntd.0002636-Boni1">[38]</a>
                  </sup>
               </span> methods to uncover evidence for recombination events. Only events with 
               <i>p</i>-values≤0.01 that were confirmed by four or more methods were considered, using the
               Bonferroni correction to prevent false positive results 
               <span ref-type="bibr" rid="pntd.0002636-Bland1">
                  <sup>
                     <a href="#pntd.0002636-Bland1">[39]</a>
                  </sup>
               </span>, as implemented in the RDP program 
               <span ref-type="bibr" rid="pntd.0002636-Martin1">
                  <sup>
                     <a href="#pntd.0002636-Martin1">[31]</a>
                  </sup>
               </span>. In addition, the occurrence of recombination in genomes was also investigated with
               the Rec-HMM program that estimates breakpoints based on the Phylo-HMM approach, which
               models tree topology changes over the columns of a multiple alignment 
               <span ref-type="bibr" rid="pntd.0002636-Westesson1">
                  <sup>
                     <a href="#pntd.0002636-Westesson1">[40]</a>
                  </sup>
               </span>. Moreover, potential intra-gene recombination was also inspected with RDP using individual
               gene datasets, and the incompatibility among phylogenies inferred from genes (NS5
               and E) was also investigated with GiRaF v1.01 
               <span ref-type="bibr" rid="pntd.0002636-Nagarajan1">
                  <sup>
                     <a href="#pntd.0002636-Nagarajan1">[41]</a>
                  </sup>
               </span> that searches incompatible clades among posterior set of trees (PST) obtained from
               different genomic regions with threshold of 70% for incompatible clades. The PST was
               obtained during Monte Carlo Markov chain (MCMC) stationarity using four chains, one
               ‘cold’ and three ‘heated’, after 20 million of generations, sampling every 5000 generations
               using MrBayes v3.2.1 
               <span ref-type="bibr" rid="pntd.0002636-Ronquist1">
                  <sup>
                     <a href="#pntd.0002636-Ronquist1">[42]</a>
                  </sup>
               </span>.
            </p>
         </div>
         <div id="s2g">
            <h3>Phylogenetic analyses</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The phylogenetic signal content of the sequence datasets to phylogenetic reconstruction
               was investigated by Likelihood mapping method 
               <span ref-type="bibr" rid="pntd.0002636-Strimmer1">
                  <sup>
                     <a href="#pntd.0002636-Strimmer1">[43]</a>
                  </sup>
               </span>, implemented in TREE-PUZZLE v5.2 
               <span ref-type="bibr" rid="pntd.0002636-SchmidtH1">
                  <sup>
                     <a href="#pntd.0002636-SchmidtH1">[44]</a>
                  </sup>
               </span>. The concordance among gene (E and NS5) datasets without recombinant sequences was
               further assessed using a permutation test with significance level (α) of 0.05 after
               10000 permutations, implemented in the Congruence among Distance Matrices (CADM) package
               
               <span ref-type="bibr" rid="pntd.0002636-Campbell1">
                  <sup>
                     <a href="#pntd.0002636-Campbell1">[45]</a>
                  </sup>
               </span>. Phylogenetic trees were generated by Maximum Likelihood (ML) criterion using GARLI
               v2.0 
               <span ref-type="bibr" rid="pntd.0002636-Zwickl1">
                  <sup>
                     <a href="#pntd.0002636-Zwickl1">[46]</a>
                  </sup>
               </span> that uses a stochastic algorithm to estimate simultaneously the best topology, branch
               lengths and substitution model parameters that maximize the log Likelihood (lnL).
               The confidence of ML trees was assessed by the convergence of lnL scores from ten
               independent replicates. We used a substitution model based on general time reversible
               (GTR) model with gamma-distributed rate variation (Γ) and a proportion of invariant
               sites (I). Support for the topology was obtained after 1000 non-parametric bootstrap
               replicates with GARLI. Then, we summarized the bootstrap trees into one consensus
               tree to access bootstrap values, using Dendropy v3.10.1 
               <span ref-type="bibr" rid="pntd.0002636-Sukumaran1">
                  <sup>
                     <a href="#pntd.0002636-Sukumaran1">[47]</a>
                  </sup>
               </span>.
            </p>
         </div>
         <div id="s2h">
            <h3>Selection analyses</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">To infer the selection pressures acting on each gene of ZIKV, we estimated the difference
               between the non-synonymous (
               <i>dN</i>) and synonymous (
               <i>dS</i>) rates per codon site using the single likelihood ancestor counting (SLAC) algorithm
               available in HyPhy v0.99 
               <span ref-type="bibr" rid="pntd.0002636-Pond1">
                  <sup>
                     <a href="#pntd.0002636-Pond1">[48]</a>
                  </sup>
               </span>, assuming a significance level of 1% (α = 0.01). In the HyPhy output, values of ω
               are expressed as ω = 
               <i>dN</i> - 
               <i>dS</i>. Therefore, ω smaller than zero (ω&lt;0) indicate purifying, negative selection.
            </p>
         </div>
         <div id="s2i">
            <h3>Prediction of glycosylation sites</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Potential glycosylation sites that may have adaptive value were previously described
               in ZIKV proteins 
               <span ref-type="bibr" rid="pntd.0002636-Lanciotti1">
                  <sup>
                     <a href="#pntd.0002636-Lanciotti1">[17]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pntd.0002636-Kuno1">
                  <sup>
                     <a href="#pntd.0002636-Kuno1">[20]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pntd.0002636-Haddow1">
                  <sup>
                     <a href="#pntd.0002636-Haddow1">[26]</a>
                  </sup>
               </span>. Thus, we investigated partial E sequences to detect potential glycosylation sites
               using NetCGlyc v1.0 
               <span ref-type="bibr" rid="pntd.0002636-Julenius1">
                  <sup>
                     <a href="#pntd.0002636-Julenius1">[49]</a>
                  </sup>
               </span>, NetOGlyc v3.1 
               <span ref-type="bibr" rid="pntd.0002636-Julenius2">
                  <sup>
                     <a href="#pntd.0002636-Julenius2">[50]</a>
                  </sup>
               </span>, YinOYang v1.2 
               <span ref-type="bibr" rid="pntd.0002636-Gupta1">
                  <sup>
                     <a href="#pntd.0002636-Gupta1">[51]</a>
                  </sup>
               </span> and NetNGlyc v1.0 
               <span ref-type="bibr" rid="pntd.0002636-Gupta1">
                  <sup>
                     <a href="#pntd.0002636-Gupta1">[51]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pntd.0002636-Blom1">
                  <sup>
                     <a href="#pntd.0002636-Blom1">[52]</a>
                  </sup>
               </span> methods that employ algorithms based in neural networks to predict, respectively,
               C-mannosylated, mucin-type O-linked, N-acetylglucosamine (GlcNAc) and N-linked glycosylation
               sites. To infer the structural position of the predicted glycosylation sites, we modeled
               the tridimensional structures of E regions of viral polyprotein of the Micronesian
               strain (GenBank accession number ACD75819). We used the homologous sequences from
               Japanese Encephalitis virus (PDB code 3p54), West Nile virus (PDB code 2i69) and Dengue
               virus type 3 (PDB code 1uzg). The amino acids sequences were aligned using MUSCLE
               v3.7 
               <span ref-type="bibr" rid="pntd.0002636-Edgar1">
                  <sup>
                     <a href="#pntd.0002636-Edgar1">[29]</a>
                  </sup>
               </span>, a total of 1000 independent models were generated and optimized using Modeller v9.10
               
               <span ref-type="bibr" rid="pntd.0002636-Sali1">
                  <sup>
                     <a href="#pntd.0002636-Sali1">[53]</a>
                  </sup>
               </span>, and the best models were validated with PROCHECK v3.5.4 
               <span ref-type="bibr" rid="pntd.0002636-Laskowski1">
                  <sup>
                     <a href="#pntd.0002636-Laskowski1">[54]</a>
                  </sup>
               </span>.
            </p>
         </div>
         <div id="s2j">
            <h3>Phylodynamic analysis</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Maximum Clade Credibility (MCC) trees were inferred using a Markov Chain Monte Carlo
               (MCMC) Bayesian approach implemented on the program BEAST v1.6.2 
               <span ref-type="bibr" rid="pntd.0002636-Drummond1">
                  <sup>
                     <a href="#pntd.0002636-Drummond1">[55]</a>
                  </sup>
               </span> under GTR + Γ + I and a relaxed (uncorrelated lognormal) molecular clock 
               <span ref-type="bibr" rid="pntd.0002636-Drummond2">
                  <sup>
                     <a href="#pntd.0002636-Drummond2">[56]</a>
                  </sup>
               </span>. MCMC convergence was obtained for four independent runs with 50 million generations,
               which were sufficient to obtain a proper sample for the posterior at MCMC stationarity,
               assessed by effective sample sizes (ESS) above 200 inspected using Tracer v1.5 (
               <div tagxxx="ext-link">http://tree.bio.ed.ac.uk/software/tracer/</div>). Furthermore, using the concatenated sequences of E and NS5 genes, we employed a
               discrete model attributing state characters representing isolation locality, animal
               source, recombination and N- linked glycosylation on E protein of each of the strains
               with the Bayesian Stochastic Search Variable (BSSVS) algorithm 
               <span ref-type="bibr" rid="pntd.0002636-Lemey1">
                  <sup>
                     <a href="#pntd.0002636-Lemey1">[57]</a>
                  </sup>
               </span>, implemented in BEAST. This method estimates the most probable state at each node
               in the MCC trees, allowing us to reconstruct plausible ancestral states on these nodes.
               Moreover, we represented the viral migration in Google Earth (
               <div tagxxx="ext-link">http://www.google.com/earth/</div>), using the SPREAD v1.0.3 program 
               <span ref-type="bibr" rid="pntd.0002636-Bielejec1">
                  <sup>
                     <a href="#pntd.0002636-Bielejec1">[58]</a>
                  </sup>
               </span>. We evaluated the correlation among viral states and inferred phylogenies from PST
               by the parsimony score (PS), association index (AI) and monophyletic clade size (MC),
               with BaTS v1.0 
               <span ref-type="bibr" rid="pntd.0002636-Parker1">
                  <sup>
                     <a href="#pntd.0002636-Parker1">[59]</a>
                  </sup>
               </span> after 10000 null replications. In addition, we investigated the occurrence of correlated
               evolutionary change among ZIKV phenotypes (glycosylation pattern and vector host)
               along PST, employing a ML approach to test the fit of the two evolutionary models,
               one where the two traits evolve independently on the phylogenetic tree (independent
               model), and one where they evolve in a correlated way (dependent model) 
               <span ref-type="bibr" rid="pntd.0002636-Barker1">
                  <sup>
                     <a href="#pntd.0002636-Barker1">[60]</a>
                  </sup>
               </span>, using BayesTraits program (
               <div tagxxx="ext-link">http://www.evolution.rdg.ac.uk/</div>). To evaluate model suitability to ZIKV data, we estimated the marginal likelihoods
               for both models after 1000 bootstrap replications and compared Bayes factors (BF)
               between models 
               <span ref-type="bibr" rid="pntd.0002636-Suchard1">
                  <sup>
                     <a href="#pntd.0002636-Suchard1">[61]</a>
                  </sup>
               </span>, using Tracer v1.5.
            </p>
         </div>
      </div>
      <div id="s3">
         <h2>Results/Discussion</h2>
         <div id="s3a">
            <h3>Recombination among ZIKV strains</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The primary analyses with RDP suggested 13 recombination events in ZIKV complete genomes
               (
               <span ref-type="supplementary-material" rid="pntd.0002636.s006">Table S2</span>), Rec-HMM also detected genomic breakpoints with confidence in the following alignment
               positions: 1044 to 1095, 5181 to 5238, 9007 to 9132 and 9580 to 9631 (
               <span ref-type="supplementary-material" rid="pntd.0002636.s003">Figure S1</span>). Since the results obtained by both methods revealed breakpoints in the E and NS5
               genomic regions, we investigated these evidences with RDP on partial gene sequences.
               We found a single event in E sequences with estimated breakpoints near to the 414
               <sup>th</sup> and 1065
               <sup>th</sup> site of E gene reaching nine viral strains: ArA986, HD78788, ArA27101, ArA27290,
               ArA27096, ArA27443, ArA27407, ArA27106 and ArA982. These results were found by Bootscan,
               Maxchi, Chimaera, SiSscan and 3Seq methods and supported by significant 
               <i>p</i>-values of 1.31E-5, 2.85E-3, 1.59E-3, 1.79E-29 and 6.85E-19, respectively. Likewise,
               only one recombination event was detected in NS5 sequences with estimated breakpoints
               near sites 1581 and 2152 of the NS5 gene from strains ArD158084, ArB1362 and ArD157995.
               These findings were confirmed by Bootscan, Maxchi, Chimaera, SiSscan and 3Seq methods
               and supported by significant 
               <i>p</i>-values of 9.93E-9, 3.32E-7, 3.32E-7, 5.27E-28 and 7.65E-24, respectively. These potential
               recombinant sequences were excluded from further analyses to avoid inferential biases
               
               <span ref-type="bibr" rid="pntd.0002636-Posada2">
                  <sup>
                     <a href="#pntd.0002636-Posada2">[62]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pntd.0002636-Schierup1">
                  <sup>
                     <a href="#pntd.0002636-Schierup1">[63]</a>
                  </sup>
               </span>. To perform the phylogenetic analysis we concatenated E and NS5 sequences and replaced
               inferred recombinant fragments with missing data. This is in line with the use of
               Maximum Likelihood approaches, which is fairly robust to the introduction of gaps
               
               <span ref-type="bibr" rid="pntd.0002636-Wiens1">
                  <sup>
                     <a href="#pntd.0002636-Wiens1">[64]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pntd.0002636-Wiens2">
                  <sup>
                     <a href="#pntd.0002636-Wiens2">[65]</a>
                  </sup>
               </span>. In addition, we found incompatibilities between E and NS5 phylogenies using GiRaF.
               The three discordant strains (ArD128000, ArA1465 and ArD142623) were excluded, and
               we used 40 (31 from E and 36 from NS5) concatenated sequences for phylogenetic analysis.
               Moreover, we also found that the two remaining datasets for E and NS5 have no conflicting
               phylogenetic signal, as estimated by a CADM test (
               <i>p</i>-value = 9.99E-5 and α = 0.05). Given the limited sampling that we investigated, these
               results indicate that ZIKV may be experiencing recombination in the field, which is
               uncommon among flaviviruses 
               <span ref-type="bibr" rid="pntd.0002636-SimonLoriere1">
                  <sup>
                     <a href="#pntd.0002636-SimonLoriere1">[66]</a>
                  </sup>
               </span>. These findings remain to be properly evaluated and assessed related to their effects
               on viral spread, zoonotic maintenance and epidemiologic potential. The possibility
               that our findings could be a consequence of cross contamination among isolates seems
               highly improbable given the extreme precautions that were taken. RNA extraction and
               reverse transcription were done separately for each isolate under BSL-II cabinets,
               sequenced several times leading to identical sequences, even when processed in different
               laboratories in Sao Paulo, Brazil, and Dakar, Senegal.
            </p>
         </div>
         <div id="s3b">
            <h3>Phylogenetic analysis</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We first investigated the phylogenetic signal content in our data by reconstructing
               50000 quartets for each gene segment using the likelihood mapping method (see 
               <span ref-type="sec" rid="s2">methods</span> section). Our results indicated that NS5 and E datasets had high phylogenetic signal
               content given their lower percentage of unresolved quartets (3.2% and 3.4%, respectively),
               while 3′NCR showed less signal (16.4% of unresolved quartets) and was not considered.
               The ML trees for E (data not shown), NS5 (data not shown) and the two concatenated
               genes (
               <span ref-type="fig" rid="pntd-0002636-g001">Figure 1</span>) reinforced that ZIKV strains could be classified in three major clusters 
               <span ref-type="bibr" rid="pntd.0002636-Lanciotti1">
                  <sup>
                     <a href="#pntd.0002636-Lanciotti1">[17]</a>
                  </sup>
               </span>. Accordingly, the African strains were arranged into two groups: the MR766 prototype
               strain cluster (yellow sector on 
               <span ref-type="fig" rid="pntd-0002636-g001">Figure 1</span>) and the Nigerian cluster (green sector on 
               <span ref-type="fig" rid="pntd-0002636-g001">Figure 1</span>); and the Micronesian and Malaysian strains clustered together forming the Asian
               clade (
               <span ref-type="fig" rid="pntd-0002636-g001">Figure 1</span>), in agreement with 
               <span ref-type="bibr" rid="pntd.0002636-Haddow1">
                  <sup>
                     <a href="#pntd.0002636-Haddow1">[26]</a>
                  </sup>
               </span>. For West Africa, the strains from Côte d'Ivoire and Senegal were found in both African
               clusters, suggesting that at least two distinct lineages of ZIKV circulated in these
               countries. Interestingly, we found that the position of the Senegalese cluster, comprising
               viruses isolated from 1998 to 2001 associated with 
               <i>A. dalzieli</i>, branching as a sister group of HD78788 isolated in Senegal in 1991, was not simply
               explained by recombination (with both Giraf and RDP) or poor rooting of the tree,
               since it did not depend on the inclusion (
               <span ref-type="fig" rid="pntd-0002636-g001">Figure 1</span>) or exclusion (
               <span ref-type="supplementary-material" rid="pntd.0002636.s004">Figure S2</span>) of the Spondweni, which is a 
               <i>bonafide</i> outgroup. It was observed 65% of the time during a highly stringent maximum likelihood
               (ML) analysis with GARLI, not taking into account dates of isolation, but crucially
               it had a posterior probability of one during Bayesian Inference (BI) that do take
               into account dates of isolation. Although we cannot rule out systematic topological
               errors, BI was certainly better informed than ML, since RNA viruses evolve fast, making
               their times of isolation important parameters for phylogenetic inference. Moreover,
               since we did not find compositional or codon usage biases in those sequences and in
               agreement with the consistent BI results, we could not rule out that the long branch
               length observed was not due to a detected increase of almost 10 fold increase in the
               rate of change along that lineage, which was not caused by detectable positive selection,
               as evaluated using HyPhy.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pntd.0002636.g001</div>
               <span class="label">Figure 1</span>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Maximum likelihood phylogenetic tree inferred for concatenated of sequences from Envelope
                     and NS5 genes of Zika virus.
                  </div>
                  <p>Consensus tree summarized after 1000 non-parametric bootstrap replicates, with support
                     values greater than 60% shown in the nodes. The cluster the Ugandan MR766 prototype
                     strain was highlighted by the yellow sector and the Nigerian cluster was highlighted
                     by the green sector. The strains from Senegal and Côte d'Ivoire are shown in green
                     and orange, respectively. The tree has been rooted with the Spondweni lineage isolated
                     in South Africa was used as outgroup to root the tree.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
         </div>
         <div id="s3c">
            <h3>Selection analyses</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Selection analyses of E and NS5 genes uncovered several sites under strong negative
               selection indicated by ω&lt;0. This suggests frequent purging of deleterious polymorphisms
               in functionally important genes. Likewise, the lack of positively selected sites,
               indicated by ω&gt;0, is typical of highly adapted phenotypes and shows no detectable
               directional change on the available data. Our findings were expected, as the infection
               and transmission modes of ZIKV allow the accumulation of synonymous mutations and
               negatively selected sites 
               <span ref-type="bibr" rid="pntd.0002636-Hanada1">
                  <sup>
                     <a href="#pntd.0002636-Hanada1">[67]</a>
                  </sup>
               </span>. The alternation between arthropod vector and mammal hosts may impose several barriers
               to non-synonymous mutations in important genes 
               <span ref-type="bibr" rid="pntd.0002636-Holmes1">
                  <sup>
                     <a href="#pntd.0002636-Holmes1">[68]</a>
                  </sup>
               </span>.
            </p>
         </div>
         <div id="s3d">
            <h3>Phylodynamic analyses</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The μ and the highest posterior densities (HPD with 95% of confidence interval) estimated
               with Beast for E, NS5 and 3′NCR genomic regions were, respectively, 2.135E-3 (2.04E-3
               to 2.33221E-3), 7.1789E-4 (6.9466E-4 to 7.417E-4) and 1.1285E-3 (2.708E-4 to 2.504E-3)
               substitutions per site per year, which are similar to μ estimated other flaviviruses
               
               <span ref-type="bibr" rid="pntd.0002636-Twiddy1">
                  <sup>
                     <a href="#pntd.0002636-Twiddy1">[69]</a>
                  </sup>
               </span>. As evolutionary rates are the result of spontaneous mutations followed by selection,
               differences 
               <i>per gene</i> are expected and in accordance with their biological role, given that the NS5 is
               a polymerase and the E is a surface protein. In addition, the root date estimates
               and 95% HPDs of the phylogenetic trees for E, NS5 and 3′NCR genomic regions were,
               respectively, 1900 (1851 to 1937), 1927 (1887 to 1940) and 1923 (1874 to 1959). These
               dates suggest a recent origin for the ZIKV strains (included in this study) near to
               the beginning of the 20
               <sup>th</sup> century.
            </p>
         </div>
         <div id="s3e">
            <h3>Movement of ZIKV</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Based on our samples we inferred the most likely geographical pathway connecting ZIKV
               lineages. These results indicated that ZIKV emerged in Uganda around 1920, most probably
               between 1892 and 1943. This inference is in line with the first known ZIKV isolation
               in Uganda in 1947 
               <span ref-type="bibr" rid="pntd.0002636-Dick1">
                  <sup>
                     <a href="#pntd.0002636-Dick1">[5]</a>
                  </sup>
               </span>. We found two independent ZIKV introductions into West Africa from the Eastern portion
               of the continent (
               <span ref-type="fig" rid="pntd-0002636-g002">Figures 2</span> and 
               <span ref-type="supplementary-material" rid="pntd.0002636.s004">S2A</span>, and kml file in 
               <span ref-type="supplementary-material" rid="pntd.0002636.s001">Dataset S1</span>). The first viral introduction into Côte d'Ivoire (CI) and Senegal (SN) was related
               to the MR766 cluster (yellow lines in 
               <span ref-type="fig" rid="pntd-0002636-g002">Figure 2</span>), which possibly moved from Uganda around 1940 into Dezidougou (CI). From there,
               this lineage probably spread to Kedougou in Senegal (SN) around 1985 and to Sokala-Sobara
               (CI) around 1995. The second introduction was related to a Nigerian cluster (green
               lines in 
               <span ref-type="fig" rid="pntd-0002636-g002">Figure 2</span>), when ZIKV probably moved from Uganda to the Central African Republic and Nigeria
               around 1935. From Nigeria, the virus probably spread to Saboya (SN) around 1950 and
               from there to Dezidougou (CI) and Bandia (SN) around 1960. From Bandia, ZIKV was introduced
               into Kedougou (SN) around 1965 and from there to Burkina Faso around 1980 and to Dakar
               (SN) around 1985. Moreover, an additional ZIKV lineage from Uganda probably spread
               to Malaysia around 1945 and from there, the virus reached Micronesia around 1960,
               forming the Asian cluster 
               <span ref-type="bibr" rid="pntd.0002636-Haddow1">
                  <sup>
                     <a href="#pntd.0002636-Haddow1">[26]</a>
                  </sup>
               </span>. The correlation between viral location (coded as character states) and phylogenies
               was strongly supported by significant AI and PS values, 
               <i>p</i>-values≤1.00 E-4 (
               <span ref-type="supplementary-material" rid="pntd.0002636.s002">Dataset S2</span>). Thus, assuming an origin of ZIKV in Uganda, our findings revealed at least two
               independent exits from East Africa in agreement with the two previously proposed African
               clades 
               <span ref-type="bibr" rid="pntd.0002636-Lanciotti1">
                  <sup>
                     <a href="#pntd.0002636-Lanciotti1">[17]</a>
                  </sup>
               </span> and also pointed to a viral migratory flow from Eastern Africa to Asia. Although
               our sampling was the most comprehensive to this date, our conclusions about ZIKV movement
               are informed conjectures at best on the most plausible hypotheses on ZIKV spreading
               patterns, which are limited by the inherent biases of this type of analyses.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pntd.0002636.g002</div>
               <span class="label">Figure 2</span>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Geographic spread of ZIKV in Africa and Asia.</div>
                  <p>The directed lines connect the most probable sources and target localities of viral
                     lineages (shown by arrows), with widths proportional to the posterior probabilities
                     and values shown in red. Only plausible routes with probabilities above 50% are shown.
                     The distinct introductions into Senegal and Côte d'Ivoire were represented by different
                     colors. The estimated time to the most recent common ancestor of strains from different
                     countries are shown with 95% posterior time intervals in parenthesis and could be
                     interpreted as the oldest possible year of introduction of that lineage at that locality.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
         </div>
         <div id="s3f">
            <h3>Animal sources of ZIKV</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The association of the animal sources with viral lineages (
               <span ref-type="supplementary-material" rid="pntd.0002636.s004">Figure S2B</span>) suggested that ZIKV dispersed widely among distinct animal species without a clear
               pattern of preference, maybe associated to the enzootic cycle of ZIKV in Africa, whose
               natural cycle allows a broad range of hosts 
               <span ref-type="bibr" rid="pntd.0002636-Kuno3">
                  <sup>
                     <a href="#pntd.0002636-Kuno3">[70]</a>
                  </sup>
               </span>. Nevertheless, we found significant MC (
               <i>p</i>-value≈1.00 E-4, 
               <span ref-type="supplementary-material" rid="pntd.0002636.s002">Dataset S2</span>) for ZIKV strains isolated from 
               <i>A. dalzieli</i>, suggesting a possible important role of this zoophilic vector 
               <span ref-type="bibr" rid="pntd.0002636-Diallo1">
                  <sup>
                     <a href="#pntd.0002636-Diallo1">[71]</a>
                  </sup>
               </span> in West Africa. This association was found to be robust to the exclusion of vertebrate
               host from the analysis. The plausibility of the putative recombination events we detected
               (
               <span ref-type="supplementary-material" rid="pntd.0002636.s006">Table S2</span>), could in part be explained by mosquitoes taking sequential blood meals from distinct
               animal species harboring distinct ZIKV lineages, which is in line with ours and others
               host range findings 
               <span ref-type="bibr" rid="pntd.0002636-Kuno3">
                  <sup>
                     <a href="#pntd.0002636-Kuno3">[70]</a>
                  </sup>
               </span>. Also, when analyzing the increase of ZIKV activity in Kedougou, (where most of the
               viruses analyzed herein were collected), we noticed that ZIKV activity is much more
               frequent, with an interval of 1–2 years, compared to the 5 to 8 years cycle of dengue
               and yellow fever virus. Hence from 1972 to 2002, ZIKV emerged in 1973, 1976, 1979,
               1980 and 1981. Such frequent activity can also be an opportunity of co-circulation
               and mixing of multiple genotypes present in the forest and that may favor recombination
               among them.
            </p>
         </div>
         <div id="s3g">
            <h3>A phylodynamic context for recombination events</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The occurrence of recombination among ZIKV strains in time-scaled phylogenetic trees
               suggested that some ZIKV lineages sampled in Dezidougou (CI) in 1990 (ArA27101, ArA27290,
               ArA27096, ArA27443, ArA27407 and ArA27106) with recombinant E (
               <span ref-type="supplementary-material" rid="pntd.0002636.s004">Figure S2C</span>) shared a common ancestor around 1962 (ranging from 1951 to 1967 HPD with 95% of
               confidence interval). Likewise, the strain ArA982 was also isolated at Dezidougou
               in 1999 and its sister-group ArA986, which shared a common ancestor with the former
               around 1992 (ranging from 1981 to 1996 HPD with 95% of confidence interval), was sampled
               in the neighbor province Sokala-Sobara (CI) in 1999. Together these results indicated
               that recombination in envelope protein could be an important trend among the enzootic
               cycle of ZIKV at this region in Côte d'Ivoire, as ZIKV lineages did not show a clear
               pattern of host preference and recombination requires the infection of the same host
               by more than one viral strain. Besides, the other E recombinant strain (HD78788),
               isolated from a human case at Dakar (SN) in 1991, shared a common ancestor around
               1984 (ranging from 1976 to 1988 HPD with 95% of confidence interval) with ZIKV strains
               from Kedougou (SN). Conversely, the three NS5 recombinants did not cluster along phylogenetic
               trees (
               <span ref-type="supplementary-material" rid="pntd.0002636.s004">Figure S2C</span>), although two of them were isolated in Kedougou from 
               <i>A. dalzieli</i> mosquitoes in 2001 (ArD157995 and ArD158084) and the other (ArB1362) was isolated
               in Bouboui, Central African Republic, from 
               <i>A. africanus</i> mosquitoes in 1968. The preferential distribution of recombinant strains along phylogenies
               was supported by significant 
               <i>p</i>-values of AI and PS ≤2.00E-4 (
               <span ref-type="supplementary-material" rid="pntd.0002636.s002">Dataset S2</span>) and the adjacency patterns of E and NS5 recombinants were also confirmed by MC statistics
               (
               <span ref-type="supplementary-material" rid="pntd.0002636.s002">Dataset S2</span>).
            </p>
         </div>
         <div id="s3h">
            <h3>Glycosylation patterns in ZIKV envelope protein</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Our analyses predicted several glycosylation sites in the E protein (
               <span ref-type="fig" rid="pntd-0002636-g003">Figure 3</span>). We detected a probable motif (Asn-X-Thr) among E sequences from several ZIKV strains
               (
               <span ref-type="fig" rid="pntd-0002636-g003">Figure 3A</span>), which suggests a N-linked glycosylation site in the residue Asn-154, in agreement
               with 
               <span ref-type="bibr" rid="pntd.0002636-Lanciotti1">
                  <sup>
                     <a href="#pntd.0002636-Lanciotti1">[17]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pntd.0002636-Haddow1">
                  <sup>
                     <a href="#pntd.0002636-Haddow1">[26]</a>
                  </sup>
               </span>. This residue is located on an α-helix in the E protein structure (yellow arrow in
               
               <span ref-type="fig" rid="pntd-0002636-g003">Figure 3A</span> and yellow bead in 
               <span ref-type="fig" rid="pntd-0002636-g003">Figure 3B</span>). Our results also pointed several O-linked glycosylation sites in the E protein
               (red arrows in 
               <span ref-type="fig" rid="pntd-0002636-g003">Figure 3A</span> and red beads 
               <span ref-type="fig" rid="pntd-0002636-g003">Figure 3B</span>) but no C-mannosylated site. We found a probable mucin-type O-linked glycosylated
               site at residue Thr-170 in E protein from all ZIKV strains, and other mucin sites
               at residues Thr-245 and Thr-381 in some isolates (
               <span ref-type="fig" rid="pntd-0002636-g003">Figure 3A</span>). In addition, we also uncovered probable O-GlcNAc attachment sites at residues Ser-142,
               Ser-227, Thr-231, Ser-304, Thr-366 and Thr-381 in E from some strains (
               <span ref-type="fig" rid="pntd-0002636-g003">Figure 3A</span>).
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pntd.0002636.g003</div>
               <span class="label">Figure 3</span>
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Mapping of predicted glycosylation sites on envelope protein of ZIKV.</div>
                  <p>A) Alignment of E protein showing predicted glycosylation sites. Red arrows point
                     to O-linked glycosylation sites (Ser or Thr residues) and the yellow arrow points
                     to the N-linked glycosylation site (Asn-X-Thr motif). B) Tridimensional structure
                     of E protein. Red beads indicate O-linked glycosylation sites and the yellow bead
                     indicates the unique N-linked glycosylation site.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Given the importance of the N-linked glycosylation site around position 154 of the
               E protein for infectivity and assembly of flaviviruses 
               <span ref-type="bibr" rid="pntd.0002636-Hanna1">
                  <sup>
                     <a href="#pntd.0002636-Hanna1">[72]</a>
                  </sup>
               </span>–
               <span ref-type="bibr" rid="pntd.0002636-Mondotte1">
                  <sup>
                     <a href="#pntd.0002636-Mondotte1">[74]</a>
                  </sup>
               </span> and the fact that we observed polymorphisms in this motif (deletions and substitutions
               156 Thr/Iso), we investigated the correlation between the conservation of this motif
               (Asn-X-Thr) and phylogenies for ZIKV strains. Our results suggested that the acquisition
               of this glycosylation site is a recurrent event in the history of ZIKV, given the
               observed changes from Isoleucine to Threonine and vice-versa more than once in the
               MCC tree (
               <span ref-type="supplementary-material" rid="pntd.0002636.s004">Figure S2D</span>), supported by 
               <i>p</i>-values for AI and PS ≤7.00E-4 (
               <span ref-type="supplementary-material" rid="pntd.0002636.s002">Dataset S2</span>). However, our conclusions are limited due to serial passages of the former ZIKV
               strains (
               <span ref-type="supplementary-material" rid="pntd.0002636.s004">Figure S2D</span>) in mouse brain 
               <span ref-type="bibr" rid="pntd.0002636-Haddow1">
                  <sup>
                     <a href="#pntd.0002636-Haddow1">[26]</a>
                  </sup>
               </span>, which could result in the loss of this glycosylation site, as observed in West Nile
               virus 
               <span ref-type="bibr" rid="pntd.0002636-Chambers2">
                  <sup>
                     <a href="#pntd.0002636-Chambers2">[75]</a>
                  </sup>
               </span>.
            </p>
         </div>
         <div id="s3i">
            <h3>Correlated evolutionary change along ZIKV phylogenies</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Since it was demonstrated that the absence of an N-linked glycosylation site on the
               E protein enhances viral infectivity for C6/36 mosquito cells 
               <span ref-type="bibr" rid="pntd.0002636-Hanna1">
                  <sup>
                     <a href="#pntd.0002636-Hanna1">[72]</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pntd.0002636-Lee1">
                  <sup>
                     <a href="#pntd.0002636-Lee1">[73]</a>
                  </sup>
               </span> and E protein of ZIKV strains from 
               <i>A. dalzieli</i>, which was the unique vector source with significant MC–showed an absence of this
               glycosylation site, we investigated the correlation between this mosquito-source and
               N-linked glycosylation patterns of E protein along PST. Our results indicated the
               changes in glycosylation patterns (presence or absence) and vector (
               <i>A. dalzieli</i> or not) were correlated during ZIKV emergence, which was supported by BF for dependent
               model (BF≈47.004) greater than for them to independent model. These findings could
               be related to the enzootic cycle of ZIKV in West Africa and the zoophilic behavior
               of 
               <i>A. dalzieli</i>
               <span ref-type="bibr" rid="pntd.0002636-Diallo1">
                  <sup>
                     <a href="#pntd.0002636-Diallo1">[71]</a>
                  </sup>
               </span>, whose females take blood meals from a broad range of vertebrates, which provides
               additional evidence for the absence of host preference (as described in Animal sources
               of ZIKV). Hence, further studies are necessary to understand the consequences of our
               results to ZIKV transmission cycle in nature.
            </p>
         </div>
         <div id="s3j">
            <h3>Biological correlates of our findings</h3>
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Our analyses indicated that ZIKV may have experienced several recombination events,
               which is uncommon among flaviviruses 
               <span ref-type="bibr" rid="pntd.0002636-SimonLoriere1">
                  <sup>
                     <a href="#pntd.0002636-SimonLoriere1">[66]</a>
                  </sup>
               </span>. The recurrent loss and gain of the N-linked glycosylation site in the E protein
               could be related to mosquito-cell infectivity 
               <span ref-type="bibr" rid="pntd.0002636-Lee1">
                  <sup>
                     <a href="#pntd.0002636-Lee1">[73]</a>
                  </sup>
               </span> and the correlated loss of this glycosylation site in ZIKV strains from 
               <i>A. dalzieli</i> also provides indirect evidence for the enzootic cycle, since this vector has a zoophilic
               behavior 
               <span ref-type="bibr" rid="pntd.0002636-Diallo1">
                  <sup>
                     <a href="#pntd.0002636-Diallo1">[71]</a>
                  </sup>
               </span> that may spread ZIKV among several hosts. Crucially, our results corroborated the
               notion that at least three distinct ZIKV clusters shared a common ancestor possibly
               with Ugandan lineages around 1920, followed by two events of East to West Africa spread
               (
               <span ref-type="fig" rid="pntd-0002636-g002">Figure 2</span>): (
               <i>i</i>) one related to the MR766 cluster introduction to Côte d'Ivoire and posterior spread
               to Senegal and; (
               <i>ii</i>) other related to the Nigerian cluster introduction in Senegal and posterior dispersion
               to Côte d'Ivoire and Burkina Faso.
            </p>
         </div>
      </div>
      <div id="s4">
         <h2>Supporting Information</h2>
         <div tagxxx="supplementary-material">
            <span class="label">Dataset S1</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>Spread of ZIKV strains in Africa and Asia.</b> A kml file to picture the history of ZIKV movement into Africa and Asia during the
                  time, it is executable in Google Earth program (
                  <div tagxxx="ext-link">http://www.google.com/earth/</div>).
               </p>
               <p>(KML)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">Dataset S2</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>Significance of the correlation among phylogenies and attributes of ZIKV lineages.</b>
               </p>
               <p>(DOC)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">Figure S1</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>Recombination analysis using Rec-HMM along ZIKV genomes.</b> The dashed green lines indicate estimated breakpoints in the genomes.
               </p>
               <p>(TIF)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">Figure S2</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>Maximum clade credibility (MCC) trees for concatenated sequences summarizing lineage
                     states along a time-scaled tree, with posterior probability values shown near the
                     nodes.
                  </b> (
                  <b>A</b>) Most probable geographical location coded according to map (
                  <span ref-type="fig" rid="pntd-0002636-g002">Figure 2</span>): Uganda (UG), Central African Republic (CF), Dezidougou in Côte d'Ivoire (DE), Sokala-Sobara
                  in Côte d'Ivoire (SS), Kedougou in Senegal (KE), Saboya in Senegal (SA), Bandia in
                  Senegal (BA), Dakar in Senegal (DA), Burkina Faso (BF), Nigeria (NG), Malaysia (MY)
                  and Yap Island in the Federated States of Micronesia (FM); (
                  <b>B</b>) most probable animal source; (
                  <b>C</b>) recombination events per region; and (
                  <b>D</b>) glycosylation polymorphisms.
               </p>
               <p>(TIF)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">Table S1</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>Source, country and year of isolation from ZIKV strains used in this study.</b>
               </p>
               <p>(DOC)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">Table S2</span>
            <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <p>
                  <b>Detection of recombination events in ZIKV genomes.</b>
               </p>
               <p>(DOC)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
      </div>
      <div tagxxx="back">
         <div tagxxx="ack">
            <p xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">We thank Dr. Scott Weaver for helpful comments and critical reading of the manuscript.</p>
         </div>
         <div tag="ref-list">
            <ul>
               <div tagxxx="title">References</div>
               <li tag="ref">
                  <a name="pntd.0002636-Hayes1"></a>
                  <span class="label">1</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Hayes</span>
                        <span tagx="given-names">EB</span>,
                     </span> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Zika virus outside Africa</span>. 
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="volume">15</span>: 
                     <span tagx="fpage">1347</span>–
                     <span tagx="lpage">1350</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.3201/eid1509.090442</div>
                     </div>
                     <span class="pub-id'">[19788800]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Simpson1"></a>
                  <span class="label">2</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Simpson</span>
                        <span tagx="given-names">DI</span>,
                     </span> (
                     <span tagx="year">1964</span>) 
                     <span tagx="article-title">Zika virus infection in man</span>. 
                     <span tagx="source">Trans R Soc Trop Med Hyg</span>
                     <span tagx="volume">58</span>: 
                     <span tagx="fpage">335</span>–
                     <span tagx="lpage">338</span>
                     <span class="pub-id'">[14175744]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Bearcroft1"></a>
                  <span class="label">3</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Bearcroft</span>
                        <span tagx="given-names">WG</span>,
                     </span> (
                     <span tagx="year">1956</span>) 
                     <span tagx="article-title">Zika virus infection experimentally induced in a human volunteer</span>. 
                     <span tagx="source">Trans R Soc Trop Med Hyg</span>
                     <span tagx="volume">50</span>: 
                     <span tagx="fpage">442</span>–
                     <span tagx="lpage">448</span>
                     <span class="pub-id'">[13380987]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Foy1"></a>
                  <span class="label">4</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Foy</span>
                        <span tagx="given-names">BD</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kobylinski</span>
                        <span tagx="given-names">KC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chilson Foy</span>
                        <span tagx="given-names">JL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Blitvich</span>
                        <span tagx="given-names">BJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Travassos da Rosa</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">Probable non-vector-borne transmission of Zika virus, Colorado, USA</span>. 
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="volume">17</span>: 
                     <span tagx="fpage">880</span>–
                     <span tagx="lpage">882</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.3201/eid1705.101939</div>
                     </div>
                     <span class="pub-id'">[21529401]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Dick1"></a>
                  <span class="label">5</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Dick</span>
                        <span tagx="given-names">GWA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kitchen</span>
                        <span tagx="given-names">SF</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Haddow</span>
                        <span tagx="given-names">AJ</span>,
                     </span> (
                     <span tagx="year">1952</span>) 
                     <span tagx="article-title">Zika virus. I. Isolations and serological specificity</span>. 
                     <span tagx="source">Trans R Soc Trop Med Hyg</span>
                     <span tagx="volume">46</span>: 
                     <span tagx="fpage">509</span>–
                     <span tagx="lpage">520</span>
                     <span class="pub-id'">[12995440]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Fagbami1"></a>
                  <span class="label">6</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Fagbami</span>
                        <span tagx="given-names">AH</span>,
                     </span> (
                     <span tagx="year">1979</span>) 
                     <span tagx="article-title">Zika virus infections in Nigeria: virological and seroepidemiological investigations
                        in Oyo State
                     </span>. 
                     <span tagx="source">J Hyg (Lond)</span>
                     <span tagx="volume">83</span>: 
                     <span tagx="fpage">213</span>–
                     <span tagx="lpage">219</span>
                     <span class="pub-id'">[489960]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Robin1"></a>
                  <span class="label">7</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Robin</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Mouchet</span>
                        <span tagx="given-names">J</span>,
                     </span> (
                     <span tagx="year">1978</span>) 
                     <span tagx="article-title">Serological and entomological study on yellow fever in Sierra Leone</span>. 
                     <span tagx="source">Bull Soc Pathol Exot Filiales</span>
                     <span tagx="volume">68</span>: 
                     <span tagx="fpage">249</span>–
                     <span tagx="lpage">258</span>
                     <span class="pub-id'">[1243735]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Jan1"></a>
                  <span class="label">8</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Jan</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Languillat</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Renaudet</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Robin</span>
                        <span tagx="given-names">Y</span>,
                     </span> (
                     <span tagx="year">1978</span>) 
                     <span tagx="article-title">A serological survey of arboviruses in Gabon</span>. 
                     <span tagx="source">Bull Soc Pathol Exot Filiales</span>
                     <span tagx="volume">71</span>: 
                     <span tagx="fpage">140</span>–
                     <span tagx="lpage">146</span>
                     <span class="pub-id'">[743766]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-McCrae1"></a>
                  <span class="label">9</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">McCrae</span>
                        <span tagx="given-names">AW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kirya</span>
                        <span tagx="given-names">BG</span>,
                     </span> (
                     <span tagx="year">1982</span>) 
                     <span tagx="article-title">Yellow fever and Zika virus epizootics and enzootics in Uganda</span>. 
                     <span tagx="source">Trans R Soc Trop Med Hyg</span>
                     <span tagx="volume">76</span>: 
                     <span tagx="fpage">552</span>–
                     <span tagx="lpage">562</span>
                     <span class="pub-id'">[6304948]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Saluzzo1"></a>
                  <span class="label">10</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Saluzzo</span>
                        <span tagx="given-names">JF</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gonzalez</span>
                        <span tagx="given-names">JP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hervé</span>
                        <span tagx="given-names">JP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Georges</span>
                        <span tagx="given-names">AJ</span>,
                     </span> (
                     <span tagx="year">1981</span>) 
                     <span tagx="article-title">Serological survey for the prevalence of certain arboviruses in the human population
                        of the south-east area of Central African Republic
                     </span>. 
                     <span tagx="source">Bull Soc Pathol Exot Filiales</span>
                     <span tagx="volume">74</span>: 
                     <span tagx="fpage">490</span>–
                     <span tagx="lpage">499</span>
                     <span class="pub-id'">[6274526]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Monlun1"></a>
                  <span class="label">11</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Monlun</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Zeller</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Le Guenno</span>
                        <span tagx="given-names">B</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Traoré-Lamizana</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hervy</span>
                        <span tagx="given-names">JP</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">1993</span>) 
                     <span tagx="article-title">Surveillance of the circulation of arbovirus of medical interest in the region of
                        eastern Senegal
                     </span>. 
                     <span tagx="source">Bull Soc Pathol Exot</span>
                     <span tagx="volume">86</span>: 
                     <span tagx="fpage">21</span>–
                     <span tagx="lpage">28</span>
                     <span class="pub-id'">[8099299]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-AkouaKoffi1"></a>
                  <span class="label">12</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Akoua-Koffi</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Diarrassouba</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Bénié</span>
                        <span tagx="given-names">VB</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ngbichi</span>
                        <span tagx="given-names">JM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Bozoua</span>
                        <span tagx="given-names">T</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2001</span>) 
                     <span tagx="article-title">Investigation surrounding a fatal case of yellow fever in Côte d'Ivoire in 1999</span>. 
                     <span tagx="source">Bull Soc Pathol Exot</span>
                     <span tagx="volume">94</span>: 
                     <span tagx="fpage">227</span>–
                     <span tagx="lpage">230</span>
                     <span class="pub-id'">[11681215]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Darwish1"></a>
                  <span class="label">13</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Darwish</span>
                        <span tagx="given-names">MA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hoogstraal</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Roberts</span>
                        <span tagx="given-names">TJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ahmed</span>
                        <span tagx="given-names">IP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Omar</span>
                        <span tagx="given-names">F</span>,
                     </span> (
                     <span tagx="year">1983</span>) 
                     <span tagx="article-title">A sero-epidemiological survey for certain arboviruses (Togaviridae) in Pakistan</span>. 
                     <span tagx="source">Trans R Soc Trop Med Hyg</span>
                     <span tagx="volume">77</span>: 
                     <span tagx="fpage">442</span>–
                     <span tagx="lpage">445</span>
                     <span class="pub-id'">[6314612]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Marchette1"></a>
                  <span class="label">14</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Marchette</span>
                        <span tagx="given-names">NJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Garcia</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rudnick</span>
                        <span tagx="given-names">A</span>,
                     </span> (
                     <span tagx="year">1969</span>) 
                     <span tagx="article-title">Isolation of Zika virus from Aedes aegypti mosquitoes in Malaysia</span>. 
                     <span tagx="source">Am J Trop Med Hyg</span>
                     <span tagx="volume">18</span>: 
                     <span tagx="fpage">411</span>–
                     <span tagx="lpage">415</span>
                     <span class="pub-id'">[4976739]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Olson1"></a>
                  <span class="label">15</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Olson</span>
                        <span tagx="given-names">JG</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ksiazek</span>
                        <span tagx="given-names">TG</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Suhandiman</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Triwibowo</span>,
                     </span> (
                     <span tagx="year">1981</span>) 
                     <span tagx="article-title">Zika virus, a cause of fever in Central Java, Indonesia</span>. 
                     <span tagx="source">Trans R Soc Trop Med Hyg</span>
                     <span tagx="volume">75</span>: 
                     <span tagx="fpage">389</span>–
                     <span tagx="lpage">393</span>
                     <span class="pub-id'">[6275577]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Duffy1"></a>
                  <span class="label">16</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Duffy</span>
                        <span tagx="given-names">MR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chen</span>
                        <span tagx="given-names">T-H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hancock</span>
                        <span tagx="given-names">WT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Powers</span>
                        <span tagx="given-names">AM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kool</span>
                        <span tagx="given-names">JL</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Zika virus outbreak on Yap Island, Federated States of Micronesia</span>. 
                     <span tagx="source">N Engl J Med</span>
                     <span tagx="volume">360</span>: 
                     <span tagx="fpage">2536</span>–
                     <span tagx="lpage">2543</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1056/NEJMoa0805715</div>
                     </div>
                     <span class="pub-id'">[19516034]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Lanciotti1"></a>
                  <span class="label">17</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Lanciotti</span>
                        <span tagx="given-names">RS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kosoy</span>
                        <span tagx="given-names">OL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Laven</span>
                        <span tagx="given-names">JJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Velez</span>
                        <span tagx="given-names">JO</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lambert</span>
                        <span tagx="given-names">AJ</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2008</span>) 
                     <span tagx="article-title">Genetic and serologic properties of Zika virus associated with an epidemic, Yap State,
                        Micronesia, 2007
                     </span>. 
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="volume">14</span>: 
                     <span tagx="fpage">1232</span>–
                     <span tagx="lpage">1239</span>
                     <span class="pub-id'">[18680646]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Heang1"></a>
                  <span class="label">18</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Heang</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Yasuda</span>
                        <span tagx="given-names">CY</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sovann</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Haddow</span>
                        <span tagx="given-names">AD</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Travassos da Rosa</span>
                        <span tagx="given-names">AP</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Zika virus infection, cambodia, 2010</span>. 
                     <span tagx="source">Emerg Infect Dis</span>
                     <span tagx="volume">18</span>: 
                     <span tagx="fpage">349</span>–
                     <span tagx="lpage">351</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.3201/eid1802.111224</div>
                     </div>
                     <span class="pub-id'">[22305269]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Chambers1"></a>
                  <span class="label">19</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Chambers</span>
                        <span tagx="given-names">TJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hahn</span>
                        <span tagx="given-names">CS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Galler</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rice</span>
                        <span tagx="given-names">CM</span>,
                     </span> (
                     <span tagx="year">1990</span>) 
                     <span tagx="article-title">Flavivirus genome organization, expression, and replication</span>. 
                     <span tagx="source">Annu Rev Microbiol</span>
                     <span tagx="volume">44</span>: 
                     <span tagx="fpage">649</span>–
                     <span tagx="lpage">688</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1146/annurev.mi.44.100190.003245</div>
                     </div>
                     <span class="pub-id'">[2174669]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Kuno1"></a>
                  <span class="label">20</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Kuno</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chang</span>
                        <span tagx="given-names">G-JJ</span>,
                     </span> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">Full-length sequencing and genomic characterization of Bagaza, Kedougou, and Zika
                        viruses
                     </span>. 
                     <span tagx="source">Arch Virol</span>
                     <span tagx="volume">152</span>: 
                     <span tagx="fpage">687</span>–
                     <span tagx="lpage">696</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1007/s00705-006-0903-z</div>
                     </div>
                     <span class="pub-id'">[17195954]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Lindenbach1"></a>
                  <span class="label">21</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Lindenbach</span>
                        <span tagx="given-names">BD</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rice</span>
                        <span tagx="given-names">CM</span>,
                     </span> (
                     <span tagx="year">2003</span>) 
                     <span tagx="article-title">Molecular biology of flaviviruses</span>. 
                     <span tagx="source">Adv Virus Res</span>
                     <span tagx="volume">59</span>: 
                     <span tagx="fpage">23</span>–
                     <span tagx="lpage">61</span>
                     <span class="pub-id'">[14696326]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Kuno2"></a>
                  <span class="label">22</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Kuno</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chang</span>
                        <span tagx="given-names">GJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tsuchiya</span>
                        <span tagx="given-names">KR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Karabatsos</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Cropp</span>
                        <span tagx="given-names">CB</span>,
                     </span> (
                     <span tagx="year">1998</span>) 
                     <span tagx="article-title">Phylogeny of the genus Flavivirus</span>. 
                     <span tagx="source">J Virol</span>
                     <span tagx="volume">72</span>: 
                     <span tagx="fpage">73</span>–
                     <span tagx="lpage">83</span>
                     <span class="pub-id'">[9420202]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Gould1"></a>
                  <span class="label">23</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">de Lamballerie</span>
                        <span tagx="given-names">X</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Zanotto</span>
                        <span tagx="given-names">PM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">EC</span>,
                     </span> (
                     <span tagx="year">2003</span>) 
                     <span tagx="article-title">Origins, evolution, and vector/host coadaptations within the genus Flavivirus</span>. 
                     <span tagx="source">Adv Virus Res</span>
                     <span tagx="volume">59</span>: 
                     <span tagx="fpage">277</span>–
                     <span tagx="lpage">314</span>
                     <span class="pub-id'">[14696332]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Gaunt1"></a>
                  <span class="label">24</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gaunt</span>
                        <span tagx="given-names">MW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sall</span>
                        <span tagx="given-names">AA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">de Lamballerie</span>
                        <span tagx="given-names">X</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Falconar</span>
                        <span tagx="given-names">AK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Dzhivanian</span>
                        <span tagx="given-names">TI</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2001</span>) 
                     <span tagx="article-title">Phylogenetic relationships of flaviviruses correlate with their epidemiology, disease
                        association and biogeography
                     </span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">82</span>: 
                     <span tagx="fpage">1867</span>–
                     <span tagx="lpage">1876</span>
                     <span class="pub-id'">[11457992]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Zanotto1"></a>
                  <span class="label">25</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Zanotto</span>
                        <span tagx="given-names">PM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gould</span>
                        <span tagx="given-names">EA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gao</span>
                        <span tagx="given-names">GF</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Harvey</span>
                        <span tagx="given-names">PH</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">EC</span>,
                     </span> (
                     <span tagx="year">1996</span>) 
                     <span tagx="article-title">Population dynamics of flaviviruses revealed by molecular phylogenies</span>. 
                     <span tagx="source">Proc Natl Acad Sci U S A</span>
                     <span tagx="volume">93</span>: 
                     <span tagx="fpage">548</span>–
                     <span tagx="lpage">553</span>
                     <span class="pub-id'">[8570593]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Haddow1"></a>
                  <span class="label">26</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Haddow</span>
                        <span tagx="given-names">AD</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Schuh</span>
                        <span tagx="given-names">AJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Yasuda</span>
                        <span tagx="given-names">CY</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kasper</span>
                        <span tagx="given-names">MR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Heang</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">Genetic Characterization of Zika Virus Strains: Geographic Expansion of the Asian
                        Lineage
                     </span>. 
                     <span tagx="source">PLoS Negl Trop Dis</span>
                     <span tagx="volume">6</span>: 
                     <span tagx="fpage">e1477</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1371/journal.pntd.0001477</div>
                     </div>
                     <span class="pub-id'">[22389730]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Digoutte1"></a>
                  <span class="label">27</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Digoutte</span>
                        <span tagx="given-names">JP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Calvo-Wilson</span>
                        <span tagx="given-names">MA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Mondo</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Traore-Lamizana</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Adam</span>
                        <span tagx="given-names">F</span>,
                     </span> (
                     <span tagx="year">1992</span>) 
                     <span tagx="article-title">Continuous cell lines and immune ascitic fluid pools in arbovirus detection</span>. 
                     <span tagx="source">Res Virol</span>
                     <span tagx="volume">143</span>: 
                     <span tagx="fpage">417</span>–
                     <span tagx="lpage">422</span>
                     <span class="pub-id'">[1297177]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Faye1"></a>
                  <span class="label">28</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Faye</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Faye</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Dupressoir</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Weidmann</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ndiaye</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2008</span>) 
                     <span tagx="article-title">One-step RT-PCR for detection of Zika virus</span>. 
                     <span tagx="source">J Clin Virol</span>
                     <span tagx="volume">43</span>: 
                     <span tagx="fpage">96</span>–
                     <span tagx="lpage">101</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1016/j.jcv.2008.05.005</div>
                     </div>
                     <span class="pub-id'">[18674965]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Edgar1"></a>
                  <span class="label">29</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Edgar</span>
                        <span tagx="given-names">RC</span>,
                     </span> (
                     <span tagx="year">2004</span>) 
                     <span tagx="article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. 
                     <span tagx="source">Nucleic Acids Res</span>
                     <span tagx="volume">32</span>: 
                     <span tagx="fpage">1792</span>–
                     <span tagx="lpage">1797</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1093/nar/gkh340</div>
                     </div>
                     <span class="pub-id'">[15034147]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Gouy1"></a>
                  <span class="label">30</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gouy</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Guindon</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gascuel</span>
                        <span tagx="given-names">O</span>,
                     </span> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">SeaView version 4: A multiplatform graphical user interface for sequence alignment
                        and phylogenetic tree building
                     </span>. 
                     <span tagx="source">Mol Biol Evol</span>
                     <span tagx="volume">27</span>: 
                     <span tagx="fpage">221</span>–
                     <span tagx="lpage">224</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1093/molbev/msp259</div>
                     </div>
                     <span class="pub-id'">[19854763]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Martin1"></a>
                  <span class="label">31</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Martin</span>
                        <span tagx="given-names">DP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lemey</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lott</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Moulton</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Posada</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">RDP3: a flexible and fast computer program for analyzing recombination</span>. 
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="volume">26</span>: 
                     <span tagx="fpage">2462</span>–
                     <span tagx="lpage">2463</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1093/bioinformatics/btq467</div>
                     </div>
                     <span class="pub-id'">[20798170]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Martin2"></a>
                  <span class="label">32</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Martin</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rybicki</span>
                        <span tagx="given-names">E</span>,
                     </span> (
                     <span tagx="year">2000</span>) 
                     <span tagx="article-title">RDP: detection of recombination amongst aligned sequences</span>. 
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="volume">16</span>: 
                     <span tagx="fpage">562</span>–
                     <span tagx="lpage">563</span>
                     <span class="pub-id'">[10980155]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Padidam1"></a>
                  <span class="label">33</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Padidam</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sawyer</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Fauquet</span>
                        <span tagx="given-names">CM</span>,
                     </span> (
                     <span tagx="year">1999</span>) 
                     <span tagx="article-title">Possible emergence of new geminiviruses by frequent recombination</span>. 
                     <span tagx="source">Virology</span>
                     <span tagx="volume">265</span>: 
                     <span tagx="fpage">218</span>–
                     <span tagx="lpage">225</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1006/viro.1999.0056</div>
                     </div>
                     <span class="pub-id'">[10600594]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Posada1"></a>
                  <span class="label">34</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Posada</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Crandall</span>
                        <span tagx="given-names">KA</span>,
                     </span> (
                     <span tagx="year">2001</span>) 
                     <span tagx="article-title">Evaluation of methods for detecting recombination from DNA sequences: computer simulations</span>. 
                     <span tagx="source">Proc Natl Acad Sci U S A</span>
                     <span tagx="volume">98</span>: 
                     <span tagx="fpage">13757</span>–
                     <span tagx="lpage">13762</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1073/pnas.241370698</div>
                     </div>
                     <span class="pub-id'">[11717435]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Smith1"></a>
                  <span class="label">35</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Smith</span>
                        <span tagx="given-names">JM</span>,
                     </span> (
                     <span tagx="year">1992</span>) 
                     <span tagx="article-title">Analyzing the mosaic structure of genes</span>. 
                     <span tagx="source">J Mol Evol</span>
                     <span tagx="volume">34</span>: 
                     <span tagx="fpage">126</span>–
                     <span tagx="lpage">129</span>
                     <span class="pub-id'">[1556748]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Martin3"></a>
                  <span class="label">36</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Martin</span>
                        <span tagx="given-names">DP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Posada</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Crandall</span>
                        <span tagx="given-names">KA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Williamson</span>
                        <span tagx="given-names">C</span>,
                     </span> (
                     <span tagx="year">2005</span>) 
                     <span tagx="article-title">A modified bootscan algorithm for automated identification of recombinant sequences
                        and recombination breakpoints
                     </span>. 
                     <span tagx="source">AIDS Res Hum Retroviruses</span>
                     <span tagx="volume">21</span>: 
                     <span tagx="fpage">98</span>–
                     <span tagx="lpage">102</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1089/aid.2005.21.98</div>
                     </div>
                     <span class="pub-id'">[15665649]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Gibbs1"></a>
                  <span class="label">37</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gibbs</span>
                        <span tagx="given-names">MJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Armstrong</span>
                        <span tagx="given-names">JS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gibbs</span>
                        <span tagx="given-names">AJ</span>,
                     </span> (
                     <span tagx="year">2000</span>) 
                     <span tagx="article-title">Sister-scanning: a Monte Carlo procedure for assessing signals in recombinant sequences</span>. 
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="volume">16</span>: 
                     <span tagx="fpage">573</span>–
                     <span tagx="lpage">582</span>
                     <span class="pub-id'">[11038328]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Boni1"></a>
                  <span class="label">38</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Boni</span>
                        <span tagx="given-names">MF</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Posada</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Feldman</span>
                        <span tagx="given-names">MW</span>,
                     </span> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">An Exact Nonparametric Method for Inferring Mosaic Structure in Sequence Triplets</span>. 
                     <span tagx="source">Genetics</span>
                     <span tagx="volume">176</span>: 
                     <span tagx="fpage">1035</span>–
                     <span tagx="lpage">1047</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1534/genetics.106.068874</div>
                     </div>
                     <span class="pub-id'">[17409078]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Bland1"></a>
                  <span class="label">39</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Bland</span>
                        <span tagx="given-names">JM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Altman</span>
                        <span tagx="given-names">DG</span>,
                     </span> (
                     <span tagx="year">1995</span>) 
                     <span tagx="article-title">Multiple significance tests: the Bonferroni method</span>. 
                     <span tagx="source">BMJ</span>
                     <span tagx="volume">310</span>: 
                     <span tagx="fpage">170</span>
                     <span class="pub-id'">[7833759]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Westesson1"></a>
                  <span class="label">40</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Westesson</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">I</span>,
                     </span> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Accurate detection of recombinant breakpoints in whole-genome alignments</span>. 
                     <span tagx="source">PLoS Comput Biol</span>
                     <span tagx="volume">5</span>: 
                     <span tagx="fpage">e1000318</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1371/journal.pcbi.1000318</div>
                     </div>
                     <span class="pub-id'">[19300487]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Nagarajan1"></a>
                  <span class="label">41</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Nagarajan</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kingsford</span>
                        <span tagx="given-names">C</span>,
                     </span> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">GiRaF: robust, computational identification of influenza reassortments via graph mining</span>. 
                     <span tagx="source">Nucleic Acids Res</span>
                     <span tagx="volume">39</span>: 
                     <span tagx="fpage">e34</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1093/nar/gkq1232</div>
                     </div>
                     <span class="pub-id'">[21177643]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Ronquist1"></a>
                  <span class="label">42</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Ronquist</span>
                        <span tagx="given-names">F</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Teslenko</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">van der Mark</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ayres</span>
                        <span tagx="given-names">DL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Darling</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2012</span>) 
                     <span tagx="article-title">MrBayes 3.2: Efficient Bayesian Phylogenetic Inference and Model Choice Across a Large
                        Model Space
                     </span>. 
                     <span tagx="source">Syst Biol</span>
                     <span tagx="volume">61</span>: 
                     <span tagx="fpage">539</span>–
                     <span tagx="lpage">42</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1093/sysbio/sys029</div>
                     </div>
                     <span class="pub-id'">[22357727]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Strimmer1"></a>
                  <span class="label">43</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Strimmer</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">von Haeseler</span>
                        <span tagx="given-names">A</span>,
                     </span> (
                     <span tagx="year">1997</span>) 
                     <span tagx="article-title">Likelihood-mapping: a simple method to visualize phylogenetic content of a sequence
                        alignment
                     </span>. 
                     <span tagx="source">Proc Natl Acad Sci U S A</span>
                     <span tagx="volume">94</span>: 
                     <span tagx="fpage">6815</span>–
                     <span tagx="lpage">6819</span>
                     <span class="pub-id'">[9192648]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-SchmidtH1"></a>
                  <span class="label">44</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Schmidt H</span>
                        <span tagx="given-names">a</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Strimmer</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vingron</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">von Haeseler</span>
                        <span tagx="given-names">A</span>,
                     </span> (
                     <span tagx="year">2002</span>) 
                     <span tagx="article-title">TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel
                        computing
                     </span>. 
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="volume">18</span>: 
                     <span tagx="fpage">502</span>–
                     <span tagx="lpage">504</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1093/bioinformatics/18.3.502</div>
                     </div>
                     <span class="pub-id'">[11934758]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Campbell1"></a>
                  <span class="label">45</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Campbell</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Legendre</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lapointe</span>
                        <span tagx="given-names">F-J</span>,
                     </span> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">The performance of the Congruence Among Distance Matrices (CADM) test in phylogenetic
                        analysis
                     </span>. 
                     <span tagx="source">BMC Evol Biol</span>
                     <span tagx="volume">11</span>: 
                     <span tagx="fpage">64</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1186/1471-2148-11-64</div>
                     </div>
                     <span class="pub-id'">[21388552]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Zwickl1"></a>
                  <span class="label">46</span>
                  <div tagxxx="mixed-citation">Zwickl D (2006) Genetic algorithm approaches for the phylogenetic analysis of large
                     biological sequence datasets under the maximum likelihood criterion. The University
                     of Texas at Austin.
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Sukumaran1"></a>
                  <span class="label">47</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Sukumaran</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Holder</span>
                        <span tagx="given-names">MT</span>,
                     </span> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">DendroPy: a Python library for phylogenetic computing</span>. 
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="volume">26</span>: 
                     <span tagx="fpage">1569</span>–
                     <span tagx="lpage">1571</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1093/bioinformatics/btq228</div>
                     </div>
                     <span class="pub-id'">[20421198]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Pond1"></a>
                  <span class="label">48</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Pond</span>
                        <span tagx="given-names">SLK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Frost</span>
                        <span tagx="given-names">SDW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Muse S</span>
                        <span tagx="given-names">V</span>,
                     </span> (
                     <span tagx="year">2005</span>) 
                     <span tagx="article-title">HyPhy: hypothesis testing using phylogenies</span>. 
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="volume">21</span>: 
                     <span tagx="fpage">676</span>–
                     <span tagx="lpage">679</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1093/bioinformatics/bti079</div>
                     </div>
                     <span class="pub-id'">[15509596]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Julenius1"></a>
                  <span class="label">49</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Julenius</span>
                        <span tagx="given-names">K</span>,
                     </span> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">NetCGlyc 1.0: prediction of mammalian C-mannosylation sites</span>. 
                     <span tagx="source">Glycobiology</span>
                     <span tagx="volume">17</span>: 
                     <span tagx="fpage">868</span>–
                     <span tagx="lpage">876</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1093/glycob/cwm050</div>
                     </div>
                     <span class="pub-id'">[17494086]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Julenius2"></a>
                  <span class="label">50</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Julenius</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Mølgaard</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gupta</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Brunak</span>
                        <span tagx="given-names">S</span>,
                     </span> (
                     <span tagx="year">2005</span>) 
                     <span tagx="article-title">Prediction, conservation analysis, and structural characterization of mammalian mucin-type
                        O-glycosylation sites
                     </span>. 
                     <span tagx="source">Glycobiology</span>
                     <span tagx="volume">15</span>: 
                     <span tagx="fpage">153</span>–
                     <span tagx="lpage">164</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1093/glycob/cwh151</div>
                     </div>
                     <span class="pub-id'">[15385431]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Gupta1"></a>
                  <span class="label">51</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gupta</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Brunak</span>
                        <span tagx="given-names">S</span>,
                     </span> (
                     <span tagx="year">2002</span>) 
                     <span tagx="article-title">Prediction of glycosylation across the human proteome and the correlation to protein
                        function
                     </span>. 
                     <span tagx="source">Pac Symp Biocomput</span>
                     <span tagx="fpage">310</span>–
                     <span tagx="lpage">322</span>
                     <span class="pub-id'">[11928486]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Blom1"></a>
                  <span class="label">52</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Blom</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sicheritz-Pontén</span>
                        <span tagx="given-names">T</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gupta</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gammeltoft</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Brunak</span>
                        <span tagx="given-names">S</span>,
                     </span> (
                     <span tagx="year">2004</span>) 
                     <span tagx="article-title">Prediction of post-translational glycosylation and phosphorylation of proteins from
                        the amino acid sequence
                     </span>. 
                     <span tagx="source">Proteomics</span>
                     <span tagx="volume">4</span>: 
                     <span tagx="fpage">1633</span>–
                     <span tagx="lpage">1649</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1002/pmic.200300771</div>
                     </div>
                     <span class="pub-id'">[15174133]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Sali1"></a>
                  <span class="label">53</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Sali</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Blundell</span>
                        <span tagx="given-names">TL</span>,
                     </span> (
                     <span tagx="year">1993</span>) 
                     <span tagx="article-title">Comparative protein modelling by satisfaction of spatial restraints</span>. 
                     <span tagx="source">J Mol Biol</span>
                     <span tagx="volume">234</span>: 
                     <span tagx="fpage">779</span>–
                     <span tagx="lpage">815</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1006/jmbi.1993.1626</div>
                     </div>
                     <span class="pub-id'">[8254673]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Laskowski1"></a>
                  <span class="label">54</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Laskowski</span>
                        <span tagx="given-names">RA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">MacArthur</span>
                        <span tagx="given-names">MW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Moss</span>
                        <span tagx="given-names">DS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Thornton</span>
                        <span tagx="given-names">JM</span>,
                     </span> (
                     <span tagx="year">1993</span>) 
                     <span tagx="article-title">PROCHECK: a program to check the stereochemical quality of protein structures</span>. 
                     <span tagx="source">J Appl Crystallogr</span>
                     <span tagx="volume">26</span>: 
                     <span tagx="fpage">283</span>–
                     <span tagx="lpage">291</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1107/S0021889892009944</div>
                     </div>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Drummond1"></a>
                  <span class="label">55</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Drummond</span>
                        <span tagx="given-names">AJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rambaut</span>
                        <span tagx="given-names">A</span>,
                     </span> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">BEAST: Bayesian evolutionary analysis by sampling trees</span>. 
                     <span tagx="source">BMC Evol Biol</span>
                     <span tagx="volume">7</span>: 
                     <span tagx="fpage">214</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1186/1471-2148-7-214</div>
                     </div>
                     <span class="pub-id'">[17996036]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Drummond2"></a>
                  <span class="label">56</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Drummond</span>
                        <span tagx="given-names">AJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ho</span>
                        <span tagx="given-names">SYW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Phillips</span>
                        <span tagx="given-names">MJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rambaut</span>
                        <span tagx="given-names">A</span>,
                     </span> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">Relaxed phylogenetics and dating with confidence</span>. 
                     <span tagx="source">PLoS Biol</span>
                     <span tagx="volume">4</span>: 
                     <span tagx="fpage">e88</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1371/journal.pbio.0040088</div>
                     </div>
                     <span class="pub-id'">[16683862]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Lemey1"></a>
                  <span class="label">57</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Lemey</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rambaut</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Drummond</span>
                        <span tagx="given-names">AJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Suchard</span>
                        <span tagx="given-names">MA</span>,
                     </span> (
                     <span tagx="year">2009</span>) 
                     <span tagx="article-title">Bayesian phylogeography finds its roots</span>. 
                     <span tagx="source">PLoS Comput Biol</span>
                     <span tagx="volume">5</span>: 
                     <span tagx="fpage">e1000520</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1371/journal.pcbi.1000520</div>
                     </div>
                     <span class="pub-id'">[19779555]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Bielejec1"></a>
                  <span class="label">58</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Bielejec</span>
                        <span tagx="given-names">F</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rambaut</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Suchard</span>
                        <span tagx="given-names">MA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lemey</span>
                        <span tagx="given-names">P</span>,
                     </span> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">SPREAD: spatial phylogenetic reconstruction of evolutionary dynamics</span>. 
                     <span tagx="source">Bioinformatics</span>
                     <span tagx="volume">27</span>: 
                     <span tagx="fpage">2910</span>–
                     <span tagx="lpage">2912</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1093/bioinformatics/btr481</div>
                     </div>
                     <span class="pub-id'">[21911333]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Parker1"></a>
                  <span class="label">59</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Parker</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rambaut</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Pybus</span>
                        <span tagx="given-names">OG</span>,
                     </span> (
                     <span tagx="year">2008</span>) 
                     <span tagx="article-title">Correlating viral phenotypes with phylogeny: accounting for phylogenetic uncertainty</span>. 
                     <span tagx="source">Infect Genet Evol</span>
                     <span tagx="volume">8</span>: 
                     <span tagx="fpage">239</span>–
                     <span tagx="lpage">246</span>
                     <span class="pub-id'">[17921073]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Barker1"></a>
                  <span class="label">60</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Barker</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Pagel</span>
                        <span tagx="given-names">M</span>,
                     </span> (
                     <span tagx="year">2005</span>) 
                     <span tagx="article-title">Predicting functional gene links from phylogenetic-statistical analyses of whole genomes</span>. 
                     <span tagx="source">PLoS Comput Biol</span>
                     <span tagx="volume">1</span>: 
                     <span tagx="fpage">e3</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1371/journal.pcbi.0010003</div>
                     </div>
                     <span class="pub-id'">[16103904]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Suchard1"></a>
                  <span class="label">61</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Suchard</span>
                        <span tagx="given-names">MA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Weiss</span>
                        <span tagx="given-names">RE</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sinsheimer</span>
                        <span tagx="given-names">JS</span>,
                     </span> (
                     <span tagx="year">2001</span>) 
                     <span tagx="article-title">Bayesian Selection of Continuous-Time Markov Chain Evolutionary Models</span>. 
                     <span tagx="source">Mol Biol Evol</span>
                     <span tagx="volume">18</span>: 
                     <span tagx="fpage">1001</span>–
                     <span tagx="lpage">1013</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1093/oxfordjournals.molbev.a003872</div>
                     </div>
                     <span class="pub-id'">[11371589]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Posada2"></a>
                  <span class="label">62</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Posada</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Crandall K</span>
                        <span tagx="given-names">a</span>,
                     </span> (
                     <span tagx="year">2002</span>) 
                     <span tagx="article-title">The effect of recombination on the accuracy of phylogeny estimation</span>. 
                     <span tagx="source">J Mol Evol</span>
                     <span tagx="volume">54</span>: 
                     <span tagx="fpage">396</span>–
                     <span tagx="lpage">402</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1007/s00239-001-0034-9</div>
                     </div>
                     <span class="pub-id'">[11847565]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Schierup1"></a>
                  <span class="label">63</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Schierup</span>
                        <span tagx="given-names">MH</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hein</span>
                        <span tagx="given-names">J</span>,
                     </span> (
                     <span tagx="year">2000</span>) 
                     <span tagx="article-title">Consequences of recombination on traditional phylogenetic analysis</span>. 
                     <span tagx="source">Genetics</span>
                     <span tagx="volume">156</span>: 
                     <span tagx="fpage">879</span>–
                     <span tagx="lpage">891</span>
                     <span class="pub-id'">[11014833]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Wiens1"></a>
                  <span class="label">64</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Wiens</span>
                        <span tagx="given-names">JJ</span>,
                     </span> (
                     <span tagx="year">2003</span>) 
                     <span tagx="article-title">Missing Data, Incomplete Taxa, and Phylogenetic Accuracy</span>. 
                     <span tagx="source">Syst Biol</span>
                     <span tagx="volume">52</span>: 
                     <span tagx="fpage">528</span>–
                     <span tagx="lpage">538</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1080/10635150390218330</div>
                     </div>
                     <span class="pub-id'">[12857643]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Wiens2"></a>
                  <span class="label">65</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Wiens</span>
                        <span tagx="given-names">JJ</span>,
                     </span> (
                     <span tagx="year">2006</span>) 
                     <span tagx="article-title">Missing data and the design of phylogenetic analyses</span>. 
                     <span tagx="source">J Biomed Inform</span>
                     <span tagx="volume">39</span>: 
                     <span tagx="fpage">34</span>–
                     <span tagx="lpage">42</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1016/j.jbi.2005.04.001</div>
                     </div>
                     <span class="pub-id'">[15922672]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-SimonLoriere1"></a>
                  <span class="label">66</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Simon-Loriere</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">EC</span>,
                     </span> (
                     <span tagx="year">2011</span>) 
                     <span tagx="article-title">Why do RNA viruses recombine?</span>
                     <span tagx="source">Nat Rev Microbiol</span>
                     <span tagx="volume">9</span>: 
                     <span tagx="fpage">617</span>–
                     <span tagx="lpage">626</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1038/nrmicro2614</div>
                     </div>
                     <span class="pub-id'">[21725337]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Hanada1"></a>
                  <span class="label">67</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Hanada</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Suzuki</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gojobori</span>
                        <span tagx="given-names">T</span>,
                     </span> (
                     <span tagx="year">2004</span>) 
                     <span tagx="article-title">A large variation in the rates of synonymous substitution for RNA viruses and its
                        relationship to a diversity of viral infection and transmission modes
                     </span>. 
                     <span tagx="source">Mol Biol Evol</span>
                     <span tagx="volume">21</span>: 
                     <span tagx="fpage">1074</span>–
                     <span tagx="lpage">1080</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1093/molbev/msh109</div>
                     </div>
                     <span class="pub-id'">[15014142]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Holmes1"></a>
                  <span class="label">68</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">EC</span>,
                     </span> (
                     <span tagx="year">2003</span>) 
                     <span tagx="article-title">Patterns of intra- and interhost nonsynonymous variation reveal strong purifying selection
                        in dengue virus
                     </span>. 
                     <span tagx="source">J Virol</span>
                     <span tagx="volume">77</span>: 
                     <span tagx="fpage">11296</span>–
                     <span tagx="lpage">11298</span>
                     <span class="pub-id'">[14512579]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Twiddy1"></a>
                  <span class="label">69</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Twiddy</span>
                        <span tagx="given-names">SS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">EC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rambaut</span>
                        <span tagx="given-names">a</span>,
                     </span> (
                     <span tagx="year">2003</span>) 
                     <span tagx="article-title">Inferring the Rate and Time-Scale of Dengue Virus Evolution</span>. 
                     <span tagx="source">Mol Biol Evol</span>
                     <span tagx="volume">20</span>: 
                     <span tagx="fpage">122</span>–
                     <span tagx="lpage">129</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1093/molbev/msg010</div>
                     </div>
                     <span class="pub-id'">[12519914]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Kuno3"></a>
                  <span class="label">70</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Kuno</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chang</span>
                        <span tagx="given-names">G-JJ</span>,
                     </span> (
                     <span tagx="year">2005</span>) 
                     <span tagx="article-title">Biological transmission of arboviruses: reexamination of and new insights into components,
                        mechanisms, and unique traits as well as their evolutionary trends
                     </span>. 
                     <span tagx="source">Clin Microbiol Rev</span>
                     <span tagx="volume">18</span>: 
                     <span tagx="fpage">608</span>–
                     <span tagx="lpage">637</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1128/CMR.18.4.608-637.2005</div>
                     </div>
                     <span class="pub-id'">[16223950]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Diallo1"></a>
                  <span class="label">71</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Diallo</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Thonnon</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Traore-Lamizana</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Fontenille</span>
                        <span tagx="given-names">D</span>,
                     </span> (
                     <span tagx="year">1999</span>) 
                     <span tagx="article-title">Vectors of Chikungunya virus in Senegal: current data and transmission cycles</span>. 
                     <span tagx="source">Am J Trop Med Hyg</span>
                     <span tagx="volume">60</span>: 
                     <span tagx="fpage">281</span>–
                     <span tagx="lpage">286</span>
                     <span class="pub-id'">[10072152]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Hanna1"></a>
                  <span class="label">72</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Hanna</span>
                        <span tagx="given-names">SL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Pierson</span>
                        <span tagx="given-names">TC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sanchez</span>
                        <span tagx="given-names">MD</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ahmed</span>
                        <span tagx="given-names">AA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Murtadha</span>
                        <span tagx="given-names">MM</span>,
                     </span>, 
                     <i>et al.</i> (
                     <span tagx="year">2005</span>) 
                     <span tagx="article-title">N-linked glycosylation of west nile virus envelope proteins influences particle assembly
                        and infectivity
                     </span>. 
                     <span tagx="source">J Virol</span>
                     <span tagx="volume">79</span>: 
                     <span tagx="fpage">13262</span>–
                     <span tagx="lpage">13274</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1128/JVI.79.21.13262-13274.2005</div>
                     </div>
                     <span class="pub-id'">[16227249]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Lee1"></a>
                  <span class="label">73</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Lee</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Leang</span>
                        <span tagx="given-names">SK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Davidson</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lobigs</span>
                        <span tagx="given-names">M</span>,
                     </span> (
                     <span tagx="year">2010</span>) 
                     <span tagx="article-title">Both E protein glycans adversely affect dengue virus infectivity but are beneficial
                        for virion release
                     </span>. 
                     <span tagx="source">J Virol</span>
                     <span tagx="volume">84</span>: 
                     <span tagx="fpage">5171</span>–
                     <span tagx="lpage">5180</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1128/JVI.01900-09</div>
                     </div>
                     <span class="pub-id'">[20219924]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Mondotte1"></a>
                  <span class="label">74</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Mondotte</span>
                        <span tagx="given-names">JA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lozach</span>
                        <span tagx="given-names">P-Y</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Amara</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gamarnik A</span>
                        <span tagx="given-names">V</span>,
                     </span> (
                     <span tagx="year">2007</span>) 
                     <span tagx="article-title">Essential role of dengue virus envelope protein N glycosylation at asparagine-67 during
                        viral propagation
                     </span>. 
                     <span tagx="source">J Virol</span>
                     <span tagx="volume">81</span>: 
                     <span tagx="fpage">7136</span>–
                     <span tagx="lpage">7148</span>
                     <div tagxxx="comment">doi:
                        <div tagxxx="ext-link">10.1128/JVI.00116-07</div>
                     </div>
                     <span class="pub-id'">[17459925]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pntd.0002636-Chambers2"></a>
                  <span class="label">75</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Chambers</span>
                        <span tagx="given-names">TJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Halevy</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nestorowicz</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rice</span>
                        <span tagx="given-names">CM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lustig</span>
                        <span tagx="given-names">S</span>,
                     </span> (
                     <span tagx="year">1998</span>) 
                     <span tagx="article-title">West Nile virus envelope proteins: nucleotide sequence analysis of strains differing
                        in mouse neuroinvasiveness
                     </span>. 
                     <span tagx="source">J Gen Virol</span>
                     <span tagx="volume">79</span> (
                     <div tagxxx="issue">Pt 10</div>) 
                     <span tagx="fpage">2375</span>–
                     <span tagx="lpage">2380</span>
                     <span class="pub-id'">[9780042]</span>
                  </div>
               </li>
            </ul>
         </div>
      </div>
   </div>
</html>