<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml">
   <div xmlns="" tagx="article">
      <head>
         <style type="text/css"> table, tr, td {border : thin;}</style>
         <span class="citation_publisher">Public Library of Science</span>
         <h2 class="citation-title">Recombinase Polymerase Amplification Assay for Rapid Diagnostics of Dengue Infection</h2>
         <span class="doi">10.1371/journal.pone.0129682</span>
         <div class="contrib-group">contrib: 
            	  CONT
            	  
            <span class="citation_author">Ahmed Abd El Wahed</span>
            <span class="citation_author_institution_ref">1</span>
            <span class="citation_author">Pranav Patel</span>
            <span class="citation_author_institution_ref">3</span>
            <span class="citation_author">Oumar Faye</span>
            <span class="citation_author_institution_ref">4</span>
            <span class="citation_author">Sasikanya Thaloengsok</span>
            <span class="citation_author_institution_ref">5</span>
            <span class="citation_author">Doris Heidenreich</span>
            <span class="citation_author_institution_ref">6</span>
            <span class="citation_author">Ponpan Matangkasombut</span>
            <span class="citation_author_institution_ref">5</span>
            <span class="citation_author">Khajohnpong Manopwisedjaroen</span>
            <span class="citation_author_institution_ref">5</span>
            <span class="citation_author">Anavaj Sakuntabhai</span>
            <span class="citation_author_institution_ref">8</span>
            <span class="citation_author">Amadou A. Sall</span>
            <span class="citation_author_institution_ref">4</span>
            <span class="citation_author">Frank T. Hufert</span>
            <span class="citation_author_institution_ref">9</span>
            <span class="citation_author">Manfred Weidmann</span>
            <span class="citation_author_institution_ref">10</span>
            	  CONT
            	  
            <div tagxxx="contrib">
               <span class="name'">
                  <span tagx="surname">Lin</span>
                  <span tagx="given-names">Baochuan</span>,
               </span>
               <div tagxxx="role">Academic Editor</div>
               <span ref-type="aff" rid="edit1"></span>
            </div>
         </div>
         <span class="email">[m.w.weidmann@stir.ac.uk]</span>
         <span class="conflict">[Competing Interests: The authors have declared that no competing interests exist.]</span>
         <span class="contribs">[Conceived and designed the experiments: AAEW PP AAS FTH MW. Performed the experiments:
            AAEW PP OF ST DH PM KM AS AAS FTH MW. Analyzed the data: AAEW PP OF ST DH PM KM AS
            AAS FTH MW. Contributed reagents/materials/analysis tools: PP OF PM AS AAS. Wrote
            the paper: AAEW MW. Critical revision of the manuscript: AAEW PP OF ST DH PM KM AS
            AAS FTH MW. Development of the DENV RPA assays: AAEW PP DH MW. Field trial in Senegal:
            AAEW PP OF AAS MW. Field trial in Thailand: AAEW ST PM KM AS FTH MW.]
         </span>
         <span class="citation_pubdate">[2015-6-15]</span>
         <span class="citation_copyright">[Copyright 2015: Abd El Wahed et al]</span>
         <span class="citation_licence">[Licence This is an open access article distributed under the terms of the Creative
            Commons Attribution License, which permits unrestricted use, distribution, and reproduction
            in any medium, provided the original author and source are credited]
         </span>
         <span class="citation_funding">[This study was funded by a grant from the Deutsche Gesellschaft für Internationale
            Zusammenarbeit (GIZ) contract no: VN 811-40270. http://www.giz.de. The Bangkok cohort
            study was funded by the Office of the Higher Education Commission and Mahidol University
            under the National Research Universities Initiative, and European Commission Seventh
            Framework Programme [FP7/2007-2013] for the DENFREE project under Grant Agreement
            number 282 378. http://cordis.europa.eu/fp7/health/. The funders had no role in design
            of the study, data collection and analysis, decision to publish, or preparation of
            the manuscript.]
         </span>
         <span class="citation_pagecount">[]</span>
         <span class="citation_data">[Data AvailabilityAll relevant data are within the paper and its Supporting Information
            files.]
         </span>
         <div tagxxx="notes">
            <div tagxxx="title">Data Availability</div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">All relevant data are within the paper and its Supporting Information files.</p>
         </div>
      </head>
      <div id="abstract" tag="abstract">
         <h2>Abstract</h2>
         <div id="sec001">
            <h2>Background</h2>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Over 2.5 billion people are exposed to the risk of contracting dengue fever (DF).
               Early diagnosis of DF helps to diminish its burden on public health. Real-time reverse
               transcription polymerase amplification assays (RT-PCR) are the standard method for
               molecular detection of the dengue virus (DENV). Real-time RT-PCR analysis is not suitable
               for on-site screening since mobile devices are large, expensive, and complex. In this
               study, two RT-recombinase polymerase amplification (RT-RPA) assays were developed
               to detect DENV1-4.
            </p>
         </div>
         <div id="sec002">
            <h2>Methodology/Principal Findings</h2>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Using two quantitative RNA molecular standards, the analytical sensitivity of a RT-RPA
               targeting the 3´non-translated region of DENV1-4 was found to range from 14 (DENV4)
               to 241 (DENV1-3) RNA molecules detected. The assay was specific and did not cross
               detect other Flaviviruses. The RT-RPA assay was tested in a mobile laboratory combining
               magnetic-bead based total nucleic acid extraction and a portable detection device
               in Kedougou (Senegal) and in Bangkok (Thailand). In Kedougou, the RT-RPA was operated
               at an ambient temperature of 38°C with auxiliary electricity tapped from a motor vehicle
               and yielded a clinical sensitivity and specificity of 98% (n=31) and 100% (n=23),
               respectively. While in the field trial in Bangkok, the clinical sensitivity and specificity
               were 72% (n=90) and 100%(n=41), respectively.
            </p>
         </div>
         <div id="sec003">
            <h2>Conclusions/Significance</h2>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">During the first 5 days of infection, the developed DENV1-4 RT-RPA assays constitute
               a suitable accurate and rapid assay for DENV diagnosis. Moreover, the use of a portable
               fluorescence-reading device broadens its application potential to the point-of-care
               for outbreak investigations.
            </p>
         </div>
      </div>
      <div id="sec004">
         <h2>Introduction</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Dengue virus (DENV) is a mosquito-transmitted virus that causes mild ((dengue fever
            (DF)) to severe disease ((dengue hemorrhagic fever (DHF) and dengue shock syndrome
            (DSS)) in humans [
            <span ref-type="bibr" rid="pone.0129682.ref001">
               <sup>
                  <a href="#pone.0129682.ref001">1</a>
               </sup>
            </span>]. DENV consists of four serotypes (DENV1-4) each comprising multiple genotypes and
            belongs to the genus 
            <i>Flavivirus</i> of the family 
            <i>Flaviviridae</i> [
            <span ref-type="bibr" rid="pone.0129682.ref002">
               <sup>
                  <a href="#pone.0129682.ref002">2</a>
               </sup>
            </span>].
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Before 1970 DENV outbreaks were reported in nine countries but has since expanded
            to more than 100 countries, with an estimated 96 million apparent and 294 million
            unapparent cases in 2010 [
            <span ref-type="bibr" rid="pone.0129682.ref003">
               <sup>
                  <a href="#pone.0129682.ref003">3</a>
               </sup>
            </span>]. Two and a half billion people are at risk of infection worldwide [
            <span ref-type="bibr" rid="pone.0129682.ref004">
               <sup>
                  <a href="#pone.0129682.ref004">4</a>
               </sup>
            </span>]. Since specific treatment or a vaccine are currently not available [
            <span ref-type="bibr" rid="pone.0129682.ref005">
               <sup>
                  <a href="#pone.0129682.ref005">5</a>
               </sup>
            </span>], early detection plays a key role for initiation of control and preventive measures
            in dengue endemic regions such as mosquito control and social mobilization [
            <span ref-type="bibr" rid="pone.0129682.ref006">
               <sup>
                  <a href="#pone.0129682.ref006">6</a>
               </sup>
            </span>].
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">DENV is detectable in blood up to 5–7 days after the onsets of symptoms. From the
            5
            <sup>th</sup> day, diagnosis depends mainly on detection of specific IgM and IgG antibodies using
            ELISA methods [
            <span ref-type="bibr" rid="pone.0129682.ref007">
               <sup>
                  <a href="#pone.0129682.ref007">7</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pone.0129682.ref008">
               <sup>
                  <a href="#pone.0129682.ref008">8</a>
               </sup>
            </span>]. Therefore, both virus and antibody detection are crucial for the identification
            of infected cases. The reference method of DENV diagnosis is virus isolation [
            <span ref-type="bibr" rid="pone.0129682.ref009">
               <sup>
                  <a href="#pone.0129682.ref009">9</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pone.0129682.ref010">
               <sup>
                  <a href="#pone.0129682.ref010">10</a>
               </sup>
            </span>], which takes a week and requires highly equipped laboratories. The most widely used
            methods are antigen detection in ELISA or rapid diagnostic tests (RDTs) formats [
            <span ref-type="bibr" rid="pone.0129682.ref008">
               <sup>
                  <a href="#pone.0129682.ref008">8</a>
               </sup>
            </span>]. Antigen detection is based on the non-structural protein 1 (NS1) of the DENV [
            <span ref-type="bibr" rid="pone.0129682.ref011">
               <sup>
                  <a href="#pone.0129682.ref011">11</a>
               </sup>
            </span>]. RDTs are fast (10–15 minutes) and suitable for point-of-care screening. However,
            RDTs clinical sensitivity (21–99%) and specificity (77%–98%) vary greatly [
            <span ref-type="bibr" rid="pone.0129682.ref007">
               <sup>
                  <a href="#pone.0129682.ref007">7</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pone.0129682.ref008">
               <sup>
                  <a href="#pone.0129682.ref008">8</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pone.0129682.ref012">
               <sup>
                  <a href="#pone.0129682.ref012">12</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pone.0129682.ref013">
               <sup>
                  <a href="#pone.0129682.ref013">13</a>
               </sup>
            </span>]. The gold standard method for molecular detection of DENV is real-time RT-PCR, which
            detects DENV RNA within 60–90 minutes [
            <span ref-type="bibr" rid="pone.0129682.ref014">
               <sup>
                  <a href="#pone.0129682.ref014">14</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pone.0129682.ref016">
               <sup>
                  <a href="#pone.0129682.ref016">16</a>
               </sup>
            </span>]. Real-time RT-PCR is highly sensitive and specific [
            <span ref-type="bibr" rid="pone.0129682.ref014">
               <sup>
                  <a href="#pone.0129682.ref014">14</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pone.0129682.ref016">
               <sup>
                  <a href="#pone.0129682.ref016">16</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pone.0129682.ref018">
               <sup>
                  <a href="#pone.0129682.ref018">18</a>
               </sup>
            </span>]. It requires precautions (carry over prevention systems and physical separation
            of pipetting sites), sophisticated equipment, and is cold chain dependent, which makes
            it difficult to implement at the point-of-need.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Isothermal DNA amplification methods represent an alternative to real-time PCR. There
            is only one development for the application of Nucleic acid sequence based amplification
            (NASBA) [
            <span ref-type="bibr" rid="pone.0129682.ref019">
               <sup>
                  <a href="#pone.0129682.ref019">19</a>
               </sup>
            </span>] but several applications of reverse transcription Loop mediated isothermal amplification
            (RT-LAMP) for the detection and differentiation of DENV serotypes have been described
            [
            <span ref-type="bibr" rid="pone.0129682.ref020">
               <sup>
                  <a href="#pone.0129682.ref020">20</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pone.0129682.ref023">
               <sup>
                  <a href="#pone.0129682.ref023">23</a>
               </sup>
            </span>]. RT-LAMP reactions are performed at 60°C, tests run for more than 30 minutes, and
            results are measured either by turbidity index or by visual qualitative fluorescence
            detection of SYBR Green [
            <span ref-type="bibr" rid="pone.0129682.ref024">
               <sup>
                  <a href="#pone.0129682.ref024">24</a>
               </sup>
            </span>]. In contrast, real-time recombinase polymerase amplification (RPA) amplifies at
            39–42°C and uses a fluorescent exo-probe for detection [
            <span ref-type="bibr" rid="pone.0129682.ref025">
               <sup>
                  <a href="#pone.0129682.ref025">25</a>
               </sup>
            </span>]. The RPA assay is very fast (3–15 minutes) and can be operated on a portable device,
            the tubescanner (19x17.5 cm). In this study, a-point-of-need reverse transcription
            RPA (RT-RPA) assays for the detection of DENV1-4 without differentiation between the
            serotypes were developed and evaluated with samples from Senegal and Thailand. A mobile
            RPA unit was deployed to Kedougou in Senegal and to Bangkok, Thailand.
         </p>
      </div>
      <div id="sec005">
         <h2>Methods</h2>
         <div id="sec006">
            <h3>Ethics Statement</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Human samples tested in Senegal were provided by the WHO collaborating center for
               Abovirus and Viral Hemorrhagic Fever at the Institute Pasteur de Dakar (IPD). The
               IPD has the required ethical approvals from the Senegalese National Health Research
               Council and the signed consent forms from patients. The ethical committee refernce
               number is 2472. Human samples tested during the field trial in Thailand were used
               after an approval of Faculty of Medicine, Vajira Hospital, Bangkok, Thailand Research
               Ethics Committee (Certificate of approval number: COA29/2012) and signed consent forms
               from patients.
            </p>
         </div>
         <div id="sec007">
            <h3>Viruses</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Robert Koch Institute, Berlin, Germany provided the viruses and/or nucleic acids used
               in this study. Viruses are listed in 
               <span ref-type="table" rid="pone.0129682.t001">
                  <a href="#pone.0129682.t001">Table 1</a>
               </span>.
            </p>
            <div id="pone.0129682.t001">
               <div tagxxx="object-id">10.1371/journal.pone.0129682.t001</div>
               <span class="label">Table 1</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">List of viral RNA tested.</div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <colgroup>
                        <col></col>
                        <col></col>
                        <col></col>
                        <col></col>
                     </colgroup>
                     <thead>
                        <tr>
                           <th>Name</th>
                           <th>Strain</th>
                           <th>DENV1-3 RT-RPA</th>
                           <th>DENV4 RT-RPA</th>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>Dengue virus serotype 1</td>
                           <td>VR344 (Thai 1958)</td>
                           <td>+</td>
                           <td>-</td>
                        </tr>
                        <tr>
                           <td>Dengue virus serotype 2</td>
                           <td>VR345 (TH-36)</td>
                           <td>+</td>
                           <td>+</td>
                        </tr>
                        <tr>
                           <td>Dengue virus serotype 3</td>
                           <td>VR216 (H87)</td>
                           <td>+</td>
                           <td>-</td>
                        </tr>
                        <tr>
                           <td>Dengue virus serotype 4</td>
                           <td>VR217 (H241)</td>
                           <td>-</td>
                           <td>+</td>
                        </tr>
                        <tr>
                           <td>Yellow Fever virus</td>
                           <td>17D</td>
                           <td>-</td>
                           <td>-</td>
                        </tr>
                        <tr>
                           <td>Yellow Fever virus</td>
                           <td>Asibi</td>
                           <td>-</td>
                           <td>-</td>
                        </tr>
                        <tr>
                           <td>Tick Borne Encepahlitis virus-FE</td>
                           <td>Far eastern subtype</td>
                           <td>-</td>
                           <td>-</td>
                        </tr>
                        <tr>
                           <td>Chikungunya virus</td>
                           <td>LR 2006</td>
                           <td>-</td>
                           <td>-</td>
                        </tr>
                        <tr>
                           <td>Zika virus</td>
                           <td>MR766</td>
                           <td>-</td>
                           <td>-</td>
                        </tr>
                        <tr>
                           <td>Tick-borne encephalitis virus</td>
                           <td>K23</td>
                           <td>-</td>
                           <td>-</td>
                        </tr>
                        <tr>
                           <td>Japanese encephalitis virus</td>
                           <td>ATCC SA14142</td>
                           <td>-</td>
                           <td>-</td>
                        </tr>
                        <tr>
                           <td>West Nile virus</td>
                           <td>Israel</td>
                           <td>-</td>
                           <td>-</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <table-wrap-foot xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div class="fn" id="t001fn001">
                     <p>DENV1-3 RT-RPA assay detected DENV1-3 but not other viral RNA. DENV4 RT-RPA identified
                        DENV2 and DENV4.
                     </p>
                  </div>
                  <div class="fn" id="t001fn002">
                     <p>+, positive;-, negative.</p>
                  </div>
               </table-wrap-foot>
            </div>
         </div>
         <div id="sec008">
            <h3>Generation of molecular RNA standard</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">According to a previously published protocol, two 
               <i>in vitro</i> transcribed RNA standards were prepared [
               <span ref-type="bibr" rid="pone.0129682.ref026">
                  <sup>
                     <a href="#pone.0129682.ref026">26</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pone.0129682.ref027">
                  <sup>
                     <a href="#pone.0129682.ref027">27</a>
                  </sup>
               </span>]. PCR primers used for the amplification of the target region for ligation into the
               TA cloning vector pCRII are listed in 
               <span ref-type="table" rid="pone.0129682.t002">
                  <a href="#pone.0129682.t002">Table 2</a>
               </span>. DENV1-3 and DEN4 RNA molecular standards cover 242 and 248 bp of the 3’ non-translated
               region (3’NTR) (10462–10703 nt and 10398–10645 nt of GenBank accession numbers AY662691.1
               and GU289913.1), respectively. The RNA standards were tested by using a published
               real-time RT-PCR [
               <span ref-type="bibr" rid="pone.0129682.ref028">
                  <sup>
                     <a href="#pone.0129682.ref028">28</a>
                  </sup>
               </span>]. The Light Cycler 2.0 and the LightCycler 480 RNA Master Hydrolysis Probes kit (Roche,
               Manheim, Germany) were used. The following temperature profile was used: RT step 63°C/3
               minutes, initial activation at 95°C/30 seconds, 45 cycles of 95°C/15 seconds and 60°C/60
               seconds, and a final cooling step of 40°C/30 seconds.
            </p>
            <div id="pone.0129682.t002">
               <div tagxxx="object-id">10.1371/journal.pone.0129682.t002</div>
               <span class="label">Table 2</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Sequence of primers and probes for the construction of molecular standards, real-time
                     RT-PCR and RT-RPA assays.
                  </div>
               </caption>
               <div tagxxx="alternatives">
                  <div tagxxx="graphic"></div>
                  <table xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                     <colgroup>
                        <col></col>
                        <col></col>
                        <col></col>
                     </colgroup>
                     <thead>
                        <tr>
                           <th>Name</th>
                           <th>Sequence 5’-3’</th>
                           <th>Amplicon size bp</th>
                        </tr>
                     </thead>
                     <div tagxxx="tbody">
                        <tr>
                           <td>DENV1-3-STD-UP</td>
                           <td>
                              <div tagxxx="monospace">CTGTACGCACGGTGTAGCAGAC</div>
                           </td>
                           <td>242</td>
                        </tr>
                        <tr>
                           <td>DENV1-3-STD-DP</td>
                           <td>
                              <div tagxxx="monospace">CCTGTTGATTCAACAGCACCATTC</div>
                           </td>
                        </tr>
                        <tr>
                           <td>DENV4-STD-UP</td>
                           <td>
                              <div tagxxx="monospace">CTGTACGCGTGGCATATTG</div>
                           </td>
                           <td>248</td>
                        </tr>
                        <tr>
                           <td>DENV4-STD-DP</td>
                           <td>
                              <div tagxxx="monospace">CCTGTTGGATCAACAACACC</div>
                           </td>
                        </tr>
                        <tr>
                           <td>DENV-PCR-FP</td>
                           <td>
                              <div tagxxx="monospace">AAGGACTAGAGGTTAKAGGAGACCC</div>
                           </td>
                           <td>85</td>
                        </tr>
                        <tr>
                           <td>DENV-PCR-RP</td>
                           <td>
                              <div tagxxx="monospace">CTGHRGAGACAGCAGGATCTCTG</div>
                           </td>
                        </tr>
                        <tr>
                           <td>DENV-PCR-P</td>
                           <td>
                              <div tagxxx="monospace">FAM-AACAGCATATTGACGCTGGGARAGAC-TAMRA</div>
                           </td>
                        </tr>
                        <tr>
                           <td>DENV1-3-RPA-FP13</td>
                           <td>
                              <div tagxxx="monospace">ATTCAACAGCACCATTCCATTTTCTGGCGTTCTGTG</div>
                           </td>
                           <td>97</td>
                        </tr>
                        <tr>
                           <td>DENV1-3-RPA-RP4</td>
                           <td>
                              <div tagxxx="monospace">AACAGCATATTGACGCTGGGAGAGACCAGAGATC</div>
                           </td>
                        </tr>
                        <tr>
                           <td>DENV4-RPA-FP3</td>
                           <td>
                              <div tagxxx="monospace">CATCTTGCGGCGCTCTGTGCCTGGATTGA</div>
                           </td>
                           <td>86</td>
                        </tr>
                        <tr>
                           <td>DENV4-RPA-RP2</td>
                           <td>
                              <div tagxxx="monospace">CACAAAAACAGCATATTGACGCTGGGAAAG</div>
                           </td>
                        </tr>
                        <tr>
                           <td>DENV-RPA-P3</td>
                           <td>
                              <div tagxxx="monospace">ATATTGACGCTGGGAGAGACCAGAGATCCTGC(BHQ1-dT)(THF)(FAM-dT)CTCCTCAGCATCATTC-(Phosphate)</div>
                           </td>
                           <td>N/A</td>
                        </tr>
                     </div>
                  </table>
               </div>
               <table-wrap-foot xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div class="fn" id="t002fn001">
                     <p>Bp, base pair; DENV1-3-STD-UP/DP are for DENV1-3 RT-PCR; DENV4-STD-UP/DP are for DENV4
                        RT-PCR; DENV-PCR-FP/RP/P, real-time RT-PCR primers and Taqman probe (FAM/TAMRA); DENV1-3-RPA-FP,
                        forward and reverse primers for DENV1-3 RT-RPA; DENV4-RPA-FP, forward and reverse
                        primers for DENV4 RT-RPA; DENV-RPA-P, exo-probe; BHQ1-dT: thymidine nucleotide carrying
                        Blackhole quencher1, THF: tetrahydrofuran spacer, FAM-dT: thymidine nucleotide carrying
                        Fluorescein. N/A, non applicable.
                     </p>
                  </div>
               </table-wrap-foot>
            </div>
         </div>
         <div id="sec009">
            <h3>RT-RPA primers and exo-probes</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Two RT-RPA assays were developed to cover DENV1-4, one to detect DENV1-3 and another
               to detect DENV4. The assays were not used for distinguishing between DENV serotypes.
               Nineteen forward primers (FP), 5 reverse primers (RP), and 3 exo-probes (exo-Ps) (
               <span ref-type="supplementary-material" rid="pone.0129682.s001">S1 Fig</span>) were used to select the combination producing the highest analytical sensitivity
               for the DENV1-3 RT-RPA assay. Three FP, 2 RP, and 2 exo-P were tested for the DENV4
               RT-RPA assay (
               <span ref-type="supplementary-material" rid="pone.0129682.s002">S2 Fig</span>). Oligonucleotides were synthesized by TIB MOLBIOL (Berlin, Germany).
            </p>
         </div>
         <div id="sec010">
            <h3>RT-RPA assay conditions</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In the laboratory, the RT-RPA assay was carried out using the TwistAmp exo kit (TwistDx,
               Cambridge, UK) and the reverse transcriptase (RT) Transcriptor (Roche, Mannheim, Germany)
               was added as described [
               <span ref-type="bibr" rid="pone.0129682.ref029">
                  <sup>
                     <a href="#pone.0129682.ref029">29</a>
                  </sup>
               </span>, 
               <span ref-type="bibr" rid="pone.0129682.ref030">
                  <sup>
                     <a href="#pone.0129682.ref030">30</a>
                  </sup>
               </span>]. In the field trials in Senegal and Thailand, the RT-RPA assay was performed using
               the ready-to-use TwistAmp RT exo (TwistDx, Cambridge, UK) as described previously
               [
               <span ref-type="bibr" rid="pone.0129682.ref026">
                  <sup>
                     <a href="#pone.0129682.ref026">26</a>
                  </sup>
               </span>].
            </p>
         </div>
         <div id="sec011">
            <h3>Analytical sensitivity of RT-RPA assays</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">A dilution range from 10
               <sup>7</sup> to 10
               <sup>1</sup> RNA molecules/μl of the molecular standards in eight replicates was used to determine
               the analytical sensitivity of RT-RPA assays. A semi-log and probit regression analyses
               were performed as stated in the statistical analysis section (see below).
            </p>
         </div>
         <div id="sec012">
            <h3>Specificity and cross-reactivity of RT-RPA assays</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The specificity of the DENV RT-RPA assays was determined by testing 64 DENV-real-time
               RT-PCR negative plasma samples (41 samples during the field trial in Thailand and
               23 in Senegal). In addition, genomic RNA of the Yellow Fever virus, Tick-borne encephalitis
               virus, Tick-borne encephalitis virus-Far East, Chikungunya virus, Zika virus, Japanese
               encephalitis virus and West Nile virus were tested for cross-reactivity.
            </p>
         </div>
         <div id="sec013">
            <h3>Differentiation between specific and non-specific signals of the RT-RPA</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Real-time fluorescence development in the RT-RPA reaction was measured with the tubescanner
               (Qiagen Lake Constance GmbH, Stockach, Germany). Fluorescence signals were analyzed
               by using the tubescanner studio software (Qiagen Lake Constance GmbH, Stockach, Germany).
               It allows an analysis of accumulative fluorescence intensity over time by threshold
               validation to identify the increase of fluorescence over time above the mean background
               signal (raw data, Fig 
               <span ref-type="fig" rid="pone.0129682.g001">1A</span> and 
               <span ref-type="fig" rid="pone.0129682.g001">1C</span>). Additionally, slope validation is used to verify that the increase of fluorescence
               occurs at a sufficiently high rate, this can be displayed as a 1
               <sup>st</sup> derivative analysis (1
               <sup>st</sup> derivative analysis, Fig 
               <span ref-type="fig" rid="pone.0129682.g001">1B</span> and 
               <span ref-type="fig" rid="pone.0129682.g001">1D</span>).
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pone.0129682.g001</div>
               <span class="label">Fig 1</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Differentiation between specific and non-specific signals of the RT-RPA assay.</div>
                  <p>A and C are real-time fluorescence intensity; B and D are the 1
                     <sup>st</sup> derivative analysis. Specific DNA amplification represented by progressive fluorescence
                     development in both views, A and B, while non-specific not. Black line shows specific
                     amplification where blue line shows no amplification.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Specific amplification (black curves 
               <span ref-type="fig" rid="pone.0129682.g001">Fig 1</span>) should present ascending curves indicating increase of florescence over time in
               both analyses. Negative or non-specific amplification (blue curves 
               <span ref-type="fig" rid="pone.0129682.g001">Fig 1</span>) might in a few cases develop fluorescence in the raw data view but never in the
               1
               <sup>st</sup> derivative analysis. The derivative analysis is therefore necessary to confirm true
               amplification.
            </p>
         </div>
         <div id="sec014">
            <h3>RT-RPA mobile laboratory</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The mobile laboratory was organized into four sites in close proximity, which included
               the extraction, the master mix, the sample mix, and the detection-sites (
               <span ref-type="fig" rid="pone.0129682.g002">Fig 2</span>). The RNA extraction was done by using a extraction kit based on magnetic beads (Dynabeads
               SILANE viral NA, Life Technologies, Darmstadt, Germany) according to the manufacturer
               instructions. The DENV RT-RPA assays were performed by using the ready-to-use TwistAmp
               exo RT kits (TwistDx, Cambridge, UK), dried DENV1-3 and DENV4 
               <i>in vitro</i> transcribed RNA (using the RNAstable kit, Biomatrica, Inc. San Diego, CA, USA), and
               dried RT-RPA oligonucleotides (TIB MOLBIOL, Syntheselabor, Berlin, Germany).
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pone.0129682.g002</div>
               <span class="label">Fig 2</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">RPA mobile laboratory.</div>
                  <p>The extraction area encompassing magnetic separator stand, vortex, rotator, 1.5–2
                     ml eppendorf tube rack, automatic 100–1000 μl micopipette, micropipette tips, digital
                     timer, 1.5 ml disposable plastic Eppendorf tubes, and a waste container with autoclavable
                     plastic bags. Both master mix and sample mix areas contain vortex, minicentrifuge,
                     automatic 1–10 and 10–100 μl micopipettes, micropipette tips, scissor, and 0.2 ml
                     tubes rack. The detection was done using the tubescanner (Twista device, TwistDx,
                     Cambridge, UK). In addition to a waste container, gloves, disposable towels, and aluminum
                     box with trolley (740x490x450 mm, ZARGES, Weilheim, Germany).
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Oligonucleotide sets were ordered as dried pellets at an absolute concentration of
               840 pmol primers/400 pmol probe. The addition of 200 μl water yielded a stock solution
               of 4.2 μM/2 μM from which the final concentration of 420 nM/200 nM of the RT-RPA reaction
               was derived by 10-fold dilution. One 200 μl 40x stock solution thus allowed pipette
               eight 9-volume master mixes to fill eight 8-tube strips for the tubescanner device.
               For the positive control, one volume of the master-mix was added to a dried positive
               control tube.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">All reagents used for the mobile laboratory were cold-chain independent, i.e. used
               stored, and transported at ambient temperature. Electricity was either tapped from
               a motor vehicle battery via inverter (HPL 1200-D-12 inverter 12V 1200W) or supplied
               via solar panel and power pack (Yeti 1250 set, GOALZERO, South Bluffdale, UT, USA).
            </p>
         </div>
         <div id="sec015">
            <h3>Field trial in the Kedougou region, Senegal</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The mobile RPA was tested in the field station of the IPD in Kedougou, Senegal. Additionally,
               the mobile RPA laboratory was transferred to a health care station in Bandafasi, Kedougou
               for use in very simple conditions with auxiliary electricity tapped from a motor vehicle.
               Inactivated DENV1-4 cell culture supernatants (heating at 56°C/1 hour and gamma irradiation
               with 30 kGy) and thirty-one DENV RNA positive samples were tested in triplicate.
            </p>
         </div>
         <div id="sec016">
            <h3>Evaluation of RT-RPA assay using RNA extracts in Bangkok, Thailand</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">RNA extracted from ninety selected samples, positive by SD Dengue Duo Rapid Test (Standard
               Diagnostic, Inc, Republic of Korea), by QIAamp Viral RNA Mini Kit (Qiagen, Hilden,
               Germany) and stored between 6–18 months at -80°C. RNA was retested simultaneously
               with real-time RT-PCR and the RT-RPA assays, each sample was tested at least twice.
               The real-time RT-PCR was performed by using the Light Cycler 480 RNA Master Hydrolysis
               Probes kit (Roche, Germany) using the following temperature profile: 61°C for 10 minutes,
               95°C for 2 minutes followed by 45 cycles of 95°C for 15 seconds and 60°C for 30 seconds
               on a 7500 Fast Real-Time PCR system. Data were analyzed using the 7500 Fast Real-Time
               PCR software provided by Applied Biosystem 7500 Fast Real-Time PCR system (Life technologies,
               USA). The primers and probe targeted the NS5 gene (
               <span ref-type="supplementary-material" rid="pone.0129682.s008">S1 Table</span>, [
               <span ref-type="bibr" rid="pone.0129682.ref014">
                  <sup>
                     <a href="#pone.0129682.ref014">14</a>
                  </sup>
               </span>]).
            </p>
         </div>
         <div id="sec017">
            <h3>Statistical analysis</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The semi-log regression analyses of the analytical sensitivity of RT-RPA assays were
               performed using PRISM (Graphpad Software Inc., San Diego, CA, USA) and the probit
               anylsis by STATISTICA (StatSoft, Hamburg, Germany). Comparison between real-time RT-PCR
               and RT-RPA for the detection of DENV was performed by linear regression analysis using
               Prism.
            </p>
         </div>
      </div>
      <div id="sec018">
         <h2>Results</h2>
         <div id="sec019">
            <h3>Analytical sensitivity of DENV1-3, and DENV4 RT-RPA assays</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Due to the variability of DENV sequences in the 3’NTR, two RT-RPA assays were designed,
               one for the detection of DENV1-3 and one for the detection of DENV4. The primers DENV1-3-RPA-FP13
               and RP4 and the primers DENV4-RPA-FP3 and RP2 both in combination with DENV-RPA-P3
               yielded analytical sensitivities between 10–100 RNA molecules (
               <span ref-type="table" rid="pone.0129682.t002">
                  <a href="#pone.0129682.t002">Table 2</a>
               </span> and 
               <span ref-type="fig" rid="pone.0129682.g003">Fig 3</span>). Most of the other primer and exo-P combinations produced either non-specific amplification
               (
               <span ref-type="supplementary-material" rid="pone.0129682.s003">S3 Fig</span>) or analytical sensitivity between 10
               <sup>5</sup>–10
               <sup>3</sup> RNA molecules (
               <span ref-type="supplementary-material" rid="pone.0129682.s004">S4 Fig</span>). Using the data of eight RT-RPA runs on the quantitative RNA standard, a semi-log
               regression (
               <span ref-type="fig" rid="pone.0129682.g004">Fig 4</span>) and probit regression analysis were performed. For both assays, the runtime was
               3–7 minutes at an efficiency of 0.24 and 0.27 calculated from the linear slopes (-0.2366
               and -0.1655) of the semi-log standard regression (E = 10
               <sup>(-1/slope)-</sup>1) for DEN1-3 and DEN4 RT-RPA assays, respectively. Probit regression yielded a sensitivity
               at 95% of 241 and 14 RNA molecules detected for the DENV1-3 and the DENV4 RT-RPA assays,
               respectively. Additionally, inactivated whole DENV1-4 (strains are listed in 
               <span ref-type="table" rid="pone.0129682.t001">
                  <a href="#pone.0129682.t001">Table 1</a>
               </span>) were spiked into plasma samples. Serial dilutions of each spiked sample were tested
               with real-time RT-PCR and RT-RPA assays simultaneously (
               <span ref-type="supplementary-material" rid="pone.0129682.s009">S2 Table</span>). Limits of detection in RT-RPA were 237, 618, 363, and 383 RNA molecules detected/reaction
               of DENV serotypes 1, 2, 3, and 4, respectively (
               <span ref-type="fig" rid="pone.0129682.g005">Fig 5</span>).
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pone.0129682.g003</div>
               <span class="label">Fig 3</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Analytical sensitivity of DENV RT-RPA assays.</div>
                  <p>A, DENV1-3 and B, DENV4 RT-RPA assays. Fluorescence development via real-time detection
                     in one RT-RPA run by using a dilution range of 10
                     <sup>7</sup>–10
                     <sup>1</sup> RNA molecules/μl of the DENV1-3 and DENV4 RNA molecular standards (Graph generated
                     by ESEquant tubescanner studio software). The sensitivity was 100 and 10 RNA copies
                     for DENV1-3 and DENV4 RT-RPA, respectively. Data of 8 RT-RPA runs is compiled in 
                     <span ref-type="fig" rid="pone.0129682.g004">Fig 4</span>. The signal for ten RNA copies is very weak. The box in the lower right corner of
                     Fig 3B magnifies the fluorescence signals for the ten RNA copies and the negative
                     control. 10
                     <sup>7</sup> represented by black line; 10
                     <sup>6</sup>, gray; 10
                     <sup>5</sup>, red; 10
                     <sup>4</sup>, blue; 10
                     <sup>3</sup>, green; 10
                     <sup>2</sup>, cyan; 10
                     <sup>1</sup>, dark khaki; negative control, orange.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pone.0129682.g004</div>
               <span class="label">Fig 4</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Reproducibility of DENV RT-RPA assays.</div>
                  <p>A, DENV1-3 and B, DENV4 RT-RPA assays. Semi-logarithmic regression of the data collected
                     from eight DENV RT-RPA test runs on the RNA standard using PRISM. Both assays yielded
                     results between 3–7 minutes. In DENV1-3 RT-RPA assay, 10
                     <sup>7</sup>–10
                     <sup>3</sup> RNA molecules were detected 8 out of 8 runs, 10
                     <sup>2</sup> in 1 out of 8 and 10 copies was not identified. In DENV4 RT-RPA assay, 10
                     <sup>7</sup>–10
                     <sup>2</sup> RNA molecules were detected 8 out of 8 runs and 10 copies in 6/ out of 8. In Fig
                     4B, the value for 10 RNA copies was consistently 5.3 minutes in all 6 cases.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pone.0129682.g005</div>
               <span class="label">Fig 5</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Performance of DENV RT-RPA assays on spiked plasma samples.</div>
                  <p>A, sample spiked with DENV1; B, DENV2; C, DENV3; D, DENV4. DENV serotypes 1–4 were
                     spiked into plasma samples. Serial dilutions of each of the spiked sample were tested
                     simultaneously with real-time RT-PCR and RT-RPA assays (
                     <span ref-type="supplementary-material" rid="pone.0129682.s009">S2 Table</span>). Limits of detection in RT-RPA assays were 237, 618, 363, and 383 RNA copies of
                     DENV serotypes 1, 2, 3, and 4, respectively.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
         </div>
         <div id="sec020">
            <h3>Specificity and cross-reactivity of DENV1-3, and DENV4 RT-RPA assays</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Sixty-four DENV-PCR negative samples were tested by both DENV RT-RPA assays. Non-specific
               amplification was not detected. DENV1-3 RT-RPA assay detected DENV serotypes 1, 2,
               and 3, while DENV4 RT-RPA identified DENV serotypes 2 and 4 (
               <span ref-type="table" rid="pone.0129682.t001">
                  <a href="#pone.0129682.t001">Table 1</a>
               </span>). No cross detection with viruses listed in 
               <span ref-type="table" rid="pone.0129682.t002">
                  <a href="#pone.0129682.t002">Table 2</a>
               </span> was observed.
            </p>
         </div>
         <div id="sec021">
            <h3>RPA mobile laboratory in Kedougou region in Senegal</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In the field, setting up the mobile laboratory including hooking electricity from
               a motor vehicle battery took about 20 minutes. A magnetic beads based extraction method
               was used to avoid the use of a centrifuge and the generation of aerosols.
            </p>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Using the inactivated DENV1-4 spiked plasma as well as 31 DENV positive samples, the
               DENV RT-RPA assays and the mobile laboratory format were tested in a healthcare center
               without electricity in Bandafassi (
               <span ref-type="supplementary-material" rid="pone.0129682.s005">S5 Fig</span>) to simulate an outbreak situation. Extraction and RT-RPA assays were carried out
               at an ambient temperature of 38°C. The RT-RPA assays were used successfully to detect
               DENV RNA in the spiked plasma. In comparison to the real-time RT-PCR results, 30 out
               of 31 DENV-3-positive samples were positive in RT-RPA assays (clinical sensitivity
               98%, 
               <span ref-type="fig" rid="pone.0129682.g006">Fig 6</span>). Samples with high cycle threshold (Ct) value (35–38) in real-time RT-PCR were detected
               by RT-RPA in a maximum of 8 minutes.
            </p>
            <div tagxxx="fig">
               <div tagxxx="object-id">10.1371/journal.pone.0129682.g006</div>
               <span class="label">Fig 6</span>
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <div tagxxx="title">Comparison between real-time RT-PCR (X-axis) and RT-RPA (Y-axis) for the detection
                     of DENV in 31 clinical samples in Senegal.
                  </div>
                  <p>Linear regression analysis of real-time RT-PCR cycle threshold values (Ct, X-axis)
                     and RT-RPA threshold time in minutes (TT, Y-axis) were determined by PRISM (R
                     <sup>2</sup> = 0.39). The RT-RPA is much faster than the real-time RT-PCR even with samples with
                     high Ct value.
                  </p>
               </caption>
               <div tagxxx="graphic"></div>
            </div>
         </div>
         <div id="sec022">
            <h3>Assay performance on RNA extracts in Thailand</h3>
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In Bangkok, Thailand, the RNA of ninety plasma samples extracted and tested between
               2012–2013 by RT-PCR, and stored at -80°C was tested. RNA was simultaneously screened
               by RT-RPA assays and serotype specific real-time RT-PCR (
               <span ref-type="supplementary-material" rid="pone.0129682.s008">S1 Table</span>). The sensitivity was 72% and 94.4% for RT-RPA and real-time RT-PCR, respectively
               (
               <span ref-type="supplementary-material" rid="pone.0129682.s010">S3 Table</span>). RT-RPA assays sensitivities for DENV serotypes were 66.6% (16/24), 44% (4/9), 80.9%
               (34/42), and 60% (6/10) for DENV1, 2, 3, and 4, respectively. Five samples were only
               positive in RT-RPA and therefore the DENV serotype was not determined. No correlation
               was found between threshold time values of RT-RPA and Ct of the real-time RT-PCR (
               <span ref-type="supplementary-material" rid="pone.0129682.s006">S6 Fig</span>, R
               <sup>2</sup> = 0.179).
            </p>
         </div>
      </div>
      <div id="sec023">
         <h2>Discussion</h2>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In their review, Peeling et al., [
            <span ref-type="bibr" rid="pone.0129682.ref008">
               <sup>
                  <a href="#pone.0129682.ref008">8</a>
               </sup>
            </span>] mention characteristics of the future diagnostic test for early detection of DENV.
            It should be rapid, sensitive, specific, stable at high temperatures (&gt;30°C), and
            cheap. Currently point-of-care DENV detection is carried out with RDT assays. RDTs
            are simple and rapid but of limited sensitivity and specificity [
            <span ref-type="bibr" rid="pone.0129682.ref007">
               <sup>
                  <a href="#pone.0129682.ref007">7</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pone.0129682.ref013">
               <sup>
                  <a href="#pone.0129682.ref013">13</a>
               </sup>
            </span>]. Real-time RT-PCR is highly sensitive, usually performed in highly equipped laboratories
            and not suitable for point-of-care testing. In contrast, RPA technology is as sensitive
            and specific as real-time RT-PCR, isothermal and easy to use at point-of-need [
            <span ref-type="bibr" rid="pone.0129682.ref026">
               <sup>
                  <a href="#pone.0129682.ref026">26</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pone.0129682.ref029">
               <sup>
                  <a href="#pone.0129682.ref029">29</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pone.0129682.ref035">
               <sup>
                  <a href="#pone.0129682.ref035">35</a>
               </sup>
            </span>]. In addition, RPA is fast (3–5 minutes including the reverse transcription step)
            and can be performed on a portable device (tubescanner). In this study, two RT-RPA
            assays for detection of DENV (DENV1-3 and DENV4) were developed which cover but do
            not differentially determine DENV1-4, as required for molecular epidemiological studies.
            Distinguishing DENV1-4 at the point-of-care is not clinically relevant. The assays
            were very sensitive with a respective limit of detection of 241 and 14 molecules detected,
            highly specific, and were evaluated as mobile tests in a low resource setting in Kedougou
            in Senegal.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Several isothermal amplification technologies were developed in the last two decades,
            e.g. RT-LAMP [
            <span ref-type="bibr" rid="pone.0129682.ref024">
               <sup>
                  <a href="#pone.0129682.ref024">24</a>
               </sup>
            </span>], NASBA [
            <span ref-type="bibr" rid="pone.0129682.ref036">
               <sup>
                  <a href="#pone.0129682.ref036">36</a>
               </sup>
            </span>], nicking enzyme amplification reaction [
            <span ref-type="bibr" rid="pone.0129682.ref037">
               <sup>
                  <a href="#pone.0129682.ref037">37</a>
               </sup>
            </span>], rolling circle amplification methods [
            <span ref-type="bibr" rid="pone.0129682.ref038">
               <sup>
                  <a href="#pone.0129682.ref038">38</a>
               </sup>
            </span>], helicase dependent amplification [
            <span ref-type="bibr" rid="pone.0129682.ref039">
               <sup>
                  <a href="#pone.0129682.ref039">39</a>
               </sup>
            </span>], and RPA [
            <span ref-type="bibr" rid="pone.0129682.ref025">
               <sup>
                  <a href="#pone.0129682.ref025">25</a>
               </sup>
            </span>]. They differ in the amplification temperature, run time, number of primers used
            for DNA amplification, detection method (probe-based or intercalating dye dependent)
            and commercial availability.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Many RT-LAMP assays have been developed for DENV detection using between 6–24 primers
            [
            <span ref-type="bibr" rid="pone.0129682.ref020">
               <sup>
                  <a href="#pone.0129682.ref020">20</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pone.0129682.ref023">
               <sup>
                  <a href="#pone.0129682.ref023">23</a>
               </sup>
            </span>] and yielding results in 30 minutes based on visual inspection (turbidity index,
            fluorescence) [
            <span ref-type="bibr" rid="pone.0129682.ref020">
               <sup>
                  <a href="#pone.0129682.ref020">20</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pone.0129682.ref023">
               <sup>
                  <a href="#pone.0129682.ref023">23</a>
               </sup>
            </span>]. While SYBR Green detection of the RT-LAMP reaction could basically also be done
            with the tubescanner, RPA is much faster (3–15 minutes) and utilizes only three oligonucleotides
            (two primers, one exo-P). Recent advances in software development to assist LAMP design
            may help overcome challenges for highly variant viruses, the tolerance for mismatches
            in LAMP assays however still needs to be explored [
            <span ref-type="bibr" rid="pone.0129682.ref040">
               <sup>
                  <a href="#pone.0129682.ref040">40</a>
               </sup>
            </span>]. The tolerance of RPA oligonucleotide sets and in particular, of the exo-P probe
            portions to mismatches [
            <span ref-type="bibr" rid="pone.0129682.ref026">
               <sup>
                  <a href="#pone.0129682.ref026">26</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pone.0129682.ref033">
               <sup>
                  <a href="#pone.0129682.ref033">33</a>
               </sup>
            </span>] are advantageous for RPA amplicon design but remain subject to testing.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Previous work had shown that for RNA viruses the best sensitivity is achieved with
            an exo-P, encompassing 15 nt 5’ (short section), and 30 nt 3’ (long section) of a
            basic site mimic (Tetrahydrofuran, THF) and a phosphate at the 3’ end of the exo-P
            [
            <span ref-type="bibr" rid="pone.0129682.ref025">
               <sup>
                  <a href="#pone.0129682.ref025">25</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pone.0129682.ref026">
               <sup>
                  <a href="#pone.0129682.ref026">26</a>
               </sup>
            </span>]. The phosphate blocks any primer type interference of the long section with the
            RPA reaction after Exonuclease nicking [
            <span ref-type="bibr" rid="pone.0129682.ref025">
               <sup>
                  <a href="#pone.0129682.ref025">25</a>
               </sup>
            </span>]. In contrast of all tested probes, DENV-RPA-P3 consisting of the standard inverse
            exo-P structure with a long 32 nt section 5’ and a short 16 nt section 3’ of the THF
            site (
            <span ref-type="table" rid="pone.0129682.t002">
               <a href="#pone.0129682.t002">Table 2</a>
            </span>) yielded the highest sensitivity. Here the long section can act as a primer and contribute
            to, the sensitivity of the assay [
            <span ref-type="bibr" rid="pone.0129682.ref025">
               <sup>
                  <a href="#pone.0129682.ref025">25</a>
               </sup>
            </span>] as observed for other RPA assays [
            <span ref-type="bibr" rid="pone.0129682.ref029">
               <sup>
                  <a href="#pone.0129682.ref029">29</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pone.0129682.ref030">
               <sup>
                  <a href="#pone.0129682.ref030">30</a>
               </sup>
            </span>].
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Of all reverse primers tested DENV1-3 RP4 and DENV4 RP2 yielded the highest assay
            sensitivities although overlapping the long exo-P 3’ section of DENV-RPA-P3 by 28
            nt (
            <span ref-type="supplementary-material" rid="pone.0129682.s001">S1</span> and 
            <span ref-type="supplementary-material" rid="pone.0129682.s002">S2</span> Figs). The best forward primer for the DENV4 assay DENV4 FP3 overlapped the short
            exo-P 3’ end by 6 nt (
            <span ref-type="supplementary-material" rid="pone.0129682.s002">S2 Fig</span>). Non-specific amplification was not observed in any of these cases. Therefore, the
            selection of primer and probe format is still down to trial and error. The factors
            that determine efficient strand invasion may be a combination of binding enthalpy
            and local secondary structure. Although advances in calculating these values for PCR
            primers have been made, they cannot as yet be deduced for RPA [
            <span ref-type="bibr" rid="pone.0129682.ref041">
               <sup>
                  <a href="#pone.0129682.ref041">41</a>
               </sup>
            </span>].
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">DEN RPA P3 is shared by both assays but carries 4 mismatches (
            <span ref-type="supplementary-material" rid="pone.0129682.s002">S2</span> and 
            <span ref-type="supplementary-material" rid="pone.0129682.s007">S7</span> Figs) in the smaller 13 nt 3’ section of the probe. Efficient RPA tolerates mismatches
            per primer which should not accumulate at the termini or at the center of the primers
            and in concordance with what is known about PCR primers mismatches at the 3’ end of
            the primers impede the RPA reaction [
            <span ref-type="bibr" rid="pone.0129682.ref042">
               <sup>
                  <a href="#pone.0129682.ref042">42</a>
               </sup>
            </span>]. As a whole RPA amplification systems appear to tolerate the presence of 5–9 mismatches
            in a primer and probe set [
            <span ref-type="bibr" rid="pone.0129682.ref026">
               <sup>
                  <a href="#pone.0129682.ref026">26</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pone.0129682.ref033">
               <sup>
                  <a href="#pone.0129682.ref033">33</a>
               </sup>
            </span>]. We assume that the long 32 nt 5’ section of DEN RPA P3, which matches almost perfectly
            to DENV1-4, allows for tolerance of the accumulation of the mismatches in the smaller
            3’ portion of the probe since it does not conflict with the principle of dissociation
            of the smaller section the exo-P after Exonuclease action. Shifting the mismatches
            into the smaller section of the probes may thus be exploited for the design of exo-P
            for the detection of other highly variable RNA target sequences. In contrast, real-time
            PCR, which utilized shorter primers and probes, is more sensitive to the presence
            of mismatches in the target sequence [
            <span ref-type="bibr" rid="pone.0129682.ref043">
               <sup>
                  <a href="#pone.0129682.ref043">43</a>
               </sup>
            </span>]. The linearity of the assay also does not represent a correlation of linearity and
            sensitivity as seen in real-time PCR assays [
            <span ref-type="bibr" rid="pone.0129682.ref029">
               <sup>
                  <a href="#pone.0129682.ref029">29</a>
               </sup>
            </span>].
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Real-time RT-PCR can be used for both qualitative and quantitative analysis of RNA
            in a sample. In contrast, RT-RPA can currently only be used for qualitative detection
            of RNA. As shown in 
            <span ref-type="fig" rid="pone.0129682.g006">Fig 6</span>, the TT values of the RT-RPA can be classified into two groups, before 3 and after
            5 minutes. The main reason is the interruption of the fluorescence read by the mixing
            step after three minutes, which is crucial for the assay sensitivity. During mixing,
            fluoresces signal are not measured (Figs 
            <span ref-type="fig" rid="pone.0129682.g003">3</span> and 
            <span ref-type="fig" rid="pone.0129682.g006">6</span>), therefore, TT values can`t be calculated. This however does not lead to false negatives.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The characteristics of the mobile laboratory (23kg including the aluminium case) are
            1) easy transportation by car and airplane, 2) power source from motor vehicle batteries
            or solar panel with power pack, 3) easy implementation in low resource settings. A
            magnetic bead extraction was applied to avoid the creation of aerosols and because
            trials at the Institute Pasteur in Paris in a climate chamber have shown that most
            commercial centrifuges short-circuited at 80% humidity and temperatures above 38°C
            (Jean-Claude Manuguerra, personal communication). All reagents were cold chain independent
            and performed well at 38–40°C ambient temperature. We were able to run the laboratory
            in the open air and observed no influence of dust on the assay quality. We will continue
            to follow this concept to allow easy deployment of up to date molecular detection
            to infrastructure poor settings. Further work will attempt to reduce pipetting steps
            by drying RPA primers and probe into the RPA reagent pellet, implementing a multiplex
            RT-RPA assay combining the two DENV RPA assays into one reaction and identifying even
            simpler extraction protocols.
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">The DENV RT-RPA assays were evaluated with plasma samples and RNA extracts in Senegal
            and Thailand. The RT-RPA clinical sensitivities were 98% for samples tested in Senegal
            and 72% for RNA extracts in Thailand, whereas real-time RT-PCR sensitivity was 98%
            and 94.4%, respectively. The lower clinical sensitivity of the RT-RPA assays on the
            RNA extracts in Thailand might not be due to the assay detection limit, which is between
            10–100 RNA copies (Figs 
            <span ref-type="fig" rid="pone.0129682.g003">3</span>, 
            <span ref-type="fig" rid="pone.0129682.g004">4</span> and 
            <span ref-type="fig" rid="pone.0129682.g005">5</span>). As shown in 
            <span ref-type="supplementary-material" rid="pone.0129682.s010">S3 Table</span>, most of the samples, which were negative in RT-RPA had Ct values between 14–30 in
            real-time RT-PCR. Genetic variations between Asian and African DENV strain could not
            be the reason for the difference in the assay sensitivity [
            <span ref-type="bibr" rid="pone.0129682.ref044">
               <sup>
                  <a href="#pone.0129682.ref044">44</a>
               </sup>
            </span>–
            <span ref-type="bibr" rid="pone.0129682.ref047">
               <sup>
                  <a href="#pone.0129682.ref047">47</a>
               </sup>
            </span>]. As mentioned above, the RPA targeted the conserved 3’ NTR of the DENV genome and
            RPA amplification can tolerate 5–9 mismatches. In Senegal, RNA was extracted from
            plasma samples and screened directly with the RT-RPA assays, while in Thailand, RNA
            stored at -80°C for 6–18 months was tested. The integrity of the RNA is affected by
            freezing and thawing, albeit, freezing of clinical samples has a less pronounced effect
            [
            <span ref-type="bibr" rid="pone.0129682.ref048">
               <sup>
                  <a href="#pone.0129682.ref048">48</a>
               </sup>
            </span>, 
            <span ref-type="bibr" rid="pone.0129682.ref049">
               <sup>
                  <a href="#pone.0129682.ref049">49</a>
               </sup>
            </span>]. It appears that in contrast to real-time RT-PCR [
            <span ref-type="bibr" rid="pone.0129682.ref050">
               <sup>
                  <a href="#pone.0129682.ref050">50</a>
               </sup>
            </span>], RT-RPA performance is influenced by the quality of the RNA confirming an observation
            we described earlier [
            <span ref-type="bibr" rid="pone.0129682.ref026">
               <sup>
                  <a href="#pone.0129682.ref026">26</a>
               </sup>
            </span>].
         </p>
         <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">In conclusion, two DENV RT-RPA assays were developed for rapid identification of DENV1-4.
            RPA was easy to implement in low resource settings and high ambient temperatures did
            not affect its performance.
         </p>
      </div>
      <div id="sec024">
         <h2>Supporting Information</h2>
         <div tagxxx="supplementary-material">
            <span class="label">S1 Fig</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">DENV1-3 RT-RPA primers and probes sequence aligned with the DENV1-3 amplicon.</div>
               <p>Three RPA exo probes (P), 19 forward primers (FP), and 5 reverse primers (RP) were
                  tested to select combinations yielding the highest analytical DENV1-3 RT-RPA sensitivity.
                  FP13, RP4, and P3 produced the best RT-RPA assay sensitivity. NNN are sites of the
                  quencher and fluorophore in following order (BHQ1-dT) (Tetrahydrofuran) (FAM-dT).
                  RC is the reverse complementary of the original sequence used in the experiment.
               </p>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S2 Fig</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">DENV4 RT-RPA primers and probes sequences aligned with the DENV4 amplicon.</div>
               <p>Two RPA exo probes (P), 3 forward primers (FP), and 2 reverse primers (RP) were tested
                  to select combinations yielding the highest analytical DENV4 RT-RPA sensitivity. FP3,
                  RP2, and P3 produced the best RT-RPA assay sensitivity. NNN are sites of the quencher
                  and fluorophore in following order (BHQ1-dT) (Tetrahydrofuran) (FAM-dT). RC is the
                  reverse complementary of the original sequence used in the experiment.
               </p>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S3 Fig</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">DENV1-3 RT-RPA assay with non-specific amplification.</div>
               <p>Fluorescence development via real-time detection by using a dilution range of 10
                  <sup>7</sup>–10
                  <sup>1</sup> RNA molecules/μl of the DENV1-3 molecular standard. 10
                  <sup>7</sup> represented by black line; 10
                  <sup>6</sup>, gray; 10
                  <sup>5</sup>, red; 10
                  <sup>4</sup>, blue; 10
                  <sup>3</sup>, green; 10
                  <sup>2</sup>, cyan; 10
                  <sup>1</sup>, dark khaki; negative control, orange.
               </p>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S4 Fig</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">DENV1-3 RT-RPA assay with 10
                  <sup>4</sup> analytical sensitivity.
               </div>
               <p>Fluorescence development via real-time detection by using a dilution range of 10
                  <sup>7</sup>–10
                  <sup>1</sup> RNA molecules/μl of the DENV1-3 molecular standard. 10
                  <sup>7</sup> represented by black line; 10
                  <sup>6</sup>, gray; 10
                  <sup>5</sup>, red; 10
                  <sup>4</sup>, blue; 10
                  <sup>3</sup>, green; 10
                  <sup>2</sup>, cyan; 10
                  <sup>1</sup>, dark khaki; negative control, orange.
               </p>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S5 Fig</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">RPA mobile laboratory in a local hospital in Bandafasi in Kedougou, Senegal.</div>
               <p>A, preparation to transfer the RPA mobile laboratory. B, Bandafasi in Kedougou region.
                  C, local hospital. D, laboratory at the local hospital. E, Power supply from motor
                  vehicle battery and convertor. F, RPA mobile laboratory operated by power from motor
                  vehicle battery.
               </p>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S6 Fig</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Comparison between RT-RPA (Y-axis) and real-time RT-PCR (X-axis) for the detection
                  of DENV in 90 clinical samples collected in Thailand between 2012–2013.
               </div>
               <p>Linear regression analysis of RT-RPA threshold time in minutes (TT, Y-axis) and real-time
                  RT-PCR cycle threshold values (Ct, X-axis) were determined using PRISM. R squared
                  value was 0.179.
               </p>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S7 Fig</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Alignment of DENV RT-RPA primers and exo-probe sequences with the consensus sequences
                  of 3’NTR of DENV1-4 using Geneious (V: 6.1.5, Biomatters Limited, New Zealand).
               </div>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S1 Table</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Primer and probe set sequences of the real-time RT-PCR performed during the field
                  trial in Thailand.
               </div>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S2 Table</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Results of screening spiked plasma samples with inactivated whole DENV1-4 with real-time
                  RT-PCR and RT-RPA assays.
               </div>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
         <div tagxxx="supplementary-material">
            <span class="label">S3 Table</span>
            <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
               <div tagxxx="title">Results of testing 90 DENV-positive RNA extracts using DENV RT-RPA assays and real-time
                  RT-PCR.
               </div>
               <p>(DOCX)</p>
            </caption>
            <div tagxxx="media">
               <caption xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">
                  <p>Click here for additional data file.</p>
               </caption>
            </div>
         </div>
      </div>
      <div tagxxx="back">
         <div tagxxx="ack">
            <p xmlns:ali="http://www.niso.org/schemas/ali/1.0" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:mml="http://www.w3.org/1998/Math/MathML">Authors thank Phillipe Buchy and Veasna Duong, Institut Pasteur of Cambodia, for providing
               materials of the real-time RT-PCR performed during the field trial in Thailand as
               well as Dr. Pratap Singhasivanon, Faculty of Tropical Medicine, Mahidol University,
               Thailand and Dr. Sawangjit Suraamornkul, Faculty of Medicine, Vajira hospital, Thailand,
               for oversee Bangkok cohort. We thank Shereen Petersen for English proofreading.
            </p>
         </div>
         <div tag="ref-list">
            <ul>
               <div tagxxx="title">References</div>
               <li tag="ref">
                  <a name="pone.0129682.ref001"></a>
                  <span class="label">1</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gan</span>
                        <span tagx="given-names">VC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lye</span>
                        <span tagx="given-names">DC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Thein</span>
                        <span tagx="given-names">TL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Dimatatac</span>
                        <span tagx="given-names">F</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tan</span>
                        <span tagx="given-names">AS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Leo</span>
                        <span tagx="given-names">YS</span>,
                     </span>. 
                     <span tagx="article-title">Implications of discordance in world health organization 1997 and 2009 dengue classifications
                        in adult dengue
                     </span>. 
                     <span tagx="source">PloS one</span>. 
                     <span tagx="year">2013</span>;
                     <span tagx="volume">8</span>(
                     <div tagxxx="issue">4</div>):
                     <span tagx="fpage">e60946</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1371/journal.pone.0060946</div>
                     </div> ; PubMed Central PMCID: PMC3613419.
                     <span class="pub-id'">[23573291]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref002"></a>
                  <span class="label">2</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Daep</span>
                        <span tagx="given-names">CA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Munoz-Jordan</span>
                        <span tagx="given-names">JL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Eugenin</span>
                        <span tagx="given-names">EA</span>,
                     </span>. 
                     <span tagx="article-title">Flaviviruses, an expanding threat in public health: focus on dengue, West Nile, and
                        Japanese encephalitis virus
                     </span>. 
                     <span tagx="source">Journal of neurovirology</span>. 
                     <span tagx="year">2014</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1007/s13365-014-0285-z</div>
                     </div> .
                     <span class="pub-id'">[25287260]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref003"></a>
                  <span class="label">3</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Bhatt</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gething</span>
                        <span tagx="given-names">PW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Brady</span>
                        <span tagx="given-names">OJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Messina</span>
                        <span tagx="given-names">JP</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Farlow</span>
                        <span tagx="given-names">AW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Moyes</span>
                        <span tagx="given-names">CL</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">The global distribution and burden of dengue</span>. 
                     <span tagx="source">Nature</span>. 
                     <span tagx="year">2013</span>;
                     <span tagx="volume">496</span>(
                     <div tagxxx="issue">7446</div>):
                     <span tagx="fpage">504</span>–
                     <span tagx="lpage">7</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1038/nature12060</div>
                     </div> ; PubMed Central PMCID: PMC3651993.
                     <span class="pub-id'">[23563266]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref004"></a>
                  <span class="label">4</span>
                  <div tagxxx="mixed-citation">WHO. Dengue fact sheets. Available: 
                     <div tagxxx="ext-link">http://wwwwprowhoint/mediacentre/factsheets/fs_09032012_Dengue/en/</div>. 2014.
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref005"></a>
                  <span class="label">5</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Thisyakorn</span>
                        <span tagx="given-names">U</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Thisyakorn</span>
                        <span tagx="given-names">C</span>,
                     </span>. 
                     <span tagx="article-title">Latest developments and future directions in dengue vaccines</span>. 
                     <span tagx="source">Therapeutic advances in vaccines</span>. 
                     <span tagx="year">2014</span>;
                     <span tagx="volume">2</span>(
                     <div tagxxx="issue">1</div>):
                     <span tagx="fpage">3</span>–
                     <span tagx="lpage">9</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1177/2051013613507862</div>
                     </div> ; PubMed Central PMCID: PMC3991153.
                     <span class="pub-id'">[24757522]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref006"></a>
                  <span class="label">6</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Stewart Ibarra</span>
                        <span tagx="given-names">AM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Luzadis</span>
                        <span tagx="given-names">VA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Borbor Cordova</span>
                        <span tagx="given-names">MJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Silva</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ordonez</span>
                        <span tagx="given-names">T</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Beltran Ayala</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">A social-ecological analysis of community perceptions of dengue fever and Aedes aegypti
                        in Machala, Ecuador
                     </span>. 
                     <span tagx="source">BMC public health</span>. 
                     <span tagx="year">2014</span>;
                     <span tagx="volume">14</span>:
                     <span tagx="fpage">1135</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1186/1471-2458-14-1135</div>
                     </div> ; PubMed Central PMCID: PMC4240812.
                     <span class="pub-id'">[25370883]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref007"></a>
                  <span class="label">7</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Blacksell</span>
                        <span tagx="given-names">SD</span>,
                     </span>. 
                     <span tagx="article-title">Commercial dengue rapid diagnostic tests for point-of-care application: recent evaluations
                        and future needs?
                     </span>
                     <span tagx="source">Journal of biomedicine &amp; biotechnology</span>. 
                     <span tagx="year">2012</span>;
                     <span tagx="volume">2012</span>:
                     <span tagx="fpage">151967</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1155/2012/151967</div>
                     </div> ; PubMed Central PMCID: PMC3357944.
                     <span class="pub-id'">[22654479]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref008"></a>
                  <span class="label">8</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Peeling</span>
                        <span tagx="given-names">RW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Artsob</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Pelegrino</span>
                        <span tagx="given-names">JL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Buchy</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Cardosa</span>
                        <span tagx="given-names">MJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Devi</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Evaluation of diagnostic tests: dengue</span>. 
                     <span tagx="source">Nature reviews Microbiology</span>. 
                     <span tagx="year">2010</span>;
                     <span tagx="volume">8</span>(
                     <div tagxxx="issue">12 Suppl</div>):
                     <span tagx="fpage">S30</span>–
                     <span tagx="lpage">8</span>. .
                     <span class="pub-id'">[21548185]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref009"></a>
                  <span class="label">9</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Medina</span>
                        <span tagx="given-names">F</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Medina</span>
                        <span tagx="given-names">JF</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Colon</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vergne</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Santiago</span>
                        <span tagx="given-names">GA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Munoz-Jordan</span>
                        <span tagx="given-names">JL</span>,
                     </span>. 
                     <span tagx="article-title">Dengue virus: isolation, propagation, quantification, and storage</span>. 
                     <span tagx="source">Current protocols in microbiology</span>. 
                     <span tagx="year">2012</span>;Chapter 15:Unit 15D 2. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1002/9780471729259.mc15d02s27</div>
                     </div> .
                     <span class="pub-id'">[23184594]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref010"></a>
                  <span class="label">10</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Yamada</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Takasaki</span>
                        <span tagx="given-names">T</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nawa</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kurane</span>
                        <span tagx="given-names">I</span>,
                     </span>. 
                     <span tagx="article-title">Virus isolation as one of the diagnostic methods for dengue virus infection</span>. 
                     <span tagx="source">J Clin Virol</span>. 
                     <span tagx="year">2002</span>;
                     <span tagx="volume">24</span>(
                     <div tagxxx="issue">3</div>):
                     <span tagx="fpage">203</span>–
                     <span tagx="lpage">9</span>. .
                     <span class="pub-id'">[11856621]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref011"></a>
                  <span class="label">11</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Paranavitane</span>
                        <span tagx="given-names">SA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gomes</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kamaladasa</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Adikari</span>
                        <span tagx="given-names">TN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Wickramasinghe</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Jeewandara</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Dengue NS1 antigen as a marker of severe clinical disease</span>. 
                     <span tagx="source">BMC infectious diseases</span>. 
                     <span tagx="year">2014</span>;
                     <span tagx="volume">14</span>:
                     <span tagx="fpage">570</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1186/s12879-014-0570-8</div>
                     </div> ; PubMed Central PMCID: PMC4222370.
                     <span class="pub-id'">[25366086]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref012"></a>
                  <span class="label">12</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Blacksell</span>
                        <span tagx="given-names">SD</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Jarman</span>
                        <span tagx="given-names">RG</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gibbons</span>
                        <span tagx="given-names">RV</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tanganuchitcharnchai</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Mammen</span>
                        <span tagx="given-names">MP</span>
                        <div tagxxx="suffix">Jr.</div>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nisalak</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Comparison of seven commercial antigen and antibody enzyme-linked immunosorbent assays
                        for detection of acute dengue infection
                     </span>. 
                     <span tagx="source">Clinical and vaccine immunology: CVI</span>. 
                     <span tagx="year">2012</span>;
                     <span tagx="volume">19</span>(
                     <div tagxxx="issue">5</div>):
                     <span tagx="fpage">804</span>–
                     <span tagx="lpage">10</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1128/CVI.05717-11</div>
                     </div> ; PubMed Central PMCID: PMC3346317.
                     <span class="pub-id'">[22441389]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref013"></a>
                  <span class="label">13</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Felix</span>
                        <span tagx="given-names">AC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Romano</span>
                        <span tagx="given-names">CM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Centrone Cde</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Rodrigues</span>
                        <span tagx="given-names">CL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Villas-Boas</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Araujo</span>
                        <span tagx="given-names">ES</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Low sensitivity of NS1 protein tests evidenced during a dengue type 2 virus outbreak
                        in Santos, Brazil, in 2010
                     </span>. 
                     <span tagx="source">Clinical and vaccine immunology: CVI</span>. 
                     <span tagx="year">2012</span>;
                     <span tagx="volume">19</span>(
                     <div tagxxx="issue">12</div>):
                     <span tagx="fpage">1972</span>–
                     <span tagx="lpage">6</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1128/CVI.00535-12</div>
                     </div> ; PubMed Central PMCID: PMC3535870.
                     <span class="pub-id'">[23100478]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref014"></a>
                  <span class="label">14</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Hue</span>
                        <span tagx="given-names">KD</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tuan</span>
                        <span tagx="given-names">TV</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Thi</span>
                        <span tagx="given-names">HT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Bich</span>
                        <span tagx="given-names">CT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Anh</span>
                        <span tagx="given-names">HH</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Wills</span>
                        <span tagx="given-names">BA</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Validation of an internally controlled one-step real-time multiplex RT-PCR assay for
                        the detection and quantitation of dengue virus RNA in plasma
                     </span>. 
                     <span tagx="source">J Virol Methods</span>. 
                     <span tagx="year">2011</span>;
                     <span tagx="volume">177</span>(
                     <div tagxxx="issue">2</div>):
                     <span tagx="fpage">168</span>–
                     <span tagx="lpage">73</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/j.jviromet.2011.08.002</div>
                     </div> .
                     <span class="pub-id'">[21843553]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref015"></a>
                  <span class="label">15</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Johnson</span>
                        <span tagx="given-names">BW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Russell</span>
                        <span tagx="given-names">BJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lanciotti</span>
                        <span tagx="given-names">RS</span>,
                     </span>. 
                     <span tagx="article-title">Serotype-specific detection of dengue viruses in a fourplex real-time reverse transcriptase
                        PCR assay
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>. 
                     <span tagx="year">2005</span>;
                     <span tagx="volume">43</span>(
                     <div tagxxx="issue">10</div>):
                     <span tagx="fpage">4977</span>–
                     <span tagx="lpage">83</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1128/JCM.43.10.4977-4983.2005</div>
                     </div> ; PubMed Central PMCID: PMC1248506.
                     <span class="pub-id'">[16207951]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref016"></a>
                  <span class="label">16</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Lai</span>
                        <span tagx="given-names">YL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Chung</span>
                        <span tagx="given-names">YK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tan</span>
                        <span tagx="given-names">HC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Yap</span>
                        <span tagx="given-names">HF</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Yap</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ooi</span>
                        <span tagx="given-names">EE</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Cost-effective real-time reverse transcriptase PCR (RT-PCR) to screen for Dengue virus
                        followed by rapid single-tube multiplex RT-PCR for serotyping of the virus
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>. 
                     <span tagx="year">2007</span>;
                     <span tagx="volume">45</span>(
                     <div tagxxx="issue">3</div>):
                     <span tagx="fpage">935</span>–
                     <span tagx="lpage">41</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1128/JCM.01258-06</div>
                     </div> ; PubMed Central PMCID: PMC1829098.
                     <span class="pub-id'">[17215345]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref017"></a>
                  <span class="label">17</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Najioullah</span>
                        <span tagx="given-names">F</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Viron</span>
                        <span tagx="given-names">F</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Cesaire</span>
                        <span tagx="given-names">R</span>,
                     </span>. 
                     <span tagx="article-title">Evaluation of four commercial real-time RT-PCR kits for the detection of dengue viruses
                        in clinical samples
                     </span>. 
                     <span tagx="source">Virology journal</span>. 
                     <span tagx="year">2014</span>;
                     <span tagx="volume">11</span>:
                     <span tagx="fpage">164</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1186/1743-422X-11-164</div>
                     </div> ; PubMed Central PMCID: PMC4177702.
                     <span class="pub-id'">[25219286]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref018"></a>
                  <span class="label">18</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Pongsiri</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Praianantathavorn</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Theamboonlers</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Payungporn</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Poovorawan</span>
                        <span tagx="given-names">Y</span>,
                     </span>. 
                     <span tagx="article-title">Multiplex real-time RT-PCR for detecting chikungunya virus and dengue virus</span>. 
                     <span tagx="source">Asian Pacific journal of tropical medicine</span>. 
                     <span tagx="year">2012</span>;
                     <span tagx="volume">5</span>(
                     <div tagxxx="issue">5</div>):
                     <span tagx="fpage">342</span>–
                     <span tagx="lpage">6</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/S1995-7645(12)60055-8</div>
                     </div> .
                     <span class="pub-id'">[22546647]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref019"></a>
                  <span class="label">19</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Wu</span>
                        <span tagx="given-names">SJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lee</span>
                        <span tagx="given-names">EM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Putvatana</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Shurtliff</span>
                        <span tagx="given-names">RN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Porter</span>
                        <span tagx="given-names">KR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Suharyono</span>
                        <span tagx="given-names">W</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Detection of dengue viral RNA using a nucleic acid sequence-based amplification assay</span>. 
                     <span tagx="source">J Clin Microbiol</span>. 
                     <span tagx="year">2001</span>;
                     <span tagx="volume">39</span>(
                     <div tagxxx="issue">8</div>):
                     <span tagx="fpage">2794</span>–
                     <span tagx="lpage">8</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1128/JCM.39.8.2794-2798.2001</div>
                     </div> ; PubMed Central PMCID: PMC88241.
                     <span class="pub-id'">[11473994]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref020"></a>
                  <span class="label">20</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Li</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Fang</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Zhou</span>
                        <span tagx="given-names">B</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ni</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Shen</span>
                        <span tagx="given-names">Q</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Zhang</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Simultaneous detection and differentiation of dengue virus serotypes 1–4, Japanese
                        encephalitis virus, and West Nile virus by a combined reverse-transcription loop-mediated
                        isothermal amplification assay
                     </span>. 
                     <span tagx="source">Virology journal</span>. 
                     <span tagx="year">2011</span>;
                     <span tagx="volume">8</span>:
                     <span tagx="fpage">360</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1186/1743-422X-8-360</div>
                     </div> ; PubMed Central PMCID: PMC3149006.
                     <span class="pub-id'">[21777455]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref021"></a>
                  <span class="label">21</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Parida</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Horioke</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ishida</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Dash</span>
                        <span tagx="given-names">PK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Saxena</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Jana</span>
                        <span tagx="given-names">AM</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Rapid detection and differentiation of dengue virus serotypes by a real-time reverse
                        transcription-loop-mediated isothermal amplification assay
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>. 
                     <span tagx="year">2005</span>;
                     <span tagx="volume">43</span>(
                     <div tagxxx="issue">6</div>):
                     <span tagx="fpage">2895</span>–
                     <span tagx="lpage">903</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1128/JCM.43.6.2895-2903.2005</div>
                     </div> ; PubMed Central PMCID: PMC1151941.
                     <span class="pub-id'">[15956414]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref022"></a>
                  <span class="label">22</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Sahni</span>
                        <span tagx="given-names">AK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Grover</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sharma</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Khan</span>
                        <span tagx="given-names">ID</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kishore</span>
                        <span tagx="given-names">J</span>,
                     </span>. 
                     <span tagx="article-title">Reverse transcription loop-mediated isothermal amplification (RT-LAMP) for diagnosis
                        of dengue
                     </span>. 
                     <span tagx="source">Medical journal, Armed Forces India</span>. 
                     <span tagx="year">2013</span>;
                     <span tagx="volume">69</span>(
                     <div tagxxx="issue">3</div>):
                     <span tagx="fpage">246</span>–
                     <span tagx="lpage">53</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/j.mjafi.2012.07.017</div>
                     </div> ; PubMed Central PMCID: PMC3862550.
                     <span class="pub-id'">[24600118]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref023"></a>
                  <span class="label">23</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Teoh</span>
                        <span tagx="given-names">BT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sam</span>
                        <span tagx="given-names">SS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tan</span>
                        <span tagx="given-names">KK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Johari</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Danlami</span>
                        <span tagx="given-names">MB</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hooi</span>
                        <span tagx="given-names">PS</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Detection of dengue viruses using reverse transcription-loop-mediated isothermal amplification</span>. 
                     <span tagx="source">BMC infectious diseases</span>. 
                     <span tagx="year">2013</span>;
                     <span tagx="volume">13</span>:
                     <span tagx="fpage">387</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1186/1471-2334-13-387</div>
                     </div> ; PubMed Central PMCID: PMC3846474.
                     <span class="pub-id'">[23964963]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref024"></a>
                  <span class="label">24</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Notomi</span>
                        <span tagx="given-names">T</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Okayama</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Masubuchi</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Yonekawa</span>
                        <span tagx="given-names">T</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Watanabe</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Amino</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Loop-mediated isothermal amplification of DNA</span>. 
                     <span tagx="source">Nucleic acids research</span>. 
                     <span tagx="year">2000</span>;
                     <span tagx="volume">28</span>(
                     <div tagxxx="issue">12</div>):
                     <span tagx="fpage">E63</span> ; PubMed Central PMCID: PMC102748.
                     <span class="pub-id'">[10871386]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref025"></a>
                  <span class="label">25</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Piepenburg</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Williams</span>
                        <span tagx="given-names">CH</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Stemple</span>
                        <span tagx="given-names">DL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Armes</span>
                        <span tagx="given-names">NA</span>,
                     </span>. 
                     <span tagx="article-title">DNA detection using recombination proteins</span>. 
                     <span tagx="source">PLoS Biol</span>. 
                     <span tagx="year">2006</span>;
                     <span tagx="volume">4</span>(
                     <div tagxxx="issue">7</div>):
                     <span tagx="fpage">e204</span> Epub 2006/06/08. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1371/journal.pbio.0040204</div>
                     </div> ; PubMed Central PMCID: PMC1475771.
                     <span class="pub-id'">[16756388]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref026"></a>
                  <span class="label">26</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Abd El Wahed</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">El-Deeb</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">El-Tholoth</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Abd El Kader</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ahmed</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hassan</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">A portable reverse transcription recombinase polymerase amplification assay for rapid
                        detection of foot-and-mouth disease virus
                     </span>. 
                     <span tagx="source">PloS one</span>. 
                     <span tagx="year">2013</span>;
                     <span tagx="volume">8</span>(
                     <div tagxxx="issue">8</div>):
                     <span tagx="fpage">e71642</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1371/journal.pone.0071642</div>
                     </div> ; PubMed Central PMCID: PMC3748043.
                     <span class="pub-id'">[23977101]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref027"></a>
                  <span class="label">27</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Weidmann</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Meyer-Konig</span>
                        <span tagx="given-names">U</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hufert</span>
                        <span tagx="given-names">FT</span>,
                     </span>. 
                     <span tagx="article-title">Rapid detection of herpes simplex virus and varicella-zoster virus infections by real-time
                        PCR
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>. 
                     <span tagx="year">2003</span>;
                     <span tagx="volume">41</span>(
                     <div tagxxx="issue">4</div>):
                     <span tagx="fpage">1565</span>–
                     <span tagx="lpage">8</span>. Epub 2003/04/12. .
                     <span class="pub-id'">[12682146]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref028"></a>
                  <span class="label">28</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Wagner</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">de With</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Huzly</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hufert</span>
                        <span tagx="given-names">F</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Weidmann</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Breisinger</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Nosocomial acquisition of dengue</span>. 
                     <span tagx="source">Emerging infectious diseases</span>. 
                     <span tagx="year">2004</span>;
                     <span tagx="volume">10</span>(
                     <div tagxxx="issue">10</div>):
                     <span tagx="fpage">1872</span>–
                     <span tagx="lpage">3</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.3201/eid1010.031037</div>
                     </div> ; PubMed Central PMCID: PMC3323263.
                     <span class="pub-id'">[15504282]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref029"></a>
                  <span class="label">29</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Euler</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Wang</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Heidenreich</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Patel</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Strohmeier</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hakenberg</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Development of a panel of recombinase polymerase amplification assays for detection
                        of biothreat agents
                     </span>. 
                     <span tagx="source">J Clin Microbiol</span>. 
                     <span tagx="year">2013</span>;
                     <span tagx="volume">51</span>(
                     <div tagxxx="issue">4</div>):
                     <span tagx="fpage">1110</span>–
                     <span tagx="lpage">7</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1128/JCM.02704-12</div>
                     </div> ; PubMed Central PMCID: PMC3666764.
                     <span class="pub-id'">[23345286]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref030"></a>
                  <span class="label">30</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Euler</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Wang</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Nentwich</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Piepenburg</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hufert</span>
                        <span tagx="given-names">FT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Weidmann</span>
                        <span tagx="given-names">M</span>,
                     </span>. 
                     <span tagx="article-title">Recombinase polymerase amplification assay for rapid detection of Rift Valley fever
                        virus
                     </span>. 
                     <span tagx="source">J Clin Virol</span>. 
                     <span tagx="year">2012</span> Epub 2012/06/12. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/j.jcv.2012.05.006</div>
                     </div> .
                     <span class="pub-id'">[22683006]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref031"></a>
                  <span class="label">31</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Abd El Wahed</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Patel</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Heidenreich</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hufert</span>
                        <span tagx="given-names">FT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Weidmann</span>
                        <span tagx="given-names">M</span>,
                     </span>. 
                     <span tagx="article-title">Reverse transcription recombinase polymerase amplification assay for the detection
                        of middle East respiratory syndrome coronavirus
                     </span>. 
                     <span tagx="source">PLoS currents</span>. 
                     <span tagx="year">2013</span>;
                     <span tagx="fpage">5</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1371/currents.outbreaks.62df1c7c75ffc96cd59034531e2e8364</div>
                     </div> ; PubMed Central PMCID: PMC3871419.
                     <span class="pub-id'">[24459611]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref032"></a>
                  <span class="label">32</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Amer</span>
                        <span tagx="given-names">HM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Abd El Wahed</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Shalaby</span>
                        <span tagx="given-names">MA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Almajhdi</span>
                        <span tagx="given-names">FN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Hufert</span>
                        <span tagx="given-names">FT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Weidmann</span>
                        <span tagx="given-names">M</span>,
                     </span>. 
                     <span tagx="article-title">A new approach for diagnosis of bovine coronavirus using a reverse transcription recombinase
                        polymerase amplification assay
                     </span>. 
                     <span tagx="source">J Virol Methods</span>. 
                     <span tagx="year">2013</span>;
                     <span tagx="volume">193</span>(
                     <div tagxxx="issue">2</div>):
                     <span tagx="fpage">337</span>–
                     <span tagx="lpage">40</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/j.jviromet.2013.06.027</div>
                     </div> .
                     <span class="pub-id'">[23811231]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref033"></a>
                  <span class="label">33</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Boyle</span>
                        <span tagx="given-names">DS</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lehman</span>
                        <span tagx="given-names">DA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Lillis</span>
                        <span tagx="given-names">L</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Peterson</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Singhal</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Armes</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Rapid Detection of HIV-1 Proviral DNA for Early Infant Diagnosis Using Recombinase
                        Polymerase Amplification
                     </span>. 
                     <span tagx="source">mBio</span>. 
                     <span tagx="year">2013</span>;
                     <span tagx="volume">4</span>(
                     <div tagxxx="issue">2</div>). Epub 2013/04/04. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1128/mBio.00135-13</div>
                     </div> .
                     <span class="pub-id'">[23549916]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref034"></a>
                  <span class="label">34</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Escadafal</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Faye</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sall</span>
                        <span tagx="given-names">AA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Faye</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Weidmann</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Strohmeier</span>
                        <span tagx="given-names">O</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Rapid molecular assays for the detection of yellow fever virus in low-resource settings</span>. 
                     <span tagx="source">PLoS neglected tropical diseases</span>. 
                     <span tagx="year">2014</span>;
                     <span tagx="volume">8</span>(
                     <div tagxxx="issue">3</div>):
                     <span tagx="fpage">e2730</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1371/journal.pntd.0002730</div>
                     </div> ; PubMed Central PMCID: PMC3945292.
                     <span class="pub-id'">[24603874]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref035"></a>
                  <span class="label">35</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Euler</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Wang</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Otto</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tomaso</span>
                        <span tagx="given-names">H</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Escudero</span>
                        <span tagx="given-names">R</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Anda</span>
                        <span tagx="given-names">P</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Recombinase Polymerase Amplification Assay for Rapid Detection of Francisella tularensis</span>. 
                     <span tagx="source">J Clin Microbiol</span>. 
                     <span tagx="year">2012</span>;
                     <span tagx="volume">50</span>(
                     <div tagxxx="issue">7</div>):
                     <span tagx="fpage">2234</span>–
                     <span tagx="lpage">8</span>. Epub 2012/04/21. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1128/JCM.06504-11</div>
                     </div> .
                     <span class="pub-id'">[22518861]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref036"></a>
                  <span class="label">36</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Compton</span>
                        <span tagx="given-names">J</span>,
                     </span>. 
                     <span tagx="article-title">Nucleic acid sequence-based amplification</span>. 
                     <span tagx="source">Nature</span>. 
                     <span tagx="year">1991</span>;
                     <span tagx="volume">350</span>(
                     <div tagxxx="issue">6313</div>):
                     <span tagx="fpage">91</span>–
                     <span tagx="lpage">2</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1038/350091a0</div>
                     </div> .
                     <span class="pub-id'">[1706072]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref037"></a>
                  <span class="label">37</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Van Ness</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Van Ness</span>
                        <span tagx="given-names">LK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Galas</span>
                        <span tagx="given-names">DJ</span>,
                     </span>. 
                     <span tagx="article-title">Isothermal reactions for the amplification of oligonucleotides</span>. 
                     <span tagx="source">Proceedings of the National Academy of Sciences of the United States of America</span>. 
                     <span tagx="year">2003</span>;
                     <span tagx="volume">100</span>(
                     <div tagxxx="issue">8</div>):
                     <span tagx="fpage">4504</span>–
                     <span tagx="lpage">9</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1073/pnas.0730811100</div>
                     </div> ; PubMed Central PMCID: PMC404692.
                     <span class="pub-id'">[12679520]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref038"></a>
                  <span class="label">38</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Gusev</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sparkowski</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Raghunathan</span>
                        <span tagx="given-names">A</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ferguson</span>
                        <span tagx="given-names">H</span>
                        <div tagxxx="suffix">Jr.</div>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Montano</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Bogdan</span>
                        <span tagx="given-names">N</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Rolling circle amplification: a new approach to increase sensitivity for immunohistochemistry
                        and flow cytometry
                     </span>. 
                     <span tagx="source">The American journal of pathology</span>. 
                     <span tagx="year">2001</span>;
                     <span tagx="volume">159</span>(
                     <div tagxxx="issue">1</div>):
                     <span tagx="fpage">63</span>–
                     <span tagx="lpage">9</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/S0002-9440(10)61674-4</div>
                     </div> ; PubMed Central PMCID: PMC1850404.
                     <span class="pub-id'">[11438455]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref039"></a>
                  <span class="label">39</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Vincent</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Xu</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Kong</span>
                        <span tagx="given-names">H</span>,
                     </span>. 
                     <span tagx="article-title">Helicase-dependent isothermal DNA amplification</span>. 
                     <span tagx="source">EMBO reports</span>. 
                     <span tagx="year">2004</span>;
                     <span tagx="volume">5</span>(
                     <div tagxxx="issue">8</div>):
                     <span tagx="fpage">795</span>–
                     <span tagx="lpage">800</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1038/sj.embor.7400200</div>
                     </div> ; PubMed Central PMCID: PMC1249482.
                     <span class="pub-id'">[15247927]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref040"></a>
                  <span class="label">40</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Torres</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vitalis</span>
                        <span tagx="given-names">EA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Baker</span>
                        <span tagx="given-names">BR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gardner</span>
                        <span tagx="given-names">SN</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Torres</span>
                        <span tagx="given-names">MW</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Dzenitis</span>
                        <span tagx="given-names">JM</span>,
                     </span>. 
                     <span tagx="article-title">LAVA: an open-source approach to designing LAMP (loop-mediated isothermal amplification)
                        DNA signatures
                     </span>. 
                     <span tagx="source">BMC bioinformatics</span>. 
                     <span tagx="year">2011</span>;
                     <span tagx="volume">12</span>:
                     <span tagx="fpage">240</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1186/1471-2105-12-240</div>
                     </div> ; PubMed Central PMCID: PMC3213686.
                     <span class="pub-id'">[21679460]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref041"></a>
                  <span class="label">41</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">SantaLucia</span>
                        <span tagx="given-names">J</span>
                        <div tagxxx="suffix">Jr</div>,
                     </span>. 
                     <span tagx="article-title">A unified view of polymer, dumbbell, and oligonucleotide DNA nearest-neighbor thermodynamics</span>. 
                     <span tagx="source">Proceedings of the National Academy of Sciences of the United States of America</span>. 
                     <span tagx="year">1998</span>;
                     <span tagx="volume">95</span>(
                     <div tagxxx="issue">4</div>):
                     <span tagx="fpage">1460</span>–
                     <span tagx="lpage">5</span>. ; PubMed Central PMCID: PMC19045.
                     <span class="pub-id'">[9465037]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref042"></a>
                  <span class="label">42</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Daher</span>
                        <span tagx="given-names">RK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Stewart</span>
                        <span tagx="given-names">G</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Boissinot</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Boudreau</span>
                        <span tagx="given-names">DK</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Bergeron</span>
                        <span tagx="given-names">MG</span>,
                     </span>. 
                     <span tagx="article-title">Influence of sequence mismatches on the specificity of Recombinase Polymerase Amplification
                        technology
                     </span>. 
                     <span tagx="source">Mol Cell Probes</span>. 
                     <span tagx="year">2014</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/j.mcp.2014.11.005</div>
                     </div> .
                     <span class="pub-id'">[25481659]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref043"></a>
                  <span class="label">43</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Whiley</span>
                        <span tagx="given-names">DM</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sloots</span>
                        <span tagx="given-names">TP</span>,
                     </span>. 
                     <span tagx="article-title">Sequence variation can affect the performance of minor groove binder TaqMan probes
                        in viral diagnostic assays
                     </span>. 
                     <span tagx="source">J Clin Virol</span>. 
                     <span tagx="year">2006</span>;
                     <span tagx="volume">35</span>(
                     <div tagxxx="issue">1</div>):
                     <span tagx="fpage">81</span>–
                     <span tagx="lpage">3</span>. Epub 2005/07/09. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1016/j.jcv.2005.05.002</div>
                     </div> .
                     <span class="pub-id'">[16002331]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref044"></a>
                  <span class="label">44</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Carneiro</span>
                        <span tagx="given-names">AR</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Cruz</span>
                        <span tagx="given-names">AC</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vallinoto</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Melo Dde</span>
                        <span tagx="given-names">V</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Ramos</span>
                        <span tagx="given-names">RT</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Medeiros</span>
                        <span tagx="given-names">DB</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Molecular characterisation of dengue virus type 1 reveals lineage replacement during
                        circulation in Brazilian territory
                     </span>. 
                     <span tagx="source">Memorias do Instituto Oswaldo Cruz</span>. 
                     <span tagx="year">2012</span>;
                     <span tagx="volume">107</span>(
                     <div tagxxx="issue">6</div>):
                     <span tagx="fpage">805</span>–
                     <span tagx="lpage">12</span>. .
                     <span class="pub-id'">[22990972]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref045"></a>
                  <span class="label">45</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">EC</span>,
                     </span>. 
                     <span tagx="article-title">Molecular epidemiology and evolution of emerging infectious diseases</span>. 
                     <span tagx="source">British medical bulletin</span>. 
                     <span tagx="year">1998</span>;
                     <span tagx="volume">54</span>(
                     <div tagxxx="issue">3</div>):
                     <span tagx="fpage">533</span>–
                     <span tagx="lpage">43</span>. .
                     <span class="pub-id'">[10326282]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref046"></a>
                  <span class="label">46</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Holmes</span>
                        <span tagx="given-names">KC</span>,
                     </span>. 
                     <span tagx="article-title">A molecular model for muscle contraction</span>. 
                     <span tagx="source">Acta crystallographica Section A, Foundations of crystallography</span>. 
                     <span tagx="year">1998</span>;
                     <span tagx="volume">54</span>(
                     <div tagxxx="issue">Pt 6 Pt 1</div>):
                     <span tagx="fpage">789</span>–
                     <span tagx="lpage">97</span>. .
                     <span class="pub-id'">[9859194]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref047"></a>
                  <span class="label">47</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Messer</span>
                        <span tagx="given-names">WB</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Gubler</span>
                        <span tagx="given-names">DJ</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Harris</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sivananthan</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">de Silva</span>
                        <span tagx="given-names">AM</span>,
                     </span>. 
                     <span tagx="article-title">Emergence and global spread of a dengue serotype 3, subtype III virus</span>. 
                     <span tagx="source">Emerging infectious diseases</span>. 
                     <span tagx="year">2003</span>;
                     <span tagx="volume">9</span>(
                     <div tagxxx="issue">7</div>):
                     <span tagx="fpage">800</span>–
                     <span tagx="lpage">9</span>. ; PubMed Central PMCID: PMC3023445.
                     <span class="pub-id'">[12899133]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref048"></a>
                  <span class="label">48</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Botling</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Edlund</span>
                        <span tagx="given-names">K</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Segersten</span>
                        <span tagx="given-names">U</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Tahmasebpoor</span>
                        <span tagx="given-names">S</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Engstrom</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Sundstrom</span>
                        <span tagx="given-names">M</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Impact of thawing on RNA integrity and gene expression analysis in fresh frozen tissue</span>. 
                     <span tagx="source">Diagnostic molecular pathology: the American journal of surgical pathology, part B</span>. 
                     <span tagx="year">2009</span>;
                     <span tagx="volume">18</span>(
                     <div tagxxx="issue">1</div>):
                     <span tagx="fpage">44</span>–
                     <span tagx="lpage">52</span>. 
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1097/PDM.0b013e3181857e92</div>
                     </div> .
                     <span class="pub-id'">[19214109]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref049"></a>
                  <span class="label">49</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Mutter</span>
                        <span tagx="given-names">GL</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Zahrieh</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Liu</span>
                        <span tagx="given-names">C</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Neuberg</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Finkelstein</span>
                        <span tagx="given-names">D</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Baker</span>
                        <span tagx="given-names">HE</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Comparison of frozen and RNALater solid tissue storage methods for use in RNA expression
                        microarrays
                     </span>. 
                     <span tagx="source">BMC genomics</span>. 
                     <span tagx="year">2004</span>;
                     <span tagx="volume">5</span>:
                     <span tagx="fpage">88</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1186/1471-2164-5-88</div>
                     </div> ; PubMed Central PMCID: PMC534099.
                     <span class="pub-id'">[15537428]</span>
                  </div>
               </li>
               <li tag="ref">
                  <a name="pone.0129682.ref050"></a>
                  <span class="label">50</span>
                  <div tagxxx="mixed-citation">
                     <span class="name'">
                        <span tagx="surname">Santiago</span>
                        <span tagx="given-names">GA</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vergne</span>
                        <span tagx="given-names">E</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Quiles</span>
                        <span tagx="given-names">Y</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Cosme</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Vazquez</span>
                        <span tagx="given-names">J</span>,
                     </span>, 
                     <span class="name'">
                        <span tagx="surname">Medina</span>
                        <span tagx="given-names">JF</span>,
                     </span>, 
                     <i>et al.</i>
                     <span tagx="article-title">Analytical and clinical performance of the CDC real time RT-PCR assay for detection
                        and typing of dengue virus
                     </span>. 
                     <span tagx="source">PLoS neglected tropical diseases</span>. 
                     <span tagx="year">2013</span>;
                     <span tagx="volume">7</span>(
                     <div tagxxx="issue">7</div>):
                     <span tagx="fpage">e2311</span>
                     <div tagxxx="comment">doi: 
                        <div tagxxx="ext-link">10.1371/journal.pntd.0002311</div>
                     </div> ; PubMed Central PMCID: PMC3708876.
                     <span class="pub-id'">[23875046]</span>
                  </div>
               </li>
            </ul>
         </div>
      </div>
   </div>
</html>